<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<title>Sentences that contain nextstrain</title>
</head>
<body>
<h1>Sentences that contain nextstrain</h1>
<p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37178807">37178807</a> The emergence, impact, and evolution of human metapneumovirus variants from 2014 to 2021 in Spain.<ul><li>WGS was performed with Illumina, and evolutionary analyses with Datamonkey and <b>Nextstrain</b>.</ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37144832">37144832</a> Validation of a new strategy for the identification of SARS-CoV-2 variants by sequencing the spike gene by Sanger.<ul><li>The data obtained were analyzed on the <b>Nextstrain</b> and PANGO Lineages platforms.</ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37144832">37144832</a> Validation of a new strategy for the identification of SARS-CoV-2 variants by sequencing the spike gene by Sanger.<ul><li>Los datos obtenidos fueron analizados en las plataformas <b>Nextstrain</b> y PANGO Lineages.</ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35602577">35602577</a> [Validation of a new strategy for the identification of SARS-CoV-2 variants by sequencing the spike gene by Sanger].<ul><li>The data obtained were analyzed on the <b>Nextstrain</b> and PANGO Lineages platforms.</ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35602577">35602577</a> [Validation of a new strategy for the identification of SARS-CoV-2 variants by sequencing the spike gene by Sanger].<ul><li>Los datos obtenidos fueron analizados en las plataformas <b>Nextstrain</b> y PANGO Lineages.</ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37172461">37172461</a> A European perspective of phylogenomics, sublineages, geographical distribution, epidemiology, and mutational landscape of mpox virus: Emergence pattern may help to fight the next public health emergency in Europe.<ul><li>For analysis, we use various advanced servers like <b>Nextstrain</b>, Taxonium, MpoxSpectrum, etc. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37077184">37077184</a> Temporal dynamics and fatality of SARS-CoV-2 variants in Bangladesh.<ul><li>The phylogenetic analysis revealed multiple introductions of SARS-CoV-2 variant(s) into Bangladesh with at least 22 <b>Nextstrain</b> clades and 107 Pangolin lineages with respect to the SARS-CoV-2 reference genome of Wuhan/Hu-1/2019. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36702307">36702307</a> Towards real-time monitoring of COVID-19 nosocomial clusters using SARS-CoV-2 genomes in a university hospital of the French Alps.<ul><li>Experience of <b>Nextstrain</b> [1,2] and its approach adapted to the local context encouraged us to carry out real-time monitoring of COVID-19 nosocomial clusters in our establishment, the Grenoble Alpes University Hospital.</ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112836">37112836</a> Understanding Mutations in Human SARS-CoV-2 Spike Glycoprotein: A Systematic Review &amp; Meta-Analysis.<ul><li>We performed comprehensive analyses of spike protein mutations in the significant variant clade of SARS-CoV-2, using the data available on the <b>Nextstrain</b> server. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37035035">37035035</a> Joint visualization of seasonal influenza serology and phylogeny to inform vaccine composition.<ul><li>To simplify this decision-making process, we have created an interactive tool for visualizing serological data that has been integrated into <b>Nextstrain</b>'s real-time phylogenetic visualization framework, Auspice. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36845975">36845975</a> Off-season circulation and characterization of enterovirus D68 with respiratory and neurological presentation using whole-genome sequencing.<ul><li>Applying phylodynamic and evolutionary analysis on <b>Nextstrain</b> and Datamonkey respectively, we report a highly diverse virus with an evolutionary rate of 3.05&#x2009;&#xD7;&#x2009;10<sup>-3</sup> substitutions per year (entire EV-D68 genome) and a positive episodic/diversifying selection with persistent yet undetected circulation likely driving evolution. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36708096">36708096</a> Molecular evolution of the human monkeypox virus.<ul><li>Currently, the classification of hMPXV according to the <b>nextstrain</b> occurs in five big clades (1A, A.1, A.2, A.1.1, and B.1).&#xA0;According to the resurgence of smallpox-like disease caused by hMPXV and the spread of the virus to the European and American continents, in the present study, we review and summarize the molecular evolution of the hMPXV, determining the molecular evolution of the main clades.&#xA0;A total of 442 hMPXV whole-genome sequences&#xA0;with available information from the country and sampling date (between October 2017 and 2022), were obtained and evaluated using the Bayesian method.&#xA0;The clade B.1 which is currently circulating was the most frequent (n&#x2009;=&#x2009;415; 93.9%). </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36638615">36638615</a> L-shaped distribution of the relative substitution rate (c/&#x3BC;) observed for SARS-COV-2's genome, inconsistent with the selectionist theory, the neutral theory and the nearly neutral theory but a near-neutral balanced selection theory: Implication on "neutralist-selectionist" debate.<ul><li>We named this hybrid mechanism as near-neutral balanced selection theory (NNBST) and examined if it was followed by SARS-CoV-2 using the three independent sets of SARS-CoV-2 genomes selected by the <b>Nextstrain</b> team. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36686884">36686884</a> Characterization and phylogenetic analysis of Iranian SARS-CoV-2 genomes: A phylogenomic study.<ul><li>The phylogenetic analysis of genomic sequences showed the presence of eight GISAID clades, namely GH, GR, O, GRY, G, GK, L, and GV, and six <b>Nextstrain</b> clades; that is, 19A, 20A, 20B, 20I (alpha, V1), 20H (Beta, V2), and 21I (Delta) in Iran. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36579822">36579822</a> COVID-19 on the Nile: a cross-sectional investigation of COVID-19 among Nile River cruise travellers returning to the United States, February-march 2020.<ul><li>Phylogenetic analyses supported the hypothesis that travelers were most likely infected in Egypt, with most sequences in <b>Nextstrain</b> clade 20A 93% (87/94). </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36611300">36611300</a> Complete Genomic Characterisation and Mutation Patterns of Iraqi SARS-CoV-2 Isolates.<ul><li>Three established global platforms GISAID, <b>Nextstrain</b>, and PANGO were used for the classification of isolates into distinct clades, variants, and lineages. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36555300">36555300</a> Quasispecies Analysis of SARS-CoV-2 of 15 Different Lineages during the First Year of the Pandemic Prompts Scratching under the Surface of Consensus Genome Sequences.<ul><li>We compared intra-sample genetic diversity and global genetic diversity available from <b>Nextstrain</b>. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36780250">36780250</a> Transmission cluster characteristics of global, regional, and lineage-specific SARS-CoV-2 phylogenies.<ul><li>We used the <b>Nextstrain</b>'s nCov open global all-time phylogeny (September 2022, 2,698 strains, 2,243 for North America, 499 for Delta21A, and 543 for Omicron20M), with <b>Nextstrain</b>'s clade annotation and Pango lineages. <li>Phylopart and DYNAMITE clusters moderately-to-highly agree with the Pango nomenclature and the <b>Nextstrain</b>'s clade. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36547242">36547242</a> Genomic Evidence Suggests Viral Persistence of SARS-CoV-2 for 386 Days in Health Worker: A Case Report from Santiago of Chile.<ul><li>RNA sequencing analyses indicated that the first positive diagnosis (8 June 2020) corresponded to a SARS-CoV-2 variant belonging to Clade <b>Nextstrain</b> 20A. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36116149">36116149</a> Bioinformatics pipeline unveils genetic variability to synthetic vaccine design for Indian SARS-CoV-2 genomes.<ul><li>We have thus performed multiple sequence alignment of 4996 Indian SARS-CoV-2 genomes as a case study using MAFFT followed by phylogenetic analysis using <b>Nextstrain</b> to identify virus clades. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36144387">36144387</a> Towards Efficient and Accurate SARS-CoV-2 Genome Sequence Typing Based on Supervised Learning Approaches.<ul><li>For <b>Nextstrain</b> and GISAID clades, a template matching-based method is proposed to quantify the differences between viral clades and to play an important role in classification evaluation. <li>Based on an accumulated database of &gt;1 million SARS-CoV-2 genome sequences, performance evaluations show that the proposed system has a typing accuracy of 99.879%, 97.732%, and 96.291% for <b>Nextstrain</b>, Pangolin, and GISAID clades, respectively. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36118194">36118194</a> Spatiotemporal dynamics of HIV-1 CRF63_02A6 sub-epidemic.<ul><li>Ancestral state reconstruction and origin date were estimated using <b>Nextstrain</b>. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36008511">36008511</a> SARS-CoV-2 genome variations and evolution patterns in Egypt: a multi-center study.<ul><li>The virus evolution spread has been tracked by three developing databases: GISAID, <b>Nextstrain</b> and PANGO to understand its circulating variants. <li>According to <b>Nextstrain</b> clades, most of our sequenced samples belonged to clades 20A and 20D, which in addition to clade 20B were present from the beginning of sample collection in May 2020. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016297">36016297</a> Evaluation of EPISEQ SARS-CoV-2 and a Fully Integrated Application to Identify SARS-CoV-2 Variants from Several Next-Generation Sequencing Approaches.<ul><li>This application performs in one click a quality check, a reference-based genome assembly, and the analysis of the generated consensus sequence as to coverage of the reference genome, mutation screening and variant identification according to the up-to-date <b>Nextstrain</b> clade and Pango lineage. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35638826">35638826</a> Genome Sequencing of Omicron Variants of SARS-CoV-2 Circulating in Bangladesh during the Third Wave of the COVID-19 Pandemic.<ul><li>The presence of different sublineages of Omicron was noted, among which sublineage BA.2 (<b>Nextstrain</b> clade 21L) was the most prevalent one.</ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35445774">35445774</a> Omicron - The new SARS-CoV-2 challenge?<ul><li>Omicron is made up of several sublineages, where the most common are BA.1 and BA.2 (or <b>Nextstrain</b> clade 21K and 21L, respectively). </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34547443">34547443</a> Characterisation of SARS-CoV-2 clades based on signature SNPs unveils continuous evolution.<ul><li>Subsequently, phylogenetic analyses are performed using <b>Nextstrain</b> to identify virus clades. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35766502">35766502</a> Genetic and Antigenic Characterization of an Expanding H3 Influenza A Virus Clade in U.S. Swine Visualized by <b>Nextstrain</b>.<ul><li>We created an implementation of <b>Nextstrain</b> to visualize the emergence, spatial spread, and genetic evolution of H3 IAV in swine, identifying two C-IVA clades that emerged in 2017 and cocirculated in multiple U.S. <li>Our study identified genetic and antigenic factors contributing to its resurgence by linking comprehensive phylodynamic analyses with empirical wet-lab experiments and visualized these evolutionary analyses in a <b>Nextstrain</b> implementation. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35759503">35759503</a> Genome sequence diversity of SARS-CoV-2 obtained from clinical samples in Uzbekistan.<ul><li>The "<b>Nextstrain</b>" clustered the largest number of SARS-CoV-2 strains into the Delta clade (n = 32; 82%), followed by two Alpha-originated (n = 4; 10,3%) and 20A (n = 3; 7,7%) clades. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35822110">35822110</a> Evolutionary traits of Tick-borne encephalitis virus: Pervasive non-coding RNA structure conservation and molecular epidemiology.<ul><li>Moreover, we provide an updated phylogeny of TBEV, building on more than 220 publicly available complete genomes, and investigate the molecular epidemiology and phylodynamics with <b>Nextstrain</b>, a web-based visualization framework for real-time pathogen evolution.</ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35050521">35050521</a> Transmission dynamics and mutational prevalence of the novel Severe acute respiratory syndrome coronavirus-2 Omicron Variant of Concern.<ul><li>The transmission dynamics and Omicron SARS-CoV-2 divergence was studied using GISAID and <b>Nextstrain</b> which provides information about the genetic sequences, epidemiological, geographical, and species-specific data of human, avian, and animal viruses. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34894077">34894077</a> Novel and extendable genotyping system for human respiratory syncytial virus based on whole-genome sequence analysis.<ul><li>We also provide an RSV classification tool to perform RSV genotype assignment and a publicly accessible up-to-date instance of <b>Nextstrain</b> where the phylogenetic relationship of all genotypes can be explored.</ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35528635">35528635</a> First Identification of Reinfection by a Genetically Different Variant of SARS-CoV-2 in a Homeless Person from the Metropolitan Area of Santiago, Chile.<ul><li>Genomic sequencing identified a different <b>Nextstrain</b> classification clade (20A and 20B) and PANGO lineage, with a divergence of 4 single nucleotide variants (SNVs) in S and ORF1ab genes, suggesting reinfection by different viral variants. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35231666">35231666</a> Covidex: An ultrafast and accurate tool for SARS-CoV-2 subtyping.<ul><li>It is a shiny web app that allows an ultra-fast and accurate classification of SARS-CoV-2 genome sequences into the three most used nomenclature systems (GISAID, <b>Nextstrain</b>, Pango lineages). </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35344550">35344550</a> Phylogenetic analysis of 17271 Indian SARS-CoV-2 genomes to identify temporal and spatial hotspot mutations.<ul><li>In this regard, multiple sequence alignment of 17271 Indian SARS-CoV-2 sequences is performed using MAFFT followed by their phylogenetic analysis using <b>Nextstrain</b>. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35458383">35458383</a> SARS-CoV-2: Evolution and Emergence of New Viral Variants.<ul><li>This review aimed to highlight the functional virus classification used by the World Health Organization (WHO), Phylogenetic Assignment of Named Global Outbreak (PANGO), Global Initiative on Sharing All Influenza Data (GISAID), and <b>Nextstrain</b>, an open-source project to harness the scientific and public health potential of pathogen genome data, the chronological emergence of viral variants of concern (VOCs) and variants of interest (VOIs), the major findings related to the rate of spread, and the mutations in the spike protein that are involved in the evasion of the host immune responses elicited by prior SARS-CoV-2 infections and by the protection induced by vaccination.</ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35084216">35084216</a> Whole-Genome Sequencing of SARS-CoV-2 Strains from Asymptomatic Individuals in India.<ul><li>Nine strains were found to be <b>Nextstrain</b> clade 20B (PANGO lineage B.1.1.519, GISAID clade GR), and two were identified as clade 20A (PANGO lineage B.1.619, GISAID clade G). </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34700068">34700068</a> Dynamics of SARS-CoV-2 variants of concern (VOC) in Bangladesh during the first half of 2021.<ul><li>Within the study timeframe, at least eight <b>Nextstrain</b> clades were found: 20A, 20B, 20C, 20H (Beta, V2), 20I (Alpha, V1), 20&#xA0;J (Gamma, V3), 21A (Delta), 21D (Eta), and six GISAID clades: four main (G, GH, GR, GRY) and two minors (GV, O) with an introduction of VOC B.1.1.7/Alpha, B.1.351/Beta and B.1.617.2/Delta. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34787497">34787497</a> SARS-CoV-2 Variants and Their Relevant Mutational Profiles: Update Summer 2021.<ul><li>We focused on variants and on specific relevant mutations reported by GISAID, <b>Nextstrain</b>, Outbreak.info, Pango, and Stanford database websites that were associated with any clinical/diagnostic impact, according to published manuscripts. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34946210">34946210</a> Genomic Epidemiology of SARS-CoV-2 Divulge B.1, B.1.36, and B.1.1.7 as the Most Dominant Lineages in First, Second, and Third Wave of SARS-CoV-2 Infections in Pakistan.<ul><li>And to infer our SARS-CoV-2 time-scaled and divergence phylogenetic trees, 3804 selected global reference sequences plus 400 Pakistani samples were used for the <b>Nextstrain</b> analysis with Wuhan/Hu-1/2019, as reference genome. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34469548">34469548</a> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees.<ul><li>Our database uses the recently proposed mutation-annotated tree (MAT) format to efficiently encode the tree with branches labeled with parsimony-inferred mutations, as well as <b>Nextstrain</b> clade and Pango lineage labels at clade roots. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34508827">34508827</a> Dynamics prediction of emerging notable spike protein mutations in SARS-CoV-2 implies a need for updated vaccines.<ul><li>This questions the current vaccines efficacies that were based on the <b>Nextstrain</b> clade 19A that first documented in Wuhan and lacks any variants. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34125658">34125658</a> Rapidly emerging SARS-CoV-2 B.1.1.7 sub-lineage in the United States of America with spike protein D178H and membrane protein V70L mutations.<ul><li>Phylogenetic analysis and transmission inference with <b>Nextstrain</b> suggest this sub-lineage likely originated in either California or Washington. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34898481">34898481</a> Emergence of Novel SARS-CoV-2 variants in India: second wave.<ul><li>Phylogenetic analysis were carried out using <b>Nextstrain</b>.</ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34790342">34790342</a> On the association between SARS-COV-2 variants and COVID-19 mortality during the second wave of the pandemic in Europe.<ul><li>For 38 European countries, data on numbers of COVID-19 deaths, SARS-COV-2 variants spread through time using <b>Nextstrain</b> classification, demographic and health characteristics were collected. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34698524">34698524</a> Scalable Reconstruction of SARS-CoV-2 Phylogeny with Recurrent Mutations.<ul><li>We show that SPHERE phylogeny is more stable than <b>Nextstrain</b>'s, and that it accurately infers known transmission links from the early pandemic. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34571275">34571275</a> Clinical outcomes in patients infected with different SARS-CoV-2 variants at one hospital during three phases of the COVID-19 epidemic in Marseille, France.<ul><li>A single centre retrospective cohort study was conducted in 1760 patients infected with SARS-CoV-2 of <b>Nextstrain</b> clades 20A, 20B, and 20C (first phase, February-May 2020), Pangolin lineages B.1.177 (we named Marseille-2) and B.1.160 (Marseille-4) variants (second phase, June-December 2020), and B.1.1.7 (alpha), B.1.351 (beta), P.1 (gamma) and A.27 (Marseille-501) variants (third phase, January 2021-today). </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34702899">34702899</a> Detailed phylogenetic analysis tracks transmission of distinct SARS-COV-2 variants from China and Europe to West Africa.<ul><li>We subjected the sequences to phylogenetic analysis employing the <b>nextstrain</b> pipeline. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34477544">34477544</a> Whole-genome-based phylogenomic analysis of the Belgian 2016-2017 influenza A(H3N2) outbreak season allows improved surveillance.<ul><li>A new approach, considering the whole genome, was investigated based on using powerful phylogenomic tools including beast and <b>Nextstrain</b>, which substantially improved phylogenetic classification. <li>as an outbreak is ongoing) via <b>Nextstrain</b> allowed positioning of the Belgian isolates into the globally circulating context. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34276267">34276267</a> [SARS-CoV-2 variants, a still unfinished story].<ul><li>The nomenclature to design the variants uses a colloquial form referred to the country or place of detection, a code from the "Pangolin" database and one from the "<b>Nextstrain</b>" page. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34276267">34276267</a> [SARS-CoV-2 variants, a still unfinished story].<ul><li>En la nomenclatura de las variantes se utiliza una forma &#xAB;coloquial&#xBB; que suele hacer referencia al pa&#xED;s o lugar de detecci&#xF3;n, un c&#xF3;digo de la base de datos &#xAB;Pangol&#xED;n&#xBB; y uno de la p&#xE1;gina &#xAB;<b>Nextstrain</b>&#xBB;. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33991677">33991677</a> Spreading of a new SARS-CoV-2 N501Y spike variant in a new lineage.<ul><li>These spike N501Y genomes are the first of <b>Nextstrain</b> clade 19B. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34458642">34458642</a> Molecular characterization of SARS-CoV-2 from Bangladesh: implications in genetic diversity, possible origin of the virus, and functional significance of the mutations.<ul><li>Our phylogeny analysis identified most of the isolates belonged to the <b>Nextstrain</b> clade 20B (86%) and GISAID clade GR (88%). </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34372565">34372565</a> Variant Analysis of SARS-CoV-2 Genomes from Belgian Military Personnel Engaged in Overseas Missions and Operations.<ul><li>The SARS-CoV-2 variants belonged to major clades 19B, 20A, and 20B (<b>Nextstrain</b> nomenclature), and included "variant of interest" B.1.525, "variant under monitoring" A.27, as well as lineages B.1.214, B.1, B.1.1.254, and A (pangolin nomenclature), some of which are internationally monitored due to the specific mutations they harbor. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33821270">33821270</a> A daily-updated database and tools for comprehensive SARS-CoV-2 mutation-annotated trees.<ul><li>Our database uses the recently-proposed mutation-annotated tree (MAT) format to efficiently encode the tree with branches labeled with parsimony-inferred mutations as well as <b>Nextstrain</b> clade and Pango lineage labels at clade roots. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33930542">33930542</a> Evidence of SARS-CoV-2 reinfection within the same clade in Ecuador: A case study.<ul><li>Both the infection and the reinfection strains were assigned as <b>Nextstrain</b> 20B, Pangolin lineage B.1.1 and GISAID clade O. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34204305">34204305</a> The Functional Consequences of the Novel Ribosomal Pausing Site in SARS-CoV-2 Spike Glycoprotein RNA.<ul><li>Due to population variants, the <b>Nextstrain</b> divergence measure of the CCU codon is extremely large. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34097716">34097716</a> Detection of R.1 lineage severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) with spike protein W152L/E484K/G769V mutations in Japan.<ul><li>The associated virus strain was classified into the GR clade (Global Initiative on Sharing Avian Influenza Data [GISAID]), 20B clade (<b>Nextstrain</b>), and R.1 lineage (Phylogenetic Assignment of Named Global Outbreak [PANGO] Lineages). </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34095334">34095334</a> High Prevalence of SARS-CoV-2 Genetic Variation and D614G Mutation in Pediatric Patients With COVID-19.<ul><li>Epidemiological investigation revealed major representation of 3 of 5 major <b>Nextstrain</b> clades (20A, 20B, and 20C) consistent with multiple introductions of SARS-CoV-2 in Southern California.</ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34067890">34067890</a> Structural Analysis of the Novel Variants of SARS-CoV-2 and Forecasting in North America.<ul><li>From the <b>Nextstrain</b> global database, we identified prevalent mutations of S-RBD of SARS-CoV-2 from December 2019 to April 2021. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34065789">34065789</a> Molecular Analysis of SARS-CoV-2 Circulating in Bangladesh during 2020 Revealed Lineage Diversity and Potential Mutations.<ul><li>Phylogenetic analysis revealed circulation of seven GISAID clades G, GH, GR, GRY, L, O, and S or five <b>Nextstrain</b> clades 20A, 20B, 20C, 19A, and 19B in the country during the study period. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34063166">34063166</a> Phylogenetic Analysis of the 2020 West Nile Virus (WNV) Outbreak in Andalusia (Spain).<ul><li>As part on this effort, we implemented a local <b>Nextstrain</b> server, which has become a constituent piece of regional epidemiological surveillance, wherein forthcoming genomes of environmental samples or, eventually, future outbreaks, will be included.</ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33316067">33316067</a> Phylogenetic Analysis of SARS-CoV-2 Data Is Difficult.<ul><li>Numerous studies covering some aspects of SARS-CoV-2 data analyses are being published on a daily basis, including a regularly updated phylogeny on <b>nextstrain</b>.org. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33785459">33785459</a> Emergence and outcomes of the SARS-CoV-2 'Marseille-4' variant.<ul><li>The 1038 SARS-CoV-2 whole genome sequences obtained in our laboratory by next-generation sequencing with Illumina technology were analysed using Nextclade and <b>nextstrain</b>/ncov pipelines and IQ-TREE. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34124300">34124300</a> Computational Analysis and Phylogenetic Clustering of SARS-CoV-2 Genomes.<ul><li>Toward this goal, we have created a comprehensive protocol for the analysis and phylogenetic clustering of SARS-CoV-2 genomes using <b>Nextstrain</b>, a powerful open-source tool for the real-time interactive visualization of genome sequencing data. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33817389">33817389</a> Mink SARS-CoV-2 Infection in Poland - Short Communication.<ul><li>Phylogenetic analysis of full-length nucleotide sequences of two SARS-CoV-2 isolates revealed that they belong to the 20B <b>Nextstrain</b> clade. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33567556">33567556</a> Dynamic Molecular Epidemiology Reveals Lineage-Associated Single-Nucleotide Variants That Alter RNA Structure in Chikungunya Virus.<ul><li>We provide an interactive view of CHIKV molecular epidemiology built on <b>Nextstrain</b>, a web-based visualization framework for real-time tracking of pathogen evolution. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33468702">33468702</a> SARS-CoV-2 Genomic Variation in Space and Time in Hospitalized Patients in Philadelphia.<ul><li>We found that all of our isolates contained the D614G substitution in the viral spike and belong to lineages variously designated B.1, <b>Nextstrain</b> clade 20A or 20C, and GISAID clade G or GH. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33434037">33434037</a> First report of Sri Lankan cassava mosaic virus and Cassava Mosaic Disease in Laos.<ul><li>A phylogenetic tree for SLCMV and a link to the open SLCMV <b>Nextstrain</b> map (Hadfield et al., 2018) is included in Supplementary Figure 2. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33818474">33818474</a> Phylogenetic classification of the whole-genome sequences of SARS-CoV-2 from India &amp; evolutionary trends.<ul><li>Phylogenetic and sequence analysis of a total of 3014 Indian SARS-CoV-2 sequences from 20 different States/Union Territories (January to September 2020) from the Global Initiative on Sharing All Influenza Data (GISAID) database was performed to observe the clustering of <b>Nextstrain</b> and Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin) lineages with the GISAID clades. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33818474">33818474</a> Phylogenetic classification of the whole-genome sequences of SARS-CoV-2 from India &amp; evolutionary trends.<ul><li>An integration of the GISAID, <b>Nextstrain</b> and Pangolin classifications is also provided. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32837981">32837981</a> Amino acid variation analysis of surface spike glycoprotein at 614 in SARS-CoV-2 strains.<ul><li>Here, we studied 489 SARS-CoV-2 genomes obtained from 32 countries from the <b>Nextstrain</b> database and performed phylogenetic tree analysis by clade, country, and genotype of the surface spike glycoprotein (S protein) at site 614. </ul></p><p>2020 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32615316">32615316</a> Characterizations of SARS-CoV-2 mutational profile, spike protein stability and viral transmission.<ul><li>We have identified frequently mutated residues by aligning ~660 SARS-CoV-2 genomes and validated in 10,000 datasets available in GISAID <b>Nextstrain</b>. </ul></p><p>2020 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33052335">33052335</a> idCOV: a pipeline for quick clade identification of SARS-CoV-2 isolates.<ul><li>Using a public dataset, we show that idCOV can make equivalent calls as annotated by <b>Nextstrain</b>.org on all three common clade systems using user uploaded FastQ files directly. </ul></p><p>2020 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33031371">33031371</a> Real-time tracking of Tomato brown rugose fruit virus (ToBRFV) outbreaks in the Netherlands using <b>Nextstrain</b>.<ul><li><b>Nextstrain</b> was used to visualize these potential connections. <li>This paper describes a <b>Nextstrain</b> build containing ToBRFV genomes up to and including November 2019. <li>Organizations are invited to share data or materials for inclusion in the <b>Nextstrain</b> build, which can be accessed at https://<b>nextstrain</b>.nrcnvwa.nl/ToBRFV/20191231.</ul></p><p>2020 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32941419">32941419</a> Genetic grouping of SARS-CoV-2 coronavirus sequences using informative subtype markers for pandemic spread visualization.<ul><li>In addition, we show the relationship of ISMs to phylogenetic reconstructions of SARS-CoV-2 evolution, and therefore, ISMs can play an important complementary role to phylogenetic tree-based analysis, such as is done in the <b>Nextstrain</b> project. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32817965">32817965</a> Genomic Diversity of SARS-CoV-2 During Early Introduction into the United States National Capital Region.<ul><li>These genomes belong to all five major <b>Nextstrain</b>-defined clades, suggesting multiple introductions into the region and underscoring the diversity of the regional epidemic. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32281388">32281388</a> New Genomes from the Congo Basin Expand History of CRF01_AE Origin and Dissemination.<ul><li>The new sequences and analysis presented in this study expand the molecular history of the CRF01_AE clade, and are illustrated in an interactive Next Strain phylogenetic tree, map, and timeline at (https://<b>nextstrain</b>.org/community/EduanWilkinson/hiv-1_crf01).</ul></p><p>2020 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32470049">32470049</a> Origin of HAV strains responsible for 2016-2017 outbreak among MSM: Viral phylodynamics in Lazio region.<ul><li>The HAV dataset is available on interactive phylodynamic platform https://<b>nextstrain</b>.org to real-time update of future outbreaks.</ul></p><p>2020 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31671157">31671157</a> Twenty years of West Nile virus spread and evolution in the Americas visualized by <b>Nextstrain</b>.<ul><li>To bring new attention to one of the most important mosquito-borne viruses in the Americas, we provide an interactive review using <b>Nextstrain</b>: a visualization tool for real-time tracking of pathogen evolution (<b>nextstrain</b>.org/WNV/NA). <li>Using <b>Nextstrain</b>, we use virus genomics to investigate the emergence of West Nile virus in the U S, followed by its rapid spread, evolution in a new environment, establishment of endemic transmission, and subsequent international spread. <li>We also provide a brief online narrative that parallels this review to further explain the data and highlight key epidemiological and evolutionary features (<b>nextstrain</b>.org/narratives/twenty-years-of-WNV). </ul></p><p>2020 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31037220">31037220</a> Intra- and interpatient evolution of enterovirus D68 analyzed by whole-genome deep sequencing.<ul><li>The new data along with publicly available EV-D68 sequences are included in an interactive phylodynamic analysis on <b>nextstrain</b>.org/enterovirus/d68 to facilitate timely EV-D68 tracking in the future.</ul></p><p>2019 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>Nextstrain</b>: real-time tracking of pathogen evolution.<ul><li><b>Nextstrain</b> consists of a database of viral genomes, a bioinformatics pipeline for phylodynamics analysis, and an interactive visualization platform. <li><b>Nextstrain</b> compiles our current understanding into a single accessible location, open to health professionals, epidemiologists, virologists and the public alike.</ul></p><p>2019 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>Nextstrain</b>: real-time tracking of pathogen evolution.<ul><li>All code (predominantly JavaScript and Python) is freely available from github.com/<b>nextstrain</b> and the web-application is available at <b>nextstrain</b>.org.</ul></p></body>
