<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<title>Sentences that contain nextclade</title>
</head>
<body>
<h1>Sentences that contain nextclade</h1>
<p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37117393">37117393</a> Minimal requirements for ISO15189 validation and accreditation of three next generation sequencing procedures for SARS-CoV-2 surveillance in clinical setting.<ul><li>Additionally, the typing results emerging from each of the three sequencing methods were compared using three widely known nomenclatures (WHO, Pangolineage, and <b>Nextclade</b>). </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37077184">37077184</a> Temporal dynamics and fatality of SARS-CoV-2 variants in Bangladesh.<ul><li>The clade and Pango lineages were assigned by using <b>Nextclade</b> v2.8.1. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36448028">36448028</a> Dynamics of SARS-CoV-2 variants characterized during different COVID-19 waves in Mali.<ul><li>Mutations were noted using the GISAID and <b>Nextclade</b> platforms.</ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36778275">36778275</a> Investigations on SARS-CoV-2 and other coronaviruses in mink farms in France at the end of the first year of COVID-19 pandemic.<ul><li>All SARS-CoV-2 genomes detected were nested within the 20A clade (<b>Nextclade</b>), together with SARS-CoV-2 genomes from humans sampled at the same period. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36576175">36576175</a> Characterization of <i>SARS-CoV-2</i> omicron variants from Iran and evaluation of the effect of mutations on the spike, nucleocapsid, ORF8, and ORF9b proteins function.<ul><li>In this study, genomics variations were evaluated for 96 samples isolated from Iran from March to July 2022 using the <b>Nextclade</b> web server and informatics tools. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36611300">36611300</a> Complete Genomic Characterisation and Mutation Patterns of Iraqi SARS-CoV-2 Isolates.<ul><li>Analysis of the isolates of this study showed that all the sequences from the third wave were clustered in the GK clades and the 21J (Delta) clade according to the GISAID and <b>Nextclade</b> systems, while the PANGO system revealed that six sequences were B.1.617.2 and four sequences were of the AY.33 lineage. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560792">36560792</a> Genomic Epidemiology of SARS-CoV-2 in Western Burkina Faso, West Africa.<ul><li>Genomic sequences were assigned to phylogenetic clades using <b>NextClade</b> and to Pango lineages using pangolin. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35905853">35905853</a> Analysis of the mutation dynamics of SARS-CoV-2 genome in the samples from Georgia State of the United States.<ul><li><b>NextClade</b>, an online tool was used for the analysis of the sequences using Wuhan-Hu-1/2019 as a reference genome. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36417247">36417247</a> Molecular Epidemiological Investigations of Localized SARS-CoV-2 Outbreaks-Utility of Public Algorithms.<ul><li>These tools include phylogenetic clustering and mutational analysis utilizing <b>Nextclade</b> and Ultrafast Sample placement on Existing tRee (UShER). </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36140622">36140622</a> Molecular Characterization of Whole-Genome SARS-CoV-2 from the First Suspected Cases of the XE Variant in the Lazio Region, Italy.<ul><li>Whole-Genome Sequencing (WGS) performed on an Ion Torrent GSS5 platform according to Italian flash surveys showed genomes corresponding to the PANGOLIN unclassified lineage and the <b>Nextclade</b> XE clade. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35723286">35723286</a> Automated Library Construction and Analysis for High-Throughput Nanopore Sequencing of SARS-CoV-2.<ul><li>Sequences were analyzed using <b>Nextclade</b>, assigning both a clade and quality score to each sample.</ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35723286">35723286</a> Automated Library Construction and Analysis for High-Throughput Nanopore Sequencing of SARS-CoV-2.<ul><li>The use of samples with known threshold cycle (Ct) values enabled normalization, acted as a quality control check, and revealed a strong correlation between sample Ct values and successful analysis, with 85% of samples with Ct&#x2009;&lt;&#x2009;30 achieving a "good" <b>Nextclade</b> score. </ul></p><p>2023 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35818397">35818397</a> First case of within-host co-infection of different SARS-CoV-2 variants in Ecuador.<ul><li>When examining the phylogenetic analysis in <b>NextClade</b> and Pangolin webservers, and considering the genotyping findings, conflicting results were obtained.</ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746651">35746651</a> Omicron Sub-Lineages (BA.1.1.529 + BA.*) Current Status in Ecuador.<ul><li>In this work, we use all Omicron sequences from Ecuador available at GISAID until March 2022 and the software <b>Nextclade</b> and Pangolin to identify which lineages circulate in this country. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35459088">35459088</a> Comparison of SARS-CoV-2 sequencing using the ONT GridION and the Illumina MiSeq.<ul><li>Our results show that the MiSeq had a significantly higher number of consensus genomes classified by <b>Nextclade</b> as good and mediocre compared to the GridION. </ul></p><p>2022 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35045124">35045124</a> Genomic surveillance of SARS-CoV-2 in the state of Delaware reveals tremendous genomic diversity.<ul><li>Pangolin and <b>Nextclade</b> were used to classify these sequences into 16 unique clades and 88 lineages. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34946210">34946210</a> Genomic Epidemiology of SARS-CoV-2 Divulge B.1, B.1.36, and B.1.1.7 as the Most Dominant Lineages in First, Second, and Third Wave of SARS-CoV-2 Infections in Pakistan.<ul><li>Finally, maximum likelihood based phylogenetic tree was built by using the Nextstrain and coverage map was created by employing <b>Nextclade</b>. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34855869">34855869</a> Patient health records and whole viral genomes from an early SARS-CoV-2 outbreak in a Quebec hospital reveal features associated with favorable outcomes.<ul><li>Dimensionality reduction identified two subclades with mutations of clinical interest, namely in the Spike protein, that evaded supervised lineage assignment methods-including Pangolin and <b>NextClade</b> supervised lineage assignment tools. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33991677">33991677</a> Spreading of a new SARS-CoV-2 N501Y spike variant in a new lineage.<ul><li>SARS-CoV-2 sequences obtained from human nasopharyngeal samples by Illumina next-generation sequencing were analysed using <b>Nextclade</b> and an in-house Python script and were compared using BLASTn to the GISAID database. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34362060">34362060</a> A Possible Role of Remdesivir and Plasma Therapy in the Selective Sweep and Emergence of New SARS-CoV-2 Variants.<ul><li>These variants include the Marseille-4 lineage (<b>Nextclade</b> 20A.EU2) in Europe, the 20I/501Y.V1 variant first detected in the UK, the 20H/501Y.V2 variant first detected in South Africa, and the 20J/501Y.V3 variant first detected in Brazil. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33857895">33857895</a> Rapid screening for SARS-CoV-2 variants of concern in clinical and environmental samples using nested RT-PCR assays targeting key mutations of the spike protein.<ul><li>Prior to application to field samples, the discriminatory potential of this PCR assay was explored using GISAID and <b>Nextclade</b>. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33785459">33785459</a> Emergence and outcomes of the SARS-CoV-2 'Marseille-4' variant.<ul><li>The 1038 SARS-CoV-2 whole genome sequences obtained in our laboratory by next-generation sequencing with Illumina technology were analysed using <b>Nextclade</b> and nextstrain/ncov pipelines and IQ-TREE. </ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33737129">33737129</a> Genomic surveillance of SARS-CoV-2 in the Republic of Congo.<ul><li>Subsequently, different SARS-CoV-2 lineages were assigned using PANGOLIN and <b>Nextclade</b>.</ul></p><p>2021 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33737129">33737129</a> Genomic surveillance of SARS-CoV-2 in the Republic of Congo.<ul><li>The Congolese SARS-CoV-2 sequences were shown to belong to lineage B1 and <b>Nextclade</b> 20A and 20C, which split them into distinct clusters, indicating two separate introductions of the virus into the Republic of Congo.</ul></p></body>
