<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<title>Sentences that contain nextstrain</title>
</head>
<body>
<h1>Sentences that contain nextstrain</h1>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37165019">37165019</a> <b>(2)</b> SARS-CoV-2 viral variants can rapidly be identified for clinical decision making and population surveillance using a high-throughput digital droplet PCR assay<ul><li> Thus far, there have been four major waves of infection: the first lineage to spread globally, B.1 (referred here as original sequence, <b>Nextstrain</b> clade 20A), followed by the VOCs Alpha, Delta, and now Omicron and its sub-lineages.<li>We designed two multiplex ddPCR reactions that target a total of seven viral sequences which allow the discrimination between the variants Alpha, Beta, Gamma, Delta, Delta K417N, Lambda, Mu, Omicron BA.1 and BA.2, and the original sequence (USA-WA1/2020, &#x201C;Washington Isolate&#x201D;, <b>NextStrain</b> Clade 20A).</ul></p>
<p><u>No PMID</u> <b>(0)</b> Divergent mutations of Delta and Omicron variants: key players behind differential viral attributes across the COVID-19 waves<ul><li> The corresponding nucleotide sequences of mutation was obtained from Next Clade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37155081">37155081</a> <b>(4)</b> Consensus insertion/deletions and amino acid variations of all coding and noncoding regions of the SARS-CoV-2 Omicron clades, including the XBB and BQ.1 lineages<ul><li> This variant has swiftly diverged into at least eight clades based on the data available in GISAID (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>).<li>Fifteen to 25 sequences of lineages 21K, 21L, 22A, 22B, 22C, 22D, 22E, and 22F, as well as XBB and BQ.1, were randomly selected from the GISAID <b>Nextstrain</b> phylogeny on October 31, 2022 and downloaded.<li>In this study, we confirm the clade separation reported by <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37148902">37148902</a> <b>(4)</b> Virological characteristics of the SARS-CoV-2 omicron XBB.1.16 variant<ul><li>In late February, 2023, some sublineages of the SARS-CoV-2 omicron XBB variant harbouring the F486P substitution in the spike protein (eg, XBB.1.5 and XBB.1.9) predominated worldwide, according to <b>Nextstrain</b>.<li>For more on <bold><b>Nextstrain</b></bold> see <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m" id="interrefs10">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37134083">37134083</a> <b>(8)</b> Metagenomic next-generation sequencing to characterize potential etiologies of non-malarial fever in a cohort living in a high malaria burden area of Uganda<ul><li>The <b>Nextstrain</b> platform [<xref rid="pgph.0001675.ref034" ref-type="bibr"><u>34</u></xref>] was used to perform phylogenetic inference of the SARS-CoV-2, Influenza A, and RSV genomes obtained from this analysis.<li> Within our <b>Nextstrain</b> build, we specified that all sequences sampled from Uganda available on GISAID should be included in the phylogenetic analysis.<li> Within our <b>Nextstrain</b> build, we specified that all sequences sampled from Africa available from EpiFlu should be included in the phylogenetic analysis, along with the Influenza A viruses sequenced as part of this study.<li> Within our <b>Nextstrain</b> build, we specified that all sequences sampled from Africa available from EpiRSV should be included in the phylogenetic analysis, along with the RSV viruses sequenced as part of this study.<li>We used <b>Nextstrain</b> Augur to perform a multiple sequence alignment of input sequences using <italic toggle="yes">MAFFT [<xref rid="pgph.0001675.ref037" ref-type="bibr"><u>37</u></xref>]</italic> and to strip the multiple sequence alignment to the reference genome, which removes sequence insertions that introduce gaps in the reference sequence.<li> <b>Nextstrain</b> phylogenetic trees were exported as JSON files by the pipeline, which we then visualized and explored in the web browser using <b>Nextstrain</b> Auspice.<li> The R code used to produce <xref rid="pgph.0001675.g002" ref-type="fig"><u>Fig 2</u></xref>, <b>Nextstrain</b> trees generated in this analysis, and corresponding Snakefiles are available on GitHub at: <ext-link xlink:href="https://github.com/czbiohub/rapid-response-prism" ext-link-type="uri">https://github.com/czbiohub/rapid-response-prism</ext-link>.<li> [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37039029">37039029</a> <b>(1)</b> Reinfection rate and disease severity of the BA.5 Omicron SARS-CoV-2 lineage compared to previously circulating variants of concern in the Canary Islands (Spain)<ul><li> The tree also includes relationships with other 1,504 genome sequences representative of major clades (light grey), labelled based on <b>Nextstrain</b> definitions, from Nextclade v2.13.0 and Auspice v2.24.2 (as of 2023-03-24), and sampled between December 2019 and July 2022 for reference.</ul></p>
<p><u>No PMID</u> <b>(7)</b> A European perspective of phylogenomics, sublineages, geographical distribution, epidemiology, and mutational landscape of mpox virus: Emergence pattern may help to fight the next public health emergency in Europe<ul><li> Here we have used the two servers and developed two types of phylogenomic trees: The <b>nextstrain</b> server <xref rid="bib20" ref-type="bibr"><u>[20]</u></xref> and the Taxonium server <xref rid="bib21" ref-type="bibr"><u>[21]</u></xref>.<li> For country-wise phylogenomics analysis, we have used the <b>nextstrain</b> server <xref rid="bib20" ref-type="bibr"><u>[20]</u></xref>.<li>Here, we have used the <b>nextstrain</b> server <xref rid="bib20" ref-type="bibr"><u>[20]</u></xref> to illustrate the divergence and scatterplot of hMPXV1 genome clusters in different European countries.<li>To depict the geographical distribution of hMPXV1 sublineages in Europe, we have also used the <b>nextstrain</b> server <xref rid="bib20" ref-type="bibr"><u>[20]</u></xref>.<li>To analyze the entropy diversity of hMPXV1 genomes from European countries, we have also used the open data sources of the <b>nextstrain</b> server <xref rid="bib20" ref-type="bibr"><u>[20]</u></xref>.<li>To evaluate the entropy diversity of hMPXV1 genomes from European countries, we have also used the open data sources of the <b>nextstrain</b> server <xref rid="bib20" ref-type="bibr"><u>[20]</u></xref>.<li>All hMPXV1 genome sequences were retrieved from GenBank of NCBI and analyzed using the <b>nextstrain</b> server <xref rid="bib20" ref-type="bibr"><u>[20]</u></xref> and other servers.&#xA0;Finally, a flowchart has been depicted to illustrate the different steps of our study (<xref rid="fig0005" ref-type="fig">Fig.<li> [[20]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[21]](          <person-group person-group-type="author">            <name>              <surname>Sanderson</surname>              <given-names>T.</given-names>            </name>          </person-group>          <article-title>Taxonium, a web-based tool for exploring large phylogenetic trees</article-title>          <source>eLife</source>          <volume>11</volume>          <year>2022</year>          <object-id pub-id-type="publisher-id">e82392</object-id>        ) [[20]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[20]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[20]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[20]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[20]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[20]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37120637">37120637</a> <b>(1)</b> CRISPR-Cas13a-powered electrochemical biosensor for the detection of the L452R mutation in clinical samples of SARS-CoV-2 variants<ul><li> During the period of this study, BA.2 and BA.5 are the two main subtypes rampaging worldwide, because BA.3 does not meet the &#x201C;<b>Nextstrain</b> clade definition criteria.&#x201C; BA.4 and BA.5 resist most broad neutralizing antibodies because of their multiple mutated sites and the mutation at the spike amino acid 452 [<xref ref-type="bibr" rid="CR3"><u>3</u></xref>].<li> [3](Shrestha LB, Foster C, Rawlinson W, Tedla N, Bull RA. Reviews in Medical Virology n.d., n/a, e2381.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37115761">37115761</a> <b>(1)</b> Insights from a workplace SARS-CoV-2 specimen collection program, with genomes placed into global sequence phylogeny<ul><li>Consensus sequences for each sample were analyzed by UShER [<xref rid="pone.0285042.ref021" ref-type="bibr"><u>21</u></xref>] and placed into a global phylogenetic tree sorted by <b>Nextstrain</b> clade.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37101215">37101215</a> <b>(0)</b> SARS-CoV-2 surveillance-based on municipal solid waste leachate in Brazil<ul><li> <xref ref-type="bibr" rid="CR41"><u>2021</u></xref>) and Nextclade version v.0.14.2 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) (Aksamentov et al.<li> [2021](O&#x2019;Toole &#xC1;, Scher E, Underwood A, Jackson B, Hill V, McCrone JT, Colquhoun R, Ruis C, Abu-Dahab K, Taylor B, Yeats C, du Plessis L, Maloney D, Medd N, Attwood SW, Aanensen DM, Holmes EC, Pybus OG, Rambaut A (2021) Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool. Virus Evol 7(2):veab064. 10.1093/ve/veab064) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37149443">37149443</a> <b>(3)</b> Efficacy of primary series AZD1222 (ChAdOx1 nCoV-19) vaccination against SARS-CoV-2 variants of concern: Final analysis of a randomized, placebo-controlled, phase 1b/2 study in South African adults (COV005)<ul><li> The whole genomes were compared against the global reference dataset using a custom pipeline based on a local version of <b>NextStrain</b> for variant classification of sequenced isolates.<li> It performs alignment of genotypes in Nextalign (<ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/nextalign" id="ir010">https://github.com/neherlab/nextalign</ext-link>), phylogenetic tree inference in IQ-Tree, tree dating <xref rid="b0070" ref-type="bibr"><u>[14]</u></xref> and ancestral state construction and annotation (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="ir015">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> [[14]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35923324">35923324</a> <b>(5)</b> Genomic epidemiology reveals the dominance of Hennepin County in transmission of SARS-CoV-2 in Minnesota from 2020&#x2013;2022<ul><li> Using this data, they derived county data on synonymous and non-synonymous single nucleotide variants (SNVs), and performed a variety of phylogenetic analyses (using <b>Nextstrain</b> [<xref rid="R5" ref-type="bibr"><u>5</u></xref>] and BEAST2 [<xref rid="R6" ref-type="bibr"><u>6</u></xref>]), determined R0 for their region, and examined the number and timing of introductions to the two counties and how each introduction subsequently impacted the local transmission [<xref rid="R4" ref-type="bibr"><u>4</u></xref>].<li>We down-sampled Minnesota genomes nearly proportional to the number of COVID-19 cases per county and then added additional genomes from NCBI GenBank [<xref rid="R3" ref-type="bibr"><u>3</u></xref>] as part of an international dataset used in <b>Nextstrain</b> [<xref rid="R5" ref-type="bibr"><u>5</u></xref>] (see <xref rid="S8" ref-type="sec"><u>Methods</u></xref>).<li> S4</xref>) that included SARS-CoV-2 genome sequences from the 20 counties with the greatest number of reported COVID-19 cases as of February 28, 2022 as well as a global representation of sequences available via GenBank as part of an open access dataset from <b>Nextstrain</b> (<xref rid="SD1" ref-type="supplementary-material"><u>Table S2</u></xref>) [<xref rid="R23" ref-type="bibr"><u>23</u></xref>].<li> We created initial an phylogenetic tree via <b>Nextstrain</b>&#x2019;s <italic toggle="yes">augur tree</italic> command [<xref rid="R5" ref-type="bibr"><u>5</u></xref>] with IQTree and rooted the tree based on Wuhan-1 (MN908947) [<xref rid="R27" ref-type="bibr"><u>27</u></xref>].<li> We removed additional sequences that had potential misassigned clades or produced inconsistences with the phylogenetic structure as shown in <b>Nextstrain</b>&#x2019;s global all-time subsampled dataset [<xref rid="R29" ref-type="bibr"><u>29</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30958812">30958812</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33144575">33144575</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33196830">33196830</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](<source>Genomic epidemiology of SARS-CoV-2 with subsampling focused globally over the past 6 months</source>. <year>2022</year> [<comment>cited</comment><date-in-citation>2022 May 1</date-in-citation>]; <comment>Available from: <ext-link xlink:href="https://nextstrain.org/ncov/open/global/6m" ext-link-type="uri">https://nextstrain.org/ncov/open/global/6m</ext-link>.</comment>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [27](<name><surname>Wu</surname><given-names>F.</given-names></name>, <etal/><source>Severe acute respiratory syndrome coronavirus 2 isolate Wuhan-Hu-1, complete genome</source>. <year>2020</year><month>Mar</month><day>18</day> [<comment>cited</comment><date-in-citation>2022 May 1</date-in-citation>]; <comment>Available from: <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MN908947" ext-link-type="uri">https://www.ncbi.nlm.nih.gov/nuccore/MN908947</ext-link>.</comment>) [29](<name><surname>Bedford</surname><given-names>T.</given-names></name><source>Genomic epidemiology of SARS-CoV-2 with subsampling focused globally since pandemic start</source>. <year>2023</year> [<comment>cited</comment><date-in-citation>2023 Apr 13</date-in-citation>]; <comment>Available from: <ext-link xlink:href="https://nextstrain.org/ncov/gisaid/global/all-time" ext-link-type="uri">https://nextstrain.org/ncov/gisaid/global/all-time</ext-link>.</comment>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37099540">37099540</a> <b>(3)</b> Evaluation of miniaturized Illumina DNA preparation protocols for SARS-CoV-2 whole genome sequencing<ul><li> QC metrics were generated using Nextclade, a tool that identifies differences between your sequences and a reference sequence used by <b>Nextstrain</b> [<xref rid="pone.0283219.ref021" ref-type="bibr"><u>21</u></xref>], which uses these differences to assign your sequences to clades, and reports potential sequence quality issues in your data.<li> Nextclade is part of the <b>Nextstrain</b> host of tools that have been pivotal in the real-time tracking of SARS-CoV-2 and other infectious disease pathogens globally.<li> In the case of SARS-CoV-2, <b>Nextstrain</b> predefined criteria for the quality of genomes to be included in downstream analysis.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37093599">37093599</a> <b>(1)</b> Neutralizing Monoclonal Antibody Use and COVID-19 Infection Outcomes<ul><li> FASTQ files<sup><xref rid="zoi230307r22" ref-type="bibr"><u>22</u></xref></sup> were processed through a common pipeline to produce World Health Organization (WHO) variant calls for all SARS-CoV-2 variants and <b>Nextstrain</b> clade classifications<sup><xref rid="zoi230307r23" ref-type="bibr"><u>23</u></xref></sup> for Delta variants.<li> [22](Cock PJA, Fields CJ, Goto N, Heuer ML, Rice PM. The Sanger FASTQ file format for sequences with quality scores, and the Solexa/Illumina FASTQ variants. <italic>Nucleic Acids Res</italic>. 2010;38(6):1767-1771. <ext-link xlink:href="http://doi:10.1093/nar/gkp1137" ext-link-type="uri">doi:10.1093/nar/gkp1137</ext-link>) [23](Nextstrain: real-time tracking of pathogen evolution. Nextstrain.org. Accessed March 19, 2023. <ext-link xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37087497">37087497</a> <b>(2)</b> HaploCoV: unsupervised classification and rapid detection of novel emerging variants of SARS-CoV-2<ul><li> Genomic positions associated with large insertions or deletions &gt;3&#x2009;nt in size were masked by the mask-alignment.py as available from <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur">https://github.com/<b>nextstrain</b>/augur</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36848460">36848460</a> <b>(8)</b> Application of Phylodynamic Tools to Inform the Public Health Response to COVID-19: Qualitative Analysis of Expert Opinions<ul><li> Bioinformatics and data visualization tools that use phylogenetic trees, such as <b>Nextstrain</b> [<xref rid="ref12" ref-type="bibr"><u>12</u></xref>], COVID-19 CoV Genetics (COVID-19 CG) [<xref rid="ref13" ref-type="bibr"><u>13</u></xref>], and Ultrafast Sample placement on Existing tRees (UShER) [<xref rid="ref14" ref-type="bibr"><u>14</u></xref>], have rapidly gained popularity.<li> Since the beginning of the pandemic, the <b>Nextstrain</b> tool has traced SARS-CoV-2 by adding sequences in real time to a global epidemic tree, aiding with the localization of new infections within existing clusters.<li> For example, heavily subsampled, custom <b>Nextstrain</b> analyses have been used to investigate the possible patient 0 in Italy [<xref rid="ref15" ref-type="bibr"><u>15</u></xref>] and undetected transmission at the beginning of the pandemic in the United States [<xref rid="ref16" ref-type="bibr"><u>16</u></xref>].<li>Tell us about your use (or knowledge of) phylogenetic tools for public health (eg, Microbe-Trace, HIV-TRACE, or <b>Nextstrain</b>).<li>Tell us about your use (or knowledge of) phylogenetic tools for public health (eg, Microbe-Trace, HIV-TRACE, or <b>Nextstrain</b>).<li> Journalists see all these nice graphs from <b>Nextstrain</b>, and they make their own conclusions, but they are not epidemiologists.<li>I think 1 of the things that could be improved in <b>Nextstrain</b> is the background data set because unless you have a specific instance for your region, the conclusions you'll be making from the general background data set available will be quite biased and probably wrong.<li> Open source phylogenetic and data visualization platforms for monitoring SARS-CoV-2 genomic epidemiology, such as <b>Nextstrain</b> [<xref rid="ref12" ref-type="bibr"><u>12</u></xref>], have rapidly gained popularity for their ability to illuminate spatial-temporal transmission patterns worldwide.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33620031">33620031</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) [15](          <person-group person-group-type="author">            <name>              <surname>Scir&#xE9;</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Vaughan</surname>              <given-names>T</given-names>            </name>            <name>              <surname>Nadeau</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Stadler</surname>              <given-names>T</given-names>            </name>          </person-group>          <article-title>Phylodynamic analyses based on 11 genomes from the Italian outbreak: insights on the R0, the number of cases through time, and the time of epidemic origin</article-title>          <source>virological.org</source>          <year>2020</year>          <month>3</month>          <day>6</day>          <date-in-citation content-type="access-date">2023-02-28</date-in-citation>          <comment><ext-link xlink:href="https://virological.org/t/phylodynamic-analyses-based-on-11-genomes-from-the-italian-outbreak/426" ext-link-type="uri">https://virological.org/t/phylodynamic-analyses-based-on-11-genomes-from-the-italian-outbreak/426</ext-link></comment>        ) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32913002">32913002</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37111512">37111512</a> <b>(0)</b> SARS-CoV-2 Variants by Whole-Genome Sequencing in a University Hospital in Bangkok: First to Third COVID-19 Waves<ul><li> The consensus sequences were assessed using NextClade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, version 1.9.0, accessed on 1 October 2021) for sequence clade assignment, identification quantification of mutations, and sequence quality analyses.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37151582">37151582</a> <b>(3)</b> NGS implementation for monitoring SARS-CoV-2 variants in Chicagoland: An institutional perspective, successes and challenges<ul><li> This labor-intensive effort was quickly abandoned when we began using more specialized open source software packages provided by <b>Nextstrain</b>, Pangolin, and Nextclade, reducing the necessity of manual analysis.<li> After using <b>Nextstrain</b> (<xref rid="ref9" ref-type="bibr"><u>9</u></xref>) for a few months, we recognized that variant nomenclature was evolving away from <b>Nextstrain</b> clade names to Pango lineage (<xref rid="ref10" ref-type="bibr"><u>10</u></xref>) and WHO labels.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37081087">37081087</a> <b>(1)</b> Comparison of SARS-CoV-2 whole genome sequencing using tiled amplicon enrichment and bait hybridization<ul><li>The consensus sequences were annotated for virus clade information using Nextclade (<b>NEXTstrain</b> version 1.14.1) and Pangolin (pangolin lineage version v3.1.20 with pangoLEARN version 2022-04-28).<li> The first 300 missing sites are not penalized (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/nextclade/en/stable/user/algorithm/07-quality-control.html">https://docs.<b>nextstrain</b>.org/projects/nextclade/en/stable/user/algorithm/07-quality-control.html</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112997">37112997</a> <b>(1)</b> Epidemiological Surveillance Reveals the Rise and Establishment of the Omicron SARS-CoV-2 Variant in Brazil<ul><li> A dataset (<italic toggle="yes">n</italic> = 1640) containing public reference genomes classified by <b>Nextstrain</b> [<xref rid="B42-viruses-15-01017" ref-type="bibr"><u>42</u></xref>] as Omicron lineages (<italic toggle="yes">n</italic> = 1531) (acknowledgments are available in <xref rid="app1-viruses-15-01017" ref-type="app"><u>Supplementary File S5</u></xref>) and genomes generated in this study (<italic toggle="yes">n</italic> = 109) was aligned using Minimap2 [<xref rid="B36-viruses-15-01017" ref-type="bibr"><u>36</u></xref>].<li> [42](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29750242">29750242</a>) </ul></p>
<p><u>No PMID</u> <b>(0)</b> A CRISPR-enhanced metagenomic NGS test to improve pandemic preparedness<ul><li><hr/></td></tr><tr><td>Kraken2</td><td>Wood et&#xA0;al., 2019<xref rid="bib39" ref-type="bibr"><sup>34</sup></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/DerrickWood/kraken2/wiki/Manual" id="intref0010">https://github.com/DerrickWood/kraken2/wiki/Manual</ext-link></td></tr><tr><td>Bracken</td><td>Lu et&#xA0;al., 2017<xref rid="bib38" ref-type="bibr"><sup>35</sup></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://ccb.jhu.edu/software/bracken/index.shtml" id="intref0015">https://ccb.jhu.edu/software/bracken/index.shtml</ext-link></td></tr><tr><td>CZ ID</td><td>Chan Zuckerberg Biohub, San Francisco, CA</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/chanzuckerberg/idseq-workflows" id="intref0020">https://github.com/chanzuckerberg/idseq-workflows</ext-link></td></tr><tr><td>NextClade</td><td>Aksamentov et&#xA0;al., 2021<xref rid="bib22" ref-type="bibr"><sup>22</sup></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="intref0025">https://clades.<b>nextstrain</b>.org</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37071654">37071654</a> <b>(3)</b> Variant-specific introduction and dispersal dynamics of SARS-CoV-2 in New York City &#x2013; from Alpha to Omicron<ul><li> sequences which were used in the <b>Nextstrain</b> [<xref rid="ppat.1011348.ref046" ref-type="bibr"><u>46</u></xref>] build focused on North America (but that also includes genomic sequences from the other continents).<li> Specifically, for each variant, we used the North American <b>Nextstrain</b> build that was available on the last collection date of the considered variant in our data set.<li>Time-scaled phylogenies were inferring using a two-step procedure consisting of (i) first estimating a maximum-likelihood (ML) phylogeny using IQ-TREE 2.2.0.3 [<xref rid="ppat.1011348.ref047" ref-type="bibr"><u>47</u></xref>] and (ii) subsequently time-calibrating the resulting maximum-likelihood tree using TreeTime 0.7.4 [<xref rid="ppat.1011348.ref048" ref-type="bibr"><u>48</u></xref>], specifying an evolutionary rate of 8x10<sup>-4</sup> substitutions per site per year (s/s/y), as in the <b>Nextstrain</b> workflow.<li> [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [47](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [48](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a> <b>(2)</b> Variant-specific introduction and dispersal dynamics of SARS-CoV-2 in New York City &#x2013; from Alpha to Omicron<ul><li> Notably there is also some reliance on <b>NextStrain</b> sampling - how would this impact the results.<li> Notably there is also some reliance on <b>NextStrain</b> sampling - how would this impact the results.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112975">37112975</a> <b>(1)</b> Mpox Virus: Its Molecular Evolution and Potential Impact on Viral Epidemiology<ul><li> The clade that each sequence belongs to was assigned using nucleotide substitutions defined by the <b>Nextstrain</b> team [<xref rid="B92-viruses-15-00995" ref-type="bibr"><u>92</u></xref>].<li> [92](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><u>No PMID</u> <b>(24)</b> CurSa: scripts to curate metadata and sample genomes from GISAID for analysis and display in <b>nextstrain</b> and microreact<ul><li> <b>Nextstrain</b> and Microreact are two popular and fine platforms used by the community to study and visualize the evolution of pathogens in a geographical context [<xref rid="bpad007-B3" ref-type="bibr"><u>3</u></xref>, <xref rid="bpad007-B4" ref-type="bibr"><u>4</u></xref>].<li> The two main components of <b>Nextstrain</b> are Augur and Auspice.<li>These inconsistencies impair the performance of <b>Nextstrain</b> because geographic localities showing varying names in metadata and reference files from Augur will not be properly georeferenced.<li>Moreover, running a <b>Nextstrain</b> analysis with 4500 sequences on a personal computer (MacPro 3.5&#x2009;GHz 6-Core Intel Xeon E5, 16 GB 1866&#x2009;MHz DDR3) takes about 28&#x2009;h to complete.<li> Attempting a comprehensive <b>Nextstrain</b> analysis in a personal computer is impractical if not impossible for those countries with a large number of sequences.<li> Therefore, scientists attempting an evolutionary study of SARS-CoV-2 face the problem of how to adequately sample a set of input sequences for practical <b>Nextstrain</b> analysis.<li><b>Nextstrain</b>, via Augur and accompanying Python scripts, offers strategies to (i) filter and then (ii) sample genome sequences at different geographic resolutions.<li> Here, we also provide a script designated to facilitate the sampling of SARS-CoV-2 genome sequences downloaded from GISAID to make a phylogenomic analysis in <b>Nextstrain</b> and their display in Microreact.<li><b>Nextstrain</b> provides via Augur a method to filter sequences from specific regions and/or dates.<li> Here, we provide a simple alternative to the method provided by Augur to subsample sequences for <b>Nextstrain</b> analysis.<li>Once the genome sequences have been analyzed with <b>Nextstrain</b> (via Augur) and a multiple sequence alignment and a phylogenetic tree are generated, the script <monospace>create_microreact.pl</monospace> can be used to generate the file that is required by Microreact to display the sequences in a geographical context.<li>CurSa scripts can be used in combination with those provided by <b>Nextstrain</b>/Augur (<xref rid="bpad007-F4" ref-type="fig"><u>Fig.&#xA0;4</u></xref>).<li><b>Nextstrain</b> analysis via Augur and display with Auspice.<li>Finally, we copied the outfiles obtained with <monospace>sample.pl</monospace> to the <monospace>data/</monospace> directory and the curated files <monospace>color_ordering.tsv</monospace> and <monospace>lat_longs.tsv</monospace> to the <monospace>defaults/</monospace> directory within the local <b>Nextstrain</b> build.<li> And we configured Auspice to contextualize the genome sequences from Mexico with those GenBank sequences of the Global sample provided by <b>Nextstrain</b> (<xref rid="sup1" ref-type="supplementary-material"><u>Supplementary File S1</u></xref>, Result S5).<li>Phylogenetic analysis of sampled sequences inferred with <b>Nextstrain</b>.<li>To display the phylogenetic analysis performed with <b>Nextstrain</b> in Microreact, we executed the script <monospace>create_microreact.pl</monospace> (<xref rid="sup1" ref-type="supplementary-material"><u>Supplementary File S1</u></xref>, Result S6).<li> The <monospace>aligned.fasta</monospace> file is the result of running <b>Nextstrain</b> on the sampled set of sequences and is found in <monospace>ncov</monospace><monospace>/results/</monospace><monospace>yourconfigfile</monospace><monospace>/</monospace> within the local <b>Nextstrain</b> build.<li>Sampled sequences can be visualized in Microreact all together with the phylogeny inferred with <b>Nextstrain</b>.<li>Phylogenetic analyses in <b>Nextstrain</b> assisted by CurSa scripts were used to describe the diversity of SARS-CoV-2 prior vaccination stages in Mexico [<xref rid="bpad007-B5" ref-type="bibr"><u>5</u></xref>]; and are used to maintain Mexstrain (<ext-link xlink:href="https://ira.cinvestav.mx/mexstrain/" ext-link-type="uri">https://ira.cinvestav.mx/mexstrain/</ext-link>) a web page designated to facilitate the visualization of the phylogeny of SARS-CoV-2 in Mexico.<li>CurSa is a suite of Perl scripts designated to curate metadata files and sample genome sequences from SARS-CoV-2 (that were downloaded from GISAID) for phylogenomic analysis in <b>Nextstrain</b> and display in Microreact.<li> CurSa scripts allow the user to sample a moderate number of sequences to run <b>Nextstrain</b> on a personal computer and to correct inconsistencies in metadata files regarding the geographic location of the samples.<li> The set of sequences sampled with CurSa are a starting point to more sophisticated subsampling methods implemented in <b>Nextstrain</b> if desired.<li> [3](<person-group person-group-type="author"><string-name><surname>Hadfield</surname><given-names>J</given-names></string-name>, <string-name><surname>Megill</surname><given-names>C</given-names></string-name>, <string-name><surname>Bell</surname><given-names>SM</given-names></string-name></person-group><etal>et al</etal><article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics</source><year>2018</year>;<volume>134</volume>:<fpage>4121</fpage>&#x2013;<lpage>3</lpage>. doi:<pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>) [4](<person-group person-group-type="author"><string-name><surname>Argim&#xF3;n</surname><given-names>S</given-names></string-name>, <string-name><surname>Abudahab</surname><given-names>K</given-names></string-name>, <string-name><surname>Goater</surname><given-names>RJE</given-names></string-name></person-group><etal>et al</etal><article-title>Microreact: visualizing and sharing data for genomic epidemiology and phylogeography</article-title>. <source>Microb Genom</source><year>2016</year>;<volume>302</volume>:<fpage>e000093</fpage>.doi:<pub-id pub-id-type="doi">10.1099/mgen.0.000093</pub-id>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34846283">34846283</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36927408">36927408</a> <b>(0)</b> Phylogenetic landscape of Monkeypox Virus (MPV) during the early outbreak in New York City, 2022<ul><li> GenBank accession number: NC_063383.1) using the NextClade tool v1.5.4 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37077184">37077184</a> <b>(18)</b> Temporal dynamics and fatality of SARS&#x2010;CoV&#x2010;2 variants in Bangladesh<ul><li> Based on the potential threats of these viral variants in terms of transmission, disease severity, immune escape, and so forth, the Center for Disease Control and Prevention (CDC) has classified SARS&#x2010;CoV&#x2010;2 variants into four categories, Variant of Interest (VOI), Variant of Concern (VOC), Variants Being Monitored (VBM) and Variant of High Consequences (VOHC).<xref rid="hsr21209-bib-0016" ref-type="bibr"><sup>16</sup></xref> Different variants of SARS&#x2010;CoV&#x2010;2 are also named according to different nomenclature systems such as Phylogenetic Assignment of Named Global Outbreak (PANGO), GISAID, and <b>Nextstrain</b>.<li> Based on the potential threats of these viral variants in terms of transmission, disease severity, immune escape, and so forth, the Center for Disease Control and Prevention (CDC) has classified SARS&#x2010;CoV&#x2010;2 variants into four categories, Variant of Interest (VOI), Variant of Concern (VOC), Variants Being Monitored (VBM) and Variant of High Consequences (VOHC).<xref rid="hsr21209-bib-0016" ref-type="bibr"><sup>16</sup></xref> Different variants of SARS&#x2010;CoV&#x2010;2 are also named according to different nomenclature systems such as Phylogenetic Assignment of Named Global Outbreak (PANGO), GISAID, and <b>Nextstrain</b>.<xref rid="hsr21209-bib-0014" ref-type="bibr"><sup>14</sup></xref>, <xref rid="hsr21209-bib-0017" ref-type="bibr"><sup>17</sup></xref> <b>Nextstrain</b> is an open&#x2010;source project<xref rid="hsr21209-bib-0018" ref-type="bibr"><sup>18</sup></xref> to harness the scientific and public health potential of pathogen genome data to define the chronological emergence of new variants of SARS&#x2010;CoV&#x2010;2.<li> This massive genome sequencing effort has informed us about the evolution and spread of variants of SARS&#x2010;CoV&#x2010;2.<xref rid="hsr21209-bib-0026" ref-type="bibr"><sup>26</sup></xref> In this study, we have primarily used the <b>Nextstrain</b> and PANGO nomenclature, which names strains according to the year they most spread.<li> Through an in&#x2010;depth investigation into the emergence and pursuit of VOC, such as Alpha, Beta, Gamma, and Delta, and VOIs, such as Eta, several <b>Nextstrain</b> clades and/or PANGO lineages, this study reported the emergence and spread of the SARS&#x2010;CoV&#x2010;2 variants, clades and lineages, their evolution, transmissibility and association with relevant clinical traits (IFR and CFR) in Bangladesh from March 2020 to October 2022.<li> The clade and Pango lineages were assigned by using Nextclade v2.8.1 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> All the metadata (Submission date, strain, GISAID ID number, region/location, division, length, host, age, sex, Pangolin lineage, GISAID/<b>Nextstrain</b> clade, originating and submitting lab, authors) from GISAID were collected for analysis (Supporting Information: Data&#xA0;<xref rid="hsr21209-suppl-0001" ref-type="supplementary-material"><u>S1</u></xref>).<li> The average CFR was calculated by the number of monthly deaths and number of confirmed COVID&#x2010;19 cases.<xref rid="hsr21209-bib-0027" ref-type="bibr"><sup>27</sup></xref>, <xref rid="hsr21209-bib-0028" ref-type="bibr"><sup>28</sup></xref> Stacked bar plot and line plot were visualized in R and RStudio ggplot2 package.<xref rid="hsr21209-bib-0029" ref-type="bibr"><sup>29</sup></xref> We used <b>Nextstrain</b> SARS&#x2010;CoV&#x2010;2 phylogenetic tree generation pipeline (<ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>) to reconstruct the phylogenetic tree with <b>Nextstrain</b> clade of the sequence.<li> The average CFR was calculated by the number of monthly deaths and number of confirmed COVID&#x2010;19 cases.<xref rid="hsr21209-bib-0027" ref-type="bibr"><sup>27</sup></xref>, <xref rid="hsr21209-bib-0028" ref-type="bibr"><sup>28</sup></xref> Stacked bar plot and line plot were visualized in R and RStudio ggplot2 package.<xref rid="hsr21209-bib-0029" ref-type="bibr"><sup>29</sup></xref> We used <b>Nextstrain</b> SARS&#x2010;CoV&#x2010;2 phylogenetic tree generation pipeline (<ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>) to reconstruct the phylogenetic tree with <b>Nextstrain</b> clade of the sequence.<xref rid="hsr21209-bib-0018" ref-type="bibr"><sup>18</sup></xref> MAFFT v7.470 was used to align the sequences to the Wuhan (Wuhan/Hu&#x2010;1/2019) reference genome.<xref rid="hsr21209-bib-0030" ref-type="bibr"><sup>30</sup></xref> IQ&#x2010;TREE v1.6.12<xref rid="hsr21209-bib-0031" ref-type="bibr"><sup>31</sup></xref> and TreeTime,<xref rid="hsr21209-bib-0032" ref-type="bibr"><sup>32</sup></xref> <b>Nextstrain</b>'s phylodynamic pipeline, were used to perform phylogenetic analysis.<li> <b>Nextstrain</b> clade's results contained 22 unique clades (Figure&#xA0;<xref rid="hsr21209-fig-0002" ref-type="fig"><u>2</u></xref>) whereas Pangolin's contained 107 lineages across these clades/subclades (Figure&#xA0;<xref rid="hsr21209-fig-0003" ref-type="fig"><u>3</u></xref>).<li> The maximum likelihood phylogenetic tree was constructed using <b>Nextstrain</b>'s SARS&#x2010;CoV&#x2010;2 phylogenetic tree generation pipeline and the default parameters were used for IQ&#x2010;TREE.<li>Temporal dynamics (evolution) of SARS&#x2010;CoV&#x2010;2 variants and <b>Nextstrain</b> clades in Bangladesh.<li> By comparing temporal distribution and prevalence of the <b>Nextstrain</b> clades of the SARS&#x2010;CoV&#x2010;2 in Bangladesh, we found that the Delta variant clades were circulating from October 2020 to February 2022 in Bangladesh (Figure&#xA0;<xref rid="hsr21209-fig-0002" ref-type="fig"><u>2A</u></xref>).<li> Eight <b>Nextstrain</b> clades (e.g., 21K, 21L, 21M, 22A&#x2010;22D, 22F) of Omicron variant were detected in Bangladesh from November 2021 to October 2022 (Figure&#xA0;<xref rid="hsr21209-fig-0002" ref-type="fig"><u>2A</u></xref>).<li>The SARS&#x2010;CoV&#x2010;2 <b>Nextstrain</b> clades 20A and 20B were found to be circulated in Bangladesh from April 2020 to January 2021 (Figure&#xA0;<xref rid="hsr21209-fig-0002" ref-type="fig"><u>2A</u></xref>).<li> We found that <b>Nextstrain</b> clades 19A and 19B were circulating in Bangladesh between March 2021 and May 2021 (Figure&#xA0;<xref rid="hsr21209-fig-0002" ref-type="fig"><u>2A</u></xref>) with a relatively lower prevalence rate (1.0%) (Figure&#xA0;<xref rid="hsr21209-fig-0002" ref-type="fig"><u>2B</u></xref>).<li> Among the three <b>Nextstrain</b> clades of Delta variant, 23 Pangolin lineages were found to be circulating in Bangladesh, and of them, the B.1.617.2 was detected as the mostly prevalent SARS&#x2010;CoV&#x2010;2 lineage during May 2021 to January 2022, with relatively higher prevalence in June (80.31%) and July (88.55%) followed 77.54% in September, 75.0% in October and 41.26% November of 2021 (Figure&#xA0;<xref rid="hsr21209-fig-0003" ref-type="fig"><u>3</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37058241">37058241</a> <b>(1)</b> SARS-CoV-2 BW lineage, a fast-growing Omicron variant from southeast Mexico bearing relevant escape mutations<ul><li> Filtered sequences (BA.5.6.2&#x2009;=&#x2009;1,077, BQ.1&#x2009;=&#x2009;12,118, BW.1&#x2009;=&#x2009;386, and BW.1.1&#x2009;=&#x2009;301) were then aligned with Nextclade&#x2019;s Nextalign default procedure (part of the <b>Nextstrain</b> pipeline) v2.11.0 and v.2.3.0 [<xref ref-type="bibr" rid="CR18"><u>18</u></xref>] using Wuhan-Hu-1 (NC_045512.2) as the reference genome, and BQ.1 variants were extracted from the alignment.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37066129">37066129</a> <b>(0)</b> Influenza A(H3N2) infection followed by separate COVID-19 infection<ul><li>We used the Nextclade web tool v.2.6.21 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) to reconstruct a phylogenetic tree with the four sequenced genomes and all of the 366 SARS-CoV-2 sequences from Rio Grande do Sul sampled during January 2022 and available in the GISAID (Global Initiative on Sharing All Influenza Data) database (<ext-link xlink:href="https://www.gisaid.org" ext-link-type="uri">https://www.gisaid.org</ext-link>) on April 19, 2022.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37059737">37059737</a> <b>(5)</b> Afro-TB dataset as a large scale genomic data of <bold><italic>Mycobacterium tuberuclosis</italic></bold> in Africa<ul><li>Phylogenetic analysis was performed on the VCF files of the Afro-TB dataset using <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR29"><u>29</u></xref></sup>, and comparative analysis was visualized using iTol<sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup>.<li> The mutation table can serve as a reference for comparative resistance analysis and the VCF files in AFRO-TB are ready to be used for <b>Nextstrain</b> analysis Fig.&#xA0;<xref rid="Fig4" ref-type="fig"><u>4</u></xref>.<li>Phylogenetic tree of 500 <italic>Mycobacterium tuberculosis</italic> isolats using <b>Nextstrain</b>.<li> The selected VCFs were merged using bcftools and used as input to the augur software<sup><xref ref-type="bibr" rid="CR32"><u>32</u></xref></sup> implemented in <b>Nextstrain</b>.<li> Metadata for each sample was retrieved from the dataset and used for time tracking and geographic distribution visualization in <b>Nextstrain</b> (4).<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30931475">30931475</a>) [32](Huddleston, J. <italic>et al</italic>. Augur: a bioinformatics toolkit for phylogenetic analyses of human pathogens. <italic>Journal of open source software</italic><bold>6</bold> (2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112945">37112945</a> <b>(2)</b> A Multiplexed, Tiled PCR Method for Rapid Whole-Genome Sequencing of Infectious Spleen and Kidney Necrosis Virus (ISKNV) in Tilapia<ul><li> Augur bioinformatics toolkit (version 3.0.6) [<xref rid="B35-viruses-15-00965" ref-type="bibr"><u>35</u></xref>] (github.com/<b>Nextstrain</b>/augur) was used to process the genomes.<li> Augur output was exported and visualised in auspice (github.com/<b>Nextstrain</b>/auspice) [<xref rid="B34-viruses-15-00965" ref-type="bibr"><u>34</u></xref>].<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17131065">17131065</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37059256">37059256</a> <b>(5)</b> Analysis of SARS-CoV-2 variants from patient specimens in Nevada from October 2020 to August 2021<ul><li> SARS-CoV-2 clades and lineage groups were assigned using the <b>Nextstrain</b> and Phylogenetic Assignment of Named Global Outbreak Lineages (pangolin) criteria (<xref rid="bb0125" ref-type="bibr"><u>O'Toole et al., 2021</u></xref>; <xref rid="bb0160" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>; <xref rid="bb0055" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li>In our samples we identified 9 different clades, as defined by the <b>Nextstrain</b> nomenclature.<li> The <b>Nextstrain</b> clades 20A, 20B, 20C, 20D, 20G, 20H, 20I, 21A and 21F were detected during this time.<li> Major clades from <b>Nextstrain</b> and pangolin lineages are indicated at the branch points.<li> Today both <b>Nextstrain</b> clades and pangolin lineages are used to categorize SARS-CoV-2 variants based on mutations in their nucleotide sequence.<li> [O'Toole et al., 2021](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Underwood</surname>              <given-names>A.</given-names>            </name>            <etal/>          </person-group>          <article-title>Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool</article-title>          <source>Virus Evol.</source>          <volume>7</volume>          <issue>2</issue>          <year>2021</year>          <pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>          <comment>veab064</comment>        ) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37041231">37041231</a> <b>(0)</b> Structural basis of spike RBM-specific human antibodies counteracting broad SARS-CoV-2 variants<ul><li> Variant calling and annotation were performed using the Nextclade website (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37124419">37124419</a> <b>(3)</b> Multiplexed on-yeast serological assay for immune escape screening of SARS-CoV-2 variants<ul><li>Three different receptor binding domains (RBD) constructs belonging to the most frequent spike protein S1 from SARS-CoV-2 variants of concern were designed by extracting the phylogenetic sequences of RBD variants from the <b>Nextstrain</b> platform (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="intref0055">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>) and employing codon-optimization for <italic>Saccharomyces cerevisiae</italic>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37039557">37039557</a> <b>(10)</b> Evolution of the SARS-CoV-2 Mutational Spectrum<ul><li> We tallied counts separately for each <b>Nextstrain</b> clade (<xref rid="msad085-B1" ref-type="bibr">Aksamentov et al.<li> (<italic toggle="yes">A</italic>) Principal component analysis (PCA) of mutation spectra of <b>Nextstrain</b> clades with sufficient sequences (each point is a clade).<li> 2021</xref>) to subset the mutation-annotated tree on samples from each <b>Nextstrain</b> clade, and then extract the mutations on each branch of the subsetted mutation-annotated trees.<li> 1 mutation that was a reversion to either the Wuhan-Hu-1 reference genome (Genbank <ext-link xlink:href="NC_045512.2" ext-link-type="DDBJ/EMBL/GenBank">NC_045512.2</ext-link>), or &gt;1 mutation that was a reversion to the founder for that <b>Nextstrain</b> clade as defined by <xref rid="msad085-B43" ref-type="bibr"><u>Neher (2022)</u></xref> (see <ext-link xlink:href="https://github.com/neherlab/SC2_variant_rates/blob/62c525dc4238385ec0755b40658f3007fdbfab1a/data/clade_gts.json" ext-link-type="uri">https://github.com/neherlab/SC2_variant_rates/blob/62c525dc4238385ec0755b40658f3007fdbfab1a/data/clade_gts.json</ext-link>).<li>the different influenza virus lineages at <b>nextstrain</b>.org/groups/neherlab.<li>RSV-A and RSV-B at <b>nextstrain</b>.org/rsv.<li>Enterovirus A71 at <b>nextstrain</b>.org/groups/neherlab/ev/a71.<li>Enterovirus D68 at <b>nextstrain</b>.org/enterovirus/d68 (<xref rid="msad085-B22" ref-type="bibr">Hodcroft et al.<li>Dengue virus 1&#x2013;4 at <b>nextstrain</b>.org/dengue.<li>West Nile Virus (WNV) <b>nextstrain</b>.org/WNV/NA (<xref rid="msad085-B16" ref-type="bibr">Hadfield et al.<li> [Neher (2022)](<person-group person-group-type="author"><string-name><surname>Neher</surname><given-names>RA.</given-names></string-name></person-group><year>2022</year>. <comment>Contributions of adaptation and purifying selection to SARS-CoV-2 evolution. 2022.08.22.504731. Available from:</comment><ext-link xlink:href="https://www.biorxiv.org/content/10.1101/2022.08.22.504731v1" ext-link-type="uri">https://www.biorxiv.org/content/10.1101/2022.08.22.504731v1</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37041143">37041143</a> <b>(0)</b> Comparative epidemic expansion of SARS-CoV-2 variants Delta and Omicron in the Brazilian State of Amazonas<ul><li> Genomes were evaluated for mutation calling and quality with Nextclade Web 1.13.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>)<sup><xref ref-type="bibr" rid="CR57"><u>57</u></xref></sup>.<li> [57](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open-Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37030387">37030387</a> <b>(2)</b> An improved method for determining frequency of multiple variants of SARS-CoV-2 in wastewater using qPCR assays<ul><li> According to Global Initiative on Sharing Avian Influenza Data (GISAID) data summarized by <ext-link ext-link-type="uri" xlink:href="http://nextrain.org" id="ir0010">nextrain.org</ext-link> (<xref rid="bb0080" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), the nucleotide 28,880&#x2013;28,882 (AA N:203) has one of the highest entropy levels in the SARS-CoV-2 genome, at 0.972, with nucleotides 28,883&#x2013;28,885 (AA N:204) also having high entropy level of 0.711 (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org" id="ir0015"><b>nextstrain</b>.org</ext-link>; accessed April 1, 2022).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37024253">37024253</a> <b>(2)</b> Monitoring SARS-CoV-2 incidence and seroconversion among university students and employees: a longitudinal cohort study in California, June&#x2013;August 2020<ul><li> The genomes were then processed through the <b>Nextstrain</b> Auger pipeline with other genomes from GISAID to construct a maximum likelihood tree.<li> A maximum likelihood phylogeny constructed from 357 genomes sequenced by the Innovative Genomics Institute between May and July 2020 constructed using <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37064325">37064325</a> <b>(1)</b> The impact of mutation sets in receptor-binding domain&#xA0;of SARS-CoV-2 variants on the stability of RBD&#x2013;ACE2 complex<ul><li> A large number of virus variants have been identified, and various nomenclatures are now used to classify them, including GISAID [<xref rid="B4" ref-type="bibr"><u>4</u></xref>], <b>Nextstrain</b> [<xref rid="B5" ref-type="bibr"><u>5</u></xref>] and PANGO [<xref rid="B6" ref-type="bibr"><u>6</u></xref>].<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37018390">37018390</a> <b>(4)</b> A conserved oligomerization domain in the disordered linker of coronavirus nucleocapsid proteins<ul><li>Mutation data were based on consensus sequences of SARS-CoV-2 isolates submitted to the GISAID (<xref rid="R55" ref-type="bibr"><italic toggle="yes">55</italic></xref>) and downloaded on 10 August 2022 as preprocessed metadata file by the <b>Nextstrain</b> team (<ext-link xlink:href="http://<b>nextstrain</b>.org" ext-link-type="uri"><b>nextstrain</b>.org</ext-link>) (<xref rid="R72" ref-type="bibr"><italic toggle="yes">72</italic></xref>).<li> As described previously (<xref rid="R21" ref-type="bibr"><italic toggle="yes">21</italic></xref>), only high-quality sequences based on multiple criteria evaluated in <b>Nextstrain</b> analysis were included in a set of 3.83 million genomes, and a threshold of 10 observations of any mutation was used to filter adventitious sequencing errors.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37014792">37014792</a> <b>(1)</b> Zooanthroponotic transmission of SARS-CoV-2 and host-specific viral mutations revealed by genome-wide phylogenetic analysis<ul><li> To do so, we used <b>Nextstrain</b> (<xref rid="bib15" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) to calculate a proximity matrix based on pairwise substitutions between every animal-derived sequence and all ~8 million human-derived sequences.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37143627">37143627</a> <b>(0)</b> Clinical Characteristics of SARS-CoV-2 Omicron Cases in Pune, Maharashtra, India<ul><li> Lineage identification and clade analysis were performed using Pangolin (<ext-link xlink:href="https://github.com/hCoV-2019/pangolin" ext-link-type="uri">https://github.com/hCoV-2019/pangolin</ext-link>) and Nextclade software (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>/).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36878014">36878014</a> <b>(0)</b> SARS-CoV-2 Omicron Replacement of Delta as Predominant Variant, Puerto Rico<ul><li> We conducted time-calibrated phylogenetic analyses locally using the ncov augur/auspice pipeline (<ext-link xlink:href="https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/index.html" ext-link-type="uri">https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/index.html</ext-link>) with a custom subsample from the Puerto Rico dataset in GISAID and a custom subsample of contextual genomes derived from the GISAID NextRegion-North America dataset representative of the Americas, with an emphasis on the United States and the Caribbean region (<xref rid="SD1" ref-type="supplementary-material"><u>Appendix</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36754279">36754279</a> <b>(8)</b> Rapid Lineage Assignment of Severe Acute Respiratory Syndrome Coronavirus 2 Cases through Automated Library Preparation, Sequencing, and Bioinformatic Analysis<ul><li> The resulting consensus assemblies were then analyzed with Pangolin<xref rid="bib5" ref-type="bibr"><sup>5</sup></xref> (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0055"><italic>https://github.com/cov-lineages/pangolin</italic></ext-link>, last accessed December 29, 2022) and NextClade (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="intref0060"><italic>https://github.com/<b>nextstrain</b>/nextclade</italic></ext-link>, last accessed October 12, 2022) to perform lineage and clade designations, respectively.<li>The Titan_Augur_Run workflow was utilized to perform phylogenetic and cluster analysis of the SARS-CoV-2 data sets<italic>.</italic> Titan_Augur_Run executed subcommands from the <b>NextStrain</b> Augur Toolkit (<ext-link ext-link-type="uri" xlink:href="http://github.com/<b>nextstrain</b>/ncov" id="intref0070"><italic>github.com/<b>nextstrain</b>/ncov</italic></ext-link>) through the use of a modified version of The Broad Institute's sarscov2_<b>nextstrain</b> WDL workflow (<ext-link ext-link-type="uri" xlink:href="https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/workflows/sarscov2_<b>nextstrain</b>.wdl" id="intref0075"><italic>https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/workflows/sarscov2_<b>nextstrain</b>.wdl</italic></ext-link>, last accessed December 12, 2022) to construct SARS-CoV-2 maximal likelihood and time trees as well as an Auspice-compatible JSON file for interactive visualization.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37003304">37003304</a> <b>(1)</b> The multiple roles of nsp6 in the molecular pathogenesis of SARS-CoV-2<ul><li> Phylogenetic analyses using <b>Nextstrain</b> reveal that &#x394;SGF emerged independently among these variants, suggesting a fitness advantage (<xref rid="bib29" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bib72" ref-type="bibr"><u>Ricciardi et al., 2022</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Ricciardi et al., 2022](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35551511">35551511</a>) </ul></p>
<li> [Hadfield et al., 2018](bib29) [Ricciardi et al., 2022](bib72) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37000821">37000821</a> <b>(1)</b> Genomic surveillance identifies SARS-CoV-2 transmission patterns in local university populations, Wisconsin, USA, 2020&#x2013;2022<ul><li> The existing SARS-CoV-2 sequences that were subsampled from the global dataset (<b>Nextstrain</b>, accessed on 10 October 2022) and 537 sequences from this study were illustrated in the Nextclade to discover the nature of clustering of sequences (Table S3).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37110335">37110335</a> <b>(1)</b> Molecular In-Depth on the Epidemiological Expansion of SARS-CoV-2 XBB.1.5<ul><li>The first genomic epidemiology of SARS-CoV-2 XBB.1.5 Omicron subvariant was reconstructed using <b>nextstrain</b>/ncov [<xref rid="B24-microorganisms-11-00912" ref-type="bibr"><u>24</u></xref>] on a dataset (n = 1174) including all complete genomes belonging to the GSAID Clade Omicron [<xref rid="B11-microorganisms-11-00912" ref-type="bibr"><u>11</u></xref>].<li> [24](          <article-title>GitHub Depository</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/ncov" ext-link-type="uri">https://github.com/nextstrain/ncov</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2023-01-20">(accessed on 20 January 2023)</date-in-citation>        ) [11](          <person-group person-group-type="author"><collab>GISAID</collab></person-group>          <article-title>Genomic Epidemiology of SARS-CoV-2 with Subsampling Focused Globally over the Past 6 Months</article-title>          <comment>Available online: <ext-link xlink:href="https://gisaid.org/phylodynamics/global/nextstrain/" ext-link-type="uri">https://gisaid.org/phylodynamics/global/nextstrain/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2023-02-01">(accessed on 1 February 2023)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112869">37112869</a> <b>(2)</b> The Omicron Lineages BA.1 and BA.2 (<italic toggle="yes">Betacoronavirus</italic> SARS-CoV-2) Have Repeatedly Entered Brazil through a Single Dispersal Hub<ul><li> For all datasets, background <b>NextStrain</b> sequences were included, (<b>NextStrain</b> builds 21K and 21L, as of 18 August 2022) [<xref rid="B33-viruses-15-00888" ref-type="bibr"><u>33</u></xref>].<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36989354">36989354</a> <b>(1)</b> Transmissible SARS-CoV-2 variants with resistance to clinical protease inhibitors<ul><li> The generated phylogenetic trees were visualized using Auspice.us from <b>NextStrain</b> (<xref rid="R57" ref-type="bibr"><italic toggle="yes">57</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112852">37112852</a> <b>(9)</b> Tracking the Genomic Evolution of SARS-CoV-2 for 29 Months in South Korea<ul><li> The gene sequences were analyzed according to the naming system, including the Pango lineage (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>, accessed on 1 February 2023), <b>Nextstrain</b> clade (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, accessed on 1 February 2023), and WHO label (<uri xlink:href="https://covariants.org/">https://covariants.org/</uri>, accessed on 1 February 2023).<li> In comparison with a frequency plot from <b>NextStrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time">https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time</uri>, accessed on 1 February 2023), Beta (20H), Gamma (20J), and Omicron (22A and 22B) was not found in South Korea.<li> Moreover, the Pango lineage, <b>Nextstrain</b>, and WHO nomenclature systems have been applied for SARS-CoV-2 genomes [<xref rid="B12-viruses-15-00873" ref-type="bibr"><u>12</u></xref>].<li> This variant was assigned to B.1.1.7 in the Pango lineage and 20I in the <b>NextStrain</b> clade.<li> This variant was assigned to B.1.617.2 according to the Pango lineage, and comprised 21A, 20I, and 21J in the <b>NextStrain</b> clade.<li> This variant comprised B.1.427 and B.1.429 in the Pango lineage and 21C in the <b>NextStrain</b> clade.<li> 21 K and 21 L, which were assigned as BA.1 and BA.2 in the <b>NextStrain</b> clade, respectively.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37051600">37051600</a> <b>(0)</b> A framework for real-time monitoring, analysis and adaptive sampling of viral amplicon nanopore sequencing<ul><li> The SNPs were called using nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) with the output data available in <xref rid="s10" ref-type="sec"><u>Supplementary Files S3.3, S3.4</u></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36971577">36971577</a> <b>(0)</b> Complete Genome Sequences of Monkeypox Virus from a French Clinical Sample and the Corresponding Isolated Strain, Obtained Using Nanopore Sequencing<ul><li> Both viral genomic sequences belonged to lineage B.1 and clade IIb, according to Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="B8" ref-type="bibr"><u>8</u></xref>).<li> [8](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><volume>6</volume>:<fpage>3773</fpage>. doi:<pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36972312">36972312</a> <b>(3)</b> Increased flexibility of the SARS-CoV-2 RNA-binding site causes resistance to remdesivir<ul><li> Importantly, amino acid substitutions E796G and C799F in NSP12 were observed only in the P10 with RDV virus and there have been no reports of these two mutations to date, according to <b>Nextstrain</b> [<xref rid="ppat.1011231.ref027" ref-type="bibr"><u>27</u></xref>].<li> According to <b>Nextstrain</b>, the same mutations had been detected in NSP1, NSP4 and NSP6, but only a few cases of each mutation had been reported.<li> Besides, the deletion of 15 nucleotides in S, start-loss of ORF3A and S194P in N were observed in every single virus and detected by MiSeq sequence as well and these mutations have not been reported in <b>Nextstrain</b>.<li> [27](Nextstrain. Genomic epidemiology of novel coronavirus -Global sampling. Available from: <ext-link xlink:href="https://nextstrain.org/ncov/gisaid/global" ext-link-type="uri">https://nextstrain.org/ncov/gisaid/global</ext-link>. Accessed 8 December 2022.) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=Nextstrain. Genomic epidemiology of novel coronavirus -Global sampling. Available from: <ext-link xlink:href="https://nextstrain.org/ncov/gisaid/global" ext-link-type="uri">https://nextstrain.org/ncov/gisaid/global</ext-link>. Accessed 8 December 2022.">Nextstrain. Genomic epidemiology of novel coronavirus -Global sampling. Available from: <ext-link xlink:href="https://nextstrain.org/ncov/gisaid/global" ext-link-type="uri">https://nextstrain.org/ncov/gisaid/global</ext-link>. Accessed 8 December 2022.</a> <b>(1)</b> Increased flexibility of the SARS-CoV-2 RNA-binding site causes resistance to remdesivir<ul><li>Because of the limits of Sanger sequencing, how sure are the authors that all of the mutations found in the serial passaging can occur on the same individual viral genome, like the R10/C799F/E796G mutant that was constructed? Is there information on how many separate replicates or lineages were created during the passaging? Was there only one? Were viruses from passages other than P10 sequenced? What were the controls? Were titers of the passages measured? If there was more than one lineage of viruses that was passaged, it would establish replicability and show that these mutations primarily arise in vitro, since they are not present in the <b>NextStrain</b> database.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112835">37112835</a> <b>(2)</b> Immunosuppression as a Hub for SARS-CoV-2 Mutational Drift<ul><li> Phylogenetic reconstruction was performed by the full <b>Nextstrain</b> v11.0 pipeline [<xref rid="B27-viruses-15-00855" ref-type="bibr"><u>27</u></xref>], with the global open-data sequences used for context (<uri xlink:href="https://registry.opendata.aws/ncbi-covid-19">https://registry.opendata.aws/ncbi-covid-19</uri>, accessed on 20 February 2023).<li> The time to most recent common ancestor (TMRCA) was estimated using <b>Nextstrain</b> v11.0 [<xref rid="B27-viruses-15-00855" ref-type="bibr"><u>27</u></xref>], with the default maximum likelihood model and 95% confidence intervals.<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112836">37112836</a> <b>(3)</b> Understanding Mutations in Human SARS-CoV-2 Spike Glycoprotein: A Systematic Review &amp; Meta-Analysis<ul><li>The present study was designed to perform comprehensive in silico analyses of significant mutations in spike protein across different geographic regions over time, using the <b>Nextstrain</b> web server (<uri xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</uri>; accessed on 1 March 2023) and reported literature.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36977434">36977434</a> <b>(2)</b> Cellular electrical impedance to profile SARS-CoV-2 fusion inhibitors and to assess the fusogenic potential of spike mutants<ul><li> 5</xref>A, spikes carrying the D614G mutation, as present in all variants since early 2020 (<b>Nextstrain</b> clade 20&#xA0;A and its descendants), retained a similar fusogenic efficiency as compared to the original Wuhan-Hu-1 strain S.<li> (A) Comparison of impedance signal of A549.ACE2<sup>+</sup> cells overlayed with HEK293T cells transfected with plasmid DNA coding for SARS-CoV-2 S either from Wuhan-Hu-1, carrying D614 (blue) or a mutant with G614 (red) as found in the <b>Nextstrain</b> clade 20&#xA0;A and its descendants.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37028004">37028004</a> <b>(3)</b> SARS-CoV-2 molecular testing and whole genome sequencing following RNA recovery from used BinaxNOW COVID-19 antigen self tests<ul><li> Alignments were performed using Nextalign within the <b>Nextstrain</b> v3.2.4 pipeline <xref rid="bib0015" ref-type="bibr"><u>[15]</u></xref>.<li> For ease of visualization, the Georgia dataset was downsampled to 100 sequences using <b>Nextstrain</b> subsampling scheme &#x201C;all&#x201D;.<li> Maximum likelihood phylogenetic trees were constructed using default settings of the <b>Nextstrain</b> SARS-CoV-2 Workflow with TreeTime v0.8.6 8.<li> [[15]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37035035">37035035</a> <b>(15)</b> Joint visualization of seasonal influenza serology and phylogeny to inform vaccine composition<ul><li> The result is a new component of <b>Nextstrain</b>&#x2019;s interactive phylogenetic visualization platform that we call the measurements panel.<li> Below, we describe the measurements panel and provide two case studies that demonstrate the practical value of this tool for vaccine composition decisions with the interactive visualizations available at <ext-link xlink:href="https://<b>nextstrain</b>.org/community/blab/measurements-panel/flu/seasonal/h3n2/ha" ext-link-type="uri"><b>nextstrain</b>.org/community/blab/measurements-panel/flu/seasonal/h3n2/ha</ext-link>.<li>We implemented this design as a new interactive measurements panel within <b>Nextstrain</b>&#x2019;s visualization tool, Auspice (version 2.43.0), which is freely available on GitHub (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/auspice" ext-link-type="uri">github.com/<b>nextstrain</b>/auspice</ext-link>) under the AGPLv3 license.<li> The visualization requires a minimum of two JavaScript Object Notation (JSON) files that are produced by the <b>Nextstrain</b> bioinformatics toolkit, Augur (<xref rid="B13" ref-type="bibr"><u>Huddleston et al., 2021</u></xref>).<li> For complete technical details about the data structure used by the measurements panel, see the <ext-link xlink:href="https://github.com/<b>nextstrain</b>/augur/blob/797af0d7fa3291574b76823ae22563f52b2ab33c/augur/data/schema-measurements.json" ext-link-type="uri">measurements sidecar JSON schema document</ext-link>.<li> Visualizations can be also shared with others through <ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri"><b>nextstrain</b>.org/</ext-link>.<li> Full documentation for sharing analyses through <b>Nextstrain</b> can be found at <ext-link xlink:href="https://docs.<b>nextstrain</b>.org/page/guides/share/" ext-link-type="uri">docs.<b>nextstrain</b>.org/page/guides/share/</ext-link>.<li>We evaluated the new measurements panel by constructing a <b>Nextstrain</b> analysis (<xref rid="B10" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) with previously published HA gene sequences and HI titers (<xref rid="B4" ref-type="bibr"><u>Bedford et al., 2014</u></xref>).<li> The resulting visualization for this paper has been shared <italic>via</italic> <b>Nextstrain</b> community and can be viewed at <ext-link xlink:href="https://<b>nextstrain</b>.org/community/blab/measurements-panel/flu/seasonal/h3n2/ha" ext-link-type="uri"><b>nextstrain</b>.org/community/blab/measurements-panel/flu/seasonal/h3n2/ha</ext-link>.<li> [Huddleston et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Bedford et al., 2014](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24497547">24497547</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36974160">36974160</a> <b>(2)</b> Molecular evolutionary model based on phylogenetic and mutation analysis of SARS-CoV-2 spike protein sequences from Asian countries: A phylogenomic approach<ul><li> Further, <b>Nextstrain</b> enables the visualization of the alignment with respective mutation at multiple sites aiming to understand epidemiological understanding and improve outbreak response [<xref rid="bib28" ref-type="bibr"><u>28</u></xref>].<li> <b>Nextstrain</b> visualized and tracked the mutational sites along multiple sequencing genome sequences which assisted in the generation of phylogenomic relationships among strains.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36946746">36946746</a> <b>(2)</b> Ancestral SARS-CoV-2-Driven Antibody Repertoire Diversity in an Unvaccinated Individual Correlates with Expanded Neutralization Breadth<ul><li>Currently, the COVID-19 pandemic is primarily driven by the Omicron lineage variants globally (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36992509">36992509</a> <b>(2)</b> Evaluation of SARS-CoV-2 ORF7a Deletions from COVID-19-Positive Individuals and Its Impact on Virus Spread in Cell Culture<ul><li> Lineage and genome variants were accessed using the web tool Nextclade (c) 2020&#x2013;2022 version 1.14.0 (<b>Nextstrain</b> developers) [<xref rid="B32-viruses-15-00801" ref-type="bibr"><u>32</u></xref>] through fasta files generated from NGS sequencing.<li>Nevertheless, data from the assembled virus genomes submitted to Nextclade on the <b>Nextstrain</b> website showed the same variants indicating that the viruses infecting all the family members belong to the B.1.1.33 lineage.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36993525">36993525</a> <b>(4)</b> Gain without pain: Adaptation and increased virulence of Zika virus in vertebrate host without fitness cost in mosquito vector<ul><li> Importantly, NS4A E19G has also been identified in a naturally isolated ZIKV genome from a pregnant woman in Honduras in 2016, reported in <b>Nextstrain</b> and has not been characterized prior to this study.<li> <b>Nextstrain</b>: <ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36993216">36993216</a> <b>(0)</b> Reduced control of SARS-CoV-2 infection is associated with lower mucosal antibody responses in pregnant women<ul><li> Clades were determined using Nextclade beta v 1.13.2 (<ext-link xlink:href="http://clades.<b>nextstrain</b>.org" ext-link-type="uri">clades.<b>nextstrain</b>.org</ext-link>, Last accessed March 30, 2022), and lineages were determined with Pangolin COVID-19 lineage Assigner [<xref rid="R19" ref-type="bibr"><u>19</u></xref>].<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36932280">36932280</a> <b>(1)</b> Subregional origins of emerging SARS-CoV-2 variants during the second pandemic wave in C&#xF4;te d&#x2019;Ivoire<ul><li> We next excluded sequences identified as being of low quality by NextClade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>), those with missing sampling dates, those with&#x2009;&lt;&#x2009;90% coverage, those with&#x2009;&gt;&#x2009;40 SNPs, those with&#x2009;&gt;&#x2009;10 ambiguous base-calls per genome, and those with clustered SNPs.<li>We retrieved respective sequence data sets compiled by Emma Hodcroft and Richard Neher (Neherlab) for Alpha (4886 sequences) and Eta (4965 sequences) variants from <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR13"><u>13</u></xref>] on September 30th 2021, and merged each data set with high quality sequences generated from C&#xF4;te d&#x2019;Ivoire for variant Alpha (33 sequences) and Eta (45 sequences), respectively.<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36931337">36931337</a> <b>(1)</b> At the crossroads of epidemiology and biology: Bridging the gap between SARS-CoV-2 viral strain properties and epidemic wave characteristics<ul><li> Source: Gisaid/<b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36868218">36868218</a> <b>(5)</b> A pseudovirus system enables deep mutational scanning of the full SARS-CoV-2 spike<ul><li>The approach for calculating the enrichment of observed versus expected mutations is inspired by the strategy used in Neher<xref rid="bib75" ref-type="bibr"><sup>75</sup></xref> Briefly, we downloaded the pre-built UShER tree of all publicly available SARS-CoV-2 sequences<sup>25</sup> as of Sept-26-2022from <ext-link ext-link-type="uri" xlink:href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/09/26/public-2022-09-26.all.masked.nextclade.pangolin.pb.gz" id="intref0230">http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/09/26/public-2022-09-26.all.masked.nextclade.pangolin.pb.gz</ext-link>We then used matUtils25 to separate the tree by the pre-annotated <b>Nextstrain</b> clades, and extract the number of unique occurrences of each nucleotide mutation along the tree branches for each clade.<li> Overall, this process provided a list of the counts of nucleotide mutations for each <b>Nextstrain</b> clade after filtering branches and mutations/sites expected to be affected by various possible errors.<li>We then calculated the relative rates of each type of the 12 possible nucleotide mutations at four-fold synonymous third codon positions by counting the mutation at such positions in each <b>Nextstrain</b> clade, and then normalizing the fraction of these positions that had each nucleotide identity at each site, retaining only <b>Nextstrain</b> clades with at least 5000 such mutation counts.<li> Next, we aggregated the expected and observed counts across all <b>Nextstrain</b> clades, and also across all nucleotide mutations that encoded for the same amino-acid mutation to get expected and observed counts for amino-acid mutations, and only retained amino-acid mutations with at least 20 expected counts.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36933724">36933724</a> <b>(6)</b> Wastewater surveillance uncovers regional diversity and dynamics of SARS-CoV-2 variants across nine states in the USA<ul><li> The PCoA plot shows lineages from each VOC lineage cluster according to their respective clades in the global phylogenetic tree as described by <b>Nextstrain</b> (<xref rid="bb0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and highlights the observed temporal shift in the viral population.<li> The VOC lineages have been labeled based on the name used in the <b>Nextstrain</b> phylogenetic tree.<li> The first columns group those variants per VOC lineage which are color coded by the specific <b>Nextstrain</b> clade of the associated lineages.<li> The first columns group those variants per VOC lineage which are color-coded by the specific <b>Nextstrain</b> clade of the associated lineages.<li> The first columns group those variants per VOC lineage which are color-coded by the specific <b>Nextstrain</b> clade of the associated lineages.<li> The first columns group those variants per VOC lineage which are color-coded by the specific <b>Nextstrain</b> clade of the associated lineages.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36989935">36989935</a> <b>(1)</b> Genomic characterization and molecular evolution of SARS-CoV-2 in Rio Grande do Sul State, Brazil<ul><li> The strains were identified using the dynamic nomenclature implemented in Pangolin (Rambaut, 2020) (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0035">https://github.com/cov-lineages/pangolin</ext-link>) and global clades and mutations using <b>Nextstrain</b> from Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0040">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36968018">36968018</a> <b>(1)</b> Bioinformatic approaches to draft the viral genome sequence of Canary Islands cases related to the multicountry mpox virus 2022-outbreak<ul><li> A phylogenetic analysis was performed using both IQ-TREE v.2.2.0.3 <xref rid="bib58" ref-type="bibr"><u>[58]</u></xref> with the K3Pu+F+I model as the best-predicted model and default parameters and using a local instance of <b>Nextstrain</b> <xref rid="bib59" ref-type="bibr"><u>[59]</u></xref>.<li> [[58]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [[59]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36989761">36989761</a> <b>(1)</b> A systematic review on impact of SARS-CoV-2 infection<ul><li> At present, SARS-CoV-2 variants lineage nomenclatures have been procured by open access databases, that includes, Global Initiative on Sharing All Influenza Data (GISAID) (<xref rid="bib103" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), <b>Nextstrain</b>, and Pango (<xref rid="bib100" ref-type="bibr"><u>P&#xE9;rez-Abeledo and Moreno, 2021</u></xref>).<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [P&#xE9;rez-Abeledo and Moreno, 2021](          <person-group person-group-type="author">            <name>              <surname>P&#xE9;rez-Abeledo</surname>              <given-names>Marta</given-names>            </name>            <name>              <surname>Moreno</surname>              <given-names>J.C.Sanz</given-names>            </name>          </person-group>          <article-title>SARS-CoV-2 variants, a still unfinished story</article-title>          <source>Vacunas</source>          <volume>22</volume>          <issue>3</issue>          <year>2021</year>          <fpage>167</fpage>          <lpage>173</lpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36998684">36998684</a> <b>(1)</b> Translating virome analyses to support biosecurity, on-farm management, and crop breeding<ul><li> Open access software pipelines (e.g., Nextflow, Python, R and Linux) and repositories of viral sequence data (e.g., <b>NextStrain</b>) should be promoted.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36916974">36916974</a> <b>(0)</b> Transmission of SARS-CoV-2 Omicron Variant under a Dynamic Clearance Strategy in Shandong, China<ul><li> The genetic clade for each SARS-CoV-2 consensus sequence was determined using Nextclade v1.11.0 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37069849">37069849</a> <b>(5)</b> Seasonal effects decouple SARS-CoV-2 haplotypes worldwide<ul><li> These genomic changes can be traced by conducting a phylogenomic analysis of genomic sequences sampled throughout the disease timeline.<xref rid="f1" ref-type="fig"><u>Figure 1A</u></xref> shows a timetree reconstructed using the<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri"><b>Nextstrain</b></ext-link> pipeline.<li> As of July 2022, 26 <b>Nextstrain</b> clades defined by a year-letter nomenclature (beginning with clade 19A) and differing by at least two mutations from their parent major clades (<xref rid="ref27" ref-type="bibr">Hodcroft<italic toggle="yes">et al.<li> The timetree of 2,906 genomes randomly sampled between December 2019 and July 29, 2022 was obtained from<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri"><b>Nextstrain</b></ext-link>.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36914815">36914815</a> <b>(1)</b> Benchmarking machine learning robustness in Covid-19 genome sequence classification<ul><li> On the other hand, this amount of data exceeds the capacity of the more traditional phylogenetic methods, such as <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR9"><u>9</u></xref></sup> or even the more recent IQTREE2<sup><xref ref-type="bibr" rid="CR10"><u>10</u></xref></sup> by several orders of magnitude&#x2014;a Big Data challenge.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36899062">36899062</a> <b>(1)</b> Immune correlates analysis of a phase 3 trial of the AZD1222 (ChAdOx1 nCoV-19) vaccine<ul><li> Based on <b>Nextstrain</b>.org<sup><xref ref-type="bibr" rid="CR38"><u>38</u></xref></sup>, we estimated the prevalence of circulating lineages during the periods of participant follow-up for correlates assessment (starting 7&#x2009;days post D57) in the U.S.<li> [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36897843">36897843</a> <b>(1)</b> The clinical characteristics of pediatric patients infected by SARS-CoV-2 Omicron variant and whole viral genome sequencing analysis<ul><li><b>Nextstrain</b> clade</th>                  <th align="center" rowspan="1" colspan="1">PANGO Sub-lineage</th>                </tr>              </thead>              <tbody>                <tr>                  <td align="center" rowspan="1" colspan="1">1bce1b5b.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36961814">36961814</a> <b>(2)</b> Identification of the viral and cellular microRNA interactomes during SARS-CoV-2 infection<ul><li>Measure of entropy (<xref rid="mmc1" ref-type="supplementary-material"><u>Figure&#xA0;S4</u></xref>A) was analyzed at <b>nextstrain</b>.org.<li> Diversity was plotted for the miRNA binding regions using per codon Shannon entropy data acquired from <b>nextstrain</b>.org/ncov/gisaid/global by downloading data and selecting &#x201C;diversity&#x201D;.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36970680">36970680</a> <b>(1)</b> Variation in synonymous evolutionary rates in the SARS-CoV-2 genome<ul><li> For example, <b>Nextstrain</b> uses the number of mutations at each site or the entropy of change to represent the site's degree of variability based on phylogenetic trees and some viral infection experiments assessing specific mutations (Hadfield et al., <xref rid="B20" ref-type="bibr"><u>2018</u></xref>; Zhang L.<li> [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36980953">36980953</a> <b>(2)</b> High Throughput SARS-CoV-2 Genome Sequencing from 384 Respiratory Samples Using the Illumina COVIDSeq Protocol<ul><li> <b>Nextstrain</b> clades and Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) were identified with the Nextclade web application [<xref rid="B26-genes-14-00681" ref-type="bibr"><u>26</u></xref>,<xref rid="B27-genes-14-00681" ref-type="bibr"><u>27</u></xref>,<xref rid="B28-genes-14-00681" ref-type="bibr"><u>28</u></xref>] and Pangolin tool [<xref rid="B29-genes-14-00681" ref-type="bibr"><u>29</u></xref>,<xref rid="B30-genes-14-00681" ref-type="bibr"><u>30</u></xref>], respectively.<li>Of 376 samples processed for sequencing with the modified Illumina COVIDSeq protocol, 356 (94.7%) yielded interpretable genome sequences as defined by Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 2 September 2022).<li> Otherwise, both protocols assigned the variants to the same clades based on the nomenclature defined by <b>Nextstrain</b>, suggesting that a majority (i.e., 279 of 356 [78.4%] genomes) of the samples belonged to clade 21K while 65 (18.2%) belonged to clade 21J.<li> [26](          <article-title>Nextclade Web 2.12.0, Nextclade CLI 2.12.0</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-09-12">(accessed on 12 September 2022)</date-in-citation>        ) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [28](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [29](          <article-title>Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN)</article-title>          <comment>Available online: <ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-09-12">(accessed on 12 September 2022)</date-in-citation>        ) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36906180">36906180</a> <b>(0)</b> Investigation of hospital discharge cases and SARS-CoV-2 introduction into Lothian care homes<ul><li> Positive control material is designated PANGO lineage B.1 and contains seven mutations (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="intref0030">https://clades.<b>nextstrain</b>.org</ext-link>) [<xref rid="bib19" ref-type="bibr"><u>19</u></xref>].<li> [19](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J&#xA0;Open Source Softw</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36892291">36892291</a> <b>(0)</b> SARS-CoV-2 Exposure in Norway Rats (<named-content content-type="genus-species">Rattus norvegicus</named-content>) from New York City<ul><li> Analyses were performed and visualized using <ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>.<li> Nucleotide sequences were aligned using MAFFT v7.471, and the mutations were analyzed using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36992424">36992424</a> <b>(5)</b> How SARS-CoV-2 Big Data Are Challenging Viral Taxonomy Rules<ul><li>The <b>NextStrain</b> naming system stems from the WHO VOC/VOI definition [<xref rid="B6-viruses-15-00715" ref-type="bibr"><u>6</u></xref>] but is definitely more transparent and objective (&gt;20% global frequency for &#x2265;2 months, &gt;30% regional frequency for &#x2265;2 months; &gt;0.05 per day growth in frequency where it is circulating and has reached &gt;5% regional frequency).<li> Nevertheless, <b>NextStrain</b> still suffers from delays (frequencies occurring much after growth advantages are clear), arbitrary cutoffs, and the vague definition of &#x201C;region&#x201D; (again, is not China a region?).<li> Genetic distance, as applied by <b>NextStrain</b> for SARS-CoV-2 and many other viruses (<uri xlink:href="https://<b>nextstrain</b>.org/pathogens">https://<b>nextstrain</b>.org/pathogens</uri>, accessed on 14 September 2022), is apparently a more reliable parameter, despite currently varying across genera and based on arbitrary cutoffs (e.g., 30% for Ebolaviruses, 89% for Gemyniviruses).<li> [6](          <article-title>SARS-CoV-2 Clade Naming Strategy for 2022</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/blog/2022-04-29-SARS-CoV-2-clade-naming-2022" ext-link-type="uri">https://nextstrain.org/blog/2022-04-29-SARS-CoV-2-clade-naming-2022</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2023-01-23">(accessed on 23 January 2023)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35379194">35379194</a> <b>(8)</b> Genomic epidemiology of the Los Angeles COVID-19 outbreak and the early history of the B.1.43 strain in the USA<ul><li> We performed a phylogenetic analysis with <b>NextStrain</b> using these 286 LA County genomes together with 3,809 genomes sampled from across the world [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>].<li> We filtered these genomes using the <b>Nextstrain</b> pipeline [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>].<li>We performed phylogenetic analysis of all LA County genomes using <b>Nextstrain</b> (v1.16.7) [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>].<li> To achieve this, we utilized proximity sampling and allowed 20 samples per country, year, and month combination, and 10 contextual samples per country and year combination (see <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.github.io/ncov/">https://<b>nextstrain</b>.github.io/ncov/</ext-link> for a more detailed description of how sampling works in <b>Nextstrain</b>).<li> These genomes were run through the entire <b>Nextstrain</b> pipeline, and we explored the results and exported the trees from the Auspice web application (v2.16.0).<li> <bold>Figure S5.</bold> Phylogenetic tree of 247 B.1.43 and 987 other SARS-CoV-2 genomes from around the world built using <b>Nextstrain</b>.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35444655">35444655</a> <b>(0)</b> Genetic Surveillance of Five SARS-CoV-2 Clinical Samples in Henan Province Using Nanopore Sequencing<ul><li> The lineage of genome was analyzed using the online website pangolin (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>), and Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>) was used for tracking the clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35390503">35390503</a> <b>(7)</b> Tracking of SARS-CoV-2 Alpha variant (B.1.1.7) in Palestine<ul><li> <b>Nextstrain</b>, a website for real-time tracking of pathogen evolution, was used to identify the Palestinian SARS-CoV-2 isolates; (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/asia?f_country=Palestine" id="ir0035">https://<b>nextstrain</b>.org/ncov/asia?f_country=Palestine</ext-link>) (<xref rid="bb0090" ref-type="bibr"><b>Nextstrain</b>, 2021</xref>).<li> The graph is compiled based on data and graphs from <b>Nextstrain</b> online platform (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/asia?f_country=Palestine" id="ir0005">https://<b>nextstrain</b>.org/ncov/gisaid/asia?f_country=Palestine</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35371608">35371608</a> <b>(2)</b> Clinical Characteristics, Transmissibility, Pathogenicity, Susceptible Populations, and Re-infectivity of Prominent COVID-19 Variants<ul><li> There are three naming systems for tracking and scientifically reporting the genetic evolution of new coronaviruses: GISAID, <b>Nextstrain</b>, and Pango.<li><b>Nextstrain</b> clade</th>              <th valign="middle" align="center" rowspan="1" colspan="1">Earliest documented samples<break/>(Time and Place)</th>              <th valign="middle" align="center" rowspan="1" colspan="1">Date of designation in WHO</th>              <th valign="middle" align="center" rowspan="1" colspan="1">Mutations</th>            </tr>          </thead>          <tbody>            <tr>              <td colspan="7" valign="middle" align="center" rowspan="1">Variants of Concern (VOCs).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35318922">35318922</a> <b>(3)</b> SARS-CoV-2 Outbreak among Malayan Tigers and Humans, Tennessee, USA, 2020<ul><li>Most viral sequences clustered into <b>NextStrain</b> Clade 20G and Pangolin lineage B.1.2, which correspond to the predominant clades observed for human specimens from Tennessee during the time of the outbreak at the zoo.<li> OK170070) also clustered in SARS-CoV-2 clade 20G by <b>Nextstrain</b> tree (<xref rid="F2" ref-type="fig"><u>Figure 2</u></xref>).<li> Phylogenetic relationships were inferred through approximate maximum-likelihood analyses implemented in TreeTime (<xref rid="R13" ref-type="bibr"><italic>13</italic></xref>) by using the <b>NextStrain</b> pipeline (<xref rid="R14" ref-type="bibr"><italic>14</italic></xref>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35317702">35317702</a> <b>(2)</b> Adaptive evolution of West Nile virus facilitated increased transmissibility and prevalence in New York State<ul><li> Recently, there has been identification of the NY10 genotype on distinctly different, and geographically distant backgrounds (<b>nextstrain</b>.org/wnv/na).<li> Importantly, while our previous studies only identified the NY10 genotype in NYS, it is now present throughout the continental US, suggesting that it may have a broad adaptive advantage that could drive similar increases in WNV activity in other regions (<b>nextstrain</b>.org/wnv/na).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35367360">35367360</a> <b>(0)</b> Identification, propagation and molecular characterization of SARS-CoV-2 delta variant isolated from Egyptian COVID-19 patients<ul><li>Further analysis was performed to detect the genetic mutation using Nextclade Tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0010">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> Constructed by open access tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0005">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35361789">35361789</a> <b>(1)</b> Human seasonal influenza under COVID-19 and the potential consequences of influenza lineage elimination<ul><li> for an interactive visualization of current seasonal influenza circulation, we refer the reader to the <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR67"><u>67</u></xref></sup> platform.<li> [67](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35361754">35361754</a> <b>(1)</b> Protein-based SARS-CoV-2 spike vaccine booster increases cross-neutralization against SARS-CoV-2 variants of concern in non-human primates<ul><li>Description of the reporter virus particles (RVPs) used in the pseudovirus neutralization assay.</p></caption><table frame="hsides" rules="groups"><thead><tr><th>Pango lineage</th><th>WHO name</th><th>Strain (<b>Nextstrain</b>)</th><th>Sequence source</th><th>Catalog number</th><th>Lot</th><th>Mutations relative to Wuhan D614</th></tr></thead><tbody><tr><td>A.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35358277">35358277</a> <b>(6)</b> Profiling of the most reliable mutations from sequenced SARS-CoV-2 genomes scattered in Uzbekistan<ul><li>Global phylogenetic tree was produced using local COVID-19 patient-derived sequences and the reference genome NC_045512.2, using <ext-link xlink:href="<b>nextstrain</b>.org" ext-link-type="uri"><b>nextstrain</b>.org</ext-link> website (<xref rid="pone.0266417.g003" ref-type="fig"><u>Fig 3</u></xref>).<li> These variants also have grouped with African and Near East variants in the global phylogenetic tree provided by <ext-link xlink:href="<b>nextstrain</b>.org" ext-link-type="uri"><b>nextstrain</b>.org</ext-link>.<li>Global phylogenetic tree provided by <ext-link xlink:href="<b>nextstrain</b>.org" ext-link-type="uri"><b>nextstrain</b>.org</ext-link> including COVID-19 samples from Uzbekistan.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35352979">35352979</a> <b>(1)</b> The Impact of Evolving SARS-CoV-2 Mutations and Variants on COVID-19 Vaccines<ul><li>As of January 25, 2022, 11 global clades of SARS-CoV-2 according to GISAID nomenclature (<xref rid="B34" ref-type="bibr"><u>34</u></xref>) and 25 according to <b>NextStrain</b> nomenclature (<xref rid="B35" ref-type="bibr"><u>35</u></xref>) have been identified.<li> [34](<collab>GISAID</collab>. <year>2021</year>. <ext-link xlink:href="https://www.gisaid.org/phylodynamics/global/nextstrain/" ext-link-type="uri">https://www.gisaid.org/phylodynamics/global/nextstrain/</ext-link>. <date-in-citation content-type="access-date">Accessed 14 July, 2021</date-in-citation>.) [35](<collab>Nextclade</collab>. <year>2021</year>. <article-title>Nextstrain/ncov</article-title>. <ext-link xlink:href="https://github.com/nextstrain/ncov/blob/master/defaults/clades.tsv" ext-link-type="uri">https://github.com/nextstrain/ncov/blob/master/defaults/clades.tsv</ext-link>. <date-in-citation content-type="access-date">Accessed 14 July, 2021</date-in-citation>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35378751">35378751</a> <b>(2)</b> SARS-CoV-2 and other respiratory pathogens are detected in continuous air samples from congregate settings<ul><li> Lineage/clade assignments were also confirmed using NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>" ext-link-type="uri">https://clades.<b>nextstrain</b></ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35352942">35352942</a> <b>(1)</b> Update on SARS-CoV-2 Omicron Variant of Concern and Its Peculiar Mutational Profile<ul><li>On the 26 November 2021, the World Health Organization (WHO) has officially disclosed the emergence of a novel SARS-CoV-2 variant of concern (VOC) defined as Omicron (Pango Lineage B.1.1.529 and <b>Nextstrain</b> Clade 21K), only 48 h after its detection.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35352028">35352028</a> <b>(1)</b> Tackling the emerging threat of antifungal resistance to human health<ul><li> Phylodynamic inference and building interactive online portals (such as <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR131"><u>131</u></xref></sup> or Microreact<sup><xref ref-type="bibr" rid="CR132"><u>132</u></xref></sup>) that are available to researchers and clinicians alike enable tracing of transmission events.<li> [131](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [132](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28348833">28348833</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35366482">35366482</a> <b>(2)</b> Strain wars 2: Binding constants, enthalpies, entropies, Gibbs energies and rates of binding of SARS-CoV-2 variants<ul><li>Various SARS-CoV-2 strains were observed epidemiologically to fight each other for survival (<xref rid="bib38" ref-type="bibr"><b>Nextstrain</b>, 2021</xref>; <xref rid="bib15" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> This led to appearance of multiple strains with a single or multiple mutations, mostly on the spike protein gene (<xref rid="bib38" ref-type="bibr"><b>Nextstrain</b>, 2021</xref>; <xref rid="bib15" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35458441">35458441</a> <b>(12)</b> Comparative Analysis of SARS-CoV-2 Variants of Concern, Including Omicron, Highlights Their Common and Distinctive Amino Acid Substitution Patterns, Especially at the Spike ORF<ul><li>Mutations in SARS-CoV-2 are used to construct phylogenetic lineages and clades according to three nomenclature systems, PANGO, Global Initiative on Sharing All Influenza Data (GISAID), and <b>Nextstrain</b> [<xref rid="B4-viruses-14-00707" ref-type="bibr"><u>4</u></xref>,<xref rid="B7-viruses-14-00707" ref-type="bibr"><u>7</u></xref>,<xref rid="B11-viruses-14-00707" ref-type="bibr"><u>11</u></xref>,<xref rid="B12-viruses-14-00707" ref-type="bibr"><u>12</u></xref>,<xref rid="B13-viruses-14-00707" ref-type="bibr"><u>13</u></xref>].<li> As an extra quality control, we investigated whether the mutations detected by our analyses were also observed in the sampled GISAID sequences (of the corresponding variant lineage) at the <b>Nextstrain</b> webserver [<xref rid="B4-viruses-14-00707" ref-type="bibr"><u>4</u></xref>,<xref rid="B7-viruses-14-00707" ref-type="bibr"><u>7</u></xref>].<li> 97%) of HF-AAS in a certain lineage were also observed in the <b>Nextstrain</b>/GISAID data.<li> 51%) of the lower-frequency mutations (5&#x2013;49% frequency in a given lineage) were also observed in the <b>Nextstrain</b>/GISAID webserver dataset.<li> This may be attributed to the much lower number of representative sequences (for a particular lineage) used in the <b>Nextstrain</b>/GISAID webserver dataset.<li>We manually investigated the frequency trends of these 69 mutations for each lineage separately in <b>Nextstrain</b>/GISAID, but we did not observe any notable decreasing trend over time.<li> Of note, the <b>Nextstrain</b>/GISAID webserver displays a sample of ~3400 total SARS-CoV-2 sequences; thus, frequency trends for certain lineages are based on a small number of samples.<li> This analysis was performed on the <b>Nextstrain</b>/GISAID webserver (8 January 2022).<li> In order to determine if such events were recurrent (homoplasy/convergent evolution) mutations, or were inherited from a common ancestor (of the six VoC/VoI lineages), we investigated the distribution of each of these mutations in the <b>Nextstrain</b> phylogenetic tree that was constructed from more than 3400 representative sequences from various clades/lineages (see <xref rid="app1-viruses-14-00707" ref-type="app"><u>Supplementary File S2</u></xref>).<li>This study compared the amino acid substitution (AAS) patterns of five VoC lineages (Alpha, Beta, Gamma, Delta, Omicron) and one VoI lineage (Lambda) that are relatively close to Omicron (based on the <b>Nextstrain</b> phylogenetic trees).<li> As a quality control, the mutations that we detected (especially the high frequency ones, &#x2265;50%) were further validated by <b>Nextstrain</b>/GISAID.<li> Manual inspection of the <b>Nextstrain</b>/GISAID phylogenetic tree shows that most of these successful lineages are deep branches, close to the root.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34108654">34108654</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35341301">35341301</a> <b>(1)</b> Impact of SARS-CoV-2 Variants on the Analytical Sensitivity of rRT-PCR Assays<ul><li>For real-time investigation and tracking of SARS-CoV-2 evolution, three nomenclature systems have been established by the Global Initiative of Sharing All Influenza Data (GISAID) (<xref rid="B2" ref-type="bibr"><u>2</u></xref>), <b>Nextstrain</b> (<xref rid="B3" ref-type="bibr"><u>3</u></xref>), and Pango (<xref rid="B4" ref-type="bibr"><u>4</u></xref>).<li> [2](<collab>Global Initiative of Sharing All Influenza Data</collab>. <year>2021</year>. <article-title>Clade and lineage nomenclature aids in genomic epidemiology studies of active hCoV-19 viruses</article-title>. <ext-link xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>. <date-in-citation content-type="access-date">Accessed 1 March 2022</date-in-citation>.) [3](<person-group person-group-type="author"><string-name><surname>Bedford</surname><given-names>T</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Updated Nextstrain SARS-CoV-2 clade naming strategy</article-title>. <source>Nextstrain.</source><ext-link xlink:href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming" ext-link-type="uri">https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>. <date-in-citation content-type="access-date">Accessed 1 March 2022</date-in-citation>.) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35378755">35378755</a> <b>(1)</b> Genomic epidemiology reveals the impact of national and international restrictions measures on the SARS-CoV-2 epidemic in Brazil<ul><li> A unique set of external references (n=7,992) were obtained by including all the non-Brazilian and non-Paraguayan sequences from the global and South American <b>NextStrain</b> builds (date of access; June 30<sup>th</sup>, 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35347305">35347305</a> <b>(3)</b> Global landscape of SARS-CoV-2 genomic surveillance and data sharing<ul><li>There are several web-based platforms that have provided up-to-date visualizations of SARS-CoV-2 genomic data and geographic distribution of variants, such as <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR38"><u>38</u></xref></sup> and outbreak.info<sup><xref ref-type="bibr" rid="CR39"><u>39</u></xref></sup>.<li> For example, the currently hosted regional <b>Nextstrain</b> pages for SARS-CoV-2 are, by necessity, subsampled from the 5 million sequences in GISAID to around 3,000&#x2013;4,000 sequences<sup><xref ref-type="bibr" rid="CR38"><u>38</u></xref></sup>.<li> in Pangolin (v3.1.16) and <b>Nextstrain</b> (Web 1.7.4) nomenclature systems by conveniently selecting about 9,000 sequences sampled in the midpoint of December 2020 and March, June and September 2021; we found the consistent degree reached a level of 100.0% and 99.9%, respectively (Supplementary Table <xref rid="MOESM1" ref-type="media"><u>8</u></xref>).<li> [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [39](Mullen, J. L. et al. outbreak.info: a standardized, open-source database of COVID-19 resources and epidemiology data. <ext-link ext-link-type="uri" xlink:href="https://outbreak.info/">https://outbreak.info/</ext-link> (2021).) [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35344550">35344550</a> <b>(4)</b> Phylogenetic analysis of 17271 Indian SARS-CoV-2 genomes to identify temporal and spatial hotspot mutations<ul><li>Motivated by the aforementioned studies, in this work we have performed multiple sequence alignment (MSA) of 17271 Indian SARS-CoV-2 genomes using multiple alignment using fast fourier transform (MAFFT) [<xref rid="pone.0265579.ref018" ref-type="bibr"><u>18</u></xref>] followed by their phylogenetic analysis using <b>Nextstrain</b> [<xref rid="pone.0265579.ref019" ref-type="bibr"><u>19</u></xref>] to eventually identify hotspot mutations both month-wise (temporal) and state-wise (spatial).<li> Initially, multiple sequence alignment (MSA) of 17271 Indian SARS-CoV-2 genomes is performed using MAFFT which is followed by their phylogenetic analysis using <b>Nextstrain</b>, thereby leading to the identification of mutation points as SNPs.<li> On the other hand, by taking the advantage of <b>Nextstrain</b>, in this work the evolution and geographic distribution of SARS-CoV-2 genomes are visualised by creating the metadata in our High Performance Computing environment.<li> Thereafter, their phylogenetic analysis using <b>Nextstrain</b> reveals 5 virus clades viz.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Phylogenetic analysis of 17271 Indian SARS-CoV-2 genomes to identify temporal and spatial hotspot mutations<ul><li>You have performed multiple sequence alignments of 5734 sequences of SARS-CoV-2 using MAFFT and phylogenetic analysis using <b>Nextstrain</b>.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> Phylogenetic analysis of 17271 Indian SARS-CoV-2 genomes to identify temporal and spatial hotspot mutations<ul><li>You have performed multiple sequence alignments of 5734 sequences of SARS-CoV-2 using MAFFT and phylogenetic analysis using <b>Nextstrain</b>.<li>Please note that hotspot mutations are characterized by PROVEAN, Polyphen-2 and I-mutant 2.0 after their locations have been identified by <b>Nextstrain</b>.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35348639">35348639</a> <b>(2)</b> hCoronavirusesDB: an integrated bioinformatics resource for human coronaviruses<ul><li> GISAID (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>) and Next Strain (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>) provide valuable gene and protein sequences for SARS-CoV-2 (<xref rid="R11" ref-type="bibr"><u>11</u></xref>, <xref rid="R12" ref-type="bibr"><u>12</u></xref>).<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [12](<person-group person-group-type="author"><string-name><surname>Shu</surname><given-names>Y.</given-names></string-name> and <string-name><surname>McCauley</surname><given-names>J.</given-names></string-name></person-group> (<year>2017</year>) <article-title>GISAID: global initiative on sharing all influenza data&#x2013;from vision to reality</article-title>. <source><italic toggle="yes">Eurosurveillance</italic></source>, <volume>22</volume>, <page-range>30494</page-range>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35368908">35368908</a> <b>(0)</b> Phylogenetic-based inference reveals distinct transmission dynamics of SARS-CoV-2 lineages Gamma and P.2 in Brazil<ul><li><hr/></td></tr><tr><td>Viralflow v.0.0.6</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/dezordi/ViralFlow" id="intref0035">https://github.com/dezordi/ViralFlow</ext-link></td><td><xref rid="bib6" ref-type="bibr"><u>Dezordi et&#xA0;al., 2022</u></xref></td></tr><tr><td>Geneious v10.2.6</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com/" id="intref0040">https://www.geneious.com/</ext-link></td><td>N/A</td></tr><tr><td>Pangolin</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0045">https://github.com/cov-lineages/pangolin</ext-link></td><td><xref rid="bib26" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td></tr><tr><td>Nextclade</td><td><ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0050">https://clades.<b>nextstrain</b>.org/</ext-link>.<li><hr/></td></tr><tr><td>Viralflow v.0.0.6</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/dezordi/ViralFlow" id="intref0035">https://github.com/dezordi/ViralFlow</ext-link></td><td><xref rid="bib6" ref-type="bibr"><u>Dezordi et&#xA0;al., 2022</u></xref></td></tr><tr><td>Geneious v10.2.6</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com/" id="intref0040">https://www.geneious.com/</ext-link></td><td>N/A</td></tr><tr><td>Pangolin</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0045">https://github.com/cov-lineages/pangolin</ext-link></td><td><xref rid="bib26" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td></tr><tr><td>Nextclade</td><td><ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0050">https://clades.<b>nextstrain</b>.org/</ext-link></td><td><xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref></td></tr><tr><td>MAFFT v7.467</td><td><ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/" id="intref0055">https://mafft.cbrc.jp/alignment/software/</ext-link></td><td><xref rid="bib14" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref></td></tr><tr><td>IQ-TREE v2.1.2</td><td><ext-link ext-link-type="uri" xlink:href="http://www.iqtree.org/" id="intref0060">http://www.iqtree.org</ext-link></td><td><xref rid="bib22" ref-type="bibr"><u>Nguyen et&#xA0;al., 2015</u></xref></td></tr><tr><td>TempEst</td><td><ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/tempest/" id="intref0065">http://tree.bio.ed.ac.uk/software/tempest/</ext-link></td><td><xref rid="bib27" ref-type="bibr"><u>Rambaut et&#xA0;al., 2016</u></xref></td></tr><tr><td>BEAST v1.10</td><td><ext-link ext-link-type="uri" xlink:href="https://beast.community/index.html" id="intref0070">https://beast.community/index.html</ext-link></td><td><xref rid="bib35" ref-type="bibr"><u>Suchard et&#xA0;al., 2018</u></xref></td></tr><tr><td>TRACER v1.7</td><td><ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/tracer/" id="intref0075">http://tree.bio.ed.ac.uk/software/tracer/</ext-link></td><td><xref rid="bib25" ref-type="bibr"><u>Rambaut et&#xA0;al., 2018</u></xref></td></tr><tr><td>Augur</td><td><ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/augur/en/stable/index.html" id="intref0080">https://docs.<b>nextstrain</b>.org/projects/augur/en/stable/index.html</ext-link>.<li> Mutational composition of these sequences was analyzed in Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0155">https://clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>) and due to highly variable patterns of substitutions, we have retained in the dataset only those genomes with the full-set of P.2 and Gamma lineages defining synapomorphies, thus excluding from the analysis all Gamma-like sequences (<xref rid="mmc1" ref-type="supplementary-material"><u>Table&#xA0;S2</u></xref>.<li> [Dezordi et&#xA0;al., 2022](          <person-group person-group-type="author">            <name>              <surname>Dezordi</surname>              <given-names>F.Z.</given-names>            </name>            <name>              <surname>Neto</surname>              <given-names>A.M.D.S.</given-names>            </name>            <name>              <surname>Campos</surname>              <given-names>T.L.</given-names>            </name>            <name>              <surname>Jeronimo</surname>              <given-names>P.M.C.</given-names>            </name>            <name>              <surname>Aksenen</surname>              <given-names>C.F.</given-names>            </name>            <name>              <surname>Almeida</surname>              <given-names>S.P.</given-names>            </name>            <name>              <surname>Wallau</surname>              <given-names>G.L.</given-names>            </name>            <collab>Fiocruz Covid-Genomic Surveillance Network</collab>          </person-group>          <article-title>ViralFlow: a versatile automated workflow for SARS-CoV-2 genome assembly, lineage assignment, mutations and intrahost variant detection</article-title>          <source>Viruses</source>          <volume>14</volume>          <year>2022</year>          <pub-id pub-id-type="doi">10.3390/v14020217</pub-id>        ) [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Dezordi et&#xA0;al., 2022](          <person-group person-group-type="author">            <name>              <surname>Dezordi</surname>              <given-names>F.Z.</given-names>            </name>            <name>              <surname>Neto</surname>              <given-names>A.M.D.S.</given-names>            </name>            <name>              <surname>Campos</surname>              <given-names>T.L.</given-names>            </name>            <name>              <surname>Jeronimo</surname>              <given-names>P.M.C.</given-names>            </name>            <name>              <surname>Aksenen</surname>              <given-names>C.F.</given-names>            </name>            <name>              <surname>Almeida</surname>              <given-names>S.P.</given-names>            </name>            <name>              <surname>Wallau</surname>              <given-names>G.L.</given-names>            </name>            <collab>Fiocruz Covid-Genomic Surveillance Network</collab>          </person-group>          <article-title>ViralFlow: a versatile automated workflow for SARS-CoV-2 genome assembly, lineage assignment, mutations and intrahost variant detection</article-title>          <source>Viruses</source>          <volume>14</volume>          <year>2022</year>          <pub-id pub-id-type="doi">10.3390/v14020217</pub-id>        ) [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Nguyen et&#xA0;al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [Rambaut et&#xA0;al., 2016](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27774300">27774300</a>) [Suchard et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29942656">29942656</a>) [Rambaut et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29718447">29718447</a>) [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35400102">35400102</a> <b>(4)</b> Measuring How Recombination Re-shapes the Evolutionary History of PRRSV-2: A Genome-Based Phylodynamic Analysis of the Emergence of a Novel PRRSV-2 Variant<ul><li> Maximum clade credibility (MCC) trees of each fragment were built using TreeAnnotator v.1.10.4 and visualized on the <b>Nextstrain</b> platform (<xref rid="B39" ref-type="bibr"><u>39</u></xref>).<li> The nucleotide substitution rate at the ancestral branch of the novel L1C (inclusive of the 2018 sequence) was lower in some fragments than the overall mean rate, whereas the ancestral branch of the more recent 2020&#x2013;2021 epidemic samples had a higher rate than the mean (<xref rid="T1" ref-type="table"><u>Table 1</u></xref>, trees in <b>Nextstrain</b>: &#x201C;<ext-link xlink:href="https://<b>nextstrain</b>.org/community/NakarinP/prrsv/" ext-link-type="uri">https://<b>nextstrain</b>.org/community/NakarinP/prrsv/</ext-link>&#x201D;).<li> [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35561179">35561179</a> <b>(25)</b> TopHap: rapid inference of key phylogenetic structures from common haplotypes in large genome collections with limited diversity<ul><li><italic toggle="yes">Annotations using <b>Nextstrain</b> and PANGO classifications.<li><italic toggle="yes">Annotations using <b>Nextstrain</b> and PANGO classifications.</italic> To compare <italic toggle="yes">TopHap</italic> phylogeny with the <b>Nextstrain</b> classification, we annotated all the <italic toggle="yes">TopHap</italic> haplotypes using the presence and absence of diagnostic <b>Nextstrain</b> mutations (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov</ext-link>).<li> The <b>Nextstrain</b> clade ID was annotated based on their diagnostic mutations and is provided at the far right.<li> Similarly, <italic toggle="yes">TopHap</italic> phylogeny agreed well with <b>Nextstrain</b> and PANGO trees (<xref rid="btac186-F5" ref-type="fig"><u>Fig.&#xA0;5</u></xref>).<li>The comparison of <italic toggle="yes">TopHap</italic> phylogeny with the (<bold>a</bold>) <b>Nextstrain</b> and (<bold>b</bold>) PANGO phylogenies.<li> The haplotypes with concerning mutations are indicated by using WHO IDs, and 20A EU2 and 20E (EU1) are <b>Nextstrain</b> clade IDs.<li> These haplotypes were identified by annotating PANGO and <b>Nextstrain</b> lineage for each genome.<li> Using these genome annotations, we paired each <italic toggle="yes">TopHap</italic> haplotype with the major PANGO and <b>Nextstrain</b> lineage, which contained the WHO annotation.<li> Evolutionary relationship of lineages with concerning mutations by (<bold>b</bold>) <b>Nextstrain</b> and (<bold>c</bold>) <italic toggle="yes">TopHap</italic>.<li> placements in <italic toggle="yes">TopHap</italic> are consistent with those in the <b>Nextstrain</b> taxonomy (<xref rid="btac186-F6" ref-type="fig"><u>Fig.&#xA0;6b and c</u></xref>).<li> For example, <b>Nextstrain</b> and <italic toggle="yes">TopHap</italic> infer WHO-ALPHA, WHO-GAMMA and WHO-LAMBDA to be sister lineages.<li> So, we used WHO-OMICRON&#x2019;s diagnostic mutations listed on the <b>Nextstrain</b> website (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov</ext-link>) to place it in the 1MG <italic toggle="yes">TopHap</italic> phylogeny.<li> This placement agrees with <b>Nextstrain</b>s&#x2019; inference (<xref rid="btac186-F6" ref-type="fig"><u>Fig.&#xA0;6b and c</u></xref>).<li>We find that <b>Nextstrain</b> and PANGO phylogeny broadly agree with 68KG and 1MG <italic toggle="yes">TopHap</italic> phylogenies, except for the root placement (<xref rid="btac186-F3" ref-type="fig"><u>Figs&#xA0;3</u></xref>, <xref rid="btac186-F5" ref-type="fig"><u>5</u></xref> and <xref rid="btac186-F6" ref-type="fig"><u>6</u></xref>).<li> For example, clade 19A is at the root of the <b>Nextstrain</b> phylogeny, but <italic toggle="yes">TopHap</italic> phylogenies (using the bat/pangolin outgroups) suggest that Clade 19A is derived.<li> The bootstrap support was modest (&gt;66%) for the root of the <italic toggle="yes">TopHap</italic> phylogeny, but no bootstrap replicates supported the <b>Nextstrain</b> rooting, and &lt;34% supported the PANGO rooting.<li> Also, <xref rid="btac186-B3" ref-type="bibr"><u>Bloom (2021)</u></xref> reported two evolutionary scenarios with this mutation <italic toggle="yes">x</italic> (<xref rid="btac186-F7" ref-type="fig"><u>Fig.&#xA0;7b and c</u></xref>), which led us to consider five alternative scenarios based on <italic toggle="yes">TopHap</italic>, MOA, <xref rid="btac186-B3" ref-type="bibr"><u>Bloom (2021)</u></xref>, <b>Nextstrain</b> and PANGO (<xref rid="btac186-F7" ref-type="fig"><u>Fig.&#xA0;7</u></xref>).<li> we attached the haplotype with the <italic toggle="yes">x</italic> mutation into the phylogenies of <italic toggle="yes">TopHap</italic> (<xref rid="btac186-F7" ref-type="fig"><u>Fig.&#xA0;7a and b</u></xref> for two equally parsimonious solutions), MOA (<xref rid="btac186-F7" ref-type="fig"><u>Fig.&#xA0;7b</u></xref>), <b>Nextstrain</b> (<xref rid="btac186-F7" ref-type="fig"><u>Fig.&#xA0;7e</u></xref>) and PANGO (<xref rid="btac186-F7" ref-type="fig"><u>Fig.&#xA0;7d</u></xref>).<li> Five root positions are explored in which the haplotype with mutation <italic toggle="yes">x</italic> has been added to the <italic toggle="yes">TopHap</italic> phylogeny in <xref rid="btac186-F3" ref-type="fig"><u>Figure&#xA0;3</u></xref> &#xA0;<bold>(a</bold> and <bold>b)</bold>, <xref rid="btac186-B8" ref-type="bibr">Kumar <italic toggle="yes">et al.</italic> (2021)</xref> mutational history (<bold>b</bold>), <xref rid="btac186-B3" ref-type="bibr"><u>Bloom (2021)</u></xref> phylogeny (<bold>b</bold> and <bold>c</bold>), PANGO classification (<bold>d</bold>) and the <b>Nextstrain</b> classification (<bold>e</bold>).<li> In the MP analysis, scenarios A, B and C were equally parsimonious, and D and E (PANGO and <b>Nextstrain</b>, respectively) were less parsimonious by 1 and 3 mutations.<li> [Bloom (2021)](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34398234">34398234</a>) [Bloom (2021)](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34398234">34398234</a>) [Bloom (2021)](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34398234">34398234</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35243660">35243660</a> <b>(11)</b> First cases of infection with the 21L/BA.2 Omicron variant in Marseille, France<ul><li>SARS&#x2010;CoV&#x2010;2 variants have been detected since summer 2020<xref rid="jmv27695-bib-0001" ref-type="bibr"><sup>1</sup></xref>, <xref rid="jmv27695-bib-0002" ref-type="bibr"><sup>2</sup></xref> and have been of critical interest with regard to viral transmissibility, viral load, and escape to natural or vaccine immunity.<xref rid="jmv27695-bib-0003" ref-type="bibr"><sup>3</sup></xref>, <xref rid="jmv27695-bib-0004" ref-type="bibr"><sup>4</sup></xref> The Omicron variant is currently the predominant variant of concern in many countries worldwide (<ext-link xlink:href="https://covariants.org/per-country" ext-link-type="uri">https://covariants.org/per-country</ext-link>).<xref rid="jmv27695-bib-0005" ref-type="bibr"><sup>5</sup></xref>, <xref rid="jmv27695-bib-0006" ref-type="bibr"><sup>6</sup></xref> It has been reported to show considerable escape to antibodies elicited by vaccination<xref rid="jmv27695-bib-0007" ref-type="bibr"><sup>7</sup></xref>, <xref rid="jmv27695-bib-0008" ref-type="bibr"><sup>8</sup></xref> and to be associated with lower clinical severity including in our center.<xref rid="jmv27695-bib-0008" ref-type="bibr"><sup>8</sup></xref>, <xref rid="jmv27695-bib-0009" ref-type="bibr"><sup>9</sup></xref> It was first detected in early November in Botswana and thereafter, in many countries, its incidence has rapidly exceeded that of the Delta variant that had predominated since the summer of 2021 (<ext-link xlink:href="https://covariants.org/per-country" ext-link-type="uri">https://covariants.org/per-country</ext-link>).<xref rid="jmv27695-bib-0005" ref-type="bibr"><sup>5</sup></xref>, <xref rid="jmv27695-bib-0006" ref-type="bibr"><sup>6</sup></xref> In fact, Omicron, or clade 21M, is composed of three branches corresponding to three variants named <b>Nextstrain</b> clade<xref rid="jmv27695-bib-0010" ref-type="bibr"><sup>10</sup></xref>, <xref rid="jmv27695-bib-0011" ref-type="bibr"><sup>11</sup></xref> 21K (or Pangolin lineage<xref rid="jmv27695-bib-0012" ref-type="bibr"><sup>12</sup></xref> BA.1), 21L (or BA.2), and lineage BA.3.<li>Nucleotide and amino acid changes in viral genomes relative to the Wuhan&#x2010;Hu&#x2010;1 isolate genome were obtained using the Nextclade tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li>Nucleotide and amino acid changes in viral genomes relative to the Wuhan&#x2010;Hu&#x2010;1 isolate genome were obtained using the Nextclade tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<xref rid="jmv27695-bib-0010" ref-type="bibr"><sup>10</sup></xref>, <xref rid="jmv27695-bib-0011" ref-type="bibr"><sup>11</sup></xref> <b>Nextstrain</b> clades and Pangolin lineages were determined using the Nextclade web application (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>)<xref rid="jmv27695-bib-0010" ref-type="bibr"><sup>10</sup></xref>, <xref rid="jmv27695-bib-0011" ref-type="bibr"><sup>11</sup></xref> and Pangolin web application (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>),<xref rid="jmv27695-bib-0012" ref-type="bibr"><sup>12</sup></xref> respectively.<li> Table&#xA0;<xref rid="jmv27695-tbl-0001" ref-type="table"><u>1A</u></xref>).<xref rid="jmv27695-bib-0016" ref-type="bibr"><sup>16</sup></xref> Finally, phylogeny was reconstructed with the <b>nextstrain</b>/ncov tool (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) then visualized with Auspice (<ext-link xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/" ext-link-type="uri">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).<li>Phylogeny performed with the <b>nextstrain</b>/ncov tool (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) shows that the nine 21L/BA.2 Omicron variant genomes obtained in our institute were part of three clusters.<li> Phylogenetic tree was built using the <b>nextstrain</b>/ncov tool (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) then visualized with Auspice (<ext-link xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/" ext-link-type="uri">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35225672">35225672</a> <b>(5)</b> Omicron: What Makes the Latest SARS-CoV-2 Variant of Concern So Concerning?<ul><li> By the time of revision about 4 weeks later, the Omicron VOC had already outcompeted the Delta VOC and was responsible for &#x223C;63% of new SARS-CoV-2 infections worldwide (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).<li> Retrieved from <b>Nextstrain</b> on 18 January 2022 (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global?m=div" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global?m=div</ext-link>) and modified.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35458389">35458389</a> <b>(2)</b> Genomic Epidemiology of SARS-CoV-2 in Tocantins State and the Diffusion of P.1.7 and AY.99.2 Lineages in Brazil<ul><li>All sequences were analyzed with Nextclade v1.7.2 [<xref rid="B17-viruses-14-00659" ref-type="bibr"><u>17</u></xref>] (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 12 January 2022) to determine the clade and the number of gap regions.<li> The first round of subsampling was performed by using <b>Nextstrain</b>&#x2019;s bioinformatics toolkit [<xref rid="B23-viruses-14-00659" ref-type="bibr"><u>23</u></xref>].<li> Following the <b>Nextstrain</b> workflow [<xref rid="B23-viruses-14-00659" ref-type="bibr"><u>23</u></xref>], the initial evolutionary clock rate was set to 8 &#xD7; 10<sup>&#x2212;4</sup> substitutions per site per year.<li> [17](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35350201">35350201</a> <b>(1)</b> Trivalent NDV-HXP-S vaccine protects against phylogenetically distant SARS-CoV-2 variants of concern in mice<ul><li> This is of special interest because Omicron (B.1.1.529 or BA.1) has quickly diverged into different sub-lineages (BA.1.1, BA.2 and BA.3) and has gained prevalence globally (<ext-link xlink:href="http://www.<b>nextstrain</b>.org/ncov" ext-link-type="uri">www.<b>nextstrain</b>.org/ncov</ext-link>).<li> Phylogenetic tree and frequency timelines were obtained from the <b>Nextstrain</b>/ncov Project within the GISAID initiative (<ext-link xlink:href="http://www.<b>nextstrain</b>.org/ncov" ext-link-type="uri">www.<b>nextstrain</b>.org/ncov</ext-link>) under CC-BY[<xref rid="R29" ref-type="bibr"><u>29</u></xref>, <xref rid="R30" ref-type="bibr"><u>30</u></xref>].<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35364015">35364015</a> <b>(22)</b> Rapid and parallel adaptive mutations in spike S1 drive clade success in SARS-CoV-2<ul><li> For example, the common ancestor of globally circulating SARS-CoV-2 viruses is currently no earlier than January 2020, corresponding to the base of clade 20A or lineage B.1 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="intref0015">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).<li> Despite possessing some proof-reading capacity (a relatively rare function for an&#xA0;RNA virus), SARS-CoV-2 has been accumulating roughly 24&#x2013;25 substitutions per year (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15" id="intref0020">https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15</ext-link>; <xref rid="bib9" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li><hr/></td></tr><tr><td>Augur, version 12.0.0</td><td><xref rid="bib12" ref-type="bibr"><u>Huddleston et&#xA0;al., 2021</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0030">https://github.com/<b>nextstrain</b>/augur</ext-link>.<li><hr/></td></tr><tr><td>Augur, version 12.0.0</td><td><xref rid="bib12" ref-type="bibr"><u>Huddleston et&#xA0;al., 2021</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0030">https://github.com/<b>nextstrain</b>/augur</ext-link></td></tr><tr><td>TreeTime, version 0.8.2</td><td><xref rid="bib27" ref-type="bibr"><u>Sagulenko et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/treetime" id="intref0035">https://github.com/neherlab/treetime</ext-link></td></tr><tr><td>IQ-TREE, version 2.1.2</td><td><xref rid="bib24" ref-type="bibr"><u>Nguyen et&#xA0;al., 2015</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/Cibiv/IQ-TREE" id="intref0040">https://github.com/Cibiv/IQ-TREE</ext-link></td></tr><tr><td>Nextclade</td><td><xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="intref0045">https://clades.<b>nextstrain</b>.org</ext-link>.<li> <xref rid="bib28" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>) on July 29, 2021 and curated by the <b>Nextstrain</b> nCoV ingest pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov-ingest" id="intref0065">https://github.com/<b>nextstrain</b>/ncov-ingest</ext-link>).<li> These genomes were aligned with Nextalign (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/nextclade/en/latest/user/nextalign-cli.html" id="intref0070">https://docs.<b>nextstrain</b>.org/projects/nextclade/en/latest/user/nextalign-cli.html</ext-link>) and masked to minimize error in phylogenetic inference associated with problematic amplicon sites.<li> shorter than 27,000&#xA0;bp of A, C, T, or G bases, missing complete date information, annotated with a date prior to October 2019, flagged with more than 20 mutations above the expected number based on the mutational clock rate, or flagged by Nextclade (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/nextclade/en/latest/user/algorithm/07-quality-control.html" id="intref0075">https://docs.<b>nextstrain</b>.org/projects/nextclade/en/latest/user/algorithm/07-quality-control.html</ext-link>; <xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>) with one or more clusters of 6 or more private differences in a 100-nucleotide window.<li> The primary analysis was conducted on 9,544 genomes collected on or before May 15, 2021, and the phylogeny reconstructed from these data can be found at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15" id="intref0080">https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15</ext-link>.<li>Phylogenies for seasonal influenza A/H3N2 and A/H1N1pdm were built using the <b>Nextstrain</b> pipeline from <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/seasonal-flu" id="intref0090">https://github.com/<b>nextstrain</b>/seasonal-flu</ext-link>.<li> This resulted in a tree with a total of 19,694 genomes, sampled on or before May 15, 2021 and distributed roughly evenly over geography and time (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15/20k" id="intref0100">https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15/20k</ext-link>).<li> The Asia-specific tree can be interactively-viewed at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15/asia" id="intref0105">https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15/asia</ext-link>, and other regional trees can be accessed by substituting the region&#x2019;s name at the end of the URL.<li> The November 15, 2020 tree can be viewed at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2020-11-15" id="intref0110">https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2020-11-15</ext-link>, and all other dates can be accessed by changing the date at the end of the URL.<li>Interactive versions of all phylogenies used in this study can be found at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15" id="intref0115">https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15</ext-link>.<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Huddleston et&#xA0;al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [Huddleston et&#xA0;al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [Sagulenko et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Nguyen et&#xA0;al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) [Shu and McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35133988">35133988</a> <b>(1)</b> T cell response to intact SARS-CoV-2 includes coronavirus cross-reactive and variant-specific components <ul><li> To estimate the global prevalence of SARS-CoV-2 aa variants, the worldwide SARS-CoV-2 sequence data set hosted at GISAID (<xref rid="B92" ref-type="bibr"><u>92</u></xref>) was accessed (<xref rid="B103" ref-type="bibr"><u>103</u></xref>) using the mutation details routine, while lineages containing specific mutations were queried via <b>Nextstrain</b> (<xref rid="B23" ref-type="bibr"><u>23</u></xref>), which accesses a representative subset of GISAID.<li> [92](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [103](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=6311903">6311903</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35341044">35341044</a> <b>(2)</b> E-Volve: understanding the impact of mutations in SARS-CoV-2 variants spike protein on antibodies and ACE2 affinity through patterns of chemical interactions at protein interfaces<ul><li> In October of 2020, from the B.1.617 clade, emerged the B.1.617.2 SARS-CoV-2 strain&#x2014;known as the Delta variant, by the WHO nomenclature, or as 21A, by the <b>Nextstrain</b> nomenclature.<li> This occurrence may lead to augmentation for long-term transmissibility of the SARS-CoV-2 VOCs, for being impactful on the capacity of viral replication, and further dominating more basal strains as visible on real-time tracking by <b>Nextstrain</b> (<xref rid="ref-23" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35341058">35341058</a> <b>(1)</b> The dynamics of circulating SARS-CoV-2 lineages in Bogor and surrounding areas reflect variant shifting during the first and second waves of COVID-19 in Indonesia<ul><li> GTR+F+I evolutionary model) was build using Augur utility (github.com/<b>Nextstrain</b>/augur) from the Nexstrain bioinformatic pipeline&#xA0;(<xref rid="ref-16" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), employing MAFFT&#xA0;(<xref rid="ref-19" ref-type="bibr"><u>Katoh &amp; Standley, 2016</u></xref>) and IQ-TREE&#xA0;(<xref rid="ref-33" ref-type="bibr"><u>Nguyen et al., 2015</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Katoh &amp; Standley, 2016](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27153688">27153688</a>) [Nguyen et al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35458383">35458383</a> <b>(5)</b> SARS-CoV-2: Evolution and Emergence of New Viral Variants<ul><li> In this work, we describe the classification used by the World Health Organization (WHO), Phylogenetic Assignment of Named Global Outbreak (PANGO), Global Initiative on Sharing All Influenza Data (GISAID), and <b>Nextstrain</b>, an open-source project to harness the scientific and public health potential of pathogen genome data to define the chronological emergence of new variants of SARS-CoV-2, which are classified as variants of concern (VOCs) and variants of interest (VOIs), as well as discussing the ability of each variant to evade the humoral immune response.<li> Other common nomenclature systems included those recommended by the GISAID [<xref rid="B15-viruses-14-00653" ref-type="bibr"><u>15</u></xref>], PANGO [<xref rid="B16-viruses-14-00653" ref-type="bibr"><u>16</u></xref>], and <b>Nextstrain</b> [<xref rid="B17-viruses-14-00653" ref-type="bibr"><u>17</u></xref>].<li> Finally, <b>Nextstrain</b> classifies SARS-CoV-2 into 14 major clades: 19A, 19B, and 20A-20L.<li> The incorporated mutations in the genome of the new viral variants caused organizations and online platforms like WHO, PANGO, GISAID, and <b>Nextstrain</b> to classify these mutations in variants, clades, or lineages, and identify their global spread and evaluate the potential of emergence of other new viral variants.<li> Nevertheless, the strengths of our work are (a) definition of viral variants by available platforms (WHO, PANGO, GISAID, and <b>Nextstrain</b>), (b) the chronological emergence of viral variants, and (c) description of mutations associated with the viral spread and evasion of the host immune response.<li> [15](          <article-title>GISAID Clade and Lineage Nomenclature Aids in Genomic Epidemiology Studies of Active hCoV-19 Viruses</article-title>          <comment>Available online: <ext-link xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-02-04">(accessed on 4 February 2022)</date-in-citation>        ) [16](          <article-title>PANGO Latest Epidemiological Lineages of SARS-CoV-2, 03 February 2022</article-title>          <comment>Available online: <ext-link xlink:href="https://cov-lineages.org/index.html" ext-link-type="uri">https://cov-lineages.org/index.html</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-02-04">(accessed on 4 February 2022)</date-in-citation>        ) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35264078">35264078</a> <b>(3)</b> SARS-CoV-2 reinfection and COVID-19 severity<ul><li> A phylogenetic tree was built with the <b>Nextstrain</b>/ncov tool (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov)" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov)</ext-link> [<xref rid="CIT0014" ref-type="bibr"><u>14</u></xref>] that performs maximum-likelihood phylogeny using IQ-TREE [<xref rid="CIT0015" ref-type="bibr"><u>15</u></xref>] and then visualized with Auspice (<ext-link xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/" ext-link-type="uri">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35386683">35386683</a> <b>(1)</b> SARS-CoV-2 Mutations and COVID-19 Clinical Outcome: Mutation Global Frequency Dynamics and Structural Modulation Hold the Key<ul><li>Phylogenetic analysis of 246 SARS-CoV-2 genomes using PANGOLIN (lineage) and <b>Nextstrain</b> (clades) showed the dominant occurrence of B.6 (29.2%), B.1.36 (9.82%), B.1 (10.56%), B.1.1 (4.47%), B.1.1.306 (4.47%), and B.6.6 (4.47%), whereas clade distribution highlighted the dominant prevalence of 19A (52.84%) followed by 20A (34.55%), as mentioned in <xref rid="ST1" ref-type="supplementary-material"><bold>Supplementary Table S1</bold></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35387187">35387187</a> <b>(2)</b> Description of Symptoms Caused by the Infection of the SARS-CoV-2 B.1.621 (Mu) Variant in Patients With Complete CoronaVac Vaccination Scheme: First Case Report From Santiago of Chile<ul><li> <bold>(E)</bold> Representative phylogenetic tree placement of all NPSs using <b>Nextstrain</b> clades classification, all samples belong to clade 21H.<li> Representative MiSeq and NextSeq 2000 sequencing confirmed the presence of the Mu variant of SARS-CoV-2, being classified on the <b>Nextstrain</b> Clade 21H (NextClade software version 1.9, Biozentrum, University of Basel, Swiss) (<xref rid="F1" ref-type="fig"><u>Figure 1E</u></xref>) and following the World Health Organization (WHO) recommendations (<xref rid="B16" ref-type="bibr"><u>16</u></xref>).<li> [16](<person-group person-group-type="author"><collab>World Health Organization. Tracking SARS-CoV-2 Variants.</collab></person-group> (<year>2021</year>). Available online at: <ext-link xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants" ext-link-type="uri">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants</ext-link> (accessed October 18, 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35328105">35328105</a> <b>(1)</b> Coronavirus Genomes and Unique Mutations in Structural and Non-Structural Proteins in Pakistani SARS-CoV-2 Delta Variants during the Fourth Wave of the Pandemic<ul><li>Genomic sequences in the current study were subjected to MAFT server [<xref rid="B27-genes-13-00552" ref-type="bibr"><u>27</u></xref>] and <b>Nextstrain</b> [<xref rid="B28-genes-13-00552" ref-type="bibr"><u>28</u></xref>] for phylogenetic analysis.<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27189561">27189561</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35331783">35331783</a> <b>(1)</b> mRNA vaccines for COVID-19 and diverse diseases<ul><li>In October 2020 in South Africa, the B.1.351, known as <b>Nextstrain</b> clade 20H, was first detected and believed to be 50% more transmissible than the original variant.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35311567">35311567</a> <b>(1)</b> SARS-CoV-2 Lineage Tracking, and Evolving Trends Seen during Three Consecutive Peaks of Infection in Delhi, India: a Clinico-Genomic Study<ul><li><bold>(ii) Evolutionary relationship among detected lineages.</bold> The phylogenetic relationship of all the lineages detected in our data set shows eight major clade members as per <b>Nextstrain</b> clade assignment (<xref rid="B9" ref-type="bibr"><u>9</u></xref>).<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33758205">33758205</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35311562">35311562</a> <b>(5)</b> Mapping Data to Deep Understanding: Making the Most of the Deluge of SARS-CoV-2 Genome Sequences<ul><li>The first widely used tool for tracking SARS-CoV-2 genomic variation was the <b>Nextstrain</b> project, <ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>.<li> <b>Nextstrain</b>, originally developed as a general tool for viruses, was adapted to offer clade definitions for SARS-CoV-2 based on phylogenetic analysis (<xref rid="B23" ref-type="bibr"><u>23</u></xref>).<li> The Pango nomenclature (<ext-link xlink:href="https://cov-lineages.org/" ext-link-type="uri">https://cov-lineages.org/</ext-link>), developed specifically for SARS-CoV-2, has largely supplanted <b>Nextstrain</b> clade definitions (<xref rid="B25" ref-type="bibr"><u>25</u></xref>).<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35335115">35335115</a> <b>(0)</b> A Rapid and Consistent Method to Identify Four SARS-CoV-2 Variants during the First Half of 2021 by RT-PCR<ul><li> Mutations of external datasets were defined by uploading the FASTA sequences to the NextClade website (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 1 January 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35371561">35371561</a> <b>(1)</b> The early SARS-CoV-2 epidemic in Senegal was driven by the local emergence of B.1.416 and the introduction of B.1.1.420 from Europe<ul><li> Initially, a list of headers for a globally representative, Africa-excluded, genome dataset was downloaded from the daily <b>NextStrain</b> update (along with 20 extra Delta variant genomes, totaling <italic toggle="yes">N</italic>&#x2009;=&#x2009;3,283 sequences) on 23 April 2021 set as the final day for sequence collection in our study.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35302731">35302731</a> <b>(1)</b> TheInteraction of Hypericin with SARS-CoV-2Reveals a Multimodal Antiviral Activity<ul><li> SARS-CoV-2 HCoV-19/Italy/310902/46/2020strain (GISAID code: EPI_ISL_9011947), belonging to clade 20A (<b>Nextstrain</b>naming), was propagated in the same cell line and incubated at 37&#xB0;C with 5% CO<sub>2</sub>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35303887">35303887</a> <b>(1)</b> SARS-CoV-2: tracing the origin, tracking the evolution<ul><li> Tools, like <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>], can use the registered genomes for phylogeographic analysis, permitting fast identification of the origin of new variants/mutations of the virus.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35300748">35300748</a> <b>(0)</b> COVID-19 cases in spectators returning to Finland from UEFA Euro 2020 matches in Saint Petersburg<ul><li> Adapter, low quality (quality score &lt;30) and short (&lt;50&#xA0;nt) sequences were removed using Trimmomatic [<xref rid="ref2" ref-type="bibr"><u>2</u></xref>], followed by assembly using BWA-MEM [<xref rid="ref3" ref-type="bibr"><u>3</u></xref>], variant calling using LoFreq [<xref rid="ref4" ref-type="bibr"><u>4</u></xref>] and consensus calling using SAMtools [<xref rid="ref5" ref-type="bibr"><u>5</u></xref>] implemented in HaVoC pipeline [<xref rid="ref6" ref-type="bibr"><u>6</u></xref>], followed by lineage annotation with Nextclade (clades.<b>nextstrain</b>.org) and Pangolin (<uri xlink:href="https://cov-lineages.org/resources/pangolin.html">https://cov-lineages.org/resources/pangolin.html</uri><underline>)</underline>.<li> [2](<string-name><surname>Bolger</surname><given-names>AM</given-names></string-name>, <string-name><surname>Lohse</surname><given-names>M</given-names></string-name> and <string-name><surname>Usadel</surname><given-names>B</given-names></string-name> (<year>2014</year>) <article-title>Trimmomatic: a flexible trimmer for Illumina sequence data</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>30</volume>, <fpage>2114</fpage>&#x2013;<lpage>2120</lpage>.) [3](<string-name><surname>Li</surname><given-names>H</given-names></string-name> (<year>2013</year>) Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv:1303.3997v2.) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23066108">23066108</a>) [5](<string-name><surname>Li</surname><given-names>H</given-names></string-name><etal/> (<year>2009</year>) <article-title>The sequence alignment/map format and SAMtools</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>25</volume>, <fpage>2078</fpage>&#x2013;<lpage>2079</lpage>.) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34273961">34273961</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35502202">35502202</a> <b>(2)</b> Tracking SARS-CoV-2 mutations and variants through the COG-UK-MutationExplorer<ul><li> CoV-GLUE<ext-link xlink:href="http://cov-glue.cvr.gla.ac.uk" ext-link-type="uri">http://cov-glue.cvr.gla.ac.uk</ext-link>,<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>; and <ext-link xlink:href="https://covariants.org" ext-link-type="uri">https://covariants.org</ext-link>&#x2014;last accessed date:16 March 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35371559">35371559</a> <b>(3)</b> Replacement of the Gamma by the Delta variant in Brazil: Impact of lineage displacement on the ongoing pandemic<ul><li> This was achieved by the inclusion of sequences in the <b>NextStrain</b> Delta build (<ext-link xlink:href="https://<b>nextstrain</b>.org/groups/neherlab/ncov/21A.Delta" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/neherlab/ncov/21A.Delta</ext-link>) at the time of writing and a random selection of 9,000 Delta sequences from the GISAID database.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35317350">35317350</a> <b>(1)</b> Exploring a prolonged enterovirus C104 infection in a severely ill patient using nanopore sequencing<ul><li> 2018</xref>) and the augur pipeline implemented in <b>Nextstrain</b> (<xref rid="R12" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35297757">35297757</a> <b>(1)</b> Unusual SARS-CoV-2 intrahost diversity reveals lineage superinfection<ul><li>A reference alignment was created using <sc>mafft</sc> [<xref rid="R33" ref-type="bibr"><u>33</u></xref>] with the 6167 genomes, which represents the genomes used in <b>Nextstrain</b> [<xref rid="R34" ref-type="bibr"><u>34</u></xref>] global phylogeny with N content less than 5&#x200A;% accessed on 24 May 2021 and Brazilian genomes obtained through a cd-hit-est [<xref rid="R35" ref-type="bibr"><u>35</u></xref>] clusterization of genomes present on GISAID at 16 March 2021 with high-quality and with more than 99.8&#x200A;% sequence identity and from the same Brazilian state.<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16731699">16731699</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35301314">35301314</a> <b>(2)</b> De novo emergence of a remdesivir resistance mutation during treatment of persistent SARS-CoV-2 infection in an immunocompromised patient: a case report<ul><li> Phylogenetic analysis found that sequenced genomes belonged to a single lineage (Pango B1) within <b>Nextstrain</b> clade 20&#x2009;C (Supplementary Fig.&#xA0;<xref rid="MOESM1" ref-type="media"><u>1</u></xref>), indicating that viral genomes identified during the course of illness were derived from intra-host diversification following infection with a single strain.<li>Phylogenic trees were generated using the <b>Nextstrain</b> software package (version 3.0.6)<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup>.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35313589">35313589</a> <b>(2)</b> Detecting SARS-CoV-2 Variants in Wastewater and Their Correlation With Circulating Variants in the Communities<ul><li> SARS-CoV-2 sequences of the randomly selected clinical samples from the individuals of the Reno-Sparks metropolitan area during November 1, 2020, and June 30, 2021, were displayed through <ext-link xlink:href="http://www.auspice.us/" ext-link-type="uri">www.auspice.us</ext-link> (genomic epidemiology of novel coronavirus built on <ext-link xlink:href="http://www.<b>nextstrain</b>/ncov" ext-link-type="uri">www.<b>nextstrain</b>/ncov</ext-link>) (<xref rid="F3" ref-type="fig">Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35310621">35310621</a> <b>(2)</b> Determinants of SARS-CoV-2 transmission to guide vaccination strategy in an urban area<ul><li> For subsetting, we included 30 genomes per country and month, whereby all genomes from Basel-City were retained, totalling 3,495 genomes, using the <b>nextstrain</b> software v.2.0.0 (<b>nextstrain</b>.org) and augur v.8.0.0 (<xref rid="R21" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35337031">35337031</a> <b>(0)</b> Molecular Epidemiology of SARS-CoV-2 in Tunisia (North Africa) through Several Successive Waves of COVID-19<ul><li> A Nextclade tree was also constructed using the online tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 10 March 2022)).<li> <xref rid="viruses-14-00624-f001" ref-type="fig"><u>Figure 1</u></xref> shows the phylogenetic tree built using the Nextclade online tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 10 March 2022)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35326463">35326463</a> <b>(1)</b> Multi-Design Differential Expression Profiling of COVID-19 Lung Autopsy Specimens Reveals Significantly Deregulated Inflammatory Pathways and SFTPC Impaired Transcription<ul><li>Clades were assigned according to the nomenclature defined by <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35304093">35304093</a> <b>(0)</b> Tracking SARS-CoV-2 variants by entire S-gene analysis using long-range RT-PCR and Sanger sequencing<ul><li>The amino acid mutations and types of VOC were determined by Nextclade (v1.12.0, <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="ir010">https://clades.<b>nextstrain</b>.org</ext-link>) <xref rid="b0055" ref-type="bibr"><u>[11]</u></xref>.<li> [[11]](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35369526">35369526</a> <b>(2)</b> A New Way to Trace SARS-CoV-2 Variants Through Weighted Network Analysis of Frequency Trajectories of Mutations<ul><li> mutations against a global reference dataset provided by GISAID using the pipeline proposed by <b>Nextstrain</b> (<xref rid="B6" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> Either phylogenetic-tree-based partition of GISAID (<xref rid="B7" ref-type="bibr"><u>Han et al., 2019</u></xref>), <b>Nextstrain</b> (<xref rid="B6" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and Pangolin (<xref rid="B22" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), or genetic-feature-based grouping of CMMs (<xref rid="B21" ref-type="bibr"><u>Qin et al., 2021</u></xref>) and ISMs (<xref rid="B37" ref-type="bibr"><u>Zhao et al., 2020</u></xref>), captured viral subtype features according to historical data, resulting in lag signals of classification, and then false subtyping at the early stage of their emergence delayed the public health response.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Han et al., 2019](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30854550">30854550</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Qin et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34121111">34121111</a>) [Zhao et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32941419">32941419</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35327034">35327034</a> <b>(1)</b> Healthcare Management: A Bibliometric Analysis Based on the Citations of Research Articles Published between 1967 and 2020<ul><li> Since the beginning of the pandemic on platforms such as GISAID or <b>Nextstrain</b>, the amount of data shared has increased significantly.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35107305">35107305</a> <b>(1)</b> Development of a One-Step Qualitative RT-PCR Assay to Detect the SARS-CoV-2 Omicron (B.1.1.529) Variant in Respiratory Specimens<ul><li> The genomic analysis demonstrated that this Omicron VOC is classified into the Pangolin lineage B.1.1.529 and the <b>Nextstrain</b> clade 21K (<ext-link xlink:href="https://www.cdc.gov/coronavirus/2019-ncov/science/science-briefs/scientific-brief-omicron-variant.html" ext-link-type="uri">https://www.cdc.gov/coronavirus/2019-ncov/science/science-briefs/scientific-brief-omicron-variant.html</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35296517">35296517</a> <b>(2)</b> SARS-CoV-2 spike E156G/&#x394;157-158 mutations contribute to increased infectivity and immune escape<ul><li> The sequences were classified into SARS-CoV-2 strains and were analyzed for frequency of insertion and deletion at each amino acid in the NTD of spike sequence using Nextrain (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35294336">35294336</a> <b>(0)</b> Targeted Sanger sequencing to recover key mutations in SARS-CoV-2 variant genome assemblies produced by next-generation sequencing<ul><li> The resulting consensus sequences were analysed in Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and the mutational sets were compared.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35615298">35615298</a> <b>(1)</b> Confirmed Severe Acute Respiratory Syndrome Coronavirus 2 Reinfections After a Second Wave With Predominance of Lambda in Lima and Callao, Peru<ul><li>We included subjects with 2 episodes of laboratory-confirmed SARS-CoV-2 infection by reverse-transcriptase polymerase chain reaction (RT-PCR) with 90 or more days between both episodes and presence of 2 different genetic clades of the virus between the first and second samples, according to the PANGOLIN case classification system registered in <b>Nextstrain</b> and GISAID [<xref rid="CIT0010" ref-type="bibr"><u>10</u></xref>].<li> [10](<person-group person-group-type="author"><string-name><surname>Pampa-Espinoza</surname><given-names>L</given-names></string-name>, <string-name><surname>Silva-Valencia</surname><given-names>J</given-names></string-name>, <string-name><surname>Fern&#xE1;ndez</surname><given-names>M</given-names></string-name>, <string-name><surname>Padilla-Rojas</surname><given-names>C</given-names></string-name>, <string-name><surname>Solari</surname><given-names>L.</given-names></string-name></person-group><article-title>Reinfections are more frequent than currently considered in countries with high incidence of COVID-19 cases due to stringent definitions.</article-title><source>Clin Infect Dis</source><year>2021</year>. doi: <pub-id pub-id-type="doi">10.1093/cid/ciab783</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35292834">35292834</a> <b>(3)</b> COVID-19 pandemic: the delta variant, T-cell responses, and the efficacy of developing vaccines<ul><li>Various classification systems, including Nexstrain system (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) and PANGOLIN classification system (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/resources/pangolin.html/">https://cov-lineages.org/resources/pangolin.html/</ext-link>), have been developed to keep track of all the new emerging variants as the pandemic has progressed.<li><b>Nextstrain</b> clade</th><th align="left">WHO nomenclature</th><th align="left"/><th align="left"/><th align="left"/><th align="left"/></tr></thead><tbody><tr><td align="left">1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35328562">35328562</a> <b>(0)</b> An Upgrade on the Surveillance System of SARS-CoV-2: Deployment of New Methods for Genetic Inspection<ul><li>The consensus sequence was then uploaded in Nextclade (<uri>clades.<b>nextstrain</b>.org</uri>, last accession 14 December 2021) and the mutation list was analyzed against the results obtained from the other tools.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35285705">35285705</a> <b>(12)</b> Specific Detection of SARS-CoV-2 Variants B.1.1.7 (Alpha) and B.1.617.2 (Delta) Using a One-Step Quantitative PCR Assay<ul><li> In order to establish that the selected mutations were indeed unique to the Delta lineage, global analysis of SC-2 genomes was performed using the <b>NextStrain</b> website (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>) using the deletion in positions 22045 to 22050 as a search term.<li> (B) Global analysis of SC-2 genomes available in the <b>NextStrain</b> database (<ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>) showing that this 6-bp deletion is present only in the Delta lineage (highlighted in purple).<li> (B) Global analysis of SC-2 genomes available in the <b>NextStrain</b> database (<ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>) showing that this 6-bp deletion is present only in the Delta lineage (highlighted in purple).<li> Uniqueness of the selected mutations was confirmed by performing global analysis for the selected mutations in the <b>NextStrain</b> website to identify other lineages with identical mutations (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35337007">35337007</a> <b>(1)</b> Next-Generation Sequencing for Confronting Virus Pandemics<ul><li> 20A&#x2013;J, 21A&#x2013;D,F&#x2013;K,L&#x2013;M) have been reported using <b>Nextstrain</b> classification [<xref rid="B9-viruses-14-00600" ref-type="bibr"><u>9</u></xref>], and 1728 lineages [<xref rid="B88-viruses-14-00600" ref-type="bibr"><u>88</u></xref>] designated using Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) [<xref rid="B85-viruses-14-00600" ref-type="bibr"><u>85</u></xref>] software.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [88](          <person-group person-group-type="author"><name><surname>O&#x2019;Toole</surname><given-names>&#xC1;.</given-names></name><name><surname>Scher</surname><given-names>E.</given-names></name><name><surname>Rambaut</surname><given-names>A.</given-names></name></person-group>          <article-title>SARS-CoV-2. Lineages. PANGOLIN</article-title>          <year>2022</year>          <comment>Available online: <ext-link xlink:href="https://cov-lineages.org/resources/pangolin.html" ext-link-type="uri">https://cov-lineages.org/resources/pangolin.html</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-02-04">(accessed on 4 February 2022)</date-in-citation>        ) [85](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35218683">35218683</a> <b>(2)</b> Molecular detection of SARS&#x2010;CoV&#x2010;2 and differentiation of Omicron and Delta variant strains<ul><li> Although phylogenetically, Omicron variants belong to clades 21K and 21L, both are part of a larger clade, 21&#xA0;M (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>), which is based on mutations they mutually possess; the variant is further divided into BA.1 and BA.2 subtypes, and some reports have also proposed the BA.3 subtype.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35288252">35288252</a> <b>(1)</b> Risk of SARS-CoV-2 transmission upon return to work in RNA-positive healthcare workers<ul><li> This study used procedures taken from [github.com/<b>nextstrain</b>/ncov] including the clock rate, reference genome and site masking.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35229308">35229308</a> <b>(4)</b> SARS&#x2010;COV&#x2010;2 genomic monitoring in the state of S&#xE3;o Paulo unveils two emerging AY.43 sublineages<ul><li>Phylogenetic analysis was performed using the <b>Nextstrain</b> v3.0.3 SARS&#x2010;CoV&#x2010;2 workflow.<li> To reconstruct the phylogeny of the AY.43 hypothetical sublineages we used 1616 SARS&#x2010;CoV&#x2010;2 sequences generated from the Butantan Network distributed between the epidemiological Weeks 41&#x2212;43 (GISAID accession ID in&#xA0;File&#xA0;<xref rid="jmv27674-suppl-0001" ref-type="supplementary-material"><u>S1</u></xref>) as input for the <b>Nextstrain</b> workflow.<li>A whole phylogenetic interactive tree, from which the AY.43 cluster was initially observed, can be accessed at the following link: <ext-link xlink:href="https://<b>nextstrain</b>.org/fetch/repositorio.butantan.gov.br/bitstream/butantan/3990/1/ncov_EpiWeek_forty-one-to-forty-three.json" ext-link-type="uri">https://<b>nextstrain</b>.org/fetch/repositorio.butantan.gov.br/bitstream/butantan/3990/1/ncov_EpiWeek_forty-one-to-forty-three.json</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35526849">35526849</a> <b>(3)</b> Genomics of Ancient Pathogens: First Advances and Prospects<ul><li><b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>), collaboration between researchers in Seattle, USA and Basel, Switzerland.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35512873">35512873</a> <b>(2)</b> Multiple SARS-CoV-2 Reinfections: A Case Series of Thrice-Infected Individuals<ul><li>Predominant circulating strain at the time of the infection in the individual&#x2019;s state per <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org" id="intref0015"><b>nextstrain</b>.org</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35272746">35272746</a> <b>(0)</b> Molecular epidemiology of the SARS-CoV-2 variant Omicron BA.2 sub-lineage in Denmark, 29 November 2021 to 2 January 2022<ul><li> Mutations were derived from consensus genomes by use of Nextclade CLI (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) [<xref rid="r19" ref-type="bibr"><u>19</u></xref>].<li> [19](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source>J Open Source Softw</source>. <year>2021</year>;<volume>6</volume>(<issue>67</issue>):<fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35493113">35493113</a> <b>(2)</b> COVID-19 Vaccines and SARS-CoV-2 Transmission in the Era of New Variants: A Review and Perspective<ul><li> Variant(s) listed in the table is the dominant sequence at the time and location of the study, as per <b>Nextstrain</b>.org.<li> Variant(s) listed in table is the dominant sequence at the time and location of the study, as per <b>Nextstrain</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36712796">36712796</a> <b>(3)</b> Continuous genomic diversification of long polynucleotide fragments drives the emergence of new SARS-CoV-2 variants of concern<ul><li>The multiple sequence alignment (MSA) of subsampled SARS-CoV-2 genomes (3,095 genomes randomly sampled from &#x201C;<italic toggle="yes">Open data</italic>&#x201D;) was derived from the <b>Nextstrain</b> database (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>), accessed on 20th December, 2021 (<xref rid="bib43" ref-type="bibr"><u>43</u></xref>).<li> [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35336980">35336980</a> <b>(1)</b> COVID-19 at a Glance: An Up-to-Date Overview on Variants, Drug Design and Therapies<ul><li> Three nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages are currently in use: Pango [<xref rid="B44-viruses-14-00573" ref-type="bibr"><u>44</u></xref>,<xref rid="B45-viruses-14-00573" ref-type="bibr"><u>45</u></xref>] (available online) [<xref rid="B46-viruses-14-00573" ref-type="bibr"><u>46</u></xref>], Global Initiative on Sharing All Influenza Data (GISAID) [<xref rid="B47-viruses-14-00573" ref-type="bibr"><u>47</u></xref>,<xref rid="B48-viruses-14-00573" ref-type="bibr"><u>48</u></xref>] and <b>NextStrain</b> [<xref rid="B49-viruses-14-00573" ref-type="bibr"><u>49</u></xref>,<xref rid="B50-viruses-14-00573" ref-type="bibr"><u>50</u></xref>].<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [45](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33514928">33514928</a>) [46](          <article-title>PANGO Lineages</article-title>          <comment>Available online: <ext-link xlink:href="https://cov-lineages.org" ext-link-type="uri">https://cov-lineages.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-02-01">(accessed on 1 February 2022)</date-in-citation>        ) [47](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [48](          <article-title>Clade and Lineage Nomenclature Aids in Genomic Epidemiology Studies of Active hCoV-19 Viruses. GISAID</article-title>          <comment>Available online: <ext-link xlink:href="https://go.nature.com/3pgSIt6" ext-link-type="uri">https://go.nature.com/3pgSIt6</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-02-01">(accessed on 1 February 2022)</date-in-citation>        ) [49](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [50](          <person-group person-group-type="author"><name><surname>Bedford</surname><given-names>T.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Updated Nextstrain SARS-CoV-2 Clade Naming Strategy. Nextstrain</article-title>          <comment>Available online: <ext-link xlink:href="https://go.nature.com/3c9Riep" ext-link-type="uri">https://go.nature.com/3c9Riep</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-02-01">(accessed on 1 February 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35336792">35336792</a> <b>(2)</b> PWM2Vec: An Efficient Embedding Approach for Viral Host Specification from Coronavirus Spike Sequences<ul><li> Upon similar methods, the state-of-the-art <b>Nextstrain</b>&#xA0;[<xref rid="B24-biology-11-00418" ref-type="bibr"><u>24</u></xref>] and IQTREE2&#xA0;[<xref rid="B25-biology-11-00418" ref-type="bibr"><u>25</u></xref>] have been developed.<li> For example, <b>Nextstrain</b>&#xA0;[<xref rid="B24-biology-11-00418" ref-type="bibr"><u>24</u></xref>] is able to construct trees on thousands of sequences, whereas IQTREE2&#xA0;[<xref rid="B25-biology-11-00418" ref-type="bibr"><u>25</u></xref>] is able to scale to tens of thousands of sequences.<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35266951">35266951</a> <b>(4)</b> Evolutionary history of the SARS-CoV-2 Gamma variant of concern(P.1): a perfect storm<ul><li> <b>Nextstrain</b> clade 20H/V2) in South Africa,which appeared in May-August 2020 (<xref rid="B95" ref-type="bibr">Tegally<italic>et al</italic>., 2020</xref>; <xref rid="B128" ref-type="bibr"><u>WorldHealth Organization, 2021c</u></xref>).<li> <b>Nextstrain</b> clade 20I/V1) emerged in southeast England inSeptember-November 2020 and is rapidly spreading toward fixation (<xref rid="B19" ref-type="bibr">Davies <italic>et al</italic>., 2021a</xref>; <xref rid="B128" ref-type="bibr"><u>World Health Organization, 2021b</u></xref>,<xref rid="B129" ref-type="bibr"><u>c</u></xref>).<li> <b>Nextstrain</b> clade 20J/V3) with the following key<italic>S</italic> mutations: L18F, T20N, P26S, D138Y, R190S, K417T, E484K,N501Y, D614G, H655Y, T1027I, and V1176F.<li>In May 2021, a new VOC was recognized: Delta (B.1.617, <b>Nextstrain</b> clade 21A) whichhas three sub-lineages: B.1.617.1, B.1.617.2, and B.1.617.3.<li> [WorldHealth Organization, 2021c](          <person-group person-group-type="author"><collab>World Health Organization</collab></person-group>          <year>2021</year>          <source>Weekly epidemiological update on COVID-19 - 25 May 2021</source>          <comment>World Health Organization (2021c) Weekly epidemiological update onCOVID-19 - 25 May 2021, <ext-link xlink:href="https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---25-may-2021" ext-link-type="uri">https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---25-may-2021</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-27">27 May,2021</date-in-citation>        ) [World Health Organization, 2021b](          <person-group person-group-type="author"><collab>World Health Organization</collab></person-group>          <year>2021</year>          <source>Weekly epidemiological update on COVID-19 - 25 May 2021</source>          <comment>World Health Organization (2021c) Weekly epidemiological update onCOVID-19 - 25 May 2021, <ext-link xlink:href="https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---25-may-2021" ext-link-type="uri">https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---25-may-2021</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-27">27 May,2021</date-in-citation>        ) [c](          <person-group person-group-type="author"><collab>World Health Organization</collab></person-group>          <year>2021</year>          <source>Weekly epidemiological update on COVID-19 - 29 June 2021</source>          <comment>World Health Organization (2021d) Weekly epidemiological update onCOVID-19 - 29 June 2021, <ext-link xlink:href="https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---29-june-2021" ext-link-type="uri">https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---29-june-2021</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-07-01">1 July,2021</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35262410">35262410</a> <b>(0)</b> Vaccine-Induced Antibody Responses against SARS-CoV-2 Variants-Of-Concern Six Months after the BNT162b2 COVID-19 mRNA Vaccination<ul><li> Sequence reads were mapped to the reference genome with BWA-MEM, and amino acid changes were identified with Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35336968">35336968</a> <b>(1)</b> CovDif, a Tool to Visualize the Conservation between SARS-CoV-2 Genomes and Variants<ul><li> Also, online resources such as <b>Nextstrain</b> allow fast and real-time tracking of the SARS-CoV-2 genome-wide diversity along with evolutionary relationships and geographical location [<xref rid="B44-viruses-14-00561" ref-type="bibr"><u>44</u></xref>].<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35636819">35636819</a> <b>(1)</b> The Successes and Challenges of SARS-CoV-2 Molecular Testing in the United States<ul><li> In addition to GISAID and other data repository tools (e.g., NCBI GenBank), open software platforms such as <b>Nextstrain</b> and Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) have facilitated the assignments of SARS-CoV-2 lineages, particularly for laboratories with limited bioinformatics skills.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35258772">35258772</a> <b>(4)</b> Omicron variant (B.1.1.529) of SARS-CoV-2: understanding mutations in the genome, S-glycoprotein, and antibody-binding regions<ul><li>The mutational event diversity of SARS-CoV-2 and S-glycoprotein of Omicron and another lineage were analyzed to reveal the number of mutations in specific positions using <b>Nextstrain</b> server [<xref ref-type="bibr" rid="CR36"><u>36</u></xref>, <xref ref-type="bibr" rid="CR37"><u>37</u></xref>].<li> The <b>Nextstrain</b> server, a database used to analyzed viral genomes, fetched the sequence of SARS-CoV-2 from the GISAID database to plot the mutational event diversity.<li> We have used the&#xA0;<b>Nextstrain</b> server&#xA0;[<xref ref-type="bibr" rid="CR36"><u>36</u></xref>]&#xA0;for the analysis.<li> We have used the&#xA0;<b>Nextstrain</b> server&#xA0;[<xref ref-type="bibr" rid="CR36"><u>36</u></xref>]&#xA0;for the analysis.<li> [36](Mathieu E, Ritchie H, Ortiz-Ospina E, Roser M, Hasell J, Appel C et al. A global database of COVID-19 vaccinations. Nature human behaviour. 2021:1&#x2013;7.) [37](Nextstrain. Nextstrain SARS-CoV-2 resources, on Nextstrain. 2021. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link> (Acessed on 27 December 2021).) [36](Mathieu E, Ritchie H, Ortiz-Ospina E, Roser M, Hasell J, Appel C et al. A global database of COVID-19 vaccinations. Nature human behaviour. 2021:1&#x2013;7.) [36](Mathieu E, Ritchie H, Ortiz-Ospina E, Roser M, Hasell J, Appel C et al. A global database of COVID-19 vaccinations. Nature human behaviour. 2021:1&#x2013;7.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35104245">35104245</a> <b>(1)</b> Adaptive immune responses in vaccinated patients with symptomatic SARS-CoV-2 Alpha infection<ul><li> P.1 (Gamma variant), B.1.526 (Iota variant), B.1.311, and early B.1 lineage (19A <b>Nextstrain</b>) accounted for the remaining 4.<li> Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) was used for clade assignment, sequence quality check, and phylogenetic tree construction.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35255849">35255849</a> <b>(1)</b> Using genomic epidemiology of SARS-CoV-2 to support contact tracing and public health surveillance in rural Humboldt County, California<ul><li> All phylogenetic analyses to support public health genomic surveillance and to generate figures for this paper were conducted using the <b>Nextstrain</b> tool suite, described in [<xref ref-type="bibr" rid="CR13"><u>13</u></xref>].<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35371558">35371558</a> <b>(1)</b> The rise and spread of the SARS-CoV-2 AY.122 lineage in Russia<ul><li> in <b>Nextstrain</b> nomenclature) and occur in &gt;85&#x2009;per cent of Delta samples both in Russia and globally are not shown: RdRp:G671S, exonuclease:A394V, nsp6:T77A, nsp3:A488S, nsp3:P1228L, nsp6:V120V, ORF7b:T40I, nsp3:P1469S, N:G215C, nsp4:D144D, nsp4:V167L, and nsp4:T492I.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35308526">35308526</a> <b>(0)</b> Case Report: SARS-CoV-2 Gamma Isolation From Placenta of a Miscarriage in Midwest, Brazil<ul><li> The nucleotide and amino acid features were observed by the online tools NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and CoVSurver (<ext-link xlink:href="https://mendel.bii.a-star.edu.sg/METHODS/corona/beta/" ext-link-type="uri">https://mendel.bii.a-star.edu.sg/METHODS/corona/beta/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35248659">35248659</a> <b>(3)</b> Role of genomics in combating COVID-19 pandemic<ul><li> So many user-friendly web-based tools were created to overcome the problem of data processing and interpretation, for example, Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="ir280">https://pangolin.cog-uk.io/</ext-link>) for lineage assignment, <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="ir285">https://<b>nextstrain</b>.org/</ext-link>), CoVizu (<ext-link ext-link-type="uri" xlink:href="http://filogeneti.ca/covizu/" id="ir290">http://filogeneti.ca/covizu/</ext-link>), and Microreact (<ext-link ext-link-type="uri" xlink:href="https://microreact.org/" id="ir295">https://microreact.org/</ext-link>) for data visualization.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35246548">35246548</a> <b>(2)</b> Homotypic protection against influenza in a pediatric cohort in Managua, Nicaragua<ul><li> Sequences were aligned using mafft v7.475<sup><xref ref-type="bibr" rid="CR40"><u>40</u></xref></sup> and assigned to clades based on defining amino acid positions using the align_clades.py script available from the seasonal influenza build of <b>Nextstrain</b>, with visual confirmation that viruses from the same clade clustered monophyletically using IQ-TREE 2.0.3<sup><xref ref-type="bibr" rid="CR41"><u>41</u></xref></sup>.<li> We assigned strains to major historical clades with <b>Nextstrain</b> (Augur 12.0.0, and Auspice 2.25.1).<li> [40](          <person-group person-group-type="author">            <name>              <surname>Katoh</surname>              <given-names>K</given-names>            </name>            <name>              <surname>Standley</surname>              <given-names>DM</given-names>            </name>          </person-group>          <article-title>MAFFT multiple sequence alignment software version 7: improvements in performance and usability</article-title>          <source>Mol. Biol. Evolut.</source>          <year>2013</year>          <volume>30</volume>          <fpage>772</fpage>          <lpage>780</lpage>          <pub-id pub-id-type="doi">10.1093/molbev/mst010</pub-id>        ) [41](          <person-group person-group-type="author">            <name>              <surname>Nguyen</surname>              <given-names>LT</given-names>            </name>            <name>              <surname>Schmidt</surname>              <given-names>HA</given-names>            </name>            <name>              <surname>von Haeseler</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Minh</surname>              <given-names>BQ</given-names>            </name>          </person-group>          <article-title>IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</article-title>          <source>Mol. Biol. Evolut.</source>          <year>2015</year>          <volume>32</volume>          <fpage>268</fpage>          <lpage>274</lpage>          <pub-id pub-id-type="doi">10.1093/molbev/msu300</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35245285">35245285</a> <b>(2)</b> An investigation of spatial-temporal patterns and predictions of the coronavirus 2019 pandemic in Colombia, 2020&#x2013;2021<ul><li> For clustering, they are complemented by the sequences from other geographical regions across the globe, downsampled to n = 1470 representative sequences using a country specific sub-sampling algorithm of <b>Nextstrain</b>.<li> We use the sequence subsample from <b>Nextstrain</b> (<ext-link xlink:href="http://www.<b>nextstrain</b>.org/" ext-link-type="uri">www.<b>nextstrain</b>.org</ext-link>) global analysis as of July 4, 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35309205">35309205</a> <b>(0)</b> Genomic Snapshot of SARS-CoV-2 in Migrants Entering Through Mediterranean Sea Routes<ul><li> Viral clade and lineage have been inferred using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and Pangolin (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>) assigners, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35241105">35241105</a> <b>(2)</b> Mini-XT, a miniaturized tagmentation-based protocol for efficient sequencing of SARS-CoV-2<ul><li>Phylogenetic analysis was performed using the &#x2018;<b>Nextstrain</b>&#x2019; collection of tools (<b>nextstrain</b>.cli 3.0.3) [<xref ref-type="bibr" rid="CR41"><u>41</u></xref>] to analyse our consensus fasta sequences.<li> [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35241661">35241661</a> <b>(9)</b> Antibody escape and global spread of SARS-CoV-2 lineage A.27<ul><li> A preliminary unrooted ML phylogenetic analysis of 1386 complete A.27 genomes and a representative set of 2516 sequences from an Africa-focused <b>Nextstrain</b> build, capturing the global SARS-CoV-2 diversity, showed sufficient temporal signal in a root-to-tip regression analysis.<li>To put the A.27 sequences within the global context, we combined all 851&#x2009;A.27 sequences from the RKI with 535&#x2009;A.27 sequences from GISAID, along with 2545 sequences from the Africa-focused <b>Nextstrain</b> build (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/africa">https://<b>nextstrain</b>.org/ncov/gisaid/africa</ext-link>)<sup><xref ref-type="bibr" rid="CR55"><u>55</u></xref></sup> for a total of 3907 sequences after removing duplicate entries.<li> We limited our selection of sequences from GISAID and <b>Nextstrain</b> to those collected up to June 1st to remain consistent with the sampling period of the RKI sequences.<li> isolate, following the <b>Nextstrain</b> SARS-CoV-2 workflow (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) and assuming a strict molecular clock and a skyline coalescent model.<li> Such a relatively low number of outliers is to be expected as the <b>Nextstrain</b> workflow already performs a similar data cleaning step.<li> [55](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35247550">35247550</a> <b>(1)</b> SARS-CoV-2&#x2019;s high rate of genetic mutation under immune selective pressure: from oropharyngeal B.1.1.7 to intrapulmonary B.1.533 in a vaccinated patient<ul><li>The sequences obtained from the 3 complete viral genomes were analyzed with CLC Genomics Workbench 21 (QIAGEN Aarhus, Silkeborgvej 2 Prismet DK-8000 Aarhus C, Denmark) and uploaded to PANGOLIN (<xref rid="bib0008" ref-type="bibr"><u>O'Toole&#xA0;et&#xA0;al., 2021</u></xref>) and <b>Nextstrain</b> (<xref rid="bib0005" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>) to identify and confirm the correct lineage and to GISAID (<xref rid="bib0003" ref-type="bibr"><u>Elbe&#xA0;et&#xA0;al., 2017</u></xref>) to obtain a cluster of single-nucleotide polymorphisms (SNPs) from these genomic sequences.<li> [O'Toole&#xA0;et&#xA0;al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [Hadfield&#xA0;et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Elbe&#xA0;et&#xA0;al., 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35235585">35235585</a> <b>(5)</b> Travel ban effects on SARS-CoV-2 transmission lineages in the UAE as inferred by genomic epidemiology<ul><li> Software toolkits that are available for this type of probabilistic evolutionary analyses include TreeTime and the <b>Nextstrain</b>/Augur tool suite [<xref rid="pone.0264682.ref006" ref-type="bibr"><u>6</u></xref>, <xref rid="pone.0264682.ref007" ref-type="bibr"><u>7</u></xref>].<li><b>Nextstrain</b>/Augur generates a richly annotated phylogeny (augur refine), with ancestral nodes holding information on time and location, including an estimate of confidence for both.<li> The phylogeny is visualized using Auspice (<ext-link xlink:href="https://<b>nextstrain</b>.github.io/auspice/" ext-link-type="uri">https://<b>nextstrain</b>.github.io/auspice/</ext-link>), which provides a rich set of exploration options.<li> The exact parameter settings are provided in <b>nextstrain</b>UAE.sh.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35658106">35658106</a> <b>(3)</b> Evolutionary dynamics of the severe acute respiratory syndrome coronavirus 2 genomes<ul><li> Moreover, based on these two sites and other SNVs, GISAID (<ext-link xlink:href="http://gisaid.org" ext-link-type="uri">http://gisaid.org</ext-link>) divides SARS-CoV-2 genomes into four major groups (S, L, V, and G), In contrast, <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>)&#xA0;[<xref rid="j_mr-2021-0035_ref_106" ref-type="bibr"><u>105</u></xref>] categorized SARS-CoV-2 variants into five major clades (19A, 19B, 20A, 20B, and 20C).<li> [105](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35225190">35225190</a> <b>(2)</b> Use of serial testing to interrupt a severe acute respiratory coronavirus virus 2 (SARS-CoV-2) outbreak on a hospital medical floor&#x2014;Minnesota, October&#x2013;December 2020<ul><li>Respiratory specimens positive for SARS-CoV-2 were submitted to the Minnesota Department of Health (MDH) Public Health Laboratory for WGS, which was conducted by using previously described methods.<sup><xref rid="r14" ref-type="bibr"><u>14</u></xref></sup> Phylogenetic relationships, including distinct clustering of viral whole-genome sequences,<sup><xref rid="r15" ref-type="bibr"><u>15</u></xref></sup> were inferred based on nucleotide differences via IQ-TREE (<uri xlink:href="http://www.iqtree.org/">http://www.iqtree.org/</uri>) using general time reversible substitution models<sup><xref rid="r16" ref-type="bibr"><u>16</u></xref></sup> as a part of the <b>Nextstrain</b> (https://<b>nextstrain</b>.org/) workflow.<li> [14](r14) [15](r15) [16](r16) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35324585">35324585</a> <b>(1)</b> Genetic Analysis of Influenza A/H1N1pdm Strains Isolated in Bangladesh in Early 2020<ul><li> the dataset included the most related strains obtained from a BLAST analysis conducted in EpiFlu GISAID, the earlier strains of 6B.1A5A + 187V/A and 6B.1A5A + 156K in influenza genomic epidemiology provided by <b>Nextstrain</b> in GISAID [<xref rid="B24-tropicalmed-07-00038" ref-type="bibr"><u>24</u></xref>], and the reference strains of 6B.1A and 6B.1A1&#x2013;A7.<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35231666">35231666</a> <b>(5)</b> Covidex: An ultrafast and accurate tool for SARS-CoV-2 subtyping<ul><li> Additionally, different naming systems have been developed, most notably: GISAID (<xref rid="bb0025" ref-type="bibr"><u>GISAID, 2020</u></xref>), <b>Nextstrain</b> (<xref rid="bb0010" ref-type="bibr"><u>Clade Naming for SARS-CoV-2, 2021</u></xref>) and Pango lineages (<xref rid="bb0050" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>).<li> It is an app that allows fast and accurate classification of viral genomes in all three (GISAID, <b>Nextstrain</b> and Pango) nomenclatures simultaneously.<li> All three models performed well, with an accuracy of 0.9777, 0.9952 and 0.9656 for GISAID, <b>Nextstrain</b> and Pango models, respectively (<xref rid="t0005" ref-type="table"><u>Table 1</u></xref>).<li> Additionally, multi-class AUC were even higher for GISAID and Pango models, but lower for the <b>Nextstrain</b> model.<li> Multi-class AUC is the mean AUC from all pairwise class comparisons.</p></caption><alt-text id="al0015">Table 1</alt-text><table frame="hsides" rules="groups"><thead><tr><th>Model</th><th>Number of classes</th><th>Number of trees</th><th>Training dataset size</th><th>Testing dataset size</th><th>Accuracy</th><th>Multi-class AUC</th></tr></thead><tbody><tr><td>GISAID</td><td>10</td><td>1000</td><td>66,126</td><td>13,230</td><td>0.9777</td><td>0.9931</td></tr><tr><td><b>Nextstrain</b>.<li> [GISAID, 2020](          <person-group person-group-type="author">            <name>              <surname>GISAID</surname>            </name>          </person-group>          <part-title>Clade and lineage nomenclature</part-title>          <source>Clade and Lineage Nomenclature</source>          <year>2020</year>          <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" id="ir0035">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>          <comment>(accessed 3.22.21)</comment>        ) [Clade Naming for SARS-CoV-2, 2021](          <person-group person-group-type="author">            <name>              <surname>Clade Naming for SARS-CoV-2</surname>            </name>          </person-group>          <part-title>Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstrain.org</part-title>          <year>2021</year>          <comment>(URL (accessed 3.22.20))</comment>        ) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35280916">35280916</a> <b>(1)</b> Assessment of Mutations Associated With Genomic Variants of SARS-CoV-2: RT-qPCR as a Rapid and Affordable Tool to Monitoring Known Circulating Variants in Chile, 2021<ul><li> Then, libraries with 48&#x2013;96 samples were performed according to the manufactures instructions (<xref rid="B27" ref-type="bibr"><u>27</u></xref>) and for each sequence, the clade and lineage were identified according to the nomenclature of <b>Nextstrain</b> and Pangolin, respectively.<li> [27](<person-group person-group-type="author"><collab>Technologies OxN,. Amplicon barcoding with Native Barcoding Expansion 96 (EXP-NBD196, SQK-LSK109) Protocol, Version: NBA_9102_v109_revF_09Jul2020. Oxford Nanopore Technologies</collab></person-group> (<year>2021</year>). Available from: <ext-link xlink:href="https://community.nanoporetech.com/protocols/amplicon-barcoding-with-native-" ext-link-type="uri">https://community.nanoporetech.com/protocols/amplicon-barcoding-with-native-barcoding-expansion-96-exp-nbd196-and-sqk/v/nba_9102_v109_revf_09jul2020#faq-tab&amp;</ext-link> (accessed December 9, 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35212747">35212747</a> <b>(1)</b> Transmission of and Infection With COVID-19 Among Vaccinated and Unvaccinated Attendees of an Indoor Wedding Reception in Minnesota<ul><li> The SARS-CoV-2 lineages were identified using pangolin (pangolin version 3.1.16, pangoLEARN 2021-11-18, and pango-designation version 1.2.102).<sup><xref rid="zoi220035r10" ref-type="bibr"><u>10</u></xref>,<xref rid="zoi220035r11" ref-type="bibr"><u>11</u></xref></sup> Initial phylogenetic relationships were identified via <b>Nextstrain</b>; Ultrafast Sample Placement on Existing Trees (UShER) was used to identify similar sequences; and Interactive Tree of Life (iTOL) version 6.3 was used to make the final visualization.<li> [10](<person-group><collab>StaPH-B Toolkit Documentation</collab></person-group>. Monroe. Accessed August 27, 2021. <ext-link xlink:href="https://staph-b.github.io/staphb_toolkit/workflow_docs/monroe/" ext-link-type="uri">https://staph-b.github.io/staphb_toolkit/workflow_docs/monroe/</ext-link>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35213571">35213571</a> <b>(6)</b> Whole genome sequence analysis showing unique SARS-CoV-2 lineages of B.1.524 and AU.2 in Malaysia<ul><li>Mutation analyses were carried out using Nextclade v.1.5.2, a web-based analysis server (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) by comparing against a wild-type of Wuhan-Hu-1 (NC_045512.2).<li> Based on <b>Nextstrain</b> clade analysis, there were 22 SARS-CoV-2 lineages actively dispersed in Malaysia in 2021, divided into 10 clades (<xref rid="pone.0263678.g002" ref-type="fig"><u>Fig 2A</u></xref>; <xref rid="pone.0263678.s003" ref-type="supplementary-material"><u>S2 File</u></xref>).<li>(A) <b>Nextstrain</b> clade distribution (inset) of 986 complete genomes of Malaysia SARS-CoV-2 D614G variant (filled circles) (B) <b>Nextstrain</b> clade and lineage distribution (inset) of the Pahang SARS-CoV-2 D614G variants (filled circle).<li> Please see <xref rid="pone.0263678.s003" ref-type="supplementary-material"><u>S2 File</u></xref> for the fully annotated tree (C) Phylogenetic relationship of the Pahang SARS-CoV-2 D614G variants (blue branches) with the selected neighbouring representatives (in <b>Nextstrain</b> clade tree) of SARS-CoV-2 genome sequences of African (green branches), Indonesian (red branches) and Malaysian lineages (yellow branches).<li> A separate phylogenetic analysis was conducted with neigbouring representative sequences (in <b>Nextstrain</b> clade tree) of African, Indonesian and Malaysian lineages (<xref rid="pone.0263678.g002" ref-type="fig"><u>Fig 2C</u></xref>) with Pahang SARS-CoV-2 D614G variants genomes.<li>Total 419 complete genomes of B.1.524 and 311 complete genomes of AU.2 were uploaded to Nextclade v.1.5.2 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) to analyse dominant mutations occur in spike protein of Malaysian lineages.<li>Using Nextclade v.1.5.2 for clade assignment, mutation calling and sequence quality checks (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>), 1356 complete genomes of the D614G variant were analysed for sequence quality and mutations.<li>(A) <b>Nextstrain</b> clade mutation analysis (vertical bars) of 986 Malaysian SARS-CoV-2 genomes, where Pahang SARS-CoV-2 D614G variants are highlighted in the box.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35202422">35202422</a> <b>(1)</b> Forecasting emergence of COVID-19 variants of concern<ul><li>Resources such as GISAID [<xref rid="pone.0264198.ref022" ref-type="bibr"><u>22</u></xref>] and <b>Nextstrain</b> [<xref rid="pone.0264198.ref023" ref-type="bibr"><u>23</u></xref>] have made phylogenetic analysis of tens of thousands of SARS-CoV-2 isolates available.<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35196783">35196783</a> <b>(0)</b> Spread of Gamma (P.1) Sub-Lineages Carrying Spike Mutations Close to the Furin Cleavage Site and Deletions in the N-Terminal Domain Drives Ongoing Transmission of SARS-CoV-2 in Amazonas, Brazil<ul><li> Subsequently, we evaluated the consensus files for quality using Nextclade tool v1.5.2 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>), those with more than 1% of ambiguities &#x201C;Ns&#x201D; had the FASTQ files imported into Geneious Prime 2021 for trimming and assembling using a customized workflow employing BBDuk and BBMap tools (v38.84) and the NC_045512.2 RefSeq as a template with carefully visually inspection.</ul></p>
<p><u>No PMID</u> <b>(3)</b> AI-powered drug repurposing for developing COVID-19 treatments<xref rid="fn0010" ref-type="fn">*</xref><ul><li><italic>Pathogen genome</italic></td></tr><tr><td>Pathogen genome data of SARS-CoV-2</td><td><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/help/coronavirus/SARS-CoV-2" id="ir0050">https://<b>nextstrain</b>.org/help/coronavirus/SARS-CoV-2</ext-link>.<li><italic>Pathogen genome</italic></td></tr><tr><td>Pathogen genome data of SARS-CoV-2</td><td><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/help/coronavirus/SARS-CoV-2" id="ir0050">https://<b>nextstrain</b>.org/help/coronavirus/SARS-CoV-2</ext-link></td><td><b>Nextstrain</b> is an open-source project to harness the scientific and public health potential of pathogen genome data.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35194675">35194675</a> <b>(3)</b> Misidentification of the SARS-CoV-2 Mu variant using commercial mutation screening assays<ul><li> The resulting sequences were then run on NextClade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) and Pangolin COVID-19 Lineage Assigner (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>) for final classification.<li> After our finding, we performed a search of Mu strains with the K417N mutation on <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>), where we found that sequences matching these criteria had already been included in the database since June 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35273977">35273977</a> <b>(3)</b> SARS-CoV-2 Mutations and Their Impact on Diagnostics, Therapeutics and Vaccines<ul><li> The <b>Nextstrain</b> is also an open-source database providing phylogenetic and phylodynamic analysis of SARS-CoV-2 variants grouped into clades based on the year and serial alphabets, 19A, 19B, 20A, 20B, etc.<li> (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35224469">35224469</a> <b>(0)</b> Assessment of the Abbott BinaxNOW SARS-CoV-2 rapid antigen test against viral variants of concern<ul><li><hr/></td></tr><tr><td>Rosalind Diagnostic Monitoring (DxM) system</td><td>RadX</td><td><ext-link ext-link-type="uri" xlink:href="https://radx.rosalind.bio/" id="intref0020">https://radx.rosalind.bio</ext-link></td></tr><tr><td>GraphPad Prism</td><td>GraphPad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.graphpad.com/scientific-software/prism/" id="intref0025">https://www.graphpad.com/scientific-software/prism/</ext-link></td></tr><tr><td>NextClade</td><td>NextClade</td><td><ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0030">https://clades.<b>nextstrain</b>.org/</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35224467">35224467</a> <b>(1)</b> A potent alpaca-derived nanobody that neutralizes SARS-CoV-2 variants<ul><li> The tree was generated in <b>Nextstrain</b> of all available variants with the VOCs used in this study highlighted and labeled.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35372784">35372784</a> <b>(1)</b> Sagacious epitope selection for vaccines, and both antibody-based therapeutics and diagnostics: tips from virology and oncology<ul><li> Nonetheless, in the search for conserved regions in pathogens, large sequence databases can provide some insights (such as HBVdb [<xref rid="ref64" ref-type="bibr"><u>64</u></xref>], Los Alamos Hepatitis C Virus [<xref rid="ref65" ref-type="bibr"><u>65</u></xref>], hemorrhagic fever virus [<xref rid="ref66" ref-type="bibr"><u>66</u></xref>], HIV [<xref rid="ref67" ref-type="bibr"><u>67</u></xref>, <xref rid="ref68" ref-type="bibr"><u>68&#x2013;69</u></xref>], Global Initiative on Sharing All Influenza Data [<xref rid="ref70" ref-type="bibr"><u>70</u></xref>], and <b>Nextstrain</b> [<xref rid="ref71" ref-type="bibr"><u>71</u></xref>]), but bearing in mind the constraints of extracellular targets for both neutralization of viruses for therapeutics and detection in diagnostics that do not require preprocessing to release internal intracellular contents.<li> [64](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23125365">23125365</a>) [65](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15377502">15377502</a>) [66](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22064861">22064861</a>) [67](<person-group person-group-type="author"><collab>Los Alamos National Laboratory</collab></person-group>. <source>HIV Sequence Database</source>, <year>2021</year>. [<ext-link xlink:href="http://www.hiv.lanl.gov/" ext-link-type="uri">http://www.hiv.lanl.gov/</ext-link>]) [68&#x2013;69](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12875108">12875108</a>) [70](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [71](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35342619">35342619</a> <b>(4)</b> Methods for sequencing the pandemic: benefits of rapid or high-throughput processing<ul><li>For subsequent, more sophisticated phylogenetic analyses, phylogenetic trees were constructed in <b>NextStrain</b><sup><xref rid="ref-16" ref-type="bibr"><u>16</u></xref></sup>, with genomes from GISAID subsampled by uploading our genomes of interest to the UCSC UShER (UShER: Ultrafast Sample placement on Existing tRee) tool<sup><xref rid="ref-37" ref-type="bibr"><u>37</u></xref></sup>, identifying relevant genomes (those most related to our genomes of interest), and further reducing that set of genomes when necessary with genome-sampler<sup><xref rid="ref-18" ref-type="bibr"><u>18</u></xref>,<xref rid="ref-38" ref-type="bibr"><u>38</u></xref></sup>.<li> <b>Nextstrain</b><sup><xref rid="ref-16" ref-type="bibr"><u>16</u></xref></sup> and GISAID<sup><xref rid="ref-15" ref-type="bibr"><u>15</u></xref></sup> have been crucial to the SARS-CoV-2 scientific community for global and local epidemiologic understanding, and tools for smart subsampling (e.g.<li>For retrospective studies, time can allow for more robust phylogenetic analyses including smart subsamplers<sup><xref rid="ref-38" ref-type="bibr"><u>38</u></xref></sup>, such as <b>NextStrain</b><sup><xref rid="ref-16" ref-type="bibr"><u>16</u></xref></sup> and other commonly used phylogenetic tools; however, their employment can significantly add time to a rapid response.<li> This reduces the input dataset to subsamplers such as genome-sampler<sup><xref rid="ref-38" ref-type="bibr"><u>38</u></xref></sup>, which significantly reduces the computation time for further subsampling based on geography and time, which in turn significantly reduces the computation time for a <b>NextStrain</b> analysis.<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33024970">33024970</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32887735">32887735</a>) [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33500774">33500774</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33500774">33500774</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33500774">33500774</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35265640">35265640</a> <b>(12)</b> Population Genomics Approaches for Genetic Characterization of SARS-CoV-2 Lineages<ul><li>In attempt to track the transmission and spread of emerging lineages, several lineage nomenclatures have been proposed, the most commonly used ones being the clades from <b>Nextstrain</b> and GISAID (phylogenetic approach) and Pangolin annotations (decision tree approach implemented in pangoLEARN) (<xref rid="B2" ref-type="bibr"><u>2</u></xref>).<li> The lineage with S:A222V corresponds to 20E in <b>Nextstrain</b> annotations (G in GISAID Clade annotation) and was first reported by Rambault and colleagues (<xref rid="B25" ref-type="bibr"><u>25</u></xref>, <xref rid="B26" ref-type="bibr"><u>26</u></xref>).<li> The lineage with S:501Y mainly corresponds to the <b>Nextstrain</b> 20I, now commonly known as the Alpha variant, which accumulated 22 co-occurring high frequency mutations during its expansion (<xref rid="F2" ref-type="fig"><u>Figure 2A</u></xref>) (<xref rid="B25" ref-type="bibr"><u>25</u></xref>&#x2013;<xref rid="B27" ref-type="bibr"><u>27</u></xref>).<li> After thinning using Gblocks (<xref rid="B35" ref-type="bibr"><u>35</u></xref>) to remove unresolved parts of the alignment (see Methods), the TMRCA was estimated to be in October 2019 and the mutational rate estimate was 21.60 mutations/year, in line with <b>Nextstrain</b> and other estimates (<xref rid="B36" ref-type="bibr"><u>36</u></xref>, <xref rid="B37" ref-type="bibr"><u>37</u></xref>).<li>Our haplotype definition is well in line with <b>Nextstrain</b>'s lineage definition and GISAID clade annotation system (<xref rid="SM1" ref-type="supplementary-material"><u>Supplementary Figure S2A</u></xref>), though some distinctions exist.<li> For instance, the haplotype approach can differentiate sequences from haplotype I and IV, differing by two substitutions (genomic positions 26144,14805), and from III and IX, differing at position 25583, whereas <b>Nextstrain</b> does not differentiate these lineages, grouping them into 19A and 20A, respectively.<li> These inconsistencies between the genetic background of different Pangolin lineages and the greater granularity observed compared to <b>Nextstrain</b> lineages justify the usage of haplotype categories for the analyses of genetic evolution of SARS-CoV-2.<li> These events are also consistent with observations from Singh and Yi (<xref rid="B41" ref-type="bibr"><u>41</u></xref>) who tracked the spread of the corresponding <b>Nextstrain</b> clades (XV: <b>Nextstrain</b> 20I; XVII: <b>Nextstrain</b> 20H).<li> Inspecting additional PCs allowed us to detect the emergence of subgroups in specific haplotypes (<xref rid="SM1" ref-type="supplementary-material"><u>Supplementary Figure S6</u></xref>), with, for example, a diverging group in haplotype VII (also known as GISAID L lineage and <b>Nextstrain</b> 19B lineage) which seemed to have accumulated additional mutations compared to other sequences in this lineage early on (PC13 and PC14 in <xref rid="SM1" ref-type="supplementary-material"><u>Supplementary Figure S6A</u></xref>), which could represent an early mutational jump that did not spread widely.<li> For the Pangolin or <b>Nextstrain</b> naming, it is unclear how different lineages are from each other, and, therefore, it is not obvious to know on which genomic background a lineage is occurring.<li> [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33772259">33772259</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33767447">33767447</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33772259">33772259</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33501460">33501460</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=10742046">10742046</a>) [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34506640">34506640</a>) [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33864026">33864026</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35150013">35150013</a> <b>(5)</b> Omicron BA.2 lineage spreads in clusters and is concentrated in Denmark<ul><li> This Phylogenetic tree was created using the <b>Nextstrain</b> online tool, with options such as nCoV, GISAID sequences, global sequences, PANGO Lineage, and CLOCK tree in this online tool for the Omicron variant (<ext-link xlink:href="https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/" ext-link-type="uri">https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/</ext-link>, January 14, 2022).<li> This Phylogenetic tree was created using the <b>Nextstrain</b> online tool, with options such as nCoV, GISAID sequences, global sequences, PANGO Lineage, and Rectangular tree in this online tool for the Omicron variant (<ext-link xlink:href="https://www.gisn/aid.org/phylodynamics/global/nextstrai" ext-link-type="uri">https://www.gisn/aid.org/phylodynamics/global/nextstrai</ext-link>, January 14, 2022).<li> This Phylogenetic tree was created using the <b>Nextstrain</b> online tool, with options such as nCoV, GISAID sequences, Denmark sequences, PANGO Lineage, and Rectangular tree in this online tool for the Omicron variant (<ext-link xlink:href="https://www.gisaid.org/phylodynamics/denmark/" ext-link-type="uri">https://www.gisaid.org/phylodynamics/denmark/</ext-link>, January 14, 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35178586">35178586</a> <b>(9)</b> Emergence in southern France of a new SARS-CoV-2 variant harbouring both N501Y and E484K substitutions in the spike protein<ul><li> SARS-CoV-2 genomes were classified into Nextclade and Pangolin lineages using web applications (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>; <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</ext-link>) [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>&#x2013;<xref ref-type="bibr" rid="CR11"><u>11</u></xref>].<li> Phylogenies were reconstructed using the <b>nextstrain</b>/ncov tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) and visualized using Auspice (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).<li> Phylogenetic analysis was performed using the <b>nextstrain</b>/ncov tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) and visualized using Auspice (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).<li> This figure is adapted from screenshots of the nextclade web application (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>) [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>, <xref ref-type="bibr" rid="CR10"><u>10</u></xref>].<li>Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>)&#xA0;identified a 20A lineage.<li> A phylogenetic analysis performed using the <b>nextstrain</b>/ncov tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) also showed that the B.1.640 and IHU variants were most closely related to each other but comprised two divergent branches (Fig.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](          <name>            <surname>Aksamentov</surname>            <given-names>I</given-names>          </name>          <name>            <surname>Roemer</surname>            <given-names>C</given-names>          </name>          <name>            <surname>Hodcroft</surname>            <given-names>EB</given-names>          </name>          <name>            <surname>Neher</surname>            <given-names>RA</given-names>          </name>          <year>2021</year>          <data-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</data-title>          <source>Zenodo</source>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35206580">35206580</a> <b>(2)</b> Classical and Next-Generation Vaccine Platforms to SARS-CoV-2: Biotechnological Strategies and Genomic Variants<ul><li>Real-time tracking of SARS-CoV-2 evolution and the appearance of new variants involves the use of epidemiological and molecular tools and the website <uri xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</uri> (accessed on 5 January 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35208920">35208920</a> <b>(5)</b> Genomic Diversity of SARS-CoV-2 in Algeria and North African Countries: What We Know So Far and What We Expect?<ul><li> The viral clades were assigned by the Nextclade tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> accessed on 20 December 2021) [<xref rid="B20-microorganisms-10-00467" ref-type="bibr"><u>20</u></xref>] and through the UShER web interface from the University of California, Santa Cruz (<uri xlink:href="https://genome.ucsc.edu/cgi-bin/hgPhyloPlace">https://genome.ucsc.edu/cgi-bin/hgPhyloPlace</uri> accessed on 20 December 2021).<li>The Nextclade web tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri> accessed on 20 December 2021) and the online COVID-19 genome annotator &#x2018;coronapp&#x2019; [<xref rid="B23-microorganisms-10-00467" ref-type="bibr"><u>23</u></xref>] were used to perform mutation signature calling and SNP profile defining of total genome sequences from North African countries by checking amino acid substitutions, deletion or insertion mutations on specific regions, including; spike surface glycoprotein (S), polyprotein 1ab (nsp1-nsp16), structural proteins (S, E, M, and N) and other accessory proteins.<li> Expanded phylogenetic analysis was conducted to examine the genetic divergence of Algerian samples against global representative SARS-CoV-2 genomes present in the <b>Nextstrain</b> database.<li> The <b>Nextstrain</b> clade assignment revealed that the analyzed genomes formed fifteen distinct clades with 20A, 20D and 21J (Delta) being the most common with 26.4%, 24.1% and 18.7% respectively, followed by 20I (Alpha) and 20B with 8.1%&#x2013;6.7% each.<li>Due to the naturally expanding genetic diversity of SARS-CoV-2 viruses, extensive molecular surveillance and efforts to understand the patterns of the global spread of the pandemic have been introduced including the three main nomenclatures, PANGO lineages (PANGO, Phylogenetic Assignment of Named Outbreak LINeages) [<xref rid="B19-microorganisms-10-00467" ref-type="bibr"><u>19</u></xref>], <b>Nextstrain</b> clades [<xref rid="B20-microorganisms-10-00467" ref-type="bibr"><u>20</u></xref>] and GISAID classification.<li> Given the epidemiological behaviors, current evidence supports that VOCs, including Delta and the newly emerged Omicron variant, have rapidly escalated, becoming predominant in the globe and replacing previously circulating variants (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri>) (accessed on 12 February 2022), adding up to a complex epidemiological scenario.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32793182">32793182</a>) [19](          <person-group person-group-type="author"><name><surname>Rambaut</surname><given-names>A.</given-names></name><name><surname>Loman</surname><given-names>N.</given-names></name><name><surname>Pybus</surname><given-names>O.</given-names></name><name><surname>Barclay</surname><given-names>W.</given-names></name><name><surname>Barrett</surname><given-names>J.</given-names></name><name><surname>Carabell</surname><given-names>A.</given-names></name><name><surname>Connor</surname><given-names>T.</given-names></name><name><surname>Peacock</surname><given-names>T.</given-names></name><name><surname>Robertson</surname><given-names>D.L.</given-names></name><name><surname>Volz</surname><given-names>E.</given-names></name><etal/></person-group>          <article-title>Preliminary Genomic Characterisation of an Emergent SARS-CoV-2 Lineage in the UK Defined by a Novel Set of Spike Mutations</article-title>          <comment>Available online: <ext-link xlink:href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563" ext-link-type="uri">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-12-20">(accessed on 20 December 2021)</date-in-citation>        ) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34729567">34729567</a> <b>(1)</b> Associations between SARS-CoV-2 variants and risk of COVID-19 hospitalization among confirmed cases in Washington State: a retrospective cohort study<ul><li> Sequence quality was determined using Nextclade version 1.0.1 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> These were all assigned a <b>Nextstrain</b> clade making the distinction clear (<xref rid="R15" ref-type="bibr"><u>15</u></xref>).<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34108654">34108654</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35171035">35171035</a> <b>(2)</b> Genomic Surveillance of SARS-CoV-2 Lineages Indicates Early Circulation of P.1 (Gamma) Variant of Concern in Southern Brazil<ul><li> All the amino acid mutations are based on Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).<li> accessed on <ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link> 20 July 2021) and whose significance is unknown so far.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35171960">35171960</a> <b>(7)</b> Infection clusters can elevate risk of diagnostic target failure for detection of SARS-CoV-2<ul><li>To understand the phylogenetic context of viral genomes from the Southern San Joaquin Valley Region, a phylogenetic tree of 2519 SARS-CoV-2 genomes was inferred using the <b>Nextstrain</b> tool suite [<xref rid="pone.0264008.ref017" ref-type="bibr"><u>17</u></xref>].<li> The inferential procedure was performed as follows: within <b>Nextstrain</b> Augur sequences were aligned using nextalign v0.1.6, a maximum likelihood phylogenetic tree was constructed with IQTREE v2.0.3 [<xref rid="pone.0264008.ref018" ref-type="bibr"><u>18</u></xref>], and the molecular clock was used to temporally resolve the tree with TreeTime v0.8.1 [<xref rid="pone.0264008.ref019" ref-type="bibr"><u>19</u></xref>].<li> The final trees were exported as JSON files for visualization in <b>Nextstrain</b> Auspice, a Javascript visualization package that enables interactive exploration of the <b>Nextstrain</b> phylogenetic trees.<li> Figures are adapted from <b>Nextstrain</b> visualizations presented in Auspice.<li> Most of the sequenced viruses clustered in <b>Nextstrain</b> clade 20C (120 viruses, 44%), followed by 20G (66 viruses, 24%), 20A (45 viruses, 16%), and 20B (31 viruses, 11%).<li>Temporally-resolved phylogenetic tree of 2519 SARS-CoV-2 whole genomes colored by <b>Nextstrain</b> clade designation.<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35167393">35167393</a> <b>(3)</b> An Update on the Status of Vaccine Development for SARS-CoV-2 Including Variants. Practical Considerations for COVID-19 Special Populations<ul><li> Three mainly used nomenclature systems are GISAID (Global Initiative on Sharing Avian Influenza Database), <b>Nextstrain</b> Platform and PANGOLIN (Phylogenetic Assignment of Named Global Outbreak Lineages) systems.<li> thirteen major clades (19A, 19B, 20A&#x2013;20J and 21A) have been identified in <b>Nextstrain</b> Platform and six major lineages (A, B, B.1, B.1.1, B.1.177, B.1.1.7) have been identified in PANGOLIN system.<li><b>Nextstrain</b> Clade</th>                <th align="left" rowspan="1" colspan="1">Additional Mutations</th>                <th align="left" rowspan="1" colspan="1">Earliest Documented Samples</th>                <th align="left" rowspan="1" colspan="1">Date of Designation</th>              </tr>            </thead>            <tbody>              <tr>                <td colspan="7" rowspan="1">A) Variant of Concerns (VOCs).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35242782">35242782</a> <b>(0)</b> Comparative Genomics and Characterization of SARS-CoV-2 P.1 (Gamma) Variant of Concern From Amazonas, Brazil<ul><li> The clade assignment, mutation calling, and sequence quality check were performed by the Nextclade web server (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35164530">35164530</a> <b>(5)</b> Spike mutations in SARS-CoV-2 AY sublineages of the&#xA0;Delta variant of concern: implications for the future of the pandemic<ul><li> The most commonly used in the technical literature so far have been the ones generated by GISAID (<ext-link xlink:href="http://www.gisaid.org" ext-link-type="uri">www.gisaid.org</ext-link>), <b>NextStrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>), PANGOLIN (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>)&#xA0;and Public Health England, but since May 2021 the WHO introduced a simplified nomenclature based on the Greek alphabet, focusing on variants of concern (VOC), variants of interest and variants under monitoring.<li> Such variant was originally named B.1.617.2 by PANGOLIN phylogeny, 21A/S:478K by <b>NextStrain</b>, VUI-21APR-02 by Public Health England, G/452R.V3 by GISAID&#xA0;and finally referred to as &#x2018;Delta&#x2019;&#xA0;VOC in the simplified WHO nomenclature.<li> B.1.1.529 or BA.1/BA.2&#xA0;in PANGOLIN, 21K/21L/21M in <b>NextStrain</b>, GH/484A in GISAID, or VUI-2021-NOV01 in Public Health England).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35237535">35237535</a> <b>(1)</b> SARS-CoV-2: Recent Variants and Clinical Efficacy of Antibody-Based Therapy<ul><li><bold><b>Nextstrain</b> clade</bold>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35220006">35220006</a> <b>(1)</b> A 336-nucleotide in-frame deletion in ORF7a gene of SARS-CoV-2 identified in genomic surveillance by next-generation sequencing<ul><li> The genome was classified as <b>Nextstrain</b> clade 20I <xref rid="bib0011" ref-type="bibr"><u>[11]</u></xref> from Pango lineage B.1.1.7 <xref rid="bib0012" ref-type="bibr"><u>[12]</u></xref>.<li> [[11]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[12]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35198645">35198645</a> <b>(1)</b> Assessment of Virological Contributions to COVID-19 Outcomes in a Longitudinal Cohort of Hospitalized Adults<ul><li> Phylogenetic analysis revealed that these viruses belonged to 4 primary clades, consistent with the population structure of the epidemic in Chicago during the study (<b>Nextstrain</b> clades 19A, 20A, 20B, and 20C; <xref rid="F4" ref-type="fig"><u>Figure 4A</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35494175">35494175</a> <b>(6)</b> Emergence and spread of a sub-lineage of SARS-CoV-2 Alpha variant B.1.1.7 in Europe, and with further evolution of spike mutation accumulations shared with the Beta and Gamma variants<ul><li> These are the standard quality assessment parameters utilized in NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="R1" ref-type="bibr">Aksamentov et&#xA0;al.<li>Phylogenetic analysis and phylogeographical analyses were carried out to infer the transmission routes of B.1.1.7 in Europe (<xref rid="R4" ref-type="bibr"><u>Dellicour, Rose, and Pybus 2016</u></xref>) with a custom build of the SARS-CoV-2 <b>NextStrain</b> build (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="R12" ref-type="bibr">Hadfield et&#xA0;al.<li> Briefly, it allows for the filtering of genomes, the alignment of genomes in NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="R1" ref-type="bibr">Aksamentov et&#xA0;al.<li> The subsampling of the GISAID data set for genomes was performed through a standard subsampling procedure applied in the <b>NextStrain</b> ncov pipeline (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="R12" ref-type="bibr">Hadfield et&#xA0;al.<li> [Dellicour, Rose, and Pybus 2016](<person-group person-group-type="author"><string-name><surname>Dellicour</surname> &#xA0;<given-names>S.</given-names></string-name>, <string-name><surname>Rose</surname> &#xA0;<given-names>R.</given-names></string-name>, and <string-name><surname>Pybus</surname> &#xA0;<given-names>O. G.</given-names></string-name></person-group> (<year>2016</year>) &#x2018;<article-title>Explaining the Geographic Spread of Emerging Epidemics: A Framework for Comparing Viral Phylogenies and Environmental Landscape Data</article-title>&#x2019;, <source><italic toggle="yes">BMC Bioinformatics</italic></source>, <volume>17</volume>: <page-range>82</page-range>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35214741">35214741</a> <b>(1)</b> Autochthonous Outbreak of SARS-CoV-2 Omicron Variant in Booster-Vaccinated (3 Doses) Healthcare Workers in Southern Italy: Just the Tip of the Iceberg?<ul><li> Phylogenetic analysis was performed using the Nextclade sequence analysis webapp (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 14 February 2021).<li> Clade information was described using the GISAID and <b>Nextstrain</b> nomenclature, and lineage information was described using the Pangolin nomenclature.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35216872">35216872</a> <b>(3)</b> The Coronavirus pandemic &#x2013; 2022: Viruses, variants &amp; vaccines<ul><li>Phylogenetic relationships among the main viral variants adapted from <b>Nextstrain</b> <xref rid="bib21" ref-type="bibr"><u>[21]</u></xref>, <xref rid="bib22" ref-type="bibr"><u>[22]</u></xref> (see <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org" id="ir0005"><b>nextstrain</b>.org</ext-link>).<li> [[21]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[22]](&#x201C;Nextstrain.&#x201D; <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org" id="ir0035">&#x3008;https://nextstrain.org&#x3009;</ext-link> (accessed Feb. 02, 2022).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35158042">35158042</a> <b>(0)</b> Introduction of the SARS-CoV-2 Beta variant from Comoros into the Marseille geographical area<ul><li> Assembled sequences were analysed for their classification using the Nextclade online tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0025">https://clades.<b>nextstrain</b>.org/</ext-link>) [<xref rid="bib14" ref-type="bibr"><u>14</u></xref>], the Pangolin online tool (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/pangolin.html" id="intref0030">https://cov-lineages.org/pangolin.html</ext-link>) [<xref rid="bib15" ref-type="bibr"><u>15</u></xref>], and an in-house script written in Python as previously described [<xref rid="bib2" ref-type="bibr"><u>2</u></xref>].<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35197938">35197938</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35148677">35148677</a> <b>(1)</b> Pango lineage designation and assignment using SARS-CoV-2 spike gene nucleotide sequences<ul><li> It differs from other SARS-CoV-2 nomenclature systems (such as GISAID and <b>NextStrain</b>) in that it (i) aims to capture the ongoing, leading edge of pandemic transmission, (ii) its lineages are intended to represent epidemiologically-relevant events, and (iii) it examines the phylogenetic structure of the pandemic at high resolution.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35147176">35147176</a> <b>(1)</b> Household Transmission of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Alpha Variant&#x2014;United States, 2021<ul><li> Phylogenetic relations between SARS-CoV-2 sequences within households were inferred using maximum likelihood analyses implemented in TreeTime using the <b>Nextstrain</b> pipeline [<xref rid="CIT0015" ref-type="bibr"><u>15</u></xref>].<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35685574">35685574</a> <b>(2)</b> Genomic Analysis of AZD1222 (ChAdOx1) Vaccine Breakthrough Infections in the City of Mumbai<ul><li> The <b>Nextstrain</b> tool was used for phylogenetic analysis and comparison with a previously deposited dataset of 700 SARS-CoV-2 isolates, generated by our group during the first wave of COVID-19 in Mumbai.<li> The SARS-CoV-2 isolates were assigned to <b>Nextstrain</b>-defined clades and PANGO lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35118687">35118687</a> <b>(2)</b> The emergence and epidemic characteristics of the highly mutated SARS&#x2010;CoV&#x2010;2 Omicron variant<ul><li>Genome sequencing showed that the Omicron variant belonged to Pango line B.1.1.529, <b>Nextstrain</b> branch 21k, and GISAID branch GR/484A (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>).<li> The amino acid mutations of the Omicron&#xFF08;21K&#xFF09;spike protein contain 28 amino acid substitutions, three deletions, and one insertion (A67 V, &#x394;69&#x2013;70, T95I, G142D, &#x394;143&#x2013;145, &#x394;211, L212I, ins214EPE, G339D, S371 L, S373P, S375F, K417N, N440K, G446S, S477N, T478K, E484A, Q493R, G496S, Q498R, N501Y, Y505H, T547K, D614G, H655Y, N679K, P681H, N764K, D796Y, N856K, Q954H, N969K, L981F).<xref rid="jmv27643-bib-0015" ref-type="bibr"><sup>15</sup></xref> On December&#xA0;7, the Omicron sublineage was detected in South Africa, and the <b>Nextstrain</b> defined it as 21 L, Omicron (21&#xA0;L, BA.2).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35281784">35281784</a> <b>(2)</b> SARS&#x2010;CoV&#x2010;2 Omicron variant: A next phase of the COVID&#x2010;19 pandemic and a call to arms for system sciences and precision medicine<ul><li> (A) The main spike mutations found from the isolated viruses of Omicron B.1.1.529, Copyright permission Nature 2022.<xref rid="mco2119-bib-0004" ref-type="bibr"><sup>4</sup></xref> (B) Phylogenetic analysis of SARS&#x2010;CoV&#x2010;2 clusters mutation (<italic toggle="yes">Source</italic>: https://<b>nextstrain</b>.org/groups/neherlab/ncov/21K.Omicron?l&#xA0;=&#xA0;radial).<li> (D) Illustration of spike protein mutation of Omicron (right side) in comparison of Delta (left side), Copyright permission Wiley 2022.<xref rid="mco2119-bib-0013" ref-type="bibr"><sup>13</sup></xref> (E) Frequency of infection over the time, since December 2021 Omicron is contributing 99%, and Delta is around 1% (<italic toggle="yes">Source</italic>: https://<b>nextstrain</b>.org/groups/neherlab/ncov/21K.Omicron?l&#xA0;=&#xA0;radial).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35234624">35234624</a> <b>(1)</b> Footprints of SARS-CoV-2 genome diversity in Pakistan, 2020&#x2013;2021<ul><li>Phylogenetic analysis of domestic strains using <b>NextStrain</b> (<xref rid="bib5" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) revealed the presence of 20A clade during the first wave of COVID-19.<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35139811">35139811</a> <b>(1)</b> Linked nosocomial COVID-19 outbreak in three facilities for people with intellectual and developmental disabilities due to SARS-CoV-2 variant B.1.1.519 with spike mutation T478K in the Netherlands<ul><li> The consensus sequences were used to construct a phylogenetic tree with ncov pipeline v3 of <b>nextstrain</b> with all B.1.1.519 and Dutch genomes in the Global Initiative on Sharing All Influenza Data (GISAID) (15-Apr-2021) as a reference.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35165664">35165664</a> <b>(3)</b> Omicron: A highly transmissible SARS-CoV-2 variant<ul><li>Phylogenetic relationships among SARS-CoV-2 <b>Nextstrain</b> clades (<ext-link ext-link-type="uri" xlink:href="https://covariants.org/" id="ir0005">https://covariants.org/</ext-link>).<li>Rising frequency of Omicron in comparison with other SARS-CoV-2 variants in the world (Date of access: 20 January 2022, available from <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/sars-cov-2" id="ir0010"><b>nextstrain</b>.org/sars-cov-2</ext-link>.).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35143560">35143560</a> <b>(1)</b> A curated dataset of peste des petits ruminants virus sequences for molecular epidemiological analyses<ul><li> This method has the advantage that it can be applied objectively and incorporated into a pipeline for processing new data, as has been implemented, for example, in the <b>NextStrain</b> monitoring site [<xref rid="pone.0263616.ref045" ref-type="bibr"><u>45</u></xref>].<li> [45](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35215963">35215963</a> <b>(15)</b> Characterization of the First SARS-CoV-2 Isolates from Aotearoa New Zealand as Part of a Rapid Response to the COVID-19 Pandemic<ul><li> Further phylogenetic analysis using 70 contemporary SARS-CoV-2 sequences from different lineages downloaded from the GISAID database (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>, accessed on 30 April 2020) (<xref rid="viruses-14-00366-f004" ref-type="fig"><u>Figure 4</u></xref>D), as well as blast comparisons using a series of databases, i.e., <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/">https://<b>nextstrain</b>.org/ncov/</uri>, accessed on 30 April 2020), PANGO Lineages (<uri xlink:href="https://cov-lineages.org/index.html">https://cov-lineages.org/index.html</uri>, accessed on 30 April 2020), and COVID-19 Research at UCSC (<uri xlink:href="https://hgwdev.gi.ucsc.edu/covid19.html">https://hgwdev.gi.ucsc.edu/covid19.html</uri>, accessed on 30 April 2020), classified the sequences of the first New Zealand SARS-CoV-2 isolates as <b>Nextstrain</b> Clade 19A, Pango Lineage B.55, and GISAID clade L for NZ1; <b>Nextstrain</b> Clade 19A, Pango Lineage B.31, and GISAID clade V for NZ2; <b>Nextstrain</b> Clade 20A, Pango Lineage B.1, and GISAID clade G for NZ3; <b>Nextstrain</b> Clade 20C, Pango Lineage B.1.369, and GISAID clade GH for NZ4; <b>Nextstrain</b> Clade 20C, Pango Lineage B.1, and GISAID clade GH for NZ5; <b>Nextstrain</b> Clade 19A, Pango Lineage B.31, and GISAID clade V for NZ6; and <b>Nextstrain</b> Clade 20A, Pango Lineage B.2, and GISAID clade G for NZ7 (<xref rid="viruses-14-00366-f004" ref-type="fig"><u>Figure 4</u></xref>E).<li> Nine larger clades have been described in GISAID, 19 clades by <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/">https://<b>nextstrain</b>.org/ncov/</uri> accessed on 2 June 2020), while an increasing number of PANGO lineages (<uri xlink:href="https://cov-lineages.org">https://cov-lineages.org</uri> accessed on 2 June 2020) are used to track the transmission and spread of SARS-CoV-2 worldwide, including several variants of interest (VOI) or concern (VOC) [<xref rid="B60-viruses-14-00366" ref-type="bibr"><u>60</u></xref>].<li> exonuclease proofreading ability provided by the nsp14 enzyme [<xref rid="B63-viruses-14-00366" ref-type="bibr"><u>63</u></xref>], the virus is accumulating a number of non-synonymous mutations (amino acid substitutions) across the entire genome, showing higher diversity in the spike and nucleocapsid genes (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri>, accessed on 10 January 2022), which has led to the generation and selection of novel SARS-CoV-2 variants [<xref rid="B9-viruses-14-00366" ref-type="bibr"><u>9</u></xref>,<xref rid="B13-viruses-14-00366" ref-type="bibr"><u>13</u></xref>].<li> [60](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34075212">34075212</a>) [63](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32853546">32853546</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34880490">34880490</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35169797">35169797</a> <b>(8)</b> Plasticity in structure and assembly of SARS-CoV-2 nucleocapsid protein<ul><li> Conspicuously, the Delta variant containing the G215C mutation arose without accompanying changes in the spike protein (<b>Nextstrain</b> clade 21J) and has dramatically outperformed and virtually displaced other Delta variant clades, assuming worldwide dominance in 2021 (<xref rid="R34" ref-type="bibr"><u>34</u></xref>, <xref rid="R35" ref-type="bibr"><u>35</u></xref>) (<xref rid="F1" ref-type="fig">Fig.<li>Among 2.49 million SARS-CoV-2 sequences uploaded to GISAID and subsequently preprocessed at <ext-link xlink:href="http://<b>Nextstrain</b>.org" ext-link-type="uri"><b>Nextstrain</b>.org</ext-link> (<xref rid="R13" ref-type="bibr"><u>13</u></xref>) as of November 29, 2021, we find 8.2 million instances of amino acid changes in the N-protein.<li> From inspection of phylogenetic trees at <ext-link xlink:href="http://<b>Nextstrain</b>.org" ext-link-type="uri"><b>Nextstrain</b>.org</ext-link>, a majority of mutations arose multiple times independently, but usually persisted only briefly.<li>Mutation data were based on sequenced SARS-CoV-2 genomes submitted to the GISAID (Global Initiative on Sharing All Influenza Data), and downloaded on November 29, 2021 as preprocessed file by the <b>Nextstrain</b> team (<ext-link xlink:href="http://<b>nextstrain</b>.org" ext-link-type="uri"><b>nextstrain</b>.org</ext-link>) (<xref rid="R13" ref-type="bibr"><u>13</u></xref>) containing 2.49 million sequences.<li> [34](<name><surname>Stern</surname><given-names>A</given-names></name>, <etal/> (<year>2021</year>) <article-title>The unique evolutionary dynamics of the SARS-CoV-2 Delta variant Israel Consortium of SARS-CoV-2 sequencing</article-title>. <source>medRxiv</source>:<comment>2021.08.05.21261642.</comment>) [35](<name><surname>Marchitelli</surname><given-names>V</given-names></name>, <etal/> (<year>2021</year>) <article-title>Evidence for the dependence of the SARS-Cov-2 Delta high diffusivity on the associated N&#x202F;: G215C nucleocapsid mutation</article-title>. <source>Res Sq</source>:<fpage>1</fpage>&#x2013;<lpage>9</lpage>.) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35120605">35120605</a> <b>(9)</b> SARS-CoV-2 prolonged infection during advanced HIV disease evolves extensive immune escape<ul><li> Coronavirus Typing Tool</td><td>(<xref rid="bib55" ref-type="bibr"><u>Vilsker et al., 2019</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="https://www.genomedetective.com" id="intref0045">https://www.genomedetective.com</ext-link></td></tr><tr><td>bcftools 1.7-2 mpileup</td><td>(<xref rid="bib56" ref-type="bibr"><u>Li et al., 2009</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="https://samtools.github.io/bcftools/bcftools.html" id="intref0050">https://samtools.github.io/bcftools/bcftools.html</ext-link></td></tr><tr><td>Geneious</td><td>Biomatters</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com/" id="intref0055">https://www.geneious.com/</ext-link></td></tr><tr><td>SARS-CoV-2 <b>NextStrain</b>.<li> Coronavirus Typing Tool</td><td>(<xref rid="bib55" ref-type="bibr"><u>Vilsker et al., 2019</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="https://www.genomedetective.com" id="intref0045">https://www.genomedetective.com</ext-link></td></tr><tr><td>bcftools 1.7-2 mpileup</td><td>(<xref rid="bib56" ref-type="bibr"><u>Li et al., 2009</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="https://samtools.github.io/bcftools/bcftools.html" id="intref0050">https://samtools.github.io/bcftools/bcftools.html</ext-link></td></tr><tr><td>Geneious</td><td>Biomatters</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com/" id="intref0055">https://www.geneious.com/</ext-link></td></tr><tr><td>SARS-CoV-2 <b>NextStrain</b></td><td>https://<b>nextstrain</b>.org/ncov/gisaid/global.<li> Coronavirus Typing Tool</td><td>(<xref rid="bib55" ref-type="bibr"><u>Vilsker et al., 2019</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="https://www.genomedetective.com" id="intref0045">https://www.genomedetective.com</ext-link></td></tr><tr><td>bcftools 1.7-2 mpileup</td><td>(<xref rid="bib56" ref-type="bibr"><u>Li et al., 2009</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="https://samtools.github.io/bcftools/bcftools.html" id="intref0050">https://samtools.github.io/bcftools/bcftools.html</ext-link></td></tr><tr><td>Geneious</td><td>Biomatters</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com/" id="intref0055">https://www.geneious.com/</ext-link></td></tr><tr><td>SARS-CoV-2 <b>NextStrain</b></td><td>https://<b>nextstrain</b>.org/ncov/gisaid/global</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0060">https://github.com/<b>nextstrain</b>/ncov</ext-link>.<li> We analyzed sequences from the six different time points (D6, D20, D34, D71, D106 and D190) against a global reference dataset of 3883 genomes using a custom build of the SARS-CoV-2 <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0095">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> All sequences from the participant clustered in a monophyletic clade (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/ngs-sa/COVID19-AHRI-2021.05.27?label=clade:HIV%20patient" id="intref0100">https://<b>nextstrain</b>.org/groups/ngs-sa/COVID19-AHRI-2021.05.27?label=clade:HIV%20patient</ext-link>) that are well separated from the rest of the phylogeny.<li> [Vilsker et al., 2019](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30124794">30124794</a>) [Li et al., 2009](          <person-group person-group-type="author">            <name>              <surname>Li</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Handsaker</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Wysoker</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Fennell</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Ruan</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Homer</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Marth</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Abecasis</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Durbin</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>1,000 Genome Project Data Processing</article-title>          <source>Subgroup</source>          <volume>25</volume>          <year>2009</year>          <fpage>2078</fpage>          <lpage>2079</lpage>        ) [Vilsker et al., 2019](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30124794">30124794</a>) [Li et al., 2009](          <person-group person-group-type="author">            <name>              <surname>Li</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Handsaker</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Wysoker</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Fennell</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Ruan</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Homer</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Marth</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Abecasis</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Durbin</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>1,000 Genome Project Data Processing</article-title>          <source>Subgroup</source>          <volume>25</volume>          <year>2009</year>          <fpage>2078</fpage>          <lpage>2079</lpage>        ) [Vilsker et al., 2019](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30124794">30124794</a>) [Li et al., 2009](          <person-group person-group-type="author">            <name>              <surname>Li</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Handsaker</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Wysoker</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Fennell</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Ruan</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Homer</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Marth</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Abecasis</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Durbin</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>1,000 Genome Project Data Processing</article-title>          <source>Subgroup</source>          <volume>25</volume>          <year>2009</year>          <fpage>2078</fpage>          <lpage>2079</lpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34931886">34931886</a> <b>(3)</b> Escape from recognition of SARS-CoV-2 Beta variant spike epitopes but overall preservation of T cell immunity<ul><li> Phylogenetic clade classification of the genomes in this study consisted of analyzing them against a global reference dataset using a custom pipeline based on a local version of <b>NextStrain</b> (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="R65" ref-type="bibr"><u>65</u></xref>).<li> [65](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35112360">35112360</a> <b>(4)</b> SARS&#x2010;CoV&#x2010;2 Omicron variant is spreading in different parts of the world in three different trends<ul><li> In phylogenetic analysis, the Gamma variant is grouped with the Alpha variant, but the Beta variant is significantly different from the Alpha and Gamma variants (Figure&#xA0;<xref rid="jmv27646-fig-0001" ref-type="fig"><u>1C</u></xref>) (<ext-link xlink:href="https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/" ext-link-type="uri">https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/</ext-link>; accessed January 14, 2022).<li> (C) The phylogenetic tree displays the phylogenetic relationship of different variants (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35130722">35130722</a> <b>(1)</b> Subcellular Detection of SARS-CoV-2 RNA in Human Tissue Reveals Distinct Localization in Alveolar Type 2 Pneumocytes and Alveolar Macrophages<ul><li> We used the probe designer described above to design nonoverlapping 52-mer oligonucleotides with a target Gibbs free energy for binding of &#x2212;60 (allowable Gibbs free energy [&#x2212;70, &#x2212;50]) to the N and ORF1a regions of the SARS-CoV-2 genome, targeting only the 3,000- to 8,000-nucleotide region of the latter because it was the most conserved region among the strains circulating at the time as determined using <b>nextstrain</b> (<xref rid="B44" ref-type="bibr"><u>44</u></xref>).<li> [44](<collab>Nextstrain</collab>. <year>2021</year>. <article-title>Nextstrain</article-title>. <ext-link xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35136154">35136154</a> <b>(0)</b> Development and validation of a high throughput SARS-CoV-2 whole genome sequencing workflow in a clinical laboratory<ul><li><sup>a</sup>SARS-CoV-2 clades were assigned with Nextclade v1.3.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li><sup>a</sup>SARS-CoV-2 clades were assigned with Nextclade v1.3.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) and lineages were assigned with Pangolin v3.1.11 (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>).<li><sup>b</sup>SARS-CoV-2 clades were assigned with Nextclade v1.3.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35197938">35197938</a> <b>(16)</b> Analysis of SARS-CoV-2 Variants From 24,181 Patients Exemplifies the Role of Globalization and Zoonosis in Pandemics<ul><li> Several SARS-CoV-2 specific classification and naming systems exist including those of GISAID<sup><xref rid="footnote3" ref-type="fn"><u>3</u></xref></sup> (<xref rid="B3" ref-type="bibr"><u>Alm et al., 2020</u></xref>), <b>Nextstrain</b><sup><xref rid="footnote4" ref-type="fn"><u>4</u></xref></sup> (<xref rid="B34" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="B2" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>), Pangolin<sup><xref rid="footnote5" ref-type="fn"><u>5</u></xref></sup> (<xref rid="B58" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), or the World Health Organization (WHO)<sup><xref rid="footnote6" ref-type="fn"><u>6</u></xref></sup>.<li> Classification and naming systems have been specifically proposed for SARS-CoV-2 including those of GISAID (see footnote 3) (<xref rid="B3" ref-type="bibr"><u>Alm et al., 2020</u></xref>), <b>Nextstrain</b> (see footnote 4) (<xref rid="B34" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="B2" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>), Pangolin (see footnote 5) (<xref rid="B58" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), and the WHO (see footnote 6) (<xref rid="boxenv-star-2" ref-type="boxed-text"><u>Box 2</u></xref>).<li><bold><b>Nextstrain</b> classification (<ext-link xlink:href="https://<b>nextstrain</b>.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming" ext-link-type="uri">https://<b>nextstrain</b>.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>) (<xref rid="B34" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="B2" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>):</bold> Defines major clades that reach &#x2265; 20% global frequency for &#x2265; two months, &#x2265; 30% regional frequency for &#x2265; two months, or correspond to a &#x2018;variant of concern&#x2019; (VOC).<li> Phylogeny reconstruction was performed using the <b>nextstrain</b>/ncov tool (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) then visualized with Auspice (<ext-link xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/" ext-link-type="uri">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).<li> Marseille variants or WHO clades, and Pangolin, or <b>Nextstrain</b> clades are indicated.<li><b>Nextstrain</b> clade.<li> They corresponded to <b>Nextstrain</b> clades 19A and 20B that briefly co-circulated in our geographical area during the first period of the SARS-CoV-2 epidemic (<xref rid="F5" ref-type="fig"><u>Figures 5</u></xref>&#x2013;<xref rid="F7" ref-type="fig"><u>7</u></xref>, <xref rid="SM1" ref-type="supplementary-material"><u>Supplementary Figure 4</u></xref>, and <xref rid="SM1" ref-type="supplementary-material"><u>Supplementary Table 3</u></xref>; <xref rid="B38" ref-type="bibr"><u>IHU M&#xE9;diterran&#xE9;e Infection, 2021a</u></xref>,<xref rid="B39" ref-type="bibr"><u>b</u></xref>).<li> Compared with viruses of the <b>Nextstrain</b> clade 20A that circulated between February and May 2020 in Marseille, patients infected with the Marseille-1 (B.1.416) variant were less likely to report dyspnea and rhinitis and to be hospitalized (<xref rid="B16" ref-type="bibr"><u>Dao et al., 2021</u></xref>).<li> In addition, patients infected with the Marseille-4 variant (B.1.160) were more likely to exhibit fever and to be hospitalized than those infected with the Marseille-1 variant, and they were also more likely to exhibit fever than those infected with a virus of <b>Nextstrain</b> clade 20A that circulated between February and May 2020 in Marseille.<li> Also, patients infected with N501Y-harboring SARS-CoV-2 variants [namely, Alpha (B.1.1.7), Beta (B.1.351), and Gamma (P.1) variants] were less likely to be hospitalized than those infected with the Marseille-4 variant and with viruses of <b>Nextstrain</b> clade 20A that circulated between February and May 2020 in Marseille (<xref rid="B16" ref-type="bibr"><u>Dao et al., 2021</u></xref>).<li> <b>Nextstrain</b> clades and Pangolin lineages provided in the present study were determined with the Nextclade web application (see footnote 4) (<xref rid="B34" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="B2" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) and the Pangolin web application (see footnote 5) (<xref rid="B58" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), respectively.<li>Phylogeny reconstructions based on the SARS-CoV-2 genomes obtained in our laboratory were performed for genome sequences larger than 24,000 nucleotides obtained with the Illumina technology using the <b>nextstrain</b>/ncov tool<sup><xref rid="footnote17" ref-type="fn"><u>17</u></xref></sup> then visualized with Auspice<sup><xref rid="footnote18" ref-type="fn"><u>18</u></xref></sup> or FigTree v1.4.4<sup><xref rid="footnote19" ref-type="fn"><u>19</u></xref></sup>; the limitation of sampling per date of collection of respiratory specimens was deactivated.<li> [3](<a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) [Alm et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) [4](<a href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><name><surname>Neher</surname><given-names>R. A.</given-names></name></person-group> (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J. Open Source Softw.</italic></source><volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>) [5](<a href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</a>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [6](<a href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</a>) [Alm et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><name><surname>Neher</surname><given-names>R. A.</given-names></name></person-group> (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J. Open Source Softw.</italic></source><volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><name><surname>Neher</surname><given-names>R. A.</given-names></name></person-group> (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J. Open Source Softw.</italic></source><volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>) [IHU M&#xE9;diterran&#xE9;e Infection, 2021a](<collab>IHU M&#xE9;diterran&#xE9;e Infection</collab> (<year>2021a</year>). <source><italic>Evolution of the Weekly Incidence of SARS-CoV-2 Mutants and Variants Extrapolated to the Total Number of Cases, Based on Their Proportions of Genotyped Cases.</italic></source><publisher-loc>Marseille</publisher-loc>: <publisher-name>HU M&#xE9;diterran&#xE9;e Infection</publisher-name>. Available online at: <pub-id pub-id-type="doi">10.35081/ffd4-1y77</pub-id><comment>(accessed September 9, 2021)</comment>.) [b](<collab>IHU M&#xE9;diterran&#xE9;e Infection</collab> (<year>2021b</year>). <source><italic>Introduction of SARS-CoV-2 Mutants and Variants in the Marseille Geographical Area Through Travel From Abroad.</italic></source><publisher-loc>Marseille</publisher-loc>: <publisher-name>HU M&#xE9;diterran&#xE9;e Infection</publisher-name>. Available online at: <pub-id pub-id-type="doi">10.35081/nz2g-f980</pub-id><comment>(accessed September 9, 2021)</comment>.) [Dao et al., 2021](<person-group person-group-type="author"><name><surname>Dao</surname><given-names>T. L.</given-names></name><name><surname>Hoang</surname><given-names>V. T.</given-names></name><name><surname>Nguyen</surname><given-names>N. N.</given-names></name><name><surname>Delerce</surname><given-names>J.</given-names></name><name><surname>Chaudet</surname><given-names>H.</given-names></name><name><surname>Levasseur</surname><given-names>A.</given-names></name><etal/></person-group> (<year>2021</year>). <article-title>Clinical outcomes in COVID-19 patients infected with different SARS-CoV-2 variants in Marseille, France.</article-title><source><italic>Clin. Microbiol. Infect.</italic></source><volume>10</volume><fpage>1516.e1</fpage>&#x2013;<lpage>1516.e6</lpage>.) [Dao et al., 2021](<person-group person-group-type="author"><name><surname>Dao</surname><given-names>T. L.</given-names></name><name><surname>Hoang</surname><given-names>V. T.</given-names></name><name><surname>Nguyen</surname><given-names>N. N.</given-names></name><name><surname>Delerce</surname><given-names>J.</given-names></name><name><surname>Chaudet</surname><given-names>H.</given-names></name><name><surname>Levasseur</surname><given-names>A.</given-names></name><etal/></person-group> (<year>2021</year>). <article-title>Clinical outcomes in COVID-19 patients infected with different SARS-CoV-2 variants in Marseille, France.</article-title><source><italic>Clin. Microbiol. Infect.</italic></source><volume>10</volume><fpage>1516.e1</fpage>&#x2013;<lpage>1516.e6</lpage>.) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><name><surname>Neher</surname><given-names>R. A.</given-names></name></person-group> (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J. Open Source Softw.</italic></source><volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [17](<a href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</a>) [18](<a href="https://docs.nextstrain.org/projects/auspice/en/stable/">https://docs.nextstrain.org/projects/auspice/en/stable/</a>) [19](<a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35206930">35206930</a> <b>(3)</b> Reshaping the Healthcare Sector with Economic Policy Measures Based on COVID-19 Epidemic Severity: A Global Study<ul><li> Variables on health outcomes were obtained from WHO&#x2019;s Global Health Observatory data repository <uri xlink:href="https://apps.who.int/gho/data/node.main">https://apps.who.int/gho/data/node.main</uri> (accessed on 10 December 2020) and <b>Nextstrain</b> <uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri> (Hadfield et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35156006">35156006</a> <b>(2)</b> Epidemiology and genetic diversity of SARS-CoV-2 lineages circulating in Africa<ul><li>Worldwide, we observed 782 Pango lineages, nine GISAID clades and 10 <b>Nextstrain</b> clades.<li>Using <b>Nextstrain</b> clade nomenclature (<xref rid="mmc1" ref-type="supplementary-material"><u>Figure&#xA0;S3</u></xref>), we observed that 20A.EU1 was the dominant circulating clade in Europe followed by 20B and 20A.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35156075">35156075</a> <b>(1)</b> Seroprevalence of anti-SARS-CoV-2 antibodies in Colombia, 2020: A population-based study<ul><li> The sequences were compiled and combined with representative genome sequences from the South America-focused subsampling available from <b>NextStrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35123472">35123472</a> <b>(2)</b> <italic>Escherichia coli</italic> recombinant expression of SARS-CoV-2 protein fragments<ul><li> Spike protein entropy/diversity data from the <b>Nextstrain</b> SARS-CoV-2 latest global analysis open data was downloaded on January 6th 2022 and plotted using GraphPad Prism 9 [<xref ref-type="bibr" rid="CR37"><u>37</u></xref>].<li> As the COVID-19 pandemic has progressed many amino acid changes have been observed in the SARS-CoV-2 spike protein, and spike protein amino acid diversity data, obtained from <b>Nextstrain</b>.org [<xref ref-type="bibr" rid="CR37"><u>37</u></xref>], is shown in Fig.&#xA0;<xref rid="Fig1" ref-type="fig"><u>1</u></xref>E.<li> [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35123509">35123509</a> <b>(4)</b> The molecular determinants of antigenic drift in a novel avian influenza A (H9N2) variant virus<ul><li>A time-scaled phylogeny of H9N2 HA was analyzed using the <b>Nextstrain</b> platform [<xref ref-type="bibr" rid="CR24"><u>24</u></xref>] (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/avian-flu">https://github.com/<b>nextstrain</b>/avian-flu</ext-link>).<li> Tree and visualization were generated using the <b>Nextstrain</b> platform (see <xref rid="Sec3" ref-type="sec"><u>Methods</u></xref>).<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35215919">35215919</a> <b>(2)</b> Comparison of SARS-CoV-2 Evolution in Paediatric Primary Airway Epithelial Cell Cultures Compared with Vero-Derived Cell Lines<ul><li> However, non-P residues (e.g., serine) are occasionally found in nature but are rare (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 7 December 2021), which suggests a functional defect in vivo.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35050521">35050521</a> <b>(2)</b> Transmission dynamics and mutational prevalence of the novel Severe acute respiratory syndrome coronavirus&#x2010;2 Omicron Variant of Concern<ul><li> SARS&#x2010;CoV&#x2010;2 variants are categorized as Variant of Concern (VOC), Variant of Interest (VOI), and Variants under monitoring (VUM) to focus global research and awareness, as well as to lead the on&#x2010;going worldwide responses to the COVID&#x2010;19 pandemic.<xref rid="jmv27611-bib-0002" ref-type="bibr"><sup>2</sup></xref>, <xref rid="jmv27611-bib-0003" ref-type="bibr"><sup>3</sup></xref> Additionally, the variants have been classified based on the genomic sequencing data available at The Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN), Global Initiative on Sharing All Influenza Data (GISAID), and <b>Nextstrain</b>.<li>To determine the Omicron SARS&#x2010;CoV&#x2010;2 divergence, we employed the <b>Nextstrain</b>, which offers the most recent worldwide genomic sequencing of the SARS&#x2010;CoV&#x2010;2 data as soon as it is released by GISAID (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35242361">35242361</a> <b>(0)</b> Limited genomic reconstruction of SARS-CoV-2 transmission history within local epidemiological clusters<ul><li> We assigned the consensus sequences to a SARS-CoV-2 clade with Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>; last accessed 20 December 2021) and to a SARS-CoV-2 PANGO lineage (<xref rid="R54" ref-type="bibr">Rambaut et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35051367">35051367</a> <b>(13)</b> Transmission from vaccinated individuals in a large SARS-CoV-2 Delta variant outbreak<ul><li><hr/></td></tr><tr><td>LoFreq version 2.1.5</td><td><xref rid="bib27" ref-type="bibr"><u>Wilm et&#xA0;al., 2012</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://csb5.github.io/lofreq/" id="intref0025">https://csb5.github.io/lofreq/</ext-link></td></tr><tr><td>viral-ngs 2.1.28</td><td>Broad Institute</td><td><ext-link ext-link-type="uri" xlink:href="https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master" id="intref0030">https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master</ext-link></td></tr><tr><td><b>Nextstrain</b> v3.0.3.<li><hr/></td></tr><tr><td>LoFreq version 2.1.5</td><td><xref rid="bib27" ref-type="bibr"><u>Wilm et&#xA0;al., 2012</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://csb5.github.io/lofreq/" id="intref0025">https://csb5.github.io/lofreq/</ext-link></td></tr><tr><td>viral-ngs 2.1.28</td><td>Broad Institute</td><td><ext-link ext-link-type="uri" xlink:href="https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master" id="intref0030">https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master</ext-link></td></tr><tr><td><b>Nextstrain</b> v3.0.3</td><td><xref rid="bib12" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_<b>nextstrain</b>_aligned_input:master" id="intref0035">https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_<b>nextstrain</b>_aligned_input:master</ext-link>.<li><hr/></td></tr><tr><td>LoFreq version 2.1.5</td><td><xref rid="bib27" ref-type="bibr"><u>Wilm et&#xA0;al., 2012</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://csb5.github.io/lofreq/" id="intref0025">https://csb5.github.io/lofreq/</ext-link></td></tr><tr><td>viral-ngs 2.1.28</td><td>Broad Institute</td><td><ext-link ext-link-type="uri" xlink:href="https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master" id="intref0030">https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master</ext-link></td></tr><tr><td><b>Nextstrain</b> v3.0.3</td><td><xref rid="bib12" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_<b>nextstrain</b>_aligned_input:master" id="intref0035">https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_<b>nextstrain</b>_aligned_input:master</ext-link></td></tr><tr><td>FastTree version 2.1.11</td><td><xref rid="bib19" ref-type="bibr"><u>Price et&#xA0;al., 2009</u></xref> and <xref rid="bib20" ref-type="bibr"><u>2010</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="http://microbesonline.org/fasttree/" id="intref0040">http://www.microbesonline.org/fasttree/</ext-link></td></tr><tr><td>baltic</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/evogytis/baltic" id="intref0045">https://github.com/evogytis/baltic</ext-link></td></tr><tr><td>EpiEstim v4.0.1</td><td><xref rid="bib7" ref-type="bibr"><u>Cori et&#xA0;al., 2013</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/EpiEstim/index.html" id="intref0050">https://cran.r-project.org/web/packages/EpiEstim/index.html</ext-link></td></tr><tr><td>outbreaker2 (version 1.1.2)</td><td><xref rid="bib3" ref-type="bibr"><u>Campbell et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/outbreaker2/index.html" id="intref0055">https://cran.r-project.org/web/packages/outbreaker2/index.html</ext-link></td></tr><tr><td>Nextclade CLI version 1.3.0</td><td><xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="intref0060">https://github.com/<b>nextstrain</b>/nextclade</ext-link>.<li>We constructed a maximum-likelihood (ML) phylogenetic tree (<xref rid="bib22" ref-type="bibr"><u>Sagulenko et&#xA0;al., 2018</u></xref>) with associated visualizations using a SARS-CoV-2-tailored Augur pipeline (<xref rid="bib14" ref-type="bibr"><u>Huddleston et&#xA0;al., 2021</u></xref>) (<italic>sarscov2_<b>nextstrain</b>_aligned_input</italic>), part of the <b>Nextstrain</b> project (<xref rid="bib12" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>), adapted from <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0080">github.com/<b>nextstrain</b>/ncov</ext-link>, with the entirety of ARTICv3 amplicons 64, 72, and 73 (Delta dropout regions) masked from tree construction.<li> First, we used a focal weighted subsampling scheme (using the <b>Nextstrain</b> script priorities.py) to prioritize genomes genetically, geographically, and temporally close to our outbreak-associated genomes (<ext-link ext-link-type="uri" xlink:href="https://github.com/broadinstitute/<b>nextstrain</b>-builds/blob/main/builds/broad-usa-builds.yaml" id="intref0085">https://github.com/broadinstitute/<b>nextstrain</b>-builds/blob/main/builds/broad-usa-builds.yaml#L125</ext-link>).<li> To do so, we assigned a binary trait to each genome in the phylogeny, outbreak-associated or not outbreak-associated, and used <b>Nextstrain</b>&#x2019;s ancestral inference (<xref rid="bib12" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) to infer the state of that trait for each internal node in the tree.<li> [Wilm et&#xA0;al., 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23066108">23066108</a>) [Wilm et&#xA0;al., 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23066108">23066108</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Wilm et&#xA0;al., 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23066108">23066108</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Price et&#xA0;al., 2009](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19377059">19377059</a>) [2010](          <person-group person-group-type="author">            <name>              <surname>Price</surname>              <given-names>M.N.</given-names>            </name>            <name>              <surname>Dehal</surname>              <given-names>P.S.</given-names>            </name>            <name>              <surname>Arkin</surname>              <given-names>A.P.</given-names>            </name>          </person-group>          <article-title>FastTree 2--approximately maximum-likelihood trees for large alignments</article-title>          <source>PLOS One</source>          <volume>5</volume>          <year>2010</year>          <object-id pub-id-type="publisher-id">e9490</object-id>          <pub-id pub-id-type="doi">10.1371/journal.pone.0009490</pub-id>        ) [Cori et&#xA0;al., 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24043437">24043437</a>) [Campbell et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30343663">30343663</a>) [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [Sagulenko et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Huddleston et&#xA0;al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35124261">35124261</a> <b>(1)</b> Neutralizing antibody activity against the B.1.617.2 (delta) variant 8&#xA0;months after two-dose vaccination with BNT162b2 in health care workers<ul><li> For the B.1.617.2 (delta) variant, next-generation sequencing (NGS) sequencing was performed, and lineage, variant assignment, and clade classification of the SARS-CoV-2 sequence was carried out using Pangolin and Nextclade as part of the <b>Nextstrain</b> framework.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35231807">35231807</a> <b>(1)</b> Emergence and onward transmission of a SARS-CoV-2 E484K variant among household contacts of a bamlanivimab-treated patient<ul><li> For phylogenetic inference (i.e., to determine the hierarchy of case relationships), sequences were integrated with associated metadata and aligned on a local implementation of <b>NextStrain</b> <xref rid="bib0027" ref-type="bibr"><u>[27]</u></xref> using augur and displayed via a web browser using auspice.<li> [[27]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35060146">35060146</a> <b>(1)</b> Characteristics of the first 1119 SARS&#x2010;CoV&#x2010;2 Omicron variant cases, in Marseille, France, November&#x2212;December 2021<ul><li> Mutation detection was performed with Nextclade web application (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>)<xref rid="jmv27613-bib-0008" ref-type="bibr"><sup>8</sup></xref> and freebayes (<ext-link xlink:href="https://github.com/freebayes/freebayes" ext-link-type="uri">https://github.com/freebayes/freebayes</ext-link>).<li> <b>Nextstrain</b> clades and Pangolin lineages were determined using Nextclade and Pangolin web application (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>),<xref rid="jmv27613-bib-0010" ref-type="bibr"><sup>10</sup></xref> respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35112893">35112893</a> <b>(3)</b> SARS-CoV-2 Genome Sequence Obtained from Ethiopia<ul><li>A phylogenetic analysis was conducted on the 15 sequenced samples using the <b>Nextstrain</b> bioinformatics platform (<xref rid="B9" ref-type="bibr"><u>9</u></xref>, <xref rid="B10" ref-type="bibr"><u>10</u></xref>).<li> A <b>Nextstrain</b> analysis reveals that SARS-CoV-2 samples from Ethiopia belong to three different <b>Nextstrain</b> clades, as follows: 19B, 20A, and 20I/501Y.V1.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33509990">33509990</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35112895">35112895</a> <b>(1)</b> Coding-Complete Genome Sequences of a Delta Subvariant (AY.33) of SARS-CoV-2 Obtained from Moroccan COVID-19 Patients<ul><li> The quality of the sequences was monitored using the Nextclade web tool version 1.7.2 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>), although the determination of the mutations was carried out using the web application Cov-GLUE (<ext-link xlink:href="http://cov-glue.cvr.gla.ac.uk/" ext-link-type="uri">http://cov-glue.cvr.gla.ac.uk/</ext-link>), and the lineage and sublineage were obtained using the web application Pangolin (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>).<li>Phylogenetic tree of SARS-CoV-2 generated from <b>Nextstrain</b> using the Nextclade maximum likelihood module.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35115515">35115515</a> <b>(1)</b> Multiple expansions of globally uncommon SARS-CoV-2 lineages in Nigeria<ul><li> Viruses from lineage A and derivates spread throughout the world early in the pandemic, but are now relatively rare globally, accounting for &lt;0.5% of isolates sequenced and deposited in August 2021 (<b>nextstrain</b>.org).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35134129">35134129</a> <b>(0)</b> Prolonged Shedding of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) at High Viral Loads Among Hospitalized Immunocompromised Persons Living With Human Immunodeficiency Virus (HIV), South Africa<ul><li> Clade and lineage assignments were made using the online Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and Pangolin (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) applications, which also enable identification of known variants of concern as well as novel mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35123044">35123044</a> <b>(0)</b> Clinical and genomic data of sars-cov-2 detected in maternal&#x2013;fetal interface during the first wave of infection in Brazil<ul><li> v.3.1.14) and Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0010">https://clades.<b>nextstrain</b>.org/</ext-link>; v.1.5.4).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35106601">35106601</a> <b>(1)</b> Ongoing Recombination in SARS-CoV-2 Revealed through Genealogical Reconstruction<ul><li> We used a SARS-CoV-2 phylogeny from <b>Nextstrain</b> (<xref rid="msac028-B6" ref-type="bibr">Hadfield et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35123068">35123068</a> <b>(1)</b> SARS-CoV-2 RT-PCR positivity of individuals subsequent to completing quarantine upon entry into a country during a transmission-free period<ul><li> Emerging time and the associating lineage of variants were determined by <b>Nextstrain</b> and Global Evaluation of SARS-CoV-2 Sequences (GESS).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35134624">35134624</a> <b>(5)</b> Analysis of SARS-CoV-2 synonymous codon usage evolution throughout the COVID-19 pandemic<ul><li> Similar results were observed using the <b>Nextstrain</b> web portal, with selected data from GISAID and coloring by emerging lineage and date respectively (<xref rid="appsec1" ref-type="sec">Fig.<li> S1</xref>, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="intref0055">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>, accessed 2021-09-22) (<xref rid="bib27" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/open/north-america" id="intref0075">https://<b>nextstrain</b>.org/ncov/open/north-america</ext-link>.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35105380">35105380</a> <b>(0)</b> Neutrophils predominate the immune signature of cerebral thrombi in COVID-19 stroke patients<ul><li>Importantly, we further confirmed the viral PCR detection by sequencing, which revealed presence of the SARS-CoV-2 clade 20A, identified by NextClade v 1.6.0 (clades.<b>nextstrain</b>.org) (data not shown).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34906290">34906290</a> <b>(2)</b> SARS-CoV-2 Circulation, Guinea, March 2020&#x2013;July 2021<ul><li> We analyzed the remaining 136 (54%) sequences by using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) and Pangolin software (<ext-link xlink:href="https://cov-lineages.org" ext-link-type="uri">https://cov-lineages.org</ext-link>).<li>*Clades and lineages are respectively assigned according to Nextclade definition (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/docs/src/reference/naming_clades.md" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov/blob/master/docs/src/reference/naming_clades.md</ext-link>) and PANGO lineages list (<ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link>) at the same assignment date (August 14, 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35144878">35144878</a> <b>(2)</b> A SARS-CoV-2 Delta variant containing mutation in the probe binding region used for RT-qPCR test in Japan exhibited atypical PCR amplification and might induce false negative result<ul><li> Then, the resulting 250 viral genomes containing the G29234A mutation were added to the <b>Nextstrain</b> build [<xref rid="bib35" ref-type="bibr"><u>35</u></xref>] as &#x2018;focal&#x2019; sequences.<li><b>Nextstrain</b> [<xref rid="bib35" ref-type="bibr"><u>35</u></xref>] generated phylogenetic tree with viral strains containing A allele at 29234 loci of the SARS-CoV-2 genome (highlighted in orange).<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35105893">35105893</a> <b>(14)</b> SARS-CoV-2 genomes from Saudi Arabia implicate nucleocapsid mutations in host response and increased viral load<ul><li>A maximum-likelihood phylogenetic analysis revealed that samples from Saudi Arabia represent five major <b>Nextstrain</b> clades<sup><xref ref-type="bibr" rid="CR12"><u>12</u></xref></sup>, 19A-B and 20A-C (Fig.&#xA0;<xref rid="Fig2" ref-type="fig"><u>2a</u></xref>).<li> The approximate relationships between <b>Nextstrain</b> clades and Pangolin lineages<sup><xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup> are listed in Supplementary Table&#xA0;<xref rid="MOESM1" ref-type="media"><u>S4</u></xref>).<li><bold>a</bold> Global Time-scaled phylogeny of 952 Saudi samples colored by <b>Nextstrain</b> clades.<li> <bold>b</bold> Distributions of importation dates for the five <b>Nextstrain</b> (<b>nextstrain</b>.org) clades found in Saudi Arabia colored by clade.<li>A dated phylogeny of global samples showed that samples with the R203K/G204R SNPs are predominantly found in <b>Nextstrain</b> clades 20A, 20B, and 20C, and do not form a monophyletic group (Supplementary Fig.&#xA0;<xref rid="MOESM1" ref-type="media"><u>S4</u></xref>).<li>To generate the phylogeny of Saudi samples with a global context, a total of 308,012 global sequences were downloaded from GISAID on 31 December 2020, filtered and processed using <b>Nextstrain</b> pipeline<sup><xref ref-type="bibr" rid="CR12"><u>12</u></xref></sup>.<li> <b>Nextstrain</b> protocol was followed for the above-mentioned steps.<li>Importation date estimates for <b>Nextstrain</b> clades.<li> Sequences were grouped by <b>Nextstrain</b> clade for analysis using the <b>Nextstrain</b>_clade parameter<sup><xref ref-type="bibr" rid="CR12"><u>12</u></xref></sup> in the GISAID metadata table.<li> Additional international sequences were selected for each of the included <b>Nextstrain</b> clade based on Tamura Nei 93 distance with the C program tn93 (v1.0.6) (github.com/veg/tn93)<sup><xref ref-type="bibr" rid="CR70"><u>70</u></xref></sup>.<li>For each <b>Nextstrain</b> clade a maximum-likelihood phylogeny was produced with IQTree (v1.6.12) with an HKY substitution model<sup><xref ref-type="bibr" rid="CR67"><u>67</u></xref>,<xref ref-type="bibr" rid="CR71"><u>71</u></xref></sup>.<li> The variants were queried to count the distribution of triplets among various <b>Nextstrain</b> clades (Supplementary Fig.&#xA0;<xref rid="MOESM1" ref-type="media"><u>S6</u></xref>).<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [70](          <person-group person-group-type="author">            <name>              <surname>Tamura</surname>              <given-names>K</given-names>            </name>            <name>              <surname>Nei</surname>              <given-names>M</given-names>            </name>          </person-group>          <article-title>Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees</article-title>          <source>Mol. Biol. evolution</source>          <year>1993</year>          <volume>10</volume>          <fpage>512</fpage>          <lpage>526</lpage>        ) [67](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [71](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=3934395">3934395</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35198984">35198984</a> <b>(1)</b> Structural Comparison and Drug Screening of Spike Proteins of Ten SARS-CoV-2 Variants<ul><li>SARS-CoV-2 Delta (B.1.617.2) strain was first identified in India in December 2020, which has become a major epidemic strain worldwide accounting for more than 80% of new cases (<b>nextstrain</b>/ncov) [<xref rid="B47" ref-type="bibr"><u>47</u></xref>].<li> [47](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35174189">35174189</a> <b>(2)</b> Viral Load in COVID-19 Patients: Implications for Prognosis and Vaccine Efficacy in the Context of Emerging SARS-CoV-2 Variants<ul><li> In many Sub-Sahara Africa countries, the beta variant was responsible for more than 50% of infections (<ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35101081">35101081</a> <b>(3)</b> VirStrain: a strain identification tool for RNA viruses<ul><li> Available tools and websites that can monitor the mutations in strain genomes such as NextFlu and <b>NextStrain</b> [<xref ref-type="bibr" rid="CR3"><u>3</u></xref>, <xref ref-type="bibr" rid="CR15"><u>15</u></xref>] also take genomes as input and assign the genome into major genetic groups.<li> In the output of VirStrain, this first reference genome is located in a cluster with other 47 SARS-CoV-2 strains, which all belong to clade 19A defined by <b>nextstrain</b>.<li> Strain1 belongs to clade 20C and strain2 belongs to clade 20B defined by <b>nextstrain</b> [<xref ref-type="bibr" rid="CR3"><u>3</u></xref>].<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35215887">35215887</a> <b>(5)</b> The Spread of SARS-CoV-2 Variant Omicron with a Doubling Time of 2.0&#x2013;3.3 Days Can Be Explained by Immune Evasion<ul><li>Omicron, the novel SARS-CoV-2 Variant of Concern (VOC, Pango lineage B.1.1.529, <b>Nextstrain</b> clade identifier 21K) was first collected in South Africa on 2 November 2021, (GISAID [<xref rid="B1-viruses-14-00294" ref-type="bibr"><u>1</u></xref>] sequence accession ID: EPI_ISL_8182767).<li> Altogether, Omicron has 51 amino-acid level mutations, and its closest known sibling has 15 mutations (GISAID sequence accession ID: EPI_ISL_622806) with only nine common mutations, implying a distance of 42 mutations from the last common ancestor (based on the phylogenetic tree generated by <b>Nextstrain</b> [<xref rid="B3-viruses-14-00294" ref-type="bibr"><u>3</u></xref>]).<li> The Omicron phylogenetic tree (with last update on 3 January 2022) was retrieved from <b>Nextstrain</b> [<xref rid="B4-viruses-14-00294" ref-type="bibr"><u>4</u></xref>]; sample collection dates of genome sequences selected by <b>Nextstrain</b> were retrieved from GISAID.<li> This is lower than the average (global) SARS-CoV-2 mutation accumulation rate equal to approximately 0.45/week (based on the <b>Nextstrain</b> [<xref rid="B3-viruses-14-00294" ref-type="bibr"><u>3</u></xref>] estimate as of 15 January 2021 [<xref rid="B20-viruses-14-00294" ref-type="bibr"><u>20</u></xref>]).<li> [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34855502">34855502</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [20](          <article-title>Nextstrain Clock</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/ncov/gisaid/global?l=clock" ext-link-type="uri">https://nextstrain.org/ncov/gisaid/global?l=clock</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-01-15">(accessed on 15 January 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35093310">35093310</a> <b>(1)</b> Emergence of SARS-CoV-2 Omicron (B.1.1.529) variant, salient features, high global health concerns and strategies to counter it amid ongoing COVID-19 pandemic<ul><li> The WHO established a group of experts as Technical Advisory Group on SARS-CoV-2 Virus Evolution (TAG-EV) to track, monitor, and evaluate the evolving situation on the pandemic virus, the WHO COVID-19 reference laboratory network, members of GISAID, <b>Nextstrain</b>, Pango, and scientific representatives from several nations and institutions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35123183">35123183</a> <b>(2)</b> A review on evolution of emerging SARS-CoV-2 variants based on spike glycoprotein<ul><li> Initially, 77681 global SARS-CoV-2 genomes are considered for multiple sequence alignment using MAFFT followed by their phylogenetic analysis using <b>Nextstrain</b>.<li> To study these variants in this work, initially multiple sequence alignment of 77681 global SARS-CoV-2 genomic sequences collected from January 2020 to July 2021 is carried out using MAFFT followed by their phylogenetic analysis using <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35277373">35277373</a> <b>(4)</b> The mutational dynamics of the SARS-CoV-2 virus in serial passages <italic>in vitro</italic><ul><li>Viral genome assembly and screening for distinct mutations was performed online using <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org" id="intref0025"><b>nextstrain</b>.org</ext-link> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv" id="intref0030">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv</ext-link>; <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0035">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35104582">35104582</a> <b>(13)</b> Early introduction, dispersal and evolution of Delta SARS-CoV-2 in Southern Brazil, late predominance of AY.99.2 and AY.101 related lineages<ul><li> PANGO and <b>Nextstrain</b> lineage assignments were applied to characterize the consensus sequences.<li> Phylogenetic tree was constructed including all SARS-CoV-2 complete genomes available from GISAID (between November 2019 and December 2021) through the Nextclade tool on <b>Nextstrain</b> server (<xref rid="bib0001" ref-type="bibr"><u>Aksamentov&#xA0;et&#xA0;al., 2021</u></xref>).<li><b>Nextstrain</b> workflow (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="interref0001">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="bib0022" ref-type="bibr"><b>Nextstrain</b>,&#xA0;2021a</xref>) was used for the temporal analysis of amino acid changes in the Delta variant genomes sequenced in this study.<li> Panel A shows the phylogenetic tree of all circulating lineages, developed through the <b>Nextstrain</b> server (Nextclade tool), using GISAID data.<li> The current global estimate (<xref rid="bib0023" ref-type="bibr"><b>Nextstrain</b>,&#xA0;2021b</xref>) including multiple variants of concern/interest suggests a similar rate of approximately 25.2 substitutions per year.<li>The Delta variant spread was remarkable in Brazil as in other countries, becoming the predominant variant in a matter of months since its first descriptions in the middle of 2021 (<xref rid="bib0023" ref-type="bibr"><b>Nextstrain</b>,&#xA0;2021b</xref>).<li> This variant, called Omicron by WHO and clade 21&#xA0;K by <b>Nextstrain</b>, is already the predominant variant in some countries despite its recent first description (late November in South Africa).<li> Updated Nexstrain data (<xref rid="bib0024" ref-type="bibr"><b>Nextstrain</b>,&#xA0;2021c</xref>) shows that, in contrast to Delta, this variant does not descend from recent variants and it's phylogenetically closer to viruses circulating in 2020.<li> By looking into <b>Nextstrain</b> updated molecular clock analysis (<xref rid="bib0024" ref-type="bibr"><b>Nextstrain</b>,&#xA0;2021c</xref>), all the so far sequenced Omicron viruses have numbers of mutations higher than the average.<li> [Aksamentov&#xA0;et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35603268">35603268</a> <b>(12)</b> SARS-CoV-2 introductions and early dynamics of the epidemic in Portugal<ul><li>We explored the diversity of INSA sequences using a variety of nomenclature strategies, namely <b>Nextstrain</b> (using <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>; 9 November 2020), GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</ext-link>; 23 July 2020) and Phylogenetic Assignment of Named Global Outbreak LINeages (cov-lineages.org) (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>; 16 October 2020)<sup><xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup>.<li> While <b>Nextstrain</b> and GISAID clade nomenclatures provide a less detailed categorisation of globally circulating diversity, cov-lineages.org classification is focused on identifying highly specific lineages that are actively transmitting in the population.<li> This site gives access to &#x201C;situation reports of the study and provides interactive data navigation using both <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR11"><u>11</u></xref></sup> and Microreact (<ext-link ext-link-type="uri" xlink:href="https://microreact.org/">https://microreact.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR12"><u>12</u></xref></sup> tools.<li> As of 23 July 2020, genomic and phylogenetic analysis were performed using the SARS-CoV-2 <b>Nextstrain</b> pipeline version from 23 March 2020 (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>), with slight modifications<sup><xref ref-type="bibr" rid="CR15"><u>15</u></xref></sup>.<li> Most of the analyzed genomes (89.8%) belong to the 20A/G, 20B/GR, and 20C/GH clades (<b>Nextstrain</b>/GISAID nomenclatures), which display, among other genetic markers, the D614G amino acid replacement in the Spike protein.<li> When applying the Pango lineages (cov-lineages.org) nomenclature, the main B lineage (roughly covering 19A/20A/20B/20C <b>Nextstrain</b> clades) was dominant in March, showing ample diversification into sublineages.<li> Of note, 108 out of the 277 introductions were seeded by <b>Nextstrain</b> 20B clade viruses.<li> In particular, our data uncovered the importation of extensive SARS-CoV-2 genomic diversity during the early epidemic in Portugal, as supported by the identification of at least 277 seeding events, covering all <b>Nextstrain</b>/GISAID clades and dozens of Pango lineages.<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28348833">28348833</a>) [15](          <person-group person-group-type="author">            <name>              <surname>Borges</surname>              <given-names>V</given-names>            </name>            <etal/>          </person-group>          <article-title>Massive dissemination of a SARS-CoV-2 Spike Y839 variant in Portugal</article-title>          <source>Emerg. Microbes Infect.</source>          <year>2020</year>          <volume>2</volume>          <fpage>1</fpage>          <lpage>58</lpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35207482">35207482</a> <b>(1)</b> Emergence of SARS-CoV-2 Variants in the World: How Could This Happen?<ul><li> For example, the GISAID, <b>Nextstrain</b>, and Pango classifications refer to genetic lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35118480">35118480</a> <b>(0)</b> A Quick Displacement of the SARS-CoV-2 variant Delta with Omicron: Unprecedented Spike in COVID-19 Cases Associated with Fewer Admissions and Comparable Upper Respiratory Viral Loads<ul><li> Query mutations were manually confirmed with Integrated Genomics viewer (IGV) (Version 2.8.10), clades were determined using Nextclade beta v 0.12.0 (<ext-link xlink:href="http://clades.<b>nextstrain</b>.org" ext-link-type="uri">clades.<b>nextstrain</b>.org</ext-link>), and lineages were determined with Pangolin COVID-19 lineage Assigner (COG-UK (cog-uk.io)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35107520">35107520</a> <b>(1)</b> Open-source real-time quantitative RT-PCR-based on a RNA standard for the assessment of SARS-CoV-2 viral load<ul><li> For analytical specificity analysis, 4026 sequences of the <b>Nextstrain</b> global dataset (18 May 2021) were downloaded from the GISAID database.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35083577">35083577</a> <b>(1)</b> Limited spread of a rare spike E484K-harboring SARS-CoV-2 in Marseille, France<ul><li> They were also classified into GISAID [<xref ref-type="bibr" rid="CR10"><u>10</u></xref>] clade GR, <b>Nextstrain</b> clade 20B [<xref ref-type="bibr" rid="CR11"><u>11</u></xref>], and PANGO lineage R.1 [<xref ref-type="bibr" rid="CR12"><u>12</u></xref>].<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35084216">35084216</a> <b>(3)</b> Whole-Genome Sequencing of SARS-CoV-2 Strains from Asymptomatic Individuals in India<ul><li> The <b>Nextstrain</b> online browser and CoVsurver mutation app were used to assign the clades and identify the mutations, respectively.<li> Among the 11 sequenced samples, 9 were found to be <b>Nextstrain</b> clade 20B (PANGO lineage B.1.1.519, GISAID clade GR), and two were identified as clade 20A (PANGO lineage B.1.619, GISAID clade G).<li><b>Nextstrain</b> clade, GISAID clade, PANGO lineage</th>                <th rowspan="2" colspan="1">Amino acid changes</th>              </tr>              <tr>                <th rowspan="1" colspan="1">Age</th>                <th rowspan="1" colspan="1">Gender<xref rid="ngtab1-1" ref-type="table-fn"><sup><italic toggle="yes">a</italic></sup></xref></th>              </tr>            </thead>            <tbody>              <tr>                <td rowspan="1" colspan="1">1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35208761">35208761</a> <b>(0)</b> Comparative Analysis of Five Multiplex RT-PCR Assays in the Screening of SARS-CoV-2 Variants<ul><li> Lineage and clade data, quality controls, mutations, missing nucleotides and gaps were analyzed and confirmed with the support of Pangolin (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri> accessed on 15 November 2021) and Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> accessed on 15 November 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35215848">35215848</a> <b>(3)</b> Rapid Biosensor of SARS-CoV-2 Using Specific Monoclonal Antibodies Recognizing Conserved Nucleocapsid Protein Epitopes<ul><li> We also investigated the frequencies of amino acid substitutions in SARS-CoV-2 NP due to viral mutations using an open-source database tool (<b>Nextstrain</b>, <uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri>, accessed on 10 January 2022) (<xref rid="viruses-14-00255-f001" ref-type="fig"><u>Figure 1</u></xref>B).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35215852">35215852</a> <b>(13)</b> Epidemiology and Genetic Analysis of SARS-CoV-2 in Myanmar during the Community Outbreaks in 2020<ul><li>To clarify the genotypes of strains in Myanmar, 10 sequences in this study and an additional 31 sequences already registered in the public database were analyzed by <b>Nextstrain</b> to find out Nexstrain and GISAID clades (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri> (accessed on 30 November 2021)) [<xref rid="B14-viruses-14-00259" ref-type="bibr"><u>14</u></xref>,<xref rid="B20-viruses-14-00259" ref-type="bibr"><u>20</u></xref>,<xref rid="B21-viruses-14-00259" ref-type="bibr"><u>21</u></xref>].<li>To further characterize the genomic shifts over time and trace the transmission pattern of how SARS-CoV-2 viruses were circulating in Myanmar, phylogenetic analysis using <b>Nextstrain</b> version 8.0 with default setting was performed against 6195 global strains which were sampled collected from December 2019 to September 2020, and 41 Myanmar strains including 10 strains obtained in this study (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</uri> (accessed on 30 November 2021)) [<xref rid="B20-viruses-14-00259" ref-type="bibr"><u>20</u></xref>,<xref rid="B21-viruses-14-00259" ref-type="bibr"><u>21</u></xref>].<li>The genetic analysis of 41 strains (10 from this study and 31 downloaded from GISAID) showed that first, five Myanmar sequences during April to early May 2020 belonged to GISAID clade O, <b>Nextstrain</b> clade 19A, and PANGOLIN B.6 (<xref rid="viruses-14-00259-f003" ref-type="fig"><u>Figure 3</u></xref>A and <xref rid="app1-viruses-14-00259" ref-type="app"><u>Table S2</u></xref>) [<xref rid="B14-viruses-14-00259" ref-type="bibr"><u>14</u></xref>].<li> Thereafter, the genotypes diversified and changed into GISAID clade G (<b>Nextstrain</b> 20A; PANGOLIN B.1.210), GR (<b>Nextstrain</b> 20B; PANGOLIN B.1.1 and B.1.1.174), and GH (<b>Nextstrain</b> 20A; PANGOLIN B.1.36) during May and June 2020, mostly related to the overseas travels such as India and China (<xref rid="app1-viruses-14-00259" ref-type="app"><u>Table S2</u></xref>) [<xref rid="B14-viruses-14-00259" ref-type="bibr"><u>14</u></xref>].<li> The genome analysis during the second wave in August&#x2013;September showed that a total of 29 sequences, 10 sequences in this study and 19 sequences from the other study [<xref rid="B14-viruses-14-00259" ref-type="bibr"><u>14</u></xref>], belonged to GISAID clade GH, <b>Nextstrain</b> clade 20A, and the PANGOLIN lineage B1.36.16 (<xref rid="viruses-14-00259-f003" ref-type="fig"><u>Figure 3</u></xref>A, <xref rid="app1-viruses-14-00259" ref-type="app"><u>Tables S1 and S2</u></xref>).<li> The time-aware phylogeny and phylogeographic analysis by <b>Nextstrain</b> using 41 Myanmar strains and 6195 global strains showed that there are multiple introductions from other foreign countries, such as China at the beginning of the first wave and then changed to India and Bangladesh in the second wave (<xref rid="viruses-14-00259-f003" ref-type="fig"><u>Figure 3</u></xref>B&#x2013;D, <xref rid="app1-viruses-14-00259" ref-type="app"><u>Video S1</u></xref>).<li> The previous report from the other group in Myanmar identified that nearly all of the sequences from the local cases in the early pandemic spread in April 2020 were PANGOLIN B.6 lineage (GISAID clade O and <b>Nextstrain</b> 19A) [<xref rid="B14-viruses-14-00259" ref-type="bibr"><u>14</u></xref>].<li>During the second wave from August to September 2020, Myanmar sequences belonged to the clade GH (<b>Nextstrain</b> clade 20A or PANGOLIN lineage B1.36.16).<li> Our <b>Nextstrain</b> analysis and ML phylogenetic analysis showed that 29 SARS-CoV-2 viruses detected in Myanmar under the GH or PANGOLIN lineage B.1.36.16 clade in August and September were in close relationship with those in Bangladesh and India.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33986354">33986354</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19505945">19505945</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19505945">19505945</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33986354">33986354</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33986354">33986354</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33986354">33986354</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33986354">33986354</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34757848">34757848</a> <b>(5)</b> A Structurally Conserved RNA Element within SARS-CoV-2 ORF1a RNA and S mRNA Regulates Translation in Response to Viral S Protein-Induced Signaling in Human Lung Cells<ul><li> The black histogram above the colored bar shows the degree of diversity among currently available SARS-CoV-2 sequences at each position of the genome (data from <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/</ext-link>), thus illustrating mutation host spots.<li> The <b>Nextstrain</b> (<xref rid="B51" ref-type="bibr"><u>51</u></xref>) SARS-CoV-2 resource (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/</ext-link>) was used for graphic visualization of the data.<li> [51](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35279413">35279413</a> <b>(3)</b> Characteristics of SARS-CoV-2 transmission in a medium-sized city with traditional communities during the early COVID-19 epidemic in China<ul><li> For the selection of foreign SARS-CoV-2 genomes, we referred to the global SARS-CoV-2 phylogeny built with <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0020">https://<b>nextstrain</b>.org/ncov/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35140714">35140714</a> <b>(2)</b> Omicron: A Heavily Mutated SARS-CoV-2 Variant Exhibits Stronger Binding to ACE2 and Potently Escapes Approved COVID-19 Therapeutic Antibodies<ul><li> The unrooted phylogenetic tree was constructed from the <b>Nextstrain</b> servers.<li> The unrooted phylogenetic tree was constructed from the <b>Nextstrain</b> (<xref rid="B43" ref-type="bibr"><u>43</u></xref>) servers using ~4,000 full-length SARS-CoV-2 sequences from the GISAID (<xref rid="B44" ref-type="bibr"><u>44</u></xref>) database with reference to Wuhan-Hu-1/2019 as a reference sequence.<li> [43](<person-group person-group-type="author"><name><surname>Hadfield</surname><given-names>J</given-names></name><name><surname>Megill</surname><given-names>C</given-names></name><name><surname>Bell</surname><given-names>SM</given-names></name><name><surname>Huddleston</surname><given-names>J</given-names></name><name><surname>Potter</surname><given-names>B</given-names></name><name><surname>Callender</surname><given-names>C</given-names></name><etal/></person-group>. <article-title>Nextstrain: Real-Time Tracking of Pathogen Evolution</article-title>. <source>Bioinformatics</source> (<year>2018</year>) <volume>34</volume>(<issue>23</issue>):<page-range>4121&#x2013;3</page-range>. doi: <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>) [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35093719">35093719</a> <b>(4)</b> Comparing susceptibility and contagiousness in concurrent outbreaks with a non-VOC and the VOC SARS-CoV-2 variant B.1.1.7 in daycare centers in Hamburg, Germany<ul><li> Pangolin lineage and <b>Nextstrain</b> clade assignment of consensus sequences were performed using the pangolin (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0010">https://github.com/cov-lineages/pangolin</ext-link>) and Nextclade (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="intref0015">https://github.com/<b>nextstrain</b>/nextclade</ext-link>) packages.<li> Phylogenetic analysis and tree visualization were performed using <b>Nextstrain</b> which relies on maximum likelihood ancestral state reconstruction of discrete traits in combination with inferred probability distributions of ancestral state of each node (<xref rid="bib11" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35075154">35075154</a> <b>(5)</b> Viral infection and transmission in a large, well-traced outbreak caused by the SARS-CoV-2 Delta variant<ul><li> We also included a set of reference sequences, comprising 50 genomes randomly selected from each of 13 defined <b>NextStrain</b> clades (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) and the notified VOCs (Alpha, Beta, Gamma, Delta).<li><bold>a</bold> A time-resolved phylogenetic tree was estimated using the <b>NextStrain</b> pipeline and includes (i) Guangdong sequences collected from local infections and imported cases, January 2020&#x2013;June 2021, and (ii) reference sequences from different genetic lineages.<li> The <b>nextstrain</b> pipeline<sup><xref ref-type="bibr" rid="CR29"><u>29</u></xref></sup> was used to analyze and visualize the genetic distribution of SARS-CoV-2 infections and its dynamic change in Guangdong between January 2020 and June 2021.<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35075180">35075180</a> <b>(5)</b> Mutations in viral nucleocapsid protein and endoRNase are discovered to associate with COVID19 hospitalization risk<ul><li>All of the Washington viral genomes obtained from GISAID were classified by <b>nextstrain</b>, GISAID-clade, and lineage.<li> To assess the relationship between the haplotype and <b>nextstrain</b> classification, we tabulated their cross-table frequencies (Table <xref rid="Tab5" ref-type="table"><u>5</u></xref>).<li>Relationships of identified SNV haplotype (t19839-g28881-g28882-g28883) with classifications of <b>Nextstrain</b> and clades by GISAID.<li><b>Nextstrain</b>.<li> We used submission dates as a proxy for collection time, and used their classification by <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR18"><u>18</u></xref></sup>, clade by GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org">https://www.gisaid.org</ext-link>) and lineage by PANGO<sup><xref ref-type="bibr" rid="CR19"><u>19</u></xref></sup>.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35215820">35215820</a> <b>(1)</b> Spike Gene Evolution and Immune Escape Mutations in Patients with Mild or Moderate Forms of COVID-19 and Treated with Monoclonal Antibodies Therapies<ul><li> The mutations found in the S gene and the <b>Nextstrain</b> clade were determined for each strain with the Nextclade online tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35150638">35150638</a> <b>(12)</b> Mutations in SARS-CoV-2 variants of concern link to increased spike cleavage and virus transmission<ul><li> The phylogenetic tree was generated with <b>Nextstrain</b> with 7,059 genomes sampled from worldwide data deposited in the GISAID database from December 2019 to September 2020 for representation of the H655Y substitution over time.<li> The phylogenetic tree was generated using <b>NextStrain</b>, and analysis was performed using a sample of 13,847 genomes focused on the most prevalent substitutions between S:655 and S:701 between February 2020 and June 2021 from GISAID database.<li><hr/></td></tr><tr><td>Graphpad Prism</td><td>Graphpad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.graphpad.com/scientific-software/prism/" id="intref0065">https://www.graphpad.com/scientific-software/prism/</ext-link></td></tr><tr><td>SnapGene 5.3.2</td><td>SnapGene</td><td><ext-link ext-link-type="uri" xlink:href="https://www.snapgene.com/" id="intref0070">https://www.snapgene.com/</ext-link></td></tr><tr><td>ImageJ</td><td>NIH</td><td><ext-link ext-link-type="uri" xlink:href="https://imagej.nih.gov/ij/download.html" id="intref0075">https://imagej.nih.gov/ij/download.html</ext-link></td></tr><tr><td>Artic Consortium protocol</td><td>Artic Consortium</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0080">https://artic.network/ncov-2019</ext-link></td></tr><tr><td>SARS-CoV-2 genome assembly</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/mjsull/COVID_pipe" id="intref0085">https://github.com/mjsull/COVID_pipe</ext-link></td></tr><tr><td>ONT software MinKNOW v4.3.7</td><td>Oxford Nanopore</td><td><ext-link ext-link-type="uri" xlink:href="https://nanoporetech.com/" id="intref0090">https://nanoporetech.com/</ext-link></td></tr><tr><td>Guppy 5.0.12</td><td>Oxford Nanopore</td><td><ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/sso/login?next_url=%2Fdownloads%2Fguppy%2Frelease_notes" id="intref0095">https://community.nanoporetech.com/sso/login?next_url=%2Fdownloads%2Fguppy%2Frelease_notes</ext-link></td></tr><tr><td>Minimap2 2.17-r941</td><td><xref rid="bib29" ref-type="bibr"><u>Li, 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/lh3/minimap2" id="intref0100">https://github.com/lh3/minimap2</ext-link></td></tr><tr><td>Nanopolish 0.13.2</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/jts/nanopolish" id="intref0105">https://github.com/jts/nanopolish</ext-link></td></tr><tr><td>COBALT multiple alignment</td><td><xref rid="bib39" ref-type="bibr"><u>Papadopoulos and Agarwala, 2007</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/tools/cobalt/re_cobalt.cgi" id="intref0110">https://www.ncbi.nlm.nih.gov/tools/cobalt/re_cobalt.cgi</ext-link></td></tr><tr><td><b>Nextstrain</b> v7 for SARS-CoV-2.<li><hr/></td></tr><tr><td>Graphpad Prism</td><td>Graphpad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.graphpad.com/scientific-software/prism/" id="intref0065">https://www.graphpad.com/scientific-software/prism/</ext-link></td></tr><tr><td>SnapGene 5.3.2</td><td>SnapGene</td><td><ext-link ext-link-type="uri" xlink:href="https://www.snapgene.com/" id="intref0070">https://www.snapgene.com/</ext-link></td></tr><tr><td>ImageJ</td><td>NIH</td><td><ext-link ext-link-type="uri" xlink:href="https://imagej.nih.gov/ij/download.html" id="intref0075">https://imagej.nih.gov/ij/download.html</ext-link></td></tr><tr><td>Artic Consortium protocol</td><td>Artic Consortium</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0080">https://artic.network/ncov-2019</ext-link></td></tr><tr><td>SARS-CoV-2 genome assembly</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/mjsull/COVID_pipe" id="intref0085">https://github.com/mjsull/COVID_pipe</ext-link></td></tr><tr><td>ONT software MinKNOW v4.3.7</td><td>Oxford Nanopore</td><td><ext-link ext-link-type="uri" xlink:href="https://nanoporetech.com/" id="intref0090">https://nanoporetech.com/</ext-link></td></tr><tr><td>Guppy 5.0.12</td><td>Oxford Nanopore</td><td><ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/sso/login?next_url=%2Fdownloads%2Fguppy%2Frelease_notes" id="intref0095">https://community.nanoporetech.com/sso/login?next_url=%2Fdownloads%2Fguppy%2Frelease_notes</ext-link></td></tr><tr><td>Minimap2 2.17-r941</td><td><xref rid="bib29" ref-type="bibr"><u>Li, 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/lh3/minimap2" id="intref0100">https://github.com/lh3/minimap2</ext-link></td></tr><tr><td>Nanopolish 0.13.2</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/jts/nanopolish" id="intref0105">https://github.com/jts/nanopolish</ext-link></td></tr><tr><td>COBALT multiple alignment</td><td><xref rid="bib39" ref-type="bibr"><u>Papadopoulos and Agarwala, 2007</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/tools/cobalt/re_cobalt.cgi" id="intref0110">https://www.ncbi.nlm.nih.gov/tools/cobalt/re_cobalt.cgi</ext-link></td></tr><tr><td><b>Nextstrain</b> v7 for SARS-CoV-2</td><td><xref rid="bib17" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0115">https://github.com/<b>nextstrain</b>/ncov</ext-link>.<li> The downloaded data was processed through the augur pipeline with <b>Nextstrain</b> v7 for SARS-CoV-2 (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0170">https://github.com/<b>nextstrain</b>/ncov</ext-link>) for sequence alignment and curation with default parameters (<xref rid="bib17" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> The relative proportion of their occurrence time and <b>Nextstrain</b> clade (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming/" id="intref0175">http://<b>nextstrain</b>.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming/</ext-link>) was plotted in R with ggplot2 (Wickham) v3.3.5.<li>Phylogenetic analysis was performed using the same dataset from above to build a time-calibrated phylogenetic tree with <b>Nextstrain</b>, to visualize the distribution of H655Y (up to 2020-09-30) and other prevalent substitutions along the 655-701 region (up to 2021-06-23).<li> [Li, 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29750242">29750242</a>) [Papadopoulos and Agarwala, 2007](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17332019">17332019</a>) [Li, 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29750242">29750242</a>) [Papadopoulos and Agarwala, 2007](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17332019">17332019</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35242337">35242337</a> <b>(3)</b> Rapidly rising cases with Omicron in Senegal<ul><li> Phylogenetic position of Omicron isolated in IRESSEF (Phylogeny reconstruction was performed using the <b>nextstrain</b>/ncov tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0020">https://github.com/<b>nextstrain</b>/ncov</ext-link>) then visualised with Auspice (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/" id="intref0025">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35064154">35064154</a> <b>(3)</b> Regional and temporal coordinated mutation patterns in SARS-CoV-2 spike protein revealed by a clustering and network analysis<ul><li> The US Center for Disease Control and Prevention (CDC) distinguishes the different variants by Greek letters while the <b>Nextstrain</b> team (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2">https://<b>nextstrain</b>.org/sars-cov-2</ext-link>) suggests using 18 major clades as from 19 A to 21F 19B, and 20A-20I.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35123650">35123650</a> <b>(0)</b> SARS-CoV-2 breakthrough infections elicit potent, broad, and durable neutralizing antibody responses<ul><li> Sequence quality was determined using Nextclade version 1.0.0-alpha.8 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0035">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35721434">35721434</a> <b>(4)</b> SARS-CoV-2 viral dynamics of the first 1000 sequences from Vietnam and neighbouring ASEAN countries<ul><li> The lineages were obtained using the Pangolin tool (<xref rid="bib0005" ref-type="bibr"><u>Rambaut&#xA0;et&#xA0;al., 2020</u></xref>) Subsequently, using the aligned sequences, a phylogenetic tree was reconstructed to show the different lineages, using the <b>Nextstrain</b> algorithm, as implemented in the web tool (<xref rid="bib0001" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>).<li>Phylogenetic tree representing SARS-CoV-2 variants over space and time from nine ASEAN countries represented by different colours, reconstructed using <b>Nextstrain</b> algorithms (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="interref0001">https://<b>nextstrain</b>.org/</ext-link>).<li> [Rambaut&#xA0;et&#xA0;al., 2020](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Holmes</surname>              <given-names>EC</given-names>            </name>            <name>              <surname>O'Toole</surname>              <given-names>&#xC1;</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>JT</given-names>            </name>            <name>              <surname>Ruis</surname>              <given-names>C</given-names>            </name>            <etal/>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>          <source>Nature Microbiology</source>          <volume>5</volume>          <issue>11</issue>          <year>2020</year>          <fpage>1403</fpage>          <lpage>1407</lpage>        ) [Hadfield&#xA0;et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35086043">35086043</a> <b>(0)</b> Detection of SARS-CoV-2 variants by Abbott molecular, antigen, and serological tests<ul><li> Sequences containing lineage-defining mutations were obtained from GISAID <xref rid="bib0011" ref-type="bibr"><u>[11]</u></xref> for <italic>in silico</italic> analysis by customized application of the NextClade tool (clades.<b>nextstrain</b>.org) to compare individual assay target sequences for Abbott ID NOW COVID-19, <italic>m</italic>2000 RealTi<italic>m</italic>e SARS-CoV-2, Alinity m SARS-CoV-2, and Alinity m Resp-4-Plex molecular assays; the BinaxNOW COVID-19 Ag Card and Panbio COVID-19 Ag Rapid Test Device; and the ARCHITECT/Alinity i SARS-CoV-2 IgG and AdviseDx IgM assays, Panbio COVID-19 IgG assay, and ARCHITECT/Alinity i AdviseDx SARS-CoV-2 IgG II assay.<li> [[11]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35215068">35215068</a> <b>(4)</b> Local Emergence of a del HV69-70 SARS-CoV-2 Variant in Burgundy, France<ul><li> The <b>Nextstrain</b> Clade 19B emerged in late 2019, but early in the pandemic, it was overtaken by clades 20A, 20B and 20C [<xref rid="B3-pathogens-11-00124" ref-type="bibr"><u>3</u></xref>,<xref rid="B4-pathogens-11-00124" ref-type="bibr"><u>4</u></xref>], which harbor the D614G substitution believed to increase viral transmission, but not pathogenicity [<xref rid="B5-pathogens-11-00124" ref-type="bibr"><u>5</u></xref>,<xref rid="B6-pathogens-11-00124" ref-type="bibr"><u>6</u></xref>], a conclusion that is supported by experimental evidence [<xref rid="B7-pathogens-11-00124" ref-type="bibr"><u>7</u></xref>].<li> Genomic epidemiology of SARS-CoV-2 in 2020 was estimated by analyzing two Europe-focused subsamplings performed using <b>Nextstrain</b> [<xref rid="B17-pathogens-11-00124" ref-type="bibr"><u>17</u></xref>].<li> The <b>Nextstrain</b> clade 20B included 41% of the strains (92 out of 202 European strains).<li> Among the selected sequences from <b>Nextstrain</b> during the study period, from weeks 38 to 47, 2020 (<italic toggle="yes">n</italic> = 86/3738), nine had the 6-nucleotide HV deletion and most were found in Switzerland and Liechtenstein.<li> [3](          <person-group person-group-type="author"><collab>Nextstrain</collab></person-group>          <article-title>Genomic Epidemiology of Novel Coronavirus&#x2014;Global Subsampling</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/ncov" ext-link-type="uri">https://nextstrain.org/ncov</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-12-20">(accessed on 20 December 2021)</date-in-citation>        ) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33275900">33275900</a>) [7](          <person-group person-group-type="author"><name><surname>Hou</surname><given-names>Y.J.</given-names></name><name><surname>Chiba</surname><given-names>S.</given-names></name><name><surname>Halfmann</surname><given-names>P.</given-names></name><name><surname>Ehre</surname><given-names>C.</given-names></name><name><surname>Kuroda</surname><given-names>M.</given-names></name><name><surname>Dinnon</surname><given-names>K.H.</given-names><suffix>III</suffix></name><name><surname>Leist</surname><given-names>S.R.</given-names></name><name><surname>Schafer</surname><given-names>A.</given-names></name><name><surname>Nakajima</surname><given-names>N.</given-names></name><name><surname>Takanashi</surname><given-names>K.</given-names></name><etal/></person-group>          <article-title>SARS-CoV-2 D614G Variant Exhibits Enhanced Replication ex vivo and Earlier Transmission in vivo</article-title>          <source>bioRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.09.28.317685</pub-id>        ) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35075454">35075454</a> <b>(6)</b> Genomic surveillance of SARS-CoV-2 in Puerto Rico reveals emergence of an autochthonous lineage and early detection of variants<ul><li> We then used the standard ncov augur/auspice multiple input workflow available in the <b>Nextstrain</b> platform<sup><xref rid="R44" ref-type="bibr"><u>44</u></xref></sup> (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) to subsample contextual genomes and phylogenetic inference with time-stamped trees.<li> The resulting global ML tree was visualized with <b>Nextstrain</b> auspice<sup><xref rid="R44" ref-type="bibr"><u>44</u></xref></sup> and annotated with iTol for region of origin and emerging variants<sup><xref rid="R48" ref-type="bibr"><u>48</u></xref></sup>.<li> A custom subsampling scheme was selected on the <b>Nextstrain</b> ncov workflow to select genomes from samples collected between November 1<sup>st</sup>, 2020 and February 28<sup>th</sup>, 2021.<li>The date of the most recent common ancestor (tMRCA) determined by the <b>Nextstrain</b> ML phylogenetic inference was confirmed with Bayesian coalescent analyses for B.1.588, Alpha VOC and Delta VOC lineage sub-trees.<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [48](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17050570">17050570</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35058525">35058525</a> <b>(3)</b> Nosocomial transmission clusters and lineage diversity characterized by SARS-CoV-2 genomes from two large hospitals in Paris, France, in 2020<ul><li> Databases and tools such as GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</ext-link>) and <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>), gathering and providing real time analysis of the SARS-CoV-2 sequences&#x2019; diversity, helped monitor the global spread of the epidemic.<li> This variant (<b>nextstrain</b> clade 20A) caused large outbreaks in early 2020 and became the most prevalent lineage by April in most European countries and worldwide, suggesting a possible increased infectivity<sup><xref ref-type="bibr" rid="CR6"><u>6</u></xref></sup>.<li> <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR24"><u>24</u></xref></sup> and PANGO lineages<sup><xref ref-type="bibr" rid="CR25"><u>25</u></xref></sup> SARS-CoV-2 classifications were used to assign a clade and lineage to each strain, respectively.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34545361">34545361</a> <b>(18)</b> Rapid and parallel adaptive mutations in spike S1 drive clade success in SARS-CoV-2<ul><li> For example, the common ancestor of globally circulating SARS-CoV-2 viruses is currently no earlier than January 2020, corresponding to the base of clade 20A or lineage B.1 (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri"><b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).<li> Despite possessing some proof-reading capacity (a relatively rare function for an RNA virus), SARS-CoV-2 has been accumulating roughly 24&#x2013;25 substitutions per year (<ext-link xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15?l=clock" ext-link-type="uri"><b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15?l=clock</ext-link>, [<xref rid="R7" ref-type="bibr"><u>7</u></xref>]).<li>All primary analyses in this manuscript were performed using data downloaded from the GISAID EpiCoV database (<ext-link xlink:href="https://gisaid.org/" ext-link-type="uri">gisaid.org</ext-link>, [<xref rid="R2" ref-type="bibr"><u>2</u></xref>]) on July 29, 2021 and curated by the <b>Nextstrain</b> nCoV ingest pipeline (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov-ingest" ext-link-type="uri">github.com/<b>nextstrain</b>/ncov-ingest</ext-link>).<li> These genomes were aligned with Nextalign (<ext-link xlink:href="https://docs.<b>nextstrain</b>.org/projects/nextclade/en/latest/user/nextalign-cli.html" ext-link-type="uri">docs.<b>nextstrain</b>.org/projects/nextclade/en/latest/user/nextalign-cli.html</ext-link>) and masked to minimize error in phylogenetic inference associated with problematic amplicon sites.<li> Masked alignments were filtered to exclude strains that were known outliers, sequenced due to &#x2018;S dropout&#x2019;, mis-annotated with an admin division of &#x2018;USA&#x2019;, shorter than 27,000 bp of A, C, T, or G bases, missing complete date information, annotated with a date prior to October 2019, flagged with more than 20 mutations above the expected number based on the mutational clock rate, or flagged by Nextclade (<ext-link xlink:href="https://docs.<b>nextstrain</b>.org/projects/nextclade/en/latest/user/algorithm/07-quality-control.html" ext-link-type="uri">docs.<b>nextstrain</b>.org/projects/nextclade/en/latest/user/algorithm/07-quality-control.html</ext-link>) with one or more clusters of 6 or more private differences in a 100-nucleotide window.<li> The primary analysis was conducted on 9544 genomes collected on or before May 15, 2021, and the phylogeny reconstructed from these data can be found at <ext-link xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15" ext-link-type="uri"><b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15</ext-link>.<li>Phylogenies for seasonal influenza A/H3N2 and A/H1N1pdm were built using the <b>Nextstrain</b> pipeline from <ext-link xlink:href="https://github.com/<b>nextstrain</b>/seasonal-flu" ext-link-type="uri">github.com/<b>nextstrain</b>/seasonal-flu</ext-link>.<li> This resulted in a tree with a total of 19,694 genomes, sampled on or before May 15,2021 and distributed roughly evenly over geography and time (<ext-link xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15/20k" ext-link-type="uri"><b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15/20k</ext-link>).<li> The Asia-specific tree can be interactively-viewed at <ext-link xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15/asia" ext-link-type="uri"><b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2021-05-15/asia</ext-link>, and other regional trees can be accessed by substituting the region&#x2019;s name at the end of the URL.<li> The November 15, 2020 tree can be viewed at <ext-link xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2020-11-15" ext-link-type="uri"><b>nextstrain</b>.org/groups/blab/ncov/adaptive-evolution/2020-11-15</ext-link>, and all other dates can be accessed by changing the date at the end of the URL.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [2](<name><surname>Shu</surname><given-names>Y</given-names></name>, <name><surname>McCauley</surname><given-names>J</given-names></name> (<year>2017</year>) <article-title>GISAID: Global initiative on sharing all influenza data - from vision to reality</article-title>. <source>Euro Surveill</source><volume>22</volume>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35043153">35043153</a> <b>(4)</b> Coronavirus GenBrowser for monitoring the transmission and evolution of SARS-CoV-2<ul><li> The WashU Virus Genome Browser provides <b>Nextstrain</b>-based phylogenetic-tree view and genomic-coordinate, track-based view of genomic features of viruses [<xref rid="ref7" ref-type="bibr"><u>7</u></xref>].<li> The pathogen genomics platform <b>Nextstrain</b> allows analysis of genomic sequences of approximately 4000 strains of SARS-CoV-2 and investigation of its evolution [<xref rid="ref8" ref-type="bibr"><u>8</u></xref>], which cannot timely analyze millions of increasing genomic sequences.<li>Similar to <b>NextStrain</b> [<xref rid="ref8" ref-type="bibr"><u>8</u></xref>] and the WashU Virus Genome Browse [<xref rid="ref7" ref-type="bibr"><u>7</u></xref>], the CGB uses a tree-based file format to store SARS-CoV-2 genomic mutations.<li> However, the <b>Nextstrain</b>-based phylogenetic-tree view only supports about 4000 strains of SARS-CoV-2 [<xref rid="ref7" ref-type="bibr"><u>7</u></xref>, <xref rid="ref8" ref-type="bibr"><u>8</u></xref>].<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908257">32908257</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908257">32908257</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908257">32908257</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35046397">35046397</a> <b>(2)</b> A phase 2 single center open label randomised control trial for convalescent plasma therapy in patients with severe COVID-19<ul><li> Clade Nomenclature of <b>Nextstrain</b> was used to visualize the phylogenetic tree<sup><xref ref-type="bibr" rid="CR44"><u>44</u></xref></sup>.<li> Our analysis from all the available sequences (331 samples) in GISAID from the region of study (Kolkata) till date (10th April, 2021), 60.42% of these sequences are of <b>Nextstrain</b> clade 20&#x2009;A, while clade 20B is observed at a frequency of 26.59% (Supplemental Table&#xA0;<xref rid="MOESM1" ref-type="media"><u>3</u></xref>).<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35065122">35065122</a> <b>(0)</b> Use of sewage surveillance for COVID-19 to guide public health response: A case study in Hong Kong<ul><li> The called consensus was identified to the specific lineage by Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0010">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> The phylogenetic tree was created by Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0015">https://clades.<b>nextstrain</b>.org/</ext-link>).<li>The consensus-level phylogeny was resolved by placing these three consensus genome sequences on a high-quality SARS-CoV-2 reference tree constructed based on clade-defining mutations (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0020">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35118097">35118097</a> <b>(1)</b> The Variation of SARS-CoV-2 and Advanced Research on Current Vaccines<ul><li><bold><b>Nextstrain</b> clade</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>First identified</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>Spike protein mutations</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>Characteristics</bold></th>            </tr>          </thead>          <tbody>            <tr>              <td valign="top" align="left" rowspan="1" colspan="1">Alpha.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35001392">35001392</a> <b>(5)</b> Distinct mutations and lineages of SARS&#x2010;CoV&#x2010;2 virus in the early phase of COVID&#x2010;19 pandemic and subsequent 1&#x2010;year global expansion<ul><li> The evolutionary rate was estimated to be 27.1 subs per year, which was very similar to the evolutionary rate (26.7 subs per year) estimated by <ext-link xlink:href="http://<b>Nextstrain</b>.org" ext-link-type="uri"><b>Nextstrain</b>.org</ext-link> from 4616 viral genomes sampled between December 2019 and April 2020 (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global?l=clock" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global?l=clock</ext-link>).<li> Although it is roughly consistent with the estimated evolutionary rate (~22 subs per year according to <b>Nextstrain</b> by December 2020), the underlying mechanism of those sequential increments of concurrent mutations is yet to be carefully unveiled.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35063123">35063123</a> <b>(1)</b> Breakthrough infections with SARS-CoV-2 omicron despite mRNA vaccine booster dose<ul><li>All seven individuals were infected with omicron (PANGO lineage B.1.1.529, <b>Nextstrain</b> clade 21K).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35037418">35037418</a> <b>(1)</b> Severe Acute Respiratory Syndrome Type 2&#x2010;Causing Coronavirus: Variants and Preventive Strategies<ul><li> To avoid confusion and stigmatization due to different variant names in a rapidly changing environment like this, WHO is also using two main categories for classification purposes (VoC and VoI or VUI), and recently a Greek naming system was introduced to combine all major scientific variant naming systems like PANGO, GISAID, and <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35039944">35039944</a> <b>(2)</b> Competing endogenous RNA network mediated by circ_3205 in SARS-CoV-2 infected cells<ul><li> From December 2019 to September 2021, 3,508 SARS-CoV-2 genomes were sampled and 20 clades were identified around the world, according to the GISAID database (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35233544">35233544</a> <b>(6)</b> SARS-CoV-2 Beta and Delta variants trigger Fc effector function with increased cross-reactivity<ul><li><hr/></td></tr><tr><td>Genome Detective 1.132</td><td>Genome Detective</td><td><ext-link ext-link-type="uri" xlink:href="https://www.genomedetective.com" id="intref0050">https://www.genomedetective.com</ext-link></td></tr><tr><td>Coronavirus Typing Tool</td><td>Cleemput et&#xA0;al., 2020</td><td>N/A</td></tr><tr><td>Geneious software</td><td>Biomatters Ltd</td><td>N/A</td></tr><tr><td><b>NextStrain</b>.<li><hr/></td></tr><tr><td>Genome Detective 1.132</td><td>Genome Detective</td><td><ext-link ext-link-type="uri" xlink:href="https://www.genomedetective.com" id="intref0050">https://www.genomedetective.com</ext-link></td></tr><tr><td>Coronavirus Typing Tool</td><td>Cleemput et&#xA0;al., 2020</td><td>N/A</td></tr><tr><td>Geneious software</td><td>Biomatters Ltd</td><td>N/A</td></tr><tr><td><b>NextStrain</b></td><td>Hadfield et&#xA0;al., 2018</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0055">https://github.com/<b>nextstrain</b>/ncov</ext-link>.<li> Phylogenetic clade classification of the genomes in this study consisted of analyzing them against a global reference dataset using a custom pipeline based on a local version of <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0015">https://github.com/<b>nextstrain</b>/ncov</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35038741">35038741</a> <b>(1)</b> The Emergence of SARS-CoV-2 Variants of Concern Is Driven by Acceleration of the Substitution Rate<ul><li>We downloaded 100 randomly selected sequences in the global <b>NextStrain</b> SARS-CoV-2 build of August 2021 (<xref rid="msac013-B26" ref-type="bibr">Hadfield <italic toggle="yes">et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35051459">35051459</a> <b>(3)</b> Detection of the Omicron (B.1.1.529) variant of SARS-CoV-2 in aircraft wastewater<ul><li> Besides the jurisdiction specific data, the analysis utilized the public data from GISAID (<ext-link ext-link-type="uri" xlink:href="http://www.gisaid.org" id="ir0035">www.gisaid.org</ext-link>) and services from <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="ir0040">https://<b>nextstrain</b>.org</ext-link> and <ext-link ext-link-type="uri" xlink:href="https://austrakka.net.au" id="ir0045">https://austrakka.net.au</ext-link>.<li>The consensus genomes of the aircraft wastewater sample (i.e., A12) positive for the del(69&#x2013;70) mutation by RT-qPCR was compared to the latest SARS-CoV-2 genomes, including the 21 K(Omicron), through the clade assignment, mutation identification, consensus genome quality checking tools, and phylogenetic placement and visualization, provided by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="ir0055">https://clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="bb0015" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>).<li> [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35069059">35069059</a> <b>(1)</b> Will the COVID-19 pandemic end with the Delta and Omicron variants?<ul><li> To date, a total of 407 pango lineages of SARS-CoV-2 variants have been identified (<b>Nextstrain</b> <xref ref-type="bibr" rid="CR49"><u>2021</u></xref>).<li> [2021](Nextstrain (2021) Genomic epidemiology of novel coronavirus&#x2013;Global subsampling. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/gisaid/global">https://nextstrain.org/ncov/gisaid/global</ext-link>. Accessed 30 November 2021) </ul></p>
<p><u>No PMID</u> <b>(1)</b> Artificial intelligence-based solutions for COVID-19<ul><li> They listed UpCode, <b>NextStrain</b>, JHU CSSE, Thebaselab, the BBC, the New York Times, and Health Map among the central panels.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35095827">35095827</a> <b>(6)</b> Molecular Characterization of Severe Acute Respiratory Syndrome Coronavirus 2 Isolates From Central Inner Sardinia<ul><li> The sequences were then attributed to a PANGOLIN (Phylogenetic Assignment of Named Global Outbreak LINeages) lineage with PANGOLIN COVID-19 Lineage Assigner v.2.4.2 software (<xref rid="B17" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) and to a <b>NextStrain</b> clade with the Nextclade v0.14.2 web tool (<xref rid="B5" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The same sequences were then attributed to a <b>NextStrain</b> clade, to validate PANGOLIN results.<li>When attributing the same sequences to a <b>NextStrain</b> clade, 8 clades were identified: 20A, 20B, 20C, 20D, 20E (EU1), 20H/501Y.V2 and 20I/501Y.V1.<li> Almost all <b>NextStrain</b> clades in our dataset seem to respect some general correlations when compared to PANGOLIN lineages.<li>When PANGOLIN lineages and <b>NextStrain</b> clades were assigned to the Nuoro sequences, it was possible to assess how the I wave, from March to June 2020, experienced the circulation of the two lineages B.1 and B.1.1 in a proportion of 38.5 and 61.5%, respectively.<li> A discrepancy emerged during the attribution of the <b>NextStrain</b> clade to sample COV1241 was explained when we identified two synonymous mutations present in all B.1.1.7 samples of the III wave and absent in COV1241.<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062359">35062359</a> <b>(2)</b> Prevalence of SARS-CoV-2 Variants of Concern and Variants of Interest in COVID-19 Breakthrough Infections in a Hospital in Monterrey, Mexico<ul><li>Global reference sequences were selected from the global dataset (<italic toggle="yes">n</italic> = 3493) obtained from <b>Nextstrain</b> [<xref rid="B14-viruses-14-00154" ref-type="bibr"><u>14</u></xref>] and were retrieved from the Global Initiative for Sharing All Influenza Data (GISAID) database.<li> Afterward, <b>Nextstrain</b> was used to analyze these sequences.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35055707">35055707</a> <b>(0)</b> Symptomatic SARS-CoV-2 Infection with Ageusia after Two mRNA Vaccine Doses<ul><li> Nextclade software (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 21 April 2021) was used to assign clade and call mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35033580">35033580</a> <b>(1)</b> SARS-CoV-2 RNA loads in Vietnamese children<ul><li> All were assigned to the SARS-CoV-2 Delta variant (sub-lineage AY57) by Pangolin, and lineage 21I by <b>Nextstrain</b> (<bold>Supplementary Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35025877">35025877</a> <b>(1)</b> A short plus long-amplicon based sequencing approach improves genomic coverage and variant detection in the SARS-CoV-2 genome<ul><li> It might be better to show the <b>Nextstrain</b> plot with these samples highlighted to put them in context of the global viral diversity.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35025877">35025877</a> <b>(2)</b> A short plus long-amplicon based sequencing approach improves genomic coverage and variant detection in the SARS-CoV-2 genome<ul><li> It might be better to show the <b>Nextstrain</b> plot with these samples highlighted to put them in context of the global viral diversity.<li> Instead, we have provided a maximum likelihood phylogenetic analysis and a <b>Nextstrain</b> plot on study samples.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34668774">34668774</a> <b>(1)</b> Key Substitutions in the Spike Protein of SARS-CoV-2 Variants Can Predict Resistance to Monoclonal Antibodies, but Other Substitutions Can Modify the Effects<ul><li> The full genomic tree was adapted from <b>nextstrain</b>/ncov (<xref rid="B65" ref-type="bibr"><u>65</u></xref>, <xref rid="B66" ref-type="bibr"><u>66</u></xref>).<li> [65](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [66](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35019707">35019707</a> <b>(8)</b> Molecular and Antigenic Characterization of Avian H9N2 Viruses in Southern China<ul><li> Data and the analysis are from <b>Nextstrain</b> (<xref rid="B58" ref-type="bibr"><u>58</u></xref>).<li> Data and the analysis are from <b>Nextstrain</b> (<xref rid="B58" ref-type="bibr"><u>58</u></xref>).<li> Referring to the global online database of AIV H9N2 in <b>Nextstrain</b>, all these mutations evolved after 2013 and became dominant in Southern China.<li>We then studied the real-time dynamic changing of these potential antigenic sites by generating a phylogenetic tree of the HA protein at every site on one practical online database (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>) (<xref rid="B58" ref-type="bibr"><u>58</u></xref>).<li>Data and analysis from <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/flu/avian/h9n2/ha" ext-link-type="uri">https://<b>nextstrain</b>.org/flu/avian/h9n2/ha</ext-link>) consisted of 1,092 of 1,332 genomes sampled between January 1976 and June 2019.<li> [58](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [58](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [58](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35020754">35020754</a> <b>(1)</b> SARS-CoV-2 variants with reduced infectivity and varied sensitivity to the BNT162b2 vaccine are developed during the course of infection<ul><li> Global swab sampling and sequencing of bulk RNA from infected individuals has allowed to evaluate the inter-host SARS-CoV-2 variability while sequencing databases such as Global Initiative for Sharing All Influenza Data (GISAID) and <b>Nextstrain</b> allow real-time evaluation of SARS-CoV-2 inter-host variability and facilitate tracking of SARS-CoV-2 global spread [<xref rid="ppat.1010242.ref007" ref-type="bibr"><u>7</u></xref>].<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32793182">32793182</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35022216">35022216</a> <b>(1)</b> Epistatic models predict mutable sites in SARS-CoV-2 proteins and epitopes<ul><li> According to <b>Nextstrain</b> (<xref rid="r10" ref-type="bibr"><u>10</u></xref>) global analysis (May 2021, 3,883 genomes), no mutational event has occurred for 58% of the entire proteome, while only 14% has experienced more than two events.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35723403">35723403</a> <b>(1)</b> Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Mutational Pattern in the Fourth Pandemic Phase in Greece<ul><li>Different designations have been given to the variants by different phylogenetic classification systems (e.g., Pango lineage, <b>Nextstrain</b>, GISAID).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35087848">35087848</a> <b>(3)</b> HiSpike Method for High-Throughput Cost Effective Sequencing of the SARS-CoV-2 Spike Gene<ul><li>The SARS-CoV-2 variants Alpha (GISAID GRY, Pango lineage B.1.1.7), and to lesser extent Beta (GISAID GH/501Y.V2, Pango lineage B.1.351) and Gamma (GISAID GR/501Y.V3, Pango lineage P.1) (<xref rid="B28" ref-type="bibr"><u>28</u></xref>&#x2013;<xref rid="B30" ref-type="bibr"><u>30</u></xref>) dominated worldwide outbreaks during the first half of 2021 (GISAID-<b>NextStrain</b>).<li> Since June 2021 the Delta variant (GISAID G/478K.V1, Pango lineage B.1.617.2 and all AY sublineages) completely overtook other variants and is leading in prevalence worldwide (GISAID-<b>NextStrain</b>).<li> In November 2021 the world was shocked by the appearance of the heavily mutated SARS-CoV-2 variant Omicron (GISAID GR/484A, Pango lineage B.1.1.529 and <b>Nextstrain</b> clade 21K).<li> FASTA sequences were uploaded to Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33690265">33690265</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33417829">33417829</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35056592">35056592</a> <b>(0)</b> SARS-CoV-2 Evolution and Spike-Specific CD4+ T-Cell Response in Persistent COVID-19 with Severe HIV Immune Suppression<ul><li> We built the consensus sequences using iVar consensus, with a minimum variant allele frequency (VAF) threshold of 0.5 and a minimum depth of 10 reads, assigning them to a clade with Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, accessed on 19 October 2021) and to a lineage using Pangolin [<xref rid="B7-microorganisms-10-00143" ref-type="bibr"><u>7</u></xref>].<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062327">35062327</a> <b>(6)</b> Discriminatory Weight of SNPs in Spike SARS-CoV-2 Variants: A Technically Rapid, Unambiguous, and Bioinformatically Validated Laboratory Approach<ul><li>To characterize mutations related to the different SARS-CoV-2 variants, we arbitrarily divided the spike (S) gene into eight portions and designed a pair of primers on each portion (<xref rid="viruses-14-00123-t001" ref-type="table"><u>Table 1</u></xref>, <xref rid="viruses-14-00123-f001" ref-type="fig"><u>Figure 1</u></xref>) and a pair of primers for the N gene in position 29,174&#x2013;29,530, which was useful to identify the <b>Nextstrain</b> cluster B.1.160 lineage.<li>Through an initial screening of the <b>Nextstrain</b> database we selected mutations that can be considered &#x201C;unique&#x201D; or characteristic of specific variants to develop a mathematical treatment.<li> In particular, out of 80 viral RNAs, we characterized 56 wild-type (70%), 22 samples belonging to the Alpha clone (27.5%), and two (2.5%) RNAs exhibiting mutations attributable to the <b>Nextstrain</b> cluster B.1.160 clone.<li> It should be noted that, owing to the lack of the A222V mutation, the <b>Nextstrain</b> Cluster 20A.EU.1 variant was overlooked by that test, causing a drop to 0.84 in the estimated overall sensitivity.<li> Indeed, it should be evidenced that the <b>Nextstrain</b> Cluster 20A.EU.1 represents a minor variant and its prevalence was probably overestimated in our model (leading to an underestimation of the overall sensitivity).<li>In any case, the A222V mutation, the only unique mutation for the <b>Nextstrain</b> Cluster 20A.EU.1&#x201D; variant, can be sequenced using three primer combinations constantly provided in our work (FS-1F/FS-2R; FS-2F/FS-3R; and FS-3F/FS-4R).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062324">35062324</a> <b>(1)</b> SARS-CoV-2 Reverse Zoonoses to Pumas and Lions, South Africa<ul><li> The 43,322 sequences were subsampled to 1545 in <b>Nextstrain</b> based on genetic proximity to the study sequences [<xref rid="B32-viruses-14-00120" ref-type="bibr"><u>32</u></xref>].<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35017107">35017107</a> <b>(1)</b> Evidence of SARS-CoV-2 symptomatic reinfection in four healthcare professionals from the same hospital despite the presence of antibodies<ul><li> Clade information using the GISAID and <b>Nextstrain</b> nomenclatures is shown.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35021068">35021068</a> <b>(2)</b> Emergence of a recurrent insertion in the N-terminal domain of the SARS-CoV-2 spike glycoprotein<ul><li>SARS-CoV-2 genomes were analyzed with the <b>nextstrain</b> <italic>augur</italic> pipeline (https://github.com/<b>nextstrain</b>/augur).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35000004">35000004</a> <b>(0)</b> Emergence of two distinct variants of SARS-CoV-2 and an explosive second wave of COVID-19: the experience of a tertiary care hospital in Pune, India<ul><li> The Nextclade web tool (v0.14.2) (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>) was used to compare the sequences to those of reference strains for clade assignment.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35042229">35042229</a> <b>(3)</b> Rapid epidemic expansion of the SARS-CoV-2 Omicron variant in southern Africa<ul><li> Preliminary maximum-likelihood phylogenies identified the Omicron&#xA0;BA.1 sequences as a monophyletic clade rooted within the B.1.1 lineage (<b>Nextstrain</b> clade 20B), with no clear basal progenitor (Fig.<li> (2) a representative subsampling of global data from the publicly maintained global build of Nexstrain (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>); and (3) the top thirty BLAST hits when querying GISAID BLAST for BA.1 and BA.2 sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35018222">35018222</a> <b>(0)</b> First detection of SARS-CoV-2 variant B.1.1.529 (Omicron) in Ecuador<ul><li>Following the Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0010">https://clades.<b>nextstrain</b>.org/</ext-link>) [<xref rid="bib8" ref-type="bibr"><u>8</u></xref>] and Pangolin (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0015">https://github.com/cov-lineages/pangolin</ext-link>) package [<xref rid="bib9" ref-type="bibr"><u>9</u></xref>], the annotated genome clustered into 21K clade and BA.1 lineage respectively, which correspond to Omicron VOC.<li> [8](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J&#xA0;Open Source Softw</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [9](Centre for Genomic Pathogen and Surveillance. Pangolin app 2021:1. Centre for genomic pathogen and surveillance (accessed 21 December, 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062308">35062308</a> <b>(4)</b> A Pre-Vaccination Baseline of SARS-CoV-2 Genetic Surveillance and Diversity in the United States<ul><li> Sequences from VOCs: Alpha (Pango lineage B.1.1.7, <b>Nextstrain</b> 20I, GISAID clade GR, originally isolated in the U.K.), Beta (B.1.351, 20H, GH, South Africa), Gamma (P.1, 20J, GR, Brazil), and Epsilon (B.1.427+B.1.429, 21C, GH, United States) were accessed from gisaid.org on 1 April 2021; and Delta (B.1.617.2, 21A, GK, India) were accessed from gisaid.org on 27 July 2021 (see also <xref rid="sec2dot5-viruses-14-00104" ref-type="sec"><u>Section 2.5</u></xref>) [<xref rid="B9-viruses-14-00104" ref-type="bibr"><u>9</u></xref>,<xref rid="B10-viruses-14-00104" ref-type="bibr"><u>10</u></xref>,<xref rid="B11-viruses-14-00104" ref-type="bibr"><u>11</u></xref>].<li> These included: PANGO lineages (<uri xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</uri>, accessed 17 December 2020) [<xref rid="B25-viruses-14-00104" ref-type="bibr"><u>25</u></xref>], <b>NextStrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</uri>, accessed 17 December 2020) [<xref rid="B26-viruses-14-00104" ref-type="bibr"><u>26</u></xref>], Global Initiative on Sharing Avian Influenza Data (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>, accessed 17 December 2020) [<xref rid="B9-viruses-14-00104" ref-type="bibr"><u>9</u></xref>,<xref rid="B10-viruses-14-00104" ref-type="bibr"><u>10</u></xref>], Outbreak.info (<uri xlink:href="https://outbreak.info/situation-reports">https://outbreak.info/situation-reports</uri>, accessed 17 December 2020) [<xref rid="B27-viruses-14-00104" ref-type="bibr"><u>27</u></xref>], Observable (<uri xlink:href="https://observablehq.com/@spond/linkage-disequilibirum-in-sars-cov-2">https://observablehq.com/@spond/linkage-disequilibirum-in-sars-cov-2</uri>, accessed 17 December 2020), Virological forum (<uri xlink:href="https://virological.org/">https://virological.org/</uri>, accessed 17 December 2020), Los Alamos National Laboratory (<uri xlink:href="https://cov.lanl.gov/content/index">https://cov.lanl.gov/content/index</uri>, accessed 17 December 2020), and the U.S.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [10](          <person-group person-group-type="author"><name><surname>Shu</surname><given-names>Y.</given-names></name><name><surname>McCauley</surname><given-names>J.</given-names></name></person-group>          <article-title>GISAID: Global initiative on sharing all influenza data&#x2014;From vision to reality</article-title>          <source>Euro Surveill. Bull. Eur. Sur Les Mal. Transm. Eur. Commun. Dis. Bull.</source>          <year>2017</year>          <volume>22</volume>          <fpage>30494</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>        ) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34535792">34535792</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [10](          <person-group person-group-type="author"><name><surname>Shu</surname><given-names>Y.</given-names></name><name><surname>McCauley</surname><given-names>J.</given-names></name></person-group>          <article-title>GISAID: Global initiative on sharing all influenza data&#x2014;From vision to reality</article-title>          <source>Euro Surveill. Bull. Eur. Sur Les Mal. Transm. Eur. Commun. Dis. Bull.</source>          <year>2017</year>          <volume>22</volume>          <fpage>30494</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>        ) [27](          <person-group person-group-type="author"><name><surname>Julia</surname><given-names>L.</given-names></name><name><surname>Mullen</surname><given-names>G.T.</given-names></name><name><surname>Latif</surname><given-names>A.A.</given-names></name><name><surname>Alkuzweny</surname><given-names>M.</given-names></name><name><surname>Cano</surname><given-names>M.</given-names></name><name><surname>Haag</surname><given-names>E.</given-names></name><name><surname>Zhou</surname><given-names>J.</given-names></name><name><surname>Zeller</surname><given-names>M.</given-names></name><name><surname>Hufbauer</surname><given-names>E.</given-names></name><name><surname>Matteson</surname><given-names>N.</given-names></name><etal/></person-group>          <article-title>Outbreak.info</article-title>          <comment>Available online: <ext-link xlink:href="https://outbreak.info/" ext-link-type="uri">https://outbreak.info/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2020-12-17">(accessed on 17 December 2020)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34999740">34999740</a> <b>(0)</b> High-Throughput Adaptable SARS-CoV-2 Screening for Rapid Identification of Dominant and Emerging Regional Variants<ul><li> ThermoFisher Scientific) as previously described.<sup><xref rid="CIT0010" ref-type="bibr"><u>10</u></xref></sup> Pangolin (pangolin.cog-uk.io, v.3.1.3, 8/6/2021) and NextClad (clades.<b>nextstrain</b>.org, v.1.5.4) were used for lineage or clade assignment, respectively.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32451533">32451533</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34995482">34995482</a> <b>(2)</b> mRNA-based COVID-19 vaccine boosters induce neutralizing immunity against SARS-CoV-2 Omicron variant<ul><li>(A) Phylogenetic tree of SARS-CoV-2 variants with sampling dates shows emergence of Omicron variant by December 2021 (adapted from <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0105"><b>nextstrain</b>.org</ext-link>; updated as of December 14, 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34992214">34992214</a> <b>(3)</b> Inferring the stabilization effects of SARS-CoV-2 variants on the binding with ACE2 receptor<ul><li> The phylogenetic tree in Fig.&#xA0;<xref rid="Fig1" ref-type="fig"><u>1</u></xref>d was obtained using the publicly available interactive visualization platform <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR34"><u>34</u></xref></sup> and it is based on the contribution of 4025 genomes sampled between December 2019 and April 2021, stored and elaborated in the Nextstation database and bioinformatics pipeline for phylodynamics analysis<sup><xref ref-type="bibr" rid="CR34"><u>34</u></xref></sup>.<li> A full list of the sequence authors is available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2">https://<b>nextstrain</b>.org/sars-cov-2</ext-link>.<li> [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35071285">35071285</a> <b>(2)</b> Isolation and Identification of a Rare Spike Gene Double-Deletion SARS-CoV-2 Variant From the Patient With High Cycle Threshold Value<ul><li> Alignment and sequence comparisons of KMUH-1 and KMUH-2 to the reference sequence NC_045512.2 The SARS-CoV-2 isolate Wuhan-Hu-1/2019 was performed using Nextclade v1.7.3 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="B38" ref-type="bibr"><u>38</u></xref>) to verify the results of the virus detail report from GISAID.<li> The viruses are shown as the virus name (GISAIG) or accession number (GenBank)/sample collection year (date/pangolin_lineage/GISAID_clade/<b>nextstrain</b>_clade).<li> The phylogenetic tree data were visualized using Auspice v2.32.0 powered by <b>Nextstrain</b> (<xref rid="B53" ref-type="bibr"><u>53</u></xref>).<li> [38](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>. <source>Nextclade: Clade Assignment, Mutation Calling Quality Control for Viral Genomes.</source> (<year>2021</year>). Available online at: <ext-link xlink:href="https://zenodo.org/record/5607694" ext-link-type="uri">https://zenodo.org/record/5607694</ext-link> (accessed December 20, 2021).) [53](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34999288">34999288</a> <b>(1)</b> The dynamic change of SARS-CoV-2 variants in Sierra Leone<ul><li> The subtrees were generated by Ultrafast Sample placement on Existing tRee (UShER) and explored using the <b>Nextstrain</b> platform.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35096645">35096645</a> <b>(1)</b> Experimental Models of COVID-19<ul><li> <b>Nextstrain</b> divides the variants into five clades (19A, 19B, 20A, 20B, and 20C), while GISAID divides them into seven (L, O, V, S, G, GH, and GR).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34982246">34982246</a> <b>(2)</b> Genomic Characterization of SARS-CoV2 from Peshawar Pakistan Using Next-Generation Sequencing<ul><li> A total of 85 SARS-CoV2 genomes including the current sequenced genome (accession number: MW242667) were clustered using Augur <b>Nextstrain</b>&#x2019;s phylodynamic pipeline (Supplementary Material) [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>].<li> The selected genomes were clustered using Augur, the <b>Nextstrain</b> phylogenetic pipeline.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34842496">34842496</a> <b>(0)</b> A year living with SARS-CoV-2: an epidemiological overview of viral lineage circulation by whole-genome sequencing in Barcelona city (Catalonia, Spain)<ul><li> Furthermore, it performs lineage/clade determination and mutation characterization by using updated Pangolin (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>) and NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) nomenclatures.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34995777">34995777</a> <b>(1)</b> A CLUSTER OF SARS&#x2010;COV&#x2010;2 DELTA VARIANT OF CONCERN ADDITIONALLY HARBORING F490S, NORTHERN LOMBARDY, ITALY<ul><li> by the World Health Organization (WHO), VOC-21APR-02 by Public Health England, 21A/S:478K by <b>NextStrain</b>, and G/452R.V3 by GISAID.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062736">35062736</a> <b>(3)</b> Analysis of Neutralization Titers against SARS-CoV-2 in Health-Care Workers Vaccinated with Prime-Boost mRNA&#x2013;mRNA or Vector&#x2013;mRNA COVID-19 Vaccines<ul><li> The sequence of this SARS-CoV-2 isolate (DK-AHH1) can be found in GenBank (accession number MZ049597) [<xref rid="B23-vaccines-10-00075" ref-type="bibr"><u>23</u></xref>] and belongs to the <b>Nextstrain</b> Clade 20C.<li> In addition, the swab was submitted for viral sequencing as previously described [<xref rid="B23-vaccines-10-00075" ref-type="bibr"><u>23</u></xref>], and the SARS-CoV-2 isolate was found to be an alpha variant (B.1.1.7, Genbank accession OK041529, <b>Nextstrain</b> Clade 20I (Alpha V1)).<li>During this study, one vaccinated individual became infected with a SARS-CoV-2 variant belonging to <b>Nextstrain</b> Clade 20I (Alpha V1), which allowed a unique opportunity to analyze longitudinal neutralization to heterologous and autologous virus following vaccination.<li> [23](          <person-group person-group-type="author"><name><surname>Ramirez</surname><given-names>S.</given-names></name><name><surname>Fernandez-Antunez</surname><given-names>C.</given-names></name><name><surname>Galli</surname><given-names>A.</given-names></name><name><surname>Underwood</surname><given-names>A.</given-names></name><name><surname>Pham</surname><given-names>L.V.</given-names></name><name><surname>Ryberg</surname><given-names>L.A.</given-names></name><name><surname>Feng</surname><given-names>S.</given-names></name><name><surname>Pedersen</surname><given-names>M.S.</given-names></name><name><surname>Mikkelsen</surname><given-names>L.S.</given-names></name><name><surname>Belouzard</surname><given-names>S.</given-names></name><etal/></person-group>          <article-title>Overcoming Culture Restriction for SARS-CoV-2 in Human Cells Facilitates the Screening of Compounds Inhibiting Viral Replication</article-title>          <source>Antimicrob. Agents Chemother.</source>          <year>2021</year>          <volume>65</volume>          <fpage>e00097-21</fpage>          <pub-id pub-id-type="doi">10.1128/AAC.00097-21</pub-id>        ) [23](          <person-group person-group-type="author"><name><surname>Ramirez</surname><given-names>S.</given-names></name><name><surname>Fernandez-Antunez</surname><given-names>C.</given-names></name><name><surname>Galli</surname><given-names>A.</given-names></name><name><surname>Underwood</surname><given-names>A.</given-names></name><name><surname>Pham</surname><given-names>L.V.</given-names></name><name><surname>Ryberg</surname><given-names>L.A.</given-names></name><name><surname>Feng</surname><given-names>S.</given-names></name><name><surname>Pedersen</surname><given-names>M.S.</given-names></name><name><surname>Mikkelsen</surname><given-names>L.S.</given-names></name><name><surname>Belouzard</surname><given-names>S.</given-names></name><etal/></person-group>          <article-title>Overcoming Culture Restriction for SARS-CoV-2 in Human Cells Facilitates the Screening of Compounds Inhibiting Viral Replication</article-title>          <source>Antimicrob. Agents Chemother.</source>          <year>2021</year>          <volume>65</volume>          <fpage>e00097-21</fpage>          <pub-id pub-id-type="doi">10.1128/AAC.00097-21</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34979019">34979019</a> <b>(0)</b> Serum Levels of Proinflammatory Lipid Mediators and Specialized Proresolving Molecules Are Increased in Patients With Severe Acute Respiratory Syndrome Coronavirus 2 and Correlate With Markers of the Adaptive Immune Response<ul><li> While it is possible that there may be mismatches between the antigen used in our assay and the strain of infecting virus that mean that antibodies are present which are undetected by our assay, at the time of sampling there was minimal genetic diversity in United Kingdom isolates (<ext-link xlink:href="http://www.<b>nextstrain</b>.org" ext-link-type="uri">www.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35229003">35229003</a> <b>(1)</b> Assessment of Coronavirus Disease 2019 Intervention Strategies in the Nordic Countries Using Genomic Epidemiology<ul><li>The data were generated from [<xref rid="CIT0003" ref-type="bibr"><u>3</u></xref>] by (1) downloading and aligning all available Nordic SARS-CoV-2 genomes from GISAID ([<ext-link xlink:href="http://www.gisaid.org" ext-link-type="uri">www.gisaid.org</ext-link>] with 67&#xA0;918 Nordic genomes) and (2) the <b>NextStrain</b> global build as of March 22, 2021 (3437 global genomes).<li> [3](<person-group person-group-type="author"><string-name><surname>Duchene</surname><given-names>S</given-names></string-name>, <string-name><surname>Featherstone</surname><given-names>L</given-names></string-name>, <string-name><surname>de Blasio</surname><given-names>BF</given-names></string-name>, <string-name><surname>Holmes</surname><given-names>EC</given-names></string-name>, <string-name><surname>Bohlin</surname><given-names>J</given-names></string-name>, <string-name><surname>Pettersson</surname><given-names>JH-O.</given-names></string-name></person-group><article-title>The impact of public health interventions in the Nordic countries during the first year of SARS-CoV-2 transmission and evolution.</article-title><source>Euro Surveill</source><year>2021</year>; <volume>26</volume>:<fpage>2001996</fpage>. doi:<pub-id pub-id-type="doi">10.2807/1560-7917.ES.2021.26.44.2001996</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062282">35062282</a> <b>(1)</b> The Remarkable Evolutionary Plasticity of Coronaviruses by Mutation and Recombination: Insights for the COVID-19 Pandemic and the Future Evolutionary Paths of SARS-CoV-2<ul><li> Currently, three nomenclature systems are being used: the GISAID (Global Initiative on Sharing All Influenza Data), <b>Nextstrain</b>, and Pango [<xref rid="B19-viruses-14-00078" ref-type="bibr"><u>19</u></xref>,<xref rid="B20-viruses-14-00078" ref-type="bibr"><u>20</u></xref>,<xref rid="B21-viruses-14-00078" ref-type="bibr"><u>21</u></xref>,<xref rid="B22-viruses-14-00078" ref-type="bibr"><u>22</u></xref>].<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34108654">34108654</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35074623">35074623</a> <b>(3)</b> Rapid and sensitive detection of SARS-CoV-2 variants in nasopharyngeal swabs and wastewaters<ul><li> On January 12, 2020, thanks to next-generation sequencing (NGS) approaches, carried out by different research institutes, the first complete genome of SARS-CoV-2 was determined (<xref rid="bib0009" ref-type="bibr"><u>Kim&#xA0;et al., 2020</u></xref>) and released via the GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/CoV2020/" id="interref0002">https://www.gisaid.org/CoV2020/</ext-link>) and <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="interref0003">https://<b>nextstrain</b>.org/ncov</ext-link>) databases.<li> Clade assignment was performed using COVIDEX (<xref rid="bib0002" ref-type="bibr"><u>Cacciabue&#xA0;et al., 2020</u></xref>) and Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0004">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> (A) Phylogenetic tree reporting the major SARS-CoV-2 circulating clades (adapted from <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0001">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [Kim&#xA0;et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32330414">32330414</a>) [Cacciabue&#xA0;et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Cacciabue</surname>              <given-names>M</given-names>            </name>            <name>              <surname>Aguilera</surname>              <given-names>P</given-names>            </name>            <name>              <surname>Gismondi</surname>              <given-names>MI</given-names>            </name>            <name>              <surname>Taboga</surname>              <given-names>O.</given-names>            </name>          </person-group>          <article-title>Covidex: an ultrafast and accurate tool for virus subtyping</article-title>          <source>bioRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.08.21.261347</pub-id>          <comment>2020.08.21.261347</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34843661">34843661</a> <b>(3)</b> Systematic Genomic and Clinical Analysis of Severe Acute Respiratory Syndrome Coronavirus 2 Reinfections and Recurrences Involving the Same Strain<ul><li> The phylogeny includes the 3,339 genomes from <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>), extracted from the Europe-focused subsampling, through April 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34807815">34807815</a> <b>(2)</b> Emergence of SARS-CoV-2 Delta Variant, Benin, May&#x2013;July 2021<ul><li> In addition, our data show that it took &#x2248;2 months for Delta VOC strains to become predominant in Benin, which is roughly comparable to findings from India, where Delta became the predominant variant detected by genomic surveillance, with a frequency of 16% in March 2021 to 83% in April 2021 (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36777001">36777001</a> <b>(1)</b> Stress adaptation signature into the functional units of spike, envelope, membrane protein and ssRNA of SARS-CoV-2<ul><li><bold>Structural Simulation with diversity: </bold>Variant of concerns (VOCs) were reported by CDC i.e alpha, beta, gamma, delta, and omicron from the five most heavily affected countries (China, USA, India, South Africa, and Australia) also represented by &#x201C;<b>Nextstrain</b> / ncov / open / global / 6m&#x201D;.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062728">35062728</a> <b>(2)</b> Development of an Oral <italic toggle="yes">Salmonella</italic>-Based Vaccine Platform against SARS-CoV-2<ul><li>The sequence of SARS-CoV-2 virus Spike gene was elicited from valid databases (e.g., <uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>, <uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, <uri xlink:href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</uri>) (asscssed on 18 July 2020) and SARS-CoV-2 S-2P, comprising proline substitutions at residues K986 and V987, was chosen as the target antigen.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062281">35062281</a> <b>(3)</b> Intra-Host SARS-CoV-2 Evolution in the Gut of Mucosally-Infected <italic toggle="yes">Chlorocebus aethiops</italic> (African Green Monkeys)<ul><li> The S2103F mutation in this protein has been detected at low frequency in human samples, including in India [<xref rid="B29-viruses-14-00077" ref-type="bibr"><u>29</u></xref>], but is not restricted to a particular clade and, through the course of the pandemic, has never been detected in greater than 1% of all sequenced isolates (<uri><b>nextstrain</b>.org</uri> accessed on 29 November 2021 [<xref rid="B30-viruses-14-00077" ref-type="bibr"><u>30</u></xref>,<xref rid="B31-viruses-14-00077" ref-type="bibr"><u>31</u></xref>]).<li> This mutation lies in the N-terminal domain of the spike protein (<xref rid="viruses-14-00077-f001" ref-type="fig"><u>Figure 1</u></xref>C) and is detected at low frequency in multiple viral lineages and is a defining variant of the B.1.351 (beta) lineage (<b>nextstrain</b>.org).<li> The H655Y mutation lies immediately adjacent to the furin cleavage site (<xref rid="viruses-14-00077-f001" ref-type="fig"><u>Figure 1</u></xref>C) and has been detected in multiple viral lineages, with the P.1 (gamma) and B.1.1.529 (omicron) lineages being most prominent (<b>nextstrain</b>.org).<li> [29](          <person-group person-group-type="author"><name><surname>Roy</surname><given-names>S.</given-names></name><name><surname>Nath</surname><given-names>H.</given-names></name><name><surname>Mallick</surname><given-names>A.</given-names></name><name><surname>Biswas</surname><given-names>S.</given-names></name></person-group>          <article-title>SARS-CoV-2 has observably higher propensity to accept uracil as nucleotide 1 substitution: Prevalence of amino acid substitutions and their predicted functional 2 implications in circulating SARS-CoV-2 in India up to</article-title>          <source>bioRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.10.07.329771</pub-id>        ) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35005525">35005525</a> <b>(1)</b> Origin and evolutionary analysis of the SARS-CoV-2 Omicron variant<ul><li> The time-scaled phylogenetic trees were constructed using <b>NextStrain</b><xref rid="b0115" ref-type="bibr"><sup>23</sup></xref> and Treetime<xref rid="b0120" ref-type="bibr"><sup>24</sup></xref> and visualized with FigTree v1.4.4.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35045382">35045382</a> <b>(0)</b> A deletion in the N gene of SARS-CoV-2 may reduce test sensitivity for detection of SARS-CoV-2<ul><li> The genetic lineages of the sequences was determined using Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin, pangolin.cog-uk.io) and Nextclade (clades.<b>nextstrain</b>.org).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34962592">34962592</a> <b>(3)</b> Detection of SARS-CoV-2-RNA in post-mortem samples of human eyes<ul><li> Phylogenetic analyses were achieved with the web and analysis platform Auspice using the SARS-Cov-2 build (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) and the bioinformatic toolkit augur.<li> <b>Nextstrain</b>/Pangolin/WHO nomenclature clades are indicated above the main branches.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34961403">34961403</a> <b>(1)</b> An update on host immunity correlates and prospects of re-infection in COVID-19<ul><li><bold><b>Nextstrain</b></bold><break/><bold>clade</bold></th>              <th align="center">Country of origin</th>              <th align="center">Date of designation</th>            </tr>          </thead>          <tbody valign="top">            <tr>              <td rowspan="4" align="left"><bold>Variants of concern (VOC)</bold>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34954792">34954792</a> <b>(4)</b> CoV-Spectrum: analysis of globally shared SARS-CoV-2 data to identify and characterize new variants<ul><li> We currently provide two instances of CoV-Spectrum: one that uses data provided by GISAID (<xref rid="btab856-B7" ref-type="bibr"><u>Elbe and Buckland-Merrett, 2017</u></xref>), and another one that uses data from NCBI GenBank provided through <b>Nextstrain</b> (<xref rid="btab856-B9" ref-type="bibr">Hadfield <italic toggle="yes">et al.<li> We clean the location data with <b>Nextstrain</b>&#x2019;s geo location rules (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov-ingest/blob/master/source-data/gisaid_geoLocationRules.tsv" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov-ingest/blob/master/source-data/gisaid_geoLocationRules.tsv</ext-link>) and we run Nextclade (<xref rid="btab856-B1" ref-type="bibr">Aksamentov <italic toggle="yes">et al.<li> [Elbe and Buckland-Merrett, 2017](<person-group person-group-type="author"><string-name><surname>Elbe</surname><given-names>S.</given-names></string-name>, <string-name><surname>Buckland-Merrett</surname><given-names>G.</given-names></string-name></person-group> (<year>2017</year>) <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>. <source>Global Chall</source>., <volume>1</volume>, <fpage>33</fpage>&#x2013;<lpage>46</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34968782">34968782</a> <b>(2)</b> Mutations on RBD of SARS-CoV-2 Omicron variant result in stronger binding to human ACE2 receptor<ul><li> The tree is generated at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0025">https://<b>nextstrain</b>.org</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35047415">35047415</a> <b>(0)</b> Learning From Biological and Computational Machines: Importance of SARS-CoV-2 Genomic Surveillance, Mutations and Risk Stratification<ul><li> SARS-CoV-2 clades were assigned using Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34954396">34954396</a> <b>(3)</b> Evidence for a mouse origin of the SARS-CoV-2 Omicron variant<ul><li> The progenitors of the other four VOCs (Alpha, Beta, Gamma, and Delta) were retrieved from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0015">https://<b>nextstrain</b>.org/</ext-link>) (<xref rid="bib9" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34951469">34951469</a> <b>(1)</b> NGSocomial Infections: High-Resolution Views of Hospital-Acquired Infections Through Genomic Epidemiology<ul><li> however, platforms such as NCBI Pathogen Detection for bacterial outbreaks and <b>Nextstrain</b> for viral WGS data have helped unite the field in visualization and analysis strategies [<xref rid="CIT0116" ref-type="bibr"><u>116</u></xref>, <xref rid="CIT0117" ref-type="bibr"><u>117</u></xref>].<li> [116](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33095870">33095870</a>) [117](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35003200">35003200</a> <b>(10)</b> Genetic Drift Versus Climate Region Spreading Dynamics of COVID-19<ul><li> SARS-CoV-2 possesses a single-strand RNA genome (<xref rid="B22" ref-type="bibr"><u>Mousavizadeh and Ghasemi, 2020</u></xref>), prone to acquire genomic mutations (<ext-link xlink:href="http://<b>nextstrain</b>.org/ncov/europe" ext-link-type="uri"><b>nextstrain</b>.org/ncov/</ext-link>; <ext-link xlink:href="http://www.gisaid.org/" ext-link-type="uri">www.gisaid.org/</ext-link>).<li> Still, SARS-CoV-2-bearing distinct sets of mutations were isolated in different regions of the world (<b>nextstrain</b>.org/ncov/global), leaving the question open as to whether viral genetic drift, driving distinct disease evolution, could account for heterogeneous disease courses across different geographic areas.<li>SARS-CoV-2 genomic RNA sequences and country-correlated data were obtained from <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global" ext-link-type="uri"><b>nextstrain</b>.org/ncov/global</ext-link>.<li> Phylogeny trees for such mutation clusters were then obtained for drawing distinct evolutionary branches of SARS-CoV-2 (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/europe?branchLabel=aa" ext-link-type="uri"><b>nextstrain</b>.org/ncov/europe?branchLabel=aa</ext-link>) (<xref rid="F2" ref-type="fig"><u>Figure 2</u></xref>, <xref rid="s10" ref-type="sec"><u>Supplementary Figures S2&#x2013;S9</u></xref>).<li>Sequence mutation analysis revealed different branches of acquired mutations, i.e., distinct groups of viral genome mutations (haplotypes), at sites of major diffusion in Europe (<ext-link xlink:href="http://<b>nextstrain</b>.org/ncov/europe" ext-link-type="uri"><b>nextstrain</b>.org/ncov/europe</ext-link>) (<xref rid="F2" ref-type="fig"><u>Figure 2</u></xref>, <xref rid="s10" ref-type="sec"><u>Supplementary Figures S2&#x2013;S9</u></xref>).<li> The D614G mutation appeared early in Europe (inferred date, January 6, 2020) (<b>nextstrain</b>.org/ncov/; <ext-link xlink:href="http://www.gisaid.org/" ext-link-type="uri">www.gisaid.org/</ext-link>) (<xref rid="B16" ref-type="bibr"><u>Korber et al., 2020</u></xref>) and was subsequently found to spread evenly across European countries.<li> [Mousavizadeh and Ghasemi, 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32265180">32265180</a>) [Korber et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34940844">34940844</a> <b>(2)</b> Yearlong COVID-19 Infection Reveals Within-Host Evolution of SARS-CoV-2 in a Patient With B-Cell Depletion<ul><li> <xref rid="sup2" ref-type="supplementary-material"><u>Supplementary Table 1</u></xref>) were generated using <b>Nextstrain</b> with default parameters [<xref rid="CIT0007" ref-type="bibr"><u>7</u></xref>].<li> Consensus sequences were mapped onto a phylogenetic tree containing 266 background samples from Maryland collected between May 2020 and March 2021 (<xref rid="sup2" ref-type="supplementary-material"><u>Supplementary Table 1</u></xref>), using the publicly available <b>Nextstrain</b> software package (<xref rid="F1" ref-type="fig"><u>Figure 1B</u></xref>) [<xref rid="CIT0007" ref-type="bibr"><u>7</u></xref>].<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062227">35062227</a> <b>(1)</b> Emerging SARS-CoV-2 Genotypes Show Different Replication Patterns in Human Pulmonary and Intestinal Epithelial Cells<ul><li> The classification of the isolates was determined using the Nextclade tool (<b>Nextstrain</b>, Nextclade: <uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 20 May 2021 and 27 November 2021) and the Pangolin classification tool (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri> (accessed on 20 May 2021), and for the name of the mutation.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34937172">34937172</a> <b>(6)</b> Identification of Hemagglutinin Mutations Caused by Neuraminidase Antibody Pressure<ul><li> To identify whether NA antibody pressure is involved in HA antigenic evolution, the evolutionary correlations between NA and HA of H9N2 IAVs in China were analyzed online with <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/flu/avian/h9n2" ext-link-type="uri">https://<b>nextstrain</b>.org/flu/avian/h9n2</ext-link>).<li> The major branch of the phylogenetic tree of full-length HA and NA genes of H9N2 IAVs isolated in China from 1994 to 2020 was generated with <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/flu/avian/h9n2" ext-link-type="uri">https://<b>nextstrain</b>.org/flu/avian/h9n2</ext-link>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34993157">34993157</a> <b>(1)</b> Clinical, Virological, Immunological, and Genomic Characterization of Asymptomatic and Symptomatic Cases With SARS-CoV-2 Infection in India<ul><li>The mutational and phylogenetic analysis was carried out using the <b>Nextstrain</b> tool as described before (<xref rid="B19" ref-type="bibr"><u>Raghav et&#xA0;al., 2020</u></xref>).<li> [Raghav et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33329480">33329480</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062214">35062214</a> <b>(2)</b> Tracking SARS-CoV-2 Spike Protein Mutations in the United States (January 2020&#x2014;March 2021) Using a Statistical Learning Strategy<ul><li> The <b>NextStrain</b> methodology [<xref rid="B17-viruses-14-00009" ref-type="bibr"><u>17</u></xref>] can model dynamic changes in variant proportions, while an alternative approach aligns sequence data to a matrix of binary indicators for the presence of variants, and systematically evaluates each mutant as a potential variant [<xref rid="B18-viruses-14-00009" ref-type="bibr"><u>18</u></xref>].<li> However, with the exception of the <b>NextStrain</b> methodology [<xref rid="B3-viruses-14-00009" ref-type="bibr"><u>3</u></xref>], these methods do not directly take into account sequence collection time, nor explicitly incorporate highly granular geographic information.<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32742035">32742035</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33991487">33991487</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34922338">34922338</a> <b>(0)</b> Infection With the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Delta Variant Is Associated With Higher Recovery of Infectious Virus Compared to the Alpha Variant in Both Unvaccinated and Vaccinated Individuals<ul><li> Mutations were visually confirmed with Integrated Genomics viewer (IGV) (Version 2.8.10), clades were determined using Nextclade beta v 0.12.0 (clades.<b>nextstrain</b>.org), and lineages were determined with Pangolin COVID-19 lineage Assigner (COG-UK (cog-uk.io)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34960813">34960813</a> <b>(3)</b> Prediction of SARS-CoV-2 Variant Lineages Using the S1-Encoding Region Sequence Obtained by PacBio Single-Molecule Real-Time Sequencing<ul><li>Data analysis: the resulting sequence data were processed using DRAGEN COVID Lineage (v3.5.1) (Illumina, CA, USA) to obtain consensus sequences, <b>Nextstrain</b> clades (<uri xlink:href="https://<b>nextstrain</b>.org/sars-cov-2">https://<b>nextstrain</b>.org/sars-cov-2</uri>, accessed on 31 August 2021), and Pangolin lineages (<uri xlink:href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</uri>, accessed on 31 August 2021) when at least 5 SARS-CoV-2 probes were detected.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34976008">34976008</a> <b>(13)</b> Geographical Landscape and Transmission Dynamics of SARS-CoV-2 Variants Across India: A Longitudinal Perspective<ul><li> The comprehensive data analysis with in-depth phylogeny, lineage, co-analysis, and an interactive visualization tool by <b>Nextstrain</b>, adds to the strength of the study (<xref rid="B14" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The dashboard would augment other existing ones, inclusive of <ext-link xlink:href="https://<b>nextstrain</b>.org/community/banijolly/Phylovis/COVID-India" ext-link-type="uri">https://<b>nextstrain</b>.org/community/banijolly/Phylovis/COVID-India</ext-link>, with an aim to strengthen the SARS-CoV-2 genome surveillance and information dissemination.<li> Clade, lineage, and mutation information was processed by the <b>Nextstrain</b> system to produce a divergence plot with respect to lineages and clade distribution across Indian states/UTs (<xref rid="F6" ref-type="fig"><u>Figures 6A,B</u></xref>).<li>To enable an understanding of the transmission and evolution of the SARS-CoV-2 genomes that were analyzed in this study, we built a transmission map to display transmission patterns of the phylogenetic analysis by utilizing the <b>Nextstrain</b> system by employing collection date, location, and sequence similarity data.<li> <b>Nextstrain</b> allows for users to define lineages and clades.<li> In order to systematically track the evolution of the SARS-CoV-2 genome, we plan to continue the addition of new SARS-CoV-2 genomes in regular intervals to the <b>Nextstrain</b> system.<li> Other efforts toward building an interactive display of India-specific SARS-CoV-2 genomic data have been observed from India such as <ext-link xlink:href="https://<b>nextstrain</b>.org/community/banijolly/Phylovis/COVID-India" ext-link-type="uri">https://<b>nextstrain</b>.org/community/banijolly/Phylovis/COVID-India</ext-link> (<xref rid="B19" ref-type="bibr"><u>Jolly and Scaria, 2021</u></xref>) and <ext-link xlink:href="https://data.ccmb.res.in/gear19/" ext-link-type="uri">https://data.ccmb.res.in/gear19/</ext-link>, that use data from GISAID and other genomic surveillance initiatives.<li> Henceforth, 6,928 sequences from the phylogenetic analysis were used to produce a phylogenetic tree, transmission map, mutation spectrum, and genomic divergence using the <b>Nextstrain</b> package Augur, containing python scripts for various filtering steps throughout the <b>Nextstrain</b>&#x2019;s pipeline (<xref rid="B16" ref-type="bibr"><u>Huddleston et al., 2021</u></xref>).<li> Some changes were made to the original commands provided by <b>Nextstrain</b>&#x2019;s SARS-CoV-2 build on Github (<xref rid="B12" ref-type="bibr"><u>GitHub., 2021</u></xref>) to include our samples in the phylogenetic tree.<li> For generating an interactive visualization web tool for the obtained phylogeographic data, Auspice (<b>Nextstrain</b>) is used.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Jolly and Scaria, 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34124300">34124300</a>) [Huddleston et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [GitHub., 2021](<collab>GitHub</collab> (<year>2021</year>). <article-title>Viral Genome Alignment, Clade Assignment, Mutation Calling, and Quality Checks</article-title>. <comment>Available at: <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link> (Accessed July 6, 2021)</comment>. ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34975897">34975897</a> <b>(0)</b> Anti-SARS-CoV-2 Immunoglobulin Isotypes, and Neutralization Activity Against Viral Variants, According to BNT162b2-Vaccination and Infection History<ul><li> SARS-CoV-2 sequences were classified using the Pangolin COVID-19 Lineage Assigner tool v.2.3.2 (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>) and Nextclade v.0.14.1 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34923158">34923158</a> <b>(0)</b> Genome sequencing of SARS-CoV-2 reveals the prevalence of variant B 1.1.7 in Egypt<ul><li> Using open access server (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/tree" id="ir0005">https://clades.<b>nextstrain</b>.org/tree</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34946210">34946210</a> <b>(3)</b> Genomic Epidemiology of SARS-CoV-2 Divulge B.1, B.1.36, and B.1.1.7 as the Most Dominant Lineages in First, Second, and Third Wave of SARS-CoV-2 Infections in Pakistan<ul><li>To infer our SARS-CoV-2 time-scaled and divergence phylogenetic tree, selected global reference sequences were used for the <b>Nextstrain</b> analysis as of 10 July 2021.<li> Using the <b>Nextstrain</b> metadata [<xref rid="B23-microorganisms-09-02609" ref-type="bibr"><u>23</u></xref>] to identify the accessions of interest, the latest whole-genome sequence alignment from the GISAID database was downloaded.<li> Finally, the maximum likelihood phylogenetic tree was built by using the <b>Nextstrain</b> pipelines which incorporates Augur for generation of phylogenetic tree and Auspice for visualizations.<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35262032">35262032</a> <b>(0)</b> SARS-CoV-2 loads in urine, sera and stool specimens in association with clinical features of COVID-19 patients<ul><li> The FASTA file with consensus sequences was uploaded in the Nextclade Beta (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0001">https://clades.<b>nextstrain</b>.org/</ext-link>) <xref rid="bib0011" ref-type="bibr"><u>[11]</u></xref> to identify mutations profile, considering PANGO linages.<li> [[11]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34913861">34913861</a> <b>(3)</b> Context-aware genomic surveillance reveals hidden transmission of a carbapenemase-producing <italic toggle="yes">Klebsiella pneumoniae</italic><ul><li> Positive selection was assessed by first aligning all sequences for a particular gene using <named-content content-type="mono-space">nextalign</named-content> (no version, unpublished, <ext-link xlink:href="https://github.com/<b>nextstrain</b>/nextclade" ext-link-type="uri">github.com/<b>nextstrain</b>/nextclade</ext-link>).<li> A recent example of this is <named-content content-type="mono-space"><b>nextstrain</b></named-content>, where the virus genomics community converged on a set of protocols and databases [<xref rid="R92" ref-type="bibr"><u>92</u></xref>], which allowed a data-driven public health response.<li> [92](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34946199">34946199</a> <b>(3)</b> Simplified Point-of-Care Full SARS-CoV-2 Genome Sequencing Using Nanopore Technology<ul><li>Consensus FASTA files were uploaded to the <b>Nextstrain</b> webapp (Nextclade.<li> Available online: <uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri> [accessed on 14 December 2021]) to perform phylogenetic analyses [<xref rid="B14-microorganisms-09-02598" ref-type="bibr"><u>14</u></xref>].<li> Phylogenetic analyses with <b>Nextstrain</b> using the FASTA consensus files obtained from the interARTIC or the Geneious Prime pipelines resulted in considerably different phylogenetic distances in clade 20I, showing that the bioinformatics pipeline influences the result (<xref rid="app1-microorganisms-09-02598" ref-type="app"><u>Figure S3</u></xref>).<li>Single or batch high-quality consensus FASTA formatted sequences were used for phylogenetic tree visualization and variant calling using the <b>Nextstrain</b> webapp (Nextclade.<li> Available online: <uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri> [accessed on 14 December 2021]) [<xref rid="B14-microorganisms-09-02598" ref-type="bibr"><u>14</u></xref>].<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34931174">34931174</a> <b>(5)</b> Striking lineage diversity of severe acute respiratory syndrome coronavirus 2 from non-human sources<ul><li> These reference genomes were downloaded from Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0015">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> Here, Single Nucleotide Polymorphisms (SNPs), as well as Insertions and Deletions (InDels), were screened using a package within the <b>Nextstrain</b> tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="ir0020">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> <bold>A.</bold> Phylogenetic analysis based on whole-genome alignment using <b>Nextstrain</b> [<xref rid="bb0055" ref-type="bibr"><u>11</u></xref>].<li> <bold>A.</bold> Phylogenetic analysis based on whole-genome alignment using <b>Nextstrain</b> [<xref rid="bb0055" ref-type="bibr"><u>11</u></xref>].<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34920116">34920116</a> <b>(2)</b> Zoonotic spill-over of SARS-CoV-2: mink-adapted virus in humans<ul><li> Phylogenetic analysis was performed using the procedure recommended by <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0010"><b>Nextstrain</b>.org</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34911747">34911747</a> <b>(0)</b> Monitoring the SPREAD of the SARS-CoV-2 lineage B.1.621 in Udine, Italy<ul><li> As evidenced in <xref rid="F1" ref-type="fig"><u>figure 1A</u></xref>, our samples overlap with the other B.1.621 sequences in Nextclade V.2.29.1web application (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34940505">34940505</a> <b>(2)</b> The Development of SARS-CoV-2 Variants: The Gene Makes the Disease<ul><li>More than five million genome sequences have been deposited in open-source platforms such as GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>, accessed on 2 December 2021), Nexstrain (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 2 December 2021) [<xref rid="B135-jdb-09-00058" ref-type="bibr"><u>135</u></xref>] and NCBI Virus (<uri xlink:href="https://www.ncbi.nlm.nih.gov/labs/virus/">https://www.ncbi.nlm.nih.gov/labs/virus/</uri>, accessed on 2 December 2021).<li> [135](          <person-group person-group-type="author"><collab>Nextstrain</collab></person-group>          <article-title>Genomic Epidemiology of Novel Coronavirus&#x2014;Global subsampling</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/ncov/global" ext-link-type="uri">https://nextstrain.org/ncov/global</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-09-15">(accessed on 15 September 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34910368">34910368</a> <b>(3)</b> Transmission of SARS&#x2010;CoV&#x2010;2 from humans to a 16&#x2010;year&#x2010;old domestic cat with comorbidities in Pennsylvania, USA<ul><li>The data sets of 1343 SARS&#x2010;CoV&#x2010;2 genomes deposited in GISAID from North America (sampled between December 2019 and December 2020) were used for phylogenetic analysis using the standard protocol from <b>Nextstrain</b>/Nextclade (<ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" ext-link-type="uri">https://<b>nextstrain</b>.org/sars&#x2010;cov&#x2010;2/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35039782">35039782</a> <b>(0)</b> Identification of a novel SARS-CoV-2 P.1 sub-lineage in Brazil provides new insights about the mechanisms of emergence of variants of concern<ul><li> 2017</xref>) and their mutational profile was investigated using the Nextclade tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34931186">34931186</a> <b>(12)</b> TopHap: Rapid inference of key phylogenetic structures from common haplotypes in large genome collections with limited diversity<ul><li>Annotations using <b>Nextstrain</b> and PANGO classifications.<li>To compare <italic toggle="yes">TopHap</italic> phylogeny with the <b>Nextstrain</b> classification, we annotated all the <italic toggle="yes">TopHap</italic> haplotypes using the presence and absence of diagnostic <b>Nextstrain</b> mutations (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov" ext-link-type="uri">https://<b>Nextstrain</b>.org/ncov</ext-link>).<li> The placement of these VOCs in <italic toggle="yes">TopHap</italic> is like that in the <b>Nextstrain</b> (<xref rid="F6" ref-type="fig">Fig.<li> For example, both <b>Nextstrain</b> and <italic toggle="yes">TopHap</italic> reconstruct WHO-ALPHA, WHO-GAMMA, and WHO-LAMBDA to be sister lineages.<li>We find that <b>Nextstrain</b> and PANGO phylogeny broadly agree with 68KG and 1MG <italic toggle="yes">TopHap</italic> phylogenies, except for the root position (<xref rid="F3" ref-type="fig">Fig.<li> For example, clade 19A is at the root of the <b>Nextstrain</b> phylogeny, but <italic toggle="yes">TopHap</italic> phylogenies using the bat/pangolin outgroups suggest that Clade 19A is a derived clade.<li> But, no bootstrap replicates supported the <b>Nextstrain</b> rooting, and &lt;34% supported the PANGO rooting.<li> (2021)</xref>, <xref rid="R3" ref-type="bibr"><u>Bloom (2021)</u></xref>, <b>Nextstrain</b>, and PANGO (<xref rid="F7" ref-type="fig">Fig.<li> MP suggests scenarios A, B, and C to be equally parsimonious, and D and E (PANGO and <b>Nextstrain</b>, respectively) to be less parsimonious by 1 and 3 mutations.<li> [Bloom (2021)](<name><surname>Bloom</surname><given-names>J.D.</given-names></name> (<year>2021</year>) <article-title>Recovery of deleted deep sequencing data sheds more light on the early wuhan sars-cov-2 epidemic</article-title>. <source>Mol. Biol. Evol.</source>, <pub-id pub-id-type="doi">10.1093/molbev/msab246</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34905108">34905108</a> <b>(2)</b> Emerging SARS-CoV-2 Variants: Genetic Variability and Clinical Implications<ul><li> The Global Initiative on Sharing All Influenza Data (GISAID), <b>NextStrain</b>, and Phylogenetic Assignment of Named Global Outbreak (Pango) are the most common naming conventions that are being used by the scientific community worldwide to address the SARS-CoV-2 variants [<xref ref-type="bibr" rid="CR17"><u>17</u></xref>].<li> To overcome this problem, <b>NextStrain</b> devised a system using phylogenetic and phylodynamic analysis to identify evolutionary stable clades and sub-clades [<xref ref-type="bibr" rid="CR18"><u>18</u></xref>].<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33735881">33735881</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33915216">33915216</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34970567">34970567</a> <b>(9)</b> CalmBelt: Rapid SARS-CoV-2 Genome Characterization for Outbreak Tracking<ul><li> These laboratories can share and analyse the SARS-CoV-2 whole-genome sequences at the regional and population level via GISAID (<xref rid="B9" ref-type="bibr"><u>9</u></xref>) and genomic analytic platforms such as <b>Nextstrain</b> (<xref rid="B8" ref-type="bibr"><u>8</u></xref>), allowing large-scale pathogen evolution tracking.<li> Compared to existing analytic frameworks such as GISAID and <b>Nextstrain</b> (<xref rid="B8" ref-type="bibr"><u>8</u></xref>, <xref rid="B9" ref-type="bibr"><u>9</u></xref>), CalmBelt addresses the issue of resolution required for local outbreak investigations by using data driven analysis based on whole-genome sequence and sample collection information as an alternative strategy to the pre-defined phylogenetic tree and machine learning model of the PANGO lineage.<li> Similar to <b>Nextstrain</b>, we applied TreeTime (<xref rid="B26" ref-type="bibr"><u>26</u></xref>) to generate time stamped phylogenies (<b>Nextstrain</b>'s transmission tree) of the input genomes.<li>As highlighted in the <b>Nextstrain</b> web portal (<xref rid="B28" ref-type="bibr"><u>28</u></xref>), &#x201C;copying mistakes&#x201D; that accumulate in the genome could facilitate tracking of transmission routes and dynamics, while a phylogenetic tree could infer locations of each case.<li>Similar to <b>Nextstrain</b> (<xref rid="B8" ref-type="bibr"><u>8</u></xref>), nucleotide diversity was calculated for each genome position.<li> We note that CalmBelt shows information from all genomes without subsampling, a pre-processing step used in <b>Nextstrain</b>.<li>Several web applications for SARS-CoV-2 sequence analyses, such as <b>Nextstrain</b> (<xref rid="B9" ref-type="bibr"><u>9</u></xref>), Audacity, PrimerChecker (<xref rid="B9" ref-type="bibr"><u>9</u></xref>) and CoVariants (<xref rid="B41" ref-type="bibr"><u>41</u></xref>), have been recently developed.<li> However, none of these tools (except <b>Nextstrain</b>) focus on integrating custom user input and sample collection dates to create a specialized report for outbreak investigations and control mitigations in the healthcare settings.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [28](<person-group person-group-type="author"><collab>How to Interpret the Phylogenetic Trees &#x2014; Nextstrain Documentation</collab></person-group>. Available online at: <ext-link xlink:href="https://docs.nextstrain.org/en/latest/learn/interpret/how-to-read-a-tree.html" ext-link-type="uri">https://docs.nextstrain.org/en/latest/learn/interpret/how-to-read-a-tree.html</ext-link> (accessed September 12, 2021).) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [41](<person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>E</given-names></name></person-group>. <article-title>CoVariants</article-title>. <source>CoVariants</source> Available online at: <ext-link xlink:href="https://covariants.org/" ext-link-type="uri">https://covariants.org/</ext-link> (accessed July 25, 2021). ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34946187">34946187</a> <b>(2)</b> Comprehensive Contact Tracing, Testing and Sequencing Show Limited Transmission of SARS-CoV-2 between Children in Schools in Norway, August 2020 to May 2021<ul><li> Sequences were aligned in a codon-aware manner to the reference genome Wuhan-Hu-1 (GenBank accession no: MN908947.3) using the program Nextalign v.0.2.0 (available at <uri xlink:href="https://github.com/<b>nextstrain</b>/nextclade">https://github.com/<b>nextstrain</b>/nextclade</uri>, accessed on 1 December 2021), and a maximum-likelihood phylogenetic tree was created using IQTREE v.2.0.3 (available at <uri xlink:href="https://iqtree.org">https://iqtree.org</uri>, accessed on 1 December 2021) [<xref rid="B18-microorganisms-09-02587" ref-type="bibr"><u>18</u></xref>].<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34905574">34905574</a> <b>(5)</b> Local occurrence and fast spread of B.1.1.7 lineage: A glimpse into Friuli Venezia Giulia<ul><li> GISAID-deposited genomes have been analyzed in different open-source project among which <b>NextStrain</b> has gained a well-deserved popularity (<ext-link xlink:href="http://www.<b>nextstrain</b>.org/" ext-link-type="uri">www.<b>nextstrain</b>.org</ext-link>).<li> Furthermore, it named evolutionary stable lineages based on the year of emergence and a letter (e.g., <b>Nextstrain</b> Clade: 19A, 19B, 20A, 20B and so on) (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global?l=radial" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global?l=radial</ext-link>; version v2.29.1), despite some sub lineages have been identified with additional information (e.g., 20E/EU1) but with no systematic rule.<li> Clade and lineages of 479 specimens were defined with the Nextclade v2.29.1 web application (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>), the GISAID CoVsurver mutation App (<ext-link xlink:href="https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/" ext-link-type="uri">https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/</ext-link>), the pangolin web app developed by the Centre for Genomic Pathogen Surveillance&#x2013;version v.3.1.11 - (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) and the Ultrafast Sample placement on Existing tRee (UShER) web tool by UCSC (<ext-link xlink:href="https://hgwdev-angie.gi.ucsc.edu/cgi-bin/hgPhyloPlace" ext-link-type="uri">https://hgwdev-angie.gi.ucsc.edu/cgi-bin/hgPhyloPlace</ext-link>).<li>Approximate correlation between the major SARS-CoV-2 PANGO lineage, GISAID <b>Nextstrain</b> clade and WHO nomenclature.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34905574">34905574</a> <b>(1)</b> Local occurrence and fast spread of B.1.1.7 lineage: A glimpse into Friuli Venezia Giulia<ul><li> the WHO classification should be indicated in a separate column instead of in brackets in the <b>Nextstrain</b> clade column.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34905574">34905574</a> <b>(7)</b> Local occurrence and fast spread of B.1.1.7 lineage: A glimpse into Friuli Venezia Giulia<ul><li>People speak about 19 A 20 A clades (line 40-41) without referring to <b>Nextstrain</b> nomenclature and its references (add website link and version).<li> to GISAID or <b>Nextstrain</b> assignement supports.<li> Notice date of assignement by using <b>Nextstrain</b> and Pangolin (line 101 103) due to update in Continuous.<li> Its of <b>Nextstrain</b> assignement miss and added concerning 1508 seq (i.e.: is it clade 20Ia and for all?).<li>Table 1: this relationship between GISAIS/<b>Nextstrain</b> and Pangolin nomenclature systems is well described in Alma et al, (<ext-link xlink:href="https://doi.org/10.2807/1560-7917.ES.2020.25.32.2001410" ext-link-type="uri">https://doi.org/10.2807/1560-7917.ES.2020.25.32.2001410</ext-link>).<li> Or table1 should be redesigned (Column 1 GISAID, Column 2 <b>Nextstrain</b>, Colum 3 Pangolin) and put on suppl.<li> the WHO classification should be indicated in a separate column instead of in brackets in the <b>Nextstrain</b> clade column.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34931192">34931192</a> <b>(0)</b> Delta breakthrough infections elicit potent, broad and durable neutralizing antibody responses<ul><li> Sequence quality was determined using Nextclade version 1.0.0-alpha.8 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34966754">34966754</a> <b>(1)</b> IDbSV: An Open-Access Repository for Monitoring SARS-CoV-2 Variations and Evolution<ul><li> In the same context, after over a year of COVID-19, several tools have been developed including <b>Nextstrain</b> (<xref rid="B9" ref-type="bibr"><u>9</u></xref>), BioAider (<xref rid="B10" ref-type="bibr"><u>10</u></xref>), Coronapp (<xref rid="B11" ref-type="bibr"><u>11</u></xref>), CoV-Seq (<xref rid="B12" ref-type="bibr"><u>12</u></xref>), ViruSurf (<xref rid="B13" ref-type="bibr"><u>13</u></xref>), NGDC (<xref rid="B14" ref-type="bibr"><u>14</u></xref>), CoV-GLUE (<xref rid="B15" ref-type="bibr"><u>15</u></xref>), Favicov (<xref rid="B16" ref-type="bibr"><u>16</u></xref>), and IDP 2.0 (<xref rid="B17" ref-type="bibr"><u>17</u></xref>) to provide analysis of SARS-CoV-2 sequences.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32904401">32904401</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33205830">33205830</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32931441">32931441</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33045721">33045721</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32102777">32102777</a>) [15](<person-group person-group-type="author"><name><surname>Singer</surname><given-names>J</given-names></name><name><surname>Gifford</surname><given-names>R</given-names></name><name><surname>Cotten</surname><given-names>M</given-names></name><name><surname>Robertson</surname><given-names>D</given-names></name></person-group>. <source>CoV-GLUE: A Web Application for Tracking SARS-CoV-2 Genomic Variation</source>. (<year>2020</year>). <pub-id pub-id-type="doi">10.20944/preprints202006.0225.v1</pub-id>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34643704">34643704</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33985433">33985433</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35039785">35039785</a> <b>(2)</b> A comprehensive evolutionary and epidemiological characterization of insertion and deletion mutations in SARS-CoV-2 genomes<ul><li> The phylogenetic tree is constructed using the <b>nextstrain</b> augur pipeline and based on a set of reference sequences downloaded from GISAID.<li>We used <b>nextstrain</b> augur tool (<xref rid="R11" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34897035">34897035</a> <b>(2)</b> Multicenter study evaluating one multiplex RT-PCR assay to detect SARS-CoV-2, influenza A/B, and respiratory syncytia virus using the LabTurbo AIO open platform: epidemiological features, automated sample-to-result, and high-throughput testing<ul><li>To identify pathogen evolution relationships, assembly sequences were uploaded to the Nextclade website (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) developed by <b>Nextstrain</b> [<xref rid="r28" ref-type="bibr"><u>28</u></xref>].<li> TSGH sequences were aligned to the reference sequences of major clades of SARS-CoV (20I) grouped using <b>Nextstrain</b> and phylogenetic tree annotated with these alignment definitions, using lists of grouped clade-defining mutations via Augur workflow [<xref rid="r29" ref-type="bibr"><u>29</u></xref>] supported by Nextclade.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34893946">34893946</a> <b>(0)</b> Case Series: Convalescent Plasma Therapy for Patients with COVID-19 and Primary Antibody Deficiency<ul><li> The clades were classified with the webservers of Nextclade (clades.<b>nextstrain</b>.org) and Pangolin (pangolin.cog-uk.io).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34960743">34960743</a> <b>(0)</b> Nucleocapsid (N) Gene Mutations of SARS-CoV-2 Can Affect Real-Time RT-PCR Diagnostic and Impact False-Negative Results<ul><li> Viral strains were classified and mutations analyzed using the software tools Pangolin (<uri xlink:href="http://pangolin.cog-uk.io/">http://pangolin.cog-uk.io/</uri>, accessed on 5 August 2021), and nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 5 August 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34956160">34956160</a> <b>(1)</b> A Case Report: Genetically Distinct Severe Acute Respiratory Syndrome Coronavirus-2 Variant Causing Reinfection<ul><li> Phylogenetic analysis showed that the lineage B.1.1.25 was detected in the first episode, whereas the lineage B.1.1.7 was detected in the second episode according to the Pangolin lineage classification.<sup><xref rid="footnote2" ref-type="fn"><u>2</u></xref></sup> According to the <b>Nextstrain</b> classification, the strains from the first and second episodes were placed in clades 20B and 20I, respectively (<xref rid="F2" ref-type="fig"><u>Figure 2</u></xref>).<li> [2](<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2021.792514/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fmicb.2021.792514/full#supplementary-material</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34943537">34943537</a> <b>(0)</b> Clinical Evaluation of a New Antigen-Based COVID-19 Rapid Diagnostic Test from Symptomatic Patients<ul><li> The SARS-CoV-2 clade was detected using the Nextclade v1.5.2 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 3 October 2021)) with default parameters [<xref rid="B25-diagnostics-11-02300" ref-type="bibr"><u>25</u></xref>].<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34880295">34880295</a> <b>(0)</b> Emergence of novel combinations of SARS-CoV-2 spike receptor binding domain variants in Senegal<ul><li> Genomes were assembled using BLAST and sequence NC_045512 as a reference, followed by clade assignment and mutation analysis with the NextClade tool (clades.<b>nextstrain</b>.org) and lineage assignments with the Pangolin tool<sup><xref ref-type="bibr" rid="CR12"><u>12</u></xref></sup>.<li> [12](O&#x2019;Toole, &#xC1;., Scher, E., Underwood, A., Jackson, B., Hill, V., McCrone, J. T., Ruis, C., Abu-Dahab, K., Taylor, B., Yeats, C., du Plessis, L., Aanensen, D., Holmes, E., Pybus, O. &amp; Rambaut, A. Pangolin: Lineage assignment in an emerging pandemic as an epidemiological tool. githubcom/cov-lineages/pangolin (2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36168505">36168505</a> <b>(1)</b> Fundamentals of genomic epidemiology, lessons learned from the coronavirus disease 2019 (COVID-19) pandemic, and new directions<ul><li> These labels do not replace existing classifications by GISAID (<uri xlink:href="https://gisaid.org/">https://gisaid.org/</uri>),<sup><xref rid="r95" ref-type="bibr"><u>95</u></xref></sup> <b>Nextstrain</b> (<uri xlink:href="https://nexstrain.org/">https://nexstrain.org/</uri>),<sup><xref rid="r96" ref-type="bibr"><u>96</u></xref></sup> and Pango lineages (<uri xlink:href="https://cov-lineages.org">https://cov-lineages.org/</uri>).<li> [95](r95) [96](r96) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34943523">34943523</a> <b>(2)</b> Low Represented Mutation Clustering in SARS-CoV-2 B.1.1.7 Sublineage Group with Synonymous Mutations in the E Gene<ul><li> The most employed tools used to report all useful information about the different viral clades and suitable to analyze and classify the variant of the sequenced SARS-CoV-2 strains are Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN, cov-lineages) [<xref rid="B11-diagnostics-11-02286" ref-type="bibr"><u>11</u></xref>] and <b>Nextstrain</b> [<xref rid="B12-diagnostics-11-02286" ref-type="bibr"><u>12</u></xref>].<li> Pangolin allows a user to assign a SARS-CoV-2 genome sequence the most likely lineage to SARS-CoV-2 query sequences, whereas <b>Nextstrain</b> CoVariants provides an overview of SARS-CoV-2 variants and mutations that are of interest, specifying what mutations define a variant, what impact they might have, and where variants are found.<li> [11](          <person-group person-group-type="author"><name><surname>O&#x2019;Toole</surname><given-names>A.</given-names></name><name><surname>Scher</surname><given-names>E.</given-names></name><name><surname>Underwood</surname><given-names>A.</given-names></name><name><surname>Jackson</surname><given-names>B.</given-names></name><name><surname>Hill</surname><given-names>V.</given-names></name><name><surname>McCrone</surname><given-names>J.T.</given-names></name><name><surname>Ruis</surname><given-names>C.</given-names></name><name><surname>Abu-Dahab</surname><given-names>K.</given-names></name><name><surname>Taylor</surname><given-names>B.</given-names></name><name><surname>Yeats</surname><given-names>C.</given-names></name><etal/></person-group>          <article-title>Pangolin: Lineage Assignment in an Emerging Pandemic as an Epidemiological Tool</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-11-03">(accessed on 3 November 2021)</date-in-citation>        ) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34952247">34952247</a> <b>(0)</b> Genomic analysis of SARS-CoV-2 variants of concern identified from the ChAdOx1 nCoV-19 immunized patients from Southwest part of Bangladesh<ul><li> Mutations and amino acid substitutions were retrieved from Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intr0005">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34909798">34909798</a> <b>(5)</b> SARS-CoV-2 evolved during advanced HIV disease immunosuppression has Beta-like escape of vaccine and Delta infection elicited immunity<ul><li> We analyzed sequences from the six different time points (D6, D20, D34, D71, D106 and D190) against a global reference dataset of 3883 genomes using a custom build of the SARS-CoV-2 <b>NextStrain</b> (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> All sequences from the participant clustered in a monophyletic clade (<ext-link xlink:href="https://<b>nextstrain</b>.org/groups/ngs-sa/COVID19-AHRI-2021.05.27?label=clade:HIV%20patient" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/ngs-sa/COVID19-AHRI-2021.05.27?label=clade:HIV%20patient</ext-link>) that are well separated from the rest of the phylogeny.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34938276">34938276</a> <b>(1)</b> Whole Genome Sequencing of Drug Resistant and Drug Susceptible <italic toggle="yes">Mycobacterium tuberculosis</italic> Isolates From Tigray Region, Ethiopia<ul><li>Additionally, variant call format files from Snippy were used to generate a visualization in <b>Nextstrain</b> (<xref rid="B18" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34908852">34908852</a> <b>(0)</b> First Report of SARS-CoV-2 Lineage B.1.1.7 (Alpha Variant) in Ecuador, January 2021<ul><li> The genome was processed using the Artic bioinformatics pipeline, assembled in the online tool Genome Detective using default parameters<xref rid="cit0009" ref-type="bibr"><u>9</u></xref> and submitted to Nextclade (<underline><ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link></underline>) for preliminary taxonomic analysis.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33298935">33298935</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34946123">34946123</a> <b>(1)</b> Microorganisms as Shapers of Human Civilization, from Pandemics to Even Our Genomes: Villains or Friends? A Historical Approach<ul><li> GISAID and <b>Nextstrain</b> [<xref rid="B124-microorganisms-09-02518" ref-type="bibr"><u>124</u></xref>] are high quality tools that can be used for sequence display and analysis have been critical to studying origins, epidemiological studies, and tracking viral variant transmission routes [<xref rid="B125-microorganisms-09-02518" ref-type="bibr"><u>125</u></xref>].<li> [124](          <article-title>Nextstrain</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-07-04">(accessed on 4 July 2021)</date-in-citation>        ) [125](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32793182">32793182</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34959672">34959672</a> <b>(1)</b> Monoclonal Antibodies against SARS-CoV-2: Current Scenario and Future Perspectives<ul><li>The established nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages by GISAID, <b>Nextstrain</b>, and Pango are currently and will remain in use by scientists and in scientific research.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34871297">34871297</a> <b>(1)</b> Mutational landscape of SARS-CoV-2 genome in Turkey and impact of mutations on spike protein structure<ul><li> There are different classification strategies, developed by different organizations like PANGO lineage [<xref rid="pone.0260438.ref023" ref-type="bibr"><u>23</u></xref>], GISAID [<xref rid="pone.0260438.ref049" ref-type="bibr"><u>49</u></xref>], <b>Nextstrain</b> [<xref rid="pone.0260438.ref050" ref-type="bibr"><u>50</u></xref>], Public Health England [<xref rid="pone.0260438.ref051" ref-type="bibr"><u>51</u></xref>], and World Health Organization [<xref rid="pone.0260438.ref020" ref-type="bibr"><u>20</u></xref>].<li> [23](PANGO lineages. Lineage Description List [Internet]. <ext-link xlink:href="https://cov-lineages.org/lineage_description_list.html" ext-link-type="uri">https://cov-lineages.org/lineage_description_list.html</ext-link>.) [49](GISAID. Clade and lineage nomenclature, March 2, 2021 [Internet]. 2021. <ext-link xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>.) [50](Hodcroft EB, Hadfield J, Neher RA, Bedford T. 2020. <ext-link xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming" ext-link-type="uri">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>.) [51](Public Health England&#x2014;GOV.UK. Coronavirus (COVID-19) Rules, guidence and support. <ext-link xlink:href="https://www.gov.uk/government/organisations/public-health-england" ext-link-type="uri">https://www.gov.uk/government/organisations/public-health-england</ext-link>.) [20](World Health Organization. Tracking SARS-CoV-2 variants. 2021. <ext-link xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" ext-link-type="uri">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34896696">34896696</a> <b>(2)</b> Emergence of a novel SARS-CoV-2 Pango lineage B.1.1.526 in West Bengal, India<ul><li> 2</xref>A, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/fetch/genome.ucsc.edu/trash/ct/singleSubtreeAuspice_genome_3511c_b706f0.json?l=radial" id="intr0020">https://<b>nextstrain</b>.org/fetch/genome.ucsc.edu/trash/ct/singleSubtreeAuspice_genome_3511c_b706f0.json?l=radial</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34909794">34909794</a> <b>(1)</b> SARS-CoV-2 incidence, transmission and reinfection in a rural and an urban setting: results of the PHIRST-C cohort study, South Africa, 2020&#x2013;2021<ul><li> and confirmed reinfection as distinct <b>Nextstrain</b> clades on sequencing or variant PCR between rRT-PCR-positive specimens meeting the temporal criteria for possible or probable.</ul></p>
</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34960694">34960694</a> <b>(1)</b> Semi-Supervised Pipeline for Autonomous Annotation of SARS-CoV-2 Genomes<ul><li>As an alternative to an autonomous genome annotation method, alignments to the Wuhan reference genome [<xref rid="B1-viruses-13-02426" ref-type="bibr"><u>1</u></xref>] can be completed using tools such as <b>NextStrain</b>&#x2019;s Augur [<xref rid="B14-viruses-13-02426" ref-type="bibr"><u>14</u></xref>], Bowtie2 [<xref rid="B15-viruses-13-02426" ref-type="bibr"><u>15</u></xref>], or UCSC SARS-CoV-2 genome browser [<xref rid="B3-viruses-13-02426" ref-type="bibr"><u>3</u></xref>].<li> [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32015508">32015508</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22388286">22388286</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908258">32908258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34854726">34854726</a> <b>(0)</b> Genome Sequencing of the SARS-CoV-2 Delta (B.1.617.2) Variant of Concern Detected in Bangladesh<ul><li> The Pangolin lineage was determined and the phylogenetic analysis performed using the default parameters of Nextclade version 1.7.1 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/tree" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/tree</ext-link>) (<xref rid="B7" ref-type="bibr"><u>7</u></xref>).<li> The FASTA file was uploaded in Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/results" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/results</ext-link>), and a phylogenetic tree was created using <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MZ437368" ext-link-type="uri" assigning-authority="insdc">MZ437368</ext-link> to represent the Delta lineage (B.1.617.2/20A/S:478K) in Bangladesh.<li> [7](<collab>Nextstrain Developers</collab>. <year>2020</year>. <article-title>Nextclade: clade assignment, mutation calling, and sequence quality checks</article-title>. <ext-link xlink:href="https://clades.nextstrain.org/tree" ext-link-type="uri">https://clades.nextstrain.org/tree</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34854728">34854728</a> <b>(1)</b> Coding-Complete Genome Sequences of Alpha and Delta SARS-CoV-2 Variants from Kamphaeng Phet Province, Thailand, from May to July 2021<ul><li><b>Nextstrain</b> clade</th>                <th rowspan="1" colspan="1">Pangolin lineage (v2021-09-28)</th>                <th rowspan="1" colspan="1">No.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34855869">34855869</a> <b>(7)</b> Patient health records and whole viral genomes from an early SARS-CoV-2 outbreak in a Quebec hospital reveal features associated with favorable outcomes<ul><li> The most popular lineage assignment tools are Pangolin [<xref rid="pone.0260714.ref012" ref-type="bibr"><u>12</u></xref>], which uses a supervised learning approach to classify sequences, and <b>Nextstrain</b>/Nextclade [<xref rid="pone.0260714.ref013" ref-type="bibr"><u>13</u></xref>], which use condensed phylogeny and phylogenetic placement methods, respectively.<li>Lineage classification was performed using the Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin) software package (version 2.1.10) proposed by [<xref rid="pone.0260714.ref012" ref-type="bibr"><u>12</u></xref>], using default parameters, as well as using the <b>Nextstrain</b>: real-time tracking of pathogen evolution (SARS-CoV-2 pipeline:2021/09/15).<li> <b>Nextstrain</b> was built with a multiple input build-config (no filtering) using our sequences and metadata along with GISAID sequences and metadata (on 2021/09/16) [<xref rid="pone.0260714.ref013" ref-type="bibr"><u>13</u></xref>].<li> The definition of haplotypes based on these 20 mutations found in samples from this study, and their corresponding <b>NextStrain</b> clades, is reported in <xref rid="pone.0260714.s012" ref-type="supplementary-material"><u>S5 Table</u></xref>.<li> We note that haplotype III and IX are grouped within the same Nexstrain clade, despite the fact that they differ at position 25563, and both differ from haplotype II (<b>NextStrain</b> clade 20C) at position 14408.<li><b>Nextstrain</b> genomic epidemiology of SARS-CoV-2.<li>Time-resolved phylogenetic tree of all genomes reported in our cohort (red dots) in the context of subsequent viral evolution (subsampled genomes from Nexstrain) during the COVID-19 pandemic (as of September 2021) produced via the <b>Nextstrain</b> SARS-CoV-2 pipeline.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Patient health records and whole viral genomes from an early SARS-CoV-2 outbreak in a Quebec hospital reveal features associated with favorable outcomes<ul><li>- Did the author run pangoling in house or used the pangoling classification already in <b>nextstrain</b>?.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(5)</b> Patient health records and whole viral genomes from an early SARS-CoV-2 outbreak in a Quebec hospital reveal features associated with favorable outcomes<ul><li> We have therefore included a supplementary figure that illustrates the time-resolved phylogenetic relationship between the genomes reported in this study and those subsequently sequenced from across the world (from <b>NextStrain</b>), as well as a sentence describing this at the end of the first paragraph of the &#x201C;Unsupervised machine learning outperforms supervised methods at discriminating between viral subclades&#x201D; section.<li> However, both mutations are rarely observed in viral genomes sampled after our study (based on their representation in <b>NextStrain</b>).<li> Did the author run pangoling in house or used the pangoling classification already in <b>nextstrain</b>?.<li> Moreover, following the reviewers interest in describing amino acid variants, we have uploaded the metadata and .json configuration files associated with the <b>NextStrain</b>/Auspice visualization we report in the Supplementary Figure 2 to the study&#x2019;s github repository so that users can explore these data interactively using the <b>NextStrain</b> software packages (alternatively, the fasta sequences can be directly uploaded to the NextClade webportal).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Patient health records and whole viral genomes from an early SARS-CoV-2 outbreak in a Quebec hospital reveal features associated with favorable outcomes<ul><li>- The authors need to state that their method worked well compared to Pangolin/<b>Nextstrain</b> only for this specific data set, and this cannot be generalized to other data-sets due to the lack of more extensive experimentation using larger sequencing set and more lineages.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Patient health records and whole viral genomes from an early SARS-CoV-2 outbreak in a Quebec hospital reveal features associated with favorable outcomes<ul><li>- The authors need to state that their method worked well compared to Pangolin/<b>Nextstrain</b> only for this specific data set, and this cannot be generalized to other data-sets due to the lack of more extensive experimentation using larger sequencing set and more lineages.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34808082">34808082</a> <b>(3)</b> SARS-CoV-2 B.1.1.7 Variant Infection in Malayan Tigers, Virginia, USA<ul><li> We used <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>) for phylogenetic analysis of tiger-derived sequences and other B.1.1.7 sequences downloaded from GISAID (<ext-link xlink:href="https://www.gisaid.org" ext-link-type="uri">https://www.gisaid.org</ext-link>) on April 15, 2021 (<xref rid="R2" ref-type="bibr"><italic>2</italic></xref>,<xref rid="R3" ref-type="bibr"><italic>3</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34821382">34821382</a> <b>(1)</b> Establishing reference sequences for each clade of SARS&#x2010;CoV&#x2010;2 to provide a basis for virus variation and function research<ul><li> Current commonly used typing methods include the Chinese typing method, Pangolin typing method, GISAID typing method, and <b>Nextstrain</b> typing method.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34846601">34846601</a> <b>(4)</b> Evolution of the SARS-CoV-2 genome and emergence of variants of concern<ul><li> Moreover, other bioinformatics platforms including <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) were developed that continuously monitor the submitted sequences and enable real-time tracking and visualization of sequence variations.<li> This haplotype is named clade G in GISAID and 20A in <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34850820">34850820</a> <b>(2)</b> Managing the life cycle of a portfolio of open data resources at the SIB Swiss Institute of Bioinformatics<ul><li><ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri"><b>Nextstrain</b></ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34917076">34917076</a> <b>(1)</b> The Global Epidemic of the SARS-CoV-2 Delta Variant, Key Spike Mutations and Immune Escape<ul><li> The established nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages by <b>Nextstrain</b>, GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>), and Pango are currently in use by scientists.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35017872">35017872</a> <b>(1)</b> Phylodynamic Pattern of Genetic Clusters, Paradigm Shift on Spatio-Temporal Distribution of Clades, and Impact of Spike Glycoprotein Mutations of SARS-CoV-2 Isolates from India<ul><li> The Nextclade-<b>Nextstrain</b> pipeline (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) was used for studies on phylogenetic analysis and clustering patterns of the S gene.[<xref rid="ref3" ref-type="bibr"><u>3</u></xref>] Further, the Average evolutionary divergence was estimated using Kimura-2 parameter model.<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34811754">34811754</a> <b>(0)</b> Dynamics of spreading of SARS&#x2010;CoV&#x2010;2 in a Belgian hemodialysis facility: The importance of the analysis of viral strains<ul><li>Nextclade Beta version 0.8.1 was used as a first sequence aligner, allowing comparison to other documented SARS&#x2010;CoV&#x2010;2 strains and clade assignment (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34846283">34846283</a> <b>(14)</b> Phylogenomics and population genomics of SARS-CoV-2 in Mexico during the pre-vaccination stage reveals variants of interest B.1.1.28.4 and B.1.1.222 or B.1.1.519 and the nucleocapsid mutation S194L associated with symptoms<ul><li>The Mexstrain-<b>Nextstrain</b> data used for this study can be found at: <ext-link xlink:href="http://www.ira.cinvestav.mx/ncov.evol.mex.aspx" ext-link-type="uri">http://www.ira.cinvestav.mx/ncov.evol.mex.aspx.<li>Emerging variants can be detected by their phylogenetic position provided by the <b>Nextstrain</b> [<xref rid="R12" ref-type="bibr"><u>12</u></xref>] and Pangolin platforms, the latter being the primary source of classification and nomenclature of SARS-CoV-2 [<xref rid="R13" ref-type="bibr"><u>13</u></xref>].<li> We also downloaded from GISAID all genome sequences represented in <b>Nextstrain</b> SARS-CoV-2 global analysis (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global</ext-link>, date: March 2021).<li> We joined the above set of sequences (and their associated metadata) into a single set for phylogenomic analyses by using in-house Perl scripts (<ext-link xlink:href="https://github.com/luisdelaye/Mexstrain" ext-link-type="uri">https://github.com/luisdelaye/Mexstrain</ext-link>), giving place to our ad hoc Mexstrain platform (<ext-link xlink:href="http://www.ira.cinvestav.mx/ncov.evol.mex.aspx" ext-link-type="uri">http://www.ira.cinvestav.mx/ncov.evol.mex.aspx</ext-link>) for phylogenomic analyses using a local version of <b>Nextstrain</b> [<xref rid="R12" ref-type="bibr"><u>12</u></xref>].<li> For the rest of the parameters, we used those provided by the ncov <b>Nextstrain</b> build.<li> All 1554 Mexican SARS-CoV-2 sequences were compared against MN908947 as a reference used by <b>Nextstrain</b> (the sequence MN908947 used by <b>Nextstrain</b> version 3.0.3 is identical to NC_045512.2).<li> To identify homologous positions between Mexican and reference sequences, we used the multiple sequence alignment generated by our local <b>Nextstrain</b> installation.<li>To further identify potential mutations of concern and characterize their phylogenetic associations, as well as their co-occurrence and frequency at the clade level, we constructed an ad hoc database for Mexican genome sequences, called Mexstrain, which includes selected references to allow for the correct establishment of phylogenetic relationships in the <b>Nextstrain</b> platform.<li> (Mexico/SLP-InDRE_192/2020 and Mexico/SLP-InDRE_454/2021), which belong to <b>Nextstrain</b> clade 20A (different sub-lineages within the Pangolin lineage B.1).<li>Motivated by these results, we analysed the prevalence of mutation N:S194L in the extended <b>Nextstrain</b> phylogeny, and found three major sub-clades within the 20A clade with high prevalence: (i) two closely related sub-clades dominated by the B.1 lineage, where most of the Mexican genomes are placed, including more than one-fifth of all sequences available for the period of analysis (342 out of 1554); and (ii) a mainly Asian sub-clade that includes the emerging lineage B.1.36.# (where # refers to many sub-lineages, e.g.<li> (<bold>a</bold>) Worldwide occurrence of the N:S194L mutation using <b>NextStrain</b>.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34873595">34873595</a> <b>(0)</b> Tracking SARS-COV-2 Variants Using Nanopore Sequencing in Ukraine in Summer 2021<ul><li> Mutation calling and sequence quality were analysed using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and Geneious Prime 2021.2.2 software (Biomatters, New Zealand).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34912372">34912372</a> <b>(4)</b> Hotspot Mutations in SARS-CoV-2<ul><li>Motivated by the aforementioned analysis, in this work, we have performed multiple sequence alignment (MSA) of 71,038 SARS-CoV-2 genomes using MAFFT (<xref rid="B15" ref-type="bibr"><u>Katoh et al., 2002</u></xref>) followed by their phylogenetic analysis using <b>Nextstrain</b> (<xref rid="B12" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) to visualize the virus evolution.<li> Initially, MSA of 71,038 global SARS-CoV-2 genomes is performed using MAFFT, which is followed by their phylogenetic analysis using <b>Nextstrain</b>.<li> On the other hand, <b>Nextstrain</b> is a collection of open-source tools, which is useful for understanding the evolution and spread of pathogen, particularly during an outbreak.<li> Initially, MSA of 71,038 global SARS-CoV-2 genomes across 98 countries is carried out using MAFFT followed by their phylogenetic analysis using <b>Nextstrain</b>, which revealed five clades: 19A, 19B, 20A, 20B, and 20C.<li> [Katoh et al., 2002](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34839413">34839413</a> <b>(8)</b> Occurrence of a substitution or deletion of SARS-CoV-2 spike amino acid 677 in various lineages in Marseille, France<ul><li> Interestingly, these strains were reported as belonging to six independent sublineages, including one, two, and three in <b>Nextstrain</b> clades 20A, 20B, and 20G, respectively.<li> The tree was built using the <b>Nextstrain</b> tool (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/index.html">https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/index.html</ext-link>) [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>] that performs maximum-likelihood phylogeny using IQ-TREE [<xref ref-type="bibr" rid="CR21"><u>21</u></xref>] then visualised with Auspice (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).<li> The nature and number of nucleotide changes in the SARS-CoV-2 genomes and of amino acid changes in the SARS-CoV-2 proteins were obtained using the Nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/results">https://clades.<b>nextstrain</b>.org/results</ext-link>).<li> Classification into <b>Nextstrain</b> clades was performed with the Nextclade online tool at URL: <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) and classification into Pangolin lineages was performed using the Pangolin online tool at URL: <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</ext-link> [<xref ref-type="bibr" rid="CR22"><u>22</u></xref>].<li> The phylogenetic tree was built using the <b>Nextstrain</b> tool (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/index.html">https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/index.html</ext-link>) [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>] that performs maximum-likelihood phylogeny using IQ-TREE [<xref ref-type="bibr" rid="CR21"><u>21</u></xref>], and was visualised with Auspice (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).<li>adapted from screenshots of an output of the <b>Nextstrain</b> tool (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/index.html">https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/index.html</ext-link>) [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>].<li> They involved three, one, and three viral strains classified in <b>Nextstrain</b> clades 20A, 20B, and 20G, respectively.<li> reported the emergence late 2020 of variants harbouring substitutions Q677H/P in the spike of viruses that were classified into sublineages of <b>Nextstrain</b> clades 20B and 20G as well as into <b>Nextstrain</b> clade 20A [<xref ref-type="bibr" rid="CR12"><u>12</u></xref>], and the Q677H substitution has been identified in the B.1.525 lineage first described in Nigeria [<xref ref-type="bibr" rid="CR33"><u>33</u></xref>] and as a fast growing mutation in variants of concern [<xref ref-type="bibr" rid="CR31"><u>31</u></xref>].<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33758901">33758901</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34729568">34729568</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33804556">33804556</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34848213">34848213</a> <b>(0)</b> Genomic monitoring of the SARS-CoV-2 B1.1.7 (WHO VOC Alpha) in the Sao Paulo state, Brazil<ul><li> We assessed genome consensus sequence quality using Nextclade v0.8.1 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="interref0005">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34845455">34845455</a> <b>(3)</b> Unsupervised clustering analysis of SARS-Cov-2 population structure reveals six major subtypes at early stage across the world<ul><li>All downloaded genomes were mapped to the reference genome of SARS-CoV-2 (GenBank Accession Number: <ext-link xlink:href="NC_045512.2" ext-link-type="DDBJ/EMBL/GenBank">NC_045512.2</ext-link>) following <b>Nextstrain</b> pipeline [<xref rid="R21" ref-type="bibr"><u>21</u></xref>].<li> Considering many putatively artefactual mutations and the gaps in sequences are located at the beginning and end of the alignment, we masked the first 130 bps and last 50 bps in mutation calling following <b>Nextstrain</b> pipeline.<li> The phylogeny is rooted following <b>Nextstrain</b> pipeline using FigTree v1.4.4.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34817214">34817214</a> <b>(3)</b> Whole-Genome Sequences of SARS-CoV-2 Isolates from the Dominican Republic<ul><li>Phylogenetic tree analysis was conducted using the <b>Nextstrain</b> bioinformatics platform (<ext-link xlink:href="http://<b>nextstrain</b>.org/ncov" ext-link-type="uri">http://<b>nextstrain</b>.org/ncov</ext-link>) with the maximum likelihood option and the JTT matrix (<xref rid="B12" ref-type="bibr"><u>12</u></xref><xref rid="B13" ref-type="bibr"><u>&#x2013;</u></xref><xref rid="B15" ref-type="bibr"><u>15</u></xref>).<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=1633570">1633570</a>) [&#x2013;](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15318951">15318951</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34870090">34870090</a> <b>(1)</b> Comprehensive literature review on COVID-19 vaccines and role ofSARS-CoV-2 variants in the pandemic<ul><li>Currently, there exists two open-source real-time software tools to analyze andassign nomenclature of genetic variations in the SARS-CoV-2 virus: <b>Nextstrain</b> andPANGOLIN.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34586866">34586866</a> <b>(1)</b> Genetic and Antigenic Characterization of an Influenza A(H3N2) Outbreak in Cambodia and the Greater Mekong Subregion during the COVID-19 Pandemic, 2020<ul><li> The GISAID data were selected based on criteria used by <b>Nextstrain</b> (<xref rid="B23" ref-type="bibr"><u>23</u></xref>), in which a greater frequency of recent samples were sampled along with key reference sequences.<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34880867">34880867</a> <b>(2)</b> Immune Evasive Effects of SARS-CoV-2 Variants to COVID-19 Emergency Used Vaccines<ul><li> Beta, known as B.1.351 in Pango lineage (<xref rid="B10" ref-type="bibr"><u>10</u></xref>), 20H (V2) in <b>nextstrain</b> clade, GH/501Y.V2 in GISAID clade, was first identified in South Africa, in May 2020.<li> Gamma, known as P.1 in Pango lineage (<xref rid="B11" ref-type="bibr"><u>11</u></xref>), 20J (V3) in <b>nextstrain</b> clade, GR/501Y.V3 in GISAID clade, was first identified in Brazil, in November 2020.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33501460">33501460</a>) [11](<person-group person-group-type="author"><name><surname>Faria</surname><given-names>NR</given-names></name><name><surname>Mellan</surname><given-names>TA</given-names></name><name><surname>Whittaker</surname><given-names>C</given-names></name><name><surname>Claro</surname><given-names>IM</given-names></name><name><surname>Candido</surname><given-names>DDS</given-names></name><name><surname>Mishra</surname><given-names>S</given-names></name><etal/></person-group>. <article-title>Genomics and Epidemiology of the P.1 SARS-CoV-2 Lineage in Manaus, Brazil</article-title>. <source>Science</source> (<year>2021</year>) <volume>372</volume>(<issue>6544</issue>):<page-range>815&#x2013;21</page-range>. doi:&#xA0;<pub-id pub-id-type="doi">10.1126/science.abh2644</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34822976">34822976</a> <b>(1)</b> Multiple introductions of SARS-CoV-2 B.1.1.214 lineages from mainland Japan preceded the third wave of the COVID-19 epidemic in Hokkaido<ul><li>Phylodynamic analysis [<xref rid="bib9" ref-type="bibr"><u>9</u></xref>], including subsampling, alignment, phylogenetic inference, temporal dating of ancestral nodes, and discrete trait geographic reconstruction, as well as inference of the most likely transmission events, was performed using the <b>Nextstrain</b> pipeline suite [<xref rid="bib10" ref-type="bibr"><u>10</u></xref>], as previously described [<xref rid="bib11" ref-type="bibr"><u>11</u></xref>].<li> [9](          <person-group person-group-type="author">            <name>              <surname>Volz</surname>              <given-names>E.M.</given-names>            </name>            <name>              <surname>Koelle</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>          </person-group>          <article-title>Viral phylodynamics</article-title>          <source>PLoS Comput Biol</source>          <volume>9</volume>          <year>2013</year>          <object-id pub-id-type="publisher-id">e1002947</object-id>          <pub-id pub-id-type="doi">10.1371/journal.pcbi.1002947</pub-id>        ) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33853975">33853975</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34495709">34495709</a> <b>(3)</b> Emergence of SARS-CoV-2 Variant B.1.575.2, Containing the E484K Mutation in the Spike Protein, in Pamplona, Spain, May to June 2021<ul><li> The viral lineage classifications were performed with the Global Initiative on Sharing Avian Influenza Data (GISAID) EpiCoV database (<ext-link xlink:href="https://www.gisaid.org" ext-link-type="uri">https://www.gisaid.org</ext-link>), <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>), and the PANGO lineage system (<ext-link xlink:href="https://cov-lineages.org" ext-link-type="uri">https://cov-lineages.org</ext-link>) (<xref rid="B9" ref-type="bibr"><u>9</u></xref><xref rid="B10" ref-type="bibr"><u>&#x2013;</u></xref><xref rid="B11" ref-type="bibr"><u>11</u></xref>).<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [&#x2013;](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34812427">34812427</a> <b>(1)</b> Metaviromic identification of discriminative genomic features in SARS-CoV-2 using machine learning<ul><li>Due to the urgency of the pandemic, SARS-CoV-2 genomes have been sequenced at an unprecedented rate, with more than 1 million sequences available through the Global Initiative on Sharing All Influenza Data (GISAID).<xref rid="bib31" ref-type="bibr"><sup>32</sup></xref><sup>,</sup><xref rid="bib32" ref-type="bibr"><sup>33</sup></xref> Other resources such as the <b>Nextstrain</b> project provide genomic epidemiology analyses on the number of accumulated mutational events across the SARS-CoV-2 genome (<xref rid="mmc1" ref-type="supplementary-material"><u>Figure&#xA0;S7</u></xref>D).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34788739">34788739</a> <b>(2)</b> The Dominance of Severe Acute Respiratory Syndrome Coronavirus 2 B.1.617 and Its Sublineages and Associations with Mortality during the COVID-19 Pandemic in India between 2020 and 2021<ul><li>A phylogenetic dendrogram was constructed based on the complete genome sequences of 15740 Indian SARS-COV-2 strains using the Nextclade web-based tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> The selected sequences were uploaded to Nextclade webtool and aligned against genome sequence of SARS-COV-2 (Wuhan-Hu-1/2019) to assign <b>Nextstrain</b> clades and construct phylogeny trees of Indian SARS-COV-2 strains.<li>(<bold>A</bold>) Overall distribution of severe acute respiratory syndrome coronavirus 2 (SARS-COV-2) clades according to <b>Nextstrain</b> classification.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34798321">34798321</a> <b>(3)</b> SARS-CoV-2 introduction and lineage dynamics across three epidemic peaks in Southern Brazil: massive spread of P.1<ul><li> Consensus sequences were submitted to Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="ir0010">https://clades.<b>nextstrain</b>.org</ext-link>) for quality inspection and to determine the mutational profile.<li> In addition, two phylogenomic analysis were performed, one concatenating our dataset of 90 genome sequences with the <b>Nextstrain</b>&#x2019; South America dataset (2961 sequences) and other concatenating only sequences classified as P.1 and P.1-like from Brazil available in GISAID (between December 2019 and April 2021).<li> Analyses were carried out using <b>Nextstrain</b> ncov workflow (<ext-link ext-link-type="uri" xlink:href="https://github.com/" id="ir0020">https://github.com/</ext-link>) (default parameters), which includes alignment, phylogenetic reconstruction, geographic and ancestral trait reconstruction, and inference of transmission events.<li> <b>Nextstrain</b> uses by default the software IQ-Tree and the substitution model GTR.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34787497">34787497</a> <b>(6)</b> SARS-CoV-2 Variants and Their Relevant Mutational Profiles: Update Summer 2021<ul><li> Mutations in black refer to the mutations reported by <b>Nextstrain</b>, Outbreak.info, Pango lineages, and Stanford database websites, while mutations in gray are those that we identified by analyzing entire high-quality viral genome sequences from GISAID (<italic toggle="yes">n</italic>&#x2009;=&#x2009;1,223,383).<li> Mutations in black refer to the mutations reported by the <b>Nextstrain</b>, Outbreak.info, Pango lineages, and Stanford database websites, while mutations in gray are those that we identified by analyzing entire high-quality viral genome sequences from GISAID (<italic toggle="yes">n</italic>&#x2009;=&#x2009;1,223,383).<li>The mutations and the variants cited in this review are those that were highlighted by at least one among the GISAID (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri" specific-use="live">https://www.gisaid.org/</ext-link>), <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri" specific-use="live">https://<b>nextstrain</b>.org/</ext-link>), Outbreak.info (<ext-link xlink:href="https://outbreak.info/" ext-link-type="uri" specific-use="live">https://outbreak.info/</ext-link>), Pango (<ext-link xlink:href="https://cov-lineages.org/" ext-link-type="uri" specific-use="live">https://cov-lineages.org/</ext-link>), and Stanford (<ext-link xlink:href="https://covdb.stanford.edu/" ext-link-type="uri" specific-use="live">https://covdb.stanford.edu/</ext-link>) database websites (<xref rid="B3" ref-type="bibr"><u>3</u></xref>, <xref rid="B18" ref-type="bibr"><u>18</u></xref><xref rid="B19" ref-type="bibr"><u>&#x2013;</u></xref><xref rid="B21" ref-type="bibr"><u>21</u></xref>) and that have been associated with any clinical or diagnostic impact according to published manuscripts present on PubMed or preprint manuscripts on bioRxiv or medRxiv.<li>The nomenclature was reported by the <b>Nextstrain</b> website.<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [&#x2013;](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<person-group person-group-type="author"><string-name><surname>Tzou</surname><given-names>P</given-names></string-name>, <string-name><surname>Tao</surname><given-names>K</given-names></string-name>, <string-name><surname>Nouhin</surname><given-names>J</given-names></string-name>, <string-name><surname>Rhee</surname><given-names>S-Y</given-names></string-name>, <string-name><surname>Hu</surname><given-names>B</given-names></string-name>, <string-name><surname>Pai</surname><given-names>S</given-names></string-name>, <string-name><surname>Parkin</surname><given-names>N</given-names></string-name>, <string-name><surname>Shafer</surname><given-names>R</given-names></string-name></person-group>. <year>2020</year>. <article-title>Coronavirus Antiviral Research Database (CoV-RDB): an online database designed to facilitate comparisons between candidate anti-coronavirus compounds</article-title>. <source>Viruses</source><volume>12</volume>:<fpage>1006</fpage>. doi:<pub-id pub-id-type="doi">10.3390/v12091006</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34787439">34787439</a> <b>(1)</b> Postvaccination SARS-COV-2 among Health Care Workers in New Jersey: A Genomic Epidemiological Study<ul><li>The resulting SARS-CoV-2 viral genome sequences were characterized by Nextclade CLI (v1) (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) to assign <b>Nextstrain</b> clades (<xref rid="B37" ref-type="bibr"><u>37</u></xref>).<li> [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34787498">34787498</a> <b>(0)</b> Prolonged SARS-CoV-2 Positivity in Immunocompetent Patients: Virus Isolation, Genomic Integrity, and Transmission Risk<ul><li>The novel genome sequences were classified with the pangolin tool v3.1.11 (pango v1.2.76, pangoLEARN model from 17 September 2021) (<xref rid="B44" ref-type="bibr"><u>44</u></xref>) and the NextClade Web application v1.7.1 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>), which was also used to annotate all mutations.<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34805437">34805437</a> <b>(3)</b> SARS-CoV-2 Genomic Surveillance Reveals Little Spread From a Large University Campus to the Surrounding Community<ul><li> The genomes presented here consisted of several different viral clades, mostly <b>Nextstrain</b> clades 20A, 20C, and 20G (<xref rid="sup1" ref-type="supplementary-material"><u>Supplementary Figure 1</u></xref>).<li> Genomes from Cluster A were part of <b>Nextstrain</b> clade 20B and Pango lineage B.1.1.304 (TMRCA, August 26<sup>;</sup> 95% CI, August 3&#x2013;September 26).<li> Genomes from Cluster B were part of <b>Nextstrain</b> clade 20C and Pango lineage B.1.593 (TMRCA, July 29; 95% CI, July 3&#x2013;August 28).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34789293">34789293</a> <b>(0)</b> Re-emergence of Gamma-like-II and emergence of Gamma-S:E661D SARS-CoV-2 lineages in the south of Brazil after the 2021 outbreak<ul><li> In parallel, we also assigned the viral lineages using the Nextclade web application (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34828410">34828410</a> <b>(1)</b> Relative Consolidation of the Kappa Variant Pre-Dates the Massive Second Wave of COVID-19 in India<ul><li> <b>Nextstrain</b> [<xref rid="B12-genes-12-01803" ref-type="bibr"><u>12</u></xref>] (available online: <uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 9 August 2021) was used for quality assessment and phylogenetic placement.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><u>No PMID</u> <b>(1)</b> Comparative analysis of the COVID-19 sequences in India and investigate the trialogue relationship between the miR- NA with Coronavirus and Gut Microbiota.<ul><li><bold>Methods:</bold> Selective COVID 19 genome sequences with special reference from India by NCBI genome database and <b>Nextstrain</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34785628">34785628</a> <b>(2)</b> Why are some coronavirus variants more infectious?<ul><li> Figure created from <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link> and colored as per clades.</ul></p>
</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34781748">34781748</a> <b>(5)</b> Evolutionary and Phenotypic Characterization of Two Spike Mutations in European Lineage 20E of SARS-CoV-2<ul><li> In many countries, the challenge of detecting new mutations of interest in severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) involves sequencing representative genomes from circulating viruses, sharing sequence information on public databases (e.g., GISAID [<xref rid="B1" ref-type="bibr"><u>1</u></xref>]), and analyzing them in real time using platforms such as <b>Nextstrain</b> (<xref rid="B2" ref-type="bibr"><u>2</u></xref>).<li>The phylogeny in <xref rid="movS1" ref-type="supplementary-material"><u>Video S1</u></xref>, composed by 10,450 sequences, was built with the <b>Nextstrain</b> pipeline (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/augur" ext-link-type="uri">https://github.com/<b>nextstrain</b>/augur</ext-link>) to monitor and visualize temporal and geographical transmission of 1163.7.<li>Geographical transmission of 1163.7 visualized with <b>Nextstrain</b> build.<li> [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34785663">34785663</a> <b>(5)</b> Organ-specific genome diversity of replication-competent SARS-CoV-2<ul><li> Clades for each sequence were assigned via Nextclade (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade/">https://github.com/<b>nextstrain</b>/nextclade/</ext-link>).<li> Using the <b>nextstrain</b> toolkit v6, a maximum-likelihood phylogenetic tree was constructed via IQ-Tree v1.6.9, using a GTR substitution model and performing 100 bootstraps with annotation and visualization done via iTol v6 (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/">https://github.com/<b>nextstrain</b>/ncov/</ext-link>)<sup><xref ref-type="bibr" rid="CR53"><u>53</u></xref>,<xref ref-type="bibr" rid="CR54"><u>54</u></xref></sup>.<li> [53](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [54](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27095192">27095192</a>) </ul></p>
<p><u>No PMID</u> <b>(4)</b> The seasonal behaviour of COVID-19 and its galectin-like culprit of the viral spike<ul><li> 6</xref>shows an example maximum likelihood tree reconstructed from 3615 sampled genomes using the <b>Nextstrain</b> online system (<xref rid="bb0315" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) with groups labelled as either GISAID clades or emerging VOCs.<li> A maximum likelihood phylogenetic tree of 3615 genomes sampled from more than a million GISAID worldwide sequences between December 2019 and September 2021 obtained from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="ir0015">https://<b>nextstrain</b>.org</ext-link>) shows groups of taxa (leaves of the tree indicated with circles) labelled as either GISAID clades or emerging variants of concern (VOCs) and their origins along branches of the trees.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34773976">34773976</a> <b>(2)</b> In depth analysis of Cyprus-specific mutations of SARS-CoV-2 strains using computational approaches<ul><li> The <b>nextstrain</b> [<xref ref-type="bibr" rid="CR17"><u>17</u></xref>] pipeline was downloaded locally and the commands for filtering, aligning and constructing phylogeny were used according to the <b>nextstrain</b>&#x2019;s best practices.<li> [17](Hadfield J, Megill C, Bell SM, Huddleston J, Potter B, Callender C, et al. NextStrain: real-time tracking of pathogen evolution. Bioinformatics. 2018;34:4121&#x2013;3.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34822776">34822776</a> <b>(4)</b> Nucleocapsid mutations R203K/G204R increase the infectivity, fitness, and virulence of SARS-CoV-2<ul><li><hr/></td></tr><tr><td>GISAID</td><td>Freunde von GISAID e.V.</td><td><ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org" id="intref0130">https://www.gisaid.org</ext-link></td></tr><tr><td>CoVdb</td><td>(<xref rid="bib75" ref-type="bibr"><u>Zhu et&#xA0;al., 2021a</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://covdb.popgenetics.net" id="intref0135">http://covdb.popgenetics.net</ext-link></td></tr><tr><td>The epidemic data of COVID-19</td><td>Global Change Data Lab</td><td><ext-link ext-link-type="uri" xlink:href="https://ourworldindata.org/covid-deaths" id="intref0140">https://ourworldindata.org/covid-deaths</ext-link></td></tr><tr><td><b>Nextstrain</b>.<li><hr/></td></tr><tr><td>GISAID</td><td>Freunde von GISAID e.V.</td><td><ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org" id="intref0130">https://www.gisaid.org</ext-link></td></tr><tr><td>CoVdb</td><td>(<xref rid="bib75" ref-type="bibr"><u>Zhu et&#xA0;al., 2021a</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://covdb.popgenetics.net" id="intref0135">http://covdb.popgenetics.net</ext-link></td></tr><tr><td>The epidemic data of COVID-19</td><td>Global Change Data Lab</td><td><ext-link ext-link-type="uri" xlink:href="https://ourworldindata.org/covid-deaths" id="intref0140">https://ourworldindata.org/covid-deaths</ext-link></td></tr><tr><td><b>Nextstrain</b></td><td>(<xref rid="bib21" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0145">https://<b>nextstrain</b>.org</ext-link>.<li> The source data are from GISAID and <b>Nextstrain</b> (<xref rid="bib21" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> [Zhu et&#xA0;al., 2021a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33009914">33009914</a>) [Zhu et&#xA0;al., 2021a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33009914">33009914</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34755363">34755363</a> <b>(0)</b> Evolution of spike mutations following antibody treatment in two immunocompromised patients with persistent COVID&#x2010;19 infection<ul><li> The global phylogenetic placement was determined using the Nextclade database (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34767579">34767579</a> <b>(5)</b> Clinical and molecular epidemiology of influenza viruses from Romanian patients hospitalized during the 2019/20 season<ul><li>The generated consensus sequences were aligned against the reference sequences retrieved from the GISAID database, which were selected as follows: other available sequences from Romania during the 2019&#x2013;2020 influenza season, sequences of the influenza strains recommended for the 2019/2020 vaccine and clade defining sequences indicated by <b>Nextstrain</b> platform (<ext-link xlink:href="https://<b>nextstrain</b>.org/flu/seasonal" ext-link-type="uri">https://<b>nextstrain</b>.org/flu/seasonal</ext-link>).<li> Further analysis using phylogenetic tools and clade defining sequences (<ext-link xlink:href="https://<b>nextstrain</b>.org/flu/seasonal" ext-link-type="uri">https://<b>nextstrain</b>.org/flu/seasonal</ext-link>) showed that 4/18 belonged to 6b1.A.183P -5a clade (highlighted in blue in <xref rid="pone.0258798.g003" ref-type="fig"><u>Fig 3</u></xref>), while the other sequences were assigned to 6b1.A.187A clade (marked in yellow in the tree).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34763751">34763751</a> <b>(3)</b> The SARS-CoV-2 Lambda variant and its neutralisation efficiency following vaccination with Comirnaty, Israel, April to June 2021<ul><li>A phylogenetic tree of imported and circulating SARS-CoV-2 lineages in Israel in May&#x2013;April 2021, constructed via the <b>Nextstrain</b> Augur pipeline [<xref rid="r5" ref-type="bibr"><u>5</u></xref>], shows the dominating Alpha and Delta variants in Israel (<xref rid="f2" ref-type="fig"><u>Figure 2A</u></xref>).<li> The phylogenetic tree was constructed via <b>Nextstrain</b> Augur pipeline [<xref rid="r5" ref-type="bibr"><u>5</u></xref>].<li> The phylogenetic tree was constructed via <b>Nextstrain</b> Augur pipeline [<xref rid="r5" ref-type="bibr"><u>5</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34858480">34858480</a> <b>(2)</b> Evolutionary and Antigenic Profiling of the Tendentious D614G Mutation of SARS-CoV-2 in Gujarat, India<ul><li> Occurrence of D614G mutation over time was visualized using <b>NextStrain</b> platform (<xref rid="B7" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) where data is enabled from the GISAID.<li> <b>NextStrain</b> visualization analysis can process up to 3000 genomic sequences at a time.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34790342">34790342</a> <b>(6)</b> On the association between SARS-COV-2 variants and COVID-19 mortality during the second wave of the pandemic in Europe<ul><li> Therefore, in this study we collected country-level data on the COVID-19 mortality during the second (winter) wave of the pandemic in Europe and investigated whether the change of the proportion of any SARS-COV-2 variant, including the B.1.1.7 and 11 other variants identified by <b>Nextstrain</b> up to 25 February 2021 [<xref rid="cit0016" ref-type="bibr"><u>16</u></xref>,<xref rid="cit0017" ref-type="bibr"><u>17</u></xref>], has an association with COVID-19 cumulative mortality or with the height of the second-wave COVID-19 mortality peak.<li>The main explanatory variables of interest, assumed to have a potential association with the above outcomes, were average proportions of SARS-COV-2 sequences among identified sequences in the same period used to form <b>Nextstrain</b> clades.<li> Twelve (12) clades (19A, 19B, 20A, 20A.EU2, 20B, 20&#xA0;C, 20D, 20E.EU1, 20&#xA0;G, 20&#xA0;H/501Y.V2, 20I/501Y.V1 (B.1.1.7), 20&#xA0;J/501Y.V3) were identified by <b>Nextstrain</b> from December 2019 to March 2021 in the European region [<xref rid="cit0017" ref-type="bibr"><u>17</u></xref>,<xref rid="cit0018" ref-type="bibr"><u>18</u></xref>].<li> In particular, the dataset included information on date and country where a given strain was observed, as well as GISAID and <b>Nextstrain</b> clade to which a given strain was classified.<li> classification used in this study was developed by <b>Nextstrain</b> group.<li> <b>Nextstrain</b> is an open-source project providing real-time phylogenetic analysis of SARS-CoV-2 variants, grouping them into clades [<xref rid="cit0016" ref-type="bibr"><u>16&#x2013;18</u></xref>].<li> [16](<person-group person-group-type="author"><string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name></person-group>. <article-title>CoVariants: SARS-CoV-2 mutations and variants of interest</article-title>; <year>2021</year><date-in-citation>[cited 2021 <month>Mar</month><day>15]</day></date-in-citation>. <comment>Available from:</comment><ext-link xlink:href="https://covariants.org/" ext-link-type="uri">https://covariants.org/</ext-link>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<person-group person-group-type="author"><string-name><surname>Bedford</surname><given-names>T</given-names></string-name>, <string-name><surname>Neher</surname><given-names>R</given-names></string-name>, <string-name><surname>Hadfield</surname><given-names>J</given-names></string-name>, et al</person-group>. <article-title>Nextstrain - real-time tracking of pathogen evolution</article-title>; <year>2021</year><date-in-citation>[cited 2021 <month>Mar</month><day>15]</day></date-in-citation>. <comment>Available from:</comment><ext-link xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link>) [16&#x2013;18](<person-group person-group-type="author"><string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name></person-group>. <article-title>CoVariants: SARS-CoV-2 mutations and variants of interest</article-title>; <year>2021</year><date-in-citation>[cited 2021 <month>Mar</month><day>15]</day></date-in-citation>. <comment>Available from:</comment><ext-link xlink:href="https://covariants.org/" ext-link-type="uri">https://covariants.org/</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34901253">34901253</a> <b>(1)</b> SARS-CoV-2 Infection in One Cat and Three Dogs Living in COVID-19-Positive Households in Madrid, Spain<ul><li> Clade assignment and phylogenetic visualization were determined using <b>Nextstrain</b> online software (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) through the UShER web interface from the University of California, Santa Cruz (<ext-link xlink:href="https://genome.ucsc.edu/cgi-bin/hgPhyloPlace" ext-link-type="uri">https://genome.ucsc.edu/cgi-bin/hgPhyloPlace</ext-link>) (<xref rid="B14" ref-type="bibr"><u>14</u></xref>, <xref rid="B15" ref-type="bibr"><u>15</u></xref>).<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34757384">34757384</a> <b>(0)</b> A stem-loop RNA RIG-I agonist protects against acute and chronic SARS-CoV-2 infection in mice<ul><li> We used Nextclade v0.14.2 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) to generate a phylogenetic tree and to compile a list of amino acid changes in the virus isolates as compared with the Wuhan-Hu-1 reference strain (<xref rid="sup1" ref-type="supplementary-material"><u>Table S1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34756072">34756072</a> <b>(4)</b> Sequencing Using a Two-Step Strategy Reveals High Genetic Diversity in the S Gene of SARS-CoV-2 after a High-Transmission Period in Tunis, Tunisia<ul><li> Indeed, two different lineages, A and B, were proposed by the Phylogenetic Assignment of Named Global Outbreak (PANGO) lineage nomenclature, while a classification into 11 different clades (19-A, 19-B, and 20-A to 20-I) was proposed by the <b>Nextstrain</b> resources and another classification into 9 clades (S, L, O, V, G, GH, GR, GRY, and GV) was proposed by Global Initiative on Sharing All Influenza Data (GISAID).<li> The figure also shows the classification of the Tunisian sequences according to the PANGO and the <b>Nextstrain</b> classifications.<li> The phylogenetic distribution of these sequences clearly shows the presence of 3 subclusters, called subclusters 1a, 1b, and 1c and classified as clade G/20A, GV/20A-C and GH/20C, respectively, according to the GISAID/<b>Nextstrain</b> nomenclatures.<li> the sequences also belong to the PANGO B lineage and to the 20B clade of the <b>Nextstrain</b> nomenclature.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34758478">34758478</a> <b>(1)</b> Pre-existing polymerase-specific T cells expand in abortive seronegative SARS-CoV-2<ul><li> The alignment was subsampled to include 800 of each defined <b>NextStrain</b> phylogenetic clade, as provided by GISAID metadata.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34755818">34755818</a> <b>(1)</b> SARS-CoV-2 variant N.9 identified in Rio de Janeiro, Brazil<ul><li> The mutational profile was investigated using the Nextclade v.0.14.2 (https://clades.<b>nextstrain</b>.org, Nextclade 2020-2021 <b>Nextstrain</b> developers), and the CoVSurver mutations App (https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/, A*STAR Bioinformatics Institute, Singapore).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37124703">37124703</a> <b>(5)</b> CoVizu: Rapid analysis and visualization of the global diversity of SARS-CoV-2 genomes<ul><li> For example, <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>, last accessed 22 October 2021; (<xref rid="R7" ref-type="bibr">Hadfield et&#xA0;al.<li> Throughout the SARS-CoV-2 pandemic, <b>Nextstrain</b> has featured prominently in global variant tracking and scientific communication, and in some cases it has directly influenced public health decision-making (<xref rid="R1" ref-type="bibr">Bedford et&#xA0;al.<li>However, the data visualization maintained by <b>Nextstrain</b> is limited in practice to fewer than about 5,000 genome sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34835044">34835044</a> <b>(3)</b> Circulation and Evolution of SARS-CoV-2 in India: Let the Data Speak<ul><li> Last accessed 30 October 2021) [<xref rid="B5-viruses-13-02238" ref-type="bibr"><u>5</u></xref>], <b>NextStrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri>; Last accessed 30 October 2021) [<xref rid="B6-viruses-13-02238" ref-type="bibr"><u>6</u></xref>], and PANGOLIN (Phylogenetic Assignment of Named Global Outbreak LINeages; <uri xlink:href="https://cov-lineages.org/resources/pangolin.html">https://cov-lineages.org/resources/pangolin.html</uri>; Last accessed 30 October 2021) [<xref rid="B7-viruses-13-02238" ref-type="bibr"><u>7</u></xref>] proposed unique lineage nomenclatures.<li> Lineages were assigned to these entries using NextClade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>; Last accessed 30 October 2021) [<xref rid="B6-viruses-13-02238" ref-type="bibr"><u>6</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35822801">35822801</a> <b>(1)</b> VirusLab: A Tool for Customized SARS-CoV-2 Data Analysis<ul><li> Others community-curated efforts pre-existed the COVID-19 pandemic and they were dedicated to viruses in general, e.g., <b>NextStrain</b> [<xref rid="B17-biotech-10-00027" ref-type="bibr"><u>17</u></xref>], iVar [<xref rid="B18-biotech-10-00027" ref-type="bibr"><u>18</u></xref>], and viralrecon/nf-core [<xref rid="B19-biotech-10-00027" ref-type="bibr"><u>19</u></xref>].<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30606230">30606230</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32055031">32055031</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34934643">34934643</a> <b>(1)</b> Newer variants of COVID-19, newer challenges of whole-genome strategy in India: A public health perspective<ul><li> An analysis of 361 complete genome sequences from India showed that five global clades were circulating in India&#x2013;old <b>Nextstrain</b> clades B, B4, A2a, A3, and a distinct clade A3i.[<xref rid="ref11" ref-type="bibr"><u>11</u></xref>].<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32362650">32362650</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34732275">34732275</a> <b>(2)</b> SARS-CoV-2 variants and effectiveness of vaccines: a review of current evidence<ul><li> <uri xlink:href="https://www.gisaid.org/">www.gisaid.org/</uri>) [<xref rid="ref16" ref-type="bibr"><u>16</u></xref>] and <b>Nextstrain</b> (<b>nextstrain</b>.org/) [<xref rid="ref17" ref-type="bibr"><u>17</u></xref>] as well as the epidemiological tool Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN; <uri xlink:href="https://cov-lineages.org/">cov-lineages.org/</uri>) [<xref rid="ref18" ref-type="bibr"><u>18</u></xref>].<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [17](<string-name><surname>Hadfield</surname><given-names>J</given-names></string-name><etal/> (<year>2018</year>) <article-title>NextStrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>34</volume>, <fpage>4121</fpage>&#x2013;<lpage>4123</lpage>.) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34738512">34738512</a> <b>(3)</b> The impact of public health interventions in the Nordic countries during the first year of SARS-CoV-2 transmission and evolution<ul><li> To obtain a representative subset of the global diversity, we also selected 3,437 genomes from the latest <b>NextStrain</b> global build on 22 March 2021 [<xref rid="r7" ref-type="bibr"><u>7</u></xref>].<li> We repeated this procedure 10 times, and in each obtained a sequence alignment with all data from <b>NextStrain</b> and with the number of genomes from the Nordic countries proportional to the prevalence of the virus (<xref rid="t1" ref-type="table"><u>Table</u></xref>).<li> between 12,179 and 11,860 Nordic genomes, plus the 3,437 from the <b>NextStrain</b> build).<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34777554">34777554</a> <b>(2)</b> The third booster vaccination dose against COVID-19: indication for circulating SARS-CoV-2 variants<ul><li>Unfortunately, the number of identified, circulating, SARS-CoV-2 variants is alarming in a relatively large number of countries that struggle with the pandemic (supportive data representing December 2019&#x2013;September 2021 can be accessed through <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34777552">34777552</a> <b>(1)</b> The Mu strain: the last but not least circulating &#x2018;variant of interest&#x2019;&#xA0;potentially affecting the COVID-19 pandemic<ul><li> The Mu variant is classified as B.1.621 (PANGO lineage) or 21H (<b>nextstrain</b> clade).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34737266">34737266</a> <b>(1)</b> Within-host evolution of SARS-CoV-2 in an immunosuppressed COVID-19 patient as a source of immune escape variants<ul><li> Accordingly, the predominant virus genotypes initially found in swab samples during the first weeks of the persistent infection of the immunocompromised patient were relatively stable and grouped into <b>Nextstrain</b> clade 20B, Pangolin lineage B.1.1, together with simultaneously circulating variants (Fig.&#xA0;<xref rid="Fig2" ref-type="fig"><u>2a, b</u></xref>).<li> The clades were classified with the webservers of Nextclade (clades.<b>nextstrain</b>.org) and Pangolin (pangolin.cog-uk.io).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34835031">34835031</a> <b>(8)</b> Rapid Response to SARS-CoV-2 in Aotearoa New Zealand: Implementation of a Diagnostic Test and Characterization of the First COVID-19 Cases in the South Island<ul><li> Further phylogenetic analysis using 70 contemporary SARS-CoV-2 sequences from different lineages, as well as blast comparisons using a series of databases, i.e., <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/">https://<b>nextstrain</b>.org/ncov/</uri>, accessed on 15 April 2020), PANGO Lineages (<uri xlink:href="https://cov-lineages.org/index.html">https://cov-lineages.org/index.html</uri>, accessed on 15 April 2020), Los Alamos COVID-19 Viral Genome Analysis Pipeline (<uri xlink:href="https://cov.lanl.gov/content/index">https://cov.lanl.gov/content/index</uri>, accessed on 15 April 2020), and COVID-19 Research at UCSC (<uri xlink:href="https://hgwdev.gi.ucsc.edu/covid19.html">https://hgwdev.gi.ucsc.edu/covid19.html</uri>, accessed on 15 April 2020), classified these two new SARS-CoV-2 sequences as: <b>Nextstrain</b> Clade 19A, Pango Lineage B.55, and GISAID clade B(<italic toggle="yes">L</italic>) for NZ/Queenstown/01 and <b>Nextstrain</b> Clade 19A, Pango Lineage B.31, and GISAID clade B.2(<italic toggle="yes">O</italic>) for NZ/Dunedin/01 (<xref rid="viruses-13-02222-f005" ref-type="fig"><u>Figure 5</u></xref>B).<li> Nine larger clades have been described in GISAID, 19 clades by <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/">https://<b>nextstrain</b>.org/ncov/</uri>, accessed on 31 August 2021), while an increasing number of PANGO lineages (<uri xlink:href="https://cov-lineages.org">https://cov-lineages.org</uri>, accessed on 31 August 2021) are used to track the transmission and spread of SARS-CoV-2 worldwide, including several variants of interest or concern [<xref rid="B35-viruses-13-02222" ref-type="bibr"><u>35</u></xref>].<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34075212">34075212</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34742926">34742926</a> <b>(0)</b> Emergence of B.1.1.318 SARS-CoV-2 viral lineage and high incidence of alpha B.1.1.7 variant of concern in the Republic of Gabon<ul><li> SARS-CoV-2 genomes were classified into lineages using Phylogenetic Assignment of Named Global Outbreak LINeages (pangolin) (<xref rid="bib0005" ref-type="bibr"><u>O'Toole&#xA0;et&#xA0;al.&#xA0;2021</u></xref>) and viral clades by Nextclade Beta (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0002">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [O'Toole&#xA0;et&#xA0;al.&#xA0;2021](          <person-group person-group-type="author">            <name>              <surname>O'Toole</surname>              <given-names>&#xC1;ine</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>Emily</given-names>            </name>            <name>              <surname>Underwood</surname>              <given-names>Anthony</given-names>            </name>            <name>              <surname>Jackson</surname>              <given-names>Ben</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>Verity</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>John T</given-names>            </name>            <name>              <surname>Colquhoun</surname>              <given-names>Rachel</given-names>            </name>            <etal/>          </person-group>          <article-title>Assignment of Epidemiological Lineages in an Emerging Pandemic Using the Pangolin Tool</article-title>          <source>Virus Evolution</source>          <volume>7</volume>          <issue>2</issue>          <year>2021</year>          <fpage>1</fpage>          <lpage>9</lpage>          <pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34732835">34732835</a> <b>(4)</b> Characterization of the SARS-CoV-2 genomes in Egypt in first and second waves of infection<ul><li> The lower plot includes the rate of mutations per sample over different time points in the Egyptian samples (Source: <b>nextstrain</b>.org).<li>Using the Pangolin and <b>Nextstrain</b> methods of lineage classification, isolates in clade 1 were assigned mainly to Pangolin lineage B.1.1.1 and B.1 and <b>Nextstrain</b> clade 20D and 20A.<li> The majority of isolates in Clade 3 belonged to Pangolin lineage A and <b>Nextstrain</b> clade 19B.</ul></p>
<p><u>No PMID</u> <b>(10)</b> First whole-genome analysis of the novel coronavirus (SARS-CoV-2) obtained from COVID-19 patients from five districts in Western Serbia<ul><li> In parallel, another classification system was developed through the <b>Nextstrain</b>.org platform (<uri xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</uri>), which tracks SARS-CoV-2 evolution in real-time.<li> The <b>Nextstrain</b> classification organises sequences into major clades marked by the year in which the particular SARS-CoV-2 strains have emerged and a letter (i.e.<li> Nevertheless, both GISAID and <b>Nextstrain</b> classifications are based on dynamic SARS-CoV-2 phylogenetic frameworks, which identify those viral lineages that contribute most to the active spread and those that became inactive.<li> The <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>) platform was used for the real-time tracking of SARS-Cov-2 genome evolution [<xref rid="ref12" ref-type="bibr"><u>12</u></xref>].<li> 1a</xref>), or into 20A <italic toggle="yes">vs.</italic> 20B clade, according to the <b>Nextstrain</b> analysis (<xref rid="fig01" ref-type="fig">Fig.<li>Classification of 48 SARS-Cov-2 isolates from Western Serbia based on GISAID (a) and <b>Nextstrain</b> (b) nomenclature.<li> [12](<string-name><surname>Hadfield</surname><given-names>J</given-names></string-name><etal/> (<year>2018</year>) <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>34</volume>, <fpage>4121</fpage>&#x2013;<lpage>4123</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34728747">34728747</a> <b>(5)</b> Genomic diversity of SARS-CoV-2 in Oxford during United Kingdom&#x2019;s first national lockdown<ul><li>To better understand and visualize the genetic changes that occurred in the viral genome throughout the lockdown period, we performed phylogenetic analyses using the sequences we generated together with a globally representative reference dataset (obtained from <b>Nextstrain</b>, sampled from GISAID<sup><xref ref-type="bibr" rid="CR25"><u>25</u></xref>&#x2013;<xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>) (Fig.&#xA0;<xref rid="Fig3" ref-type="fig"><u>3</u></xref>A&#x2013;C)<italic>.<li> Smith-Waterman phylogenetic alignment of (<bold>A</bold>) spring and (<bold>B</bold>) summer samples on globally representative reference data sets (obtained from <b>Nextstrain</b>, sampled from GISAID<sup><xref ref-type="bibr" rid="CR25"><u>25</u></xref>&#x2013;<xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>).<li>Phylogenetic analysis was performed via <b>Nextstrain</b>&#x2019;s online tool Nextclade (version 0.14.2).<li> Sequenced SARS-CoV-2 genomes as well as globally representative reference dataset (December 2019-June 2021, obtained from <b>Nextstrain</b>, sampled from GISAID<sup><xref ref-type="bibr" rid="CR25"><u>25</u></xref>&#x2013;<xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>) were clustered using Augur, the phylodynamic pipeline provided by <b>Nextstrain</b>, and visualized using Auspice.<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34586058">34586058</a> <b>(5)</b> Interventions to Disrupt Coronavirus Disease Transmission at a University, Wisconsin, USA, August&#x2013;October 2020<ul><li> We used all available full-length sequences from Dane County through January 31, 2021, for phylogenetic analysis using the tools implemented in <b>Nextstrain</b> custom builds (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="R24" ref-type="bibr"><italic>24</italic></xref><italic>,</italic><xref rid="R25" ref-type="bibr"><italic>25</italic></xref>).<li> We built time-resolved and divergence phylogenetic trees using standard <b>Nextstrain</b> tools and scripts.<li> In addition, sequencing of Dane County specimens in <b>Nextstrain</b> represented a small proportion of the total number infections within the county (&#x2248;3.0%) and were sampled nonrandomly among clients of a large testing provider.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34651569">34651569</a> <b>(1)</b> Spike protein evolution in the SARS-CoV-2 Delta variant of concern: a case series from Northern Lombardy<ul><li> The WHO has recently dubbed VOCs with a simplified Greek alphabet nomenclature, easy to recall than former phylogenies provided by GISAID, PANGOLIN, <b>NextStrain</b>, and Public Health England (PHE).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34725257">34725257</a> <b>(3)</b> Mass spectrometric identification of immunogenic SARS-CoV-2 epitopes and cognate TCRs<ul><li> The <b>Nextstrain</b> project (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>), an open-source project that provides a continually updated view of publicly available data alongside powerful analytic and visualization tools to aid epidemiological understanding and improve outbreak response, provides a means to analyze genetic diversity across the SARS-CoV-2 genome.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34771026">34771026</a> <b>(2)</b> Robust Saliva-Based RNA Extraction-Free One-Step Nucleic Acid Amplification Test for Mass SARS-CoV-2 Monitoring<ul><li> 10<sup>&#x2212;4</sup> substitutions/site/year (approximately 24,600 substitutions/genome/year, <uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global?l=clock">https://<b>nextstrain</b>.org/ncov/gisaid/global?l=clock</uri>, accessed on 23 October 2021) [<xref rid="B23-molecules-26-06617" ref-type="bibr"><u>23</u></xref>] with mutations concentrated within hotspots, one of which is the S1 protein.<li> [23](          <person-group person-group-type="author"><name><surname>Obermeyer</surname><given-names>F.</given-names></name><name><surname>Schaffner</surname><given-names>S.F.</given-names></name><name><surname>Jankowiak</surname><given-names>M.</given-names></name><name><surname>Barkas</surname><given-names>N.</given-names></name><name><surname>Pyle</surname><given-names>J.D.</given-names></name><name><surname>Park</surname><given-names>D.J.</given-names></name><name><surname>MacInnis</surname><given-names>B.L.</given-names></name><name><surname>Luban</surname><given-names>J.</given-names></name><name><surname>Sabeti</surname><given-names>P.C.</given-names></name><name><surname>Lemieux</surname><given-names>J.E.</given-names></name></person-group>          <article-title>Analysis of 2.1 million SARS-CoV-2 genomes identifies mutations associated with transmissibility</article-title>          <source>medRxiv</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.1101/2021.09.07.21263228</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34717546">34717546</a> <b>(1)</b> SARS-CoV-2 surveillance in Italy through phylogenomic inferences based on Hamming distances derived from pan-SNPs, -MNPs and -InDels<ul><li>Faced to the rareness of other tools dedicated to lineage identification of SARS-CoV-2 (<b>Nextstrain</b> [<xref ref-type="bibr" rid="CR113"><u>113</u></xref>], GISAID [<xref ref-type="bibr" rid="CR114"><u>114</u></xref>] and PhenoGraph-based [<xref ref-type="bibr" rid="CR115"><u>115</u></xref>]), the workflow PANGOLIN has been implemented in the GENPAT workflow (Fig.<li> [113](Bedford T, Hodcroft EB, Neher RA. Updated Nextstrain SARS-CoV-2 clade naming strategy. Retrieved 16 March 2021. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming">https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>.) [114](          <person-group person-group-type="author">            <collab>GISAID</collab>          </person-group>          <source>Clade and lineage nomenclature aids in genomic epidemiology studies of active hCoV-19 viruses</source>          <year>2021</year>        ) [115](Yang Z-K, Pan L, Zhang Y, Luo H, Gao F. Data-driven identification of SARS-CoV-2 subpopulations using PhenoGraph and binary-coded genomic data. Brief Bioinform. 2021;00(00):1&#x2013;11.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34718395">34718395</a> <b>(5)</b> A comprehensive review of the analysis and integration of omics data for SARS-CoV-2 and COVID-19<ul><li> In addition, the <b>Nextstrain</b> lists publicly available SARS-CoV-2 analyses that used <b>Nextstrain</b> from groups all over the world.<li><b>Nextstrain</b>.<li><ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34725629">34725629</a> <b>(2)</b> DNA Mutations via Chern&#x2013;Simons Currents<ul><li> To be more precise, we used the tool <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>Nextclade, yielding the graph of Fig.&#xA0;<xref rid="Fig5" ref-type="fig"><u>5</u></xref>.<li> This figure shows the aforementioned evolution of the sequences (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34713795">34713795</a> <b>(5)</b> Genomic epidemiology reveals multiple introductions of SARS-CoV-2 followed by community and nosocomial spread, Germany, February to May 2020<ul><li> <b>Nextstrain</b> nomenclature clades are indicated above the main branches.<li> <b>Nextstrain</b> nomenclature clades are indicated above the main branches.<li>To allow better interaction with the sequence data shown in this study, we build a Narrative using <b>Nextstrain</b> that can be accessed here: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/narratives/axgraf/SARS-CoV-2/Munich">https://<b>nextstrain</b>.org/community/narratives/axgraf/SARS-CoV-2/Munich</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34777297">34777297</a> <b>(5)</b> Genomic Epidemiology Reveals Multiple Introductions of Severe Acute Respiratory Syndrome Coronavirus 2 in Niigata City, Japan, Between February and May 2020<ul><li> The lineage or clade classification of SARS-CoV-2 was performed using those of the Global Initiative on Sharing All Influenza Data (GISAID) database<sup><xref rid="footnote1" ref-type="fn"><u>1</u></xref></sup>, PANGOLIN (version 2.0<sup><xref rid="footnote2" ref-type="fn"><u>2</u></xref></sup> (<xref rid="B52" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) and <b>Nextstrain</b> team version 8.0<sup><xref rid="footnote3" ref-type="fn"><u>3</u></xref></sup> (<xref rid="B19" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) (<xref rid="DS3" ref-type="supplementary-material"><u>Supplementary Table 2</u></xref>).<li> Global strains in the present study were collected from December 2019 to May 2020, which is used in the global phylogenetic tree, constructed by <b>Nextstrain</b> team version 8.0 (see Text Footnote 3) (<xref rid="B19" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), and the Japanese strains from January to May 2020 excluding clade O (other) and airport quarantine isolates in Japan downloaded on December 04, 2020.<li>Retrospectively trace the dispersal dynamics of SARS-CoV-2 transmission patterns in Niigata City and other global/local areas, phylogeographic analysis was performed by <b>Nextstrain</b> team version 8.0 (see Text Footnote 3) (<xref rid="B19" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) against 292 global strains, 3,968 Japanese strains, and all 47 strains obtained in this present study, which had accurate collection date information.<li> Global strains in the present study were collected from December 2019 to May 2020 and were used to construct the global phylogenetic tree using <b>Nextstrain</b> team version 8.0 (see Text Footnote 3) (<xref rid="B19" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and all Japanese strains from January to May 2020 downloaded on August 31, 2021.<li>To further explore the dispersal dynamics of SARS-CoV-2 transmission patterns in Niigata City and other global/local areas, we also employed a phylogeographic inference analytic approach, using <b>Nextstrain</b> team version 8.0 (see Text Footnote 3) (<xref rid="B19" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), on the whole genomes of all 47 isolates from Niigata that had accurate collection date information (<xref rid="DS1" ref-type="supplementary-material"><u>Supplementary Figure 2</u></xref> and <xref rid="DS1" ref-type="supplementary-material"><u>Supplementary Table 5</u></xref>).<li> [1](<a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) [2](<a href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</a>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [3](<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2021.749149/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fmicb.2021.749149/full#supplementary-material</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34706766">34706766</a> <b>(12)</b> A small number of early introductions seeded widespread transmission of SARS-CoV-2 in Qu&#xE9;bec, Canada<ul><li>Raw and time-scaled phylogenomic trees were built using the <b>NextStrain</b> pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>">https://github.com/<b>nextstrain</b></ext-link>, version 1.16.2) [<xref ref-type="bibr" rid="CR30"><u>30</u></xref>] installed in a conda environment (<ext-link ext-link-type="uri" xlink:href="https://github.com/conda/conda">https://github.com/conda/conda</ext-link>, version 4.8.3).<li> This pipeline uses the Augur toolkit (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur">https://github.com/<b>nextstrain</b>/augur</ext-link>, version 7.0.2) [<xref ref-type="bibr" rid="CR31"><u>31</u></xref>] to filter, align/mask genomic sequences, build trees (divergence and time-scaled), and produce an output file processed by the Auspice web interface (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/auspice">https://github.com/<b>nextstrain</b>/auspice</ext-link>, version 2.16.0) [<xref ref-type="bibr" rid="CR30"><u>30</u></xref>] to explore phylodynamic and phylogenomic data.<li> <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> We defined Qu&#xE9;bec transmission lineages as descendants of a unique introduction event in the phylogeny, and then annotated these based upon Pangolin (PANGO) [<xref ref-type="bibr" rid="CR12"><u>12</u></xref>] and <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR30"><u>30</u></xref>] lineage nomenclatures.<li> Note that PANGO or <b>Nextstrain</b> are viral phylogenetic lineages used for taxonomic purposes, which are distinct from Qu&#xE9;bec transmission lineages, which we define at higher phylogenetic resolution as descendants of a single introduction event, and thus represent a partially observed transmission chain.<li> The heatmap shows <b>Nextstrain</b> clade designations for all sequences in the tree.<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [31](Huddleston J, Hadfield J, Sibley TR, Lee J, Fay K, Ilcisin M, et al. Augur: a bioinformatics toolkit for phylogenetic analyses of human pathogens. J Open Source Software. 2021;6(57) Available from: http://dx.doi.org/10.21105/joss.02906.) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34718705">34718705</a> <b>(1)</b> The UCSC Genome Browser database: 2022 update<ul><li> The web interface generates custom tracks for the uploaded variants and subtrees, downloadable summary files, and JSON files for display by <b>nextstrain</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34754251">34754251</a> <b>(3)</b> Fault Lines in India&#x2019;s COVID-19 Management: Lessons Learned and Future Recommendations<ul><li> The centralized genome databank needs to be integrated with International SARS-CoV2 gene sequencing consortiums such as GISAID (<underline><ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link></underline>), ViPER (<underline><ext-link xlink:href="https://www.viprbrc.org/brc/home.spg?decorator=vipr" ext-link-type="uri">https://www.viprbrc.org/brc/home.spg?decorator=vipr</ext-link></underline>), NIAID (<underline><ext-link xlink:href="https://www.niaid.nih.gov/research/systems-biology-consortium" ext-link-type="uri">https://www.niaid.nih.gov/research/systems-biology-consortium</ext-link></underline>), NCBI SARS-CoV2 Resource (<underline><ext-link xlink:href="https://www.ncbi.nlm.nih.gov/sars-cov-2" ext-link-type="uri">https://www.ncbi.nlm.nih.gov/sars-cov-2</ext-link></underline>), <b>NextStrain</b> (<underline><ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2</ext-link></underline>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34704780">34704780</a> <b>(6)</b> The Emergence of Sars-CoV-2 Variant Lambda (C.37) in South America<ul><li> By July 2021, 3,100 genome sequences from Peru were available on GISAID, comprising 60 circulating PANGO lineages (<ext-link xlink:href="https://<b>nextstrain</b>.org/community/quipupe/Peru_lambda" ext-link-type="uri">https://<b>nextstrain</b>.org/community/quipupe/Peru_lambda</ext-link>).<li> Complete information of other circulating PANGO lineages can be accessed from <ext-link xlink:href="https://<b>nextstrain</b>.org/community/quipupe/Peru_lambda" ext-link-type="uri">https://<b>nextstrain</b>.org/community/quipupe/Peru_lambda</ext-link>.<li> A list of authors and related metadata from all sequences included in our analyses can be downloaded from <ext-link xlink:href="https://<b>nextstrain</b>.org/community/quipupe/Peru_lambda" ext-link-type="uri">https://<b>nextstrain</b>.org/community/quipupe/Peru_lambda</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34735950">34735950</a> <b>(1)</b> Analysis of 329,942 SARS-CoV-2 records retrieved from GISAID database<ul><li> Based on the mutations, eight distinct viral clades had been reported by GISAID and twelve by <b>Nextstrain</b> by March 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34702903">34702903</a> <b>(1)</b> Data-driven analysis of amino acid change dynamics timely reveals SARS-CoV-2 variant emergence<ul><li> Pangolin<sup><xref ref-type="bibr" rid="CR13"><u>13</u></xref></sup>, <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR14"><u>14</u></xref></sup>, GISAID<sup><xref ref-type="bibr" rid="CR1"><u>1</u></xref></sup>).<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34702899">34702899</a> <b>(14)</b> Detailed phylogenetic analysis tracks transmission of distinct SARS-COV-2 variants from China and Europe to West Africa<ul><li> Graphics were generated using the <b>nextstrain</b> pipeline including software Augur (version 7.0.2, <ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/augur/en/stable/index.html">https://docs.<b>nextstrain</b>.org/projects/augur/en/stable/index.html</ext-link>) and TreeTime (version 0.7.6, <ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/treetime">https://github.com/neherlab/treetime</ext-link>)<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref>,<xref ref-type="bibr" rid="CR31"><u>31</u></xref></sup>.<li>The phylogenetic tree can be viewed interactively via the <b>nextstrain</b>.org framework under the URL: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/wwruck/wa">https://<b>nextstrain</b>.org/community/wwruck/wa</ext-link>.<li> The branch determined by the mutation G22486T (Supplementary Figure <xref rid="MOESM2" ref-type="media"><u>1</u></xref>) may reflect migration routes because in the <b>nextstrain</b> analysis of the entire Africa there are also samples from Mali and Tunisia in this branch (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/africa?f_region=Africa">https://<b>nextstrain</b>.org/ncov/africa?f_region=Africa</ext-link>, accessed August 14 2020).<li> G22486T may reflect migration routes because in the <b>nextstrain</b> analysis of Africa as a whole there are also Tunisian samples in this branch (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/africa?f_region=Africa">https://<b>nextstrain</b>.org/ncov/africa?f_region=Africa</ext-link>, accessed Jun 26th, 2020).<li> Graphics were generated using the software FigTree (version 1.4.4, <ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</ext-link>) for annotation of the phylogenetic tree and the <b>nextstrain</b> pipeline including software Augur (version 7.0.2, <ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/augur/en/stable/index.html">https://docs.<b>nextstrain</b>.org/projects/augur/en/stable/index.html</ext-link>) and TreeTime (version 0.7.6, <ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/treetime">https://github.com/neherlab/treetime</ext-link>)<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref>,<xref ref-type="bibr" rid="CR31"><u>31</u></xref></sup>.<li> Figure adapted from the website <ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org">www.<b>nextstrain</b>.org</ext-link> (accessed June 22, 2021, CC-BY<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref>,<xref ref-type="bibr" rid="CR31"><u>31</u></xref></sup>).<li>The phylogenetic tree was constructed using a pipeline adapted from the Zika virus pipeline on the <b>nextstrain</b>.org web page<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup> employing the Augur<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup>, the MAFFT<sup><xref ref-type="bibr" rid="CR29"><u>29</u></xref></sup> and the IQ-tree<sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup> software.<li> Connections between countries were based on the <b>nextstrain</b> Africa analysis and our own auspice analysis.<li>Variant distribution plots were downloaded from the website <b>nextstrain</b>.org<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref>,<xref ref-type="bibr" rid="CR31"><u>31</u></xref></sup> on June 22nd, 2021 filtering for Africa and the distinct countries Gambia, Ghana, Nigeria and Senegal.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [31](Sagulenko, P., Puller, V. &amp; Neher, R. A. TreeTime: Maximum-likelihood phylodynamic analysis. <italic>Virus Evol.</italic><bold>4</bold>(1). 10.1093/ve/vex042 (2018).) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [31](Sagulenko, P., Puller, V. &amp; Neher, R. A. TreeTime: Maximum-likelihood phylodynamic analysis. <italic>Virus Evol.</italic><bold>4</bold>(1). 10.1093/ve/vex042 (2018).) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [31](Sagulenko, P., Puller, V. &amp; Neher, R. A. TreeTime: Maximum-likelihood phylodynamic analysis. <italic>Virus Evol.</italic><bold>4</bold>(1). 10.1093/ve/vex042 (2018).) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [31](Sagulenko, P., Puller, V. &amp; Neher, R. A. TreeTime: Maximum-likelihood phylodynamic analysis. <italic>Virus Evol.</italic><bold>4</bold>(1). 10.1093/ve/vex042 (2018).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34702945">34702945</a> <b>(3)</b> Generator based approach to analyze mutations in genomic datasets<ul><li> We consider sequences in the <b>nextstrain</b><sup><xref ref-type="bibr" rid="CR6"><u>6</u></xref></sup> clades - 19A, 20A, 20B and 20C.<li> We apply the same technique for comparing 20<italic>A</italic> with other <b>nextstrain</b> clades&#x2014;20B (561 samples), 20C (232 samples) (see Fig.<li> For example, an accuracy of 1 in region 24 (23000&#x2013;24000) in (<bold>a</bold>) indicates the detection of spike protein mutation (A23403G or D614G) in <b>nextstrain</b> clade 20A when compared against 19A.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34704394">34704394</a> <b>(2)</b> Detection of SARS&#x2010;CoV&#x2010;2 in two cats during the second wave of the COVID&#x2010;19 pandemic in France<ul><li> This H57 mutation is widely distributed around the world and was present in about 70% of the sequences reported in France between October and December 2020 according to <b>Nextstrain</b> (https://<b>nextstrain</b>.org/).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34760721">34760721</a> <b>(5)</b> Host Response to SARS-CoV2 and Emerging Variants in Pre-Existing Liver and Gastrointestinal Diseases<ul><li> The <b>Nextstrain</b> clades naming is the third nomenclature system, which follows Year-Letter nomenclature to label clades that persist for at least several months and have significant geographic spread (20% frequency&#x2014;globally at any time point).<li> In the year 2020, Alpha variant (Pango Lin, B.1.1.7/GSIAID, GRY/<b>Nextstrain</b>, 20I clade) emerged from UK, Beta variant (Pango Lin, B.1.351/GSIAID, GH/501Y.V2/<b>Nextstrain</b>, 20H clade) emerged from South Africa, and Gamma variant (Pango Lin, P.1/GSIAID, GR/501Y.V3/<b>Nextstrain</b>, 20J clade) emerged from Brazil, whereas Delta variant (Pango Lin, B.1.617.2, AY.1/2/3/GSIAID, G/478K.V1/<b>Nextstrain</b>, 21A clade) recently emerged from India in the year 2021 (<xref rid="B121" ref-type="bibr"><u>WHO, 2021</u></xref>).<li> [WHO, 2021](<person-group person-group-type="author"><collab>WHO</collab></person-group> (<year>2021</year>) <source>Tracking SARS-CoV-2 Variants</source>. Available at: <uri xlink:href="https://wwwwhoint/en/activities/tracking-SARS-CoV-2-variants/">https://wwwwhoint/en/activities/tracking-SARS-CoV-2-variants/</uri>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34710479">34710479</a> <b>(2)</b> COVID-19: A review of newly formed viral clades, pathophysiology, therapeutic strategies and current vaccination tasks<ul><li> The clades such as 19A, 19B, 20A, 20B, and 20C are the currently named clades in this system (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming" id="ir0140">https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35058179">35058179</a> <b>(1)</b> Identification of potent small molecule inhibitors of SARS-CoV-2 entry<ul><li> Modified figure from https://<b>nextstrain</b>.</ul></p>
<p><u>No PMID</u> <b>(4)</b> SARS-CoV-2 variants, a still unfinished story<sup><xref ref-type="fn" rid="d36e814">&#x2606;</xref></sup><ul><li>Name <b>Nextstrain</b>.<li> form that usually refers to the country or place where the variant was detected (British strain), a code from the Pangolin database<xref rid="bib0020" ref-type="bibr"><sup>4</sup></xref> (B.1.1.7), and one based on the <b>Nextstrain</b> page<xref rid="bib0025" ref-type="bibr"><sup>5</sup></xref> (20I/501Y.V1).<li><b>Nextstrain</b> clade</th><th align="center">GISAID clade/lineage<xref rid="tblfn0015" ref-type="table-fn"><u>a</u></xref></th></tr></thead><tbody><tr><td align="left">Variants of concern.<li>More recently, there has been widespread alarm about the emergence and spread in India of the new variant B.1.617,<xref rid="bib0130" ref-type="bibr"><sup>26</sup></xref> also known as VUI (Variant Under Investigation)-21APR-01<xref rid="bib0135" ref-type="bibr"><u>27</u></xref>, <xref rid="bib0140" ref-type="bibr"><u>28</u></xref> and 20A according to &#x201C;<b>Nextstrain</b>&#x201D;.<li> [27](GOV.UK. Variants: distribution of cases data. 20 May 2021. Public Health England. [Accessed 24 May 2021]. <ext-link ext-link-type="uri" xlink:href="https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data#Variant11" id="intr0040">https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data#Variant11</ext-link>.) [28](          <person-group person-group-type="author">            <name>              <surname>Yadav</surname>              <given-names>P.D.</given-names>            </name>            <name>              <surname>Sapkal</surname>              <given-names>G.N.</given-names>            </name>            <name>              <surname>Abraham</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Ella</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Deshpande</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Patil</surname>              <given-names>D.Y.</given-names>            </name>            <etal/>          </person-group>          <article-title>Neutralization of variant under investigation B.1.617 with sera of BBV152 vaccinees</article-title>          <source>Clin Infect Dis.</source>          <year>2021</year>          <object-id pub-id-type="publisher-id">ciab411</object-id>          <pub-id pub-id-type="doi">10.1093/cid/ciab411</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34700068">34700068</a> <b>(4)</b> Dynamics of SARS-CoV-2 variants of concern (VOC) in Bangladesh during the first half of 2021<ul><li>The file of sequence data in FASTA format was then imported into the web-based software Nextclade v1.5.2 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0035">https://clades.<b>nextstrain</b>.org/</ext-link>) for quality assurance, clade assignment, lineage distribution, and diversity analysis.<li>Within the specified timeframe, genome sequences from Bangladesh revealed the presence of eight <b>Nextstrain</b> clades: 20A, 20B, 20C, 20H (Beta, V2), 20I (Alpha, V1), 20&#xA0;J (Gamma, V3), 21A (Delta), 21D (Eta), and six GISAID clades: G, GH, GR, GRY, GV, and O (<xref rid="fig1" ref-type="fig">Fig.<li> a) Phylogenetic clusters and clades as generated by <b>Nextstrain</b> for the dataset of 820 high-quality Bangladeshi SARS CoV-2 genomes in comparison with the 1794 global reference sequences.<li>Scatter plot showing evolutionary relationship and divergence limit based on <b>Nextstrain</b> phylogeny.<li>Many efforts have been made to share information about the SARS CoV-2 through open platforms such as GISAID EpiCoV (<xref rid="bib27" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>) and <b>Nextstrain</b> (<xref rid="bib13" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) in order to better understand the molecular epidemiology, classification, and evolution of the virus in order to control and manage the disease.<li> [Shu and McCauley, 2017](          <person-group person-group-type="author">            <name>              <surname>Shu</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>McCauley</surname>              <given-names>J.</given-names>            </name>          </person-group>          <article-title>GISAID: global initiative on sharing all influenza data - from vision to reality</article-title>          <source>Euro Surveill.</source>          <volume>22</volume>          <year>2017</year>          <object-id pub-id-type="publisher-id">30494</object-id>          <pub-id pub-id-type="doi">10.2807/1560-7917.es.2017.22.13.30494</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34697612">34697612</a> <b>(3)</b> Unlocking SARS-CoV-2 detection in low- and middle-income countries<ul><li> The latter is further evidenced by several remarkable open-access initiatives, like BEARmix (<xref rid="bib28" ref-type="bibr"><u>Graham et&#xA0;al., 2021</u></xref>), Addgene (<ext-link ext-link-type="uri" xlink:href="https://www.addgene.org/" id="intref0010">https://www.addgene.org/</ext-link>), BEI Resources (<ext-link ext-link-type="uri" xlink:href="https://www.beiresources.org/" id="intref0015">https://www.beiresources.org/</ext-link>), <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0020">https://<b>nextstrain</b>.org/</ext-link>), and GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="intref0025">https://www.gisaid.org/</ext-link>).<li> [Graham et&#xA0;al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33534838">33534838</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34672705">34672705</a> <b>(0)</b> Genome Sequences of SARS-CoV-2 Sublineage B.1.617.2 Strains from 12 Children in Chattogram, Bangladesh<ul><li>A phylogenetic tree was constructed on 3 August 2021 using the Nextclade v1.5.2 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) with default parameters (<xref rid="B3" ref-type="bibr"><u>3</u></xref>).<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34673905">34673905</a> <b>(1)</b> In-Depth Investigation of Conjunctival Swabs and Tear Fluid of Symptomatic COVID-19 Patients, an Observational Cohort Study<ul><li> Genome quality control was performed using Nextclade, phylogenetic analysis was carried out utilizing <b>Nextstrain</b> pipelines, phylogenetic tree visualization was generated in Auspice, and Pangolin (Phylogenetic Assignment of Named Global Outbreak Lineages) lineages were assigned using the Pangolin COVID-19 Lineage Assigner web application.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34688034">34688034</a> <b>(3)</b> Evidence for retained spike-binding and neutralizing activity against emerging SARS-CoV-2 variants in serum of COVID-19 mRNA vaccine recipients<ul><li> The time-calibrated tree was built with <b>Nextstrain</b> v7 for SARS-CoV-2 with default parameters (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="interref0001">https://github.com/<b>nextstrain</b>/ncov</ext-link>) <xref rid="bib0022" ref-type="bibr"><u>[22]</u></xref>.<li> [[22]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34379531">34379531</a> <b>(10)</b> Acquisition of the L452R Mutation in the ACE2-Binding Interface of Spike Protein Triggers Recent Massive Expansion of SARS-CoV-2 Variants<ul><li> This prompted us to perform an in-depth follow-up analysis of the L452R-carrying nCoV strains in our samples and a determination of their prevalence and clonal origin on a global scale using publicly available nCoV genomes and analytical tools of GISAID and <b>Nextstrain</b> databases.<li>The latest global analysis of SARS-CoV-2 genomes from the <b>Nextstrain</b> database (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global</ext-link>) was used to determine viral genomes with the presence of L452 mutations in the spike protein (as of 12 June 2021).<li> This expansion has been linked to a single viral variant from PANGO lineage B.1.427/B.1.429, the <b>Nextstrain</b> clade 21C, which was originally designated CAL.20C and recently designated VOI epsilon (used here).<li> A <b>Nextstrain</b>-generated phylogenetic tree of a subset of the lineages in which L452R occurs, based on genomes randomly sampled from GISAID by <b>Nextstrain</b> on the same day (i.e., 12 June 2021), is depicted in <xref rid="F3" ref-type="fig">Fig.<li><b>Nextstrain</b>-generated radial cladogram of SARS-CoV-2 genetic variants (as of 12 June 2021).<li> (At the date of analysis, CAL.20A strains were not among the sampled genomes in the <b>Nextstrain</b> database, and only three B.1.232 [all non-CAL.20A] lineages were found).<li> Nomenclature: PANGO lineage followed by <b>Nextstrain</b> clade in parenthesis and the current VOC/VOI designation.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34696534">34696534</a> <b>(0)</b> Near-Complete Genome of SARS-CoV-2 Delta (AY.3) Variant Identified in a Dog in Kansas, USA<ul><li>SARS-CoV-2 sequence lineages were determined in Pangolin (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>) and the Nextclade tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>; accessed 25 August 2021) assigned clades and were used to perform the phylogenetic analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34738021">34738021</a> <b>(3)</b> Evolution, Interspecies Transmission, and Zoonotic Significance of Animal Coronaviruses<ul><li> Many researchers use the Rob Lanfear's method (<xref rid="B449" ref-type="bibr"><u>449</u></xref>) and the maximum likelihood method with FastTree software (MicrobesOnline, Berkeley, CA, USA) to construct a phylogenetic tree and sequence alignment with the best setting determined by Global Initiative for Sharing All Influenza Data database (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>) and <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>) (<xref rid="B450" ref-type="bibr"><u>450</u></xref>).<li> [449](<person-group person-group-type="author"><name><surname>Lanfear</surname><given-names>R</given-names></name></person-group>. <article-title>A global phylogeny of SARS-CoV-2 sequences from GISAID</article-title>. <source>Zenodo</source>. (<year>2020</year>). <pub-id pub-id-type="doi">10.5281/zenodo.3958883</pub-id>) [450](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33335314">33335314</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34733906">34733906</a> <b>(2)</b> Emergence of a New Lineage 1C Variant of Porcine Reproductive and Respiratory Syndrome Virus 2 in the United States<ul><li> In order to evaluate if this genetic cluster represented the emergence of a new variant within its sub-lineage (1C), a phylogenetic tree with cases detected up to December 2020 was constructed using <b>NextStrain</b> (<xref rid="B12" ref-type="bibr"><u>12</u></xref>).<li> Additionally, whole genome sequence (WGS) was obtained for a subset of 16 cases with available diagnostic specimens at the UMN VDL and a phylogenetic tree was constructed using <b>NextStrain</b> comparing those with 365 North America GenBank publicly available PRRSV2 WGS.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34672129">34672129</a> <b>(1)</b> SARS-CoV-2 Variants of Concern<ul><li>The nomenclature systems for naming SARS-COV-2 lineage by GISAID, <b>Nextstrain</b>, and Pango are currently used by researchers.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34673335">34673335</a> <b>(3)</b> SARS-CoV-2: Current trends in emerging variants, pathogenesis, immune responses, potential therapeutic, and vaccine development strategies<ul><li> Before the naming of SARS-CoV-variants by the WHO, there were some databases and softwares by different nomenclature for real-time tracking of SARS-CoV-2 evolution and clades distribution by assessing the whole genomic sequences of virus in global such as <b>Nextstrain</b>, GISAID, PANGOLIN, and other websites such as CoVariants <xref rid="b0275" ref-type="bibr"><u>[55]</u></xref>, <xref rid="b0280" ref-type="bibr"><u>[56]</u></xref>, <xref rid="b0285" ref-type="bibr"><u>[57]</u></xref>, <xref rid="b0290" ref-type="bibr"><u>[58]</u></xref>.<li> According to these databases, <b>Nextstrain</b> had identified 11 major clades (19A, 19B, and 20A&#x2013;20I) as of January 2021 <xref rid="b0275" ref-type="bibr"><u>[55]</u></xref>, GISAID had identified eight global clades (S, O, L, V, G, GH, GR, and GV) <xref rid="b0280" ref-type="bibr"><u>[56]</u></xref>, and PANGOLIN had identified six major lineages (A, B, B.1, B.1.1, B.1.177, B.1.1.7) as of February 2021 <xref rid="b0285" ref-type="bibr"><u>[57]</u></xref>.<li><b>Nextstrain</b> clades</th><th rowspan="2">Notable and elevated risk variants</th><th rowspan="2">WHO label</th><th colspan="2">First detection.<li> [[55]](Nextstrain: Genomic Epidemiology of Novel Coronavirus - Global Subsampling 2020, 2021. https://nextstrain.org/sars-cov-2.) [[56]](GISAID- Global Initiative on Sharing All Influenza Data, Clade and Lineage Nomenclature Aids in Genomic Epidemiology Studies of Active hCoV-19 Viruses, 2021. https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/.) [[57]](pangolin: Software package for assigning SARS-CoV-2 genome sequences to global lineages, 2021. https://github.com/cov-lineages/pangolin.) [[58]](CoVariants. Enabled by data from GISAID, Powered by Nextstrain, 2021. https://covariants.org/.) [[55]](Nextstrain: Genomic Epidemiology of Novel Coronavirus - Global Subsampling 2020, 2021. https://nextstrain.org/sars-cov-2.) [[56]](GISAID- Global Initiative on Sharing All Influenza Data, Clade and Lineage Nomenclature Aids in Genomic Epidemiology Studies of Active hCoV-19 Viruses, 2021. https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/.) [[57]](pangolin: Software package for assigning SARS-CoV-2 genome sequences to global lineages, 2021. https://github.com/cov-lineages/pangolin.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34888394">34888394</a> <b>(1)</b> A Multidimensional Cross-Sectional Analysis of Coronavirus Disease 2019 Seroprevalence Among a Police Officer Cohort: The PoliCOV-19 Study<ul><li>The viral strain consisting of the mutation D614G in the S protein (<b>Nextstrain</b> clade 20A and its descendants) was the dominating circulating variant in 2020 in Switzerland [<xref rid="CIT0015" ref-type="bibr"><u>15</u></xref>].<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34098568">34098568</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34645288">34645288</a> <b>(1)</b> Rapid development of analytical methods for evaluating pandemic vaccines: a COVID-19 perspective<ul><li> Open-source initiatives for tracking variants and providing pathogen genome data, such as the National Center for Biotechnology Information (NCBI) SARS-CoV-2 Resources [<xref rid="B11" ref-type="bibr"><u>11</u></xref>], Protein Data Bank Japan (PDBJ) [<xref rid="B12" ref-type="bibr"><u>12</u></xref>] and <b>Nextstrain</b> [<xref rid="B13" ref-type="bibr"><u>13</u></xref>], have helped to ensure assays remain up to date and relevant.<li> [11](<article-title>NCBI SARS-CoV-2 Resources</article-title> (<year>2021</year>). <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/sars-cov-2/" ext-link-type="uri">https://www.ncbi.nlm.nih.gov/sars-cov-2/</ext-link>) [12](<article-title>Protein Data Bank Japan (PDBJ)</article-title> (<year>2021</year>). <ext-link xlink:href="https://pdbj.org/featured/covid-19" ext-link-type="uri">https://pdbj.org/featured/covid-19</ext-link>) [13](<article-title>Nextstrain</article-title> (<year>2021</year>). <ext-link xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34650153">34650153</a> <b>(3)</b> Surveillance of SARS-CoV-2 lineage B.1.1.7 in Slovakia using a novel, multiplexed RT-qPCR assay<ul><li>In Dec 2020, Rambaut et al.<sup><xref ref-type="bibr" rid="CR1"><u>1</u></xref></sup> reported the genomic characterization of a distinct phylogenetic cluster named lineage B.1.1.7 (also referred to as 20I/501Y.V1 by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>), Variant of Concern (VOC) 202012/01, or Alpha by WHO), briskly spreading over the past four weeks in the United Kingdom.<li> [1](Rambaut, A. <italic>et al. Preliminary Genomic Characterisation of an Emergent SARS-CoV-2 Lineage in the UK Defined by a Novel Set of Spike Mutations. Virological</italic> (2020). <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</ext-link>. Accessed 13 January 2021.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34696495">34696495</a> <b>(7)</b> High-Throughput Next-Generation Sequencing Respiratory Viral Panel: A Diagnostic and Epidemiologic Tool for SARS-CoV-2 and Other Viruses<ul><li>Inference and visualization of the phylogeny of the SARS-CoV-2 sequences were performed through the <b>Nextstrain</b> Command-Line Interface (CLI) tool, utilizing the associated augur and auspice toolkits [<xref rid="B1-viruses-13-02063" ref-type="bibr"><u>1</u></xref>].<li> The metadata for the final tree were integrated to allow for both temporal and geographical visualization in the accompanying <b>Nextstrain</b> instance.<li>The final instance has been posted through the <b>Nextstrain</b> community platform [<xref rid="B16-viruses-13-02063" ref-type="bibr"><u>16</u></xref>].<li> Using the open-source platform <b>Nextstrain</b>, we built an interactive visualization of the phylogenic analysis of our SARS-CoV-2 isolates.<li> Through our <b>Nextstrain</b> thread, it was observed that the clade for pangolin lineage B contains certain distant variants, including P4715L in ORF1ab, Q57H in ORF 3a, and S84L in ORF8 covarying with the D614G spike protein mutation, which were found to be the most prevalent in the early phase of the pandemic in the state of Georgia.<li> To this end, we built an interactive visualization of the phylogenetic analysis of our samples using the open-source platform <b>Nextstrain</b>, which could reflect the phylogeny and circulating diversity during the early SARS epidemic in Georgia.<li> Through our <b>Nextstrain</b> thread, we found that the clade for pangolin lineage B that contains certain distant variants covarying with the D614G spike protein mutation had become increasingly prevalent at the early phase of the pandemic in the state of Georgia.<li> [1](          <article-title>COVID-19 Dashboard by the Center for Systems Science and Engineering (CSSE) at Johns Hopkins University (JHU)</article-title>          <comment>Available online: <ext-link xlink:href="https://coronavirus.jhu.edu/map.html" ext-link-type="uri">https://coronavirus.jhu.edu/map.html</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-02">(accessed on 2 May 2021)</date-in-citation>        ) [16](          <article-title>Nextstrain Build for SARS-COV-2 in, GA</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/community/Bahl-Lab-IOB/SARS_CoV2_Augusta_Edu@main" ext-link-type="uri">https://nextstrain.org/community/Bahl-Lab-IOB/SARS_CoV2_Augusta_Edu@main</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-02">(accessed on 2 May 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34688376">34688376</a> <b>(6)</b> Prior infection with SARS-CoV-2 boosts and broadens Ad26.COV2.S immunogenicity in a variant-dependent manner<ul><li><hr/></td></tr><tr><td>Genome Detective 1.132</td><td>Genome Detective</td><td><ext-link ext-link-type="uri" xlink:href="https://www.genomedetective.com" id="intref0165">https://www.genomedetective.com</ext-link></td></tr><tr><td>Coronavirus Typing Tool</td><td><xref rid="bib8" ref-type="bibr"><u>Cleemput et&#xA0;al., 2020</u></xref></td><td>N/A</td></tr><tr><td>Geneious software</td><td>Biomatters Ltd</td><td>N/A</td></tr><tr><td><b>NextStrain</b>.<li><hr/></td></tr><tr><td>Genome Detective 1.132</td><td>Genome Detective</td><td><ext-link ext-link-type="uri" xlink:href="https://www.genomedetective.com" id="intref0165">https://www.genomedetective.com</ext-link></td></tr><tr><td>Coronavirus Typing Tool</td><td><xref rid="bib8" ref-type="bibr"><u>Cleemput et&#xA0;al., 2020</u></xref></td><td>N/A</td></tr><tr><td>Geneious software</td><td>Biomatters Ltd</td><td>N/A</td></tr><tr><td><b>NextStrain</b></td><td><xref rid="bib13" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0170">https://github.com/<b>nextstrain</b>/ncov</ext-link>.<li> Phylogenetic clade classification of the genomes in this study consisted of analyzing them against a global reference dataset using a custom pipeline based on a local version of <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0210">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="bib13" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> [Cleemput et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32108862">32108862</a>) [Cleemput et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32108862">32108862</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34721349">34721349</a> <b>(1)</b> Strategy and Performance Evaluation of Low-Frequency Variant Calling for SARS-CoV-2 Using Targeted Deep Illumina Sequencing<ul><li> Lineage B.1.1.7, termed Variant of Concern (VOC) 202012/01 by Public Health England (PHE) (<xref rid="B50" ref-type="bibr"><u>Public Health England, 2021</u></xref>), 20I/501Y.V1 by <b>Nextstrain</b> (<xref rid="B9" ref-type="bibr"><u>Centers for Disease Control and Prevention [CDC], 2021</u></xref>) and alpha variant by the World Health Organization (<xref rid="B66" ref-type="bibr"><u>WHO, 2021</u></xref>), was first reported in the United Kingdom but became the dominant strain in many European countries until the emergence of the delta variant since mid-April 2021 (<xref rid="B45" ref-type="bibr"><u>Mishra et al., 2021</u></xref>).<li> [Public Health England, 2021](<collab>Public Health England.</collab> (<year>2021</year>). <source><italic toggle="yes">Variants of concern or under investigation.</italic></source> Available online at: <ext-link xlink:href="https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data" ext-link-type="uri">https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data</ext-link> [accessed March 4, 2021]) [Centers for Disease Control and Prevention [CDC], 2021](<collab>Centers for Disease Control and Prevention [CDC].</collab> (<year>2021</year>). <source><italic toggle="yes">SARS-CoV-2 Variants.</italic></source> Available online at: <ext-link xlink:href="https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/variant-surveillance/variant-info.html" ext-link-type="uri">https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/variant-surveillance/variant-info.html</ext-link> [accessed March 4, 2021]) [WHO, 2021](<collab>WHO</collab> (<year>2021</year>). <source><italic toggle="yes">Tracking SARS-CoV-2 variants.</italic></source> Available online at: <ext-link xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" ext-link-type="uri">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link> [accessed June 23, 2021]) [Mishra et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34401689">34401689</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34609003">34609003</a> <b>(1)</b> The global epidemic of SARS&#x2010;CoV&#x2010;2 variants and their mutational immune escape<ul><li>The established nomenclature systems for naming and tracking SARS&#x2010;CoV&#x2010;2 genetic lineages by GISAID, <b>Nextstrain</b> and Pango are currently and will remain in use by scientists (<ext-link xlink:href="https://covlineages.org/resources/pangolin.html" ext-link-type="uri">https://covlineages.org/resources/pangolin.html</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34634927">34634927</a> <b>(2)</b> Two-Component Nanoparticle Vaccine Displaying Glycosylated Spike S1 Domain Induces Neutralizing Antibody Response against SARS-CoV-2 Variants<ul><li>The recent emergence of SARS-CoV-2 variants (<ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2</ext-link>) highlights the importance of a robust vaccine production platform with good scalability and modularity for rapid adaptation to novel variants.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34634797">34634797</a> <b>(3)</b> The Ensembl COVID-19 resource: ongoing integration of public SARS-CoV-2 data<ul><li>The first set of variants we integrated were from the <b>Nextstrain</b> project which creates phylogenetic trees for tracking pathogen evolution based on virus subsamples (<xref rid="B16" ref-type="bibr"><u>16</u></xref>).<li> We converted their SARS-CoV-2 data release from 08-04-2020 to VCF for integration into our system and display frequency distributions by country and <b>Nextstrain</b>-inferred clade.<li> Variants can be displayed as separate tracks in the genome browser: those from ENA, those from <b>Nextstrain</b> and those observed in more than one sample in either project as shown in Figure&#xA0;<xref rid="F3" ref-type="fig"><u>3</u></xref>.<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34634820">34634820</a> <b>(1)</b> SCovid: single-cell atlases for exposing molecular characteristics of COVID-19 across 10 human tissues<ul><li> GISAID (<xref rid="B18" ref-type="bibr"><u>18</u></xref>), <b>Nextstrain</b> (<xref rid="B19" ref-type="bibr"><u>19</u></xref>), GESS (<xref rid="B20" ref-type="bibr"><u>20</u></xref>) and European Nucleotide Archive (<xref rid="B21" ref-type="bibr"><u>21</u></xref>) collected SARS-CoV-2 strains of different patients all around the world and provided tools to analyse sequences.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33045727">33045727</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31722421">31722421</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34596904">34596904</a> <b>(0)</b> Temporal spread and evolution of SARS&#x2010;CoV&#x2010;2 in the second pandemic wave in Brazil<ul><li> We identified nucleotide and amino acids substitutions for P.1 and P.2 lineages using Nextclade v0.8.1 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and Pangolin v2 (<ext-link xlink:href="https://github.com/covlineages/pangolin" ext-link-type="uri">https://github.com/covlineages/pangolin</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35223081">35223081</a> <b>(2)</b> Spatial and temporal coevolution of N2 neuraminidase and H1 and H3 hemagglutinin genes of influenza A virus in US swine<ul><li> Each N2 genetic clade was visualized using a local <b>Nextstrain</b> (<xref rid="R24" ref-type="bibr">Hadfield et&#xA0;al.<li> Amino acid mutations were visualized across the phylogenetic backbone of each N2 genetic clade using <b>Nextstrain</b> (Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34627988">34627988</a> <b>(2)</b> A new SARS-CoV-2 variant with high lethality poorly detected by RT-PCR on nasopharyngeal samples: an observational study<ul><li> Whole-genome sequencing (WGS) from lower respiratory tract samples of several cases identified a previously unknown variant of SARS-CoV-2 belonging to clade GH/20C (GISAID/<b>Nextstrain</b> nomenclatures) carrying several amino acid substitutions or deletions in the spike (S) protein, which received the B.1.616 Pango lineage designation.<li> (A) Subsampled global phylogenetic maximum likelihood tree of SARS-CoV-2 with annotated <b>Nextstrain</b> clades next to the corresponding nodes and tips highlighted only for sequences from the Pangolin B.1.616 lineage.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34642698">34642698</a> <b>(1)</b> Comparative transmissibility of SARS-CoV-2 variants Delta and Alpha in New England, USA<ul><li> Consensus sequences were uploaded to <b>Nextstrain</b> and Pango for quality control, strain and clade identification.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34642698">34642698</a> <b>(1)</b> Comparative transmissibility of SARS-CoV-2 variants Delta and Alpha in New England, USA<ul><li> Consensus genomes obtained from these processes were compared using the <b>Nextstrain</b> clade tool (<xref rid="B7" ref-type="bibr"><u>7</u></xref>), with which the proper clade was identified.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34637888">34637888</a> <b>(4)</b> The mutational landscape of SARS-CoV-2 variants diversifies T&#xA0;cell targets in an HLA-supertype-dependent manner<ul><li> (<xref rid="bib26" ref-type="bibr"><u>Huddleston et&#xA0;al., 2020</u></xref>) who used SANTA-SIM to simulate influenza A/H3N2 that has a yearly substitution rate approximately twice as high as SARS-CoV-2 [&#x223C;48,824 substitutions/year (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/flu/seasonal/h3n2/ha/2y?l=clock" id="intref0105">https://<b>nextstrain</b>.org/flu/seasonal/h3n2/ha/2y?l=clock</ext-link>) vs.<li> &#x223C;24.5 substitution/year (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?l=clock" id="intref0110">https://<b>nextstrain</b>.org/ncov/global?l=clock</ext-link>)], we chose 400 generations/year, with the mutation rate per position per generation set to 2.04E-6 (yearly substitution rate/(generations in one year <sup>&#x2217;</sup> genome size)).<li> [Huddleston et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32876050">32876050</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34607353">34607353</a> <b>(1)</b> A multi-step and multi-scale bioinformatic protocol to investigate potential SARS-CoV-2 vaccine targets<ul><li> This sequence is broadly used as a standard reference and, because of its early identification, is used as the origin genome in phylogenetic trees produced by <b>Nextstrain</b>, COVID-19 Genomics UK (COG-UK) and the China National Center for Bioinformation project.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34642697">34642697</a> <b>(2)</b> Year-long COVID-19 infection reveals within-host evolution of SARS-CoV-2 in a patient with B cell depletion<ul><li>Phylogenetic trees were generated using the <b>Nextstrain</b> software package and default parameters [<xref rid="R18" ref-type="bibr"><u>18</u></xref>].<li> Consensus sequences were mapped onto a phylogenetic tree containing 266 background samples from Maryland from the time of the first to the second presentation using the publicly available <b>Nextstrain</b> software package [<xref rid="R18" ref-type="bibr"><u>18</u></xref>].<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34630935">34630935</a> <b>(3)</b> N-terminal domain mutations of the spike protein are structurally implicated in epitope recognition in emerging SARS-CoV-2 strains<ul><li> <b>Nextstrain</b> and WHO nomenclature regarding the major SARS-CoV-2 clades was applied to our analysis for specific variants <xref rid="b0115" ref-type="bibr"><u>[23]</u></xref>, <xref rid="b0120" ref-type="bibr"><u>[24]</u></xref>.<li>As of June 2021, <b>Nextstrain</b> has identified 13 major clades (19A&#x2013;B, 20A&#x2013;20J and 21A).<li>In order to identify whether SARS-CoV-2 strains harboring mutations in NTD loops are associated with greater prevalence, we investigated <b>Nextstrain</b> data with a focus on WHO reported VOCs (Alpha, Beta, Gamma, Delta), VOIs (Eta, Iota, Kappa and Lambda) and strains under monitoring (B.1.1.318, B1.1.375 and B.1.1.616) that are characterized by high transmissibility.<li> [[23]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33761326">33761326</a>) [[24]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32698192">32698192</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34678659">34678659</a> <b>(1)</b> Intrathecal inflammatory responses in the absence of SARS-CoV-2 nucleic acid in the CSF of COVID-19 hospitalized patients<ul><li> A maximum likelihood phylogenetic tree of 772 previously reported SARS-CoV-2 genomes [<xref rid="bb0110" ref-type="bibr"><u>22</u></xref>] was downloaded from <b>NextStrain</b> [<xref rid="bb0115" ref-type="bibr"><u>23</u></xref>,<xref rid="bb0120" ref-type="bibr"><u>24</u></xref>] and annotated in FigTree (v1.4.4).<li> [22](          <person-group person-group-type="author">            <name>              <surname>Lemieux</surname>              <given-names>J.E.</given-names>            </name>            <name>              <surname>Siddle</surname>              <given-names>K.J.</given-names>            </name>            <name>              <surname>Shaw</surname>              <given-names>B.M.</given-names>            </name>            <etal/>          </person-group>          <article-title>Phylogenetic analysis of SARS-CoV-2 in Boston highlights the impact of superspreading events</article-title>          <source>Science [online serial]</source>          <year>2021</year>          <fpage>371</fpage>          <comment>Accessed at</comment>          <pub-id pub-id-type="doi">10.1126/science.abe3261</pub-id>        ) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [24](          <person-group person-group-type="author">            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Puller</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>TreeTime: maximum-likelihood phylodynamic analysis</article-title>          <comment>[online]</comment>          <source>Virus Evol.</source>          <year>2018</year>          <pub-id pub-id-type="doi">10.1093/ve/vex042</pub-id>          <comment>Accessed at</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34593004">34593004</a> <b>(0)</b> Transmission event of SARS-CoV-2 delta variant reveals multiple vaccine breakthrough infections<ul><li> Swift analysis pipeline produced variant vcf file, consensus genome, pangolin lineage, and Nextclade assignment (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34314668">34314668</a> <b>(4)</b> Emergence of SARS-COV-2 Spike Protein Escape Mutation Q493R after Treatment for COVID-19<ul><li> We used the Sanger method to sequence the spike gene as reported (<xref rid="R6" ref-type="bibr"><italic>6</italic></xref>), analyzed sequences by using <b>NextStrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>), and deposited sequences in GenBank.<li> <b>NextStrain</b> clade 20I/501Y.V1; GenBank accession no.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34599175">34599175</a> <b>(3)</b> Emergence and spread of SARS-CoV-2 lineage B.1.620 with variant of concern-like mutations and deletions<ul><li> This core set was subsequently combined with 250 randomly selected sequences from the <b>Nextstrain</b> global analysis on April 29, 2021 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link><sup><xref ref-type="bibr" rid="CR34"><u>34</u></xref></sup>) to provide context for the B.1.620 analysis, plus an additional two reference sequences: Wuhan/Hu-1/2019 and Wuhan/WH01/2019.<li> [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34696408">34696408</a> <b>(4)</b> Real-Time Genomic Surveillance during the 2021 Re-Emergence of the Yellow Fever Virus in Rio Grande do Sul State, Brazil<ul><li> YFV-subsets combined with the newly determined genomes were used to construct a time-scaled tree in <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</uri>, accessed on 23 August 2021).<li>To examine the spatial and evolutionary dynamics of the dispersion of YFV in the epizootic reported here, we made an analysis using a subset of sequences belonging to the South America 1 genotype on <b>Nextstrain</b>.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34596015">34596015</a> <b>(2)</b> Nosocomial outbreak caused by the SARS-CoV-2 Delta variant in a highly vaccinated population, Israel, July 2021<ul><li> Phylogenetic trees were constructed using <b>NextStrain</b>&#x2019;s Augur pipeline and visualised with auspice [<xref rid="r8" ref-type="bibr"><u>8</u></xref>].<li>The tree was constructed using <b>Nextstrain</b>&#x2019;s Augur pipeline and visualised with Auspice [<xref rid="r8" ref-type="bibr"><u>8</u></xref>].<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34594042">34594042</a> <b>(4)</b> The first wave of the COVID-19 epidemic in Spain was associated with early introductions and fast spread of a dominating genetic variant<ul><li> In contrast to clades from other European countries, these SECs belong to early lineages in the epidemic (A in Pango, 19B in <b>NextStrain</b>).<li><bold>a</bold>, ML phylogenetic tree of Spanish sequences indicating the identified SECs (inner circle), the Pango lineage (middle circle) and the <b>NextStrain</b> clade (outer circle).<li> <b>NextStrain</b> tracking tools for viral spatial spread suggest additional SEC8-related early seedings in Madrid, Pa&#xED;s Vasco, Andaluc&#xED;a and La Rioja regions (Supplementary Data <xref rid="MOESM1" ref-type="media"><u>2</u></xref>), which might have involved other countries, not exclusively Italy.<li>Video showing the transmission dynamics of SEC8 within Spain from 2020-01-16 to 2020-07-19, obtained from <b>NextStrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34585731">34585731</a> <b>(5)</b> COVIDium: a COVID-19 resource compendium<ul><li> LitCovid (<xref rid="R14" ref-type="bibr"><u>14</u></xref>) allows users to access numerous curated published peer-reviewedliterature on COVID-19, whereas <b>Nextstrain</b> COVID-19 genetic epidemiology (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) is an open-source platform providingSARS-CoV-2 genome data for different analyses.<li>GISAID, GESS, Pangolin COVID-19 Lineage Assigner, VADR&#x2014;ViralAnnotation DefineR, SARS-CoV-2 alignment screen, idCOV, MCCS, UCSC SARS-CoV-2genome browser, GENOME DETECTIVE VIRUS TOOL, 2019nCoVR, Phyloscanner, CoV2ID,GLEAMviz, COVID-19 data portal, NCBI nucleotide sequences, SARS-CoV-2 data hub,ViPR SARS-CoV-2, VirHostNet, WOLFRAM, VAPOR, Ensembl COVID-19 resource, VBRCgenome analysis tools, <b>nextstrain</b>, covidex, Haplotype Explorer, COVID-Align,CorGAT, CoV-Seq, SARS-COV-2 genome browser, COVID-19 Viral Genome AnalysisPipeline, VirusSeeker, COVIDOUTCOME&#x2014;Estimating COVID Severity Based on Mutationsin the SARS-CoV-2 Genome, COVIDier, CovRadar, CoVPipe, SNPnexus COVID, METATRYPStandalone Software.<li>BEAST 2, Phylogeographic reconstruction, Hypothesis Testing usingPhylogenies (HyPhy), COPASI, MapMyCorona, <b>Nextstrain</b>, covidex, pangolin COVID-19lineage assigner, 2019nCoVR, Phyloscanner, SARS-CoV-2 alignment screen, 2019nCoVR,CoV-GLUE, coronavirus typing tool, phylomeDB coronavirus phylomes, SARS-CoV-2analysis workflow, Haploflow, PoSeiDon.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33166392">33166392</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34588460">34588460</a> <b>(3)</b> Genomic sequencing of SARS-CoV-2 in Rwanda reveals the importance of incoming travelers on lineage diversity<ul><li>We downloaded all SARS-CoV-2 genomes from the available <b>nextstrain</b> build<sup><xref ref-type="bibr" rid="CR23"><u>23</u></xref></sup> with Africa-focused subsampling (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/africa">https://<b>nextstrain</b>.org/ncov/africa</ext-link>) on February 23, 2021.<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Genomic sequencing of SARS-CoV-2 in Rwanda reveals the importance of incoming travelers on lineage diversity<ul><li> Here, B 1.1.25 variant was the most predominant in the first wave but in the ending phase of the first wave presence of the Alpha variant was observed in the <b>Nextstrain</b> phylogenetic tree (<bold><xref rid="pone.0258019.s011" ref-type="supplementary-material"><u>S1 Fig</u></xref></bold>).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34679489">34679489</a> <b>(1)</b> Fast SARS-CoV-2 Variant Detection Using Snapback Primer High-Resolution Melting<ul><li> Forward and reverse primers were designed to bind in a region with lower mutational rate as observed by entropy using <b>NextStrain</b> analysis [<xref rid="B33-diagnostics-11-01788" ref-type="bibr"><u>33</u></xref>].<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34581669">34581669</a> <b>(1)</b> Tiled-ClickSeq for targeted sequencing of complete coronavirus genomes with simultaneous capture of RNA recombination and minority variants<ul><li> Phylogenetic tree reconstruction using <b>NextStrain</b> (<xref rid="bib13" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) placed 10 of the isolates in the A2a clade (<xref rid="fig3" ref-type="fig"><u>Figure 3D</u></xref>) and two isolates (WRCEVA_00508, WRCEVA_00513) were Clade B/B1.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34592415">34592415</a> <b>(4)</b> Investigating the first stage of the COVID-19 pandemic in Ukraine using epidemiological and genomic data<ul><li>For the first stage of the analysis, we ustilized a SARS-COV-2-specific pipeline implemented by <b>Nextstrain</b> (<xref rid="bib0210" ref-type="bibr"><u>github.com, 2021</u></xref>).<li> These genomes were complemented by a representative subsample of 6479 sequences from other geographic regions obtained using a country-specific subsampling algorithm of <b>Nextstrain</b>.<li> of <b>nextstrain</b> (<xref rid="bib0225" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> Similarly, by <b>Nextstrain</b> classification the distribution of lineages is: 19A - 1.7%, 20A - 51.7% and 20B - 46.7% (<xref rid="fig0010" ref-type="fig">Fig.<li> [github.com, 2021](          <person-group person-group-type="author">            <name>              <surname>github.com</surname>            </name>          </person-group>          <part-title>https://github.com/nextstrain/ncov, 2021. Accessed: 2020-11-01.</part-title>          <year>2021</year>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34631911">34631911</a> <b>(20)</b> High throughput nanopore sequencing of SARS-CoV-2 viral genomes from patient samples<ul><li> state of Illinois, which are available on the GISAID (Global Initiative on Sharing All Influenza Data) database and publicly viewable on <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/illinois-gagnon-public/ncov/gagnon">https://<b>nextstrain</b>.org/groups/illinois-gagnon-public/ncov/gagnon</ext-link>).<li> Phylogenetic analyses can be performed using <b>Nextstrain</b> [<xref rid="ref15" ref-type="bibr"><u>15</u></xref>] and results visualized on <b>nextstrain</b>.org.<li> A convenient way to observe sequences and determine the quality is to use the NextClade website (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> each sequence corresponds to, which are the phylogenetic categories the <b>Nextstrain</b> pipeline separates sequences into.<li>Finally, if the sequences are going to be placed into a phylogenetic context using the <b>Nextstrain</b> pipeline (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>), they must also meet requirements there as well.<li> <b>Nextstrain</b> rejects any sample that lacks metadata (for example time and location of collection, length of sequence) or is too ambiguous.<li> <b>Nextstrain</b> also estimates the mutation rate of SARS-CoV-2.<li>The <b>Nextstrain</b> software can be installed by following their SARS-CoV-2 analysis tutorial (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.github.io/ncov/">https://<b>nextstrain</b>.github.io/ncov/</ext-link>).<li> Briefly, after the software is installed, all sequence FASTA files must be added in bulk to a sequences.fasta file in the <b>Nextstrain</b> data directory, and the accompanying metadata for those sequences must be added to the metadata.tsv file in the same location.<li> More permanent hosting for these analyses can be setup <italic>via</italic> the <b>Nextstrain</b> website (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/en/latest/guides/share/<b>nextstrain</b>-groups.html">https://docs.<b>nextstrain</b>.org/en/latest/guides/share/<b>nextstrain</b>-groups.html</ext-link>).<li> An example of our laboratory&#x2019;s <b>Nextstrain</b> build with genome sequences from Illinois is shown in <bold><xref ref-type="fig" rid="fig004"><u>Figure 4</u></xref>.<li><bold>Working Protocol S13.</bold> Visualizing data in Nextclade and <b>Nextstrain</b>.<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34680431">34680431</a> <b>(0)</b> OmniSARS2: A Highly Sensitive and Specific RT-qPCR-Based COVID-19 Diagnostic Method Designed to Withstand SARS-CoV-2 Lineage Evolution<ul><li> We then assigned them to a SARS-CoV-2 clade with Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri> (accessed on 6 July 2021)) and to a SARS-CoV-2 PANGO lineage [<xref rid="B5-biomedicines-09-01314" ref-type="bibr"><u>5</u></xref>] with Pangolin [<xref rid="B18-biomedicines-09-01314" ref-type="bibr"><u>18</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34607084">34607084</a> <b>(2)</b> High-throughput sequencing of SARS-CoV-2 in wastewater provides insights into circulating variants<ul><li> Our SNV analysis shows that there is no particular region of the genome that is a SNV hotspot and this mirrors what is observed in clinical-derived samples (see updates on SNVs analysis of clinical data at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="PC_link4PM3u9neaR">https://<b>nextstrain</b>.org/ncov/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34571275">34571275</a> <b>(5)</b> Clinical outcomes in patients infected with different SARS-CoV-2 variants at one hospital during three phases of the COVID-19 epidemic in Marseille, France<ul><li> In addition, both variants have spread to several countries located in six WHO regions (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="ir0010">https://<b>nextstrain</b>.org/ncov/global</ext-link>; <xref rid="bb0145" ref-type="bibr"><u>PAHO, 2021</u></xref>).<li> The phylogenetic analysis was performed using the <b>nextstrain</b>/ncov tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="ir0035">https://github.com/<b>nextstrain</b>/ncov</ext-link>) that uses the IQ-TREE software (<xref rid="bb0140" ref-type="bibr"><u>Minh et al., 2020</u></xref>) for phylogenetic tree building then the Auspice software for tree visualisation (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/latest/releases/v2.html" id="ir0040">https://docs.<b>nextstrain</b>.org/projects/auspice/en/latest/releases/v2.html</ext-link>).<li> [PAHO, 2021](          <person-group person-group-type="author">            <name>              <surname>PAHO</surname>            </name>          </person-group>          <article-title>Epidemiological Update: Variants of SARS-CoV-2 in the Americas - 24 March 2021 - PAHO/WHO | Pan American Health Organization</article-title>          <ext-link ext-link-type="uri" xlink:href="https://www.paho.org/en/documents/epidemiological-update-variants-sars-cov-2-americas-24-march-2021" id="ir0100">https://www.paho.org/en/documents/epidemiological-update-variants-sars-cov-2-americas-24-march-2021</ext-link>          <year>2021</year>          <comment>(accessed May 11, 2021)</comment>        ) [Minh et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34524690">34524690</a> <b>(6)</b> Assessment of two&#x2010;pool multiplex long&#x2010;amplicon nanopore sequencing of SARS&#x2010;CoV&#x2010;2<ul><li> Local <b>Nextstrain</b>/Nextclade v0.13.0 was also implemented for clade assignment and variant annotation.<li> The phylogenetic tree was generated with <b>Nextstrain</b>/Augur using its default subsampling scheme and focusing on country Ethiopia, region Africa, where 1,960 samples were subsampled between December 2019 and February 2021; the tree was visualized using the <b>Nextstrain</b>/Auspice tool.<li> According to <b>Nextstrain</b> (<xref rid="B10" ref-type="bibr"><u>10</u></xref>), the phylogenetic tree revealed that all of the isolates belonged to <b>Nextstrain</b> clade 20A and GISAID clade GH.<li> The figure was created using <b>Nextstrain</b>.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34659798">34659798</a> <b>(10)</b> The impact of global lineage dynamics, border restrictions, and emergence of the B.1.1.7 lineage on the SARS-CoV-2 epidemic in Norway<ul><li> In the case of SARS-CoV-2, both GISAID and <b>NextStrain</b> (<xref rid="R6" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) have become essential platforms for tracking the dispersal of viral variants and mutations globally.<li> On the <b>NextStrain</b> platform, there are multiple schemes for defining clades and lineages.<li> The B.1.1.7 sweep was also clearly manifested in the Simpson&#x2019;s diversity (<xref rid="R24" ref-type="bibr"><u>Simpson 1949</u></xref>) estimated for the same periods, both at the level of <b>NextStrain</b> clades and Pangolin lineages (<xref rid="F2" ref-type="fig">Fig.<li> The dot plots on the right illustrate the corresponding Simpson diversity estimates at the level of <b>NextStrain</b> clades and Pangolin lineages.<li> (A) Dated phylogeny generated using the <b>NextStrain</b> pipeline used to infer imports to and local transmission of SARS-CoV-2 in Norway.<li> The wealth of sequence data shared and analysed via GISAID (<xref rid="R4" ref-type="bibr"><u>Elbe and Buckland-Merrett 2017</u></xref>) and <b>NextStrain</b> (<xref rid="R6" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) puts us in a unique position to understand the dynamics of the pandemic in response to both intrinsic viral characteristics, such as the evolution of increased transmissibility, as well as infection control measures, such as travel restrictions and other NPIs.<li>The international dataset was created using the <b>Nextstrain</b> pipeline (<xref rid="R6" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>), client version 3.0.3.<li> Briefly, we used pre-selected Norwegian isolates as described above and set the <b>Nextstrain</b> workflow to sample up to 2,500 contextual samples based on genetic proximity from the rest of the world.<li> The time-resolved phylogeny was generated with the <b>Nextstrain</b> toolkit &#x2018;augur refine&#x2019; command (<xref rid="R9" ref-type="bibr"><u>Huddleston et&#xA0;al., 2021</u></xref>), which uses the program TreeTime (<xref rid="R22" ref-type="bibr"><u>Sagulenko, Puller, and Neher 2018</u></xref>).<li>We used the phylogeny from the <b>Nextstrain</b> build as input for phylogeographic analyses.<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Simpson 1949](<person-group person-group-type="author"><string-name><surname>Simpson</surname>&#xA0;<given-names>E. H.</given-names></string-name></person-group> (<year>1949</year>) &#x2018;<article-title>Measurement of Diversity</article-title>&#x2019;, <source><italic toggle="yes">Nature</italic></source>, <volume>163</volume>: <page-range>688</page-range>.) [Elbe and Buckland-Merrett 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Huddleston et&#xA0;al., 2021](<person-group person-group-type="author"><string-name><surname>Huddleston</surname>&#xA0;<given-names>John</given-names></string-name></person-group>&#xA0;<etal>et&#xA0;al.</etal> (<year>2021</year>) &#x2018;<article-title>Augur: A Bioinformatics Toolkit for Phylogenetic Analyses of Human Pathogens</article-title>&#x2019;, <source><italic toggle="yes">Journal of Open Source Software</italic></source>, <volume>6</volume>: 2906.doi: <pub-id pub-id-type="doi">10.21105/joss.02906</pub-id>.) [Sagulenko, Puller, and Neher 2018](<person-group person-group-type="author"><string-name><surname>Sagulenko</surname>&#xA0;<given-names>Pavel</given-names></string-name>, <string-name><given-names>Vadim</given-names><surname>Puller</surname></string-name>, and <string-name><given-names>Richard A.</given-names><surname>Neher</surname></string-name></person-group> (<year>2018</year>) &#x2018;<article-title>TreeTime: Maximum-Likelihood Phylodynamic Analysis</article-title>&#x2019;, <source><italic toggle="yes">Virus Evolution</italic></source>, <volume>4</volume>: <page-range>vex042</page-range>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34696340">34696340</a> <b>(3)</b> Molecular Characterization of Imported and Autochthonous Dengue in Northeastern Spain<ul><li> Available sequences for each DENV serotype and genotype were selected in <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 1 April 2021), Virus Pathogen Resources (VIPR, <uri xlink:href="https://www.viprbrc.org/brc/home.spg?decorator=vipr">https://www.viprbrc.org/brc/home.spg?decorator=vipr</uri>, accessed on 1 June 2021) and GenBank (<uri xlink:href="https://www.ncbi.nlm.nih.gov/genbank/">https://www.ncbi.nlm.nih.gov/genbank/</uri>, accessed on 1 June 2021) for phylogenetic analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34696342">34696342</a> <b>(1)</b> The Immune Response to SARS-CoV-2 and Variants of Concern<ul><li>Since the emergence of SARS-CoV-2 in December 2019 till July 2021, 3945 variants have been reported in <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34619611">34619611</a> <b>(3)</b> Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy<ul><li> We then confirmed and visualized these findings by generating radial plots from subsampled GISAID sequences using <b>nextstrain</b>.org (<xref rid="bib0011" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>).<li> Radial plots were generated on <b>nextstrain</b>.org (<xref rid="bib0011" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>).<li> Radial plots were generated on <b>nextstrain</b>.org (<xref rid="bib0011" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>) with sequences subsampled global sequencing data subsampled from GISAID (<xref rid="bib0008" ref-type="bibr"><u>Elbe&#xA0;and Buckland-Merrett,&#xA0;2017</u></xref>).<li> [Hadfield&#xA0;et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield&#xA0;et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield&#xA0;et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Elbe&#xA0;and Buckland-Merrett,&#xA0;2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35602196">35602196</a> <b>(2)</b> Reduced subgenomic RNA expression is a molecular indicator of asymptomatic SARS-CoV-2 infection<ul><li>To understand the pathophysiology of COVID-19 infection, a major effort has been made on fully decoding the SARS-CoV-2 genome and its genetic variation, specifically the single nucleotide variants (SNVs)<sup><xref ref-type="bibr" rid="CR12"><u>12</u></xref>,<xref ref-type="bibr" rid="CR13"><u>13</u></xref></sup> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>).<li> [12](Alm, E. et al. Geographical and temporal distribution of SARS-CoV-2 clades in the WHO European Region, January to June 2020. <italic>Euro Surveill</italic>10.2807/1560-7917.ES.2020.25.32.2001410 (2020).) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34696160">34696160</a> <b>(1)</b> Shooting at a Moving Target&#x2014;Effectiveness and Emerging Challenges for SARS-CoV-2 Vaccine Development<ul><li>The first generation of vaccine prototypes against SARS-CoV-2 are based on the S protein amino acid sequence of the early Wuhan-Hu-1 (<b>Nextstrain</b>: clade A) isolate.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34560289">34560289</a> <b>(1)</b> SARS-CoV-2 B.1.1.7 lineage rapidly spreads and replaces R.1 lineage in Japan: Serial and stationary observation in a community<ul><li>The viral clade and lineage classifications were conducted by using <b>Nextstrain</b> (<xref rid="bb0040" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), and Phylogenetic Assignment of Named Global Outbreak (PANGO) Lineages (<xref rid="bb0135" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34560183">34560183</a> <b>(4)</b> Does immune recognition of SARS-CoV2 epitopes vary between different ethnic groups?<ul><li><b>Nextstrain</b>/augur pipeline (accessed on 21st April 2020 from <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="interref0010">https://github.com/<b>nextstrain</b>/ncov</ext-link>) was used to align 40,342 SARS-CoV2 genome sequences downloaded from GISAID using Wuhan-Hu-1/2019, as a reference sequence using MAFFT (<xref rid="bib0028" ref-type="bibr"><u>Katoh&#xA0;and Standley,&#xA0;2013</u></xref>).<li> Conforming to the <b>Nextstrain</b> protocol, 130 nucleotides from 5&#x2032;-end and 50 nucleotides from the 3&#x2032;-end as well as single nucleotide positions 18,529, 29,849, 29,851, 29,853 were masked from multiple sequence alignment (MSA).<li> [Katoh&#xA0;and Standley,&#xA0;2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34544043">34544043</a> <b>(2)</b> Case Report: Paucisymptomatic College-Age Population as a Reservoir for Potentially Neutralization-Resistant Severe Acute Respiratory Syndrome Coronavirus 2 Variants<ul><li> (<bold>A</bold>) Clade assignment and visual generated using the <b>Nextstrain</b> tool (clades.<b>nextstrain</b>.org), with distance from the center based on the number of mutations relative to the SARS-CoV-2/Wuhan-Hu-1 strain, accession number MN908947.<li> New sequences reported in this study are shown as black dots, with variants designated BV-1, BV-2, and BV-3 indicated, with 4,000 other sequences from the <b>Nextstrain</b> globally subsampled daily dataset, accessed May 10, 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34544043">34544043</a> <b>(1)</b> Case Report: Paucisymptomatic College-Age Population as a Reservoir for Potentially Neutralization-Resistant Severe Acute Respiratory Syndrome Coronavirus 2 Variants<ul><li> A diagnostic of consensus sequences was performed with the <b>Nextstrain</b> workflow version 2.0.0.post1 (<xref rid="B18" ref-type="bibr"><u>18</u></xref>).<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=14783086">14783086</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34506640">34506640</a> <b>(1)</b> Molecular evolutionary characteristics of SARS&#x2010;CoV&#x2010;2 emerging in the United States<ul><li> The <b>Nextstrain</b> website estimates that the annual nucleotide evolution rate is 8&#x2009;&#xD7;&#x2009;10<sup>&#x2212;4</sup> substitution per nucleotide per year based on current statistics.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34547443">34547443</a> <b>(13)</b> Characterisation of SARS-CoV-2 clades based on signature SNPs unveils continuous evolution<ul><li> To achieve this, multiple alignment using fast fourier transform (MAFFT) <xref rid="b0110" ref-type="bibr"><u>[22]</u></xref> is used for multiple sequence alignment (MSA) followed by <b>Nextstrain</b> <xref rid="b0115" ref-type="bibr"><u>[23]</u></xref> for performing phylogenetic analysis to identify virus clades.<li> Initially, multiple sequence alignment of Datasets A and B are carried out using MAFFT followed by phylogenetic analyses using <b>Nextstrain</b> which resulted in the identification of virus clades.<li>On the other hand, <b>Nextstrain</b> is a collection of open-source tools which is useful for understanding the evolution and spread of pathogen, particularly during an outbreak.<li> Using <b>Nextstrain</b>, proper and meaningful visualisation of a large number of virus sequences can be achieved.<li> There are a substantial number of tools in <b>Nextstrain</b> which perform phylodynamic analysis <xref rid="b0120" ref-type="bibr"><u>[24]</u></xref> which ranges from subsampling, alignment, phylogenetic inference to temporal dating of ancestral nodes and discrete trait geographic reconstruction and inference of the most likely transmission events.<li> The spread and evolution of virus genomes can be visualised at <b>nextstrain</b>.org using auspice.<li>Once the virus clades are identified using <b>Nextstrain</b>, clade specific aligned sequences are used to identify the mutation points as substitutions specifically SNPs in each clade.<li> In this regard, initially multiple sequence alignment of Dataset A with 15359 and Dataset B with 3033 SARS-CoV-2 genomic sequences are carried out using MAFFT followed by their phylogenetic analysis using <b>Nextstrain</b>.<li>As a result of phylogenetic analysis by <b>Nextstrain</b>, 5 clades are identified viz.<li>Initially, multiple sequence alignment of Dataset A with 15359 and Dataset B with 3033 SARS-CoV-2 genomes using MAFFT are performed followed by phylogenetic analyses using <b>Nextstrain</b> to identify virus clades.<li>In this work, multiple sequence alignment of 18392 SARS-CoV-2 sequences is carried out using MAFFT followed by phylogenetic analyses using <b>Nextstrain</b> where 15359 global dataset without Indian (Dataset A) and dataset of 3033 exclusive Indian (Dataset B) SARS-CoV-2 genomes are considered separately to identify the virus clades.<li>All the files which include dataset (raw and aligned sequences, metadata for <b>Nextstrain</b> and JSON files as outputs of <b>Nextstrain</b>), codes, supplementary PDF and videos of clade specific virus evolution and transmission in 71 countries are available at &#x201C;http://www.nitttrkol.ac.in/indrajit/projects/COVID-Evolution-SignatureSNPs-18&#xA0;K/&#x201D;.<li> [[22]](K. Katoh, K. Misawa, K. Kuma, et al., MAFFT: A novel method for rapid multiple sequence alignment based on fast fourier transform, Nucleic Acids Research 30 (14) (2002) 3059&#x2013;3066. doi:https://doi: 10.1093/nar/gkf436.) [[23]](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics (Oxford, England)</source>          <volume>34</volume>          <year>2018</year>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [[24]](          <person-group person-group-type="author">            <name>              <surname>Volz</surname>              <given-names>E.M.</given-names>            </name>            <name>              <surname>Koelle</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>          </person-group>          <article-title>Viral phylodynamics</article-title>          <source>PLoS Computer Biol.</source>          <volume>9</volume>          <issue>3</issue>          <year>2013</year>          <object-id pub-id-type="publisher-id">e1002947</object-id>          <pub-id pub-id-type="doi">10.1371/journal.pcbi.1002947</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34578452">34578452</a> <b>(3)</b> Multiple Occurrences of a 168-Nucleotide Deletion in SARS-CoV-2 ORF8, Unnoticed by Standard Amplicon Sequencing and Variant Calling Pipelines<ul><li> <b>Nextstrain</b> analysis was carried out according to the official workflow (<uri xlink:href="https://github.com/<b>nextstrain</b>">https://github.com/<b>nextstrain</b></uri>, accessed on 5 May 2021) [<xref rid="B21-viruses-13-01870" ref-type="bibr"><u>21</u></xref>,<xref rid="B22-viruses-13-01870" ref-type="bibr"><u>22</u></xref>].<li> All sequences possess at least 21 SNVs compared to the Wuhan-Hu-1 reference (MN908947.3) and were assigned to Clades 20B by Nextclade (clades.<b>nextstrain</b>.org) and M.1 (B.1.1.294.1) by Pangolin (cov-lineages.org, accessed on 17 September 2021), respectively.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(0)</b> Multiple Occurrences of a 168-Nucleotide Deletion in SARS-CoV-2 ORF8, Unnoticed by Standard Amplicon Sequencing and Variant Calling Pipelines<ul><li> NextClade v1.5.0 web tool (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">clades.<b>nextstrain</b>.org/</ext-link>) was employed for clade determination and mutation calling.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34535792">34535792</a> <b>(3)</b> The biological and clinical significance of emerging SARS-CoV-2 variants<ul><li> Two commonly used systems have been developed for epidemiological surveillance: the Phylogenetic Assignment of Named Global Outbreak (PANGO) lineage<sup><xref ref-type="bibr" rid="CR22"><u>22</u></xref></sup> and <b>NextStrain</b><sup><xref ref-type="bibr" rid="CR23"><u>23</u></xref></sup> systems.<li><bold>a</bold> | Phylogenetic tree based on subsampling of globally circulating sequences created by <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global"><b>NextStrain</b></ext-link> (CC BY 4.0).<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [23](Hodcroft, E. CoVariants: SARS-CoV-2 mutations and variants of interest (NextStrain). <italic>CoVariants</italic><ext-link ext-link-type="uri" xlink:href="https://covariants.org/">https://covariants.org/</ext-link> (2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34805425">34805425</a> <b>(2)</b> SARS-CoV-2 Epidemiology on a Public University Campus in Washington State<ul><li> Viral sequences were aligned and phylogenetic trees constructed using <b>Nextstrain</b> augur software [<xref rid="CIT0020" ref-type="bibr"><u>20</u></xref>].<li> Trees were visualized using <b>Nextstrain</b> auspice.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34535691">34535691</a> <b>(3)</b> Exploiting genomic surveillance to map the spatio-temporal dispersal of SARS-CoV-2 spike mutations in Belgium across 2020<ul><li> For this purpose, we first retrieved all non-Belgian sequences present in the European <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR12"><u>12</u></xref></sup> build on December 1, 2020.<li> <b>Nextstrain</b> uses a subsampling scheme in order to construct a real-time snapshot tree of the larger SARS-CoV-2 epidemic and phylogenetic diversity.<li> To replicate the <b>Nextstrain</b> workflow as closely as possible, we specified a clock rate of 8&#x2009;&#xD7;&#x2009;10<sup>&#x2013;4</sup> in TreeTime and set to filter samples that deviated more than four interquartile ranges from the root-to-tip regression<sup><xref ref-type="bibr" rid="CR8"><u>8</u></xref></sup>.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32407507">32407507</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34537361">34537361</a> <b>(1)</b> Monitoring the SARS-CoV-2 pandemic: screening algorithm with single nucleotide polymorphism detection for the rapid identification of established and emerging variants<ul><li> Sequence quality was validated using <b>NextStrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35602211">35602211</a> <b>(2)</b> SARS-CoV-2 transmission dynamics in Belarus in 2020 revealed by genomic and incidence data analysis<ul><li>For the phylogeny reconstruction, we utilized the SARS-CoV-2-specific phylogenetic inference pipeline implemented in <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR23"><u>23</u></xref></sup>.<li> To obtain a representative sample with those background sequences, a country-specific <b>Nextstrain</b> context subsampling was used<sup><xref ref-type="bibr" rid="CR23"><u>23</u></xref></sup>.<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34536502">34536502</a> <b>(1)</b> Case-control study evaluating risk factors for SARS-CoV-2 outbreak amongst healthcare personnel at a tertiary care center<ul><li> Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="interref0001">https://clades.<b>nextstrain</b>.org</ext-link>) and GISAID EpiCoV (<ext-link ext-link-type="uri" xlink:href="https://www.epicov.org" id="interref0002">https://www.epicov.org</ext-link>) were then used to track the origin of this strain and assign it to its corresponding clade.<li> The strain identified in this cluster of cases was identified as belonging to <b>Nextstrain</b> clade 20A and traced back to a likely importation from Europe in March 2020.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34519774">34519774</a> <b>(2)</b> Infectious Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) in Exhaled Aerosols and Efficacy of Masks During Early Mild Infection<ul><li> SARS-CoV-2 lineages and mutations were determined using PANGOLIN and <b>Nextstrain</b> tools [<xref rid="CIT0017" ref-type="bibr"><u>17</u></xref>, <xref rid="CIT0018" ref-type="bibr"><u>18</u></xref>].<li> Two (3%) of the 66 fine-aerosol samples collected from participants while wearing face masks were culture-positive, including one from a person infected with the alpha variant 2 days post onset and one from a person with a <b>Nextstrain</b> clade 20G virus 3 days post onset.<li> [17](<person-group person-group-type="author"><string-name><surname>O&#x2019;Toole</surname><given-names>&#xC1;</given-names></string-name>, <string-name><surname>Hill</surname><given-names>V</given-names></string-name>, <string-name><surname>McCrone</surname><given-names>JT</given-names></string-name>, <string-name><surname>Scher</surname><given-names>E</given-names></string-name>, <string-name><surname>Rambaut</surname><given-names>A</given-names></string-name></person-group>. <comment>cov-lineages/pangolin. CoV-lineages, 2021</comment>. Available at: <ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link>. Accessed <comment>15 July 2021</comment>.) [18](<person-group person-group-type="author"><string-name><surname>Bedford</surname><given-names>T</given-names></string-name>, <string-name><surname>Nehe</surname><given-names>R</given-names></string-name>, <string-name><surname>Hadfield</surname><given-names>J</given-names></string-name>, <etal>et al</etal></person-group><comment>Nextstrain SARS-CoV-2 resources</comment>, <year>2020</year>. Available at: <ext-link xlink:href="https://nextstrain.org/sars-cov-2/" ext-link-type="uri">https://nextstrain.org/sars-cov-2/</ext-link>. Accessed <comment>29 July 2021</comment>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34760282">34760282</a> <b>(0)</b> Genomic epidemiology reveals the reduction of the introduction and spread ofSARS-CoV-2 after implementing control strategies in Republic of Korea,2020<ul><li> The lineages and clades ofSARS-CoV-2 sequences were assigned by PANGOLIN (<xref rid="R22" ref-type="bibr"><u>Rambautet&#xA0;al., 2020</u></xref>) (<ext-link xlink:href="https://cov-lineages.org/" ext-link-type="uri">https://cov-lineages.org/</ext-link>) and Nextclade Beta (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [Rambautet&#xA0;al., 2020](<person-group person-group-type="author"><string-name><surname>Rambaut</surname><given-names>A.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2020</year>) &#x2018;<article-title>A Dynamic Nomenclature Proposalfor SARS-CoV-2 Lineages to Assist Genomic Epidemiology</article-title>&#x2019;,<source><italic toggle="yes">Nature Microbiology</italic></source>, <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>7</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=<person-group person-group-type="author"><string-name><surname>Rambaut</surname><given-names>A.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2020</year>) &#x2018;<article-title>A Dynamic Nomenclature Proposalfor SARS-CoV-2 Lineages to Assist Genomic Epidemiology</article-title>&#x2019;,<source><italic toggle="yes">Nature Microbiology</italic></source>, <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>7</lpage>."><person-group person-group-type="author"><string-name><surname>Rambaut</surname><given-names>A.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2020</year>) &#x2018;<article-title>A Dynamic Nomenclature Proposalfor SARS-CoV-2 Lineages to Assist Genomic Epidemiology</article-title>&#x2019;,<source><italic toggle="yes">Nature Microbiology</italic></source>, <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>7</lpage>.</a> <b>(2)</b> Genomic epidemiology reveals the reduction of the introduction and spread ofSARS-CoV-2 after implementing control strategies in Republic of Korea,2020<ul><li>The sequence of SARS-CoV-2 virus Spike gene was elicited regarding the valid databases (e.g., <uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>, <uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, <uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</uri>) (asscssed on 18 July 2020) and SARS-CoV-2 S-2P, comprising proline substitutions at residues K986 and V987, was chosen as the target antigen.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=<person-group person-group-type="author"><string-name><surname>Rambaut</surname><given-names>A.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2020</year>) &#x2018;<article-title>A Dynamic Nomenclature Proposalfor SARS-CoV-2 Lineages to Assist Genomic Epidemiology</article-title>&#x2019;,<source><italic toggle="yes">Nature Microbiology</italic></source>, <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>7</lpage>."><person-group person-group-type="author"><string-name><surname>Rambaut</surname><given-names>A.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2020</year>) &#x2018;<article-title>A Dynamic Nomenclature Proposalfor SARS-CoV-2 Lineages to Assist Genomic Epidemiology</article-title>&#x2019;,<source><italic toggle="yes">Nature Microbiology</italic></source>, <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>7</lpage>.</a> <b>(7)</b> Genomic epidemiology reveals the reduction of the introduction and spread ofSARS-CoV-2 after implementing control strategies in Republic of Korea,2020<ul><li>Genomic surveillance of SARS-CoV-2 across Brazilian regions was performed using the <b>Nextstrain</b> platform (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</uri>, accessed on 30 June 2021), an open-source program that generates updated phylogeny with interactive visualization of publicly available SARS-CoV-2 genomes.<li> Because of the large number of sequenced genomes, we conducted a subsampling of Brazilian genomes from GISAID by using <b>Nextstrain</b>&#x2019;s bioinformatics toolkit, which includes python3 scripts for preparing GISAID data for processing by Augur [<xref rid="B22-viruses-13-01806" ref-type="bibr"><u>22</u></xref>].<li>To date, twenty major clades of SARS-CoV-2 were defined by <b>Nextstrain</b> [<xref rid="B21-viruses-13-01806" ref-type="bibr"><u>21</u></xref>] (19A, 19B, 20A, 20B, 20C, 20D, 20E (EU1), 20F, 20G, 20H (Beta, V2), 20I (Alpha, V1), 20J (Gamma, V3), 21A (Delta), 21B (Kappa), 21C (Epsilon), 21D (Eta), 21E (Theta), 21F (Iota), 21G (Lambda), and 21H), based on global frequency and characteristic mutational events observed in the genomes.<li>It is important to point out that different nomenclatures for SARS-CoV-2 have been proposed, including by <b>Nextstrain</b>, cov-lineages.org [<xref rid="B23-viruses-13-01806" ref-type="bibr"><u>23</u></xref>], and GISAID [<xref rid="B17-viruses-13-01806" ref-type="bibr"><u>17</u></xref>].<li> Based on <b>Nextstrain</b> clades, Brazilian genomes were classified into nine clades, with the majority belonging to clade 20B (<italic toggle="yes">n</italic> = 2724; 50.91%) and 20J (Gamma, V3) (2516; 47.21%).<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34522191">34522191</a> <b>(2)</b> The viral phoenix: enhanced infectivity and immunity evasion of SARS-CoV-2 variants<ul><li><b>Nextstrain</b> Clade<sup>(b)</sup></th><th align="left">WHO Classification</th><th align="left">Pango Lineage<sup>(a)</sup></th><th align="left">Sequence change</th><th align="left">Spike Protein Substitutions</th><th align="left">First Identified</th></tr></thead><tbody><tr><td align="left" rowspan="11">20I/501Y.V1.<li><sup>b</sup><b>Nextstrain</b>, a collaboration between researchers in Seattle, USA and Basel, Switzerland, provides&#xA0;an open-source tool for visualizing the genetics of outbreaks.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34522191">34522191</a> <b>(2)</b> The viral phoenix: enhanced infectivity and immunity evasion of SARS-CoV-2 variants<ul><li> Mutations of interest were determined using a local instance of nextclade-cli version 0.14.2 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" ext-link-type="uri">https://github.com/<b>nextstrain</b>/nextclade</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34508827">34508827</a> <b>(6)</b> Dynamics prediction of emerging notable spike protein mutations in SARS-CoV-2 implies a need for updated vaccines<ul><li> It is worth mentioning that the pandemic has triggered international efforts to build up vast and significant genome evolution within real-time tracking databases such as <b>Nextstrain</b>.<li> <b>Nextstrain</b> has its unique naming and grouping system for SARS-CoV-2 variants into what is called clades; <b>Nextstrain</b> relies on Python scripts to maintain pathogen genomic sequences reposited initially in open source databases with an ability to perform robust phylodynamic data curation, analysis and visualization with a continuous update to bind the scientific and public health potentials [<xref rid="bib12" ref-type="bibr"><u>12</u></xref>].<li> We used <b>Nextstrain</b> clades for Similarity Matrix and Superimposition because these tools work on the overall two dimensional (2D) and three dimensional structures (3D) with multiple mutations usually found in one clade.<li> A set of 39 PDB structures of the wild type and the mutant clades of <b>Nextstrain</b>, listed in <xref rid="fig2" ref-type="fig"><u>Fig.&#xA0;2</u></xref>, <xref rid="fig3" ref-type="fig"><u>Fig.&#xA0;3</u></xref>, were built by phyre2 at (<ext-link ext-link-type="uri" xlink:href="http://www.sbg.bio.ic.ac.uk/phyre2" id="intref0035">http://www.sbg.bio.ic.ac.uk/phyre2</ext-link>) [<xref rid="bib13" ref-type="bibr"><u>13</u></xref>], the GISAID accessions of the chosen clades in this study; Alpha, Beta, Gamma, Delta, Epsilon, Eta Theta, Iota, Kappa, and Lambda are listed in Table&#xA0;S in the appendix.<li>It is well-noticed that the mutation D614G is found in all clades reported in the <b>Nextstrain</b> database as mentioned in the introduction; this has led us to question if the current emergency use authorization (EUA)-vaccines have taken into consideration the vaccine design to be effective against the D614G as a basal and dominant mutation or even other mutations of interest.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25950237">25950237</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34494886">34494886</a> <b>(4)</b> Complete Analysis of the Epidemiological Scenario around a SARS-CoV-2 Reinfection: Previous Infection Events and Subsequent Transmission<ul><li> Similarly, a dating analysis available at <b>Nextstrain</b>.org (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>) shows that the strain identified in the second episode belongs to a clade, 20A.EU1, that had not been described in GISAID before the end of June 2020 (<xref rid="figS2" ref-type="supplementary-material"><u>Fig.&#xA0;S2</u></xref>).<li> Clade nomenclature was assigned using <b>Nextstrain</b> (<xref rid="B16" ref-type="bibr"><u>16</u></xref>).<li> [16](<person-group person-group-type="author"><string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Hadfield</surname><given-names>J</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name>, <string-name><surname>Bedford</surname><given-names>T</given-names></string-name></person-group>. <year>2020</year>. <article-title>Year-letter genetic clade naming for SARS-CoV-2 on Nextstrain.org</article-title>. <ext-link xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming" ext-link-type="uri">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34497354">34497354</a> <b>(4)</b> Genomics, social media and mobile phone data enable mapping of SARS-CoV-2 lineages to inform health policy in Bangladesh<ul><li>To investigate the emergence of lineages of concern Alpha (B.1.1.7) and Beta (B.1.351) in Bangladesh, a <b>Nextstrain</b> phylogenetic tree was built (Fig.<li>Bangladesh genomes in the context of a <b>Nextstrain</b> time-scaled global phylogeny.<li><bold>a</bold>, <b>Nextstrain</b> time-scaled phylogeny of 1,489 representative global sequences of SARS-CoV-2 sampled from GISAID on 20 April 2021.<li>To update our phylogenetic analysis with our recent 2021 samples, we constructed a <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR25"><u>25</u></xref></sup> build using their global subsampling scheme across 1,045,491 sequences downloaded from GISAID on 20 April 20 2021.<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34556337">34556337</a> <b>(1)</b> Superspreading in the emergence of COVID-19 variants<ul><li> According to the maximum likelihood tree of <b>Nextstrain</b><xref rid="p0200" ref-type="p"><u>iii</u></xref>, mutation A23063T, which belongs to the sequence motif of B.1.1.7, B.1.351, and P.1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34556337">34556337</a> <b>(1)</b> Superspreading in the emergence of COVID-19 variants<ul><li> 1) For one, the authors reference the other competing web applications in use, <b>Nextstrain</b> and MicroReact.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34536875">34536875</a> <b>(4)</b> SARS-CoV-2 variant detection at a university dormitory using wastewater genomic tools<ul><li> Phylogenetic analysis was conducted on March 1, 2021, by using <b>Nextstrain</b> and open-source tools for visualizing viral genomes (<xref rid="bb0035" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> <b>Nextstrain</b> clade memberships of genomes were inferred using <b>nextstrain</b>-cli v1.16.5 and applied to the FastME tree.<li> The outer ring is colored according to assigned <b>Nextstrain</b> clade.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34488872">34488872</a> <b>(3)</b> Late reinfection with a different severe acute respiratory syndrome coronavirus-2 clade in a patient with refractory arterial hypertension: a case report<ul><li>Phylogenetic analysis of both specimens was done using web server <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>), with the SARS-CoV-2 database Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) to identify the clade, mutation calling, and phylogenetic placement of the SARS-CoV-2 genomes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34488225">34488225</a> <b>(0)</b> SARS-CoV-2 B.1.617.2 Delta variant replication and immune evasion<ul><li> Following this, all sequences were passed through Nextclade v0.15 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) to determine the number of gap regions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34509862">34509862</a> <b>(1)</b> Antiviral evaluation of hydroxyethylamine analogs: Inhibitors of SARS-CoV-2 main protease (3CLpro), a virtual screening and simulation approach<ul><li>The mutations reported from 35,000 SARS-COV-2 genomes sequenced worldwide (GISAID) analyzed by the Next-strain server (<b>Nextstrain</b>.org) were mapped w.r.t.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34512928">34512928</a> <b>(1)</b> AutoVEM2: A flexible automated tool to analyze candidate key mutations and epidemic trends for virus<ul><li> Besides, compared with the phylogenetic tree building based lineage identification tools such as PANGO lineages and <b>NextStrain</b> clades <xref rid="b0130" ref-type="bibr"><u>[26]</u></xref>, <xref rid="b0135" ref-type="bibr"><u>[27]</u></xref>, the efficiency of AutoVEM2 is much higher because of mutation filtering and haplotype-based variation tracking.<li> [[26]](&#xC1;ine O&#x2019;Toole, Emily Scher, Anthony Underwood, Ben Jackson, Verity Hill, John T McCrone, Rachel Colquhoun, Chris Ruis, Khalil Abu-Dahab, Ben Taylor, Corin Yeats, Louis Du Plessis, Daniel Maloney, Nathan Medd, Stephen W Attwood, David M Aanensen, Edward C Holmes, Oliver G Pybus, Andrew Rambaut, Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool, Virus Evolution, 2021, veab064.) [[27]](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>            </name>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics</source>          <year>2018</year>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34507232">34507232</a> <b>(2)</b> Near-roadway air pollution associated with COVID-19 severity and mortality &#x2013; Multiethnic cohort study in Southern California<ul><li> Since several more transmissible SARS-CoV-2 variants have been found after our study period (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="ir015"><italic>https://<b>nextstrain</b>.org/ncov/gisaid/global</italic></ext-link>), future studies are needed to compare the NRAP effect across various SARS-CoV-2 variants and across earlier <italic>vs.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34873448">34873448</a> <b>(1)</b> Transient acquisition of cross-species infectivity during the evolution of SARS-CoV-2<ul><li> (F) Time-resolved frequency distribution of SARS-CoV-2 variants with or without the N501Y mutation (based on the <b>Nextstrain</b> project).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34481867">34481867</a> <b>(4)</b> SARS-CoV-2, Zika viruses and mycoplasma: Structure, pathogenesis and some treatment options in these emerging viral and bacterial infectious diseases<ul><li> Another open-source site for sequences of SARS-CoV-2 genomes, <b>NEXTSTRAIN</b>, has identified at least five clades with slightly different criteria (19A, 19B, 20A, 20B, and 20C).<li> The image was obtained by the GISAID initiative and <b>NEXTSTRAIN</b> site (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="ir0035">https://<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34493992">34493992</a> <b>(2)</b> Emergence of the Delta Plus variant of SARS-CoV-2 in Iran<ul><li> This is supported by data presented by <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="ir8820"><underline>https://<b>nextstrain</b>.org</underline></ext-link>, showing the temporal emergence of the cases infected with new viral variants from 19 December 2020 to 12 February 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34477544">34477544</a> <b>(20)</b> Whole-genome-based phylogenomic analysis of the Belgian 2016&#x2013;2017 influenza A(H3N2) outbreak season allows improved surveillance<ul><li> Additional reference sequences were selected from phylogenetic trees in ECDC reports and <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">www.<b>nextstrain</b>.org</ext-link>) [<xref rid="R39" ref-type="bibr"><u>39</u></xref>] (Table S1, available in the online version of this article).<li>A local <b>Nextstrain</b> instance [<xref rid="R39" ref-type="bibr"><u>39</u></xref>], allowing lightweight phylogenomics comparison with much more genomes than possible with <sc>beast</sc>, was built using sequenced samples complemented with GISAID sequences.<li> The local <b>Nextstrain</b> instance was then constructed as detailed in the Supplementary Methods.<li> Red arrows indicate (coloured) reference names assigned to the various groups as defined by the WHO/ECDC, and coloured names without red arrows indicate additional references selected from ECDC reports and <b>nextstrain</b>.org.<li> Coloured names indicate additional references selected from ECDC reports and <b>nextstrain</b>.org.<li> Therefore, more reference sequences were selected manually from phylogenetic trees in ECDC reports and <b>nextstrain</b>.org [<xref rid="R39" ref-type="bibr"><u>39</u></xref>] by building provisional trees to evaluate whether the Belgian samples could be classified.<li> Belgian samples belonging to group &#x2018;3C2a1&#x2019;, however, could be classified because they clustered with some of the additional reference strains that were selected from ECDC reports and <b>nextstrain</b>.org [<xref rid="R39" ref-type="bibr"><u>39</u></xref>].<li>Increasing the number of genomes considered with a custom-built <b>Nextstrain</b> instance.<li> An alternative approach was explored by using <b>Nextstrain</b> to reconstruct an in-house instance of the Belgian samples supplemented with more than 2500 publicly available A(H3N2) genomes available in the GISAID database, because <b>Nextstrain</b> is a framework meant for real-time analysis (i.e.<li> When comparing the custom-built <b>Nextstrain</b> tree (<xref rid="F3" ref-type="fig">Fig.<li>Time-resolved overview of influenza samples from the Belgian 2016&#x2013;2017 outbreak season in the context of globally circulating influenza strains based on an in-house <b>Nextstrain</b> instance using only whole-genome sequences.<li> Coloured taxon labels indicate additional references selected from ECDC reports and <b>nextstrain</b>.org.<li> Our study evaluated the feasibility of shifting classification for influenza surveillance to a whole-genome-based approach rather than solely the HA gene, also incorporating many more references selected from ECDC reports and <b>nextstrain</b>.org [<xref rid="R39" ref-type="bibr"><u>39</u></xref>], to improve definition of genetic groups.<li> Therefore, an alternative approach was explored using a custom-built <b>Nextstrain</b> instance, which offers several advantages.<li> <b>Nextstrain</b> can analyse several hundreds to thousands of genomes much faster compared to <sc>beast</sc> and presents a real-time view.<li> Although the public influenza instance hosted at <b>nextstrain</b>.org [<xref rid="R39" ref-type="bibr"><u>39</u></xref>] is based solely on the HA and NA genes, we demonstrated the feasibility of using whole-genome information.<li> The feasibility of using <b>Nextstrain</b> with whole genomes also allows provisionally assigning samples to already existing or newly emerging groups to select suitable references to perform more powerful phylogenomics investigation with methods such as <sc>beast</sc>.<li> Tools optimized for real-time analysis, such as <b>Nextstrain</b>, facilitate contrasting seasonal local outbreaks to the globally circulating context and can provide a quick response.<li> [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34482101">34482101</a> <b>(3)</b> The real-life impact of vaccination on COVID-19 mortality in Europe and Israel<ul><li>The main explanatory variables of interest were proportion of vaccinated inhabitants (vaccination coverage), as well as average proportions of SARS-CoV-2 variants calculated across strains forming 12 <b>Nextstrain</b> clades.<li>Data on COVID-19 deaths and vaccination were obtained from Our World in Data on 15 April 2021.<xref rid="bib24" ref-type="bibr"><sup>24</sup></xref> Metadata on SARS-CoV-2 virus variants (clades) identified up to mid-April 2021 were downloaded from the <b>Nextstrain</b> platform.<li> Mean proportions of variants were calculated based on daily data on variants proportions across observed strains used to form <b>Nextstrain</b> clades (variants), between 29 January 2020 and 15 April 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34482101">34482101</a> <b>(1)</b> The real-life impact of vaccination on COVID-19 mortality in Europe and Israel<ul><li> In summary, these genomes provide strong evidence that this cat had been infected by the SARS-CoV-2 Alpha variant and <b>Nextstrain</b> analysis consistently clustered these sequences in the 20I clade (<xref rid="viruses-13-01759-f003" ref-type="fig"><u>Figure 3</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34482101">34482101</a> <b>(4)</b> The real-life impact of vaccination on COVID-19 mortality in Europe and Israel<ul><li> Phylogenies are updated and published on a daily basis on <b>nextstrain</b>.org, which is crucial for quickly identifying and tracking emergent strains.<li> (ii) clades proposed by the GISAID (Global Initiative on Sharing All Influenza Data) [<xref rid="B9-viruses-13-01764" ref-type="bibr"><u>9</u></xref>], (iii) lineages suggested by the PANGOLIN (Phylogenetic Assignment of Named Global Outbreak LINeages) tool [<xref rid="B19-viruses-13-01764" ref-type="bibr"><u>19</u></xref>], (iv) the years-and-letter nomenclature code introduced by <b>Nextstrain</b>, all combined with information about date and geographic location, when and where the respective samples were collected.<li> The GR-clades occupy pre-tip areas in both, sample and SNV space in agreement with the <b>nextstrain</b>-tree analysis (<xref rid="viruses-13-01764-f0A10" ref-type="fig"><u>Figure A10</u></xref>).<li> <b>Nextstrain</b> uses a Year-Letter nomenclature to label clades that persist for at least several months and have significant geographic spread [<xref rid="B6-viruses-13-01764" ref-type="bibr"><u>6</u></xref>].<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(7)</b> The real-life impact of vaccination on COVID-19 mortality in Europe and Israel<ul><li> In Italy, the predominant clades from September 2020 to January 2021 were represented by GISAID clade G (lineage B.1, <b>Nextstrain</b> clade 20A) and related clades GR (lineage B.1.1.1, <b>Nextstrain</b> clade 20B and lineage C, <b>Nextstrain</b> clade 20D) and GV (lineage B.1.177, <b>Nextstrain</b> clade 20E, EU1) [<xref rid="B6-jpm-11-00882" ref-type="bibr"><u>6</u></xref>,<xref rid="B7-jpm-11-00882" ref-type="bibr"><u>7</u></xref>,<xref rid="B8-jpm-11-00882" ref-type="bibr"><u>8</u></xref>].<li> Moreover, different Variants Of Concerns (VOCs) occurred, with the alpha variant (&#x3B1;V) (lineage B.1.1.7, GISAID clade GRY, <b>Nextstrain</b> clade 20I/501.Y.V1) and the beta variant (lineage B.1.351, GISAID clade GH, <b>Nextstrain</b> clade 20H/501.Y.V2) being the most relevant for public safety [<xref rid="B9-jpm-11-00882" ref-type="bibr"><u>9</u></xref>].<li> In particular, the &#x3B1;V dramatically raised in the first part of 2021, whereas the delta variant (&#x3B4;V) (lineage B.1.617.2, GISAID clade G/478K.V1, <b>Nextstrain</b> clade 20A/S.478K) recently showed a dramatic increase, raising the need of maintaining the measures to counteract the spreading of infection.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34391446">34391446</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34200331">34200331</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34211506">34211506</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34211506">34211506</a> <b>(0)</b> The real-life impact of vaccination on COVID-19 mortality in Europe and Israel<ul><li> a total of 23 mutations with 10 amino acid substitutions, three deletions, and one stop codon were observed in our analyzed sequence (<xref rid="tab1" ref-type="table"><u>Table&#xA0;1</u></xref>) (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34211506">34211506</a> <b>(1)</b> The real-life impact of vaccination on COVID-19 mortality in Europe and Israel<ul><li> Phylogenetic tree reconstruction using <b>NextStrain</b> (<xref rid="R45" ref-type="bibr"><u>45</u></xref>) placed 10 of the isolates in the A2a clade (<xref rid="F3" ref-type="fig"><u>Fig 3D</u></xref>) and two isolates (WRCEVA_00508, WRCEVA_00513) were Clade B/B1.<li> [45](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34423763">34423763</a> <b>(9)</b> Severe Acute Respiratory Syndrome Coronavirus 2 in Farmed Mink (<italic>Neovison vison</italic>), Poland<ul><li>The dataset consisted of all genetic sequences of SARS-CoV-2 from this study (Poland, Germany, Lithuania, Latvia, Estonia, Russia, and Ukraine), which was completed as the representative pool Europe by Nexstrain (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/europe" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/europe</ext-link>) and resulted in 5,778 entries.<li> We performed phylogenetic analysis by using the procedure recommended by <b>Nextstrain</b> with modifications in the subsampling region filtering procedure, in which the number of sequences per country was 40 (<xref rid="R21" ref-type="bibr"><italic>21</italic></xref>).<li> We used Augur toolkit version 10.1.1 (<b>Nextstrain</b>) for phylogenetic analysis and Auspice version 2.10.1 (<b>Nextstrain</b>) for visualization.<li> Possible time of divergence for samples was inferred by using the TreeTime pipeline (<ext-link xlink:href="https://www.treetime.com" ext-link-type="uri">https://www.treetime.com</ext-link>) implemented in the <b>Nextstrain</b> analysis and presented in the phylogenetic tree (<xref rid="R22" ref-type="bibr"><italic>22</italic></xref>).<li> Visualization was achieved by using <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/europe" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/europe</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34153623">34153623</a> <b>(2)</b> Lessons for preparedness and reasons for concern from the early COVID-19 epidemic in Iran<ul><li> The samples are randomly selected from 11 major clades as identified by <b>NextStrain</b> in 2020 (i.e., 19A, 19B, 20A, 20B, 20C, 20D, 20E, 20 F, 20 G, 501Y.V1, and 501Y.V2).<li> The report on 2020-03-16 by <b>NextStrain</b> and findings from returned travellers to Australia and New Zealand showed that sequences from cases with a travel history to Iran correspond to a distinct clade of SARS-CoV-2 (<xref rid="bib0100" ref-type="bibr"><u>Hodcroft et al., 2020</u></xref>; <xref rid="bib0045" ref-type="bibr"><u>Eden et al., 2020</u></xref>).<li> [Hodcroft et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>M&#xFC;ller</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Wagner</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Ilcisin</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>          </person-group>          <part-title>Genomic Analysis of COVID-19 Spread (Nextstrain Situation Report)</part-title>          <comment>2020-03-13; Available from:</comment>          <year>2020</year>          <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13" id="intr0020">https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13</ext-link>        ) [Eden et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32296544">32296544</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34540874">34540874</a> <b>(2)</b> Whole Genome Sequencing of SARS-CoV-2 Strains in COVID-19 Patients From Djibouti Shows Novel Mutations and Clades Replacing Over Time<ul><li> size of the amplification fragment: 114 bp), UK variant (<b>Nextstrain</b> clade 20I; primers in the N gene; size of the amplification fragment: 110 bp), and South African variant (<b>Nextstrain</b> clade 20H; primers in nsp2; size of the amplification fragment: 119 bp) strains, were used for the rapid identification of variants.<li> Sequences were analyzed using Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) (<xref rid="B20" ref-type="bibr"><u>20</u></xref>) and an in-house Python script.<li>Phylogenetic tree (obtained using the Nexcladevv0.14.4 software (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>), including four South African variants of SARS-CoV-2 strains from the second COVID-19 wave in Djibouti compared to forty SARS-CoV-2 sequences (clade 20H) available through GISAID and automatically selected using Nextclade.<li>Phylogenetic tree (obtained using the Nexcladevv0.14.4 software (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>), including the UK variant of SARS-CoV-2 strain from the second COVID-19 wave in Djibouti compared to six SARS-CoV-2 sequences (clade 20I) available through GISAID.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34481060">34481060</a> <b>(23)</b> Investigating the possible origin and transmission routes of SARS-CoV-2 genomes and variants of concern in Bangladesh<ul><li>An epidemiological analysis on genome samples from Bangladesh revealed regional sources of the isolates including variants rather than country-specific origin information (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/CHRF-Genomics/ncovBangladesh@main" id="ir0050">https://<b>nextstrain</b>.org/community/CHRF-Genomics/ncovBangladesh@main</ext-link>).<li> Moreover, <b>Nextstrain</b> (<xref rid="bb0040" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) considered insufficient samples from Bangladesh for the global phylogenetics build of SARS-CoV-2 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="ir0055">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li>The metadata of G1 subset samples was analyzed with Python version 3.8.5 (<xref rid="bb0155" ref-type="bibr"><u>Van Rossum and Drake, 2009</u></xref>) to investigate the <b>Nextstrain</b> clade distribution among divisions, as well as the gender and age-group distribution in Bangladeshi samples.<li> The complete phylogenetic analysis pipeline is available online (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="ir0065">https://github.com/<b>nextstrain</b>/ncov</ext-link>, version 3).<li> For tree visualizations, auspice version 0.5.0 (<ext-link ext-link-type="uri" xlink:href="https://auspice.us" id="ir0070">https://auspice.us</ext-link>) by <b>Nextstrain</b> (<xref rid="bb0040" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) was used.<li> In the initial phylogeny, each branch tip carrying a Bangladeshi genome present in available <b>Nextstrain</b> clades were identified, and that branch was then inspected backwards in the tree until all the closely related exogenous strains and their country of origin were determined.<li> In B.1.1.7 and B.1.351 variant-focused phylogenies, only the 20I/501Y&#xB7;V1 and 20H/501Y&#xB7;V2 <b>Nextstrain</b> clades were checked, respectively, for the origin prediction.<li> The overall <b>Nextstrain</b> clade distribution across all the major divisional cities of Bangladesh (<xref rid="f0005" ref-type="fig">Fig.<li> (B) Distribution of <b>Nextstrain</b> clades and Bangladeshi Divisions in SARS-CoV-2 samples isolated in Bangladesh (<italic>n</italic>&#xA0;=&#xA0;1009).<li> Out of 86,926 analyzed genome sequences from 49 countries and territories including Bangladesh in the G2 genome subset, the tree was constructed using a total of 1966 subsampled complete genomes with IQ-TREE version 2.0.3 (<xref rid="bb0080" ref-type="bibr"><u>Minh et al., 2020</u></xref>) and TreeTime version 0.8.1 (<xref rid="bb0105" ref-type="bibr"><u>Sagulenko et al., 2018</u></xref>) utilizing the generalized time-reversible (GTR) model and visualized using auspice version 0.5.0 (<ext-link ext-link-type="uri" xlink:href="https://auspice.us" id="ir0005">https://auspice.us</ext-link>) by <b>Nextstrain</b> (<xref rid="bb0040" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The interactive version of the tree can be accessed at: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/nahid18/ncov-bd@main/initial" id="ir0010">https://<b>nextstrain</b>.org/community/nahid18/ncov-bd@main/initial</ext-link>.<li>Estimated origins of initial representative samples of Bangladesh.</p></caption><alt-text id="al0035">Table 2</alt-text><table frame="hsides" rules="groups"><thead><tr><th><b>Nextstrain</b> Clade</th><th>Bangladeshi Sample</th><th>Estimated Origin</th></tr></thead><tbody><tr><td>19A.<li> The tree can be interactively explored at: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/nahid18/ncov-bd@main/uk" id="ir0015">https://<b>nextstrain</b>.org/community/nahid18/ncov-bd@main/uk</ext-link>.<li>Estimated origins of B.1.1.7 variant representative samples of Bangladesh.</p></caption><alt-text id="al0040">Table 3</alt-text><table frame="hsides" rules="groups"><thead><tr><th><b>Nextstrain</b> Clade</th><th>Bangladeshi Sample</th><th>Estimated Origin</th></tr></thead><tbody><tr><td rowspan="14">20I/501Y&#xB7;V1.<li> Pink colored branch tips are the representatives of all 47 B.1.351 variant samples of Bangladesh, which are distinctively grouped under a common ancestor suggesting local transmission of the variant The tree is available for interactive inspection at: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/nahid18/ncov-bd@main/sa" id="ir0020">https://<b>nextstrain</b>.org/community/nahid18/ncov-bd@main/sa</ext-link>.<li>Estimated origins of B.1.351 variant representative samples of Bangladesh.</p></caption><alt-text id="al0045">Table 4</alt-text><table frame="hsides" rules="groups"><thead><tr><th><b>Nextstrain</b> Clade</th><th>Bangladeshi Sample</th><th>Estimated Origin</th></tr></thead><tbody><tr><td rowspan="6">20H/501Y&#xB7;V2.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Van Rossum and Drake, 2009](          <person-group person-group-type="author">            <name>              <surname>Van Rossum</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Drake</surname>              <given-names>F.L.</given-names>            </name>          </person-group>          <part-title>Python 3 Reference Manual</part-title>          <year>2009</year>          <publisher-name>CreateSpace</publisher-name>          <publisher-loc>Scotts Valley, CA</publisher-loc>          <comment>Available from:</comment>          <pub-id pub-id-type="doi">10.5555/1593511</pub-id>          <comment>[Internet]</comment>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Minh et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [Sagulenko et al., 2018](          <person-group person-group-type="author">            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Puller</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>TreeTime: maximum-likelihood phylodynamic analysis</article-title>          <source>Virus Evol.</source>          <volume>4</volume>          <year>2018</year>          <pub-id pub-id-type="doi">10.1093/ve/vex042</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33835528">33835528</a> <b>(8)</b> Multiple clades of SARS&#x2010;CoV&#x2010;2 were introduced to Thailand during the first quarter of 2020<ul><li> Thailand was the first country to report a confirmed COVID&#x2010;19 case outside China, identified in a traveler who arrived at the Suvarnabhumi Airport in Bangkok from Wuhan, China on January 8, 2020.<xref rid="mim12883-bib-0001" ref-type="bibr"><sup>1</sup></xref>, <xref rid="mim12883-bib-0002" ref-type="bibr"><sup>2</sup></xref> As of November 14, 2020, there were 3866 confirmed COVID&#x2010;19 cases reported in Thailand with 60 deaths.<xref rid="mim12883-bib-0003" ref-type="bibr"><sup>3</sup></xref> Clade analysis from the <b>Nextstrain</b> database<xref rid="mim12883-bib-0004" ref-type="bibr"><sup>4</sup></xref> demonstrated that five clades of SARS&#x2010;CoV&#x2010;2 were found worldwide (as of November 13, 2020, from 184,320 full genomes), namely, clades 19A, 19B, 20A, 20B, and 20C, with more than 700 lineages characterized with Pangolin (Phylogenetic Assignment of Named Global Outbreak LINeages) software 5.0.<li> The phylogenetic tree was constructed using Nextclade webapp (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).<li> All sequences were further identified for viral clade and lineage using the <b>Nextstrain</b> database<xref rid="mim12883-bib-0004" ref-type="bibr"><sup>4</sup></xref> and Pangolin software,<xref rid="mim12883-bib-0005" ref-type="bibr"><sup>5</sup></xref> respectively.<li><b>Nextstrain</b>_clade</th>            <th align="center" valign="bottom" rowspan="1" colspan="1">Pangolin_lineage</th>          </tr>        </thead>        <tbody valign="top">          <tr>            <td align="left" valign="top" rowspan="1" colspan="1">EPI_ISL_437620.<li>Six pangolin viral lineages were identified in our study including A, A.6, B, B.1, B.1.8, and B.58 (three <b>Nextstrain</b> viral clades including 19A, 19B, and 20A), as detailed in Table&#xA0;<xref rid="mim12883-tbl-0001" ref-type="table"><u>1</u></xref>.<li> Five <b>Nextstrain</b> viral clades were found in Thailand from the specimens collected between January and March 2020, as was publicized in GISAID on November 13, 2020<xref rid="mim12883-bib-0006" ref-type="bibr"><sup>6</sup></xref> (<italic toggle="yes">n</italic>&#x2009;=&#x2009;243; Figure&#xA0;<xref rid="mim12883-fig-0001" ref-type="fig"><u>1</u></xref> and Table S<xref rid="mim12883-tbl-0001" ref-type="table"><u>1</u></xref>).<li>Sequencing data were uploaded to Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>, accessed on April 4, 2020) for phylogenetic analysis.<li> Five <b>Nextstrain</b> clades were indicated; 19A (purple color), 19B (dark blue), 20A (blue), 20B (emerald green), and 20C (leaf green).<li> EU1 <b>Nextstrain</b> clade) found in Thailand in October 2020 is in lime green circle [Color figure can be viewed at <ext-link xlink:href="http://wileyonlinelibrary.com" ext-link-type="uri">wileyonlinelibrary.com</ext-link>].<li> All five current viral <b>Nextstrain</b> clades were identified among 243 viral sequences sampled between January and March 2020, reflecting considerable viral diversity since the early spread of the virus outside China.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34469548">34469548</a> <b>(7)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li> Platforms like <b>Nextstrain</b> (<xref rid="msab264-B8" ref-type="bibr">Hadfield et al.<li> Specifically, we are maintaining and openly sharing a daily-updated database of mutation-annotated trees (MATs) containing global SARS-CoV-2 sequences from public databases, without any downsampling (other than for quality control, see <xref rid="sup1" ref-type="supplementary-material"><u>supplementary methods</u></xref>, <xref rid="sup1" ref-type="supplementary-material"><u>Supplementary Material online</u></xref>), including annotations for <b>Nextstrain</b> clades (<xref rid="msab264-B8" ref-type="bibr">Hadfield et al.<li> We also use the <italic toggle="yes">matUtils annotate</italic> command (<xref rid="sup1" ref-type="supplementary-material"><u>supplementary methods</u></xref>, <xref rid="sup1" ref-type="supplementary-material"><u>Supplementary Material online</u></xref>) to add <b>Nextstrain</b> clade and Pango lineage annotations to individual branches of our MAT.<li> As of June 9, 2021, our MAT consists of 834,521 sequences, includes 14 <b>Nextstrain</b> clade and 895 Pango lineage annotations for all samples, and is only 65 MB, or 14 MB when gzip-compressed (<xref rid="sup1" ref-type="supplementary-material">supplementary fig.<li> <b>Nextstrain</b> (<xref rid="msab264-B8" ref-type="bibr">Hadfield et al.<li> Both methods ensure that the clades remain monophyletic, but we use the second approach to label <b>Nextstrain</b> clades and Pango lineages in our SARS-CoV-2 MAT database since it can be automated using available data (<xref rid="sup1" ref-type="supplementary-material"><u>supplementary methods</u></xref>, <xref rid="sup1" ref-type="supplementary-material"><u>Supplementary Material online</u></xref>).<li> <italic toggle="yes">matUtils annotate</italic> has high congruence with <b>Nextstrain</b> clades and Pango lineage annotations (<xref rid="sup1" ref-type="supplementary-material"><u>supplementary table S1</u></xref>, <xref rid="sup1" ref-type="supplementary-material"><u>Supplementary Material online</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34469548">34469548</a> <b>(0)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li> Complete genomic sequences were analyzed with the NextClade online software (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, accessed on 21 July 2021) to assign the clade and call amino acid mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34469548">34469548</a> <b>(23)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li><b>Nextstrain</b> described as 20C/S:452R.<li><b>Nextstrain</b> described as 20A/S:478K.<li><b>Nextstrain</b> described as 20C.<li><b>Nextstrain</b> described as20C.<li> The phylogenetic tree of all circulating lineages was developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The phylogenetic trees of significant VOCs were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The phylogenetic trees of significant VOIs were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The scatterplot of all circulating lineages was developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The scatterplots of significant VOCs were developed before 27 June 2021 through the <b>Nextstrain</b> server, which is using GISAID data.<li> The scatterplots of significant VOIs were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The geographical distribution and transmission pattern of all circulating lineages were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The geographical distributions and transmission patterns of all circulating lineages were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The geographical distributions and transmission patterns of all circulating lineages were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The frequency pattern of all circulating lineages was developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The frequency patterns of significant VOCs were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The frequency patterns of significant VOIs were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The entropy diversity and event diversity of all circulating lineages were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> Entropy diversity and event diversity of significant VOCs were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> The entropy diversities and event diversities of significant VOIs were developed before 27 June 2021 through the <b>Nextstrain</b> server, using GISAID data.<li> For further data collection, we used several databases and servers, such as <b>Nextstrain</b> (SARS-CoV-2 resources) (<xref rid="B28" ref-type="bibr"><u>28</u></xref>, <xref rid="B29" ref-type="bibr"><u>29</u></xref>), Pango lineages (<xref rid="B30" ref-type="bibr"><u>30</u></xref>), Pango lineages on GitHub (<xref rid="B31" ref-type="bibr"><u>31</u></xref>), and COVID-3D (<xref rid="B32" ref-type="bibr"><u>32</u></xref>, <xref rid="B33" ref-type="bibr"><u>33</u></xref>).<li> <b>Nextstrain</b> uses the data from GISAID.<li> We analyzed and retrieved the data from the <b>Nextstrain</b> server in April 2021.<li>For data analysis, we used several servers, such as <b>Nextstrain</b> (SARS-CoV-2 resources) (<xref rid="B26" ref-type="bibr"><u>26</u></xref>, <xref rid="B27" ref-type="bibr"><u>27</u></xref>), Pango lineages (<xref rid="B30" ref-type="bibr"><u>30</u></xref>), Pango lineages on GitHub (<xref rid="B31" ref-type="bibr"><u>31</u></xref>), and COVID-3D (<xref rid="B32" ref-type="bibr"><u>32</u></xref>).<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](<collab>Nextstrain</collab>. <year>2021</year>. Nextstrain SARS-CoV-2 resources, on Nextstrain. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://nextstrain.org/sars-cov-2/" ext-link-type="uri">https://nextstrain.org/sars-cov-2/</ext-link>. <date-in-citation content-type="access-date">Accessed 14 April 2021</date-in-citation>.) [30](<person-group person-group-type="author"><string-name><surname>Banchich</surname><given-names>A</given-names></string-name>, <string-name><surname>O&#x2019;Toole</surname><given-names>&#xC1;</given-names></string-name></person-group>. <year>2021</year>. PANGO lineages. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://cov-lineages.org/index.html" ext-link-type="uri">https://cov-lineages.org/index.html</ext-link>. <date-in-citation content-type="access-date">Accessed 14 April 2021</date-in-citation>.) [31](<collab>GitHub</collab>. <year>2021</year>. cov-lineages/pangolin. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link>. <date-in-citation content-type="access-date">Accessed 14 April 2021</date-in-citation>) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908256">32908256</a>) [33](<person-group person-group-type="author"><string-name><surname>Portelli</surname><given-names>S</given-names></string-name>, <string-name><surname>Olshansky</surname><given-names>M</given-names></string-name>, <string-name><surname>Rodrigues</surname><given-names>CHM</given-names></string-name>, <string-name><surname>D'Souza</surname><given-names>EN</given-names></string-name>, <string-name><surname>Myung</surname><given-names>Y</given-names></string-name>, <string-name><surname>Silk</surname><given-names>M</given-names></string-name>, <string-name><surname>Alavi</surname><given-names>A</given-names></string-name>, <string-name><surname>Pires</surname><given-names>DEV</given-names></string-name>, <string-name><surname>Ascher</surname><given-names>DB</given-names></string-name></person-group>. <year>2021</year>. <article-title>Author correction: exploring the structural distribution of genetic variation in SARS-CoV-2 with the COVID-3D online resource</article-title>. <source>Nat Genet</source><volume>53</volume>:<fpage>254</fpage>&#x2013;<lpage>254</lpage>. doi:<pub-id pub-id-type="doi">10.1038/s41588-020-00775-x</pub-id>.) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33757278">33757278</a>) [27](<person-group person-group-type="author"><string-name><surname>Zimmer</surname><given-names>JCaC</given-names></string-name></person-group>. <year>2021</year>. <article-title>Coronavirus variants and mutations, on The New York Times</article-title>. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.nytimes.com/interactive/2021/health/coronavirus-variant-tracker.html" ext-link-type="uri">https://www.nytimes.com/interactive/2021/health/coronavirus-variant-tracker.html</ext-link>. <date-in-citation content-type="access-date">Accessed 14 April 2021</date-in-citation>.) [30](<person-group person-group-type="author"><string-name><surname>Banchich</surname><given-names>A</given-names></string-name>, <string-name><surname>O&#x2019;Toole</surname><given-names>&#xC1;</given-names></string-name></person-group>. <year>2021</year>. PANGO lineages. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://cov-lineages.org/index.html" ext-link-type="uri">https://cov-lineages.org/index.html</ext-link>. <date-in-citation content-type="access-date">Accessed 14 April 2021</date-in-citation>.) [31](<collab>GitHub</collab>. <year>2021</year>. cov-lineages/pangolin. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link>. <date-in-citation content-type="access-date">Accessed 14 April 2021</date-in-citation>) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908256">32908256</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908256">32908256</a> <b>(1)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li> Lineage B.6 (belonging to <b>Nextstrain</b> clade 19A) was a major contributor of SARS-CoV-2 during the early pandemic transmission in Malaysia and is known to have spread from there to India [<xref rid="B30-pathogens-10-01109" ref-type="bibr"><u>30</u></xref>].<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33253226">33253226</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33253226">33253226</a> <b>(6)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li> Global SARS-CoV-2 genomic sequencing efforts have contributed data of thousands (2,503,415) of strains into public database such as, GISAID and <b>Nextstrain</b>, NCBI SARS-CoV-2 Resources (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.ncbi.nlm.nih.gov/sars-cov-2/" ext-link-type="uri">https://www.ncbi.nlm.nih.gov/sars-cov-2/</ext-link>).<li>The fasta files were used for phylogenetic tree reconstruction using <b>NEXTSTRAIN</b>&#x2019;s (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.<b>nextstrain</b>.org/" ext-link-type="uri">https://www.<b>nextstrain</b>.org/</ext-link>) augur (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.docs.<b>nextstrain</b>.org/projects/augur/en/stable/" ext-link-type="uri">https://www.docs.<b>nextstrain</b>.org/projects/augur/en/stable/</ext-link>) pipeline.<li><b>Nextstrain</b> analysis.<li>Expanded phylogenetic analysis to examine the genetic divergence of strains was conducted against a representative subset of 825 Asian SARS-CoV-2 genomes present in <b>Nextstrain</b> database using data for the period January 2020 to January 2021.<li>Time-resolved phylogenetic distribution of genomic epidemiology of SAR-CoV-2 focused on Asian subsampling (Screenshot of the current <b>Nextstrain</b> display in SVG format are protected by CC-BY license).<li>Subsequently, <b>Nextstrain</b> analysis was run for global temporal and spatial classification of the isolates.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33253226">33253226</a> <b>(1)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li> Although only few genomes were selected elsewhere for our phylogenetic inference as shown in <xref rid="pone.0241942.g001" ref-type="fig"><u>Fig 1</u></xref>, similar topology was observed when genomes deposited on <b>Nextstrain</b> [<xref rid="pone.0241942.ref012" ref-type="bibr"><u>12</u></xref>] a public repository for SARS-CoV-2 sequences were included as shown in <xref rid="pone.0241942.g002" ref-type="fig"><u>Fig 2</u></xref>.<li> [12](Nextstrain SARS-CoV-2 resources [Internet]. [cited 2020 June 11]. Available from <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://nextstrain.org/sars-cov-2" ext-link-type="uri">https://nextstrain.org/sars-cov-2</ext-link>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34803444">34803444</a> <b>(6)</b> Genomic chronicle of SARS-CoV-2: a mutational analysis with over 1 million genome sequences<ul><li><b>NextStrain</b> (2021).<li> <b>Nextstrain</b>: Analysis and Visualization of Pathogen Sequence Data [online] Website: https://<b>nextstrain</b>.org/ [accessed 28 April 2021].<li><b>NextStrain</b> (2020).<li> <b>Nextstrain</b>: analysis and visualization of pathogen sequence data [online] Website: https://<b>nextstrain</b>.org/[ accessed 17 May 2020].</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34803439">34803439</a> <b>(1)</b> An update comprehensive review on the status of COVID-19: vaccines, drugs, variants and neurological symptoms<ul><li><b>Nextstrain</b> clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34448936">34448936</a> <b>(2)</b> Emergence and spread of the potential variant of interest (VOI) B.1.1.519 of SARS-CoV-2 predominantly present in Mexico<ul><li> A phylogenomic analysis of genomic sequences using the <b>Nextstrain</b> tool showed that the viruses in the lineage B.1.1.519 (B.1.1.1.222+T478K+P681H+T732A) group independently of the lineage B.1.1.222 sequences, strongly suggesting that this variant should be classified as a variant of interest (VOI) (Fig.<li> The phylogenomic tree was powered a CC-BY-4.0 license and attribution of <b>nextstrain</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34448987">34448987</a> <b>(2)</b> A SARS-CoV-2 mutant from B.1.258 lineage with &#x2206;H69/&#x2206;V70 deletion in the Spike protein circulating in Central Europe in the fall 2020<ul><li>SARS-CoV-2 strain was provided by Laborat&#xF3;rio de V&#xED;rus Respirat&#xF3;rio e do Sarampo, Instituto Oswaldo Cruz, Fiocruz (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</uri>: Brazil/RJ-314/2020 or GISAID #414045).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34435953">34435953</a> <b>(3)</b> Effects of common mutations in the SARS-CoV-2 Spike RBD and its ligand, the human ACE2 receptor on binding affinity and kinetics<ul><li> Constructed using TreeTime (<xref rid="bib35" ref-type="bibr"><u>Sagulenko et al., 2018</u></xref>) from the <b>Nextstrain</b> Global (<xref rid="bib12" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) sample of SARS-CoV-2 sequences from the GISAID database (<xref rid="bib39" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>) (accessed 15 April 2021, N = 4017).<li> The variants are labelled with their clade designation from <b>Nextstrain</b> (<xref rid="bib12" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and/or PANGO lineage (<xref rid="bib31" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), where relevant.<li> The RBD mutations were collated from CoVariants (<xref rid="bib14" ref-type="bibr"><u>Hodcroft, 2021</u></xref>) and <b>Nextstrain</b>.<li> [Sagulenko et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Shu and McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hodcroft, 2021](          <person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>EB</given-names></name></person-group>          <year iso-8601-date="2021">2021</year>          <article-title>Covariants: SARS-COV-2 mutations and variants of interest</article-title>          <ext-link xlink:href="https://covariants.org/" ext-link-type="uri">https://covariants.org/</ext-link>          <date-in-citation iso-8601-date="2021-04-15">April 15, 2021</date-in-citation>        ) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34454775">34454775</a> <b>(1)</b> Relevance of immune response and vaccination strategies of SARS-CoV-2 in the phase of viral red queen dynamics<ul><li><bold><b>Nextstrain</b> clade</bold>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34431762">34431762</a> <b>(2)</b> Evolution and emergence of multidrug-resistant <italic toggle="yes">Mycobacterium tuberculosis</italic> in Chisinau, Moldova<ul><li> tuberculosis</ext-link></named-content></italic> isolates, doing model testing in BEAST2 [<xref rid="R9" ref-type="bibr"><u>9</u></xref>], phylogenetic reconstruction and discrete trait analysis in BEAST 1.10 [<xref rid="R10" ref-type="bibr"><u>10</u></xref>], and the <italic toggle="yes">IQtree</italic> and <italic toggle="yes">timetree</italic> tools available through <b>Nextstrain</b> <italic toggle="yes">Augur</italic> version 6.2.0 [<xref rid="R11" ref-type="bibr"><u>11</u></xref>] (Supplementary Methods and Table S1, available in the online version of this article).<li> The phylogenetic tree was estimated using the <italic toggle="yes">IQtree</italic> and <italic toggle="yes">timetree</italic> tools in <b>Nextstrain</b> <italic toggle="yes">Augur</italic> 6.2.0 [<xref rid="R11" ref-type="bibr"><u>11</u></xref>].<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28166715">28166715</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29942656">29942656</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> Evolution and emergence of multidrug-resistant <italic toggle="yes">Mycobacterium tuberculosis</italic> in Chisinau, Moldova<ul><li> The genomes of these two samples harbored 14 mutations (versus Wuhan-Hu-1) and were assigned <b>Nextstrain</b> clade 20C, Pangolin lineage B.1.369, and GISAID clade GH (<xref rid="fig1" ref-type="fig"><u>Fig.&#xA0;1B</u></xref>).<li>The resulting SARS-CoV-2 viral genome sequences were uploaded to Nextclade server (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) to assign <b>Nextstrain</b> clades (<xref rid="B23" ref-type="bibr"><u>23</u></xref>).<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> Evolution and emergence of multidrug-resistant <italic toggle="yes">Mycobacterium tuberculosis</italic> in Chisinau, Moldova<ul><li> Introduced stop codons are rare among the submitted genome assemblies tracking the evolution of SARS-CoV-2 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://<b>nextstrain</b>.org" ext-link-type="uri"><b>nextstrain</b>.org</ext-link>) but are distributed all along the genome (<xref rid="pone.0244468.s004" ref-type="supplementary-material"><u>S4 Fig</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34428777">34428777</a> <b>(11)</b> Emergence and expansion of SARS-CoV-2 B.1.526 after identification in New York<ul><li> An interactive version of this figure is available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/ny/B.1.526">https://<b>nextstrain</b>.org/groups/blab/ncov/ny/B.1.526</ext-link>.<li> Data in <bold>e</bold>, <bold>f</bold> were visualized using <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>) and made available through a CC-BY-4.0 license.<li> <xref rid="Fig2" ref-type="fig"><u>2a</u></xref>) was conducted using the <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR32"><u>32</u></xref></sup> workflow at <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>, which aligns sequences against the Wuhan Hu-1 reference using nextalign (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade">https://github.com/<b>nextstrain</b>/nextclade</ext-link>), constructs a maximum-likelihood phylogenetic tree via IQ-TREE<sup><xref ref-type="bibr" rid="CR33"><u>33</u></xref></sup>, estimates molecular clock branch lengths via TreeTime<sup><xref ref-type="bibr" rid="CR34"><u>34</u></xref></sup> and reconstructs nucleotide and amino acid changes (also via TreeTime).<li> <xref rid="Fig4" ref-type="fig"><u>4e, f</u></xref>) was similarly conducted using the same <b>Nextstrain</b> workflow with the addition of performing ancestral trait reconstruction of the geographic &#x2018;division&#x2019; attribute of 933 SARS-CoV-2 genomes downloaded from GISAID on 6 Jun 2021.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34437928">34437928</a> <b>(2)</b> Emergence of Q493R mutation in SARS-CoV-2 spike protein during bamlanivimab/etesevimab treatment and resistance to viral clearance<ul><li> Clades and lineages were assigned to the genomes according to the <b>Nextstrain</b> nomenclature (version 1.0.0) and the PANGOLIN package (version 2021.06.15), respectively.<li> Mutation analysis of the Spike region was provided by <b>Nextstrain</b> and GISAID CoVsuver tool.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34437928">34437928</a> <b>(10)</b> Emergence of Q493R mutation in SARS-CoV-2 spike protein during bamlanivimab/etesevimab treatment and resistance to viral clearance<ul><li> Both of these sites are polymorphic deep in the SARS-CoV-2 phylogeny near the branch point for clade 20A (<b>Nextstrain</b> clade nomenclature).<li>All available full-length sequences from Wisconsin through February 16, 2021 were used for phylogenetic analysis using the tools implemented in <b>Nextstrain</b> custom builds (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) [<xref rid="ppat.1009849.ref054" ref-type="bibr"><u>54</u></xref>,<xref rid="ppat.1009849.ref055" ref-type="bibr"><u>55</u></xref>].<li> Phylogenetic trees were built using the standard <b>Nextstrain</b> tools and scripts [<xref rid="ppat.1009849.ref054" ref-type="bibr"><u>54</u></xref>,<xref rid="ppat.1009849.ref055" ref-type="bibr"><u>55</u></xref>].<li>We used the global <b>Nextstrain</b> [<xref rid="ppat.1009849.ref054" ref-type="bibr"><u>54</u></xref>] phylogenetic tree (<b>nextstrain</b>.org/ncov/global) accessed on February 24, 2021 to query whether mutations detected within-host are detected on the global tree.<li> Using the tree JSON output from the <b>Nextstrain</b> pipeline [<xref rid="ppat.1009849.ref054" ref-type="bibr"><u>54</u></xref>], we traversed the tree from root to tip.<li>A full-genome phylogenetic tree built showing 6306 Wisconsin consensus sequences with the <b>Nextstrain</b> pipeline is shown.<li> <b>Nextstrain</b> clade labels are shown on the corresponding branch.<li> [54](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [55](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [54](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [55](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [54](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [54](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34537474">34537474</a> <b>(6)</b> Impact of cobas PCR Media freezing on SARS-CoV-2 viral RNA integrity and whole genome sequencing analyses<ul><li> To simulate national surveillance and assessment of circulating viral clades, we independently compared the pre- and post-freezing sequence datasets with published and well described SARS-CoV-2 reference genomes submitted to <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" id="interref0002">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>) (<xref rid="bib0004" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> Radial rooted phylogenetic representation of SARS-CoV-2 reference sequence submitted to <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" id="interref0001">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34458642">34458642</a> <b>(12)</b> Molecular characterization of SARS-CoV-2 from Bangladesh: implications in genetic diversity, possible origin of the virus, and functional significance of the mutations<ul><li> A team of scientists has developed an open source bioinformatics and visualization toolkit named <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org" id="intref0010">www.<b>nextstrain</b>.org</ext-link>) for real-time tracking of pathogen evaluation including SARS-CoV-2 (<xref rid="bib47" ref-type="bibr"><b>nextstrain</b>-github, 2020</xref>).<li>We have used open source bioinformatics visualization platform <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org" id="intref0025"><b>nextstrain</b>.org</ext-link>) for phylogeny analysis of our sequences (<xref rid="bib31" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> Pairwise sequence alignment and clade assignment was done using web based <b>Nextstrain</b> tool Nextclade beta version 0.4.9 (<xref rid="bib46" ref-type="bibr"><u>Nextclade, 2020</u></xref>).<li> Further detailed information of different clusters was derived from a preformed interactive web-based tree developed from a subsample of global sequences (~5000) by neighbour joining method in <b>Nextstrain</b> web interface (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0030">https://<b>nextstrain</b>.org/ncov/global</ext-link>, date accessed on 30 August, 2020).<li>After phylogeny analysis we have found, our 324 isolates were distributed among all the <b>nextstrain</b> clades where 20B clade was the most frequent (86%) (<xref rid="fig4" ref-type="fig"><u>Figure&#xA0;4</u></xref>).<li>Radial presentation of phylogeny tree formed from 324 SARS-CoV-2 genomes from Bangladesh compared to the reference SARS-CoV-2 sequence shows the isolates belonged to all of the <b>nextstrain</b> clades and the 20B clade is the most prominent.<li>Location-wise of distribution of isolates from different clades.</p></caption><alt-text id="alttext0065">Table&#xA0;3</alt-text><table frame="hsides" rules="groups"><thead><tr><th>Location (Division)</th><th><b>Nextstrain</b>/GISAID Clade</th><th>No.<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Nextclade, 2020](          <source>NEXTCLADE</source>          <year>2020</year>          <comment>Available:</comment>          <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0080">https://clades.nextstrain.org/</ext-link>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34419470">34419470</a> <b>(7)</b> CovidPhy: A tool for phylogeographic analysis of SARS-CoV-2 variation<ul><li> One of the most popular tool is <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0030">https://<b>nextstrain</b>.org</ext-link> (<xref rid="bib8" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>)), which provides a maximum likelihood phylogeny built on a massive amount of SARS-CoV-2 genomes, and which allows to investigate the phylodynamics of the virus since the beginning of the pandemic.<li> However, nomenclature of the <b>Nextstrain</b> phylogeny is limited to a few nodes among hundreds, it does not follow systematic criteria for naming clades, and it is not stable since it has undergone several changes over the last few months.<li> The tool Nextclade, which is part of the <b>Nextstrain</b> project, allows to classify SARS-CoV-2 fasta sequences into 18 major clades.<li> Thereby, the differing naming for SARS-CoV-2 genetic lineages by GISAID, <b>Nextstrain</b> and Pango will remain by the scientific community, so that it seems most reasonable to keep the consistent nomenclature employed in the source mentioned.<li> In addition, note also that the minimal nomenclature scheme employed by <b>Nextstrain</b> does not follows a systematic criteria for naming branches and the nomenclature is dynamic (see comments in (<xref rid="bib5" ref-type="bibr"><u>G&#xF3;mez-Carballa et al., 2020a</u></xref>)); this is also a common feature of the nomenclature used by <xref rid="bib15" ref-type="bibr">Rambaut et al.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [G&#xF3;mez-Carballa et al., 2020a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32878977">32878977</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34490316">34490316</a> <b>(1)</b> Severe SARS-CoV-2 Breakthrough Reinfection With Delta Variant After Recovery From Breakthrough Infection by Alpha Variant in a Fully Vaccinated Health Worker<ul><li> The assembled SARS-CoV-2 genomes were assigned clade using <b>Nextstrain</b> (<xref rid="B11" ref-type="bibr"><u>11</u></xref>).<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34370324">34370324</a> <b>(1)</b> Genomic analysis reveals a rapid spread and predominance of lambda (C.37) SARS&#x2010;COV&#x2010;2 lineage in Peru despite circulation of variants of concern<ul><li> (C) Phylogenetic analysis of the main circulating variants in Peru implemented in <b>NextStrain</b> platform considering the genomes previously reported in the Global Initiative on Sharing Avian Influenza Data and those sequenced in this article.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34467932">34467932</a> <b>(1)</b> Wastewater SARS-CoV-2 monitoring as a community-level COVID-19 trend tracker and variants in Ohio, United States<ul><li> Viral sequences were strain-typed using Pangolin (<xref rid="bb0245" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) and <b>NextStrain</b> criteria (<xref rid="bb0285" ref-type="bibr">T.<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34409452">34409452</a> <b>(1)</b> Cluster characterization of SARS-CoV-2 in military personnel deployed to Egypt and subsequent introduction of B.1.1.7 and C.36 lineages to Colombia<ul><li> In addition to the B.1.1.7 groups, a collection of 1902 high-quality reference genomes embracing different lineages used in Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) for phylogenetic inferences was included and built from GISAID (Global Initiative on Sharing All Influenza Data) database (downloaded on 19 May 2021).<li> The complete dataset with 1965 sequences was used to perform phylogenetic inference using <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34438144">34438144</a> <b>(1)</b> Community-level SARS-CoV-2 sequence diversity revealed by wastewater sampling<ul><li> These correspond to <b>NextStrain</b> clades 20I, 20H and 20J, respectively (<xref rid="bb0060" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34451505">34451505</a> <b>(2)</b> Monitoring the Circulation of SARS-CoV-2 Variants by Genomic Analysis of Wastewater in Marseille, South-East France<ul><li> <b>Nextstrain</b> clade 19B) [<xref rid="B44-pathogens-10-01042" ref-type="bibr"><u>44</u></xref>].<li> The Bio-T Kit<sup>&#xAE;</sup> SARS-CoV-2 UK and N501Y variants (Biosellal, BIOTK125, Dardily, France) was used for the putative detection of the <b>Nextstrain</b> variants 20I (V1) (UK), 20H (V2) (SA) and 20J (V3) (Br) by real-time RT-PCR (qRT-PCR) with an exogenous internal positive control (IPC).<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33991677">33991677</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34391446">34391446</a> <b>(0)</b> Circulating SARS-CoV-2 variants in Italy, October 2020&#x2013;March 2021<ul><li> 1.4.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34389814">34389814</a> <b>(1)</b> SARS-CoV-2 N501Y variants of concern and their potential transmission by mouse<ul><li> The graph was made from data adopted from <b>Nextstrain</b> SARS-CoV-2 resources.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34433082">34433082</a> <b>(1)</b> Acute SARS-CoV-2 infection is associated with an increased abundance of bacterial pathogens, including <italic>Pseudomonas aeruginosa</italic> in the nose<ul><li> Genomes with greater than 90% coverage and 10X depth were retained for phylogenetic analysis with <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34400021">34400021</a> <b>(1)</b> Aparici&#xF3;n <italic>de novo</italic> de la mutaci&#xF3;n E484K en una variante del linaje B.1.1.7 de SARS-CoV-2<ul><li> la webApp <b>Nextstrain</b><xref rid="bib0100" ref-type="bibr"><sup>10</sup></xref> tanto para la asignaci&#xF3;n del clado como para la visualizaci&#xF3;n de las mutaciones.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34452471">34452471</a> <b>(1)</b> Retrospective Study of the Upsurge of Enterovirus D68 Clade D1 among Adults (2014&#x2013;2018)<ul><li> The open reading frame (ORF) and VP1 nucleotide sequence datasets were subjected to phylogenetic analyses with the <b>Nextstrain</b> pipeline [<xref rid="B11-viruses-13-01607" ref-type="bibr"><u>11</u></xref>].<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34392145">34392145</a> <b>(1)</b> High throughput diagnostics and dynamic risk assessment of SARS-CoV-2 variants of concern<ul><li> Publicly available databases and bioinformatics initiatives <xref rid="bib0056" ref-type="bibr"><u>[56]</u></xref> (<italic>e.g.</italic> GISAID, PANGO, Pangolin or <b>Nextstrain</b>) [<xref rid="bib0004" ref-type="bibr"><u>4</u></xref>,<xref rid="bib0057" ref-type="bibr"><u>[57]</u></xref>, <xref rid="bib0058" ref-type="bibr"><u>[58]</u></xref>, <xref rid="bib0059" ref-type="bibr"><u>[59]</u></xref>, <xref rid="bib0060" ref-type="bibr"><u>[60]</u></xref>], aim to provide real-time surveillance but in reality take between two and six weeks for sequencing and analysis <xref rid="bib0050" ref-type="bibr"><u>[50]</u></xref>.<li> [[56]](COVID-19 Genomics UK Consortium. Public data &amp; analysis n.d. <ext-link ext-link-type="uri" xlink:href="https://www.cogconsortium.uk/tools-analysis/public-data-analysis-2/" id="interref0016">https://www.cogconsortium.uk/tools-analysis/public-data-analysis-2/</ext-link> (accessed March 19, 2021).) [4](O'Toole &#xC1;., Scher E., Underwood A., Jackson B., Hill V., McCrone J. T., et&#xA0;al. Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool. Virus Evolution 2021; veab064. <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/lineage_list.html" id="ir74858">https://cov-lineages.org/lineage_list.html</ext-link> (accessed August 8, 2021). <pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>) [[57]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [[58]](          <person-group person-group-type="author">            <name>              <surname>Shu</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>McCauley</surname>              <given-names>J.G.</given-names>            </name>          </person-group>          <article-title>Global initiative on sharing all influenza data&#x2013;from vision to reality</article-title>          <source>Eurosurveillance</source>          <volume>22</volume>          <year>2017</year>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>        ) [[59]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [[60]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[50]](          <person-group person-group-type="author">            <name>              <surname>Leung</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Shum</surname>              <given-names>M.H.</given-names>            </name>            <name>              <surname>Leung</surname>              <given-names>G.M.</given-names>            </name>            <name>              <surname>Lam</surname>              <given-names>T.T.</given-names>            </name>            <name>              <surname>Wu</surname>              <given-names>JT.</given-names>            </name>          </person-group>          <article-title>Early transmissibility assessment of the N501Y mutant strains of SARS-CoV-2 in the United Kingdom, October to November 2020</article-title>          <source>Eurosurveillance</source>          <volume>26</volume>          <year>2021</year>          <object-id pub-id-type="publisher-id">2002106</object-id>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2020.26.1.2002106</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34401863">34401863</a> <b>(3)</b> 2019nCoVR&#x2014;A comprehensive genomic resource for SARS-CoV-2 variant surveillance<ul><li> It could be helpful if it is cross-referenced with other viral assignments, such as those from WHO and <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0035">https://<b>nextstrain</b>.org/</ext-link>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34391593">34391593</a> <b>(1)</b> Enhancing influenza vaccine immunogenicity and efficacy through infection mimicry using silk microneedles<ul><li> We used the probe designer described above to design non-overlapping 52-mer oligos with a target Gibbs free energy for binding of &#x2212;60 (allowable Gibbs free energy [&#x2212;70, &#x2212;50]) to the N and ORF1a regions of the SARS-CoV-2 genome, targeting only the 3000&#x2013;8000 nt region of the latter because it was the most conserved region among the strains circulating at the time as determined using <b>nextstrain</b><sup><xref rid="R43" ref-type="bibr"><u>43</u></xref></sup>.<li> [43](<collab>Nextstrain</collab>. <comment><ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link>.</comment>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=<collab>Nextstrain</collab>. <comment><ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link>.</comment>"><collab>Nextstrain</collab>. <comment><ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link>.</comment></a> <b>(5)</b> Enhancing influenza vaccine immunogenicity and efficacy through infection mimicry using silk microneedles<ul><li> Chinese lineage [<xref rid="ref17" ref-type="bibr"><u>17</u></xref>] and <b>Nextstrain</b> clade [<xref rid="ref18" ref-type="bibr"><u>18</u></xref>] typing are SNP-based classification methods, which primarily depend on marker mutations or mutations with significant frequency and geographic spread.<li> In fact, GISAID classification is not strictly based on phylogenetic relationships, but relies more on several marker variants, which could be observed in <b>NextStrain</b> website (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/" ext-link-type="uri">https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/</ext-link>) [<xref rid="ref18" ref-type="bibr"><u>18</u></xref>].<li> Moreover, <b>Nextstrain</b> team classified virus isolates with mutations of more than 20% global frequency as new branches and Rambaut <italic toggle="yes">et&#xA0;al</italic>.<li> [17](<person-group person-group-type="author"><string-name><surname>Tang</surname> &#xA0;<given-names>X</given-names></string-name>, <string-name><surname>Wu</surname> &#xA0;<given-names>C</given-names></string-name>, <string-name><surname>Li</surname> &#xA0;<given-names>X</given-names></string-name>, <etal>et&#xA0;al.</etal></person-group> &#xA0;<article-title>On the origin and continuing evolution of SARS-CoV-2</article-title>. <source>Natl Sci Rev</source> &#xA0;<year>2020</year>;<volume>7</volume>(<issue>6</issue>):<fpage>1012</fpage>&#x2013;<lpage>23</lpage>.) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34390857">34390857</a> <b>(0)</b> SARS-CoV-2 variant of concern P.1 (Gamma) infection in young and middle-aged patients admitted to the intensive care units of a single hospital in Salvador, Northeast Brazil, February 2021<ul><li> Genomes were submitted to the Nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="interref0005">https://clades.<b>nextstrain</b>.org</ext-link>) for quality control assessment and the Pangolin tool (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="interref0006">https://pangolin.cog-uk.io/</ext-link>) for SARS-CoV-2 lineage assignment.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34378952">34378952</a> <b>(1)</b> SARS-CoV-2 Isolates Show Impaired Replication in Human Immune Cells but Differential Ability to Replicate and Induce Innate Immunity in Lung Epithelial Cells<ul><li> At the present rate of global spreading, SARS-CoV-2 is accumulating 2 to 3 mutations a month with a maximum of &#x223C;55 amino acid changes to date (GISAID and <b>Nextstrain</b> data), which accounts for less than 0.2% of the genome.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34376689">34376689</a> <b>(1)</b> Tracking the introduction and spread of SARS-CoV-2 in coastal Kenya<ul><li>We used a modified <b>NextStrain</b><sup><xref ref-type="bibr" rid="CR22"><u>22</u></xref></sup> Augur based pipeline to perform quality control and phylogenetic analysis for a total of 406 sequences from the Kenyan coast.<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34452444">34452444</a> <b>(6)</b> SHARED: An International Collaboration to Unravel Hepatitis C Resistance<ul><li>To provide broader access to the SHARED data, we are in the process of setting up an interactive data visualization platform, HCV <b>Nextstrain</b>, adapted from GISAID, which also provides the platform for SARS-CoV2 [<xref rid="B28-viruses-13-01580" ref-type="bibr"><u>28</u></xref>].<li> HCV <b>Nextstrain</b> integrates the phylogenomic data with the geographic information collected by SHARED.<li> With detailed epidemiological and risk factor information, HCV <b>Nextstrain</b> can potentially be used to track resistant HCV transmission networks.<li> A preview version of HCV <b>Nextstrain</b> is available at <uri xlink:href="https://hcv<b>nextstrain</b>.hcvdb.ubc.ca">https://hcv<b>nextstrain</b>.hcvdb.ubc.ca</uri> (accessed on 21 July 2021).<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> SHARED: An International Collaboration to Unravel Hepatitis C Resistance<ul><li> While the Pango system has been widely adopted, some other nomenclature systems have also been used such as <b>Nextstrain</b> and GISAID [<xref rid="CIT0073" ref-type="bibr"><u>73</u></xref>, <xref rid="CIT0074" ref-type="bibr"><u>74</u></xref>], all of which require manual curation to define which clades on the phylogenetic tree should be named.<li><b>Nextstrain</b> name.<li> [73](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [74](<person-group person-group-type="author"><string-name><surname>Shu</surname> &#xA0;<given-names>Y</given-names></string-name>, <string-name><surname>McCauley</surname> &#xA0;<given-names>J</given-names></string-name></person-group>. <article-title>GISAID: global initiative on sharing all influenza data - from vision to reality</article-title>. <source>Euro Surveill</source> &#xA0;<year>2017</year>; <volume>22</volume>(<issue>13</issue>). pii=30494.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34375700">34375700</a> <b>(1)</b> Outbreak or pseudo-outbreak? Integrating SARS-CoV-2 sequencing to validate infection control practices in a dialysis facility<ul><li> Differences between individual sequences were evaluated at <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="interref0001">https://clades.<b>nextstrain</b>.org</ext-link>.<li> For phylogenetic inference (ie to determine the hierarchy of case relationships), sequences were integrated with associated metadata (diagnosis date and location) and aligned on a local implementation of <b>NextStrain</b><xref rid="bib0012" ref-type="bibr"><sup>12</sup></xref> using augur and displayed via a web browser using auspice.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34445204">34445204</a> <b>(2)</b> Novel Nested-Seq Approach for SARS-CoV-2 Real-Time Epidemiology and In-Depth Mutational Profiling in Wastewater<ul><li> This finding is in line with the genomic data of the GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri> accessed on 1 May 2021) and Nexstrain (<uri xlink:href="https://<b>nextstrain</b>.org/sars-cov-2">https://<b>nextstrain</b>.org/sars-cov-2</uri> accessed on 1 May 2021) databases, highlighting the significant prevalence of the G164 strain in percentage of &gt;98% and ~100% worldwide and in Europe, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34075382">34075382</a> <b>(1)</b> CovidExpress: an interactive portal for intuitive investigation on SARS-CoV-2 related transcriptomes<ul><li> For instance, <b>Nextstrain</b> and GISAID databases allow sharing and comprehensive real-time analysis to monitor the virus evolution and adaptation (Hadfield <italic toggle="yes">et al</italic>; Shu &amp; McCauley) since the first sequencing data were available (Wu <italic toggle="yes">et al</italic>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34373859">34373859</a> <b>(2)</b> Investigating the first stage of the COVID-19 pandemic in Ukraine using epidemiological and genomic data<ul><li> These genomes were utilized to construct a maximum likelihood phylogeny using <b>Nextstrain</b> build for SARS-CoV-2 with the default country-specific subsampling settings [<xref rid="R42" ref-type="bibr"><u>42</u></xref>].<li> Similarly, by <b>Nextstrain</b> classification the distribution of lineages is: 19A - 1.7 %, 20A - 51.7 % and 20B - 46.7 % (<xref rid="F2" ref-type="fig"><u>Figure 2</u></xref>).<li> [42](<source><ext-link xlink:href="http://github.com" ext-link-type="uri">github.com</ext-link></source>, <comment><ext-link xlink:href="https://github.com/nextstrain/ncov" ext-link-type="uri">https://github.com/nextstrain/ncov</ext-link></comment>, <year>2021</year>. <comment>Accessed:</comment><date-in-citation>2020-11-01</date-in-citation>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34442750">34442750</a> <b>(1)</b> Distribution of SARS-CoV-2 Lineages in the Czech Republic, Analysis of Data from the First Year of the Pandemic<ul><li> Following the emergence of new variants, the constant need for updates of variant nomenclature [<xref rid="B4-microorganisms-09-01671" ref-type="bibr"><u>4</u></xref>,<xref rid="B5-microorganisms-09-01671" ref-type="bibr"><u>5</u></xref>] has been reflected by the GISAID nomenclature system [<xref rid="B2-microorganisms-09-01671" ref-type="bibr"><u>2</u></xref>] and the <b>Nextstrain</b> nomenclature system [<xref rid="B3-microorganisms-09-01671" ref-type="bibr"><u>3</u></xref>], including a dynamic virus nomenclature system which reflects the spread of the SARS-CoV-2 [<xref rid="B6-microorganisms-09-01671" ref-type="bibr"><u>6</u></xref>].<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32269081">32269081</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33184173">33184173</a>) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34451453">34451453</a> <b>(4)</b> Predominance of the SARS-CoV-2 Lineage P.1 and Its Sublineage P.1.2 in Patients from the Metropolitan Region of Porto Alegre, Southern Brazil in March 2021<ul><li>After running the <b>Nextstrain</b> workflow using quality control and subsampling approaches, we obtained a dataset of 8635 time- and geographical-representative genomes.<li> Finally, we assessed the consensus sequences quality using Nextclade v0.14.2 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>; accessed on 4 May 2021).<li> accessed on 4 May 2021) and global clades and mutations using Nextclade v0.14.2 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>; accessed on 4 May 2021).<li> These sequences were subjected to analysis inside the <b>NextStrain</b> ncov pipeline [<xref rid="B52-pathogens-10-00988" ref-type="bibr"><u>52</u></xref>] (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</uri>; accessed on 4 May 2021).<li> [52](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34363852">34363852</a> <b>(4)</b> Mutation hotspots and spatiotemporal distribution of SARS-CoV-2 lineages in Brazil, February 2020-2021<ul><li> These sequences were then subjected to analysis inside <b>NextStrain</b> ncov pipeline (https://github.com/<b>nextstrain</b>/ncov) (<xref rid="bib0026" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>) through a Brazilian-focused subsampling scheme using time- and worldwide-representative contextual samples.<li> <b>Nextstrain</b> clades are represented in key branches.<li> <b>Nextstrain</b> clades are represented in key branches.<li> [Hadfield&#xA0;et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34363772">34363772</a> <b>(1)</b> Performance of saliva and mid-turbinate swabs for detection of the beta variant in South Africa<ul><li>Differences between before and after beta VOC replacement were assessed for the following parameters: diagnostic validity of saliva and mid-turbinate sampling relative to nasopharyngeal swabs, mean Ct differences, pre-test probability of having SARS-CoV-2, and <b>Nextstrain</b> clades (<xref rid="sec1" ref-type="sec"><u>appendix 2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33655278">33655278</a> <b>(6)</b> Emergence and Expansion of the SARS-CoV-2 Variant B.1.526 Identified in New York<ul><li> 2A</xref>) was conducted using the <b>Nextstrain</b><sup><xref rid="R32" ref-type="bibr"><u>32</u></xref></sup> workflow at <ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link> which aligns sequences against the Wuhan-Hu-1 reference via nextalign (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/nextclade" ext-link-type="uri">https://github.com/<b>nextstrain</b>/nextclade</ext-link>), constructs a maximum-likelihood phylogenetic tree via IQ-TREE<sup><xref rid="R33" ref-type="bibr"><u>33</u></xref></sup>, estimates molecular clock branch lengths via TreeTime<sup><xref rid="R34" ref-type="bibr"><u>34</u></xref></sup> and reconstructs nucleotide and amino acid changes also via TreeTime.<li> 4E</xref>&#x2013;<xref rid="F4" ref-type="fig"><u>F</u></xref>) was similarly conducted using the same <b>Nextstrain</b> workflow with the addition of performing ancestral trait reconstruction of the geographic &#x201C;division&#x201D; attribute of 933 SARS-CoV-2 genomes downloaded from <ext-link xlink:href="http://gisaid.org" ext-link-type="uri">gisaid.org</ext-link> on Jun 6, 2021.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><u>No PMID</u> <b>(1)</b> Insights from genomes and genetic epidemiology of SARS-CoV-2 isolates from the state of Andhra Pradesh<ul><li>Phylogenetic analysis was performed following the <b>Nextstrain</b> protocol for analysis of SARS-CoV-2 genomes using the genomes sequenced in this study and the dataset of 3058 genomes from India deposited in the GISAID (Supplementary Table S2) [<xref rid="ref10" ref-type="bibr"><u>10</u></xref>&#x2013;<xref rid="ref12" ref-type="bibr"><u>12</u></xref>].<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34124300">34124300</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34408729">34408729</a> <b>(3)</b> Whole-Genome-Based <italic>Helicobacter pylori</italic> Geographic Surveillance: A Visualized and Expandable Webtool<ul><li> In addition to genomic typing, <italic>Hp</italic>TT also provides a phylogenetic and geographic visualization tool based on the <b>Nextstrain</b> framework (<xref rid="B14" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The online visualization service was performed based on the CodeIgniter framework<sup><xref ref-type="fn" rid="footnote1"><u>1</u></xref></sup>, tree visualization was analyzed by the augur<sup><xref ref-type="fn" rid="footnote2"><u>2</u></xref></sup> bioinformatics tool and the auspice<sup><xref ref-type="fn" rid="footnote3"><u>3</u></xref></sup> visualization tool imbedded in the <b>Nextstrain</b> (<xref rid="B14" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) open source project.<li>Except for the typing tool, the <b>Nextstrain</b> framework was also embedded in our website.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [1](<a href="https://www.codeigniter.com/">https://www.codeigniter.com/</a>) [2](<a href="https://github.com/nextstrain/augur">https://github.com/nextstrain/augur</a>) [3](<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2021.687259/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fmicb.2021.687259/full#supplementary-material</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34409009">34409009</a> <b>(3)</b> The Emergence and Spread of Novel SARS-CoV-2 Variants<ul><li><bold>B.1.1.7:</bold> It is a new variant under investigation (VUI 202012/01 or variant of concern, VOC 202012/01) of the 501Y lineage, based on the B.1.1.7 lineage of PANGO lineage, GISAID clade GR/501Y.V1, <b>Nextstrain</b> clade 20I/501Y.V1 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34352360">34352360</a> <b>(2)</b> A discernable increase in the severe acute respiratory syndrome coronavirus 2 R.1 lineage carrying an E484K spike protein mutation in Japan<ul><li> Phylogenetic analysis was performed using the <b>Nextstrain</b> analysis (<xref rid="bb0015" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> C) Phylogenetic analysis of the R.1 lineage by <b>Nextstrain</b> analysis, along with the B.1.1.316 lineage which is the most recent common ancestor (<xref rid="bb0015" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34314049">34314049</a> <b>(1)</b> Introduction of SARS&#x2010;COV&#x2010;2 C.37 (WHO VOI lambda) from Peru to Italy<ul><li> Full genome sequencing (GISAID entry EPI_ISL_2759089) showed deletion &#x394;246&#x2010;252 and missense amino acid mutations G75V, T76I, L452Q, F490S, and T859N, which are hallmarks of SARS&#x2010;CoV&#x2010;2 C.37 clade according to PANGOLIN phylogeny (GR/452Q.V1 in GISAID phylogeny, 20D clade in <b>NextStrain</b>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34368787">34368787</a> <b>(4)</b> An optimized, amplicon-based approach for sequencing of SARS-CoV-2 from patient samples using COVIDSeq assay on Illumina MiSeq sequencing platforms<ul><li><hr/></td></tr><tr><td>bcl2fastq Conversion Software</td><td>Illumina</td><td>NA</td></tr><tr><td>Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN)</td><td>Center for Genomic Pathogen Surveillance</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0030">https://github.com/cov-lineages/pangolin</ext-link></td></tr><tr><td><b>Nextstrain</b> open-source toolkit.<li><hr/></td></tr><tr><td>bcl2fastq Conversion Software</td><td>Illumina</td><td>NA</td></tr><tr><td>Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN)</td><td>Center for Genomic Pathogen Surveillance</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0030">https://github.com/cov-lineages/pangolin</ext-link></td></tr><tr><td><b>Nextstrain</b> open-source toolkit</td><td><xref rid="bib4" ref-type="bibr"><u>Hadfield et&#xA0;al.,2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>" id="intref0035">https://github.com/<b>nextstrain</b></ext-link>.<li> For performing the phylogenetic analysis, we use the open-source tool-kit <b>Nextstrain</b> following a recently detailed protocol (<xref rid="bib5" ref-type="bibr"><u>Jolly and Scaria 2021</u></xref>).<li> [Hadfield et&#xA0;al.,2018](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics</source>          <volume>1</volume>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [Jolly and Scaria 2021](          <person-group person-group-type="author">            <name>              <surname>Jolly</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Scaria</surname>              <given-names>V.</given-names>            </name>          </person-group>          <article-title>Computational analysis and phylogenetic clustering of SARS-CoV-2 genomes</article-title>          <source>Bio Protoc.</source>          <volume>11</volume>          <year>2021</year>          <fpage>e3999</fpage>          <pub-id pub-id-type="doi">10.21769/BioProtoc.3999</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34138695">34138695</a> <b>(1)</b> Transmission Dynamics of Severe Acute Respiratory Syndrome Coronavirus 2 in High-Density Settings, Minnesota, USA, March&#x2013;June 2020<ul><li> Output from Augur was visualized by using Auspice as hosted by the <b>nextstrain</b> team (<ext-link xlink:href="http://auspice.us" ext-link-type="uri">http://auspice.us</ext-link>) (<xref rid="R27" ref-type="bibr"><italic>27</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34452375">34452375</a> <b>(1)</b> Severe SARS-CoV-2 Infection in a Cat with Hypertrophic Cardiomyopathy<ul><li> Phylogenetic analysis using <b>Nextstrain</b> [<xref rid="B26-viruses-13-01510" ref-type="bibr"><u>26</u></xref>] placed two SNPs, both encoding amino acid changes (A15659G, ORF1b: D731G; C29376T, N: P368L) on the terminal branch leading to the sequence from the cat.<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(3)</b> Severe SARS-CoV-2 Infection in a Cat with Hypertrophic Cardiomyopathy<ul><li> Phylogenetic results, paired with demographic and epidemiologic data, were visualized in <b>NextStrain</b> and displayed using baltic (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/evogytis/baltic" ext-link-type="uri">https://github.com/evogytis/baltic</ext-link>) [<xref rid="CIT0018" ref-type="bibr"><u>18</u></xref>].<li> We found that SCC contained all the major global clades circulating at the time, as defined by <b>NextStrain</b> (19A, 19B, 20A, 20B, 20C) [<xref rid="CIT0018" ref-type="bibr"><u>18</u></xref>].<li> <b>NextStrain</b> clades are labeled 19A&#x2013;20C.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34325753">34325753</a> <b>(2)</b> Phylogenetic analysis of SARS-CoV-2 lineage development across the first and second waves in Eastern Germany in 2020: insights into the cause of the second wave<ul><li> Phylogenetic maximum likelihood and time trees were constructed using the SARS-CoV-2-specific procedures taken from <uri xlink:href="https://github.com/<b>nextstrain</b>/ncov">github.com/<b>nextstrain</b>/ncov</uri> [<xref rid="ref4" ref-type="bibr"><u>4</u></xref>, <xref rid="ref5" ref-type="bibr"><u>5</u></xref>].<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [5](<string-name><surname>Hadfield</surname><given-names>J</given-names></string-name><etal/> (<year>2018</year>) <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>34</volume>, <fpage>4121</fpage>&#x2013;<lpage>4123</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=<string-name><surname>Hadfield</surname><given-names>J</given-names></string-name><etal/> (<year>2018</year>) <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>34</volume>, <fpage>4121</fpage>&#x2013;<lpage>4123</lpage>."><string-name><surname>Hadfield</surname><given-names>J</given-names></string-name><etal/> (<year>2018</year>) <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>34</volume>, <fpage>4121</fpage>&#x2013;<lpage>4123</lpage>.</a> <b>(8)</b> Phylogenetic analysis of SARS-CoV-2 lineage development across the first and second waves in Eastern Germany in 2020: insights into the cause of the second wave<ul><li> When neutralization activity was tested in different <italic toggle="yes">in vitro</italic> experiments, Ab #169 (immunoglobulin G<sub>1,</sub> IgG<sub>1</sub>) displayed the greatest neutralization potency in the Plaque Reduction Neutralization Test (PRNT) against isolates from <b>NextStrain</b> clades 19A and 19B: half maximal inhibitory concentration (IC<sub>50</sub>) was 0.049&#xA0;&#x3BC;g/mL and 0.02&#xA0;&#x3BC;g/mL, respectively.<li> In particular, PRNT showed complete loss of function against WHO variants of concern (VOC) gamma (also named P.1 by PANGOLIN or 20&#xA0;J/501Y.V3 by <b>NextStrain</b>) [<xref rid="cit0018" ref-type="bibr"><u>18&#x2013;20</u></xref>], beta (also named B.1.351 by PANGOLIN and 20&#xA0;H/501Y.V2 by <b>NextStrain</b>) [<xref rid="cit0018" ref-type="bibr"><u>18</u></xref>,<xref rid="cit0019" ref-type="bibr"><u>19</u></xref>,<xref rid="cit0021" ref-type="bibr"><u>21</u></xref>], and variants of interests (VOI) zeta (also named P.2 by PANGOLIN and 20B/S.484&#xA0;K by <b>NextStrain</b>) [<xref rid="cit0022" ref-type="bibr"><u>22</u></xref>], epsilon (also named B.1.427/B.1.419 by PANGOLIN and 21&#xA0;C by <b>NextStrain</b>) [<xref rid="cit0023" ref-type="bibr"><u>23</u></xref>,<xref rid="cit0024" ref-type="bibr"><u>24</u></xref>], and iota (also named B.1.526 by PANGOLIN and 21&#xA0;F by <b>NextStrain</b>) [<xref rid="cit0025" ref-type="bibr"><u>25</u></xref>].<li> Efficacy of bamlanivimab was preserved against VOC alpha (also named B.1.1.7 by PANGOLIN and 20I/501Y.V1 by <b>NextStrain</b>) [<xref rid="cit0021" ref-type="bibr"><u>21</u></xref>], but was lost in the related E484K-positive eta VOI (also named B.1.525 in PANGOLIN and 21D in <b>NextStrain</b>) [<xref rid="cit0026" ref-type="bibr"><u>26</u></xref>].<li> [18&#x2013;20](<person-group person-group-type="author"><string-name><surname>Hoffmann</surname><given-names>M</given-names></string-name>, <string-name><surname>Arora</surname><given-names>P</given-names></string-name>, <string-name><surname>Gro&#xDF;</surname><given-names>R</given-names></string-name></person-group>. et al. <article-title>SARS-CoV-2 variants B.1.351 and B.1.1.248: escape from therapeutic 1 antibodies and antibodies induced by infection and vaccination 2 3</article-title>. <source>bioRxiv</source>. Internet]. <year>2021</year> [cited 2021 <month>Apr</month><day>26</day>];2021.02.11.430787. Available from. . <pub-id pub-id-type="doi">10.1101/2021.02.11.430787</pub-id>) [18](<person-group person-group-type="author"><string-name><surname>Hoffmann</surname><given-names>M</given-names></string-name>, <string-name><surname>Arora</surname><given-names>P</given-names></string-name>, <string-name><surname>Gro&#xDF;</surname><given-names>R</given-names></string-name></person-group>. et al. <article-title>SARS-CoV-2 variants B.1.351 and B.1.1.248: escape from therapeutic 1 antibodies and antibodies induced by infection and vaccination 2 3</article-title>. <source>bioRxiv</source>. Internet]. <year>2021</year> [cited 2021 <month>Apr</month><day>26</day>];2021.02.11.430787. Available from. . <pub-id pub-id-type="doi">10.1101/2021.02.11.430787</pub-id>) [19](<person-group person-group-type="author"><string-name><surname>Liu</surname><given-names>H</given-names></string-name>, <string-name><surname>Wei</surname><given-names>P</given-names></string-name>, <string-name><surname>Zhang</surname><given-names>Q</given-names></string-name></person-group>. et al. <article-title>501Y.V2 and 501Y.V3 variants of SARS-CoV-2 lose binding to Bamlanivimab in vitro</article-title>. <source>bioRxiv Prepr Serv Biol</source>. Internet]. <year>2021</year> [cited 2021 <month>Apr</month><day>26</day>]; Available from. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.ncbi.nlm.nih.gov/pubmed/33619479" ext-link-type="uri">http://www.ncbi.nlm.nih.gov/pubmed/33619479</ext-link>) [21](<person-group person-group-type="author"><string-name><surname>Widera</surname><given-names>M</given-names></string-name>, <string-name><surname>Wilhelm</surname><given-names>A</given-names></string-name>, <string-name><surname>Hoehl</surname><given-names>S</given-names></string-name></person-group>. et al. <article-title>Bamlanivimab does not neutralize two SARS-CoV-2 variants carrying E484K in vitro</article-title>. <source>medRxiv</source>. Internet]. <year>2021</year> [cited 2021 Apr 26];2021.02.24.21252372. Available from&#x2026; . . <pub-id pub-id-type="doi">10.1101/2021.02.24.21252372</pub-id>) [22](<person-group person-group-type="author"><string-name><surname>Widera</surname><given-names>M</given-names></string-name>, <string-name><surname>Wilhelm</surname><given-names>A</given-names></string-name>, <string-name><surname>Hoehl</surname><given-names>S</given-names></string-name></person-group>, et al. Limited neutralization of authentic SARS-CoV-2 variants carrying E484K in vitro. J Infect Dis. <year>2021</year> Jul 5:jiab355. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http:10.1093/infdis/jiab355.Onlineaheadofprint" ext-link-type="uri">10.1093/infdis/jiab355.Onlineaheadofprint</ext-link>.) [23](<person-group person-group-type="author"><string-name><surname>Starr</surname><given-names>TN</given-names></string-name>, <string-name><surname>Greaney</surname><given-names>AJ</given-names></string-name>, <string-name><surname>Dingens</surname><given-names>AS</given-names></string-name></person-group>. et al. <article-title>Complete map of SARS-CoV-2 RBD mutations that escape the monoclonal antibody LY-CoV555 and its cocktail with LY-CoV016</article-title>. <source>Cell reports Med</source> [Internet]. <year>2021</year> [cited 2021 Apr 26];<volume>2</volume>:<fpage>100255</fpage>.) [24](<person-group person-group-type="author"><string-name><surname>McCallum</surname><given-names>M</given-names></string-name>, <string-name><surname>Bassi</surname><given-names>J</given-names></string-name>, <string-name><surname>De Marco</surname><given-names>A</given-names></string-name></person-group>. et al. <article-title>SARS-CoV-2 immune evasion by variant B.1.427/B.1.429</article-title>. <source>bioRxiv</source>. Internet]. <year>2021</year> [cited 2021 <month>Apr</month><day>26</day>];2021.03.31.437925. Available from. <pub-id pub-id-type="doi">10.1101/2021.03.31.437925</pub-id>) [25](<person-group person-group-type="author"><string-name><surname>Zhou</surname><given-names>H</given-names></string-name>, <string-name><surname>Dcosta</surname><given-names>BM</given-names></string-name>, <string-name><surname>Samanovic</surname><given-names>MI</given-names></string-name>, et al</person-group>. <article-title>B.1.526 SARS-CoV-2 variants identified in New York City are neutralized by vaccine-elicited and therapeutic monoclonal antibodies</article-title>. <source>bioRxiv Prepr Serv Biol</source>. <year>2021</year> Internet]. [cited 2021 Apr 26]; Available from; . <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.ncbi.nlm.nih.gov/pubmed/33791698" ext-link-type="uri">http://www.ncbi.nlm.nih.gov/pubmed/33791698</ext-link>) [21](<person-group person-group-type="author"><string-name><surname>Widera</surname><given-names>M</given-names></string-name>, <string-name><surname>Wilhelm</surname><given-names>A</given-names></string-name>, <string-name><surname>Hoehl</surname><given-names>S</given-names></string-name></person-group>. et al. <article-title>Bamlanivimab does not neutralize two SARS-CoV-2 variants carrying E484K in vitro</article-title>. <source>medRxiv</source>. Internet]. <year>2021</year> [cited 2021 Apr 26];2021.02.24.21252372. Available from&#x2026; . . <pub-id pub-id-type="doi">10.1101/2021.02.24.21252372</pub-id>) [26](<person-group person-group-type="author"><string-name><surname>Focosi</surname><given-names>D</given-names></string-name>, <string-name><surname>Tuccori</surname><given-names>M</given-names></string-name>, <string-name><surname>Baj</surname><given-names>A</given-names></string-name>, et al</person-group>. <article-title>SARS-CoV-2 variants: a synopsis of in vitro efficacy data of convalescent plasma, currently marketed vaccines, and monoclonal antibodies. Viruses</article-title>. <publisher-loc>Switzerland</publisher-loc>; <year>2021</year>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34395364">34395364</a> <b>(1)</b> Molecular Phylogenesis and Spatiotemporal Spread of SARS-CoV-2 in Southeast Asia<ul><li> demonstrated the emergence of European and North American mutant variants of SARS-CoV-2 in South-East Asia and constructed a transmission map using <b>Nextstrain</b> with 329 sequences (<xref rid="B24" ref-type="bibr"><u>24</u></xref>).<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32701194">32701194</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a> <b>(2)</b> Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool<ul><li> 2021</xref>) and complements two other SARS-CoV-2 nomenclature systems (<b>NextStrain</b> and GISAID) that focus on broader phylogenetic &#x2018;clades&#x2019;, and which incorporate criteria for minimum prevalence and persistence.<li> At the time of writing, there are 1,293 Pango lineages, compared to 12 and 9 clades for the <b>NextStrain</b> and GISAID nomenclatures, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34394051">34394051</a> <b>(8)</b> Genomic Surveillance and Phylodynamic Analyses Reveal the Emergence of Novel Mutations and Co-mutation Patterns Within SARS-CoV-2 Variants Prevalent in India<ul><li>We found that, for Indian sequences GR, GH, and G (GISAID) or 20B and 20A (<b>Nextstrain</b>) (<xref rid="B31" ref-type="bibr"><b>Nextstrain</b>, 2021</xref>), the clades were primarily the major prevalent clades in the middle and later half of the year 2020.<li> The GISAID clade distribution of the 54 sequences is slightly different from the national distribution with majority of O clade representation, and the same is true for <b>Nextstrain</b> clades with a prevalence of 20A clade (<xref ref-type="supplementary-material" rid="FS1"><u>Supplementary Figures 1C,D</u></xref>).<li> <xref ref-type="fig" rid="F1"><u>Figure 1</u></xref> shows the distribution of the different GISAID and <b>Nextstrain</b> clades in three time points.<li> Similarly, <b>Nextstrain</b> clades 20B and 20A are found to be more prevalent in the latter half of the year 2020.<li> Distribution of GISAID <bold>(A)</bold> and <b>Nextstrain</b> <bold>(B)</bold> clades across India for three different times spans: &#x201C;Term1&#x201D; (December 2019&#x2013;March 2020), &#x201C;Term2&#x201D; (April 2020&#x2013;July 2020), and &#x201C;Term3&#x201D; (August 2020&#x2013;December 2020), respectively.<li> clade (GISAID) and 20A clade (<b>Nextstrain</b>) sequences compared to both the overall Indian distribution and the sequences retrieved during the &#x201C;Term3&#x201D; period as well.<li> Our analysis clearly shows the gradual fixation of the more prevalent clades such as GR (GISAID) and 20B (<b>Nextstrain</b>) across time within the viral population extracted from Indian patients.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34323977">34323977</a> <b>(2)</b> M2-Deficient Single-Replication Influenza Vaccine&#x2013;Induced Immune Responses Associated With Protection Against Human Challenge With Highly Drifted H3N2 Influenza Strain<sup><xref rid="jiab374-FM1" ref-type="author-notes">&#xA0;</xref></sup><ul><li> Color coding (antigenic advance, tree model) and clade designations are taken from <b>Nextstrain</b>.org [<xref rid="jiab374-CIT0010" ref-type="bibr"><u>10</u></xref>].<li>HA clade designations and the phylogenetic relationships between the viruses shown in <xref rid="jiab374-F1" ref-type="fig"><u>Figure 1</u></xref> were taken from <b>nextstrain</b>.org [<xref rid="jiab374-CIT0010" ref-type="bibr"><u>10</u></xref>].<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34452356">34452356</a> <b>(2)</b> Rise and Fall of SARS-CoV-2 Lineage A.27 in Germany<ul><li>We used all available A.27 genomes and the selection of A genomes from <b>NextStrain</b> [<xref rid="B15-viruses-13-01491" ref-type="bibr"><u>15</u></xref>] global build to assemble a dataset for phylogenetic analyses including 718 A.27 genomes from RKI and GISAID.<li>For phylogenetic analysis, we assembled a dataset of 718 A.27 genomes from RKI and GISAID and a selection of additional A genomes from <b>NextStrain</b> [<xref rid="B15-viruses-13-01491" ref-type="bibr"><u>15</u></xref>] as an outgroup.<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(3)</b> Rise and Fall of SARS-CoV-2 Lineage A.27 in Germany<ul><li>&#x2003;<b>Nextstrain</b><sup><xref rid="bibr92-11779322211035921" ref-type="bibr"><u>92</u></xref></sup>.<li><b>Nextstrain</b> is an open-source project consisting of adatabase of viral genomes, a bioinformatics pipeline andinteractive visualization platform for phylodynamicsanalysis.<li>Advanced computing environment AWS Batch, which allowsusers to launch and monitor more reproducible <b>Nextstrain</b>build in cloud.<li> [92](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35036827">35036827</a> <b>(2)</b> Integrating Bioinformatics Tools Into Inquiry-Based Molecular Biology Laboratory Education Modules<ul><li> Specifically, they learn how to interpret BLAST results (max score, query cover, E-value, and percent identity) and create phylogenetic trees of coronavirus sequence divergence using the open-source software <b>Nextstrain</b> (<xref rid="R11" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> Data analyzed from NCBI BLAST and <b>Nextstrain</b> are used to answer important questions regarding SARS-CoV-2 spread.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34394197">34394197</a> <b>(3)</b> poreCov-An Easy to Use, Fast, and Robust Workflow for SARS-CoV-2 Genome Reconstruction <italic>via</italic> Nanopore Sequencing<ul><li>After the genome reconstruction <italic>via</italic> ARTIC, each reconstructed and polished genome is further analyzed <italic>via</italic> pangolin (lineage determination<sup><xref rid="footnote5" ref-type="fn"><u>5</u></xref></sup>), <b>nextstrain</b> (<b>nextstrain</b> clades, mutations, and deletions) (<xref rid="B9" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), and president (calculating various genome quality metrics<sup><xref rid="footnote6" ref-type="fn"><u>6</u></xref></sup>).<li> The last columns of the table highlight the determined lineages based on the pangolin and <b>nextstrain</b> nomenclatures and include information about substitutions and deletions based on the nextclade alignments (<xref rid="B9" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [5](<a href="github.com/cov-lineages/pangolin">github.com/cov-lineages/pangolin</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](<a href="gitlab.com/RKIBioinformaticsPipelines/president">gitlab.com/RKIBioinformaticsPipelines/president</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34255352">34255352</a> <b>(3)</b> The variants question: What is the problem?<ul><li> The outbreak-causing SARS-CoV-2 belongs to COG-UK lineage B.1.1.401 and <b>Nextstrain</b> clade 20B, carrying the spike amino acid change D614G (<xref rid="tab1" ref-type="table"><u>Table&#xA0;1</u></xref>).<li> <b>Nextstrain</b> 20B clade markers are labeled with an asterisk (*).<li> <b>Nextstrain</b> clade 20A.EU1), likely emerging in Spain by summer 2020, that markedly increased in frequency worldwide (<xref rid="B18" ref-type="bibr"><u>18</u></xref>).<li> Clade and lineage assignments were performed using Nextclade (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin) (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) (<xref rid="B24" ref-type="bibr"><u>24</u></xref>), respectively.<li> [18](<person-group person-group-type="author"><string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Zuber</surname><given-names>M</given-names></string-name>, <string-name><surname>Nadeau</surname><given-names>S</given-names></string-name>, <string-name><surname>Crawford</surname><given-names>KH</given-names></string-name>, <string-name><surname>Bloom</surname><given-names>JD</given-names></string-name>, <string-name><surname>Veesler</surname><given-names>D</given-names></string-name>, <etal>et al</etal></person-group>. <year>2020</year>. <article-title>Emergence and spread of a SARS-CoV-2 variant through Europe in the summer of 2020</article-title>. <source>MedRxiv</source> doi:<pub-id pub-id-type="doi">10.1101/2020.10.25.20219063v2</pub-id>.) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34315427">34315427</a> <b>(1)</b> COVID-19 false dichotomies and a comprehensive review of the evidence regarding public health, COVID-19 symptomatology, SARS-CoV-2 transmission, mask wearing, and reinfection<ul><li> Next-generation viral genome sequencing showed that the sequence variability between the two virus isolates belonging to <b>Nextstrain</b> clade 20C was too great to be explained by evolution within the patient alone.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34508652">34508652</a> <b>(5)</b> Emergence of an early SARS-CoV-2 epidemic in the United States<ul><li>(E) Relative <b>NextStrain</b> clade prevalence per U.S.<li>(F) Shannon evenness of <b>NextStrain</b> clades per U.S.<li> To investigate how many times SARS-CoV-2 was introduced into Louisiana, we first conducted a high-level genomic analysis by comparing <b>NextStrain</b> clade distributions of all available SARS-CoV-2 sequences from the continental U.S.<li><hr/></td></tr><tr><td>Pangolin v2.0</td><td><xref rid="bib50" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0025">https://github.com/cov-lineages/pangolin</ext-link></td></tr><tr><td>NextClade v0.12.0</td><td><xref rid="bib64" ref-type="bibr"><u>Hadfield et al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="intref0030">https://github.com/<b>nextstrain</b>/nextclade</ext-link>.<li> [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34218488">34218488</a> <b>(1)</b> A review of the antiviral activity of Chitosan, including patented applications and its potential use against COVID&#x2010;19<ul><li> For evolution analysis, 1,753 full-genome sequences from each <b>NextStrain</b> (<xref rid="B26" ref-type="bibr"><u>26</u></xref>) clade were filtered and cluster random sampled from the GISAID EpiFlu database.<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34320390">34320390</a> <b>(0)</b> Cross-neutralization of SARS-CoV-2 B.1.1.7 and P.1 variants in vaccinated, convalescent and P.1 infected<ul><li> Chromatograms were assembled and edited with the DNAStar 7.1.0 SeqMan module and imported in the clade assignment tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0002">https://clades.<b>nextstrain</b>.org/</ext-link>) to determine the variant.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34362060">34362060</a> <b>(3)</b> A Possible Role of Remdesivir and Plasma Therapy in the Selective Sweep and Emergence of New SARS-CoV-2 Variants<ul><li> Then, in August 2020, another epidemic occurred with a different variant we named Marseille-4 (also named <b>Nextstrain</b> 20A.EU2 clade), whose genome comprised 13 mutations compared to the Wuhan-Hu-1 strain and that we were able to link to a mink strain [<xref rid="B9-jcm-10-03276" ref-type="bibr"><u>9</u></xref>].<li>This is, for instance, the case of the variants Marseille-4 [<xref rid="B9-jcm-10-03276" ref-type="bibr"><u>9</u></xref>] (<b>Nextstrain</b> clade 20A.EU2) and Marseille-501 [<xref rid="B2-jcm-10-03276" ref-type="bibr"><u>2</u></xref>] (Pangolin lineage A.27) in Southern France and in Europe [<xref rid="B28-jcm-10-03276" ref-type="bibr"><u>28</u></xref>,<xref rid="B29-jcm-10-03276" ref-type="bibr"><u>29</u></xref>] and of variants 20I/501Y.V1 (lineage B.1.1.7) in the United Kingdom [<xref rid="B30-jcm-10-03276" ref-type="bibr"><u>30</u></xref>,<xref rid="B31-jcm-10-03276" ref-type="bibr"><u>31</u></xref>], 20H/501Y.V2 (lineage B.1.351) in South Africa [<xref rid="B32-jcm-10-03276" ref-type="bibr"><u>32</u></xref>], 20J/501Y.V3 (lineage B.1.281.1) in Brazil [<xref rid="B33-jcm-10-03276" ref-type="bibr"><u>33</u></xref>], and of lineage B.1.617 in India [<xref rid="B34-jcm-10-03276" ref-type="bibr"><u>34</u></xref>].<li> The viral variants that we detected in July 2020 and named Marseille-4 (later classified as <b>Nextstrain</b> clade 20A.EU2) [<xref rid="B9-jcm-10-03276" ref-type="bibr"><u>9</u></xref>,<xref rid="B12-jcm-10-03276" ref-type="bibr"><u>12</u></xref>] harbor amino acid substitution S477N located in the spike RBD [<xref rid="B9-jcm-10-03276" ref-type="bibr"><u>9</u></xref>,<xref rid="B12-jcm-10-03276" ref-type="bibr"><u>12</u></xref>,<xref rid="B45-jcm-10-03276" ref-type="bibr"><u>45</u></xref>].<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33785459">33785459</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33785459">33785459</a>) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33184236">33184236</a>) [28](          <person-group person-group-type="author">            <name>              <surname>Colson</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Levasseur</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Delerce</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Pinault</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Dudouet</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Devaux</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Fournier</surname>              <given-names>P.-E.</given-names>            </name>            <name>              <surname>Scola</surname>              <given-names>B.L.</given-names>            </name>            <name>              <surname>Lagier</surname>              <given-names>J.-C.</given-names>            </name>            <name>              <surname>Raoult</surname>              <given-names>D.</given-names>            </name>          </person-group>          <article-title>Spreading of a new SARS-CoV-2 N501Y spike variant in a new lineage</article-title>          <source>Clin. Microbiol. Infect.</source>          <year>2021</year>          <volume>12</volume>          <fpage>1198</fpage>          <pub-id pub-id-type="doi">10.1016/j.cmi.2021.05.006</pub-id>        ) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33900195">33900195</a>) [30](          <person-group person-group-type="author">            <name>              <surname>Davies</surname>              <given-names>N.G.</given-names>            </name>            <name>              <surname>Barnard</surname>              <given-names>R.C.</given-names>            </name>            <name>              <surname>Jarvis</surname>              <given-names>C.I.</given-names>            </name>            <name>              <surname>Kucharski</surname>              <given-names>A.J.</given-names>            </name>            <name>              <surname>Munday</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Pearson</surname>              <given-names>C.A.B.</given-names>            </name>            <name>              <surname>Russell</surname>              <given-names>T.W.</given-names>            </name>            <name>              <surname>Tully</surname>              <given-names>D.C.</given-names>            </name>            <name>              <surname>Abbott</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Gimma</surname>              <given-names>A.</given-names>            </name>            <etal/>          </person-group>          <article-title>Estimated Transmissibility and Severity of Novel SARS-CoV-2 Variant of Concern 202012/01 in England</article-title>          <source>medRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.12.24.20248822</pub-id>        ) [31](          <person-group person-group-type="author">            <name>              <surname>Leung</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Shum</surname>              <given-names>M.H.</given-names>            </name>            <name>              <surname>Leung</surname>              <given-names>G.M.</given-names>            </name>            <name>              <surname>Lam</surname>              <given-names>T.T.</given-names>            </name>            <name>              <surname>Wu</surname>              <given-names>J.T.</given-names>            </name>          </person-group>          <article-title>Early transmissibility assessment of the N501Y mutant strains of SARS-CoV-2 in the United Kingdom, October to November 2020</article-title>          <source>Eurosurveillance</source>          <year>2021</year>          <volume>26</volume>          <fpage>2002106</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2020.26.1.2002106</pub-id>        ) [32](          <person-group person-group-type="author">            <name>              <surname>Tegally</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Wilkinson</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Giovanetti</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Iranzadeh</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Fonseca</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Giandhari</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Doolabh</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Pillay</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>San</surname>              <given-names>E.J.</given-names>            </name>            <name>              <surname>Msomi</surname>              <given-names>N.</given-names>            </name>            <etal/>          </person-group>          <article-title>Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) line-age with multiple spike mutations in South Africa</article-title>          <source>medRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.12.21.20248640</pub-id>        ) [33](          <person-group person-group-type="author">            <name>              <surname>Faria</surname>              <given-names>N.R.</given-names>            </name>            <name>              <surname>Claro</surname>              <given-names>I.M.</given-names>            </name>            <name>              <surname>Candido</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Moyses Franco</surname>              <given-names>L.A.</given-names>            </name>            <name>              <surname>Andrade</surname>              <given-names>P.S.</given-names>            </name>            <name>              <surname>Coletti</surname>              <given-names>T.M.</given-names>            </name>            <name>              <surname>Silva</surname>              <given-names>C.A.M.</given-names>            </name>            <name>              <surname>Sales</surname>              <given-names>F.C.</given-names>            </name>            <name>              <surname>Manuli</surname>              <given-names>E.R.</given-names>            </name>            <name>              <surname>Aguiar</surname>              <given-names>R.S.</given-names>            </name>            <etal/>          </person-group>          <article-title>Genomic Characterisation of an Emergent SARS-CoV-2 Lineage in Manaus: Preliminary Findings</article-title>          <source>Virological</source>          <year>2021</year>          <fpage>1</fpage>          <lpage>9</lpage>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586">https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-26">(accessed on 26 May 2021)</date-in-citation>        ) [34](          <person-group person-group-type="author">            <name>              <surname>Cherian</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Potdar</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Jadhav</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Yadav</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Gupta</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Das</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Das</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Agarwal</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Singh</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Abraham</surname>              <given-names>P.</given-names>            </name>            <etal/>          </person-group>          <article-title>Convergent evolution of SARS-CoV-2 spike mutations, L452R, E484Q and P681R, in the second wave of COVID-19 in Maharashtra, India</article-title>          <source>MedRxiv</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.1101/2021.04.22.440932</pub-id>        ) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33785459">33785459</a>) [12](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Domman</surname>              <given-names>D.B.</given-names>            </name>            <name>              <surname>Snyder</surname>              <given-names>D.J.</given-names>            </name>            <name>              <surname>Oguntuyo</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Van Diest</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Densmore</surname>              <given-names>K.H.</given-names>            </name>            <name>              <surname>Schwalm</surname>              <given-names>K.C.</given-names>            </name>            <name>              <surname>Femling</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Carroll</surname>              <given-names>J.L.</given-names>            </name>            <name>              <surname>Scott</surname>              <given-names>R.S.</given-names>            </name>            <etal/>          </person-group>          <article-title>Emergence in late 2020 of multiple lineages of SARS-CoV-2 Spike protein variants affecting amino acid position 677</article-title>          <source>MedRxiv</source>          <year>2021</year>          <fpage>21251658</fpage>          <pub-id pub-id-type="doi">10.1101/2021.02.12.21251658</pub-id>        ) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33785459">33785459</a>) [12](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Domman</surname>              <given-names>D.B.</given-names>            </name>            <name>              <surname>Snyder</surname>              <given-names>D.J.</given-names>            </name>            <name>              <surname>Oguntuyo</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Van Diest</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Densmore</surname>              <given-names>K.H.</given-names>            </name>            <name>              <surname>Schwalm</surname>              <given-names>K.C.</given-names>            </name>            <name>              <surname>Femling</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Carroll</surname>              <given-names>J.L.</given-names>            </name>            <name>              <surname>Scott</surname>              <given-names>R.S.</given-names>            </name>            <etal/>          </person-group>          <article-title>Emergence in late 2020 of multiple lineages of SARS-CoV-2 Spike protein variants affecting amino acid position 677</article-title>          <source>MedRxiv</source>          <year>2021</year>          <fpage>21251658</fpage>          <pub-id pub-id-type="doi">10.1101/2021.02.12.21251658</pub-id>        ) [45](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32225176">32225176</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34332650">34332650</a> <b>(3)</b> Memory B cell repertoire for recognition of evolving SARS-CoV-2 spike<ul><li>(B) Heatmap showing binding of 119 mAbs to <b>Nextstrain</b> cluster 20A.EU1 (A222V), Danish mink variant (&#x394;69-70 and Y453F), B.1.1.7 (&#x394;69-70, &#x394;144, N501Y, A570D, P681H, T716I, S982A, and D1118H), B.1.351 (L18F, D80A, D215G, &#x394;242-244, K417N, E484K, N501Y, and A701V), and P.1 (L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, H655Y, T1027I, and D1176F) (top), and NTD deletion variants (bottom).<li>Flow plots for binding of 7 mAbs to <b>Nextstrain</b> cluster 20A.EU1 (A222V), Danish mink variant (&#x394;69-70 and Y453F), B.1.1.7 (&#x394;69-70, &#x394;144, N501Y, A570D, P681H, T716I, S982A, D1118H), B.1.351 (L18F, D80A, D215G, &#x394;241-243, K417N, E484K, N501Y, A701V), P.1 (L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, H655Y, T1027I, D1176F) and NTD deletion variants.<li> 158762), recurring NTD deletions as described (<xref rid="bib24" ref-type="bibr"><u>McCarthy et&#xA0;al., 2021</u></xref>) (&#x394;69-70, &#x394;141-144, &#x394;144, &#x394;146, &#x394;210, &#x394;243-244), <b>Nextstrain</b> cluster 20A.EU1 (A222V), Danish mink variant (&#x394; 69-70 and Y453F), UK B.1.1.7 (&#x394;69-70, &#x394;144, N501Y, A570D, P681H, T716I, S982A, D1118H), SA B.1.351 (L18F, D80A, D215G, &#x394;242-244, K417N, E484K, N501Y, A701V) and P.1 (L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, H655Y, T1027I, D1176F).<li> [McCarthy et&#xA0;al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33536258">33536258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36700092">36700092</a> <b>(3)</b> A novel method of literature mining to identify candidate COVID-19 drugs<ul><li> Since the outbreak of SARS-CoV-2 in December 2019, the most mutated strain has accumulated 39 mutations within 13&#x2009;months, as seen on the <b>Nextstrain</b> (<xref rid="vbab013-B2" ref-type="bibr">Hadfield <italic toggle="yes">et al.<li> Since the outbreak of SARS-CoV-2 in December 2019, the most mutated strain has accumulated 39 mutations within 13&#x2009;months, as seen on the <b>Nextstrain</b> (<xref rid="vbab013-B2" ref-type="bibr">Hadfield <italic toggle="yes">et al.</italic>, 2018</xref>) web page (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34294731">34294731</a> <b>(1)</b> resistancebank.org, an open-access repository for surveys of antimicrobial resistance in animals<ul><li> These include, amongst others, Microreact<sup><xref ref-type="bibr" rid="CR59"><u>59</u></xref></sup> and <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR60"><u>60</u></xref></sup>, and are complementary to the phenotypic information provided on <italic>resistancebank.org</italic>.<li> [59](          <person-group person-group-type="author">            <name>              <surname>Argim&#xF3;n</surname>              <given-names>S</given-names>            </name>            <etal/>          </person-group>          <article-title>Microreact: visualizing and sharing data for genomic epidemiology and phylogeography</article-title>          <source>Microb. genomics</source>          <year>2016</year>          <volume>2</volume>          <fpage>1</fpage>          <lpage>11</lpage>          <pub-id pub-id-type="doi">10.1099/mgen.0.000093</pub-id>        ) [60](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34308277">34308277</a> <b>(9)</b> A benchmarking study of SARS-CoV-2 whole-genome sequencing protocols using COVID-19 patient samples<ul><li> From a global phylogeny generated via <b>Nextstrain</b> (<xref rid="bib17" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) (<xref rid="mmc1" ref-type="supplementary-material"><u>Figure&#xA0;S8</u></xref>A), We found that variants 3 and 7 from Loma Linda (CA, USA) were most related to the cohort from Australia, variant 6 was related to the cohort from MI (USA), variants 5 and 9 were related to the cases from CA, and variants 8 and 10 were mostly related to the cohort from Greece.<li> 2009</xref></td><td><ext-link ext-link-type="uri" xlink:href="http://source.jalview.org/gitweb/?p=jalview.git;a=summary" id="intref0075">http://source.jalview.org/gitweb/?p=jalview.git;a=summary</ext-link></td></tr><tr><td><b>Nextstrain</b>.<li> 2018</xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0080">https://github.com/<b>nextstrain</b>/ncov</ext-link>.<li>Phylogenetic analysis was performed using <b>Nextstrain</b> pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0135">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="bib17" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) on a local Linux machine.<li> The <b>Nextstrain</b> pipeline was run using the combined 417 public and 5 LLU SARS-CoV-2 WGS sequencing data.<li> The output JASON file from <b>Nextstrain</b> pipeline was viewed using auspice (<ext-link ext-link-type="uri" xlink:href="https://auspice.us" id="intref0145">https://auspice.us</ext-link>) (<xref rid="bib17" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) (<xref rid="mmc1" ref-type="supplementary-material"><u>Figure&#xA0;S8</u></xref>).<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> A benchmarking study of SARS-CoV-2 whole-genome sequencing protocols using COVID-19 patient samples<ul><li> We used the sequence subsample from <b>Nextstrain</b> (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.<b>nextstrain</b>.org/" ext-link-type="uri">www.<b>nextstrain</b>.org</ext-link>) global analysis as of August 15, 2020.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34298174">34298174</a> <b>(4)</b> Limited capacity of SARS-CoV-2 variants testing in Japan: A secondary analysis using publicly available data<ul><li> According to recent data from the genomic surveillance reported to the GISAID platform (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/" id="PC_link7mwKXGezZW">https://<b>nextstrain</b>.org/ncov/</ext-link>), the variants of concern alpha and delta are currently present in Japan, in addition to the clades 19A, 20C, and 20B (<xref rid="fig1" ref-type="fig">Fig.<li>Genomic sequences analyzed by GISAID from Japan, July 18, 2021 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global?f_country=Japan" id="intref0025">https://<b>nextstrain</b>.org/ncov/gisaid/global?f_country=Japan</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34354952">34354952</a> <b>(0)</b> Case Report: Discovery a Novel SARS-CoV-2 Variant in a Six-Months Long-Term Swab Positive Female Suffering From Non-Hodgkin Lymphoma<ul><li> Furthermore, Nextclade (<uri xlink:type="simple" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>) and PANGO (<uri xlink:type="simple" xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</uri>) free tools were used to identify the clade or lineage, respectively, which are defined by specific mutation signatures.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34098050">34098050</a> <b>(1)</b> Healthcare workers acquired COVID-19 disease from patients? An investigation by phylogenomics<ul><li>WGS revealed that all samples from the HCWs and patients were clustered into the same SARS-CoV-2 lineage (Pangolin lineage B.1.36.27/GISAID clade GH/<b>Nextstrain</b> clade 20A), which predominated in the recent wave of COVID-19 outbreak in Hong Kong.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34260066">34260066</a> <b>(4)</b> SARS&#x2010;CoV&#x2010;2 reinfection in a cancer patient with a defective neutralizing humoral response<ul><li> Clade information as inferred by <b>Nextstrain</b> and Pangolin nomenclatures is shown.<li> Whole&#x2010;genome sequencing and phylogenetic analysis of RNA from the first two specimens showed that the viral genome found at Day 0 could be grouped in the <b>Nextstrain</b> clade 20B and Pangolin lineage B.1.1, while the strain isolated on Day 110 belonged to the <b>Nextstrain</b> clade 20A and Pangolin lineage B.1 (Figure&#xA0;<xref rid="jmv27200-fig-0001" ref-type="fig"><u>1B</u></xref>).<li> However, when we sequenced the RNA from the third sample harvested on Day 115, we detected again the <b>Nextstrain</b> clade 20B, suggesting that the first infection strain had never been cleared completely.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34281387">34281387</a> <b>(3)</b> Evolutionary Tracking of SARS-CoV-2 Genetic Variants Highlights an Intricate Balance of Stabilizing and Destabilizing Mutations<ul><li>The availability of whole-genome sequences of SARS-CoV-2 in public repositories such as Global Initiative on Sharing All Influenza Data (GISAID) and real-time data visualization pipeline <b>NextStrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>) offers a great opportunity for scientists to track the evolutionary path of this virus (<xref rid="B11" ref-type="bibr"><u>11</u></xref>, <xref rid="B12" ref-type="bibr"><u>12</u></xref>).<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34372615">34372615</a> <b>(0)</b> Viral Load and Patterns of SARS-CoV-2 Dissemination to the Lungs, Mediastinal Lymph Nodes, and Spleen of Patients with COVID-19 Associated Lymphopenia<ul><li>Identification of SARS-CoV-2 subtype was carried out by comparing sequence data with the reference genome on the Nextclade v1.1.0 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 20 April 2021) platform.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34281608">34281608</a> <b>(2)</b> SARS-CoV-2: from its discovery to genome structure, transcription, and replication<ul><li> The SARS&#x2010;CoV&#x2010;2 genome exhibits significant genetic diversity since its discovery (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2">https://<b>nextstrain</b>.org/sars-cov-2</ext-link>) and has displayed over 7123 unique single nucleotide mutations/modifications among 12,754 complete US genome sequences by September 11, 2020 [<xref ref-type="bibr" rid="CR32"><u>32</u></xref>], or 29% of the genome positions over forty thousand SARS-CoV-2 genomes worldwide [<xref ref-type="bibr" rid="CR33"><u>33</u></xref>].<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33589648">33589648</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33045727">33045727</a>) </ul></p>
</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34241897">34241897</a> <b>(4)</b> Genomic monitoring unveil the early detection of the SARS&#x2010;CoV&#x2010;2 B.1.351 (beta) variant (20H/501Y.V2) in Brazil<ul><li>A representative subset of 3852 genomes obtained from GISAID<xref rid="jmv27190-bib-0009" ref-type="bibr"><sup>9</sup></xref> was obtained following the <b>Nextstrain</b> guidelines (<ext-link xlink:href="https://<b>nextstrain</b>.github.io/ncov/customizing-analysis.html" ext-link-type="uri">https://<b>nextstrain</b>.github.io/ncov/customizing-analysis.html</ext-link>).<li> These sequences were separated by downsampling as implied by <b>Nextstrain</b> using the most suitable parameters like space and sequence distribution.<li>Mutational profile was investigated by using the Nextclade tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) to describe substitutions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34405126">34405126</a> <b>(3)</b> Molecular Diagnosis of SARS-CoV-2: Assessing and Interpreting Nucleic Acid and Antigen Tests<ul><li> Since the emergence of SARS-CoV-2, the Global Initiative on Sharing Avian Influenza Data (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/about-us/history/">GISAID</ext-link>), <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>Nextstrain</b></ext-link>, the National Center for Biotechnology Information (<ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/sars-cov-2/">NCBI</ext-link>), and the China National Center for Bioinformation (<ext-link ext-link-type="uri" xlink:href="https://bigd.big.ac.cn/ncov/?lang=en">CNCB</ext-link>) have been aggregating full and partial SARS-CoV-2 genome sequence from infections around the world.<li> the most recent updates on naming strategies have been written by Trevor Bedford and <b>Nextstrain</b> colleagues [<xref rid="R63" ref-type="bibr"><u>63</u></xref>].<li> [63](<person-group person-group-type="author"><string-name><surname>Bedford</surname><given-names>T</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <article-title>Neher TBaRA, editor2021</article-title>. [<date-in-citation>cited 2021</date-in-citation>]. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming">https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34273961">34273961</a> <b>(1)</b> <italic>HAVoC</italic>, a bioinformatic pipeline for reference-based consensus assembly and lineage assignment for SARS-CoV-2 sequences<ul><li> To identify to what variant a sequenced SARS-CoV-2 belongs to, it is usually analyzed and classified with pangolin [<xref ref-type="bibr" rid="CR15"><u>15</u></xref>] or Nextclade, which is a part of <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR16"><u>16</u></xref>].<li> [15](O&#x2019;Toole &#xC1;, Scher E, Underwood A, Jackson B, Hill V, McCrone JT, et al. pangolin: lineage assignment in an emerging pandemic as an epidemiological tool. <ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</ext-link>. Accessed 12 Feb 2021.) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34272947">34272947</a> <b>(1)</b> An Update on Severe Acute Respiratory Syndrome Coronavirus 2 Diversity in the US National Capital Region: Evolution of Novel and Variants of Concern<ul><li> Clades were determined using Nextclade beta version 0.12.0 (clades.<b>nextstrain</b>.org) [<xref rid="CIT0026" ref-type="bibr"><u>26</u></xref>], and lineages were determined with the Pangolin coronavirus disease 2019 (COVID-19) lineage assigner (COG-UK; cog-uk.io).<li> The tree was generated using <b>Nextstrain</b> [<xref rid="CIT0026" ref-type="bibr"><u>26</u></xref>].<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34337269">34337269</a> <b>(6)</b> Conformational Variability Correlation Predictionof Transmissibility and Neutralization Escape Ability for MultipleMutation SARS-CoV-2 Strains using SSSCPreds<ul><li>After March 2020, only two mutations, RNA-directed RNA polymeraseP323L (ORF1ab P4715L, ORF1b P314L) and spike protein D614G had nearlyoverwhelmed the original mutation sites (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614">https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614</uri>).<li> B.1.1.7 is prevalent all over the world, but the N501Y frequencyof phylogeny is about 60&#x2013;70% (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=gt-S_501">https://<b>nextstrain</b>.org/ncov/global?c=gt-S_501</uri>).<li> As for the receptor-binding motif (RBM) of spike protein, beforeB.1.1.7, B.1.351, and P.1, the mutation of S477N has expanded in Australiaand in Europe but has not led to an extension of the pandemic (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=gt-S_477">https://<b>nextstrain</b>.org/ncov/global?c=gt-S_477</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34464903">34464903</a> <b>(0)</b> Diagnostic pre-screening method based on N-gene dropout or delay to increase feasibility of SARS-CoV-2 VOC B.1.1.7 detection<ul><li> Clade assignment and annotated mutations are generated through the nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0002">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34274478">34274478</a> <b>(4)</b> No association between the SARS-CoV-2 variants and mortality rates in the Eastern Mediterranean Region<ul><li> For clade analysis, we used the Nextclade tool from the <b>Nextstrain</b> project (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir015">https://clades.<b>nextstrain</b>.org/</ext-link>) (<xref rid="b0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and GISAID clade designation provided by CoVsurver analysis.<li>Next, we used the Nextclade tool from <b>Nextstrain</b> to assign clades (GISAID nomenclature) to the sequences (<xref rid="f0015" ref-type="fig">Fig.<li> Of all sequences from EMR countries, 65.8% belonged to the G (20A per <b>Nextstrain</b> clades), GR (20B), GH (20C) and GV (20A.EU1) clades, which emerged following the introduction of the S_D614G variant (<xref rid="f0015" ref-type="fig">Fig.<li> Red arrow points to the introduction of S_D614G, marking the emergence of G, GR and GH clades (reproduced from <b>Nextstrain</b>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> No association between the SARS-CoV-2 variants and mortality rates in the Eastern Mediterranean Region<ul><li>The clades were analyzed with Next Strain classification for SARS&#x2010;CoV&#x2010;2 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/ncov" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov</ext-link>) and clade&#x2010;specific nonsynonymous mutations were identified.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(6)</b> No association between the SARS-CoV-2 variants and mortality rates in the Eastern Mediterranean Region<ul><li>Pango&#xA0;lineage<xref rid="jmv27188-tbl1-note-0002" ref-type="table-fn"><sup>a</sup></xref>&#x2013;<b>Nextstrain</b><xref rid="jmv27188-tbl1-note-0003" ref-type="table-fn"><sup>b</sup></xref> name</th>              <th align="left" valign="bottom" rowspan="1" colspan="1">First detected country</th>              <th align="left" valign="bottom" rowspan="1" colspan="1">Sequences isolated in Turkey and their VOC distribution, <italic toggle="yes">n</italic> (%)</th>            </tr>          </thead>          <tbody valign="top">            <tr>              <td align="left" valign="top" rowspan="1" colspan="1">B.1.1.7&#x2013;20I/501Y.V1.<li><b>Nextstrain</b> has organized the variants into clades defined by specific signature mutations.<li> We determine the clade distribution according to the three nomenclature systems (Pango lineage, <b>Nextstrain</b>, and GISAID) to understand the phylogenetic context and genomic diversity of the isolates in relation to global data.<li> Pango lineage,<xref rid="jmv27188-bib-0011" ref-type="bibr"><sup>11</sup></xref> <b>Nextstrain</b><xref rid="jmv27188-bib-0012" ref-type="bibr"><sup>12</sup></xref> clade, and GISAID<xref rid="jmv27188-bib-0026" ref-type="bibr"><sup>26</sup></xref> clade assignments were carried out using the online tools of the respective nomenclature systems.<li>GISAID and <b>Nextstrain</b> clade distribution.<li> These sequences were grouped into two main <b>Nextstrain</b> clades.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34274369">34274369</a> <b>(4)</b> A one-step real-time RT-PCR assay for simultaneous typing of SARS-CoV-2 mutations associated with the E484K and N501Y spike protein amino-acid substitutions<ul><li> Specifically, the B.1.1.7 lineage (Alpha variant, or variant V1/20I <b>Nextstrain</b> clade; 17 amino-acid substitutions) emerged in southeast England in November 2020 and rapidly spread towards fixation.<li> The B.1.351 lineage (Beta variant, V2/20H <b>Nextstrain</b> clade; 17 amino-acid substitutions) was initially reported in South Africa in December 2020.<li> Additionally, the P.1 lineage (Gamma variant, V3/20 J <b>Nextstrain</b> clade; 17 amino-acid substitutions) was reported in Brazil in January 2021 (<xref rid="bib0005" ref-type="bibr"><u>Abdool Karim and de Oliveira, 2021</u></xref>).<li> It should be also noted that, besides Beta and Gamma variants, both aforementioned MOCs are also present in strains belonging to the B.1.621 lineage (<b>Nextstrain</b> clade 21H).<li> [Abdool Karim and de Oliveira, 2021](          <person-group person-group-type="author">            <name>              <surname>Abdool Karim</surname>              <given-names>S.S.</given-names>            </name>            <name>              <surname>de Oliveira</surname>              <given-names>T.</given-names>            </name>          </person-group>          <article-title>New SARS-CoV-2 variants &#x2014; clinical, public health, and vaccine implications</article-title>          <source>N. Engl. J. Med.</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.1056/NEJMc2100362</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34272411">34272411</a> <b>(3)</b> Invasion, establishment, and spread of invasive mosquitoes from the <italic>Culex coronator</italic> complex in urban areas of Miami-Dade County, Florida<ul><li>Phylogenomic analyses were carried out using <b>Nextstrain</b>, a suite of tools that includes subsampling, alignment, phylogenetic reconstruction, geographic and ancestral trait reconstruction, and inference of transmission events [<xref rid="CIT0031" ref-type="bibr"><u>31</u></xref>].<li> For these analyses, our dataset of 340 local genome sequences was concatenated with the <b>Nextstrain</b> South America dataset (build 2021-03-03) (Supplementary Data 2).<li> In order to visualize the <b>Nextstrain</b> build, the JSON file should be loaded at the following link: <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://auspice.us/" ext-link-type="uri">https://auspice.us/</ext-link>.<li> [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34358046">34358046</a> <b>(1)</b> West Nile Virus in Brazil<ul><li>A.1 (<b>Nextstrain</b> 19B).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36793768">36793768</a> <b>(10)</b> Recombinant SARS-CoV-2 genomes circulated at low levels over the first year of the pandemic<ul><li> of which the most widely used are the <b>Nextstrain</b> (<xref rid="R20" ref-type="bibr"><b>Nextstrain</b>: Genomic Epidemiology of Novel Coronavirus 2021</xref>), GISAID (<xref rid="R5" ref-type="bibr"><u>GISAID 2021</u></xref>), and Pangolin (<xref rid="R23" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) systems.<li> While Pangolin provides a fine-scaled classification of viral diversity, <b>Nextstrain</b> and GISAID provide broad-scale classification schemes that cluster SARS-CoV-2 genomes into a relatively small number of distinct clades, particularly during the first year of the virus&#x2019;s circulation.<li> Here, we relied on <b>Nextstrain</b>&#x2019;s classification scheme to detect genomes collected prior to mid-February 2021 that are potentially recombinant, but increase its resolution to 14 monophyletic clades with &#x2265;50&#x2009;per cent bootstrap support to expand the number of variant sites that contribute to our detection of potential recombinant genomes (<xref rid="F1" ref-type="fig">Fig.<li> Fourteen monophyletic clades with &#x2265;50&#x2009;per cent bootstrap support were identified and named based on <b>Nextstrain</b> nomenclature (<xref rid="R8" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> Clade boundaries are in full agreement with <b>Nextstrain</b> clades (e.g.<li> Assuming a substitution rate of 8&#x2009;&#xD7;&#x2009;10<sup>&#x2212;4</sup> substitutions per site per year (<xref rid="R20" ref-type="bibr"><b>Nextstrain</b>: Genomic Epidemiology of Novel Coronavirus 2021</xref>), corresponding to a substitution rate of <italic toggle="yes">s&#x2009;</italic>=&#x2009;0.066 substitutions per genome per day, a clonal evolutionary ancestor would be expected to be a distance of <italic toggle="yes">st</italic> nucleotides away from the focal sequence if collected <italic toggle="yes">t</italic> days prior to the focal sequence.<li> Clades were named according to the <b>Nextstrain</b> clade to which they belong, with added suffixes of &#x2018;-1&#x2019;, &#x2018;-2&#x2019;, etc.<li> to denote clades at finer resolution than those available under the <b>Nextstrain</b> system.<li> The nucleotide distance threshold was calculated as the maximum distance that would be expected in &gt;99&#x2009;per cent of cases given a substitution rate of 8&#x2009;&#xD7;&#x2009;10<sup>&#x2212;4</sup> substitutions per site per year (<xref rid="R20" ref-type="bibr"><b>Nextstrain</b>: Genomic Epidemiology of Novel Coronavirus 2021</xref>), within a time frame of 50&#x2009;days prior to the collection date of the potential recombinant.<li> [GISAID 2021](<person-group person-group-type="author"><collab>Global Initiative on Sharing All Influenza Data (GISAID)</collab></person-group>. (<year>2021</year>) <article-title>Clade and Lineage Nomenclature Aids in Genomic Epidemiology Studies of Active hCoV-19 Viruses</article-title>. &lt;<ext-link xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>&gt; <comment>accessed 11 Jan 2020</comment>.) [Rambaut et al., 2020](<person-group person-group-type="author"><string-name><surname>Rambaut</surname>&#xA0;<given-names>A.</given-names></string-name></person-group>&#xA0;<etal>et al.</etal> (<year>2020</year>) &#x2018;<article-title>A Dynamic Nomenclature Proposal for SARS-CoV-2 Lineages to Assist Genomic Epidemiology</article-title>&#x2019;, <source><italic toggle="yes">Nature Microbiology</italic></source>, <volume>5</volume>: <fpage>1403</fpage>&#x2013;<lpage>7</lpage>.doi: <pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id>.) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34265245">34265245</a> <b>(1)</b> The success of SARS-CoV-2 vaccines and challenges ahead<ul><li>To address the complex naming convention for the variants, on the basis of lineages defined by the Global Initiative on Sharing All Influenza Data (GISAID), Pango, and <b>Nextstrain</b> platforms, the WHO has proposed a simplified nomenclature using the Greek alphabet, which will be updated as needed (<ext-link ext-link-type="uri" xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" id="intref0030">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34261487">34261487</a> <b>(2)</b> SARS-CoV-2: is there neuroinvasion?<ul><li>Scientific name:<break/>Pangolin<break/><b>Nextstrain</b></th><th align="left">Defined by</th><th align="left">Important mutations</th><th align="left">Countries reported (first detected)</th><th align="left">Increased transmissibility</th><th align="left">Increased COVID-19 mortality or severity</th><th align="left">Effect on vaccines</th></tr></thead><tbody><tr><td align="left">UK variant.<li>Bold scientific name is pangolin, non-bold is <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34271187">34271187</a> <b>(4)</b> Genetic and evolutionary analysis of SARS-CoV-2 circulating in the region surrounding Islamabad, Pakistan<ul><li>Demographic characteristics of COVID-19 cases analyzed in the study.</p></caption><alt-text id="al0020">Table 1</alt-text><table frame="hsides" rules="groups"><thead><tr><th><bold>Sample Name</bold></th><th><bold>Date of collection</bold></th><th><bold>Location</bold></th><th><bold>Gender</bold></th><th><bold>Age</bold></th><th><bold>Patient status</bold></th><th><bold>Duration of symptoms</bold></th><th><bold>Other infected family members</bold></th><th><bold>Travel history within Pakistan</bold></th><th><bold>Ct value</bold></th><th><bold>Quantity of amplified product (ng/ul)</bold></th><th><bold>Quantity of indexed product (ng/ul)</bold></th><th><bold>Row reads</bold></th><th><bold>Coverage</bold></th><th><bold>Depth</bold></th><th><bold><b>Nextstrain</b> clade</bold></th><th><bold>No of Mutations</bold></th><th><bold>Pangolin lineage assiged</bold></th><th><bold>Genbank Accession number</bold></th></tr></thead><tbody><tr><td>hC0V-19/pakistan/NIH-421800/2020.<li> Resulting consensus sequences were analyzed through PANGOLIN v2.1.7 (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin/" id="ir0025">https://github.com/cov-lineages/pangolin/</ext-link>) and <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/ncov" id="ir0030">http://<b>nextstrain</b>.org/ncov</ext-link>) (<xref rid="bb0045" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Genetic and evolutionary analysis of SARS-CoV-2 circulating in the region surrounding Islamabad, Pakistan<ul><li> Visualization was achieved with Auspice and <b>Nextstrain</b> [<xref rid="CIT0011" ref-type="bibr"><u>11</u></xref>].<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(7)</b> Genetic and evolutionary analysis of SARS-CoV-2 circulating in the region surrounding Islamabad, Pakistan<ul><li> Platforms like <b>Nextstrain</b> (<xref rid="R7" ref-type="bibr">Hadfield et al.<li> Specifically, we are maintaining and openly sharing a daily-updated database of mutation-annotated trees (MATs) containing global SARS-CoV-2 sequences from public databases, including annotations for <b>Nextstrain</b> clades (<xref rid="R7" ref-type="bibr">Hadfield et al.<li> We also use the <italic toggle="yes">matUtils annotate</italic> command (see <xref rid="SD2" ref-type="supplementary-material"><u>Supplementary Methods</u></xref>) to add <b>Nextstrain</b> clade and Pango lineage annotations to individual branches of our MAT.<li> As of June 9, 2021, our MAT consists of 834,521 sequences, includes 14 <b>Nextstrain</b> clade and 895 Pango lineage annotations for all samples, and is only 65 MB, or 14 MB in its gzip-compressed form (<xref rid="SD2" ref-type="supplementary-material"><u>Supplementary Figure 1</u></xref>, <xref rid="SD2" ref-type="supplementary-material"><u>Supplementary Table S1</u></xref>).<li> <b>Nextstrain</b> (<xref rid="R7" ref-type="bibr">Hadfield et al.<li> Both ways of annotating ensure that the clades remain monophyletic, but we use the second approach to label <b>Nextstrain</b> clades and Pango lineages in our SARS-CoV-2 MAT database since it can be automated using available data (see <xref rid="SD2" ref-type="supplementary-material"><u>Supplementary Methods</u></xref>).<li> <italic toggle="yes">matUtils annotate</italic> has high congruence with <b>Nextstrain</b> clades and Pango lineage annotations (<xref rid="SD2" ref-type="supplementary-material"><u>Supplementary Table S1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34352474">34352474</a> <b>(3)</b> COVID-19, the first pandemic in the post-genomic era<ul><li><bold>(a)</bold> Daily counts of genome assemblies (y-axis) shared on GISAID (current to 5/7/2021) over the course of the pandemic for all <b>NextStrain</b> clades.<li> This resource has supported several initiatives allowing visualisation and mutation characterisation including <b>NextStrain</b> [<xref rid="bib0100" ref-type="bibr"><u>20</u></xref>], CoVizu [<xref rid="bib0105" ref-type="bibr"><u>21</u></xref>], <ext-link ext-link-type="uri" xlink:href="http://cov-lineages.org" id="intr0020">cov-lineages.org</ext-link> [<xref rid="bib0110" ref-type="bibr">22<sup>&#x2022;&#x2022;</sup></xref>], outbreak.info (<ext-link ext-link-type="uri" xlink:href="https://outbreak.info/" id="intr0025">https://outbreak.info/</ext-link>) and CoVariants (<ext-link ext-link-type="uri" xlink:href="https://covariants.org" id="intr0030">https://covariants.org</ext-link>).<li> with both GISAID [<xref rid="bib0090" ref-type="bibr"><u>18</u></xref>,<xref rid="bib0095" ref-type="bibr"><u>19</u></xref>] and <b>NextStrain</b> [<xref rid="bib0100" ref-type="bibr"><u>20</u></xref>] providing schemes (<xref rid="fig0010" ref-type="fig"><u>Figure 2</u></xref>).<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](          <person-group person-group-type="author">            <name>              <surname>Poon</surname>              <given-names>A.F.Y.</given-names>            </name>          </person-group>          <article-title>CoVizu: rapid analysis and visualization of the global diversity of SARS-CoV-2 genomes - SARS-CoV-2 coronavirus/software and tools - virological</article-title>          <source>Virological</source>          <year>2021</year>          <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/covizu-rapid-analysis-and-visualization-of-the-global-diversity-of-sars-cov-2-genomes/678" id="intr0050">https://virological.org/t/covizu-rapid-analysis-and-visualization-of-the-global-diversity-of-sars-cov-2-genomes/678</ext-link>        ) [18](          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G.</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Glob Challenges</source>          <volume>1</volume>          <year>2017</year>          <fpage>33</fpage>          <lpage>46</lpage>        ) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34255632">34255632</a> <b>(29)</b> TNet: Transmission Network Inference Using Within-Host Strain Diversity and its Application to Geographical Tracking of COVID-19 Spread<ul><li> These include BEASTlier <xref rid="ref18" ref-type="bibr"><u>[18]</u></xref>, SCOTTI <xref rid="ref5" ref-type="bibr"><u>[5]</u></xref>, phybreak <xref rid="ref21" ref-type="bibr"><u>[21]</u></xref>, TransPhylo <xref rid="ref7" ref-type="bibr"><u>[7]</u></xref>, phyloscanner <xref rid="ref37" ref-type="bibr"><u>[37]</u></xref>, <b>Nextstrain</b>/Augur <xref rid="ref17" ref-type="bibr"><u>[17]</u></xref>, and BadTrIP <xref rid="ref4" ref-type="bibr"><u>[4]</u></xref>.<li> Using available epidemiological ground truth data, we found that the COVID-19 transmission network inferred using our framework was significantly more accurate than the corresponding network inferred by the popular <b>Nextstrain</b> tool <xref rid="ref17" ref-type="bibr"><u>[17]</u></xref>.<li> For example, the popular <b>Nextstrain</b> tool (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>) computes and provides a regularly updated SARS-CoV-2 phylogeny and associated transmission network between geographical regions <xref rid="ref17" ref-type="bibr"><u>[17]</u></xref>.<li> We compared the resulting transmission networks against those inferred by the widely used <b>Nextstrain</b> tool, evaluating inference accuracy using the available epidemiological information about country of exposure for each SARS-CoV-2 genome used in the analysis.<li> The resulting 11 phylogenies were rooted and dated using TreeTime <xref rid="ref28" ref-type="bibr"><u>[28]</u></xref>, which is also used by the <b>Nextstrain</b> pipeline.<li>We applied TNet, random sampling, and the <b>Nextstrain</b>/Augur tool to this dataset to infer international (country-to-country) transmission networks.<li> Since <b>Nextstrain</b>/augur is not based on sampling, we computed the geographical transmission network for <b>Nextstrain</b> by using the maximum likelihood phylogeny from RAxML.<li><italic>Overall Accuracies of the Methods Based on Ground-Truth.</italic> By comparing the international transmission networks inferred by the three methods against the known country of exposure available for each SARS-CoV-2 sequence, we found that TNet significantly outperformed <b>Nextstrain</b> and that random sampling dramatically outperformed both <b>Nextstrain</b> and TNet.<li> Specifically, <b>Nextstrain</b>, TNet, and random sampling were able to correctly determine the country of exposure correctly for 67, 71, and 85 percent of the sequences, respectively.<li>Accuracy of <b>Nextstrain</b>, TNet, and random sampling on the COVID-19 dataset based on the known country of exposure available for each SARS-CoV-2 sequence.<li> Surprisingly, TNet still outperforms <b>Nextstrain</b> in this analysis.<li> On the other hand, while <b>Nextstrain</b> infers that the majority of spread from China occurred in January 2020, TNet and random sampling both infer that the majority of the spread from China occurred in December 2019.<li> Notably, both TNet and random sampling identity USA as an early and important contributor to the spread of COVID-19, while <b>Nextstrain</b> does not include USA in its top five list until March 2020.<li>Top five spreader countries inferred by <b>Nextstrain</b>, TNet, and Random Sampling during each month from December 2019 through April 2020.<li>Top five receiver countries inferred by <b>Nextstrain</b>, TNet, and Random Sampling during each month from December 2019 through April 2020.<li> S1, available online), we find that there is generally good agreement between the top five lists of TNet and random sampling for the months December 2019 through February 2020, but that they have significant differences from the top five lists inferred by <b>Nextstrain</b> for the same periods.<li> However, as with spreader countries, there are considerable differences between the top five countries for each period inferred by <b>Nextstrain</b> and those inferred by TNet or random sampling.<li> (Note that for 10 of the 1801 sequences the state of exposure was a country or state that was not included in our analysis.) As before, we found that TNet significantly outperformed <b>Nextstrain</b> and that random sampling dramatically outperformed both <b>Nextstrain</b> and TNet.<li> Specifically, <b>Nextstrain</b>, TNet, and random sampling were able to correctly determine the state of exposure correctly for 65, 73, and 86 percent of the sequences, respectively.<li> As these figures show, TNet and random sampling generally agree in their lists of top spreaders and receivers, but that those lists differ significantly from those inferred by <b>Nextstrain</b>.<li> For instance, <b>Nextstrain</b> infers Virginia and Pennsylvania as the top two spreader states during February 2020, but these states do not feature in the top five spreader lists for TNet and random sampling during any time period.<li>Top five spreader states in USA inferred by <b>Nextstrain</b>, TNet, and Random Sampling during each month from Dec.<li>Top five receiver states in USA inferred by <b>Nextstrain</b>, TNet, and Random Sampling during each month from Dec.<li> We also show how TNet as well as the closely related random sampling method (also implemented in TNet) can be used to infer geographical transmission networks and our analysis using large-scale COVID-19 data demonstrates how TNet and random sampling both significantly outperform the popular <b>Nextstrain</b>/Augur method.<li> [[18]](<person-group person-group-type="author"><string-name><given-names>M.</given-names><surname>Hall</surname></string-name>, <string-name><given-names>M.</given-names><surname>Woolhouse</surname></string-name>, and <string-name><given-names>A.</given-names><surname>Rambaut</surname></string-name></person-group>, &#x201C;<article-title>Epidemic reconstruction in a phylogenetics framework: Transmission trees as partitions of the node set</article-title>,&#x201D; <source>PLoS Comp. Biol.</source>, vol. <volume>11</volume>, no. <issue>12</issue>, <year>2015</year>, Art. no. e1004613.) [[5]](<person-group person-group-type="author"><string-name><given-names>N.</given-names><surname>De Maio</surname></string-name>, <string-name><given-names>C.-H.</given-names><surname>Wu</surname></string-name>, and <string-name><given-names>D. J.</given-names><surname>Wilson</surname></string-name></person-group>, &#x201C;<article-title>SCOTTI: Efficient reconstruction of transmission within outbreaks with the structured coalescent</article-title>,&#x201D; <source>PLoS Comput. Biol.</source>, vol. <volume>12</volume>, no. <issue>9</issue>, pp. <fpage>1</fpage>&#x2013;<lpage>23</lpage>, <year>2016</year>.) [[21]](<person-group person-group-type="author"><string-name><given-names>D.</given-names><surname>Klinkenberg</surname></string-name>, <string-name><given-names>J. A.</given-names><surname>Backer</surname></string-name>, <string-name><given-names>X.</given-names><surname>Didelot</surname></string-name>, <string-name><given-names>C.</given-names><surname>Colijn</surname></string-name>, and <string-name><given-names>J.</given-names><surname>Wallinga</surname></string-name></person-group>,&#x201C;<article-title>Simultaneous inference of phylogenetic and transmission trees in infectious disease outbreaks</article-title>,&#x201D; <source>PLoS Comput. Biol.</source>, vol. <volume>13</volume>, no. <issue>5</issue>, <year>2017</year>, Art. no. e1005495.) [[7]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28100788">28100788</a>) [[37]](<person-group person-group-type="author"><string-name><given-names>C.</given-names><surname>Wymant</surname></string-name>, <etal>et al.</etal></person-group>, &#x201C;<article-title>PHYLOSCANNER: Inferring transmission from within- and between-host pathogen genetic diversity</article-title>,&#x201D; <source>Mol. Biol. Evol.</source>, vol. <volume>35</volume>, no. <issue>3</issue>, pp. <fpage>719</fpage>&#x2013;<lpage>733</lpage>, <year>2017</year>.) [[17]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[4]](<person-group person-group-type="author"><string-name><given-names>N.</given-names><surname>De Maio</surname></string-name>, <string-name><given-names>C. J.</given-names><surname>Worby</surname></string-name>, <string-name><given-names>D. J.</given-names><surname>Wilson</surname></string-name>, and <string-name><given-names>N.</given-names><surname>Stoesser</surname></string-name></person-group>, &#x201C;<article-title>Bayesian reconstruction of transmission within outbreaks using genomic variants</article-title>,&#x201D; <source>PLoS Comput. Biol.</source>, vol. <volume>14</volume>, no. <issue>4</issue>, pp. <fpage>1</fpage>&#x2013;<lpage>23</lpage>, <year>2018</year>.) [[17]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[17]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[28]](<person-group person-group-type="author"><string-name><given-names>P.</given-names><surname>Sagulenko</surname></string-name>, <string-name><given-names>V.</given-names><surname>Puller</surname></string-name>, and <string-name><given-names>R. A.</given-names><surname>Neher</surname></string-name></person-group>, &#x201C;<article-title>Treetime: Maximum-likelihood phylodynamic analysis</article-title>,&#x201D; <source>Virus Evol.</source>, vol. <volume>4</volume>, no. <issue>1</issue>, <year>2018</year>, Art. no. vex042.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34271202">34271202</a> <b>(1)</b> Monoclonal Antibody Therapy in a Vaccine Breakthrough SARS-CoV-2 Hospitalized Delta (B.1.617.2) Variant Case<ul><li> The proportion of known genetic variants of SARS-CoV-2 strains as reported in <b>NextStrain</b> (version 03-07-2021) from across the world are shown (bottom).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34257311">34257311</a> <b>(1)</b> Reprogrammed CRISPR-Cas13b suppresses SARS-CoV-2 replication and circumvents its mutational escape through mismatch tolerance<ul><li> Analysis of virus phylogenetics globally enabled by the GISAID<sup><xref ref-type="bibr" rid="CR13"><u>13</u></xref></sup> and <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR14"><u>14</u></xref></sup> databases revealed recurrent hotspot mutations in several viral subunits that likely confer selective advantage and/or alter pathogenicity<sup><xref ref-type="bibr" rid="CR15"><u>15</u></xref>,<xref ref-type="bibr" rid="CR16"><u>16</u></xref></sup>.<li> [13](Shu, Y. &amp; McCauley, J. GISAID: Global initiative on sharing all influenza data &#x2013; from vision to reality. <italic>Eurosurveillance</italic> (2017) 10.2807/1560-7917.ES.2017.22.13.30494.) [14](Hadfield, J. et al. NextStrain: Real-time tracking of pathogen evolution. <italic>Bioinformatics</italic> (2018) 10.1093/bioinformatics/bty407.) [15](Young, B. E. et al. Effects of a major deletion in the SARS-CoV-2 genome on the severity of infection and the inflammatory response: an observational cohort study. <italic>Lancet</italic> (2020) 10.1016/s0140-6736(20)31757-8.) [16](Pachetti, M. et al. Emerging SARS-CoV-2 mutation hot spots include a novel RNA-dependent-RNA polymerase variant. <italic>J</italic>. <italic>Transl. Med</italic>. <bold>18</bold>, (2020).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34276267">34276267</a> <b>(4)</b> Variantes de SARS-CoV-2, una historia todav&#xED;a inacabada<ul><li>Denominaci&#xF3;n<break/><b>Nextstrain</b>.<li> que suele hacer referencia al pa&#xED;s o lugar de detecci&#xF3;n (cepa Brit&#xE1;nica), un c&#xF3;digo de la base de datos Pangol&#xED;n<xref rid="bib0225" ref-type="bibr"><sup>4</sup></xref> (B.1.1.7), y uno basado en la p&#xE1;gina <b>Nextstrain</b><xref rid="bib0230" ref-type="bibr"><sup>5</sup></xref> (20I/501Y.V1).<li>Clado <b>Nextstrain</b></th><th align="left"><xref rid="tblfn0015" ref-type="table-fn"><u>*</u></xref>GISAID clado/linaje</th></tr></thead><tbody><tr><td align="left">Variantes de preocupaci&#xF3;n.<li>M&#xE1;s recientemente han suscitado gran alarma las noticias relacionadas con la aparici&#xF3;n y difusi&#xF3;n en India de la nueva variante B.1.617 <xref rid="bib0335" ref-type="bibr"><sup>26</sup></xref>, tambi&#xE9;n conocida como VUI (Variant Under Investigation)-21APR-01 <xref rid="bib0340" ref-type="bibr"><u>27</u></xref>, <xref rid="bib0345" ref-type="bibr"><u>28</u></xref> y 20A seg&#xFA;n &#xAB;<b>Nextstrain</b>&#xBB;<xref rid="bib0240" ref-type="bibr"><sup>7</sup></xref>.<li> [27](GOV.UK. Variants: distribution of cases data. 20 May 2021. Public Health England. [Consultado 24 May 2021]. <ext-link ext-link-type="uri" xlink:href="https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data" id="intr0030">https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data#Variant11</ext-link>.) [28](          <person-group person-group-type="author">            <name>              <surname>Yadav</surname>              <given-names>P.D.</given-names>            </name>            <name>              <surname>Sapkal</surname>              <given-names>G.N.</given-names>            </name>            <name>              <surname>Abraham</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Ella</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Deshpande</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Patil</surname>              <given-names>D.Y.</given-names>            </name>            <etal/>          </person-group>          <article-title>Neutralization of variant under investigation B.1.617 with sera of BBV152 vaccinees</article-title>          <source>Clin Infect Dis.</source>          <volume>ciab411</volume>          <year>2021</year>          <pub-id pub-id-type="doi">10.1093/cid/ciab411</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34372565">34372565</a> <b>(21)</b> Variant Analysis of SARS-CoV-2 Genomes from Belgian Military Personnel Engaged in Overseas Missions and Operations<ul><li> The resulting auspice JSON was processed in baltic v.0.1.5 (<uri xlink:href="https://github.com/evogytis/baltic">https://github.com/evogytis/baltic</uri> accessed on 6 July 2021) to rename clades from <b>Nextstrain</b> nomenclature to pangolin nomenclature (20J/501Y.V3 as P.1, 20H/501Y.V2 as B.1.351, 20I/501Y.V1 as B.1.1.7, 20E (EU1) as B.1.177, 20A.EU2 as B.1.160, 20D as C.16, 20G as B.1.2, and 20F as D.2) and collapsed.<li> Additional clades not yet recognized in <b>Nextstrain</b> nomenclature&#x2014;B.1.617.2, A.23.1, and B.1.526&#x2014;were identified and collapsed as well.<li>These genomes were first analyzed in <b>Nextstrain</b> [<xref rid="B24-viruses-13-01359" ref-type="bibr"><u>24</u></xref>], an open-source project that provides a continuously updated phylogenetic analysis of the evolution and relationships of publicly available SARS-CoV-2 genomes.<li> <b>Nextstrain</b> consists of an automated pipeline that includes subsampling, alignment, maximum-likelihood phylogenetic inference, temporal dating of ancestral nodes, and ancestral geographic location reconstruction.<li> The <b>Nextstrain</b> phylogeny is typically rooted by using the 2019 Wuhan samples as outgroup sequences [<xref rid="B24-viruses-13-01359" ref-type="bibr"><u>24</u></xref>].<li> Given that the <b>Nextstrain</b> application generates a frequency plot over time of all the SARS-CoV-2 lineages that make up its data set, we have created custom frequency plots for the purpose of this study.<li>SARS-CoV-2 variants were assigned to &#x201C;<b>Nextstrain</b> clades&#x201D; using the Nextclade tool [<xref rid="B28-viruses-13-01359" ref-type="bibr"><u>28</u></xref>].<li><xref ref-type="fig" rid="viruses-13-01359-f004"><u>Figure 4</u></xref> shows the global SARS-CoV-2 phylogeny, obtained by augmenting the global <b>Nextstrain</b> build (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri> accessed on 6 July 2021) with the genomes obtained in this study.<li> The three Niger genotypes (Niger 1&#x2013;3) belong to pango lineage A and <b>Nextstrain</b> clade 19B (<xref ref-type="fig" rid="viruses-13-01359-f002"><u>Figure 2</u></xref> and <xref ref-type="fig" rid="viruses-13-01359-f003"><u>Figure 3</u></xref>), which was first observed in travelers from Wuhan in the early days of the outbreak [<xref rid="B4-viruses-13-01359" ref-type="bibr"><u>4</u></xref>] and became common in North America and Europe [<xref rid="B5-viruses-13-01359" ref-type="bibr"><u>5</u></xref>,<xref rid="B35-viruses-13-01359" ref-type="bibr"><u>35</u></xref>,<xref rid="B36-viruses-13-01359" ref-type="bibr"><u>36</u></xref>], but has now declined worldwide (<xref ref-type="fig" rid="viruses-13-01359-f003"><u>Figure 3</u></xref>).<li>The DRC 1 genotype belongs to pango lineage B.1.214 and <b>Nextstrain</b> clade 20A (<xref ref-type="fig" rid="viruses-13-01359-f002"><u>Figure 2</u></xref> and <xref ref-type="fig" rid="viruses-13-01359-f003"><u>Figure 3</u></xref>) and contains the signature mutations P314L in ORF1b and D614G in the spike (S) protein.<li>The Afghanistan 1 genotype was assigned to pango lineage B.1.1.254 and <b>Nextstrain</b> clade 20B (<xref ref-type="fig" rid="viruses-13-01359-f002"><u>Figure 2</u></xref> and <xref ref-type="fig" rid="viruses-13-01359-f003"><u>Figure 3</u></xref>), and harbors the characteristic mutations G28881A, G28882A, and G28883C that confer R203K and G204R amino acid changes in the N gene, and a G50N amino acid substitution in ORF14, depending on the translation frame.<li> The Mali 1 genotype was shown to belong to pango lineage B.1 and <b>Nextstrain</b> clade 20A (<xref ref-type="fig" rid="viruses-13-01359-f002"><u>Figure 2</u></xref> and <xref ref-type="fig" rid="viruses-13-01359-f003"><u>Figure 3</u></xref>).<li> The Mali 2 and 3 genotypes were assigned to pango lineage A.27 and <b>Nextstrain</b> clade 19B (<xref ref-type="fig" rid="viruses-13-01359-f002"><u>Figure 2</u></xref> and <xref ref-type="fig" rid="viruses-13-01359-f003"><u>Figure 3</u></xref>).<li>The Mali 4 genotype, retrieved from two military service men, was assigned to pango lineage B.1.525 (also known as VUI-21FEB-03, VUI-202102/03, or UK1188) (<xref ref-type="fig" rid="viruses-13-01359-f002"><u>Figure 2</u></xref> and <xref ref-type="fig" rid="viruses-13-01359-f003"><u>Figure 3</u></xref>) and <b>Nextstrain</b> clade 20A [<xref rid="B58-viruses-13-01359" ref-type="bibr"><u>58</u></xref>].<li>Given the difficulties in visualizing such a large collection of genomes for lineage B.1.525, we constructed a dedicated <b>Nextstrain</b> build to enable a thorough exploration of our results: <uri xlink:href="https://<b>nextstrain</b>.org/community/GuyBaele/sars-cov-2-belgium/B.1.525">https://<b>nextstrain</b>.org/community/GuyBaele/sars-cov-2-belgium/B.1.525</uri> (accessed on 8 June 2021).<li> GISAID entries corresponding to genomes with imprecise sampling dates were discarded to generate a sufficiently responsive <b>Nextstrain</b> instance and to limit the file sizes (to &lt;100 Mb) so that the visualization could still be made publicly available.<li>Our study showed that during the period from May 2020 to April 2021, 21 Belgian soldiers returned to Belgium from overseas missions in Niger, the DRC, Afghanistan and Mali harboring nine distinct SARS-CoV-2 genotypes, belonging to three major <b>Nextstrain</b> clades (19B, 20A and 20B) (<xref ref-type="fig" rid="viruses-13-01359-f002"><u>Figure 2</u></xref> and <xref ref-type="fig" rid="viruses-13-01359-f003"><u>Figure 3</u></xref>).<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [28](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-13">(accessed on 13 May 2021)</date-in-citation>        ) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33170902">33170902</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32742035">32742035</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32793182">32793182</a>) [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32509310">32509310</a>) [58](          <person-group person-group-type="author">            <collab>Public Health England (PHE)</collab>          </person-group>          <article-title>Research and Analysis. Variants: Distribution of Cases Data</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data">https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-14">(accessed on 14 May 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34359323">34359323</a> <b>(0)</b> Limitation of Screening of Different Variants of SARS-CoV-2 by RT-PCR<ul><li> For a complete analysis and lineage determination, we employed two websites: Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 10 May 2021) and pangolineage (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>, accessed on 10 May 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34356077">34356077</a> <b>(3)</b> Correlates of SARS-CoV-2 Variants on Deaths, Case Incidence and Case Fatality Ratio among the Continents for the Period of 1 December 2020 to 15 March 2021<ul><li>In comparison with the reference sequence of SARS-CoV-2 Wuhan-Hu-1 strain (Accession NC_045512.2), phylogenetic analysis was performed for the complete coding region of all isolates using <b>Nextstrain</b> software [<xref rid="B14-genes-12-01061" ref-type="bibr"><u>14</u></xref>].<li> Clades 20 (including 20A, 20B, 20C, 20E (EU1), 20F, 20G, 20H/501Y.V2, 20I/501Y.V1 and 20J/501Y.V2) by <b>Nextstrain</b> nomenclature, are also known as clades G, GR and GH by GISAID nomenclature (<xref ref-type="fig" rid="genes-12-01061-f006"><u>Figure 6</u></xref>).<li> Clades 20 included 20A, 20B, 20C, 20E(EU1), 20F, 20G, 20H/501Y.V2, 20I/501Y.V1 and 20J/501Y.V2 by <b>Nextstrain</b> nomenclature, which is also known as clades G, GR and GH by GISAID nomenclature [<xref rid="B50-genes-12-01061" ref-type="bibr"><u>50</u></xref>].<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [50](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34268529">34268529</a> <b>(0)</b> Transmission event of SARS-CoV-2 Delta variant reveals multiple vaccine breakthrough infections<ul><li> Swift analysis pipeline produced variant vcf file, consensus genome, pangolin lineage and Nextclade assignment (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34713175">34713175</a> <b>(4)</b> Immune Profile of SARS-CoV-2 Variants of Concern<ul><li> The two major classifications have been produced by GISAID (<ext-link xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>) and <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov</ext-link>).<li> <b>Nextstrain</b>, in particular, assigns nomenclature through the designation of SARS-CoV-2 clades to label well-defined clades that reached geographic spread with significant frequency (<xref rid="B21" ref-type="bibr"><u>21</u></xref>).<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><u>No PMID</u> <b>(3)</b> Research and recommended resources on Zika virus, pathology, and control<ul><li>Regulatory bodies and relevant organizations.</p></caption><alt-text id="al0030">Table 1</alt-text><table frame="hsides" rules="groups"><tbody><tr><td>ASEAN Plus Three Field Epidemiology Training Network (ASEAN&#xA0;+&#xA0;3 FETN)<break/><ext-link ext-link-type="uri" xlink:href="http://www.aseanplus3fetn.net/" id="ir0010">http://www.aseanplus3fetn.net/</ext-link></td></tr><tr><td>ASEAN-Emergency Operation Centre (EOC) Network<break/><ext-link ext-link-type="uri" xlink:href="https://asean.org/" id="ir0015">https://asean.org/</ext-link></td></tr><tr><td>Brazilian Health Ministry<break/><ext-link ext-link-type="uri" xlink:href="https://www.saude.gov.br/component/tags/tag/combate-aedes" id="ir0020">https://www.saude.gov.br/component/tags/tag/combate-aedes</ext-link></td></tr><tr><td>Bureau of Epidemiology (Thailand)-Zika<break/><ext-link ext-link-type="uri" xlink:href="http://www.boe.moph.go.th/boedb/surdata/disease.php?ds=87" id="ir0025">http://www.boe.moph.go.th/boedb/surdata/disease.php?ds=87</ext-link></td></tr><tr><td>Centers for Disease Control and Prevention (CDC)-Zika Virus<break/><ext-link ext-link-type="uri" xlink:href="https://www.cdc.gov/zika/" id="ir0030">https://www.cdc.gov/zika/</ext-link></td></tr><tr><td>Centers for Disease Control and Prevention-Zika Virus Statistics and Maps<break/><ext-link ext-link-type="uri" xlink:href="https://www.cdc.gov/zika/reporting/index.html" id="ir0035">https://www.cdc.gov/zika/reporting/index.html</ext-link></td></tr><tr><td>European Centre for Disease Prevention and Control (EDCD)<break/><ext-link ext-link-type="uri" xlink:href="https://www.ecdc.europa.eu/en/zika-virus-disease" id="ir0040">https://www.ecdc.europa.eu/en/zika-virus-disease</ext-link></td></tr><tr><td>Evandro Chagas Institute (IEC)<break/><ext-link ext-link-type="uri" xlink:href="https://www.iec.gov.br/saiba-mais-virus-zika/" id="ir0045">https://www.iec.gov.br/saiba-mais-virus-zika/</ext-link></td></tr><tr><td>Fiocruz<break/><ext-link ext-link-type="uri" xlink:href="http://www.fiocruz.br" id="ir0050">www.fiocruz.br</ext-link></td></tr><tr><td>Funda&#xE7;ao Oswaldo Cruz&#x2014;FIOCRUZ<break/><ext-link ext-link-type="uri" xlink:href="https://portal.fiocruz.br/zika" id="ir0055">https://portal.fiocruz.br/zika</ext-link></td></tr><tr><td>Instituto Nacional de Saude da Mulher, Crian&#xE7;a e Adolescente<break/><ext-link ext-link-type="uri" xlink:href="http://www.iff.fiocruz.br" id="ir0060">www.iff.fiocruz.br</ext-link></td></tr><tr><td>John Hopkins Center for Health Security<break/><ext-link ext-link-type="uri" xlink:href="https://www.centerforhealthsecurity.org/" id="ir0065">https://www.centerforhealthsecurity.org/</ext-link></td></tr><tr><td>Ministry of Health of Brazil<break/><ext-link ext-link-type="uri" xlink:href="https://www.saude.gov.br/saude-de-a-z/zika-virus" id="ir0070">https://www.saude.gov.br/saude-de-a-z/zika-virus</ext-link></td></tr><tr><td>Ministry of Health-New Zealand<break/><ext-link ext-link-type="uri" xlink:href="https://www.health.govt.nz/your-health/conditions-and-treatments/diseases-and-illnesses/zika-virus" id="ir0075">https://www.health.govt.nz/your-health/conditions-and-treatments/diseases-and-illnesses/zika-virus</ext-link></td></tr><tr><td>National Health Service (NHS)<break/><ext-link ext-link-type="uri" xlink:href="https://www.nhs.uk/conditions/zika/" id="ir0080">https://www.nhs.uk/conditions/zika/</ext-link></td></tr><tr><td>National Institute of Allergy and Infectious Diseases (NIAID)<break/><ext-link ext-link-type="uri" xlink:href="https://www.niaid.nih.gov/diseases-conditions/addressing-zika" id="ir0085">https://www.niaid.nih.gov/diseases-conditions/addressing-zika</ext-link></td></tr><tr><td><b>Nextstrain</b><break/><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/zika" id="ir0090">https://<b>nextstrain</b>.org/zika</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34305996">34305996</a> <b>(5)</b> Creation and Worldwide Utilisation of New COVID-19 Online Information Hub for Genetics Health Professionals, Patients and Families<ul><li> Other websites have been established to provide more summarised information for professionals, such as <b>Nextstrain</b> (with its animated representations of sequence-based coronavirus global epidemiological data) and also the European Reference Networks.<li> These included sources relating to (a) COVID-19-related practical advice for healthcare, (b) Nature, Science, Frontiers, LitCovid (PubMed), and RCSB Protein Data Bank (PDB) hubs for Coronavirus research publications, (c) the COVID-19 pages of international organisations such as the World Health Organisation, European Centre for Disease Prevention and Control, and European Reference Networks, and (d) <b>Nextstrain</b>.<li><b>Nextstrain</b>.<li>                <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>              .</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34305996">34305996</a> <b>(4)</b> Creation and Worldwide Utilisation of New COVID-19 Online Information Hub for Genetics Health Professionals, Patients and Families<ul><li> Nextclade beta version 0.14.2 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://clades.<b>nextstrain</b>.org" ext-link-type="uri">clades.<b>nextstrain</b>.org</ext-link>) by <b>Nextstrain</b> (<xref rid="B11" ref-type="bibr"><u>11</u></xref>) was utilized in the phylogenetic analysis, which assigned both sequences to SARS-CoV-2 clade 20B (<xref rid="fig1" ref-type="fig"><u>Fig.&#xA0;1</u></xref>).<li> Nextclade beta version 0.14.2 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://clades.<b>nextstrain</b>.org" ext-link-type="uri">clades.<b>nextstrain</b>.org</ext-link>) was used to construct the tree on 4 May 2021, with the red and dark blue circles representing the positions of the hCoV-19/Bangladesh/JU-WMSRC-Dhaka-5/2020 (GISAID accession no.<li> According to the <b>Nextstrain</b> community build of Bangladesh (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/community/CHRF-Genomics/ncovBangladesh%40main" ext-link-type="uri">https://<b>nextstrain</b>.org/community/CHRF-Genomics/ncovBangladesh@main</ext-link>), the 9 nonsynonymous mutations common to the two studied genomes are also prevalent in other Bangladeshi SARS-CoV-2 genome sequences.<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34305826">34305826</a> <b>(4)</b> Introduction and Characteristics of SARS-CoV-2 in North-East of Romania During the First COVID-19 Outbreak<ul><li> Thus, the possible routes of introduction of SARS-CoV-2 in Romania, observed when inferring phylogenies using <b>Nextstrain</b> based on more than 30.000 GISAID sequences support the relationship between Romanian genomes and the genomes from the countries mentioned above.<li> When taking into account the European samples subset provided by <b>Nextstrain</b>, 28 of our samples from Suceava belong to 20A clade, while two belong to the 20B clade (<xref ref-type="fig" rid="F4"><u>Figure 4</u></xref>).<li>Phylogenetic analysis of SARS-CoV-2 emergence in Suceava and other Romanian regions within <b>Nextstrain</b> phylogeny/clades (showing genomes as filtered by <b>Nextstrain</b> visualization platform).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34273860">34273860</a> <b>(1)</b> Rapid SARS-CoV-2 variant monitoring using PCR confirmed by whole genome sequencing in a high-volume diagnostic laboratory<ul><li> Nonetheless, careful choice of metrics to monitor sequencing coverage and quality, as applied here, are necessary in order to avoid erroneous and misleading lineage assignment using both <italic>Pangolin</italic> and <italic><b>Nextstrain</b></italic> bioinformatic toolkits.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34273860">34273860</a> <b>(0)</b> Rapid SARS-CoV-2 variant monitoring using PCR confirmed by whole genome sequencing in a high-volume diagnostic laboratory<ul><li> Clades were determined using Nextclade beta v 0.12.0 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://clades.<b>nextstrain</b>.org" ext-link-type="uri">clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="R7" ref-type="bibr"><u>7</u></xref>), and lineages were determined with Pangolin COVID-19 lineage Assigner (COG-UK (cog-uk.io)).<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34234167">34234167</a> <b>(2)</b> Genomic epidemiology of SARS-CoV-2 in the UAE reveals novel virus mutation, patterns of co-infection and tissue specific host immune response<ul><li> We built a maximum likelihood phylogenetic tree using the <b>Nextstrain</b> pipeline; Augur v6.4.3 and MAFFT v7.455 for multiple sequence alignment and IQtree v1.6.12 for phylogenetic tree construction <sup><xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup>.<li> Clades were defined following the <b>Nextstrain</b> nomenclature<sup><xref ref-type="bibr" rid="CR16"><u>16</u></xref></sup>.<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=6247931">6247931</a>) [16](A. Rambaut, E. C. Holmes, V. Hill, A. OToole, J. McCrone, C. Ruis, L. du Plessis, O. Pybus. <italic>Nat. Miocrobiol</italic>. 10.1101/2020.04.17.046086 (2020).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34252764">34252764</a> <b>(6)</b> A direct capture method for purification and detection of viral nucleic acid enables epidemiological surveillance of SARS-CoV-2<ul><li>Genomic locations of known signature variants were obtained from <b>Nextstrain</b> (<xref rid="bb0095" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) (accessed 12 March 2021) and UCSC Table Browser (<xref rid="bb0130" ref-type="bibr"><u>Karolchik et al., 2004</u></xref>) (SARS-CoV-2 Jan.<li> Virtually all the SARS-CoV-2 sequences contained mutations corresponding to <b>Nextstrain</b> clade 20A.<li> Mutations found in <b>Nextstrain</b> clades 20B, 20C, and 20G were present in the November, December, and January samples, though mutations specific to strain 20B were present at 2.0% frequency in January, below the 10% frequency cutoff we had set for making a positive call.<li> Although merely suggestive due to small sample size, this does correspond to <b>Nextstrain</b> reported data which also suggests a decrease in the prevalence of this strain over the sampling period.<li> More transmissible strains were not observed in this data, consistent with <b>Nextstrain</b> reports for such strains initially appearing in this geographic area approximately 2 months after the sampling period.<li>sars cov-2 known variants by <b>nextstrain</b>clade.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Karolchik et al., 2004](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=14681465">14681465</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=14681465">14681465</a> <b>(1)</b> A direct capture method for purification and detection of viral nucleic acid enables epidemiological surveillance of SARS-CoV-2<ul><li> Graph adapted from <b>Nextstrain</b> [<xref rid="pmed.1003656.ref054" ref-type="bibr"><u>54</u></xref>].<li> [54](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34230209">34230209</a> <b>(3)</b> Long-chain polyphosphates impair SARS-CoV-2 infection and replication<ul><li> Overall, by <b>Nextstrain</b> sequence analysis (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>), we showed that the Korean strain (EPI_ISL_426163) belonged to the 19B clade, the Italian strain (EPI_ISL_514432-S66) belonged to the 20A clade, and the British strain (EPI_ISL_736997) belonged to the 20I/501Y.V1 (B.1.1.7) clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34230209">34230209</a> <b>(5)</b> Long-chain polyphosphates impair SARS-CoV-2 infection and replication<ul><li> Clade assignment was performed using Nextclade v0.12.0 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" ext-link-type="uri">https://github.com/<b>nextstrain</b>/nextclade</ext-link>).<li>Phylogenetic tree construction utilized the <b>Nextstrain</b> pipeline to align sequences, reconstruct maximum&#x2010;likelihood and time&#x2010;resolved phylogenetic trees and to infer nucleotide and amino acid substitutions across the phylogeny.<li> The tree, continually updated with additional Washington sequences collected between November 2020 and February 2021, can be viewed at: <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/wa/nov20-feb21" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/blab/ncov/wa/nov20-feb21</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34246830">34246830</a> <b>(3)</b> Analysis of SARS-CoV-2 nucleocapsid phosphoprotein N variations in the binding site to human 14-3-3 proteins<ul><li> Within this region, the most prominent peak in the entropy plot of the SARS-CoV-2 N protein corresponded to sites 203 and 204, which are among the most variable of the complete SARS-CoV-2 proteome, which can be observed at the GISAID website (GISAID - <b>NextStrain</b>).<li> These two appeared conjointly during December 2019 (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/" id="intref0050">https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/</ext-link>), which correlates with our result of co-evolution between them (<xref rid="fig2" ref-type="fig"><u>Fig.&#xA0;2</u></xref>C and D).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34212944">34212944</a> <b>(1)</b> A critical overview of computational approaches employed for COVID-19 drug discovery<ul><li> Another notable project, <b>Nextstrain</b>, is tracking viral evolution on the daily basis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34594942">34594942</a> <b>(0)</b> Genome Characterization of the First Outbreak of COVID-19 Delta Variant B.1.617.2 &#x2014; Guangzhou City, Guangdong Province, China, May 2021<ul><li> (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) (<xref rid="b1" ref-type="bibr"><u>1</u></xref>).<li> [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34580015">34580015</a> <b>(1)</b> SARS-CoV-2 Mutations and their Viral Variants<ul><li> To date, this variant has become the dominant form, replacing the wild type (WT) according to the mutation levels in the world presented in the <b>Nextstrain</b> database.The H49Y variant is produced with the C/T change at the 21.707 positions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33880483">33880483</a> <b>(2)</b> Coincident rapid expansion of two SARS-CoV-2 lineages with enhanced infectivity in Nigeria<ul><li> Viruses from lineage A viruses spread throughout the world early in the pandemic, but are now relatively rare globally, accounting for only 0.3% of isolates sequenced and deposited in January 2021 (<ext-link xlink:href="http://<b>nextstrain</b>.org" ext-link-type="uri"><b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34372500">34372500</a> <b>(14)</b> Molecular Evolution and Epidemiological Characteristics of SARS COV-2 in (Northwestern) Poland<ul><li> Three main nomenclatures have been introduced for SARS-CoV-2, including <b>Nextstrain</b> clades [<xref rid="B5-viruses-13-01295" ref-type="bibr"><u>5</u></xref>], PANGO lineages (PANGO, Phylogenetic Assignment of Named Outbreak LINeages) [<xref rid="B6-viruses-13-01295" ref-type="bibr"><u>6</u></xref>], and GISAID groups [<xref rid="B7-viruses-13-01295" ref-type="bibr"><u>7</u></xref>].<li> referred to as PANGO) and NEXTCLADE v0.14.2 (<uri xlink:href="https://github.com/<b>nextstrain</b>/nextclade">https://github.com/<b>nextstrain</b>/nextclade</uri> (accessed on 15 March 2021); referred to as <b>Nextstrain</b>).<li>The predominant variant in the analyzed dataset was PANGO B.1.221/<b>Nextstrain</b> 20A (n = 170, 15.37%).<li> Nineteen (1.72%) sequences were identified as <b>Nextstrain</b> 19A clade and a single isolate (0.09%) as 19B, reflecting the events of virus introduction to Poland directly from China.<li> <b>Nextstrain</b> and PANGO lineage identification methods were used to assign the sequence groups.<li> Genomes in Clade 1 were genetically diverse and classified as lineages 20B and 20I/501Y.V1 by <b>Nextstrain</b> and 11 PANGO variants (B.1.1.*), with B.1.1.153 (n = 10, 8.20%) being the most common.<li> All of the isolates in Clade 2 were <b>Nextstrain</b> 20A clade and PANGO B.1.1.258 lineage, while sequences from Clade 3 belonged to the 20A <b>Nextstrain</b> group and PANGO B.1.1.221 variant.<li>When analyzed in a broader national context with the remaining 984 Polish sequences included into the dataset, the three clades identified above contained 74.41% (n = 823) of sequences, with clade 1 (B.1.1*) consisting of 500 (45.21%) sequences belonging to the three <b>Nextstrain</b> clades (20B (n = 364, 32.91%), 20I/501Y.V1 (123, 11.12%), 20D (13, 1.18%)) (for details see <xref ref-type="app" rid="app1-viruses-13-01295"><u>Supplementary Material</u></xref>).<li> Clade 2 contained 160 (14.47%) sequences (only <b>Nextstrain</b> 20A, PANGO B.1.1.258 variant), while for Clade 3 we reported 163 (14.74%) sequences which corresponded to <b>Nextstrain</b> clade 20A and the PANGO B.1.1.221 variant (<xref ref-type="fig" rid="viruses-13-01295-f003"><u>Figure 3</u></xref>).<li> Clade 3 was the PANGO B.1.221 (<b>Nextstrain</b> 20A) strain with S98F mutation.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [7](          <person-group person-group-type="author">            <name>              <surname>Alm</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Broberg</surname>              <given-names>E.K.</given-names>            </name>            <name>              <surname>Connor</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Komissarov</surname>              <given-names>A.B.</given-names>            </name>            <name>              <surname>Maurer-Stroh</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Melidou</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Pereyaslov</surname>              <given-names>D.</given-names>            </name>            <etal/>          </person-group>          <article-title>Geographical and temporal distribution of SARS-CoV-2 clades in the WHO European Region, January to June 2020</article-title>          <source>Eurosurveillance</source>          <year>2020</year>          <volume>25</volume>          <fpage>2001410</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2020.25.32.2001410</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34707899">34707899</a> <b>(5)</b> Genetic Diversity and Evolution of the Biological Features of the Pandemic SARS-CoV-2<ul><li> Based ondata from viral sequences and information on the geographical origin of thesamples in GISAID, another information resource, <b>Nextstrain</b>(https://<b>nextstrain</b>.org) [<xref rid="R63" ref-type="bibr"><u>63</u></xref>] publisheshosts phylogenetic, geographic, and genomic analyses of SARSCoV- 2.<li> Using theGISAID database and <b>Nextstrain</b> resource, the evolution of a virus can bemonitored in real time.<li> The <b>Nextstrain</b> analysis predicts the occurrence ofapproximately 26 substitutions in the SARS-CoV-2 genome per year.<li> A phylogenetic analysis of SARS-CoV-2 isolates fromRussia showed that most samples correspond to the B.1, B.1.1, and B.1* lineages(PANGOLIN nomenclature) or to the G, GR, and GH clades (GISAID nomenclature),which are widespread in Europe [<xref rid="R106" ref-type="bibr"><u>106</u></xref>].In this case, the most prevalent genetic lineage is GR/20B/B.1.1(GISAID/<b>Nextstrain</b>/Pangolin nomenclature, respectively) [<xref rid="R107" ref-type="bibr"><u>107</u></xref>].<li> [63](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [106](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33510171">33510171</a>) [107](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33842111">33842111</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33979566">33979566</a> <b>(0)</b> Possible Human-to-Dog Transmission of SARS-CoV-2, Italy, 2020<ul><li> The Pangolin COVID-19 Lineage Assigner (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>) and Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) assigned the sequences to the lineage B.1.177 (denoted by Nextclade as 20A.EU1) in Europe.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34212838">34212838</a> <b>(1)</b> Neutralising capacity against Delta (B.1.617.2) and other variants of concern following Comirnaty (BNT162b2, BioNTech/Pfizer) vaccination in health care workers, Israel<ul><li> Identification of the variants were done with Pangolin COVID-19 lineage assigner (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>) and specific mutations were identified with a custom python code and Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li>B.1.617 and B.1.618 (non-VOC) lineages, and their background lineage B.1 (original virus) were constructed using <b>Nextstrain</b>&#x2019;s Augur pipeline and visualised with Auspice [<xref rid="r9" ref-type="bibr"><u>9</u></xref>].<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34197207">34197207</a> <b>(0)</b> New SARS-CoV-2 Variant from Jordan<ul><li> The phylogenetic tree was constructed using the Nextclade tool v 0.14.3 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34211026">34211026</a> <b>(1)</b> High-precision and cost-efficient sequencing for real-time COVID-19 surveillance<ul><li> However, diverse mutations in viral proteins, as observed in GISAID<sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref>,<xref ref-type="bibr" rid="CR21"><u>21</u></xref></sup> and <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR22"><u>22</u></xref></sup>, may lower COVID-19 vaccine effectiveness<sup><xref ref-type="bibr" rid="CR16"><u>16</u></xref>,<xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup>, as observed in influenza vaccines<sup><xref ref-type="bibr" rid="CR23"><u>23</u></xref>,<xref ref-type="bibr" rid="CR24"><u>24</u></xref></sup>.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33930298">33930298</a>) [17](Zhou, D. <italic>et al.</italic> Evidence of escape of SARS-CoV-2 variant B.1.351 from natural and vaccine-induced sera. <italic>Cell</italic>. <bold>184</bold>, 2348&#x2013;2361 (2021).) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27061888">27061888</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29109276">29109276</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34210893">34210893</a> <b>(1)</b> SARS-CoV-2 immune evasion by the B.1.427/B.1.429 variant of concern<ul><li> The two lineages B.1.427 and B.1.429 (belonging to clade 20C according to <b>Nextstrain</b> designation) share the same S mutations (S13I in the signal peptide, W152C in the NTD, and L452R in the RBD) but harbor different mutations in other SARS-CoV-2 genes (<xref rid="R42" ref-type="bibr"><italic toggle="yes">42</italic></xref>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34192736">34192736</a> <b>(1)</b> Untangling introductions and persistence in COVID-19 resurgence in Europe<ul><li> There have been advances in recent years in the computational efficiency and workflow standardisation possible for phylogenetic analyses that have made it easier to use these methods for real-time investigation of outbreaks, for example, through the development of the <b>Nextstrain</b> project (<xref rid="bib9" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bib14" ref-type="bibr"><u>Huddleston et al., 2021</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Huddleston et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a> <b>(3)</b> Untangling introductions and persistence in COVID-19 resurgence in Europe<ul><li> Although extending this particular algorithm to include phylogenetic data is outside the scope of the study, the manuscript is unfairly dismissive of such approaches, especially when a location specific build of <b>nextstrain</b> can be created with a programming burden similar to that of the algorithm described in this study.<li>3) Although extending this particular algorithm to include phylogenetic data is outside the scope of this study, it should address the value of such approaches, especially when a location specific build of <b>nextstrain</b> can be created with a programming burden similar to that of the algorithm described in this study.<li> I recognise that extending this particular algorithm to include phylogenetic data is outside the scope of work for this project, but the manuscript is unfairly dismissive of such approaches, especially when a location specific build of <b>nextstrain</b> can created with a programming burden similar to that of the algorithm described in this study.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a> <b>(5)</b> Untangling introductions and persistence in COVID-19 resurgence in Europe<ul><li> Although extending this particular algorithm to include phylogenetic data is outside the scope of the study, the manuscript is unfairly dismissive of such approaches, especially when a location specific build of <b>nextstrain</b> can be created with a programming burden similar to that of the algorithm described in this study.<li>3) Although extending this particular algorithm to include phylogenetic data is outside the scope of this study, it should address the value of such approaches, especially when a location specific build of <b>nextstrain</b> can be created with a programming burden similar to that of the algorithm described in this study.<li>&#x201C;There have been advances in recent years in the computational efficiency and workflow standardisation possible for phylogenetic analyses that have made it easier to use these methods for real-time investigation of outbreaks, for example through the development of the <b>Nextstrain</b> project<sup>[28, 29]</sup>.<li> I recognise that extending this particular algorithm to include phylogenetic data is outside the scope of work for this project, but the manuscript is unfairly dismissive of such approaches, especially when a location specific build of <b>nextstrain</b> can created with a programming burden similar to that of the algorithm described in this study.<li>&#x201C;There have been advances in recent years in the computational efficiency and workflow standardisation possible for phylogenetic analyses that have made it easier to use these methods for real-time investigation of outbreaks, for example through the development of the <b>Nextstrain</b> project<sup>[28, 29]</sup>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34181711">34181711</a> <b>(3)</b> Characterization of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection Clusters Based on Integrated Genomic Surveillance, Outbreak Analysis and Contact Tracing in an Urban Setting<ul><li>Sample clades, variants and other summary statistics were computed with the NextClade tool of <b>NextStrain</b> [<xref rid="CIT0021" ref-type="bibr"><u>21</u></xref>].<li> <italic toggle="yes">C</italic>, Clade composition of surveillance samples by sample collection week, using the <b>NextStrain</b> [<xref rid="CIT0021" ref-type="bibr"><u>21</u></xref>] color scheme.<li> colours are assigned according to the <b>NextStrain</b> [<xref rid="CIT0021" ref-type="bibr"><u>21</u></xref>] clade system.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34219999">34219999</a> <b>(3)</b> Notable and Emerging Variants of SARS-CoV-2 Virus: A Quick Glance<ul><li><b>Nextstrain</b>.org/ncov.<li> Wuhan/hCoV-19/ /WIV04/2019 sequence is considered as reference or zero sequence and it belongs to the A lineage of PANGO / 19B clade of <b>Nextstrain</b>/S clade of GISAID.<li>According to <b>Nextstrain</b>.org/ncov: Total 11 major clades were distinguished on the basis of analysis of clades.<b>nextstrain</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33906227">33906227</a> <b>(6)</b> Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Sequence Characteristics of Coronavirus Disease 2019 (COVID-19) Persistence and Reinfection<ul><li>Sequences were analyzed for mutations using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and snp-sites (<ext-link xlink:href="https://github.com/sanger-pathogens/snp-sites" ext-link-type="uri">https://github.com/sanger-pathogens/snp-sites</ext-link>).<li> Second, to identify unique characteristics of reinfecting viruses, each of the first and reinfection sequences were compared with circulating sequences in the community as defined by the same <b>Nextstrain</b> clade sampled within 1 month obtained from the same geographic location uploaded to GISAID (<xref rid="sup1" ref-type="supplementary-material"><u>Supplementary Table 2</u></xref>, <xref rid="sup1" ref-type="supplementary-material"><u>Supplementary Methods</u></xref>, <xref rid="sup3" ref-type="supplementary-material"><u>Supplementary Data 2</u></xref>).<li> The slope of the trendline was compared with the latest global clock rate (29 March 2021) as estimated by <b>NextStrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global/" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global/</ext-link>).<li> Before antiviral treatment, the rate of sequence changes over time appeared faster than the <b>Nextstrain</b> estimate for the global rate of SARS-CoV-2 evolution (dotted purple line in <xref rid="F3" ref-type="fig">Figure 3<italic toggle="yes">B</italic></xref>, <xref rid="sup1" ref-type="supplementary-material">Supplementary Figure 5<italic toggle="yes">B</italic></xref>).<li> The purple dash-dotted line is the global rate estimate obtained from <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34125658">34125658</a> <b>(4)</b> Rapidly emerging SARS-CoV-2 B.1.1.7 sub-lineage in the United States of America with spike protein D178H and membrane protein V70L mutations<ul><li>Phylogenetic analysis was conducted using the <b>NextStrain</b> phylogenetic pipeline (version 3.0.1) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>).<li> 2609 genomes were kept for final phylogenetic and transmission analysis using <b>Nextstrain</b>, rooted by NC_045512 (<xref rid="F0001" ref-type="fig"><u>Figure 1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34267839">34267839</a> <b>(1)</b> Review of COVID-19 Variants and COVID-19 Vaccine Efficacy: What the Clinician Should Know?<ul><li> <b>Nextstrain</b> curates and analyses the genetics of SARS-CoV-2, and is a collaborative effort of researchers between Seattle, USA and Basel, Switzerland [<xref rid="R12" ref-type="bibr"><u>12</u></xref>].<li> [12](          <comment>Emma B. Hodcroft. 2021. &#x201C;CoVariants: SARS-CoV-2 Mutations and Variants of Interest.&#x201D; <ext-link ext-link-type="uri" xlink:href="https://covariants.org/">https://covariants.org/</ext-link></comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34172783">34172783</a> <b>(1)</b> Limited intestinal inflammation despite diarrhea, fecal viral RNA and SARS-CoV-2-specific IgA in patients with acute COVID-19<ul><li> Phylogenetic analysis was performed using <b>Nextstrain</b> build for SARS-CoV-2 genomes with default parameters<sup><xref ref-type="bibr" rid="CR60"><u>60</u></xref></sup>.<li> The mutation profiles were analyzed with the tool NextClade v0.3.6 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [60](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34166484">34166484</a> <b>(0)</b> Variants of Concern Are Overrepresented Among Postvaccination Breakthrough Infections of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) in Washington State<ul><li> <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) and NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) tools.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34188776">34188776</a> <b>(3)</b> Evolution, correlation, structural impact and dynamics of emerging SARS-CoV-2 variants<ul><li> Additionally, we used available <b>Nextstrain</b> global SARS-CoV-2 analyses web portal <xref rid="b0220" ref-type="bibr"><u>[44]</u></xref> to infer the evolutionary relationship among the variants as well as the first date of emergence, which confirmed our results (<xref rid="s0085" ref-type="sec">Supplementary Fig.<li>We also used <b>Nextstrain</b>&#x2019;s NextClade <xref rid="b0220" ref-type="bibr"><u>[44]</u></xref> tool to further analyze and validate the evolutionary relationships between variants from variant sequences (n&#xA0;=&#xA0;6,052); B.1.1.7 (n&#xA0;=&#xA0;1,637), B.1.351 (n&#xA0;=&#xA0;1,069), P.1 (n&#xA0;=&#xA0;1,926), and Cal.20C (n&#xA0;=&#xA0;4,182).<li> The resultant alignment was submitted into <b>NextStrain</b>&#x2019;s NextClade <xref rid="b0220" ref-type="bibr"><u>[44]</u></xref>, which generated a Phylogeny using the sequences.<li> [[44]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[44]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[44]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34174479">34174479</a> <b>(0)</b> Molecular surveillance of the on-going SARS-COV-2 epidemic in Ribeirao Preto City, Brazil<ul><li> The reference SARS-CoV-2 strains composing the phylogenetic tree were obtained from an available dataset from the Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="ir0020">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34218651">34218651</a> <b>(2)</b> Review of the early reports of the epidemiological characteristics of the B.1.1.7 variant of SARS-CoV-2 and its spread worldwide<ul><li> from the <b>Nextstrain</b> clade naming strategy combining the year and letter in a system developed 5 years ago for seasonal influenza and extended for SARS-CoV-2 since January 2020 (<b>nextstrain</b>.org) [<xref rid="b15-j-phrp-2021-0037" ref-type="bibr"><u>15</u></xref>], and &#x201C;501Y.V1&#x201D; which denotes the replacement of asparagine (N) by tyrosine (Y) in the RBD of the spike protein at position 501.<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34202835">34202835</a> <b>(2)</b> Heparan Sulfate Binding Cationic Peptides Restrict SARS-CoV-2 Entry<ul><li> We conducted a phylogenetic analysis using <b>NextStrain</b> [<xref rid="pone.0253578.ref039" ref-type="bibr"><u>39</u></xref>] to compare the sequences with other viruses detected through local subsampling in California and Sacramento County specifically.<li> The phylogenetic tree was generated using the <b>NextStrain</b> protocol, and compares sequences to others amplified in Sacramento County in California.<li> [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34314723">34314723</a> <b>(4)</b> The origins and potential future of SARS-CoV-2 variants of concern in the evolving COVID-19 pandemic<ul><li> <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0010">https://<b>nextstrain</b>.org</ext-link><xref rid="bib8" ref-type="bibr"><sup>8</sup></xref>; accessed 22 May 2021).<li><b>Nextstrain</b> clade<xref rid="bib8" ref-type="bibr"><sup>8</sup></xref></th><th rowspan="2">First detection location</th><th rowspan="2">VOI or VOC</th><th rowspan="2">WHO<xref rid="bib71" ref-type="bibr"><sup>71</sup></xref> designation</th><th rowspan="2">Mutations of interest on S protein</th><th colspan="3">Observed clinical effect.<li> <b>Nextstrain</b> clade names start with the year of origin and distinguish lineages that reach a global frequency of 20%, so the name of a lineage can change if it increases in frequency<xref rid="bib8" ref-type="bibr"><sup>8</sup></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34314723">34314723</a> <b>(3)</b> The origins and potential future of SARS-CoV-2 variants of concern in the evolving COVID-19 pandemic<ul><li>Name<break/>(<b>Nextstrain</b>)<sup>a</sup></th><th align="center">First detected</th><th align="center">Spike protein substitutions<sup>b</sup></th><th align="center">Characteristics</th><th align="center">Vaccine efficacy</th><th align="center">Neutralizing antibody efficacy</th></tr></thead><tbody><tr><td colspan="7" align="left"><bold>Variants of Concern</bold> (B.1.1.7, B.1.351, B.1.427, B.1.429, and P.1).<li><sup>a</sup><ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34162869">34162869</a> <b>(4)</b> COVseq is a cost-effective workflow for mass-scale SARS-CoV-2 genomic surveillance<ul><li> To this end, we used <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR23"><u>23</u></xref></sup> and a random selection of SARS-CoV-2 genomes from GISAID<sup><xref ref-type="bibr" rid="CR2"><u>2</u></xref></sup> in order to build phylogenetic trees visualizing the hierarchical distribution of the&#xA0;179 samples that we sequenced with COVseq (&#x201C;Methods&#x201D;).<li> We then used the ncov tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) built on <b>nextstrain</b><sup><xref ref-type="bibr" rid="CR23"><u>23</u></xref></sup> to generate a temporal and spatial phylogenetic tree.<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34163035">34163035</a> <b>(1)</b> Emergence and spread of a SARS-CoV-2 lineage A variant (A.23.1) with altered spike protein in Uganda<ul><li>The genomes generated in this study were classified into Pango lineages<sup><xref ref-type="bibr" rid="CR4"><u>4</u></xref></sup> using the pangolin module pangoLEARN (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</ext-link>) and into <b>Nextstrain</b> clades using Nextclade<sup><xref ref-type="bibr" rid="CR9"><u>9</u></xref></sup> (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34152105">34152105</a> <b>(2)</b> From COVID-19 Crisis to Digital Health Care Innovation<ul><li> <b>Nextstrain</b> is an open-source initiative for sharing genomic data with a bioinformatic analysis engine to visualize data from contributing scientists.<li> It is sponsored by the National Institutes of Health, the Bill and Melinda Gates Foundation, and other international organizations (<xref rid="bib12" ref-type="bibr"><b>Nextstrain</b>, 2021</xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34157877">34157877</a> <b>(8)</b> Divergent regional evolutionary histories of a devastating global amphibian pathogen<ul><li> After confirming the temporal signal, we inferred time-measured phylogenies, with our concatenated alignment and recorded sampling years, using both BEAST2 [<xref rid="RSPB20210782C49" ref-type="bibr"><u>49</u></xref>] and <b>Nextstrain</b> [<xref rid="RSPB20210782C50" ref-type="bibr"><u>50</u></xref>].<li> Comparative methods for <b>Nextstrain</b> can be found in the electronic supplementary material.<li>It is important to note that we used BEAST2 and <b>Nextstrain</b> as analytical frameworks to compare patterns between the Sierra Nevada and Central Panama but not to infer exact introduction dates.<li> For <b>Nextstrain</b>, tMRCA for Sierra Nevada samples was estimated as 1407 years from present day (95% CI 4498&#x2013;1151) and tMRCA for Central Panama was estimated as 666 years from present day (95% CI 1914&#x2013;534) (electronic supplementary material, figure S4); dynamic <b>Nextstrain</b> visualizations are available at: <uri xlink:href="https://<b>nextstrain</b>.org/community/andrew-rothstein/bd-gpl/auspice/viz">https://<b>nextstrain</b>.org/community/andrew-rothstein/bd-gpl/auspice/viz</uri>.<li> Our <b>Nextstrain</b> and BEAST2 analyses infer that <italic>Bd</italic> from the Sierra Nevada is older than <italic>Bd</italic> from Central Panama (<xref ref-type="fig" rid="RSPB20210782F3"><u>figure&#xA0;3</u></xref>, electronic supplementary material, figure S4).<li> [49](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24722319">24722319</a>) [50](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34201767">34201767</a> <b>(2)</b> SARS-CoV-2 Variants: A Synopsis of In Vitro Efficacy Data of Convalescent Plasma, Currently Marketed Vaccines, and Monoclonal Antibodies<ul><li> Each variant was reported using both the official (PANGOLIN and <b>NextStrain</b>) and the local (VUI/VOC/VOI) naming systems, and colloquial terms (e.g., &#x201C;UK variant&#x201D;) in order to provide comprehensive association.<li><b>NextStrain</b> name.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34110974">34110974</a> <b>(0)</b> Symptomatic SARS-CoV-2 infections after full schedule BNT162b2 vaccination in seropositive healthcare workers: a case series from a single institution<ul><li> Sequences were deposited on GenBank and characterized using NextClade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>), showing the synchronous circulation of different SARS-CoV-2 strains in the Lombardy region.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34156923">34156923</a> <b>(5)</b> Genomic epidemiology of the first epidemic wave of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in Palestine<ul><li> derived from multiple imports, predominately from North America [<xref rid="R16" ref-type="bibr"><u>16</u></xref>], early analysis of samples collected during the August 2020 outbreak suggest the secondary outbreak consists largely of closely related viruses assigned to the B.1.1.1 lineage (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/oceania?c=region">https://<b>nextstrain</b>.org/ncov/oceania?c=region</ext-link>) [<xref rid="R6" ref-type="bibr"><u>6</u></xref>].<li> All animal-associated genomes were removed, as well as samples flagged by <b>NextStrain</b> as &#x2018;exclude&#x2019; (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt</ext-link> as of 25/08/2020).<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33311501">33311501</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34206507">34206507</a> <b>(1)</b> A Summary of the SARS-CoV-2 Vaccines and Technologies Available or under Development<ul><li>Although the WHO is trying to introduce uniform nomenclature for all of the viral variants, there are three main clades&#x2014;GISAID (2021), <b>Nextstrain</b> (2017), and Pangolin (2020)&#x2014;used to categorize these variants.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34206453">34206453</a> <b>(2)</b> Immune Evasion of SARS-CoV-2 Emerging Variants: What Have We Learnt So Far?<ul><li> However, this does not replace the three current nomenclature systems: GISAID (Global Initiative on Sharing All Influenza Data), <b>Nextstrain</b>, and PANGO.<li> There are 8 clades of SARS-CoV-2 or hCoV-19 (S, O, L, V, G, GH, GR, and GV) identified by the GISAID database [<xref rid="B68-viruses-13-01192" ref-type="bibr"><u>68</u></xref>], 11 major clades (19A, 19B, and 20A&#x2013;20I) recognized by <b>Nextstrain</b>, while Rambaut et al.<li> [68](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34149046">34149046</a> <b>(1)</b> Editorial: Revised World Health Organization (WHO) Terminology for Variants of Concern and Variants of Interest of SARS-CoV-2<ul><li> Three scientific nomenclature systems have been used to identify and track SARS-CoV-2 genotypes, including the Global Initiative on Sharing All Influenza Data (GISAID), <b>Nextstrain</b>, and Pango [<xref rid="b3-medscimonit-27-e933622" ref-type="bibr"><u>3</u></xref>].<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a> <b>(2)</b> Editorial: Revised World Health Organization (WHO) Terminology for Variants of Concern and Variants of Interest of SARS-CoV-2<ul><li> <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> Clades were determined based on the <b>Nextstrain</b> classification system.<li> Clades were determined based on the <b>Nextstrain</b> classification system.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34186311">34186311</a> <b>(1)</b> Non-permissive SARS-CoV-2 infection in human neurospheres<ul><li> Collected data corresponds to an exposure of NP from 3 different cell lines (<xref rid="t0005" ref-type="table"><u>Table 1</u></xref>) to SARS-CoV-2 (PANGO lineage B.1, <b>Nextstrain</b> clade 20C) in one experimental infection.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34265628">34265628</a> <b>(4)</b> Study on SARS-CoV-2 strains in Iran reveals potential contribution of co-infection with and recombination between different strains to the emergence of new strains<ul><li> <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming" id="intref0020">https://<b>nextstrain</b>.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>)).<li> For example, haplotypes H1, H1a, H1b, and H1r correspond, respectively, to clades G, GR, GH, and GV in GISAID, and to clades 20A, 20B, 20C, and 20E in <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34220784">34220784</a> <b>(0)</b> The First Molecular Characterization of Serbian SARS-CoV-2 Isolates From a Unique Early Second Wave in Europe<ul><li> The tree was made online with Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) without any restrictions on use.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34207314">34207314</a> <b>(0)</b> Isolation of Viable SARS-CoV-2 Virus from Feces of an Immunocompromised Patient Suggesting a Possible Fecal Mode of Transmission<ul><li> The consensus sequence was analyzed with Nextclade web interface (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 24 May 2021) (version 0.7.5)).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34153515">34153515</a> <b>(12)</b> Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Genome Sequencing from Post-Mortem Formalin-Fixed, Paraffin-Embedded Lung Tissues<ul><li> The occurrence of variants was checked on the GISAID (using CoVsurver) and <b>Nextstrain</b> websites to detect new variants.<li> Viral sequences from eight patients with &lt;25 variants in the variant list were deposited in GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org" id="intref0020"><italic>https://www.gisaid.org</italic></ext-link>, last accessed April 4, 2021).<xref rid="bib4" ref-type="bibr"><sup>4</sup></xref> For all contributive sequences, clades were attributed using <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses" id="intref0025"><italic>https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses</italic></ext-link>, last accessed April 4, 2021; <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv" id="intref0030"><italic>https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv</italic></ext-link>, last accessed April 4, 2021)<xref rid="bib27" ref-type="bibr"><sup>27</sup></xref> and Pangolin<xref rid="bib28" ref-type="bibr"><sup>28</sup></xref> classification tools and Pangolin COVID-19 classification according to Rambaut et&#xA0;al<xref rid="bib28" ref-type="bibr"><sup>28</sup></xref> (Pangolin, <ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="intref0035"><italic>https://pangolin.cog-uk.io</italic></ext-link>, last accessed April 4, 2021).<li> Using <b>Nextstrain</b> classification, eight patients were classified as clade 20A (because of the <italic>C14408T</italic> and <italic>A23403G</italic> variants), one patient was classified as clade 20B (because of the <italic>GGG28881AAC</italic> variant), one patient was classified as clade 20C (because of the <italic>C1059T</italic> and <italic>G25563T</italic> variants), and three patients were classified as clade 20D (because of the <italic>C4002T</italic>, <italic>G10097A</italic>, <italic>C13536T</italic>, and <italic>C23731T</italic> variants) (<xref rid="fig5" ref-type="fig"><u>Figure&#xA0;5</u></xref>).<li> Variants were checked on the GISAID (using CoVsurver) and <b>Nextstrain</b> websites, and three variants that have never been described before were detected [i.e.<li>Variant Profile per Patient</p></caption><table frame="hsides" rules="groups"><thead><tr><th>Patient no.</th><th>Profile</th><th>GISAID clade</th><th>GISAID ID<xref rid="tbl3fnlowast" ref-type="table-fn"><u>&#x2217;</u></xref></th><th><b>Nextstrain</b> clade</th><th>Pangolin COVID-19 classification<xref rid="bib28" ref-type="bibr"><sup>28</sup></xref></th></tr></thead><tbody><tr><td>1.<li><b>Nextstrain</b> classification for 13 patients.<li>The data obtained in the present study allowed us to classify SARS-CoV-2 genomes using the clade nomenclature from GISAID for all contributory sequences as well as <b>Nextstrain</b> and Pangolin COVID-19 classification tools.<li> According to the clade classification, most of the 13 patients (8/13) are classified as clade 20A, 1 is classified as clade 20B, 1 is classified as clade 20C, and 3 are classified as clade 20D (<b>Nextstrain</b>, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/blog/2021-01-06-updated-sars-cov-2-clade-naming" id="intref0040"><italic>https://<b>nextstrain</b>.org/blog/2021-01-06-updated-sars-cov-2-clade-naming</italic></ext-link>, last accessed April 23, 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34153515">34153515</a> <b>(2)</b> Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Genome Sequencing from Post-Mortem Formalin-Fixed, Paraffin-Embedded Lung Tissues<ul><li>Summarized list recreated from Science Daily,<sup><xref rid="bibr42-20499361211024372" ref-type="bibr"><u>42</u></xref></sup> <b>Nextstrain</b>,<sup><xref rid="bibr43-20499361211024372" ref-type="bibr"><u>43</u></xref></sup>,** GISAID,<sup><xref rid="bibr41-20499361211024372" ref-type="bibr"><u>41</u></xref></sup>,*** Bayarri-Olmos <italic toggle="yes">et al</italic>.,<sup><xref rid="bibr27-20499361211024372" ref-type="bibr"><u>27</u></xref></sup>,**** and the New York Times.<li> Data recreated from <b>Nextstrain</b>.<li> [42](<collab>Universit&#xE0; di Bologna</collab>. <article-title>The six strains of SARS-CoV-2</article-title>. <source>Science Daily</source>. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.sciencedaily.com/releases/2020/08/200803105246.htm" ext-link-type="uri">https://www.sciencedaily.com/releases/2020/08/200803105246.htm</ext-link> (<year>2020</year>, <comment>accessed 12 January 2021</comment>).) [43](<collab>Genomic epidemiology of novel coronavirus - global subsampling</collab>. <source>Nextstrain</source>. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://nextstrain.org/sars-cov-2/" ext-link-type="uri">https://nextstrain.org/sars-cov-2/</ext-link> (<year>2021</year>, <comment>accessed 12 January 2021</comment>).) [41](<collab>SARS-CoV-2: Clade and lineage nomenclature, July 4, 2020</collab>. <article-title>GISAID</article-title>. <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link> (<year>2021</year>, <comment>accessed 12 January 2021</comment>).) [27](<person-group person-group-type="author"><name><surname>Bayarri-Olmos</surname><given-names>R</given-names></name><name><surname>Rosbjerg</surname><given-names>A</given-names></name><name><surname>Johnsen</surname><given-names>LB</given-names></name></person-group>, <etal>et al</etal>. <article-title>The SARS-CoV-2 Y453F mink variant displays a pronounced increase in ACE-2 affinity but does not challenge antibody neutralization</article-title>. <source>J Biol Chem</source><year>2021</year>; <volume>296</volume>: <fpage>100536</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34159330">34159330</a> <b>(1)</b> A stem-loop RNA RIG-I agonist confers prophylactic and therapeutic protection against acute and chronic SARS-CoV-2 infection in mice<ul><li> We used Nextclade v0.14.2 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) to generate a phylogenetic tree and to compile a list of amino acid changes in the virus isolates as compared to the Wuhan-Hu-1 reference strain (<xref rid="SD1" ref-type="supplementary-material"><u>Extended Data Table 1</u></xref>; <xref rid="F10" ref-type="fig">Extended Data Fig.<li>We used NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) to generate a phylogenetic tree to show the evolutionary relations between the four cultured viruses and other SARS-CoV-2 lineages.<li> Branches are colored by <b>Nextstrain</b> clade, with branch labels indicating the Pango lineages, and highlighted are the four cultured viruses used in this study.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34174992">34174992</a> <b>(2)</b> Antibody and B cell responses to SARS-CoV-2 infection and vaccination<ul><li> A comparison of 3,823 representative viral genomes (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0025">https://<b>nextstrain</b>.org/ncov/global</ext-link>) to the early Wuhan-Hu-1 strain identified that SARS-CoV-2 viruses had accumulated around 75 heritable non-synonymous nucleotide mutations as of January 2021 and 24 non-synonymous mutations that potentially arose independently in different viral isolates (<xref rid="bib95" ref-type="bibr"><u>Wu et&#xA0;al., 2021a</u></xref>).<li> [Wu et&#xA0;al., 2021a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33676588">33676588</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34137349">34137349</a> <b>(1)</b> SARS-CoV-2 late shedding may be infectious between immunocompromised hosts<ul><li>Whole genome sequencing of the SARS-CoV-2 samples from these patients was performed using the ARTIC protocol [<xref rid="CIT0010" ref-type="bibr"><u>10</u></xref>] (<xref rid="t0001" ref-type="table"><u>Table 1</u></xref>), followed by phylogenetic analysis with <b>Nextstrain</b> [<xref rid="CIT0011" ref-type="bibr"><u>11</u></xref>] by the Expert Microbiology Unit of the National Institute for Health and Welfare.<li> [10](Artic Network protocols for sequencing SARS-CoV-2. Available from: <ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019">https://artic.network/ncov-2019</ext-link>.) [11](Nextstrain database for pathogen genome data. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org">https://nextstrain.org</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34204305">34204305</a> <b>(7)</b> The Functional Consequences of the Novel Ribosomal Pausing Site in SARS-CoV-2 Spike Glycoprotein RNA<ul><li>Further analyses of mutations using the <b>Nextstrain</b> measure of diversity [<xref rid="B21-ijms-22-06490" ref-type="bibr"><u>21</u></xref>] suggest that the second position of the CCT codon (encoding P681; <xref ref-type="fig" rid="ijms-22-06490-f001"><u>Figure 1</u></xref>) has an unusually large <b>Nextstrain</b> diversity measure (0.772).<li> <uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>, accessed date 1 June 2021).<li>While we restricted our study to the original Wuhan isolate S protein to establish the underlying mechanism, analyses of mutations using the <b>Nextstrain</b> measure of diversity [<xref rid="B21-ijms-22-06490" ref-type="bibr"><u>21</u></xref>] suggested that the second position of the CCT codon (encoding P681, <xref ref-type="fig" rid="ijms-22-06490-f001"><u>Figure 1</u></xref>) has an unusually large <b>Nextstrain</b> diversity measure.<li> Further analyses considering frequencies of mutations were performed using the <b>Nextstrain</b> measure of diversity [<xref rid="B21-ijms-22-06490" ref-type="bibr"><u>21</u></xref>].<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34214259">34214259</a> <b>(3)</b> En las entra&#xF1;as del SARS-CoV-2: liderazgo cient&#xED;fico del Instituto Nacional de Salud<ul><li> <b>NextStrain</b>, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>) y mediante la aplicaci&#xF3;n CoV-GLUE (<ext-link ext-link-type="uri" xlink:href="http://cov-glue.cvr.gla.ac.uk">http://cov-glue.cvr.gla.ac.uk</ext-link>) (<xref rid="B10" ref-type="bibr"><u>10</u></xref>,<xref rid="B11" ref-type="bibr"><u>11</u></xref>).<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [11](          <person-group person-group-type="author">            <name>              <surname>D&#xED;az</surname>              <given-names>FJ</given-names>            </name>            <name>              <surname>Aguilar-Jim&#xE9;nez</surname>              <given-names>W</given-names>            </name>            <name>              <surname>Fl&#xF3;rez-&#xC1;lvarez</surname>              <given-names>L</given-names>            </name>            <name>              <surname>Valencia</surname>              <given-names>G</given-names>            </name>            <name>              <surname>Laiton-Donato</surname>              <given-names>K</given-names>            </name>            <name>              <surname>Franco-Mu&#xF1;oz</surname>              <given-names>C</given-names>            </name>            <etal/>          </person-group>          <article-title>Aislamiento y caracterizaci&#xF3;n de una cepa temprana de SARS-CoV-2 durante la epidemia de 2020 en Medell&#xED;n, Colombia</article-title>          <source>Biom&#xE9;dica</source>          <year>2020</year>          <volume>40</volume>          <issue>2</issue>          <fpage>148</fpage>          <lpage>158</lpage>          <pub-id pub-id-type="doi">10.7705/biomedica.5834</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=          <person-group person-group-type="author">            <name>              <surname>D&#xED;az</surname>              <given-names>FJ</given-names>            </name>            <name>              <surname>Aguilar-Jim&#xE9;nez</surname>              <given-names>W</given-names>            </name>            <name>              <surname>Fl&#xF3;rez-&#xC1;lvarez</surname>              <given-names>L</given-names>            </name>            <name>              <surname>Valencia</surname>              <given-names>G</given-names>            </name>            <name>              <surname>Laiton-Donato</surname>              <given-names>K</given-names>            </name>            <name>              <surname>Franco-Mu&#xF1;oz</surname>              <given-names>C</given-names>            </name>            <etal/>          </person-group>          <article-title>Aislamiento y caracterizaci&#xF3;n de una cepa temprana de SARS-CoV-2 durante la epidemia de 2020 en Medell&#xED;n, Colombia</article-title>          <source>Biom&#xE9;dica</source>          <year>2020</year>          <volume>40</volume>          <issue>2</issue>          <fpage>148</fpage>          <lpage>158</lpage>          <pub-id pub-id-type="doi">10.7705/biomedica.5834</pub-id>        ">          <person-group person-group-type="author">            <name>              <surname>D&#xED;az</surname>              <given-names>FJ</given-names>            </name>            <name>              <surname>Aguilar-Jim&#xE9;nez</surname>              <given-names>W</given-names>            </name>            <name>              <surname>Fl&#xF3;rez-&#xC1;lvarez</surname>              <given-names>L</given-names>            </name>            <name>              <surname>Valencia</surname>              <given-names>G</given-names>            </name>            <name>              <surname>Laiton-Donato</surname>              <given-names>K</given-names>            </name>            <name>              <surname>Franco-Mu&#xF1;oz</surname>              <given-names>C</given-names>            </name>            <etal/>          </person-group>          <article-title>Aislamiento y caracterizaci&#xF3;n de una cepa temprana de SARS-CoV-2 durante la epidemia de 2020 en Medell&#xED;n, Colombia</article-title>          <source>Biom&#xE9;dica</source>          <year>2020</year>          <volume>40</volume>          <issue>2</issue>          <fpage>148</fpage>          <lpage>158</lpage>          <pub-id pub-id-type="doi">10.7705/biomedica.5834</pub-id>        </a> <b>(2)</b> En las entra&#xF1;as del SARS-CoV-2: liderazgo cient&#xED;fico del Instituto Nacional de Salud<ul><li> The <b>NextStrain</b> methodology (<xref rid="R18" ref-type="bibr"><u>18</u></xref>) can model dynamic changes of variant proportions, while an alternative approach aligns sequence data to a matrix of binary indicators for the presence of variants, and systematically evaluates each mutant as a potential variant (<xref rid="R19" ref-type="bibr"><u>19</u></xref>).<li> However, with the exception of the <b>NextStrain</b> methodology (<xref rid="R3" ref-type="bibr"><u>3</u></xref>), these methods do not directly take into account sequence collection time, nor explicitly incorporate highly granular geographic information.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32742035">32742035</a>) [3](<name><surname>Deng</surname><given-names>X.</given-names></name>, <name><surname>Garcia-Knight</surname><given-names>M. A.</given-names></name>, <name><surname>Khalid</surname><given-names>M. M.</given-names></name>, <name><surname>Servellita</surname><given-names>V.</given-names></name>, <name><surname>Wang</surname><given-names>C.</given-names></name>, <name><surname>Morris</surname><given-names>M. K.</given-names></name>, <name><surname>Sotomayor-Gonzalez</surname><given-names>A.</given-names></name>, <name><surname>Glasner</surname><given-names>D. R.</given-names></name>, <name><surname>Reyes</surname><given-names>K. R.</given-names></name>, <name><surname>Gliwa</surname><given-names>A. S.</given-names></name>, <name><surname>Reddy</surname><given-names>N. P.</given-names></name>, <name><surname>Sanchez San Martin</surname><given-names>C.</given-names></name>, <name><surname>Federman</surname><given-names>S.</given-names></name>, <name><surname>Cheng</surname><given-names>J.</given-names></name>, <name><surname>Balcerek</surname><given-names>J.</given-names></name>, <name><surname>Taylor</surname><given-names>J.</given-names></name>, <name><surname>Streithorst</surname><given-names>J. A.</given-names></name>, <name><surname>Miller</surname><given-names>S.</given-names></name>, <name><surname>Kumar</surname><given-names>G. R.</given-names></name>, <name><surname>Sreekumar</surname><given-names>B.</given-names></name>, <name><surname>Chen</surname><given-names>P. Y.</given-names></name>, <name><surname>Schulze-Gahmen</surname><given-names>U.</given-names></name>, <name><surname>Taha</surname><given-names>T. Y.</given-names></name>, <name><surname>Hayashi</surname><given-names>J.</given-names></name>, <name><surname>Simoneau</surname><given-names>C. R.</given-names></name>, <name><surname>McMahon</surname><given-names>S.</given-names></name>, <name><surname>Lidsky</surname><given-names>P. V.</given-names></name>, <name><surname>Xiao</surname><given-names>Y.</given-names></name>, <name><surname>Hemarajata</surname><given-names>P.</given-names></name>, <name><surname>Green</surname><given-names>N. M.</given-names></name>, <name><surname>Espinosa</surname><given-names>A.</given-names></name>, <name><surname>Kath</surname><given-names>C.</given-names></name>, <name><surname>Haw</surname><given-names>M.</given-names></name>, <name><surname>Bell</surname><given-names>J.</given-names></name>, <name><surname>Hacker</surname><given-names>J. K.</given-names></name>, <name><surname>Hanson</surname><given-names>C.</given-names></name>, <name><surname>Wadford</surname><given-names>D. A.</given-names></name>, <name><surname>Anaya</surname><given-names>C.</given-names></name>, <name><surname>Ferguson</surname><given-names>D.</given-names></name>, <name><surname>Lareau</surname><given-names>L. F.</given-names></name>, <name><surname>Frankino</surname><given-names>P. A.</given-names></name>, <name><surname>Shivram</surname><given-names>H.</given-names></name>, <name><surname>Wyman</surname><given-names>S. K.</given-names></name>, <name><surname>Ott</surname><given-names>M.</given-names></name>, <name><surname>Andino</surname><given-names>R.</given-names></name>, <name><surname>Chiu</surname><given-names>C. Y.</given-names></name>, <article-title>Transmission, infectivity, and antibody neutralization of an emerging SARS-CoV-2 variant in California carrying a L452R spike protein mutation</article-title>. <source>medRxiv</source>, (<year>2021</year>).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34198622">34198622</a> <b>(1)</b> SARS-CoV-2 Variants in Lebanon: Evolution and Current Situation<ul><li> Strains S5 and S9 were analyzed in a previous study and identified as belonging to the <b>Nextstrain</b> clade 19A, a group associated with the GISAID clades L or V.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34176716">34176716</a> <b>(5)</b> Virological and genomic analysis of SARS-CoV-2 from a favipiravir clinical trial cohort<ul><li>The genomic epidemiology of SARS-CoV-2 identified in this study was analyzed using the Nextclade webpage (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0025">https://clades.<b>nextstrain</b>.org/</ext-link>) and the <b>Nextstrain</b> tools (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.github.io/ncov/" id="intref0030">https://<b>nextstrain</b>.github.io/ncov/</ext-link>).<li> Sequencing results were evaluated in the Nextclade webpage (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0035">https://clades.<b>nextstrain</b>.org/</ext-link>), and sequence data indicated as good with all QC status under the default settings were defined as high-quality.<li> For sequences registered only with the months of collection and not the dates, the collection dates were randomly assigned within the months to enable analysis through the <b>Nextstrain</b> pipeline.<li> The amino acid substitutions on SARS-CoV-2 genome were also evaluated on the Nextclade webpage (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0045">https://clades.<b>nextstrain</b>.org/</ext-link>).<li>Phylogenetic tree of SARS-CoV-2 genomes generated by <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34204754">34204754</a> <b>(2)</b> Previous SARS-CoV-2 Infection Increases B.1.1.7 Cross-Neutralization by Vaccinated Individuals<ul><li> Individuals infected during the second wave (August 2020) were almost exclusively exposed to the G614-containing 20E (EU1) lineage, which accounted for nearly 100% of new infections during the summer of 2020 (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>, accessed on 31 March 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34136351">34136351</a> <b>(1)</b> SARS-CoV-2 detection and sequencing in heart tissue associated with myocarditis and persistent arrhythmia: A case report<ul><li> This clade is the most prevalent in Europe, America, Asia, USA, and India (<b>Nextstrain</b>, 2020).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34114541">34114541</a> <b>(5)</b> The SARS-CoV-2 B.1.351 lineage (VOC &#x3B2;) is outgrowing the B.1.1.7 lineage (VOC &#x3B1;) in some French regions in April 2021<ul><li> The first, VOC &#x3B1;, which corresponds to Pango lineage B.1.1.7, <b>nextstrain</b> clade 20I/501Y.V1, and GISAID clade/lineage GRY, is currently causing the majority of infections in Europe and North America [<xref rid="r2" ref-type="bibr"><u>2</u></xref>], whereas, the second, VOC &#x3B2; (Pango lineage: B.1.351; <b>nextstrain</b> clade: 20H/501Y.V2; GISAID clade/lineage: GH/501Y.V2) is the most common variant in South Africa [<xref rid="r3" ref-type="bibr"><u>3</u></xref>].<li> <b>nextstrain</b> clade: 20J/501Y.V3; GISAID clade/lineage: GR/501Y.V3) dominates in Brazil and South America [<xref rid="r4" ref-type="bibr"><u>4</u></xref>] and the fourth VOC &#x3B4; (Pango lineage: B.1.617.2; <b>nextstrain</b> clade: 21A/S:478K; GISAID clade/lineage: G/452R.V3) caused a major epidemic wave in India [<xref rid="r5" ref-type="bibr"><u>5</u></xref>].<li> <b>nextstrain</b> clade: 20A/S484K; GISAID clade/lineage: G/484K.V3) [<xref rid="r1" ref-type="bibr"><u>1</u></xref>], but not in B.1.1.7.<li> [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33658326">33658326</a>) [3](Tegally H, Wilkinson E, Giovanetti M, Iranzadeh A, Fonseca V, Giandhari J, et al. Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa. medRxiv. 2020;2020.12.21.20248640.) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33853970">33853970</a>) [5](GISAID. "Tracking of Variants". gisaid.org. GISAID. [Accessed 30 May 2021]. Available from: <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/hcov19-variants/">https://www.gisaid.org/hcov19-variants/</ext-link>) [1](World Health Organization (WHO). Tracking-SARS-CoV-2-variants. Geneva: WHO; 2021. [Accessed 1 Jun 2021]. Available from: <ext-link ext-link-type="uri" xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34207490">34207490</a> <b>(0)</b> A Comprehensive Molecular Epidemiological Analysis of SARS-CoV-2 Infection in Cyprus from April 2020 to January 2021: Evidence of a Highly Polyphyletic and Evolving Epidemic<ul><li>Mutations of the sequences in this study were identified using the Nextclade webtool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 15 February 2021) [<xref rid="B18-viruses-13-01098" ref-type="bibr"><u>18</u></xref>].<li> [18](          <article-title>Nextstrain Nextstrain</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">https://nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-02-24">(accessed on 24 February 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34127976">34127976</a> <b>(2)</b> Mapping Potential Antigenic Drift Sites (PADS) on SARS-CoV-2 Spike in Continuous Epitope-Paratope Space<ul><li> According to <b>NextStrain</b> as of 3/22/21, the 501Y.V2 lineage has mutations at PADS 348, 352, 359, 367, <bold>382</bold>, <bold>384</bold>, 408, 427, <bold>430</bold>, <bold>434</bold>, <bold>435</bold>, 439, 440, 443, <bold>444</bold>, <bold>445</bold>, 456, <bold>450</bold>, 457, 476, and 478, where bolded residues score in the top 50% of PADS [<ext-link xlink:href="http://nexstrain.org" ext-link-type="uri">nexstrain.org</ext-link>].<li> 501Y.V3 has few available sequences (375 in <b>NextStrain</b> as of 3/22/21), and no observed mutations yet at PADS.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34166617">34166617</a> <b>(0)</b> Recurrent emergence of SARS-CoV-2 spike deletion H69/V70 and its role in the Alpha variant B.1.1.7<ul><li> Major SARS-CoV-2 clade memberships were assigned to all sequences using the Nextclade server v0.13 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0060">https://clades.<b>nextstrain</b>.org/</ext-link>), Pangolin v2.4.2 (<xref rid="bib56" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020a</u></xref>); <ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0065">https://github.com/cov-lineages/pangolin</ext-link>) and a local instance of the PangoLEARN model, dated 18<sup>th</sup> April 21:49 (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangoLEARN" id="intref0070">https://github.com/cov-lineages/pangoLEARN</ext-link>).<li> [Rambaut et&#xA0;al., 2020a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34103090">34103090</a> <b>(1)</b> Characterization of SARS-CoV-2 different variants and related morbidity and mortality: a systematic review<ul><li>B.1.351 in PANGOLIN phylogeny or 20H/501Y.V2 in <b>NextStrain</b> phylogeny.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34103497">34103497</a> <b>(8)</b> Molecular evidence of SARS-CoV-2 in New York before the first pandemic wave<ul><li> The PANGO lineage classification of the RPN pools is indicated on the right, and the <b>NextStrain</b> clades are shown as node labels.<li> Lineage B.1 is a parent lineage to multiple emerging lineages that continue to circulate in the US, including lineage B.1.2 (N:P67S), which has become a dominant lineage in the US (<b>NextStrain</b> build, 17 February 2021, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>).<li>Lineage classification of previously undetected SARS-CoV-2 in NYC.</p></caption><table frame="hsides" rules="groups"><thead><tr><th>Sample</th><th>Week ending</th><th>Ct values (<italic>ORF1ab</italic>, <italic>E</italic>)</th><th>Genome completeness (%)</th><th><b>NextStrain</b> clade</th><th>Clade-defining mutations</th><th>PANGO lineage</th><th>Lineage detection prior to 1 March</th></tr></thead><tbody><tr><td>Pool-134.<li> (1) Nextera XT Illumina amplicon sequencing as described above, (2) Nextera XT sequencing of 1.5&#x2013;2&#x2009;kb amplicons targeting only regions containing clade-defining single-nucleotide variation (SNVs) (positions 1059, 8782, 14,408, 23,403, 25,563, 28,144, 28,881, and 28,882, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming">https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>), and the Swift Normalase&#xAE; Amplicon Panel SARS-CoV-2 (Swift Bioscience COVG1V2-96, SN-5&#xD7;296 and SN-5S1A96) according to the manufacturer&#x2019;s instructions for (3) regular input and (4) low input samples.<li> Initial alignment and subsampling were done by using the <b>NextStrain</b> tool (v1.0)<sup><xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup>.<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34201088">34201088</a> <b>(1)</b> A Unique SARS-CoV-2 Spike Protein P681H Variant Detected in Israel<ul><li>Global whole genome sequencing (WGS) efforts and initiatives, such as the GISAID (global initiative on sharing all influenza data) platform [<xref rid="B2-vaccines-09-00616" ref-type="bibr"><u>2</u></xref>], <b>Nextstrain</b> [<xref rid="B3-vaccines-09-00616" ref-type="bibr"><u>3</u></xref>], and Pangolin [<xref rid="B4-vaccines-09-00616" ref-type="bibr"><u>4</u></xref>,<xref rid="B5-vaccines-09-00616" ref-type="bibr"><u>5</u></xref>], facilitate access to worldwide SARS-CoV-2 sequences, tracking different lineages of the virus and detecting emerging novel virus variants.<li> [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [5](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Anthony Underwood</surname>              <given-names>B.J.</given-names>            </name>            <name>              <surname>Verity Hill</surname>              <given-names>J.T.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>C.R.</given-names>            </name>            <name>              <surname>Khali</surname>              <given-names>A.D.</given-names>            </name>            <name>              <surname>Ben Taylor</surname>              <given-names>C.Y.</given-names>            </name>          </person-group>          <article-title>Louis du Plessis, David Aanensen, Eddie Holmes, Oliver Pybus</article-title>          <source>Pangolin: Lineage Assignment in an Emerging Pandemic as an Epidemiological Tool</source>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="github.com/cov-lineages/pangolin">github.com/cov-lineages/pangolin</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-06-01">(accessed on 1 June 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527282">34527282</a> <b>(1)</b> Phylodynamics reveals the role of human travel and contact tracing in controlling the first wave of COVID-19 in four island nations<ul><li>Genome sequencing is increasingly affordable and shared global surveillance resources such as <b>NextStrain</b> (<xref rid="R34" ref-type="bibr">Hadfield, et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34163459">34163459</a> <b>(9)</b> Molecular Epidemiology, Evolution and Reemergence of Chikungunya Virus in South Asia<ul><li> <xref rid="B74" ref-type="bibr"><b>Nextstrain</b>, 2021</xref>; <xref rid="B120" ref-type="bibr"><u>Spicher et al., 2021</u></xref>).<li>Molecular evolutionary analysis confirmed the divergence of lineages from each other in previous studies and <b>Nextstrain</b> project and recently published phylogenies and evolutionary analysis (<xref rid="B22" ref-type="bibr"><u>de Bernardi Schneider et al., 2019</u></xref>; <xref rid="B23" ref-type="bibr"><u>Deeba et al., 2020</u></xref>; <xref rid="B74" ref-type="bibr"><b>Nextstrain</b>, 2021</xref>; <xref rid="B120" ref-type="bibr"><u>Spicher et al., 2021</u></xref>).<li> In four countries, the available 157 whole genome of CHIKV in <b>Nextstrain</b> (India-88, Bangladesh-40, Sri Lanka-22, and Pakistan-7) had an estimated 2.63e<sup>&#x2013;4</sup> substitution per site per year (<xref ref-type="fig" rid="F3"><u>Figures 3A&#x2013;C</u></xref>).<li> The phylogenetic tree was retrieved and from <b>Nextstrain</b> and modified based on the most updated information on CHIKV (<xref rid="B22" ref-type="bibr"><u>de Bernardi Schneider et al., 2019</u></xref>; <xref rid="B120" ref-type="bibr"><u>Spicher et al., 2021</u></xref>).<li> direction, while the vertical scale indicated the number of substitutions per position in the genome (<xref rid="B74" ref-type="bibr"><b>Nextstrain</b>, 2021</xref>).<li> Lineages are indicated as Asian urban (AUL), AUL-America (AUL-Am), South America (SAL), Middle Africa (MAL), Indian Ocean (IOL), East Africa (EAL), Africa and Asia (AAL), Sister Taxa to ECSA (sECSA), and West Africa (WA) (<xref rid="B74" ref-type="bibr"><b>Nextstrain</b>, 2021</xref>).<li> Source: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/ViennaRNA/CHIKV">https://<b>nextstrain</b>.org/community/ViennaRNA/CHIKV</ext-link>.<li> [Spicher et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33567556">33567556</a>) [de Bernardi Schneider et al., 2019](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31470643">31470643</a>) [Deeba et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32070451">32070451</a>) [Spicher et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33567556">33567556</a>) [de Bernardi Schneider et al., 2019](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31470643">31470643</a>) [Spicher et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33567556">33567556</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34200331">34200331</a> <b>(6)</b> Nucleic Acid Testing of SARS-CoV-2<ul><li> Based on these genome sequences, the phylogenetic classification of SARS-CoV-2 was performed and the nomenclature of GISAID, Phylogenetic Assignment of Named Global Outbreak LINeages (PANGO lineage), and <b>Nextstrain</b> are widely used in scientific and clinical communities [<xref rid="B26-ijms-22-06150" ref-type="bibr"><u>26</u></xref>,<xref rid="B28-ijms-22-06150" ref-type="bibr"><u>28</u></xref>,<xref rid="B29-ijms-22-06150" ref-type="bibr"><u>29</u></xref>] The major lineages of each nomenclature are summarized in <xref rid="ijms-22-06150-t001" ref-type="table"><u>Table 1</u></xref>.<li>The clades of <b>Nextstrain</b> nomenclature were initially named according to year&#x2013;letter combinations [<xref rid="B32-ijms-22-06150" ref-type="bibr"><u>32</u></xref>].<li> For this reason, <b>Nextstrain</b> updated their major clade designation criteria with regional frequency (&gt;30%) and recognized variants of concern.<li> Currently, 12 major clades (19A, 19B, 20A, 20B, 20C, 20D, 20E, 20F, 20G, 20H/501Y.V2, 20I/501Y.V1, 20J/501Y.V3) are designated in the nomenclature of <b>Nextstrain</b>.<li> <b>Nextstrain</b> provides real-time tracking evolution and spreading of SARS-CoV-2 [<xref rid="B28-ijms-22-06150" ref-type="bibr"><u>28</u></xref>].<li> <b>Nextstrain</b> visualizes tracking of SARS-CoV-2 lineages by integrating geographic information and sequence information.<li> [26](          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G.</given-names>            </name>          </person-group>          <article-title>Data, Disease and Diplomacy: GISAID&#x2019;s Innovative Contribution to Global Health</article-title>          <source>Glob. Challenges</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [32](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>          </person-group>          <article-title>Year-Letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.Org</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-31">(accessed on 31 May 2021)</date-in-citation>        ) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(6)</b> Nucleic Acid Testing of SARS-CoV-2<ul><li>The viral clade and lineage classifications were conducted by using the Global Initiative on Sharing Avian Influenza Data (GISAID) database [<xref rid="ppat.1009619.ref027" ref-type="bibr"><u>27</u></xref>], <b>Nextstrain</b> [<xref rid="ppat.1009619.ref028" ref-type="bibr"><u>28</u></xref>], and Phylogenetic Assignment of Named Global Outbreak (PANGO) Lineages [<xref rid="ppat.1009619.ref029" ref-type="bibr"><u>29</u></xref>].<li>To examine the phylogeny based on the identified genetic variations, we analyzed SARS-CoV-2 genomic data using GISAID, <b>Nextstrain</b>, and Pangolin.<li> The sequences of our SARS-CoV-2 were assigned as GR clade (GISAID), 20B clade (<b>Nextstrain</b>), and R.1 lineage (PANGO lineage).<li> <b>Nextstrain</b>, <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>, CC-BY-4.0 license.<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](<name><surname>Rambaut</surname><given-names>A</given-names></name>, <name><surname>Holmes</surname><given-names>EC</given-names></name>, <name><surname>O&#x2019;Toole</surname><given-names>A</given-names></name>, <name><surname>Hill</surname><given-names>V</given-names></name>, <name><surname>McCrone</surname><given-names>JT</given-names></name>, <name><surname>Ruis</surname><given-names>C</given-names></name>, <etal>et al</etal>. <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>. <source>Nat Microbiol. 2020</source>; <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>1407</lpage>.) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=<name><surname>Rambaut</surname><given-names>A</given-names></name>, <name><surname>Holmes</surname><given-names>EC</given-names></name>, <name><surname>O&#x2019;Toole</surname><given-names>A</given-names></name>, <name><surname>Hill</surname><given-names>V</given-names></name>, <name><surname>McCrone</surname><given-names>JT</given-names></name>, <name><surname>Ruis</surname><given-names>C</given-names></name>, <etal>et al</etal>. <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>. <source>Nat Microbiol. 2020</source>; <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>1407</lpage>."><name><surname>Rambaut</surname><given-names>A</given-names></name>, <name><surname>Holmes</surname><given-names>EC</given-names></name>, <name><surname>O&#x2019;Toole</surname><given-names>A</given-names></name>, <name><surname>Hill</surname><given-names>V</given-names></name>, <name><surname>McCrone</surname><given-names>JT</given-names></name>, <name><surname>Ruis</surname><given-names>C</given-names></name>, <etal>et al</etal>. <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>. <source>Nat Microbiol. 2020</source>; <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>1407</lpage>.</a> <b>(4)</b> Nucleic Acid Testing of SARS-CoV-2<ul><li>From looking at this clade on the focal <b>Nextstrain</b> 484K build (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/groups/neherlab/ncov/S.E484?c=country&amp;f_pango_lineage=R.1&amp;label=mlabel:20B/C18877T" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/neherlab/ncov/S.E484?c=country&amp;f_pango_lineage=R.1&amp;label=mlabel:20B/C18877T</ext-link>), it does indeed seem to be, but sometimes in the paper it seems like perhaps this is presented as ambiguous (though this may not have been intentional, and I may be misinterpreting the writing - apologies if so).<li>Further a phylogeny would help to clarify the meaning of statement like in lines 168-169: "The circulating trend of R.1 lineage showed similar pattern (note: typo, should be 'patterns') in each country, implying that the ancestral strain acquired homegrown variants, and subsequently R.1 lineage are likely to emerge in several countries." Is there evidence of unique R.1 variants in different countries? (I actually think from the <b>Nextstrain</b> tree, there may be - but showing a tree would display this more clearly, if so.).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34042186">34042186</a> <b>(1)</b> Genome&#x2010;wide analysis of SARS&#x2010;CoV&#x2010;2 strains circulating in Vietnam: Understanding the nature of the epidemic and role of the D614G mutation<ul><li> Variant analysis was done using CLC (v10.1.1) and Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).<li> These strains are classified into clades 20A, 20B, 20C, and 20D according to <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34089348">34089348</a> <b>(1)</b> Dying of VOC-202012/01 &#x2014; multimodal investigations in a death case of the SARS-CoV-2 variant<ul><li> Samples were analyzed and visualized within the phylogenetic context by next strain (<b>nextstrain</b>.org) using data available through GISAID (gisaid.org); GISAID identifiers are indicated at the right.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34198904">34198904</a> <b>(2)</b> Phylogenetic Structure and Sequential Dominance of Sub-Lineages of PRRSV Type-2 Lineage 1 in the United States<ul><li> A phylogenetic tree with the combined UMN VDL/GenBank data set was then illustrated using <b>Nextstrain</b> 2.0.0 [<xref rid="B43-vaccines-09-00608" ref-type="bibr"><u>43</u></xref>].<li> and Canada, and built a phylogenetic tree with the combined UMN VDL/GenBank dataset using <b>Nextstrain</b> (<xref ref-type="fig" rid="vaccines-09-00608-f006"><u>Figure 6</u></xref>) [<xref rid="B43-vaccines-09-00608" ref-type="bibr"><u>43</u></xref>].<li> [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><u>No PMID</u> <b>(2)</b> The sudden appearance of SARS-CoV-2<ul><li> 1</alt-text><graphic xlink:href="f01-01-9780323828604"/><attrib>Credit: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="ir0010">https://<b>nextstrain</b>.org/ncov/global</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34083758">34083758</a> <b>(5)</b> Single-nucleotide conservation state annotation of the SARS-CoV-2 genome<ul><li> All annotations were accessed through the&#xA0;UCSC Genome Browser<sup><xref ref-type="bibr" rid="CR1"><u>1</u></xref></sup> except for nonsingleton mutations from <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup> and homoplastic mutations from a prior study<sup><xref ref-type="bibr" rid="CR18"><u>18</u></xref></sup>.<li> All annotations were accessed through the&#xA0;UCSC Genome Browser<sup><xref ref-type="bibr" rid="CR1"><u>1</u></xref></sup> except for nonsingleton mutations from <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup> and homoplastic mutations from a prior study<sup><xref ref-type="bibr" rid="CR18"><u>18</u></xref></sup>.<li> Nonsingleton mutations were identified from <b>Nextstrain</b> mutations<sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>.<li>Mutations found in SARS-CoV-2 sequences were point mutations identified by <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup> (accessed on Sept 7, 2020) from sequences available on GISAID<sup><xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup>.<li> For our analysis, to minimize putative false calls we filtered out mutations if their ancestral alleles did not match the reference genome used by <b>Nextstrain</b>, MN908947.3, such as C&#x2009;&gt;&#x2009;T at a base where T is the reference allele.<li> [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908258">32908258</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32387564">32387564</a>) [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908258">32908258</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32387564">32387564</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [17](          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Glob. Challenges</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Glob. Challenges</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ">          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Glob. Challenges</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        </a> <b>(1)</b> Single-nucleotide conservation state annotation of the SARS-CoV-2 genome<ul><li>Fuelled by the raging COVID-19 pandemic, GISAID's data is enabling more than a dozen SARS-CoV-2 variant web-based visualizations including those hosted by <b>NextStrain</b>,<sup><xref rid="ref11" ref-type="bibr"><u>11</u></xref></sup> CovMT<sup><xref rid="ref12" ref-type="bibr"><u>12</u></xref></sup> and CoVariants.<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [12](<person-group person-group-type="author"><name><surname>Alam</surname><given-names>I</given-names></name><etal/></person-group>:<article-title>CovMT: an interactive SARS-CoV-2 mutation tracker, with a focus on critical variants.</article-title><source><italic toggle="yes">Lancet Infect Dis.</italic></source><year>2021</year>;<volume>21</volume>:<fpage>602</fpage>.<pub-id pub-id-type="doi">10.1016/S1473-3099(21)00078-5</pub-id>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=<person-group person-group-type="author"><name><surname>Alam</surname><given-names>I</given-names></name><etal/></person-group>:<article-title>CovMT: an interactive SARS-CoV-2 mutation tracker, with a focus on critical variants.</article-title><source><italic toggle="yes">Lancet Infect Dis.</italic></source><year>2021</year>;<volume>21</volume>:<fpage>602</fpage>.<pub-id pub-id-type="doi">10.1016/S1473-3099(21)00078-5</pub-id>"><person-group person-group-type="author"><name><surname>Alam</surname><given-names>I</given-names></name><etal/></person-group>:<article-title>CovMT: an interactive SARS-CoV-2 mutation tracker, with a focus on critical variants.</article-title><source><italic toggle="yes">Lancet Infect Dis.</italic></source><year>2021</year>;<volume>21</volume>:<fpage>602</fpage>.<pub-id pub-id-type="doi">10.1016/S1473-3099(21)00078-5</pub-id></a> <b>(4)</b> Single-nucleotide conservation state annotation of the SARS-CoV-2 genome<ul><li> Looking at<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>, which also provides a very nice overview of SARS-CoV-2 variation, may give some hints.<li>The variant emergence graph heavily competes with the information in&#xA0;<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>, which claims to be updated daily.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=<person-group person-group-type="author"><name><surname>Alam</surname><given-names>I</given-names></name><etal/></person-group>:<article-title>CovMT: an interactive SARS-CoV-2 mutation tracker, with a focus on critical variants.</article-title><source><italic toggle="yes">Lancet Infect Dis.</italic></source><year>2021</year>;<volume>21</volume>:<fpage>602</fpage>.<pub-id pub-id-type="doi">10.1016/S1473-3099(21)00078-5</pub-id>"><person-group person-group-type="author"><name><surname>Alam</surname><given-names>I</given-names></name><etal/></person-group>:<article-title>CovMT: an interactive SARS-CoV-2 mutation tracker, with a focus on critical variants.</article-title><source><italic toggle="yes">Lancet Infect Dis.</italic></source><year>2021</year>;<volume>21</volume>:<fpage>602</fpage>.<pub-id pub-id-type="doi">10.1016/S1473-3099(21)00078-5</pub-id></a> <b>(4)</b> Single-nucleotide conservation state annotation of the SARS-CoV-2 genome<ul><li> Looking at&#xA0;<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>, which also provides a very nice overview of SARS-CoV-2 variation, may give some hints.<li><italic toggle="yes">The variant emergence graph heavily competes with the information in&#xA0;<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>, which claims to be updated daily.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34013874">34013874</a> <b>(28)</b> Molecular Epidemiology and Evolutionary Trajectory of Emerging Echovirus 30, Europe<ul><li><b>Nextstrain</b> VP1 Phylodynamic Analysis.<li>The dataset used for <b>Nextstrain</b> phylodynamic analysis comprised 1,285 sequences; 1,215 study sequences (region 1) and 70 complete VP1 sequences extracted from GenBank (<xref rid="SD1" ref-type="supplementary-material"><u>Appendix</u></xref> Figure 1).<li> We provided the resulting <b>Nextstrain</b> build for viewing (<ext-link xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/vp1" ext-link-type="uri">https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/vp1</ext-link>).<li><b>Nextstrain</b> VP1:3Dpol Tanglegram Phylodynamics.<li> We provided the resulting <b>Nextstrain</b> tanglegram build for viewing (<ext-link xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/3D:community/enterovirus-phylo/echo30-2019/vp1" ext-link-type="uri">https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/3D:community/enterovirus-phylo/echo30-2019/vp1</ext-link>) and the codes for both VP1 and 3Dpol analyses (<ext-link xlink:href="https://github.com/enterovirus-phylo/echo30-2019" ext-link-type="uri">https://github.com/enterovirus-phylo/echo30-2019</ext-link>).<li> We created an interactive representation of age and clinical features of sequences from E30 cases, which we made available on <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/vp1" ext-link-type="uri">https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/vp1</ext-link>).<li> B) Maximum likelihood trees constructed by using <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>) from which we dropped several problematic sequences, including group 5.<li>We used <b>Nextstrain</b> to create an interactive phylodynamic tree and map to explore relationships of the E30 study VP1 sequences in G1&#x2013;G6 (<ext-link xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/vp1" ext-link-type="uri">https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/vp1</ext-link>) (<xref rid="F3" ref-type="fig"><u>Figure 3</u></xref>, panel B; <xref rid="SD1" ref-type="supplementary-material"><u>Appendix</u></xref> Figure 2).<li> Interactive data are available on <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/vp1" ext-link-type="uri">https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/vp1</ext-link>).<li> We used <b>Nextstrain</b> to generate an interactive tanglegram of VP1 and 3Dpol RFs (<ext-link xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/3D:community/enterovirus-phylo/echo30-2019/vp1" ext-link-type="uri">https://<b>nextstrain</b>.org/community/enterovirus-phylo/echo30-2019/3D:community/enterovirus-phylo/echo30-2019/vp1</ext-link>) (<xref rid="F5" ref-type="fig"><u>Figure 5</u></xref>).<li> We used 110 sequences and rendered the tanglegram by using <b>Nextstrain</b> (<ext-link xlink:href="https://www.<b>nextstrain</b>.org" ext-link-type="uri">https://www.<b>nextstrain</b>.org</ext-link>).<li>Using <b>Nextstrain</b>, we visualized the various categories of demographic and clinical data, clades, and RFs.<li> Previous studies using <b>Nextstrain</b> with 2 EV-D68 datasets have shown the value of combining demographic and phylogenetic analysis, both as retrospective (<xref rid="R33" ref-type="bibr"><italic>33</italic></xref>) and real-time analysis (<xref rid="R34" ref-type="bibr"><italic>34</italic></xref>).<li> The E30 dataset and the 2 EV-D68 datasets (<xref rid="R33" ref-type="bibr"><italic>33</italic></xref>,<xref rid="R34" ref-type="bibr"><italic>34</italic></xref>) available on <b>Nextstrain</b> represent large nonpolio enterovirus datasets that support real-time tracking of viruses over time and across countries.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34100030">34100030</a> <b>(2)</b> Genomic epidemiology and associated clinical outcomes of a SARS-CoV-2 outbreak in a general adult hospital in Quebec<ul><li> The definition of haplotypes based on these 20 mutations found in samples from this study, and their corresponding <b>NextStrain</b> clades, is reported in <xref rid="SD2" ref-type="supplementary-material"><u>Supplementary Table 5</u></xref>.<li> We note that haplotype III and IX are grouped within the same Nexstrain clade, despite the fact that they differ at position 25563, and both differ from haplotype II (<b>NextStrain</b> clade 20C) at position 14408.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33536572">33536572</a> <b>(1)</b> Highly multiplexed oligonucleotide probe-ligation testing enables efficient extraction-free SARS-CoV-2 detection and viral genotyping<ul><li> In a proof-of-concept study, we designed genotyping probes for the 20 most entropic SARS-CoV-2 SNPs reported in the <b>Nextstrain</b> database (queried on 3/16/2020) [<xref rid="bib29" ref-type="bibr"><u>29</u></xref>].<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34072569">34072569</a> <b>(5)</b> Characterization of a Novel SARS-CoV-2 Genetic Variant with Distinct Spike Protein Mutations<ul><li> A phylogenetic tree was constructed using the tools implemented in <b>Nextstrain</b> custom builds (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</uri>, accessed on 28 May 2021) [<xref rid="B10-viruses-13-01029" ref-type="bibr"><u>10</u></xref>].<li>According to different classification nomenclatures, the NW sequences belong to clade 20B, according to <b>Nextstrain</b> [<xref rid="B10-viruses-13-01029" ref-type="bibr"><u>10</u></xref>]; clade GR, according GISAID; or lineage AT.1 (alias of B 1.1.370.1), according to PANGOLIN (Phylogenetic Assignment of Named Global Outbreak LINeages) [<xref rid="B14-viruses-13-01029" ref-type="bibr"><u>14</u></xref>].<li> On the <b>Nextstrain</b>-based tree, they form a separate, long branch within clade 20B (<xref ref-type="fig" rid="viruses-13-01029-f001"><u>Figure 1</u></xref>).<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33514928">33514928</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34047966">34047966</a> <b>(1)</b> Outbreak investigation of SARS-CoV-2 transmission in an emergency childcare centre<ul><li> All sequenced genomes belong to clade 20B according to the <b>Nextstrain</b> (Hadfield et al., <xref ref-type="bibr" rid="CR6"><u>2018</u></xref>) nomenclature and clade B.1.1 in the Pangolin nomenclature (Cov-lineages/pangolin.<li> [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Outbreak investigation of SARS-CoV-2 transmission in an emergency childcare centre<ul><li> Variation within the SARS-CoV-2 genome has been captured through variant calling of <b>Nextstrain</b> and ENA data, processed with Ensembl and available in the Variants category.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34042471">34042471</a> <b>(1)</b> Genome Sequencing Identified a SARS-CoV-2 Lineage B.1.1.7 Strain with a High Number of Mutations from Dhaka, Bangladesh<ul><li> The <b>Nextstrain</b> and PANGOLIN Web tools were used for comparative genomics (<xref rid="B11" ref-type="bibr"><u>11</u></xref>, <xref rid="B12" ref-type="bibr"><u>12</u></xref>).<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33514928">33514928</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34141890">34141890</a> <b>(2)</b> Overview of current state of research on the application of artificial intelligence techniques for COVID-19<ul><li> <b>NextStrain</b> generates the genomic epidemiology of COVID-19.<li> <b>Nextstrain</b> is working on the gene structure of SARS-CoV-2 to illustrate the spread of coronavirus.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34044037">34044037</a> <b>(1)</b> Genomic survey of SARS-CoV-2 vaccine breakthrough infections in healthcare workers from Kerala, India<ul><li> Phylogenetic clustering for the isolates was done using <b>Nextstrain</b> with additional SARS-CoV-2 genomes isolated from Kerala.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34033854">34033854</a> <b>(1)</b> How to choose the right real-time RT-PCR primer sets for the SARS-CoV-2 genome detection?<ul><li> 356 different lineages (Pangolin nomenclature) and 12 or 9 different clades (<b>Nextstrain</b> and GISAID nomenclature respectively) were represented including the variants of concern (VOCs) in Europe (B.1.1.7 first detection in United Kingdom; B.1.351 first detection in South Africa; P.1 first detection in Brazil).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34064143">34064143</a> <b>(2)</b> The Emerging Concern and Interest SARS-CoV-2 Variants<ul><li> The three main, generally used terminologies have been proposed (PANGO lineage, <b>Nextstrain</b>, and GISAID) and the first two of them are used in this manuscript.<li> The <b>Nextstrain</b> nomenclature system is based on a large scale diversity of SARS-CoV-2 patterns and label clades that can persist for at least several months and have significant geographic spread [<xref rid="B24-pathogens-10-00633" ref-type="bibr"><u>24</u></xref>].<li> [24](          <person-group person-group-type="author">            <collab>World Health Organization</collab>          </person-group>          <article-title>SARS-CoV-2 Genomic Sequencing for Public Health Goals</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://www.who.int/publications/i/item/WHO-2019-nCoV-genomic_sequencing-2021.1">https://www.who.int/publications/i/item/WHO-2019-nCoV-genomic_sequencing-2021.1</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-10">(accessed on 10 May 2021)</date-in-citation>        ) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34018094">34018094</a> <b>(2)</b> Update and Supplementary Articles Proteins of SARS CoV-2, Which Causes COVID-19, and the Interacting Proteins<ul><li>Name of strain: <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR8"><u>8</u></xref>], Pango lineage [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>]</th><th align="left" colspan="4">Mutation on the spike protein<sup>a</sup></th><th align="left" rowspan="2">Original geographical location</th></tr><tr><th align="left">{0&#x2013;300}</th><th align="left">{301&#x2013;600}</th><th align="left">{601&#x2013;900}</th><th align="left">{901&#x2013;1200}</th></tr></thead><tbody><tr><td align="left">1.<li>Name of strain: <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR8"><u>8</u></xref>], Pango lineage [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>]</th><th align="left" colspan="4">Mutation on the spike protein</th><th align="left" rowspan="2">Original geographical location</th></tr><tr><th align="left">{0&#x2013;300}</th><th align="left">{301&#x2013;600}</th><th align="left">{601&#x2013;900}</th><th align="left">{901&#x2013;1200}</th></tr></thead><tbody><tr><td align="left">4.<li> [8](          <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">https://nextstrain.org/</ext-link>        ) [9](          <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</ext-link>        ) [8](          <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">https://nextstrain.org/</ext-link>        ) [9](          <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</ext-link>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34029710">34029710</a> <b>(1)</b> SARS-CoV-2 variant from India to Marseille: The still active role of ports in the introduction of epidemics<ul><li>The Marseille-IHU-9101 genome, indicated by a white bold font and a black background, is classified in <b>Nextstrain</b> clade 20A and Pangolin lineage B.1.617.2.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34016042">34016042</a> <b>(6)</b> Genomic epidemiology of SARS-CoV-2 in Esteio, Rio Grande do Sul, Brazil<ul><li>Epidemiological data of the 21 sequenced samples from Esteio, RS, Brazil</p></caption><table frame="hsides" rules="groups"><thead><tr><th>GISAID Accession</th><th>Ct value</th><th>Collection month</th><th>Age range</th><th>Sex</th><th>Clinical status</th><th>Pangolin Lineage</th><th><b>Nextstrain</b> Clade</th></tr></thead><tbody><tr><td>EPI_ISL_831678.<li>After running the <b>Nextstrain</b> pipeline for quality control and subsampling, we obtained 3758 time-, geographical- and genetic-representative genomes to proceed phylogenetic inferences.<li> Finally, we assessed genome consensus sequence quality using Nextclade v0.8.1 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) and CoV-GLUE (<ext-link ext-link-type="uri" xlink:href="http://cov-glue.cvr.gla.ac.uk/">http://cov-glue.cvr.gla.ac.uk/</ext-link> [<xref ref-type="bibr" rid="CR48"><u>48</u></xref>];).<li> Moreover, we identified global virus lineages using Nextclade v0.8.1 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) and Pangolin v2.1.3 (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</ext-link> [<xref ref-type="bibr" rid="CR15"><u>15</u></xref>];).<li> Available sequences were then subjected to analysis inside the <b>NextStrain</b> ncov pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link> [<xref ref-type="bibr" rid="CR91"><u>91</u></xref>];).<li>All global sequences (until December 24, 2020) belonging to lineages B.1.1.248 (<italic>n</italic>=405) and B.1.1.33 (<italic>n</italic>=725), found in high frequency in this study, were recovered from the filtered MAFFT alignment performed inside <b>Nextstrain</b> ncov pipeline in the previous step.<li> [48](Singer J, Gifford R, Cotten M, Robertson D. CoV-GLUE: A Web Application for Tracking SARS-CoV-2 Genomic Variation. 2020. 10.20944/preprints202006.0225.v1.) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [91](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34093495">34093495</a> <b>(0)</b> Genomic Epidemiology of SARS-CoV-2 From Mainland China With Newly Obtained Genomes From Henan Province<ul><li>To identify differences between 19 genome sequences from Henan Province and reference sequences, and to find out which clades they are from and where on the SARS-CoV-2 tree they fall, we used the Nextclade web tool (beta v0.10.0, <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) to perform mutation calling and clade assignment.<li> The clade-defining mutations (shown on <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) are chosen such that assignment based on genotype works in most cases.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34016683">34016683</a> <b>(3)</b> Complete Genome Sequence of a SARS-CoV-2 Strain Sampled in Morocco in May 2020, Obtained Using Sanger Sequencing<ul><li> A total of 69 viral genome sequences are displayed, including (i) 66 genomes provided by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>) on 10 February 2021, counting the sequence reported in this announcement (in green); (ii) 2 mostly similar genome sequences from NCBI, USA/CA-CZB-1229/2020 (GenBank accession number <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MT499204.1" assigning-authority="insdc">MT499204.1</ext-link>) and TWN/CGMH-CGU-15/2020 (<ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MT374111.1" assigning-authority="insdc">MT374111.1</ext-link>) (in blue); and (iii) the reference genome of the SARS-CoV-2 Wuhan-Hu-1 isolate (<ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MN908947.3" assigning-authority="insdc">MN908947.3</ext-link>) (in red).<li>Sequence quality control, clade assignment, and mutation determination were done using the Nextclade tool v0.12.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34015049">34015049</a> <b>(5)</b> Dispersal dynamics of SARS-CoV-2 lineages during the first epidemic wave in New York City<ul><li> We inferred such a time-scaled phylogeny based on 828 viral genomic sequences collected within New York State throughout the first epidemic wave (March-May, 2020), as well as all 1,899 background sequences used in the <b>Nextstrain</b> [<xref rid="ppat.1009571.ref007" ref-type="bibr"><u>7</u></xref>] build focused on North America (which also includes genomic sequences from the other continents), for a total of 2,727 SARS-CoV-2 genomic sequences.<li> As described in the Methods section, the time-scaled phylogenetic tree was inferred following a procedure similar to the one used by the <b>Nextstrain</b> platform [<xref rid="ppat.1009571.ref007" ref-type="bibr"><u>7</u></xref>] (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>), and the resulting tree employed as a fixed topology in initial Bayesian discrete [<xref rid="ppat.1009571.ref008" ref-type="bibr"><u>8</u></xref>] and continuous [<xref rid="ppat.1009571.ref009" ref-type="bibr"><u>9</u></xref>] phylogeographic analyses.<li> (ii) we subsequently time-calibrated the resulting maximum-likelihood tree using TreeTime 0.7.4 [<xref rid="ppat.1009571.ref025" ref-type="bibr"><u>25</u></xref>], specifying a clock rate of 8x10<sup>-4</sup> substitutions per site per year (s/s/y), as in the <b>Nextstrain</b> workflow.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19779555">19779555</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20203288">20203288</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34031660">34031660</a> <b>(2)</b> Epidemiology and genetic diversity of SARS-CoV-2 lineages circulating in Africa<ul><li>Worldwide, we observed 782 Pango lineages, 9 GISAID clades and 10 <b>Nextstrain</b> clades.<li>Using <b>Nextstrain</b> clade nomenclature (<xref rid="SD4" ref-type="supplementary-material"><u>Supplementary Figure 1</u></xref>), we observed that 20A.EU1 is the dominant circulating clade in Europe followed by 20B and 20A.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34069681">34069681</a> <b>(2)</b> Global Genomic Analysis of SARS-CoV-2 RNA Dependent RNA Polymerase Evolution and Antiviral Drug Resistance<ul><li> Since selection analysis tools are limited in the number of sequences that can be handled, we down-sampled to represent all phylogenetic lineages from all countries and each month by 25 sequences using the <b>nextstrain</b> analysis pipeline v.2.0.0 (<b>nextstrain</b>.org) [<xref rid="B22-microorganisms-09-01094" ref-type="bibr"><u>22</u></xref>] resulting in 16,184 genomes that could be processed further.<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Global Genomic Analysis of SARS-CoV-2 RNA Dependent RNA Polymerase Evolution and Antiviral Drug Resistance<ul><li> All SARS-CoV-2 strains belonged to clade 19A identified by <b>Nextstrain</b>, and they harbored a single nucleotide mutation at position 11083 (G11083T transversion), leading to a nonsynonymous amino acid substitution (Leu37Phe) in nonstructural protein 6 (nsp6), as previously described from the Diamond Princess outbreak event (<xref rid="B9" ref-type="bibr"><u>9</u></xref>).<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32723824">32723824</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34001211">34001211</a> <b>(1)</b> Shortcomings of SARS-CoV-2 genomic metadata<ul><li> Databases such as the global initiative on sharing avian influenza data (GISAID) [<xref ref-type="bibr" rid="CR22"><u>22</u></xref>] and <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR23"><u>23</u></xref>] have empowered an impressive array of SARS-CoV-2 studies by maintaining SARS-CoV-2 genomic sequences and corresponding metadata.<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28537546">28537546</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34001237">34001237</a> <b>(1)</b> Global genomic pathogen surveillance to inform vaccine strategies: a decade-long expedition in pneumococcal genomics<ul><li> Popular examples of visualisation software include Microreact [<xref ref-type="bibr" rid="CR96"><u>96</u></xref>] and <b>NextStrain</b> [<xref ref-type="bibr" rid="CR97"><u>97</u></xref>].<li> [96](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28348833">28348833</a>) [97](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34067890">34067890</a> <b>(5)</b> Structural Analysis of the Novel Variants of SARS-CoV-2 and Forecasting in North America<ul><li>SARS-CoV-2 genomic sequence data was gathered from the <b>Nextstrain</b> global database (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri> accessed on 2 May 2021), and mutation site numbering and genome structure were done using the Wuhan-Hu-1/2019 genomic sequence as reference (wild-type).<li> The database created by the Global Initiative on Sharing All Influenza Data (GISAID) and the analysis tool <b>Nextstrain</b> have enabled the analysis of thousands of whole-genome sequences of SARS-CoV-2 [<xref rid="B25-viruses-13-00930" ref-type="bibr"><u>25</u></xref>].<li> The <b>Nextstrain</b> data bank groups the isolates by regions.<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34043946">34043946</a> <b>(6)</b> SARS-CoV-2 genomic surveillance identifies naturally occurring truncation of ORF7a that limits immune suppression<ul><li> Subsampling was performed using Augur utility (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0155">https://<b>nextstrain</b>.org</ext-link>) by selecting 10 genomes per country per month since the start of the pandemic.<li><hr/></td></tr><tr><td>SDS software v1.4</td><td>Applied Biosystems</td><td>4379633</td></tr><tr><td>RStudio v1.2.1335</td><td>The R project</td><td><ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0035">https://www.r-project.org/</ext-link></td></tr><tr><td>artic-ncov2019</td><td>ARTIC network</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0040">https://artic.network/ncov-2019</ext-link></td></tr><tr><td>MinKNOW software</td><td>Oxford Nanopore Technologies</td><td><ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016" id="interref0010">https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016</ext-link></td></tr><tr><td>PrimePCR analysis software</td><td>Bio-Rad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a" id="intref0045">https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a</ext-link></td></tr><tr><td>CD-HIT</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/weizhongli/cdhit" id="intref0050">https://github.com/weizhongli/cdhit</ext-link></td></tr><tr><td>BioStrings</td><td>Bioconductor</td><td><ext-link ext-link-type="uri" xlink:href="https://bioconductor.org/packages/release/bioc/html/Biostrings.html" id="intref0055">https://bioconductor.org/packages/release/bioc/html/Biostrings.html</ext-link></td></tr><tr><td>ggplot2</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/ggplot2/index.html" id="intref0060">https://cran.r-project.org/web/packages/ggplot2/index.html</ext-link></td></tr><tr><td>Augur</td><td><b>Nextstrain</b>.<li><hr/></td></tr><tr><td>SDS software v1.4</td><td>Applied Biosystems</td><td>4379633</td></tr><tr><td>RStudio v1.2.1335</td><td>The R project</td><td><ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0035">https://www.r-project.org/</ext-link></td></tr><tr><td>artic-ncov2019</td><td>ARTIC network</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0040">https://artic.network/ncov-2019</ext-link></td></tr><tr><td>MinKNOW software</td><td>Oxford Nanopore Technologies</td><td><ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016" id="interref0010">https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016</ext-link></td></tr><tr><td>PrimePCR analysis software</td><td>Bio-Rad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a" id="intref0045">https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a</ext-link></td></tr><tr><td>CD-HIT</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/weizhongli/cdhit" id="intref0050">https://github.com/weizhongli/cdhit</ext-link></td></tr><tr><td>BioStrings</td><td>Bioconductor</td><td><ext-link ext-link-type="uri" xlink:href="https://bioconductor.org/packages/release/bioc/html/Biostrings.html" id="intref0055">https://bioconductor.org/packages/release/bioc/html/Biostrings.html</ext-link></td></tr><tr><td>ggplot2</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/ggplot2/index.html" id="intref0060">https://cran.r-project.org/web/packages/ggplot2/index.html</ext-link></td></tr><tr><td>Augur</td><td><b>Nextstrain</b></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0065">https://github.com/<b>nextstrain</b>/augur</ext-link>.<li><hr/></td></tr><tr><td>SDS software v1.4</td><td>Applied Biosystems</td><td>4379633</td></tr><tr><td>RStudio v1.2.1335</td><td>The R project</td><td><ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0035">https://www.r-project.org/</ext-link></td></tr><tr><td>artic-ncov2019</td><td>ARTIC network</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0040">https://artic.network/ncov-2019</ext-link></td></tr><tr><td>MinKNOW software</td><td>Oxford Nanopore Technologies</td><td><ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016" id="interref0010">https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016</ext-link></td></tr><tr><td>PrimePCR analysis software</td><td>Bio-Rad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a" id="intref0045">https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a</ext-link></td></tr><tr><td>CD-HIT</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/weizhongli/cdhit" id="intref0050">https://github.com/weizhongli/cdhit</ext-link></td></tr><tr><td>BioStrings</td><td>Bioconductor</td><td><ext-link ext-link-type="uri" xlink:href="https://bioconductor.org/packages/release/bioc/html/Biostrings.html" id="intref0055">https://bioconductor.org/packages/release/bioc/html/Biostrings.html</ext-link></td></tr><tr><td>ggplot2</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/ggplot2/index.html" id="intref0060">https://cran.r-project.org/web/packages/ggplot2/index.html</ext-link></td></tr><tr><td>Augur</td><td><b>Nextstrain</b></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0065">https://github.com/<b>nextstrain</b>/augur</ext-link></td></tr><tr><td>Nextclade</td><td><b>Nextstrain</b>.<li><hr/></td></tr><tr><td>SDS software v1.4</td><td>Applied Biosystems</td><td>4379633</td></tr><tr><td>RStudio v1.2.1335</td><td>The R project</td><td><ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0035">https://www.r-project.org/</ext-link></td></tr><tr><td>artic-ncov2019</td><td>ARTIC network</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0040">https://artic.network/ncov-2019</ext-link></td></tr><tr><td>MinKNOW software</td><td>Oxford Nanopore Technologies</td><td><ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016" id="interref0010">https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016</ext-link></td></tr><tr><td>PrimePCR analysis software</td><td>Bio-Rad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a" id="intref0045">https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a</ext-link></td></tr><tr><td>CD-HIT</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/weizhongli/cdhit" id="intref0050">https://github.com/weizhongli/cdhit</ext-link></td></tr><tr><td>BioStrings</td><td>Bioconductor</td><td><ext-link ext-link-type="uri" xlink:href="https://bioconductor.org/packages/release/bioc/html/Biostrings.html" id="intref0055">https://bioconductor.org/packages/release/bioc/html/Biostrings.html</ext-link></td></tr><tr><td>ggplot2</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/ggplot2/index.html" id="intref0060">https://cran.r-project.org/web/packages/ggplot2/index.html</ext-link></td></tr><tr><td>Augur</td><td><b>Nextstrain</b></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0065">https://github.com/<b>nextstrain</b>/augur</ext-link></td></tr><tr><td>Nextclade</td><td><b>Nextstrain</b></td><td><ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0070">https://clades.<b>nextstrain</b>.org/</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33992693">33992693</a> <b>(1)</b> Structural Modeling of the SARS-CoV-2 Spike/Human ACE2 Complex Interface can Identify High-Affinity Variants Associated with Increased Transmissibility<ul><li> Recent emergence of fast-spreading SARS-CoV-2 lineages in UK, South Africa, Brazil, India and other countries reinforces the need for surveillance and to anticipate infectious mutations (<b>nextstrain</b>.org/sars-cov-2).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33988184">33988184</a> <b>(1)</b> Identification of and Surveillance for the SARS-CoV-2 Variants B.1.427 and B.1.429 &#x2014; Colorado, January&#x2013;March 2021<ul><li> COVID-19 B.1.427/B.1.429 variant cases were identified through tiled amplicon WGS.<xref rid="FN2" ref-type="fn"><sup>&#x2020;</sup></xref><sup>,</sup><xref rid="FN3" ref-type="fn"><sup>&#xA7;</sup></xref> Assembly of sequencing data into whole genomes was performed using CDPHE&#x2019;s publicly available Illumina and Nanopore data workflows.<xref rid="FN4" ref-type="fn"><sup>&#xB6;</sup></xref> Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin)<xref rid="FN5" ref-type="fn"><u>**</u></xref> (<xref rid="R5" ref-type="bibr"><italic>5</italic></xref>) and <b>Nextstrain</b>&#x2019;s Nextclade tools (<xref rid="R6" ref-type="bibr"><italic>6</italic></xref>) were used to assign lineage designations to each assembled genome.<li> [**](<a href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33985433">33985433</a> <b>(5)</b> IPD 2.0: To derive insights from an evolving <italic>SARS-CoV-2</italic> genome<ul><li>For clade assignment to a particular sample, IPD 2.0 uses the sample variants and compares them against the known clade-specific variants described in the <b>NextStrain</b> repository (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> Clade assessment of the sequences was performed using the NextClade module of the <b>NextStrain</b> package [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>] and used as a truth set.<li>With the inclusion of the variant database and revised clade assessment module based on the recent clade nomenclature proposed [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>], we benchmarked the <italic>SARS-CoV-2</italic> clade assessment module of IPD 2.0 against the <b>NextStrain</b> clade assignments.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34055726">34055726</a> <b>(2)</b> Applying Genomic Epidemiology to Characterize a COVID-19 Outbreak in a Developmentally Disabled Adult Group Home Setting, Arizona<ul><li> Maximum likelihood phylogenetic trees containing the outbreak genomes and a subset of other Arizona genomes for context were generated using the Wuhan-1 genome as a reference using <b>NextStrain</b> (<xref rid="B13" ref-type="bibr"><u>13</u></xref>, <xref rid="B14" ref-type="bibr"><u>14</u></xref>).<li>Maximum likelihood phylogenetic tree of 74 SARS-CoV-2 genomes from Northern Arizona, May-June 2020 generated by <b>Nextstrain</b> (<xref rid="B13" ref-type="bibr"><u>13</u></xref>, <xref rid="B14" ref-type="bibr"><u>14</u></xref>) using the Wuhan1 genome as a reference (EPI_ISL_402125), showing 18/20 samples sequenced from this outbreak form one tight clade.<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Hadfield</surname><given-names>J</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name><name><surname>Bedford</surname><given-names>T</given-names></name></person-group>. <article-title>Year-letter genetic clade naming for SARS-CoV-2 on Nextstrain.org</article-title>. <source>Virological.org</source>. (<year>2020</year>). Available online at: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/year-letter-genetic-clade-naming-for-sars-cov-2-on-nextstrain-org/498">https://virological.org/t/year-letter-genetic-clade-naming-for-sars-cov-2-on-nextstrain-org/498</ext-link> (accessed November 30, 2020).) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Hadfield</surname><given-names>J</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name><name><surname>Bedford</surname><given-names>T</given-names></name></person-group>. <article-title>Year-letter genetic clade naming for SARS-CoV-2 on Nextstrain.org</article-title>. <source>Virological.org</source>. (<year>2020</year>). Available online at: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/year-letter-genetic-clade-naming-for-sars-cov-2-on-nextstrain-org/498">https://virological.org/t/year-letter-genetic-clade-naming-for-sars-cov-2-on-nextstrain-org/498</ext-link> (accessed November 30, 2020).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33986354">33986354</a> <b>(0)</b> Surge of severe acute respiratory syndrome coronavirus 2 infections linked to single introduction of a virus strain in Myanmar, 2020<ul><li> The variant calling was done by CoV-GLUE (<ext-link ext-link-type="uri" xlink:href="http://cov-glue.cvr.gla.ac.uk/">http://cov-glue.cvr.gla.ac.uk/</ext-link>)<sup><xref ref-type="bibr" rid="CR9"><u>9</u></xref></sup> and NextClade v0.5.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR10"><u>10</u></xref></sup> to assign the clade.<li> [9](Singer, J., Gifford, R., Cotten, M. &amp; Robertson, D. CoV-GLUE: A web application for tracking SARS-CoV-2 genomic variation. 2020060225. 10.20944/preprints202006.0225.v1 (2020).) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(6)</b> Surge of severe acute respiratory syndrome coronavirus 2 infections linked to single introduction of a virus strain in Myanmar, 2020<ul><li>Phylogenetic analysis was completed using tools implemented in <b>Nextstrain</b> custom builds (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) [<xref rid="CIT0018" ref-type="bibr"><u>18</u></xref>, <xref rid="CIT0019" ref-type="bibr"><u>19</u></xref>].<li> We found that 24 (92%) of these 26 genomic sequences cluster tightly in the <b>Nextstrain</b> 20A clade on a time-resolved tree and are separated by 0&#x2013;2 fixed consensus nucleotide differences (<xref rid="F3" ref-type="fig"><u>Figure 3</u></xref>).<li> <italic toggle="yes">A,</italic> Time-resolved phylogenetic tree created using <b>Nextstrain</b> tools and nomenclature showing the team sequences contextualized with all available community sequences (<italic toggle="yes">gray</italic>) for 25 of 32 confirmed cases (78%) associated with outbreak 1; tips affiliated with outbreak 1 are colored red.<li> <italic toggle="yes">A,</italic> Time-resolved phylogenetic tree created using <b>Nextstrain</b> tools and nomenclature showing 8 of 12 available samples (67%) from outbreak 2 sequences contextualized with all available community sequences (<italic toggle="yes">light gray</italic>).<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33991677">33991677</a> <b>(2)</b> Spreading of a new SARS-CoV-2 N501Y spike variant in a new lineage<ul><li> We describe herein the first SARS-CoV-2 spike 501Y variant belonging to <b>Nextstrain</b> clade 19B.<li> Sequences were analysed using <sc>Nextclade</sc> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0025">https://clades.<b>nextstrain</b>.org/</ext-link>) [<xref rid="bib6" ref-type="bibr"><u>6</u></xref>] and an in-house <sc>Python</sc> script.<li> In addition, sequences that span codons 1&#x2013;618 of the spike gene and allow the identification of this variant by covering four of its hallmark mutations (C21614T (substitution L18F), G22468T, T22917G (L452R), A23063T (N501Y)) and by being devoid of the <b>Nextstrain</b> clade 20 A23403G mutation, were obtained from 43 other patients (deposited at: <ext-link ext-link-type="uri" xlink:href="https://doi.org/10.35081/43r6-sz33" id="intref0035">https://doi.org/10.35081/43r6-sz33</ext-link>).<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33980686">33980686</a> <b>(3)</b> How To Prepare for the Unexpected: a Public Health Laboratory Response<ul><li> For example, <b>Nextstrain</b> is an open-source project that is founded on the open sharing of genetic information to improve research and public health responses to emerging diseases (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34065789">34065789</a> <b>(7)</b> Molecular Analysis of SARS-CoV-2 Circulating in Bangladesh during 2020 Revealed Lineage Diversity and Potential Mutations<ul><li> Another nomenclature system has been used by <b>Nextstrain</b> [<xref rid="B11-microorganisms-09-01035" ref-type="bibr"><u>11</u></xref>] to designate clades of SARS-CoV-2, and it has also been followed among researchers.<li> Nextclade beta (<uri xlink:href="https://clades.<b>nextstrain</b>.org/results">https://clades.<b>nextstrain</b>.org/results</uri> (accessed on 23 February 2021) was used to check the sequence quality and to generate the clade designated tree.<li> The <b>Nextstrain</b> clade assignment revealed that Bangladeshi SARS-CoV-2 produced at least five distinct clades: 20A, 20B, 20C, 19A, and 19B (<xref ref-type="fig" rid="microorganisms-09-01035-f001"><u>Figure 1</u></xref>).<li> The lineage B.1.1.25 (<italic>n</italic> = 565) belonged to the GISAID clade GR and the <b>Nextstrain</b> clade 20B was the most prevalent lineage (71.4%) among the SARS-CoV-2 sequences reported from Bangladesh.<li> Phylogenetic analysis using the GISAID [<xref rid="B9-microorganisms-09-01035" ref-type="bibr"><u>9</u></xref>]/<b>Nextstrain</b> [<xref rid="B11-microorganisms-09-01035" ref-type="bibr"><u>11</u></xref>]/Lineage [<xref rid="B10-microorganisms-09-01035" ref-type="bibr"><u>10</u></xref>] nomenclature system identified several clades and lineages among Bangladeshi virus populations.<li> The GISAID or <b>Nextstrain</b> clades are differentiated based on their mutations in amino acid sequences.<li> Similarly, <b>Nextstrain</b> clade 20B and lineage B are also quite common in the country.<li>Molecular genomic dynamics, using GISAID, <b>Nextstrain</b>, or other bioinformatics platforms applied to SARS-CoV-2 characterization in Bangladesh, revealed many aspects of the circulating viruses already known to the authorities, especially the introduction of SARS-CoV-2 strains into Bangladesh from many other foreign countries at the beginning of the pandemic.<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34065871">34065871</a> <b>(1)</b> SARS-CoV-2 Persistent Viral Shedding in the Context of Hydroxychloroquine-Azithromycin Treatment<ul><li>A total of 21 complete (<italic toggle="yes">n</italic> = 11) or partial (approximately the first half of the spike-encoding gene, <italic toggle="yes">n</italic> = 10) genome sequences were obtained for the 34 persistent shedders, comprising 14 classified in <b>Nextstrain</b> clade 20A, 3 in clade 20B and 4 in clade 20C [<xref rid="B13-viruses-13-00890" ref-type="bibr"><u>13</u></xref>].<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33535105">33535105</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34066205">34066205</a> <b>(4)</b> Symptomatic SARS-CoV-2 Reinfection in a Healthy Healthcare Worker in Italy Confirmed by Whole-Genome Sequencing<ul><li> if the strains of the two episodes belonged to different clades, as defined in the <b>Nextstrain</b> and GISAID databases, ideally with other evidence of infection (e.g., high viral titers in each sample, detection of sgmRNA, or viral culture).<li>The phylogenetic analysis was performed using the Nextclade sequence analysis webapp (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 14 February 2021).<li> The clade information was described using the GISAID and <b>Nextstrain</b> nomenclature, and the lineage information was described using the Pangolin nomenclature.<li> In particular, the SARS-CoV-2/human/ITA/APU-POLBA21/2020 strain belongs to the GISAID clade GR, <b>Nextstrain</b> clade 20 B, PANGO lineage B.1.1.74, while the SARS-CoV-2/human/ITA/APU-POLBA20/2021 strain belongs to the GISAID clade GV, <b>Nextstrain</b> clade 20 E(EU1), PANGO lineage B.1.177 (<xref ref-type="fig" rid="viruses-13-00899-f001"><u>Figure 1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34237251">34237251</a> <b>(2)</b> Computational methods to predict the mutational landscape of the spike protein<ul><li> Sequencing results show that, although SARS-CoV-2 has a relatively low rate, the spike protein varies more than any other SARS-CoV-2 protein (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/sars-like-cov" id="intref0020">https://<b>nextstrain</b>.org/groups/blab/sars-like-cov</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33997827">33997827</a> <b>(1)</b> Genomic variation, origin tracing, and vaccine development of SARS-CoV-2: A systematic review<ul><li>The phylogenetic history of the SARS-CoV-2 was reconstructed using a maximum likelihood approach<xref rid="bib53" ref-type="bibr"><sup>53</sup></xref> by <b>Nextstrain</b><xref rid="bib54" ref-type="bibr"><sup>54</sup></xref> based on genomes of 3,991 strains<xref rid="bib55" ref-type="bibr"><sup>55</sup></xref>, and the tree was shown and manipulated in itol.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33976134">33976134</a> <b>(6)</b> SARS-CoV-2 gene content and COVID-19 mutation impact by comparing 44 <italic>Sarbecovirus</italic> genomes<ul><li>Single nucleotide variants were downloaded from the &#x201C;<b>Nextstrain</b> Vars&#x201D; track in the UCSC Table Browser on 2020-04-18 at 11:46 AM EDT.<li> We found that there were at least seven independent reversion events by classifying the genomes containing the reversions into distinct lineages using the branch-defining mutations in the <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR98"><u>98</u></xref></sup> tree for 20I/501Y.V1 updated 2021-02-08 showing 175 of 3863 genomes sampled between December 2020 and January 2021.<li> UCSC data came from <b>nextstrain</b>.org<sup><xref ref-type="bibr" rid="CR98"><u>98</u></xref></sup>, which was derived from genome sequences deposited in GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org">https://www.gisaid.org</ext-link>)<sup><xref ref-type="bibr" rid="CR97"><u>97</u></xref></sup>.<li> Right of use and publication of the underlying sequences is entirely controlled by the authors of the original resource and the contributors of individual sequences, who are acknowledged in the <b>Nextstrain</b> metadata file (Supplementary Data&#xA0;<xref rid="MOESM19" ref-type="media"><u>16</u></xref>) Our analysis provides an additional layer of annotation on their work rather than replicating or replacing it.<li> Please contact the respective authors (available via the <b>Nextstrain</b> metadata.tsv file) if you intend to carry out further research using their data.<li> Derived data, such as phylogenies, can be downloaded from <b>nextstrain</b>.org (see &#x201C;DOWNLOAD DATA&#x201D; link at bottom of page)&#x2014;please contact the relevant authors where appropriate.&#x201D;.<li> [98](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [98](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [97](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34064904">34064904</a> <b>(3)</b> Mutation in a SARS-CoV-2 Haplotype from Sub-Antarctic Chile Reveals New Insights into the Spike&#x2019;s Dynamics<ul><li> The multi-tool platform <b>Nextstrain</b> [<xref rid="B21-viruses-13-00883" ref-type="bibr"><u>21</u></xref>] was used for genome alignment (MAFFT v7.4 [<xref rid="B22-viruses-13-00883" ref-type="bibr"><u>22</u></xref>]), phylogenetic analysis (IQ-TREE v1.6 [<xref rid="B23-viruses-13-00883" ref-type="bibr"><u>23</u></xref>]), and maximum-likelihood phylodynamic analysis (Treetime [<xref rid="B24-viruses-13-00883" ref-type="bibr"><u>24</u></xref>]).<li> During phase 1, three <b>Nextstrain</b> clade 20A variants (C3037T, C14408T, and A23403G) were identified, which accounted for 43.3% of cases.<li> However, two variants of <b>Nextstrain</b> clade 20C (C1059T and G25563T) emerged during this initial phase (56.7%).<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33966626">33966626</a> <b>(4)</b> COVID-19 one year into the pandemic: from genetics and genomics to therapy, vaccination, and policy<ul><li> SARS-CoV-2 variants identified are heterogeneously distributed in geographic areas of the world (see <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) [<xref ref-type="bibr" rid="CR11"><u>11</u></xref>, <xref ref-type="bibr" rid="CR12"><u>12</u></xref>].<li> Analyses and illustrations were respectively generated using augur and auspice (by <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/"><b>nextstrain</b>.org</ext-link>) [<xref ref-type="bibr" rid="CR11"><u>11</u></xref>].<li> [11](Nextstrain (2021). &#x201C;Real-time tracking of pathogen evolution&#x201D;. Retrieved 31/03, 2021, from <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link>.) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](Nextstrain (2021). &#x201C;Real-time tracking of pathogen evolution&#x201D;. Retrieved 31/03, 2021, from <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link>.) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33896413">33896413</a> <b>(3)</b> Emerging variants of concern in SARS-CoV-2 membrane protein: a highly conserved target with potential pathological and therapeutic implications<ul><li>Phylogenetic analysis was conducted using the <b>NextStrain</b> phylogenetic pipeline (version 3.0.1) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33964965">33964965</a> <b>(4)</b> Epidemiology and evolution of Middle East respiratory syndrome coronavirus, 2012&#x2013;2020<ul><li>The whole-genome sequences were analyzed using toolkits provided by the <b>Nextstrain</b> framework [<xref ref-type="bibr" rid="CR22"><u>22</u></xref>].<li> Spatiotemporal migration dynamics of clade C of MERS-CoV using <b>Nextstrain</b>.&#xA0;<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/wqshi/merse">https://<b>nextstrain</b>.org/community/wqshi/merse</ext-link>.<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33975021">33975021</a> <b>(5)</b> A Sanger-based approach for scaling up screening of SARS-CoV-2 variants of interest and concern<ul><li> After Sanger sequencing, samples were assigned to a lineage according to the mutational profile described in <xref rid="t0005" ref-type="table"><u>Table 1</u></xref>and in WGS, according to the Pango and <b>Nextstrain</b> lineages naming assignment (<xref rid="bb0025" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bb0055" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>).<li>According to the WGS, from the twelve COVID-19 samples evaluated, six were from the B.1.1 lineage (non-VOC, <b>Nextstrain</b> clade 20B), three were P.2 (VOI, <b>Nextstrain</b> clade 20B), two were P.1 (VOC, <b>Nextstrain</b> clade 20&#xA0;J/501Y&#xB7;V3) and one was B.1.1.117 (non-VOC, <b>Nextstrain</b> clade 20B).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33961632">33961632</a> <b>(7)</b> Multiplex qPCR discriminates variants of concern to enhance global surveillance of SARS-CoV-2<ul><li> (<bold>B</bold>) The <b>Nextstrain</b> &#x201C;global build&#x201D; (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/ncov/global"><b>nextstrain</b>.org/ncov/global</ext-link>) accessed on January 22, 2021 showing the phylogenetic representation of 4,046 SARS-CoV-2 genomes colored by the presence of deletions at amino acid positions ORF1a 3575&#x2013;3677 and spike 69&#x2013;70.<li>We analyzed over 400,000 SARS-CoV-2 genomes on GISAID and used custom <b>Nextstrain</b> builds [<xref rid="pbio.3001236.ref014" ref-type="bibr"><u>14</u></xref>] to identify that a 9-nucleotide deletion in the ORF1a gene at amino acid positions 3675&#x2013;3677 (ORF1a &#x394;3675&#x2013;3677) occurs in the B.1.1.7, B.1.351, and P.1 variants, but is only found in 0.03% (103/377,011) of all other genomes (<bold><xref ref-type="fig" rid="pbio.3001236.g001"><u>Fig 1</u></xref></bold>, <bold><xref ref-type="supplementary-material" rid="pbio.3001236.s001"><u>S1 Table</u></xref></bold>).<li> Phylogenetic analysis of a subset of 4,046 SARS-CoV-2 genomes was performed using <b>Nextstrain</b> [<xref rid="pbio.3001236.ref014" ref-type="bibr"><u>14</u></xref>], downsampled as shown using the &#x201C;global build&#x201D; on January 22, 2021 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33991511">33991511</a> <b>(2)</b> Epitope profiling reveals binding signatures of SARS-CoV-2 immune response in natural infection and cross-reactivity with endemic human CoVs<ul><li> To evaluate whether the epitopes identified in our phage display assay contained sites with high incidence of mutation, we harnessed existing mutation frequency data from the <b>Nextstrain</b> database to evaluate the median and maximum entropy across all amino acids in the epitopes listed in <xref rid="tbl2" ref-type="table"><u>Table 2</u></xref> (<xref rid="bib17" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> Multiple strains of the endemic CoVs were selected to cover a wide range of circulation chronology using the <b>Nextstrain</b> database (<xref rid="bib17" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33956816">33956816</a> <b>(2)</b> A 20-year historical review of West Nile virus since its initial emergence in North America: Has West Nile virus become a neglected tropical disease?<ul><li> <italic>Grubaugh from the data available at</italic><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA"><italic>https</italic>:<italic>//<b>nextstrain</b></italic>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33952579">33952579</a> <b>(5)</b> COVID-19 rise in Bangladesh correlates with increasing detection of B.1.351 variant<ul><li> (C) The <b>Nextstrain</b> clades (Pangolin lineage) of the 204 SARS-CoV-2 sequenced at the Child Health Research Foundation.<li> Genomes with metadata are uploaded on GISAID.<xref rid="R10" ref-type="bibr"><u>10</u></xref> Additionally, we have developed and regularly maintain a Bangladesh-specific build in <b>Nextstrain</b><xref rid="R11" ref-type="bibr"><u>11</u></xref> of all SARS-CoV-2 isolates sequenced and uploaded to GISAID from entire Bangladesh.<li>The monthly distribution of the <b>Nextstrain</b> clades of SARS-CoV-2 sequences from our testing facility are shown in <xref ref-type="fig" rid="F1"><u>figure 1C</u></xref>.<li> Using genome sequencing, it was confirmed that in both cases, the first episode was caused by the <b>Nextstrain</b> clade 20A, and the second episode was by 20H/501Y.V2 (B.1.351).<li> Anonymised sequencing results are uploaded on online open-access platforms, GISAID and <b>Nextstrain</b>.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34063166">34063166</a> <b>(5)</b> Phylogenetic Analysis of the 2020 West Nile Virus (WNV) Outbreak in Andalusia (Spain)<ul><li>The results can be viewed in the <b>Nextstrain</b> [<xref rid="B40-viruses-13-00836" ref-type="bibr"><u>40</u></xref>] local server, which is now part of the Andalusian Genomic Epidemiology System (SIEGA) [<xref rid="B41-viruses-13-00836" ref-type="bibr"><u>41</u></xref>].<li> The SIEGA <b>Nextstrain</b> Server.<li>The SIEGA <b>Nextstrain</b> server [<xref rid="B41-viruses-13-00836" ref-type="bibr"><u>41</u></xref>] offers an interactive view of the complete WNV phylogeny with the four WNV sequences belonging to the Spanish 2020 outbreak in the context of the rest of the viral sequences available, belonging to worldwide outbreaks, that have been sequenced in previous years (see <xref ref-type="app" rid="app1-viruses-13-00836"><u>Table S1</u></xref>).<li> <xref ref-type="app" rid="app1-viruses-13-00836"><u>Figure S1</u></xref> shows a summarized view of the whole phylogenetic tree as displayed by the SIEGA <b>Nextstrain</b> server.<li> One of the key tools to support this surveillance is the WNV <b>Nextstrain</b> Epidemiological Local Server [<xref rid="B41-viruses-13-00836" ref-type="bibr"><u>41</u></xref>].<li> [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [41](          <article-title>SIEGA. West Nile Virus Epidemiological Local Server&#x2014;SIEGA</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="http://nextstrain.clinbioinfosspa.es/wnv">http://nextstrain.clinbioinfosspa.es/wnv</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-04-20">(accessed on 20 April 2021)</date-in-citation>        ) [41](          <article-title>SIEGA. West Nile Virus Epidemiological Local Server&#x2014;SIEGA</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="http://nextstrain.clinbioinfosspa.es/wnv">http://nextstrain.clinbioinfosspa.es/wnv</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-04-20">(accessed on 20 April 2021)</date-in-citation>        ) [41](          <article-title>SIEGA. West Nile Virus Epidemiological Local Server&#x2014;SIEGA</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="http://nextstrain.clinbioinfosspa.es/wnv">http://nextstrain.clinbioinfosspa.es/wnv</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-04-20">(accessed on 20 April 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34017870">34017870</a> <b>(17)</b> Development of a Dissemination Platform for Spatiotemporal and Phylogenetic Analysis of Avian Infectious Bronchitis Virus<ul><li>The development of modern data visualization platforms such as <b>Nextstrain</b> (<xref rid="B40" ref-type="bibr"><u>40</u></xref>) allows the simultaneous analysis of phylogenetic, epidemiological, and geographic information (<xref rid="B41" ref-type="bibr"><u>41</u></xref>).<li> We described the spatiotemporal spread and evolution of 132 IBV samples across one state of the United States and additionally built an interactive web visualization dashboard based on the <b>Nextstrain</b> platform.<li> Our derivative of the <b>Nextstrain</b> platform provides a portal for complete phylodynamic analysis by research staff while affording producers' redacted views into the origins, spread, and phylodynamic analysis that preserve anonymity across producers who contribute samples for analysis but enable producers to learn valuable information about their operations.<li>To visualize the evolution and spread of IBV in an interactive tool that allows for near real-time updates using phylogenetic, spatial, and epidemic information based on outbreak data (<xref ref-type="supplementary-material" rid="SM1"><u>Supplementary Table 5</u></xref>), we used <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>) (<xref rid="B40" ref-type="bibr"><u>40</u></xref>).<li> IBV sequences were processed in <b>Nextstrain</b>'s platform using the Augur bioinformatics toolkit (augur) (<xref rid="B67" ref-type="bibr"><u>67</u></xref>) and visualized using Auspice (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/auspice">https://github.com/<b>nextstrain</b>/auspice</ext-link>) (<xref rid="B68" ref-type="bibr"><u>68</u></xref>).<li>An overview of the modified <b>Nextstrain</b> architecture, leveraging industry-standard SSL encryption, JSON web tokens for identity management, and separate servers for web hosting and data storage to ensure information security.<li>Near Real-Time Visualization of Infectious Bronchitis Virus Spread Using the <b>Nextstrain</b> Platform.<li><b>Nextstrain</b> is a publicly accessible open-source tool that can be used for near real-time interactive tracking of pathogen spread and evolution (<xref rid="B40" ref-type="bibr"><u>40</u></xref>).<li> We utilized the <b>Nextstrain</b> platform to host our previous phylogeographic analysis.<li> Here, we have shown that the <b>Nextstrain</b> platform could be utilized to monitor the spread of IBV, helping to guide control activities in the field such as informing stakeholders about new viral emergence events and correlate current strain to vaccination strategies.<li>In this study, we provided the stakeholders with an alternative platform via <b>Nextstrain</b>, which provides updated information about IBV serotype circulation and the needed information to ensure targeted vaccination and efficient protection against current IBV serotypes.<li> Using the <b>Nextstrain</b> platform, this study provides a near real-time tool for tracking spread in North Carolina, where the collaboration between producers and researchers was the key to gather up-to-date IBV epidemiological information, maintaining the data confidentiality.<li> DR and AC designed and implemented the different security levels that accompanied <b>Nextstrain</b> visualization.<li> [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31104603">31104603</a>) [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [67](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16966358">16966358</a>) [68](<person-group person-group-type="author"><name><surname>Chiu</surname><given-names>D</given-names></name><name><surname>Agrawal</surname><given-names>G</given-names></name></person-group>. <article-title>Auspice: automatic service planning in cloud/grid environments</article-title>. In: <person-group person-group-type="editor"><name><surname>Cafaro</surname><given-names>M</given-names></name><name><surname>Aloisio</surname><given-names>G</given-names></name></person-group>, editors. <source>Grids, Clouds and Virtualization.</source><publisher-loc>London</publisher-loc>: <publisher-name>Springer</publisher-name> (<year>2011</year>). p. <fpage>93</fpage>&#x2013;<lpage>121</lpage>.) [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35262003">35262003</a> <b>(2)</b> Comparative genomic analysis demonstrates that true reinfection following SARS-CoV-2 infection is possible<ul><li>1st infection: GISAID clade V, <b>Nextstrain</b> clade 19A, Pangolin lineage B.2<break/>2nd infection: GISAID clade G, <b>Nextstrain</b> clade 20A, Pangolin lineage B.1.79.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33948037">33948037</a> <b>(2)</b> Testing at scale during the COVID-19 pandemic<ul><li> These sequences, along with metadata (including location, date and method of sampling) and tools such as <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>Nextstrain</b></ext-link><sup><xref ref-type="bibr" rid="CR13"><u>13</u></xref></sup>, could then be used to track the spread of SARS-CoV-2 strains<sup><xref ref-type="bibr" rid="CR14"><u>14</u></xref></sup>.<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32269081">32269081</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33941299">33941299</a> <b>(3)</b> Use of whole-genome sequencing to investigate a cluster of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections in emergency department personnel<ul><li> Variant positions were defined as any genomic position that had an alternate allele frequency &gt;50% in any of the sequenced samples and were determined using samtools and mpileup.<sup><xref rid="r7" ref-type="bibr"><u>7</u></xref>,<xref rid="r8" ref-type="bibr"><u>8</u></xref></sup> Sequences were assigned to a transmission cluster if they had &#x2264;2 base differences based on their variant positions.<sup><xref rid="r6" ref-type="bibr"><u>6</u></xref></sup> The consensus sequences for each strain were classified into 3 different commonly referenced clade classification systems using <b>Nextstrain</b> (<b>nextstrain</b>.org) and Pangolin (github.com/cov-lineages/pangolin).<li> Clade information based on <b>Nextstrain</b>, Global Initiative on Sharing Avian Influenza Data (GISAID), and Pangolin lineage.<li> [7](r7) [8](r8) [6](r6) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33938934">33938934</a> <b>(1)</b> Clinical Characteristics and Transmission of COVID-19 in Children and Youths During 3 Waves of Outbreaks in Hong Kong<ul><li> The circulating strains in 87.9% of the individuals with domestic infections in the third wave belonged to the Global Initiative on Sharing All Influenza Data clade GR, <b>Nextstrain</b> clade 20B, and Pangolin lineage B.1.1, in addition to D614G.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33941621">33941621</a> <b>(8)</b> Viral genomes reveal patterns of the SARS-CoV-2 outbreak in Washington State<ul><li> Since its first occurrence, this amino acid substitution from aspartate (D) to glycine (G) at position 614 of the Spike protein increased in relative frequency around the world (visible at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614">https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614</ext-link>) and now represents the vast majority of all new cases of COVID-19 (<xref rid="R3" ref-type="bibr"><italic>3</italic></xref>&#x2013;<xref rid="R5" ref-type="bibr"><italic>5</italic></xref>).<li> The local transmission clusters obtained are shown at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/wa-phylodynamics?c=cluster_size">https://<b>nextstrain</b>.org/groups/blab/ncov/wa-phylodynamics?c=cluster_size</ext-link>, and their size distribution and D614G makeup are shown in fig.<li> First, we considered the genetic diversity defined by five viral clades using the <b>Nextstrain</b> nomenclature: 19A, 19B, 20A, 20B, and 20C (fig.<li> We first built a timed tree using sequences from Washington State and from around the world using the <b>Nextstrain</b> pipeline (<xref rid="R3" ref-type="bibr"><italic>3</italic></xref>).<li> The D clade consists of all sequences with an aspartic acid at site 614 of the spike protein, and the G clade consists of all sequences with a glycine at this position (visible at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614">https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33797393">33797393</a> <b>(2)</b> Genetic Evidence and Host Immune Response in Persons Reinfected with SARS-CoV-2, Brazil<ul><li> We determined SARS-CoV-2 clades using the Nextclade software, beta version 0.14 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).<li> The original sequences used in this work are publicly available on <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov</ext-link>: GISAID EPI ISL nos.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33900175">33900175</a> <b>(0)</b> Use of Genomics to Track Coronavirus Disease Outbreaks, New Zealand<ul><li> For example, 12 SARS-CoV-2 genomes from persons returning to New Zealand from India who arrived on the same flight fell across at least 4 genomic lineages and comprised sequence divergence of up to 34 single-nucleotide polymorphisms (<ext-link xlink:href="https://www.<b>nextstrain</b>.org" ext-link-type="uri">https://www.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33900177">33900177</a> <b>(5)</b> Monitoring SARS-CoV-2 Circulation and Diversity through Community Wastewater Sequencing, the Netherlands and Belgium<ul><li> <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>) currently divides SARS-CoV-2 diversity into 12 major global clades (19A, 19B, and 20A&#x2013;20J), on the basis of high prevalence, signature mutations, and geographic spread (<xref rid="R3" ref-type="bibr"><italic>3</italic></xref>).<li> Clades (19A, 19B, 20A, 20B, and 20C) were assigned by using the Nextclade tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).<li> During the period of wastewater-sample collection, <b>Nextstrain</b> defined 5 major clades (19A, 19B, 20A, 20B, and 20C).<li> To evaluate this diversity in a comprehensive fashion, we used the <b>Nextstrain</b> clade classification system because it is based on the use of signature mutations to assign sequences to a clade (<xref rid="R3" ref-type="bibr"><italic>3</italic></xref>), enabling the association of SNPs or LFV to a particular clade, especially for genome sequences with &lt;75% coverage.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33928885">33928885</a> <b>(3)</b> SARS-CoV-2 mutations: the biological trackway towards viral fitness<ul><li> Mutations in SARS-CoV-2 proteins for respective countries were obtained from <b>NextStrain</b> open source project (<uri xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33914720">33914720</a> <b>(1)</b> COVID-19 Outbreak Associated with a SARS-CoV-2 R.1 Lineage Variant in a Skilled Nursing Facility After Vaccination Program &#x2014; Kentucky, March 2021<ul><li> Genomes were assembled using the StaPH-B Monroe pipeline,<xref rid="FN4" ref-type="fn"><sup>&#xB6;</sup></xref> followed by Nextclade<xref rid="FN5" ref-type="fn"><u>**</u></xref> for clade assignment and mutation calling, Pangolin<xref rid="FN6" ref-type="fn"><sup>&#x2020;&#x2020;</sup></xref> for lineage assignment, and <b>Nextstrain</b> for phylogenetic analysis (<xref rid="R3" ref-type="bibr"><italic>3</italic></xref>).<li> [**](<a href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33743213">33743213</a> <b>(1)</b> Multiple SARS-CoV-2 variants escape neutralization by vaccine-induced humoral immunity<ul><li>(A) Phylogenetic tree of SARS-CoV-2 variants (adapted from <b>nextstrain</b>.org) with sampling dates is illustrated with a focus on the following lineages: A (gray), B.1.1.7 (purple), B.1.1.298 (blue), B.1.429 (green), P.2 (yellow), P.1 (orange), and B.1.351 (red).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34796242">34796242</a> <b>(0)</b> The Longest Persistence of Viable SARS-CoV-2 With Recurrence of Viremia and Relapsing Symptomatic COVID-19 in an Immunocompromised Patient&#x2014;A Case Study<ul><li> We used the Nextclade and Pangolin systems (freely available at <ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link> and <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>, respectively) [<xref rid="CIT0013" ref-type="bibr"><u>13</u></xref>, <xref rid="CIT0014" ref-type="bibr"><u>14</u></xref>] for strain classification.<li> [13](Nextclade. Available at: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>. <comment>Accessed 17 December 2020</comment>.) [14](Pangolin COVID-19 Lineage Assigner. Available at: <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>. <comment>Accessed 17 December 2020</comment>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33924778">33924778</a> <b>(2)</b> Global Discrepancies between Numbers of Available SARS-CoV-2 Genomes and Human Development Indexes at Country Scales<ul><li> In our country, for example, the majority of SARS-CoV-2 strains that are currently circulating have a spike protein whose amino acid sequence differs from that used in vaccines, which corresponds to strains that no longer exist or are in the minority [<xref rid="B6-viruses-13-00775" ref-type="bibr"><u>6</u></xref>,<xref rid="B7-viruses-13-00775" ref-type="bibr"><u>7</u></xref>] (<uri xlink:href="https://<b>nextstrain</b>.org/groups/neherlab/ncov/france">https://<b>nextstrain</b>.org/groups/neherlab/ncov/france</uri>; accessed on 11 April 2021).<li> [6](          <person-group person-group-type="author">            <name>              <surname>Fournier</surname>              <given-names>P.E.</given-names>            </name>            <name>              <surname>Colson</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Levasseur</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Gautret</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Luciani</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Bedotto</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Delerce</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Brechard</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Lagier</surname>              <given-names>J.C.</given-names>            </name>            <name>              <surname>Chaudet</surname>              <given-names>H.</given-names>            </name>            <etal/>          </person-group>          <article-title>Genome sequence analysis enabled deciphering the atypical evolution of COVID-19 in Marseille, France</article-title>          <source>medRxiv</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.35088/kmct-tj43</pub-id>        ) [7](          <person-group person-group-type="author">            <name>              <surname>Fournier</surname>              <given-names>P.E.</given-names>            </name>            <name>              <surname>Colson</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Levasseur</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Devaux</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Gautret</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedotto</surname>              <given-names>M.</given-names>            </name>          </person-group>          <article-title>Emergence and outcome of the SARS-CoV-2 "Marseille-4" variant</article-title>          <source>Int. J. Infect. Dis.</source>          <year>2021</year>          <comment>online ahead of print</comment>          <pub-id pub-id-type="doi">10.1016/j.ijid.2021.03.068</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33930563">33930563</a> <b>(2)</b> Assessment of basic reproduction number (R<sub>0</sub>), spatial and temporal epidemiological determinants, and genetic characterization of SARS-CoV-2 in Bangladesh<ul><li> Finally, Next strain's SARS-CoV-2 global genomic epidemiology analysis differentiated ten major clades for this virus (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" id="ir0005">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33930542">33930542</a> <b>(1)</b> Evidence of SARS-CoV-2 reinfection within the same clade in Ecuador: A case study<ul><li> Clades were assigned using GISAID (<xref rid="bib0105" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>), <b>Nextstrain</b> (<xref rid="bib0110" ref-type="bibr"><u>Tillett et al., 2020</u></xref>), and Pangolin (<xref rid="bib0080" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>).<li> [Shu and McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [Tillett et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33058797">33058797</a>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33915337">33915337</a> <b>(1)</b> Increased viral variants in children and young adults with impaired humoral immunity and persistent SARS-CoV-2 infection: A consecutive case series<ul><li> Phylogenetic analysis and evolutionary rate estimation were performed using packages available through <b>Nextstrain</b> command-line interface (v 2.0.0.) [<xref rid="bib0025" ref-type="bibr"><u>25</u></xref>].<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33915216">33915216</a> <b>(8)</b> One year into the pandemic: Short-term evolution of SARS-CoV-2 and emergence of new lineages<ul><li>SARS-CoV-2 sequences deposited in GISAID are readily analysed in different platforms among which <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org" id="ir0025">www.<b>nextstrain</b>.org</ext-link>) has gained a well-deserved popularity.<li> The nomenclature used in <b>NextStrain</b> builds is based on the inferred phylogenetic relationships, ultimately depending on the accumulation of mutations.<li> The different names that a particular genome can receive are provided in the <b>Nextstrain</b> builds (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="ir0035">https://<b>nextstrain</b>.org/ncov</ext-link>).<li> All clades included in GISAID are represented in the table, but only a few of the lineages included in <b>NextStrain</b> and PANGO are considered.<li>Distribution of human SARS-CoV-2 sequences of human origin deposited at GISAID (on February 4, 2021) in the major clades/lineages according to the three nomenclature systems: GISAID (clades V, S, O, L, G, GH, GR, GV), <b>NextStrain</b> (only lineages 19A, 19B, 20A, 20A.EU2, 20B, 20C, 20D, 20E(EU1), 20G, 20H/501Y.V2, and 20I/501Y.V1), and PANGO (lineages A*, including 18 sublineages, and B*, including 815 sublineages).<li> VOC-202012/01 is also referred to as B.1.1.7 (PANGO nomenclature) and 20B/501Y.V1 (<b>Nextstrain</b> nomenclature).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33915212">33915212</a> <b>(2)</b> Heparin: A simplistic repurposing to prevent <italic>SARS-CoV-2</italic> transmission in light of its <italic>in-vitro</italic> nanomolar efficacy<ul><li> Further mutation data was obtained from the GISAID database and epidemiological resource <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="ir0050">https://<b>nextstrain</b>.org/</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33900399">33900399</a> <b>(3)</b> Genomic Epidemiology of SARS-CoV-2 Infection During the Initial Pandemic Wave and Association With Disease Severity<ul><li> Multiple sequence alignments were performed using MAFFT software version 7.0.<sup><xref rid="zoi210251r18" ref-type="bibr"><u>18</u></xref></sup> A maximum likelihood approach in <b>NextStrain</b><sup><xref rid="zoi210251r19" ref-type="bibr"><u>19</u></xref></sup> was used to build the phylogenetic tree, and a local installation of Auspice from <b>NextStrain</b> was used to visualize the phylogenetic tree and associated meta data (see eMethods in the <xref ref-type="supplementary-material" rid="note-ZOI210251-1-s"><u>Supplement</u></xref>).<li> The leaves are colored according to the GISAID clade, whereas the branches are labeled using <b>NextStrain</b> clade ID.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28968734">28968734</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33925854">33925854</a> <b>(1)</b> Mutation Signatures and In Silico Docking of Novel SARS-CoV-2 Variants of Concern<ul><li> (i) <b>Nextstrain</b> system has identified 11 major clades as 19A, 19B, and 20A-20I, (ii) Rambaut et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33903676">33903676</a> <b>(8)</b> Analysis and forecasting of global real time RT-PCR primers and probes for SARS-CoV-2<ul><li> We can thus examine the number of mismatches segmented by the clades, as defined by <b>Nextstrain</b>, in order to determine if primer-probe sets are skewed in performance to a particular clade.<li> for example, the American nucleoprotein primers have the highest number of mismatches for clade 19A, which <b>Nextstrain</b> defines as originating from predominantly Asian genomes, while the Chinese primer has the lowest number of mismatches for this clade.<li>Average number of mutations for each PCR test that occur within each clade, as defined by <b>NextStrain</b>.<li> The clades are defined by specific mutations at nucleotide locations, detailed by <b>Nextstrain</b> documentation, which only overlaps with the primer binding regions for 3.37% of the genomes.<li><b>Nextstrain</b> categorizes the various genetic phylogenies by clade, which is designed to denote long-term genetic changes based on mutation.<li> This indicates that variations in the primer target sequences have not yet have reached large enough statistical significance to define a new clade in the <b>Nextstrain</b> phylogeny, although the variants that are present in the primer region may cause a decrease in amplification signal within the assay.<li> For this analysis, we use the clades as indicated by <b>NextStrain</b>, which are defined by frequency and geographic spread.<li> Furthermore, <b>NextStrain</b> publishes the genome locus that defines each clade, and these loci were compared to the genome location the primer targets bind to.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33903676">33903676</a> <b>(1)</b> Analysis and forecasting of global real time RT-PCR primers and probes for SARS-CoV-2<ul><li> For example, <b>Nextstrain</b> (<xref rid="jkab126-B8" ref-type="bibr">Hadfield <italic toggle="yes">et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33922257">33922257</a> <b>(6)</b> Genomic Surveillance of Circulating SARS-CoV-2 in South East Italy: A One-Year Retrospective Genetic Study<ul><li>The Nextclade tool was used for isolate clades definition, mutation calling, and phylogenetic analyses (<b>Nextstrain</b>, Nextclade: <uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed: 28 January 2021).<li> Among these, Nextclade analysis allowed to identify three main clades: 51.27% (<italic>n</italic> = 181) of the genomes belonged to <b>Nextstrain</b> clade 20E(EU1), 29.18% (<italic>n</italic> = 103) belonged to clade 20B, and 9.63% (<italic>n</italic> = 34) belonged to clade 20A.<li>Furthermore, phylogeny reported on <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>, accessed 21 January 2021) showed that, in the global sampling, 20A and 20B (along with the 20C) were the most abundant clades, within the 11 known <b>Nextstrain</b> clades.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33975397">33975397</a> <b>(10)</b> Will Mutations in the Spike Protein of SARS-CoV-2 Lead to the Failure of COVID-19 Vaccines?<ul><li> By analyzing the genomic epidemiology of SARS-CoV-2 obtained from <b>nextstrain</b>.org (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>), we found that the viral mutations were more likely to occur in spring, autumn, and winter, and less likely in summer (as shown by white lines in <xref ref-type="fig" rid="F2">Fig.<li> The number of mutations in hCoV-19 genomes was obtained from <b>Nextstrain</b> database and showed by solid dot (<bold>B</bold>).<li> Compiled <b>Nextstrain</b> SARS-CoV-2 resources are available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>.<li>To determine mutations in SARS-CoV-2 gene sequences, we obtained, sequenced, and shared SARS-CoV-2 genomic data from the <b>Nextstrain</b> team (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>) and GISAID Mutation Tracker (<ext-link ext-link-type="uri" xlink:href="https://users.math.msu.edu/users/weig/SARS-CoV-2_Mutation_Tracker.html">https://users.math.msu.edu/users/weig/SARS-CoV-2_Mutation_Tracker.html</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33882219">33882219</a> <b>(0)</b> Vaccine Breakthrough Infections with SARS-CoV-2 Variants<ul><li> The SARS-CoV-2 genome was assembled with MEGAHIT with default parameters, and the longest sequence (30,005 nucleotides) was analyzed with Nextclade software (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) in order to assign the clade and call mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33919314">33919314</a> <b>(0)</b> A Potential SARS-CoV-2 Variant of Interest (VOI) Harboring Mutation E484K in the Spike Protein Was Identified within Lineage B.1.1.33 Circulating in Brazil<ul><li> The mutational profile was investigated using the Nextclade tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, accessed on 1 March 2021) and temporal signal was assessed by the regression analysis of the root-to-tip genetic distance against sampling dates using the program Tempest [<xref rid="B12-viruses-13-00724" ref-type="bibr"><u>12</u></xref>].<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27774300">27774300</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27774300">27774300</a> <b>(2)</b> A Potential SARS-CoV-2 Variant of Interest (VOI) Harboring Mutation E484K in the Spike Protein Was Identified within Lineage B.1.1.33 Circulating in Brazil<ul><li> The <b>Nextstrain</b> software tool is an open-source platform for exploiting the potential of genomic data from a variety of infectious disease pathogens to support epidemiological research and outbreak response with a GISAID-enabled interface of publicly available sequence data from every continent.<li> <b>Nextstrain</b> can address questions about which regions of the genome are most variable, estimate the rate of infection, and identify sequences on the phylogenetic tree by features such as gender and age of those infected.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27774300">27774300</a> <b>(1)</b> A Potential SARS-CoV-2 Variant of Interest (VOI) Harboring Mutation E484K in the Spike Protein Was Identified within Lineage B.1.1.33 Circulating in Brazil<ul><li> For each of them (Our World in Data, Worldometer, <b>Nextstrain</b> &#x2013; GISAID, EpiCoV, Oxford Government Response Tracker) the data were findable and accessible, metadata were well-defined and rich, but regarding a persistent ID for each data element, only the genetic data fulfilled this requirement.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33991487">33991487</a> <b>(9)</b> Transmission, infectivity, and neutralization of a spike L452R SARS-CoV-2 variant<ul><li> A variant, subsequently named 20C/L452R according to the <b>Nextstrain</b> nomenclature system (<xref rid="bib3" ref-type="bibr"><u>Bedford et&#xA0;al., 2021</u></xref>) or B.1.427/B.1.429 according to the Pango system (<xref rid="bib49" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020a</u></xref>) (henceforth referred to using the Pango designation to distinguish between the B.1.427 and B.1.429 lineages), was identified in 21.5% (466 of 2,172) of the genomes (<xref rid="mmc2" ref-type="supplementary-material"><u>Table S1</u></xref>).<li>Bayesian phylogenetic analysis of 1,153 genomes subsampled from a 2,519-genome dataset consisting of the 2,172 California genomes sequenced in this study and 347 representative global genomes (<xref rid="bib2" ref-type="bibr"><u>Bedford and Neher, 2020</u></xref>) identified two distinct lineages in clade 20C (<b>Nextstrain</b> designation) associated with the variant, B.1.427 and B.1.429 (<xref rid="fig2" ref-type="fig"><u>Figures 2</u></xref>A and 2C).<li> Defining single nucleotide polymorphisms (SNPs) in the B.1.427 and B.1.429 lineages are compared to each other and to other SARS-CoV-2 viruses in <b>Nextstrain</b> clade 20C.<li><hr/></td></tr><tr><td>BBTools suite, v38.87</td><td><xref rid="bib5" ref-type="bibr"><u>Bushnell, 2021</u></xref>, <ext-link ext-link-type="uri" xlink:href="https://jgi.doe.gov/data-and-tools/bbtools/" id="intref0020">https://jgi.doe.gov/data-and-tools/bbtools/</ext-link></td><td>N/A</td></tr><tr><td>MAFFT aligner v7.388</td><td><xref rid="bib32" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>, <ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/" id="intref0025">https://mafft.cbrc.jp/alignment/software/</ext-link></td><td>N/A</td></tr><tr><td>Geneious v11.1.5</td><td><xref rid="bib33" ref-type="bibr"><u>Kearse et&#xA0;al., 2012</u></xref>, <ext-link ext-link-type="uri" xlink:href="https://www.geneious.com" id="intref0030">https://www.geneious.com</ext-link></td><td>N/A</td></tr><tr><td><b>Nextstrain</b>/Augur pipeline v3.0.0.<li><hr/></td></tr><tr><td>BBTools suite, v38.87</td><td><xref rid="bib5" ref-type="bibr"><u>Bushnell, 2021</u></xref>, <ext-link ext-link-type="uri" xlink:href="https://jgi.doe.gov/data-and-tools/bbtools/" id="intref0020">https://jgi.doe.gov/data-and-tools/bbtools/</ext-link></td><td>N/A</td></tr><tr><td>MAFFT aligner v7.388</td><td><xref rid="bib32" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>, <ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/" id="intref0025">https://mafft.cbrc.jp/alignment/software/</ext-link></td><td>N/A</td></tr><tr><td>Geneious v11.1.5</td><td><xref rid="bib33" ref-type="bibr"><u>Kearse et&#xA0;al., 2012</u></xref>, <ext-link ext-link-type="uri" xlink:href="https://www.geneious.com" id="intref0030">https://www.geneious.com</ext-link></td><td>N/A</td></tr><tr><td><b>Nextstrain</b>/Augur pipeline v3.0.0</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0035">https://github.com/<b>nextstrain</b>/augur</ext-link>.<li> <xref rid="bib55" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>) (accession numbers in <xref rid="mmc2" ref-type="supplementary-material"><u>Table S1</u></xref>) and can be visualized in <b>Nextstrain</b>.<li>High-quality SARS-CoV-2 genomes (n&#xA0;= 2,519, 2,172 generated in the current study and 347 used as representative global genomes) were downloaded from the Global Initiative on Sharing of All Influenza Data (GISAID) database and processed using the <b>Nextstrain</b> bioinformatics pipeline Augur using IQTREE v1.6.<li> The resulting tree consisting of 1,153 subsampled genomes was visualized in the <b>Nextstrain</b> web application Auspice (root-to-tip divergence plot in <xref rid="fig2" ref-type="fig"><u>Figure&#xA0;2</u></xref>B) and in Geneious v11.1.5 (circular phylogenetic tree in <xref rid="fig2" ref-type="fig"><u>Figure&#xA0;2</u></xref>C) (<xref rid="bib33" ref-type="bibr"><u>Kearse et&#xA0;al., 2012</u></xref>).<li> [Bedford et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <part-title>Updated Nextstrain SARS CoV-2 clade naming strategy</part-title>          <year>2021</year>          <ext-link ext-link-type="uri" xlink:href="http://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming" id="intref0090">http://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>        ) [Rambaut et&#xA0;al., 2020a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Bedford and Neher, 2020](          <person-group person-group-type="author">            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <part-title>A Getting Started Guide to the Genomic Epidemiology of SARS-CoV-2. Nextstrain.org</part-title>          <year>2020</year>          <ext-link ext-link-type="uri" xlink:href="https://nextstrain.github.io/ncov/" id="intref0085">https://nextstrain.github.io/ncov/</ext-link>        ) [Bushnell, 2021](          <person-group person-group-type="author">            <name>              <surname>Bushnell</surname>              <given-names>B.</given-names>            </name>          </person-group>          <part-title>BBMap short read aligner, and other bioinformatic tools</part-title>          <year>2021</year>          <ext-link ext-link-type="uri" xlink:href="https://sourceforge.net/projects/bbmap/" id="intref0095">https://sourceforge.net/projects/bbmap/</ext-link>        ) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Kearse et&#xA0;al., 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22543367">22543367</a>) [Bushnell, 2021](          <person-group person-group-type="author">            <name>              <surname>Bushnell</surname>              <given-names>B.</given-names>            </name>          </person-group>          <part-title>BBMap short read aligner, and other bioinformatic tools</part-title>          <year>2021</year>          <ext-link ext-link-type="uri" xlink:href="https://sourceforge.net/projects/bbmap/" id="intref0095">https://sourceforge.net/projects/bbmap/</ext-link>        ) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Kearse et&#xA0;al., 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22543367">22543367</a>) [Shu and McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [Kearse et&#xA0;al., 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22543367">22543367</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33857576">33857576</a> <b>(1)</b> Monitoring the rise of the SARS-CoV-2 lineage B.1.1.7 in Tenerife (Spain) since mid-December 2020<ul><li> The tree also includes relationships with other 1902 genome sequences representative of major clades, labelled based on <b>Nextstrain</b> definitions, from Auspice v2.23.0 (as of 2021-03-07) and sampled between December 2019 and March 2021 for reference (light gray) (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33857576">33857576</a> <b>(2)</b> Monitoring the rise of the SARS-CoV-2 lineage B.1.1.7 in Tenerife (Spain) since mid-December 2020<ul><li> Complete, high-quality SARS-CoV-2 (n&#x2005;=&#x2005;19 922) genomes from the global COVID-19 pandemic with collection date information, which had been sequenced from samples obtained from infected persons on or before 23 March 2020, were downloaded from the Global Initiative on Sharing All Influenza Data (GISAID) database (10 August 12020 build) [<xref rid="CIT0015" ref-type="bibr"><u>15</u></xref>, <xref rid="CIT0016" ref-type="bibr"><u>16</u></xref>], expanded to include SARS-CoV-2 genomes, and processed using the <b>Nextstrain</b> bioinformatics pipeline Augur [<xref rid="CIT0017" ref-type="bibr"><u>17</u></xref>].<li> The resulting tree was visualized in the <b>Nextstrain</b> Web application Auspice [<xref rid="CIT0017" ref-type="bibr"><u>17</u></xref>] and using Geneious v11.1.5 software [<xref rid="CIT0020" ref-type="bibr"><u>20</u></xref>].<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22543367">22543367</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33871357">33871357</a> <b>(6)</b> Repeated introductions and intensive community transmission fueled a mumps virus outbreak in Washington State<ul><li> Sequences and metadata were cleaned and organized using fauna, a database system that is part of the <b>Nextstrain</b> platform.<li> Sequences were processed using <b>Nextstrain</b>&#x2019;s augur software (<xref rid="bib36" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), and filtered to include only those with at least 8000 bases and were sampled in North America in 2006 or later.<li> Using the JSON for the North American full genome mumps tree output from the <b>Nextstrain</b> pipeline (shown in <xref rid="fig2s2" ref-type="fig"><u>Figure 2&#x2014;figure supplement 2</u></xref>;&#xA0;<xref rid="bib36" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), we traversed the tree from root to tip.<li> For each subsampled data&#xA0;set, we then combined these subsampled data&#xA0;sets with the background North American sequences, and reran the <b>Nextstrain</b> pipeline.<li> This clock rate was set based on the inferred substitutions per site per year from all North American mumps genomes on <ext-link xlink:href="http://<b>nextstrain</b>.org/mumps/na" ext-link-type="uri"><b>nextstrain</b>.org/mumps/na</ext-link>.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33907756">33907756</a> <b>(2)</b> SARS-CoV-2 transmission dynamics in Belarus revealed by genomic and incidence data analysis.<ul><li>For the phylogeny reconstruction, we utilized the SARS-CoV-2-specific phylogenetic inference pipeline implemented in <b>Nextstrain</b> [<xref rid="R29" ref-type="bibr"><u>29</u></xref>].<li> To obtain a representative sample with those background sequences, a country-specific <b>Nextstrain</b> context subsampling was used [<xref rid="R29" ref-type="bibr"><u>29</u></xref>].<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33875719">33875719</a> <b>(3)</b> Global dynamics of SARS-CoV-2 clades and their relation to COVID-19 epidemiology<ul><li> Genome analysis tools were also provided by several platforms such as The China National Center for Bioinformation (<ext-link ext-link-type="uri" xlink:href="https://bigd.big.ac.cn/ncov/tool/annotation">https://bigd.big.ac.cn/ncov/tool/annotation</ext-link>)<sup><xref ref-type="bibr" rid="CR5"><u>5</u></xref></sup> <b>Nextstrain</b> project (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>)<sup><xref ref-type="bibr" rid="CR6"><u>6</u></xref></sup>, and CoV-GLUE (<ext-link ext-link-type="uri" xlink:href="http://cov-glue.cvr.gla.ac.uk">http://cov-glue.cvr.gla.ac.uk</ext-link>)<sup><xref ref-type="bibr" rid="CR7"><u>7</u></xref></sup>.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32102777">32102777</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](          <person-group person-group-type="author">            <name>              <surname>Singer</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Gifford</surname>              <given-names>R</given-names>            </name>            <name>              <surname>Cotten</surname>              <given-names>M</given-names>            </name>            <name>              <surname>Robertson</surname>              <given-names>D</given-names>            </name>          </person-group>          <article-title>CoV-GLUE: A web application for tracking SARS-CoV-2 genomic variation</article-title>          <source>Preprints</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.20944/preprints202006.0225.v1</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33864026">33864026</a> <b>(4)</b> On the origin and evolution of SARS-CoV-2<ul><li> Indeed, the global effort to understand the dynamics of SARS-CoV-2 continues to generate such data (currently, there are more than 600,000 related datasets and counting, based on the Global Initiative on Sharing All Influenza Data (GISAID) repertoire and the <b>Nextstrain</b> database).<li> Clades were defined using <b>Nextstrain</b> nomenclature based on global frequency, variation from parent clade, and year of emergence.<li> For <bold>a</bold>, <bold>b</bold> clades were named according to <b>Nextstrain</b> nomenclature, which distinguishes clades based on global frequency, year of emergence and a unique letter.<li> For <bold>a</bold>&#x2013;<bold>e</bold> data visualization was performed by <b>nextstrain</b>.org with data provided by GISAID.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33898853">33898853</a> <b>(2)</b> Considering the use of the terms strain and adaptation in prion research<ul><li> Thus, strains contain information that allows a phylogenetic reconstruction of their evolutionary history, which allows a classification to be constructed that reflects that evolutionary history, such as that for SARS-CoV-2 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0010">https://<b>nextstrain</b>.org/ncov/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33880458">33880458</a> <b>(2)</b> Epidemiology, clinical characteristics, and virologic features of COVID-19 patients in Kazakhstan: A nation-wide retrospective cohort study<ul><li> The global SARS-CoV2 phylogeny and associated metadata were downloaded from <b>Nextstrain</b><xref rid="bib0010" ref-type="bibr"><u>[10]</u></xref> and the tree was reconstructed using the <b>Nextstrain</b>/ncov pipeline.<li> [[10]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(9)</b> Epidemiology, clinical characteristics, and virologic features of COVID-19 patients in Kazakhstan: A nation-wide retrospective cohort study<ul><li> Each sample&#x2019;s consensus sequence was visually inspected using Geneious Prime software (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.geneious.com" ext-link-type="uri">https://www.geneious.com</ext-link>) and/or <b>Nextstrain</b>&#x2019;s Nextclade online tool (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li>Wisconsin-centric time-resolved and divergence phylogenetic trees (seen in <xref rid="sup1" ref-type="supplementary-material"><u>Supplementary File 1</u></xref>) were built using the standard <b>Nextstrain</b> tools and scripts [<xref rid="CIT0026" ref-type="bibr"><u>26</u></xref>].<li> An interactive view of this <b>Nextstrain</b> phylogenetic tree can be found at <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/community/gagekmoreno/Wisconsin-SARS-CoV-2/ncov/wisconsin/2021-1-8" ext-link-type="uri">https://<b>nextstrain</b>.org/community/gagekmoreno/Wisconsin-SARS-CoV-2/ncov/wisconsin/2021-1-8</ext-link>.<li> <italic toggle="yes">B,</italic> Time-resolved phylogenetic tree built using <b>Nextstrain</b> tools with all Wisconsin sequences available as of 15 January 2021.<li> <italic toggle="yes">B,</italic> Time-resolved phylogenetic tree built using <b>Nextstrain</b> tools with all Wisconsin sequences available as of 15 January 2021.<li> <italic toggle="yes">B,</italic> A time-resolved phylogenetic tree built using <b>Nextstrain</b> tools with all Wisconsin sequences available as of 15 January 2021.<li>A time-resolved phylogenetic tree built using <b>Nextstrain</b> tools for all samples collected and shared in Wisconsin from March to December 2020.<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33858931">33858931</a> <b>(2)</b> Genome Sequences of Five SARS-CoV-2 Variants from Mumbai, India, Obtained by Nanopore Sequencing<ul><li> Phylogenetic analysis was performed using the Augur v6.3.0 pipeline and visualized using Auspice v2.0, both developed by <b>Nextstrain</b>.<li> Based on the <b>Nextstrain</b> nomenclature, four out of five variants (India/MH-CMP-19/2020, India/MH-CMP-30/2020, India/MH-CMP-34/2020, and India/MH-CMP-35/2020) belong to clade 20B, whereas one variant (India/MH-CMP-31/2020) belongs to clade 20A.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33858942">33858942</a> <b>(2)</b> Killed whole-genome reduced-bacteria surface-expressed coronavirus fusion peptide vaccines protect against disease in a porcine model<ul><li> For all coronaviruses across four genera, the FP core [IEDLLF] is identical (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" xlink:show="new">https://<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33853970">33853970</a> <b>(0)</b> Genomics and epidemiology of the P.1 SARS-CoV-2 lineage in Manaus, Brazil<ul><li> Viral lineages were classified by using the Pangolin (<xref rid="R26" ref-type="bibr"><italic>26</italic></xref>) software tool (<ext-link ext-link-type="uri" xlink:href="http://pangolin.cog-uk.io">http://pangolin.cog-uk.io</ext-link>), nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>), and standard phylogenetic analysis using complete reference genomes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33924636">33924636</a> <b>(3)</b> Deepening of In Silico Evaluation of SARS-CoV-2 Detection RT-qPCR Assays in the Context of New Variants<ul><li> The UK variant, called Variant of Concern (VOC) 202012/01, belongs to Global Initiative on Sharing Avian Influenza Data (GISAID) clade GR, Pangolin lineage B.1.1.7, and <b>Nextstrain</b> clade 20I/501Y.V1 and is defined by 23 mutations: 13 non-synonymous mutations, 4 deletions, and 6 synonymous mutations [<xref rid="B30-genes-12-00565" ref-type="bibr"><u>30</u></xref>,<xref rid="B31-genes-12-00565" ref-type="bibr"><u>31</u></xref>,<xref rid="B32-genes-12-00565" ref-type="bibr"><u>32</u></xref>].<li> The SA variant belongs to GISAID clade GH, Pangolin lineage B.1.351, and <b>Nextstrain</b> clade 20H/501Y.V2 and contains up to 22 non-synonymous mutations [<xref rid="B33-genes-12-00565" ref-type="bibr"><u>33</u></xref>].<li> Finally, the Brazil variant belongs to GISAID clade GR, Pangolin lineage P.1, and <b>Nextstrain</b> clade 20J/501Y.V3 and carries 20 non-synonymous and 6 synonymous mutations [<xref rid="B34-genes-12-00565" ref-type="bibr"><u>34</u></xref>,<xref rid="B35-genes-12-00565" ref-type="bibr"><u>35</u></xref>].<li> [30](          <person-group person-group-type="author">            <collab>Public Health England</collab>          </person-group>          <article-title>Investigation of novel SARS-CoV-2 variant: Variant of Concern 202012/01. Technical Briefing 1</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959438/Technical_Briefing_VOC_SH_NJL2_SH2.pdf">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959438/Technical_Briefing_VOC_SH_NJL2_SH2.pdf</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-01-21">(accessed on 21 January 2021)</date-in-citation>        ) [31](          <person-group person-group-type="author">            <collab>Public Health England</collab>          </person-group>          <article-title>Investigation of novel SARS-CoV-2 variant: Variant of Concern 202012/01. Technical Briefing 2</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959361/Technical_Briefing_VOC202012-2_Briefing_2.pdf">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959361/Technical_Briefing_VOC202012-2_Briefing_2.pdf</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-01-21">(accessed on 21 January 2021)</date-in-citation>        ) [32](          <person-group person-group-type="author">            <name>              <surname>Davies</surname>              <given-names>N.G.</given-names>            </name>            <name>              <surname>Abbott</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Barnard</surname>              <given-names>R.C.</given-names>            </name>            <name>              <surname>Jarvis</surname>              <given-names>C.I.</given-names>            </name>            <name>              <surname>Kucharski</surname>              <given-names>A.J.</given-names>            </name>            <name>              <surname>Munday</surname>              <given-names>J.D.</given-names>            </name>            <name>              <surname>Pearson</surname>              <given-names>C.A.B.</given-names>            </name>            <name>              <surname>Russell</surname>              <given-names>T.W.</given-names>            </name>            <name>              <surname>Tully</surname>              <given-names>D.C.</given-names>            </name>            <name>              <surname>Washburne</surname>              <given-names>A.D.</given-names>            </name>            <etal/>          </person-group>          <article-title>Estimated Transmissibility and Impact of SARS-CoV-2 Lineage B.1.1.7 in England</article-title>          <source>Science</source>          <year>2021</year>          <fpage>372</fpage>          <pub-id pub-id-type="doi">10.1126/science.abg3055</pub-id>        ) [33](          <person-group person-group-type="author">            <name>              <surname>Tegally</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Wilkinson</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Giovanetti</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Iranzadeh</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Fonseca</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Giandhari</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Doolabh</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Pillay</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>San</surname>              <given-names>E.J.</given-names>            </name>            <name>              <surname>Wibmer</surname>              <given-names>K.</given-names>            </name>            <etal/>          </person-group>          <article-title>Emergence and Rapid Spread of a New Severe Acute Respiratory Syndrome-Related Coronavirus 2 (SARS-CoV-2) Lineage with Multiple Spike Mutations in South Africa</article-title>          <source>medRxiv</source>          <year>2020</year>          <fpage>10</fpage>        ) [34](          <person-group person-group-type="author">            <name>              <surname>Naveca</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>Nascimento</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Souza</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Corado</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Nascimento</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>Silva</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Costa</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Duarte</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Pessoa</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Gon&#xE7;alves</surname>              <given-names>L.</given-names>            </name>            <etal/>          </person-group>          <article-title>Phylogenetic Relationship of SARS-CoV-2 Sequences from Amazonas with Emerging Brazilian Variants Harboring Mutations E484K and N501Y in the Spike Protein</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/phylogenetic-relationship-of-sars-cov-2-sequences-from-amazonas-with-emerging-brazilian-variants-harboring-mutations-e484k-and-n501y-in-the-spike-protein/585">https://virological.org/t/phylogenetic-relationship-of-sars-cov-2-sequences-from-amazonas-with-emerging-brazilian-variants-harboring-mutations-e484k-and-n501y-in-the-spike-protein/585</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-01-15">(accessed on 15 January 2021)</date-in-citation>        ) [35](          <person-group person-group-type="author">            <name>              <surname>Naveca</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>da Costa</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Nascimento</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Souza</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Andr&#xE9;</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Nascimento</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>Costa</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Duarte</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Silva</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Mej&#xED;a</surname>              <given-names>M.</given-names>            </name>            <etal/>          </person-group>          <article-title>SARS-CoV-2 Reinfection by the New Variant of Concern (VOC) P.1 in Amazonas, Brazil</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596">https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-01-27">(accessed on 27 January 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=          <person-group person-group-type="author">            <name>              <surname>Naveca</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>da Costa</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Nascimento</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Souza</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Andr&#xE9;</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Nascimento</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>Costa</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Duarte</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Silva</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Mej&#xED;a</surname>              <given-names>M.</given-names>            </name>            <etal/>          </person-group>          <article-title>SARS-CoV-2 Reinfection by the New Variant of Concern (VOC) P.1 in Amazonas, Brazil</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596">https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-01-27">(accessed on 27 January 2021)</date-in-citation>        ">          <person-group person-group-type="author">            <name>              <surname>Naveca</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>da Costa</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Nascimento</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Souza</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Andr&#xE9;</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Nascimento</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>Costa</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Duarte</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Silva</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Mej&#xED;a</surname>              <given-names>M.</given-names>            </name>            <etal/>          </person-group>          <article-title>SARS-CoV-2 Reinfection by the New Variant of Concern (VOC) P.1 in Amazonas, Brazil</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596">https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-01-27">(accessed on 27 January 2021)</date-in-citation>        </a> <b>(1)</b> Deepening of In Silico Evaluation of SARS-CoV-2 Detection RT-qPCR Assays in the Context of New Variants<ul><li> The regular search indicates that 19 resources relate to COVID-19, such as PROSITE, SWISS-MODEL Repository, ViralZone, <b>Nextstrain</b> (<xref rid="B26" ref-type="bibr"><u>26</u></xref>), GlyConnect (<xref rid="B27" ref-type="bibr"><u>27</u></xref>), SIB COVID-19 Integrated Knowledgebase (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://covid-19-sparql.expasy.org/" ext-link-type="uri">https://covid-19-sparql.expasy.org/</ext-link>), to name a few.<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30574787">30574787</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33921216">33921216</a> <b>(4)</b> SARS-CoV-2 Reinfection in a Healthcare Worker Despite the Presence of Detectable Neutralizing Antibodies<ul><li> Pangolin lineage and <b>nextstrain</b> clade assignment of consensus sequences were performed using the pangolin (<uri xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</uri>, accessed on 7 March 2021) and nextclade (<uri xlink:href="https://github.com/<b>nextstrain</b>/nextclade">https://github.com/<b>nextstrain</b>/nextclade</uri>, accessed on 7 March 2021) packages.<li> Phylogenetic analysis and tree visualization were performed using <b>nextstrain</b> [<xref rid="B18-viruses-13-00661" ref-type="bibr"><u>18</u></xref>].<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33920487">33920487</a> <b>(3)</b> Genetic Diversity of SARS-CoV-2 over a One-Year Period of the COVID-19 Pandemic: A Global Perspective<ul><li> Multiple sequence alignments of this subset were subsequently imported to reconstruct a maximum-likelihood phylogenetic tree using <b>NextStrain</b> tools (<uri xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</uri>, the accessed date: 16 January 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33907515">33907515</a> <b>(1)</b> Developing effective siRNAs to reduce the expression of key viral genes of COVID-19<ul><li> Based on the times and locations of the viral strains that appeared in the past 12 months, we prepared a phylogenetic tree and a pangolin lineage of COVID-19 from the GISAID platform using <b>Nextstrain</b> open source project <xref rid="B25" ref-type="bibr"><u>25</u></xref>,<xref rid="B26" ref-type="bibr"><u>26</u></xref>, which showed that this virus has quickly mutated and spread across the five continents (Figure <xref ref-type="fig" rid="F1"><u>1</u></xref>A and B).<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33739917">33739917</a> <b>(1)</b> Introduction of SARS-CoV-2 variant of concern 20h/501Y.V2 (B.1.351) from Malawi to Italy<ul><li>The SARS-CoV-2 variant of concern (VOC) 202012/02 (also known as B.1.351 in PANGOLIN phylogeny or 20H/501Y.V2 in <b>NextStrain</b> phylogeny) is currently causing a major outbreak of COVID-19 in South Africa.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33840632">33840632</a> <b>(1)</b> Association of SARS-CoV-2 clades with clinical, inflammatory and virologic outcomes: An observational study<ul><li> <b>Nextstrain</b> nomenclature offers a similar high level annotation, while SARS-CoV-2 lineages (cov-lineages.org) provide a more granular view with hundreds of named lineages incorporating epidemiological and sequence information corresponding to outbreaks in different regions <xref rid="bib0010" ref-type="bibr"><u>[10]</u></xref>.<li> [[10]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34258310">34258310</a> <b>(0)</b> Prolonged SARS-CoV-2 Illness in a Patient Receiving Ocrelizumab for Multiple Sclerosis<ul><li> The isolate had 19 missense mutations, which is similar to the number of mutations observed from other members of the same clade, 20G (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org;" ext-link-type="uri">https://clades.<b>nextstrain</b>.org;</ext-link>version 0.14.0) (<xref rid="sup1" ref-type="supplementary-material"><u>Supplementary Table 2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33851162">33851162</a> <b>(2)</b> A new SARS-CoV-2 lineage that shares mutations with known Variants of Concern is rejected by automated sequence repository quality control<ul><li> Hereafter we refer to this as lineage B.1.x as it is nested within the prevalent lineage B.1 in our phylogeny (<b>Nextstrain</b> 20C).<li> Instead, we find the nucleotide substitution pattern is largely consistent with other lineages within Pango clade B.1 (<b>Nextstrain</b> 20C).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33836204">33836204</a> <b>(4)</b> SARS-CoV-2 presented moderately during two episodes of the infection with lack of antibody responses<ul><li> Phylogenetic analysis was performed using <b>NextStrain</b> online tools (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intr0010">https://<b>nextstrain</b>.org/</ext-link>).<li> The entropy of each nucleotide and amino acids position based on <b>nextstrain</b>.org is also added.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33822878">33822878</a> <b>(1)</b> Choice of assemblers has a critical impact on de novo assembly of SARS-CoV-2 genome and characterizing variants<ul><li> Around 50&#xA0;000 SARS-CoV-2 genomic sequences have been submitted to NCBI Nucleotide records and <b>Nextstrain</b> database since the first whole-genome was sequenced in January 2020 [<xref rid="ref3" ref-type="bibr"><u>3</u></xref>, <xref rid="ref4" ref-type="bibr"><u>4</u></xref>].<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32015508">32015508</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33891875">33891875</a> <b>(14)</b> Early introductions and transmission of SARS-CoV-2 variant B.1.1.7 in the United States<ul><li> For comparison, an interactive phylogenetic tree, inferred using IQ-Tree and TreeTime only, can be accessed from our custom <b>Nextstrain</b> build: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5" id="intref0315">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5</ext-link>.<li> Additional examples can be found by exploring our full dataset (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5" id="intref0030">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5</ext-link>).<li><hr/></td></tr><tr><td>R</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/" id="intref0045">https://cran.r-project.org/</ext-link></td></tr><tr><td>IQ-Tree 1.6.12</td><td><ext-link ext-link-type="uri" xlink:href="http://www.iqtree.org/" id="intref0050">http://www.iqtree.org/</ext-link></td><td><xref rid="bib30" ref-type="bibr"><u>Minh et&#xA0;al., 2020</u></xref></td></tr><tr><td>TreeTime 0.8.0</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/treetime" id="intref0055">https://github.com/neherlab/treetime</ext-link></td><td><xref rid="bib38" ref-type="bibr"><u>Sagulenko et&#xA0;al., 2018</u></xref></td></tr><tr><td>TempEst</td><td><ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/tempest/" id="intref0060">http://tree.bio.ed.ac.uk/software/tempest/</ext-link></td><td><xref rid="bib35" ref-type="bibr"><u>Rambaut et&#xA0;al., 2016</u></xref></td></tr><tr><td>TreeAnnotator</td><td><ext-link ext-link-type="uri" xlink:href="https://beast.community/treeannotator" id="intref0065">https://beast.community/treeannotator</ext-link></td><td><xref rid="bib36" ref-type="bibr"><u>Rambaut et&#xA0;al., 2018</u></xref></td></tr><tr><td>BEAST v1.10</td><td><ext-link ext-link-type="uri" xlink:href="http://beast.community" id="intref0070">http://beast.community</ext-link></td><td><xref rid="bib40" ref-type="bibr"><u>Suchard et&#xA0;al., 2018</u></xref></td></tr><tr><td>BWA</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/lh3/bwa" id="intref0075">https://github.com/lh3/bwa</ext-link></td><td><xref rid="bib25" ref-type="bibr"><u>Li and Durbin, 2010</u></xref></td></tr><tr><td>MAFFT</td><td><ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/" id="intref0080">https://mafft.cbrc.jp/alignment/software/</ext-link></td><td><xref rid="bib19" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref></td></tr><tr><td>iVar 1.2.1</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/andersen-lab/ivar" id="intref0085">https://github.com/andersen-lab/ivar</ext-link></td><td><xref rid="bib12" ref-type="bibr"><u>Grubaugh et&#xA0;al., 2019</u></xref></td></tr><tr><td>Samtools</td><td><ext-link ext-link-type="uri" xlink:href="http://samtools.sourceforge.net/" id="intref0090">http://samtools.sourceforge.net/</ext-link></td><td><xref rid="bib26" ref-type="bibr"><u>Li et&#xA0;al., 2009</u></xref></td></tr><tr><td>TrimGalore</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/FelixKrueger/TrimGalore" id="intref0095">https://github.com/FelixKrueger/TrimGalore</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/FelixKrueger/TrimGalore" id="intref0100">https://github.com/FelixKrueger/TrimGalore</ext-link></td></tr><tr><td>RAMPART</td><td>ARTIC Network</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/artic-network/rampart" id="intref0105">https://github.com/artic-network/rampart</ext-link></td></tr><tr><td>ARTIC Network Bioinformatic protocol</td><td>ARTIC Network</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html" id="intref0110">https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html</ext-link></td></tr><tr><td><b>Nextstrain</b>.<li><hr/></td></tr><tr><td>R</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/" id="intref0045">https://cran.r-project.org/</ext-link></td></tr><tr><td>IQ-Tree 1.6.12</td><td><ext-link ext-link-type="uri" xlink:href="http://www.iqtree.org/" id="intref0050">http://www.iqtree.org/</ext-link></td><td><xref rid="bib30" ref-type="bibr"><u>Minh et&#xA0;al., 2020</u></xref></td></tr><tr><td>TreeTime 0.8.0</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/treetime" id="intref0055">https://github.com/neherlab/treetime</ext-link></td><td><xref rid="bib38" ref-type="bibr"><u>Sagulenko et&#xA0;al., 2018</u></xref></td></tr><tr><td>TempEst</td><td><ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/tempest/" id="intref0060">http://tree.bio.ed.ac.uk/software/tempest/</ext-link></td><td><xref rid="bib35" ref-type="bibr"><u>Rambaut et&#xA0;al., 2016</u></xref></td></tr><tr><td>TreeAnnotator</td><td><ext-link ext-link-type="uri" xlink:href="https://beast.community/treeannotator" id="intref0065">https://beast.community/treeannotator</ext-link></td><td><xref rid="bib36" ref-type="bibr"><u>Rambaut et&#xA0;al., 2018</u></xref></td></tr><tr><td>BEAST v1.10</td><td><ext-link ext-link-type="uri" xlink:href="http://beast.community" id="intref0070">http://beast.community</ext-link></td><td><xref rid="bib40" ref-type="bibr"><u>Suchard et&#xA0;al., 2018</u></xref></td></tr><tr><td>BWA</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/lh3/bwa" id="intref0075">https://github.com/lh3/bwa</ext-link></td><td><xref rid="bib25" ref-type="bibr"><u>Li and Durbin, 2010</u></xref></td></tr><tr><td>MAFFT</td><td><ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/" id="intref0080">https://mafft.cbrc.jp/alignment/software/</ext-link></td><td><xref rid="bib19" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref></td></tr><tr><td>iVar 1.2.1</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/andersen-lab/ivar" id="intref0085">https://github.com/andersen-lab/ivar</ext-link></td><td><xref rid="bib12" ref-type="bibr"><u>Grubaugh et&#xA0;al., 2019</u></xref></td></tr><tr><td>Samtools</td><td><ext-link ext-link-type="uri" xlink:href="http://samtools.sourceforge.net/" id="intref0090">http://samtools.sourceforge.net/</ext-link></td><td><xref rid="bib26" ref-type="bibr"><u>Li et&#xA0;al., 2009</u></xref></td></tr><tr><td>TrimGalore</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/FelixKrueger/TrimGalore" id="intref0095">https://github.com/FelixKrueger/TrimGalore</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/FelixKrueger/TrimGalore" id="intref0100">https://github.com/FelixKrueger/TrimGalore</ext-link></td></tr><tr><td>RAMPART</td><td>ARTIC Network</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/artic-network/rampart" id="intref0105">https://github.com/artic-network/rampart</ext-link></td></tr><tr><td>ARTIC Network Bioinformatic protocol</td><td>ARTIC Network</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html" id="intref0110">https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html</ext-link></td></tr><tr><td><b>Nextstrain</b></td><td><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0115">https://<b>nextstrain</b>.org/</ext-link>.<li> We visualized this time-scaled maximum-likelihood phylogeny using auspice (<xref rid="bib14" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>), which can be found on our custom <b>nextstrain</b> page: <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5" id="intref0235">http://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5</ext-link>.<li> For comparison, an interactive phylogenetic tree, inferred using IQ-Tree and TreeTime only, can be accessed from our custom <b>Nextstrain</b> build: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5" id="intref0320">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5</ext-link>.<li> [Minh et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [Sagulenko et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Rambaut et&#xA0;al., 2016](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27774300">27774300</a>) [Rambaut et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29718447">29718447</a>) [Suchard et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29942656">29942656</a>) [Li and Durbin, 2010](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20080505">20080505</a>) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Grubaugh et&#xA0;al., 2019](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30621750">30621750</a>) [Li et&#xA0;al., 2009](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19505943">19505943</a>) [Minh et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [Sagulenko et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Rambaut et&#xA0;al., 2016](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27774300">27774300</a>) [Rambaut et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29718447">29718447</a>) [Suchard et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29942656">29942656</a>) [Li and Durbin, 2010](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20080505">20080505</a>) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Grubaugh et&#xA0;al., 2019](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30621750">30621750</a>) [Li et&#xA0;al., 2009](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19505943">19505943</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33813118">33813118</a> <b>(3)</b> Recommendations for accurate genotyping of SARS-CoV-2 using amplicon-based sequencing of clinical samples<ul><li> (B) Relationship between the entropy estimated for all clonal variants in clinical samples (y-axis) and the entropy of the same variants in samples collected in the same country and during the same period according to <b>Nextstrain</b> [<xref rid="bib30" ref-type="bibr"><u>30</u></xref>] (x-axis).<li> Clades (according to <b>Nextstrain</b>) are indicated.<li> The tree visualization was generated using the <b>Nextstrain</b> platform [<xref rid="bib30" ref-type="bibr"><u>30</u></xref>].<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33793462">33793462</a> <b>(6)</b> Rapid Spread of SARS-CoV-2 in a State Prison After Introduction by Newly Transferred Incarcerated Persons &#x2014; Wisconsin, August 14&#x2013;October 22, 2020<ul><li> Phylogenetic inference and visualization performed using <b>Nextstrain</b> Augur v9.0.0 (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/augur" ext-link-type="uri">https://github.com/<b>nextstrain</b>/augur</ext-link>) and <b>Nextstrain</b> Auspice v2.18.4 (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/auspice" ext-link-type="uri">https://github.com/<b>nextstrain</b>/auspice</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33857895">33857895</a> <b>(8)</b> Rapid screening for SARS-CoV-2 variants of concern in clinical and environmental samples using nested RT-PCR assays targeting key mutations of the spike protein<ul><li> The new variant belongs to clade GR, lineage B.1.1.7 (GISAID), or clade 20I/501Y.V1 (<b>Nextstrain</b>) and was discovered following an unforeseen rise in COVID-19 cases in South East England, between October and December 2020 (<xref rid="bib0045" ref-type="bibr"><u>WHO,&#xA0;2020d</u></xref>).<li>On 18 December 2020, national authorities in South Africa announced the detection of a new variant of SARS-CoV-2, belonging to clade GH, lineage B.1.351 (GISAID) or 20H/501Y.V2 (<b>Nextstrain</b>).<li> This was done using GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="interref0005">https://www.gisaid.org/</ext-link>) and Nextclade (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="interref0006">https://<b>nextstrain</b>.org/</ext-link>).<li> Both files were then analyzed using the Nextclade web tool (<b>nextstrain</b>.org) in order to determine whether our nested assay (PCR ID 980) would yield similar results to those given by a full sequence analysis.<li> The only significant exception was the 20I/501Y.V1 variant, which was incorrectly assigned to clade 20B This misclassification, however, was due to a limitation of the Nextclade web tool, which, for the purposes of phylogenetic placement and clade assignment, uses only mutations and no gaps (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade/issues/311#issuecomment-765904855" id="interref0007">https://github.com/<b>nextstrain</b>/nextclade/issues/311#issuecomment-765904855</ext-link>), and was therefore unable to take into consideration the deletions HV69&#x2013;70del and Y144del for a correct assignment of our partial sequences of the S gene.<li>Nextclade web tool phylogenetic trees for representative strains of the different variants using the full genome or the Spike partial gene fragment amplified with PCR ID 980 - <b>nextstrain</b>.org with ad hoc modifications.<li> [WHO,&#xA0;2020d](          <person-group person-group-type="author">            <collab>WHO</collab>          </person-group>          <part-title>SARS-CoV-2 Variant</part-title>          <year>2020</year>          <publisher-name>United Kingdom of Great Britain and Northern Ireland</publisher-name>          <comment>&#x2013;</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33793189">33793189</a> <b>(1)</b> Nano-Enabled COVID-19 Vaccines: Meeting the Challenges of Durable AntibodyPlus Cellular Immunity and Immune Escape<ul><li> Figure is adapted from <b>Nextstrain</b>, undera C-BY-4.0 license.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33838744">33838744</a> <b>(6)</b> Establishment of a reverse genetics system for SARS-CoV-2 using circular polymerase extension reaction<ul><li> For instance, viruses encoding the D614G mutation on the surface of the spike protein (S protein) became predominant (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0015">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> In addition, human cases infected with the lineage 501Y.V1 and 501Y.V2 viruses have been increasing, in which multiple mutations have been introduced besides D614G (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0020">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> Substitution of D614 to G on the S protein was frequently detected and became dominant in European, South American, and North American countries (<xref rid="bib6" ref-type="bibr"><u>Koyama et&#xA0;al., 2020</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0025">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> [Koyama et&#xA0;al., 2020](          <person-group person-group-type="author">            <name>              <surname>Koyama</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Weeraratne</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Snowdon</surname>              <given-names>J.L.</given-names>            </name>            <name>              <surname>Parida</surname>              <given-names>L.</given-names>            </name>          </person-group>          <article-title>Emergence of Drift Variants That May Affect COVID-19 Vaccine Development and Antibody Treatment</article-title>          <source>Pathogens</source>          <volume>9</volume>          <year>2020</year>          <fpage>324</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33567246">33567246</a> <b>(3)</b> Imported SARS-CoV-2 Variant P.1 in Traveler Returning from Brazil to Italy<ul><li> The P.1 variant also is known as B.1.1.28 in the Phylogenetic Assignment of Named Global Outbreak Lineages (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>) and as 20J/501Y.V3 in <b>NextStrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33797392">33797392</a> <b>(1)</b> Linked transmission chains of imported SARS-CoV-2 variant B.1.351 across mainland France, January 2021<ul><li>On 13 and 14 January 2021, two cases of confirmed severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection with the B.1.351 variant [<xref rid="r1" ref-type="bibr"><u>1</u></xref>] (or 20H/501Y.V2 in the <b>Nextstrain</b> classification [<xref rid="r2" ref-type="bibr"><u>2</u></xref>]) were reported by the French National Reference Laboratory (NRL) for respiratory diseases to the respective regional health agencies (ARS, regional Ministry of Health&#x2019;s bureaus) for Ile-de-France and for Pays de la Loire and to Sant&#xE9; publique France (the French public health agency).<li> [1](O&#x2019;Toole A, Scher E, Underwood A, Jackson B, Hill V, McCrone JT, et al. Pangolin: lineage assignment in an emerging pandemic as an epidemiological tool. github. [Accessed: 30 Mar2021]. Available from: <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/global_report_B.1.351.html">https://cov-lineages.org/global_report_B.1.351.html</ext-link>) [2](Bedford T, Hodcroft EB, Neher RA. Updated Nextstrain SARS-CoV-2 clade naming strategy. Nextstrain. [Accessed: 29 Mar 2021]. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming">https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34540148">34540148</a> <b>(1)</b> Bioinformatic analysis of the whole genome sequences of SARS-CoV-2 from Indonesia<ul><li> Meanwhile, complete viral genome sequence has been submitted and published to NCBI, <b>Nextstrain</b>, and GISAID databases.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33795344">33795344</a> <b>(0)</b> Genome Sequence of a SARS-CoV-2 Strain from a COVID-19 Clinical Sample from the Khagrachari District of Bangladesh<ul><li> Phylogenetic analysis using Nextclade<sup>beta</sup> version 0.13.0 (<ext-link ext-link-type="uri" xlink:href="http://clades.<b>nextstrain</b>.org">clades.<b>nextstrain</b>.org</ext-link>) assigned the new genome to the SARS-CoV-2 clade 20A (<xref ref-type="fig" rid="fig1"><u>Fig.&#xA0;1</u></xref>) (<xref rid="B6" ref-type="bibr"><u>6</u></xref>).<li> The tree was constructed on 19 February 2021 using the Nextclade<sup>beta</sup> version 0.13.0 (<ext-link ext-link-type="uri" xlink:href="http://clades.<b>nextstrain</b>.org">clades.<b>nextstrain</b>.org</ext-link>), in which the red circle represents the position of hCoV-19/Bangladesh/CU-CTG-24/2020 (GISAID accession no.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33789012">33789012</a> <b>(2)</b> Ebola Virus Transmission Initiated by Systemic Ebola Virus DiseaseRelapse<ul><li> Datataken from <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/inrb-drc/ebola-nord-kivu">https://<b>nextstrain</b>.org/community/inrb-drc/ebola-nord-kivu</ext-link>and released on NCBI GenBank database.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33795722">33795722</a> <b>(1)</b> Analysis of SARS-CoV-2 genomic epidemiology reveals disease transmission coupled to variant emergence and allelic variation<ul><li> Publicly available raw WGS were retrieved from GISAID (supplementary Table <xref rid="MOESM1" ref-type="media"><u>1</u></xref>) and <b>Nextstrain</b> (supplementary Table <xref rid="MOESM1" ref-type="media"><u>2</u></xref>) with appropriate metadata.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33916205">33916205</a> <b>(6)</b> Nosocomial Outbreak of SARS-CoV-2 in a &#x201C;Non-COVID-19&#x201D; Hospital Ward: Virus Genome Sequencing as a Key Tool to Understand Cryptic Transmission<ul><li> Phylogenetic analysis was performed using the SARS-CoV-2 <b>Nextstrain</b> pipeline [<xref rid="B20-viruses-13-00604" ref-type="bibr"><u>20</u></xref>] version from March 23, 2020 (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</uri>, accessed on 23 March 2020), as previously described [<xref rid="B18-viruses-13-00604" ref-type="bibr"><u>18</u></xref>].<li> Clade and lineage assignments were performed using Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>) and Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin) (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>, accessed on 16 December 2020) [<xref rid="B22-viruses-13-00604" ref-type="bibr"><u>22</u></xref>], respectively.<li> The outbreak-associated SARS-CoV-2 variant belongs to COG-UK lineage B.1.1 and <b>Nextstrain</b> clade 20B, carrying the Spike amino acid changes D614G and L176F (<xref ref-type="app" rid="app1-viruses-13-00604"><u>Supplementary Table S2</u></xref>).<li> These challenges were even more pronounced given: (i) the SARS-CoV-2 relatively low substitution rate (around one mutation every 2 weeks) (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>, accessed on 16 December 2020), which renders identical or nearly-identical sequences in several individuals enrolled in the transmission chain; (ii) that most cases were only detected upon massive screening, meaning that the viral genetic profile detected at the time of testing may not reflect the one at the time of transmission/infection; and (iii) that most cases were asymptomatic at time testing, hampering any reconstruction based on the chronology of incubation period and symptoms onset.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33131453">33131453</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33798758">33798758</a> <b>(4)</b> Molecular epidemiology analysis of early variants of SARS-CoV-2 reveals the potential impact of mutations P504L and Y541C (NSP13) in the clinical COVID-19 outcomes<ul><li> Two phylogenetic treemaps based on ORF1ab: p.5828P&#xA0;&gt;&#xA0;L and ORF1ab: p.5865Y&#xA0;&gt;&#xA0;C were download from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="ir0020">https://<b>nextstrain</b>.org/ncov</ext-link>) (<xref rid="bb0070" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> (A) Phylogenetic tree of strains downloaded from <b>Nextstrain</b>.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33836314">33836314</a> <b>(1)</b> Implementation of an in-house real-time reverse transcription-PCR assay for the rapid detection of the SARS-CoV-2 Marseille-4 variant<ul><li> It predominates in Marseille since August 2020, has been reported to spread in Europe since early summer and was classified as the <b>Nextstrain</b> 20A.EU2 lineage [<xref rid="bib0005" ref-type="bibr"><u>1</u></xref>,<xref rid="bib0010" ref-type="bibr"><u>2</u></xref>].<li> [1](          <person-group person-group-type="author">            <name>              <surname>Fournier</surname>              <given-names>P.E.</given-names>            </name>            <name>              <surname>Colson</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Levasseur</surname>              <given-names>A.</given-names>            </name>            <etal/>          </person-group>          <part-title>Emergence and Outcome of the SARS-CoV-2 &#x201C;Marseille-4&#x201D; Variant</part-title>          <year>2021</year>          <publisher-name>IHU Pre-prints</publisher-name>          <pub-id pub-id-type="doi">10.35081/xcrm-6t77</pub-id>        ) [2](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Zuber</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Nadeau</surname>              <given-names>S.</given-names>            </name>            <etal/>          </person-group>          <part-title>Emergence and Spread of a SARS-CoV-2 Variant Through Europe in the Summer of 2020</part-title>          <comment>medRxiv</comment>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.10.25.20219063</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33821249">33821249</a> <b>(5)</b> Simultaneous detection and mutation surveillance of SARS-CoV-2 and multiple respiratory viruses by rapid field-deployable sequencing<ul><li>(F) The SNVs detected in multiplex RPA sequencing and their position as shown in the <b>Nextstrain</b> data portal (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0055">https://<b>nextstrain</b>.org</ext-link>).<li> Variant calling was performed using samtools (v1.9) and bcftools (v1.9).<xref rid="bib32" ref-type="bibr"><sup>32</sup></xref> The detected variants were filtered by position (within the targeted regions) and compared with the data in <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0040"><b>Nextstrain</b>.org</ext-link> as of Jun 2, 2020.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33861950">33861950</a> <b>(2)</b> Emergence and rapid transmission of SARS-CoV-2 B.1.1.7 in the United States<ul><li><hr/></td></tr><tr><td>DRAGEN COVIDSeq Test Pipeline v.1.3.0.28</td><td>Illumina</td><td>N/A</td></tr><tr><td>Pangolin v2.0</td><td><xref rid="bib24" ref-type="bibr"><u>O&#x2019;Toole et&#xA0;al., 2021b</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0025">https://github.com/cov-lineages/pangolin</ext-link></td></tr><tr><td>NextClade v0.12.0</td><td><xref rid="bib18" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="intref0030">https://github.com/<b>nextstrain</b>/nextclade</ext-link>.<li> [O&#x2019;Toole et&#xA0;al., 2021b](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.T.</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>E.</given-names>            </name>          </person-group>          <part-title>Phylogenetic Assignment of Named Global Outbreak LINeages</part-title>          <year>2021</year>          <publisher-name>Pangolin</publisher-name>        ) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33794345">33794345</a> <b>(4)</b> Severe acute respiratory syndrome coronavirus 2 escape mutants and protective immunity from natural infections or immunizations<ul><li> Surveillance of mutations has been implemented by different databases and bioinformatics platforms, for instance GISAID [<xref rid="bib3" ref-type="bibr"><u>3</u></xref>], the EU COVID19 open data [<xref rid="bib4" ref-type="bibr"><u>4</u></xref>], the NCBI SARS-CoV-2 resource [<xref rid="bib5" ref-type="bibr"><u>5</u></xref>], <b>Nextstrain</b> [<xref rid="bib6" ref-type="bibr"><u>6</u></xref>], and the China Centre for Bioinformation [<xref rid="bib7" ref-type="bibr"><u>7</u></xref>].<li> There are also now three nomenclature systems, developed by <b>Nextstrain</b> [<xref rid="bib6" ref-type="bibr"><u>6</u></xref>], the University of Edinburgh PANGO [<xref rid="bib8" ref-type="bibr"><u>8</u></xref>] and GISAID [<xref rid="bib3" ref-type="bibr"><u>3</u></xref>], respectively, all widely used in the field.<li> Updates of detections of spike gene mutations for each GISAID clade are posted daily on the <italic>CoVariants</italic> website [<xref rid="bib3" ref-type="bibr"><u>3</u></xref>,<xref rid="bib9" ref-type="bibr"><u>9</u></xref>], together with geographic maps of their proportional distribution [<xref rid="bib10" ref-type="bibr"><u>10</u></xref>], and visualized through collaboration with <sc><b>Nextstrain</b></sc>.<li> The current version of the <sc><b>Nextstrain</b></sc> data visualization software allows switching between GISAID and PANGO lineage designations, and the WHO is working on a unifying system for nomenclature.<li> [3](          <person-group person-group-type="author">            <collab>GSAID</collab>          </person-group>          <article-title>hCoV-19 spike glycoprotein mutation surveillance dashboard</article-title>          <comment>Available at:</comment>          <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/hcov19-mutation-dashboard/" id="intref0010">https://www.gisaid.org/hcov19-mutation-dashboard/</ext-link>        ) [4](          <article-title>European COVID-19 data platform</article-title>          <comment>Available at:</comment>          <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/hcov19-variants/" id="intref0015">https://www.gisaid.org/hcov19-variants/</ext-link>        ) [5](          <article-title>NCBI SARS-CoV-2 resource</article-title>          <comment>Available at:</comment>          <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/sars-cov-2/" id="intref0020">https://www.ncbi.nlm.nih.gov/sars-cov-2/</ext-link>        ) [6](          <article-title>Nextstrain</article-title>          <comment>Available at:</comment>          <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/" id="intref0025">https://nextstrain.org/</ext-link>        ) [7](          <article-title>China centre for Bioinformation</article-title>          <comment>Available at:</comment>          <ext-link ext-link-type="uri" xlink:href="https://bigd.big.ac.cn/ncov/?lang=en" id="intref0030">https://bigd.big.ac.cn/ncov/?lang=en</ext-link>        ) [6](          <article-title>Nextstrain</article-title>          <comment>Available at:</comment>          <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/" id="intref0025">https://nextstrain.org/</ext-link>        ) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [3](          <person-group person-group-type="author">            <collab>GSAID</collab>          </person-group>          <article-title>hCoV-19 spike glycoprotein mutation surveillance dashboard</article-title>          <comment>Available at:</comment>          <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/hcov19-mutation-dashboard/" id="intref0010">https://www.gisaid.org/hcov19-mutation-dashboard/</ext-link>        ) [3](          <person-group person-group-type="author">            <collab>GSAID</collab>          </person-group>          <article-title>hCoV-19 spike glycoprotein mutation surveillance dashboard</article-title>          <comment>Available at:</comment>          <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/hcov19-mutation-dashboard/" id="intref0010">https://www.gisaid.org/hcov19-mutation-dashboard/</ext-link>        ) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33536258">33536258</a>) [10](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>          </person-group>          <article-title>CoVariants</article-title>          <comment>Available at:</comment>          <ext-link ext-link-type="uri" xlink:href="https://covariants.org/" id="intref0035">https://covariants.org/</ext-link>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33779548">33779548</a> <b>(3)</b> Saliva TwoStep for rapid detection of asymptomatic SARS-CoV-2 carriers<ul><li>Regions of the SARS-CoV-2 genome that are conserved among strains were identified using genome diversity data from <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global"><b>nextstrain</b>.org/ncov/global</ext-link>).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36890186">36890186</a> <b>(1)</b> A high-resolution melt curve toolkit to identify lineage-defining SARS-CoV-2 mutations<ul><li> Automated rapid variant calling was carried out using EPI2ME Desktop Agent v3.3.0 with the ARTIC&#x2009;+&#x2009;<b>NextStrain</b> analysis pipeline, and via the vcf file generated via ARTIC.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36995936">36995936</a> <b>(8)</b> Rapid escape of new SARS-CoV-2 Omicron variants from BA.2-directed antibody responses<ul><li> BA.1 caused the first Omicron wave, closely followed by a combination of BA.1 sub-lineages BA.1.1 and BA.2 in early 2021 (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/lineage_list" id="intref0045">https://cov-lineages.org/lineage_list</ext-link>, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/nextclade/sars-cov-2" id="intref0050">https://<b>nextstrain</b>.org/nextclade/sars-cov-2</ext-link>).<li> BA.2 variants (including derivatives of BA.5)&#x2014;variants with long phylogenetic branch lengths, multiple antigenic mutations, and a lack of genetic intermediates, for example BA.2.75, BJ.1, BS.1, BA.2.10.4, and BA.2.3.20,<xref rid="bib19" ref-type="bibr"><sup>19</sup></xref> and (2) antigenic drift, seen both in BA.5<xref rid="bib24" ref-type="bibr"><sup>24</sup></xref> and within these second-generation BA.2 lineages, notably BQ.1 and BA.2.75 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/nextclade/sars-cov-2/21L" id="intref0060">https://<b>nextstrain</b>.org/nextclade/sars-cov-2/21L</ext-link>) (<xref rid="fig1" ref-type="fig"><u>Figures&#xA0;1</u></xref>A&#x2013;1C).<li> The rooted phylogenetic tree<xref rid="bib26" ref-type="bibr"><sup>26</sup></xref> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/nextclade/sars-cov-2" id="intref0075">https://<b>nextstrain</b>.org/nextclade/sars-cov-2</ext-link>) shown in <xref rid="fig1" ref-type="fig"><u>Figure&#xA0;1</u></xref>C provides a helpful grouping of the lineages.<li>The phylogenetic tree was generated by pruning the Nextclade reference tree (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/nextclade/sars-cov-2/21L" id="intref0215">https://<b>nextstrain</b>.org/nextclade/sars-cov-2/21L</ext-link>) which contains one sequence per Pango lineage.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36960050">36960050</a> <b>(3)</b> Immune damage mechanisms of COVID-19 and novel strategies in prevention and control of epidemic<ul><li> A phylogenetic tree showing 3,064 genomes sampled between December 2019 and January 2023 is available on <b>Nextstrain</b> website (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time</ext-link>), which can help us better understand changing patterns of different lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36898405">36898405</a> <b>(2)</b> Efficacy of antivirals and bivalent mRNA vaccines against SARS-CoV-2 isolate CH.1.1<ul><li>SARS-CoV-2 subvariants BQ.1 (a BA.5 subvariant) and XBB (a BA.2 subvariant) are currently the most widespread variant globally, according to <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time" id="interrefs10"><b>Nextstrain</b></ext-link>; however, other variants are being closely monitored as they emerge in specific regions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36992191">36992191</a> <b>(1)</b> Molecular Characterization and Selection of Indigenous SARS-CoV-2 Delta Variant for the Development of the First Inactivated SARS-CoV-2 Vaccine of Pakistan<ul><li> The second wave was observed from October to December 2020, and all reported sequences nested in a separate clade (<b>Nextstrain</b> clade classification).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36921604">36921604</a> <b>(2)</b> Toward a global virus genomic surveillance network<ul><li> There were also other nomenclature systems developed in 2020, such as the <b>Nextstrain</b> clade system, which aimed to describe the phylogeny in a broader and more stable way, by having criteria for minimum size and persistence; as compared to pango lineages which are finer scale to capture outbreaks and ongoing transmission in close to real-time.<li> This can be mitigated by the fast turnaround of sequences into readable reports or dashboards like <b>Nextstrain</b>,<xref rid="bib57" ref-type="bibr"><sup>57</sup></xref> allowing local sample collectors to act on their data where necessary, and to see where their region fits into the wider picture.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36963096">36963096</a> <b>(4)</b> Genomic epidemiology of SARS-CoV-2 during the first four waves in Mozambique<ul><li> Additionally, global, and African representative datasets were included in these analyses and accessed through the <b>NextStrain</b> platform (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).<li> Additional South African references from other provinces were permitted as part of the African or global builds of <b>NextStrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36992400">36992400</a> <b>(14)</b> Systematic Guidelines for Effective Utilization of COVID-19 Databases in Genomic, Epidemiologic, and Clinical Research<ul><li> Raw sequence data on SARS-CoV-2 and its variants have been submitted to GISAID, <b>Nextstrain</b>, the University of California Santa Cruz (UCSC) Genome Browser for COVID-19 Research on the UCSC page, and the COVID-19 Data Portal powered by EMBL-European Bioinformatics Institute (EBI).<li> The WHO, Pangolin, and Nextclade, powered by <b>Nextstrain</b>, are considered to be the main classifying entities [<xref rid="B26-viruses-15-00692" ref-type="bibr"><u>26</u></xref>,<xref rid="B27-viruses-15-00692" ref-type="bibr"><u>27</u></xref>,<xref rid="B28-viruses-15-00692" ref-type="bibr"><u>28</u></xref>].<li> <b>Nextstrain</b> visualizes its genomic data as phylogenetic trees that display clade and variant information.<li> We determined that GISAID, the WHO, the NCBI SARS-CoV-2 Resources, interaction data between coronavirus RNAs and host proteins (CovInter), T-cell COVID-19 Atlas (T-CoV), Immune escape variants in SARS-CoV-2 (ESC), DockCoV2, USCS Genome Browser, COVID-19 Data Portal, cov-lineages.org, Protein Data Bank (PDB), COVID CG, and <b>Nextstrain</b> included this category of data.<li>(3) To obtain data on VOCs, we checked <b>Nextstrain</b>&#x2019;s latest global analysis table.<li> To check Omicron&#x2019;s sublineage data and date information, <b>Nextstrain</b> was used.<li> Upon clicking on <b>Nextstrain</b>&#x2019;s latest global analysis&#x2014;GISAID or latest global analysis&#x2014;open data, the SARS-CoV-2 genomic epidemiology was checked.<li>(2) To obtain the latest data on the Omicron lineage, we searched for BA.5 information through <b>Nextstrain</b>, specifically for BA.5 date information;.<li>(4) Through <b>Nextstrain</b>, we checked not just BA.5 but also BA.2, so we used cov-lineages.org to obtain and analyze data on BA.5 and BA.2 variants;.<li> From information on the currently circulating VOC Omicron, <b>Nextstrain</b> was used to check BA.5 and BA.2.<li> Upon clicking on <b>Nextstrain</b>&#x2019;s latest global analysis&#x2014;GISAID tab or latest global analysis&#x2014;open data tab, genomic epidemiology data of BA.5 and BA.2 can be checked;.<li>(2) Utilizing <b>Nextstrain</b>, we verified the earliest date and most recently observed date for the currently dominant strains BA.1, BA.2, BA.3, BA.4, and BA.5;.<li> To check the sublineages and date information on the currently circulating VOC Omicron, <b>Nextstrain</b> was used.<li> Upon clicking on <b>Nextstrain</b>&#x2019;s latest global analysis&#x2014;GISAID tab or latest global analysis&#x2014;open data tab, BA.5 and BA.2 data can be accessed;.<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33735881">33735881</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34149046">34149046</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36862099">36862099</a> <b>(0)</b> Sequencing directly from antigen-detection rapid diagnostic tests in Belgium, 2022: a gamechanger in genomic surveillance?<ul><li> Sequences were analysed with Nexclade v 2.0.0 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37012880">37012880</a> <b>(2)</b> Update on the effectiveness of COVID-19 vaccines on different variants of SARS-CoV-2<ul><li> There are Global Initiative on Sharing All Influenza Data (GISAID), <b>Nextstrain</b>, and Pango genetic lineages nomenclature.<li>B.1.17 variant (20I/501Y.V1 in <b>Nextstrain</b> nomenclature system) also known as Alpha variant, is a previously circulating VOC.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37038489">37038489</a> <b>(1)</b> Phylogenetic Changes in SARS-CoV-2 Virus in Bosnian-Herzegovinian Population Over the Period of Two Years<ul><li><b>Nextstrain</b> software was used to confirm the lineages and clades of the SARS-CoV-2 virus that were present in Bosnia and Herzegovina at the time of the research.</ul></p>
<p><u>No PMID</u> <b>(1)</b> Navigating Pandemic Crises: Encountering the DigitalCommons<ul><li> For example, in February 2020, researchers at theopen science project <b>Nextstrain</b> analyzed genomic data shared on the open datasharing platform GISAID.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37123451">37123451</a> <b>(3)</b> A multiplex method for detection of SARS-CoV-2 variants based on MALDI-TOF mass spectrometry<ul><li> According to the interpretation criteria formulated from GISAID (<ext-link ext-link-type="uri" xlink:href="https://gisaid.org/resources/statements-clarifications/" id="ir025">https://gisaid.org/resources/statements-clarifications/</ext-link>) and <b>Nextstrain</b> database (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="ir030">https://<b>nextstrain</b>.org/</ext-link>) for the SARS-CoV-2 MS method (<xref rid="t0015" ref-type="table"><u>Table 3</u></xref>), the viral variants of Delta, Omicron BA.1, Omicron BA.2, and Omicron BA.5 were distinguished.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36703245">36703245</a> <b>(1)</b> Intrahost Monkeypox Virus Genome Variation in Patient with Early Infection, Finland, 2022<ul><li> The curated dataset of MPXV reference genomes was downloaded from <b>Nextstrain</b> and aligned by using Nextalign (<xref rid="R5" ref-type="bibr"><italic>5</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36863537">36863537</a> <b>(1)</b> Genetic and structural genome-based survey reveals the low potential for epidemiological expansion of the SARS-CoV-2 XBB.1.5 sublineage<ul><li>The phylogenomic reconstruction, available at https://gisaid.org/phylodynamics/global/<b>nextstrain</b>/, depicts an evolutionary pattern of XBB.1.5 very similar to that of previous, recent variants such as BA.2.75,<xref rid="bib2" ref-type="bibr"><sup>2</sup></xref> BA.2.12.1 and BQ.1<xref rid="bib3" ref-type="bibr"><sup>3</sup></xref> that appeared as an evolutionary blind background.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36857133">36857133</a> <b>(4)</b> Antitarget,Anti-SARS-CoV-2 Leads, Drugs, andthe Drug Discovery&#x2013;Genetics Alliance Perspective<ul><li> Adapted from <uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/groups/neherlab/ncov/global?c=clade_membership&amp;p=grid&amp;r=division">https://<b>nextstrain</b>.org/groups/neherlab/ncov/global?c=clade_membership&amp;p=grid&amp;r=division</uri>.<li> Adapted from <uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/groups/neherlab/ncov/global?c=clade_membership&amp;p=grid&amp;r=division">https://<b>nextstrain</b>.org/groups/neherlab/ncov/global?c=clade_membership&amp;p=grid&amp;r=division</uri>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36853059">36853059</a> <b>(1)</b> Unique Genomic Epidemiology of COVID-19 in the White Mountain Apache Tribe, April to August 2020, Arizona<ul><li> Phylogenetic analyses were conducted in <b>Nextstrain</b> (<xref rid="B42" ref-type="bibr"><u>42</u></xref>), supplemented with genomes from GISAID (<xref rid="B41" ref-type="bibr"><u>41</u></xref>) selected by the genome sampler (<xref rid="B43" ref-type="bibr"><u>43</u></xref>) for context, and with BEAST 1.10.5 (<xref rid="B44" ref-type="bibr"><u>44</u></xref>) to estimate the timing of relevant events.<li> [42](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34934514">34934514</a>) [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33500774">33500774</a>) [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29942656">29942656</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36841870">36841870</a> <b>(1)</b> Lung tropism in hospitalized patients following infection with SARS-CoV-2 variants from D614G to Omicron BA.2<ul><li>The viral clade and lineage classifications were conducted using <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR23"><u>23</u></xref></sup> and Phylogenetic Assignment of Named Global Outbreak Lineages<sup><xref ref-type="bibr" rid="CR24"><u>24</u></xref></sup>.<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36992333">36992333</a> <b>(1)</b> Development of Next Generation Vaccines against SARS-CoV-2 and Variants of Concern<ul><li>Ever since the detection of the first SARS-CoV-2 viral variant, it became clear that a suitable naming scheme had to be implemented to designate and keep track of novel emerging variants despite the existing nomenclatures used by the GISAID and <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36746267">36746267</a> <b>(3)</b> Monkeypox virus (MPXV) genomics: A mutational and phylogenomic analyses of B.1 lineages<ul><li> This result is consistent with that reported by <b>Nextstrain</b> [January 24, 2022] (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/monkeypox/hmpxv1?dmin=2021-07-03&amp;label=clade:B.1" id="intref0015">https://<b>nextstrain</b>.org/monkeypox/hmpxv1?dmin=2021-07-03&amp;label=clade:B.1</ext-link> (accessed October 17, 2022)), which determined that the B.1 lineage emerged in the European continent at that time generating the first outbreaks of the virus, and subsequently spreading to other continents such as America, Oceania and Asia.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36963002">36963002</a> <b>(1)</b> Implementation of genomic surveillance of SARS-CoV-2 in the Caribbean: Lessons learned for sustainability in resource-limited settings<ul><li> CoV-GLUE (<ext-link xlink:href="http://cov-glue.cvr.gla.ac.uk/#/home" ext-link-type="uri">http://cov-glue.cvr.gla.ac.uk/#/home</ext-link>), UShER [<xref rid="pgph.0001455.ref014" ref-type="bibr"><u>14</u></xref>] and Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) were used for exploratory analysis prior to reporting results to the originating institutions, for example to inspect mutations, resolve ambiguities, and check for long branches.<li>Availability of open-source software (e.g., Pangolin, <b>Nextstrain</b>, MicroReact, UShER, CoV-GLUE) and open public access databases (GISAID, <ext-link xlink:href="http://outbreak.info" ext-link-type="uri">outbreak.info</ext-link>) simplified data sharing and analyses.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36902520">36902520</a> <b>(1)</b> Association between the Seroprevalence of Antibodies against Seasonal Alphacoronaviruses and SARS-CoV-2 Humoral Immune Response, COVID-19 Severity, and Influenza Vaccination<ul><li> According to the Global Initiative on Sharing All Influenza Data, during March and December 2020, SARS-CoV-2 infections in Poland were primarily caused by <b>Nextstrain</b> clades 20A, 20B, 20D, and 20G that did not reveal major differences in clinical outcomes [<xref rid="B41-jcm-12-01733" ref-type="bibr"><u>41</u></xref>,<xref rid="B42-jcm-12-01733" ref-type="bibr"><u>42</u></xref>,<xref rid="B43-jcm-12-01733" ref-type="bibr"><u>43</u></xref>].<li> [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35011858">35011858</a>) [42](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34571275">34571275</a>) [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33400131">33400131</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851795">36851795</a> <b>(3)</b> Search of Novel Small Molecule Inhibitors for the Main Protease of SARS-CoV-2<ul><li> We searched the data from GISAID (<uri xlink:href="https://gisaid.org/">https://gisaid.org/</uri> (accessed on 23 January 2023)) and <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri> (accessed on 23 January 2023)) with 2910 SARS-CoV-2 genomes samples in the past six months (updated on 23 January 2023) [<xref rid="B24-viruses-15-00580" ref-type="bibr"><u>24</u></xref>].<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851797">36851797</a> <b>(15)</b> Whole-Genome Analysis of Influenza A(H3N2) and B/Victoria Viruses Detected in Myanmar during the COVID-19 Pandemic in 2021<ul><li>To differentiate the genetic clades of circulated Myanmar viruses from those of viruses from other countries, genomic similarity and geographical topology were determined using an open-source database of the public health pathogen genome, <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</uri>, (accessed on 15 February 2023)).<li> <b>Nextstrain</b> provides continually updated publicly available data enabled by GISAD.<li> Similarly, a comparative phylogenetic tree with tanglegrams was automatically created by <b>Nextstrain</b> for the B/Victoria virus [<xref rid="B28-viruses-15-00583" ref-type="bibr"><u>28</u></xref>].<li> To confirm the topology of HA and NA indicated by <b>Nextstrain</b> and to assess reassortment in other segments, we constructed Maximum Likelihood (ML) phylogenetic trees for all eight segments for A(H3N2) and B/Victoria.<li> Additional viruses were randomly selected and downloaded from the GISAID between January 2021 and October 2022 from different countries, if they showed high proximity to our strains according to <b>Nextstrain</b> phylogenies.<li>The phylogeny of the HA segment, including global influenza A(H3N2) strains generated by <b>Nextstrain</b>, demonstrated predominant circulation of clade 2 and its subclades from August 2020 and co-circulation of clades 1, 1a and 3C.2a1b.1a.<li> The <b>Nextstrain</b> analysis of the HA segment showed that the viruses from Myanmar were close to those from Bahrain, Australia, Qatar, Sri Lanka, Bangladesh and Nepal, which belong to Middle East and South Asia, circulated between June 2021 and January 2022 (<xref rid="viruses-15-00583-f002" ref-type="fig"><u>Figure 2</u></xref>b).<li>Next, we assessed the evolution of influenza A(H3N2) Myanmar viruses between the HA and NA segments created by <b>Nextstrain</b> [<xref rid="B27-viruses-15-00583" ref-type="bibr"><u>27</u></xref>].<li>To confirm the topology that was found using <b>Nextstrain</b>, we constructed ML phylogenetic trees for each of eight segments by MEGA6 (<xref rid="app1-viruses-15-00583" ref-type="app"><u>Figure S1</u></xref>).<li>To investigate the evolution of B/Victoria viruses, we compared the tanglegram of the HA and NA segments of the viruses from Myanmar and vaccine strains of the 2021 and 2022 seasons by <b>Nextstrain</b> [<xref rid="B28-viruses-15-00583" ref-type="bibr"><u>28</u></xref>].<li> For the current study, we used a ready-made dataset in <b>Nextstrain</b>; however, we further intend to analyze the changes in transmission routes of influenza viruses to Myanmar in the post-pandemic era by constructing our own dataset to run using the <b>Nextstrain</b> algorithm.<li> Third, to provide a more detailed and focused analysis, we did not generate our own genetic dataset for influenza viruses to run in the <b>Nextstrain</b> interface, and we did not use an individual software package, such as BEAST, for the phylogenetic analysis with an emphasis on time-scaled trees or for geo-temporal map construction with estimated transmission routes [<xref rid="B48-viruses-15-00583" ref-type="bibr"><u>48</u></xref>].<li> [28](          <person-group person-group-type="author"><name><surname>Lee</surname><given-names>J.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name><name><surname>Bedford</surname><given-names>T.</given-names></name></person-group>          <article-title>Real-Time Tracking of Influenza B/Vic Evolution</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/flu/seasonal/vic/ha/2y" ext-link-type="uri">https://nextstrain.org/flu/seasonal/vic/ha/2y</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2023-02-17">(accessed on 17 February 2023)</date-in-citation>        ) [27](          <person-group person-group-type="author"><name><surname>Lee</surname><given-names>J.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name><name><surname>Bedford</surname><given-names>T.</given-names></name></person-group>          <article-title>Real-Time Tracking of Influenza A/H3N2 Evolution</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/flu/seasonal/h3n2/ha/2y" ext-link-type="uri">https://nextstrain.org/flu/seasonal/h3n2/ha/2y</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2023-02-17">(accessed on 17 February 2023)</date-in-citation>        ) [28](          <person-group person-group-type="author"><name><surname>Lee</surname><given-names>J.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name><name><surname>Bedford</surname><given-names>T.</given-names></name></person-group>          <article-title>Real-Time Tracking of Influenza B/Vic Evolution</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/flu/seasonal/vic/ha/2y" ext-link-type="uri">https://nextstrain.org/flu/seasonal/vic/ha/2y</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2023-02-17">(accessed on 17 February 2023)</date-in-citation>        ) [48](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29942656">29942656</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851799">36851799</a> <b>(3)</b> Dynamics of Early Establishment of SARS-CoV-2 VOC Omicron Lineages in Minas Gerais, Brazil<ul><li> As background, these datasets included all BA.1 (<b>NextStrain</b> clade 21k) and BA.2 (<b>NextStrain</b> clade 21L) sequences from these variants <b>NextStrain</b> builds, respectively (captured on 18 August 2022, BA.1 = 4644, BA.2 = 4477).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36845037">36845037</a> <b>(4)</b> One-year monitoring SARS-CoV-2 RNA surface contamination in hospitals reveals no correlation with organic material and negative pressure as a limiting factor for contamination<ul><li> PANGO lineages were determined using Pangolin v3.1.5 [<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0020">https://github.com/cov-lineages/pangolin</ext-link>, <ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1038/s41564-020-0770-5" id="intref0025">https://doi.org/10.1038/s41564-020-0770-5</ext-link>] and <b>Nextstrain</b> clade assignment was performed using Nextclade CLI v1.10.3 [<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="intref0030">https://github.com/<b>nextstrain</b>/nextclade</ext-link>, <ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1093/bioinformatics/bty407" id="intref0035">https://doi.org/10.1093/bioinformatics/bty407</ext-link>].<li>We conducted a phylogenetic analysis using <b>NextStrain</b> (Hadfield et al., 2018) to compare the obtained genomes with others (from Uruguay) available in the database.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36815201">36815201</a> <b>(1)</b> Emergence, phylogeography, and adaptive evolution of mpox virus<ul><li> The emergence of clade IIb responsible for the current outbreaks have been estimated to have originated as early as 2021.25 (95% HPD:2020.47&#x2013;2021.12), implying an earlier ancestor for the 2022 MPXV than that estimates by <b>Nextstrain</b> (2022.01).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36801471">36801471</a> <b>(2)</b> An overview of the vaccine platforms to combat COVID-19 with a focus on the subunit vaccines<ul><li> from GISAID (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m" id="intref0030">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36800345">36800345</a> <b>(0)</b> Transcriptionally active nasopharyngeal commensals and opportunistic microbial dynamics define mild symptoms in the COVID 19 vaccination breakthroughs<ul><li> Clade assignment to all the genomes was done using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36795504">36795504</a> <b>(3)</b> Letter to the editor: Highly pathogenic influenza A(H5N1) viruses in farmed mink outbreak contain a disrupted second sialic acid binding site in neuraminidase, similar to human influenza A viruses<ul><li>Phylogenetic analysis using <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>, accessed: 1 Feb 2023 [<xref rid="r5" ref-type="bibr"><u>5</u></xref>]) indicated that the I396M substitution, seldom observed in other H5N1 viruses, was acquired in an avian host immediately before emergence of the H5N1 virus in mink.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36791146">36791146</a> <b>(2)</b> Host heterogeneity and epistasis explain punctuated evolution of SARS-CoV-2<ul><li><bold>(A)</bold> Full, time-dependent Hamming distance distribution (UK data, GenBank via <b>Nextstrain</b> [<xref rid="pcbi.1010896.ref002" ref-type="bibr"><u>2</u></xref>]).<li> We have used the open, GenBank-derived dataset of aligned SARS-CoV-2 sequences from <b>Nextstrain</b> [<xref rid="pcbi.1010896.ref002" ref-type="bibr"><u>2</u></xref>].<li> [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36786990">36786990</a> <b>(3)</b> Differences in Presentation of SARS-CoV-2 Omicron Strain Variant BA.1&#x2013;BA.5 Peptides by HLA Molecules<ul><li>According to the <b>Nextstrain</b> portal (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>) data, the dominant variant of SARS-CoV-2 in 2022 is the Omicron strain.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36788246">36788246</a> <b>(2)</b> Resistance of Omicron subvariants BA.2.75.2, BA.4.6, and BQ.1.1 to neutralizing antibodies<ul><li> Sequences of the variants after one passage on IGROV-1 cells identified BA.4.6 and BQ.1.1 (Pango lineage B.1.1.529.4.6 and B.1.1.529.5.3.1.1.1.1.1.1, respectively according to <b>Nextstrain</b>, GISAID accession ID: BA.4.6: EPI_ISL_15729633 and BQ.1.1: EPI_ISL_15731523), indicating that no adaptative mutations were generated during this short culture period.<li> Sequencing of the virus confirmed the presence of BA.2.75.2 (Pango lineage B.1.1.529.2.75.2, according to <b>Nextstrain</b>, GISAID accession ID: E EPI_ISL_15731524).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851749">36851749</a> <b>(3)</b> Characterization of Influenza A(H1N1)pdm09 Viruses Isolated in the 2018&#x2013;2019 and 2019&#x2013;2020 Influenza Seasons in Japan<ul><li> According to the sequences registered in GISAID, the detection rate has gradually increased (0.17%, 2.10%, and 21.98% in the 2017&#x2013;2018, 2018&#x2013;2019, and 2019&#x2013;2020 seasons, respectively), and according to the <b>Nextstrain</b> project (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 29 December 2020) [<xref rid="B30-viruses-15-00535" ref-type="bibr"><u>30</u></xref>], in 2021 and 2022, A(H1N1)pdm09 viruses possessing the N156K substitution (6B.1A.5a.2) co-circulated with viruses possessing the D187A and Q189E substitutions (6B.1A.5a.1).<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36824759">36824759</a> <b>(6)</b> Olivar: fully automated and variant aware primer design for multiplex tiled amplicon sequencing of pathogen genomes<ul><li>To demonstrate Olivar&#x2019;s ability to avoid high-risk regions in a target genome, we used the publicly available SARS-CoV-2 data on <b>Nextstrain</b> <sup><xref rid="R19" ref-type="bibr"><u>19</u></xref></sup> where Shannon&#x2019;s entropy of each base in the reference genome is provided (<xref rid="F2" ref-type="fig"><u>Figure 2a</u></xref>).<li> Calculation of <b>Nextstrain</b> entropy can be found in Methods.<li>Calculation of <b>Nextstrain</b> nucleotide entropy.<li><b>Nextstrain</b> <sup><xref rid="R19" ref-type="bibr"><u>19</u></xref></sup> calculates Shannon&#x2019;s entropy for each nucleotide position of the reference SARS-CoV-2 genome based on MSA of genomes available on GISAID <sup><xref rid="R9" ref-type="bibr"><u>9</u></xref></sup>.<li> We used the reference genome provided by <b>Nextstrain</b> (GenBank <ext-link xlink:href="MN908947.3" ext-link-type="DDBJ/EMBL/GenBank">MN908947.3</ext-link>) without a BLAST database of non-targeted sequences.<li> For well-studied pathogens, such as SARS-CoV-2 and human monkeypox virus, the MSA Olivar uses is updated in real-time from the public database GISAID <sup><xref rid="R9" ref-type="bibr"><u>9</u></xref></sup> and coordinates and frequencies of nucleotide change are available at <b>Nextstrain</b> <sup><xref rid="R19" ref-type="bibr"><u>19</u></xref></sup>.<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34934514">34934514</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34934514">34934514</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36844451">36844451</a> <b>(4)</b> Delta-Omicron recombinant escapes therapeutic antibody neutralization<ul><li><hr/></td></tr><tr><td>GraphPad Prism 8.0e</td><td>GraphPad Prism Software, Inc.</td><td>N/A</td></tr><tr><td>IQ-TREE v2.2.0</td><td>IQ-TREE</td><td>N/A</td></tr><tr><td>BEAST v1.10.5</td><td>BEAST</td><td>N/A</td></tr><tr><td><b>Nextstrain</b> CLI v6.0.0.<li><hr/></td></tr><tr><td>GraphPad Prism 8.0e</td><td>GraphPad Prism Software, Inc.</td><td>N/A</td></tr><tr><td>IQ-TREE v2.2.0</td><td>IQ-TREE</td><td>N/A</td></tr><tr><td>BEAST v1.10.5</td><td>BEAST</td><td>N/A</td></tr><tr><td><b>Nextstrain</b> CLI v6.0.0</td><td><b>Nextstrain</b>.<li> The web-based Nextclade v1.14<xref rid="bib40" ref-type="bibr"><sup>40</sup></xref> and Auspice v2.36 phylogenomic visualization features of <b>Nextstrain</b><xref rid="bib14" ref-type="bibr"><sup>14</sup></xref> were used to examine viral genome clade assignment and mutation calling.<li> Mutations were identified with Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0030">https://clades.<b>nextstrain</b>.org/</ext-link>) and lineage assignment was performed with Pangolin v3.1.19 (pangoLearn 2022-01-22)<xref rid="bib22" ref-type="bibr"><sup>22</sup></xref> for both consensus genomes, which identified the first part of the recombinant genome as AY.45, i.e., a sublineage of the Delta variant of concern (VOC), and the second part as Omicron BA.1.<li>Phylogenetic analyses were done using the <b>Nextstrain</b> CLI package WSL on Windows.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36781860">36781860</a> <b>(1)</b> Emergence and spread of two SARS-CoV-2 variants of interest in Nigeria<ul><li> Nextclade v1.3.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) was used to assign the sequences to globally circulating viral clades and to investigate mutations in the genome that could affect primer amplification during PCR.<li> Treetime<sup><xref ref-type="bibr" rid="CR40"><u>40</u></xref></sup> v0.92 was used for quantifying the molecular clock and identifying ancestral phylogeny in the augur pipeline of <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR41"><u>41</u></xref></sup> v3.0.3.<li> [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851719">36851719</a> <b>(3)</b> SARSMutOnto: An Ontology for SARS-CoV-2 Lineages and Mutations<ul><li> [<xref rid="B25-viruses-15-00505" ref-type="bibr"><u>25</u></xref>], and <b>Nextstrain</b>.<li> [<xref rid="B26-viruses-15-00505" ref-type="bibr"><u>26</u></xref>] and <b>Nextstrain</b>&#xA0; [<xref rid="B27-viruses-15-00505" ref-type="bibr"><u>27</u></xref>] nomenclatures provide an overview of clade trends; the Pango nomenclatures offer detailed information on lineages, allowing early prediction of local lineage expansion.<li> variant, while GISAID used &#x2018;GR/501Y.V1&#x2019;, and <b>Nextstrain</b> used the name &#x2018;20I/S:501Y.V1&#x2019;.<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [26](          <person-group person-group-type="author"><collab>GISAID</collab></person-group>          <article-title>Clade and Lineage Nomenclature Aids in Genomic Epidemiology Studies of Active hCoV-19 Viruses</article-title>          <year>2021</year>          <comment>Available online: <ext-link xlink:href="https://gisaid.org/resources/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://gisaid.org/resources/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-08-20">(accessed on 20 August 2022)</date-in-citation>        ) [27](          <person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Hadfield</surname><given-names>J.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name><name><surname>Bedfor</surname><given-names>T.</given-names></name></person-group>          <article-title>Year-Letter Genetic Clade Naming for SARS-CoV-2 on Nextstrain.org</article-title>          <year>2020</year>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming" ext-link-type="uri">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-25">(accessed on 25 July 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36831052">36831052</a> <b>(5)</b> Computational Analysis Predicts Correlations among Amino Acids in SARS-CoV-2 Proteomes<ul><li> These could imply that both B.1.177 (EU1) and B.1.160 (EU2) share the same ancestry as confirmed by the phylogenetic relationships of <b>Nextstrain</b> SARS-CoV-2 clades, reporting that B.1.177 (EU1) is an offspring of EU2.<li> Herein, the phylogenetic relationship of SARS-CoV-2 clades generated by <b>Nextstrain</b> [<xref rid="B98-biomedicines-11-00512" ref-type="bibr"><u>98</u></xref>] was used (available at <uri xlink:href="https://github.com/<b>nextstrain</b>/ncov-clades-schema">https://github.com/<b>nextstrain</b>/ncov-clades-schema</uri> accessed on 12 December 2022) [<xref rid="app1-biomedicines-11-00512" ref-type="app"><u>Figure S2</u></xref>].<li> Furthermore, the phylogenetic tree by <b>Nextstrain</b> shows that BA.4 and B.1.177 are highly unrelated among the SARS-CoV-2 variants [<xref rid="app1-biomedicines-11-00512" ref-type="app"><u>Figure S2</u></xref>].<li> [98](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36845975">36845975</a> <b>(5)</b> Off-season circulation and characterization of enterovirus D68 with respiratory and neurological presentation using whole-genome sequencing<ul><li> A time-scaled phylogenetic analysis using the augur pipeline (v7.0.2) and TimeTree v0.8.1 (<xref rid="ref39" ref-type="bibr"><u>Sagulenko et al., 2018</u></xref>) implemented in <b>Nextstrain</b> (<xref rid="ref10" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) was performed (<xref rid="SM2" ref-type="supplementary-material"><u>Supplementary Table S3</u></xref>).<li> Applying a time-scaled tree using the <b>Nextstrain</b> pipeline, we estimated the evolutionary rate for the entire EV-D68 genome to have 3.05&#x2009;&#xD7;&#x2009;10<sup>&#x2212;3</sup> substitutions per site per year.<li> A total of 872 sequences were run through the <b>Nextstrain</b> pipeline.<li> The <b>Nextstrain</b> pipeline automatically removed problematic sequences during tree refinement (<italic>n</italic>&#x2009;=&#x2009;12 references) and uploading to auspice (<italic>n</italic>&#x2009;=&#x2009;7 references), this included EVD68_GR_05_02.12.19 (<xref rid="SM2" ref-type="supplementary-material"><u>Supplementary Table S8</u></xref>).<li>Using the <b>Nextstrain</b> pipeline, we could perform more real-time tracking of EV-D68 evolution and spread overtime (<xref rid="fig1" ref-type="fig"><u>Figure 1</u></xref>).<li> [Sagulenko et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36834888">36834888</a> <b>(0)</b> Accurate Detection of SARS-CoV-2 by Next-Generation Sequencing in Low Viral Load Specimens<ul><li> automatic plugin (Thermo Fisher Scientific), as well as the Nextclade tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>), against the reference genome from Wuhan (GenBank accession number NC_045512.2), to achieve the complete viral genome sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36797096">36797096</a> <b>(2)</b> Persistent SARS-COV-2 infection in vaccinated individual with three doses of COVID-19 vaccine<ul><li> Lineage prediction and variant calling was performed using Pangolin v4.1.2 (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="ir020">https://github.com/cov-lineages/pangolin</ext-link>) and Nextclade v2.6.0 (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="ir025">https://github.com/<b>nextstrain</b>/nextclade</ext-link>) software.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36833680">36833680</a> <b>(2)</b> Inflammatory Biomarkers Differ among Hospitalized Veterans Infected with Alpha, Delta, and Omicron SARS-CoV-2 Variants<ul><li> The dates based on phylogenetic trees were retrieved from <b>Nextstrain</b> [<xref rid="B13-ijerph-20-02987" ref-type="bibr"><u>13</u></xref>].<li> The <b>Nextstrain</b> is an online SARS-CoV-2 database that maps the evolution and epidemiology of the virus [<xref rid="B14-ijerph-20-02987" ref-type="bibr"><u>14</u></xref>,<xref rid="B15-ijerph-20-02987" ref-type="bibr"><u>15</u></xref>].<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35427787">35427787</a>) [15](          <person-group person-group-type="author"><name><surname>Paudel</surname><given-names>S.</given-names></name><name><surname>Dahal</surname><given-names>A.</given-names></name><name><surname>Bhattarai</surname><given-names>H.K.</given-names></name></person-group>          <article-title>Temporal Analysis of SARS-CoV-2 Variants during the COVID-19 Pandemic in Nepal</article-title>          <source>COVID</source>          <year>2021</year>          <volume>1</volume>          <fpage>423</fpage>          <lpage>434</lpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36758824">36758824</a> <b>(1)</b> Monkeypox virus: A review<ul><li> According to the <b>Nextstrain</b> database, the human monkeypox viruses identified in Iran are part of the B.1 lineage, which originated in Europe and has since spread globally [<xref rid="bib60" ref-type="bibr"><u>60</u></xref>].<li> [60](          <person-group person-group-type="author">            <name>              <surname>Luna</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Ram&#xED;rez</surname>              <given-names>A.L.</given-names>            </name>            <name>              <surname>Mu&#xF1;oz</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Ballesteros</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Pati&#xF1;o</surname>              <given-names>L.H.</given-names>            </name>            <name>              <surname>Casta&#xF1;eda</surname>              <given-names>S.A.</given-names>            </name>            <name>              <surname>Bonilla-Aldana</surname>              <given-names>D.K.</given-names>            </name>            <name>              <surname>Paniz-Mondolfi</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Ram&#xED;rez</surname>              <given-names>J.D.</given-names>            </name>          </person-group>          <article-title>Phylogenomic analysis of the monkeypox virus (MPXV) 2022 outbreak: emergence of a novel viral lineage?</article-title>          <source>Trav. Med. Infect. Dis.</source>          <volume>49</volume>          <year>2022 Sep 1</year>          <object-id pub-id-type="publisher-id">102402</object-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36753510">36753510</a> <b>(1)</b> Genomic and phenotypic analyses suggest moderate fitness differences among Zika virus lineages<ul><li>We used the phylogeny and diversity data on <b>Nextstrain</b> [<xref rid="pntd.0011055.ref021" ref-type="bibr"><u>21</u></xref>] to select major clades that are defined by nonsynonymous mutations, where the amino acid site had a Shannon entropy of greater than 0.2 as of February 2020.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36825152">36825152</a> <b>(3)</b> Variant-specific deleterious mutations in the SARS-CoV-2 genome reveal immune responses and potentials for prophylactic vaccine development<ul><li>Sequence analysis, alignment, and clustering were performed using NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>), an advanced tool for SARS-CoV-2 sequence analysis (<xref rid="B4" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>).<li> (Analysed by <b>Nextstrain</b>: <ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>).<li> [Aksamentov et al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group> (<year>2021</year>). <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J. Open Source Softw.</source><volume>6</volume> (<issue>67</issue>), <fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36708091">36708091</a> <b>(1)</b> Transmission of monkeypox/mpox&#xA0;virus: A narrative review of environmental, viral, host, and population factors in relation to the 2022 international outbreak<ul><li>Sequencing by Isidro and colleagues also found that the circulating MPXV appears to have descended from the clade sampled in cases from Nigeria, Singapore, Israel, and the UK between 2017 and 2019.<xref rid="jmv28534-bib-0052" ref-type="bibr"><sup>52</sup></xref> <b>Nextstrain</b>, an open&#x2010;source project that allows for tracking of pathogen evolution in real time, performed molecular clock analysis of this virus, which showed extensive diversity in MPXV' descendant lineages (A.1, A.2, A.1.1, and B.1).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36833334">36833334</a> <b>(4)</b> Molecular Evolution of SARS-CoV-2 during the COVID-19 Pandemic<ul><li> Another nomenclature was presented by <b>Nextstrain</b> [<xref rid="B27-genes-14-00407" ref-type="bibr"><u>27</u></xref>], a project that phylogenetically classifies SARS-CoV-2 genomes available from the GISAID database [<xref rid="B28-genes-14-00407" ref-type="bibr"><u>28</u></xref>].<li> The nomenclature of <b>Nextstrain</b> is based on the different clades that were detected over time.<li> These clades often display a direct association with PANGO lineages (i.e., GISAID GR corresponds to <b>Nextstrain</b> 20B and PANGO B.1.1.*).<li> A correspondence between PANGO, <b>Nextstrain</b>, GISAID, and WHO nomenclatures can be found in [<xref rid="B26-genes-14-00407" ref-type="bibr"><u>26</u></xref>].<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36729117">36729117</a> <b>(3)</b> Vaccine effectiveness estimates from an early-season influenza A(H3N2) epidemic, including unique genetic diversity with reassortment, Canada, 2022/23<ul><li> for cross reference, current <b>Nextstrain</b> terminology is as follows for corresponding ECDC subgroups i (&#x2018;2b&#x2019;), ii (&#x2018;2a.3a.1&#x2019;), iii (&#x2018;2a.1&#x2019;) and iv (&#x2018;2a.1b&#x2019;) [<xref rid="r12" ref-type="bibr"><u>12</u></xref>,<xref rid="r14" ref-type="bibr"><u>14</u></xref>].<li> Genetically characterised influenza A(H3N2) variants contributing to influenza vaccine effectiveness analyses with associated substitutions and ECDC subgroup designations, followed by <b>Nextstrain</b> nomenclature, noted in curly brackets.<li> When that occurred is unknown, with analysis of internal gene segments ongoing, but evidence of the same reassortment this season is found among NA phylogenies available in <b>Nextstrain</b> [<xref rid="r14" ref-type="bibr"><u>14</u></xref>].<li> [12](European Centre for Disease Prevention and Control (ECDC). Influenza virus characterization, summary Europe, November 2022. Stockholm: ECDC; 2023. Available from: <ext-link xlink:href="https://www.ecdc.europa.eu/en/publications-data/influenza-virus-characterization-summary-europe-november-2022" ext-link-type="uri">https://www.ecdc.europa.eu/en/publications-data/influenza-virus-characterization-summary-europe-november-2022</ext-link>) [14](Lee J, Neher R, Bedford T. Real-time tracking of influenza A/H3N2 evolution. Nextstrain. [Accessed: 30 Jan 2023]. Available from: <ext-link xlink:href="https://nextstrain.org/flu/seasonal/h3n2/ha/2y" ext-link-type="uri">https://nextstrain.org/flu/seasonal/h3n2/ha/2y</ext-link>) [14](Lee J, Neher R, Bedford T. Real-time tracking of influenza A/H3N2 evolution. Nextstrain. [Accessed: 30 Jan 2023]. Available from: <ext-link xlink:href="https://nextstrain.org/flu/seasonal/h3n2/ha/2y" ext-link-type="uri">https://nextstrain.org/flu/seasonal/h3n2/ha/2y</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36732505">36732505</a> <b>(1)</b> High-depth sequencing characterization of viral dynamics across tissues in fatal COVID-19 reveals compartmentalized infection<ul><li> We constructed a phylogenetic tree using the sarscov2_<b>nextstrain</b>_aligned_input pipeline in viral-ngs.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36648111">36648111</a> <b>(1)</b> Metagenomic Sequencing of Monkeypox Virus, Northern Mexico<ul><li>We used <b>Nextstrain</b>&#x2019;s mpox bioinformatic processing workflow (<xref rid="R7" ref-type="bibr"><italic>7</italic></xref>) for our initial analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36692451">36692451</a> <b>(6)</b> Early Introduction and Community Transmission of SARS-CoV-2 Omicron Variant, New York, New York, USA<ul><li> we used consensus sequences of these viruses as input to Nextclade version 1.10.1 (<xref rid="R19" ref-type="bibr"><italic>19</italic></xref>) for quality control, mutation calling, and <b>Nextstrain</b> clade assignment.<li> We also constructed a phylogenetic tree of those Omicron viruses in NYC using <b>Nextstrain</b> SARS-CoV-2 workflow version 3.0.6 (<xref rid="R21" ref-type="bibr"><italic>21</italic></xref>) and visualized it as timescaled using Auspice version 2.33.0 (<ext-link xlink:href="https://auspice.us" ext-link-type="uri">https://auspice.us</ext-link>), which is part of the <b>Nextstrain</b> workflow.<li> We used <b>Nextstrain</b> SARS-CoV-2 workflow version 3.0.6 (<xref rid="R21" ref-type="bibr"><italic>21</italic></xref>) to construct this phylogenetic tree and Auspice to visualize it as divergence-scaled.<li> To investigate the genetic relationship between viruses from travelers and locals, we analyzed the 392 Omicron viruses in a timescaled phylogenetic tree, which we constructed using <b>Nextstrain</b> workflow and visualized as timescaled using Auspice (<xref rid="R21" ref-type="bibr"><italic>21</italic></xref>) (<xref rid="F7" ref-type="fig"><u>Figure 7</u></xref>).<li> We constructed this phylogenetic tree using <b>Nextstrain</b> workflow and visualized it as timescaled using Auspice (<xref rid="R21" ref-type="bibr"><italic>21</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36596565">36596565</a> <b>(2)</b> Sentinel Surveillance System Implementation and Evaluation for SARS-CoV-2 Genomic Data, Washington, USA, 2020&#x2013;2021<ul><li>To determine variability of genomic data, we constructed phylogenetic trees for 4 scenarios using the <b>Nextstrain</b> (<xref rid="R18" ref-type="bibr"><italic>18</italic></xref>) pipeline for SARS-CoV-2.<li> During this period, most (63%) sequences were classified as <b>Nextstrain</b> clade 20B (Pango lineage B.1.1), 23% were clade 19B (Pango lineage A), 9% were clade 20A (Pango lineage B.1) and 5% were clade 20C.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36723867">36723867</a> <b>(4)</b> Quantitative profiling of N-glycosylation of SARS-CoV-2 spike protein variants<ul><li> B) The evolution of the 7 selective emerging SARS-CoV-2 spike variants in this study with each variant represented with World Health Organization designation (Greek alphabet annotation), Pango lineage (nomenclature lineage designation with letter and numerical value), and <b>Nextstrain</b> clade (year-letter generic clade annotation), respectively.<li> 2021</xref>), and <b>Nextstrain</b> clade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) (<xref rid="ref26" ref-type="bibr">Hadfield et&#xA0;al.<li> The only discrepancy was that the Gamma variant was clustered with Kappa and Delta variants in the current study, whereas it was grouped with the Alpha Variant monitored by <b>Nextstrain</b> Clades and PANGOLIN (<xref rid="ref26" ref-type="bibr">Hadfield et&#xA0;al.<li> One possible reason is that the PANGO nomenclature and the <b>Nextstrain</b> analysis provide a genetic lineage to classify the viral variants based on genomic sequences that may differ from an evolutionary lineage or to the changes in glycosylation profiles that we analysed (<xref rid="ref94" ref-type="bibr">Tang et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36744069">36744069</a> <b>(1)</b> Sequence analysis of SARS-CoV-2 Delta variant isolated from Makassar, South Sulawesi, Indonesia<ul><li>The SARS-CoV-2 variant belonging to lineage B.1.617.2, described the Delta variant, is also termed 21A/S: 478&#xA0;K by <b>Nextstrain</b> and G/478&#xA0;K.V1 by the GISAID database.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36719912">36719912</a> <b>(1)</b> White-tailed deer (<italic toggle="yes">Odocoileus virginianus</italic>) may serve as a wildlife reservoir for nearly extinct SARS-CoV-2 variants of concern<ul><li> Phylogenetic analysis of both datasets was performed by using procedures implemented in <b>Nextstrain</b> (<xref rid="r36" ref-type="bibr"><u>36</u></xref>).<li> [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36032980">36032980</a> <b>(5)</b> Demographic and Viral-Genetic Analyses of COVID-19 Severity in Bahrain Identify Local Risk Factors and a Protective Effect of Polymerase Mutations<ul><li> We then filtered sequences based on quality summaries implemented in Nextclade using default parameters for SARS-CoV-2 (<xref rid="R2" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>, <ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).<li>We assigned samples to phylogenetic clades using <b>Nextstrain</b> clade definitions as implemented in the Nextclade software (<xref rid="R2" ref-type="bibr">Aksamentov et al.<li> 2021</xref>, <ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).<li> In order to retain these individuals in the data set, sampling date was imputed as the median date within the <b>Nextstrain</b>-assigned cluster, age was imputed as the median overall age, and gender was imputed by coding as 0/1 and setting missing individuals to the mean value.<li> Non-genetic variables were analyzed first in the absence of any viral genetic information, such as <b>Nextstrain</b> clade assignments and PCs.<li> as designated by <b>Nextstrain</b>, as well as the first seven principal components (PCs) of the genetic relatedness matrix.<li>The association between burden scores and COVID-19 was tested using logistic regression, including all demographic and health information, as well as the top seven principal components (PCs) and <b>Nextstrain</b> clade designations, as covariates.<li> [Aksamentov et al., 2021](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <name><surname>Roemer</surname><given-names>C</given-names></name>, <name><surname>Hodcroft</surname><given-names>EB</given-names></name>, &amp; <name><surname>Neher</surname><given-names>RA</given-names></name>, (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source>, <volume>6</volume>(<issue>67</issue>), <fpage>3773</fpage>, <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36908697">36908697</a> <b>(3)</b> COVID-19 and <italic toggle="yes">Evolution, Medicine, and Public Health</italic><ul><li> In the early months of the pandemic, for example, many of us spent hours on <b>NextStrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>) examining the most up-to-date phylogenies of SARS-CoV-2 to help make sense of its global movement.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36791724">36791724</a> <b>(5)</b> Accelerated SARS-CoV-2 intrahost evolution leading to distinct genotypes during chronic infection<ul><li> Our estimate for the global evolutionary rate, based on a careful random sampling of representative genomes from GISAID per month per variant, is within the expected range of what is reported in other studies that use the same regression method.<xref rid="bib8" ref-type="bibr"><sup>8</sup></xref><sup>,</sup><xref rid="bib50" ref-type="bibr"><sup>50</sup></xref> It is worth noting that estimates of the background rate of evolution vary due to different methodologies and downsampling used: 8&#xA0;&#xD7;&#xA0;10<sup>&#x2212;4</sup> s/s/y is commonly used in phylodynamic analyses,<xref rid="bib51" ref-type="bibr"><sup>51</sup></xref><sup>,</sup><xref rid="bib52" ref-type="bibr"><sup>52</sup></xref><sup>,</sup><xref rid="bib53" ref-type="bibr"><sup>53</sup></xref> and the current (June 2022) <b>Nextstrain</b> estimate is approximately 9.9&#xA0;&#xD7;&#xA0;10<sup>&#x2212;4</sup> s/s/y.<li><hr/></td></tr><tr><td>R</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0040">https://www.R-project.org/</ext-link></td></tr><tr><td>APE</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/ape/index.html" id="intref0045">https://cran.r-project.org/web/packages/ape/</ext-link></td></tr><tr><td>Phytools</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/phytools/index.html" id="intref0050">https://cran.r-project.org/web/packages/phytools/</ext-link></td></tr><tr><td>Augur</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0055">https://github.com/<b>nextstrain</b>/augur</ext-link>.<li> To understand the genetic relationship of the consensus SARS-CoV-2 genomes from the chronic infection and other WHO-designated SARS-CoV-2 variants (<ext-link ext-link-type="uri" xlink:href="https://www.who.int/activities/tracking-SARS-CoV-2-variants" id="intref0180">https://www.who.int/activities/tracking-SARS-CoV-2-variants</ext-link>), we constructed phylogenetic trees with branches resolved by time and genetic divergence, i.e., number of mutations, using the <b>Nextstrain</b> pipeline (version 3.0.3).<li> To understand the genetic relationship of the consensus SARS-CoV-2 genomes from the chronic infection and other WHO-designated SARS-CoV-2 variants (<ext-link ext-link-type="uri" xlink:href="https://www.who.int/activities/tracking-SARS-CoV-2-variants" id="intref0180">https://www.who.int/activities/tracking-SARS-CoV-2-variants</ext-link>), we constructed phylogenetic trees with branches resolved by time and genetic divergence, i.e., number of mutations, using the <b>Nextstrain</b> pipeline (version 3.0.3).<xref rid="bib91" ref-type="bibr"><sup>91</sup></xref> We used Nextalign (version 1.10.2) (<ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/nextalign" id="intref0185">https://github.com/neherlab/nextalign</ext-link>) and Augur (version 11.1.2),<xref rid="bib92" ref-type="bibr"><sup>92</sup></xref> implemented in the <b>Nextstrain</b> pipeline, to filter out the genomes based on sampling dates, construct maximum likelihood phylogenies with the generalized time-reversible (GTR) model using IQ-TREE (version 2.0.3),<xref rid="bib93" ref-type="bibr"><sup>93</sup></xref> refine and reconstruct mutations on the phylogeny, and estimate the effective population size (N<sub>e</sub>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36768752">36768752</a> <b>(1)</b> Detection of High Level of Co-Infection and the Emergence of Novel SARS CoV-2 Delta-Omicron and Omicron-Omicron Recombinants in the Epidemiological Surveillance of Andalusia<ul><li> Mutations specific to lineages are retrieved from LAPIS (Lightweight API for Sequences), which uses fully public data pre-processed and hosted by <b>Nextstrain</b> from NCBI GenBank, as in sc2rf software [<xref rid="B19-ijms-24-02419" ref-type="bibr"><u>19</u></xref>].<li> [19](          <person-group person-group-type="author"><name><surname>Schimmel</surname><given-names>L.</given-names></name><name><surname>Poon</surname><given-names>A.</given-names></name><name><surname>Symalla</surname><given-names>T.</given-names></name></person-group>          <article-title>Sc2rf-SARS-Cov-2 Recombinant Finder</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/lenaschimmel/sc2rf" ext-link-type="uri">https://github.com/lenaschimmel/sc2rf</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-11-22">(accessed on 22 November 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36718149">36718149</a> <b>(1)</b> Evolutionary analysis of Omicron variant BF.7 and BA.5.2 pandemic in China<ul><li> The time-scaled phylogenetic tree was constructed using <b>NextStrain</b><xref rid="b0060" ref-type="bibr"><sup>12</sup></xref> and Treetime.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36814518">36814518</a> <b>(2)</b> How the Commonwealth of the Northern Mariana Islands stalled COVID-19 for 22 months and managed its first significant community transmission<ul><li> <b>Nextstrain</b> (<xref rid="R11" ref-type="bibr"><italic>11</italic></xref>) was used to conduct all phylogenetic analyses.<li> Following the standard <b>Nextstrain</b> Augur (<xref rid="R12" ref-type="bibr"><italic>12</italic></xref>) pipeline, nucleotide alignment was conducted with MAFFT, (<xref rid="R13" ref-type="bibr"><italic>13</italic></xref>) maximum-likelihood phylogenetic trees were created with IQ-TREE2, (<xref rid="R14" ref-type="bibr"><italic>14</italic></xref>) time-resolved phylogenetic trees were created with TreeTime (<xref rid="R15" ref-type="bibr"><italic>15</italic></xref>) and results were visualized using Auspice.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36693089">36693089</a> <b>(2)</b> Correlated substitutions reveal SARS-like coronaviruses recombine frequently with a diverse set of structured gene pools<ul><li>We used the 191 WGS used in the current <italic toggle="yes"><b>Nextstrain</b></italic> build for SL-CoVs&#xA0;(<xref rid="r52" ref-type="bibr"><u>52</u></xref><xref rid="r53" ref-type="bibr"/>&#x2013;<xref rid="r54" ref-type="bibr"><u>54</u></xref>) and aligned these to the NCBI reference genome for SARS-CoV-2 (see <italic toggle="yes">Materials and Methods</italic> for details).<li> As new subvariants of SARS-CoV-2 arose during the peer review of this manuscript, we performed an updated analysis in July 2022 using <italic toggle="yes"><b>Nextstrain</b></italic>&#x2019;s subsampling of SARS-CoV-2 sequences from across the globe from the last 6 mo (<ext-link xlink:href="https://www.pnas.org/lookup/doi/10.1073/pnas.2206945119#supplementary-materials" ext-link-type="uri" xlink:show="new"><italic toggle="yes">SI&#xA0;Appendix</italic>, Fig.<li> [52](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32320687">32320687</a>) [54](<person-group person-group-type="author"><string-name><given-names>P.</given-names><surname>Sagulenko</surname></string-name>, <string-name><given-names>V.</given-names><surname>Puller</surname></string-name>, <string-name><given-names>R. A.</given-names><surname>Neher</surname></string-name></person-group>, <article-title>TreeTime: Maximum-likelihood phylodynamic analysis</article-title>. <source>Virus Evol.</source><volume><bold>4</bold></volume>, <fpage>1</fpage>&#x2013;<lpage>9</lpage> (<year>2018</year>).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36755900">36755900</a> <b>(2)</b> Early detection of SARS-CoV-2 variants through dynamic co-mutation network surveillance<ul><li>The phylogenetic-tree-based methods such as Pangolin (<xref rid="B20" ref-type="bibr"><u>20</u></xref>), <b>Nextstrain</b> (<xref rid="B21" ref-type="bibr"><u>21</u></xref>), and GISAID (<xref rid="B22" ref-type="bibr"><u>22</u></xref>) have been consistently proposed for SARS-CoV-2 variant detection and their evolution surveillance.<li> Second, their subtyping fineness either results in excess burden on variant surveillance (e.g., Pangolin with &gt;2,000 lineages, so far) or delayed detection and communication of dangerous variants (e.g., <b>Nextstrain</b> with 31 clades and GISAID with 11 clades) (<xref rid="B9" ref-type="bibr"><u>9</u></xref>).<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30854550">30854550</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35369526">35369526</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36702307">36702307</a> <b>(4)</b> Towards real-time monitoring of COVID-19 nosocomial clusters using SARS-CoV-2 genomes in a university hospital of the French Alps<ul><li> Inspired by the <b>Nextstrain</b> experience <xref rid="b0005" ref-type="bibr"><u>[1]</u></xref>, <xref rid="b0010" ref-type="bibr"><u>[2]</u></xref> and associated with the development of a local clinical data warehouse (PREDIMED) <xref rid="b0015" ref-type="bibr"><u>[3]</u></xref>, we developed a software tool to query electronic health records (EHR) in view of identifying potential nosocomial pathways and clusters of PCR-confirmed COVID-19 patients and of calculating genetic distances between their PCR-confirmed and nanopore-sequenced samples, thereby identifying single-lineage infections between patients.<li>Variant frequencies generated with 2021-year genomic data extracted from our <b>Nextstrain</b> visualization platform (top), COVID-19 nosocomial clusters identified in 2021 by the Infection Prevention and Control unit and sequenced by the Virology unit (bottom).<li> scaled phylogenies of a Delta SARS-CoV-2 nosocomial cluster extracted from our local instance of the <b>Nextstrain</b> visualization platform.<li>We integrated the <b>Nextstrain</b> visualization platform with our software tool and its web-reporting interface in a comprehensive platform called InfeCtrl (&#x201C;Infect Control&#x201D;), which is dedicated to real-time epidemic surveillance.<li> [[1]](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinforma Oxf Engl</source>          <volume>34</volume>          <issue>23</issue>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>        ) [[2]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [[3]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35673198">35673198</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36690803">36690803</a> <b>(1)</b> Neutralization activity of IgG antibody in COVID&#x2011;19&#x2011;convalescent plasma against SARS-CoV-2 variants<ul><li> As described in the Methods section, the tested plasma samples were obtained from patients who were infected with COVID-19 between June 2020 and March 2021, and thus, it is considered that most IgGs were from convalescent plasmas of non-VOCs (<b>Nextstrain</b> clade 20B).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36828492">36828492</a> <b>(7)</b> Mpox, Caused by the MPXV of the Clade IIb Lineage, Goes Global<ul><li>A total of 528 mpox genomes (as of 3 October 2022) from six continents were selected and used for phylogenetic analysis based on the online analysis tool (built with <b>nextstrain</b>/monkeypox (<uri xlink:href="https://<b>nextstrain</b>.org/mpox/mpxv">https://<b>nextstrain</b>.org/mpox/mpxv</uri>, accessed on 23 December 2022)).<li>As of 23 December 2022, 617 mpox genomes from six continents have been published (<uri xlink:href="https://<b>nextstrain</b>.org/mpox/mpxv">https://<b>nextstrain</b>.org/mpox/mpxv</uri>, accessed on 23 December 2022), among which the number per clade area was as follows: I (<italic toggle="yes">n</italic> = 48), IIa (<italic toggle="yes">n</italic> = 26), and IIb (<italic toggle="yes">n</italic> = 543).<li> Furthermore, mpox sequences have been obtained from 12 different hosts (<uri xlink:href="https://<b>nextstrain</b>.org/mpox/mpxv">https://<b>nextstrain</b>.org/mpox/mpxv</uri>), including <italic toggle="yes">Homo sapiens</italic> (<italic toggle="yes">n</italic> = 570), <italic toggle="yes">Pan troglodytes verus</italic> (<italic toggle="yes">n</italic> = 12), <italic toggle="yes">Cricetomys Gambians</italic> (<italic toggle="yes">n</italic> = 1), <italic toggle="yes">Crocidura littoralis</italic> (<italic toggle="yes">n</italic> = 1), <italic toggle="yes">Cynomys</italic> (<italic toggle="yes">n</italic> = 1), <italic toggle="yes">Funisciurus</italic> (<italic toggle="yes">n</italic> = 1), <italic toggle="yes">Funisciurus anerythrus</italic> (<italic toggle="yes">n</italic> = 1), <italic toggle="yes">Funisciurus bayonii</italic> (<italic toggle="yes">n</italic> = 1), Gliridae (<italic toggle="yes">n</italic> = 1), <italic toggle="yes">Malacomys longipes</italic> (<italic toggle="yes">n</italic> = 1), and Platyrrhini (<italic toggle="yes">n</italic> = 1).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36658311">36658311</a> <b>(6)</b> Plagued by a cryptic clock: insight and issues from the global phylogeny of <italic>Yersinia pestis</italic><ul><li> All images are produced by <b>nextstrain</b>.org are by CC BY.<li> All images are produced by <b>nextstrain</b>.org are by CC BY.<li> All images are produced by <b>nextstrain</b>.org are by CC BY.<li> All phylogenetic data, visualizations are publicly accessible and interactive at: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/ktmeaton/yersinia-pestis">https://<b>nextstrain</b>.org/community/ktmeaton/yersinia-pestis</ext-link>.<li>Data visualization was performed using the python package seaborn (v0.11.1)<sup><xref ref-type="bibr" rid="CR72"><u>72</u></xref></sup> and Auspice (v2.23.0)<sup><xref ref-type="bibr" rid="CR73"><u>73</u></xref></sup>, a component of the <b>Nextstrain</b> visualization suite.<li> [72](          <person-group person-group-type="author">            <name>              <surname>Waskom</surname>              <given-names>ML</given-names>            </name>          </person-group>          <article-title>Seaborn: statistical data visualization</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3021</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03021</pub-id>        ) [73](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36814938">36814938</a> <b>(5)</b> Avian H7N9 influenza viruses are evolutionarily constrained by stochastic processes during replication and transmission in mammals<ul><li> All coding region changes are reported using H7 numbering for the H7N9 viruses and H1 numbering for the H1N1 virus, consistent with the numbering schemes used in <b>Nextstrain</b> (<xref rid="R17" ref-type="bibr">Hadfield et&#xA0;al.<li> To look for such a signal, we compared non-synonymous iSNVs detected in this study (<italic toggle="yes">n</italic>&#x2009;=&#x2009;262) to non-synonymous single nucleotide polymorphisms (SNPs) found at the most distal nodes in <b>Nextstrain</b>&#x2019;s H7N9 phylogenetic trees for all gene segments (<italic toggle="yes">n</italic>&#x2009;=&#x2009;2,071 unique isolates) (<xref rid="R17" ref-type="bibr">Hadfield et&#xA0;al.<li> Among the list of variants shared between our data and <b>Nextstrain</b>&#x2019;s, we looked for those that were proportionally overrepresented in sequences from humans.<li>Considering all iSNVs we detected, around half (46.6&#x2009;per cent, 77/165) of the Anhui/1 iSNVs and 36.1&#x2009;per cent (35/97) of the GD/3 iSNVs can be found in at least one bird or human H7N9 sequence on <b>Nextstrain</b>.<li>H7N9 phylogenies obtained from <b>Nextstrain</b> (<xref rid="R17" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36633196">36633196</a> <b>(2)</b> Whole&#x2010;genome sequencing of hMPXV1 in five Italian cases confirms the occurrence of the predominant epidemic lineage<ul><li>Based on the distinctive genomic and epidemiological features, the virus causing the 2022 monkeypox outbreak was renamed hMPXV1.<xref rid="jmv28493-bib-0004" ref-type="bibr"><sup>4</sup></xref> To date, several hMPXV1 genome sequences were deposited in public repositories and classified in distinct B.1 lineages (B.1.1&#x2013;B1.1.14) (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li>The mutation pattern was analyzed by the NextClade online tool using as a reference the strain MPXV_USA_2022_MA001 in NC_063383 coordinates (pseudo_ON563414) (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>, v2.8.1, last access&#xA0;November 25, 2022).<li> These genomes, along with the five generated herein, were analyzed with the <b>Nextstrain</b> suite (v5.0.1)<xref rid="jmv28493-bib-0014" ref-type="bibr"><sup>14</sup></xref> to infer phylogenetic relationship.<li> The tree was generated using the <b>Nextstrain</b> tool and lineage and geographic location coding are reported in the legend.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36839431">36839431</a> <b>(0)</b> Epidemiological, Clinical, and Virological Investigation of the First Four Cases of Monkeypox in Cartagena during the 2022 Outbreak<ul><li> Consensus sequences were deposited in NCBI, and clades and lineages were determined using Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 3 November 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851491">36851491</a> <b>(3)</b> Origin, Genetic Variation and Molecular Epidemiology of SARS-CoV-2 Strains Circulating in Sardinia (Italy) during the First and Second COVID-19 Epidemic Waves<ul><li>The CoVsurver mutations app of GISAID [<xref rid="B34-viruses-15-00277" ref-type="bibr"><u>34</u></xref>] was used to classify the sequences according to the different <b>Nextstrain</b> clades, as well as to screen Sardinian sequences for any nucleotide and amino acidic mutations in comparison with the reference genome Cov19/Wuhan/W1V04/.<li> Individual subtrees containing 500 sequences each were extracted and visualized using Auspice [<xref rid="B45-viruses-15-00277" ref-type="bibr"><u>45</u></xref>] and <b>NextStrain</b> [<xref rid="B46-viruses-15-00277" ref-type="bibr"><u>46</u></xref>].<li>The full SARS-CoV-2 tree obtained from all GISAID sequences using Usher [<xref rid="B43-viruses-15-00277" ref-type="bibr"><u>43</u></xref>] and <b>Nextstrain</b> [<xref rid="B46-viruses-15-00277" ref-type="bibr"><u>46</u></xref>] evidenced that the 55 Sardinian SARS-CoV-2 samples are grouped into 18 different disconnected subtrees (<xref rid="viruses-15-00277-f005" ref-type="fig"><u>Figure 5</u></xref> and <xref rid="app1-viruses-15-00277" ref-type="app"><u>Table S4</u></xref>).<li> [34](          <article-title>GISAID</article-title>          <comment>Available online: <ext-link xlink:href="https://gisaid.org/database-features/covsurver-mutations-app/" ext-link-type="uri">https://gisaid.org/database-features/covsurver-mutations-app/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-10">(accessed on 10 May 2021)</date-in-citation>        ) [45](          <article-title>AUSPICE</article-title>          <comment>Available online: <ext-link xlink:href="https://auspice.us" ext-link-type="uri">https://auspice.us</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-11-07">(accessed on 7 November 2022)</date-in-citation>        ) [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36648012">36648012</a> <b>(0)</b> Evoluci&#xF3;n temporal de las variantes de SARSCoV-2 circulantes en el &#xE1;rea sanitaria de C&#xE1;diz<ul><li> en el &#xC1;rea de Bioinfoirm&#xE1;tica Cl&#xED;nica de la Fundaci&#xF3;n Progreso y Salud, ensamblando con la herramienta nf-core/viralrecon (<ext-link xlink:href="https://nfco.re/viralrecon" ext-link-type="uri">https://nfco.re/viralrecon</ext-link>) [<xref rid="ref5" ref-type="bibr"><u>5</u></xref>], asignando los diferentes clados y anotando las mutaciones presentes con Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) [<xref rid="ref6" ref-type="bibr"><u>6</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32055031">32055031</a>) [6](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>E</given-names></string-name>, <string-name><surname>Neher</surname><given-names>R</given-names></string-name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source>. <year>2021</year><month>Nov</month><day>30</day>;<volume>6</volume>(<issue>67</issue>):<fpage>3773</fpage>. doi: <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851484">36851484</a> <b>(2)</b> Simple Wastewater Preparation Protocol Applied to Monitor the Emergence of the Omicron 21L/BA.2 Variant by Genome Sequencing<ul><li> To date, one of the most recent of these is the Omicron variant (B.1.1.529), which arrived from South Africa [<xref rid="B6-viruses-15-00268" ref-type="bibr"><u>6</u></xref>] including sub-lineage variants such as 21L (<b>Nextstrain</b> classification (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>; access date: 16 January 2023)) or BA.2 (Pangolin classification (<uri xlink:href="https://cov-lineages.org/resources/pangolin.html">https://cov-lineages.org/resources/pangolin.html</uri>; access date: 16 January 2023)), which was first highly prevalent in Denmark, then spread to across the whole of Europe [<xref rid="B7-viruses-15-00268" ref-type="bibr"><u>7</u></xref>,<xref rid="B8-viruses-15-00268" ref-type="bibr"><u>8</u></xref>].<li> A custom Python script was used to associate mutations present in the output file generated by the CLC genomics workbench with a database of mutations of SARS-CoV-2 variants built by our institute based on the <b>Nextstrain</b> tool [<xref rid="B39-viruses-15-00268" ref-type="bibr"><u>39</u></xref>].<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34982466">34982466</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35101887">35101887</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35150013">35150013</a>) [39](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36686884">36686884</a> <b>(3)</b> Characterization and phylogenetic analysis of Iranian SARS&#x2010;CoV&#x2010;2 genomes: A phylogenomic study<ul><li>The maximum likelihood phylogenetic tree by <b>Nextstrain</b> online (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) showed six and five clades in the samples obtained from the 24 cities of Iran and Tehran, respectively (Figure&#xA0;<xref rid="hsr21052-suppl-0003" ref-type="supplementary-material"><u>1S</u></xref> a and b).<li> GISAID,<xref rid="hsr21052-bib-0015" ref-type="bibr"><sup>15</sup></xref> <b>Nextstrain</b>,<xref rid="hsr21052-bib-0016" ref-type="bibr"><sup>16</sup></xref> and lineage<xref rid="hsr21052-bib-0017" ref-type="bibr"><sup>17</sup></xref> nomenclature systems indicated several lineages and clades among Iranian virus populations distinguished by the mutation of amino acid sequences.<li> Similarly, the <b>Nextstrain</b> clade 20B and the B.1.1 and B.1.1.413 lineages were very common in the cities' and Tehran's sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36690516">36690516</a> <b>(4)</b> Outbreak investigation in a COVID-19 designated hospital: The combination of phylogenetic analysis and field epidemiology study suggesting airborne transmission<ul><li> A phylogenetic tree was constructed and visualized using <b>Nextstrain</b> analysis (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0015">https://github.com/<b>nextstrain</b>/ncov</ext-link>)<xref rid="bib25" ref-type="bibr"><sup>25</sup></xref> and Auspice (<ext-link ext-link-type="uri" xlink:href="https://auspice.us/" id="intref0020">https://auspice.us/</ext-link>).<li> All strains from Hospital A belong to clade 20I (alpha, V1) in a time-scale tree that is constructed using <b>Nextstrain</b> tool (3a).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680285">36680285</a> <b>(6)</b> Detection and Molecular Characterization of the SARS-CoV-2 Delta Variant and the Specific Immune Response in Companion Animals in Switzerland<ul><li>For the phylogenetic analysis, the <b>Nextstrain</b> ncov pipeline (GitHub&#x2014;<b>nextstrain</b>/ncov; <uri xlink:href="https://github.com/<b>nextstrain</b>/ncov/">https://github.com/<b>nextstrain</b>/ncov/</uri>; accessed on 9 May 2022) was used [<xref rid="B49-viruses-15-00245" ref-type="bibr"><u>49</u></xref>].<li> Following the <b>Nextstrain</b> pipeline, sites prone to low sequencing accuracy were masked prior to the phylogenetic inference: sites 21846, 21987, 22992, 23012, 23063, 23604, and 24410, as well as 100 sites at the beginning and 200 at the end of the sequences.<li> Genetic similarity was calculated using <b>Nextstrain</b> priority protocol based on pairwise SNP distance.<li> [49](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36717455">36717455</a> <b>(1)</b> Biology of the SARS-CoV-2 Coronavirus<ul><li>For the first time, a genome derived from two different variants of SARS CoV-2 (20A and 20B according to the <b>NextStrain</b> classification) was reported in February 2021 [<xref ref-type="bibr" rid="CR65"><u>65</u></xref>].<li> [65](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33540596">33540596</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33794262">33794262</a> <b>(3)</b> Incidence of delayed asymptomatic COVID-19 recurrences in a 6-month longitudinal study<ul><li> Genome sequencing of SARS-CoV-2 was performed on nasopharyngeal samples following ARTIC amplicon sequencing protocol for MinIon version V3- Phylogenetic analysis was done using webserver <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="interref0001">https://<b>nextstrain</b>.org/</ext-link>), with the SARS-CoV-2 database Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0002">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33855294">33855294</a> <b>(5)</b> Possible Involvement of Central Nervous System in COVID-19 and Sequence Variability of SARS-CoV-2 Revealed in Autopsy Tissue Samples: A Case Report<ul><li>The clade designation system<sup><xref rid="bibr19-2632010X211006096" ref-type="bibr"><u>19</u></xref></sup> was used by downloading the <b>Nextstrain</b> Metadata set on the May 28, 2020.<li> The phylogenetic analysis (<xref ref-type="fig" rid="fig2-2632010X211006096"><u>Figure 2</u></xref>) showed that all 4 virus samples belonged to the A2a clade (corresponding to clade 20B by the classification system implemented in Nextclade: <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/results">https://clades.<b>nextstrain</b>.org/results</ext-link>).<li> Phylogenetic tree rooted with the Wuhan-Hu-1 sequence (NC_045512) and GISAID sequences, shown for example EPI ISL 420347 (GISAID identifier) A2 (<b>Nextstrain</b> clade) and the 4 patient sequences (spinal, tracheal, right lung, and left lung).<li> Most European SARS-CoV-2-sequences on <b>Nextstrain</b> as per May 28, 2020, belong to this clade (3177 of 4082; 77.8%).<li> 99.03%) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/europe">https://<b>nextstrain</b>.org/ncov/europe</ext-link>).<li> [19](<collab>Nextstrain</collab>. <article-title>Nextstrain: analysis and visualization of pathogen sequence data</article-title>, <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link> (<year>2020</year>, <comment>accessed 22 December 2020</comment>).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33785459">33785459</a> <b>(7)</b> Emergence and outcomes of the SARS-CoV-2 &#x2018;Marseille-4&#x2019; variant<ul><li>The 1038 SARS-CoV-2 whole genome sequences obtained in our laboratory were analysed using the Nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intr0040">https://clades.<b>nextstrain</b>.org/</ext-link>) (<xref rid="bib0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and an in-house script written in Python.<li> Phylogenetic trees were reconstructed using the <b>nextstrain</b>/ncov tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intr0045">https://github.com/<b>nextstrain</b>/ncov</ext-link>) and visualized with Auspice software (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/" id="intr0050">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).<li> The first case of infection with the Marseille-4 variant, named 20A.EU2 in the <b>Nextstrain</b> classification (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intr0080">https://clades.<b>nextstrain</b>.org/</ext-link>) (<xref rid="bib0095" ref-type="bibr"><u>Hodcroft et al., 2020</u></xref>), was detected in a German patient on March 24, 2020.<li> Phylogenetic trees were reconstructed and visualized using the <b>Nextstrain</b> pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/" id="intr0010">https://github.com/<b>nextstrain</b>/ncov/</ext-link>) (<xref rid="bib0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hodcroft et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Zuber</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Nadeau</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Comas</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Gonzalez Candelas</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>SeqCOVID-SPAIN Consortium</surname>            </name>          </person-group>          <article-title>Emergence and spread of a SARS-CoV-2 variant through Europe in the summer of 2020</article-title>          <source>medRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.10.25.20219063</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33855277">33855277</a> <b>(2)</b> Synergistic Inhibition of SARS-CoV-2 ReplicationUsing Disulfiram/Ebselen andRemdesivir<ul><li> This is because the proteins essential for RNAreplication such as the RTC core are the most conserved among coronaviruses.Indeed, analysis of the SARS-CoV-2 gene sequences show the least divergencein the gene encoding these conserved domains (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33815862">33815862</a> <b>(4)</b> Molecular detection of SARS-CoV-2 being challenged by virus variation and asymptomatic infection<ul><li> Up to January 2021, the <b>Nextstrain</b> platform had collected a total of 3,819 SARS-CoV-2 genome sequences from different countries and regions around the world.<li>The phylogenetic tree of complete genome sequences of 3,819 SARS-CoV-2 was obtained and analyzed with <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/nCoV/global?l=unrooted" id="intref0010">https://<b>nextstrain</b>.org/nCoV/global?l=unrooted</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33770098">33770098</a> <b>(11)</b> Genomic variation and epidemiology of SARS-CoV-2 importation and early circulation in Israel<ul><li> The main nomenclature systems of SARS-CoV-2 clades include <b>Nextstrain</b>, who name a new major clade when it reaches a frequency of 20% globally by using a year-letter genetic clade naming [<xref rid="pone.0243265.ref005" ref-type="bibr"><u>5</u></xref>], and GISAID, who use the statistical distribution of genome distances in phylogenetic clusters and name the clades by the actual letters of the defining marker mutations of each cluster [<xref rid="pone.0243265.ref004" ref-type="bibr"><u>4</u></xref>].<li> According to <b>Nextstrain</b>&#x2019;s nomenclature system [<xref rid="pone.0243265.ref005" ref-type="bibr"><u>5</u></xref>], five globally circulating SARS-CoV-2 clades are currently defined&#x2013; 19A (the root clade) and 19B, that originated in Asia and are still widespread there, and clades 20A, B and C now dominate global infections and are widespread in Europe [<xref rid="pone.0243265.ref006" ref-type="bibr"><u>6</u></xref>, <xref rid="pone.0243265.ref007" ref-type="bibr"><u>7</u></xref>].<li> C18877T emerging from clade 20, C13730T emerging from clade 19) via <b>Nextstrain</b>&#x2019;s global genomic epidemiology analysis [<xref rid="pone.0243265.ref005" ref-type="bibr"><u>5</u></xref>] and may become a major clade once they reach sufficient global frequency/spread.<li> Clade nomenclature was attained from <b>Nextstrain</b> [<xref rid="pone.0243265.ref005" ref-type="bibr"><u>5</u></xref>].<li>To define imported and circulating clades in Israel, we applied the <b>Nextstrain</b> nomenclature (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv</ext-link>), that includes the originating clade 19A and its derivation 19B, and the emerging clades 20 and its derivations 20B and 20C with the spike mutation D614G [<xref rid="pone.0243265.ref006" ref-type="bibr"><u>6</u></xref>] that had widely spread in Europe since mid-February [<xref rid="pone.0243265.ref005" ref-type="bibr"><u>5</u></xref>].<li> The novel positions were associated with defined clades via <b>Nextstrain</b> auspice visualization tool [<xref rid="pone.0243265.ref005" ref-type="bibr"><u>5</u></xref>].<li> Visualization with <b>Nextstrain</b> global analysis (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) showed that these mutated positions are not specifically unique to Israel and were observed in several SARS-CoV-2 genome sequences worldwide.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33806013">33806013</a> <b>(3)</b> SARS-CoV-2 N501Y Introductions and Transmissions in Switzerland from Beginning of October 2020 to February 2021&#x2014;Implementation of Swiss-Wide Diagnostic Screening and Whole Genome Sequencing<ul><li> 15 genomes per country and per month in Europe, and 8 genomes per country and per month for the rest of the world (contextual samples) totaling 10,316 genomes, using the <b>nextstrain</b> software v.2.0.0.post1 (<b>nextstrain</b>.org) and augur v.10.3.8 [<xref rid="B39-microorganisms-09-00677" ref-type="bibr"><u>39</u></xref>].<li> The distribution and details of the B.1.258 lineage is further described on <b>nextstrain</b>.org, <uri xlink:href="https://cov-lineages.org/lineages/lineage_B.1.258.html">https://cov-lineages.org/lineages/lineage_B.1.258.html</uri> and <uri xlink:href="https://covariants.org/variants/S.N439K">https://covariants.org/variants/S.N439K</uri> (accessed on 3 March 2021).<li> [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33761008">33761008</a> <b>(1)</b> Diverse SARS-CoV-2 variants preceded the initial COVID-19 outbreak in Croatia<ul><li> 10<sup>-3</sup> nucleotide substitution per site per year) (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org">www.<b>nextstrain</b>.org</ext-link>.) [<xref ref-type="bibr" rid="CR8"><u>8</u></xref>, <xref ref-type="bibr" rid="CR19"><u>19</u></xref>, <xref ref-type="bibr" rid="CR20"><u>20</u></xref>, <xref ref-type="bibr" rid="CR26"><u>26</u></xref>, <xref ref-type="bibr" rid="CR27"><u>27</u></xref>, <xref ref-type="bibr" rid="CR31"><u>31</u></xref>].<li> UTR, Nsp3, RdRp, spike, and ORF3a genes (Supplementary Table 2), which has been also observed globally (<b>nextstrain</b>.org) [<xref ref-type="bibr" rid="CR26"><u>26</u></xref>, <xref ref-type="bibr" rid="CR27"><u>27</u></xref>] and might indicate a strong fitness and/or adaptation advantage [<xref ref-type="bibr" rid="CR15"><u>15</u></xref>, <xref ref-type="bibr" rid="CR21"><u>21</u></xref>].<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32878977">32878977</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30976793">30976793</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32793182">32793182</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32387564">32387564</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32167180">32167180</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15222897">15222897</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32387564">32387564</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32167180">32167180</a>) [15](          <person-group person-group-type="author">            <name>              <surname>Korber</surname>              <given-names>B</given-names>            </name>            <name>              <surname>Fischer</surname>              <given-names>WM</given-names>            </name>            <name>              <surname>Gnanakaran</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Yoon</surname>              <given-names>H</given-names>            </name>            <name>              <surname>Theiler</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Abfalterer</surname>              <given-names>W</given-names>            </name>            <name>              <surname>Hengartner</surname>              <given-names>N</given-names>            </name>            <name>              <surname>Giorgi</surname>              <given-names>EE</given-names>            </name>            <name>              <surname>Bhattacharya</surname>              <given-names>T</given-names>            </name>            <name>              <surname>Foley</surname>              <given-names>B</given-names>            </name>            <name>              <surname>Hastie</surname>              <given-names>KM</given-names>            </name>            <name>              <surname>Parker</surname>              <given-names>MD</given-names>            </name>            <name>              <surname>Partridge</surname>              <given-names>DG</given-names>            </name>            <name>              <surname>Evans</surname>              <given-names>CM</given-names>            </name>            <name>              <surname>Freeman</surname>              <given-names>TM</given-names>            </name>            <name>              <surname>de Silva</surname>              <given-names>TI</given-names>            </name>            <name>              <surname>Sheffield</surname>              <given-names>C-GG</given-names>            </name>            <name>              <surname>McDanal</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Perez</surname>              <given-names>LG</given-names>            </name>            <name>              <surname>Tang</surname>              <given-names>H</given-names>            </name>            <name>              <surname>Moon-Walker</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Whelan</surname>              <given-names>SP</given-names>            </name>            <name>              <surname>LaBranche</surname>              <given-names>CC</given-names>            </name>            <name>              <surname>Saphire</surname>              <given-names>EO</given-names>            </name>            <name>              <surname>Montefiori</surname>              <given-names>DC</given-names>            </name>          </person-group>          <article-title>Tracking changes in SARS-CoV-2 Spike: evidence that D614G increases infectivity of the COVID-19 virus</article-title>          <source>Cell</source>          <year>2020</year>          <volume>182</volume>          <issue>812&#x2013;827</issue>          <fpage>e819</fpage>        ) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33106671">33106671</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33269368">33269368</a> <b>(14)</b> Emergence and spread of a SARS-CoV-2 variant through Europe in the summer of 2020<ul><li> This variant (<b>Nextstrain</b> clade 20A) seeded large outbreaks in Europe in early 2020 and subsequently dominated the outbreaks in the Americas, thereby largely replacing previously circulating lineages.<li> Focal phylogenies for these, and other variants mentioned in this paper, as well as updated phylogenies of SARS-CoV-2 in Europe and individual European countries can be found at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/neherlab"><b>nextstrain</b>.org/groups/neherlab</ext-link>.<li> While initial introductions of the variant likely originated from Spain, 20E (EU1) cases outside of Spain surpassed those in Spain in late September and later cross-border transmissions likely originated in other European countries (see <xref rid="SD2" ref-type="supplementary-material"><u>Fig S4B</u></xref> and 20E (EU1) <b>Nextstrain</b> build online).<li> Unlike the initial introduction, later sequences in Norway or Iceland often cluster more closely with diversity in European countries other than Spain, which may suggest further introductions came from third countries (see 20E (EU1) <b>Nextstrain</b> build online).<li>We use the <b>Nextstrain</b> pipeline for our phylogenetic analyses <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/">https://github.com/<b>nextstrain</b>/ncov/</ext-link> (<xref rid="R4" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> For computational feasibility, ease of interpretation, and to balance disparate sampling efforts between countries, the <b>Nextstrain</b>-maintained runs subsample the available genomes across time and geography, resulting in final builds of ~5,000 genomes each.<li> After sub-sampling, the 20E (EU1) cluster within the <b>Nextstrain</b> build contains 5,145 sequences, 3,369 of which are unique (accounting for Ns).<li>Sequences were downloaded from GISAID at the end of January and analysed using the <b>nextstrain</b>/ncov work-flow, using a cutoff date of the 30 Sept (<xref rid="F3" ref-type="fig"><u>Fig 3</u></xref>) or 30 Nov (all other analyses).<li>The sub-sampling of the standard <b>Nextstrain</b> analysis means that we are not able to visualise the true size or phylogenetic structure of the cluster in question.<li> sample in the <b>nextstrain</b>/ncov pipeline.<li> This selection will exclude any sequences with no coverage or reversions at these positions, but the similarity-based sampling during the <b>Nextstrain</b> run will identify these, as well as any other nearby sequences, and incorporate them into the dataset.<li> [Hadfield et al., 2018](<name><surname>Hadfield</surname><given-names>J.</given-names></name>, <name><surname>Megill</surname><given-names>C.</given-names></name>, <name><surname>Bell</surname><given-names>S. M.</given-names></name>, <name><surname>Huddleston</surname><given-names>J.</given-names></name>, <name><surname>Potter</surname><given-names>B.</given-names></name>, <name><surname>Callender</surname><given-names>C.</given-names></name>, <name><surname>Sagulenko</surname><given-names>P.</given-names></name>, <name><surname>Bedford</surname><given-names>T.</given-names></name>, <name><surname>Neher</surname><given-names>R. A.</given-names></name>, <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>, <source>Bioinformatics</source> (<year>2018</year>).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33760807">33760807</a> <b>(3)</b> SARS-CoV-2 variants reveal features critical for replication in primary human cells<ul><li> Clades were determined by <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link> [<xref rid="pbio.3001006.ref064" ref-type="bibr"><u>64</u></xref>].<li> The tree is colored based on SARS-CoV-2 <b>Nextstrain</b> clades, with circles indicating the positions of 14 SARS-CoV-2 isolates generated in this study.<li> [64](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33778788">33778788</a> <b>(1)</b> A comprehensive analysis and resource to use CRISPR-Cas13 for broad-spectrum targeting of RNA viruses<ul><li> Each crRNA in crRNA-MP5 was able to target&#xA0;almost 99% of SARS-CoV-2 isolates, and the crRNA-MP5 together can target&#xA0;all of the known SARS-CoV-2 strains regardless of the variations in different SARS-CoV-2 clades as identified in <b>Nextstrain</b><xref rid="bib21" ref-type="bibr"><sup>21</sup></xref> (<xref rid="fig4" ref-type="fig"><u>Figure&#xA0;4</u></xref>C).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33766711">33766711</a> <b>(1)</b> SARS-CoV-2 variants lacking ORF8 occurred in farmed mink and pangolin<ul><li> vison</italic>) with ORF8 nonsense mutations were classified into clades using the Nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir010">https://clades.<b>nextstrain</b>.org/</ext-link>) and the Phylogenetic Assignment of Named Global Outbreak LINeages (PANGOLIN) tool running in GISAID (<xref rid="b0165" ref-type="bibr"><u>Rambaut et al., 2020a</u></xref>).<li> Tree obtained from the <b>Nextstrain</b> website.<li> [Rambaut et al., 2020a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33778182">33778182</a> <b>(2)</b> Comparative analysis of mutational hotspots in the spike protein of SARS-CoV-2 isolates from different geographic origins<ul><li>As of September 30, 2020, SARS-CoV-2 full genome sequences were downloaded from GISAID (<ext-link ext-link-type="uri" xlink:href="http://gisaid.org" id="ir0005">http://gisaid.org</ext-link>), <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org" id="ir0010">www.<b>nextstrain</b>.org</ext-link>), and NCBI virus repository (<ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/labs/virus" id="ir0015">http://www.ncbi.nlm.nih.gov/labs/virus</ext-link>), and then the 113,381 high-quality sequences were collected after removing duplicate samples, partially sequenced genomes, and samples with unclear collection data.<li>Based on these PCR-based sequencing results, we analyzed the frequency of the mutation in the 113,381 high-quality S protein sequences available from the GISAID, <b>Nextstrain</b>, and NCBI virus repositories as of September 30, 2020.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33766552">33766552</a> <b>(1)</b> Discovery of a SARS-CoV-2 variant from the P.1 lineage harboring K417T/E484K/N501Y mutations in Kofu, Japan<ul><li><bold>Phylogenetic tree analysis of SARS-CoV-2.</bold> (<bold>A</bold>) Sequencing data were uploaded to Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0001">https://clades.<b>nextstrain</b>.org/</ext-link>) for phylogenetic analysis.<li> Schematic tree from <b>Nextstrain</b> showing clade evolution in Japan since February 2020.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33778795">33778795</a> <b>(6)</b> Phylogenomic analysis of COVID-19 summer and winter outbreaks in Hong Kong: An observational study<ul><li> We previously reported that the early cases of the third wave belong to a single lineage B.1.1.63 (previously designated as genetic cluster HK1), which is within the Global Initiative on Sharing All Influenza Data (GISAID) clade GR and <b>Nextstrain</b> clade 20B <xref rid="bib0006" ref-type="bibr"><u>[6]</u></xref>.<li> We described the genetic information using GISAID <xref rid="bib0021" ref-type="bibr"><u>[21]</u></xref>, <b>Nextstrain</b> <xref rid="bib0022" ref-type="bibr"><u>[22]</u></xref>, and PANGO lineage <xref rid="bib0023" ref-type="bibr"><u>[23]</u></xref> nomenclatures.<li> (c) GISAID and <b>Nextstrain</b> clade distribution of imported cases in Hong Kong.<li> Specimens collected on 19th September 2020 and on 25th September 2020, respectively) (PANGO lineage B.1.36.27, GISAID clade GH and <b>Nextstrain</b> 20A) (<xref rid="fig0002" ref-type="fig"><u>Fig.&#xA0;2</u></xref>a and <xref rid="fig0002" ref-type="fig"><u>2</u></xref>b).<li> Since 16th September 2020, genetically diverse genomes in different GISAID or <b>Nextstrain</b> clades were found among imported cases (<xref rid="fig0002" ref-type="fig"><u>Fig.&#xA0;2</u></xref>c).<li> First, we excluded viral genomes that have been considered not suitable for analysis by the TreeTime, the phylodynamic program used by <b>Nextstrain</b>.<li> [[6]](          <person-group person-group-type="author">            <name>              <surname>To</surname>              <given-names>K.K.</given-names>            </name>            <name>              <surname>Chan</surname>              <given-names>W.M.</given-names>            </name>            <name>              <surname>Ip</surname>              <given-names>J.D.</given-names>            </name>            <name>              <surname>Chu</surname>              <given-names>A.W.</given-names>            </name>            <name>              <surname>Tam</surname>              <given-names>A.R.</given-names>            </name>            <name>              <surname>Liu</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Unique SARS-CoV-2 clusters causing a large COVID-19 outbreak in Hong Kong</article-title>          <source>Clin. Infect. Dis.</source>          <year>2020</year>          <ext-link ext-link-type="uri" xlink:href="http://10.1093/cid/ciaa1119" id="interref0003">10.1093/cid/ciaa1119</ext-link>        ) [[21]](GISAID. Clade and lineage nomenclature aids in genomic epidemiology studies of active hCoV-19 viruses. Available at <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" id="interref0007">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>. Accessed on July 19, 2020. 2020) [[22]](Hodcroft E.B., Hadfield J., Neher R.A., Bedford T. Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org. Available at <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming" id="interref0008">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>. Accessed on July 25, 2020.) [[23]](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Holmes</surname>              <given-names>E.C.</given-names>            </name>            <name>              <surname>O'Toole</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.T.</given-names>            </name>            <name>              <surname>Ruis</surname>              <given-names>C.</given-names>            </name>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>          <source>Nat. Microbiol.</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33755704">33755704</a> <b>(3)</b> Whole-genome sequencing of SARS-CoV-2 reveals the detection of G614 variant in Pakistan<ul><li> For further classification of SARS-CoV-2 strains detected in Pakistan, all available sequences were categorized into clades defined by the <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/en/latest/tutorials/SARS-CoV-2/steps/naming_clades.html">https://docs.<b>nextstrain</b>.org/en/latest/tutorials/SARS-CoV-2/steps/naming_clades.html</ext-link>).<li> The <b>Nextstrain</b> classification has grouped SARS-CoV-2 strains into five major global clades: 19A, 19B, 20A, 20B and 20C.<li>According to the <b>Nextstrain</b> classification and phylogenetic analysis, Pakistani strains (NIH-45090, NIH-45143, and NIH-45579) of 20A clade were closely related to the strains from Oman, Slovakia, United States, and indigenous strain EPI_ISL_513925.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33666147">33666147</a> <b>(5)</b> Lessons learned 1 year after SARS-CoV-2 emergence leading to COVID-19 pandemic<ul><li> The GISAID and <b>Nextstrain</b> systems were used since the beginning of the pandemic, and the clades or lineages are defined by signature mutations.<li> The <b>Nextstrain</b> is divided into 19 (A, B) and 20 (A-J) according to the year and order when the clade emerged.<li> Although GISAID and <b>Nextstrain</b> nomenclatures are useful in understanding the virus evolution in a macroscopic scale, these systems are not able to delineate more detailed outbreak cluster information.<li> The Pango lineage system has a much finer resolution than GISAID or <b>Nextstrain</b>, and is particularly useful to capture the emergence of novel variants.<li> Clade information as inferred by <b>Nextstrain</b> or Pango lineage are shown.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33749660">33749660</a> <b>(4)</b> Genomic diversity of SARS-CoV-2 during early introduction into the Baltimore&#x2013;Washington metropolitan area<ul><li> Efforts to describe this diversity during the early phase of the pandemic have resulted in multiple clade (or lineage) designation systems including those used by the Global Information Sharing of All Influenza Data (GISAID), the <b>NextStrain</b> platform, and COVID-19 Genomics UK consortium (<xref rid="B14" ref-type="bibr"><u>14</u></xref>&#x2013;<xref rid="B16" ref-type="bibr"><u>16</u></xref>).<li> For example, <b>NextStrain</b> clade designations separate viruses that originated in China in 2019 (Clade 19) from those that were later introduced into Europe in early 2020 (Clade 20) (<xref rid="B15" ref-type="bibr"><u>15</u></xref>).<li>We used a customized <b>Nextstrain</b> Snakemake pipeline (<xref rid="B12" ref-type="bibr"><u>12</u></xref>) using augur v9.0.0 on the final data set, which included the 886 Global 1K data set plus all removed regional groups (including all 114 JHHS sequences and all published Virginia sequences), resulting in 1279 sequences used in our phylogenetic analysis.<li> Finally, clades were assigned to sequences using <b>Nextstrain</b> (<xref rid="B15" ref-type="bibr"><u>15</u></xref>) and the Pango nomenclature system (<xref rid="B14" ref-type="bibr"><u>14</u></xref>) (see <xref ref-type="supplementary-material" rid="sd"><u>Supplemental Table 3</u></xref>).<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) [15](          <comment>Hodcroft EB, et al. Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link> Updated August 2, 2020. Accessed February 12, 2021</comment>        ) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](          <comment>Hodcroft EB, et al. Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link> Updated August 2, 2020. Accessed February 12, 2021</comment>        ) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33810168">33810168</a> <b>(6)</b> Multiple Early Introductions of SARS-CoV-2 to Cape Town, South Africa<ul><li>Nextclade v0.14.0 (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, last accessed 21 March 2021) [<xref rid="B18-viruses-13-00526" ref-type="bibr"><u>18</u></xref>] was used as quality assurance to report potential sequence quality issues, to identify differences between the Tygerberg sequences and the Wuhan-Hu-1 reference sequence, and to identify clades.<li> This subsample, along with 46 of the 47 genotypes from Cape Town, was analyzed on the <b>NextStrain</b> platform (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>, last accessed 21 March 2021) [<xref rid="B18-viruses-13-00526" ref-type="bibr"><u>18</u></xref>] using the standard COVID-19 build, with slight modifications.<li>Bayesian coalescent analyses were performed on clades 19A, 20A, and 20B of the <b>NextStrain</b> build to confirm the estimated date of origin for SARS-CoV-2 as proposed in recent literature [<xref rid="B24-viruses-13-00526" ref-type="bibr"><u>24</u></xref>], to infer the estimated date to the most recent common ancestor (MRCA) for major lineages, and to infer the estimated dates of viral introductions into Cape Town.<li> Furthermore, because of the downsampling of the datasets, the ancestral state reconstruction will be different to that observed in the <b>Nextstrain</b> build.<li>The remainder of the Cape Town sequences were classified into <b>Nextstrain</b> clade 20A or pangolin lineages B.1 (<italic>n</italic> = 1) and B.1.8 (<italic>n</italic> = 27); and clade 20B; or lineages B.1.1.1, B.1.1.161, (<italic>n</italic> = 1 each), and B.1.1.29 (<italic>n</italic> = 12).<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33235813">33235813</a>) </ul></p>
<p><u>No PMID</u> <b>(2)</b> Significance of deep learning for Covid-19: state-of-the-art review<ul><li> Some projects like <b>Nextstrain</b> (<b>Nextstrain</b>, <xref ref-type="bibr" rid="CR80"><u>2020</u></xref>) are searching at the genetic assortment of coronaviruses, to distinguish the worldwide outbreak of Covid-19 by assuming the family tree of hundreds of openly collective genomes of Covid-19.<li> [2020](Nextstrain (2020) <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">https://nextstrain.org/</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33815459">33815459</a> <b>(4)</b> Genomic Variations in SARS-CoV-2 Genomes From Gujarat: Underlying Role of Variants in Disease Epidemiology<ul><li> This cluster is visualized at <ext-link ext-link-type="uri" xlink:href="http://covid.gbrc.org.in/<b>nextstrain</b>.php">http://covid.gbrc.org.in/<b>nextstrain</b>.php</ext-link> using the <b>Nextstrain</b> virus genome analysis pipeline.<li> In this study, we have reported a distinct cluster of coronavirus under 20A clade of <b>Nextstrain</b> and proposed it as 20D as per the next strain analysis or GHJ as per the GISAID analysis, predominantly present in the Gujarat genomes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33758785">33758785</a> <b>(1)</b> Pan-India novel coronavirus SARS-CoV-2 genomics and global diversity analysis in spike protein<ul><li> Clades analysis of the SARS-CoV-2 genomes from India classified under 7 clusters a identified by GISAID and <b>Nextstrain</b> consortium as: G, GH, GR, S, L, V and O.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33735949">33735949</a> <b>(5)</b> OverCOVID: an integrative web portal for SARS-CoV-2 bioinformatics resources<ul><li>GISAID, <b>nextstrain</b>, covidex, GESS, pangolin, COVID-19 genotyping tool, SARS-CoV-2 alignment screen, UCSC SARS-CoV-2 genome browser, 2019nCoVR, CoV2ID, COVID-19 data portal, NCBI nucleotide sequences, SARS-CoV-2 data hub, ViPR SARS-CoV-2, WOLFRAM.<li><b>Nextstrain</b>, covidex, pangolin COVID-19 lineage assigner, 2019nCoVR, SARS-CoV-2 alignment screen, 2019nCoVR, CoV-GLUE, coronavirus typing tool, phylomeDB coronavirus phylomes, SARS-CoV-2 analysis workflow.<li> Users can also find a table showing the clades and corresponding marker variations provided by the <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33740028">33740028</a> <b>(9)</b> SARS-CoV-2 outbreak in a tri-national urban area is dominated by a B.1 lineage variant linked to a mass gathering event<ul><li>The virus genome of <italic>Patient 1</italic> carries a synonymous mutation at C313T in <italic>ORF1ab</italic>, which is found in samples from Israel, Hungary, Japan, USA, Argentina, Greece, India, Brazil, Morocco, and Netherlands among others (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>), all sharing an unsampled hypothetical common ancestor that emerged around February 25<sup>th</sup> (CI February 23-26<sup>th</sup>).<li> as of August 12<sup>th</sup> 2020), filtering for genomes belonging <b>nextstrain</b> emerging clade 20A/15324 and sampled until March 23<sup>rd</sup> (N = 2,856), we find that the C15324T mutation has subsequently been observed in other countries but remains most prevalent in Switzerland (N<sub>GISAID</sub> = 57/213, 26.8%; N<sub>GISAID+this study</sub> = 386/675, 57.2%; first genome 42173111 from March 2<sup>nd</sup>) (<bold><xref ref-type="fig" rid="ppat.1009374.g004"><u>Fig 4D</u></xref></bold> and <xref ref-type="supplementary-material" rid="ppat.1009374.s009"><u>S3 Table</u></xref>).<li> Metadata and consensus sequences of the Basel samples and global data from GISAID were combined for further joint analysis, which were performed using custom R scripts and the <b>nextstrain</b> command line interface analysis pipeline v.2.0.0 (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>) and augur v.8.0.0 [<xref rid="ppat.1009374.ref050" ref-type="bibr"><u>50</u></xref>].<li> Non-human derived viruses as well as sequences with other ambiguous characters (Us), as well as those from cruise ships, and duplicated sequences defined by the <b>nextstrain</b> team as of June 24<sup>th</sup> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/">https://github.com/<b>nextstrain</b>/ncov/</ext-link>) were excluded using <italic>augur filter</italic> [<xref rid="ppat.1009374.ref050" ref-type="bibr"><u>50</u></xref>] resulting in 2,485 sequences for the final phylogenetic analysis dataset.<li> We used Nextclade version 0.3.5 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org</ext-link>) to infer genomic mutations and filtered for sequences that contained C15324T.<li> [50](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [50](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33753180">33753180</a> <b>(3)</b> Novel indel mutation in the N gene of SARS-CoV-2 clinical samples that were diagnosed positive in a commercial RT-PCR assay<ul><li>To identify if this insertion/deletion (indel) mutation has been previously reported, we further investigated publicly available genomes from the GISAID (<ext-link ext-link-type="uri" xlink:href="http://gisaid.org" id="intr0005">http://gisaid.org</ext-link>), <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org" id="intr0010">www.<b>nextstrain</b>.org</ext-link>), and NCBI virus repository (<ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/labs/virus" id="intr0015">http://www.ncbi.nlm.nih.gov/labs/virus</ext-link>) as of January 15, 2021.<li> Through the analysis of the publicly available viral genomes from the <b>Nextstrain</b> SARS-CoV-2 resources, we also identified that the novel 12 nt deletions at positions 28890&#x2013;28901 are located at variable region of the viral N gene (<xref rid="bib0040" ref-type="bibr"><b>Nextstrain</b> SARS-CoV-2 resources, 2021</xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33735881">33735881</a> <b>(3)</b> SARS-CoV-2 Lineages and Sub-Lineages Circulating Worldwide: A Dynamic Overview<ul><li>This global effort was also joined by <b>Nextstrain</b> [<xref rid="B20" ref-type="bibr"><u>20</u></xref> ], an open-source project started in 2018 which integrates the SARS-CoV-2 genomic data updated daily (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) to interactively visualize the phylogeny of this emerging viral pathogen to better understand its dispersion dynamic and improve outbreak response [<xref rid="B21" ref-type="bibr"><u>21</u></xref> , <xref rid="B22" ref-type="bibr"><u>22</u></xref> , <xref rid="B23" ref-type="bibr"><u>23</u></xref> ].<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>          </person-group>          <article-title>Phylodynamic analysis | 176 genomes | 2020 Mar 6 [Internet]</article-title>          <source>Virological</source>          <year>2020</year>          <month>1</month>          <comment>[cited 2020 Dec 11]. Available from: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/phylodynamic-analysis-176-genomes-6-mar-2020/356">https://virological.org/t/phylodynamic-analysis-176-genomes-6-mar-2020/356</ext-link></comment>        ) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32913002">32913002</a>) [23](          <person-group person-group-type="author">            <name>              <surname>Su</surname>              <given-names>YC</given-names>            </name>            <name>              <surname>Anderson</surname>              <given-names>DE</given-names>            </name>            <name>              <surname>Young</surname>              <given-names>BE</given-names>            </name>            <name>              <surname>Zhu</surname>              <given-names>F</given-names>            </name>            <name>              <surname>Linster</surname>              <given-names>M</given-names>            </name>            <name>              <surname>Kalimuddin</surname>              <given-names>S</given-names>            </name>            <etal/>          </person-group>          <article-title>Discovery of a 382-nt deletion during the early evolution of SARS-CoV-2</article-title>          <source>bioRxiv</source>          <year>2020</year>          <month>3</month>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33725111">33725111</a> <b>(6)</b> Whole genome analysis of more than 10 000 SARS-CoV-2 virus unveils global genetic diversity and target region of NSP6<ul><li>Moreover, to understand the spread and evolution of SARS-CoV-2, <b>Nextstrain</b> [<xref rid="ref35" ref-type="bibr"><u>35</u></xref>] is used to create phylogenetic tree.<li> <b>Nextstrain</b> is a collection of open-source tools, which helps to understand pathogen spread and evolution, especially during outbreak.<li> <b>Nextstrain</b> is one of the very recent tools through which evolution of large number of virus genomes can be visualised properly and meaningfully.<li> There are a sequence of tools in <b>Nextstrain</b> that perform phylodynamic analysis [<xref rid="ref36" ref-type="bibr"><u>36</u></xref>] that includes subsampling, alignment, phylogenetic inference, temporal dating of ancestral nodes and discrete trait geographic reconstruction and inference of the most likely transmission events.<li> The spread and evolution of virus genomes can be visualised at <b>nextstrain</b>.org using auspice.<li> Furthermore, SNPs are visualised using protein structures and to show the genetic diversity, phylogenetic analysis has been performed using <b>Nextstrain</b>.<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [36](<person-group person-group-type="author"><string-name><surname>Volz</surname> &#xA0;<given-names>EM</given-names></string-name>, <string-name><surname>Koelle</surname> &#xA0;<given-names>K</given-names></string-name>, <string-name><surname>Bedford</surname> &#xA0;<given-names>T</given-names></string-name></person-group>. <article-title>Viral phylodynamics</article-title>. <source>PLoS Comput Biol</source> &#xA0;<year>2013</year>; <volume>9</volume>(<issue>3</issue>):e1002947.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33815467">33815467</a> <b>(6)</b> Initial Insights Into the Genetic Epidemiology of SARS-CoV-2 Isolates From Kerala Suggest Local Spread From Limited Introductions<ul><li> The patient had traveled from Wuhan, China (Yadav et al., <xref rid="B54" ref-type="bibr"><u>2020</u></xref>), the genomic identity of which mapped to the <b>Nextstrain</b> clade B of SARS-CoV-2 (Somasundaram et al., <xref rid="B45" ref-type="bibr"><u>2020</u></xref>).<li> The early months of the pandemic (March and April) was dominated by the clade I/A3i, a distinct phylogenetic cluster reported from the genomes in India, while by late April, a shift in clade prevalence was observed as most states showed an increased representation of the <b>Nextstrain</b> clade A2a (Banu et al., <xref rid="B2" ref-type="bibr"><u>2020</u></xref>).<li>Phylogenetic analysis was performed according to the pipeline provided by <b>Nextstrain</b> (Hadfield et al., <xref rid="B12" ref-type="bibr"><u>2018</u></xref>).<li> The phylogenetic tree was constructed and refined to a molecular clock phylogeny using the Augur framework provided by <b>Nextstrain</b> and was visualized using Auspice.<li> All 113 genomes from this study were found to cluster under the globally predominant <b>Nextstrain</b> clade A2a (GISAID clade G and GH).<li> In contrast, one of the previous genomes available from Kerala (EPI_ISL_413523, submitted by National Institute of Virology, Pune, India), which is also one of the first SARS-CoV-2 genomes sequenced in India, belongs to the <b>Nextstrain</b> clade B (Yadav et al., <xref rid="B54" ref-type="bibr"><u>2020</u></xref>).<li> [2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32242873">32242873</a>) [2020](<person-group person-group-type="author"><name><surname>Somasundaram</surname><given-names>K.</given-names></name><name><surname>Mondal</surname><given-names>M.</given-names></name><name><surname>Lawarde</surname><given-names>A.</given-names></name></person-group> (<year>2020</year>). <article-title>Genomics of Indian SARS-CoV-2: implications in genetic diversity, possible origin and spread of virus</article-title>. <source>medRxiv</source> 2020.04.25.20079475. <pub-id pub-id-type="doi">10.1101/2020.04.25.20079475</pub-id>) [2020](<person-group person-group-type="author"><name><surname>Banu</surname><given-names>S.</given-names></name><name><surname>Jolly</surname><given-names>B.</given-names></name><name><surname>Mukherjee</surname><given-names>P.</given-names></name><name><surname>Singh</surname><given-names>P.</given-names></name><name><surname>Khan</surname><given-names>S.</given-names></name><name><surname>Zaveri</surname><given-names>L.</given-names></name><etal/></person-group>. (<year>2020</year>). <article-title>A distinct phylogenetic cluster of Indian SARS-CoV-2 isolates</article-title>. <source>bioRxiv</source> 2020.05.31.126136. <pub-id pub-id-type="doi">10.1101/2020.05.31.126136</pub-id>) [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32242873">32242873</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33731472">33731472</a> <b>(1)</b> Machine Learning Prediction and Experimental Validation of Antigenic Drift in H3 Influenza A Viruses in Swine<ul><li> Linear models can mitigate issues of collinearity by implementing approaches such as ridge regression in antigen bridges (<xref rid="B24" ref-type="bibr"><u>24</u></xref>) or lasso regression used by <b>Nextstrain</b> (<xref rid="B19" ref-type="bibr"><u>19</u></xref>, <xref rid="B31" ref-type="bibr"><u>31</u></xref>), but these approaches may result in models that are more difficult to interpret biologically.<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23820391">23820391</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26951657">26951657</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31385805">31385805</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33758855">33758855</a> <b>(1)</b> DNA spike-ins enable confident interpretation of SARS-CoV-2 genomic data from amplicon-based sequencing<ul><li> We used the sarscov2_<b>nextstrain</b> workflow to perform a Massachusetts-weighted subsampling of samples from 1 November 2020 &#x2013; 1 November 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33727249">33727249</a> <b>(2)</b> Phylo-geo-network and haplogroup analysis of 611 novel coronavirus (SARS-CoV-2) genomes from India<ul><li>Subtype analysis-SARS Cov 2 <b>nextstrain</b> (<xref rid="bib27" ref-type="bibr"><u>Hadfield et al, 2018</u></xref>)</th>              </tr>            </thead>            <tbody>              <tr>                <td align="left" rowspan="1" colspan="1">Proposed.<li> Furthermore, the viral subtypes of the studied genomes from the Indian population was checked using &#x201C;SARS Cov 2 <b>Nextstrain</b>&#x201D; classification model of Covidex (<ext-link ext-link-type="uri" xlink:href="https://cacciabue.shinyapps.io/shiny2/">https://cacciabue.shinyapps.io/shiny2/</ext-link>), a Web-based subtyping tool (<xref rid="bib2" ref-type="bibr"><u>Cacciabue et al, 2020</u></xref><italic>Preprint</italic>).<li> [Hadfield et al, 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Cacciabue et al, 2020](<person-group person-group-type="author"><string-name><surname>Cacciabue</surname><given-names>M</given-names></string-name>, <string-name><surname>Aguilera</surname><given-names>P</given-names></string-name>, <string-name><surname>Gismondi</surname><given-names>MI</given-names></string-name>, <string-name><surname>Taboga</surname><given-names>O</given-names></string-name></person-group> (<year>2020</year>) <article-title>Covidex: An ultrafast and accurate tool for virus subtyping</article-title>. <source><italic toggle="yes">BioRxiv</italic></source><pub-id pub-id-type="doi">10.1101/2020.08.21.261347</pub-id><comment>(Preprint posted August 21, 2020)</comment>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34611501">34611501</a> <b>(7)</b> Genomic characterization and phylogenetic analysis of the first SARS-CoV-2 variants introduced in Lebanon<ul><li> The resulting alignment was used in masking terminal regions and gaps with <b>Nextstrain</b>&#x2019;s custom Python script (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov/tree/master/scripts">https://github.com/<b>nextstrain</b>/ncov/tree/master/scripts</uri>).<li>We assigned genetic lineages based on three commonly used systems including the recently proposed dynamic classification using Pangolin tool v1.1.14 (<uri xlink:href="https://github.com/hCoV-2019/pangolin">https://github.com/hCoV-2019/pangolin</uri>) (<xref rid="ref-21" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), the <b>Nextstrain</b> classification and GISAID&#x2019;s internal classification.<li>The SARS-CoV-2 isolates recovered from Lebanon clustered, and according to <b>Nextstrain</b>&#x2019;s classification, in three distinct clades namely: 19A (93% bootstrap support), 20A (81% bootstrap support) and 20B (98% bootstrap support) (<xref ref-type="fig" rid="fig-2">Fig.<li> <b>Nextstrain</b> clades were defined through different clade colors with light blue denoting the 19A clade, dark blue representing the 20B clade and gold designating the 20A clade.<li> Detailed analysis of the European sub-clusters by <b>Nextstrain</b> (<xref rid="ref-9" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) revealed that the UK outbreak was largely rooted in B.1 and B.1.1 lineages which was in agreement with the analysis from GISAID&#x2019;s EpiCoV<sup>&#x2122;</sup> database (<xref rid="ref-23" ref-type="bibr"><u>Shu &amp; McCauley, 2017</u></xref>).<li> [Rambaut et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Holmes</surname>              <given-names>EC</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V</given-names>            </name>            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>JT</given-names>            </name>            <name>              <surname>Ruis</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Du Plessis</surname>              <given-names>L</given-names>            </name>            <name>              <surname>Pybus</surname>              <given-names>OG</given-names>            </name>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 to assist genomic epidemiology</article-title>          <source>bioRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.04.17.046086</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Shu &amp; McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33758853">33758853</a> <b>(5)</b> Recombinant SARS-CoV-2 genomes are currently circulating at low levels<ul><li> Multiple nomenclature schemes have been introduced to describe the major lineages that are currently in circulation, the most widely used of which are the <b>Nextstrain</b><sup><xref rid="R13" ref-type="bibr"><u>13</u></xref></sup>, GISAID<sup><xref rid="R14" ref-type="bibr"><u>14</u></xref></sup>, and Pangolin<sup><xref rid="R15" ref-type="bibr"><u>15</u></xref></sup> systems.<li> While Pangolin provides a fine-scaled classification of viral diversity, <b>Nextstrain</b> and GISAID provide a broad-scale classification scheme, clustering genomes into five or seven clades, respectively.<li> To increase the number of variant sites that contribute to our detection of recombinant genomes, we increased the resolution of the <b>Nextstrain</b> scheme by further dividing their clades into 14 monophyletic clades with at least 50% bootstrap support (<xref rid="F1" ref-type="fig">Fig.<li> Clades were named according to the <b>Nextstrain</b> clade to which they belong, with added suffixes of &#x2018;&#x2212;1&#x2019;, &#x2018;&#x2212;2&#x2019;, etc.<li> to denote clades at finer resolution than those available under the <b>Nextstrain</b> system.<li> [13](<source>Nextstrain: Genomic epidemiology of novel coronavirus.</source><comment>Available at: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link>. (Accessed:</comment><date-in-citation>11th January 2020</date-in-citation><comment>)</comment>) [14](<collab>Global Initiative on Sharing All Influenza Data (GISAID)</collab>. <source>Clade and lineage nomenclature aids in genomic epidemiology studies of active hCoV-19 viruses.</source><comment>Available at: <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>. (Accessed:</comment><date-in-citation>11th January 2020</date-in-citation><comment>)</comment>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33737129">33737129</a> <b>(1)</b> Genomic surveillance of SARS-CoV-2 in the Republic of Congo<ul><li> SARS-CoV-2 genomes were classified into lineages using Phylogenetic Assignment of Named Global Outbreak LINeages (PANGOLIN) (<xref rid="bib0060" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) and viral clades were assigned by Nextclade Beta (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intr0010">https://clades.<b>nextstrain</b>.org/</ext-link>) and <b>Nextstrain</b> (<xref rid="bib0025" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33758901">33758901</a> <b>(4)</b> PCR assay to enhance global surveillance for SARS-CoV-2 variants of concern<ul><li>We analyzed over 400,000 SARS-CoV-2 genomes on GISAID and used custom <b>Nextstrain</b> builds<sup><xref rid="R15" ref-type="bibr"><u>15</u></xref></sup> to identify that a 9 nucleotide deletion in the ORF1a gene at amino acid positions 3675-3677 (ORF1a &#x394;3675-3677) occurs in the B.1.1.7, B.<li> Phylogenetic analysis of a subset of 4,046 SARS-CoV-2 genomes was performed using <b>Nextstrain</b><sup><xref rid="R15" ref-type="bibr"><u>15</u></xref></sup>, downsampled as shown using the &#x201C;global build&#x201D; on 2021-01-22 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33712587">33712587</a> <b>(7)</b> Shotgun transcriptome, spatial omics, and isothermal profiling of SARS-CoV-2 infection reveals unique host responses, viral diversification, and drug interactions<ul><li> We compared the 155 full-length NYPH-WCMC sequences with 46,581 SARS-CoV-2 sequences obtained from a recent GISAID build (downloaded on 6/16/2020) using the <b>Nextstrain</b> pipeline<sup><xref ref-type="bibr" rid="CR7"><u>7</u></xref></sup> (Fig.&#xA0;<xref rid="Fig3" ref-type="fig"><u>3a</u></xref>, see Methods).<li> Branches and nodes are colored according to the <b>Nextstrain</b> clade affiliation.<li>We then used a New York State-focused build of <b>Nextstrain</b> resulting in the incorporation of our 155 genomes with 3541 sequences from GISAID (Fig.&#xA0;<xref rid="Fig3" ref-type="fig"><u>3a</u></xref>).<li> Analyzing worldwide groupings annotated by the <b>Nextstrain</b> database, we found a high proportion (&gt;74%, 116/155) of our assemblies were associated with clade 20C (Fig.&#xA0;<xref rid="Fig3" ref-type="fig"><u>3b</u></xref>), which overall represents 17% of strains in the GISAID build.<li> All sequence filtering, alignments, phylogenetic inference, temporal ordering of sequences, and geographic reconstruction of likely transmission events were done using <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR7"><u>7</u></xref>,<xref ref-type="bibr" rid="CR44"><u>44</u></xref>,<xref ref-type="bibr" rid="CR45"><u>45</u></xref></sup>.<li> <b>Nextstrain</b> was configured to generate a New York State-focused build.<li> Phylogenetic trees were created using <b>Nextstrain</b>&#x2019;s augur as described above, and visualized using the ggtree package in R<sup><xref ref-type="bibr" rid="CR47"><u>47</u></xref></sup>.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [45](Sagulenko, P., Puller, V. &amp; Neher, R. A., TreeTime: Maximum-likelihood phylodynamic analysis. Virus Evol. 10.1093/ve/vex042 (2018).) [47](Yu, L. et al. Rapid colorimetric detection of COVID-19 coronavirus using a reverse transcriptional loop-mediated isothermal amplification (RT-LAMP) diagnostic platform: iLACO. medrxiv 10.1101/2020.02.20.20025874v1 (2020).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33688657">33688657</a> <b>(5)</b> SARS-CoV-2 501Y.V2 (B.1.351) elicits cross-reactive neutralizing antibodies<ul><li> We confirmed clade classification by analyzing our sequenced genomes against a global reference dataset using a custom pipeline based on a local version of <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="R11" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> It performs alignment of genomes in MAFFT (<xref rid="R14" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>), phylogenetic tree inference in IQ-Tree V1.6.9 (<xref rid="R25" ref-type="bibr"><u>Nguyen et al., 2015</u></xref>), tree dating and ancestral state construction and annotation (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Nguyen et al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33594373">33594373</a> <b>(5)</b> Early introductions and community transmission of SARS-CoV-2 variant B.1.1.7 in the United States<ul><li> Additional examples can be found by exploring our full dataset (<ext-link xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5" ext-link-type="uri"><b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5</ext-link>).<li> We visualized this time-scaled maximum-likelihood phylogeny using auspice (<xref rid="R12" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), which can be found on our custom <b>nextstrain</b> page: <ext-link xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5" ext-link-type="uri"><b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper5</ext-link>.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33707530">33707530</a> <b>(1)</b> Bioinformatics-based prediction of conformational epitopes for Enterovirus A71 and Coxsackievirus A16<ul><li> Entropy based on amino acid residues was calculated to perform diversity analysis using Augur package of <b>Nextstrain</b> platform<sup><xref ref-type="bibr" rid="CR74"><u>74</u></xref></sup>.<li> [74](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33768104">33768104</a> <b>(1)</b> Clinical, Serological, Whole Genome Sequence Analyses to Confirm SARS-CoV-2 Reinfection in Patients From Mumbai, India<ul><li> In patients meeting the above criteria, genomic testing revealing differing clades as defined in <b>Nextstrain</b> (<xref rid="B27" ref-type="bibr"><u>27</u></xref>) and GISAID of SARS-CoV-2 between the first and second infection is considered the best evidence of SARS-CoV-2 reinfection.<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33817389">33817389</a> <b>(4)</b> Mink SARS-CoV-2 Infection in Poland &#x2013; Short Communication<ul><li> 14-02-2021, <ext-link ext-link-type="uri" xlink:href="http://https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/)</ext-link>.<li> Phylogenetic analysis indicated that these viruses belong to the 20B <b>Nextstrain</b> clade (<xref ref-type="fig" rid="j_jvetres-2021-0017_fig_001">Fig.<li><b>Nextstrain</b> clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33803490">33803490</a> <b>(2)</b> SARS-CoV-2 Molecular Transmission Clusters and Containment Measures in Ten European Regions during the First Pandemic Wave<ul><li> Those files were used as input in <b>Nextstrain</b>&#x2019;s &#x201C;augur&#x201D; pipeline [<xref rid="B43-life-11-00219" ref-type="bibr"><u>43</u></xref>] after excluding duplicates.<li> Re-rooting, resolving of polytomies, inferring of internal node dates, labeling of internal nodes, pruning of sequences and maximum-likelihood phylodynamics analysis was performed with TreeTime [<xref rid="B50-life-11-00219" ref-type="bibr"><u>50</u></xref>] using the default parameters that are pre-optimized for SARS-CoV-2 by <b>Nextstrain</b>.<li> [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [50](          <person-group person-group-type="author">            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Puller</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>TreeTime: Maximum-likelihood phylodynamic analysis</article-title>          <source>Virus Evol.</source>          <year>2018</year>          <volume>4</volume>          <pub-id pub-id-type="doi">10.1093/ve/vex042</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33758899">33758899</a> <b>(5)</b> Transmission, infectivity, and antibody neutralization of an emerging SARS-CoV-2 variant in California carrying a L452R spike protein mutation<ul><li> A novel variant, subsequently named 20C/L452R according to the <b>NextStrain</b> nomenclature system (<xref rid="R2" ref-type="bibr"><u>Bedford et al., 2021</u></xref>) or B.1.427/B.1.429 according to the Pango system (<xref rid="R42" ref-type="bibr"><u>Rambaut et al., 2020a</u></xref>) (henceforth referred to using the Pango designation to distinguish between the B.1.427 and B.1.429 lineages), was identified in 21.1% (459 of 2,172) of the genomes (<xref rid="SD2" ref-type="supplementary-material"><u>Supplementary Table 1</u></xref>).<li>Bayesian phylogenetic analysis of 1,166 genomes subsampled from a 2,519-genome dataset consisting of the 2,172 California genomes sequenced in this study and 347 representative global genomes (<xref rid="R3" ref-type="bibr"><u>Bedford and Neher, 2020</u></xref>) identified two distinct lineages in clade 20C (<b>Nextstrain</b> designation) associated with the novel variant, B.1.427 and B.1.429 (<xref rid="F2" ref-type="fig"><u>Figure 2B</u></xref>).<li> <xref rid="R47" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>) (accession numbers in <xref rid="SD2" ref-type="supplementary-material"><u>Supplementary Table 1</u></xref>) and can be visualized in <b>NextStrain</b>.<li>High-quality SARS-CoV-2 genomes (n=2,519, 2,172 generated in the current study and 347 used as representative global genomes) were downloaded from the Global Initiative on Sharing of All Influenza Data (GISAID) database and processed using the <b>NextStrain</b> bioinformatics pipeline Augur using IQTREE v1.6.<li> The resulting tree was visualized in the <b>NextStrain</b> web application Auspice and in Geneious v11.1.5 (<xref rid="R27" ref-type="bibr"><u>Kearse et al., 2012</u></xref>).<li> [Bedford et al., 2021](<name><surname>Bedford</surname><given-names>T.</given-names></name>, <name><surname>Hodcroft</surname><given-names>E.B.</given-names></name>, and <name><surname>Neher</surname><given-names>R.A.</given-names></name> (<year>2021</year>). <source>Updated Nextstrain SARS-CoV-2 clade naming strategy (NextStrain)</source>.) [Rambaut et al., 2020a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Bedford and Neher, 2020](<name><surname>Bedford</surname><given-names>T.</given-names></name>, and <name><surname>Neher</surname><given-names>R.</given-names></name> (<year>2020</year>). <article-title>A Getting Started Guide to the Genomic Epidemiology of SARS-CoV-2</article-title>. In <source>Nextstrain documentation</source> (<collab><ext-link ext-link-type="uri" xlink:href="http://Nextstrain.org">Nextstrain.org</ext-link></collab>).) [Shu and McCauley, 2017](<name><surname>Shu</surname><given-names>Y.</given-names></name>, and <name><surname>McCauley</surname><given-names>J.</given-names></name> (<year>2017</year>). <article-title>GISAID: Global initiative on sharing all influenza data - from vision to reality</article-title>. <source>Euro Surveill</source><volume>22</volume>.) [Kearse et al., 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22543367">22543367</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33842111">33842111</a> <b>(3)</b> A Case of Moderately Severe COVID-19 in a Healthcare Worker in Russia: Virus Isolation and Full Genome Sequencing<ul><li> [<xref rid="REF8" ref-type="bibr"><u>8</u></xref>] and implemented in Pangolin software), and 20B clade (<b>Nextstrain</b> nomenclature) (Figure <xref ref-type="fig" rid="FIG3"><u>3</u></xref>), suggesting that the studied isolate is well adapted for spreading among the human population.<li>The phylogram was provided by the Nextclade tool of the <b>Nextstrain</b> web-service (<ext-link ext-link-type="uri" xlink:href="clades.<b>nextstrain</b>.org">clades.<b>nextstrain</b>.org</ext-link>).<li> Of particular interest among detected amino acid substitutions is spike D614G mutation, which is present in almost all genomes of viral strains belonging to 20A, 20B, and 20C clades (<b>Nextstrain</b> nomenclature).<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33688665">33688665</a> <b>(5)</b> SARS-CoV-2 transmission in intercollegiate athletics not fully mitigated with daily antigen testing<ul><li>Phylogenetic analysis was completed using tools implemented in <b>Nextstrain</b> custom builds (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>)[<xref rid="R18" ref-type="bibr"><u>18</u></xref>,<xref rid="R19" ref-type="bibr"><u>19</u></xref>].<li> Time-resolved and divergence phylogenetic trees were built using the standard <b>Nextstrain</b> tools and scripts[<xref rid="R18" ref-type="bibr"><u>18</u></xref>,<xref rid="R19" ref-type="bibr"><u>19</u></xref>].<li> We found that 24 (92%) of these 26 genomic sequences cluster tightly in the <b>Nextstrain</b> 20A clade on a time-resolved tree and are separated by 0&#x2013;2 fixed consensus nucleotide differences (<xref rid="F3" ref-type="fig"><u>Figure 3</u></xref>).<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33807682">33807682</a> <b>(1)</b> CNBP Binds and Unfolds In Vitro G-Quadruplexes Formed in the SARS-CoV-2 Positive and Negative Genome Strands<ul><li>The nucleotide and amino acid variations of SARS-CoV-2 genome and their associated data were searched by using the <b>Nextstrain</b> tool for analysis and visualization of 325,005 SARS-CoV-2 full genomic sequences sampled by different laboratories worldwide between 10 January 2020 and 11 January 2021, available in GISAID database (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>, accessed on 11 January 2021) [<xref rid="B52-ijms-22-02614" ref-type="bibr"><u>52</u></xref>].<li> [52](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33807854">33807854</a> <b>(3)</b> Population-Predicted MHC Class II Epitope Presentation of SARS-CoV-2 Structural Proteins Correlates to the Case Fatality Rates of COVID-19 in Different Countries<ul><li> (b) secondary effects that may also impact CFR, like (i) overwhelming the health system capacity as observed in some countries due to overshooting numbers of patients, (ii) varying strain heterogeneity (see, e.g., current SARS-CoV-2 variation at <uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>; 4992 genomes sampled December 2019 till the end of September 2020 according to <b>Nextstrain</b> database [<xref rid="B23-ijms-22-02630" ref-type="bibr"><u>23</u></xref>]&#x2014;accessed 15 October 2020) and (iii) different protective measures are likely not yet compromising the SARS-CoV-2 associated CFR at this comparatively early time point.<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Population-Predicted MHC Class II Epitope Presentation of SARS-CoV-2 Structural Proteins Correlates to the Case Fatality Rates of COVID-19 in Different Countries<ul><li>Region used are the ones defined in <b>Nextstrain</b>/flu (<xref rid="msab065-B6" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33661076">33661076</a> <b>(27)</b> Open science, COVID-19, and the news: Exploring controversies in the circulation of early SARS-CoV-2 genomic epidemiology research<ul><li> These sites included <ext-link ext-link-type="uri" xlink:href="http://Virological.org">Virological.org</ext-link>, <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link>, several investigators&#x2019; websites and blogs, select GitHub repositories, Twitter, preprint websites, and the COVID-19 Open Research Dataset community.<li>In this case, we consider coverage of SARS-CoV-2 and the <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> project in <italic>USA Today</italic>.<li> Established in 2018, <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> is housed at the University of Washington in Seattle (<xref rid="R25" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), and attracted media attention early in the COVID-19 pandemic.<li> (<xref rid="R47" ref-type="bibr"><b>NextStrain</b>.org, n.d.<li>The <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> team was credited with generating a range of novel insights during the early COVID-19 pandemic, including the early discovery of community transmission in the United States (<xref rid="R8" ref-type="bibr"><u>Bedford, 2020</u></xref>).<li> Investigators associated with <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> regularly disseminate findings through peer-reviewed papers, but also through non-peer reviewed publications such as blog posts.<li> One such blog post by Trevor Bedford, a <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link>-affiliated investigator, was posted on 2 March 2020, and written in lay terminology (<xref rid="R8" ref-type="bibr"><u>Bedford, 2020</u></xref>).<li> a claim made without clear citation but implicitly substantiated by <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> (<xref rid="R64" ref-type="bibr"><u>Weise, 2020</u></xref>).<li> The information is uploaded to a website called <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> that shows how the virus is migrating and splitting into similar but new subtypes.<li> First, it shows that using platforms such as blogs, tools like <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link>, and other public-facing venues to share early genomic epidemiology results can enable reporters to make unsubstantiated claims.<li> efforts to educate the public via open platforms such as <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> can contribute to, rather than ameliorate, confusion about the meaning of genomic pathogen data.<li> Further, these claims not only circulated via preprints, but also jumped from <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> into the mainstream press.<li> The comment was a secondary analysis of published studies and <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> visualisations.<li> (2020)</xref> and fallout from the paper, the effects of which also appear in the <ext-link ext-link-type="uri" xlink:href="http://<b>NextStrain</b>.org"><b>NextStrain</b>.org</ext-link> and &#x2018;mutant virus&#x2019; cases.<li> [Hadfield et al., 2018](<name><surname>Hadfield</surname><given-names>J</given-names></name>, <name><surname>Megill</surname><given-names>C</given-names></name>, <name><surname>Bell</surname><given-names>SM</given-names></name>, <name><surname>Huddleston</surname><given-names>J</given-names></name>, <name><surname>Potter</surname><given-names>B</given-names></name>, <name><surname>Callender</surname><given-names>C</given-names></name>, <name><surname>Sagulenko</surname><given-names>P</given-names></name>, <name><surname>Bedford</surname><given-names>T</given-names></name>, &amp; <name><surname>Neher</surname><given-names>RA</given-names></name> (<year>2018</year>). <article-title>Nextstrain: Real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics (Oxford, England)</source>, <volume>34</volume>(<issue>23</issue>), <fpage>4121</fpage>&#x2013;<lpage>4123</lpage>. <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>) [Bedford, 2020](<name><surname>Bedford</surname><given-names>T</given-names></name> (<year>2020</year>, <month>3</month><day>2</day>). <article-title>Cryptic transmission of novel coronavirus revealed by genomic epidemiology</article-title>. <source>Bedford Lab Blog</source>. <comment><ext-link ext-link-type="uri" xlink:href="https://bedford.io/blog/ncov-cryptic-transmission/">https://bedford.io/blog/ncov-cryptic-transmission/</ext-link>.</comment>) [Bedford, 2020](<name><surname>Bedford</surname><given-names>T</given-names></name> (<year>2020</year>, <month>3</month><day>2</day>). <article-title>Cryptic transmission of novel coronavirus revealed by genomic epidemiology</article-title>. <source>Bedford Lab Blog</source>. <comment><ext-link ext-link-type="uri" xlink:href="https://bedford.io/blog/ncov-cryptic-transmission/">https://bedford.io/blog/ncov-cryptic-transmission/</ext-link>.</comment>) [Weise, 2020](<name><surname>Weise</surname><given-names>E</given-names></name> (<year>2020</year>, <month>3</month><day>27</day>). <article-title>8 strains of the coronavirus are circling the globe. Here&#x2019;s what clues they&#x2019;re giving scientists</article-title>. <source>USA Today</source>. <comment><ext-link ext-link-type="uri" xlink:href="https://www.usatoday.com/story/news/nation/2020/03/27/scientists-track-coronavirus-strains-mutation/5080571002/">https://www.usatoday.com/story/news/nation/2020/03/27/scientists-track-coronavirus-strains-mutation/5080571002/</ext-link></comment>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33662015">33662015</a> <b>(3)</b> Genomic characterization and evolution of SARS-CoV-2 of a Canadian population<ul><li> In the tree, 129 viruses from a Canadian population were dispersed across 9 different SARS-CoV-2 lineages, including A1a, A2a, A3, A6, A7, B, B1, B2, and B4 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=clade_membership">https://<b>nextstrain</b>.org/ncov/global?c=clade_membership</ext-link>) (<bold><xref ref-type="fig" rid="pone.0247799.g003"><u>Fig 3</u></xref></bold>).<li> The clades were defined by referring to the classification of <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://nextrain.org/">https://nextrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33748571">33748571</a> <b>(1)</b> SARS-CoV-2 Genome from the Khyber Pakhtunkhwa Province of Pakistan<ul><li> however,its effect needs to be elucidated for a better understanding of immunogenicity.In January 2020, clades 19A/L/V/O (<b>Nextstrain</b>/GISAID nomenclatures)were more widespread in Europe than the 20A/G clades; however, smallnumbers of sequenced viral genomes and travel history in Europe duringthe early pandemic days may affect these findings.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33667722">33667722</a> <b>(1)</b> Different selection dynamics of S and RdRp between SARS-CoV-2 genomes with and without the dominant mutations<ul><li> As a result, the high amount of frequently updated data on viral genomes on databases such as GISAID (<xref rid="bb0020" ref-type="bibr"><u>Elbe and Buckland-Merrett, 2017</u></xref>) and <b>NextStrain</b> (<xref rid="bb0040" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) provides researchers with invaluable resources to track the evolution of the virus.<li> [Elbe and Buckland-Merrett, 2017](          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G.</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Global Chall.</source>          <volume>1</volume>          <year>2017</year>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33501446">33501446</a> <b>(2)</b> SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma<ul><li>We, and others, recently described a new SARS-CoV-2 lineage in South Africa, defined as <b>Nextstrain</b> clade 20H/501Y.V2 (PANGOLin lineage B.1.351)<sup><xref rid="R12" ref-type="bibr"><u>12</u></xref></sup>.<li>The closed prefusion SARS-CoV-2 spike protein from <b>Nextstrain</b><sup><xref rid="R21" ref-type="bibr"><u>21</u></xref></sup> clade 501Y.V2 (PANGOLin lineage B.1.351, (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</ext-link>) was modelled on PDBID 6VXX<sup><xref rid="R22" ref-type="bibr"><u>22</u></xref></sup>, while a model of the RBD was averaged using neutralizing domain-bound structures compared to the ACE2-bound PDBID 6LZG<sup><xref rid="R23" ref-type="bibr"><u>23</u></xref></sup>.<li> [12](<name><surname>Tegally</surname><given-names>H.</given-names></name>, <etal/><article-title>Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa</article-title>. <source>medRxiv</source> (<year>2020</year>).) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33306958">33306958</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32275855">32275855</a>) </ul></p>
<p><u>No PMID</u> <b>(2)</b> Influenza B, C and D Viruses (<italic>Orthomyxoviridae</italic>)<ul><li> The continued evolution of IBV HA genes is presented on <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="interref3">https://<b>nextstrain</b>.org</ext-link>.</ul></p>
<p><u>No PMID</u> <b>(5)</b> Database and Analytical Resources for Viral Research Community<ul><li>List of databases and webtools</p></caption><alt-text id="at0025">Table 1</alt-text><table frame="hsides" rules="groups"><thead><tr><th>Category</th><th>Name</th><th>Types of data/services</th><th>Weblink</th></tr></thead><tbody><tr><td><italic>Databases</italic></td><td>GenBank</td><td>Gene and genome sequences</td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/genbank/" id="interref1">https://www.ncbi.nlm.nih.gov/genbank/</ext-link></td></tr><tr><td/><td>UniProt</td><td>Protein sequences</td><td><ext-link ext-link-type="uri" xlink:href="https://www.uniprot.org" id="interref2">https://www.uniprot.org</ext-link></td></tr><tr><td/><td>RefSeq</td><td>Curated genome and protein sequences</td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/refseq/" id="interref3">https://www.ncbi.nlm.nih.gov/refseq/</ext-link></td></tr><tr><td/><td>Protein Data Bank (PDB)</td><td>3D protein structures</td><td><ext-link ext-link-type="uri" xlink:href="http://www.pdb.org" id="interref4">www.pdb.org</ext-link></td></tr><tr><td/><td>Immune Epitope Database (IEDB)</td><td>Experimental data on B cell and T cell epitopes</td><td><ext-link ext-link-type="uri" xlink:href="http://www.iedb.org" id="interref5">www.iedb.org</ext-link></td></tr><tr><td/><td>Virus Particle Explorer (VIPERdb)</td><td>Structures of viruses with icosahedral virions</td><td><ext-link ext-link-type="uri" xlink:href="http://viperdb.scripps.edu/" id="interref6">http://viperdb.scripps.edu/</ext-link></td></tr><tr><td/><td>Viral Pathogen Resource (ViPR)</td><td>Comprehensive collection of multiple data types on high priority human pathogenic and related viruses and an integrated suite of analytical and visualization capabilities</td><td><ext-link ext-link-type="uri" xlink:href="https://www.viprbrc.org/" id="interref7">https://www.viprbrc.org/</ext-link></td></tr><tr><td/><td>Influenza Research Database (IRD)</td><td>Comprehensive collection of influenza virus-related data and an integrated suite of analytical and visualization capabilities</td><td><ext-link ext-link-type="uri" xlink:href="https://www.fludb.org" id="interref8">https://www.fludb.org</ext-link></td></tr><tr><td/><td>Global Initiative on Sharing All Influenza Data (GISAID)</td><td>Access-controlled resource of influenza sequence information and related epidemiological data</td><td><ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org" id="interref9">https://www.gisaid.org</ext-link></td></tr><tr><td/><td>FluNet</td><td>Global web-based influenza surveillance data collection</td><td><ext-link ext-link-type="uri" xlink:href="https://www.who.int/influenza/gisrs_laboratory/flunet/en/" id="interref10">https://www.who.int/influenza/gisrs_laboratory/flunet/en/</ext-link></td></tr><tr><td/><td>Influenza Virus Resource</td><td>Influenza genomic and protein sequences</td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/genomes/FLU/Database/nph-select.cgi?go=database" id="interref11">https://www.ncbi.nlm.nih.gov/genomes/FLU/Database/nph-select.cgi?go=database</ext-link></td></tr><tr><td/><td>Hepatitis B Virus Database (HBVDb)</td><td>Nucleotide and protein sequence information for analysis of drug resistance profiling</td><td><ext-link ext-link-type="uri" xlink:href="https://hbvdb.ibcp.fr/HBVdb/" id="interref12">https://hbvdb.ibcp.fr/HBVdb/</ext-link></td></tr><tr><td/><td>Human Immunodeficiency Virus (HIV)</td><td>Genome and protein sequences and immunological epitopes</td><td><ext-link ext-link-type="uri" xlink:href="https://www.hiv.lanl.gov/content/index" id="interref13">https://www.hiv.lanl.gov/content/index</ext-link></td></tr><tr><td/><td>ViralZone</td><td>highly curated and extensive virus knowledgebase</td><td><ext-link ext-link-type="uri" xlink:href="https://viralzone.expasy.org" id="interref14">https://viralzone.expasy.org</ext-link></td></tr><tr><td/><td>Prokaryotic Virus Ortholog Groups (pVOGs)</td><td>Bacteriophage protein orthology information</td><td><ext-link ext-link-type="uri" xlink:href="http://dmk-brain.ecn.uiowa.edu/VOG/" id="interref15">http://dmk-brain.ecn.uiowa.edu/VOG/</ext-link></td></tr><tr><td/><td>Plant Viruses Online</td><td>Plant virus gene and protein sequences and structures</td><td><ext-link ext-link-type="uri" xlink:href="http://sdb.im.ac.cn/vide/refs.htm" id="interref16">http://sdb.im.ac.cn/vide/refs.htm</ext-link></td></tr><tr><td><italic>Webtools</italic></td><td>IDSeq</td><td>Real-time pathogen detection from metagenomes</td><td><ext-link ext-link-type="uri" xlink:href="https://idseq.net" id="interref17">https://idseq.net</ext-link></td></tr><tr><td/><td>Virome</td><td>Viral detection from environmental metagenomes</td><td><ext-link ext-link-type="uri" xlink:href="http://virome.dbi.udel.edu" id="interref18">http://virome.dbi.udel.edu</ext-link></td></tr><tr><td/><td>VirusDetect</td><td>viral detection from small RNA datasets using both <italic>de novo</italic> and reference-based assemblies</td><td><ext-link ext-link-type="uri" xlink:href="http://virusdetect.feilab.net/cgi-bin/virusdetect/index.cgi" id="interref19">http://virusdetect.feilab.net/cgi-bin/virusdetect/index.cgi</ext-link></td></tr><tr><td/><td>Viral Genome ORF Reader (VIGOR)</td><td>Homology-driven viral gene prediction and genome annotation</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/JCVenterInstitute/VIGOR4" id="interref20">https://github.com/JCVenterInstitute/VIGOR4</ext-link></td></tr><tr><td/><td>STRING-Viruses</td><td>Assessment of viral-host protein-protein interactions using visualization tools such as Cytoscape</td><td><ext-link ext-link-type="uri" xlink:href="http://viruses.string-db.org" id="interref21">http://viruses.string-db.org</ext-link></td></tr><tr><td/><td><b>NextStrain</b>.<li>List of databases and webtools</p></caption><alt-text id="at0025">Table 1</alt-text><table frame="hsides" rules="groups"><thead><tr><th>Category</th><th>Name</th><th>Types of data/services</th><th>Weblink</th></tr></thead><tbody><tr><td><italic>Databases</italic></td><td>GenBank</td><td>Gene and genome sequences</td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/genbank/" id="interref1">https://www.ncbi.nlm.nih.gov/genbank/</ext-link></td></tr><tr><td/><td>UniProt</td><td>Protein sequences</td><td><ext-link ext-link-type="uri" xlink:href="https://www.uniprot.org" id="interref2">https://www.uniprot.org</ext-link></td></tr><tr><td/><td>RefSeq</td><td>Curated genome and protein sequences</td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/refseq/" id="interref3">https://www.ncbi.nlm.nih.gov/refseq/</ext-link></td></tr><tr><td/><td>Protein Data Bank (PDB)</td><td>3D protein structures</td><td><ext-link ext-link-type="uri" xlink:href="http://www.pdb.org" id="interref4">www.pdb.org</ext-link></td></tr><tr><td/><td>Immune Epitope Database (IEDB)</td><td>Experimental data on B cell and T cell epitopes</td><td><ext-link ext-link-type="uri" xlink:href="http://www.iedb.org" id="interref5">www.iedb.org</ext-link></td></tr><tr><td/><td>Virus Particle Explorer (VIPERdb)</td><td>Structures of viruses with icosahedral virions</td><td><ext-link ext-link-type="uri" xlink:href="http://viperdb.scripps.edu/" id="interref6">http://viperdb.scripps.edu/</ext-link></td></tr><tr><td/><td>Viral Pathogen Resource (ViPR)</td><td>Comprehensive collection of multiple data types on high priority human pathogenic and related viruses and an integrated suite of analytical and visualization capabilities</td><td><ext-link ext-link-type="uri" xlink:href="https://www.viprbrc.org/" id="interref7">https://www.viprbrc.org/</ext-link></td></tr><tr><td/><td>Influenza Research Database (IRD)</td><td>Comprehensive collection of influenza virus-related data and an integrated suite of analytical and visualization capabilities</td><td><ext-link ext-link-type="uri" xlink:href="https://www.fludb.org" id="interref8">https://www.fludb.org</ext-link></td></tr><tr><td/><td>Global Initiative on Sharing All Influenza Data (GISAID)</td><td>Access-controlled resource of influenza sequence information and related epidemiological data</td><td><ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org" id="interref9">https://www.gisaid.org</ext-link></td></tr><tr><td/><td>FluNet</td><td>Global web-based influenza surveillance data collection</td><td><ext-link ext-link-type="uri" xlink:href="https://www.who.int/influenza/gisrs_laboratory/flunet/en/" id="interref10">https://www.who.int/influenza/gisrs_laboratory/flunet/en/</ext-link></td></tr><tr><td/><td>Influenza Virus Resource</td><td>Influenza genomic and protein sequences</td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/genomes/FLU/Database/nph-select.cgi?go=database" id="interref11">https://www.ncbi.nlm.nih.gov/genomes/FLU/Database/nph-select.cgi?go=database</ext-link></td></tr><tr><td/><td>Hepatitis B Virus Database (HBVDb)</td><td>Nucleotide and protein sequence information for analysis of drug resistance profiling</td><td><ext-link ext-link-type="uri" xlink:href="https://hbvdb.ibcp.fr/HBVdb/" id="interref12">https://hbvdb.ibcp.fr/HBVdb/</ext-link></td></tr><tr><td/><td>Human Immunodeficiency Virus (HIV)</td><td>Genome and protein sequences and immunological epitopes</td><td><ext-link ext-link-type="uri" xlink:href="https://www.hiv.lanl.gov/content/index" id="interref13">https://www.hiv.lanl.gov/content/index</ext-link></td></tr><tr><td/><td>ViralZone</td><td>highly curated and extensive virus knowledgebase</td><td><ext-link ext-link-type="uri" xlink:href="https://viralzone.expasy.org" id="interref14">https://viralzone.expasy.org</ext-link></td></tr><tr><td/><td>Prokaryotic Virus Ortholog Groups (pVOGs)</td><td>Bacteriophage protein orthology information</td><td><ext-link ext-link-type="uri" xlink:href="http://dmk-brain.ecn.uiowa.edu/VOG/" id="interref15">http://dmk-brain.ecn.uiowa.edu/VOG/</ext-link></td></tr><tr><td/><td>Plant Viruses Online</td><td>Plant virus gene and protein sequences and structures</td><td><ext-link ext-link-type="uri" xlink:href="http://sdb.im.ac.cn/vide/refs.htm" id="interref16">http://sdb.im.ac.cn/vide/refs.htm</ext-link></td></tr><tr><td><italic>Webtools</italic></td><td>IDSeq</td><td>Real-time pathogen detection from metagenomes</td><td><ext-link ext-link-type="uri" xlink:href="https://idseq.net" id="interref17">https://idseq.net</ext-link></td></tr><tr><td/><td>Virome</td><td>Viral detection from environmental metagenomes</td><td><ext-link ext-link-type="uri" xlink:href="http://virome.dbi.udel.edu" id="interref18">http://virome.dbi.udel.edu</ext-link></td></tr><tr><td/><td>VirusDetect</td><td>viral detection from small RNA datasets using both <italic>de novo</italic> and reference-based assemblies</td><td><ext-link ext-link-type="uri" xlink:href="http://virusdetect.feilab.net/cgi-bin/virusdetect/index.cgi" id="interref19">http://virusdetect.feilab.net/cgi-bin/virusdetect/index.cgi</ext-link></td></tr><tr><td/><td>Viral Genome ORF Reader (VIGOR)</td><td>Homology-driven viral gene prediction and genome annotation</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/JCVenterInstitute/VIGOR4" id="interref20">https://github.com/JCVenterInstitute/VIGOR4</ext-link></td></tr><tr><td/><td>STRING-Viruses</td><td>Assessment of viral-host protein-protein interactions using visualization tools such as Cytoscape</td><td><ext-link ext-link-type="uri" xlink:href="http://viruses.string-db.org" id="interref21">http://viruses.string-db.org</ext-link></td></tr><tr><td/><td><b>NextStrain</b></td><td>Rapid, real-time tracking and prediction of spatio-temporal spread of infection during infectious outbreaks</td><td><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="interref22">https://<b>nextstrain</b>.org</ext-link>.<li> Likewise, <b>NextStrain</b> (see &#x201C;Relevant Websites section&#x201D;) is a webtool that enables rapid, real-time tracking of evolving pathogen populations during infectious outbreaks.<li> <b>NextStrain</b> is an open source system that tracks mutation marker data on pathogen phylogenies to make inferences about epidemiologically-relevant parameters such as spatio-temporal spread of the infection within a host population.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33680867">33680867</a> <b>(9)</b> COVID-19 outbreak in Malaysia: Decoding D614G mutation of SARS-CoV-2 virus isolated from an asymptomatic case in Pahang<ul><li> The frequencies of G614 over D614 virus variants were analysed using <b>Nextstrain</b> SARS-CoV-2 resources database (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="ir030">https://<b>nextstrain</b>.org/</ext-link>).<li> Moreover, the data stated here was supported by the data reported in the <b>Nextstrain</b> SARS-CoV-2 resources database (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="ir045">https://<b>nextstrain</b>.org/</ext-link>) shown in <xref rid="f0005" ref-type="fig">Fig.<li> The figure was generated using <b>Nextstrain</b> SARS-CoV-2 resources database (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/asia%3fc%3dgt-S_614%26f_division%3dMalaysia" id="ir005">https://<b>nextstrain</b>.org/ncov/asia?c=gt-S_614&amp;f_division=Malaysia</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33647375">33647375</a> <b>(7)</b> Investigation of intra-hospital SARS-CoV-2 transmission using nanopore whole-genome sequencing<ul><li>Phylogenetic analysis, <b>Nextstrain</b> clade assortment, and pangolin lineage assignment.<li>Samples were assorted to clades according to the <b>Nextstrain</b> nomenclature [<xref rid="bib20" ref-type="bibr"><u>20</u></xref>].<li>Assessment of intra-hospital SARS-CoV-2 transmission</p></caption><alt-text id="alttext0025">Table&#xA0;I</alt-text><table frame="hsides" rules="groups"><thead><tr><th>Case</th><th>Sample date</th><th>Ward</th><th>Outbreak as defined by contact tracing</th><th>Outbreak as defined by WGS</th><th><b>Nextstrain</b> clade</th><th>Pangolin lineage</th><th>Study-unique variants</th><th>Interpretation</th></tr></thead><tbody><tr><td>HCW1.<li> Mutations shown according to <b>Nextstrain</b> classification system.<li>Phylogenetic analysis, <b>Nextstrain</b> clade assortment, and pangolin lineage assignment.<li> Eleven of the samples were classified as clade 20A, with three samples clustering within the <b>Nextstrain</b> emerging clade 20A/20268G.<li> In the one suspected outbreak that was refuted, the samples belonged to different <b>Nextstrain</b> major clades and pangolin lineages, with several different variants reported.<li> [20](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>          </person-group>          <article-title>Year-letter genetic clade naming for SARS-CoV-2 on nextstrain.org</article-title>          <source>Nextstrain</source>          <volume>2020</volume>          <year>2020</year>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33635912">33635912</a> <b>(2)</b> Transmission of SARS-CoV-2 in domestic cats imposes a narrow bottleneck<ul><li>Focal <b>Nextstrain</b> build of S H655Y sequences.<li> set for a <b>Nextstrain</b> phylogenetic analysis, to which &#x2019;context&#x2019; sequences were added, with the most genetically similar sequences given priority.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33637076">33637076</a> <b>(1)</b> Characterization of hospital airborne SARS-CoV-2<ul><li> Genome sequences were submitted to GenBank [<xref ref-type="bibr" rid="CR17"><u>17</u></xref>] (accession number MW047086) and analyzed using <b>NextStrain</b> [<xref ref-type="bibr" rid="CR18"><u>18</u></xref>].<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26590407">26590407</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33637820">33637820</a> <b>(6)</b> Hospitalization and mortality associated with SARS-CoV-2 viral clades in COVID-19<ul><li> The <b>Nextstrain</b> sequencing consortium<sup><xref ref-type="bibr" rid="CR9"><u>9</u></xref></sup> has identified 2785 variants, while the GISAID initiative has collected over 50,000 full length sequences as of 6-28-20<sup><xref ref-type="bibr" rid="CR10"><u>10</u></xref>&#x2013;<xref ref-type="bibr" rid="CR12"><u>12</u></xref></sup>.<li> Analysis of these sequences reveals 5 (<b>Nextstrain</b>) or 6 (GISAID) sequence clusters, with as many as 18 new mutations in each cluster distinct from the original SARS-CoV-2 virus<sup><xref ref-type="bibr" rid="CR6"><u>6</u></xref></sup>.<li> When mapped onto GISAID and <b>NextStrain</b> clades, we find in clade 1 that 89 correspond to clades GH/20C, 6 map to G/20A, and 2 map to G/20B.<li><b>Nextstrain</b> clade</th><th align="left">Clade 1</th><th align="left">Clade 2</th><th align="left">Clade 3</th><th align="left">GISAID</th></tr></thead><tbody><tr><td align="left">19A.<li> When mapped onto GISAID and <b>Nextstrain</b> clades, we find in clade 1 that 90 correspond to clades GH/20C (GISAID/<b>NextStrain</b>), while in clade 2, 86 correspond to S/19B.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33361812">33361812</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32167180">32167180</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32442528">32442528</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33676232">33676232</a> <b>(0)</b> SARS-CoV-2 variants combining spike mutations and the absence of ORF8 may be more transmissible and require close monitoring<ul><li>All sequences have been classified into clades using the Nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0015">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33630927">33630927</a> <b>(2)</b> Transcriptomic profiling and genomic mutational analysis of Human coronavirus (HCoV)-229E -infected human cells<ul><li> For genomic analysis we used the bioinformatics toolkit Augur (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>-augur.readthedocs.io/en/stable/">https://<b>nextstrain</b>-augur.readthedocs.io/en/stable/</ext-link>).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33633229">33633229</a> <b>(1)</b> Structural insights into SARS-CoV-2 spike protein and its natural mutants found in Mexican population<ul><li> To date, this mutant has become the dominant circulating protein displacing the WT, according to the levels of mutations worldwide, presented on the <b>nextstrain</b> database (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org">www.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33676588">33676588</a> <b>(11)</b> One year of SARS-CoV-2 evolution<ul><li> By comparing 3,823 representative viral genomes (referring to the result of <b>Nextstrain</b>, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0020">https://<b>nextstrain</b>.org/ncov/global</ext-link>, as of January 11<sup>th</sup>, 2021) to the early reference strain (EPI_ISL_402125), we found that SARS-CoV-2 accumulated about 0.035 amino acid mutations per day on average within the past year (<xref rid="fig1" ref-type="fig"><u>Figure&#xA0;1</u></xref>A), while S proteins showed a nonlinear variation pattern, which might be the result of different selection pressures on the whole genome compared to the S protein.<li> The heritable amino acid mutations in the major strain clades identified by <b>Nextstrain</b> (<xref rid="bib4" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) are shown hierarchically in the phylogenetic tree (<xref rid="fig1" ref-type="fig"><u>Figure&#xA0;1</u></xref>B).<li>(B) Phylogenetic tree with fixed amino acid mutations for representative SARS-CoV-2 strains selected by <b>Nextstrain</b> until January 11<sup>th</sup>, 2021.<li> Viral strains were divided into hierarchical clusters as those in <b>Nextstrain</b>, including four big clusters (19A, 19B, 20A, and 20B) and eight small clusters (20C, 20D, 20E, 20F, 20G, 20H, 20I, and 20J).<li>(B) Genetic diversity across residues of the RBD using representative SARS-CoV-2 strains selected by <b>Nextstrain</b> updated on January 11<sup>th</sup>, 2021.<li> Therefore, to ensure a representative dataset with an equitable and balanced spatiotemporal sequence distribution, we used the selected sequences and phylogenetic tree generated by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0030">https://<b>nextstrain</b>.org/ncov/global</ext-link>) as of January 11<sup>th</sup>, 2021 for further analysis.<li> The <b>Nextstrain</b> clade scheme was used to group the sequences.<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33620031">33620031</a> <b>(6)</b> COVID-19 CG enables SARS-CoV-2 mutation and lineage tracking by locations and dates of interest<ul><li> (2) Based on best practices, we filter out sequences on <b>NextStrain</b>&#x2019;s exclusion list, non-human SARS-CoV-2&#xA0;genomes,&#xA0;&lt;29,700,700 nt, or with&#xA0;&gt;5% ambiguous base calls (<xref rid="bib28" ref-type="bibr"><u>van Dorp et al., 2020b</u></xref>).<li> <b>NextStrain</b> uses the 100% identical Wuhan-Hu-1 (<xref rid="bib17" ref-type="bibr"><u>Langmead et al., 2009</u></xref>).<li> <b>NextStrain</b> (<xref rid="bib13" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) visualizes real-time tracking of SARS-CoV-2 evolution on a global and continental level, using phylogenetic trees, geographical maps, and entropy and frequency plots; CoV-GLUE (<xref rid="bib23" ref-type="bibr"><u>Singer et al., 2020</u></xref>) is a browsable database of amino acid replacements and indels; the UCSC SARS-CoV-2 Genome Browser is an extension of their widely used browser that enables layering of annotation tracks and new features such as conservation with similar viruses, immune epitopes, primers, and CRISPR guides (<xref rid="bib8" ref-type="bibr"><u>Fernandes et al., 2020</u></xref>); the COVID-19 Viral Genome Analysis Pipeline (<xref rid="bib15" ref-type="bibr"><u>Korber et al., 2020</u></xref>) allows for exploring mutations geographically and over time with a focus on the spike protein; COVIDep (<xref rid="bib1" ref-type="bibr"><u>Ahmed et al., 2020</u></xref>) is a browser for real-time reporting of vaccine target recommendations; Genomic Signature Analysis of Virus Causing COVID-19 (<xref rid="bib2" ref-type="bibr"><u>Bauer et al., 2020</u></xref>) visualizes the similarity between SARS-CoV-2 genomes in 2D space; SARS-CoV-2 Alignment Screen (<xref rid="bib27" ref-type="bibr"><u>van Dorp et al., 2020a</u></xref>; <xref rid="bib28" ref-type="bibr"><u>van Dorp et al., 2020b</u></xref>) is a visualization tool providing the distribution of SNPs and homoplasies; the WashU Virus Genome Browser (<xref rid="bib9" ref-type="bibr"><u>Flynn et al., 2020</u></xref>) provides different tools such as a phylogenetic tree and a genomic-coordinate track-based view of viral sequencing data.<li>Based on best practices, we filter out sequences meeting any of the following criteria: (1) present on the <b>NextStrain</b>&#x2019;s exclusion list (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt</ext-link>;&#xA0;<xref rid="bib13" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), (2) genomes from non-humans (animals, environmental samples, etc.), (3) genome length less than 29,700 nt, or (4)&#xA0;&gt;5% ambiguous base calls.<li> [van Dorp et al., 2020b](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33239633">33239633</a>) [Langmead et al., 2009](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19261174">19261174</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Singer et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Singer</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Gifford</surname>              <given-names>R</given-names>            </name>            <name>              <surname>Cotten</surname>              <given-names>M</given-names>            </name>            <name>              <surname>Robertson</surname>              <given-names>D</given-names>            </name>          </person-group>          <year>2020</year>          <article-title>CoV-GLUE: a web application for tracking SARS-CoV-2 genomic variation</article-title>          <source>Preprints</source>          <pub-id pub-id-type="doi">10.20944/preprints202006.0225.v1</pub-id>        ) [Fernandes et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908258">32908258</a>) [Korber et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) [Ahmed et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32555466">32555466</a>) [Bauer et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32306500">32306500</a>) [van Dorp et al., 2020a](          <person-group person-group-type="author">            <name>              <surname>van Dorp</surname>              <given-names>L</given-names>            </name>            <name>              <surname>Acman</surname>              <given-names>M</given-names>            </name>            <name>              <surname>Richard</surname>              <given-names>D</given-names>            </name>            <name>              <surname>Shaw</surname>              <given-names>LP</given-names>            </name>            <name>              <surname>Ford</surname>              <given-names>CE</given-names>            </name>            <name>              <surname>Ormond</surname>              <given-names>L</given-names>            </name>            <name>              <surname>Owen</surname>              <given-names>CJ</given-names>            </name>            <name>              <surname>Pang</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Tan</surname>              <given-names>CCS</given-names>            </name>            <name>              <surname>Boshier</surname>              <given-names>FAT</given-names>            </name>            <name>              <surname>Ortiz</surname>              <given-names>AT</given-names>            </name>            <name>              <surname>Balloux</surname>              <given-names>F</given-names>            </name>          </person-group>          <year>2020a</year>          <article-title>Emergence of genomic diversity and recurrent mutations in SARS-CoV-2</article-title>          <source>Infection, Genetics and Evolution</source>          <volume>83</volume>          <elocation-id>104351</elocation-id>          <pub-id pub-id-type="doi">10.1016/j.meegid.2020.104351</pub-id>        ) [van Dorp et al., 2020b](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33239633">33239633</a>) [Flynn et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908257">32908257</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> COVID-19 CG enables SARS-CoV-2 mutation and lineage tracking by locations and dates of interest<ul><li>The authors should note many of the existing sequence resources available including: the UCSC SARS-CoV-2 browser (https://genome.ucsc.edu/covid19.html) , the Wash U Genome browser (https://virusgateway.wustl.edu/), <b>Nextstrain</b> (https://<b>nextstrain</b>.org/) as well as the COG-UK efforts (https://www.cogconsortium.uk/, including CoV-GLUE and Pangolin which is cited, but only in the Materials and methods).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> COVID-19 CG enables SARS-CoV-2 mutation and lineage tracking by locations and dates of interest<ul><li>The authors should note many of the existing sequence resources available including: the UCSC SARS-CoV-2 browser (https://genome.ucsc.edu/covid19.html) , the Wash U Genome browser (https://virusgateway.wustl.edu/), <b>Nextstrain</b> (https://<b>nextstrain</b>.org/) as well as the COG-UK efforts (https://www.cogconsortium.uk/, including CoV-GLUE and Pangolin which is cited, but only in the Materials and methods).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33672421">33672421</a> <b>(1)</b> Human-to-Cat SARS-CoV-2 Transmission: Case Report and Full-Genome Sequencing from an Infected Pet and Its Owner in Northern Italy<ul><li>The strains showed 99.9% nucleotide identity, and both belonged to the <b>Nextstrain</b> clade 19A (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 29 May 2020)), corresponding to the B lineage of Pangolin (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri> (accessed on 29 May 2020)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33693018">33693018</a> <b>(3)</b> Case Report: Identification of SARS-CoV-2 in Cerebrospinal Fluid by Ultrahigh-Depth Sequencing in a Patient With Coronavirus Disease 2019 and Neurological Dysfunction<ul><li> To determine the <b>Nextstrain</b> clade of the sequence obtained in this study, 29 SARS-CoV-2 genome sequences belong to 19A/19B/20A/20B/20C/20D, according to the clade classification scheme of SARS-CoV-2 genomes by <b>Nextstrain</b>, were retrieved from GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</ext-link>).<li> We obtained a 29,857 bp SARS-CoV-2 genome (EPI_ISL_412386) (<xref ref-type="fig" rid="F4"><u>Figure 4</u></xref>) belonging to <b>Nextstrain</b> clade 19A.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33671631">33671631</a> <b>(1)</b> Genomic Epidemiology of SARS-CoV-2 in Madrid, Spain, during the First Wave of the Pandemic: Fast Spread and Early Dominance by D614G Variants<ul><li>Lineages were assigned following the &#x2019;PANGO scheme [<xref rid="B16-microorganisms-09-00454" ref-type="bibr"><u>16</u></xref>] and using the Pangolin 2.0 web app (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri> (accessed on 3 February 2021)), and the <b>NextStrain</b> year-letter scheme in the NextClade web page (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 3 February 2021)).<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33619344">33619344</a> <b>(1)</b> In silico detection of SARS-CoV-2 specific B-cell epitopes and validation in ELISA for serological diagnosis of COVID-19<ul><li> Raw data for estimating the effects of variants can alternatively be sourced directly from public databases, such as <b>NextStrain</b><sup><xref ref-type="bibr" rid="CR16"><u>16</u></xref></sup>, ViPR<sup><xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup> and the CNCB<sup><xref ref-type="bibr" rid="CR18"><u>18</u></xref></sup>.<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22006842">22006842</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32102777">32102777</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33691216">33691216</a> <b>(4)</b> Insights into SARS-CoV-2 evolution, potential antivirals, and vaccines<ul><li><b>Nextstrain</b> [<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" id="intref0015">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>] open source tracking introduced a new nomenclature system, in which a novel clade is introduced when the frequency of circulation reaches approximately 20% for worldwide strains (<xref rid="bib45" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The <b>Nextstrain</b> classification was modified to include labels correspond to an emerging variant and the prominent spike mutation and currently new clades 20D, 20E, 20F, 20G, 20H/501Y.V2, and 20I/501Y.V1 have been added (<xref rid="bib13" ref-type="bibr"><u>Bedford et al., 2021</u></xref>) as shown in <xref rid="tbl2" ref-type="table"><u>Table 2</u></xref>.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Bedford et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <part-title>Updated Nextstain SARS-CoV-2 Clade Naming Strategy</part-title>          <year>2021</year>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33594385">33594385</a> <b>(10)</b> Emergence in late 2020 of multiple lineages of SARS-CoV-2 Spike protein variants affecting amino acid position 677<ul><li>In late January of 2021, our two independent SARS-CoV-2 genomic surveillance programs, based at the University of New Mexico Health Sciences (UNM HSC) in Albuquerque, New Mexico and the Louisiana State University Health Sciences Center (LSUHS) in Shreveport, Louisiana, each noticed increasing numbers of PANGO lineage B.1.2 (<xref rid="R28" ref-type="bibr">Rambaut <italic toggle="yes">et al.</italic>, 2020</xref>) / <b>Nextstrain</b> clade 20G viruses carrying an S:Q677P mutation, and that this variant had increased in frequency in samples collected in late 2020 to mid January (<xref rid="F1" ref-type="fig">FIG.<li> Four of these occurred within clade 20G, a large clade that has accounted for over 30% of US sequences since October 2020, and as of the beginning of February 2021 represents approximately 50% of all SARS-CoV-2 sequence data in the USA (<xref rid="R25" ref-type="bibr"><italic toggle="yes"><b>Nextstrain</b> Build, North America, Feb 04, 2021</italic>, 2021</xref>).<li> <b>Nextstrain</b> build of 5874 sequences was prepared as described in <xref rid="R14" ref-type="bibr"><u>Hodcroft et al, 2020</u></xref>, initially selecting for sequences with any mutation at position 23592 or 23593.<li> Both of these are then processed through the open-source <b>Nextstrain</b> &#x1F9;cov` pipeline to produce a time-resolved phylogenetic analysis.<li> A time-stamped version of the <b>Nextstrain</b> build used in this manuscript can be found here: <ext-link xlink:href="https://<b>nextstrain</b>.org/groups/neherlab/ncov/S.Q677/2020-02-04" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/neherlab/ncov/S.Q677/2020-02-04</ext-link>, and the most recently updated version of this build can be viewed here: <ext-link xlink:href="https://<b>nextstrain</b>.org/groups/neherlab/ncov/S.Q677" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/neherlab/ncov/S.Q677</ext-link>.<li> The resulting JSON file from the <b>Nextstrain</b> pipeline was used to visualize the phylogenetic trees using the baltic package (<ext-link xlink:href="https://github.com/evogytis/baltic" ext-link-type="uri">https://github.com/evogytis/baltic</ext-link>).<li> [Hodcroft et al, 2020](<name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><etal/> (<year>2020</year>) &#x2018;<article-title>Emergence and spread of a SARS-CoV-2 variant through Europe in the summer of 2020</article-title>&#x2019;, <source>medRxiv : the preprint server for health sciences</source>. doi: <pub-id pub-id-type="doi">10.1101/2020.10.25.20219063</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33618012">33618012</a> <b>(2)</b> Clinical, virologic and immunologic features of a mild case of SARS-CoV-2 reinfection<ul><li> Although both viral sequences belong to the 20A clade (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" id="intref0010">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>), sequencing revealed that the two viruses recovered 6&#xA0;months apart were different (&gt;99.9% bp identity) and clustered with viruses circulating locally in the hospital clusters during each of the corresponding episodes (<xref rid="appsec1" ref-type="sec"><u>Supplementary Fig.&#xA0;S1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33737495">33737495</a> <b>(2)</b> SARS-CoV-2 genomics: An Indian perspective on sequencing viral variants<ul><li>New <b>Nextstrain</b> Nomenclature (Year-Letter) (Hodcroft et al.<li> The clade nomenclature (19 and 20) as specified by <b>Nextstrain</b> is given within the box, while the GISAID clade is mentioned in parentheses.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33609027">33609027</a> <b>(14)</b> Whole-genome Sequencing of SARS-CoV-2: Using Phylogeny and Structural Modeling to Contextualize Local Viral Evolution<ul><li>The <b>Nextstrain</b> conda environment, Auger (bioinformatics tooling) and Auspice (the <b>Nextstrain</b> open-source visualization tool (<ext-link ext-link-type="uri" xlink:href="https://auspice.us">https://auspice.us</ext-link>), was downloaded and installed locally.<li>The <b>Nextstrain</b> conda environment, Auger (bioinformatics tooling) and Auspice (the <b>Nextstrain</b> open-source visualization tool (<ext-link ext-link-type="uri" xlink:href="https://auspice.us">https://auspice.us</ext-link>), was downloaded and installed locally.<sup><xref rid="R18" ref-type="bibr"><u>18</u></xref></sup> The <b>Nextstrain</b> SARS-CoV-2 tutorial, including snakemake file with auger commands, was downloaded (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.github.io/ncov">https://<b>nextstrain</b>.github.io/ncov</ext-link>) and utilized as the foundation of the CIRS SARS-CoV-2 pathogen build.<li>To build phylogenetic trees in <b>Nextstrain</b>, the Snakemake command was run, referencing CIRS sequences and the <b>Nextstrain</b> CIRS SARS-CoV-2 pathogen build, and Auspice was used to visual the output .json file in the <b>Nextstrain</b> environment.<li>To visualize the phylogeny of the 59MDW samples, we created a pathogen build for a locally installed version of <b>Nextstrain</b>, an open-source visualization tool from the GISAID (gisaid.org) (Hadfield).<li> First, we modified the <b>Nextstrain</b> SARS-CoV-2 tutorial, which contained 419 global SARS-CoV-2 genomes, by adding an additional 48 specimens sequenced between May 2020 and August 2020, bringing the total number of global SARS-CoV-2 sequences in the build to 467.<li> These global strains were collected in different regions and classified into one of five distinct clades identified in <b>Nextstrain</b>&#x2014;19A, 19B, 20A, 20B, and 20C.<li> This file was then run in Auspice, the program to render <b>Nextstrain</b> visualizations, creating time and divergence phylogenies, diversity plots, and map-based visualizations of the data (<xref ref-type="fig" rid="F1">Fig.<li> The SARS-CoV-2 phylogeny was constructed in a local version of <b>Nextstrain</b> (Auspice.us) and shows the evolutionary relationship of 59MDW SARS-CoV-2 genomes (black dots) and a subsample of publicly available SARSCoV-2 genomes collected worldwide (<italic>n</italic>&#x2009;=&#x2009;301, colored by viral clade [see legend]).<li> Finally, we did an in-depth analysis of a cluster of infections in the 20C clade by zooming in on the cluster in the <b>Nextstrain</b> divergence phylogeny and using IGV VCF alignment in IGV (Fig.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Whole-genome Sequencing of SARS-CoV-2: Using Phylogeny and Structural Modeling to Contextualize Local Viral Evolution<ul><li> In this sense, SARS-CoV-2 genomes analysis has shown mutations that have emerged independently in multiple times.<sup><xref rid="cit0041" ref-type="bibr"><u>41</u></xref></sup> Interestingly, three sites in Orf1ab and one in Spike protein have experienced numerous recurrent mutations (&gt;15 events) that are of particular interest in the context of adaptation of SARS-CoV-2 to the human host.<sup><xref rid="cit0041" ref-type="bibr"><u>41</u></xref></sup> Consistent with that, characterizing frequently mutated residues by aligning ~660 SARS-CoV-2 genomes and validated in 10,000 datasets has shown that Orf1ab and Spike protein contained numerous nonsynonymous mutations, after available in GISAID <b>Nextstrain</b>.<li> [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32387564">32387564</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32387564">32387564</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33820739">33820739</a> <b>(22)</b> Genomic surveillance of Nevada patients revealed prevalence of unique SARS-CoV-2 variants bearing mutations in the RdRp gene<ul><li> We used a subsampling of sequence data from <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0010"><b>Nextstrain</b>.org</ext-link> to assess the frequency of D614G in the United States and globally during the same time period (March 6 to June 5) (<xref rid="fig2" ref-type="fig"><u>Fig.&#xA0;2</u></xref>B and 2C).<li> <bold>C</bold>: Global proportion of D614G in the shown regions during the same time period from a subsampling of sequences deposited in <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0070"><b>Nextstrain</b>.org</ext-link>.<li>Worldwide, there are currently 5 main clades (19A, 19B, 20A, 20B, 20C) of SARS-CoV-2 differentiated based on specific nucleotide profiles in the Year-letter scheme of <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="intref0015">https://clades.<b>nextstrain</b>.org</ext-link>.<li> We used a subsampling of <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0020"><b>Nextstrain</b>.org</ext-link> data to assess the frequency of clades in the United States and globally by continent during the same time period (March 6 to June 5).<li> <bold>D</bold>: Pie charts show the proportion of clades from global regions during the same time period from a subsampling of sequences deposited in <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0075"><b>Nextstrain</b>.org</ext-link>.<li> We used a subsampling of <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0025"><b>Nextstrain</b>.org</ext-link> data to assess the frequency of P323&#xA0;L/F in the United States and globally during the same time period (March 6 to June 5).<li> <bold>B</bold>: Circular dendrogram representing the distribution of amino acid change at residue 323 of nsp12 from a global subsampling of sequences deposited in <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0080"><b>Nextstrain</b>.org</ext-link> from March 6 to June 5, the larger dots indicate specimens from Nevada.<li> <bold>D</bold>: Proportion of P323&#xA0;L/F from a subsampling of sequences deposited in <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0085"><b>Nextstrain</b>.org</ext-link> for the United States and <bold>E</bold>: global regions from March 6 to June 5.<li> <bold>B</bold>: Circular dendrogram of global subsample of sequences from <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0090"><b>Nextstrain</b>.org</ext-link> with NV specimens indicated by larger dots.<li> Subsamples of sequences from <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0095"><b>Nextstrain</b>.org</ext-link> are used to generate the proportion of 379C&gt;A in <bold>D</bold>: the indicated states within the United States and <bold>E</bold> internationally.<li>A set of 3,644 complete, high-coverage SARS-CoV-2 genomes, subsample of data reported in the July 15, 2020, <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0030"><b>Nextstrain</b>.org</ext-link> global, were used to obtain the metadata from GISAID and combined with our own samples to determine their phylogenetic placement (<xref rid="bib7" ref-type="bibr"><u>Elbe and Buckland-Merrett, 2017</u></xref>; <xref rid="bib23" ref-type="bibr"><b>Nextstrain</b>, 2020</xref>).<li> The combined sets of global and Nevadan samples were aligned together, with metadata, by the <italic>augur</italic> phylodynamic pipelines of the <italic>ncov</italic> build of the <italic><b>nextstrain</b></italic> command-line tool, version 2.0.0.post1 (<xref rid="bib10" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> [Elbe and Buckland-Merrett, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33596239">33596239</a> <b>(5)</b> High throughput detection and genetic epidemiology of SARS-CoV-2 using COVIDSeq next-generation sequencing<ul><li> Clades were assigned according to the nomenclature defined by <b>Nextstrain</b> [<xref rid="pone.0247115.ref015" ref-type="bibr"><u>15</u></xref>] The resulting phylogenetic tree suggests that out of 469 COVIDSeq genomes, 451 genomes (96%) fell into the A2a clade while 14 genomes (3%) mapped to the I/A3i clade, a distinct clade previously defined for genomes from India [<xref rid="pone.0247115.ref016" ref-type="bibr"><u>16</u></xref>].<li>A) Phylogenetic trees generated by <b>Nextstrain</b>.<li> Mutation information provided by <b>Nextstrain</b> [<xref rid="pone.0247115.ref033" ref-type="bibr"><u>33</u></xref>] till August 08, 2020, was also used for comparison.<li> The phylogenetic network was built using the analysis protocol for SARS-CoV-2 genomes provided by <b>Nextstrain</b> [<xref rid="pone.0247115.ref015" ref-type="bibr"><u>15</u></xref>].<li> The resulting phylogenetic tree was viewed using Auspice, an interactive visualization web-application provided by <b>Nextstrain</b>.<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33200080">33200080</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27706213">27706213</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> High throughput detection and genetic epidemiology of SARS-CoV-2 using COVIDSeq next-generation sequencing<ul><li> The clade names have been following the <b>NextStrain</b> as well as PANGOLIN clade classifications which is widely followed in literature.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> High throughput detection and genetic epidemiology of SARS-CoV-2 using COVIDSeq next-generation sequencing<ul><li> <b>Nextstrain</b> analysis [<xref rid="CIT0009" ref-type="bibr"><u>9</u></xref>] revealed that all of the 4 viral genomes belonged to the 20B clade, and carry the D614G mutation in spike.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33619503">33619503</a> <b>(3)</b> Saliva TwoStep for rapid detection of asymptomatic SARS-CoV-2 carriers<ul><li>Regions of the SARS-CoV-2 genome that are conserved among strains were identified using genome diversity data from <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/ncov/global"><b>nextstrain</b>.org/ncov/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33592074">33592074</a> <b>(2)</b> Prospective observational study and serosurvey of SARS-CoV-2 infection in asymptomatic healthcare workers at a Canadian tertiary care center<ul><li> A multiple sequence alignment of all consensus reads and the MN908947.3 reference was generated, then used to build a phylogenetic tree using augur (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur">https://github.com/<b>nextstrain</b>/augur</ext-link>).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33587028">33587028</a> <b>(9)</b> A comprehensive profile of genomic variations in the SARS-CoV-2 isolates from the state of Telangana, India<ul><li>A binary matrix was constructed for each sample, which was processed by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) indicating whether a mutation of interest was present or absent.<li>The consensus fasta files generated for both Illumina and Nanopore data were subjected to phylogenetic analysis using the <b>Nextstrain</b> pipeline with recommended default criteria for filtering, multiple sequence alignment (MSA) and nucleotide substitution calculations.<li> The nomenclature of <b>Nextstrain</b>-assigned phylogenetic clades is based on the naming scheme proposed by Rambaut <italic>et al</italic>.<li> We also performed <b>Nextstrain</b> analysis separately on our Illumina and Nanopore datasets and observed that the clusters obtained did not show any segregation due to the platform used and were distributed across the clades (data not shown), thus ruling out the possibility of incorporation of any bias due to the sequencing method used.<li>Phylodynamic tree of the samples as analysed using <b>Nextstrain</b> (a).<li> Time-resolved phylogenetic tree from <b>Nextstrain</b> analysis; (b) distribution of clades across the dataset; (c) symptoms vs case distribution for samples classified as belonging to 20B clade.<li>(a) Mapping of the 17 high-frequency mutations on the phylodynamic tree calculated using <b>Nextstrain</b>; (b) pairwise correlation coefficient matrix for the aforementioned mutation set; the numbers within the cells indicate <italic>P</italic>-values; (c) associated dendrogram obtained after hierarchical clustering of mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33615284">33615284</a> <b>(2)</b> COVID-19 cryptic transmission and genetic information blackouts: Need for effective surveillance policy to better understand disease burden<ul><li> <b>Nextstrain</b><xref rid="bib6" ref-type="bibr"><u>[5]</u></xref> was used to conduct phylogenetic analysis.<li> At the time point of September 2020, <b>Nextstrain</b> defined 5 major clades.<li> [[5]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33588037">33588037</a> <b>(1)</b> Transmission dynamics of SARS-CoV-2 on the <italic>Diamond Princess</italic> uncovered using viral genome sequence analysis<ul><li> All retained sequences were randomly resampled up to two per country, year, and month using <b>nextstrain</b> CLI (v.1.16.5) (<xref rid="b0070" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) to maintain the geographic and temporal diversity of the virus.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33589297">33589297</a> <b>(1)</b> Recurrent COVID-19 including evidence of reinfection and enhanced severity in thirty Brazilian healthcare workers<ul><li> SARS-CoV-2 lineage assignments were performed with Pangolin<xref rid="bib0011" ref-type="bibr"><sup>11</sup></xref> and phylogenetic analysis of sequenced viral genomes was conducted using <b>Nextstrain</b>, described in supplementary methods.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33643227">33643227</a> <b>(1)</b> Early Diffusion of SARS-CoV-2 Infection in the Inner Area of the Italian Sardinia Island<ul><li> The global distribution and prevalence of each amino acid substitution have been evaluated using the <b>Nextstrain</b> genomic epidemiology site<sup><xref ref-type="fn" rid="footnote4"><u>4</u></xref></sup> and the Tracking Mutations tool as part of the COVID-19 viral genome analysis pipeline from Los Alamos National Laboratory<sup><xref ref-type="fn" rid="footnote5"><u>5</u></xref></sup>.<li> [4](<a href="https://nextstrain.org/ncov/">https://nextstrain.org/ncov/</a>) [5](<a href="https://cov.lanl.gov">https://cov.lanl.gov</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33580111">33580111</a> <b>(0)</b> SARS-CoV-2 genomic surveillance in Rond&#xF4;nia, Brazilian Western Amazon<ul><li> Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) and CoVsurver reports&#xA0;(<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/">https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/</ext-link>) are shown in the Supplementary Material (table <xref rid="MOESM3" ref-type="media"><u>S2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33580132">33580132</a> <b>(3)</b> Phylogenomics reveals viral sources, transmission, and potential superinfection in early-stage COVID-19 patients in Ontario, Canada<ul><li> The pipeline follows the <b>Nextstrain</b> analysis<sup><xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup> but includes additional steps to remove genome sequences from divergent lineages not informative for reconstructing transmission.<li> This is based on an average of 24.225&#xA0;bp substitutions per year in the <b>Nextstrain</b> analysis<sup><xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup> of the GISAID<sup><xref ref-type="bibr" rid="CR18"><u>18</u></xref>,<xref ref-type="bibr" rid="CR19"><u>19</u></xref></sup> database.<li><b>Nextstrain</b> phylogenetic tree of local cases of SARS-CoV-2 and the most similar reference sequences in the GISAID database.<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33570490">33570490</a> <b>(8)</b> The Spike D614G mutation increases SARS-CoV-2 infection of multiple human cell types<ul><li> This visualization was produced by the <b>Nextstrain</b> webtool using GISAID genomes (n&#xA0;=&#xA0;3,866 genomes samples from January 2020 to May 2020).<li><b>NextStrain</b>.<li>                <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>              .<li>For temporal tracking of D614G mutations in SARS-CoV-2 genomes, we used the <b>Nextstrain</b> analysis tool (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>) with data obtained from GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</ext-link>) (<xref rid="bib6" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bib23" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>).<li> With the <b>Nextstrain</b> webtool, we visualized 3866 genomes using the &#x2018;clock&#x2019; layout with sample coloring based on Spike 614 mutation status.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Shu and McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33574104">33574104</a> <b>(2)</b> Coding-Complete Genome Sequences of NITMA1086 and NITMA1139, Two SARS-CoV-2 Isolates from Belagavi District, Karnataka State, India, Harboring the D614G Mutation<ul><li>Phylogenetic comparison of these two genome sequences with globally detected viral strains was performed using the <b>Nextstrain</b> tool (with an inbuilt global data set from GISAID) (<xref rid="B7" ref-type="bibr"><u>7</u></xref>).<li>Phylogenetic tree displaying sequenced SARS-CoV-2 strains (with reference to GISAID global data set), in accordance to the clades assigned for the virus during phylogenetic analysis using the Augur toolkit (with default parameters) run through the <b>Nextstrain</b> server.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33571105">33571105</a> <b>(14)</b> The origin and early spread of SARS-CoV-2 in Europe<ul><li> The intermixing of these lineages in different countries and regions suggests that the virus was transmitted across borders many times (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" xlink:show="new">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> This lineage is defined by a characteristic amino acid substitution at position 314 in the ORF1b gene from proline to leucine and was provisionally named the A2a lineage by the <b>Nextstrain</b> team, later renamed to 20A.<li> lineages assigned using <b>Nextstrain</b> (<xref rid="r10" ref-type="bibr"><u>10</u></xref>)].<li> This German sample is part of a smaller clade that is closely related to the larger clade of A2a sequences and which was originally named the A2 lineage but was later included in the larger 19A (<b>Nextstrain</b> nomenclature) or B (pangolin nomenclature) lineage (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" xlink:show="new">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> In contrast, at the time of this paper&#x2019;s original submission, <b>Nextstrain</b> placed this ancestor in the United Kingdom with 100% confidence (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/europe" xlink:show="new">https://<b>nextstrain</b>.org/ncov/europe</ext-link>).<li> This <b>Nextstrain</b> result may have been an artifact of disproportionately high sequencing effort in the United Kingdom since biased sampling violates the assumptions of the &#x201C;mugration&#x201D; method employed (<xref rid="r26" ref-type="bibr"><u>26</u></xref>).<li>We prepared a sequence alignment from data publicly available on GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" xlink:show="new">https://www.gisaid.org/</ext-link>) on 1 April 2020 using the <b>Nextstrain</b> pipeline for SARS-CoV-2 (<xref rid="r10" ref-type="bibr"><u>10</u></xref>).<li> Short sequences (&lt;25,000 bases), sequences without fully specified collection dates, and sequences in the <b>Nextstrain</b> exclude list (<xref rid="r40" ref-type="bibr"><u>40</u></xref>) (duplicate sequences from the same case, or with suspicious amounts of nucleotide divergence) were excluded.<li> To eliminate sites identified by the <b>Nextstrain</b> team as prone to sequencing errors (<xref rid="r41" ref-type="bibr"><u>41</u></xref>), we masked the first 130 and final 50 sites from the alignment, as well as sites 18,529, 29,849, 29,851, and 29,853.<li> [10](Nextstrain, nextstrain/ncov: Nextstrain build for novel coronavirus (nCoV), <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</ext-link>. Accessed 2 April 2020.) [26](<person-group person-group-type="author"><string-name><given-names>P.</given-names><surname>Sagulenko</surname></string-name>, <string-name><given-names>R.</given-names><surname>Neher</surname></string-name></person-group>, Inference of transition between discrete characters and &#x2018;mugration&#x2019; models&#x2014;TreeTime 0.7.6 documentation, <ext-link ext-link-type="uri" xlink:href="https://treetime.readthedocs.io/en/latest/tutorials/mugration.html#sampling-biases">https://treetime.readthedocs.io/en/latest/tutorials/mugration.html#sampling-biases</ext-link>. Accessed 21 May 2020.) [10](Nextstrain, nextstrain/ncov: Nextstrain build for novel coronavirus (nCoV), <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</ext-link>. Accessed 2 April 2020.) [40](Nextstrain, ncov/exclude.txt at master nextstrain/ncov GitHub, <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov/blob/master/defaults/exclude.txt">https://github.com/nextstrain/ncov/blob/master/defaults/exclude.txt</ext-link>. Accessed 11 November 2020.) [41](Nextstrain, ncov/parameters.yaml at master nextstrain/ncov GitHub, <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov/blob/master/defaults/parameters.yaml">https://github.com/nextstrain/ncov/blob/master/defaults/parameters.yaml</ext-link>. Accessed 11 November 2020.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33594355">33594355</a> <b>(2)</b> SARS-CoV-2 European resurgence foretold: interplay of introductions and persistence by leveraging genomic and mobility data<ul><li> Genomic surveillance demonstrated that a new variant (lineage B.1.177 <sup><xref rid="R8" ref-type="bibr"><u>8</u></xref></sup>, 20A.EU1 [<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>]) that emerged in Spain in early summer has spread to multiple locations in Europe <sup><xref rid="R9" ref-type="bibr"><u>9</u></xref></sup>.<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [9](<name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><etal/><article-title>Emergence and spread of a SARS-CoV-2 variant through Europe in the summer of 2020</article-title>. <source>medRxiv</source> (<year>2020</year>) doi:<pub-id pub-id-type="doi">10.1101/2020.10.25.20219063</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33568759">33568759</a> <b>(1)</b> Genomic mutations and changes in protein secondary structure and solvent accessibility of SARS-CoV-2 (COVID-19 virus)<ul><li>A recent situation report<sup><xref ref-type="bibr" rid="CR25"><u>25</u></xref></sup> by <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup> on genomic epidemiology of novel coronavirus using 5193 publicly shared COVID-19 genomes shows that SARS-CoV-2 on average accumulates changes at a rate of &#xA0;24 substitutions per year.<li> [25](Bell, S. M. <italic>et al.</italic> Genomic analysis of COVID-19: Situation report 2020-05-15. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/narratives/ncov/sit-rep/2020-05-15">https://nextstrain.org/narratives/ncov/sit-rep/2020-05-15</ext-link> (2020).) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33563320">33563320</a> <b>(1)</b> CoronaHiT: high-throughput sequencing of SARS-CoV-2 genomes<ul><li> This has enabled real-time public health surveillance of the spread and evolution of the pandemic through interactive tools such as <b>NextStrain</b> [<xref ref-type="bibr" rid="CR7"><u>7</u></xref>].<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33572274">33572274</a> <b>(2)</b> Understanding Viral Infection Mechanisms and Patient Symptoms for the Development of COVID-19 Therapeutics<ul><li> More research should be done to clarify the above (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33572190">33572190</a> <b>(2)</b> Molecular Epidemiology Surveillance of SARS-CoV-2: Mutations and Genetic Diversity One Year after Emerging<ul><li>A maximum likelihood phylogenetic tree of the 2213 SARS-CoV-2 genomes analyzed in this study on a background of 1888 reference genomes was constructed in <b>Nextstrain</b> [<xref rid="B70-pathogens-10-00184" ref-type="bibr"><u>70</u></xref>] on 30 November 2020.<li> In addition, we obtained a maximum likelihood phylogenetic tree (named Nexstrain phylogenetic tree) of SARS-CoV-2 obtained from <b>Nextstrain</b> [<xref rid="B70-pathogens-10-00184" ref-type="bibr"><u>70</u></xref>] on the same date in order to localize the nucleotide changes and <italic>dN</italic> substitutions into their respective clades together with divergence times.<li> [70](          <article-title>Nextstrain/Ncov/Global</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2020-11-30">(accessed on 30 November 2020)</date-in-citation>        ) [70](          <article-title>Nextstrain/Ncov/Global</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2020-11-30">(accessed on 30 November 2020)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33597826">33597826</a> <b>(5)</b> Current mutatome of SARS-CoV-2 in Turkey reveals mutations of interest<ul><li> (Elbe and Buckland&#x2010;Merrett, 2017) and <b>Nextstrain</b><b>Nextstrain</b> (2015).<li> <b>Nextstrain</b> [online].<li> Our findings reveal the presence of three main clades of SARS-CoV-2 in Turkey, roughly analogous to 19A, 20A, and 20B as described in <b>NextStrain</b>, with a preponderance of high mutability variants (Eskier et al., 2020a, 2020b, 2020c) compared to international isolates.<li> Future plans include implementation of <b>Nextstrain</b> clade and branch information in the phylogenetic tree to aid the user in comparisons with international sequencing data.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33567556">33567556</a> <b>(31)</b> Dynamic Molecular Epidemiology Reveals Lineage-Associated Single-Nucleotide Variants That Alter RNA Structure in Chikungunya Virus<ul><li> <b>Nextstrain</b> (<uri xlink:href="https://www.<b>nextstrain</b>.org">https://www.<b>nextstrain</b>.org</uri> (accessed on 20 January 2021)) [<xref rid="B46-genes-12-00239" ref-type="bibr"><u>46</u></xref>] is an open source project that makes pathogen phylogenetic data easily accessible to researchers and the interested public, thus facilitating research efforts in the field of pathogen evolution and epidemiology.<li> <b>Nextstrain</b> allows for the set up so-called community builds that employ the <b>Nextstrain</b> software stack to construct custom real-time phylodynamics resources.<li> The <b>Nextstrain</b> community builds have become increasingly popular, and were used, for example, to highlight the genomic epidemiology of the 2018&#x2013;2020 Ebola virus outbreak in DRC [<xref rid="B47-genes-12-00239" ref-type="bibr"><u>47</u></xref>], and showcase the mutational dynamics of SARS-CoV-2 superspreading events in Austria [<xref rid="B48-genes-12-00239" ref-type="bibr"><u>48</u></xref>].<li> We report here the availability of a custom <b>Nextstrain</b> build for CHIKV that encompasses 924 publicly available genomes.<li> We compiled all temporal and geographic metadata available in these genome records to create data sets for building the <b>Nextstrain</b> instance.<li> CHIKV <b>Nextstrain</b>.<li>We employed the workflow management system <monospace>Snakemake</monospace> [<xref rid="B53-genes-12-00239" ref-type="bibr"><u>53</u></xref>] to build a pipeline for rapid deployment and reproducibility of the CHIKV <b>Nextstrain</b> build.<li> Isolates with unknown lineage association were assigned to lineages via the the time-resolved phylogenetic tree produced by <b>Nextstrain</b> iteratively.<li> The standard <monospace>augur</monospace> pipeline was then applied to construct all relevant data for <b>Nextstrain</b> visualization [<xref rid="B46-genes-12-00239" ref-type="bibr"><u>46</u></xref>].<li>For characterizing SNVs that affect CHIKV RNA structure formation, we performed local RNA structure prediction with <monospace>RNALfold</monospace> [<xref rid="B54-genes-12-00239" ref-type="bibr"><u>54</u></xref>] in the reference strain of our <b>Nextstrain</b> build (KT327163.2), limited to sequence lengths of 150 nt and filtered for thermodynamically stable structures.<li> These were then intersected with <b>Nextstrain</b> genome diversity data, yielding a set of 759 candidate RNAs that overlap either one or more variable sites of the CHIKV genome.<li>The CHIKV Nexstrain instance is available at <uri xlink:href="https://<b>nextstrain</b>.org/community/ViennaRNA/CHIKV">https://<b>nextstrain</b>.org/community/ViennaRNA/CHIKV</uri> (accessed on 20 January 2021).<li> A <b>Nextstrain</b> Build for Chikungunya Virus.<li>In an attempt to provide a publicly available epidemiological and phylogeographical interactive visualization of CHIKV spread, we created a custom <b>Nextstrain</b> build that encompasses all currently available CHIKV genome data.<li> Our community build is available via <uri xlink:href="https://<b>nextstrain</b>.org/community/ViennaRNA/CHIKV">https://<b>nextstrain</b>.org/community/ViennaRNA/CHIKV</uri> (accessed on 20 January 2021) and comprises 924 genomes, which represents a substantial increase compared to the 590 genomes considered in the previous most comprehensive study of CHIKV phylogeny [<xref rid="B16-genes-12-00239" ref-type="bibr"><u>16</u></xref>].<li> The <b>Nextstrain</b> phylogeny is based on a maximum-likelihood tree, which is used to infer a timed tree with TreeTime [<xref rid="B57-genes-12-00239" ref-type="bibr"><u>57</u></xref>] (<xref ref-type="fig" rid="genes-12-00239-f001"><u>Figure 1</u></xref>), making available the time of the most recent common ancestor (TMRCA) associated with each individual lineage/major clade of interest (<xref rid="genes-12-00239-t002" ref-type="table"><u>Table 2</u></xref>).<li>The CHIKV <b>Nextstrain</b> build allows the user to filter data and change the visualization according to preferences, utilizing a set of filters such as date ranges, multiple tree visualization layouts, feature filters and colors.<li> Besides visualizing phylogeography-related characteristics, <b>Nextstrain</b> provides information about the diversity of the underlying sequence data as normalized Shannon entropy in a separate diversity panel.<li> The analysis of nucleotide divergence within the full genome sequences available on the <b>Nextstrain</b> build enabled us to explore lineage-associated genomic variants that change the stability and fold of locally stable RNA structures in silico.<li> To this end, we performed local RNA secondary structure prediction in the reference strain of our <b>Nextstrain</b> build (KT327163.2, clustering with the AUL-Am lineage), aiming at characterizing structural elements that show increased thermodynamic stability, as expressed by z-score statistics.<li> We intersected loci that fold into locally stable RNA structures with genome diversity data from <b>Nextstrain</b> to obtain regions of the CHIKV genome that both fold into stable RNA structures and contain one or more single-nucleotide mutations.<li> To this end, we build on the <b>Nextstrain</b> [<xref rid="B46-genes-12-00239" ref-type="bibr"><u>46</u></xref>] framework to provide an interactive phylodynamics resource of CHIKV that reveals spatiotemporal and epidemiological facets of global virus dissemination.<li> From this perspective, our <b>Nextstrain</b> instance provides a reasonable amount of data to conduct a more in-depth investigation of the outbreaks that have recently occurred.<li> The consistency of the dates between <b>Nextstrain</b> TimeTree calculations and previously described TMRCAs in the literature is also reassuring of our ability to provide this additional information on the CHIKV <b>Nextstrain</b> instance.<li>We asked whether <b>Nextstrain</b> can be utilized to infer novel insight into RNA structure-association of individual clades.<li> As <b>Nextstrain</b> is particularly convenient for discriminating characteristics of viral clades/lineages, we set out to expand the sequence-centric approach to computation of lineage-associated structural traits.<li> By combining <b>Nextstrain</b> genome diversity data with RNA structure prediction methods, we could associate sequence variants observed in different CHIKV lineages with alternative RNA structure formation.<li> [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [47](          <person-group person-group-type="author">            <name>              <surname>Kinganda-Lusamaki</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Black</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Mukadi</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Mbala-Kingebeni</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Pratt</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Aziza</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Diagne</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>White</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Bisento</surname>              <given-names>N.</given-names>            </name>            <etal/>          </person-group>          <article-title>Operationalizing genomic epidemiology during the Nord-Kivu Ebola outbreak, Democratic Republic of the Congo</article-title>          <source>medRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.06.08.20125567</pub-id>        ) [48](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33229462">33229462</a>) [53](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22908215">22908215</a>) [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [54](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33374382">33374382</a>) [16](          <person-group person-group-type="author">            <name>              <surname>De Bernardi Schneider</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Ochsenreiter</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Hostager</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Hofacker</surname>              <given-names>I.L.</given-names>            </name>            <name>              <surname>Janies</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Wolfinger</surname>              <given-names>M.T.</given-names>            </name>          </person-group>          <article-title>Updated Phylogeny of Chikungunya Virus Suggests Lineage-Specific RNA Architecture</article-title>          <source>Viruses</source>          <year>2019</year>          <volume>11</volume>          <elocation-id>798</elocation-id>          <pub-id pub-id-type="doi">10.3390/v11090798</pub-id>        ) [57](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33564781">33564781</a> <b>(1)</b> Emergence of an early SARS-CoV-2 epidemic in the United States<ul><li> To investigate how many times SARS-CoV-2 was introduced into Louisiana, we first conducted a high-level genomic analysis by comparing <b>NextStrain</b> clade distributions of all available SARS-CoV-2 sequences from the continental U.S.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33558493">33558493</a> <b>(3)</b> SARS-CoV-2 D614G spike mutation increases entry efficiency with enhanced ACE2-binding affinity<ul><li> During the COVID-19 pandemic, SARS-CoV-2 has accumulated mutations throughout viral genes encoding the ORF1a, ORF1b, ORF3, ORF8, nucleocapsid (N), S proteins, etc., some of which are clade-defining (<b>Nextstrain</b>, <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/ncov">http://<b>nextstrain</b>.org/ncov</ext-link>, based on the GISAID data, <ext-link ext-link-type="uri" xlink:href="http://www.gisaid.org">www.gisaid.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33585048">33585048</a> <b>(4)</b> Evolutionary analysis and lineage designation of SARS-CoV-2 genomes<ul><li> Based on SNVs at these two sites and other sites, GISAID (<ext-link ext-link-type="uri" xlink:href="http://gisaid.org" id="ir015">http://gisaid.org</ext-link>) divided SARS-CoV-2 genomes into four major groups (S, L, V, and G), and <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="ir020">https://<b>nextstrain</b>.org</ext-link>) <xref rid="b0110" ref-type="bibr"><u>[22]</u></xref> categorized them into five major clades (19A, 19B, 20A, 20B, and 20C).<li> <xref rid="b0105" ref-type="bibr"><u>[21]</u></xref>, GISAID, or <b>Nextstrain</b> <xref rid="b0110" ref-type="bibr"><u>[22]</u></xref>.<li> [[22]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[21]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [[22]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33556558">33556558</a> <b>(12)</b> SARS-CoV-2 mutation 614G creates an elastase cleavage site enhancing its spread in high AAT-deficient regions<ul><li> This mutant spread rapidly and widely across Europe and North-America [GISAID: <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="ir0005">https://www.gisaid.org/</ext-link> and <b>Nextstrain</b>: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="ir0010">https://<b>nextstrain</b>.org/</ext-link>] outcompeting the ancestral type - 614D (<xref rid="bb0040" ref-type="bibr"><u>Biswas and Majumder, 2020</u></xref>; <xref rid="bb0145" ref-type="bibr"><u>Gudbjartsson et al., 2020</u></xref>; <xref rid="bb0175" ref-type="bibr"><u>Korber et al., 2020</u></xref>).<li>To analyze SARS-CoV-2 sequence data, we used the community standard <b>nextstrain</b>/ncov (<xref rid="bb0150" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) (<ext-link ext-link-type="uri" xlink:href="http://github.com/<b>nextstrain</b>/ncov" id="ir0015">github.com/<b>nextstrain</b>/ncov</ext-link>) pipeline developed specifically for spatial and temporal tracking of pathogens.<li> <b>Nextstrain</b>/ncov (<xref rid="bb0150" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), is an open-source pipeline for phylodynamic analysis (<xref rid="bb0290" ref-type="bibr"><u>Volz et al., 2013</u></xref>), including subsampling, alignment, phylogenetic inference, temporal dating of ancestral nodes and discrete trait geographic reconstruction as well as interactive data visualization.<li> It comprises augur (<xref rid="bb0150" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) (<ext-link ext-link-type="uri" xlink:href="http://github.com/<b>nextstrain</b>/augur" id="ir0020">github.com/<b>nextstrain</b>/augur</ext-link>) a modular bioinformatics tool used for data analysis, and auspice (<xref rid="bb0150" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) (<ext-link ext-link-type="uri" xlink:href="http://github.com/<b>nextstrain</b>/auspice" id="ir0025">github.com/<b>nextstrain</b>/auspice</ext-link>) a web-based visualization tool for phylogenomic and phylogeographic data.<li> Out of the 82644 sequences, 78616 sequences passed default QC criteria of <b>Nextstrain</b> pipeline.<li> ORF8 -L84S, ORF1a - V378I, ORF1a - L3606F, ORF1a - A3220V, ORF3a - G251V, ORF1a - L3606F, S - D614G, ORF1b - P314L and 3 other nucleotide sites were used by community (www.<b>nextstrain</b>.org) to define the clades.<li> [Biswas and Majumder, 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32474553">32474553</a>) [Gudbjartsson et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32289214">32289214</a>) [Korber et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Volz et al., 2013](          <person-group person-group-type="author">            <name>              <surname>Volz</surname>              <given-names>E.M.</given-names>            </name>            <name>              <surname>Koelle</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>          </person-group>          <article-title>Viral Phylodynamics</article-title>          <source>PLoS Comput. Biol.</source>          <volume>9</volume>          <issue>3</issue>          <year>2013</year>          <pub-id pub-id-type="doi">10.1371/journal.pcbi.1002947</pub-id>          <comment>Public Library of Science. e1002947</comment>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33545711">33545711</a> <b>(0)</b> SARS-CoV-2 evolution during treatment of chronic infection<ul><li> Major SARS-CoV-2 clade memberships were assigned to all sequences using both the Nextclade server v0.9 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) and Phylogenetic Assignment Of Named Global Outbreak Lineages (pangolin)<sup><xref rid="R32" ref-type="bibr"><u>32</u></xref></sup>.<li> [32](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>            <etal/>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>          <source>Nature Microbiology</source>          <year>2020</year>          <volume>5</volume>          <fpage>1403</fpage>          <lpage>1407</lpage>          <pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33547152">33547152</a> <b>(2)</b> Guillain-Barr&#xE9; Syndrome Outbreak in Peru 2019 Associated With <italic>Campylobacter jejuni</italic> Infection<ul><li> These 83 genomes, the <italic>C jejuni</italic> reference genome (NCTC11168),<sup><xref rid="R21" ref-type="bibr"><u>21</u></xref>,<xref rid="R22" ref-type="bibr"><u>22</u></xref></sup> and the 4 genomes of 4 <italic>C jejuni</italic> isolates assembled from the present study were used to construct a phylogenic tree from the sequence of the hypervariable LOS biosynthesis gene locus using the <b>Nextstrain</b>-Augur pipeline.<li>Phylogenetic tree built from the lipo-oligosaccharide region of 88 <italic>C jejuni</italic> species using the <b>Nextstrain</b> pipeline, colored by region.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=10688204">10688204</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17199895">17199895</a>) </ul></p>
<p><u>No PMID</u> <b>(3)</b> What is the intermediate host species of SARS-CoV-2?<ul><li> mass sequencing initiatives such as GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org">www.gisaid.org</ext-link>) have enabled scientists to establish the rate of evolution measured in real units of time and map the phylogenies of the SARS-CoV-2 through platforms such as <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) [<xref rid="B2" ref-type="bibr"><u>2</u></xref>,<xref rid="B3" ref-type="bibr"><u>3</u></xref>].<li> [2](<person-group person-group-type="author"><string-name><surname>Cohen</surname><given-names>J</given-names></string-name></person-group><article-title>Mining coronavirus genomes for clues to the outbreak's origins</article-title>. (<year>2020</year>). <ext-link ext-link-type="uri" xlink:href="http://www.sciencemag.org/news/2020/01/mining-coronavirus-genomes-clues-outbreak-s-origins">www.sciencemag.org/news/2020/01/mining-coronavirus-genomes-clues-outbreak-s-origins</ext-link>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32197085">32197085</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33558859">33558859</a> <b>(2)</b> Genetic diversity and genomic epidemiology of SARS-CoV-2 in Morocco<ul><li>To assess the genetic variation of SARS-CoV-2 in Morocco, a total of 40,390 complete genomes of SARS-CoV-2 and their corresponding metadata were retrieved from GISAID database [<xref rid="bb0045" ref-type="bibr"><u>9</u></xref>] and analyzed using the <b>Nextstrain</b> bioinformatics pipeline [<xref rid="bb0050" ref-type="bibr"><u>10</u></xref>].<li>Genomic epidemiology, using <b>Nextstrain</b>, applied to SARS-CoV-2 transmission in Morocco, revealed many aspects of the epidemic already known to the authorities, like the introduction of SARS-CoV-2 strains into Morocco from Belgium, Spain and France at the beginning of the epidemic.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33548358">33548358</a> <b>(1)</b> Introduction of Brazilian SARS-CoV-2 484K.V2 related variants into the UK<ul><li> Different characterisation is assigned by <b>Nextstrain</b> and a global system has yet to be agreed upon.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33540596">33540596</a> <b>(2)</b> Dynamics of a Dual SARS-CoV-2 Lineage Co-Infection on a Prolonged Viral Shedding COVID-19 Case: Insights into Clinical Severity and Disease Duration<ul><li>The affiliation of the two haplotypes in distinct clades, which emerged at different time points along the pandemic (see <uri xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</uri>) [<xref rid="B28-microorganisms-09-00300" ref-type="bibr"><u>28</u></xref>], renders it unlikely that they evolved intra-host from one another.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33395380">33395380</a> <b>(1)</b> Rapid Transmission of Severe Acute Respiratory Syndrome Coronavirus 2 in Detention Facility, Louisiana, USA, May&#x2013;June, 2020<ul><li> Representative full-genome sequences were downloaded on August 28, 2020, from GISAID (<ext-link xlink:href="https://www.gisaid.org" ext-link-type="uri">https://www.gisaid.org</ext-link>), and phylogenetic relations were inferred by using maximum-likelihood analyses implemented in TreeTime (<ext-link xlink:href="https://evol.bio.lmu.de/_statgen/software/treetime" ext-link-type="uri">https://evol.bio.lmu.de/_statgen/software/treetime</ext-link>) and the <b>Nextstrain</b> pipeline (<xref rid="R17" ref-type="bibr"><italic>17</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33535105">33535105</a> <b>(0)</b> Introduction into the Marseille geographical area of a mild SARS-CoV-2 variant originating from sub-Saharan Africa: An investigational study<ul><li>Sequences from complete genomes were analyzed using the Nextclade web-tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0025">https://clades.<b>nextstrain</b>.org/</ext-link>) <xref rid="bib7" ref-type="bibr"><u>[7]</u></xref>.<li> [[7]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33581746">33581746</a> <b>(2)</b> Public health actions to control new SARS-CoV-2 variants<ul><li>SARS-CoV-2 is prone to making errors in its genetic code during replication, accumulating 1&#x2013;2 nucleotide changes (mutations) every month (<b>nextstrain</b>.org/ncov/global?l = clock).<li> Dedicated sequencing and open data sharing are allowing us to track the emergence and spread of SARS-CoV-2 genetic variants in near real time (e.g., <b>nextstrain</b>.org).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33525651">33525651</a> <b>(4)</b> Summary of the Available Molecular Methods for Detection of SARS-CoV-2 during the Ongoing Pandemic<ul><li> The two major classification efforts have been produced by GISAID (<uri xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</uri>) and <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</uri>) initiatives, respectively.<li> <b>Nextstrain</b> assigns nomenclature through the designation of SARS-CoV-2 clades to label well-defined clades that reached geographic spread with significant frequency [<xref rid="B19-ijms-22-01298" ref-type="bibr"><u>19</u></xref>].<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33509997">33509997</a> <b>(1)</b> Genome Sequence of a SARS-CoV-2 VUI 202012/01 Strain Identified from a Patient Returning from London, England, to the Apulia Region of Italy<ul><li>The phylogenetic analysis was performed by using the Nextclade sequence analysis Web app (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="B4" ref-type="bibr"><u>4</u></xref>).<li> <b>Nextstrain</b> clades are indicated at the branches.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33509990">33509990</a> <b>(2)</b> Genome Sequencing of a Novel Coronavirus SARS-CoV-2 Isolate from Iraq<ul><li><b>Nextstrain</b> clade</th>              </tr>            </thead>            <tbody>              <tr>                <td rowspan="1" colspan="1">hCoV-19/Iraq/ICGEB-5T.<li> The phylogenetic tree revealed that strain hCoV-19/Iraq/ICGEB-5T belongs to the <b>Nextstrain</b> clade 20A and GISAID clade GH (<xref ref-type="fig" rid="fig1"><u>Fig.&#xA0;1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33510171">33510171</a> <b>(2)</b> Genomic epidemiology of the early stages of the SARS-CoV-2 outbreak in Russia<ul><li> Major SARS-CoV-2 lineages are labeled according to <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR81"><u>81</u></xref></sup> and PANGOLIN|GISAID nomenclature in red and blue, respectively.<li> Sequences shorter than 29,000&#x2009;bp, sequences with more than 300 positions with missing data (Ns), sequences excluded by <b>Nextstrain</b>, and samples corresponding to resequencing of the same patients were removed.<li> [81](Nextstrain. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/">https://nextstrain.org/ncov/</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33728416">33728416</a> <b>(2)</b> VERSO: A comprehensive framework for the inference of robust phylogenies and the quantification of intra-host genomic diversity of viral samples<ul><li>At the time of this writing, numerous independent laboratories around the world are isolating and sequencing SARS-CoV-2 samples and depositing them on public databases (e.g., GISAID<xref rid="bib9" ref-type="bibr"><sup>9</sup></xref>) whose data are accessible via the <b>Nextstrain</b> portal.<li>VERSO step #1 was compared with two state-of-the-art phylogenetic methods from consensus sequences: IQ-TREE,<xref rid="bib10" ref-type="bibr"><sup>10</sup></xref> the algorithmic strategy included in the <b>Nextstrain</b>-Augur pipeline,<xref rid="bib72" ref-type="bibr"><sup>72</sup></xref> and BEAST 2.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33513449">33513449</a> <b>(1)</b> Different SARS-CoV-2 haplotypes associate with geographic origin and case fatality rates of COVID-19 patients<ul><li> The substitutions are ordered on the basis of the date on which they first appeared, as inferred by <b>Nextstrain</b> (<xref rid="bb0045" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and with the most frequent ones being: S.DP.YP.QT, <ext-link ext-link-type="uri" xlink:href="http://S.DP.CL" id="ir0015">S.DP.CL</ext-link>.QT, L.DP.YP.QT, L.GL.YP.QT, <ext-link ext-link-type="uri" xlink:href="http://L.GL.YP.HT" id="ir0020">L.GL.YP.HT</ext-link>, L.GL.YP.HI, L.GP.YP.QT and <ext-link ext-link-type="uri" xlink:href="http://L.GP.YP.HT" id="ir0025">L.GP.YP.HT</ext-link> (see <xref rid="f0005" ref-type="fig">Fig.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33497392">33497392</a> <b>(2)</b> Generalized linear models provide a measure of virulence for specific mutations in SARS-CoV-2 strains<ul><li> The <b>nextstrain</b> [<xref rid="pone.0238665.ref026" ref-type="bibr"><u>26</u></xref>] pipeline was downloaded locally and the commands for filtering, aligning and constructing phylogeny were used according to the <b>nextstrain</b>&#x2019;s best practices.<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33519002">33519002</a> <b>(2)</b> The coronavirus evolves<ul><li>A project called <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="interrefs20"><b>Nextstrain</b>,</ext-link> based at the Fred Hutchinson Cancer Research Center in Seattle, compiles all published viral genome sequences and plots them on a family tree.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33484714">33484714</a> <b>(1)</b> Enterovirus D68 molecular and cellular biology and pathogenesis<ul><li> Visualization of phylogenetic data was performed using <b>Nextstrain</b> (<xref rid="bib230" ref-type="bibr"><u>230</u></xref>).<li> [230](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33482149">33482149</a> <b>(3)</b> Biochemical features and mutations of key proteins in SARS-CoV-2 and their impacts on RNA therapeutics<ul><li> 3</xref>, the database SARS-CoV-2 Variome Structure-Substrate Database is proposed as 1) to accrue and monitor all mutations for three key viral proteins (the S protein, RdRp and M<sup>pro</sup>) in the SARS-CoV-2 viral genome weekly, using sequencing data from the GISAID database and the <b>nextstrain</b> database (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" id="ir035"><underline>https://<b>nextstrain</b>.org/sars-cov-2/</underline></ext-link>); 2) to predict protein conformational changes due to mutations, and to evaluate changes of enzyme activities or antigen-antibody affinities of mutants for three key viral proteins, using computational modeling and machine learning approaches with publicly available 3D structures for the three proteins; and 3) to develop a SARS-CoV-2 Variome Structure-Substrate Database and machine learning tools for evaluating potential effects of mutants on U.S.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33463525">33463525</a> <b>(13)</b> Evidence for adaptive evolution in the receptor-binding domain of seasonal coronaviruses OC43 and 229e<ul><li> copy archived at <ext-link xlink:href="https://archive.softwareheritage.org/swh:1:dir:feb61594144d5787d91049172267d925b738079b;origin=https://github.com/blab/seasonal-cov-adaptive-evolution;visit=swh:1:snp:db7c97d981938da69e10585d540a30e869537903;anchor=swh:1:rev:83721cd000f2848d4f77d2a6da8c2d0df8a555a1/" ext-link-type="uri">swh:1:rev:83721cd000f2848d4f77d2a6da8c2d0df8a555a1</ext-link>).&#xA0;All phylogenetic trees constructed and analyzed in this manuscript can be viewed interactively at <ext-link xlink:href="https://<b>nextstrain</b>.org/community/blab/seasonal-cov-adaptive-evolution" ext-link-type="uri">https://<b>nextstrain</b>.org/community/blab/seasonal-cov-adaptive-evolution</ext-link>; <xref rid="bib31" ref-type="bibr"><u>Mattenberger, 2021</u></xref>.<li> A Snakemake file (<xref rid="bib22" ref-type="bibr"><u>K&#xF6;ster and Rahmann, 2012</u></xref>) within each HCoV directory follows the general outline of a <b>Nextstrain</b> build (<b>Nextstrain</b>, RRID:<ext-link xlink:href="https://identifiers.org/RRID/RRID:SCR_018223" ext-link-type="uri">SCR_018223</ext-link>) and was used to align each gene to a reference strain and build a time-resolved phylogeny with IQ-Tree v1 (<xref rid="bib35" ref-type="bibr"><u>Nguyen et al., 2015</u></xref>) and TimeTree (<xref rid="bib41" ref-type="bibr"><u>Sagulenko et al., 2018</u></xref>).<li> Phylogenies were viewed to identify the distribution of genotypes throughout the tree, different lineages, and signals of recombination using the <b>nextstrain</b> view command (<xref rid="bib10" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li>Influenza and measles alignments were generated by running <b>Nextstrain</b> the respective <b>Nextstrain</b> builds from <ext-link xlink:href="https://github.com/<b>nextstrain</b>/seasonal-flu" ext-link-type="uri">https://github.com/<b>nextstrain</b>/seasonal-flu</ext-link> and <ext-link xlink:href="https://github.com/<b>nextstrain</b>/measles" ext-link-type="uri">https://github.com/<b>nextstrain</b>/measles</ext-link> (<xref rid="bib10" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>;&#xA0;<b>Nextstrain</b>, RRID:<ext-link xlink:href="https://identifiers.org/RRID/RRID:SCR_018223" ext-link-type="uri">SCR_018223</ext-link>).<li> Our code for this analysis is at <monospace>antigenic_evolution/bhatt_<b>nextstrain</b>.ipynb</monospace>.<li> [Mattenberger, 2021](          <person-group person-group-type="author"><name><surname>Mattenberger</surname><given-names>F</given-names></name></person-group>          <year iso-8601-date="2021">2021</year>          <data-title>seasonal-cov-adaptive-evolution</data-title>          <source>Software Heritage</source>          <version designator="swh:1:rev:83721cd000f2848d4f77d2a6da8c2d0df8a555a1">swh:1:rev:83721cd000f2848d4f77d2a6da8c2d0df8a555a1</version>          <ext-link xlink:href="https://archive.softwareheritage.org/swh:1:dir:feb61594144d5787d91049172267d925b738079b;origin=https://github.com/blab/seasonal-cov-adaptive-evolution;visit=swh:1:snp:db7c97d981938da69e10585d540a30e869537903;anchor=swh:1:rev:83721cd000f2848d4f77d2a6da8c2d0df8a555a1/" ext-link-type="uri">https://archive.softwareheritage.org/swh:1:dir:feb61594144d5787d91049172267d925b738079b;origin=https://github.com/blab/seasonal-cov-adaptive-evolution;visit=swh:1:snp:db7c97d981938da69e10585d540a30e869537903;anchor=swh:1:rev:83721cd000f2848d4f77d2a6da8c2d0df8a555a1/</ext-link>        ) [K&#xF6;ster and Rahmann, 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22908215">22908215</a>) [Nguyen et al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [Sagulenko et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Evidence for adaptive evolution in the receptor-binding domain of seasonal coronaviruses OC43 and 229e<ul><li>(https://<b>nextstrain</b>.org/community/blab/seasonal-cov-adaptive-evolution/oc43/spike?c=country)&#x200B; , we can see that the tree forms a ladder-like topology where the tips are organized by date, rather than by country.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33468686">33468686</a> <b>(2)</b> Genome Sequencing of Sewage Detects Regionally Prevalent SARS-CoV-2 Variants<ul><li> By further comparing the probabilities of SNV overlap between patient genotypes and wastewater genotypes at the <b>NextStrain</b> &#x201C;location&#x201D; level (corresponding to counties and/or cities), we found the highest likelihood of nonrandom overlap between all wastewater genotypes observed and clinical genotypes from Alameda County (<xref ref-type="fig" rid="fig2"><u>Fig.&#xA0;2d</u></xref>), the location that the wastewater samples were also derived from.<li> Hypergeometric distributions were calculated with the stats.hypergeom function in scipy (<xref rid="B37" ref-type="bibr"><u>37</u></xref>) to compare wastewater samples to all clinical data from each <b>NextStrain</b> &#x201C;location&#x201D; with at least 20 genomes deposited.<li> [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32015543">32015543</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33468702">33468702</a> <b>(4)</b> SARS-CoV-2 Genomic Variation in Space and Time in Hospitalized Patients in Philadelphia<ul><li> Several nomenclatures have been proposed for this SARS-CoV-2 lineage&#x2014;these include lineage B.1, <b>Nextstrain</b> clade 20A or 20C, and GISAID clade G or GH; an older designation is the A2a clade (<xref rid="B16" ref-type="bibr"><u>16</u></xref>, <xref rid="B17" ref-type="bibr"><u>17</u></xref>).<li> Philadelphia sequences also all encoded P314L in the virus-encoded RdRp (ORF1b), marking them as lineage B.1, <b>Nextstrain</b> clade 20A or 20C, GISAID clade G or GH, and clade A2a (<xref rid="B16" ref-type="bibr"><u>16</u></xref>, <xref rid="B17" ref-type="bibr"><u>17</u></xref>).<li> The lineage B.1/<b>Nextstrain</b> clade 20A or 20C/GISAID clade G or GH/A2a (<xref rid="B16" ref-type="bibr"><u>16</u></xref>, <xref rid="B17" ref-type="bibr"><u>17</u></xref>) variants were circulating in New York (<xref rid="B6" ref-type="bibr"><u>6</u></xref>) prior to expansion of the epidemic in Philadelphia.<li> We find that all of the viral genomes recovered contained the spike D614G substitution suggested to promote spread among humans, and all contained the linked RdRp P314L substitution, marking them as the lineage B.1, <b>Nextstrain</b> clade 20A or 20C, GISAID clade G or GH, and clade A2a (<xref rid="B16" ref-type="bibr"><u>16</u></xref>, <xref rid="B17" ref-type="bibr"><u>17</u></xref>).<li> [16](<person-group person-group-type="author"><string-name><surname>Banu</surname><given-names>S</given-names></string-name>, <string-name><surname>Jolly</surname><given-names>B</given-names></string-name>, <string-name><surname>Mukherjee</surname><given-names>P</given-names></string-name>, <string-name><surname>Singh</surname><given-names>P</given-names></string-name>, <string-name><surname>Khan</surname><given-names>S</given-names></string-name>, <string-name><surname>Zaveri</surname><given-names>L</given-names></string-name>, <string-name><surname>Shambhavi</surname><given-names>S</given-names></string-name>, <string-name><surname>Gaur</surname><given-names>N</given-names></string-name>, <string-name><surname>Mishra</surname><given-names>RK</given-names></string-name>, <string-name><surname>Scaria</surname><given-names>V</given-names></string-name>, <string-name><surname>Sowpati</surname><given-names>DT</given-names></string-name></person-group><year>2020</year><article-title>A distinct phylogenetic cluster of Indian SARS-CoV-2 isolates</article-title>. <source>bioRxiv</source> doi:<pub-id pub-id-type="doi">10.1101/2020.05.31.126136</pub-id>.) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [16](<person-group person-group-type="author"><string-name><surname>Banu</surname><given-names>S</given-names></string-name>, <string-name><surname>Jolly</surname><given-names>B</given-names></string-name>, <string-name><surname>Mukherjee</surname><given-names>P</given-names></string-name>, <string-name><surname>Singh</surname><given-names>P</given-names></string-name>, <string-name><surname>Khan</surname><given-names>S</given-names></string-name>, <string-name><surname>Zaveri</surname><given-names>L</given-names></string-name>, <string-name><surname>Shambhavi</surname><given-names>S</given-names></string-name>, <string-name><surname>Gaur</surname><given-names>N</given-names></string-name>, <string-name><surname>Mishra</surname><given-names>RK</given-names></string-name>, <string-name><surname>Scaria</surname><given-names>V</given-names></string-name>, <string-name><surname>Sowpati</surname><given-names>DT</given-names></string-name></person-group><year>2020</year><article-title>A distinct phylogenetic cluster of Indian SARS-CoV-2 isolates</article-title>. <source>bioRxiv</source> doi:<pub-id pub-id-type="doi">10.1101/2020.05.31.126136</pub-id>.) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [16](<person-group person-group-type="author"><string-name><surname>Banu</surname><given-names>S</given-names></string-name>, <string-name><surname>Jolly</surname><given-names>B</given-names></string-name>, <string-name><surname>Mukherjee</surname><given-names>P</given-names></string-name>, <string-name><surname>Singh</surname><given-names>P</given-names></string-name>, <string-name><surname>Khan</surname><given-names>S</given-names></string-name>, <string-name><surname>Zaveri</surname><given-names>L</given-names></string-name>, <string-name><surname>Shambhavi</surname><given-names>S</given-names></string-name>, <string-name><surname>Gaur</surname><given-names>N</given-names></string-name>, <string-name><surname>Mishra</surname><given-names>RK</given-names></string-name>, <string-name><surname>Scaria</surname><given-names>V</given-names></string-name>, <string-name><surname>Sowpati</surname><given-names>DT</given-names></string-name></person-group><year>2020</year><article-title>A distinct phylogenetic cluster of Indian SARS-CoV-2 isolates</article-title>. <source>bioRxiv</source> doi:<pub-id pub-id-type="doi">10.1101/2020.05.31.126136</pub-id>.) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32471856">32471856</a>) [16](<person-group person-group-type="author"><string-name><surname>Banu</surname><given-names>S</given-names></string-name>, <string-name><surname>Jolly</surname><given-names>B</given-names></string-name>, <string-name><surname>Mukherjee</surname><given-names>P</given-names></string-name>, <string-name><surname>Singh</surname><given-names>P</given-names></string-name>, <string-name><surname>Khan</surname><given-names>S</given-names></string-name>, <string-name><surname>Zaveri</surname><given-names>L</given-names></string-name>, <string-name><surname>Shambhavi</surname><given-names>S</given-names></string-name>, <string-name><surname>Gaur</surname><given-names>N</given-names></string-name>, <string-name><surname>Mishra</surname><given-names>RK</given-names></string-name>, <string-name><surname>Scaria</surname><given-names>V</given-names></string-name>, <string-name><surname>Sowpati</surname><given-names>DT</given-names></string-name></person-group><year>2020</year><article-title>A distinct phylogenetic cluster of Indian SARS-CoV-2 isolates</article-title>. <source>bioRxiv</source> doi:<pub-id pub-id-type="doi">10.1101/2020.05.31.126136</pub-id>.) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33477951">33477951</a> <b>(3)</b> Structural Mapping of Mutations in Spike, RdRp and Orf3a Genes of SARS-CoV-2 in Influenza Like Illness (ILI) Patients<ul><li> Phylogenetic trees were constructed using the <b>NextStrain</b> nCoV automated pipeline using the assembled genomes and associated metadata as inputs (i.e., collection date and patient demographics and travel history).<li> We used temporal and spatial visualization to understand SARS-CoV-2 evolution through space and time that involved filtering large amounts of data by <b>Nextstrain</b> [<xref rid="B37-viruses-13-00136" ref-type="bibr"><u>37</u></xref>].<li> however, <b>Nextstrain</b> analysis shows the presence of GH clade sequences during the month of February reported from Senegal, Canada, Belgium, Singapore, Lebanon and France [<xref rid="B54-viruses-13-00136" ref-type="bibr"><u>54</u></xref>,<xref rid="B55-viruses-13-00136" ref-type="bibr"><u>55</u></xref>].<li> [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [54](          <person-group person-group-type="author">            <name>              <surname>G&#xE1;mbaro</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>Behillil</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Baidaliuk</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Donati</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>Albert</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Alexandru</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Vanpeene</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Bizard</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Brisebarre</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Barbet</surname>              <given-names>M.</given-names>            </name>            <etal/>          </person-group>          <article-title>Introductions and early spread of SARS-CoV-2 in France, 24 January to 23 March 2020</article-title>          <source>Eurosurveillance</source>          <year>2020</year>          <volume>25</volume>          <fpage>2001200</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2020.25.26.2001200</pub-id>        ) [55](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32371096">32371096</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33410308">33410308</a> <b>(1)</b> Pitfalls of barcodes in the study of worldwide SARS-CoV-2 variation and phylodynamics<ul><li> The authors attempt to find a parallel between the <b>Nextstrain</b> phylogeny (<xref rid="bHadfield2018" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and their signatures, which does not clarify the sections of the evolutionary tree being captured.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33490718">33490718</a> <b>(2)</b> Mutational analysis of SARS-CoV-2 <italic>ORF8</italic> during six months of COVID-19 pandemic<ul><li> Despite the fact that the mutation's rate in SARS-CoV-2 is less than that of other RNA viruses, like influenza (<ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="ir0025"><b>Nextstrain</b>.org</ext-link>), 240 different nonsynonymous mutations, from January to June 2020, were detected in the <italic>ORF8</italic> gene leading to nonsense and missense mutations of 7 and 93%, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33467166">33467166</a> <b>(10)</b> Molecular Epidemiological Analysis of the Origin and Transmission Dynamics of the HIV-1 CRF01_AE Sub-Epidemic in Bulgaria<ul><li> We used a local instance of <b>Nextstrain</b> to conduct viral dating and evolutionary rate analyses.<li> <b>Nextstrain</b> uses ML analysis implemented in TreeTime [<xref rid="B23-viruses-13-00116" ref-type="bibr"><u>23</u></xref>,<xref rid="B24-viruses-13-00116" ref-type="bibr"><u>24</u></xref>] and is used for monitoring emerging outbreaks such as COVID-19 in near-real time (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>).<li> Three HIV-1 subtype J sequences were used as the outgroup for the <b>Nextstrain</b> ML analysis.<li> The phylogenetic tree generated by ML analysis in <b>Nextstrain</b> overall shows that the majority of CRF01_AE transmission events are taking place in Asia with some regional isolation after local introduction, as seen for China, but with spread and mixing in other geographic locations as for infections in Thailand and Vietnam (<xref ref-type="fig" rid="viruses-13-00116-f003"><u>Figure 3</u></xref>A).<li><b>Nextstrain</b> ML analysis identified a large clade containing the majority of the Bulgarian CRF01_AE sequences (<xref ref-type="fig" rid="viruses-13-00116-f003"><u>Figure 3</u></xref>A).<li><xref ref-type="fig" rid="viruses-13-00116-f003"><u>Figure 3</u></xref>B shows the inferred transmission routes for the origin and spread of the outbreak cluster using <b>Nextstrain</b>.<li> Our overall CRF01_AE phylogenies, MRCA divergence dates, and evolutionary rate results from the <b>Nextstrain</b> analysis are consistent with those in the literature, including studies using complete genomes, providing strong support for our findings [<xref rid="B8-viruses-13-00116" ref-type="bibr"><u>8</u></xref>,<xref rid="B41-viruses-13-00116" ref-type="bibr"><u>41</u></xref>,<xref rid="B42-viruses-13-00116" ref-type="bibr"><u>42</u></xref>,<xref rid="B43-viruses-13-00116" ref-type="bibr"><u>43</u></xref>].<li> Near the root of the outbreak clade were sequences from Vietnam and China with the <b>Nextstrain</b> analysis inferring Vietnam as the potential origin of the other sequences in the outbreak clade.<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32281388">32281388</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23807275">23807275</a>) [42](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27698457">27698457</a>) [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28127051">28127051</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33306459">33306459</a> <b>(6)</b> SARS-CoV-2 genomic characterization and clinical manifestation of the COVID-19 outbreak in Uruguay<ul><li> We build on the existing <b>Nextstrain</b> framework, a broadly used and universally recognized analytical platform for the analysis of global SARS-CoV-2 viral genome sequence data that has accumulated since the start of the pandemic [<xref rid="CIT0013" ref-type="bibr"><u>13</u></xref>] to analyse newly sequenced SARS-CoV-2 genomes sampled in Uruguay, and combine it with Bayesian Evolutionary Analysis of Sampling Trees (BEAST).<li>To infer our SARS-CoV-2 time-scaled phylogenetic tree, we selected global reference sequences used for the <b>Nextstrain</b> analysis specific for South America as of August 6<sup>th</sup>, 2020.<li> Using the <b>Nextstrain</b> metadata to identify the accessions of interest, we then downloaded the latest whole-genome sequence alignment from the GISAID database.<li> The tree was dated using TreeTime v.0.7.6 [<xref rid="CIT0021" ref-type="bibr"><u>21</u></xref>], specifying a clock rate of 8&#x2009;&#xD7;&#x2009;10<sup>&#x2212;4</sup> substitutions per site per year to replicate the original <b>Nextstrain</b> workflow analysis as faithfully as possible.<li>To determine phylogenetic and epidemiologic characteristics of the Uruguayan SARS-CoV-2 outbreak, we performed a comprehensive set of phylogenetic analyses, including maximum-likelihood trees, haplotype networks, <b>Nextstrain</b>-based phylogenetic placements, and Bayesian phylogeographic analyses using BEAST (<xref rid="F0003" ref-type="fig"><u>Figures 3</u></xref> and <xref rid="F0004" ref-type="fig"><u>4</u></xref>, Figures S5&#x2013;S8).<li>To assess global introductions and the regional spread of SARS-CoV-2 in Uruguay, we performed discrete phylogeographic analysis using BEAST with all 73 Uruguayan sequences, complemented with 1,737 global reference sequences, based on the global subsampling dataset suggested by <b>Nextstrain</b> (Table S2).<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33460733">33460733</a> <b>(2)</b> TSP-based PCR for rapid identification of L and S type strains of SARS-CoV-2<ul><li> Several nomenclatures have been used for SARS-CoV-2 classification including by <b>Nextstrain</b>, GISAID, and <ext-link ext-link-type="uri" xlink:href="http://Cov-lineages.org" id="intref0020">Cov-lineages.org</ext-link> [<xref rid="bib18" ref-type="bibr"><u>18</u></xref>].<li> Clades based on GISAID divides the SARS-CoV-2 into 7 major clades (S, L, O, V, G, GH, GR), while clades based on <b>Nextstrain</b> categories SARS-CoV-2 into 19B (ancestral type), 19A, 20A, 20C, 20B and <ext-link ext-link-type="uri" xlink:href="http://cov-lineages.org" id="intref0025">cov-lineages.org</ext-link> categories SARS-CoV-2 into just two major clades A and B and multiple lineages [<xref rid="bib18" ref-type="bibr"><u>18</u></xref>].<li> [18](          <person-group person-group-type="author">            <name>              <surname>Alm</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Broberg</surname>              <given-names>E.K.</given-names>            </name>            <name>              <surname>Connor</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Komissarov</surname>              <given-names>A.B.</given-names>            </name>            <name>              <surname>Maurer-Stroh</surname>              <given-names>S.</given-names>            </name>          </person-group>          <article-title>Geographical and temporal distribution of SARS-CoV-2 clades in the WHO European region, january to June 2020</article-title>          <source>Euro Surveill</source>          <volume>25</volume>          <issue>32</issue>          <year>2020</year>          <fpage>2001410</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2020.25.32.2001410</pub-id>          <comment>Accessed August 14, 2020</comment>        ) [18](          <person-group person-group-type="author">            <name>              <surname>Alm</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Broberg</surname>              <given-names>E.K.</given-names>            </name>            <name>              <surname>Connor</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Komissarov</surname>              <given-names>A.B.</given-names>            </name>            <name>              <surname>Maurer-Stroh</surname>              <given-names>S.</given-names>            </name>          </person-group>          <article-title>Geographical and temporal distribution of SARS-CoV-2 clades in the WHO European region, january to June 2020</article-title>          <source>Euro Surveill</source>          <volume>25</volume>          <issue>32</issue>          <year>2020</year>          <fpage>2001410</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2020.25.32.2001410</pub-id>          <comment>Accessed August 14, 2020</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33743368">33743368</a> <b>(2)</b> Reinfection, recurrence, or delayed presentation of COVID-19? Case series and review of the literature<ul><li> Viral genomic studies showed the first episode virus particles belongs to GISAID clade V, <b>Nextstrain</b> 19A, and Pangolin Lineage B.2 with a remarkably close probability of 0.99.<li> On the other hand, the second viral genome fit in the GISAID clade V, <b>Nextstrain</b> 19A, and Pangolin Lineage B.1.79 with a probability of 0.70.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33486212">33486212</a> <b>(1)</b> Impact of virus genetic variability and host immunity for the success of COVID-19 vaccines<ul><li> While <b>Nextstrain</b> [<xref rid="bib0900" ref-type="bibr"><u>180</u></xref>,<xref rid="bib0905" ref-type="bibr"><u>181</u></xref>] and GISAID [<xref rid="bib0910" ref-type="bibr"><u>182</u></xref>] databases provide a broad categorization based on globally circulating SARS-CoV-2 diversity, cov.lineages.org database name lineages based on the outbreaks [<xref rid="bib0915" ref-type="bibr"><u>183</u></xref>].<li> [180](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics (Oxford, England)</source>          <volume>34</volume>          <issue>23</issue>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [181](          <chapter-title>Nextstrain: Genomic Epidemiology of Novel Coronavirus - Global Subsampling</chapter-title>          <year>2020</year>          <comment>Accessed: 17 October 17</comment>          <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov" id="intr0035">https://nextstrain.org/ncov</ext-link>        ) [182](          <person-group person-group-type="author">            <name>              <surname>GISAID- Global Initiative on Sharing All Influenza Data</surname>            </name>          </person-group>          <chapter-title>Clade and Lineage Nomenclature Aids in Genomic Epidemiology Studies of Active hCoV-19 Viruses</chapter-title>          <comment>4 Jul Available from:</comment>          <year>2020</year>          <publisher-name>GISAID</publisher-name>          <publisher-loc>Munich</publisher-loc>          <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" id="intr0040">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>        ) [183](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33432321">33432321</a> <b>(5)</b> Bioinformatics resources facilitate understanding and harnessing clinical research of SARS-CoV-2<ul><li> <b>Nextstrain</b> [<xref rid="ref12" ref-type="bibr"><u>12</u></xref>] is an open-source project aiming to provide real-time tracking of pathogen evolution.<li> A special SARS-CoV-2 portal is constructed (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) by <b>Nextstrain</b>, providing the phylogenic structure for different virus strains, epidemic spread features at the global and continental level, geography-based datasets and other analytics and visualizations.<li> According to the phylogenetic information from <b>Nextstrain</b>, there are five main clades for SARS-CoV-2, each can be represented by specific marker mutations.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33457109">33457109</a> <b>(6)</b> Decoding Covid-19 with the SARS-CoV-2 Genome<ul><li> A regularly updated phylogenetic tree of SARS-CoV-2 is publicly available at <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>Nextstrain</b>.org</ext-link> [<xref ref-type="bibr" rid="CR57"><u>57</u></xref>] and is displayed in Fig.&#xA0;<xref rid="Fig4" ref-type="fig"><u>4</u></xref>.<li> Figure <xref rid="Fig4" ref-type="fig"><u>4c</u></xref>, also obtained from <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>Nextstrain</b>.org</ext-link>, demonstrates that mutations have accumulated across the SARS-CoV-2 genome with hotspots frequently found within ORF1b, S and N genes.<li> Images were downloaded from https:<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/ncov/global"><b>nextstrain</b>.org/ncov/global</ext-link> on 24 September 2020 showing 4640 genomes sampled between December 2019 and September 2020 (last updated 18 September 2020).<li> [57](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>SM</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics (Oxford, England)</source>          <year>2018</year>          <volume>34</volume>          <issue>23</issue>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33416890">33416890</a> <b>(7)</b> Bioinformatics resources for SARS-CoV-2 discovery and surveillance<ul><li>&#x2003;<b>Nextstrain</b> analysis platform*.<li><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/%0d">https://<b>nextstrain</b>.org/</ext-link>.<li> For example, a BI-based analysis platform, <b>Nextstrain</b> [<xref rid="ref104" ref-type="bibr"><u>104</u></xref>], analyzes the latest SARS-CoV-2 data from GISAID [<xref rid="ref51" ref-type="bibr"><u>51</u></xref>] and visualizes the spread and evolution of all available SARS-CoV-2 strains in real time (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global/zh">https://<b>nextstrain</b>.org/ncov/global/zh</ext-link>).<li> Online resources, including the CNCB/NGDC database (<ext-link ext-link-type="uri" xlink:href="https://bigd.big.ac.cn/ncov/">https://bigd.big.ac.cn/ncov/</ext-link>), <b>Nextstrain</b> website [<xref rid="ref104" ref-type="bibr"><u>104</u></xref>] and the UCSC Genome Browser for SARS-CoV-2 (<ext-link ext-link-type="uri" xlink:href="http://genome.ucsc.edu/covid19.html">http://genome.ucsc.edu/covid19.html</ext-link>), can display the single-nucleotide polymorphisms at more than 10&#xA0;000 sites across the SARS-CoV-2 genome.<li> [104](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [51](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [104](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33518996">33518996</a> <b>(1)</b> A pandemic like no other<ul><li> and there have been just a handful of pervasive mutations since the pandemic began, according to <b>Nextstrain</b>, an open-source project tracking genetic changes in the virus as it moves through human populations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33413740">33413740</a> <b>(13)</b> Early transmissibility assessment of the N501Y mutant strains of SARS-CoV-2 in the United Kingdom, October to November 2020<ul><li> However, a later 501Y lineage (501Y Variant 2, also named as B.1.1.7 by COVID-19 Genomics Consortium UK (CoG-UK) [<xref rid="r3" ref-type="bibr"><u>3</u></xref>], 20B/501Y.V1 by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) and VOC-202012/01 by Public Health England [<xref rid="r4" ref-type="bibr"><u>4</u></xref>]) started co-circulating with the 501N lineage in England in late September and became the dominant lineage in December.<li><sup>a</sup> 501Y Variant 2 was also named B.1.1.7 by COVID-19 Genomics Consortium UK (CoG-UK) [<xref rid="r3" ref-type="bibr"><u>3</u></xref>], 20B/501Y.V1 by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) and VOC-202012/01 by Public Health England [<xref rid="r4" ref-type="bibr"><u>4</u></xref>].<li> It indicates that the recent 501Y strains in the UK, since August/September 2020, emerged from the 20B clade (<b>Nextstrain</b> nomenclature) and formed two lineages.<li><sup>a</sup> 501Y Variant 2 was also named B.1.1.7 by COVID-19 Genomics Consortium UK (CoG-UK) [<xref rid="r3" ref-type="bibr"><u>3</u></xref>], 20B/501Y.V1 by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) and VOC-202012/01 by Public Health England [<xref rid="r4" ref-type="bibr"><u>4</u></xref>].<li><sup>a</sup> 501Y Variant 2 was also named B.1.1.7 by COVID-19 Genomics Consortium UK (CoG-UK) [<xref rid="r3" ref-type="bibr"><u>3</u></xref>], 20B/501Y.V1 by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) and VOC-202012/01 by Public Health England [<xref rid="r4" ref-type="bibr"><u>4</u></xref>].<li> [3](Rambaut A, Loman N, Pybus O, Barclay W, Barrett J, Carabelli A, et al. Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations. Virological; 2020. Available from: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</ext-link>) [4](Public Health England (PHE). Investigation of novel SARS-COV-2 variant: Variant of Concern 202012/01: Technical briefing document on novel SARS-CoV-2 variant. London: PHE; 21 Dec 2020. Available from: <ext-link ext-link-type="uri" xlink:href="https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201">https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201</ext-link>) [3](Rambaut A, Loman N, Pybus O, Barclay W, Barrett J, Carabelli A, et al. Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations. Virological; 2020. Available from: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</ext-link>) [4](Public Health England (PHE). Investigation of novel SARS-COV-2 variant: Variant of Concern 202012/01: Technical briefing document on novel SARS-CoV-2 variant. London: PHE; 21 Dec 2020. Available from: <ext-link ext-link-type="uri" xlink:href="https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201">https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201</ext-link>) [3](Rambaut A, Loman N, Pybus O, Barclay W, Barrett J, Carabelli A, et al. Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations. Virological; 2020. Available from: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</ext-link>) [4](Public Health England (PHE). Investigation of novel SARS-COV-2 variant: Variant of Concern 202012/01: Technical briefing document on novel SARS-CoV-2 variant. London: PHE; 21 Dec 2020. Available from: <ext-link ext-link-type="uri" xlink:href="https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201">https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201</ext-link>) [3](Rambaut A, Loman N, Pybus O, Barclay W, Barrett J, Carabelli A, et al. Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations. Virological; 2020. Available from: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</ext-link>) [4](Public Health England (PHE). Investigation of novel SARS-COV-2 variant: Variant of Concern 202012/01: Technical briefing document on novel SARS-CoV-2 variant. London: PHE; 21 Dec 2020. Available from: <ext-link ext-link-type="uri" xlink:href="https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201">https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a> <b>(6)</b> Augur: a bioinformatics toolkit for phylogenetic analyses of human pathogens<ul><li> Bedford, 2015</xref>) and <b>Nextstrain</b> (<xref rid="R6" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) applications.<li> When <b>Nextstrain</b> replaced nextflu and expanded to support multiple viral and bacterial pathogens, each pathogen received its own copy of the original script.<li> Critically, this software architecture led to long-lived, divergent branches of untested code in version control that <b>Nextstrain</b> team members could not confidently merge without potentially breaking existing analyses.<li> Users can apply a series of Augur commands and other standard bioinformatics tools to these files to create annotated phylogenies that can be viewed in Auspice, the web application that serves <b>Nextstrain</b> (<xref rid="F1" ref-type="fig"><u>Figure 1</u></xref>).<li> <xref rid="R16" ref-type="bibr">The <b>Nextstrain</b> Team, 2020</xref>).<li> Augur can be installed from PyPI (<b>nextstrain</b>-augur) and Bioconda (augur).<li> [Hadfield et al., 2018](<name><surname>Hadfield</surname><given-names>J</given-names></name>, <name><surname>Megill</surname><given-names>C</given-names></name>, <name><surname>Bell</surname><given-names>SM</given-names></name>, <name><surname>Huddleston</surname><given-names>J</given-names></name>, <name><surname>Potter</surname><given-names>B</given-names></name>, <name><surname>Callender</surname><given-names>C</given-names></name>, <name><surname>Sagulenko</surname><given-names>P</given-names></name>, <name><surname>Bedford</surname><given-names>T</given-names></name>, &amp; <name><surname>Neher</surname><given-names>RA</given-names></name> (<year>2018</year>). <article-title>Nextstrain: Real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics</source>, <fpage>bty407</fpage>. <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33413422">33413422</a> <b>(2)</b> Transcriptome of nasopharyngeal samples from COVID-19 patients and a comparative analysis with other SARS-CoV-2 infection models reveal disparate host responses against SARS-CoV-2<ul><li>Exploring the <b>Nextstrain</b> portal [<xref ref-type="bibr" rid="CR32"><u>32</u></xref>], we noticed that our analyzed SARS-CoV-2 sequences are closely placed to the Saudi-Arabian isolates (Additional file <xref rid="MOESM2" ref-type="media"><u>2</u></xref>: Figure S1A); although, most of the other isolates of this country were placed in the major European clusters (data not shown).<li><bold>Additional file 2: Figure S1.</bold><bold>A.</bold> Snapshot of <b>Nextstrain</b> data portal showing the phylogenetic relationship of two SARS-CoV-2 isolates used in this study.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Transcriptome of nasopharyngeal samples from COVID-19 patients and a comparative analysis with other SARS-CoV-2 infection models reveal disparate host responses against SARS-CoV-2<ul><li> A phylogenetic analysis generated using <b>Nextstrain</b> (<xref rid="B7" ref-type="bibr"><u>7</u></xref>), rooted relative to the early samples from Wuhan, revealed that two of the eight samples (i.e., EPI_ISL_594155 and EPI_ISL_594156) belong to clade 20B, while the rest belong to clade 20C.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(3)</b> Transcriptome of nasopharyngeal samples from COVID-19 patients and a comparative analysis with other SARS-CoV-2 infection models reveal disparate host responses against SARS-CoV-2<ul><li>Phylogenetic tree of the genome sequences of SARS-CoV-2/human/KOR/YS006/2020 and SARS-CoV-2/human/KOR/YS008/2020, along with 58 other randomly chosen sequences, including 30 sequences of South Korean isolates of SARS-CoV-2 retrieved from <b>Nextstrain</b> (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://<b>nextstrain</b>.org" ext-link-type="uri"><b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> Transcriptome of nasopharyngeal samples from COVID-19 patients and a comparative analysis with other SARS-CoV-2 infection models reveal disparate host responses against SARS-CoV-2<ul><li> According to the NextClade tool from <b>Nextstrain</b> (<xref rid="B8" ref-type="bibr"><u>8</u></xref>), all variants were classified in the 20B clade.<li> Those PANGOLIN and <b>Nextstrain</b> clusters were previously reported as containing representative groups of sequenced Peruvian SARS-CoV-2 strains (<xref rid="B9" ref-type="bibr"><u>9</u></xref>).<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<person-group person-group-type="author"><string-name><surname>Juscamayta-L&#xF3;pez</surname><given-names>E</given-names></string-name>, <string-name><surname>Tarazona</surname><given-names>D</given-names></string-name>, <string-name><surname>Valdivia</surname><given-names>F</given-names></string-name>, <string-name><surname>Rojas</surname><given-names>N</given-names></string-name>, <string-name><surname>Carhuaricra</surname><given-names>D</given-names></string-name>, <string-name><surname>Maturrano</surname><given-names>L</given-names></string-name>, <string-name><surname>Gavil&#xE1;n</surname><given-names>R</given-names></string-name></person-group>. <year>2020</year>. <article-title>Phylogenomic reveals multiple introductions and early spread of SARS-CoV-2 into Peru</article-title>. <source>BioRxiv</source> doi:<pub-id pub-id-type="doi">10.1101/2020.09.14.296814</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=<person-group person-group-type="author"><string-name><surname>Juscamayta-L&#xF3;pez</surname><given-names>E</given-names></string-name>, <string-name><surname>Tarazona</surname><given-names>D</given-names></string-name>, <string-name><surname>Valdivia</surname><given-names>F</given-names></string-name>, <string-name><surname>Rojas</surname><given-names>N</given-names></string-name>, <string-name><surname>Carhuaricra</surname><given-names>D</given-names></string-name>, <string-name><surname>Maturrano</surname><given-names>L</given-names></string-name>, <string-name><surname>Gavil&#xE1;n</surname><given-names>R</given-names></string-name></person-group>. <year>2020</year>. <article-title>Phylogenomic reveals multiple introductions and early spread of SARS-CoV-2 into Peru</article-title>. <source>BioRxiv</source> doi:<pub-id pub-id-type="doi">10.1101/2020.09.14.296814</pub-id>."><person-group person-group-type="author"><string-name><surname>Juscamayta-L&#xF3;pez</surname><given-names>E</given-names></string-name>, <string-name><surname>Tarazona</surname><given-names>D</given-names></string-name>, <string-name><surname>Valdivia</surname><given-names>F</given-names></string-name>, <string-name><surname>Rojas</surname><given-names>N</given-names></string-name>, <string-name><surname>Carhuaricra</surname><given-names>D</given-names></string-name>, <string-name><surname>Maturrano</surname><given-names>L</given-names></string-name>, <string-name><surname>Gavil&#xE1;n</surname><given-names>R</given-names></string-name></person-group>. <year>2020</year>. <article-title>Phylogenomic reveals multiple introductions and early spread of SARS-CoV-2 into Peru</article-title>. <source>BioRxiv</source> doi:<pub-id pub-id-type="doi">10.1101/2020.09.14.296814</pub-id>.</a> <b>(5)</b> Transcriptome of nasopharyngeal samples from COVID-19 patients and a comparative analysis with other SARS-CoV-2 infection models reveal disparate host responses against SARS-CoV-2<ul><li> <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MN328061" ext-link-type="uri" assigning-authority="genbank">MN328061</ext-link>) and subsequently contextualized in the dengue virus serotype 2 build of <b>Nextstrain</b> (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/dengue/denv2" ext-link-type="uri">https://<b>nextstrain</b>.org/dengue/denv2</ext-link>) among 1,202 available dengue virus serotype 2 genomes sampled between 1944 and 2019 (<xref rid="B6" ref-type="bibr"><u>6</u></xref>) (<xref rid="fig1" ref-type="fig"><u>Fig.&#xA0;1A</u></xref>).<li>Phylogenetic trees of dengue virus serotype 2 (DENV2) strains rendered using <b>Nextstrain</b>.<li> <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MN328061" ext-link-type="uri" assigning-authority="genbank">MN328061</ext-link>) contextualized with 1,202 dengue virus serotype 2 genomes available on <b>Nextstrain</b>.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33299552">33299552</a> <b>(6)</b> A proposed molecular mechanism for pathogenesis of severe RNA-viral pulmonary infections<ul><li> The particular strains that were selected were those that showed maximum divergence from one other based on analyses by<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>NextStrain</b></ext-link> (which tracks the genomic epidemiology of SARS-CoV-2<sup><xref rid="ref-44" ref-type="bibr"><u>44</u></xref></sup>).<li> Four Influenza A strains (H3N2) from four separate clades (analogous to the SARS-CoV-2 strain selection procedure using <b>NextStrain</b><sup><xref rid="ref-44" ref-type="bibr"><u>44</u></xref></sup>; A/Denmark/316/2020; A/England/323/2019; A/Singapore/TT0333/2019; and A/Sydney/1017/2018) along with the two Influenza A strains previously selected (A/swine/Ontario/104-25/2012 and A/Duck/Shanghai/C84/2009) were analyzed and their genomes scanned for the presence of strong RNPS1, SRSF1 and hnRNP A1 binding sites (extended data<sup><xref rid="ref-39" ref-type="bibr"><u>39</u></xref></sup> Section 1 &#x2013; Table 4).<li> <b>NextStrain</b> (which provides real-time tracking of the<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org./ncov/">SARS-CoV-2</ext-link> and<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org./flu/">Influenza A</ext-link>) was utilized to choose divergent strains of either virus by selecting strains from separate clades, i.e.<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [39](<person-group person-group-type="author"><name><surname>Rogan</surname><given-names>PK</given-names></name><name><surname>Mucaki</surname><given-names>EJ</given-names></name><name><surname>Shirley</surname><given-names>BC</given-names></name></person-group>:<article-title>Characteristics of human and viral RNA binding sites and site clusters recognized by SRSF1 and RNPS1</article-title><source><italic toggle="yes">Zenodo.</italic></source><year>2020</year><pub-id pub-id-type="doi">10.5281/zenodo.3737089</pub-id>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33402722">33402722</a> <b>(3)</b> Estimating and explaining the spread of COVID-19 at the county level in the USA<ul><li>Before analyzing the full GISAID data, we analyzed a subset from <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR29"><u>29</u></xref></sup> na&#xEF;vely, without engaging the specific hypothesis that the G614 mutation increased transmission.<li> This na&#xEF;ve analysis considered strains from <b>Nextstrain</b> clades 19A, 19B, 20A, 20B, and 20C; clades 19A and 19B do not contain the G614 mutation, but the other clades do.<li> Nonetheless, we repeated analyses using alternative datasets giving county-level characteristics, and also an alternative dataset on SARS-CoV-2 strains (<xref rid="MOESM2" ref-type="media"><u>Supplementary Methods</u></xref>: Analysis of <b>Nextstrain</b> metadata of SARS-CoV-2 strains), and all of the conclusions were the same.<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33400131">33400131</a> <b>(4)</b> European context of the diversity and phylogenetic position of SARS-CoV-2 sequences from Polish COVID-19 patients<ul><li> A parallel school of SARS-CoV-2 nomenclature, with year-based clades A, B, and C, is used in the US-based ncov-<b>NextStrain</b> database (Gudbjartsson et al.<li> however, references to <b>NextStrain</b> database names were also provided where possible (Table <xref rid="Tab1" ref-type="table"><u>1</u></xref>).<li> GISAID clades are as in Mercatelli and Giorgio (<xref ref-type="bibr" rid="CR27"><u>2020</u></xref>), <b>NextStrain</b> as in Gudbjartsson et al.<li> derived</p></caption><table frame="hsides" rules="groups"><thead><tr><th>Nucleotide variant</th><th>Effect in protein</th><th>Localization in the genome</th><th>GISAID clades and haplogroups defined by the mutations</th><th><b>NextStrain</b> Clades</th><th>Occurrence in Polish samples</th></tr></thead><tbody><tr><td>C241T.<li> [2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32793182">32793182</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33236011">33236011</a> <b>(2)</b> Transmission of SARS-CoV-2 in domestic cats imposes a narrow bottleneck<ul><li>Focal <b>Nextstrain</b> build of S H655Y sequences.<li> set for a <b>Nextstrain</b> phylogenetic analysis, to which &#x2018;context&#x2019; sequences were added, with the most genetically similar sequences given priority.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33818474">33818474</a> <b>(26)</b> Phylogenetic classification of the whole-genome sequences of SARS-CoV-2 from India &amp; evolutionary trends<ul><li> On the other hand, <b>Nextstrain</b><xref rid="ref7" ref-type="bibr"><u>7</u></xref> classified the SARS-CoV-2 initially into about nine clades referred to as A1a, A2, A2a, A3, A6, B, B1, B2 and B4.<li> In this study, we describe the genetic lineages of the strains circulating in India as retrieved from GISAID and provide integration for the SARS-CoV-2 classification systems developed by GISAID, <b>Nextstrain</b> and Pangolin.<li> Further, the classification of the Indian sequences into the <b>Nextstrain</b> assigned new clades and the Pangolin nomenclature for clades/sub-clades was done using the respective tools directly.<li>Establishing an equivalence between the Global Initiative on Sharing All Influenza Data (GISAID), <b>Nextstrain</b> and Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin) nomenclature systems with respect to the genome sequence data from India (n=3014).<li><b>Nextstrain</b></th>              <th align="left" rowspan="1" colspan="1">Dominant Pangolin lineages</th>              <th align="left" rowspan="1" colspan="1">Major marker mutations</th>            </tr>          </thead>          <tbody>            <tr>              <td align="left" rowspan="1" colspan="1">G.<li>Distribution of the Indian sequences (n=3014) in the Global Initiative on Sharing All Influenza Data, <b>Nextstrain</b> and Phylogenetic Assignment of Named Global Outbreak LINeages clades.<li><b>Nextstrain</b> new clades<sup>#</sup></th>              <th align="center" rowspan="1" colspan="1">Pangolin</th>            </tr>          </thead>          <tbody>            <tr>              <td align="left" rowspan="1" colspan="1">O.<li>On the basis of the new nomenclature by <b>Nextstrain</b> as per the ancestral nodes, majority of the sequences fell into the cluster having ancestral nodes as 20A and 20B and others fell into clusters with nodes as 19A, 19B and 20C (<xref rid="T2" ref-type="table"><u>Supplementary Table II</u></xref>).<li> The <b>Nextstrain</b> clade assignment was retrieved as on 14 October 2020.<li> Extrapolating to the <b>Nextstrain</b> old nomenclature for classification, it could be seen that the Indian strains could be classified into clades A2a, A1a, A3, B, B4 and O (<xref rid="T4" ref-type="table"><u>Supplementary Table III</u></xref> (available from <italic><ext-link ext-link-type="uri" xlink:href="https://www.ijmr.org.in/articles/2021/153/1/images/IndianJMedRes_2021_153_1_166_311942_sm11.pdf">https://www.ijmr.org.in/articles/2021/153/1/images/IndianJMedRes_2021_153_1_166_311942_sm11.pdf</ext-link></italic>)).<li>Mutations representing the old <b>Nextstrain</b> clade and corresponding major new <b>Nextstrain</b> clade nomenclatures.<li><b>Nextstrain</b> (old clades)</th>              <th align="left" rowspan="1" colspan="1">Mutation(s) defined for the clade</th>              <th align="left" rowspan="1" colspan="1">Major <b>Nextstrain</b> new clades (defined mutation)</th>            </tr>          </thead>          <tbody>            <tr>              <td align="left" rowspan="1" colspan="1">A1a.<li><b>Nextstrain</b></th>              <th align="center" rowspan="1" colspan="1">Lineage</th>              <th align="center" rowspan="1" colspan="1">Andhra Pradesh</th>              <th align="center" rowspan="1" colspan="1">Assam</th>              <th align="center" rowspan="1" colspan="1">Bihar</th>              <th align="center" rowspan="1" colspan="1">Delhi</th>              <th align="center" rowspan="1" colspan="1">Gujarat</th>              <th align="center" rowspan="1" colspan="1">Haryana</th>              <th align="center" rowspan="1" colspan="1">Jammu and Kashmir</th>              <th align="center" rowspan="1" colspan="1">Karnataka</th>              <th align="center" rowspan="1" colspan="1">Kerala</th>              <th align="center" rowspan="1" colspan="1">Ladakh</th>            </tr>          </thead>          <tbody>            <tr>              <td align="left" rowspan="1" colspan="1">G.<li><b>Nextstrain</b> inference (<inline-supplementary-material xlink:title="local_file" xlink:href="IJMR-153-166_Suppl1.tif" xlink:type="simple" id="d24e5663" content-type="local-data">Supplementary Fig.<li> 2</inline-supplementary-material> (available from <italic><ext-link ext-link-type="uri" xlink:href="https://www.ijmr.org.in/articles/2021/153/1/images/IndianJMedRes_2021_153_1_166_311942_sm14.pdf">https://www.ijmr.org.in/articles/2021/153/1/images/IndianJMedRes_2021_153_1_166_311942_sm14.pdf</ext-link></italic>)) of the most likely transmission events (<italic><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/asia?c=clade_membership&amp;f_country=India&amp;f_region=Asia">https://<b>nextstrain</b>.org/ncov/asia?c=clade_membership&amp;f_country=India&amp;f_region=Asia</ext-link></italic>) revealed that the dominant clade B.6 (GISAID O) that emerged from 19A was introduced into India from China, Europe, South-East Asia and Middle-East while B.1 (G) and B.1.36 (GH) that emerged from 20A had their origins from Europe, Middle-East and Africa.<li> On the other hand, <b>Nextstrain</b> is based on a maximum likelihood approach as implemented in TreeTime<xref rid="ref19" ref-type="bibr"><u>19</u></xref>.<li> Considering temporal dating of ancestral nodes and discrete trait geographic reconstruction based on the SARS-CoV-2 sequences, <b>Nextstrain</b> identified five nodes that were labelled as 19A, 19B, 20A, 20B and 20C.<li>Based on the equivalence between the GISAID clade nomenclature, the new <b>Nextstrain</b> clades and the Pangolin sub-lineages, initially, <b>Nextstrain</b> clade names were <italic>ad hoc</italic> letter number combinations that were never intended to be a permanent naming system.<li> Thus, the old <b>Nextstrain</b> clade nomenclature was found to be undefined and did not reflect on the time scale of evolution.<li> We further analyzed the predominance of the strains in different Indian States based on the Pangolin and GISAID clade nomenclatures (<xref rid="T5" ref-type="table"><u>Supplementary Table IV</u></xref>) in association with their emergence times as per the <b>Nextstrain</b> new clades classification nomenclature.<li> The same <b>Nextstrain</b> clades/Pangolin lineages were found to occur in multiple GISAID clades.<li>Global transmissions as captured from <b>Nextstrain</b> analyses.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33398302">33398302</a> <b>(0)</b> Neutralising antibodies in Spike mediated SARS-CoV-2 adaptation<ul><li> Major SARS-CoV-2 clade memberships were assigned to all sequences using both the Nextclade server v0.9 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and Phylogenetic Assignment Of Named Global Outbreak Lineages (pangolin)<sup><xref rid="R35" ref-type="bibr"><u>35</u></xref></sup>.<li> [35](<name><surname>Rambaut</surname><given-names>A.</given-names></name><etal/><article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>. <source>Nature Microbiology</source><volume>5</volume>, <fpage>1403</fpage>&#x2013;<lpage>1407</lpage>, doi:<pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id> (<year>2020</year>).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33370399">33370399</a> <b>(10)</b> <italic>Helicobacter pylori</italic> diversification during chronic infection within a single host generates sub-populations with distinct phenotypes<ul><li>To display the genetic relatedness of all the isolates in the collection, a maximum likelihood tree was generated with consensus sequences from the SNP calls using the <b>Nextstrain</b> platform (Figs <xref ref-type="fig" rid="ppat.1008686.g004"><u>4A</u></xref> and <xref ref-type="supplementary-material" rid="ppat.1008686.s001"><u>S1A</u></xref>) [<xref rid="ppat.1008686.ref055" ref-type="bibr"><u>55</u></xref>].<li>(<bold>A</bold>) A maximum likelihood tree was generated from all SNPs in the collection with <b>Nextstrain</b> [<xref rid="ppat.1008686.ref055" ref-type="bibr"><u>55</u></xref>].<li> The maximum-likelihood tree (Figs <xref ref-type="fig" rid="ppat.1008686.g004"><u>4A</u></xref>, <xref ref-type="fig" rid="ppat.1008686.g005"><u>5A</u></xref> and <xref ref-type="fig" rid="ppat.1008686.g007"><u>7A</u></xref>) was built in IQtree implemented in <b>Nextstrain</b> v 1.8.1, using a general time reversible (GTR) substitution model.<li> The interactive tree can be found at the <b>Nextstrain</b> community site &lt;<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/salama-lab/Hp-J99">https://<b>nextstrain</b>.org/community/salama-lab/Hp-J99</ext-link>&gt;.<li> Time scaled trees and molecular clock rates (<xref ref-type="supplementary-material" rid="ppat.1008686.s002"><u>S2 Fig</u></xref>) were generated with TreeTime implemented within <b>Nextstrain</b>, which approximates divergence times and molecular clock rates using maximum likelihood methods [<xref rid="ppat.1008686.ref075" ref-type="bibr"><u>75</u></xref>].<li> (<bold>A</bold>) The maximum likelihood tree constructed in <b>Nextstrain</b> (<xref ref-type="fig" rid="ppat.1008686.g004"><u>Fig 4A</u></xref>, &lt;<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/salama-lab/Hp-J99">https://<b>nextstrain</b>.org/community/salama-lab/Hp-J99</ext-link>&gt;) and (<bold>B</bold>) the recombination corrected phylogeny created in ClonalFrameML with rescaled branch lengths are shown.<li> [55](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29291710">29291710</a>) [55](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29291710">29291710</a>) [75](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a> <b>(2)</b> <italic>Helicobacter pylori</italic> diversification during chronic infection within a single host generates sub-populations with distinct phenotypes<ul><li> On the <b>NextStrain</b> platform, the scale indicates &#x201C;Divergence&#x201D;, which seems to correspond to the accumulation of changes since the common ancestor and not a rate like &#x201C;substitution per site per year&#x201D; seems to indicate.<li>8.) Figure 4/5/6/7: The common ancestor of the recent group of isolates appears to only date back to a few months before the sampling date (using the year scale on the <b>Nextstrain</b> platform).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33372181">33372181</a> <b>(1)</b> In silico discovery of antigenic proteins and epitopes of SARS-CoV-2 for the development of a vaccine or a diagnostic approach for COVID-19<ul><li> In addition to this reference genome, according to <b>Nextstrain</b> results showing the prevalent variations, variant D614G corresponding for S protein, variant T175M corresponding for M protein, variants P13L, S194L, S197L, R203K, and G204R corresponding for N protein, and variants S24L, V62L, and L84S corresponding for orf8 were selected as additional proteins to investigate the immunological effects associated with antigenicity, signal peptide, BetaWrap motifs, PTMs and, epitope regions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33205709">33205709</a> <b>(4)</b> Loss of orf3b in the circulating SARS-CoV-2 strains<ul><li>SARS-CoV-2 sequences and sequence author data were downloaded from GISAID depository and <b>NextStrain</b>.org [<xref rid="CIT0016" ref-type="bibr"><u>16</u></xref>].<li>To better visualize current circulation of the two distinct orf3b genotypes, we analysed sequence statistics of available resources on <b>NextStrain</b>.org [<xref rid="CIT0016" ref-type="bibr"><u>16</u></xref>] and GISAID sequence depository (Supplementary Table S1).<li> Sequence data from Dec 2019 to Aug 2020 was obtained from <b>NextStrain</b>.<li>To determine the extent of circulation of &#x394;3b viruses, we compiled and analysed sequence data from GISAID and <b>NextStrain</b>.org by month (<xref rid="F0003" ref-type="fig"><u>Figure 3</u></xref>).<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33348368">33348368</a> <b>(3)</b> A review on viral data sources and search systems for perspective mitigation of COVID-19<ul><li> <b>NextStrain</b> [<xref rid="ref13" ref-type="bibr"><u>13</u></xref>] (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>) overviews emergent viral outbreaks based on the visualization of sequence data integrated with geographic information, serology and host species.<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33451855">33451855</a> <b>(2)</b> Dysregulation of Cell Signaling by SARS-CoV-2<ul><li> Source: latest global SARS-CoV-2 updated daily at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2" id="ir0005">https://<b>nextstrain</b>.org/sars-cov-2</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33451855">33451855</a> <b>(4)</b> Dysregulation of Cell Signaling by SARS-CoV-2<ul><li> The phylogenetic analysis was performed following the standard protocol for analysis of SARS-CoV-2 genomes provided by <b>Nextstrain</b> (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://<b>nextstrain</b>.org/ncov" ext-link-type="uri">http://<b>nextstrain</b>.org/ncov</ext-link>) (<xref rid="B2" ref-type="bibr"><u>2</u></xref>).<li> The figure was created using <b>Nextstrain</b>.<li> [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33332472">33332472</a> <b>(6)</b> Genomic epidemiology reveals multiple introductions and spread of SARS-CoV-2 in the Indian state of Karnataka<ul><li> These sequences could be placed in the global context with <b>Nextstrain</b>, an online analysis platform that allows the tracking of pathogen sequences in real time [<xref rid="pone.0243412.ref015" ref-type="bibr"><u>15</u></xref>].<li>An analysis of 361 complete genome sequences from India showed that five global clades were circulating in India&#x2013;old <b>Nextstrain</b> clades B, B4, A2a, A3, and a distinct clade A3i [<xref rid="pone.0243412.ref016" ref-type="bibr"><u>16</u></xref>].<li>Of the lineages in the state, B.1 (related to GISAID clade G, and <b>Nextstrain</b> clade A2a) and B.1.1 (related to GISAID clade GR and <b>Nextstrain</b> clade A2a) are European clades.<li>The largest cluster in the state, C1 (comprising of 67 individuals) had four sequenced samples belonging to lineage B.1 and three sequenced samples belonging to lineage B.1.1 (<b>Nextstrain</b> clade A2a).<li> Of the sequenced samples from this cluster, nine were assigned to lineage B.4 (<b>Nextstrain</b> A3).<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [16](Banu S, Jolly B, Mukherjee P, Singh P, Khan S, Zaveri L, et al. A distinct phylogenetic cluster of Indian SARS-CoV-2 isolates. bioRxiv. 2020.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33325750">33325750</a> <b>(4)</b> Genomic analysis of SARS-CoV-2 reveals local viral evolution in Ghana<ul><li> We performed basic phylogenetic analysis on SARS-CoV-2 genomic data using <b>Nextstrain</b> pipelines.<li>Phylogenetic analysis on SARS-CoV-2 using the <b>Nextstrain</b> pipelines showed time-resolved phylogenies of samples from the first and second sequencing batches (<xref ref-type="fig" rid="fig2-1535370220975351"><u>Figure 2</u></xref>).<li> Legacy clade assignment based on phenotypic variation<sup><xref rid="bibr30-1535370220975351" ref-type="bibr"><u>30</u></xref></sup> and &#x2018;<b>Nextstrain</b> clade&#x2019; assignment based on nucleotide polymorphisms.<li> The first batch of 15 sequences clustered to several clades (nomenclature-GISAID/Legacy/<b>Nextstrain</b>): 13% 19&#x2009;A/V/undetermined, 34% 19B/S/B4, 20% 20&#x2009;A/G/A2a, and 13% 20B/GR/A2a and 20% 20&#x2009;C/GH/A2a (<xref ref-type="fig" rid="fig3-1535370220975351"><u>Figure 3(a) and (b)</u></xref>).<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32335334">32335334</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32335334">32335334</a> <b>(9)</b> Genomic analysis of SARS-CoV-2 reveals local viral evolution in Ghana<ul><li> Molecular epidemiology has attempted to provide a detailed picture about the distinct lineages and substrains circulating in different geographic areas as well as about the dispersal pattern and cross-border transmissions at different time periods during the pandemic (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://<b>nextstrain</b>.org/ncov/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global</ext-link>; <xref rid="msaa314-B19" ref-type="bibr">Hadfield et&#xA0;al.<li> In almost all cases, including the daily updated virus phylogenies on the exceptional <b>nextstrain</b> platform, phylogenetic inference on the currently available virus genomes is conducted predominantly via standard maximum likelihood (ML)-based tools using default program parameters.<li> First, one can use biologically reasonable <italic toggle="yes">ad hoc</italic> criteria as we already apply them here by removing singleton and duplicate sequences, or as deployed by <b>nextstrain</b> that removes sequences that are below a certain length threshold.<li> In addition, <b>nextstrain</b> randomly subsamples sequences within predefined geospatial groups, to yield the inference process more computationally tractable.<li> 2020</xref>) or for the <b>nextstrain</b> and GISAD naming schemes.<li> The results of <b>nextstrain</b> analyses, where major clades were detected for the United States and other regions, but also a considerable number of cross border transmissions was reported, support our findings.<li> We chose the pangolin tool, as it is easy to use and the three alternative naming schemes implemented in GISAID, <b>nextstrain</b>, and the Pangolin tool do not appear to exhibit major differences (<xref rid="msaa314-B1" ref-type="bibr">Alm et&#xA0;al.<li> corresponding to lineage <italic toggle="yes">19B</italic> and <italic toggle="yes">S</italic> according to the <b>nextstrain</b> and GISAID classifications, respectively) which is considered as the root lineage of the pandemic, it is not among the earliest sampled sequences from China and other Asian regions in our analyses.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34713073">34713073</a> <b>(1)</b> Data Sharing and Global Public Health: Defining What We Mean by Data<ul><li> <b>Nextstrain</b> (<xref rid="B12" ref-type="bibr"><u>12</u></xref>).<li> [12](<person-group person-group-type="author"><collab>The Nextstrain team</collab></person-group>. <source>Genomic Epidemiology of Novel Coronavirus - Global Subsampling</source>. (<year>2020</year>). Available online at: <ext-link xlink:href="https://nextstrain.org/ncov/global" ext-link-type="uri">https://nextstrain.org/ncov/global</ext-link> (accessed September 23, 2020).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33330863">33330863</a> <b>(2)</b> Oligonucleotide Capture Sequencing of the SARS-CoV-2 Genome and Subgenomic Fragments from COVID-19 Individuals<ul><li> Introduced stop codons are rare among the submitted genome assemblies tracking the evolution of SARS-CoV2 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>), but are distributed all along the genome (<xref rid="SD1" ref-type="supplementary-material"><u>S4 Fig</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33303686">33303686</a> <b>(1)</b> Phylogenetic analysis of SARS-CoV-2 in Boston highlights the impact of superspreading events<ul><li>We constructed phylogenetic maximum likelihood (ML) and time trees with associated visualizations using the Augur pipeline (<italic>augur_with_assemblies</italic>) and SARS-CoV-2-specific procedures taken from github.com/<b>nextstrain</b>/ncov for our 772 genomes and a representative background set of 4,011 subsampled from the GISAID database on 15 June, 2020.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32909002">32909002</a> <b>(1)</b> Limited intestinal inflammation despite diarrhea, fecal viral RNA and SARS-CoV-2-specific IgA in patients with acute COVID-19<ul><li> Phylogenetic analysis was performed using <b>Nextstrain</b> build for SARS-CoV-2 genomes with default parameters[<xref rid="R56" ref-type="bibr"><u>56</u></xref>].<li> The mutation profiles were analyzed with the tool NextClade v0.3.6 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [56](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33302501">33302501</a> <b>(1)</b> Comprehensive Structural and Molecular Comparison of Spike Proteins of SARS-CoV-2, SARS-CoV and MERS-CoV, and Their Interactions with ACE2<ul><li> A sequence analysis from 2310 viral isolates from <b>Nextstrain</b> reveals that the residue at position 614 of the viral spike protein is changed from a putative ancestral aspartic acid (D) to a glycine (G) between two viral clades.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33279989">33279989</a> <b>(1)</b> Next generation sequencing of SARS-CoV-2 genomes: challenges, applications and opportunities<ul><li> The <b>Nextstrain</b> [<xref rid="ref128" ref-type="bibr"><u>128</u></xref>] and the Hyphy COVID-19 [<xref rid="ref129" ref-type="bibr"><u>129</u></xref>] portals are further notable examples of highly flexible and interactive systems for the real time monitoring of the evolution SARS-CoV-2 strains.<li> [128](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [129](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15509596">15509596</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33291299">33291299</a> <b>(2)</b> Multiple SARS-CoV-2 Introductions Shaped the Early Outbreak in Central Eastern Europe: Comparing Hungarian Data to a Worldwide Sequence Data-Matrix<ul><li> This observation is further supported by a narrative analysis on the <b>Nextstrain</b> online platform focusing on Eastern European processes of SARS-CoV-2 pandemic evolution [<xref rid="B26-viruses-12-01401" ref-type="bibr"><u>26</u></xref>].<li> In order to leverage additional support regarding this phenomenon, we applied a local <b>Nextstrain</b> database workflow in the addition of the sequences from this manuscript (<xref ref-type="app" rid="app1-viruses-12-01401"><u>Supplementary Figure S3</u></xref>) [<xref rid="B27-viruses-12-01401" ref-type="bibr"><u>27</u></xref>].<li> [26](          <article-title>Nextstrain/Narratives/Ncov/Sit-Rep/2020-05-01</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?f_region=Europe">https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?f_region=Europe</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2020-05-02">(accessed on 2 May 2020)</date-in-citation>        ) [27](          <article-title>Nextstrain</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">https://nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2020-04-29">(accessed on 29 April 2020)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33267855">33267855</a> <b>(1)</b> Does respiratory co-infection facilitate dispersal of SARS-CoV-2? investigation of a super-spreading event in an open-space office<ul><li> Phylogenetic analysis showed that all sequences form a cluster within lineage 20A of SARS-CoV-2 (<b>nextstrain</b> nomenclature) [<xref ref-type="bibr" rid="CR31"><u>31</u></xref>].<li> [31](          <person-group person-group-type="author">            <name>              <surname>Alm</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Broberg</surname>              <given-names>EK</given-names>            </name>            <name>              <surname>Connor</surname>              <given-names>T</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Komissarov</surname>              <given-names>AB</given-names>            </name>            <name>              <surname>Maurer-Stroh</surname>              <given-names>S</given-names>            </name>            <etal/>          </person-group>          <article-title>Geographical and temporal distribution of SARS-CoV-2 clades in the WHO European region, January to June 2020</article-title>          <source>Euro Surveill.</source>          <year>2020</year>          <volume>25</volume>          <issue>32</issue>          <fpage>2001410</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2020.25.32.2001410</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33269412">33269412</a> <b>(2)</b> The human coronaviruses (HCoVs) and the molecular mechanisms of SARS-CoV-2 infection<ul><li>The genomic epidemiology of novel coronavirus (Global subsampling, 93 of 3587 genomes sampled between Mar 2020 and Oct 2020, source <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?f_recency=1-2%20days%20ago&amp;l=unrooted">https://<b>nextstrain</b>.org/ncov/global?f_recency=1-2%20days%20ago&amp;l=unrooted</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33301953">33301953</a> <b>(11)</b> Next-generation computational tools and resources for coronavirus research: From detection to vaccine discovery<ul><li><b>Nextstrain</b> COVID-19 genetic epidemiology.<li><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0345">https://<b>nextstrain</b>.org</ext-link>.<li> <b>Nextstrain</b> is incorporating country-wise genomes information of SARS-CoV-2 epidemiology of novel coronavirus analyses and situation report.<li><b>Nextstrain</b>.<li><b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0165">https://<b>nextstrain</b>.org</ext-link>) is a real-time genetic tracking tool to map how the virus spread and pathogen evolution.<li> <b>Nextstrain</b> provides an updated profile of publicly existing data with powerful analytic and visualization tools for use by the global scientific community.<li> To further accelerate research on COVID-19 disease, <b>Nextstrain</b> is incorporating SARS-CoV-2 genomes and started sharing country-wise genomic epidemiology of novel coronavirus analyses and situation reports.<li> A genomic epidemiology map of different SARS-CoV-2 isolates has been demonstrated using <b>NextStrain</b> tools [<xref rid="bib110" ref-type="bibr"><u>110</u></xref>].<li> [110](          <person-group person-group-type="author">            <name>              <surname>Monteil</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Kwon</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Prado</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Hagelkr&#xFC;ys</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Wimmer</surname>              <given-names>R.A.</given-names>            </name>            <name>              <surname>Stahl</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Leopoldi</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Garreta</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Del Pozo</surname>              <given-names>C.H.</given-names>            </name>            <name>              <surname>Prosper</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>Romero</surname>              <given-names>J.P.</given-names>            </name>          </person-group>          <article-title>Inhibition of SARS-CoV-2 infections in engineered human tissues using clinical-grade soluble human ACE2</article-title>          <source>Cell</source>          <year>2020 Apr 24</year>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33219646">33219646</a> <b>(5)</b> Genomic Epidemiology of Severe Acute Respiratory Syndrome Coronavirus 2, Colombia<ul><li> We combined the 122 sequences from Colombia with 1,461 representative genome sequences from South America&#x2013;focused subsampling available from <b>NextStrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>) (<xref rid="R25" ref-type="bibr"><italic>25</italic></xref>) as of May 20, 2020 (<xref rid="SD1" ref-type="supplementary-material"><u>Appendix 1</u></xref> Table 2) plus reference MN908947.3 from the GenBank nucleotide database (accesssion no.<li> From the South America&#x2013;focused subsampling available from <b>NextStrain</b>, comparable findings were observed for other countries in South America (<xref rid="R45" ref-type="bibr"><italic>45</italic></xref>,<xref rid="R46" ref-type="bibr"><italic>46</italic></xref>), where the most frequently observed lineages were B.1 in 149 (60.8%) cases, B.1.5 in 35 (13.5%) cases, and A.5 in 14 (5.7%) cases.<li> By using all sequences in the South America&#x2013;focused subsampling available from <b>NextStrain</b>, we estimated that an average of 64 (interquartile range [IQR] 62&#x2013;67) introductions into the country have occurred but this estimate gets lower as we reduce the number of samples (sensitivity analyses) from other locations, down to 22 with the most even downsampled dataset.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33093069">33093069</a> <b>(1)</b> Sequencing identifies multiple early introductions of SARS-CoV-2 to the New York City region<ul><li> Sequences from non-human hosts, annotated by <b>Nextstrain</b> as duplicate individuals or highly divergent, with fewer than 27,000 nonambiguous nucleotides or with improperly formatted dates or location were excluded.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33205830">33205830</a> <b>(2)</b> <italic toggle="yes">Coronapp</italic>: A web application to annotate and monitor SARS&#x2010;CoV&#x2010;2 mutations<ul><li> The GISAID effort has made it possible to compare genomes on a geographical and temporal scale and an increasing number of laboratories have started to sequence COVID&#x2010;19 patient samples worldwide.<xref rid="jmv26678-bib-0013" ref-type="bibr"><sup>13</sup></xref>, <xref rid="jmv26678-bib-0014" ref-type="bibr"><sup>14</sup></xref> Several online tools have been developed to monitor the evolution of the virus from a phylogenetic perspective, such as <b>Nextstrain</b>,<xref rid="jmv26678-bib-0015" ref-type="bibr"><sup>15</sup></xref> or to visualize epidemiological data, such as the number of cases and deaths.<li>The GISAID sample ID (useful for cross&#x2010;reference with the GISAID database and other analyses based on it, e.g., <b>Nextstrain</b>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33253226">33253226</a> <b>(1)</b> SARS-CoV-2 lineage B.6 was the major contributor to early pandemic transmission in Malaysia<ul><li>Although lineage B.6 (which also belongs to GISAID clade O and <b>Nextstrain</b> clade 19A) was the most common in Malaysia, only 1,497 B.6 sequences had been reported globally out of over 105,568 sequences available as of September 20, 2020, a rate of 1.4%.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33253658">33253658</a> <b>(2)</b> Tackling zoonoses in a crowded world: Lessons to be learned from the COVID-19 pandemic<ul><li> However, based on an exploration of the evolutionary history of SARS-CoV-2, it is likely that the virus was already circulating outside of China within the first 6 weeks of being reported (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="interref0002">https://<b>nextstrain</b>.org/ncov/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33259879">33259879</a> <b>(0)</b> Evolution and genetic diversity of SARS-CoV-2 in Africa using whole genome sequences<ul><li> 10<sup>&#x2212;4</sup> as reported by Nexstrain (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org/ncov/global" id="intr0015">www.<b>nextstrain</b>.org/ncov/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33321075">33321075</a> <b>(1)</b> Predicted Cellular Immunity Population Coverage Gaps for SARS-CoV-2 Subunit Vaccines and Their Augmentation by Compact Peptide Sets<ul><li> <b>Nextstrain</b> (<xref rid="bib10" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) was used to identify ORFs and translate the sequence.<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33186228">33186228</a> <b>(3)</b> Adaptation of advanced clinical virology assays from HIV-1 to SARS-CoV-2<ul><li> The first two classification systems, GISAID and <b>Nextstrain</b>, are focused on investigating large scale diversity patterns of SARS-CoV-2 for understanding of patterns and determinants of the global spread by identifying clades that persist for at least several months and have significant geographic spread [<xref rid="R78" ref-type="bibr"><u>78</u></xref>,<xref rid="R79" ref-type="bibr"><u>79</u></xref>].<li>SARS-CoV-2 phylogenetic categorization systems <b>Nextstrain</b>, global initiative on sharing all influenza data and PANGOLIN.<li><b>Nextstrain</b> (79).<li> [78](<collab>(GISAID) GISAID</collab>. <article-title>Clade and lineage nomenclature aids in genomic epidemiology studies of active hCoV-19 viruses</article-title>. <year>2020</year>; <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aidsin-genomic-epidemiology-of-activehcov-19-viruses/">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aidsin-genomic-epidemiology-of-activehcov-19-viruses/</ext-link>) [79](<person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Hadfield</surname><given-names>J</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name><etal/></person-group><article-title>Year-letter genetic clade naming for SARS-CoV-2 on Nextstain.org</article-title>. <source>NextStrain</source><year>2020</year>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33237891">33237891</a> <b>(1)</b> COVID-19 Outbreak Associated with a 10-Day Motorcycle Rally in a Neighboring State &#x2014; Minnesota, August&#x2013;September 2020<ul><li> Phylogenetic relationships, including distinct clustering of viral whole genome sequences, were inferred based on nucleotide differences via IQ-TREE<xref rid="FN2" ref-type="fn"><sup>&#x2020;</sup></xref> using general time reversible substitution models (<xref rid="R3" ref-type="bibr"><italic>3</italic></xref>) as a part of the <b>Nextstrain</b><xref rid="FN3" ref-type="fn"><sup>&#xA7;</sup></xref> workflow (<xref rid="R4" ref-type="bibr"><italic>4</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33262895">33262895</a> <b>(5)</b> Genomic characterization of SARS-CoV-2 in Egypt<ul><li><italic>Collecting Subsampled <b>Nextstrain</b> sequences</italic>: To facilitate visualization of the phylogenetic tree, the <b>Nextstrain</b> team (www.<b>nextstrain</b>.org) subsamples the huge virus dataset into smaller collection of sequences representing different geographical areas.<li> To deal with large number of sequences, we used the <b>nextstrain</b> pipeline deployed via the mini-conda environment: First, the sequences were re-aligned using MAFFT.<li><bold>Level 1:</bold> Phylogenetic tree was constructed using the Egyptian sequences plus the 250 sequences sub-sampled from the <b>Nextstrain</b> (<xref rid="f0010" ref-type="fig">Fig.<li>Phylogenetic analysis of the Egyptian sequences plus 250 sequences sub-sampled from the <b>Nextstrain</b> dataset.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33304657">33304657</a> <b>(0)</b> Evolutionary dynamics and geographic dispersal of beta coronaviruses in African bats<ul><li> 10<sup>&#x2212;4</sup> (<uri xlink:href="https://www.<b>nextstrain</b>.org/ncov/global">www.<b>nextstrain</b>.org/ncov/global</uri>).<li> 10<sup>&#x2212;4</sup> (<uri xlink:href="https://www.<b>nextstrain</b>.org/ncov/global">www.<b>nextstrain</b>.org/ncov/global</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33131453">33131453</a> <b>(17)</b> Massive dissemination of a SARS-CoV-2 Spike Y839 variant in Portugal<ul><li> Table S1) using the SARS-CoV-2 <b>Nextstrain</b> pipeline [<xref rid="CIT0023" ref-type="bibr"><u>23</u></xref>] version from March 23, 2020 (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> of the study and providing interactive data navigation using both <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) [<xref rid="CIT0023" ref-type="bibr"><u>23</u></xref>] and Microreact (<ext-link ext-link-type="uri" xlink:href="https://microreact.org/">https://microreact.org/</ext-link>) [<xref rid="CIT0024" ref-type="bibr"><u>24</u></xref>] tools.<li> and Portugal phylogeny using <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>) and Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> According to <b>Nextstrain</b> clade definition (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>), the 1516 SARS-CoV2 genomes from Portugal (<ext-link ext-link-type="uri" xlink:href="https://microreact.org/project/nDGsJKFv7gQTj1q8CQwwKR/18a0a470">https://microreact.org/project/nDGsJKFv7gQTj1q8CQwwKR/18a0a470</ext-link>) follow, in general, the same trend observed at European level (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/europe">https://<b>nextstrain</b>.org/ncov/europe</ext-link>) [<xref rid="CIT0015" ref-type="bibr"><u>15</u></xref>,<xref rid="CIT0025" ref-type="bibr"><u>25</u></xref>].<li> GISAID accession number EPI_ISL_451307) (Table S2) [<xref rid="CIT0026" ref-type="bibr"><u>26</u></xref>], being identical (i.e, <b>Nextstrain</b> clade 20A background plus the G24077&#x2005;T SNP) to the &#x201C;founder&#x201D; Spike Y839 variant genome sequences detected in Portugal (<ext-link ext-link-type="uri" xlink:href="https://insaflu.insa.pt/covid19">https://insaflu.insa.pt/covid19</ext-link>).<li> phylogeny (using <b>Nextstrain</b> <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link> and Nextclade <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) pointed out that the D839Y amino acid change likely emerged independently in two other instances.<li> GISAID accession EPI_ISL_474528), which presents all clade-defining SNPs of <b>Nextstrain</b> Clade 20C plus four additional SNPs (including G24077&#x2005;T).<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28348833">28348833</a>) [15](Nextstrain. Available at: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov">https://nextstrain.org/ncov</ext-link>.) [25](<person-group person-group-type="author"><string-name><surname>Alm</surname><given-names>E</given-names></string-name>, <string-name><surname>Broberg E</surname><given-names>K</given-names></string-name>, <string-name><surname>Connor</surname><given-names>T</given-names></string-name>, <etal>et al.</etal></person-group><article-title>Geographical and temporal distribution of SARS-CoV-2 clades in the WHO European Region, January to June 2020</article-title>. <source>Euro Surveill</source>. <year>2020</year>;<volume>25</volume>:<fpage>32</fpage>. DOI:<pub-id pub-id-type="doi">10.2807/1560-7917.ES.2020.25.32.2001410</pub-id>.) [26](<person-group person-group-type="author"><string-name><surname>Rueca</surname><given-names>M</given-names></string-name>, <string-name><surname>Bartolini</surname><given-names>B</given-names></string-name>, <string-name><surname>Gruber</surname><given-names>CEM</given-names></string-name>, <etal>et al.</etal></person-group><article-title>Compartmentalized replication of SARS-Cov-2 in upper vs. lower respiratory tract assessed by whole genome quasispecies analysis</article-title>. <source>Microorganisms</source>. <year>2030</year>;<volume>8</volume>(<issue>9</issue>):<fpage>1302</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33239633">33239633</a> <b>(6)</b> No evidence for increased transmissibility from recurrent mutations in SARS-CoV-2<ul><li> All samples flagged by <b>NextStrain</b> as &#x2018;exclude&#x2019; (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt</ext-link>) as of 30/07/2020 were also removed.<li>All 46,723 assemblies were aligned against the Wuhan-Hu-1 reference genome (GenBank NC_045512.2, GISAID EPI_ISL_402125) using MAFFT v7.471<sup><xref ref-type="bibr" rid="CR52"><u>52</u></xref></sup> implemented in the rapid phylodynamic alignment pipeline provided by Augur 6.3.0 (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur">github.com/<b>nextstrain</b>/augur</ext-link>).<li> We also employed a less stringent approach, following the masking strategy employed by <b>NextStrain</b>, which masks only positions 18,529, 29,849, 29,851 and 29,853 as well as the first 130 and last 50 sites of the alignment.<li> [52](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33269353">33269353</a> <b>(1)</b> Timing the SARS-CoV-2 Index Case in Hubei Province<ul><li> We restricted our dataset to genomes that (i) were complete &#x2265;29,000 nt, (ii) had high coverage with &#x2264;0.5% unique amino acid mutations, (iii) had fewer than 1% &#x2018;N&#x2019;s, (iv) were not identified as potentially problematic via <b>NextStrain</b>, and (v) had a day-month-year sampling date reported.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33329480">33329480</a> <b>(8)</b> Analysis of Indian SARS-CoV-2 Genomes Reveals Prevalence of D614G Mutation in Spike Protein Predicting an Increase in Interaction With TMPRSS2 and Virus Infectivity<ul><li>Phylogenetic tree analyses of all the samples were performed using SARS-CoV-2 analysis protocol standards and tools provided by <b>Nextstrain</b> (<xref rid="B8" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) pipeline.<li> To finalize the tree for <b>Nextstrain</b> auspice visualization the tree was annotated using ancestral traits, clades, nucleotide mutation and amino acid mutation.<li>We have used <b>Nextstrain</b> year-letter clade nomenclature that started with 19A and 19B branched by C8782T and T28144C nucleotide changes and was initially prevalent in Asia during initial outbreak.<li>Phylogenetic analysis of the SARS-Cov genomes and their distribution into different <b>Nextstrain</b> defined new clades.<li> <bold>(A)</bold> A donut chart representing the sequenced sample (<italic>n</italic> = 202) distribution across the clades (clade nomenclature obtained using <b>Nextstrain</b>).<li> <bold>(C)</bold> Time tree (1,000 bootstraps) of the sequenced samples (<italic>n</italic> = 202) generated using <b>Nextstrain</b> time-tree pipeline overlaid with clinical status (condition, inner circle) of the patients during sample collection, place of migration (state, outer circle), and clade information (clades).<li> From the transmission map generated using <b>Nextstrain</b>, we observed that most of the sequences in 20A clades share sequence similarity with samples from Gujarat, which has very high prevalence of this clade (<xref ref-type="supplementary-material" rid="FS1"><u>Supplementary Figure 1E</u></xref>).<li> Our analysis depicted that all the genomes analyzed in our study were grouped into four major clades, 19A, 19B, 20A, and 20B, according to the new <b>Nextstrain</b> clade nomenclature.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33229462">33229462</a> <b>(8)</b> Genomic epidemiology of superspreading events in Austria reveals mutational dynamics and transmission properties of SARS-CoV-2<ul><li> Lines from global phylogenetic tree (left) to phylogenetic tree of all Austrian strains obtained in this study (right) indicate the phylogenetic relation and <b>Nextstrain</b> clade assignment of Austrian strains.<li> Color schemes of branches represent <b>Nextstrain</b> clade assignment (left) or phylogenetic clusters of Austrian strains (right).<li> These clusters are related to the global clades 19A, 20A, 20B, and 20C of the widely used <b>Nextstrain</b> classification (fig.<li> Using phylogenetic analysis, we found that the mutation profile defining the Tyrol-1 cluster matched the definition of the global clade 20C of the <b>Nextstrain</b> classification (fig.<li> Clade annotations for global trees were adapted from <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv</ext-link>; <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>); clusters of Austrian strains were identified on the basis of shared mutation profiles and patient location from epidemiological data.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33227343">33227343</a> <b>(2)</b> Genomic surveillance of SARS-CoV-2 in Thailand reveals mixed imported populations, a local lineage expansion and a virus with truncated ORF7a<ul><li> For example, six of the samples in the B.1 lineage in our collection are grouped tightly with virus samples commonly found in the United States of America and Europe and collected during the same period (March 2020) as visualized by <b>Nextstrain</b> Timetree (Supplementary <xref rid="fig0010" ref-type="fig">Fig.<li> Upon visual inspection in <b>Nextstrain</b>, only one Malaysian sample (MKAK-CL-2020-5096) is the closest to A/Thai-1, but only with 63 % bootstrap value by the maximum-likelihood approach and one shared lineage-specific nucleotide substitution (4,390G&#x2192;U) (Supplementary Figure 4).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33211775">33211775</a> <b>(3)</b> Characterising the epidemic spread of influenza A/H3N2 within a city through phylogenetics<ul><li> We then built an initial phylogeny from the HA segment alone by using RaxML 8.2.1 [<xref rid="ppat.1008984.ref024" ref-type="bibr"><u>24</u></xref>] and obtained branch lengths in calendar time via timetree [<xref rid="ppat.1008984.ref025" ref-type="bibr"><u>25</u></xref>] using the <b>nextstrain</b> pipeline [<xref rid="ppat.1008984.ref026" ref-type="bibr"><u>26</u></xref>].<li> We obtained around 240 local clusters (see <xref ref-type="fig" rid="ppat.1008984.g001"><u>Fig 1c</u></xref> and <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/jameshadfield/basel-flu/1">https://<b>nextstrain</b>.org/community/jameshadfield/basel-flu/1</ext-link>), suggesting that the sampled sequences were the result of around 240 influenza introductions from areas outside of Basel.<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24451623">24451623</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33206635">33206635</a> <b>(69)</b> Stability of SARS-CoV-2 phylogenies<ul><li> However, despite efforts to mitigate the impact of sequencing and assembly errors, and to provide standardized datasets for real-time analysis [<xref rid="pgen.1009175.ref021" ref-type="bibr"><u>21</u></xref>], inferred phylogenetic histories of the outbreak often differ between analyses of different research groups (Results) and these inferred histories sometimes differ between analyses performed by the same group with different data (<italic>e</italic>.<italic>g</italic>., 31 different <b>Nextstrain</b> trees produced between 3/23 and 4/30: Results).<li><b>Nextstrain</b> datasets.<li>Our analyses are built in large part on the work of <b>Nextstrain</b> [<xref rid="pgen.1009175.ref016" ref-type="bibr"><u>16</u></xref>].<li> For our purposes, we anticipate that <b>Nextstrain</b>&#x2019;s filters will minimize idiosyncratic errors and should be retained in the majority of future analyses.<li> Here, as a primary example we use 31 different alignments and phylogenetic trees inferred by <b>Nextstrain</b> from days between 3/23/2020 and 4/30/2020.<li> sites that contain the allele in the reference genome from the root of the tree (here and in <b>Nextstrain</b> this is sample Wuhan/Hu-1, obtained in December 2019 in the city of Wuhan, China, RefSeq accession NC_045512.2) and a single alternate allele.<li> Across the 4/19/2020 <b>Nextstrain</b> tree, we found 2533, 395, 94, 40, and 44 bi-allelic sites with parsimony score 1, 2, 3, 4, or 5 or more, respectively (<xref ref-type="fig" rid="pgen.1009175.g002"><u>Fig 2</u></xref>, <xref ref-type="supplementary-material" rid="pgen.1009175.s005"><u>S1 Fig</u></xref>).<li> We found that the parsimony score displays a roughly linear relationship with the log of the alternate allele count, as it does for extremal sites in <b>Nextstrain</b> trees we examined built on different days in April, but with varying slope (<xref ref-type="fig" rid="pgen.1009175.g004"><u>Fig 4</u></xref>).<li>Part of the tree we obtained from the 4/19/2020 <b>Nextstrain</b> tree (left) compared to the corresponding part of tree after removal of sites with lab-associated variants (right).<li>Though not a focus here, we emphasize that filtering for genomic regions that are difficult to assemble or align (<italic>e</italic>.<italic>g</italic>., those used by <b>Nextstrain</b> to filter the ends of chromosomes as defined here <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) should also be rigorously employed.<li>Researchers can upload their own aligned SARS-CoV-2 genome samples and phylogenetic trees to the SARS-CoV-2 Genome Browser in order to compare their phylogenetic analyses to those from <b>Nextstrain</b> and COG-UK, and also to look at the specific molecular features of the clades that their phylogenetic analysis identifies (<xref ref-type="supplementary-material" rid="pgen.1009175.s004"><u>S4 Text</u></xref>).<li><b>Nextstrain</b> phylogenies vary significantly over time.<li> For comparison, we restricted 31 <b>Nextstrain</b> trees produced between March 23, 2020 and April 30, 2020 to just the 468 samples they all have in common.<li> Note that the <b>Nextstrain</b> clade ID system was updated while we were finalizing this work [<xref rid="pgen.1009175.ref073" ref-type="bibr"><u>73</u></xref>].<li> There is therefore substantial variation in <b>Nextstrain</b> phylogenies over time.<li>Comparisons of <b>Nextstrain</b> trees over time.<li>(A) Multidimensional scaling of normalized entropy-weighted total distances among phylogenetic trees produced by <b>Nextstrain</b> from March and April.<li> (B) Relationships between <b>Nextstrain</b> phylogenies are shown in a tree-of-trees, &#x201C;meta-tree&#x201D; [<xref rid="pgen.1009175.ref067" ref-type="bibr"><u>67</u></xref>] we constructed, which displays the distances among topologies of the constitutive trees.<li> Indeed, the names used for <b>Nextstrain</b>&#x2019;s original clades (A1a, A2, A2a, A6, A7, B, B1, B2, B4) bear no relationship to the lineage names (A, B, A.1, A.2, B.1, B.2, A.1.1, etc.) suggested by the COG-UK consortium [<xref rid="pgen.1009175.ref018" ref-type="bibr"><u>18</u></xref>,<xref rid="pgen.1009175.ref019" ref-type="bibr"><u>19</u></xref>], and without a 1&#x2013;1 correspondence between the topologically defined clades in their respective phylogenetic trees, it is difficult to translate nomenclature in order to conduct precise scientific discourse pertaining to the evolutionary conclusions reached by these groups.<li>To explore the differences among available phylogenies and to provide guidelines for clade-based comparisons across possible evolutionary histories, we used our approach to identify the correspondence between the <b>Nextstrain</b> phylogeny produced on April 19, 2020 and the COG-UK phylogeny produced on April 24, 2020 (<xref ref-type="fig" rid="pgen.1009175.g011"><u>Fig 11A</u></xref>).<li> We observe good agreement between the big <b>Nextstrain</b> named clades and their corresponding best matching named clades in the COG-UK tree and vice versa (e.g., &#x201C;A2a&#x201D; clade in <b>Nextstrain</b>, &#x201C;B.1&#x201D; clade in COG-UK, etc., <xref ref-type="fig" rid="pgen.1009175.g011"><u>Fig 11B</u></xref>), suggesting that these clades are reasonably stable across different analyses.<li> <b>Nextstrain</b> named clades much earlier and many did not increase in size subsequently; others have since emerged and were named by COG-UK later.<li>Comparison of <b>Nextstrain</b> and COG-UK trees.<li>(A) A tanglegram of the <b>Nextstrain</b> tree from 4/19 (left) with the COG-UK tree from 4/24 (right).<li> (B) The COG-UK clades (which they term &#x201C;lineages&#x201D;) having the highest Jaccard similarity coefficient (J) with each <b>Nextstrain</b> (NS) named clade and vice versa, where the Jaccard similarity coefficient is computed using the set of samples from the root of that clade.<li> This contrasts with the <b>Nextstrain</b> project&#x2019;s primary goal of up-to-date transmission tracing.<li>Higher branches in our tree closely mirror a <b>Nextstrain</b> &#x201C;consensus&#x201D; tree and the COG-UK Tree.<li>To identify stable nodes across analyses we compared a <b>Nextstrain</b> &#x201C;consensus tree&#x201D; and the COG-UK tree.<li> To do this, we produced a majority rule clade consensus tree [<xref rid="pgen.1009175.ref075" ref-type="bibr"><u>75</u></xref>] for the 422 common samples in 31 trees produced by <b>Nextstrain</b> between 3/23 to 4/30, and restricted the COG-UK tree to these same samples.<li> We find exceptionally good congruence between our <b>Nextstrain</b> consensus and the COG-UK phylogenies (<xref ref-type="fig" rid="pgen.1009175.g012"><u>Fig 12A</u></xref>), even though the inference methods differed substantially.<li> Specifically, the COG-UK tree is built using a more typical bootstrapping approach [<xref rid="pgen.1009175.ref060" ref-type="bibr"><u>60</u></xref>], whereas our approach for building a <b>Nextstrain</b> &#x201C;consensus&#x201D; from trees produced on subsequent days would resemble a kind of &#x201C;bootstrapping by samples&#x201D; approach.<li>Comparison of <b>Nextstrain</b> and the COG-UK trees.<li>(A) A tanglegram of our <b>Nextstrain</b> consensus tree (left) and COG-UK tree from 4/24 (right).<li> (B) The COG-UK lineages having the highest Jaccard similarity coefficient (J) with each <b>Nextstrain</b> consensus (NS) named clade and vice versa.<li> Each tree, including our <b>Nextstrain</b> &#x201C;consensus&#x201D;, is available for visualization through the UCSC Genome Browser (<xref ref-type="fig" rid="pgen.1009175.g008"><u>Fig 8</u></xref>, <xref ref-type="supplementary-material" rid="pgen.1009175.s010"><u>S6</u></xref>).<li>Obtaining <b>Nextstrain</b> trees and genotype data.<li>We have downloaded genomic variation data from <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/ncov">http://<b>nextstrain</b>.org/ncov</ext-link>, which is ultimately processed and derived from the GISAID database [<xref rid="pgen.1009175.ref076" ref-type="bibr"><u>76</u></xref>], and transformed it into a VCF file with genotypes for all samples as assigned by <b>Nextstrain</b> and a Newick tree file, and associated files for display in the UCSC SARS-CoV-2 Genome Browser.<li> Software to perform this is described here: <ext-link ext-link-type="uri" xlink:href="https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/utils/otto/<b>nextstrain</b>Ncov/<b>nextstrain</b>.py">https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/utils/otto/<b>nextstrain</b>Ncov/<b>nextstrain</b>.py</ext-link>.<li> To do this, we first filtered variants with parsimony score greater than 4 using concurrent <b>Nextstrain</b> tree and VCF files from 4/19/2020.<li>We obtained the phylogenetic tree hosted by <b>Nextstrain</b> (accessed 4/19/2020) and used this in our comparisons of clades among trees and as our primary data object for examining apparently recurrent mutation on the tree.<li> We did separately confirm that most apparently recurrent mutations are recovered on the trees produced on different days by <b>Nextstrain</b>.<li>For comparison of clades among different tree-building approaches, we obtained variant datasets, and phylogenies from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link> accessed 4/19/2020-4/26/2020) and from COG-UK (<ext-link ext-link-type="uri" xlink:href="https://cog-uk.s3.climb.ac.uk/20200424/cog_2020-04-24_tree.newick">https://cog-uk.s3.climb.ac.uk/20200424/cog_2020-04-24_tree.newick</ext-link>, accessed 4/24/2020).<li>From the 04/19 <b>Nextstrain</b> tree, we created a &#x201C;reference phylogeny&#x201D;using IQ-TREE 2 [<xref rid="pgen.1009175.ref059" ref-type="bibr"><u>59</u></xref>,<xref rid="pgen.1009175.ref079" ref-type="bibr"><u>79</u></xref>] to build phylogenies from each of these alignments using the GTR+G nucleotide substitution model.<li> We chose Australia due to its high representation in the <b>Nextstrain</b> trees (372 samples in the 04/19 <b>Nextstrain</b> tree).<li>Comparisons across <b>Nextstrain</b> trees.<li>To understand commonalities in tree structure over time, we used MDS of a distance matrix of normalized entropy-weighted total distances among <b>Nextstrain</b> trees (pruned to 468 shared samples) spanning from March 23 to April 30.<li>Producing a <b>Nextstrain</b> consensus tree.<li>To produce a <b>Nextstrain</b> consensus tree, we first pruned all <b>Nextstrain</b> trees to a common set of samples included in each tree.<li>Alternate allele count versus parsimony score for the <b>Nextstrain</b> 4/20/2020 dataset and tree.<li>Phylogenies created using the variants from 04/19 <b>Nextstrain</b> tree without modification (left) and with lab-associated variants completely masked (right) demonstrate movement of multiple samples between sub-clades.<li>Comparisons between the reference phylogeny, built from the 04/19/2020 release of <b>Nextstrain</b>, to phylogenies built by entirely masking lab-associated variants (blue), control sites (grey), and extremal sites (brown) are shown for Robinson-Foulds <bold>(A)</bold>, Quartet <bold>(B)</bold>, Path Difference <bold>(C)</bold>, and Triples <bold>(D)</bold> scores as calculated by TreeCmp [<xref rid="pgen.1009175.ref064" ref-type="bibr"><u>64</u></xref>].<li> Horizontal lines indicate scores for phylogenies constructed after masking all lab-associated sites (blue), all control sites (grey), all extremal sites (brown), or using an unaltered <b>Nextstrain</b> 04/19/2020 dataset (black).<li>Tanglegrams for the two <b>Nextstrain</b> trees released on 04/19/2020 (left) and 04/20/2020 (right).<li>UCSC Genome Browser display of the trees from <xref ref-type="fig" rid="pgen.1009175.g011"><u>Fig 11C</u></xref> (COG-UK tree from 4/24, restricted to 422 samples in common with consensus tree of <b>Nextstrain</b> trees 3/23-4/30, and <b>Nextstrain</b> consensus tree), colored by <b>Nextstrain</b> clade assigned to sample.<li>GISAID IDs whose clade annotation changed in the <b>Nextstrain</b> tree from 4/19/2020 to 3/30/2020, and from 4/19/2020 to 4/28/2020.<li> [21](<name><surname>Bogner</surname><given-names>P</given-names></name>, <name><surname>Capua</surname><given-names>I</given-names></name>, <name><surname>Lipman</surname><given-names>DJ</given-names></name>, <name><surname>Cox</surname><given-names>NJ</given-names></name>. <article-title>A global initiative on sharing avian flu data</article-title>. <source>Nature</source>. <year>2006</year> pp. <fpage>981</fpage>&#x2013;981. <pub-id pub-id-type="doi">10.1038/442981a</pub-id>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [73](<name><surname>Hodcroft</surname><given-names>EB</given-names></name>, <name><surname>Hadfield</surname><given-names>J</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>, <name><surname>Bedford</surname><given-names>T</given-names></name>. <article-title>Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org</article-title>. In: <source>Virological</source> [Internet]. <day>2</day><month>6</month><year>2020</year> [cited 8 Jun 2020]. Available: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/year-letter-genetic-clade-naming-for-sars-cov-2-on-nextstain-org/498">https://virological.org/t/year-letter-genetic-clade-naming-for-sars-cov-2-on-nextstain-org/498</ext-link>) [67](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18853364">18853364</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [19](<name><surname>Mavian</surname><given-names>C</given-names></name>, <name><surname>Marini</surname><given-names>S</given-names></name>, <name><surname>Prosperi</surname><given-names>M</given-names></name>, <name><surname>Salemi</surname><given-names>M</given-names></name>. <article-title>A snapshot of SARS-CoV-2 genome availability up to 30th March, 2020 and its implications.</article-title><pub-id pub-id-type="doi">10.1101/2020.04.01.020594</pub-id>) [75](<name><surname>Margush</surname><given-names>T</given-names></name>, <name><surname>McMorris</surname><given-names>FR</given-names></name>. <article-title>Consensus n-trees.</article-title><source>Bulletin of Mathematical Biology</source>. <year>1981</year> pp. <fpage>239</fpage>&#x2013;<lpage>244</lpage>. <pub-id pub-id-type="doi">10.1007/bf02459446</pub-id>) [60](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29077904">29077904</a>) [76](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [59](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [79](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [64](<name><surname>Bogdanowicz</surname><given-names>D</given-names></name>, <name><surname>Giaro</surname><given-names>K</given-names></name>, <name><surname>Wr&#xF3;bel</surname><given-names>B</given-names></name>. <article-title>TreeCmp: Comparison of Trees in Polynomial Time.</article-title><source>Evolutionary Bioinformatics</source>. <year>2012</year> p. EBO.S9657. <pub-id pub-id-type="doi">10.4137/ebo.s9657</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33183397">33183397</a> <b>(3)</b> Chasing the ghost of infection past: identifying thresholds of change during the COVID-19 infection in Spain<ul><li> This, together with the initial unawareness of the population, led to a high transmission rate of the infection, which spread rapidly to neighbouring countries, the Middle East and Europe, and then the rest of the world (see <uri xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</uri> [<xref rid="ref5" ref-type="bibr"><u>5</u></xref>]).<li> as evidenced by the presence of 14 different genetic clusters identified by the <b>Nextstrain</b> project [<xref rid="ref5" ref-type="bibr"><u>5</u></xref>; accessed 27 April].<li> [5](<string-name><surname>Hadfield</surname><given-names>J</given-names></string-name><etal/> (<year>2018</year>) <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>34</volume>, <fpage>4121</fpage>&#x2013;<lpage>4123</lpage>.) [5](<string-name><surname>Hadfield</surname><given-names>J</given-names></string-name><etal/> (<year>2018</year>) <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>34</volume>, <fpage>4121</fpage>&#x2013;<lpage>4123</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34095334">34095334</a> <b>(5)</b> High Prevalence of SARS-CoV-2 Genetic Variation and D614G Mutation in Pediatric Patients With COVID-19<ul><li>Phylogenetic analysis and evolutionary rate estimation were performed using packages available through <b>Nextstrain</b> command-line interface (version 2.0.0.) [<xref rid="CIT0017" ref-type="bibr"><u>17</u></xref>].<li> <b>Nextstrain</b> clades: orange = 20B, blue = 20C, green = 20A.<li> The panel on the right shows the <b>Nextstrain</b> clade for each isolate.<li> All but 1 of the isolates from our cohort belonged to 3 of the 5 phylogenetic clades (20A, 20B, 20C) defined by <b>Nextstrain</b> (<xref ref-type="fig" rid="F1"><u>Figure 1</u></xref>; <xref ref-type="supplementary-material" rid="sup1"><u>Supplementary Figure 1</u></xref>) [<xref rid="CIT0017" ref-type="bibr"><u>17</u></xref>, <xref rid="CIT0023" ref-type="bibr"><u>23</u></xref>].<li>From a genomic epidemiology perspective, the isolates in our study almost exclusively belonged to the 3 most recent SARS-CoV-2 phylogenetic clades (20A, 20B, 20C) as defined in <b>Nextstrain</b> [<xref rid="CIT0017" ref-type="bibr"><u>17</u></xref>].<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](Nextstain.org. <comment>Year-letter genetic clade naming for SARS-CoV-2</comment>. Available at: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming</ext-link>. Accessed 31 August 2020.) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33189939">33189939</a> <b>(3)</b> Early transmission of SARS-CoV-2 in South Africa: An epidemiological and phylogenetic report<ul><li> The resulting dataset of 1848 sequences was used in a custom build on the <b>NextStrain</b> analysis platform (<xref rid="bib0055" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> To infer support for the splits in this tree topology, an additional 100 bootstrap trees in IQ-Tree under the same model parameters as <b>NextStrain</b> were inferred.<li>Bayesian coalescent analyses were performed on major lineages of the <b>NextStrain</b> build in which KZN sequences fell.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33186810">33186810</a> <b>(5)</b> A clade of SARS-CoV-2 viruses associated with lower viral loads in patient upper airways<ul><li> Isolate lineages were determined with PANGOLIN using the 2020-05-19 definitions and with <b>Nextstrain</b> using the assign_clades.py program <xref rid="bib0023" ref-type="bibr"><u>[23]</u></xref>, <xref rid="bib0024" ref-type="bibr"><u>[24]</u></xref>, <xref rid="bib0025" ref-type="bibr"><u>[25]</u></xref>.<li> Using the Pangolin and <b>Nextstrain</b> methods of lineage classification [<xref rid="bib0023" ref-type="bibr"><u>23</u></xref>,<xref rid="bib0024" ref-type="bibr"><u>24</u></xref>], isolates in Clade 1 were assigned mainly to Pangolin lineage B.1 (92% assigned to lineage B.1; 2% to B.1.1; and 6% to B.1.5) and <b>Nextstrain</b> clade 20C (61% to clade 20C, 37% to clade 20A, and 2% to clade 20B), the majority of isolates in Clade 2 belonged to Pangolin lineage A.3 (92% to A.3; 8% to A) and <b>Nextstrain</b> clade 19B (92% to clade 19B and 8% to clade 19A), and isolates in clade 3 belonged mainly to lineage A.1 (86% to A.1; 14% to A) and all belonged to <b>Nextstrain</b> clade 19B (Supplemental Table 1).<li> [[23]](O'Toole &#xC1;, McCrone JT. pangolin (phylogenetic assignment of named global outbreak lineages). <ext-link ext-link-type="uri" xlink:href="https://github.com/hCoV-2019/pangolin" id="interref0007">https://github.com/hCoV-2019/pangolin</ext-link> (accessed June 4, 2020).) [[24]](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Holmes</surname>              <given-names>EC</given-names>            </name>            <name>              <surname>O'Toole</surname>              <given-names>A</given-names>            </name>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>          <source>Nat Microbiol</source>          <year>2020</year>        ) [[25]](Nextstrain. ncov. <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov" id="interref0008">https://github.com/nextstrain/ncov</ext-link> (accessed September 13, 2020).) [23](O'Toole &#xC1;, McCrone JT. pangolin (phylogenetic assignment of named global outbreak lineages). <ext-link ext-link-type="uri" xlink:href="https://github.com/hCoV-2019/pangolin" id="interref0007">https://github.com/hCoV-2019/pangolin</ext-link> (accessed June 4, 2020).) [24](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Holmes</surname>              <given-names>EC</given-names>            </name>            <name>              <surname>O'Toole</surname>              <given-names>A</given-names>            </name>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>          <source>Nat Microbiol</source>          <year>2020</year>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33176093">33176093</a> <b>(3)</b> SARS-CoV-2 Transmission among Marine Recruits during Quarantine<ul><li>SARS-CoV-2 genomes obtained from patients worldwide and associated metadata were downloaded from the Global Initiative on Sharing All Influenza Data EpiCoV database<sup><xref rid="r12" ref-type="bibr"><u>12</u></xref></sup> on August 11, 2020 (79,840 sequences), and a subset of sequences was selected from this database with the use of the default subsampling scheme of <b>Nextstrain</b> software<sup><xref rid="r13" ref-type="bibr"><u>13</u></xref></sup> with the aim of maximizing representation of genomes obtained from patients in the United States.<li> Phylogenetic analyses of the specimens obtained from participants were performed with the v1.0-292-ga9de690 <b>Nextstrain</b> build for SARS-CoV-2 genomes with the use of default parameters.<li> Transmission and outbreak events were identified on the basis of clustering of the SARS-CoV-2 genomes obtained from study participants within the <b>Nextstrain</b> phylogenetic tree, visualized with TreeTime.<li> Transmission and outbreak events were identified on the basis of clustering of the SARS-CoV-2 genomes obtained from study participants within the <b>Nextstrain</b> phylogenetic tree, visualized with TreeTime.<sup><xref rid="r14" ref-type="bibr"><u>14</u></xref></sup> A comparative analysis of mutation profiles relative to the SARS-CoV-2 Wuhan reference genome was performed with the use of Nextclade software, version 0.3.6 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32504052">32504052</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33175831">33175831</a> <b>(3)</b> Using B cell receptor lineage structures to predict affinity<ul><li> Its ability to predict fitness in real influenza data has been shown in [<xref rid="pcbi.1008391.ref069" ref-type="bibr"><u>69</u></xref>], and it features prominently on the <b>nextstrain</b> [<xref rid="pcbi.1008391.ref070" ref-type="bibr"><u>70</u></xref>] web site <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/flu/seasonal/h3n2/ha/2y?c=lbi">https://<b>nextstrain</b>.org/flu/seasonal/h3n2/ha/2y?c=lbi</ext-link>.<li> [69](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26951657">26951657</a>) [70](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33170902">33170902</a> <b>(1)</b> Large scale genomic analysis of 3067 SARS-CoV-2 genomes reveals a clonal geo-distribution and a rich genetic variations of hotspots mutations<ul><li> Maximum-likelihood trees were inferred with IQ-TREE v1.5.5 [<xref rid="pone.0240345.ref012" ref-type="bibr"><u>12</u></xref>] under the GTR model using <b>NextStrain</b> tools.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33335743">33335743</a> <b>(2)</b> Emerging phylogenetic structure of the SARS-CoV-2 pandemic<ul><li> Real-time phylogenetic reconstruction in <b>Nextstrain</b> (<xref rid="veaa082-B10" ref-type="bibr">Hadfield et&#xA0;al.<li> We trimmed the first 130&#x202F;bp and last 50&#x202F;bp of the aligned sequences to remove potential sequencing artefacts in line with <b>Nextstrain</b> protocol (<xref rid="veaa082-B10" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33212338">33212338</a> <b>(1)</b> Surveillance of wastewater revealed peaks of SARS-CoV-2 preceding those of hospitalized patients with COVID-19<ul><li>-source project <b>Nextstrain</b>, five larger clades of SARS-CoV-2 have been identified so far (<xref rid="bib0013" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>).<li> [Hadfield&#xA0;et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33171711">33171711</a> <b>(2)</b> Analysis and Evaluation of COVID-19 Web Applications for Health Professionals: Challenges and Opportunities<ul><li> A&#xA0;similar application, <b>Nextstrain</b> SARS-CoV-2 for genome analysis has been included in the dashboards (<xref rid="healthcare-08-00466-t004" ref-type="table"><u>Table 4</u></xref>, No.<li>The <b>Nextstrain</b> SARS-CoV-2 portal for genome analysis (<xref rid="healthcare-08-00466-t004" ref-type="table"><u>Table 4</u></xref>, No.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33159066">33159066</a> <b>(1)</b> Epidemiological hypothesis testing using a phylogeographic and phylodynamic framework<ul><li> These genotypes were identified on the basis of the WNV database published on the platform <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR32"><u>32</u></xref>,<xref ref-type="bibr" rid="CR73"><u>73</u></xref></sup>.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31671157">31671157</a>) [73](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33166565">33166565</a> <b>(2)</b> Deletion in the C-terminal region of the envelope glycoprotein in some of the Indian SARS-CoV-2 genome<ul><li> Analysis of 361 genomes of SARS-CoV-2 from India revealed that there were 5 clusters, 4 of these being known clades identified by <b>Nextstrain</b>: A2a, A3, B, and B4.<li> (2020)</xref> and as indicated in Table S1, of the 34 sequences with E gene deletions, 15 belonged to 19A, one belonged to 19B, 4 belonged to 20A and 20B each and remaining sequences could not be categorised into any of the clades defined by <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33161087">33161087</a> <b>(16)</b> Trends of mutation accumulation across global SARS-CoV-2 genomes: Implications for the evolution of the novel coronavirus<ul><li> Multiple sequence alignment was also created using the Augur tool kit of the <b>Nextstrain</b> package.<li> For the labeling of clades in the phylogenetic tree, type-defining marker mutations were downloaded from the <b>Nextstrain</b> github repository which comes as a package within the <b>Nextstrain</b> tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="ir0040">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> Rules of clade-labeling followed were those mentioned in the website located at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.github.io/ncov/naming_clades.html" id="ir0045">https://<b>nextstrain</b>.github.io/ncov/naming_clades.html</ext-link>.<li> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming" id="ir0050">https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>), and (iii) the system proposed by Tang et al.<li> sampling for other geographical areas, both following custom rules laid down in <b>Nextstrain</b>: for the &#x2018;focal&#x2019; country (India), up to 300 sequences, or whatever maximum number (&lt;300) is available, per month for each year under consideration; for contextual sampling, 50 such whole-genome sequences per month per country that are genetically associated to the &#x2018;focal&#x2019; samples based on the priority call criterion called &#x2018;Proximity&#x2019;.<li>The overall evolutionary paths traced thus far by SARS-CoV-2 was delineated by labeling the 4618 global (GISAID) sequences on the phylogenetic tree using three different concepts of clade nomenclature defined in the web-based resoure <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.github.io/ncov/" id="ir0060">https://<b>nextstrain</b>.github.io/ncov/</ext-link> (<xref rid="f0005" ref-type="fig">Figs.<li> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming" id="ir0005">https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>).<li> with the year of identification followed by an alphabet) of Hodcroft et al., 2020 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming" id="ir0070">https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>), showed that the largest lineage A2 identified by PANGOLIN clade-nomenclature system, emerged in the year 2020 and evolved further into a number of sub-lineages characterized by mutations in specific nucleotide positions (these have been designated in Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33147627">33147627</a> <b>(3)</b> Computational strategies to combat COVID-19: useful tools to accelerate SARS-CoV-2 and coronavirus research<ul><li> In addition, poreCov includes a quick-time, tree-based analysis of the inputs against reference sequences using augur (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur">https://github.com/<b>nextstrain</b>/augur</ext-link>) and toytree (<ext-link ext-link-type="uri" xlink:href="https://github.com/eaton-lab/toytree">https://github.com/eaton-lab/toytree</ext-link>) for visualization.<li> For SARS-CoV-2, the default uploaded model is based on <b>Nextstrain</b> [<xref rid="ref31" ref-type="bibr"><u>31</u></xref>] and GISAID data [<xref rid="ref18" ref-type="bibr"><u>18</u></xref>].<li> [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33248470">33248470</a> <b>(15)</b> Case Study: Prolonged Infectious SARS-CoV-2 Shedding from an Asymptomatic Immunocompromised Individual with Cancer<ul><li>(B) Full SARS-CoV-2 genomes were subsampled from Washington state, representing <b>NextStrain</b> clade 19B, including the four full-genome sequences recovered from the individual and the Wuhan-Hu-1/2019 sequence and aligned using MAFFT v.1.4 (<xref rid="bib31" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>; <xref rid="bib32" ref-type="bibr"><u>Katoh et&#xA0;al., 2002</u></xref>) implemented in Geneious Prime v.2020.1.2 (<ext-link ext-link-type="uri" xlink:href="https://www.geneious.com/" id="intref0290">https://www.geneious.com/</ext-link>).<li>To visualize the temporal relationships of the patient isolates, 44 full SARS-CoV-2 genome sequences from Washington state belonging to <b>NextStrain</b> clade 19B (<ext-link ext-link-type="uri" xlink:href="http://clades.<b>nextstrain</b>.org/" id="intref0025">http://clades.<b>nextstrain</b>.org/</ext-link>) were subsampled from the GISAID database (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="intref0030">https://www.gisaid.org/</ext-link>; <xref rid="bib63" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>) representing strains collected in Washington state from February to May 2020.<li> <xref rid="bib19" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) using the HKY85 model of nucleotide substitution and a fixed molecular clock at 8e&#x2212;4 with a standard deviation of 4e&#x2212;4, as implemented in the <b>NextStrain</b> pipeline (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" id="intref0040">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>).<li> The quality of the sequences was determined by Nextclade v.0.7.5 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="interref0015">https://<b>nextstrain</b>.org/ncov/global</ext-link>), and 1,789 sequences on April 20, 385 sequences between April 20 and May 11, 268 sequences between May 11 and May 26, and 709 sequences between May 26 and June 15 were kept for further phylogenetic analysis.<li><hr/></td></tr><tr><td>MAFFT align</td><td>(<xref rid="bib31" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>, <xref rid="bib32" ref-type="bibr"><u>Katoh et&#xA0;al., 2002</u></xref>)</td><td>Geneious Prime 2020.1.2 plugin</td></tr><tr><td>Multalin sequence alignment</td><td>(<xref rid="bib13" ref-type="bibr"><u>Corpet, 1988</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://multalin.toulouse.inra.fr/multalin/" id="intref0075">http://multalin.toulouse.inra.fr/multalin/</ext-link></td></tr><tr><td>ESPript 3.0</td><td>(<xref rid="bib57" ref-type="bibr"><u>Robert and Gouet, 2014</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://espript.ibcp.fr/ESPript/ESPript/" id="intref0080">http://espript.ibcp.fr/ESPript/ESPript/</ext-link></td></tr><tr><td>Geneious Prime 2020.1.2</td><td>Geneious</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com" id="intref0085">https://www.geneious.com</ext-link></td></tr><tr><td>PhyML 3.320180621</td><td><xref rid="bib18" ref-type="bibr"><u>Guindon et&#xA0;al., 2010</u></xref></td><td>Geneious Prime 2020.1.2 plugin</td></tr><tr><td>FigTree v1.4.4</td><td><ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/figtree/" id="intref0090">http://tree.bio.ed.ac.uk/software/figtree/</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/rambaut/figtree/" id="intref0095">https://github.com/rambaut/figtree/</ext-link></td></tr><tr><td>Pangolin COVID-19 Lineage Assigner</td><td><xref rid="bib56" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="intref0100">https://pangolin.cog-uk.io</ext-link></td></tr><tr><td>Pymol Molecular Graphics System version 2.0.1</td><td>Schr&#xF6;dinger</td><td><ext-link ext-link-type="uri" xlink:href="https://www.schrodinger.com/pymol" id="intref0105">https://www.schrodinger.com/pymol</ext-link></td></tr><tr><td>Prism 8.2.0</td><td>GraphPad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.graphpad.com:443/" id="intref0110">https://www.graphpad.com:443/</ext-link></td></tr><tr><td>NextClade v0.7.5</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="intref0115">https://github.com/<b>nextstrain</b>/nextclade</ext-link>.<li><hr/></td></tr><tr><td>MAFFT align</td><td>(<xref rid="bib31" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>, <xref rid="bib32" ref-type="bibr"><u>Katoh et&#xA0;al., 2002</u></xref>)</td><td>Geneious Prime 2020.1.2 plugin</td></tr><tr><td>Multalin sequence alignment</td><td>(<xref rid="bib13" ref-type="bibr"><u>Corpet, 1988</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://multalin.toulouse.inra.fr/multalin/" id="intref0075">http://multalin.toulouse.inra.fr/multalin/</ext-link></td></tr><tr><td>ESPript 3.0</td><td>(<xref rid="bib57" ref-type="bibr"><u>Robert and Gouet, 2014</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://espript.ibcp.fr/ESPript/ESPript/" id="intref0080">http://espript.ibcp.fr/ESPript/ESPript/</ext-link></td></tr><tr><td>Geneious Prime 2020.1.2</td><td>Geneious</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com" id="intref0085">https://www.geneious.com</ext-link></td></tr><tr><td>PhyML 3.320180621</td><td><xref rid="bib18" ref-type="bibr"><u>Guindon et&#xA0;al., 2010</u></xref></td><td>Geneious Prime 2020.1.2 plugin</td></tr><tr><td>FigTree v1.4.4</td><td><ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/figtree/" id="intref0090">http://tree.bio.ed.ac.uk/software/figtree/</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/rambaut/figtree/" id="intref0095">https://github.com/rambaut/figtree/</ext-link></td></tr><tr><td>Pangolin COVID-19 Lineage Assigner</td><td><xref rid="bib56" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="intref0100">https://pangolin.cog-uk.io</ext-link></td></tr><tr><td>Pymol Molecular Graphics System version 2.0.1</td><td>Schr&#xF6;dinger</td><td><ext-link ext-link-type="uri" xlink:href="https://www.schrodinger.com/pymol" id="intref0105">https://www.schrodinger.com/pymol</ext-link></td></tr><tr><td>Prism 8.2.0</td><td>GraphPad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.graphpad.com:443/" id="intref0110">https://www.graphpad.com:443/</ext-link></td></tr><tr><td>NextClade v0.7.5</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="intref0115">https://github.com/<b>nextstrain</b>/nextclade</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=region" id="interref0020">https://<b>nextstrain</b>.org/ncov/global?c=region</ext-link>.<li> For the time tree, full SARS-CoV-2 genomes were subsampled from Washington state representing <b>NextStrain</b> clade 19B, including the four patient genomes sequences and the Wuhan-Hu-1/2019 genome sequence.<li> 2020.1.2 (<ext-link ext-link-type="uri" xlink:href="https://www.geneious.com/" id="intref0215">https://www.geneious.com/</ext-link>), a maximum likelihood tree reconstructed with PhyML v.3.1 (<xref rid="bib18" ref-type="bibr"><u>Guindon et&#xA0;al., 2010</u></xref>), and the time tree showing temporal divergence inferred in TreeTime v.0.7.6 (<xref rid="bib19" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) using the HKY85 model of nucleotide substitution and a fixed molecular clock at 8e-4 with a standard deviation of 4e-4 as implemented in the <b>NextStrain</b> pipeline (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" id="intref0220">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>).<li> The sequences were analyzed by the Nextclade server v0.7.5 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0230">https://clades.<b>nextstrain</b>.org/</ext-link>) for quality and sequences that were not of sufficient quality were discarded.<li> [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Katoh et&#xA0;al., 2002](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [Shu and McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Katoh et&#xA0;al., 2002](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [Corpet, 1988](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=2849754">2849754</a>) [Robert and Gouet, 2014](          <person-group person-group-type="author">            <name>              <surname>Robert</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Gouet</surname>              <given-names>P.</given-names>            </name>          </person-group>          <article-title>Deciphering key features in protein structures with the new ENDscript server</article-title>          <source>Nucleic Acids Res.</source>          <volume>42</volume>          <year>2014</year>          <comment>W320-4</comment>        ) [Guindon et&#xA0;al., 2010](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20525638">20525638</a>) [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Katoh et&#xA0;al., 2002](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [Corpet, 1988](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=2849754">2849754</a>) [Robert and Gouet, 2014](          <person-group person-group-type="author">            <name>              <surname>Robert</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Gouet</surname>              <given-names>P.</given-names>            </name>          </person-group>          <article-title>Deciphering key features in protein structures with the new ENDscript server</article-title>          <source>Nucleic Acids Res.</source>          <volume>42</volume>          <year>2014</year>          <comment>W320-4</comment>        ) [Guindon et&#xA0;al., 2010](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20525638">20525638</a>) [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Guindon et&#xA0;al., 2010](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20525638">20525638</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33158051">33158051</a> <b>(3)</b> Threading the Pieces Together: Integrative Perspective on SARS-CoV-2<ul><li><b>Nextstrain</b> and the Global Initiative on Sharing All Influenza Data (GISAID <uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>) are two open-source tools for the collection and visualization of pathogen genome data that help us understand pathogen evolution and epidemic spread (<xref ref-type="fig" rid="pathogens-09-00912-f001"><u>Figure 1</u></xref>).<li> The existing clades, as defined by <b>Nextstrain</b>, and their global distribution are shown in <xref rid="pathogens-09-00912-t001" ref-type="table"><u>Table 1</u></xref>.<li> The rapid increase in various platforms for high-throughput sequencing [<xref rid="B86-pathogens-09-00912" ref-type="bibr"><u>86</u></xref>], the sharing of genomic data (GISAID, JHU Dashboard, <b>Nextstrain</b>), and data analytical tools (Viral track; [<xref rid="B87-pathogens-09-00912" ref-type="bibr"><u>87</u></xref>]) with open access enable the availability of unmatched resources for the investigation of interactomics of host and pathogen using transcriptome sequencing data.<li> [86](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22890146">22890146</a>) [87](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32479746">32479746</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33316043">33316043</a> <b>(2)</b> A Phylodynamic Workflow to Rapidly Gain Insights into the Dispersal History and Dynamics of SARS-CoV-2 Lineages<ul><li> In practice, we propose to take advantage of the rapid time-scaled phylogenetic tree inference process used by the online <b>Nextstrain</b> platform (<xref rid="msaa284-B10" ref-type="bibr">Hadfield et&#xA0;al.<li>On June 10, 2020, we downloaded all Belgian SARS-CoV-2 sequences (<italic>n&#x2009;</italic>=<italic>&#x2009;</italic>740) available on GISAID, as well as all non-Belgian sequences used in <b>Nextstrain</b> to represent the overall dispersal history of the virus (<italic>n&#x2009;</italic>=<italic>&#x2009;</italic>4,309 with 126 different countries of origin).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33144575">33144575</a> <b>(9)</b> Revealing fine-scale spatiotemporal differences in SARS-CoV-2 introduction and spread<ul><li> <bold>c</bold> A divergence-based phylogenetic tree built using <b>Nextstrain</b> tools for the 122 Dane county (red) and 125 Milwaukee county (blue) sequences.<li>To further analyze the genetic differences among viruses in the two locations, we assigned clades using the <b>Nextstrain</b> nomenclature.<li><bold>a</bold> A time-resolved phylogenetic tree built using <b>Nextstrain</b> tools for 122 samples collected in Dane county.<li>All 247 available full-length sequences from Dane and Milwaukee county through 26 April 2020 were used for phylogenetic analysis using the tools implemented in <b>Nextstrain</b> custom builds (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>)<sup><xref ref-type="bibr" rid="CR4"><u>4</u></xref>,<xref ref-type="bibr" rid="CR62"><u>62</u></xref></sup>.<li> Time-resolved and divergence phylogenetic trees were built using the standard <b>Nextstrain</b> tools and scripts<sup><xref ref-type="bibr" rid="CR4"><u>4</u></xref>,<xref ref-type="bibr" rid="CR62"><u>62</u></xref></sup>.<li>An additional subsampled global phylogeny using all available sequences in GISAID as of 21 June 2020 were input into the <b>Nextstrain</b> pipeline.<li> those given only by month and year as opposed to day, month, and year) and all tips which were excluded from our global alignment using the <b>Nextstrain</b> exclusion criteria (minimum length of 27,000 nucleotides, sequences listed in the &#x201C;exclude&#x201D; configuration file, sequences with admin division listed as &#x201C;USA&#x201D;) using BioPython.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [62](Moreno, G. K., Braun, K. M. &amp; Michael, M. A. gagekmoreno/SARS-CoV-2-in-Southern-Wisconsin: SARS-CoV-2 in Southern Wisconsin (Github). <ext-link ext-link-type="uri" xlink:href="https://github.com/gagekmoreno/SARS-CoV-2-in-Southern-Wisconsin/tree/v1.0.0">https://github.com/gagekmoreno/SARS-CoV-2-in-Southern-Wisconsin/tree/v1.0.0</ext-link> (2020).) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [62](Moreno, G. K., Braun, K. M. &amp; Michael, M. A. gagekmoreno/SARS-CoV-2-in-Southern-Wisconsin: SARS-CoV-2 in Southern Wisconsin (Github). <ext-link ext-link-type="uri" xlink:href="https://github.com/gagekmoreno/SARS-CoV-2-in-Southern-Wisconsin/tree/v1.0.0">https://github.com/gagekmoreno/SARS-CoV-2-in-Southern-Wisconsin/tree/v1.0.0</ext-link> (2020).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32699851">32699851</a> <b>(2)</b> Single-nucleotide conservation state annotation of the SARS-CoV-2 genome<ul><li>Mutations found in SARS-CoV-2 sequences were point mutations identified by <b>Nextstrain</b><sup><xref rid="R28" ref-type="bibr"><u>28</u></xref></sup> (accessed on Sept 7, 2020) from sequences available on GISAID<sup><xref rid="R19" ref-type="bibr"><u>19</u></xref></sup>.<li> For our analysis, we filtered out mutations if their ancestral alleles did not match the reference genome used by <b>Nextstrain</b>, MN908947.3.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<name><surname>Elbe</surname><given-names>S.</given-names></name> &amp; <name><surname>Buckland-Merrett</surname><given-names>G.</given-names></name><article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>. <source>Glob. challenges (Hoboken, NJ)</source><volume>1</volume>, <fpage>33</fpage>&#x2013;<lpage>46</lpage> (<year>2017</year>).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33139704">33139704</a> <b>(7)</b> Full genome viral sequences inform patterns of SARS-CoV-2 spread into and within Israel<ul><li> Figure (including map) generated using <b>NextStrain</b><sup><xref ref-type="bibr" rid="CR32"><u>32</u></xref></sup>.<li> Figure generated using <b>NextStrain</b><sup><xref ref-type="bibr" rid="CR32"><u>32</u></xref></sup>.<li> All available sequences were then down-sampled to 4693 representative sequences across the globe using the latest build of <b>NextStrain</b> ncov pipeline<sup><xref ref-type="bibr" rid="CR31"><u>31</u></xref>,<xref ref-type="bibr" rid="CR32"><u>32</u></xref></sup> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>); 1195 of these 4693 sequences were from the U.S., while 1991 were from Europe.<li>Following down-sampling and manual curation, a phylogenetic tree was inferred using the <b>NextStrain</b> pipeline<sup><xref ref-type="bibr" rid="CR32"><u>32</u></xref></sup>.<li> 4693 sequences total), we assigned a country to each internal node using <b>NextStrain</b>&#x2019;s maximum-likelihood ancestral state reconstruction.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [31](Hadfield, J. et al. Twenty years of West Nile virus spread and evolution in the Americas visualized by Nextstrain. <italic>PLoS Pathog.</italic><bold>15</bold>, e1008042 (2019).) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33137190">33137190</a> <b>(1)</b> Ensembl 2021<ul><li> To complement our own analysis of the public data, we also display annotated variants from real-time pathogen surveillance resource <b>Nextstrain</b> (<xref rid="B45" ref-type="bibr"><u>45</u></xref>), using sequence data acquired from the virus data sharing platform GISAID (<xref rid="B46" ref-type="bibr"><u>46</u></xref>).<li> [45](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33147840">33147840</a> <b>(4)</b> First Phylogenetic Analysis of Malian SARS-CoV-2 Sequences Provides Molecular Insights into the Genomic Diversity of the Sahel Region<ul><li>SNP and phylogenetic analysis were performed using a local installation of the <b>nextstrain</b>.org pipeline [<xref rid="B5-viruses-12-01251" ref-type="bibr"><u>5</u></xref>].<li> At the moment, three diverging nomenclatures exist: the Pangolin lineages based on the phylogenetic assignment of named global outbreak lineages (Pangolin) algorithm [<xref rid="B19-viruses-12-01251" ref-type="bibr"><u>19</u></xref>], the GISAID clades using the actual letters of the marker mutations [<xref rid="B20-viruses-12-01251" ref-type="bibr"><u>20</u></xref>] and the <b>Nextstrain</b> clades, which are defined by year and a letter code [<xref rid="B21-viruses-12-01251" ref-type="bibr"><u>21</u></xref>].<li> As the Pangolin lineages and the <b>Nextstrain</b> clades resulted in fairly similar classifications for the Malian SARS-CoV-2 genomes (see <xref ref-type="fig" rid="viruses-12-01251-f003"><u>Figure 3</u></xref>), and it was unsure up to now which nomenclature would endure, we will discuss our results for both systems.<li> According to the <b>Nextstrain</b> nomenclature, 14 strains belonged to the 19B cluster, while the remaining 7 strains were split into the 20A (4) and 20C (3) clades.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [20](          <person-group person-group-type="author">            <collab>Freunde von GISAID, e.V</collab>          </person-group>          <article-title>Clade and Lineage Nomenclature Aids in Genomic Epidemiology Studies of Active hCoV-19 Viruses</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2020-07-22">(accessed on 22 July 2020)</date-in-citation>        ) [21](          <article-title>Year-Letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2020-06-25">(accessed on 25 June 2020)</date-in-citation>        ) </ul></p>
<p><u>No PMID</u> <b>(1)</b> Detection and genetic characterization of community-based SARS-CoV-2 infections &#x2013; New York City, March 2020<ul><li> Nucleic acid from RT-PCR&#x2013;positive specimens was then extracted and subjected to Oxford Nanopore MinION sequencing, and full genome sequences were generated using methods described previously.<xref rid="bib3" ref-type="bibr"><sup>3</sup></xref> Phylogenetic relations were inferred using the <b>Nextstrain</b> pipeline,<xref rid="bib4" ref-type="bibr"><sup>4</sup></xref> including the 36 positive SARS-CoV-2 sentinel specimens and selected full genome sequences available as of April 1, 2020, from the Global Initiative on Sharing All Influenza Data (GISAID).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33163695">33163695</a> <b>(3)</b> Genomic exploration light on multiple origin with potential parsimony-informative sites of the severe acute respiratory syndrome coronavirus 2 in Bangladesh<ul><li> Image was taken from <b>nextstrain</b>, link <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?f_country=Bangladesh" id="ir0010">https://<b>nextstrain</b>.org/ncov/global?f_country=Bangladesh</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33163105">33163105</a> <b>(4)</b> Analyses g&#xE9;nomiques et phylodynamiques du Sars-CoV-2<ul><li> popularis&#xE9;e par la plateforme <b>Nextstrain</b> [<xref rid="bib17" ref-type="bibr"><u>17</u></xref>], qui permet &#xE0; tout un chacun de visualiser des phylog&#xE9;nies telles que celle repr&#xE9;sent&#xE9;e sur la <xref rid="fig1" ref-type="fig"><u>figure 1</u></xref>.<li>Phylog&#xE9;nie d&#x2019;infections de la pand&#xE9;mie de Covid-19 obtenue par <b>Nextstrain</b>.<li>Les trois premi&#xE8;res &#xE9;tapes effectu&#xE9;es par <b>Nextstrain</b> sont relativement robustes au vu du nombre de s&#xE9;quences analys&#xE9;es (plusieurs milliers).<li> cela, les analyses de <b>Nextstrain</b> ont r&#xE9;v&#xE9;l&#xE9; une structure mondiale de pand&#xE9;mie tr&#xE8;s coh&#xE9;rente avec les relev&#xE9;s &#xE9;pid&#xE9;miologiques.<li> [17](Bell SM, Hodcroft E, M&#xFC;ller N et al. Genomic analysis of nCoV spread. Situation report 2020-05-15;2020.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33144890">33144890</a> <b>(1)</b> Die Virologie von SARS-CoV-2<ul><li> Viele der Sequenzen werden Onlinedatenbanken wie etwa <b>Nextstrain</b> zur Verf&#xFC;gung gestellt und k&#xF6;nnen von der wissenschaftlichen Gemeinschaft ausgewertet werden.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33127656">33127656</a> <b>(1)</b> Society for Immunotherapy of Cancer clinical and biomarkers data sharing resource document: Volume I&#x2014;conceptual challenges<ul><li> For example, <b>NextStrain</b> has been tracking and updating information on the progression and changes observed in the viral genome.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33137892">33137892</a> <b>(1)</b> Phylogeography of 27,000 SARS-CoV-2 Genomes: Europe as the Major Source of the COVID-19 Pandemic<ul><li> For this, we used letter labels established by GISAID/<b>nextstrain</b> (A and B) intercalated with numbers (e.g., A1, A2, B1, B2) and letters (e.g., A1a, A1b, A1a1, A1a1a2) as in the cladistic system established for human mitochondrial DNA [<xref rid="B30-microorganisms-08-01678" ref-type="bibr"><u>30</u></xref>].<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=9803269">9803269</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33119657">33119657</a> <b>(7)</b> SARS-CoV-2 transmission routes from genetic data: A Danish case study<ul><li> As such our analysis is based on significantly more data as compared to the more global, but less Denmark-specific analysis by <b>Nextstrain</b>.<li> <b>Nextstrain</b> lists 132 Danish sequences for the relevant date range (retrieved on: August 21 2020).<li>We choose to root our trees with respect to the reference sequence NC-045512.2 (SARS-CoV-2 isolate Wuhan-Hu-1), which is also the reference for <b>Nextstrain</b> [<xref rid="pone.0241405.ref026" ref-type="bibr"><u>26</u></xref>] and for [<xref rid="pone.0241405.ref019" ref-type="bibr"><u>19</u></xref>, <xref rid="pone.0241405.ref027" ref-type="bibr"><u>27</u></xref>, <xref rid="pone.0241405.ref028" ref-type="bibr"><u>28</u></xref>].<li> These haplotypes were in turn derived from (the original clades of) <b>Nextstrain</b> [<xref rid="pone.0241405.ref026" ref-type="bibr"><u>26</u></xref>].<li> In addition, we list their names in the more recent <b>Nextstrain</b> convention [<xref rid="pone.0241405.ref032" ref-type="bibr"><u>32</u></xref>] and the clades from the pangolin system that they are included in [<xref rid="pone.0241405.ref028" ref-type="bibr"><u>28</u></xref>].<li>new <b>Nextstrain</b></th>                  <th align="left" style="border-bottom:thick" rowspan="1" colspan="1">pang.<li>List of relevant haplotypes with their definition in terms of mutations as well as their new <b>Nextstrain</b> label.<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32387564">32387564</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32289214">32289214</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [32](<name><surname>Hodcroft</surname><given-names>E</given-names></name>. <source>Clade Naming &amp; Definitions</source>; <year>2020</year><month>6</month><day>2</day> Available from: <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov/blob/master/docs/clades.md">https://github.com/nextstrain/ncov/blob/master/docs/clades.md</ext-link> [cited 2020 June 12].) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33500774">33500774</a> <b>(10)</b> Reproducibly sampling SARS-CoV-2 genomes across time, geography, and viral diversity<ul><li> The widely used <b>NextStrain</b><sup><xref rid="ref-4" ref-type="bibr"><u>4</u></xref></sup> platform refers to these sequence collections in their documentation as the<italic>context sequences</italic> and the<italic>focal sequences</italic>, respectively, and we adopt that terminology here.<li> Our subsampling workflow is compatible with tools such as <b>NextStrain</b><sup><xref rid="ref-4" ref-type="bibr"><u>4</u></xref></sup>, which includes a similar but not identical subsampling process (details provided in the<italic>Discussion</italic> section).<li>Resemblance to <b>NextStrain</b> context sequence sampling workflow.<li>The <b>NextStrain</b> workflow also subsamples context sequences for its phylogenetic tree builds using augur (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur">https://github.com/<b>nextstrain</b>/augur</ext-link>) and scripts in their ncov repository (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> As far as we are aware, there is not an analog to our Step 4 in the <b>NextStrain</b> workflow.<li> At this stage, we do not claim that our workflow is better than the one used by <b>NextStrain</b>.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33100263">33100263</a> <b>(4)</b> ORF3a mutation associated with higher mortality rate in SARS-CoV-2 infection<ul><li>Amino acid changes in SARS-CoV-2 strains for each country were extracted from <b>NextStrain</b> open source project (<uri xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</uri>) [<xref rid="ref12" ref-type="bibr"><u>12</u></xref>].<li> A total of 218 viral strains from 15 countries (including China as reference) were further analysed for amino acid mutations from <b>NextStrain</b> database (online Supplementary table 2).<li> [12](<string-name><surname>Hadfield</surname><given-names>J</given-names></string-name>, <etal/> (<year>2018</year>) <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>34</volume>, <fpage>4121</fpage>&#x2013;<lpage>4123</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33195515">33195515</a> <b>(1)</b> Field-Adapted Full Genome Sequencing of Peste-Des-Petits-Ruminants Virus Using Nanopore Sequencing<ul><li> It would also have made it possible to track the outbreak in real-time using tools such as <b>Nextstrain</b> (<xref rid="B12" ref-type="bibr"><u>12</u></xref>, <xref rid="B31" ref-type="bibr"><u>31</u></xref>).<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27501264">27501264</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33132795">33132795</a> <b>(1)</b> Die Virologie von SARS-CoV-2<ul><li>Viele der Sequenzen werden Online-Datenbanken wie etwa <b>Nextstrain</b> zur Verf&#xFC;gung gestellt und k&#xF6;nnen von der wissenschaftlichen Gemeinschaft ausgewertet werden.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34756867">34756867</a> <b>(3)</b> Clinical utility of SARS-CoV-2 whole genome sequencing in deciphering source of infection<ul><li> Currently, global monitoring of SARS-CoV-2 mutation dynamics is publicly available through the Global Initiative on Sharing All Influenza Data (GISAID) and <b>Nextstrain</b> [<xref rid="bib2" ref-type="bibr"><u>2</u></xref>,<xref rid="bib3" ref-type="bibr"><u>3</u></xref>].<li>A phylogenetic tree analysis was performed locally using the Augur program available from <b>Nextstrain</b> and genome sequence data obtained in the currently reported study as well as data available from the global database EpiCov from GISAID [<xref rid="bib3" ref-type="bibr"><u>3</u></xref>].<li> analysis using <b>Nextstrain</b> showed that the presence of C11752T could be used to segregate the results into two distinctive clades.<li> [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33149541">33149541</a> <b>(3)</b> New Pathways of Mutational Change in SARS-CoV-2 Proteomes Involve Regions of Intrinsic Disorder Important for Virus Replication and Release<ul><li> Metadata was acquired from <b>NextStrain</b>.<li> The protein sequences were also categorized by month of collection, using the <b>NextStrain</b> metadata.<li>Our study explores pathways of mutational change in highly entropic sites of the SARS-CoV-2 proteome, quantifying diversity and fixation of variants with the two-state variable strategy we modified from Pan and Deem.<sup><xref rid="bibr38-1176934320965149" ref-type="bibr"><u>38</u></xref></sup> The analysis does not explore phylogenetic relationships at genomic level with distance or parsimony-based reconstruction methods, which for example are systematically pursued in the <b>Nextstrain</b> portal.<li> [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21543352">21543352</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33091066">33091066</a> <b>(2)</b> Time-lapse sentinel surveillance of SARS-CoV-2 spread in India<ul><li> Furthermore, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>Nextstrain</b></ext-link>, the open-source pathogen genome project, with data plugged-in from GISAID, demonstrates a global panoramic view of the SARS-CoV-2 pandemic.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33113345">33113345</a> <b>(2)</b> High-Density Amplicon Sequencing Identifies Community Spread and Ongoing Evolution of SARS-CoV-2 in the Southern United States<ul><li> <xref rid="bib47" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>), GenBank, and <b>Nextstrain</b> (<xref rid="bib22" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> The <b>Nextstrain</b> database (<xref rid="bib22" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) suggests that positions 18,529, 29,849, 29,851, and 29,853 may be subject to PCR or sequencing bias.<li> [Shu and McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33082451">33082451</a> <b>(2)</b> Point mutation bias in SARS-CoV-2 variants results in increased ability to stimulate inflammatory responses<ul><li>To build a phylogenetic tree of the SARS-CoV-2 sequence, Augur pipeline published by <b>Nextstrain</b> (github.com/<b>nextstrain</b>/ncov) was used<sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref>,<xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup>.<li> [27](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: Real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics (Oxford, England)</source>          <year>2018</year>          <volume>34</volume>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33082405">33082405</a> <b>(1)</b> Multiple early introductions of SARS-CoV-2 into a global travel hub in the Middle East<ul><li> SARS-CoV-2 clades annotations were performed in auspice v2.13.0 and cross-checked with <b>nextstrain</b>.org as of 12 May 2020.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33103064">33103064</a> <b>(1)</b> Optimizing open data to support one health: best practices to ensure interoperability of genomic data from bacterial pathogens<ul><li> As a result of building interoperable systems, researchers can now use public data from the INSDC in conjunction with private data from individual or industry labs, using platforms such as Integrated Rapid Infectious Disease Analysis (IRIDA) [<xref ref-type="bibr" rid="CR22"><u>22</u></xref>], INNUENDO [<xref ref-type="bibr" rid="CR23"><u>23</u></xref>], PathogenWatch [<xref ref-type="bibr" rid="CR24"><u>24</u></xref>] <b>NextStrain</b> [<xref ref-type="bibr" rid="CR13"><u>13</u></xref>], IDseq [<xref ref-type="bibr" rid="CR25"><u>25</u></xref>], and CGE Evergreen [<xref ref-type="bibr" rid="CR26"><u>26</u></xref>].<li> [22](          <person-group person-group-type="author">            <name>              <surname>Matthews</surname>              <given-names>TC</given-names>            </name>            <name>              <surname>Bristow</surname>              <given-names>FR</given-names>            </name>            <name>              <surname>Griffiths</surname>              <given-names>EJ</given-names>            </name>            <name>              <surname>Petkau</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Adam</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Dooley</surname>              <given-names>D</given-names>            </name>            <etal/>          </person-group>          <article-title>The integrated rapid infectious disease analysis (IRIDA) platform</article-title>          <source>bioRxiv</source>          <year>2018</year>          <volume>79</volume>          <fpage>381830</fpage>        ) [23](          <person-group person-group-type="author">            <name>              <surname>Llarena</surname>              <given-names>A-K</given-names>            </name>            <name>              <surname>Gon&#xE7;alves</surname>              <given-names>BFR</given-names>            </name>            <name>              <surname>Silva</surname>              <given-names>DN</given-names>            </name>            <name>              <surname>Halkilahti</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Machado</surname>              <given-names>MP</given-names>            </name>            <name>              <surname>Da Silva</surname>              <given-names>MS</given-names>            </name>            <etal/>          </person-group>          <article-title>INNUENDO: a cross-sectoral platform for the integration of genomics in the surveillance of food-borne pathogens</article-title>          <source>EFSA Supporting Publ</source>          <year>2018</year>          <volume>15</volume>          <fpage>53</fpage>        ) [24](Pathogenwatch. <ext-link ext-link-type="uri" xlink:href="https://cgps.gitbook.io/pathogenwatch">https://cgps.gitbook.io/pathogenwatch</ext-link>. Accessed 20 July 2020.) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [25](Kalantar KL, Carvalho T, Bourcy CFA de, Dimitrov B, Dingle G, Egger R, et al. IDseq &#x2013; An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring. bioRxiv. 2020. 10.1101/2020.04.07.030551.) [26](Szarvas J, Ahrenfeldt J, Cisneros JLB, Thomsen MCF, Aarestrup FM, Lund O. Large scale automated phylogenomic analysis of bacterial isolates and the Evergreen Online platform. Communications Biology. 2020;3:1&#x2013;10.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33057582">33057582</a> <b>(14)</b> Web tools to fight pandemics: the COVID-19 experience<ul><li><b>Nextstrain</b>.<li><ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/ncov/global"><b>nextstrain</b>.org/ncov/global</ext-link>.<li><b>Nextstrain</b>.<li>Screenshots of selected COVID-19 webtools described in this manuscript: the epidemiological dashboard from Johns Hopkins University (A), the time-wise phylogenetic tree representation from <b>NextStrain</b> (B) and the human/SARS-CoV-2 interactome visualized by Covex (C).<li> One of the first and most known platforms specifically developed for real-time tracking of pathogen evolution is the <b>Nextstrain</b> project [<xref rid="ref32" ref-type="bibr"><u>32</u></xref>].<li> <b>Nextstrain</b> combines a daily updated database of viral genomes, a bioinformatics pipeline for phylodynamics analysis, and an interactive visualization platform into a single open web application (<xref rid="f2" ref-type="fig"><u>Figure 2B</u></xref>), to assist virologists and epidemiologists to share and analyze data.<li> The architecture of <b>Nextstrain</b> is well designed and responds to the need for a continual surveillance to prevent uncontrolled outbreaks.<li> The existence of <b>Nextstrain</b> allowed researchers from across the world respond promptly to the emergence of COVID-19 and to track the population movements associated with outbreaks via a combination of time, geography and mutation data [<xref rid="ref33" ref-type="bibr"><u>33</u></xref>].<li> Many other web tools rely on <b>Nextstrain</b> architecture to visualize data.<li> <b>Nextstrain</b> source code (mainly JavaScript and Python) is shared on GitHub.<li> It is worthwhile to highlight that the GISAID and <b>Nextstrain</b> teams are closely collaborating: upon connection via the GISAID website, users can browse phylogenetic trees and graphs produced by the <b>Nextstrain</b> Next-hCoV-19 interactive web interface, which provides updated phylogenetic and mutational analysis of GISAID-shared data.<li>As the emergence of a novel coronavirus arose quickly, data scientists and bioinformaticians have promptly responded to the COVID-19 pandemic by developing SARS-CoV-2 specific tools to support research and advancing knowledge on biology, pathology, epidemiology and treatment of COVID-19, building on and taking inspiration from preexisting platforms like <b>Nextstrain</b> and the GISAID Initiative.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [33](<person-group person-group-type="author"><name name-style="western"><surname>Wang</surname><given-names>J-T</given-names></name>, <name name-style="western"><surname>Lin</surname><given-names>Y-Y</given-names></name>, <name name-style="western"><surname>Chang</surname><given-names>S-Y</given-names></name>, <etal>et&#xA0;al.</etal></person-group> &#xA0;<article-title>The role of phylogenetic analysis in clarifying the infection source of a COVID-19 patient</article-title>. <source>J Infect</source> &#xA0;<year>2020</year>;<volume>81</volume>:<fpage>147</fpage>&#x2013;<lpage>78</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33224265">33224265</a> <b>(1)</b> A comprehensive review about SARS-CoV-2<ul><li> Scientists from all over the world have started to deposit their analysis on the sequence alignment of the patient&#x2019;s virus infection on integrated international websites such as GISAID [<xref rid="B28" ref-type="bibr"><u>28</u></xref>], <b>nextstrain</b> [<xref rid="B29" ref-type="bibr"><u>29</u></xref>], this has immensely helped in following the infection, mapping the mutation as well as tracing the transmission of different patients everywhere.<li> [28](<article-title>&#x2018;GISAID &#x2013; Enabling rapid and open access to epidemic and pandemic virus data&#x2019;</article-title>. <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">www.gisaid.org/</ext-link>) [29](<article-title>&#x2018;Nextstrain &#x2013; Real-time tracking of pathogen evolution&#x2019;</article-title>. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">https://nextstrain.org/</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33048046">33048046</a> <b>(3)</b> The challenges of vaccine strain selection<ul><li> Moreover, they have now integrated their forecasts for H3N2 into <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>nextstrain</b>.org</ext-link>, an open-source platform that scientists and policymakers use to track the real-time evolution of a wide range of pathogens (<xref rid="bib9" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bib18" ref-type="bibr"><u>Neher and Bedford, 2015</u></xref>).<li> <b>Nextstrain</b> provides a platform to make influenza vaccine strain selection more data-driven, systematic and transparent, and to allow new forecasting methods to be integrated as they show promise.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Neher and Bedford, 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33045727">33045727</a> <b>(1)</b> GESS: a database of global evaluation of SARS-CoV-2/hCoV-19 sequences<ul><li> The GISAID also features data searching, downloading and related tools integrated with an online app Next hCoV-19 based on <b>Nextstrain</b> (<xref rid="B10" ref-type="bibr"><u>10</u></xref>), which yields both the phylogenetic analysis and geographical information of the SARS-CoV-2 samples.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33083157">33083157</a> <b>(1)</b> Mutations of SARS-CoV-2 nsp14 exhibit strong association with increased genome-wide mutation load<ul><li> Buckland-Merrett, 2017</xref>) and <b>NextStrain</b> (<xref rid="ref-9" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) provides researchers with invaluable resources to track the evolution of the virus as it spreads across the world.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33071356">33071356</a> <b>(1)</b> Food shopping under risk and uncertainty<ul><li> This virus was first detected in the city of Wuhan in China in late 2019 and rapidly spread around the globe (<xref rid="bib0195" ref-type="bibr"><b>Nextstrain</b>.org, 2020</xref>, <xref rid="bib0105" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33052352">33052352</a> <b>(6)</b> SARS-CoV-2 genomic characterization and clinical manifestation of the COVID-19 outbreak in Uruguay<ul><li> We build on the existing <b>Nextstrain</b> framework, a broadly used and universally recognized analytical platform for the analysis of global SARS-CoV-2 viral genome sequence data that has accumulated since the start of the pandemic [<xref rid="R13" ref-type="bibr"><u>13</u></xref>] to analyze newly sequenced SARS-CoV-2 genomes sampled in Uruguay, and combine it with Bayesian Evolutionary Analysis of Sampling Trees (BEAST).<li>To infer our SARS-CoV-2 time-scaled phylogenetic tree, we selected global reference sequences used for the <b>Nextstrain</b> analysis specific for South America as of August 6th, 2020.<li> Using the <b>Nextstrain</b> metadata to identify the accessions of interest, we then downloaded the latest whole-genome sequence alignment from the GISAID database.<li> 10&#x2013;4 substitutions per site per year to replicate the original <b>Nextstrain</b> workflow analysis as faithfully as possible.<li>To determine phylogenetic and epidemiologic characteristics of the Uruguayan SARS-CoV-2 outbreak, we performed a comprehensive set of phylogenetic analyses, including maximum-likelihood trees, haplotype networks, <b>Nextstrain</b>-based phylogenetic placements, and Bayesian phylogeographic analyses using BEAST (<xref rid="F3" ref-type="fig"><u>Figures 3</u></xref> and <xref rid="F4" ref-type="fig"><u>4</u></xref>, <xref rid="SD8" ref-type="supplementary-material"><u>Figures S5</u></xref>&#x2013;<xref rid="SD11" ref-type="supplementary-material"><u>S8</u></xref>).<li>To assess global introductions and the regional spread of SARS-CoV-2 in Uruguay, we performed discrete phylogeographic analysis using BEAST with all 73 Uruguayan sequences, complemented with 1737 global reference sequences, based on the global subsampling dataset suggested by <b>Nextstrain</b> (<xref rid="SD3" ref-type="supplementary-material"><u>Table S2</u></xref>).<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33050463">33050463</a> <b>(4)</b> Genomic Diversity and Hotspot Mutations in 30,983 SARS-CoV-2 Genomes: Moving Toward a Universal Vaccine for the &#x201C;Confined Virus&#x201D;?<ul><li> Using the <b>Nextstrain</b> clade nomenclature, we can identify two main clades with different divergence profiles; first and most divergent &#x201C;A2&#x201D; clade, although the first strain observed was from China.<li>We generated a phylogenetic and divergence tree, as well as a genomic epidemiology map based on the 30,983 genomes of SARS-CoV-2 using <b>NextStrain</b> tools (<uri xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</uri>) [<xref rid="B63-pathogens-09-00829" ref-type="bibr"><u>63</u></xref>].<li> [63](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33035262">33035262</a> <b>(1)</b> Preparedness needs research: How fundamental science and international collaboration accelerated the response to COVID-19<ul><li> for example, the <b>Nextstrain</b> platform [<xref rid="ppat.1008902.ref025" ref-type="bibr"><u>25</u></xref>] is being used to integrate genomic, geographical, and temporal data to visualise patterns of SARS-CoV-2 spread, whilst CoV-GLUE, an online resource created using the Genes Linked by Underlying Evolution (GLUE) [<xref rid="ppat.1008902.ref026" ref-type="bibr"><u>26</u></xref>] software environment, focuses on identifying and tracking new variants.<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30563445">30563445</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33034798">33034798</a> <b>(4)</b> Whole genome sequencing and phylogenetic classification of Tunisian SARS-CoV-2 strains from patients of the Military Hospital in Tunis<ul><li>SNP and phylogenetic analyses were performed using a local installation of the <b>nextstrain</b>.org pipeline [<xref ref-type="bibr" rid="CR12"><u>12</u></xref>].<li> Sample MHT_1 belongs to clade 20A (<b>Nextstrain</b>.org nomenclature [<xref ref-type="bibr" rid="CR13"><u>13</u></xref>] matches Pangolin lineage B.1 [<xref ref-type="bibr" rid="CR14"><u>14</u></xref>]) with the characteristic mutation D614G (SNP A23403G) of the SARS-CoV-2 spike protein assumed to enhance viral infectivity [<xref ref-type="bibr" rid="CR15"><u>15</u></xref>]; sample MHT_2 is clustering within clade 19B (Pangolin lineage A).<li>available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [13](Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming</ext-link>. Accessed on Jun 25, 2020) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32596695">32596695</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33200127">33200127</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33031371">33031371</a> <b>(29)</b> Real-time tracking of Tomato brown rugose fruit virus (ToBRFV) outbreaks in the Netherlands using <b>Nextstrain</b><ul><li> We used <b>Nextstrain</b> [<xref rid="pone.0234671.ref023" ref-type="bibr"><u>23</u></xref>], a collection of open-source bio-informatic tools, to create an interactive view of the diversity and spread of the virus.<li> Real-time virus genome sequencing and rapid dissemination of epidemiologically relevant results, which is central in the <b>Nextstrain</b> philosophy, has been mainly applied to viruses causing disease in humans, such as Zika and Ebola [<xref rid="pone.0234671.ref024" ref-type="bibr"><u>24</u></xref>].<li>At the time of preparation of this report, public ToBRFV sequences and sequences generated in this study dating from November 2014 to November 2019 were included in the ToBRFV <b>Nextstrain</b> build.<li> The ToBRFV <b>Nextstrain</b> build can be accessed via <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.nrcnvwa.nl/ToBRFV/20191231">https://<b>nextstrain</b>.nrcnvwa.nl/ToBRFV/20191231</ext-link> (last accessed: 30 July 2020).<li><b>Nextstrain</b> implementation.<li><b>Nextstrain</b> is a bioinformatics pipeline that uses two tools, Augur and Auspice [<xref rid="pone.0234671.ref023" ref-type="bibr"><u>23</u></xref>].<li> Augur (github.com/<b>Nextstrain</b>/augur) is a bioinformatics toolkit for phylogenetic analysis that uses a series of modules that produces output that subsequently can be visualized by Auspice (github.com/<b>Nextstrain</b>/auspice).<li> The ToBRFV <b>Nextstrain</b> build is deposited on GitHub (<ext-link ext-link-type="uri" xlink:href="https://github.com/NPPO-NL/<b>nextstrain</b>-ToBRFV">https://github.com/NPPO-NL/<b>nextstrain</b>-ToBRFV</ext-link>).<li> 39941596_B) in independent sequencing runs as repeatability control produced two identical ToBRFV genome sequences (named 39941596 B-1 and 39941596 B-2 in the <b>Nextstrain</b> build).<li> Clustering obtained with RAxML, which is part of the <b>Nextstrain</b> build, results in the same grouping as the Bayesian analysis for the viral strains obtained from the Dutch outbreak locations, but some rearrangements can be seen between clusters (<xref ref-type="supplementary-material" rid="pone.0234671.s003"><u>S3 Fig</u></xref>).<li><b>Nextstrain</b> build.<li>The ToBRFV <b>Nextstrain</b> build contains 63 complete ToBRFV genomes generated from material sampled between November 2014 and November 2019.<li> Based on the information provided in the build, <b>Nextstrain</b> determines the most likely transmission events, which can be animated from the webpage.<li> The public <b>Nextstrain</b> tool has been instrumental in visualizing the genomic data in the context of geographic distribution and multi-layered epidemiological traits for several viral and bacterial human pathogens including the novel coronavirus causing the ongoing COVID-19 pandemic (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>, last accessed 24-7-2020).<li> In the plant health domain, two <b>Nextstrain</b> builds are available.<li> <italic>tritici</italic> [<xref rid="pone.0234671.ref036" ref-type="bibr"><u>36</u></xref>], and one for cassava-infecting geminiviruses in Africa (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/pestdisplace/CMDAFRICA">https://<b>nextstrain</b>.org/community/pestdisplace/CMDAFRICA</ext-link>, last accessed 24-7-2020).<li>The ToBRFV <b>Nextstrain</b> build contains 63 (near) complete ToBRFV genomes, 54 of which have been generated in this study.<li>The ToBRFV <b>Nextstrain</b> build currently holds ToBRFV genomes from various outbreak locations in the Middle East, North-America and Europe.<li> The ToBRFV <b>Nextstrain</b> build will be maintained by NPPO-NL.<li>Comparison of the <b>Nextstrain</b> RAxML tree (left) and the Bayesian inference of phylogeny (right) of 63 ToBRFV genomes included in this study.<li> The <b>Nextstrain</b> tree is colored based on the country of origin, whereas the Bayesian tree is colored based on the main clusters identified for the viral genomes sequenced in this study.<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29129921">29129921</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [36](Adams TM, Neher RA, Saunders DGO. Tracking wheat yellow rust: near real-time data sharing with Nextstrain (Version 1.0) 2020. <pub-id pub-id-type="doi">10.5281/zenodo.3758079</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33050053">33050053</a> <b>(4)</b> Evolutionary Analysis of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Reveals Genomic Divergence with Implications for Universal Vaccine Efficacy<ul><li> The <b>Nextstrain</b> database is continuously updating the information on the phylogenetic analysis and genomic divergence of SARS-CoV-2 with concomitant updates on the evolutionary changes [<xref rid="B28-vaccines-08-00591" ref-type="bibr"><u>28</u></xref>].<li> The genomic divergence is still continuing as evident from <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</uri>).<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33033018">33033018</a> <b>(1)</b> The Seattle Flu Study: a multiarm community-based prospective study protocol for assessing influenza prevalence, transmission and genomic epidemiology<ul><li> Geospatial incidence maps are estimated using generalised additive mixed models with spatial, temporal, age and non-structured random effects.<xref rid="R14" ref-type="bibr"><u>14</u></xref> Phylogenetic analyses are based on the <b>Nextstrain</b> pipeline.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19966070">19966070</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33026453">33026453</a> <b>(1)</b> Analysis of Genomic Characteristics and Transmission Routes of Patients With Confirmed SARS-CoV-2 in Southern California During the Early Stage of the US COVID-19 Pandemic<ul><li> Sample percentages were calculated based on their distribution within <b>Nextstrain</b> global clades.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33123498">33123498</a> <b>(2)</b> Within-Host Diversity of SARS-CoV-2 in COVID-19 Patients With Variable Disease Severities<ul><li> Of these, 14,085, in the RNA-dependent RNA polymerase (RdRp) coding region, was not found to be a variation hotspot in SARS-CoV-2 sequences deposited in GISAID (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33031143">33031143</a> <b>(3)</b> The Direct and Indirect Impact of SARS-CoV-2 Infections on Neonates<ul><li> MN908947.3, <ext-link ext-link-type="uri" xlink:href="https://github.com/czbiohub/sc2-msspe-bioinfo">https://github.com/czbiohub/sc2-msspe-bioinfo</ext-link>).<sup><xref rid="R15" ref-type="bibr"><u>15</u></xref>,<xref rid="R16" ref-type="bibr"><u>16</u></xref></sup> Consensus sequences of the 9 SARS-CoV-2 genomes were uploaded on GISAID<sup><xref rid="R17" ref-type="bibr"><u>17</u></xref></sup> and subsequently analyzed using <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>).<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32527780">32527780</a>) [16](<collab>czbiohub/sc2-illumina-pipeline</collab>. <article-title>Chan Zuckerberg Biohub.</article-title><year>2020</year><comment><ext-link ext-link-type="uri" xlink:href="https://github.com/czbiohub/sc2-illumina-pipeline">https://github.com/czbiohub/sc2-illumina-pipeline</ext-link>. Accessed July 16, 2020</comment>.) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33022254">33022254</a> <b>(2)</b> Microbe hunting in the modern era: reflecting on a decade of microbial genomic epidemiology<ul><li> Similar epidemiological studies cross-referencing time, place, and person with pathogen genome sequences have allowed reconstruction of the spread of Ebola virus during the recent outbreaks in West Africa, and are also being used in our present day efforts to determine the origins [<xref rid="bib21" ref-type="bibr"><u>21</u></xref>] and trace the global spread of SARS-CoV-2 (see <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="intref0010">https://<b>nextstrain</b>.org/ncov</ext-link>).<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32015507">32015507</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33020502">33020502</a> <b>(3)</b> SARS-CoV-2 spike protein predicted to form complexes with host receptor protein orthologues from a broad range of mammals<ul><li> SARS-CoV-2 strains from animal infections fall among the phylogenetic diversity observed in a representative set of human strains (Fig.&#xA0;<xref rid="Fig5" ref-type="fig"><u>5</u></xref>a), as also seen in larger phylogenetic analyses available on <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32869037">32869037</a> <b>(15)</b> Genomic surveillance of Nevada patients revealed prevalence of unique SARS-CoV-2 variants bearing mutations in the RdRp gene<ul><li>A set of 3,644 complete, high-coverage SARS-CoV-2 genomes reported in the July 15, 2020 <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org"><b>Nextstrain</b>.org</ext-link> global subsample and metadata were obtained from GISAID and combined with our own samples to determine their phylogenetic placement [<xref rid="R21" ref-type="bibr"><u>21</u></xref>,<xref rid="R22" ref-type="bibr"><u>22</u></xref>].<li> The combined sets of global and Nevadan samples were aligned together, with metadata, by the <italic>augur</italic> phylodynamic pipelines of the <italic>ncov</italic> build of the <italic><b>nextstrain</b></italic> command-line tool, version 2.0.0.post1 [<xref rid="R23" ref-type="bibr"><u>23</u></xref>].<li> We used a subsampling of sequence data from <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org"><b>Nextstrain</b>.org</ext-link> to assess the frequency of D614G in the United States and globally during the same time period (March 6 to June 5) (<xref rid="F2" ref-type="fig">Fig.<li>Worldwide, there are currently 5 main clades (19A, 19B, 20A, 20B, 20C) of SARS-CoV-2 differentiated based on specific nucleotide profiles in the Year-letter scheme of <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org</ext-link>.<li> We used a subsampling of <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org"><b>Nextstrain</b>.org</ext-link> data to assess the frequency of clades in the United States and globally by continent during the same time period (March 6 to June 5).<li> We used a subsampling of <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org"><b>Nextstrain</b>.org</ext-link> data to assess the frequency of P323L/F in the United States and globally during the same time period (March 6 to June 5).<li> Dendrogram of Nevada specimens in context of other sequenced specimens.</bold> (a) Nucleotide mutation clock from <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org"><b>Nextstrain</b>.org</ext-link> with the SARS-CoV-2 genome isolated from Washington (USA/WA 1/2020) on January 24th and first specimen from Nevada (A0004) on March 5th indicated in red.<li> (b) Circular dendrogram of Nevada specimens from March 6th to June 5th positioned within a subsample of global sequences from <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org"><b>Nextstrain</b>.org</ext-link> during the same time period, the larger dots indicate specimens from Nevada.<li> (b) Pie chart of the clades in each indicated country from March 6th to June 5th generated from a subsampling of sequences deposited in <ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org"><b>Nextstrain</b>.org</ext-link>.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [22](<collab>Nextstrain</collab>. (<year>2020</year>) <article-title>Genomic epidemiology of novel coronavirus - Global subsampling</article-title>. In: <source>Nextstrainorg [Internet]</source><day>15</day><month>7</month> 2020 [<date-in-citation>cited 15 Jul 2020</date-in-citation>] Available: <comment><ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global/2020-07-15?d=tree&amp;l=clock&amp;legend=closed">https://nextstrain.org/ncov/global/2020-07-15?d=tree&amp;l=clock&amp;legend=closed</ext-link></comment>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33006981">33006981</a> <b>(3)</b> Similarity between mutation spectra in hypermutated genomes of rubella virus and in SARS-CoV-2 genomes accumulated during the COVID-19 pandemic<ul><li> Based on the published analysis of the GISAID data for a subset of around 4000 of SARS-CoV-2 isolates across the world performed with the use of the <b>Nextstrain</b> package ([<xref rid="pone.0237689.ref035" ref-type="bibr"><u>35</u></xref>] and <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?l=clock">https://<b>nextstrain</b>.org/ncov/global?l=clock</ext-link>), an average lineage of SARS-CoV-2 virus successfully transmitted from one subject to another would accumulate approximately 22 base substitutions per year (12&#x2013;13 base substitutions for the period of December&#x2014;June, 2020); a similar estimate was also obtained in [<xref rid="pone.0237689.ref036" ref-type="bibr"><u>36</u></xref>]).<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [36](<name><surname>Chiara</surname><given-names>M</given-names></name>, <name><surname>Horner</surname><given-names>DS</given-names></name>, <name><surname>Gissi</surname><given-names>C</given-names></name>, <name><surname>Pesole</surname><given-names>G</given-names></name>. <article-title>Comparative genomics suggests limited variability and similar evolutionary patterns between major clades of SARS-CoV-2</article-title>. <source>bioRxiv</source><year>2020</year>:2020.03.30.016790. <pub-id pub-id-type="doi">10.1101/2020.03.30.016790</pub-id>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32878977">32878977</a> <b>(1)</b> Mapping genome variation of SARS-CoV-2 worldwide highlights the impact of COVID-19 super-spreaders<ul><li>We used a nomenclature of clades that is as conservative as possible with <b>Nextstrain</b> but avoiding alteration of basic cladistic rules (e.g., B1 cannot be ancestral of B).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32908258">32908258</a> <b>(5)</b> The UCSC SARS-CoV-2 Genome Browser<ul><li> This sequence is widely used as a standard reference, and, because of its early identification, it is used as the root genome in phylogenetic trees produced by <b>Nextstrain</b> (<xref rid="R17" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), The COG-UK (<xref rid="R33" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) and the China National Center for Bioinformation project (<xref rid="R47" ref-type="bibr"><u>Zhao et al., 2020</u></xref>).<li> &#x201C;<b>Nextstrain</b>Vars&#x201D; (<xref rid="F9" ref-type="fig"><u>Figure 9</u></xref>) contains the time-stamped molecular phylogenetic tree produced by the <b>Nextstrain</b> team (<xref rid="R17" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) based on complete and quality-controlled viral genomes from GISAID (<xref rid="R37" ref-type="bibr"><u>Shu &amp; McCauley, 2017</u></xref>).<li> In the tree, samples are sorted according to their order in the JSON file produced by <b>Nextstrain</b> describing the phylogenetic tree, and appear in different colors according to clades identified by <b>Nextstrain</b>.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut et al., 2020](<name><surname>Rambaut</surname><given-names>A</given-names></name>, <name><surname>Holmes</surname><given-names>EC</given-names></name>, <name><surname>Hill</surname><given-names>V</given-names></name>, <name><surname>O&#x2019;Toole</surname><given-names>&#xC1;</given-names></name>, <name><surname>McCrone</surname><given-names>JT</given-names></name>, <name><surname>Ruis</surname><given-names>C</given-names></name>, <name><surname>du Plessis</surname><given-names>L</given-names></name>, &amp; <name><surname>Pybus</surname><given-names>OG</given-names></name> (<year>2020</year>). <article-title>A dynamic nomenclature proposal for SARS-CoV-2 to assist genomic epidemiology</article-title>. In <source>bioRxiv</source> (p. 2020.04.17.046086). <pub-id pub-id-type="doi">10.1101/2020.04.17.046086</pub-id>) [Zhao et al., 2020](<name><surname>Zhao</surname><given-names>W-M</given-names></name>, <name><surname>Song</surname><given-names>S-H</given-names></name>, <name><surname>Chen</surname><given-names>M-L</given-names></name>, <name><surname>Zou</surname><given-names>D</given-names></name>, <name><surname>Ma</surname><given-names>L-N</given-names></name>, <name><surname>Ma</surname><given-names>Y-K</given-names></name>, <name><surname>Li</surname><given-names>R-J</given-names></name>, <name><surname>Hao</surname><given-names>L-L</given-names></name>, <name><surname>Li</surname><given-names>C-P</given-names></name>, <name><surname>Tian</surname><given-names>D-M</given-names></name>, <name><surname>Tang</surname><given-names>B-X</given-names></name>, <name><surname>Wang</surname><given-names>Y-Q</given-names></name>, <name><surname>Zhu</surname><given-names>J-W</given-names></name>, <name><surname>Chen</surname><given-names>H-X</given-names></name>, <name><surname>Zhang</surname><given-names>Z</given-names></name>, <name><surname>Xue</surname><given-names>Y-B</given-names></name>, &amp; <name><surname>Bao</surname><given-names>Y-M</given-names></name> (<year>2020</year>). <source>The 2019 novel coronavirus resource</source>. <comment>Yi Chuan = Hereditas / Zhongguo Yi Chuan Xue HuiBian Ji, 4</comment><volume>2</volume>(<issue>2</issue>), <fpage>212</fpage>&#x2013;<lpage>221</lpage>. <pub-id pub-id-type="doi">10.16288/j.yczz.20-030</pub-id>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Shu &amp; McCauley, 2017](<name><surname>Shu</surname><given-names>Y</given-names></name>, &amp; <name><surname>McCauley</surname><given-names>J</given-names></name>. (<year>2017</year>). <article-title>GISAID: Global initiative on sharing all influenza data &#x2013; from vision to reality</article-title>. In <source>Eurosurveillance</source> (Vol. <volume>22</volume>, Issue <issue>13</issue>). <pub-id pub-id-type="doi">10.2807/1560-7917.es.2017.22.13.30494</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33024961">33024961</a> <b>(8)</b> SARS-CoV-2 gene content and COVID-19 mutation impact by comparing 44 Sarbecovirus genomes<ul><li>Single nucleotide variants were downloaded from the &#x201C;<b>Nextstrain</b> Vars&#x201D; track in the UCSC Table Browser on 2020&#x2013;04-18 at 11:46 AM EDT.<li> UCSC data came from <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link><sup><xref rid="R55" ref-type="bibr"><u>55</u></xref></sup>, which was derived from genome sequences deposited in GISAID<sup><xref rid="R35" ref-type="bibr"><u>35</u></xref></sup>.<li> Right of use and publication of the underlying sequences is entirely controlled by the authors of the original resource and the contributors of individual sequences, who are acknowledged in the <b>Nextstrain</b> metadata file included with supplementary materials.<li> Please contact the respective authors (available via the <b>Nextstrain</b> metadata.tsv file) if you intend to carry out further research using their data.<li> Derived data, such as phylogenies, can be downloaded from <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link> (see &#x201C;DOWNLOAD DATA&#x201D; link at bottom of page) - please contact the relevant authors where appropriate.&#x201D;.<li>Supplementary Material</title>      <supplementary-material content-type="local-data" id="SD1">        <label>Supplement</label>        <media xlink:href="SIGuide.doc" orientation="portrait" id="d38e1041" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD2">        <label>Supplement</label>        <media xlink:href="SupplementaryData.pdf" orientation="portrait" id="d38e1045" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD3">        <label>Supplement</label>        <media xlink:href="44Sarbecoviruses.nh.txt" orientation="portrait" id="d38e1049" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD4">        <label>Supplement</label>        <media xlink:href="44Sarbecoviruses.fa.txt" orientation="portrait" id="d38e1053" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD5">        <label>Supplement</label>        <media xlink:href="PhyloCSFGenes.bed.txt" orientation="portrait" id="d38e1057" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD6">        <label>Supplement</label>        <media xlink:href="PhyloCSFRejectedGenes.bed.txt" orientation="portrait" id="d38e1061" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD7">        <label>Supplement</label>        <media xlink:href="<b>nextstrain</b>ncovglobalmetadata.txt" orientation="portrait" id="d38e1065" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD8">        <label>Supplement</label>        <media xlink:href="SupplementaryTableS1.txt" orientation="portrait" id="d38e1069" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD9">        <label>Supplement</label>        <media xlink:href="SupplementaryTableS2.txt" orientation="portrait" id="d38e1073" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD10">        <label>Supplement</label>        <media xlink:href="SupplementaryTableS3.txt" orientation="portrait" id="d38e1077" position="anchor"/>      </supplementary-material>      <supplementary-material content-type="local-data" id="SD11">        <label>Supplement</label>        <media xlink:href="SupplementaryTableS4.txt" orientation="portrait" id="d38e1082" position="anchor"/>      </supplementary-material>    </sec>  </body>.<li> [55](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33024981">33024981</a> <b>(7)</b> Viral genomes reveal patterns of the SARS-CoV-2 outbreak in Washington State<ul><li> Since its first occurrence, this amino acid substitution from aspartate (D) to glycine (G) at position 614 of the Spike protein increased in relative frequency around the world (visible at: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614">https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614</ext-link>) and now represents the vast majority of all new cases of COVID-19 (<xref rid="R3" ref-type="bibr"><u>3</u></xref>&#x2013;<xref rid="R5" ref-type="bibr"><u>5</u></xref>).<li> The local transmission clusters obtained are shown at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/blab/ncov/wa-phylodynamics?c=cluster_size">https://<b>nextstrain</b>.org/groups/blab/ncov/wa-phylodynamics?c=cluster_size</ext-link> and their size distribution and D614G makeup shown in <xref rid="SD2" ref-type="supplementary-material"><u>Figure S1</u></xref>.<li> To do so, we first built a timed tree using sequences from Washington State and from around the world using the <b>Nextstrain</b> pipeline (<xref rid="R3" ref-type="bibr"><u>3</u></xref>).<li> The G clade consists of all sequences with a glycine at this position (visible at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614">https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614</ext-link>).<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [5](<name><surname>Yurkovetskiy</surname><given-names>L.</given-names></name>, <name><surname>Pascal</surname><given-names>K. E.</given-names></name>, <name><surname>Tompkins-Tinch</surname><given-names>C.</given-names></name>, <name><surname>Nyalile</surname><given-names>T.</given-names></name>, <name><surname>Wang</surname><given-names>Y.</given-names></name>, <name><surname>Baum</surname><given-names>A.</given-names></name>, <name><surname>Diehl</surname><given-names>W. E.</given-names></name>, <name><surname>Dauphin</surname><given-names>A.</given-names></name>, <name><surname>Carbone</surname><given-names>C.</given-names></name>, <name><surname>Veinotte</surname><given-names>K.</given-names></name>, <name><surname>Egri</surname><given-names>S. B.</given-names></name>, <name><surname>Schaffner</surname><given-names>S. F.</given-names></name>, <name><surname>Lemieux</surname><given-names>J. E.</given-names></name>, <name><surname>Munro</surname><given-names>J.</given-names></name>, <name><surname>Sabeti</surname><given-names>P. C.</given-names></name>, <name><surname>Kyratsous</surname><given-names>C.</given-names></name>, <name><surname>Shen</surname><given-names>K.</given-names></name>, <name><surname>Luban</surname><given-names>J.</given-names></name>, <article-title>SARS-CoV-2 Spike protein variant D614G increases infectivity and retains sensitivity to antibodies that target the receptor binding domain</article-title>, <source>bioRxiv</source> (<year>2020</year>), doi: <pub-id pub-id-type="doi">10.1016/j.cell.2020.09.032</pub-id>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33003465">33003465</a> <b>(3)</b> Performance of Targeted Library Preparation Solutions for SARS-CoV-2 Whole Genome Analysis<ul><li> Next to a number of solutions utilizing the advantages of RT-qPCR techniques for SARS-CoV-2 detection [<xref rid="B5-diagnostics-10-00769" ref-type="bibr"><u>5</u></xref>,<xref rid="B6-diagnostics-10-00769" ref-type="bibr"><u>6</u></xref>], the next-generation sequencing (NGS)-based protocols allows the analysis of SARS-CoV-2 genome evolution and variability and the monitoring of its spread within the global population (<b>Nextstrain</b>; <uri xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</uri>) [<xref rid="B7-diagnostics-10-00769" ref-type="bibr"><u>7</u></xref>,<xref rid="B8-diagnostics-10-00769" ref-type="bibr"><u>8</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32745627">32745627</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32824767">32824767</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32092483">32092483</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32007145">32007145</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32995794">32995794</a> <b>(3)</b> COVID-19 CG: Tracking SARS-CoV-2 mutations by locations and dates of interest<ul><li>Based on best practices, we filter out sequences meeting any of the following criteria: (1) Present on the <b>NextStrain</b>&#x2019;s exclusion list (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt</ext-link>), (2) Isolates from non-humans (animals, environmental samples, etc), (3) genome length less than 29,700 nt, or (4) &gt;5% ambiguous base calls.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33243941">33243941</a> <b>(3)</b> Characteristics of viral specimens collected from asymptomatic and fatal cases of COVID-19<ul><li> The alteration at base 18377 was seen in four other sequences submitted to <b>nextstrain</b>.org out of 3104 total genomic sequences submitted as of June 20, 2020.<li> The T&gt;C base change seen at 28187 was not previously observed according to <b>nextstrain</b>.org.<li> The polymorphisms are extremely rare in the global database (<b>nextstrain</b>.org), thusly and any association with fatality outside of the cases herein is non-existent at present.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32492206">32492206</a> <b>(7)</b> Understanding the possible origin and genotyping of the first Bangladeshi SARS&#x2010;CoV&#x2010;2 strain<ul><li>First, Bangladesh reported the first case of COVID&#x2010;19 on 8 March 2020 having a travel history from Italy.<xref rid="jmv26115-bib-0002" ref-type="bibr"><sup>2</sup></xref> The first complete sequence data is of the virus strain sampled from a 22 years aged female patient on 18 April 2020, although not mentioned, possibly the person had no recent travel history.<xref rid="jmv26115-bib-0001" ref-type="bibr"><sup>1</sup></xref> Phylodynamics data found in the <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>) showed that the time to most recent common ancestor (tMRCA) of this Bangladeshi strain was 14 March 2020 spanning the time interval range between 7 March and 23 March 2020 (Figure&#xA0;<xref rid="jmv26115-fig-0001" ref-type="fig"><u>1</u></xref>).<li> Image was taken from the <b>Nextstrain</b>,<xref rid="jmv26115-bib-0003" ref-type="bibr"><sup>3</sup></xref> link <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global?f_country=Bangladesh" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global?f_country=Bangladesh</ext-link>.<li> As <b>Nextstrain</b> or GISAID deals with a large number of data set (25,246 as of 14<sup>th</sup> April 2020) and generates an updated picture of the evolutionary timeframe using the maximum clade credibility (MCC) tree, at this stage, these results can be considered more reliable.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33013021">33013021</a> <b>(1)</b> Global Biological Threats: Novel Tools and Multi-Disciplinary Approaches to Sustainable Development<ul><li> In the ensuing 7&#xA0;months, over 80,000 SARS-CoV-2 genome sequences are available in the Global Initiative on Sharing All Influenza Data (GISAID) public database<sup><xref ref-type="bibr" rid="CR10"><u>10</u></xref></sup>, allowing real-time tracking of virus evolution and its movement around the globe, through an open-source project called <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR11"><u>11</u></xref></sup>.<li> [10](Global Initiative on Sharing All Influenza Data, 2020. <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</ext-link>. Accessed 9 Aug 2020) [11](Genomic epidemiology of novel coronavirus&#x2014;Global subsampling. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global?c=region">https://nextstrain.org/ncov/global?c=region</ext-link>. Accessed 9 Aug 2020) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32995830">32995830</a> <b>(1)</b> Reinfection with SARS-CoV-2 and Failure of Humoral Immunity: a case report.<ul><li>Viral sequencing in March was performed via rapid metagenomic next-generation sequencing (NGS),<sup><xref rid="R23" ref-type="bibr"><u>23</u></xref></sup> and in July was modified from the multiplexed PCR amplicon NGS method using the ARTIC V3 primers.<sup><xref rid="R24" ref-type="bibr"><u>24</u></xref></sup> SARS-CoV-2 clade designations and phylogenetic analyses were produced using <b>NextStrain</b>.<li> [23](<name><surname>Bedford</surname><given-names>T</given-names></name>, <name><surname>Greninger</surname><given-names>AL</given-names></name>, <name><surname>Roychoudhury</surname><given-names>P</given-names></name>, <etal/><article-title>Cryptic transmission of SARS-CoV-2 in Washington state</article-title>. <source>Science</source>. <month>9</month><year>2020</year>;doi:<pub-id pub-id-type="doi">10.1126/science.abc0523</pub-id>) [24](<name><surname>Quick</surname><given-names>J.</given-names></name><source>nCoV-2019 sequencing protocol v3</source>. <comment><ext-link ext-link-type="uri" xlink:href="https://github.com/ioshauick/artic-ncov2019/blob/master/primer_schemes/nCoV-2019/V3/nCoV-2019.tsv">https://github.com/ioshauick/artic-ncov2019/blob/master/primer_schemes/nCoV-2019/V3/nCoV-2019.tsv</ext-link></comment>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32989413">32989413</a> <b>(2)</b> Genomic Epidemiology and Recent Update on Nucleic Acid&#x2013;Based Diagnostics for COVID-19<ul><li> In this case, a number of publically available open-access platforms such as GISAID and associated sources (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>) have proven to be useful and easily accessible tools to understand the COVID-19 pandemic by using the available genomic sequence of the SARS-CoV-2 virus [<xref ref-type="bibr" rid="CR12"><u>12</u></xref>].<li> [12](GISAID. Genomic epidemiology of novel coronavirus. 2020 Available at <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov">https://nextstrain.org/ncov</ext-link> Last accessed 30th May 2020.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32972934">32972934</a> <b>(3)</b> Coding-Complete Genome Sequences of Three SARS-CoV-2 Strains from Bangladesh<ul><li> The figure was condensed using <b>Nextstrain</b> (<xref rid="B3" ref-type="bibr"><u>3</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?dmax=2020-06-10&amp;f_country=Bangladesh">https://<b>nextstrain</b>.org/ncov/global?dmax=2020-06-10&amp;f_country=Bangladesh</ext-link>).<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32973171">32973171</a> <b>(1)</b> Sequence analysis of SARS-CoV-2 genome reveals features important for vaccine design<ul><li> Furthermore, we examined genetic diversity data from <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup>, accounting for 4,675 SARS-CoV-2 genomes.<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32995774">32995774</a> <b>(11)</b> Rapid detection of inter-clade recombination in SARS-CoV-2 with Bolotie<ul><li> Additionally, 15 out of 225 potential recombinants were found present in the set of 4,039 representative genomes used by <b>NextStrain</b> (<xref rid="SD1" ref-type="supplementary-material"><u>Supplementary Table 2</u></xref>).<li> In our analysis, Bolotie identified 15 out of 4,039 sequences in the set of sequences used by <b>NextStrain</b> (<xref rid="R6" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) as recombinants or as having mutational anomalies (<xref rid="SD1" ref-type="supplementary-material"><u>Supplementary Table 2</u></xref>).<li> Minor differences between the <b>NextStrain</b> tree and the tree computed from Bolotie consensus sequences are to be expected since consensus sequences have 200 bases replaced with the reference at both 3&#x2019; and 5&#x2019; ends and do not include any structural variations.<li> Additionally, since <b>NextStrain</b> tree includes 455 isolates submitted to GISAID after we downloaded our latest set, those additional sequences are also expected to slightly alter the topology.<li> As shown in <xref rid="SD1" ref-type="supplementary-material"><u>Supplementary Table 1</u></xref>, one such discrepancy does exist between clades 1 and 3, where <b>NextStrain</b> classification adds 334 sequences from clade GH to clade G.<li> Lineages S, L, V and O were grouped together, similarly to <b>NextStrain</b> (<xref rid="R6" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), and based on the close distances observed in our independent phylogenetic analyses (<xref rid="F3" ref-type="fig"><u>Figure 3</u></xref>).<li> Mappings between clade IDs in our analysis, those defined by GISAID, and those defined by <b>NextStrain</b> are provided in <xref rid="SD1" ref-type="supplementary-material"><u>Supplementary Table 1</u></xref>.<li>To test how well information is preserved in our consensus sequences, we obtained a pre-computed tree from <b>NextStrain</b> (<xref rid="R6" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) which contained a total of 4,494 representative genomes chosen by <b>NextStrain</b>.<li> Only 4,039 genomes from those in the <b>NextStrain</b> tree were available on GISAID at the time when we obtained genome assemblies for our analysis.<li>First, we re-built the tree using the set of 4,039 genomes using the general time-reversible model as used by the <b>NextStrain</b> platform and allowing IQ-TREE (<xref rid="R13" ref-type="bibr"><u>Minh et al., 2020</u></xref>) to automatically choose the precise model.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Minh et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32966272">32966272</a> <b>(1)</b> Serial Testing for SARS-CoV-2 and Virus Whole Genome Sequencing Inform Infection Risk at Two Skilled Nursing Facilities with COVID-19 Outbreaks &#x2014; Minnesota, April&#x2013;June 2020<ul><li> Phylogenetic relationships, including distinct clustering of viral whole genome sequences, were inferred based on nucleotide differences via IQ-TREE, using general time reversible substitution models (<xref rid="R9" ref-type="bibr"><italic>9</italic></xref>) as a part of the <b>Nextstrain</b> workflow (<xref rid="R10" ref-type="bibr"><italic>10</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33518937">33518937</a> <b>(1)</b> Family tree of a deadly virus<ul><li> These are constantly slotted into an ever-growing family tree by the pathogen-tracking project <b>Nextstrain</b> at Fred Hutchinson Cancer Research Center in Seattle.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33200080">33200080</a> <b>(8)</b> A Distinct Phylogenetic Cluster of Indian Severe Acute Respiratory Syndrome Coronavirus 2 Isolates<ul><li> Furthermore, 10 high-quality genomes from each of the 10 clades, respectively, as annotated by <b>Nextstrain</b> were retrieved from GISAID and used in the analysis.<li>Phylogenetic analysis of the samples was performed as detailed previously following the standard protocol for analysis of SARS-CoV-2 genomes provided by <b>Nextstrain</b> [<xref rid="CIT0011" ref-type="bibr"><u>11</u></xref>, <xref rid="CIT0012" ref-type="bibr"><u>12</u></xref>].<li> Six of these clusters are known clades identified by <b>Nextstrain</b>: A1a, A2a, A3, B, B1, and B4 [<xref rid="CIT0017" ref-type="bibr"><u>17</u></xref>].<li>Phylogenetic clusters and clades as generated by <b>Nextstrain</b> for the dataset of 1377 high-quality Indian severe acute respiratory syndrome coronavirus 2 genomes.<li> This cluster of sequences could not be classified into any of the 10 clade sequences defined by <b>Nextstrain</b>, and it did not share the nucleotide compositions that define any of the 10 clades [<xref rid="CIT0017" ref-type="bibr"><u>17</u></xref>].<li> Therefore, to avoid potential conflict with the nomenclature followed by <b>Nextstrain</b>, we define this cluster of sequences as Clade I/A3i, for the unique occurrence as a dominant cluster among SARS-CoV-2 genome sequences from India, and also because this clade is largely formed by sequences from India (<xref ref-type="supplementary-material" rid="sup9"><u>Supplementary Figures S2</u></xref> and <xref ref-type="supplementary-material" rid="sup9"><u>S3</u></xref>).<li> This distinct cluster could not be classified into any of the 10 clade annotations as described by <b>Nextstrain</b>, and it was characterized by a unique combination of 4 variants that was shared by more than 95% of the isolates falling in the cluster.<li> As per <b>Nextstrain</b>, the Indian genomes constituted more than 30% of the global genomes for this cluster.<li> [11](<person-group person-group-type="author"><name name-style="western"><surname>Jolly</surname><given-names>B</given-names></name>, <name name-style="western"><surname>Scaria</surname><given-names>V</given-names></name></person-group><comment>Computational analysis and phylogenetic clustering of SARS-nCov-2 genomes.</comment> Available at: <ext-link ext-link-type="uri" xlink:href="https://docs.google.com/document/d/1B5NxWFwsRz_vD5Y6EwjKxkRamsPLVfs1MjVoziU1Zq0/edit?usp=sharing&amp;usp=embed_facebook">https://docs.google.com/document/d/1B5NxWFwsRz_vD5Y6EwjKxkRamsPLVfs1MjVoziU1Zq0/edit?usp=sharing&amp;usp=embed_facebook</ext-link>. Accessed <comment>31 August 2020</comment>.) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [17](<person-group person-group-type="author"><name name-style="western"><surname>Bedford</surname><given-names>T</given-names></name>, <name name-style="western"><surname>Neher</surname><given-names>R</given-names></name>, <name name-style="western"><surname>Hadfield</surname><given-names>J</given-names></name>, <etal>et al.</etal></person-group> &#xA0;<comment>Nextstrain. Nextstrain/ncov. GitHub.</comment> Available at: <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</ext-link>. Accessed <comment>30 August 2020</comment>.) [17](<person-group person-group-type="author"><name name-style="western"><surname>Bedford</surname><given-names>T</given-names></name>, <name name-style="western"><surname>Neher</surname><given-names>R</given-names></name>, <name name-style="western"><surname>Hadfield</surname><given-names>J</given-names></name>, <etal>et al.</etal></person-group> &#xA0;<comment>Nextstrain. Nextstrain/ncov. GitHub.</comment> Available at: <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</ext-link>. Accessed <comment>30 August 2020</comment>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32946527">32946527</a> <b>(2)</b> Disentangling primer interactions improves SARS-CoV-2 genome sequencing by multiplex tiling PCR<ul><li> We consider this insight will be helpful especially when revision of primer set (either for V3 or N1) become necessary in future to cope with the ever-increasing diversity of SARS-CoV-2 genomes (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32941419">32941419</a> <b>(30)</b> Genetic grouping of SARS-CoV-2 coronavirus sequences using informative subtype markers for pandemic spread visualization<ul><li> For example, the <b>Nextstrain</b> group has created a massive phylogenetic tree incorporating sequence data and applied a model of the time-based rate of mutation to create a hypothetical map of viral distribution [<xref rid="pcbi.1008269.ref006" ref-type="bibr"><u>6</u></xref>] (available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>).<li>Phylogenetic trees obtained through sequence alignment may be utilized to map viral outbreaks geographically and trace transmission chains [<xref rid="pcbi.1008269.ref022" ref-type="bibr"><u>22</u></xref>, <xref rid="pcbi.1008269.ref023" ref-type="bibr"><u>23</u></xref>] and have been applied to SARS-CoV-2 by, e.g., the <b>Nextstrain</b> group as discussed above.<li> The <b>Nextstrain</b> project has sought to address these challenges by providing their own clade definitions based on whether there are a certain number of mutations at nucleotide positions in the sequence (at least two) and naming clades based on their estimated time of emergence [<xref rid="pcbi.1008269.ref025" ref-type="bibr"><u>25</u></xref>].<li> In addition, viral evolutionary analysis, such as by the <b>Nextstrain</b> group, relies on making assumptions solely on molecular evolution (the degree of sequence similarity and branching points in defining genetic clades and other levels of organization) and not on transmission models.<li><b>Nextstrain</b> maintains a continually-updated, pre-formatted SARS-CoV-2 (novel coronavirus) sequence dataset through GISAID (this dataset also includes sequences of other novel coronavirus sampled from other hosts such as Bat).<li> Our preprocessing pipeline then begins by filtering out sequences that are less than 25000 base pairs (the same threshold used in <b>Nextstrain</b> project built for SARS-CoV-2, <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li>For the aligned sequences, we merged the sequence with the metadata provided by <b>Nextstrain</b> as of June 17, 2020, based on identification number, <bold>gisaid_epi_isl</bold>, provided by GISAID [<xref rid="pcbi.1008269.ref005" ref-type="bibr"><u>5</u></xref>].<li><b>Nextstrain</b> [<xref rid="pcbi.1008269.ref006" ref-type="bibr"><u>6</u></xref>] provides a phylogeny method to track and visualize the dynamic of SARS-CoV-2 sequences.<li> To obtain the results presented here, we downloaded the <b>Nextstrain</b> tree data from <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link> on June 17, 2020.<li> Since both the ISMs and the <b>Nextstrain</b> phylogenetic tree were generated based on the GISAID database, they may be easily compared.<li> 2) ISM clusters defined at different entropy thresholds and <b>Nextstrain</b> defined &#x201C;clades&#x201D;.<li>In <b>Nextstrain</b> data, there are 5 &#x201C;clades&#x201D;, namely, 19A, 19B, 20A, 20B and 20C, defined [<xref rid="pcbi.1008269.ref025" ref-type="bibr"><u>25</u></xref>].<li> Since our ISMs are clusters of sequences with similar genetic variations, the <b>Nextstrain</b> &#x201C;clades&#x201D; provides us a good interface to study how our entropy threshold influences the ISM definition by comparing the overlaps between <b>Nextstrain</b> &#x201C;clades&#x201D; grouped sequences and ISM grouped sequences.<li> We vary the entropy threshold to form different sets of ISM clusters of sequences and compare each set with <b>Nextstrain</b> &#x201C;clades&#x201D; using homogeneity and completeness.<li> that were determined by <b>Nextstrain</b>, in order to demonstrate how ISMs relate to evolutionary relationships predicted by phylogenetic methods.<li> In particular by comparing the ISM-defined groups of sequences identified by our pipeline with the phylogenetic tree-based clusters (i.e., clades) identified by the <b>Nextstrain</b> group [<xref rid="pcbi.1008269.ref006" ref-type="bibr"><u>6</u></xref>].<li> We do so by placing an ISM of interest at the lowest common ancestor (LCA) node of sequences containing that ISM on the phylogenetic tree produced by <b>Nextstrain</b>.<li> Conversely, we observe that the phylogenetic clades identified by <b>Nextstrain</b> are imperfect with respect to their preservation of SNV information.<li> <b>Nextstrain</b> identifies the clades based on whether they contain at least two prevalent SNVs.<li>We further assess how ISMs defined at different entropy thresholds relate to the clades identified by <b>Nextstrain</b>.<li> We compute homogeneity and completeness scores between ISM labels and <b>Nextstrain</b> clades as a function of entropy threshold.<li>The effect of the entropy threshold on ISM membership to <b>Nextstrain</b> clade representation.<li> <italic>Second</italic>, ISM-based subtypes are able to capture the majority of phylogenetic relationships between viral genomes that are represented in <italic><b>Nextstrain</b></italic> tree clades.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30546099">30546099</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23673226">23673226</a>) [25](Hodcroft EB, Hadfield J, Neher RA, Bedford T. Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org. Nextstrainorg. June 2, 2020;.) [5](<name><surname>Shu</surname><given-names>Y</given-names></name>, <name><surname>McCauley</surname><given-names>J</given-names></name>. <article-title>GISAID: Global initiative on sharing all influenza data&#x2014;from vision to reality</article-title>. <source>Eurosurveillance</source>. <year>2017</year>;<volume>22</volume>(<issue>13</issue>). <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [25](Hodcroft EB, Hadfield J, Neher RA, Bedford T. Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org. Nextstrainorg. June 2, 2020;.) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32931703">32931703</a> <b>(1)</b> Sequence Characterization and Molecular Modeling of Clinically RelevantVariants of the SARS-CoV-2 Main Protease<ul><li> five of these unique evolutionary events can be verified in<b>Nextstrain</b>&#x2019;s SARS-CoV-2 phylogenetic tree.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32991842">32991842</a> <b>(2)</b> Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant<ul><li>The frequency of the SARS-CoV-2 D614G S protein variant in published genomic data was examined using the full <b>Nextstrain</b>-curated set of sequences available from GISAID as of 25 June 2020 (<xref rid="bib14" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>; <xref rid="bib39" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>).<li> The identity of the variant at amino acid position 614 was tabulated for the full dataset and reported as frequency by date using the collection dates reported in the <b>Nextstrain</b>-curated metadata file available from GISAID (<xref rid="bib14" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>; <xref rid="bib39" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>).<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Shu and McCauley, 2017](          <person-group person-group-type="author">            <name>              <surname>Shu</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>McCauley</surname>              <given-names>J.</given-names>            </name>          </person-group>          <article-title>GISAID: Global initiative on sharing all influenza data - from vision to reality</article-title>          <source>Euro Surveill.</source>          <volume>22</volume>          <year>2017</year>          <comment>ii&#x2013;30494</comment>        ) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Shu and McCauley, 2017](          <person-group person-group-type="author">            <name>              <surname>Shu</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>McCauley</surname>              <given-names>J.</given-names>            </name>          </person-group>          <article-title>GISAID: Global initiative on sharing all influenza data - from vision to reality</article-title>          <source>Euro Surveill.</source>          <volume>22</volume>          <year>2017</year>          <comment>ii&#x2013;30494</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32925078">32925078</a> <b>(3)</b> COVID-19: Review of a 21st Century Pandemic from Etiology to Neuro-psychiatric Implications<ul><li> <b>Nextstrain</b>.org identifies 5 major clades that correspond to GISAID nomenclature (in parenthesis): 19A (L) &amp; (V), 19B (S), 20A (G), 20B (GR), 20C (GH).<li> Last updated 07-24-2020 <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global/gisaid?c=GISAID_clade&amp;p=full">https://<b>nextstrain</b>.org/ncov/global/gisaid?c=GISAID_clade&amp;p=full</ext-link> [<xref rid="ref035" ref-type="bibr"><u>35</u></xref>].<li> [35](GISAID, Genomic epidemiology of hCoV-19, <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/epiflu-applications/next-hcov-19-app/">https://www.gisaid.org/epiflu-applications/next-hcov-19-app/</ext-link>, Last Updated 27 July 2020, Accessed 27 July 2020.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32937868">32937868</a> <b>(1)</b> Spatio-Temporal Mutational Profile Appearances of Swedish SARS-CoV-2 during the Early Pandemic<ul><li> During the pandemic, there have been frequent updates for new sequenced isolates with evolving nomenclature systems for SARS-CoV-2 such as <b>Nextstrain</b>, GISAID, and PANGOLIN.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32912998">32912998</a> <b>(2)</b> The emergence of SARS-CoV-2 in Europe and North America<ul><li> For low-diversity data, a single phylogeny has a resolution that is largely not supported by the full posterior tree distribution containing several plausible phylogeographic scenarios that must be considered, all of which are compatible with the genetic data [e.g., the mutation trees in (<xref rid="R2" ref-type="bibr"><italic>2</italic></xref>) and those available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>].</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32913002">32913002</a> <b>(5)</b> Cryptic transmission of SARS-CoV-2 in Washington state<ul><li> An interactive version of this figure is available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/blab/ncov-cryptic-transmission/introductions">https://<b>nextstrain</b>.org/community/blab/ncov-cryptic-transmission/introductions</ext-link>.<li> An interactive version of this figure is available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/blab/ncov-cryptic-transmission/wa-clade">https://<b>nextstrain</b>.org/community/blab/ncov-cryptic-transmission/wa-clade</ext-link>.<li> From 29 February onward, genomic data were immediately posted to the GISAID EpiCoV sequence database (<xref rid="R9" ref-type="bibr"><italic>9</italic></xref>, <xref rid="R10" ref-type="bibr"><italic>10</italic></xref>) and analyzed alongside other public SARS-CoV-2 genomes by means of the <b>Nextstrain</b> online platform (<xref rid="R25" ref-type="bibr"><italic>25</italic></xref>) to provide immediate and public situational awareness.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33181975">33181975</a> <b>(2)</b> COVID-19 effective reproduction number dropped during Spain's nationwide dropdown, then spiked at lower-incidence regions<ul><li> But also to their international connectivity (<xref rid="bb0020" ref-type="bibr"><u>Coelho et al., 2020</u></xref>), the specific strains reaching each territory (see updated data at <b>nextstrain</b>; <xref rid="bb0065" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and the age and spatial structure of their human populations (<xref rid="bb0060" ref-type="bibr"><u>Gatto et al., 2020</u></xref>).<li> as evidenced by the presence of fourteen different genetic clusters identified by <b>nextstrain</b> (<xref rid="bb0065" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; data from 27 April 2020).<li> [Coelho et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Coelho</surname>              <given-names>M.T.P.</given-names>            </name>            <name>              <surname>Rodrigues</surname>              <given-names>J.F.M.</given-names>            </name>            <name>              <surname>Medina</surname>              <given-names>A.M.</given-names>            </name>            <name>              <surname>Scalco</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Terribile</surname>              <given-names>L.C.</given-names>            </name>            <name>              <surname>Vilela</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Diniz-Filho</surname>              <given-names>J.A.F.</given-names>            </name>            <name>              <surname>Dobrovolski</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Global expansion of COVID-19 pandemic is driven by population size and airport connections</article-title>          <source>PeerJ</source>          <volume>8</volume>          <year>2020</year>          <object-id pub-id-type="publisher-id">e9708</object-id>          <pub-id pub-id-type="doi">10.7717/peerj.9708</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Gatto et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32327608">32327608</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32779445">32779445</a> <b>(2)</b> Profiling of Initial Available SARS-CoV-2 Sequences fromIranian Related COVID-19 Patients<ul><li> Also, the entropy values of thesemutations among the 3927 full genome sequences ofthe SARS-CoV-2 have retrieved from <b>Nextstrain</b> (<xref rid="B14" ref-type="bibr"><u>14</u></xref>)(https://<b>nextstrain</b>.org/) analyses.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32900935">32900935</a> <b>(3)</b> SARS-CoV-2 detection using isothermal amplification and a rapid, inexpensive protocol for sample inactivation and purification<ul><li> Despite this lack of conservation among coronaviruses, sequencing data from clinical SARS-CoV-2 isolates from around the world, accessed through <b>Nextstrain</b> (<xref rid="r9" id="pcxrefw1ab1b2b3b1b1b7" ref-type="bibr"><u>9</u></xref>), do not show that this target region is mutating more quickly than the rest of the genome (<xref id="pcxrefw1ab1b2b3b1b1b9" ref-type="fig" rid="fig01">Fig.<li> (<italic>B</italic> and <italic>C</italic>) Measure of sequenced nucleotide mutations in a subset of global data provided by <b>Nextstrain</b> (nucleotide entropy), adapted from <b>Nextstrain</b> visual interface.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32887735">32887735</a> <b>(3)</b> An Early Pandemic Analysis of SARS-CoV-2 Population Structure and Dynamics in Arizona<ul><li> In order to assist with both local and global efforts to track the spread and evolution of this virus, we began intensive sequencing of viral genomes from across Arizona and deposition of these sequences in the GISAID database, which makes them accessible to the research community for downstream analyses, notably including real-time pathogen tracking through <b>Nextstrain</b> (<xref rid="B12" ref-type="bibr"><u>12</u></xref>).<li> Based on ancestral state reconstructions, this lineage is predicted to have first emerged in Asia or Europe (<b>Nextstrain</b>), and this lineage was observed in multiple European countries, including the large Italian outbreak, before it was first documented in New York (<b>Nextstrain</b>).<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33013809">33013809</a> <b>(1)</b> Comprehensive Genome Analysis of 6,000 USA SARS-CoV-2 Isolates Reveals Haplotype Signatures and Localized Transmission Patterns by State and by Country<ul><li> The major external resources of SARS-CoV-2 strains, genome sequences, and variants were GISAID, GenBank, CNCB, and <b>NextStrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32577641">32577641</a> <b>(7)</b> SARS-CoV-2 gene content and COVID-19 mutation impact by comparing 44 Sarbecovirus genomes<ul><li>Single nucleotide variants were downloaded from the &#x201C;<b>Nextstrain</b> Vars&#x201D; track in the UCSC Table Browser on 2020-04-18 at 11:46 AM EDT.<li> UCSC data came from <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link><sup><xref rid="R55" ref-type="bibr"><u>55</u></xref></sup>, which was derived from genome sequences deposited in GISAID<sup><xref rid="R35" ref-type="bibr"><u>35</u></xref></sup>.<li> Right of use and publication of the underlying sequences is entirely controlled by the authors of the original resource and the contributors of individual sequences, who are acknowledged in the <b>Nextstrain</b> metadata file included with <xref rid="SD1" ref-type="supplementary-material"><u>supplementary materials</u></xref>.<li> Please contact the respective authors (available via the <b>Nextstrain</b> metadata.tsv file) if you intend to carry out further research using their data.<li> Derived data, such as phylogenies, can be downloaded from <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link> (see &#x201C;<xref rid="SD2" ref-type="supplementary-material"><u>DOWNLOAD DATA</u></xref>&#x201D; link at bottom of page) - please contact the relevant authors where appropriate.&#x201D;.<li> [55](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32876050">32876050</a> <b>(8)</b> Integrating genotypes and phenotypes improves long-term forecasts of seasonal influenza A/H3N2 evolution<ul><li>To enable real-time forecasts, we integrated our forecasting framework into our existing open source pathogen surveillance application, <b>Nextstrain</b> (<xref rid="bib13" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> Prior to finalizing our model coefficients for use in <b>Nextstrain</b>, we tested whether our three best composite models could be improved by learning new coefficients per timepoint from the test data.<li>Snapshot of live forecasts on <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>nextstrain</b>.org</ext-link> from our best model (HI antigenic novelty and mutational load) for April 1, 2021.<li>Snapshot of the last two years of seasonal influenza H3N2 evolution on <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>nextstrain</b>.org</ext-link> showing the estimated distance per strain to the future population.<li> We have added this framework as a component of seasonal influenza analyses on <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>nextstrain</b>.org</ext-link> where it provides real-time forecasts for influenza researchers, decision makers, and the public.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35382097">35382097</a> <b>(6)</b> Implications of Knowledge Organization Systems for Health Information Exchange and Communication during the COVID-19 Pandemic<ul><li><b>Nextstrain</b>.<li><b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="interrefs380">https://<b>nextstrain</b>.org/</ext-link>) is an open-source initiative that uses pathogen genome data in its provision of real-time views of the spread and evolution of viruses.<li> <b>Nextstrain</b> is unique in that it tracks and is able to recreate mutations as well as convey the geographical spread of the virus and the underlying genomic data that is supporting that.<li> <b>Nextstrain</b> is being used by researchers to trace the virus back to its origin point and date in time and space and to perform analysis of its genomic variations in order to facilitate treatment, containment, and development of vaccines.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35382103">35382103</a> <b>(1)</b> How American Academic Medical/Health Sciences Libraries Responded to the COVID-19 Health Crisis: An Observational Study<ul><li>Besides publication data and COVID-19 cases, about 10% of the library guides collected a series of genomic and clinical trial datasets by linking to GenBank, <b>nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32544053">32544053</a> <b>(1)</b> Seroepidemiologic Study Designs for Determining SARS-COV-2 Transmission and Immunity<ul><li> Phylogenetic analysis has greatly benefited from development of the GISAID database (<ext-link xlink:href="https://www.gisaid.org" ext-link-type="uri">https://www.gisaid.org</ext-link>) to enable rapid sharing of genomic data and from platforms like <b>Nextstrain</b> (<xref rid="R24" ref-type="bibr"><italic>24</italic></xref>) for rapid analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32904687">32904687</a> <b>(7)</b> Temporal Detection and Phylogenetic Assessment of SARS-CoV-2 in Municipal Wastewater<ul><li>                    <hr/>                  </td>                </tr>                <tr>                  <td>SDS software v1.4</td>                  <td>Applied Biosystems</td>                  <td>4379633</td>                </tr>                <tr>                  <td>RStudio v1.2.1335</td>                  <td>The R project</td>                  <td>RRID: SCR_000432, <ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0055">https://www.r-project.org/</ext-link></td>                </tr>                <tr>                  <td>ggplot2</td>                  <td>Tidyverse</td>                  <td>RRID: SCR_014601, <ext-link ext-link-type="uri" xlink:href="https://ggplot2.tidyverse.org/" id="intref0060">https://ggplot2.tidyverse.org/</ext-link></td>                </tr>                <tr>                  <td>stats</td>                  <td>R Core Team</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html" id="intref0065">https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html</ext-link>                  </td>                </tr>                <tr>                  <td>astsa</td>                  <td>CRAN</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/astsa/index.html" id="intref0070">https://cran.r-project.org/web/packages/astsa/index.html</ext-link>                  </td>                </tr>                <tr>                  <td>spatialEco</td>                  <td>CRAN</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/spatialEco/index.html" id="intref0075">https://cran.r-project.org/web/packages/spatialEco/index.html</ext-link>                  </td>                </tr>                <tr>                  <td>MinKNOW software</td>                  <td>Oxford Nanopore Technologies</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/sso/login?next_url+%2Fdownloads" id="intref0080">https://community.nanoporetech.com/sso/login?next_url+%2Fdownloads</ext-link>                  </td>                </tr>                <tr>                  <td>artic-ncov2019</td>                  <td>ARTIC network</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0085">https://artic.network/ncov-2019</ext-link>                  </td>                </tr>                <tr>                  <td>minimap2</td>                  <td>GitHub</td>                  <td>RRID: SCR_018550, <ext-link ext-link-type="uri" xlink:href="https://github.com/lh3/minimap2" id="intref0090">https://github.com/lh3/minimap2</ext-link></td>                </tr>                <tr>                  <td>MAFFT v7.429</td>                  <td>N/A</td>                  <td>RRID: SCR_011811, <ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/index.html" id="intref0095">https://mafft.cbrc.jp/alignment/software/index.html</ext-link></td>                </tr>                <tr>                  <td>trimAl v1.2rev59</td>                  <td>N/A</td>                  <td>RRID: SCR_017334, <ext-link ext-link-type="uri" xlink:href="http://trimal.cgenomics.org/use_of_the_command_line_trimal_v1.2" id="intref0100">http://trimal.cgenomics.org/use_of_the_command_line_trimal_v1.2</ext-link></td>                </tr>                <tr>                  <td>IQTree</td>                  <td><b>Nextstrain</b>.<li>                    <hr/>                  </td>                </tr>                <tr>                  <td>SDS software v1.4</td>                  <td>Applied Biosystems</td>                  <td>4379633</td>                </tr>                <tr>                  <td>RStudio v1.2.1335</td>                  <td>The R project</td>                  <td>RRID: SCR_000432, <ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0055">https://www.r-project.org/</ext-link></td>                </tr>                <tr>                  <td>ggplot2</td>                  <td>Tidyverse</td>                  <td>RRID: SCR_014601, <ext-link ext-link-type="uri" xlink:href="https://ggplot2.tidyverse.org/" id="intref0060">https://ggplot2.tidyverse.org/</ext-link></td>                </tr>                <tr>                  <td>stats</td>                  <td>R Core Team</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html" id="intref0065">https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html</ext-link>                  </td>                </tr>                <tr>                  <td>astsa</td>                  <td>CRAN</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/astsa/index.html" id="intref0070">https://cran.r-project.org/web/packages/astsa/index.html</ext-link>                  </td>                </tr>                <tr>                  <td>spatialEco</td>                  <td>CRAN</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/spatialEco/index.html" id="intref0075">https://cran.r-project.org/web/packages/spatialEco/index.html</ext-link>                  </td>                </tr>                <tr>                  <td>MinKNOW software</td>                  <td>Oxford Nanopore Technologies</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/sso/login?next_url+%2Fdownloads" id="intref0080">https://community.nanoporetech.com/sso/login?next_url+%2Fdownloads</ext-link>                  </td>                </tr>                <tr>                  <td>artic-ncov2019</td>                  <td>ARTIC network</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0085">https://artic.network/ncov-2019</ext-link>                  </td>                </tr>                <tr>                  <td>minimap2</td>                  <td>GitHub</td>                  <td>RRID: SCR_018550, <ext-link ext-link-type="uri" xlink:href="https://github.com/lh3/minimap2" id="intref0090">https://github.com/lh3/minimap2</ext-link></td>                </tr>                <tr>                  <td>MAFFT v7.429</td>                  <td>N/A</td>                  <td>RRID: SCR_011811, <ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/index.html" id="intref0095">https://mafft.cbrc.jp/alignment/software/index.html</ext-link></td>                </tr>                <tr>                  <td>trimAl v1.2rev59</td>                  <td>N/A</td>                  <td>RRID: SCR_017334, <ext-link ext-link-type="uri" xlink:href="http://trimal.cgenomics.org/use_of_the_command_line_trimal_v1.2" id="intref0100">http://trimal.cgenomics.org/use_of_the_command_line_trimal_v1.2</ext-link></td>                </tr>                <tr>                  <td>IQTree</td>                  <td><b>Nextstrain</b></td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0105">https://github.com/<b>nextstrain</b>/augur</ext-link>                  .<li>                    <hr/>                  </td>                </tr>                <tr>                  <td>SDS software v1.4</td>                  <td>Applied Biosystems</td>                  <td>4379633</td>                </tr>                <tr>                  <td>RStudio v1.2.1335</td>                  <td>The R project</td>                  <td>RRID: SCR_000432, <ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0055">https://www.r-project.org/</ext-link></td>                </tr>                <tr>                  <td>ggplot2</td>                  <td>Tidyverse</td>                  <td>RRID: SCR_014601, <ext-link ext-link-type="uri" xlink:href="https://ggplot2.tidyverse.org/" id="intref0060">https://ggplot2.tidyverse.org/</ext-link></td>                </tr>                <tr>                  <td>stats</td>                  <td>R Core Team</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html" id="intref0065">https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html</ext-link>                  </td>                </tr>                <tr>                  <td>astsa</td>                  <td>CRAN</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/astsa/index.html" id="intref0070">https://cran.r-project.org/web/packages/astsa/index.html</ext-link>                  </td>                </tr>                <tr>                  <td>spatialEco</td>                  <td>CRAN</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/spatialEco/index.html" id="intref0075">https://cran.r-project.org/web/packages/spatialEco/index.html</ext-link>                  </td>                </tr>                <tr>                  <td>MinKNOW software</td>                  <td>Oxford Nanopore Technologies</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/sso/login?next_url+%2Fdownloads" id="intref0080">https://community.nanoporetech.com/sso/login?next_url+%2Fdownloads</ext-link>                  </td>                </tr>                <tr>                  <td>artic-ncov2019</td>                  <td>ARTIC network</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0085">https://artic.network/ncov-2019</ext-link>                  </td>                </tr>                <tr>                  <td>minimap2</td>                  <td>GitHub</td>                  <td>RRID: SCR_018550, <ext-link ext-link-type="uri" xlink:href="https://github.com/lh3/minimap2" id="intref0090">https://github.com/lh3/minimap2</ext-link></td>                </tr>                <tr>                  <td>MAFFT v7.429</td>                  <td>N/A</td>                  <td>RRID: SCR_011811, <ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/index.html" id="intref0095">https://mafft.cbrc.jp/alignment/software/index.html</ext-link></td>                </tr>                <tr>                  <td>trimAl v1.2rev59</td>                  <td>N/A</td>                  <td>RRID: SCR_017334, <ext-link ext-link-type="uri" xlink:href="http://trimal.cgenomics.org/use_of_the_command_line_trimal_v1.2" id="intref0100">http://trimal.cgenomics.org/use_of_the_command_line_trimal_v1.2</ext-link></td>                </tr>                <tr>                  <td>IQTree</td>                  <td><b>Nextstrain</b></td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0105">https://github.com/<b>nextstrain</b>/augur</ext-link>                  </td>                </tr>                <tr>                  <td>Augur</td>                  <td><b>Nextstrain</b>.<li>                    <hr/>                  </td>                </tr>                <tr>                  <td>SDS software v1.4</td>                  <td>Applied Biosystems</td>                  <td>4379633</td>                </tr>                <tr>                  <td>RStudio v1.2.1335</td>                  <td>The R project</td>                  <td>RRID: SCR_000432, <ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0055">https://www.r-project.org/</ext-link></td>                </tr>                <tr>                  <td>ggplot2</td>                  <td>Tidyverse</td>                  <td>RRID: SCR_014601, <ext-link ext-link-type="uri" xlink:href="https://ggplot2.tidyverse.org/" id="intref0060">https://ggplot2.tidyverse.org/</ext-link></td>                </tr>                <tr>                  <td>stats</td>                  <td>R Core Team</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html" id="intref0065">https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html</ext-link>                  </td>                </tr>                <tr>                  <td>astsa</td>                  <td>CRAN</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/astsa/index.html" id="intref0070">https://cran.r-project.org/web/packages/astsa/index.html</ext-link>                  </td>                </tr>                <tr>                  <td>spatialEco</td>                  <td>CRAN</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/spatialEco/index.html" id="intref0075">https://cran.r-project.org/web/packages/spatialEco/index.html</ext-link>                  </td>                </tr>                <tr>                  <td>MinKNOW software</td>                  <td>Oxford Nanopore Technologies</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/sso/login?next_url+%2Fdownloads" id="intref0080">https://community.nanoporetech.com/sso/login?next_url+%2Fdownloads</ext-link>                  </td>                </tr>                <tr>                  <td>artic-ncov2019</td>                  <td>ARTIC network</td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0085">https://artic.network/ncov-2019</ext-link>                  </td>                </tr>                <tr>                  <td>minimap2</td>                  <td>GitHub</td>                  <td>RRID: SCR_018550, <ext-link ext-link-type="uri" xlink:href="https://github.com/lh3/minimap2" id="intref0090">https://github.com/lh3/minimap2</ext-link></td>                </tr>                <tr>                  <td>MAFFT v7.429</td>                  <td>N/A</td>                  <td>RRID: SCR_011811, <ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/index.html" id="intref0095">https://mafft.cbrc.jp/alignment/software/index.html</ext-link></td>                </tr>                <tr>                  <td>trimAl v1.2rev59</td>                  <td>N/A</td>                  <td>RRID: SCR_017334, <ext-link ext-link-type="uri" xlink:href="http://trimal.cgenomics.org/use_of_the_command_line_trimal_v1.2" id="intref0100">http://trimal.cgenomics.org/use_of_the_command_line_trimal_v1.2</ext-link></td>                </tr>                <tr>                  <td>IQTree</td>                  <td><b>Nextstrain</b></td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0105">https://github.com/<b>nextstrain</b>/augur</ext-link>                  </td>                </tr>                <tr>                  <td>Augur</td>                  <td><b>Nextstrain</b></td>                  <td>                    <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0110">https://github.com/<b>nextstrain</b>/augur</ext-link>                  .<li> A maximum-likelihood phylogenetic tree was constructed from this alignment using IQTree in the Augur utility of <b>Nextstrain</b> 45,46.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32867854">32867854</a> <b>(2)</b> Emerging of a SARS-CoV-2 viral strain with a deletion in nsp1<ul><li> This is quite surprising, since corona viruses typically experience a moderate rate of mutations, due to the presence of a protein with proofreader activity (ExonN, also called nsp14), calculated in about 26 mutations per year (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?l=clock">https://<b>nextstrain</b>.org/ncov/global?l=clock</ext-link><underline>)</underline>.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32853525">32853525</a> <b>(1)</b> Impact of Early Pandemic Stage Mutations on Molecular Dynamics ofSARS-CoV-2 M<sup>pro</sup><ul><li> The SARS-CoV-2 genome is RNA-based, and virusesfrom this category have been reported to have increased rates ofmutation.<sup><xref ref-type="bibr" rid="ref14"><u>14</u></xref></sup> For instance, in HIV, this has lead toseveral levels of classification of the virus, in which certain strains canmanifest different transmissibility patterns and show differing responses toexisting therapies.<sup><xref ref-type="bibr" rid="ref15"><u>15</u></xref>,<xref ref-type="bibr" rid="ref16"><u>16</u></xref></sup> Nevertheless, coronaviruses comprise anRNA-dependent RNA polymerase genetic proofreadingmachinery,<sup><xref ref-type="bibr" rid="ref17"><u>17</u></xref>,<xref ref-type="bibr" rid="ref18"><u>18</u></xref></sup> which may be responsible for theirreduced genetic diversity, even though the persistence of the pandemic mayallow for the accumulation of immunologically important mutations.<sup><xref ref-type="bibr" rid="ref3"><u>3</u></xref></sup> From the data gathered from the GISAID database<sup><xref ref-type="bibr" rid="ref12"><u>12</u></xref></sup> and real-time subsample estimates of genetic relatednessfrom the <b>Nextstrain</b> web resource,<sup><xref ref-type="bibr" rid="ref19"><u>19</u></xref></sup> it is clear that thevirus is evolving within the human host.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30102691">30102691</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19331584">19331584</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23317093">23317093</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25451065">25451065</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32321524">32321524</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32855259">32855259</a> <b>(1)</b> Four SARS-CoV-2 Genome Sequences from Late April in Stockholm, Sweden, Reveal a Rare Mutation in the Spike Protein<ul><li>We compared the four coding-complete SARS-CoV-2 genomes from Stockholm with genomes detected globally using the three main methods available to establish relationships between different genetic variants of the virus, GISAID, <b>Nextstrain</b>, and PANGOLIN (Phylogenetic Assignment of Named Global Outbreak LINeages) (<xref rid="B6" ref-type="bibr"><u>6</u></xref>, <xref rid="B7" ref-type="bibr"><u>7</u></xref>).<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<person-group person-group-type="author"><name name-style="western"><surname>Rambaut</surname><given-names>A</given-names></name>, <name name-style="western"><surname>Holmes</surname><given-names>EC</given-names></name>, <name name-style="western"><surname>Hill</surname><given-names>V</given-names></name>, <name name-style="western"><surname>O&#x2019;Toole</surname><given-names>&#xC1;</given-names></name>, <name name-style="western"><surname>McCrone</surname><given-names>JT</given-names></name>, <name name-style="western"><surname>Ruis</surname><given-names>C</given-names></name>, <name name-style="western"><surname>Du Plessis</surname><given-names>L</given-names></name>, <name name-style="western"><surname>Pybus</surname><given-names>OG</given-names></name></person-group><year>2020</year><article-title>A dynamic nomenclature proposal for SARS-CoV-2 to assist genomic epidemiology</article-title>. <source>bioRxiv</source> doi:<pub-id pub-id-type="doi">10.1101/2020.04.17.046086</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32853241">32853241</a> <b>(7)</b> Mutation of the second sialic acid-binding site of influenza A virus neuraminidase drives compensatory mutations in hemagglutinin<ul><li> To determine the order, in which these mutations in NA and HA appeared in H9N2 viruses, phylogenetic analysis, using <b>Nextstrain</b> analytic and visualization tools (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org</ext-link>)[<xref rid="ppat.1008816.ref032" ref-type="bibr"><u>32</u></xref>](<xref ref-type="fig" rid="ppat.1008816.g005"><u>Fig 5</u></xref>), was employed.<li>Phylogenetic trees of full-length NA and HA sequences of H9N2 viruses between 1970 and 2019 were generated using <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org</ext-link>) [<xref rid="ppat.1008816.ref032" ref-type="bibr"><u>32</u></xref>].<li> Residue numbers shown for HA, correspond to residue number +8 in <b>Nextstrain</b>.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32867108">32867108</a> <b>(4)</b> Study of a SARS-CoV-2 Outbreak in a Belgian Military Education and Training Center in Maradi, Niger<ul><li> After submission of the obtained SARS-CoV-2 genome sequences to GISAID (Accession IDs: EPI_ISL_487433 to EPI_ISL_487436) they were automatically included into <b>Nextstrain</b> [<xref rid="B12-viruses-12-00949" ref-type="bibr"><u>12</u></xref>], an open-source project providing a continually updated phylogenetic analysis and visualization of the evolutionary relationships of publicly available SARS-CoV-2 genomes.<li> <b>Nextstrain</b> phylogeny is rooted relative to early samples from Wuhan.<li> The genome sequences were submitted to GISAID under accession IDs EPI_ISL_487433-6, and were re-named C2181, C2182, C2183, and C2185 in <b>Nextstrain</b> (<xref rid="viruses-12-00949-t003" ref-type="table"><u>Table 3</u></xref>).<li> Phylogenetic examination using <b>Nextstrain</b> (accessed 11 August 2020) showed that all four Maradi genomes belonged to cluster 19B of GISAID&#x2019;s clade S (<xref ref-type="fig" rid="viruses-12-00949-f002"><u>Figure 2</u></xref>).<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32864588">32864588</a> <b>(2)</b> Clinical features, diagnostics, and outcomes of patients presenting with acute respiratory illness: A retrospective cohort study of patients with and without COVID-19<ul><li> For phylogenetic analysis of SARS-CoV-2 viruses, we constructed genomes using minimap2 [<xref rid="bib0033" ref-type="bibr"><u>33</u></xref>] to align reads to the reference MN908947.3 and iVar [<xref rid="bib0034" ref-type="bibr"><u>34</u></xref>] to trim primers and call variants, then restricted to samples with at least 10-fold coverage of at least 97% (2929 kgbases) of the genome (<italic>n</italic>&#xA0;=&#xA0;10), and utilized the <b>Nextstrain</b> [<xref rid="bib0035" ref-type="bibr"><u>35</u></xref>] pipeline to build a phylogenetic tree using iqtree.<li> Most samples fell into the <b>Nextstrain</b>.org clades A2a (widely prevalent in New York) and B1 (detected in Washington State in February 2020).<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29750242">29750242</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30621750">30621750</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33145011">33145011</a> <b>(3)</b> Open science approaches to COVID-19<ul><li><b>Nextstrain</b> pathogen evolution.<li><b>Nextstrain</b> is an open source, interactive data visualisation platform that provides &#x201C;real-time tracking of pathogen evolution&#x201D;<italic>via</italic> the analysis of sequencing data<sup><xref rid="ref-41" ref-type="bibr"><u>41</u></xref></sup>.<li> In response to the current pandemic, <b>Nextstrain</b> is maintaining a SARS-CoV-2 phylogenetic tree based on the analysis of contributed sequencing data.<li> [41](<article-title>Real-Time Tracking of Pathogen Evolution</article-title>.<year>2020</year>; Accessed 24 June, 2020.<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">Reference Source</ext-link>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32854265">32854265</a> <b>(1)</b> Big Data Analytics in the Fight against Major Public Health Incidents (Including COVID-19): A Conceptual Framework<ul><li> In order to solve this problem, scientists and medical institutions in various countries are currently building various development source cloud computing platforms, such as <b>Nextstrain</b>.org [<xref rid="B39-ijerph-17-06161" ref-type="bibr"><u>39</u></xref>], which allow researchers to share and track genome sequences in real time, providing a reliable and real-time source for analyzing diseases.<li> [39](          <person-group person-group-type="author">            <collab>Nextstrain</collab>          </person-group>          <article-title>Genomic Epidemiology of Novel Coronavirus-Global Subsampling</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global?c=region">https://nextstrain.org/ncov/global?c=region</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2020-08-16">(accessed on 16 August 2020)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32840837">32840837</a> <b>(2)</b> Advancing Global Health Equity in the COVID-19 Response: Beyond Solidarity<ul><li> <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov"><b>NextStrain</b></ext-link>, for example, is an open repository that pulls in data from labs around the world that are sequencing SARS-CoV-2&#x2019;s genome and centralizes it in a phylogenetic tree (Berditchevskaia and Peach <xref ref-type="bibr" rid="CR1"><u>2020</u></xref>).<li> [2020](Berditchevskaia, A.P., and K. Peach. 2020. Coronavirus: Seven ways collective intelligence is tackling the pandemic. <italic>The Conversation,</italic> March 13<italic>.</italic>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32802963">32802963</a> <b>(1)</b> Protocol: Leveraging a demographic and health surveillance system for Covid-19 Surveillance in rural KwaZulu-Natal<ul><li> WGS will be assembled using Genome Detective<sup><xref rid="ref-40" ref-type="bibr"><u>40</u></xref></sup> and analysed using phylogenetic software applications, such as BEAST<sup><xref rid="ref-41" ref-type="bibr"><u>41</u></xref></sup> and <b>Nextstrain</b><sup><xref rid="ref-42" ref-type="bibr"><u>42</u></xref></sup>.<li> [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32108862">32108862</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29942656">29942656</a>) [42](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32817965">32817965</a> <b>(4)</b> Genomic Diversity of SARS-CoV-2 During Early Introduction into the United States National Capital Region<ul><li> Efforts to describe this diversity during the early phase of the pandemic have resulted in multiple clade designation systems including those used by the Global Information Sharing of All Influenza Data (GISAID), the <b>NextStrain</b> platform, and COVID-19 Genomics UK consortium (COG-UK)<sup><xref rid="R14" ref-type="bibr"><u>14</u></xref>&#x2013;<xref rid="R16" ref-type="bibr"><u>16</u></xref></sup>.<li> For example, <b>NextStrain</b> clade designations are broadly separated by viruses that originated in China in 2019 (Clade 19) and those that were later introduced into Europe early 2020 (Clade 20)<sup><xref rid="R15" ref-type="bibr"><u>15</u></xref></sup>.<li>We used a customized <b>Nextstrain</b> snakemake pipeline<sup><xref rid="R12" ref-type="bibr"><u>12</u></xref></sup> using augur v9.0.0 on the final dataset, which included the 886 Global 1K dataset plus all removed regional groups (including all 114 JHHS sequences and all published Virginia sequences), resulting in 1279 sequences used in our phylogenetic analysis.<li> Finally, clades were assigned to sequences using <b>Nextstrain</b><sup><xref rid="R15" ref-type="bibr"><u>15</u></xref></sup> and Pangolin<sup><xref rid="R14" ref-type="bibr"><u>14</u></xref></sup>, see <xref rid="SD8" ref-type="supplementary-material"><u>Table S3</u></xref>.<li> [14](<name><surname>Rambaut</surname><given-names>A</given-names></name>, <name><surname>Holmes</surname><given-names>EC</given-names></name>, <name><surname>Hill</surname><given-names>V</given-names></name>, <etal/><article-title>A dynamic nomenclature proposal for SARS-CoV-2 to assist genomic epidemiology</article-title>. <source>bioRxiv</source>. <year>2020</year>; : <comment>2020.04.17.046086</comment>.) [16](<name><surname>Alm</surname><given-names>E</given-names></name>, <name><surname>Broberg</surname><given-names>EK</given-names></name>, <name><surname>Connor</surname><given-names>T</given-names></name>, <etal/><article-title>Geographical and temporal distribution of SARS-CoV-2 clades in the WHO European Region, January to June 2020</article-title>. <source>Eurosurveillance</source><year>2020</year>; <volume>25</volume>: <fpage>2001410</fpage>.) [15](<name><surname>Hodcroft</surname><given-names>Emma B</given-names></name>, <name><surname>Hadfield</surname><given-names>James</given-names></name>, <name><surname>Neher</surname><given-names>Richard A</given-names></name>, <name><surname>Bedford</surname><given-names>Trevor</given-names></name>. <source>Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org.</source><year>2020</year>; <comment>published online June 6.</comment><comment><ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link></comment> (<comment>accessed</comment><date-in-citation>Aug 2, 2020</date-in-citation>).) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<name><surname>Hodcroft</surname><given-names>Emma B</given-names></name>, <name><surname>Hadfield</surname><given-names>James</given-names></name>, <name><surname>Neher</surname><given-names>Richard A</given-names></name>, <name><surname>Bedford</surname><given-names>Trevor</given-names></name>. <source>Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstain.org.</source><year>2020</year>; <comment>published online June 6.</comment><comment><ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link></comment> (<comment>accessed</comment><date-in-citation>Aug 2, 2020</date-in-citation>).) [14](<name><surname>Rambaut</surname><given-names>A</given-names></name>, <name><surname>Holmes</surname><given-names>EC</given-names></name>, <name><surname>Hill</surname><given-names>V</given-names></name>, <etal/><article-title>A dynamic nomenclature proposal for SARS-CoV-2 to assist genomic epidemiology</article-title>. <source>bioRxiv</source>. <year>2020</year>; : <comment>2020.04.17.046086</comment>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32822393">32822393</a> <b>(4)</b> Real-time public health communication of local SARS-CoV-2 genomic epidemiology<ul><li> Weekly situation reports of SARS-CoV-2 genomic epidemiology on global and continental levels have been published by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>) since January 23, 2020 [<xref rid="pbio.3000869.ref012" ref-type="bibr"><u>12</u></xref>], presenting the status of the pandemic across large geographical regions.<li> We present easily digestible descriptions of our analysis and major conclusions in both English and Spanish, alongside interactive figures facilitated by the <b>Nextstrain</b> visualization toolkit (<xref ref-type="fig" rid="pbio.3000869.g001"><u>Fig 1C and 1D</u></xref>).<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a> <b>(6)</b> Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus<ul><li> <xref rid="bib52" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>) and <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0015">https://<b>nextstrain</b>.org</ext-link>; <xref rid="bib26" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> Phylogenetic tracking of SARS-CoV-2 variants at <b>Nextstrain</b> reveals complex webs of evolutionary and geographical relationships (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="intref0035">https://<b>nextstrain</b>.org</ext-link>; <xref rid="bib26" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>); travelers globally dispersed G614 variants and likely would have introduced and reintroduced G614 variants into different locations.<li> [Shu and McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32645326">32645326</a> <b>(3)</b> Structures of Human Antibodies Bound to SARS-CoV-2 Spike Reveal Common Epitopes and Recurrent Features of Antibodies<ul><li> RRID:<ext-link ext-link-type="uri" xlink:href="rridsoftware:SCR_002320" id="intref0300">SCR_002320</ext-link></td>                </tr>                <tr>                  <td><b>Nextstrain</b>/augur.<li> RRID:<ext-link ext-link-type="uri" xlink:href="rridsoftware:SCR_002320" id="intref0300">SCR_002320</ext-link></td>                </tr>                <tr>                  <td><b>Nextstrain</b>/augur</td>                  <td>                    <xref rid="bib22" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>                  </td>                  <td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="intref0305">https://github.com/<b>nextstrain</b>/augur</ext-link>; RRID: <ext-link ext-link-type="uri" xlink:href="rridsoftware:SCR_018223" id="intref0310">SCR_018223</ext-link>.<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33343922">33343922</a> <b>(1)</b> Efficient inference, potential, and limitations of site-specific substitution models<ul><li> 2015</xref>), and ancestral sequences were inferred using TreeTime (<xref rid="veaa066-B29" ref-type="bibr"><u>Sagulenko, Puller, and Neher 2018</u></xref>) via the <b>nextstrain</b>&#x2019;s augur pipeline (<xref rid="veaa066-B9" ref-type="bibr">Hadfield et&#xA0;al.<li> [Sagulenko, Puller, and Neher 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511587">32511587</a> <b>(2)</b> Sequencing identifies multiple early introductions of SARS-CoV-2 to the New York City Region<ul><li> <xref rid="R2" ref-type="bibr"><u>ARTIC Network</u></xref>), open sharing of sequences through the GISAID (Global Initiative on Sharing All Influenza Data) repository (<xref rid="R25" ref-type="bibr"><u>Shu and McCauley 2017</u></xref>), and rapid analysis and visualization of viral phylogenies using <b>Nextstrain</b> (<xref rid="R11" ref-type="bibr">Hadfield et al.<li> Sequences from non-human hosts, annotated by <b>Nextstrain</b> as duplicate individuals or highly divergent, with &lt;27,000 non-ambiguous nucleotides, or with improperly formatted dates or location were excluded.<li> [ARTIC Network](<collab>ARTIC Network</collab>, ed. <source>ARTIC SARS-CoV-2 Protocol</source>. <comment><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019">https://artic.network/ncov-2019</ext-link></comment> (<comment>Accessed</comment><date-in-citation>April 15, 2020</date-in-citation>).) [Shu and McCauley 2017](<name><surname>Shu</surname><given-names>Y</given-names></name>, <name><surname>McCauley</surname><given-names>J</given-names></name>. <year>2017</year><article-title>GISAID: Global initiative on sharing all influenza data - from vision to reality</article-title>. <source>Euro Surveill</source><volume>22</volume>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33063054">33063054</a> <b>(1)</b> A Study on Fight Against COVID-19 from Latest Technological Intervention<ul><li> Furthermore, ML Models are also used for tracking the COVID-19 cases across the globe in Data Dashboards like <bold>UpCode, <b>NextStrain</b> and Healthmap</bold> [<xref ref-type="bibr" rid="CR2"><u>2</u></xref>], effective in drawing useful insight like the number of recoveries, number of new cases and number of deaths, which part of the region controlled the outbreak, etc.<li> [2](The best, and the worst, of the coronavirus dashboards|MIT Technology Review.) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33235813">33235813</a> <b>(2)</b> Temporal signal and the phylodynamic threshold of SARS-CoV-2<ul><li> To minimise the impact of potential sequencing errors in our alignments, we deleted obvious errors upon visual inspection and compared our phylogenetic trees to those obtained by other groups (virological.org) and those from <b>Nextstrain</b> (<xref rid="veaa061-B16" ref-type="bibr">Hadfield et&#xA0;al.<li> As a case in point, the <b>Nextstrain</b> workflow uses a fixed evolutionary rate to infer the time scale of SARS-CoV-2, avoiding potential errors in the resulting estimates of evolutionary time scales in downstream analyses (<xref rid="veaa061-B8" ref-type="bibr">Dellicour et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33110519">33110519</a> <b>(3)</b> Natural outbreaks and bioterrorism: How to deal with the two sides of the same coin?<ul><li> The use of the <b>Nextstrain</b> tool [<xref rid="R73" ref-type="bibr"><u>73</u></xref>] in the context of SARS-CoV-2 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>) perfectly illustrates the potential of such approaches in the context of spreading epidemics [<xref rid="R74" ref-type="bibr"><u>74</u></xref>].<li> [73](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [74](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31991541">31991541</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32808094">32808094</a> <b>(7)</b> Immuno-epidemiology and pathophysiology of coronavirus disease 2019 (COVID-19)<ul><li> To track the day-to-day evolution of the virus, scientists are encouraged to submit viral sequences to publicly available databases like GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/CoV2020/">https://www.gisaid.org/CoV2020/</ext-link>) where virus divergence from the original strain from Wuhan is analyzed in real time using the <b>nextstrain</b> platform (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) [<xref ref-type="bibr" rid="CR39"><u>39</u></xref>].<li> Our <b>nextstrain</b> analysis carried out on 2 May 2020 revealed that the greatest diversity of publicly reported SARS-CoV-2 clades was reported in the Asian continent.<li> Radial phylogenetic trees were generated using <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>) after the SARS-CoV-2 global dataset was filtered according to Africa, Asia, Europe, and the USA regions.<li> [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32838349">32838349</a> <b>(1)</b> A genomics network established to respond rapidly to public health threats in South Africa<ul><li> In addition, our bioinformatics team has curated a functional website that provides access to software applications, protocols, data, analytical updates using <b>Nextstrain</b>,<xref rid="bib8" ref-type="bibr"><sup>8</sup></xref> and other tools.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32816810">32816810</a> <b>(1)</b> Rift valley fever: diagnostic challenges and investment needs for vaccine development<ul><li> Recent advances in sequencing and comparative genomics for use in outbreak monitoring (eg, <b>nextstrain</b>.org) can further improve our understanding of transmission pathways.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a> <b>(15)</b> Geographical and temporal distribution of SARS-CoV-2 clades in the WHO European Region, January to June 2020<ul><li>Several nomenclatures have been introduced for SARS-CoV-2 (strains in GISAID use the name hCoV-19), including by <b>Nextstrain</b> [<xref rid="r6" ref-type="bibr"><u>6</u></xref>], GISAID [<xref rid="r7" ref-type="bibr"><u>7</u></xref>] and Rambaut et al.<li> While <b>Nextstrain</b> and GISAID clade nomenclatures aim at providing a broad-brush categorisation of globally circulating diversity, the lineages (cov-lineages.org) are meant to correspond to outbreaks.<li>Schematic comparison of the GISAID, <b>Nextstrain</b> and cov-lineages.org nomenclatures for SARS-CoV-2 sequences of world-wide origin, February&#x2013;July 2020.<li> The tree is shaded to correspond to the major lineages and all <b>Nextstrain</b> and GISAID clades.<li> Between <b>Nextstrain</b> clades 20A and 20C and GISAID clades G and GH, there are differences in the position where clades are divided (shown as grey in the main tree and lineage columns, and grey stripes in the <b>Nextstrain</b> and GISAID column).<li> Here, the grey is shown as stripes in the <b>Nextstrain</b> and GISAID clade columns to reflect how they are categorised by each methodology.<li> There was an initial period in January 2020 when the 19A/L/V/O clades (<b>Nextstrain</b>/GISAID nomenclatures) were more prevalent in total than the 20A/G clades, However, this could partially be an effect of small numbers of sequenced viral genomes in Europe early in the pandemic as well as the widespread sampling strategy focusing on cases with travel history to East Asia.<li> The <b>Nextstrain</b> 20C clade peaked at ca 20% of the sequences in early April 2020 and has since then slowly declined and almost disappeared, while the GISAID GH clade peaked at ca 30% in May 2020 and has rapidly declined since then.<li>From top to bottom: <b>Nextstrain</b>, GISAID and cov-lineages.org.<li>The geographical distribution of <b>Nextstrain</b> and GISAID clades shows that in general, clade 19B/S has been very rare, except in Spain and Kazakhstan, and frequencies of the other clades were varied for most countries in the WHO European Region (<xref ref-type="fig" rid="f3"><u>Figure 3</u></xref>).<li>Distribution of GISAID and <b>Nextstrain</b> clades of SARS-CoV-2 across countries in the WHO European Region, based on all high-quality genomes in GISAID, February&#x2013; July 2020 (n&#x202F;=&#x202F;37,187).<li> The grey areas represent sequences called differently by the GISAID and <b>Nextstrain</b> methodologies, and correspond to the grey-shaded partition of the tree in <xref ref-type="fig" rid="f1"><u>Figure 1</u></xref>.<li> Clade membership for each sequence was determined by given GISAID annotation or via the <b>Nextstrain</b> clades script, and proportions were calculated for all sequences from a given country.<li>Overall, the GISAID and <b>Nextstrain</b> nomenclatures provide similar pictures of the situation and may provide useful systems for genomic situation reporting globally.<li> [6](Nextstrain: Genomic epidemiology of novel coronavirus - Global subsampling. [Accessed: 10 Jul 2020]. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov">https://nextstrain.org/ncov</ext-link>) [7](Global Initiative on Sharing All Influenza Data (GISAID). Clade and lineage nomenclature aids in genomic epidemiology studies of active hCoV-19 viruses. Munich: GISAID; 4 Jul 2020. Available from: <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33029383">33029383</a> <b>(1)</b> Intra-genome variability in the dinucleotide composition of SARS-CoV-2<ul><li> Individual ORFs were identified using a combination of ORF finder (<ext-link xlink:href="https://www.ncbi.nlm.nih.gov/orffinder/" ext-link-type="uri">https://www.ncbi.nlm.nih.gov/orffinder/</ext-link>), visual inspection of nucleotide alignments in SSE v1.4, comparison with previous literature and information available from <b>nextstrain</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32814065">32814065</a> <b>(1)</b> The Role of Host Genetic Factors in Coronavirus Susceptibility: Review of Animal and Systematic Review of Human Literature<ul><li> Specifically, questions about <italic>ACE2</italic> have been described in the context of SARS-CoV-1 and the impact of species-specific variants in this and other genes.<xref rid="bib22" ref-type="bibr"><sup>22</sup></xref> This work has emphasized interactions of viral and host genetics.<xref rid="bib23" ref-type="bibr"><sup>23</sup></xref> This aspect bears further scrutiny in COVID-19 studies, especially given recent data regarding SARS-CoV-2 genetic changes detected in different areas of the world (e.g., see data from <b>Nextstrain</b> under <xref rid="app4" ref-type="sec"><u>Web Resources</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32834891">32834891</a> <b>(8)</b> Genome evolution of SARS-CoV-2 and its virological characteristics<ul><li> Based on the viral sequences as well as geographical and sample collection information in the GISAID database, <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>) [<xref ref-type="bibr" rid="CR27"><u>27</u></xref>] shares phylogenetic, geographical, and genomic analyses of SARS-CoV-2, illustrating the real-time evolution of SARS-CoV-2.<li> Note that <b>Nextstrain</b> has been used to analyze the phylogeny of not only SARS-CoV-2 but also other pathogenic viruses that can potentially pose a public health threat.<li> At the time of writing this article (May 28, 2020), 30,699 SARS-CoV-2 and closely related viral sequences are stored in the GISAID database, and 4308 SARS-CoV-2 genomes were analyzed in the <b>Nextstrain</b>.<li> According to the <b>Nextstrain</b>, the number of substitutions in the SARS-CoV-2 genome was estimated at approximately 26 substitutions per year.<li> However, the L-type emerged in the beginning of the COVID-19 outbreak, and the current major type of SARS-CoV-2 widely spreading all over the world is L-type as of May 21, 2020 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>).<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32848795">32848795</a> <b>(0)</b> COVID-19 Therapeutic Options Under Investigation<ul><li> Importantly, genetic analysis of SARS-CoV-2 from different countries reveals that the virus has diversified into several genetic clades (<uri xlink:type="simple" xlink:href="http://www.<b>nextstrain</b>.org/ncov/global">www.<b>nextstrain</b>.org/ncov/global</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32763945">32763945</a> <b>(5)</b> Genome Sequences of Six SARS-CoV-2 Strains Isolated in Morocco, Obtained Using Oxford Nanopore MinION Technology<ul><li> 1.5.5 under the generalized time-reversable (GTR) model (<xref rid="B8" ref-type="bibr"><u>8</u></xref>), implemented via the pipeline provided by Augur (<ext-link ext-link-type="uri" xlink:href="http://github.com/<b>nextstrain</b>/augur">github.com/<b>nextstrain</b>/augur</ext-link>).<li> Major clades were defined by amino acid and/or nucleotide substitutions and were matched to the <b>Nextstrain</b> nomenclature (<xref rid="B9" ref-type="bibr"><u>9</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>).<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32764372">32764372</a> <b>(6)</b> Comprehensive Analyses of SARS-CoV-2 Transmission in a Public Health Virology Laboratory<ul><li> Major SARS-CoV-2 worldwide clades have been proposed by nomenclature systems including <b>Nextstrain</b> [<xref rid="B8-viruses-12-00854" ref-type="bibr"><u>8</u></xref>] and the global initiative on sharing all influenza data (GISAID, <uri xlink:href="https://www.gisaid.org">https://www.gisaid.org</uri>) [<xref rid="B9-viruses-12-00854" ref-type="bibr"><u>9</u></xref>].<li> For example, using <b>Nextstrain</b>&#x2019;s nomenclature, there are currently five major clades: 19A (the root clade) and 19B, and clades 20A, B and C that are widespread in Europe and include a mutation in the spike protein, D614G, that is associated with increased infectivity and higher viral loads [<xref rid="B10-viruses-12-00854" ref-type="bibr"><u>10</u></xref>].<li>Clade annotations were attained from <b>Nextstrain</b> [<xref rid="B8-viruses-12-00854" ref-type="bibr"><u>8</u></xref>], who identify variants that define clades of interest from sequences submitted to GISAID by labs worldwide and updates them periodically in clades.tsv (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv</uri>).<li> All samples sequenced were associated with clade 20, harboring the clade&#x2019;s mutations A23403G and C14408T (<b>Nextstrain</b> nomenclature [<xref rid="B8-viruses-12-00854" ref-type="bibr"><u>8</u></xref>]).<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32649872">32649872</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32759818">32759818</a> <b>(2)</b> Use of Whole Genome Sequencing Data for a First in Silico Specificity Evaluation of the RT-qPCR Assays Used for SARS-CoV-2 Detection<ul><li> These observations match with the genomic evolution of the virus determined since its first appearance in December 2019 (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/global?d=entropy&amp;p=full">https://<b>nextstrain</b>.org/ncov/global?d=entropy&amp;p=full</uri>, accessed June 24, 2020 [<xref rid="B43-ijms-21-05585" ref-type="bibr"><u>43</u></xref>]).<li> [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32995557">32995557</a> <b>(3)</b> Integrated genomic view of SARS-CoV-2 in India<ul><li><bold>Cluster 1:</bold> The sequences (n=26) in this cluster belonged to the G-clade [variant 23403 (Spike protein D614G) as per GISAID nomeclature (Global Inititative on Sharing All Influenza Data)] Based on <b>Nextstrain</b> classification these 26 sequence belonged to A2a clade [(denoted by positions C241T; C3037T, A23403G (S: D614G); C14408T (ORF1b/RdRp: P314L)] (<xref ref-type="fig" rid="f1"><u>Figure 1A</u></xref>).<li><bold>Cluster 2:</bold> In our study large numbers of strains (n=65) belong to this unclassified cluster (as per GISAID and <b>Nextstrain</b>).<li><bold>Cluster 3:</bold> This subclass of strains (n=7) harbouring a common variant G11083T (NSP6), G1397A (NSP2) and T28668C (N-capsid) are described for the A3 clade (<b>Nextstrain</b>) in additions to G29742T (<xref ref-type="fig" rid="f1"><u>Figure 1A</u></xref>).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32398273">32398273</a> <b>(3)</b> RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses: a first look<ul><li>SARS-CoV-2-MSA-2 was generated from 739 GISAID (<xref rid="RNA076141RANC11" ref-type="bibr"><u>Elbe and Buckland-Merrett 2017</u></xref>) sequences, including all sequences described in the <b>Nextstrain</b> project metadata file as of 03-18-20 (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</uri>).<li> [Elbe and Buckland-Merrett 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32732237">32732237</a> <b>(9)</b> Genome Sequences of SARS-CoV-2 Strains Detected in Hong Kong<ul><li> Maximum-likelihood time-resolved phylogenetic analysis was conducted using TreeTime (<xref rid="B4" ref-type="bibr"><u>4</u></xref>) and <b>Nextstrain</b> (<xref rid="B5" ref-type="bibr"><u>5</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) on 12 June 2020 (default GTR substitution model and nucleotide substitution rate of 0.0008 per site per year).<li>(A) Maximum-likelihood time-resolved phylogenetic tree of 2,949 SARS-CoV-2 genome sequences sampled up to 16 April 2020 in the GISAID database on 12 June 2020 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?dmax=2020-04-16">https://<b>nextstrain</b>.org/ncov/global?dmax=2020-04-16</ext-link>).<li> (B) Close-up view of the phylogeny (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/asia?dmax=2020-04-16">https://<b>nextstrain</b>.org/ncov/asia?dmax=2020-04-16</ext-link>) of case 1 (HKSH0003, purple asterisk), case 2 (HKSH0004, purple asterisk), and 29 closely related sequences (gray for non-Asian samples and other colors for Asian samples) (late February 2020 to mid-April 2020).<li> (C) Close-up view of the phylogeny (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/asia?dmax=2020-04-16">https://<b>nextstrain</b>.org/ncov/asia?dmax=2020-04-16</ext-link>) of case 3 (HKSH0007, purple asterisk) and 13 closely related sequences (gray for non-Asian samples and other colors for Asian samples) (late February 2020 to mid-April 2020).<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32732235">32732235</a> <b>(4)</b> SARS-CoV-2 Genome Sequence from Morocco, Obtained Using Ion AmpliSeq Technology<ul><li>The phylogenetic analysis of this genome sequence was performed using the <b>Nextstrain</b> tool enabled by data from the Global Initiative on Sharing All Influenza Data (GISAID) (<xref ref-type="fig" rid="fig1"><u>Fig.&#xA0;1</u></xref>).<li> The figure was rendered using <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32738255">32738255</a> <b>(0)</b> An enzyme-based immunodetection assay to quantify SARS-CoV-2 infection<ul><li> <xref rid="bib21" ref-type="bibr"><u>Walls et al., 2020</u></xref>) (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org" id="intref0015">www.<b>nextstrain</b>.org</ext-link> (<xref rid="bib5" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>)) and has been verified to bind to SARS-CoV-2 S protein (<xref rid="bib28" ref-type="bibr"><u>Zheng et al., 2020</u></xref>).<li> The French isolate clusters with the reference Wuhan-Hu-1/2019 isolate whereas the Netherlands/01 strain can be grouped to clade A2a (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org" id="intref0020">www.<b>nextstrain</b>.org</ext-link> (<xref rid="bib5" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>)).<li> <xref rid="bib21" ref-type="bibr"><u>Walls et al., 2020</u></xref>) (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org" id="intref0025">www.<b>nextstrain</b>.org</ext-link> (<xref rid="bib5" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>)) which allows detection of the S protein of the France/IDF0372/2020 as well as the Netherlands/01 isolate and two isolates from Ulm, Southern Germany (<xref rid="appsec1" ref-type="sec">Fig.<li> [Walls et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32155444">32155444</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Zheng et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Zheng</surname>              <given-names>Z.</given-names>            </name>            <name>              <surname>Monteil</surname>              <given-names>V.M.</given-names>            </name>            <name>              <surname>Maurer-Stroh</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Yew</surname>              <given-names>C.W.</given-names>            </name>            <name>              <surname>Leong</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Mohd-Ismail</surname>              <given-names>N.K.</given-names>            </name>            <name>              <surname>Arularasu</surname>              <given-names>S.C.</given-names>            </name>            <name>              <surname>Chow</surname>              <given-names>V.T.K.</given-names>            </name>            <name>              <surname>Lin</surname>              <given-names>R.T.P.</given-names>            </name>            <name>              <surname>Mirazimi</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Hong</surname>              <given-names>W.</given-names>            </name>            <name>              <surname>Tan</surname>              <given-names>Y.-J.</given-names>            </name>          </person-group>          <article-title>Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</article-title>          <comment>bioRxiv 2020.03.06.980037</comment>          <pub-id pub-id-type="doi">10.1101/2020.03.06.980037</pub-id>          <year>2020</year>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Walls et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32155444">32155444</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32723824">32723824</a> <b>(3)</b> Haplotype networks of SARS-CoV-2 infections in the <italic>Diamond Princess</italic> cruise ship outbreak<ul><li> The estimated evolutionary rate of SARS-CoV-2, described on the <b>Nextstrain</b> website (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov?l=unrooted" xlink:show="new">https://<b>nextstrain</b>.org/ncov?l=unrooted</ext-link>, last updated on May 15, 2020), of 24.17 substitutions per genome per year on average suggests that predominant isolates in the DP-A cluster could be a very early generation of progeny from Wuhan-Hu-1 isolated on December 26, 2019.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32766579">32766579</a> <b>(2)</b> Oligonucleotide capture sequencing of the SARS-CoV-2 genome and subgenomic fragments from COVID-19 individuals<ul><li> Introduced stop codons are rare among the submitted genome assemblies tracking the evolution of SARS-CoV2 (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>), but are distributed all along the genome (<xref rid="SD5" ref-type="supplementary-material">Supplementary Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32718950">32718950</a> <b>(2)</b> The COVID-19 pandemic: diverse contexts; different epidemics&#x2014;how and why?<ul><li>Ten different circulating clades (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>nextstrain</b>.org</ext-link>; <ext-link ext-link-type="uri" xlink:href="www.gisaid.org">www.gisaid.org</ext-link>),<xref rid="R136" ref-type="bibr"><u>136</u></xref> as of May 2020.<li> [136](<person-group person-group-type="author"><collab>Nextstrain.com</collab></person-group><article-title>Genomic epidemiology of novel coronavirus</article-title>, <year>2020</year> Available: <uri xlink:href="https://nextstrain.org/ncov?r=location">https://nextstrain.org/ncov?r=location</uri> [Accessed <comment>30 May 2020</comment>].) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32721383">32721383</a> <b>(10)</b> Computationally Optimized SARS-CoV-2 MHC Class I and II Vaccine Formulations Predicted to Target Human Haplotype Distributions<ul><li> We used <b>Nextstrain</b> (<xref rid="bib21" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) to identify ORFs and translate the sequence.<li> <ext-link ext-link-type="uri" xlink:href="https://github.com/gifford-lab/PUFFIN" id="intref0075">https://github.com/gifford-lab/PUFFIN</ext-link></td><td>GitHub commit a63f6c563b7e2f7b04eac28<break/>a6cf09d8078ac3a2a with pre-trained model</td></tr><tr><td>Hapferret</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/nilsboar/HapFerret" id="intref0080">https://github.com/nilsboar/HapFerret</ext-link></td><td>GitHub commit e2381dc567cec97373acb<break/>49c09f167e46ea0bb53</td></tr><tr><td><b>Nextstrain</b>.<li> <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0085">https://github.com/<b>nextstrain</b>/ncov</ext-link>.<li>For SARS-CoV-2, we obtained the most up to date version of the GISAID database (<xref rid="bib13" ref-type="bibr"><u>Elbe and Buckland-Merrett, 2017</u></xref>) (as of 2:02pm EST May 13, 2020, see <xref rid="mmc4" ref-type="supplementary-material"><u>Table S4</u></xref>: GISAID acknowledgements) and used <b>Nextstrain</b> (<xref rid="bib21" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) to remove genomes with sequencing errors, translate the genome into proteins, and perform multiple sequence alignments (MSAs).<li> We retrieved 24,468 sequences from GISAID, and 19,288 remained after <b>Nextstrain</b> quality processing.<li> After quality processing, <b>Nextstrain</b> randomly sampled 34 genomes from every geographic region and month to produce a representative set of 5,142 genomes for evolutionary analysis.<li> <b>Nextstrain</b> definition of a &#x201C;region&#x201D; can vary from a city (e.g., &#x201C;Shanghai&#x201D;) to a larger geographical district.<li> Spatial and temporal sampling in <b>Nextstrain</b> is designed to provide a representative sampling of sequences around the world.<li>The 5,142 genomes sampled by <b>Nextstrain</b> were then translated into protein sequences and aligned.<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Elbe and Buckland-Merrett, 2017](          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G.</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Global Chall.</source>          <volume>1</volume>          <year>2017</year>          <fpage>33</fpage>          <lpage>46</lpage>        ) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32721529">32721529</a> <b>(2)</b> Isolation and phylogenetic analysis of SARS-CoV-2 variants collected in Russia during the COVID-19 outbreak<ul><li> A total of 8934 sequences were used for further analysis with the <b>Nextstrain</b> framework (<xref rid="bib0035" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>, <xref rid="bib0075" ref-type="bibr"><u>Sagulenko et al., 2018</u></xref>) to estimate the phylogeny and molecular epidemiology of the virus variants in Russia.<li> The tree and map were built using the <b>Nextstrain</b> framework.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Sagulenko et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32725080">32725080</a> <b>(2)</b> Past, present, and future of COVID-19: a review<ul><li> Sofar, viruses from A2a, A1a, and B lineages were described in Brazil, with morethan 90% of sequences belonging to the A2a lineage (<b>NEXTSTRAIN</b>,&lt;<b>nextstrain</b>.org&gt;).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33072402">33072402</a> <b>(1)</b> Towards a unified classification for human respiratory syncytial virus genotypes<ul><li> Currently, novel <italic toggle="yes">real-time</italic> surveillance tools, such as <b>Nextstrain</b>, limit themselves to subtype HRSV-A due to a shortage of whole-genome strains for the subtype HRSV-B (<xref rid="veaa052-B35" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32722343">32722343</a> <b>(14)</b> Molecular Tracing of SARS-CoV-2 in Italy in the First Three Months of the Epidemic<ul><li>The newly-characterized 59 genomes plus three previously characterized isolates by us (EPI_ISL_417445-417447) [<xref rid="B1-viruses-12-00798" ref-type="bibr"><u>1</u></xref>] were aligned with a total of 77 Italian sequences available in public databases (GISAID, <uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>) on 13 May 2020 and 452 genomes sampled in different European and Asian countries (513 and 16, respectively) representing all the different viral clades described in the <b>Nextstrain</b> platform (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>).<li>A subset of sequences assigned to the old <b>Nextstrain</b> A2 clade (classified as B lineage for Pangolin), was generated for dating the epidemic, including all the Italian sequences, one German (EPI_ISL_406862) and three Chinese isolates from Shanghai, ancestral to the A2 clade (EPI_ISL_416327, EPI_ISL_416334 and EPI_ISL_416386).<li> For this reason, we used the old <b>Nextstrain</b> and the new Pangolin (freely available at <uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>) systems for strain classification.<li> The new <b>Nextstrain</b> classification was performed by using the available script (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/docs/running.md">https://github.com/<b>nextstrain</b>/ncov/blob/master/docs/running.md</uri>).<li>Phylogenetic analysis by maximum likelihood showed that the Italian sequences were included in a single SARS-CoV-2 clade (the old <b>Nextstrain</b> A2 clade, corresponding to new Nextstain clades 20A and 20B) with the exception of three sequences: Two from Chinese patients visiting Italy at the end of January 2020 after being infected in Wuhan and one characterized by us from an Italian subject, living in Padua, sampled in March 2020, not reporting any recent trip outside Italy or contacts with subjects affected by COVID-19 (pp = 0.99) (<xref ref-type="fig" rid="viruses-12-00798-f001"><u>Figure 1</u></xref>, clade 19A).<li> The new lineage assignment of 62 Italian isolates is reported on <xref rid="viruses-12-00798-t001" ref-type="table"><u>Table 1</u></xref> with the correspondence to other naming systems (old and new <b>Nextstrain</b>).<li> The present analysis shows that all but one of 62 SARS-CoV-2 sequences obtained from 22 February to the end of April in different Northern and Central Italian areas belong to a single clade, corresponding to the Pangolin lineage B.1, the old <b>Nextstrain</b> subclade A2a and the new <b>Nextstrain</b> clades 20A and 20B (<uri xlink:href="https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming">https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming</uri>) [<xref rid="B1-viruses-12-00798" ref-type="bibr"><u>1</u></xref>,<xref rid="B12-viruses-12-00798" ref-type="bibr"><u>12</u></xref>].<li> [1](          <person-group person-group-type="author">            <name>              <surname>Zehender</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Lai</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Bergna</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Meroni</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Riva</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Balotta</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Tarkowski</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Gabrieli</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Bernacchia</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Rusconi</surname>              <given-names>S.</given-names>            </name>            <etal/>          </person-group>          <article-title>Genomic characterization and phylogenetic analysis of SARS-COV-2 in Italy</article-title>          <source>J. Med. Virol.</source>          <year>2020</year>          <volume>29</volume>          <fpage>25794</fpage>          <pub-id pub-id-type="doi">10.1002/jmv.25794</pub-id>        ) [1](          <person-group person-group-type="author">            <name>              <surname>Zehender</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>Lai</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Bergna</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Meroni</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Riva</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Balotta</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Tarkowski</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Gabrieli</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Bernacchia</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Rusconi</surname>              <given-names>S.</given-names>            </name>            <etal/>          </person-group>          <article-title>Genomic characterization and phylogenetic analysis of SARS-COV-2 in Italy</article-title>          <source>J. Med. Virol.</source>          <year>2020</year>          <volume>29</volume>          <fpage>25794</fpage>          <pub-id pub-id-type="doi">10.1002/jmv.25794</pub-id>        ) [12](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Holmes</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Ruis</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Du Plessis</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Pybus</surname>              <given-names>O.G.</given-names>            </name>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 to assist genomic epidemiology</article-title>          <source>bioRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32705298">32705298</a> <b>(1)</b> Die Virologie von SARS-CoV-2<ul><li>Viele der Sequenzen werden Online-Datenbanken wie etwa <b>Nextstrain</b> zur Verf&#xFC;gung gestellt und k&#xF6;nnen von der wissenschaftlichen Gemeinschaft ausgewertet werden.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32743569">32743569</a> <b>(2)</b> The D614G mutation in the SARS-CoV2 Spike protein increases infectivity in an ACE2 receptor dependent manner.<ul><li> After the appearance of the Spike D614G variant in the latter course of the Chinese epidemic, over time in most examined local epidemics an enrichment of the 614G Spike protein variant over the original 614D variant has been observed, leading to the hypothesis that the Spike D614G mutation is positively selected (<xref rid="R5" ref-type="bibr"><u>5</u></xref>)(<xref rid="SD1" ref-type="supplementary-material"><u>Supplemental movie</u></xref>, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614">https://<b>nextstrain</b>.org/ncov/global?c=gt-S_614</ext-link>)(<xref rid="R6" ref-type="bibr"><u>6</u></xref>).<li> [5](<name><surname>Korber</surname><given-names>B.</given-names></name>, <etal/>, <article-title>Spike mutation pipeline reveals the emergence of a more transmissible form of SARS-CoV-2</article-title>. <source>bioRxiv 4, 2020.04.29.069054</source> (<year>2020</year>).) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32711111">32711111</a> <b>(1)</b> Drugs targeting various stages of the SARS-CoV-2 life cycle: Exploring promising drugs for the treatment of Covid-19<ul><li> However, outside Wuhan, 61.3% was found to be L type and 38.4% S type.[<xref rid="bb0110" ref-type="bibr"><u>22</u></xref>] Further, <b>Nextstrain</b> analysis on 2310 viral isolates disclosed that West Coast (United States) predominates with B1 clade, whereas A2a clade, was found to be predominant in East Coast including New York, Europe, and Italy.<li> [22](          <person-group person-group-type="author">            <name>              <surname>Tang</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Wu</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Li</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Song</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>Yao</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Wu</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Duan</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>Zhang</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Wang</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>Qian</surname>              <given-names>Z.</given-names>            </name>          </person-group>          <article-title>On the origin and continuing evolution of SARS-CoV-2</article-title>          <source>Nat. Sci. Rev.</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1093/nsr/nwaa036</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32793182">32793182</a> <b>(1)</b> Geographic and Genomic Distribution of SARS-CoV-2 Mutations<ul><li> One of such tools is <b>NextStrain</b> (Hadfield et al., <xref rid="B13" ref-type="bibr"><u>2018</u></xref>), which also allows for scalable phylogenetic analyses and real time tracking of specific mutations.<li> [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32693748">32693748</a> <b>(2)</b> Reconciling early-outbreak estimates of the basic reproductive number and its uncertainty: framework and applications to the novel coronavirus (SARS-CoV-2) outbreak<ul><li> and one report was posted on <uri xlink:href="<b>nextstrain</b>.org"><b>nextstrain</b>.org</uri> [<xref rid="RSIF20200144C5" ref-type="bibr"><u>5</u></xref>] (<xref rid="RSIF20200144TB1" ref-type="table"><u>table 1</u></xref>).<li> [5](<person-group person-group-type="author"><name name-style="western"><surname>Bedford</surname><given-names>T</given-names></name>, <name name-style="western"><surname>Neher</surname><given-names>R</given-names></name>, <name name-style="western"><surname>Hadfield</surname><given-names>J</given-names></name>, <name name-style="western"><surname>Hodcroft</surname><given-names>E</given-names></name>, <name name-style="western"><surname>Ilcisin</surname><given-names>M</given-names></name>, <name name-style="western"><surname>M&#xFC;ller</surname><given-names>N</given-names></name></person-group><year>2020</year><comment>Genomic analysis of nCoV spread. Situation report 2020-01-23. <uri xlink:href="https://nextstrain.org/narratives/ncov/sit-rep/2020-01-23">https://nextstrain.org/narratives/ncov/sit-rep/2020-01-23</uri> (accessed 24 January 2020)</comment>.) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32690878">32690878</a> <b>(2)</b> Genomics and epidemiological surveillance<ul><li> To overcome the challenge in data analysis and interpretation, user-friendly web-based applications were designed for linage assignment (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">Pangolin COVID-19 Lineage Assigner</ext-link>) and to interactively visualize the circulating lineages on national and international scales (for example, <ext-link ext-link-type="uri" xlink:href="https://microreact.org/showcase">Microreact</ext-link> and <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>Nextstrain</b></ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32678072">32678072</a> <b>(1)</b> Detection and Genetic Characterization of Community-Based SARS-CoV-2 Infections &#x2014; New York City, March 2020<ul><li> Phylogenetic relations were inferred using the <b>Nextstrain</b> pipeline (<xref rid="R4" ref-type="bibr"><italic>4</italic></xref>), including the 36 positive SARS-CoV-2 sentinel specimens and selected full genome sequences available as of April 1, 2020, from the Global Initiative on Sharing All Influenza Data (GISAID) (<xref rid="R5" ref-type="bibr"><italic>5</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32637944">32637944</a> <b>(2)</b> Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant<ul><li>The frequency of the SARS-CoV-2 D614G S protein variant in published genomic data was examined using the full <b>Nextstrain</b>-curated set of sequences available from GISAID as of 25 June 2020 (<xref rid="R17" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="R42" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>).<li> The identity of the variant at amino acid position 614 was tabulated for the full dataset and reported as frequency by date using the collection dates reported in the <b>Nextstrain</b>-curated metadata file available from GISAID (<xref rid="R17" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="R42" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Shu and McCauley, 2017](<name><surname>Shu</surname><given-names>Y.</given-names></name>, and <name><surname>McCauley</surname><given-names>J.</given-names></name> (<year>2017</year>). <article-title>GISAID: Global initiative on sharing all influenza data &#x2013; from vision to reality</article-title>. <source>Eurosurveillance</source><volume>22</volume>, <fpage>ii=30494</fpage>.) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Shu and McCauley, 2017](<name><surname>Shu</surname><given-names>Y.</given-names></name>, and <name><surname>McCauley</surname><given-names>J.</given-names></name> (<year>2017</year>). <article-title>GISAID: Global initiative on sharing all influenza data &#x2013; from vision to reality</article-title>. <source>Eurosurveillance</source><volume>22</volume>, <fpage>ii=30494</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32667035">32667035</a> <b>(1)</b> Highlights in Medical Entomology, 2019: Familiar Foes and New Frontiers<ul><li> Through viral sequences isolated from mosquito pools or bird samples collected from West Nile virus surveillance, the spread of these West Nile virus genotypes can be visualized using the online interface <b>Nextstrain</b> (<xref rid="CIT0015" ref-type="bibr">Hadfield et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32676620">32676620</a> <b>(6)</b> Distinct patterns of SARS-CoV-2 transmission in two nearby communities in Wisconsin, USA<ul><li>To further analyze genetic differences among viruses in the two locations, we assigned clades using the <b>Nextstrain</b> nomenclature.<li>All 247 available full length sequences from Dane and Milwaukee County through 26 April 2020 were used for phylogenetic analysis using the tools implemented in <b>Nextstrain</b> custom builds (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) <sup><xref rid="R4" ref-type="bibr"><u>4</u></xref>,<xref rid="R52" ref-type="bibr"><u>52</u></xref></sup>.<li> Time-resolved and divergence phylogenetic trees were built using the standard <b>Nextstrain</b> tools and scripts <sup><xref rid="R4" ref-type="bibr"><u>4</u></xref>,<xref rid="R52" ref-type="bibr"><u>52</u></xref></sup>.<li>An additional subsampled global phylogeny using all available sequences in GISAID as of 21 June 2020 were input into the <b>Nextstrain</b> pipeline.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [52](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [52](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32646487">32646487</a> <b>(1)</b> Evolving geographic diversity in SARS-CoV2 and in silico analysis of replicating enzyme 3CL<sup>pro</sup> targeting repurposed drug candidates<ul><li> SARS-CoV-2 patient genome sequences deposited in GISAID database from more than 60 different countries (<bold>a</bold>) radial and (<bold>b</bold>) unrooted phylogeny created by <b>Nextstrain</b> program [<xref ref-type="bibr" rid="CR66"><u>66</u></xref>] (<bold>c</bold>) Pie chart representation of number of SARS-CoV-2 patient genomes deposited in GISAID till 15th April 2020 from six different regions; Asia (orange, 35.37%), Oceania (purple, 20.86%), North America (brown, 21.13%), Europe (blue, 20.31%), Africa (red, 10.35%) and South America (black, 16.63%).<li> [66](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32646911">32646911</a> <b>(6)</b> Coding-Complete Genome Sequences of Two SARS-CoV-2 Isolates from Early Manifestations of COVID-19 in Israel<ul><li> The Italian strain harbors a documented replacement of D614&#x2192;G in the spike protein, a replacement which is typical of European strains (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>), compared with the strain imported from Japan.<li>The radial presentation of a phylogenetic tree generated by <b>Nextstrain</b> (<xref rid="B5" ref-type="bibr"><u>5</u></xref>) is presented in <xref ref-type="fig" rid="fig1"><u>Fig.&#xA0;1</u></xref> for all the Asian strains available to date in the <b>Nextstrain</b> data set.<li> The tree was generated by the <b>Nextstrain</b> site using the bioinformatics toolkit Augur, which implements FastTree, which infers approximately maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences (<xref rid="B5" ref-type="bibr"><u>5</u></xref>, <xref rid="B7" ref-type="bibr"><u>7</u></xref>).<li> Background, strains of the Asia data set available in <b>Nextstrain</b>; dots, all strains collected in Israel; circled in red, the two strains documented in the current study, ISR_JP0320 and ISR_IT0320.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20224823">20224823</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32650631">32650631</a> <b>(1)</b> Rapid Large-Scale COVID-19 Testing during Shortages<ul><li> Phylogenetic analysis of the samples was performed as detailed previously, following the standard protocol for analysis of SARS-CoV-2 genomes as provided by <b>Nextstrain</b> [<xref rid="B19-diagnostics-10-00464" ref-type="bibr"><u>19</u></xref>].<li> [19](          <article-title>Nextstrain &#x2013; Real-Time Tracking of Pathogen Evolution</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2020-06-22">(accessed on 22 June 2020)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511611">32511611</a> <b>(1)</b> Temporal detection and phylogenetic assessment of SARS-CoV-2 in municipal wastewater<ul><li>A maximum-likelihood phylogenetic tree was constructed from this alignment using IQTree in the Augur utility of <b>Nextstrain</b> (<xref rid="R10" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="R19" ref-type="bibr"><u>Minh et al., 2020</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Minh et al., 2020](<name><surname>Minh</surname><given-names>B.Q.</given-names></name>, <name><surname>Schmidt</surname><given-names>H.A.</given-names></name>, <name><surname>Chernomor</surname><given-names>O.</given-names></name>, <name><surname>Schrempf</surname><given-names>D.</given-names></name>, <name><surname>Woodhams</surname><given-names>M.D.</given-names></name>, <name><surname>von Haeseler</surname><given-names>A.</given-names></name>, and <name><surname>Lanfear</surname><given-names>R.</given-names></name> (<year>2020</year>). <article-title>IQ-TREE 2: New models and efficient methods for phylogenetic inference in the genomic era</article-title>. <source>Mol Biol Evol</source>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32281388">32281388</a> <b>(9)</b> New Genomes from the Congo Basin Expand History of CRF01_AE Origin and Dissemination<ul><li>The final dataset was submitted to a <b>Nextstrain</b> build.<li>The final dataset was submitted to a <b>Nextstrain</b> build.<sup><xref rid="B33" ref-type="bibr"><u>33</u></xref></sup> <b>Nextstrain</b> consists of a suite of tools that take raw sequences (in fasta format) and associated metadata (e.g., time, country, publications, authors, and/or journals) as input.<li> In short, <b>Nextstrain</b> performs a sequence alignment of the input data in multiple sequence alignment based on Fast Fourier transform (MAFFT),<sup><xref rid="B34" ref-type="bibr"><u>34</u></xref></sup> which in turn is used to infer a maximum likelihood.<li> The four CRF01_AE strains isolated in this study were aligned to 371 CRF01_AE sequences from LANL, after a filtering step to achieve a more equitable temporal and geographic distribution as implemented in <b>Nextstrain</b> (final alignment; <italic>n</italic>&#x2009;=&#x2009;358).<li> Our phylogenetic and evolutionary analysis of near full-length CRF01_AE sequences in <b>Nextstrain</b> indicates that HIV-1 CRF01_AE arose in Africa in the early 1970s (1968&#x2013;1970) (<xref ref-type="fig" rid="f2">Figs.<li><b>Nextstrain</b> maximum-likelihood phylogenetic analysis of CRF01_AE whole-genome sequences.<li> The evolutionary history of the CRF01_AE strains as they exited Africa and were introduced to Asia, Europe, and the Americas is illustrated in an interactive phylogenetic tree, map, and timeline at <uri xlink:href="https://<b>nextstrain</b>.org/community/EduanWilkinson/hiv-1_crf01">https://<b>nextstrain</b>.org/community/EduanWilkinson/hiv-1_crf01</uri>.<li><b>Nextstrain</b> geographical view of CRF01_AE temporal dynamics.<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32643599">32643599</a> <b>(2)</b> Introductions and early spread of SARS-CoV-2 in France, 24 January to 23 March 2020<ul><li> While a more thorough sampling will be needed to confirm this hypothesis, the phylogenetic analysis of sequences recovered in the current study suggests that the French outbreak was mainly seeded by one or several variants of this clade, unlike what is observed for many other European countries (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/europe?f_region=Europe">https://<b>nextstrain</b>.org/ncov/europe?f_region=Europe</ext-link>) [<xref rid="r7" ref-type="bibr"><u>7</u></xref>,<xref rid="r8" ref-type="bibr"><u>8</u></xref>].<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32289214">32289214</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32222993">32222993</a>) </ul></p>
</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34467103">34467103</a> <b>(2)</b> Perspectives on Molecular Diagnostic Testing for the COVID-19 Pandemic in Delaware<ul><li> for example, the database <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>, an open-source project that enables the comparisons of COVID-19 for genomic surveillance and outbreak response.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32896836">32896836</a> <b>(1)</b> Genetic characterization of SARS-CoV-2 &amp; implications for epidemiology, diagnostics &amp; vaccines in India<ul><li> Differing nomenclatures based on diverse approaches were proposed by the different platforms such as GISAID and <b>NextStrain</b>, while a dynamic nomenclature was proposed by Rambaut <italic>et al</italic><xref rid="ref12" ref-type="bibr"><u>12</u></xref>.<li> [12](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Holmes</surname>              <given-names>EC</given-names>            </name>            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>JT</given-names>            </name>            <name>              <surname>Ruis</surname>              <given-names>C</given-names>            </name>            <etal/>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>          <source>Nat Microbiol</source>          <year>2020</year>          <comment>doi: 101038/s41564-020-0770-5</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32859867">32859867</a> <b>(3)</b> Identification of SARS-CoV-2 clusters from symptomatic cases in India<ul><li> Despite segregating into different groups, the sequences studied clustered in genotype A2a, as observed in the <b>Nextstrain</b> site (<italic><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link></italic>), analyzed for the other Indian SARS-CoV-2 sequences.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32615316">32615316</a> <b>(14)</b> Characterizations of SARS-CoV-2 mutational profile, spike protein stability and viral transmission<ul><li>To study the impact of non-synonymous mutations in the transmission and viability of the newly emerging SARS-CoV-2 sequences across the world, we looked into <b>Nextstrain</b> tool, a powerful visualization tool comprising &gt;10,000 SARS-CoV-2 samples to study the evolution of various pathogens (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="ir0090">https://<b>nextstrain</b>.org/</ext-link>) (<xref rid="bb0065" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> Please note that the genotype information on <b>Nextstrain</b> are not consistent on different dates, at least for some residues.<li> Major amino acid substitutions were mapped and visualized in the phylogeny from <b>Nextstrain</b> Next hCoV-19 App.<li> Minor (less frequently) mutated residues that showed variations in <b>Nextstrain</b> data compared to NCBI virus database were excluded from the analysis.<li> Amino acid residues of ORF1ab were split as ORF1a from 1 to 4400 and the rest as ORF1b in <b>Nextstrain</b> Next hCoV-19 App.<li> To further validate these findings, we visualized these major substitutions by observing the phylogeny of SARS-CoV-2 in <b>Nextstrain</b>, which contains a curated database of more than 10,000 SARS-CoV-2 sequences, depicted in the form of phylogenetic trees (<xref rid="f0010" ref-type="fig">Fig.<li> As observed previously, substitutions at 5828th and 5865th position co-occurred even in this large sample set of <b>Nextstrain</b> data.<li> Overall, the mutation profile that we identified with 664 samples showed excellent concordance with the <b>Nextstrain</b> data comprising ~10,000 samples.<li> The analysis of the larger <b>Nextstrain</b> dataset indicated that D614G mutation is significantly more prevalent, indeed.<li>Looking into the spatial and temporal distribution of these variants of S protein in <b>Nextstrain</b> and noting down the number of occurrences of each variant along with the country it originated with the corresponding date (<xref rid="t0025" ref-type="table"><u>Table 5</u></xref>), we find that the variant with D614G substitution is characterized by greater viability across different countries as seen over a span of time, first originating on 24th December 2019 and prevailing since last recorded date.<li> Those strains that corresponded to structures with low stabilities were consequently found to have low transmission capabilities as verified in the <b>Nextstrain</b> data.<li> However, our hypothesis and interpretation of the mutations show good concordance as evidenced by the <b>Nextstrain</b> data.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32606379">32606379</a> <b>(1)</b> The network effect: studying COVID-19 pathology with the Human Cell Atlas<ul><li> In this extended network, the HCA leverages its relationships with other communities, such as the <ext-link ext-link-type="uri" xlink:href="https://lifetime-fetflagship.eu/">LifeTime Initiative</ext-link>, the <ext-link ext-link-type="uri" xlink:href="https://chanzuckerberg.com/science/programs-resources/humancellatlas/seednetworks/">Seed Networks</ext-link> and COVID-19-specific projects funded by various philanthropic organizations as well as our colleagues in the <ext-link ext-link-type="uri" xlink:href="https://www.covid19hg.org/">human genetics</ext-link> and <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">viral genomics</ext-link> communities.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32612646">32612646</a> <b>(3)</b> COVID-19 pandemic in Panama: lessons of the unique risks and research opportunities for Latin America<ul><li> For example, are several strains of SARS-CoV-2 co-circulating in the isthmus, or is there a single strain that amplifies? Publicly available data from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>) shows that the population expansion of SARS-CoV-2 is ongoing (e.g., branching events as the virus acquired genetic mutations when entering new geographic regions), and that novel mutations arising from epidemic countries in four continents could be introduced into Panama, with no scientific knowledge on the potential consequences for viral evolution (e.g., genetic recombination among strains coming from different geographic regions co-infecting a host), including transmission and pathogenicity.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32603804">32603804</a> <b>(3)</b> Digital microbiology<ul><li> Other platforms such as microreact ([<xref rid="bib94" ref-type="bibr"><u>94</u></xref>]) or <b>nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0065">https://<b>nextstrain</b>.org/</ext-link>; [<xref rid="bib95" ref-type="bibr"><u>95</u></xref>]) visualize sequencing data either on a project basis or as semi-automated surveillance tools, which access public sources such as GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="intref0070">https://www.gisaid.org/</ext-link>) in the case of influenza or SARS-CoV-2.<li> [94](          <person-group person-group-type="author">            <name>              <surname>Argimon</surname>              <given-names>S.</given-names>            </name>          </person-group>          <article-title>Microreact: visualizing and sharing data for genomic epidemiology and phylogeography</article-title>          <source>Microb Genom</source>          <volume>2</volume>          <year>2016</year>          <object-id pub-id-type="publisher-id">e000093</object-id>          <pub-id pub-id-type="doi">10.1099/mgen.0.000093</pub-id>        ) [95](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32604724">32604724</a> <b>(8)</b> <italic>Betacoronavirus</italic> Genomes: How Genomic Information has been Used to Deal with Past Outbreaks and the COVID-19 Pandemic<ul><li> In addition to public genome repositories, open-source platforms for real-time data visualization and analysis of genomic data are also available, including <b>NextStrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</uri>) and CoV-GLUE (<uri xlink:href="http://cov-glue.cvr.gla.ac.uk">http://cov-glue.cvr.gla.ac.uk</uri>).<li> <b>NextStrain</b> is fed with sequences from the GISAID repository and uses the Augur bioinformatics toolkit (<uri xlink:href="https://github.com/<b>nextstrain</b>/augur">https://github.com/<b>nextstrain</b>/augur</uri>) for tracking molecular evolution and the Auspice software (<uri xlink:href="https://<b>nextstrain</b>.github.io/auspice/">https://<b>nextstrain</b>.github.io/auspice/</uri>) for interactive visualization of phylogenomic data.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32586872">32586872</a> <b>(2)</b> Genome Sequence of SARS-CoV-2 Isolate Cali-01, from Colombia, Obtained Using Oxford Nanopore MinION Sequencing<ul><li> The alignment was performed using MAFFT (<xref rid="B9" ref-type="bibr"><u>9</u></xref>), implemented via the rapid phylodynamic alignment pipeline provided by Augur (<ext-link ext-link-type="uri" xlink:href="http://github.com/<b>nextstrain</b>/augur">github.com/<b>nextstrain</b>/augur</ext-link>) (<xref rid="B10" ref-type="bibr"><u>10</u></xref>).<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32311094">32311094</a> <b>(6)</b> Distinct viral clades of SARS&#x2010;CoV&#x2010;2: Implications for modeling of viral spread<ul><li>Dominant viral clades in United States clade B1 (orange) and clade A2a (blue) <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/north-america?c=clade_membership&amp;r=location" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/north-america?c=clade_membership&amp;r=location</ext-link>.<li>Distribution of codon 614 in the SARS&#x2010;CoV&#x2010;2 coronavirus aspartic acid (aqua), glycine (orange) <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/north-america?c=gt-S_614&amp;r=location" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/north-america?c=gt-S_614&amp;r=location</ext-link>.<li>Inferred phylogeny of codon 614 spike protein mutation in SARS&#x2010;CoV&#x2010;2 <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/north-america?branchLabel=aa&amp;c=gt-S_614" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/north-america?branchLabel=aa&amp;c=gt-S_614</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32595351">32595351</a> <b>(2)</b> Phylogenetic analysis of SARS-CoV-2 genomes in Turkey<ul><li> We employed the front-end package (Auspice) that <b>Nextstrain</b> uses (Hadfield, et al.<li> <b>Nextstrain</b> handles this problem by grouping the datasets based on the continents.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32595352">32595352</a> <b>(6)</b> An updated analysis of variations in SARS-CoV-2 genome<ul><li>Alignment results obtained using the software developed in this study were verified with the <b>Nextstrain</b> platform (Hadfield, 2018; <b>Nextstrain</b>, 2020<xref ref-type="fn" rid="fn5"><u>5</u></xref><fn id="fn5"><label>5</label><p><b>NextStrain</b> (2020).<li> <b>Nextstrain</b>: analysis and visualization of pathogen sequence data [online] Website: https://<b>nextstrain</b>.org/ [accessed 17 May 2020].<li> It must be noted that availability of these variations is also previously reported in other studies (<b>Nextstrain</b>, 2020; Pachetti et al., 2020; Wang et al., 2020; Phan et al., 2020) with different aspects.<li> [5](fn5) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32561801">32561801</a> <b>(1)</b> COVID-19 pandemic reveals the peril of ignoring metadata standards<ul><li> Opportunities for coordination of reported data types are critical for data interoperability as contact tracing efforts and outbreak resources, such as <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR19"><u>19</u></xref></sup> and GISAID<sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup> are being developed.<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [20](          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health. <italic>Glob. Chall</italic></article-title>          <source>33&#x2010;</source>          <year>2017</year>          <volume>1</volume>          <fpage>46</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32552792">32552792</a> <b>(2)</b> Acute encephalopathy with elevated CSF inflammatory markers as the initial presentation of COVID-19<ul><li> Sequencing of this SARS-CoV-2 isolate revealed a virus similar to other SARS-CoV-2 circulating in the region (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov?s=USA/CT-Yale-009/2020">https://<b>nextstrain</b>.org/ncov?s=USA/CT-Yale-009/2020</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32542015">32542015</a> <b>(1)</b> Dynamic interactions of influenza viruses in Hong Kong during 1998-2018<ul><li> <xref rid="pcbi.1007989.t002" ref-type="table"><u>Table 2</u></xref>) roughly matched with the estimated timings of amino acid mutations from <b>NextStrain</b> [<xref rid="pcbi.1007989.ref055" ref-type="bibr"><u>55</u></xref>]: 1) HA1: S185T and HA2: S124N around 10/1/09 (CI: 7/26/09-1/16/10) and 2) HA1: D97N round 10/30/09 (CI: 09/03/09-02/07/10).<li> [55](NextStrain.org (2019) Real-time tracking of influenza A/H1N1pdm evolution. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/flu/seasonal/h1n1pdm/ha/12y?c=cTiter&amp;f_country=hong_kong">https://nextstrain.org/flu/seasonal/h1n1pdm/ha/12y?c=cTiter&amp;f_country=hong_kong</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32545799">32545799</a> <b>(3)</b> Potential Antiviral Options against SARS-CoV-2 Infection<ul><li> Transmission of the strains was visualized using <uri xlink:href="https://<b>nextstrain</b>.org/ncov/global?f_author=Aleksandr%20Ianevski%20et%20al">https://<b>nextstrain</b>.org/ncov/global?f_author=Aleksandr%20Ianevski%20et%20al</uri>.<li> In cross-referencing our sequence data with the pathogen-tracking resource <b>NextStrain</b>.org, we determined that the SARS-CoV-2 strains isolated in Trondheim originated from China, Denmark, the USA and Canada (<xref ref-type="fig" rid="viruses-12-00642-f001"><u>Figure 1</u></xref>e).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32750338">32750338</a> <b>(3)</b> On the evolutionary epidemiology of SARS-CoV-2<ul><li> Based on genomic sampling over time, the substitution rate is estimated to be 0.00084 per site per year (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org" id="intref0010">www.<b>nextstrain</b>.org</ext-link> [<xref rid="bib8" ref-type="bibr"><u>8</u></xref>]; 16 May 2020), 2- to 6-fold lower than the substitution rate for influenza (0.004&#x2013;0.005 substitutions/site/year for influenza A and 0.002 substitutions/site/year for influenza B in the haemagglutinin gene [<xref rid="bib9" ref-type="bibr"><u>9</u></xref>]).<li>Genetic diversity segregating among SARS-CoV-2 genomes (from <b>Nextstrain</b> [<xref rid="bib8" ref-type="bibr"><u>8</u></xref>]).<li>Genomic tracking tools like <b>Nextstrain</b> [<xref rid="bib8" ref-type="bibr"><u>8</u></xref>] allow us to analyze genetic variants very rapidly, as soon as their sequences become available during an outbreak.<li> Mutations with no functional significance readily fluctuate in abundance, acting like genomic fingerprints that can be used to track viral geographic spread and to reconstruct epidemiological dynamics (for example, see May 8, 2020 Situation Report on international spread from <b>Nextstrain</b>).<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26053121">26053121</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32525765">32525765</a> <b>(1)</b> Genetic cluster analysis of SARS-CoV-2 and the identification of thoseresponsible for the major outbreaks in various countries<ul><li>Global genomic surveillance of SARS-CoV-2 was implemented by means of the automatedphylogenetic analysis pipeline <b>Nextstrain</b>, which generates an interactive visualizationintegrating a phylogeny with sample metadata such as geographic location or isolation date[<xref rid="CIT0013" ref-type="bibr"><u>13</u></xref>].<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32527780">32527780</a> <b>(5)</b> Complete Genome Sequence of a Novel Coronavirus (SARS-CoV-2) Isolate from Bangladesh<ul><li> The genome was subsequently contextualized among 5,265 other genomes available in the GISAID database in the <b>Nextstrain</b> Asia build of 15 May 2020 (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org/ncov/asia">www.<b>nextstrain</b>.org/ncov/asia</ext-link>) (<xref rid="B11" ref-type="bibr"><u>11</u></xref>).<li> The figure was rendered using <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>).<li> In the <b>Nextstrain</b> global build of 15 May 2020, this mutation is predicted to have emerged at the end of January 2020 and is present in many isolates from clade A2a (<xref rid="B11" ref-type="bibr"><u>11</u></xref>).<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511359">32511359</a> <b>(5)</b> Architecture and self-assembly of the SARS-CoV-2 nucleocapsid protein<ul><li> We used the <b>NextStrain</b> database<sup><xref rid="R58" ref-type="bibr"><u>58</u></xref></sup> to visualize the prevalence of the N protein G204R mutation, which is diagnostic of the GGG&#x2192;AAC trinucleotide substitution in positions 28881&#x2013;28883.<li> To visualize SARS-CoV-2 clade identity, we used the URL &#x201C;<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=clade_membership&amp;l=unrooted">https://<b>nextstrain</b>.org/ncov/global?c=clade_membership&amp;l=unrooted</ext-link>&#x201D;.<li> To color by N protein residue 204 identity, we used the URL &#x201C;<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?c=gt-N_204&amp;l=unrooted">https://<b>nextstrain</b>.org/ncov/global?c=gt-N_204&amp;l=unrooted</ext-link>&#x201D; (screenshots taken June 2, 2020).<li> [58](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511092">32511092</a> <b>(2)</b> Predicting geographic location from genetic variation with deep neural networks<ul><li> Similarly, establishing the location of origin from biological samples is critical to anti-poaching conservation efforts (<xref rid="bib58" ref-type="bibr"><u>Wasser et al., 2004</u></xref>), and efforts to map transmission during the ongoing SARS-CoV-2 pandemic have been informed by analysis of geographically restricted genetic variants (see e.g., <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> [Wasser et al., 2004](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15459317">15459317</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32577654">32577654</a> <b>(1)</b> Landscape and Selection of Vaccine Epitopes in SARS-CoV-2<ul><li> 0.1 (approximately 98% sequence identity) in all amino acid positions across MSA-aligned SARS-CoV-2 genomes within the <b>Nextstrain</b> database<sup><xref rid="R51" ref-type="bibr"><u>51</u></xref>,<xref rid="R52" ref-type="bibr"><u>52</u></xref></sup>.<li> [51](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [52](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32524946">32524946</a> <b>(2)</b> Genetic structure of SARS-CoV-2 reflects clonal superspreading and multiple independent introduction events, North-Rhine Westphalia, Germany, February and March 2020<ul><li>More than 10,000 globally sourced SARS-CoV-2 genomes are publicly available, and powerful data sharing and analysis platforms like the Global Initiative on Sharing All Influenza Data (GISAID) EpiCoV database [<xref rid="r13" ref-type="bibr"><u>13</u></xref>] and <b>Nextstrain</b> [<xref rid="r14" ref-type="bibr"><u>14</u></xref>] enable the collaborative analysis of viral population structure on a global level.<li> Consistent with reports from Iceland [<xref rid="r18" ref-type="bibr"><u>18</u></xref>], New York [<xref rid="r19" ref-type="bibr"><u>19</u></xref>], and data on <b>Nextstrain</b>, our study has demonstrated the simultaneous circulation of distinct viral variants (i.e.<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32289214">32289214</a>) [19](Gonzalez-Reiche AS, Hernandez MM, Sullivan MJ, Ciferri B, Alshammary A, Obla A, et al. Introductions and early spread of SARS-CoV-2 in the New York City area. <italic>medRxiv</italic>, 2020.04.08.20056929 (preprint). <pub-id pub-id-type="doi">10.1126/science.abc1917</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32515358">32515358</a> <b>(2)</b> Mutations in SARS-CoV-2 viral RNA identified in Eastern India: Possible implications for the ongoing outbreak in India and impact on viral structure and host susceptibility<ul><li> We used the <b>Nextstrain</b> pipeline to process the sequence data.<li> <b>Nextstrain</b> with the augur pipeline was used to build phylogenetic tree based on the IQTREE method, which is a fast and effective stochastic algorithm to infer phylogenetic trees by maximum likelihood.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32577648">32577648</a> <b>(1)</b> Highly multiplexed oligonucleotide probe-ligation testing enables efficient extraction-free SARS-CoV-2 detection and viral genotyping<ul><li> In a proof-of-concept study, we designed genotyping probes for the 20 most entropic SARS-CoV-2 SNPs reported in the <b>Nextstrain</b> database<sup><xref rid="R28" ref-type="bibr"><u>28</u></xref></sup> (queried on 3/16/2020).<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32504743">32504743</a> <b>(1)</b> SARS-CoV-2 infection, clinical features and outcome of COVID-19 in United Kingdom nursing homes<ul><li> Phylogenetic analysis and figure obtained using <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32837981">32837981</a> <b>(15)</b> Amino acid variation analysis of surface spike glycoprotein at 614 in SARS-CoV-2 strains<ul><li>In this study, we explored the genomes of 489 SARS-CoV-2 strains derived from 88 regions in 32 countries between December 2019 and March 2020 via the <b>Nextstrain</b> database of viral genomes.<li>Phylogenetic tree of SARS-CoV-2 in <b>Nextstrain</b>.<li>The phylogenetic tree of SARS-CoV-2 was performed in <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="intref0010">https://<b>nextstrain</b>.org/ncov</ext-link>),<xref rid="bib17" ref-type="bibr"><sup>17</sup></xref><sup>,</sup><xref rid="bib18" ref-type="bibr"><sup>18</sup></xref> using the parameter with clade, countries, and genotype of surface spike glycoprotein (S protein) at site 614 with default settings, to identify the genomic epidemiology of COIVD-19.<li>The world map of the SARS-CoV-2 relating clade was performed in <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="intref0015">https://<b>nextstrain</b>.org/ncov</ext-link>).<li>A phylogenetic tree of nucleotide sequences can be used to compare genome divergence and similarity among viruses.<xref rid="bib19" ref-type="bibr"><sup>19</sup></xref><sup>,</sup><xref rid="bib25" ref-type="bibr"><sup>25</sup></xref><sup>,</sup><xref rid="bib26" ref-type="bibr"><sup>26</sup></xref> Here, we performed phylogenetic tree analysis based on the consolidation of 489 genomes of SARS-CoV-2 strains sampled from 88 regions of 32 countries between December 2019 and March 2020 from the <b>Nextstrain</b> database (<xref rid="fig1" ref-type="fig"><u>Fig.&#xA0;1</u></xref>A).<li> (A) Phylogenetic tree of 489 SARS-CoV-2 genomes from <b>Nextstrain</b>, the cases were colored by countries.<li> <bold>(B)</bold> Clade Distribution of 489 SARS-CoV-2 genomes in world map from <b>Nextstrain</b>.<li> (A) Radial phylogenetic tree of 489 SARS-CoV-2 from <b>Nextstrain</b>, the cases were colored by the amino acid of S protein at site 614.<li> <bold>(B)</bold> Rectangular phylogenetic tree of 489 SARS-CoV-2 from <b>Nextstrain</b>, the cases were colored by the amino acid of S protein at site 614.<li> By examining 489 SARS-CoV-2 genomes obtained from 32 countries sampled between December 2019 and March 2020 from the <b>Nextstrain</b> phylogenetic tree, we found that strains from mainland China were mostly distributed in Clade B and Clade undefined, and differed from strains in Clades A2 and A2a.<li> As such, we re-explored the phylogenetic tree results in <b>Nextstrain</b>, and found that of the 37 and 149 cases in Italy and Spain, 35 (94.60%) and 72 (48.32%), respectively, were found in Clades A2 and A2a (<xref rid="appsec1" ref-type="sec"><u>Supplementary Figure&#xA0;1</u></xref>), thus supporting our previous findings.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32314811">32314811</a> <b>(3)</b> Understanding evolution of SARS&#x2010;CoV&#x2010;2: A perspective from analysis of genetic diversity of RdRp gene<ul><li> This resulted in sequencing of more than 170 genomes in a span of ~60 days that are being tracked by resources such as <b>Nextstrain</b> (URL: <ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>) and <ext-link xlink:href="http://Virological.org" ext-link-type="uri">Virological.org</ext-link> (URL: <ext-link xlink:href="http://virological.org" ext-link-type="uri">virological.org</ext-link>) in real time.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32412415">32412415</a> <b>(2)</b> A Snapshot of SARS-CoV-2 Genome Availability up to April 2020 and its Implications: Data Analysis<ul><li> In particular, the US-based <b>NextStrain</b> [<xref rid="ref11" ref-type="bibr"><u>11</u></xref>] team has been posting real-time updates on the tracing of the epidemic by molecular analyses.<li> Based on this observation (which is available on <b>NextStrain</b>), a heated discussion circulated on social media about a transmission event from Germany to Italy followed by further spread from Italy to other countries.<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32470119">32470119</a> <b>(3)</b> Coronavirus3D: 3D structural visualization of COVID-19 genomic divergence<ul><li>Information on the growing genetic diversity of SARS-CoV-2 is being studied intensively and continuously updated data can be obtained from resources such as GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org">https://www.gisaid.org</ext-link>) or <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32473976">32473976</a> <b>(10)</b> SARS-CoV-2 and COVID-19: A genetic, epidemiological, and evolutionary perspective<ul><li> Some convenient, easy-to-use solutions have been provided by the <b>NextStrain</b> Platform (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="ir0045">https://<b>nextstrain</b>.org</ext-link>).<li> A complete description of the tree building method used in <b>NextStrain</b> is available elsewhere (<xref rid="bb0215" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), but most readers need only understand that millions of alternative trees are considered and only the most compatible with the data is represented.<li>To fully appreciate how successfully the recent evolution of SARS-CoV-2 has been captured in the <b>NextStrain</b> platform, it is worth first reflecting on the strengths and weaknesses in the time-stamped tree.<li> It depicts relationships among hundreds of viral sequences, sub-sampled from the thousands of sequences available at the time of writing, as rendered for real-time evaluation by the <b>Nextstrain</b> consortium (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="ir0050">https://<b>nextstrain</b>.org/ncov</ext-link>) using data from <ext-link ext-link-type="uri" xlink:href="http://GISAID.org" id="ir0055">GISAID.org</ext-link>.<li>Time-stamped maximum likelihood phylogenetic reconstruction of SARS-CoV-2 isolates deposited to <ext-link ext-link-type="uri" xlink:href="http://GISAID.org" id="ir0015">GISAID.org</ext-link> and rendered by <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="ir0020">https://<b>nextstrain</b>.org/ncov</ext-link>.<li> [Hadfield et al., 2018](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>James</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>Colin</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>Sidney M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>John</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>Barney</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>Charlton</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>Pavel</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>Trevor</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>Richard A.</given-names>            </name>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics</source>          <volume>43</volume>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511505">32511505</a> <b>(3)</b> Early transmission of SARS-CoV-2 in South Africa: An epidemiologicaland phylogenetic report<ul><li> The resulting dataset of 1848sequences was used in a custom build on the <b>NextStrain</b> analysis platform (<xref rid="R32" ref-type="bibr"><u>32</u></xref>).<li> To infer support for the splits inthis tree topology we inferred an additional 100 bootstrap trees in IQ-Treeunder the same model parameters as <b>NextStrain</b>.<li>Bayesian coalescent analyses were performed on major lineages of the<b>NextStrain</b> build in which KZN sequences fell.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32470049">32470049</a> <b>(13)</b> Origin of HAV strains responsible for 2016&#x2013;2017 outbreak among MSM: Viral phylodynamics in Lazio region<ul><li>At first, this dataset was analyzed through web-based <b>Nextstrain</b> tool, which uses a augur pipeline [<xref rid="pone.0234010.ref037" ref-type="bibr"><u>37</u></xref>], for reconstruction of viral phylogeny [<xref rid="pone.0234010.ref038" ref-type="bibr"><u>38</u></xref>, <xref rid="pone.0234010.ref039" ref-type="bibr"><u>39</u></xref>].<li> The resulting analysis is visualized by auspice and is publicly available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/INMIbioinfo/HAV_VP1-2A">https://<b>nextstrain</b>.org/community/INMIbioinfo/HAV_VP1-2A</ext-link>.<li> At first, an explorative analysis of temporal and phylogenetic relationship was done using the web-based <b>Nextstrain</b> tool (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/INMIbioinfo/HAV_VP1-2A">https://<b>nextstrain</b>.org/community/INMIbioinfo/HAV_VP1-2A</ext-link>), which allows users to interactively explore the dataset.<li>The panel A shows a phylogenetic tree as obtained by <b>Nextstrain</b>, containing VP1X2A junction sequences of HAV genome from Lazio region, Europe and worldwide.<li> The analysis of coalescent events from Bayesian MCC tree suggests for both clusters an origin dating back to 2012 (95% HPD: 2006&#x2013;2015), which was consistent with <b>Nextstrain</b> dating.<li> <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>), may represent a valuable tool for early appreciation of an impending epidemic event.<li>Panel A shows the phylogenetic tree, based on VP1X2A junction sequences of HAV genome, as obtained by <b>Nextstrain</b>, labelled by country.<li>HAV database used for <b>Nextstrain</b> tree.<li>List of HAV sequences and information, using for interactive phylodynamic analysis on <b>Nextstrain</b>.org.<li> [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31471545">31471545</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31037220">31037220</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32471856">32471856</a> <b>(1)</b> Introductions and early spread of SARS-CoV-2 in the New York City area<ul><li> As a reference, we adopted the clade nomenclature from the <b>NextStrain</b> tool (<xref rid="R5" ref-type="bibr"><italic>5</italic></xref>, <xref rid="R6" ref-type="bibr"><italic>6</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32485821">32485821</a> <b>(1)</b> Non-Mouse-Adapted H1N1pdm09 Virus as a Model for Influenza Research<ul><li>Phylogenetic analysis of H1N1pdm09 viruses was performed using the <b>nextstrain</b>.org website [<xref rid="B21-viruses-12-00590" ref-type="bibr"><u>21</u></xref>].<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32536940">32536940</a> <b>(2)</b> Rates and Rocks: Strengths and Weaknesses of Molecular Dating Methods<ul><li> Biek et al., <xref rid="B5" ref-type="bibr"><u>2015</u></xref>, and the excellent internet resource <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>, Hadfield et al., <xref rid="B34" ref-type="bibr"><u>2018</u></xref>).<li> [2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25887947">25887947</a>) [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32461612">32461612</a> <b>(3)</b> Vulnerabilities in coronavirus glycan shields despite extensive glycosylation<ul><li> Bioinformatic analysis of current SARS-CoV-2 spike genes (<italic>n</italic>&#x2009;=&#x2009;566 as of 17 March 2020) from <b>nextstrain</b><sup><xref ref-type="bibr" rid="CR61"><u>61</u></xref></sup> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>) revealed low sequence diversity and no changes in glycosylation sites (SI Fig.&#xA0;<xref rid="MOESM1" ref-type="media"><u>8</u></xref>).<li> [61](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32451533">32451533</a> <b>(4)</b> Molecular Detection of SARS-CoV-2 Infection in FFPE Samples and Histopathologic Findings in Fatal SARS-CoV-2 Cases<ul><li> Genomic epidemiology classification was performed using <b>NextStrain</b>.org on April 27, 2020.<li> Shared SARS-CoV-2 genotyping efforts and resources like the Global Initiative on Sharing All Influenza Data (GISAID, <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org">https://www.gisaid.org</ext-link>), <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>), and Johns Hopkins University Dashboard (<ext-link ext-link-type="uri" xlink:href="https://coronavirus.jhu.edu/map.html">https://coronavirus.jhu.edu/map.html</ext-link>) have enabled dynamic tracking of the evolution of the pandemic, enhanced monitoring of infection patterns, and have allowed better estimations of the affected population size in communities.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32464271">32464271</a> <b>(4)</b> Could the D614G substitution in the SARS-CoV-2 spike (S) protein be associated with higher COVID-19 mortality?<ul><li><bold>(A)</bold> Unrooted phylogenetic tree with clear distinction of the 614G and 614D clades at the S protein site (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="intr0005">https://<b>nextstrain</b>.org/ncov</ext-link>; last accessed May 2, 2020).<li> The map inside the box plot presents the distribution of S-614G and the wild-type S-614D in European populations (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="intr0015">https://<b>nextstrain</b>.org/ncov</ext-link>; last accessed May 2, 2020) labeled in the y-axis of the bar plot.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32501353">32501353</a> <b>(4)</b> An evolving crisis<ul><li> In 2018, a group of researchers set up an open-source project called <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="interrefs210"><b>Nextstrain</b></ext-link> to do just that, looking at a range of diseases from seasonal flu and measles to West Nile virus and Zika.<li> says Emma Hodcroft at the University of Basel, Switzerland, who is a co-developer of <b>Nextstrain</b>.<li> It will take unprecedented collaboration, but projects like <b>Nextstrain</b> illustrate that this is already happening.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32445875">32445875</a> <b>(2)</b> Ion torrent-based nasopharyngeal swab metatranscriptomics in COVID-19<ul><li> SNP analysis using Atlas 2 (<ext-link ext-link-type="uri" xlink:href="https://sourceforge.net/p/atlas2/wiki/Atlas-SNP/" id="intr0010">https://sourceforge.net/p/atlas2/wiki/Atlas-SNP/</ext-link>) indicated that mutations accumulate mainly in 4 ORFs (ORF1a, M, ORF8 and N) in addition to 3&#x2019;UTR, in a similar profile observed for strains derived from Brazilian patients present in NexStrain platform (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intr0015">https://<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32509472">32509472</a> <b>(1)</b> Insights on early mutational events in SARS-CoV-2 virus reveal founder effects across geographical regions<ul><li> Consistent with the latter, real-time tracking of pathogen evolution and phylogenetic analysis provided from the <b>Nextstrain</b> initiative demonstrated dissemination of SARS-CoV-2 viral signatures from USA to Australia and Europe in early February 2020 (<xref rid="ref-8" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32782825">32782825</a> <b>(1)</b> The role of genetic sequencing and analysis in the polio eradication programme<ul><li> for the 2014 Ebola outbreak and interactive phylogenetic trees using the <b>Nextstrain</b> platform have also been developed, and are provided in internal WHO reports, but are not yet publicly available (<xref rid="veaa040-B13" ref-type="bibr">Dudas et&#xA0;al.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32433217">32433217</a> <b>(4)</b> SARS-CoV-2/COVID-19: Evolving Reality, Global Response, Knowledge Gaps, and Opportunities<ul><li> The visualization is based upon sub-sampled available genome data (see more under: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>).<li> Graphic modified based on <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link> (accessed on April 27, 2020).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33184612">33184612</a> <b>(2)</b> Bringing Big Data to Bear in Environmental Public Health: Challenges and Recommendations<ul><li> Bai et al., <xref rid="B2" ref-type="bibr"><u>2018</u></xref>), disease diagnosis (Xiong et al., <xref rid="B49" ref-type="bibr"><u>2018</u></xref>), infectious disease monitoring (Milinovich et al., <xref rid="B33" ref-type="bibr"><u>2014</u></xref><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>), tracking antibiotic resistance (Li et al., <xref rid="B25" ref-type="bibr"><u>2018</u></xref>), computational chemistry (Goh et al., <xref rid="B16" ref-type="bibr"><u>2017</u></xref>), exposure and chemical-mixture modeling (Bobb et al., <xref rid="B5" ref-type="bibr"><u>2014</u></xref>; Park et al., <xref rid="B37" ref-type="bibr"><u>2017</u></xref>; Vopham et al., <xref rid="B45" ref-type="bibr"><u>2018</u></xref>) and improving classification of climate regions (Liss et al., <xref rid="B26" ref-type="bibr"><u>2014</u></xref>).<li> [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29673227">29673227</a>) [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29707349">29707349</a>) [2014](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24290841">24290841</a>) [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29793542">29793542</a>) [2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28272810">28272810</a>) [2014](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25532525">25532525</a>) [2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28950902">28950902</a>) [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29665858">29665858</a>) [2014](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25599636">25599636</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32478289">32478289</a> <b>(4)</b> The origin of SARS-CoV-2 in Istanbul: Sequencing findings from the epicenter of the pandemic in Turkey<ul><li> Viral genome sequences were examined phylogenetically together with world-wide isolates using <b>Nextstrain</b> [<xref rid="ref25" ref-type="bibr"><u>25</u></xref>, <xref rid="ref26" ref-type="bibr"><u>26</u></xref>].<li>Retrieved and adapted from <ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org">www.<b>nextstrain</b>.org</ext-link>.<li> The <b>Nextstrain</b> website (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org">www.<b>nextstrain</b>.org</ext-link>) provides real-time monitoring of viral isolates around the world, mainly based on publicly accessible GISAID data [<xref rid="ref25" ref-type="bibr"><u>25</u></xref>, <xref rid="ref26" ref-type="bibr"><u>26</u></xref>].<li> As of 1 May 2020, more than ten thousand genomes were uploaded to the GISAID database and nearly five thousand different genomes are available for analysis in <b>Nextstrain</b> online tool.<li> Phylogenetic analysis in <b>Nextstrain</b> online tool showed that three isolates from Istanbul in this study were found to be closely clustered within samples isolated mostly in Belgium (<xref ref-type="fig" rid="F3"><u>Figure 3</u></xref>).<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32419848">32419848</a> <b>(3)</b> Ideas for how informaticians can get involved with COVID-19 research<ul><li> In particular, <b>Nextstrain</b>, an open-source software package (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>), uses SARS-CoV-2 genome data to help track the spread of disease outbreaks.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32702723">32702723</a> <b>(2)</b> Acute encephalopathy with elevated CSF inflammatory markers as the initial presentation of COVID-19.<ul><li> 2 isolate revealed a virus similar to other SARS-CoV&#x2013;2 circulating in the region (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov?s=USA/CT-Yale&#x2013;009/2020">https://<b>nextstrain</b>.org/ncov?s=USA/CT-Yale&#x2013;009/2020</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32392129">32392129</a> <b>(3)</b> Screening of healthcare workers for SARS-CoV-2 highlights the role of asymptomatic carriage in COVID-19 transmission<ul><li> A SARS-CoV-2 genome collected in Wuhan in December 2019 was selected to root the tree, visualised initially on <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) and the fasta file was downloaded from GISAID (ID: EPI ISL 402123) (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</ext-link>).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32386545">32386545</a> <b>(16)</b> Coast-to-Coast Spread of SARS-CoV-2 during the Early Epidemic in the United States<ul><li> The genomic data can be visualized and interacted with at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper1" id="intref0185">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper1</ext-link>.<li> <b>Nextstrain</b> page (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper1" id="intref0010">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper1</ext-link>).<li> As of March 25, there are more than 1,000 SARS-CoV-2 genomes sequenced from around the world, including more than 350 from the United States (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="intref0015">https://<b>nextstrain</b>.org/ncov</ext-link>); however, most of the latter were obtained from a small number of states.<li><hr/></td></tr><tr><td>R</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/" id="intref0040">https://cran.r-project.org/</ext-link></td></tr><tr><td>IQ-Tree</td><td><ext-link ext-link-type="uri" xlink:href="http://www.iqtree.org/" id="intref0045">http://www.iqtree.org/</ext-link></td><td><xref rid="bib16" ref-type="bibr"><u>Nguyen et&#xA0;al., 2015</u></xref></td></tr><tr><td>augur toolkit</td><td><ext-link ext-link-type="uri" xlink:href="https://bedford.io/projects/augur/" id="intref0050">https://bedford.io/projects/augur/</ext-link></td><td><xref rid="bib11" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td></tr><tr><td>MAFFT</td><td><ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/" id="intref0055">https://mafft.cbrc.jp/alignment/software/</ext-link></td><td><xref rid="bib12" ref-type="bibr"><u>Katoh et&#xA0;al., 2002</u></xref></td></tr><tr><td>treetime</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/treetime" id="intref0060">https://github.com/neherlab/treetime</ext-link></td><td><xref rid="bib24" ref-type="bibr"><u>Sagulenko et&#xA0;al., 2018</u></xref></td></tr><tr><td>RAMPART</td><td>ARTIC Network</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/artic-network/rampart" id="intref0065">https://github.com/artic-network/rampart</ext-link></td></tr><tr><td>ARTIC Network Bioinformatic protocol</td><td>ARTIC Network</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html" id="intref0070">https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html</ext-link></td></tr><tr><td><b>Nextstrain</b>.<li><hr/></td></tr><tr><td>R</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/" id="intref0040">https://cran.r-project.org/</ext-link></td></tr><tr><td>IQ-Tree</td><td><ext-link ext-link-type="uri" xlink:href="http://www.iqtree.org/" id="intref0045">http://www.iqtree.org/</ext-link></td><td><xref rid="bib16" ref-type="bibr"><u>Nguyen et&#xA0;al., 2015</u></xref></td></tr><tr><td>augur toolkit</td><td><ext-link ext-link-type="uri" xlink:href="https://bedford.io/projects/augur/" id="intref0050">https://bedford.io/projects/augur/</ext-link></td><td><xref rid="bib11" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td></tr><tr><td>MAFFT</td><td><ext-link ext-link-type="uri" xlink:href="https://mafft.cbrc.jp/alignment/software/" id="intref0055">https://mafft.cbrc.jp/alignment/software/</ext-link></td><td><xref rid="bib12" ref-type="bibr"><u>Katoh et&#xA0;al., 2002</u></xref></td></tr><tr><td>treetime</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/treetime" id="intref0060">https://github.com/neherlab/treetime</ext-link></td><td><xref rid="bib24" ref-type="bibr"><u>Sagulenko et&#xA0;al., 2018</u></xref></td></tr><tr><td>RAMPART</td><td>ARTIC Network</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/artic-network/rampart" id="intref0065">https://github.com/artic-network/rampart</ext-link></td></tr><tr><td>ARTIC Network Bioinformatic protocol</td><td>ARTIC Network</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html" id="intref0070">https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html</ext-link></td></tr><tr><td><b>Nextstrain</b></td><td><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0075">https://<b>nextstrain</b>.org/</ext-link>.<li> The interactive Nextstain page to visualize the genomic data can be found at: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper1" id="intref0085">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper1</ext-link>.<li> ends of the alignment as well as a small number of sites that likely vary due to assembly artifacts (see <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0110">https://github.com/<b>nextstrain</b>/ncov</ext-link>), and reconstructed a phylogeny using IQ-Tree (<xref rid="bib16" ref-type="bibr"><u>Nguyen et&#xA0;al., 2015</u></xref>).<li> The data can be visualized at: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper1" id="intref0115">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2/paper1</ext-link>.<li> [Nguyen et&#xA0;al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Katoh et&#xA0;al., 2002](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [Sagulenko et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Nguyen et&#xA0;al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Katoh et&#xA0;al., 2002](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [Sagulenko et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Nguyen et&#xA0;al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32381734">32381734</a> <b>(1)</b> Sampling bias and incorrect rooting make phylogenetic network tracing of SARS-COV-2 infections unreliable<ul><li> The fact that SARS-CoV-2 sequences show some geographical clustering is not new and is nicely and interactively shown on <b>Nextstrain</b> (<xref rid="r5" ref-type="bibr"><u>5</u></xref>), but this cannot be used as a proof of biological differences unless backed by solid experimental data (<xref rid="r6" ref-type="bibr"><u>6</u></xref>).<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32071422">32071422</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32392812">32392812</a> <b>(1)</b> Pandemic Influenza Vaccines: What did We Learn from the 2009 Pandemic and are We Better Prepared Now?<ul><li> High-throughput sequencing, through use of next generation sequencing technologies, more specialist influenza databases (e.g., GISAID, IRD) and online analysis tools (e.g., <b>Nextstrain</b>, IRD, GISAID, FluSurver) are also now available, improving surveillance capacity and understanding.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32446559">32446559</a> <b>(3)</b> Stability of RNA sequences derived from the coronavirus genome in human cells<ul><li> nucleotides and were incorporated into the <b>Nextstrain</b> website (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global" id="intref0060z">https://<b>nextstrain</b>.org/ncov/global</ext-link>; last access: 2020/0) were selected, resulting in 1,116 SARS-CoV-2 genomes (<xref rid="appsec1" ref-type="sec"><u>Table&#xA0;S2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511488">32511488</a> <b>(1)</b> Clinical features, diagnostics, and outcomes of patients presenting with acute respiratory illness: a comparison of patients with and without COVID-19<ul><li> For phylogenetic analysis of SARS-CoV-2 viruses, we constructed genomes using minimap2<sup><xref rid="R32" ref-type="bibr"><u>32</u></xref></sup> to align reads to the reference MN908947.3 and iVar<sup><xref rid="R33" ref-type="bibr"><u>33</u></xref></sup> to trim primers and call variants, then restricted to samples with at least 10-fold coverage of at least 97% (29 kilobases) of the genome (n=10), and utilized the <b>Nextstrain</b><sup><xref rid="R34" ref-type="bibr"><u>34</u></xref></sup> pipeline to build a phylogenetic tree using iqtree.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29750242">29750242</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30621750">30621750</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32387564">32387564</a> <b>(13)</b> Emergence of genomic diversity and recurrent mutations in SARS-CoV-2<ul><li> The extraordinary availability of genomic data during the COVID-19 pandemic has been made possible thanks to a tremendous effort by hundreds of researchers globally depositing SARS-CoV-2 assemblies (Table S1) and the proliferation of close to real time data visualisation and analysis tools including <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="ir0030">https://<b>nextstrain</b>.org</ext-link>) and CoV-GLUE (<ext-link ext-link-type="uri" xlink:href="http://cov-glue.cvr.gla.ac.uk" id="ir0035">http://cov-glue.cvr.gla.ac.uk</ext-link>).<li> Sequence metadata was obtained from the <b>NextStrain</b> Github repository (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/tree/master/data" id="ir0050">https://github.com/<b>nextstrain</b>/ncov/tree/master/data</ext-link>).<li>Assemblies were aligned against the Wuhan-Hu-1 reference genome (NC_045512.2, EPI_ISL_402125) using MAFFT (<xref rid="bb0105" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>) implemented via the rapid phylodynamic alignment pipeline provided by Augur (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur" id="ir1050">https://github.com/<b>nextstrain</b>/augur</ext-link>).<li> Sites in the first 130&#xA0;bp and last 50&#xA0;bp of the alignment were masked, as were positions 18,529, 29,849, 29,851 and 29,853, following the protocol also advocated by <b>NextStrain</b> and to account for the fact many putatively artefactual SNPs are located at the beginning and ends of the alignment.<li> S6), and inspected the distribution of all identified homoplasies in the phylogenies from our own analyses and on the phylogenetic visualisation platform provided by <b>NextStrain</b>.<li> Dark blue: South America according to metadata annotations available on <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/tree/master/data" id="ir0005">https://github.com/<b>nextstrain</b>/ncov/tree/master/data</ext-link>).<li> [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32387409">32387409</a> <b>(4)</b> Early treatment of COVID-19 patients with hydroxychloroquine and azithromycin: A retrospective analysis of 1061 cases in Marseille, France<ul><li> Phylogenetic reconstruction was performed using NEXSTRAIN (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0020">https://<b>nextstrain</b>.org/</ext-link>) and GISAID (Global Initiative; <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="intref0025">https://www.gisaid.org/</ext-link>) [<xref rid="bib24" ref-type="bibr"><u>24</u></xref>].<li> Phylogenetic reconstruction was performed using NEXSTRAIN (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0045">https://<b>nextstrain</b>.org/</ext-link>) and GISAID (Global Initiative; <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="intref0050">https://www.gisaid.org/</ext-link>) with acknowledgments [<xref rid="bib24" ref-type="bibr"><u>24</u></xref>].<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32474553">32474553</a> <b>(17)</b> Analysis of RNA sequences of 3636 SARS-CoV-2 collected from 55 countries reveals selective sweep of one virus type<ul><li>The data dump was downloaded from <italic><ext-link ext-link-type="uri" xlink:href="www.<b>nextstrain</b>.org">www.<b>nextstrain</b>.org</ext-link></italic> (<italic><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/">https://<b>nextstrain</b>.org/ncov/</ext-link></italic>) on April 6, 2020.<li> The data dump from the <b>Nextstrain</b> portal contains information on various parameters such as viral strain name, viral sample collection data, sampled from the country and State level information as available from the submitter, viral type information, age, GISAID accession number and sequence submission date.<li> Type defining marker mutations (mostly amino acid changes) were obtained from the <b>Nextstrain</b> github repository (<italic><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link></italic>).<li> Phylodynamic analyses were performed using TreeTime<xref rid="ref11" ref-type="bibr"><u>11</u></xref>, as implemented in the <b>Nextstrain</b> pipeline<xref rid="ref12" ref-type="bibr"><u>12</u></xref>.<li> (<italic>Source</italic>: <italic><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link></italic>).<li> <italic>Source: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/">https://<b>nextstrain</b>.org/ncov/</ext-link></italic>.<li> <italic>Source: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/">https://<b>nextstrain</b>.org/ncov/</ext-link></italic>.<li> <italic>Source: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/">https://<b>nextstrain</b>.org/ncov/</ext-link></italic>.<li><italic>Source</italic>: <italic><ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/asia">https://<b>nextstrain</b>.org/ncov/asia</ext-link></italic>.<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32361161">32361161</a> <b>(1)</b> New understanding of the damage of SARS-CoV-2 infection outside the respiratory system<ul><li> The entire genome sequence of SARS-CoV-2 has been published on virological.org, <b>nextstrain</b>.org, and bioRxiv.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32537561">32537561</a> <b>(2)</b> Current and emerging diagnostic tests available for the novel COVID-19 global pandemic<ul><li> Many of these genomes have been successfully uploaded to Global Initiative on Sharing All Influenza Data -<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">GISAID</ext-link> by the WHO reference laboratories globally and<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov"><b>Nextstrain</b></ext-link>.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32333836">32333836</a> <b>(3)</b> Inhibition of SARS-CoV-2 Infections in Engineered Human Tissues Using Clinical-Grade Soluble Human ACE2<ul><li>To generate a phylogenetic tree, we created a genomic epidemiology map of different SARS-CoV-2 isolates using <b>NextStrain</b> tools (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0105">https://<b>nextstrain</b>.org/</ext-link>) (<xref rid="bib56" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32335334">32335334</a> <b>(2)</b> Emerging genetic diversity among clinical isolates of SARS-CoV-2: Lessons for today<ul><li><bold>Visual depiction of emerging genomic diversity among clinical isolates of SARS-CoV-2</bold> (A) SARS-CoV-2 separating into different clades since its identification in December 2019 till March 2020 using Wuhan-Hu-1/2019 (NC_045512) as reference strain (courtesy: <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org" id="ir0005"><b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32373634">32373634</a> <b>(3)</b> Animal Disease Surveillance in the 21st Century: Applications and Robustness of Phylodynamic Methods in Recent U.S. Human-Like H3 Swine Influenza Outbreaks<ul><li><b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>), which implements TreeTime is a futuristic working example of a web-based real-time molecular surveillance system for important human pathogens such as influenza, Ebola, Dengue, and the newly emerging corona (COVID-19) viruses.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32313804">32313804</a> <b>(2)</b> Tracing New Clinical Manifestations in Patients with COVID-19 in Chile and Its Potential Relationship with the SARS-CoV-2 Divergence<ul><li>Phylogeny of SARS-CoV-2 reconstructed by the GISAD collaboration (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>) from 1455 genomes including seven strains sequenced from Chile.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32303725">32303725</a> <b>(5)</b> Coronavirus jolts labs to warp speed<ul><li> Data from, for example, the Global Initiative on Sharing All Influenza Data (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">GISAID</ext-link>) feed into <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org"><b>Nextstrain</b></ext-link>, which is devoted to the exchange of genome data about pathogens and which offers a suite of software tools.<li> <b>Nextstrain</b> has a <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">dedicated tab</ext-link> for genetic epidemiological analysis of the novel coronavirus, with interactive data visualizations.<li><b>Nextstrain</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511596">32511596</a> <b>(5)</b> Cryptic transmission of SARS-CoV-2 in Washington State<ul><li> From 29 February onwards, new genomic data was immediately posted to the GISAID EpiCoV sequence database (<xref rid="R23" ref-type="bibr"><u>23</u></xref>, <xref rid="R24" ref-type="bibr"><u>24</u></xref>) and analyzed alongside other public SARS-CoV-2 genomes via the <b>Nextstrain</b> online platform (<xref rid="R25" ref-type="bibr"><u>25</u></xref>) to provide immediate and public situational awareness.<li>SARS-CoV-2 genomes from the global COVID-19 pandemic were downloaded from GISAID (<xref rid="R23" ref-type="bibr"><u>23</u></xref>, <xref rid="R24" ref-type="bibr"><u>24</u></xref>) and processed using the <b>Nextstrain</b> (<xref rid="R25" ref-type="bibr"><u>25</u></xref>) bioinformatics pipeline <italic>Augur</italic>to align genomes via MAFFT v7.4 (<xref rid="R33" ref-type="bibr"><u>33</u></xref>), build maximum likelihood phylogeny via IQTREE v1.6 (<xref rid="R34" ref-type="bibr"><u>34</u></xref>) and reconstruct nucleotide and amino acid changes on the ML tree.<li> This bioinformatic processing pipeline is fully documented and reproducible at <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>.<li> The resulting tree was visualized in the <b>Nextstrain</b> web application <italic>Auspice</italic> to view resulting inferences.<li> [23](<name><surname>Shu</surname><given-names>Y.</given-names></name>, <name><surname>McCauley</surname><given-names>J.</given-names></name>, <article-title>GISAID: Global initiative on sharing all influenza data - from vision to reality</article-title>. <source>Euro Surveill</source>. <volume>22</volume> (<year>2017</year>), doi:<pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](<name><surname>Shu</surname><given-names>Y.</given-names></name>, <name><surname>McCauley</surname><given-names>J.</given-names></name>, <article-title>GISAID: Global initiative on sharing all influenza data - from vision to reality</article-title>. <source>Euro Surveill</source>. <volume>22</volume> (<year>2017</year>), doi:<pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32283108">32283108</a> <b>(2)</b> Remdesivir and SARS-CoV-2: Structural requirements at both nsp12 RdRp and nsp14 Exonuclease active-sites<ul><li>As of Feb 17, 2020, 90 complete SARS-CoV-2 genome sequences have been published and analyzed in the Nexstrain repository (<ext-link ext-link-type="uri" xlink:href="http://<b>Nextstrain</b>.org" id="intref0010"><b>Nextstrain</b>.org</ext-link>), with the first sequence deposited December 2019.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32242948">32242948</a> <b>(3)</b> Response to Carletti et al, &#x201C;About the origin of the first two SARS&#x2010;CoV&#x2010;2 infections in Italy: Inference not supported by appropriate sequence analysis&#x201D;<ul><li> have been confirmed by our recent publication,<xref rid="jmv25823-bib-0001" ref-type="bibr"><sup>1</sup></xref> as well as by the results shared with the scientific committee by the <b>Nextstrain</b> (real&#x2010;time tracking of pathogen evolution) coronavirus report available on <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32290905">32290905</a> <b>(1)</b> A hundred days into the coronavirus disease (COVID-19) pandemic<ul><li> In particular, the global initiative on sharing all influenza data (GISAID) has allowed organisations like <b>nextstrain</b>.org to collectively perform analysis in almost real time.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32277969">32277969</a> <b>(3)</b> The role of phylogenetic analysis in clarifying the infection source of a COVID-19 patient<ul><li> The phylogenetic tree was generated and modified for display purposes from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov" id="interref0001">https://<b>nextstrain</b>.org/ncov</ext-link>),<xref rid="bib0009" ref-type="bibr"><sup>9</sup></xref> which uses genetic sequences and metadata from GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/CoV2020/" id="interref0002">https://www.gisaid.org/CoV2020/</ext-link>) and sequence submission date for the horizontal axis.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511306">32511306</a> <b>(3)</b> RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses<ul><li>SARS-CoV-2-MSA-2 was generated from 739 GISAID<sup><xref rid="R17" ref-type="bibr"><u>17</u></xref></sup> sequences, including all sequences described in the <b>Nextstrain</b> project metadata file as of 03&#x2013;18-20 (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34908970">34908970</a> <b>(2)</b> The Dynamics of Fear at the Time of Covid-19: A Contextual Behavioral Science Perspective<ul><li> The presence of the virus in the USA was known and documented since the end of December 2019 <ext-link xlink:href="http://<b>Nextstrain</b>.org" ext-link-type="uri">(<b>Nextstrain</b>.org</ext-link> December 31, 2020).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32292213">32292213</a> <b>(3)</b> Coronaviren: von der banalen Erk&#xE4;ltung zum schweren Lungenversagen<ul><li><b>Nextstrain</b>: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32385132">32385132</a> <b>(1)</b> Consensus for prevention and management of coronavirus disease 2019 (COVID-19) for neurologists<ul><li> Genomic sequence analysis of SARS-CoV-2 has been published on virological.org, <b>nextstrain</b>.org, bioRxiv and other academic journals.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511579">32511579</a> <b>(1)</b> A Genomic Survey of SARS-CoV-2 Reveals Multiple Introductions into Northern California without a Predominant Lineage<ul><li>The assembled SARS-CoV-2 genomes in this study were uploaded to GISAID (<xref rid="R28" ref-type="bibr"><u>28</u></xref>&#x2013;<xref rid="R29" ref-type="bibr"><u>29</u></xref>) as FASTA files, and can be visualized on a continually updated phylogenetic tree on <b>NextStrain</b> (<xref rid="R30" ref-type="bibr"><u>30</u></xref>).<li> [28](<name><surname>Shu</surname><given-names>Y.</given-names></name>, <name><surname>McCauley</surname><given-names>J.</given-names></name>, <article-title>GISAID: Global initiative on sharing all influenza data &#x2013; from vision to reality</article-title>. <source>Euro Surveill</source>. <volume>22</volume> (<year>2017</year>), doi:<pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511620">32511620</a> <b>(1)</b> Global transmission network of SARS-CoV-2: from outbreak to pandemic<ul><li> Automatic high-performance computing-based molecular near real-time surveillance systems such as <b>Nextstrain</b> [<xref rid="R28" ref-type="bibr"><u>28</u></xref>], HIV-Trace [<xref rid="R36" ref-type="bibr"><u>36</u></xref>] and GHOST [<xref rid="R40" ref-type="bibr"><u>40</u></xref>] could be instrumental in such public health global surveillance and decision making.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29401317">29401317</a>) [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29244005">29244005</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32511630">32511630</a> <b>(11)</b> Coast-to-coast spread of SARS-CoV-2 in the United States revealed by genomic epidemiology<ul><li> <b>Nextstrain</b> page (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2</ext-link>).<li> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov">https://<b>nextstrain</b>.org/ncov</ext-link>); however most of the latter were obtained from a small number of U.S.<li> ends of the alignment as well as a small number of sites that likely vary due to assembly artefacts (see <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>), and reconstructed a phylogeny using IQ-Tree (<xref rid="R15" ref-type="bibr"><u>Nguyen et al., 2015</u></xref>).<li> The data can be visualized at: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2</ext-link>.<li> The interactive Nextstain page to visualize the genomic data can be found at: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2">https://<b>nextstrain</b>.org/community/grubaughlab/CT-SARS-CoV-2</ext-link>.<li> [Nguyen et al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32198478">32198478</a> <b>(1)</b> Large scale automated phylogenomic analysis of bacterial isolates and the Evergreen Online platform<ul><li> For viral pathogens, real-time tracking and visualization of evolution has been realized for seasonal influenza with Nextflu<sup><xref ref-type="bibr" rid="CR34"><u>34</u></xref></sup>, and generalized in <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR35"><u>35</u></xref></sup>.<li> [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32209163">32209163</a> <b>(1)</b> Serological and molecular findings during SARS-CoV-2 infection: the first case study in Finland, January to February 2020<ul><li> Later sequence information in Finland (up to 2 March) showed clustering with strains circulating in Italy (see <b>nextstrain</b>.org/ncov) [<xref rid="r9" ref-type="bibr"><u>9</u></xref>].<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32155798">32155798</a> <b>(3)</b> The Nature and Extent of Plasmid Variation in <italic>Chlamydia trachomatis</italic><ul><li>Maximum likelihood analysis was performed on the dataset using TreeTime [<xref rid="B61-microorganisms-08-00373" ref-type="bibr"><u>61</u></xref>] using the <b>Nextstrain</b> platform, which consists of data curation, analysis and visualization components [<xref rid="B49-microorganisms-08-00373" ref-type="bibr"><u>49</u></xref>].<li>The plasmid phylogenetic tree can be viewed at: <uri xlink:href="https://<b>nextstrain</b>.org/community/jameshadfield/scratch/ct">https://<b>nextstrain</b>.org/community/jameshadfield/scratch/ct</uri>.<li> [61](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [49](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Harris</surname>              <given-names>S.R.</given-names>            </name>            <name>              <surname>Seth-Smith</surname>              <given-names>H.M.B.</given-names>            </name>            <name>              <surname>Parmar</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Andersson</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Giffard</surname>              <given-names>P.M.</given-names>            </name>            <name>              <surname>Schachter</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Moncada</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Ellison</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Vaulet</surname>              <given-names>M.L.G.</given-names>            </name>            <etal/>          </person-group>          <article-title>Comprehensive global genome dynamics of <italic>Chlamydia trachomatis</italic> show ancient diversification followed by contemporary mixing and recent lineage expansion</article-title>          <source>Genome Res.</source>          <year>2017</year>          <pub-id pub-id-type="doi">10.1101/gr.212647.116</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32282863">32282863</a> <b>(1)</b> Molecular immune pathogenesis and diagnosis of COVID-19<ul><li> The phylogenetic tree of complete genome sequences of coronaviruses was obtained and analyzed with <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/blab/sars-like-cov" id="intref0020">https://github.com/blab/sars-like-cov</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32101295">32101295</a> <b>(1)</b> Online Bayesian Phylodynamic Inference in BEAST with Application to Epidemic Reconstruction<ul><li> Fast maximum-likelihood-based methods, such as those adopted by <b>Nextstrain</b> (<xref rid="msaa047-B31" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32071422">32071422</a> <b>(2)</b> We shouldn&#x2019;t worry when a virus mutates during disease outbreaks<ul><li> To this end, websites like <ext-link ext-link-type="uri" xlink:href="http://virological.org/">Virological.org</ext-link> and <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>Nextstrain</b>.org</ext-link> are leading the charge.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33829127">33829127</a> <b>(3)</b> Typing methods based on whole genome sequencing data<ul><li> Some projects such as Microreact (<ext-link ext-link-type="uri" xlink:href="https://microreact.org">https://microreact.org</ext-link>) or <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) have developed tools for this purpose, but these need to be more broadly accessible and applicable for official laboratories involved in routine surveillance.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32029567">32029567</a> <b>(3)</b> Nanopore-Based Complete Genome Sequence of a Sri Lankan Cassava Mosaic Virus (<italic>Geminivirus</italic>) Strain from Thailand<ul><li> A phylogenetic tree based on <b>Nextstrain</b> analysis (<xref rid="B15" ref-type="bibr"><u>15</u></xref>) is available online (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/pestdisplace/CMDASIA?c=virus&amp;r=location">https://<b>nextstrain</b>.org/community/pestdisplace/CMDASIA?c=virus&amp;r=location</ext-link>).<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a> <b>(1)</b> IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era<ul><li> 2018</xref>), <b>Nextstrain</b> (<xref rid="msaa015-B20" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31951644">31951644</a> <b>(8)</b> Quantifying within-host diversity of H5N1 influenza viruses in humans and poultry in Cambodia<ul><li> Sequences and metadata were cleaned and organized using fauna (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/fauna">https://github.com/<b>nextstrain</b>/fauna</ext-link>), a database system part of the <b>Nextstrain</b> platform.<li> Sequences were then processed using <b>Nextstrain</b>&#x2019;s augur software[<xref rid="ppat.1008191.ref035" ref-type="bibr"><u>35</u></xref>] (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur">https://github.com/<b>nextstrain</b>/augur</ext-link>).<li>All currently available H5N1 sequences were downloaded from the Influenza Research Database and the Global Initiative on Sharing All Influenza Data and used to generate full genome phylogenies using <b>Nextstrain</b>&#x2019;s augur pipeline as shown in the trees on the left.<li>All currently available H5N1 virus sequences were downloaded from the Influenza Research Database and the Global Initiative on Sharing All Influenza Data and used to generate full genome phylogenies using <b>Nextstrain</b>&#x2019;s augur pipeline.<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31964463">31964463</a> <b>(15)</b> Co-circulation of multiple enterovirus D68 subclades, including a novel B3 cluster, across Europe in a season of expected low prevalence, 2019/20<ul><li>Phylogenetic analysis of VP1 sequence data was carried out using the <b>Nextstrain</b> augur pipeline [<xref rid="r13" ref-type="bibr"><u>13</u></xref>].<li> the analysis can be viewed at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/evd68-2019/vp1-300">https://<b>nextstrain</b>.org/community/enterovirus-phylo/evd68-2019/vp1-300</ext-link>.<li>Phylogenetic analysis of enterovirus D68 with <b>Nextstrain</b> using partial VP1 sequences (n&#x2009;=&#x2009;1,693).<li> These formed a distinct cluster within clade B3 (EU19 in <xref ref-type="fig" rid="f2"><u>Figures 2</u></xref> and <xref ref-type="fig" rid="f3"><u>3</u></xref>), which did not include previously detected B3 strains from these countries and was not well represented in the 2016 or 2018 epidemic (see <b>Nextstrain</b> analysis, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/evd68-2019/vp1-300">https://<b>nextstrain</b>.org/community/enterovirus-phylo/evd68-2019/vp1-300</ext-link>).<li> Four German strains clustered within this group in the VP4/VP2 region (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/evd68-2019/vp4vp2">https://<b>nextstrain</b>.org/community/enterovirus-phylo/evd68-2019/vp4vp2</ext-link>).<li>Phylogenetic analysis of three clusters of partial VP1 sequences of enterovirus D68 strains using <b>Nextstrain</b>, five European countries, 1 January&#x2013;6 December 2019 (n&#x2009;=&#x2009;36).<li> Five and 21 sequences, respectively, were long enough to be included in the <b>Nextstrain</b> full-length (&#x2265;&#x2009;6,000&#x202F;bp) and VP1 (&#x2265;&#x2009;700&#x202F;bp) builds; they can be viewed at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/enterovirus/d68/genome">https://<b>nextstrain</b>.org/enterovirus/d68/genome</ext-link> and <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/enterovirus/d68/vp1">https://<b>nextstrain</b>.org/enterovirus/d68/vp1</ext-link>.<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32055407">32055407</a> <b>(2)</b> Evolutionary genomics can improve prediction of species&#x2019; responses to climate change<ul><li> A successful example that has pioneered large&#x2010;scale community&#x2010;based data collection comes from human&#x2010;pathogenic organisms as Ebola, Flu, or Zika, where researchers have shared significant amounts of data during critical outbreaks and analyses were generated and published in almost real time (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org">http://<b>nextstrain</b>.org</ext-link>).</ul></p>
</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31856206">31856206</a> <b>(12)</b> Childhood immune imprinting to influenza A shapes birth year-specific risk during seasonal H1N1 and H3N2 epidemics<ul><li>We used publicly available data from <b>Nextstrain</b> [<xref rid="ppat.1008109.ref038" ref-type="bibr"><u>38</u></xref>,<xref rid="ppat.1008109.ref039" ref-type="bibr"><u>39</u></xref>], and from one previously published study [<xref rid="ppat.1008109.ref040" ref-type="bibr"><u>40</u></xref>], to calculate annual antigenic advance, which we defined as the mean antigenic distance between strains of a given lineage (pre-2009 H1N1, post-2009 H1N1 or H3N2) that circulated in consecutive seasons (<bold>Methods</bold>).<li>We obtained antigenic distance estimates from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org"><b>nextstrain</b>.org</ext-link>) [<xref rid="ppat.1008109.ref038" ref-type="bibr"><u>38</u></xref>,<xref rid="ppat.1008109.ref056" ref-type="bibr"><u>56</u></xref>], and from source data from Fig 3 in Bedford et al.<li> <b>Nextstrain</b> calculates antigenic distance using genetic data from GISAID [<xref rid="ppat.1008109.ref057" ref-type="bibr"><u>57</u></xref>], and using methods described by Neher et al.<li> estimates from <b>Nextstrain</b>, which correspond to Neher et al.&#x2019;s tree model method, and are most directly comparable to pre-2009 H1N1 estimates from [<xref rid="ppat.1008109.ref040" ref-type="bibr"><u>40</u></xref>].<li> Datasets from <b>Nextstrain</b> and Bedford et al.<li> analyzed the pre-2009 lineage, and only <b>Nextstrain</b> data analyzed the post-2009 lineage.<li> were roughly proportional to those reported on <b>Nextstrain</b>, but greater in absolute magnitude [<xref rid="ppat.1008109.ref039" ref-type="bibr"><u>39</u></xref>].<li> using the formula d<sub><b>Nextstrain</b></sub> = 0.47d<sub>Bedford</sub>.<li> The <b>Nextstrain</b> data files used in this analysis are archived within our analysis code.<li>Comparison of rescaled antigenic distance estimates from the Bedford et al., and <b>Nextstrain</b> datasets.<li> [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26951657">26951657</a>) [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24497547">24497547</a>) [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [56](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [57](<name><surname>Bogner</surname><given-names>P</given-names></name>, <name><surname>Capua</surname><given-names>I</given-names></name>, <name><surname>Lipman</surname><given-names>DJ</given-names></name>, <name><surname>Cox</surname><given-names>NJ</given-names></name>. <article-title>A global initiative on sharing avian flu data</article-title>. <source>Nature</source>. <year>2006</year>;<volume>442</volume>: <fpage>981</fpage><pub-id pub-id-type="doi">10.1038/442981a</pub-id>) [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24497547">24497547</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26951657">26951657</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31976168">31976168</a> <b>(2)</b> Evaluating probabilistic programming and fast variational Bayesian inference in phylogenetics<ul><li> In particular, systems such as <b>Nextstrain</b> (<xref rid="ref-18" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) provide a fast way to incorporate data into a visualization environment that enables exploration of phylogeny and other key aspects such as sample collection time and geographical location.<li> <b>Nextstrain</b> computes a phylogeny for input sequences using maximum likelihood-based methods.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31809512">31809512</a> <b>(3)</b> Multiscale analysis for patterns of Zika virus genotype emergence, spread, and consequence<ul><li>Analysis of ZIKV genomes show distinct patterns of emergence of nonsynonymous mutations in the years prior to the epidemic, as has been previously described [<xref rid="pone.0225699.ref006" ref-type="bibr"><u>6</u></xref>,<xref rid="pone.0225699.ref029" ref-type="bibr"><u>29</u></xref>,<xref rid="pone.0225699.ref030" ref-type="bibr"><u>30</u></xref>], and superbly documented in the <b>nextstrain</b>.org Zika website (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/zika">https://<b>nextstrain</b>.org/zika</ext-link>) [<xref rid="pone.0225699.ref031" ref-type="bibr"><u>31</u></xref>].<li> [6](<name><surname>Wang</surname><given-names>L</given-names></name>, <name><surname>Valderramos Stephanie</surname><given-names>G</given-names></name>, <name><surname>Wu</surname><given-names>A</given-names></name>, <name><surname>Ouyang</surname><given-names>S</given-names></name>, <name><surname>Li</surname><given-names>C</given-names></name>, <name><surname>Brasil</surname><given-names>P</given-names></name>, <etal>et al</etal> (<year>2016</year>) <article-title>From Mosquitos to Humans: Genetic Evolution of Zika Virus</article-title>. <source>Cell Host &amp; Microbe</source>.) [29](<name><surname>Faria</surname><given-names>NR</given-names></name>, <name><surname>Azevedo</surname><given-names>RdSdS</given-names></name>, <name><surname>Kraemer</surname><given-names>MUG</given-names></name>, <name><surname>Souza</surname><given-names>R</given-names></name>, <name><surname>Cunha</surname><given-names>MS</given-names></name>, <name><surname>Hill</surname><given-names>SC</given-names></name>, <etal>et al</etal> (<year>2016</year>) <article-title>Zika virus in the Americas: Early epidemiological and genetic findings</article-title>. <source>Science</source>.) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26980239">26980239</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
<p><u>No PMID</u> <b>(1)</b> Innovative Technologies for Advancement of WHO Risk Group 4 Pathogens Research<ul><li> Smaller data portals like virological.org and <b>nextstrain</b>.org [<xref ref-type="bibr" rid="CR175"><u>175</u></xref>] also exist and can rapidly disseminate pre-publication data.<li> [175](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31718580">31718580</a> <b>(14)</b> Genomic epidemiology supports multiple introductions and cryptic transmission of Zika virus in Colombia<ul><li>Data were cleaned and canonicalized using <b>Nextstrain</b> Fauna (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/fauna">https://github.com/<b>nextstrain</b>/fauna</ext-link>), a databasing tool that enforces a schema for organizing sequence data and sample metadata, thereby creating datasets compatible with the <b>Nextstrain</b> Augur analytic pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/augur">https://github.com/<b>nextstrain</b>/augur</ext-link>) and the <b>Nextstrain</b> Auspice visualization platform (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/auspice">https://github.com/<b>nextstrain</b>/auspice</ext-link>).<li> A full description of the <b>Nextstrain</b> pipelines can be found in Hadfield et al.<li> Briefly, <b>Nextstrain</b> Augur performs a multiple sequence alignment with MAFFT [<xref ref-type="bibr" rid="CR32"><u>32</u></xref>], which is then trimmed to the reference sequence.<li> These data are exported as JSON files that can be interactively visualized on the web using <b>Nextstrain</b> Auspice.<li> The full tree can be explored interactively at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/community/blab/zika-colombia">https://<b>nextstrain</b>.org/community/blab/zika-colombia</ext-link>.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31720122">31720122</a> <b>(1)</b> Genomics of host-pathogen interactions: challenges and opportunities across ecological and spatiotemporal scales<ul><li> In addition, open source genomic data-sharing sites and analysis platforms like <b>nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org/">http://www.<b>nextstrain</b>.org/</ext-link>) are invaluable to explore pathogen time-space variation in real-time.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31671157">31671157</a> <b>(88)</b> Twenty years of West Nile virus spread and evolution in the Americas visualized by <b>Nextstrain</b><ul><li> To fill this need, <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>nextstrain</b>.org</ext-link>) [<xref rid="ppat.1008042.ref030" ref-type="bibr"><u>30</u></xref>] was created to provide everyone the ability to explore pathogen spread and evolution using genomic epidemiology.<li> In this review, we use <b>Nextstrain</b> to uncover key epidemiological and evolutionary processes during the emergence and establishment of WNV in the Americas (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA"><b>nextstrain</b>.org/WNV/NA</ext-link>) and discuss collaborative opportunities to advance this work for future outbreak control.<li>Visualizing spread and evolution using <b>Nextstrain</b>.<li><b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/"><b>nextstrain</b>.org</ext-link>) is an open-source initiative to use pathogen genome data to provide a real-time interactive view of the spread and evolution of significant human pathogens [<xref rid="ppat.1008042.ref030" ref-type="bibr"><u>30</u></xref>].<li> Thus, <b>Nextstrain</b> was created to provide a publicly accessible and up-to-date overview of pathogen spread and evolution during outbreaks, facilitated through collaborations with subject matter experts.<li> Alongside platforms such as Virological (<ext-link ext-link-type="uri" xlink:href="http://virological.org/">virological.org</ext-link>), <b>Nextstrain</b> has become an important venue to quickly disseminate prepublication results, which is especially critical when the information can lead to public health interventions or improved public understanding [<xref rid="ppat.1008042.ref034" ref-type="bibr"><u>34</u></xref>].<li><b>Nextstrain</b> is powered by a collection of open-source bioinformatic tools to curate, analyze, and visualize available virus genomes [<xref rid="ppat.1008042.ref030" ref-type="bibr"><u>30</u></xref>].<li> To reduce runtimes, <b>Nextstrain</b> leverages TreeTime for the maximum likelihood phylogenetic analysis [<xref rid="ppat.1008042.ref035" ref-type="bibr"><u>35</u></xref>], through which the entire WNV dataset currently presented on <b>Nextstrain</b> (<italic>n</italic> = 2,267 genomes as of June 2019) can be analyzed in under an hour on a modern computer.<li>We created a WNV <b>Nextstrain</b> resource in 2018 to convey the history of WNV in the Americas and promote new research directions (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA"><b>nextstrain</b>.org/WNV/NA</ext-link>; <bold><xref ref-type="supplementary-material" rid="ppat.1008042.s001"><u>S1 Fig</u></xref></bold>).<li> All of the figures herein were created using <b>Nextstrain</b> and contain live display links in each legend to allow the reader to explore the data.<li> The analysis presented on <b>Nextstrain</b> can be recreated using our build pipeline found at <ext-link ext-link-type="uri" xlink:href="https://github.com/grubaughlab/WNV-<b>nextstrain</b>">github.com/grubaughlab/WNV-<b>nextstrain</b></ext-link>, which can be used to analyze any WNV dataset offline.<li> Additionally, we designed a new narrative function in <b>Nextstrain</b> to accompany this review&#x2014;accessible at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/narratives/twenty-years-of-WNV"><b>nextstrain</b>.org/narratives/twenty-years-of-WNV</ext-link>&#x2014;which walks users through the epidemiological findings discussed here in a more condensed and interactive format.<li> Mirroring the dynamic nature of such research, <b>Nextstrain</b> is constantly updated as new WNV genomes are shared publicly.<li> Data from specific times can be visualized on <b>Nextstrain</b> using the &#x201C;Date Range&#x201D; function.<li> A live display can be found at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=tree,map&amp;dmax=1998-12-01&amp;p=grid"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=tree,map&amp;dmax=1998-12-01&amp;p=grid</ext-link>.<li>By reconstructing the ancestral traits using TreeTime, <b>Nextstrain</b> infers state of all internal nodes of the phylogenetic tree to map the pattern of WNV spread.<li> Data from specific times can be visualized on <b>Nextstrain</b> using the &#x201C;Date Range&#x201D; function.<li> Live displays can be found at (<bold>A</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=1999-12-31&amp;f_country=USA"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=1999-12-31&amp;f_country=USA</ext-link>, (<bold>B</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2000-12-31&amp;f_country=USA"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2000-12-31&amp;f_country=USA</ext-link>, (<bold>C</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2001-12-31&amp;f_country=USA"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2001-12-31&amp;f_country=USA</ext-link>, (<bold>D</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2002-12-31&amp;f_country=USA"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2002-12-31&amp;f_country=USA</ext-link>.<li> Data from specific genotypes can be visualized on <b>Nextstrain</b> using the &#x201C;Color By: Strain&#x201D; and &#x201C;Filter by Lineage&#x201D; functions.<li> Live displays can be found at (<bold>A</bold>) <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;d=tree&amp;f_country=USA"><b>nextstrain</b>.org/WNV/NA?c=lineage&amp;d=tree&amp;f_country=USA</ext-link>, (<bold>B</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;d=map&amp;f_country=USA&amp;f_lineage=NY99"><b>nextstrain</b>.org/WNV/NA?c=lineage&amp;d=map&amp;f_country=USA&amp;f_lineage=NY99</ext-link>, (<bold>C</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;d=map&amp;f_country=USA&amp;f_lineage=WN02"><b>nextstrain</b>.org/WNV/NA?c=lineage&amp;d=map&amp;f_country=USA&amp;f_lineage=WN02</ext-link>, (<bold>D</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;d=map&amp;f_country=USA&amp;f_lineage=SW03"><b>nextstrain</b>.org/WNV/NA?c=lineage&amp;d=map&amp;f_country=USA&amp;f_lineage=SW03</ext-link>.<li> Data from specific countries can be visualized on <b>Nextstrain</b> using the &#x201C;Color By: Country&#x201D; and &#x201C;Filter by Country&#x201D; functions.<li> A live display can be found at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=country&amp;f_country=Argentina,Brazil,British-Virgin-Islands,Colombia,Mexico&amp;p=grid"><b>nextstrain</b>.org/WNV/NA?c=country&amp;f_country=Argentina,Brazil,British-Virgin-Islands,Colombia,Mexico&amp;p=grid</ext-link>.<li> Platforms such as <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA"><b>nextstrain</b>.org/WNV/NA</ext-link>) and Virological (<ext-link ext-link-type="uri" xlink:href="http://virological.org/">virological.org</ext-link>) allow these new data to be continually communicated to the public.<li> Data from specific times can be visualized on <b>Nextstrain</b> using the &#x201C;Date Range&#x201D; function.<li> Live displays of the data can be found at (<bold>A</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2004-12-31&amp;f_country=USA&amp;r=division"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2004-12-31&amp;f_country=USA&amp;r=division</ext-link>, (<bold>B</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2009-12-31&amp;dmin=2005-01-01&amp;f_country=USA&amp;r=division"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2009-12-31&amp;dmin=2005-01-01&amp;f_country=USA&amp;r=division</ext-link>, (<bold>C</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2014-12-31&amp;dmin=2010-01-01&amp;f_country=USA&amp;r=division"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2014-12-31&amp;dmin=2010-01-01&amp;f_country=USA&amp;r=division</ext-link>, (<bold>D</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2019-12-31&amp;dmin=2015-01-01&amp;f_country=USA&amp;r=division"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=map&amp;dmax=2019-12-31&amp;dmin=2015-01-01&amp;f_country=USA&amp;r=division</ext-link>.<li><b>Nextstrain</b> as a tool to visualize WNV evolution and spread.<li>A depiction of the main interactive interface on <b>Nextstrain</b> to visualize the virus (<bold>A</bold>) phylogeny (time-resolved phylogenetic tree created with TreeTime, states listed by two-letter abbreviations, countries listed by three-letter abbreviations), (<bold>B</bold>) spread (inferred patterns of spread based on the tree, circle size is relative to the number of genomes that remained local), and (<bold>C</bold>) genetic diversity (amino acid diversity plotted as entropy and positioned by codon sequence).<li> A live display, which can be used to select and zoom in on various features, can be found at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?p=grid"><b>nextstrain</b>.org/WNV/NA?p=grid</ext-link>.<li> tree view can be visualized on <b>Nextstrain</b> by toggling between the &#x201C;Tree Options: Layout&#x201D;.<li> A live display can be found at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?l=unrooted&amp;m=div&amp;p=full"><b>nextstrain</b>.org/WNV/NA?l=unrooted&amp;m=div&amp;p=full</ext-link>.<li> view can be visualized on <b>Nextstrain</b> by toggling between the &#x201C;Tree Options: Layout.&#x201D; A live display can be found at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=tree&amp;l=clock"><b>nextstrain</b>.org/WNV/NA?c=num_date&amp;d=tree&amp;l=clock</ext-link>.<li> All other alleles (nucleotide and amino acid changes) can be visualized using <b>Nextstrain</b> by entering the loci using the &#x201C;Color By: genotype&#x201D; function or by selecting a loci on the &#x201C;Diversity&#x201D; plot (i.e., <bold><xref ref-type="supplementary-material" rid="ppat.1008042.s001"><u>S1C Fig</u></xref></bold>).<li> Live displays can be found at (<bold>A</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=gt-NS4A_85"><b>nextstrain</b>.org/WNV/NA?c=gt-NS4A_85</ext-link> and (<bold>B</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=gt-NS5_314"><b>nextstrain</b>.org/WNV/NA?c=gt-NS5_314</ext-link>.<li> Data from other studies can be visualized on <b>Nextstrain</b> by using the &#x201C;Filter by Authors&#x201D; function.<li> Live displays can be found at (<bold>A</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;f_authors=Hepp%20et%20al">https://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;f_authors=Hepp%20et%20al</ext-link>, (<bold>B</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;f_authors=Duggal%20et%20al">https://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;f_authors=Duggal%20et%20al</ext-link>, and (<bold>C</bold>) <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;f_authors=Shabman%20et%20al">https://<b>nextstrain</b>.org/WNV/NA?c=lineage&amp;f_authors=Shabman%20et%20al</ext-link>.<li> Data from other states can be visualized on <b>Nextstrain</b> by using the &#x201C;Filter by State&#x201D; function.<li> A live display can be found at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/WNV/NA?f_state=NY&amp;d=tree"><b>nextstrain</b>.org/WNV/NA?f_state=NY&amp;d=tree</ext-link>.<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29129921">29129921</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31635227">31635227</a> <b>(4)</b> Antigenic Site Variation in the Hemagglutinin of Pandemic Influenza A(H1N1)pdm09 Viruses between 2009&#x2013;2017 in Ukraine<ul><li> Since the 2015&#x2013;2016 season, the vast majority of the viruses worldwide has become part of the 6B.1 subgroup according to <b>Nextstrain</b> data [<xref rid="B18-pathogens-08-00194" ref-type="bibr"><u>18</u></xref>].<li> Spread of A(H1N1)pdm09 influenza virus of genetic groups 6B.1 and 6B.2 was taken from <b>Nextstrain</b> <uri xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</uri> [<xref rid="B18-pathogens-08-00194" ref-type="bibr"><u>18</u></xref>,<xref rid="B19-pathogens-08-00194" ref-type="bibr"><u>19</u></xref>].<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31487952">31487952</a> <b>(1)</b> Understanding Enterovirus D68-Induced Neurologic Disease: A Basic Science Review<ul><li> An updated EV-D68 phylogenetic tree, including the time and location of strain isolations, can be found at <b>nextstrain</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31471545">31471545</a> <b>(13)</b> Evaluating the use of whole genome sequencing for the investigation of a large mumps outbreak in Ontario, Canada<ul><li>Comparison of our strains with genotype G mumps virus circulating in North America was facilitated by the <b>Nextstrain</b> project (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/mumps/global">https://<b>nextstrain</b>.org/mumps/global</ext-link>), which includes sequences that are not publically available in GenBank.<li> <b>Nextstrain</b> analysis estimated the date of the most recent common ancestor for each of our major clades, using an ML discrete traits model.<li>Phylogenetic tree of mumps isolates reproduced from <b>nextstrain</b>.org.<li> The differences in estimates for tMRCA from <b>Nextstrain</b> (spring 2016) and Bayesian analysis are likely due to the different models used, and the greater diversity of samples in the <b>Nextstrain</b> dataset.<li> Although the Bayesian model is a better fit than the <b>Nextstrain</b> model for rapid outbreak growth dynamics, the <b>Nextstrain</b> model has the advantage of using other strains to refine the date estimates.<li> Results of <b>Nextstrain</b>, ML and Bayesian analyses support the hypothesis that the two major clades were introduced by separate importation events in late 2016 or early 2017.<li> We uploaded our outbreak clade genomes to the <b>Nextstrain</b> project (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31452511">31452511</a> <b>(4)</b> Mapping person-to-person variation in viral mutations that escape polyclonal serum targeting influenza hemagglutinin<ul><li> The figure panels are taken from the <b>Nextstrain</b> real-time pathogen evolution website (<xref rid="bib31" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bib60" ref-type="bibr"><u>Neher and Bedford, 2015</u></xref>).<li> For these nine sites, we then took images of the amino-acid frequencies over time from the <b>Nextstrain</b> website (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) (<xref rid="bib31" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bib60" ref-type="bibr"><u>Neher and Bedford, 2015</u></xref>) and used them to create <xref ref-type="fig" rid="fig8"><u>Figure 8</u></xref>.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Neher and Bedford, 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Neher and Bedford, 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31431210">31431210</a> <b>(2)</b> Intense interseasonal influenza outbreaks, Australia, 2018/19<ul><li> For influenza A(H1N1)pdm09, the predominant clade was 6B1A 183P-5 (<b>Nextstrain</b> nomenclature [<xref rid="r16" ref-type="bibr"><u>16</u></xref>]), which accounted for 201/278 (72.3%) of A(H1N1)pdm09 viruses sequenced by the WHO CCRRI during this period.<li> This clade has been designated as 3C.2a1b based on the HA gene phylogeny (<b>Nextstrain</b> nomenclature [<xref rid="r16" ref-type="bibr"><u>16</u></xref>]) with a distinctive T131K change (i.e.<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31405370">31405370</a> <b>(1)</b> MARPLE, a point-of-care, strain-level disease diagnostics and surveillance tool for complex fungal pathogens<ul><li> By focusing on generating data from a small subset of genes, our approach is also relatively inexpensive and generates small, unified datasets that can then be readily explored using analytic and visualisation tools created for smaller bacterial and viral genomic datasets such as <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR28"><u>28</u></xref>].<li> [28](Bedford T, Neher R, Hadfield J, Hodcroft E, Sibley T, Huddleston J, et al. Nextstrain. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org">https://nextstrain.org</ext-link>.) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31385805">31385805</a> <b>(5)</b> Dengue genetic divergence generates within-serotype antigenic variation, but serotypes dominate evolutionary dynamics<ul><li>Dengue virus genotypes can be seen on <b>Nextstrain</b> (<xref rid="bib14" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> A live view of this figure is available at <ext-link xlink:href="https://<b>nextstrain</b>.org/dengue/" ext-link-type="uri"><b>nextstrain</b>.org/dengue/</ext-link>.<li> (2016)</xref> and implemented in <b>Nextstrain</b> (<xref rid="bib14" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), which assumes that antigenic evolution is driven by underlying genetic evolution.<li> <xref rid="bib9" ref-type="bibr"><u>Gao and Han, 2012</u></xref>), Biopython (<xref rid="bib6" ref-type="bibr"><u>Cock et al., 2009</u></xref>), SciPy (<xref rid="bib18" ref-type="bibr"><u>Jones et al., 2001</u></xref>), Statsmodels (<xref rid="bib48" ref-type="bibr"><u>Seabold and Perktold, 2010</u></xref>), <b>Nextstrain</b> (<xref rid="bib14" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), MAFFT (<xref rid="bib20" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>), and IQ-TREE (<xref rid="bib36" ref-type="bibr"><u>Nguyen et al., 2015</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Gao and Han, 2012](          <person-group person-group-type="author"><name><surname>Gao</surname><given-names>F</given-names></name><name><surname>Han</surname><given-names>L</given-names></name></person-group>          <year iso-8601-date="2012">2012</year>          <article-title>Implementing the Nelder-Mead simplex algorithm with&#xA0;adaptive parameters</article-title>          <source>Computational Optimization and Applications</source>          <volume>51</volume>          <fpage>259</fpage>          <lpage>277</lpage>          <pub-id pub-id-type="doi">10.1007/s10589-010-9329-3</pub-id>        ) [Cock et al., 2009](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19304878">19304878</a>) [Jones et al., 2001](          <person-group person-group-type="author"><name><surname>Jones</surname><given-names>E</given-names></name><name><surname>Oliphant</surname><given-names>T</given-names></name><name><surname>Peterson</surname><given-names>P</given-names></name></person-group>          <year iso-8601-date="2001">2001</year>          <data-title>SciPy: Open source scientific tools for Python</data-title>          <ext-link xlink:href="http://www.scipy.org/" ext-link-type="uri">http://www.scipy.org/</ext-link>        ) [Seabold and Perktold, 2010](          <person-group person-group-type="author"><name><surname>Seabold</surname><given-names>S</given-names></name><name><surname>Perktold</surname><given-names>J</given-names></name></person-group>          <year iso-8601-date="2010">2010</year>          <article-title>Statsmodels: econometric and statistical modeling with Python</article-title>          <conf-name>9th Python in Science Conference</conf-name>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Nguyen et al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a> <b>(1)</b> Dengue genetic divergence generates within-serotype antigenic variation, but serotypes dominate evolutionary dynamics<ul><li> Along with this, could you add a table that contains, for every branch that carries a non-zero db value, the list of amino acid changes that were inferred on that branch? This should be easily do-able using <b>Nextstrain</b>, and it would be informative.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a> <b>(1)</b> Dengue genetic divergence generates within-serotype antigenic variation, but serotypes dominate evolutionary dynamics<ul><li> Along with this, could you add a table that contains, for every branch that carries a non-zero db value, the list of amino acid changes that were inferred on that branch? This should be easily do-able using <b>Nextstrain</b>, and it would be informative.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31381759">31381759</a> <b>(1)</b> Global patterns of avian influenza A (H7): virus evolution and zoonotic threats<ul><li> Real-time epidemiological and sequence data were rapidly fed into models to demonstrate transmission patterns and virus evolution, and platforms such as <b>Nextstrain</b> (Hadfield <italic>et&#xA0;al</italic>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31428595">31428595</a> <b>(10)</b> Advances in Visualization Tools for Phylogenomic and Phylodynamic Studies of Viral Diseases<ul><li> (<xref rid="B46" ref-type="bibr"><u>46</u></xref>) and <b>Nextstrain</b>; (<xref rid="B19" ref-type="bibr"><u>19</u></xref>, <xref rid="B47" ref-type="bibr"><u>47</u></xref>), also see <bold>Figure 5</bold>] that seamlessly connect the different steps of increasingly complex analyses and visualization of genomic epidemiology and phylodynamics allows automating this process.<li>Focusing on real-time tracking of several viruses, <b>Nextstrain</b> (<xref rid="B19" ref-type="bibr"><u>19</u></xref>) provides up-to-date visualizations of phylogenetic and phylogeographic analyses, the latter with an animation over time similar to SpreaD3 (<xref rid="B40" ref-type="bibr"><u>40</u></xref>).<li> Recently, entire pipelines have emerged that include data curation, analysis and visualization components, with <b>Nextstrain</b> as its most popular example (<xref rid="B19" ref-type="bibr"><u>19</u></xref>).<li> The accompanying <b>Nextstrain</b> website (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) provides a continually-updated view of publicly available data alongside visualization for a number of pathogens such as West Nile virus, Ebola virus, and Zika virus.<li> For the latter virus, we provide the currently available data visualization in <b>Nextstrain</b> (at time of submission) in <xref ref-type="fig" rid="F5"><u>Figure 5</u></xref>, showing a color-coded time-scaled maximum-likelihood tree alongside an animation of Zika geographic transmissions over time as well as the genetic diversity across the genome.<li> Analysis of such outbreaks relies on public sharing of data, and <b>Nextstrain</b> has taken the lead to address data sharing concerns by preventing access to the raw genome sequences, and by clearly indicating the source of each sequence, while allowing derived data&#x2014;such as the inferred phylogenetic trees&#x2014;to be made publicly available.<li> Animations resulting from software packages such as SPREAD, SpreaD3 and <b>Nextstrain</b> can be hosted on the authors' website or they can be captured into a video file format and uploaded as supplementary materials onto the journal website.<li> One-stop and fully-integrated analysis platforms such as MicroReact (<xref rid="B46" ref-type="bibr"><u>46</u></xref>) and <b>Nextstrain</b> (<xref rid="B19" ref-type="bibr"><u>19</u></xref>) adhere to these needs by providing the necessary visualizations of virus epidemiology and evolution across time and space, and by implementing support for collaborative analyses and sharing of genomic data and analysis outputs.<li> [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28348833">28348833</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [47](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27189542">27189542</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28348833">28348833</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31308446">31308446</a> <b>(1)</b> Predicting Antigenicity of Influenza A Viruses Using biophysical ideas<ul><li> developed an online application called <italic><b>Nextstrain</b></italic> for real-time tracking and visualization of the evolution of influenza and other epidemiological viruses such as Zika and Ebola<sup><xref ref-type="bibr" rid="CR21"><u>21</u></xref></sup>.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
</ul></p>
<p><u>No PMID</u> <b>(1)</b> ZooPhy: A bioinformatics pipeline for virus phylogeography and surveillance<ul><li> One example is <b>NextStrain</b> whichcan perform real-time virus phylodynamics [<xref rid="r10" ref-type="bibr"><u>10</u></xref>].<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31104603">31104603</a> <b>(1)</b> Outbreak analytics: a developing data science for informing the response to emerging pathogens<ul><li> accessed 19 September 2018), which maps risks of emergence and spread of VHF diseases, <b>Nextstrain</b> [<xref rid="RSTB20180276C92" ref-type="bibr"><u>92</u></xref>] and Microreact [<xref rid="RSTB20180276C93" ref-type="bibr"><u>93</u></xref>], which focus on mapping pathogen evolution and epidemic spread, and HealthMap [<xref rid="RSTB20180276C94" ref-type="bibr"><u>94</u></xref>], which provides resources for the rapid detection of outbreaks.<li> [92](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [93](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28348833">28348833</a>) [94](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18096908">18096908</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31038123">31038123</a> <b>(1)</b> Comprehensive mapping of adaptation of the avian influenza polymerase protein PB2 to humans<ul><li>The phylogenetic tree was generated using <b>Nextstrain</b>&#x2019;s augur pipeline (<xref rid="bib36" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), and ancestral state reconstruction and adjustment of branch lengths according to sequence isolation date were performed with TreeTime (<xref rid="bib73" ref-type="bibr"><u>Sagulenko et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Sagulenko et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31037220">31037220</a> <b>(17)</b> Intra- and interpatient evolution of enterovirus D68 analyzed by whole-genome deep sequencing<ul><li> To facilitate exploration of the data and future analyses, we combined the whole-genome sequences reported here with publicly available genomes and implemented an interactive visualization in <b>nextstrain</b> available at <b>nextstrain</b>.org/enterovirus/d68.<li> The combined dataset was analyzed using <b>nextstrain</b> (<xref rid="vez007-B11" ref-type="bibr">Hadfield <italic>et al.<li> The repository detailing the analysis pipeline is available at <ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/enterovirus_<b>nextstrain</b>">https://github.com/neherlab/enterovirus_<b>nextstrain</b></ext-link>.<li> The resulting analysis is visualized with auspice and is available at <b>nextstrain</b>.org/enterovirus/d68.<li>The interpatient evolution and phylogenetic relationship of the EV-D68 variants from Sweden and 509 published EV-D68 sequences was investigated using the <b>nextstrain</b> platform available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/enterovirus/d68">https://<b>nextstrain</b>.org/enterovirus/d68</ext-link> (<xref rid="vez007-B11" ref-type="bibr">Hadfield <italic>et al.<li> The left panel shows a phylogenetic tree as rendered by <b>nextstrain</b> with the sequences labeled by region.<li>The <b>nextstrain</b> phylodynamic analysis showed that all Swedish EV-D68 variants from the 2016 outbreak belonged to the B3 lineage of EV-D68, which agrees with our earlier findings based on VP2/VP4 gene sequences (<xref rid="vez007-B5" ref-type="bibr">Dyrdak <italic>et al.<li>We used <b>nextstrain</b> to investigate diversity across the EV-D68 genome in the sample containing the genomes reported in this study and database genomes.<li> Amino acid substitution events on the phylogeny in <xref ref-type="fig" rid="vez007-F6"><u>Fig.&#xA0;6</u></xref> left were enumerated using <b>nextstrain</b>.<li>The BC and DE loops in the VP1 protein showed relatively high diversity, especially in positions 95, 97, 98, 142, 143, and 148, suggesting that immune selection may have impacted on EV-D68 evolution (see <xref ref-type="fig" rid="vez007-F8"><u>Fig.&#xA0;8</u></xref>, bottom panel, and for details see <b>nextstrain</b>).<li>We also used <b>nextstrain</b> to track on which branches in the EV-D68 phylogeny amino acid substitutions had evolved and been fixated.<li> The new data and previously published sequences are available on <b>nextstrain</b> to facilitate real-time tracking and further study of EV-D68 evolution and spread.<li> We hope that the <b>nextstrain</b> platform will allow for continuous collection and sharing of EV-D68 sequence data, which will be needed to obtain a better understanding of how new successful EV-D68 variants evolve and spread.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30940704">30940704</a> <b>(1)</b> Human Influenza A Virus Hemagglutinin Glycan Evolution Follows a Temporal Pattern to a Glycan Limit<ul><li> The figure was produced using <b>nextstrain</b>.org (previously nextflu) (<xref rid="B53" ref-type="bibr"><u>53</u></xref>, <xref rid="B54" ref-type="bibr"><u>54</u></xref>).<li> [53](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) [54](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30818354">30818354</a> <b>(3)</b> GenomeGraphR: A user-friendly open-source web application for foodborne pathogen whole genome sequencing data integration, analysis, and visualization<ul><li> The purposes of the currently available applications, such as Microreact (<ext-link ext-link-type="uri" xlink:href="https://microreact.org">https://microreact.org</ext-link>) [<xref rid="pone.0213039.ref024" ref-type="bibr"><u>24</u></xref>], <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>) [<xref rid="pone.0213039.ref025" ref-type="bibr"><u>25</u></xref>] or tools available within the Integrated Rapid Infectious Disease Analysis (IRIDA) project platform (<ext-link ext-link-type="uri" xlink:href="http://www.irida.ca">www.irida.ca</ext-link>) are mainly to aid epidemiological understanding of non-foodborne pathogen spread and evolution, especially in outbreak situations.<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28348833">28348833</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30755292">30755292</a> <b>(2)</b> Genetic and antigenic characterisation of influenza A(H3N2) viruses isolated in Yokohama during the 2016/17 and 2017/18 influenza seasons<ul><li> During the 2018 southern hemisphere influenza season, the clade 3C.2A1b and 3C.2A2 viruses continued to dominate (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/flu/seasonal/h3n2/ha/3y">https://<b>nextstrain</b>.org/flu/seasonal/h3n2/ha/3y</ext-link>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30696523">30696523</a> <b>(3)</b> Interim estimates of 2018/19 vaccine effectiveness against influenza A(H1N1)pdm09, Canada, January 2019<ul><li><b>Nextstrain</b> subgroup<break/>[<xref rid="r8" ref-type="bibr"><u>8</u></xref>]</th>              <th rowspan="2" valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" colspan="1">Genetic clade<sup>a</sup> with subgroup substitutions<sup>b</sup></th>              <th valign="middle" colspan="2" align="center" scope="colgroup" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1">British Columbia<sup>c</sup></th>              <th valign="middle" colspan="2" align="center" scope="colgroup" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1">Alberta<sup>d</sup></th>              <th valign="middle" colspan="2" align="center" scope="colgroup" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1">Ontario<sup>e</sup></th>              <th valign="middle" colspan="2" align="center" scope="colgroup" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1">Quebec<sup>f</sup></th>              <th valign="middle" colspan="2" align="center" scope="colgroup" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1">Overall<sup>g</sup></th>            </tr>            <tr>              <th valign="middle" colspan="1" align="center" scope="colgroup" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1">n</th>              <th valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1" colspan="1">%</th>              <th valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1" colspan="1">n</th>              <th valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1" colspan="1">%</th>              <th valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1" colspan="1">n</th>              <th valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1" colspan="1">%</th>              <th valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1" colspan="1">n</th>              <th valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1" colspan="1">%</th>              <th valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1" colspan="1">n</th>              <th valign="middle" align="center" scope="col" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" rowspan="1" colspan="1">%</th>            </tr>          </thead>          <tbody>            <tr>              <td rowspan="2" valign="middle" align="left" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" scope="row" colspan="1">6b1.A.<li> Genetic variants displayed here have been aligned with <b>nextstrain</b> subgrouping [<xref rid="r8" ref-type="bibr"><u>8</u></xref>], recognising differences in numbering approaches (HA1 and HA2).<li> Genetic variants displayed here have been aligned with <b>nextstrain</b> subgrouping [<xref rid="r8" ref-type="bibr"><u>8</u></xref>], recognising differences in numbering approaches (HA1 and HA2).<li> [8](Nextstrain. Real-time tracking of influenza A/H1N1pdm evolution 2018. [Accessed: 16 Jan 2019]. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/flu/seasonal/h1n1pdm/ha/3y">https://nextstrain.org/flu/seasonal/h1n1pdm/ha/3y</ext-link>) [8](Nextstrain. Real-time tracking of influenza A/H1N1pdm evolution 2018. [Accessed: 16 Jan 2019]. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/flu/seasonal/h1n1pdm/ha/3y">https://nextstrain.org/flu/seasonal/h1n1pdm/ha/3y</ext-link>) [8](Nextstrain. Real-time tracking of influenza A/H1N1pdm evolution 2018. [Accessed: 16 Jan 2019]. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/flu/seasonal/h1n1pdm/ha/3y">https://nextstrain.org/flu/seasonal/h1n1pdm/ha/3y</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30585228">30585228</a> <b>(0)</b> Islands as Hotspots for Emerging Mosquito-Borne Viruses: A One-Health Perspective<ul><li>Develop real-time tracking (<uri xlink:href="www.<b>nextstrain</b>.org">www.<b>nextstrain</b>.org</uri>) and forecasting using syndromic or internet search surveillance [<xref rid="B154-viruses-11-00011" ref-type="bibr"><u>154</u></xref>,<xref rid="B155-viruses-11-00011" ref-type="bibr"><u>155</u></xref>];.<li> [154](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28482810">28482810</a>) [155](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27678086">27678086</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30563445">30563445</a> <b>(2)</b> GLUE: a flexible software system for virus sequence data<ul><li><b>Nextstrain</b>.<li> Common examples of auxiliary data include the disease status of infected organisms as in the Los Alamos HCV database [<xref ref-type="bibr" rid="CR12"><u>12</u></xref>], and geographical or temporal data as in <b>nextstrain</b> [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>, <xref ref-type="bibr" rid="CR10"><u>10</u></xref>].<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15377502">15377502</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30546099">30546099</a> <b>(1)</b> Tracking virus outbreaks in the twenty-first century<ul><li> Furthermore, extensive online communities and forums such as Twitter (<ext-link ext-link-type="uri" xlink:href="https://www.twitter.com">https://www.twitter.com</ext-link>), Virological (<ext-link ext-link-type="uri" xlink:href="http://virological.org">http://virological.org</ext-link>), FluTrackers (<ext-link ext-link-type="uri" xlink:href="https://www.flutrackers.com">https://www.flutrackers.com</ext-link>), ProMED (<ext-link ext-link-type="uri" xlink:href="https://www.promedmail.org">https://www.promedmail.org</ext-link>), <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://www.<b>nextstrain</b>.org">https://www.<b>nextstrain</b>.org</ext-link>), HealthMap (<ext-link ext-link-type="uri" xlink:href="https://www.healthmap.org">https://www.healthmap.org</ext-link>) and Microreact (<ext-link ext-link-type="uri" xlink:href="https://www.microreact.org">https://www.microreact.org</ext-link>) allow rapid dissemination of unpublished results and analyses.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30193960">30193960</a> <b>(3)</b> From Theory to Practice: Translating Whole-Genome Sequencing (WGS) into the Clinic<ul><li> Influenza surveillance is arguably the most developed, with large sequencing repositories such as the GISAID database (<ext-link ext-link-type="uri" xlink:href="http://gisaid.org" id="intr0015">gisaid.org</ext-link>) and online data exploration and phylodynamics available through web tools such as NextFlu <xref rid="bib0300" ref-type="bibr"><u>[60]</u></xref> and <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org" id="intr0020">http://<b>nextstrain</b>.org</ext-link>), which also allows examination of other significant viruses including Zika, Ebola, and avian influenza.<li> [[60]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30475820">30475820</a> <b>(1)</b> Phylogenetic analysis of West Nile Virus in Maricopa County, Arizona: Evidence for dynamic behavior of strains in two major lineages in the American Southwest<ul><li> However, there is a long branch separating MG004536 from the other eight strains, and data presented on <b>Nextstrain</b> [<xref rid="pone.0205801.ref029" ref-type="bibr"><u>29</u></xref>] indicate that MG004536 is actually part of a separate entrance from Texas.<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30226188">30226188</a> <b>(3)</b> Frequent Genetic Mismatch between Vaccine Strains and Circulating Seasonal Influenza Viruses, Hong Kong, China, 1996&#x2013;2012<ul><li> (<xref rid="R23" ref-type="bibr"><italic>23</italic></xref>) and sequence information from <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30256887">30256887</a> <b>(1)</b> A systematic method for surveying data visualizations and a resulting genomic epidemiology visualization typology: GEViT<ul><li> Tools including <b>nextstrain</b> (<xref rid="bty832-B12" ref-type="bibr">Hadfield <italic>et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30231576">30231576</a> <b>(1)</b> Estimating Vaccine-Driven Selection in Seasonal Influenza<ul><li> For the H3N2 analysis, we collected sequences from GISAID and inferred clade membership and frequencies using <b>Nextstrain</b> [<xref rid="B56-viruses-10-00509" ref-type="bibr"><u>56</u></xref>].<li> [56](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>bioRxiv</source>          <year>2017</year>          <fpage>224048</fpage>          <pub-id pub-id-type="doi">10.1101/224048</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30104379">30104379</a> <b>(2)</b> Deep mutational scanning of hemagglutinin helps predict evolutionary fates of human H3N2 influenza variants<ul><li>The pylogenetic tree was generated using <b>Nextstrain</b>&#x2019;s augur pipeline (<xref rid="r85" ref-type="bibr"><u>85</u></xref>), and ancestral state reconstruction and branch length timing were performed with TreeTime (<xref rid="r86" ref-type="bibr"><u>86</u></xref>).<li> Frequency trajectories of mutations were estimated following <b>Nextstrain</b>&#x2019;s augur pipeline as first implemented in Nextflu (<xref rid="r87" ref-type="bibr"><u>87</u></xref>).<li> [85](          <person-group person-group-type="author">            <name name-style="western">              <surname>Hadfield</surname>              <given-names>J</given-names>            </name>            <etal/>          </person-group>          <year>2018</year>          <article-title>Nextstrain: Real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics</source>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [86](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [87](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(9)</b> <b>Nextstrain</b>: real-time tracking of pathogen evolution<ul><li> We have developed <b>Nextstrain</b> to visualize outbreaks in as close to real time as possible.<li><b>Nextstrain</b> consists of data curation, analysis and visualization components: Python scripts maintain a database of available sequences and related metadata, sourced from public repositories such as NCBI (<ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov">www.ncbi.nlm.nih.gov</ext-link>), GISAID (<ext-link ext-link-type="uri" xlink:href="http://www.gisaid.org">www.gisaid.org</ext-link>) and ViPR (<ext-link ext-link-type="uri" xlink:href="http://www.viprbrc.org">www.viprbrc.org</ext-link>), as well as GitHub repositories and other sources of genomic data.<li> Visualization is available through <b>nextstrain</b>.org.<li> There is a growing need for surveillance of non-influenza viruses (<xref rid="bty407-B9" ref-type="bibr">Tang <italic>et al.</italic>, 2017</xref>), and <b>Nextstrain</b> is able to be extended to most outbreaks with readily accessible genomic data, although we note the potential for recombination or low mutation rate to confound phylogenetic signal.<li>Genomic epidemiology of Zika virus as of Oct 2017 (live display at <b>nextstrain</b>.org/zika).<li><b>Nextstrain</b> tracks and reconstructs mutations across the tree and displays this information as a bar chart of entropy at each position in the genome, as well as showing the mutations inferred to occur on each branch by hovering over the tree.<li> For example, mutations identified by <xref rid="bty407-B2" ref-type="bibr">de Vries <italic>et al.</italic> (2017)</xref> are readily visible at <b>nextstrain</b>.org/avian/h7n9.<li><b>Nextstrain</b> presents a single, continuously updated overview of both endemic viral disease (seasonal influenza, dengue) as well as emergent viral outbreaks (avian influenza, Zika, Ebola), all based upon the same underlying bioinformatics architecture.<li>Analysis of such outbreaks relies on public sharing of data, and <b>Nextstrain</b> has the ability to automatically update as new sequences from a range of public databases and repositories appear.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29716518">29716518</a> <b>(3)</b> Treemmer: a tool to reduce large phylogenetic datasets with minimal loss of diversity<ul><li>The influenza A virus tree was download from <b>nextstrain</b> [<xref ref-type="bibr" rid="CR17"><u>17</u></xref>] on the 1<sup>st</sup> of December 2017.<li> To do this we used a time-calibrated influenza A / H3N2 tree with 2,080 viral sequences downloaded from <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR17"><u>17</u></xref>].<li> tree in <b>Nextstrain</b>) at 99%, 90%, 75% and 50% of the relative tree length.<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29346387">29346387</a> <b>(0)</b> Inferring the risk factors behind the geographical spread and transmission of Zika in the Americas<ul><li> [<xref rid="pntd.0006194.ref003" ref-type="bibr"><u>3</u></xref>, <xref rid="pntd.0006194.ref004" ref-type="bibr"><u>4</u></xref>, <xref rid="pntd.0006194.ref024" ref-type="bibr"><u>24</u></xref>], see also <ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org/zika">www.<b>nextstrain</b>.org/zika</ext-link>.<li> [3](<name><surname>Faria</surname><given-names>NR</given-names></name>, <name><surname>Quick</surname><given-names>J</given-names></name>, <name><surname>Claro</surname><given-names>IM</given-names></name>, <name><surname>Th&#xE9;z&#xE9;</surname><given-names>J</given-names></name>, <name><surname>de Jesus</surname><given-names>JG</given-names></name>, <name><surname>Giovanetti</surname><given-names>M</given-names></name>, <etal>et al</etal><article-title>Establishment and cryptic transmission of Zika virus in Brazil and the Americas</article-title> [Journal Article]. <source>Nature</source>. <year>2017</year>;advance online publication. Available from: <pub-id pub-id-type="doi">10.1038/nature22401</pub-id>.) [4](<name><surname>Metsky</surname><given-names>HC</given-names></name>, <name><surname>Matranga</surname><given-names>CB</given-names></name>, <name><surname>Wohl</surname><given-names>S</given-names></name>, <name><surname>Schaffner</surname><given-names>SF</given-names></name>, <name><surname>Freije</surname><given-names>CA</given-names></name>, <name><surname>Winnicki</surname><given-names>SM</given-names></name>, <etal>et al</etal><article-title>Zika virus evolution and spread in the Americas</article-title> [Journal Article]. <source>Nature</source>. <year>2017</year>;advance online publication. Available from: <pub-id pub-id-type="doi">10.1038/nature22402</pub-id>.) [24](<name><surname>Grubaugh</surname><given-names>ND</given-names></name>, <name><surname>Ladner</surname><given-names>JT</given-names></name>, <name><surname>Kraemer</surname><given-names>MUG</given-names></name>, <name><surname>Dudas</surname><given-names>G</given-names></name>, <name><surname>Tan</surname><given-names>AL</given-names></name>, <name><surname>Gangavarapu</surname><given-names>K</given-names></name>, <etal>et al</etal><article-title>Genomic epidemiology reveals multiple introductions of Zika virus into the United States</article-title> [Journal Article]. <source>Nature</source>. <year>2017</year>;advance online publication. Available from: <pub-id pub-id-type="doi">10.1038/nature22400</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a> <b>(2)</b> TreeTime: Maximum-likelihood phylodynamic analysis<ul><li> It is already in use as an integral component of the real-time time outbreak tracking tools <b>nextstrain</b> and nextflu (<xref rid="vex042-B15" ref-type="bibr"><u>Neher and Bedford, 2015</u></xref>).<li>TreeTime was developed to analyse large heterochronous viral sequence alignments and we have used TreeTime as the core component of the real-time phylogenetics pipelines <b>nextstrain</b> and nextflu (<xref rid="vex042-B15" ref-type="bibr"><u>Neher and Bedford, 2015</u></xref>).<li> [Neher and Bedford, 2015](<person-group person-group-type="author"><name name-style="western"><surname>Neher</surname><given-names>R. A.</given-names></name>, <name name-style="western"><surname>Bedford</surname><given-names>T.</given-names></name></person-group> (<year>2015</year>) &#x2018;Nextflu: Real-Time Tracking of Seasonal Influenza Virus Evolution in Humans&#x2019;, <source>Bioinformatics (Oxford, England)</source>, <volume>31</volume>: <fpage>3546</fpage>.) [Neher and Bedford, 2015](<person-group person-group-type="author"><name name-style="western"><surname>Neher</surname><given-names>R. A.</given-names></name>, <name name-style="western"><surname>Bedford</surname><given-names>T.</given-names></name></person-group> (<year>2015</year>) &#x2018;Nextflu: Real-Time Tracking of Seasonal Influenza Virus Evolution in Humans&#x2019;, <source>Bioinformatics (Oxford, England)</source>, <volume>31</volume>: <fpage>3546</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29129921">29129921</a> <b>(1)</b> Towards a genomics-informed, real-time, global pathogen surveillance system<ul><li> Genome sequencing data coupled with rich metadata would then be released in real time to web-based platforms, such as <ext-link ext-link-type="uri" xlink:href="http://www.virological.org">Virological</ext-link> for collaborative analysis and <ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org"><b>Nextstrain</b></ext-link> for analysis and visualization<sup><xref ref-type="bibr" rid="CR96"><u>96</u></xref></sup>.<li> [96](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28933344">28933344</a> <b>(1)</b> First outbreak of Zika virus in the continental United States: a modelling analysis<ul><li> Our conclusion is supported by molecular-clock analyses of 32 ZIKV isolates from Florida, reported by the collaborative project <b>nextstrain</b> [<xref rid="r29" ref-type="bibr"><u>29</u></xref>,<xref rid="r30" ref-type="bibr"><u>30</u></xref>], which suggest that importation occurred between the end of February and the end of March.<li> [29](Nextstrain. Real-time tracking of Zika virus evolution. [Accessed 13 Mar 2017]. Available from: <ext-link ext-link-type="uri" xlink:href="http://www.nextstrain.org/zika/">http://www.nextstrain.org/zika/</ext-link>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115986">26115986</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28880955">28880955</a> <b>(0)</b> Zika genomics urgently need standardized and curated reference sequences<ul><li> The widespread distribution of the amino acid substitutions (L42F in the M protein and E146K in the NS1 protein) in the recent epidemic further support the use of KJ776791 over KX369547 (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org/zika">http://www.<b>nextstrain</b>.org/zika</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28763440">28763440</a> <b>(2)</b> Developing international open science collaborations: Funder reflections on the Open Science Prize<ul><li> The project uses an online visualization platform (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org/">http://www.<b>nextstrain</b>.org</ext-link>) in which the outputs of statistical analyses of viral genome sequences can be used by public health officials for epidemiological insights within days of genome sequences having been obtained from viral pathogen samples taken from patients.<li> <b>Nextstrain</b>.org expanded upon the submitters&#x2019; prior work on influenza tracking (nextflu.org) to include many other viral pathogens.<li> used the funds the team received for the prototype <b>nextstrain</b>.org as a proof of concept to help them secure NIH funding from 2 unrelated NIH funding lines in addition to the OSP.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29202081">29202081</a> <b>(2)</b> Phylodynamic applications in 21<sup>st</sup> century global infectious disease research<ul><li> An excellent example of this type of collaboration is the &#x201C;<b>nextstrain</b>&#x201D; project (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org/">http://www.<b>nextstrain</b>.org/</ext-link>).<li> <b>Nextstrain</b> is a publicly available repository currently comprised of evolutionary datasets for Ebola, Zika, and avian and seasonal influenza viruses contributed by research groups from all over the world for the purpose of real-time tracking of viral epidemics.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28232448">28232448</a> <b>(2)</b> First Complete Genome Sequences of Zika Virus Isolated from Febrile Patient Sera in Ecuador<ul><li>The complete genomic sequences have been annotated and deposited in GenBank, under accession numbers <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/KX879603">KX879603</ext-link> to <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/KX879604">KX879604</ext-link>, and on <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/zika">http://<b>nextstrain</b>.org/zika</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28217117">28217117</a> <b>(2)</b> The Promise of Whole Genome Pathogen Sequencing for the Molecular Epidemiology of Emerging Aquaculture Pathogens<ul><li> Full genome phylogenies of Ebola and Zika viruses, and patterns of genomic variability, are freely available to explore at <b>nextstrain</b>.org/ebola and <b>nextstrain</b>.org/zika respectively.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680262">36680262</a> <b>(2)</b> A Repeat Pattern of Founder Events for SARS-CoV-2 Variants in Alaska<ul><li> Visualizations, Statistical Analyses, and <b>Nextstrain</b> Build.<li> We generated a <b>Nextstrain</b> (cli v-3.2.4) build, which generates a maximum likelihood reconstruction [<xref rid="B17-viruses-15-00222" ref-type="bibr"><u>17</u></xref>], to examine the phylogenetic relationship of BA.2.3 in Alaska compared to global sequences.<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36633573">36633573</a> <b>(1)</b> Machine Learning-Assisted Real-Time Polymerase ChainReaction and High-Resolution Melt Analysis for SARS-CoV-2 VariantIdentification<ul><li>To date, several SARS-CoV-2 nomenclature systems have been introduced,including GISIAD, PANGOLIN, <b>Nextstrain</b>, and WHO nomenclature systems.WHO has classified the virus strains based on their risk to globalpublic health: Variants of Monitoring (VOMs), Variants of Interest(VOIs), and Variants of Concern (VOCs).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36642191">36642191</a> <b>(0)</b> Genomic evidence for reinfection with different Omicron subvariants<ul><li> The data were uploaded and visualized using Nextclade (https://clades.<b>nextstrain</b>.org).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36695611">36695611</a> <b>(2)</b> Successful lung transplantation using an allograft from a COVID-19&#x2013;recovered donor: a potential role for subgenomic RNA to guide organ utilization<ul><li> Both viruses contained the ORF1b:P314L mutation, a defining mutation that separated the <b>nextstrain</b> 20A (pangolin lineage B.1) clade from 19A (pangolin lineage B), and the 28881:28883 GGG &gt; AAC mutation (N:R203K, G204R), which is a defining mutation of the <b>nextstrain</b> 20B clade (pangolin lineage B.1.1).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36629447">36629447</a> <b>(1)</b> Rapid Assessment of SARS-CoV-2 Variant-Associated Mutations in Wastewater Using Real-Time RT-PCR<ul><li> The probes were designed to target-defining mutations of each variant, as detailed in <xref rid="tab6" ref-type="table"><u>Table&#xA0;6</u></xref>, which were identified using <b>Nextstrain</b> resources (<xref rid="B36" ref-type="bibr"><u>36</u></xref>).<li> [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36703815">36703815</a> <b>(1)</b> Detection of hospital environmental contamination during SARS-CoV-2 Omicron predominance using a highly sensitive air sampling device<ul><li>The study was conducted from January to March 2022 when Singapore's first Omicron wave occurred, with Omicron BA.1 and then BA.2 representing almost all isolates by February, according to the SARS-CoV-2 pathogen tracking in <b>Nextstrain</b> database (<xref rid="B28" ref-type="bibr"><u>28</u></xref>).<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36627290">36627290</a> <b>(3)</b> SARS-CoV-2 escape from cytotoxic T cells during long-term COVID-19<ul><li> After that, we excluded sequences (1) with more than 300 positions of missing data (Ns) and gaps, (2) excluded by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/exclude.txt</ext-link>), or (3) from non-human animals other than minks, leaving us with 201,948 sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36751428">36751428</a> <b>(1)</b> Evolutionary rate of SARS-CoV-2 increases during zoonotic infection of farmed mink<ul><li> 2022</xref>) and generated a representative global human dataset by randomly selecting 200 sequences from the <b>Nextstrain</b> SARS-CoV-2 build from April 2021 (<xref rid="R32" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36579775">36579775</a> <b>(1)</b> Dynamics of nosocomial SARS&#x2010;CoV&#x2010;2 transmissions: Facing the challenge of variants of concern in a Brazilian reference hospital<ul><li> (A) Distribution of sequenced samples (<b>Nextstrain</b> server, Nextclade tool).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36625603">36625603</a> <b>(2)</b> Rapid and Flexible RT-qPCR Surveillance Platforms To Detect SARS-CoV-2 Mutations<ul><li> Consensus genome mutation calling with reference to Wuhan-Hu-1/2019 (GenBank accession number <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MN908947" ext-link-type="uri" assigning-authority="insdc">MN908947</ext-link>) was performed with Nextclade CLI (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/nextclade" ext-link-type="uri">https://github.com/<b>nextstrain</b>/nextclade</ext-link>) (v.1.2.0), and lineage designations were performed using Pangolin (<ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link>) (v.3.1.3) with the accompanying pangoLEARN model (<ext-link xlink:href="https://github.com/cov-lineages/pangoLEARN" ext-link-type="uri">https://github.com/cov-lineages/pangoLEARN</ext-link>) (v.1.2.6).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36643886">36643886</a> <b>(2)</b> The race to understand immunopathology in COVID-19: Perspectives on the impact of quantitative approaches to understand within-host interactions<ul><li> This means that each 30Kb genome will accumulate approximately 20 mutations per year, with the most recent estimate being up to 23.7 substitutions per year (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="interref0001">https://<b>nextstrain</b>.org</ext-link>; accessed 1 February 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36687432">36687432</a> <b>(0)</b> Probable vertical transmission of Alpha variant of concern (B.1.1.7) with evidence of SARS-CoV-2 infection in the syncytiotrophoblast, a case report<ul><li> RNA mutation identification used Nextclade version v0.14.2 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36638615">36638615</a> <b>(7)</b> L-shaped distribution of the relative substitution rate (c/&#x3BC;) observed for SARS-COV-2's genome, inconsistent with the selectionist theory, the neutral theory and the nearly neutral theory but a near-neutral balanced selection theory: Implication on &#x201C;neutralist-selectionist&#x201D; debate<ul><li> Please refer to the <b>Nextstrain</b> web site for the present figures: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="intref0030">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>.<li> Systematic analysis using <italic>c/&#x3BC;</italic> was done on three independent sets of SARS-CoV-2 genomes (a total of 11,198 sequences) from human patients within the first 19 months of the COVID-19 pandemic, which were selected and compiled by the <b>Nextstrain</b> team (<xref rid="tbl1" ref-type="table"><u>Table 1</u></xref>) [<xref rid="bib40" ref-type="bibr"><u>40</u></xref>,<xref rid="bib59" ref-type="bibr"><u>59</u></xref>].<li> These were selected and compiled by <b>Nextstrain</b> [<xref rid="bib40" ref-type="bibr"><u>40</u></xref>] and overlapping sequences between each dataset were removed prior to our analysis.<li> Consistent with the total NT mutation timeline from the <b>Nextstrain</b> team between Dec.<li> Our relative substitution rate (<italic>c/&#x3BC;</italic>) method was used to assign selection type to each gene, UTR and TRS of SARS-CoV-2 using the selected genome sequences by the <b>Nextstrain</b> team.<li> [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [59](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36610792">36610792</a> <b>(1)</b> Cellular APOBEC3A deaminase drives mutations in the SARS-CoV-2 genome<ul><li> In the extracted sequence dataset, the rate of total mononucleotide substitutions was &#x223C;26.9 nucleotides (nt) per year (nt/yr) in the 2.5-year period assessed, nearly identical to the real-time estimate reported by <b>Nextstrain</b> (30.6 nt/yr) (<xref rid="B42" ref-type="bibr"><u>42</u></xref>).<li> [42](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36687433">36687433</a> <b>(4)</b> Whole-genome sequencing of SARS-CoV-2 isolates from symptomatic and asymptomatic individuals in Tanzania<ul><li> NextClade Web 1.10.0 (<xref rid="B15" ref-type="bibr"><u>15</u></xref>) output was used to visualize the phylogenetic placement of the isolates in <b>NextStrain</b> Auspice 2.32.0 (<xref rid="B22" ref-type="bibr"><u>22</u></xref>).<li> Two of the four Delta isolates (Tanzania/KIDH-02B14/2021 and Tanzania/KIDH-02B15/2021) were from asymptomatic travelers arriving from India of whom samples were collected on 3 May 2021 (<xref rid="T2" ref-type="table"><u>Table 2</u></xref>), and were placed in <b>NextStrain</b> clades 21A and 21J, respectively.<li> The other two Delta isolates (Tanzania/KIDH-03B05/2021 and Tanzania/KIDH-02B01/2021) were from local symptomatic suspects, collected in Mwanza on 22 April 2021 and 25 April 2021, respectively, and both placed in <b>NextStrain</b> clade 21I.<li> Isolate Tanzania/KIDH-03B05/2021 and Tanzania/KIDH-02B01/2021 of <b>NextStrain</b> clade 21I clustered closely with isolates from neighboring Rwanda, Uganda, Kenya, and Burundi, suggesting regional circulation (<xref rid="FS2" ref-type="supplementary-material"><u>Supplementary File 2</u></xref>).<li> [15](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J Open Source Softw.</italic></source> (<year>2021</year>) <volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36684874">36684874</a> <b>(0)</b> The Delta variant wave in Tunisia: Genetic diversity, spatio-temporal distribution and evidence of the spread of a divergent AY.122 sub-lineage<ul><li> The web-based software Nextclade v1.12.0 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) was used for clade assignment, diversity and gene-wise amino acid mutations analysis (<xref rid="B32" ref-type="bibr"><u>32</u></xref>).<li> The web-based software Nextclade v1.12.0 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) was used for gene-wise amino acid mutations analysis (<xref rid="B32" ref-type="bibr"><u>32</u></xref>).<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36676732">36676732</a> <b>(3)</b> A Case of a Malignant Lymphoma Patient Persistently Infected with SARS-CoV-2 for More than 6 Months<ul><li>The comparison dataset comprised 27 representative SARS-CoV-2 genomes selected from <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 17 August 2022) by regions of interest (Japan, Asia, Europe, North and South America).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36597102">36597102</a> <b>(0)</b> A traveling SARS-CoV-2 laboratory as part of a pandemic response among vulnerable Brazilian populations<ul><li> We assessed the consensus genome sequence quality using Nextclade version 0.8.1 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680188">36680188</a> <b>(1)</b> Variability in the Clinical Course of COVID-19 in a Retrospective Analysis of a Large Real-World Database<ul><li> According to data available for our country, the defined pre-Delta period (1 March 2020&#x2013;30 June 2021) was dominated by infections with <b>Nextstrain</b> clades 20A, 20B, and 20C (till February 2021), and the alpha variant (from February 2021).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36619228">36619228</a> <b>(1)</b> Characterization of SARS-CoV-2 recombinants and emerging Omicron sublineages<ul><li> GISAID, <b>Nextstrain</b>, and Pango are the three most common nomenclature systems used during the COVID-19 pandemic.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680148">36680148</a> <b>(0)</b> Genomic Epidemiology of the SARS-CoV-2 Epidemic in Cyprus from November 2020 to October 2021: The Passage of Waves of Alpha and Delta Variants of Concern<ul><li>Mutations in the sequences in this study were identified using Nextclade Webtool Web version 1.14.1 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/Nextclade">https://clades.<b>nextstrain</b>.org/Nextclade</uri> accessed on 7 May 2022) [<xref rid="B35-viruses-15-00108" ref-type="bibr"><u>35</u></xref>].<li> Mutations were identified using the Nextclade webtool (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, date accessed 9 May 2022) [<xref rid="B35-viruses-15-00108" ref-type="bibr"><u>35</u></xref>].<li> The maximum likelihood tree was generated in Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, date accessed 05 August 2022 [<xref rid="B35-viruses-15-00108" ref-type="bibr"><u>35</u></xref>]) and highlights the emergence of mutations.<li> [35](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [35](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [35](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36603376">36603376</a> <b>(1)</b> COVID-19: A state of art on immunological responses, mutations, and treatment modalities in riposte<ul><li> The Omicron BA.1 was the original Omicron variant discovered in November 2021 and it sequences as a monophyletic clade rooted within the B.1.1 lineage (<b>Nextstrain</b> clade 20B), with no clear basal progenitor <xref rid="bib47" ref-type="bibr"><u>[47]</u></xref>.<li> [[47]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35042229">35042229</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680136">36680136</a> <b>(1)</b> One Health Investigation of SARS-CoV-2 in People and Animals on Multiple Mink Farms in Utah<ul><li> Phylogenetic relations of the study sequences were inferred using maximum likelihood analyses implemented in Treetime using the <b>NextStrain</b> pipeline [<xref rid="B37-viruses-15-00096" ref-type="bibr"><u>37</u></xref>].<li> [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680135">36680135</a> <b>(1)</b> The Cold-Adapted, Temperature-Sensitive SARS-CoV-2 Strain TS11 Is Attenuated in Syrian Hamsters and a Candidate Attenuated Vaccine<ul><li> Viral sequences were strain-typed using <b>NextStrain</b> criteria (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri> (accessed on 17 October 2022)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36576129">36576129</a> <b>(1)</b> Accurate and fast clade assignment via deep learning and frequency chaos game representation<ul><li> Using a different model for each task, all based on random forest and 6-mers as input, the reported accuracies are <inline-formula><tex-math id="TM0015" notation="LaTeX">\documentclass[12pt]{minimal}\usepackage{amsmath}\usepackage{wasysym}\usepackage{amsfonts}\usepackage{amssymb}\usepackage{amsbsy}\usepackage{upgreek}\usepackage{mathrsfs}\setlength{\oddsidemargin}{-69pt}\begin{document}}{}$97,77\%$\end{document}</tex-math></inline-formula>, <inline-formula><tex-math id="TM0016" notation="LaTeX">\documentclass[12pt]{minimal}\usepackage{amsmath}\usepackage{wasysym}\usepackage{amsfonts}\usepackage{amssymb}\usepackage{amsbsy}\usepackage{upgreek}\usepackage{mathrsfs}\setlength{\oddsidemargin}{-69pt}\begin{document}}{}$99,52\%$\end{document}</tex-math></inline-formula>, and <inline-formula><tex-math id="TM0017" notation="LaTeX">\documentclass[12pt]{minimal}\usepackage{amsmath}\usepackage{wasysym}\usepackage{amsfonts}\usepackage{amssymb}\usepackage{amsbsy}\usepackage{upgreek}\usepackage{mathrsfs}\setlength{\oddsidemargin}{-69pt}\begin{document}}{}$96,56\%$\end{document}</tex-math></inline-formula> for GISAID, <b>Nextstrain</b>, and Pango models, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36591005">36591005</a> <b>(4)</b> Advances of CRISPR-Cas13 system in COVID-19 diagnosis and treatment<ul><li> It is an enveloped virus with a single-stranded, positive-sense RNA genome of &#x223C;30,000 nucleotides in size.<xref rid="bib1" ref-type="bibr"><u>1</u></xref>, <xref rid="bib2" ref-type="bibr"><u>2</u></xref>, <xref rid="bib3" ref-type="bibr"><u>3</u></xref> The N-terminal 3/4 genome is translated into two polyproteins (pp1a and pp1ab), which are then processed into 16 nonstructural proteins that constitute the replication/translation complex.<xref rid="bib3" ref-type="bibr"><u>3</u></xref>, <xref rid="bib4" ref-type="bibr"><u>4</u></xref>, <xref rid="bib5" ref-type="bibr"><u>5</u></xref> The rest of the genome close to the C-terminus synthesizes spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins as well as accessory proteins including ORF3a, ORF3b, ORF6, ORF7a, ORF7b, ORF8, ORF9b, ORF9c, and ORF10 (<xref rid="fig1" ref-type="fig"><u>Fig.&#xA0;1</u></xref>A).<xref rid="bib3" ref-type="bibr"><sup>3</sup></xref><sup>,</sup><xref rid="bib6" ref-type="bibr"><sup>6</sup></xref><sup>,</sup><xref rid="bib7" ref-type="bibr"><sup>7</sup></xref> The SARS-CoV-2 genome has significant genetic diversity (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/sars-cov-2" id="intref0010">https://<b>nextstrain</b>.org/sars-cov-2</ext-link>) with &#x223C;300 thousand <italic>de novo</italic> mutations found to date.<li> data taken from <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m" id="intref0020">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m</ext-link>).<li> [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32015507">32015507</a>) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32160149">32160149</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32015508">32015508</a>) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32015508">32015508</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33245961">33245961</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32344679">32344679</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36575217">36575217</a> <b>(0)</b> Genomic epidemiology of SARS- CoV-2 Omicron variants in the Republic of Korea<ul><li> We used Nextclade v1.13.2 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) for quality control (QC) of 582 sequences isolated in South Korea.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36610711">36610711</a> <b>(2)</b> AccuVIR: an ACCUrate VIRal genome assembly tool for third-generation sequencing data<ul><li> For example, a popular strain-level analysis website, &#x2018;<b>Nextstrain</b>&#x2019; (<xref rid="btac827-B9" ref-type="bibr">Hadfield <italic toggle="yes">et al.<li> The accuracy of the constructed genomes can directly affect the downstream analysis such as those at <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35982678">35982678</a> <b>(1)</b> Transmissible SARS-CoV-2 variants with resistance to clinical protease inhibitors<ul><li> The generated phylogenetic trees were visualized using Auspice.us from <b>NextStrain</b> (<xref rid="R56" ref-type="bibr"><u>56</u></xref>).<li> [56](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36679891">36679891</a> <b>(1)</b> Immunological Studies to Understand Hybrid/Recombinant Variants of SARS-CoV-2<ul><li>The scientific community follows an established nomenclature system provided by the Global Initiative on Sharing All Influenza Data (GISAID), <b>NextStrain</b>, and Pango for the naming and tracking of the lineages of the SARS-CoV-2 virus.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36590492">36590492</a> <b>(2)</b> The SARS-CoV-2 Delta (B.1.617.2) variant with spike N501Y mutation in the shadow of Omicron emergence<ul><li> <b>Nextstrain</b> 21A/S:478K) and Omicron (Pango B.1.1.529 and BA lineages; <b>Nextstrain</b> 21K) are classified as variants of concern (VOC) (<xref rid="bib3" ref-type="bibr"><u>CDC, 2021</u></xref>; <xref rid="bib23" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>; <xref rid="bib32" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref>; <xref rid="bib45" ref-type="bibr"><u>WHO, 2021a</u></xref>).<li> [CDC, 2021](          <person-group person-group-type="author">            <collab>CDC</collab>          </person-group>          <part-title>SARS-CoV-2 Variant Classifications and Definitions</part-title>          <year>2021</year>          <ext-link ext-link-type="uri" xlink:href="http://www.cdc.gov/coronavirus/2019-ncov/variants/variant-classifications.html" id="intref0010">www.cdc.gov/coronavirus/2019-ncov/variants/variant-classifications.html</ext-link>        ) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [WHO, 2021a](          <person-group person-group-type="author">            <collab>WHO</collab>          </person-group>          <article-title>Tracking SARS-CoV-2 variants</article-title>          <ext-link ext-link-type="uri" xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" id="intref0055">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link>          <year>2021</year>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36620039">36620039</a> <b>(1)</b> Genomic landscape of the SARS-CoV-2 pandemic in Brazil suggests an external P.1 variant origin<ul><li>We searched for Brazilian sequence mutations using the web platform <b>Nextstrain</b><xref rid="fn0003" ref-type="fn"><sup>3</sup></xref> from FASTA nucleotide files (Hadfield et al., <xref rid="B22" ref-type="bibr"><u>2018</u></xref>).<li> [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36611300">36611300</a> <b>(1)</b> Complete Genomic Characterisation and Mutation Patterns of Iraqi SARS-CoV-2 Isolates<ul><li> The platforms are GISAID, <b>Nextstrain</b>, and PANGO [<xref rid="B20-diagnostics-13-00008" ref-type="bibr"><u>20</u></xref>,<xref rid="B23-diagnostics-13-00008" ref-type="bibr"><u>23</u></xref>,<xref rid="B24-diagnostics-13-00008" ref-type="bibr"><u>24</u></xref>,<xref rid="B25-diagnostics-13-00008" ref-type="bibr"><u>25</u></xref>,<xref rid="B26-diagnostics-13-00008" ref-type="bibr"><u>26</u></xref>].<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34108654">34108654</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36668909">36668909</a> <b>(0)</b> First Case of Monkeypox in Venezuela: Partial Complete Genome Sequence Allowed Its Grouping into the West African Clade II<ul><li> The JSON file containing the phylogenomic datasets was downloaded from Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> last accessed 20 September 2022), and was analyzed at Auspice (<uri xlink:href="https://auspice.us/">https://auspice.us/</uri> last accessed 20 September 2022) [<xref rid="B13-tropicalmed-08-00002" ref-type="bibr"><u>13</u></xref>].<li> [13](          <article-title>Auspice.Us</article-title>          <comment>Available online: <ext-link xlink:href="https://auspice.us/" ext-link-type="uri">https://auspice.us/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-27">(accessed on 27 July 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36539656">36539656</a> <b>(3)</b> Isolation and characterization of a SARS-CoV-2 variant with a Q677H mutation in the spike protein<ul><li> Although Q677H is not a characteristic of any variants of concern according to WHO, it can be generally detected in Alpha (20I), Beta (20H), Gamma (20J), Delta (21I and 21J), and Omicron (21K, 21L, 22A, 22C, and 22D) variants with peak frequencies of 100% (September 20, 2021 to December 21, 2021), 81% (July 13, 2021 to July 20, 2021), 100% (November 7, 2021 to November 29, 2021), 100% (March 22, 2022 to May 30, 2022), and 40-50% (February 21, 2022 to October 1, 2022), respectively, according to the <b>Nextstrain</b> project (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/groups/neherlab/ncov/S.Q677?c=gt-S_677">https://<b>nextstrain</b>.org/groups/neherlab/ncov/S.Q677?c=gt-S_677</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36679852">36679852</a> <b>(1)</b> Reinfection with SARS-CoV-2 and Waning Humoral Immunity: A Case Report<ul><li> SARS-CoV-2 clade designations and phylogenetic analyses were produced using <b>NextStrain</b> [<xref rid="B20-vaccines-11-00005" ref-type="bibr"><u>20</u></xref>]; sequence acknowledgements are provided in <xref rid="app1-vaccines-11-00005" ref-type="app"><u>Table S3</u></xref>.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680042">36680042</a> <b>(1)</b> A Glimpse on the Evolution of RNA Viruses: Implications and Lessons from SARS-CoV-2<ul><li> Visualisation of SARS-CoV-2 evolution and spread since the start of the pandemic, performed using <b>Nextstrain</b>, a publicly accessible bioinformatic tool for real-life tracking of pathogen evolution, shows global dominance of Omicron strains that almost completely replaced Delta and all previous variants (<xref rid="viruses-15-00001-f003" ref-type="fig"><u>Figure 3</u></xref>) [<xref rid="B180-viruses-15-00001" ref-type="bibr"><u>180</u></xref>,<xref rid="B181-viruses-15-00001" ref-type="bibr"><u>181</u></xref>].<li> [180](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [181](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36535982">36535982</a> <b>(3)</b> Challenges and developments in universal vaccine design against SARS-CoV-2 variants<ul><li> The unrooted phylogenic tree is constructed by <b>Nextstrain</b> according to the sequences of 3050 SARS-CoV-2 spike glycoproteins from GISAID collected between December 2019 and October 2022 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m?l=unrooted&amp;m=div">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m?l=unrooted&amp;m=div</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36569378">36569378</a> <b>(1)</b> Mosaic Recombination Inflicted Various SARS-CoV-2 Lineages to Emerge into Novel Virus Variants: a Review Update<ul><li> For the designation and tracking of SARS-CoV-2 new genetic lineages, established nomenclature systems by GISAID, <b>Nextstrain</b> and Pango Lineage are currently used by scientific communities [<xref ref-type="bibr" rid="CR8"><u>8</u></xref>, <xref ref-type="bibr" rid="CR9"><u>9</u></xref>].<li> [8](WHO. Tracking SARS-CoV-2 variants. (2022) <ext-link ext-link-type="uri" xlink:href="https://www.who.int/activities/tracking-SARS-CoV-2-variants">https://www.who.int/activities/tracking-SARS-CoV-2-variants</ext-link> (Accessed May 21, 2022).) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560820">36560820</a> <b>(2)</b> Comparative Analysis of Tomato Brown Rugose Fruit Virus Isolates Shows Limited Genetic Diversity<ul><li> [<xref rid="B24-viruses-14-02816" ref-type="bibr"><u>24</u></xref>] developed a <b>NextStrain</b> database to track ToBRFV outbreaks in the Netherlands and sequenced two ToBRFV population sets, each containing 54 and 47 genomes.<li> In the second <b>NextStrain</b> build, a divergent isolate recovered from seeds originating from Peru was found to represent a new genotype, and more variation is expected with more sequenced isolates [<xref rid="B29-viruses-14-02816" ref-type="bibr"><u>29</u></xref>].<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33031371">33031371</a>) [29](          <person-group person-group-type="author"><name><surname>van de Vossenberg</surname><given-names>B.T.L.H.</given-names></name><name><surname>Dawood</surname><given-names>T.</given-names></name><name><surname>Wo&#x17A;ny</surname><given-names>M.</given-names></name><name><surname>Botermans</surname><given-names>M.</given-names></name></person-group>          <article-title>First Expansion of the Public Tomato Brown Rugose Fruit Virus (ToBRFV) Nextstrain Build; Inclusion of New Genomic and Epidemiological Data</article-title>          <source>Phytofrontiers</source>          <year>2021</year>          <volume>1</volume>          <fpage>359</fpage>          <lpage>363</lpage>          <pub-id pub-id-type="doi">10.1094/PHYTOFR-01-21-0005-A</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36751558">36751558</a> <b>(0)</b> Surveillance and Analysis of SARS-CoV-2 Variant Importation &#x2014; China, January&#x2013;June 2022<ul><li>Consensus sequences were assessed for quality control by the Nextclade web tool (https://clades.<b>nextstrain</b>.org; version 2.4.1) with parameters for missing data, mixed sites, private mutations, mutation clusters, frameshifts, and stop codons.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36561171">36561171</a> <b>(12)</b> Local-Scale phylodynamics reveal differential community impact of SARS-CoV-2 in metropolitan US county<ul><li>A temporally-resolve phylogeny was created using the <b>Nextstrain</b> (<xref rid="R13" ref-type="bibr"><u>13</u></xref>) SARS-CoV-2 workflow (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>), which aligns sequences against the Wuhan Hu-1 reference using nextalign (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/nextclade" ext-link-type="uri">https://github.com/<b>nextstrain</b>/nextclade</ext-link>), infers a maximum-likelihood phylogeny using IQ-TREE (<xref rid="R14" ref-type="bibr"><u>14</u></xref>) with a GTR nucleotide substitution model, and estimates molecular clock branch lengths using TreeTime (<xref rid="R15" ref-type="bibr"><u>15</u></xref>).<li> To do so, we created specific phylogenies for Omicron (<b>Nextstrain</b> clades 21K, 21L, 21M comprising 2856 King County Sequences and 18,817 contextual sequences from around the world), Delta (<b>Nextstrain</b> clades 21A, 21I, 21J comprising 2955 King County Sequences and 19,197 contextual sequences from around the world), Alpha (<b>Nextstrain</b> clade 20I comprising 2941 King County Sequences and 15,406 contextual sequences from around the world), and all other SARS-CoV-2 lineages (2850 King County Sequences, 16,168 contextual sequences from around the world).<li>Phylogeographic reconstruction of spread around King County was conducted using the same <b>Nextstrain</b> workflow via ancestral trait reconstruction of PUMAs and North and South region geographic attributes.<li>After identifying relevant King County clusters from each of the four variant <b>Nextstrain</b> builds, we then annotated the clusters in a combined dataset.<li><b>Nextstrain</b> builds, BEAST XMLS, scripts, sequence information, and de-identified data can be found at <ext-link xlink:href="https://github.com/blab/ncov-king" ext-link-type="uri">https://github.com/blab/ncov-king</ext-link>.<li> Following the creation of time-resolved phylogenies using <b>Nextstrain</b> (<xref rid="R13" ref-type="bibr"><u>13</u></xref>), we split the sequences into local outbreak clusters using parsimony-based clustering to identify groups of sequences whose ancestral states were inferred to be in King County (see <xref rid="S2" ref-type="sec"><u>Methods</u></xref>).<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [15](<name><surname>Sagulenko</surname><given-names>P</given-names></name>, <name><surname>Puller</surname><given-names>V</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>. <article-title>TreeTime: Maximum-likelihood phylodynamic analysis</article-title>. <source>Virus Evol</source>. <year>2018</year><month>Jan</month><day>8</day>;<volume>4</volume>(<issue>1</issue>):<comment>vex042.</comment>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36526616">36526616</a> <b>(2)</b> Dysfunctional Sars-CoV-2-M protein-specific cytotoxic T lymphocytes in patients recovering from severe COVID-19<ul><li>Importantly, M<sub>198&#x2013;206</sub>-specific CD8<sup>+</sup> T cells were detected with significantly higher frequency at the periphery of convalescents suffering moderate/severe COVID-19 from late 2020 through early 2022, consistent with the reports demonstrating M protein as well conserved viral protein among various SARS-CoV-2 strains (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36573212">36573212</a> <b>(1)</b> Recent advances on human mpox<ul><li> <b>Nextstrain</b> database was used for genome sequence mutation search [<xref rid="bib17" ref-type="bibr"><u>17</u></xref>].<li> [17](          <article-title>Nextstrain.org</article-title>          <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/monkeypox/hmpxv1" id="intref0040">https://nextstrain.org/monkeypox/hmpxv1</ext-link>          <year>2022</year>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36655186">36655186</a> <b>(1)</b> Association Between Population-Level Factors and Household Secondary Attack Rate of SARS-CoV-2: A Systematic Review and Meta-analysis<ul><li> We obtained information on predominant circulating viruses during the study period from the <b>Nextstrain</b> website if such information was not available in the study [<xref rid="ofac676-B13" ref-type="bibr"><u>13</u></xref>].<li> [13](<person-group person-group-type="author"><collab>Nextstrain</collab></person-group>. <article-title>Nextstrain SARS-CoV-2 resources</article-title>. Available at: <ext-link xlink:href="https://nextstrain.org/sars-cov-2/" ext-link-type="uri">https://nextstrain.org/sars-cov-2/</ext-link>. Accessed June 10, 2022.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36654682">36654682</a> <b>(0)</b> Genomic epidemiology of SARS-CoV-2 in Cambodia, January 2020 to February 2021<ul><li> Lineages and specific mutations were determined from consensus sequences using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and PANGOLIN (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) webservers.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36590812">36590812</a> <b>(5)</b> SARS-CoV-2 and animals, a long story that doesn't have to end now: What we need to learn from the emergence of the Omicron variant<ul><li> Data from <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m</ext-link>) show how these VOC have all evolved independently from the original strain, rather than one from another (<xref rid="B19" ref-type="bibr"><u>19</u></xref>).<li> Moreover, Omicron sublineages (identified, for nomenclature easiness reasons, with aliases such as BA.1 to BA.5) arose during 2022 until BA.5 became the most circulating variant globally in the past 6 months (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m</ext-link>, accessed on 26/10/2022), also due to Omicron's shorter incubation periods and maybe milder clinical symptoms (<xref rid="B25" ref-type="bibr"><u>25</u></xref>).<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35444005">35444005</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34968376">34968376</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36632482">36632482</a> <b>(1)</b> Reversions to consensus are positively selected in HIV-1 and bias substitution rate estimates<ul><li> We then performed an MSA, including the reference HIV-1 HXB2 sequence, using Multiple Alignment using Fast Fourier Transform (<xref rid="R28" ref-type="bibr"><u>Katoh and Standley 2013</u></xref>) and the <b>Nextstrain</b> framework (<xref rid="R23" ref-type="bibr">Huddleston et&#xA0;al.<li> [Katoh and Standley 2013](<person-group person-group-type="author"><string-name><surname>Katoh</surname><given-names>K.</given-names></string-name> and <string-name><given-names>M.</given-names><surname>Standley Daron</surname></string-name></person-group> (<year>2013</year>) &#x2018;<article-title>MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability</article-title>&#x2019;, <source><italic toggle="yes">Molecular Biology and Evolution</italic> IV</source>, <volume>30</volume>: <fpage>772</fpage>&#x2013;<lpage>780</lpage>. doi: <pub-id pub-id-type="doi">10.1093/molbev/mst010</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560792">36560792</a> <b>(1)</b> Genomic Epidemiology of SARS-CoV-2 in Western Burkina Faso, West Africa<ul><li> Specifically, curated <b>Nextstrain</b> continent-representative datasets were downloaded from the GISAID database (9 March 2022) for all six continents (2620 to 3937 genomes per continent).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36561179">36561179</a> <b>(3)</b> Read2Tree: scalable and accurate phylogenetic trees from raw reads<ul><li> SARS-CoV-2 samples were obtained from <b>Nextstrain</b> open (<ext-link xlink:href="https://data.<b>nextstrain</b>.org/files/ncov/open/global/metadata.tsv.xz" ext-link-type="uri">https://data.<b>nextstrain</b>.org/files/ncov/open/global/metadata.tsv.xz</ext-link><underline>)</underline><sup><xref rid="R7" ref-type="bibr"><u>7</u></xref></sup>.<li> SRA read accessions together with the clade annotations from <b>nextstrain</b> are available in Supplementary File 1.<li> Nevertheless, the colour clustering in both trees shows that Read2Tree can be used to reliably classify samples according to the fine-grained <b>Nextstrain</b> (Hadfield et al., 2018) classification.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560780">36560780</a> <b>(2)</b> Multiple Mutations Associated with Emergent Variants Can Be Detected as Low-Frequency Mutations in Early SARS-CoV-2 Pandemic Clinical Samples<ul><li> To understand the potential impact of these mutations, the mutations were compared to emergent consensus mutations described in literature and by referencing several highly informative SARS-CoV-2 websites such as outbreak.info, covariants.org, and <b>Nextstrain</b>.org [<xref rid="B1-viruses-14-02775" ref-type="bibr"><u>1</u></xref>,<xref rid="B3-viruses-14-02775" ref-type="bibr"><u>3</u></xref>,<xref rid="B19-viruses-14-02775" ref-type="bibr"><u>19</u></xref>].<li> In particular, ORF1b, Y1464C and P1427L, and ORF8 L84S emerged early as part of clade 19B (<b>Nextstrain</b>.org designation) which includes the reference strain, but quickly disappeared as this clade was replaced by other clades.<li> [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [3](          <person-group person-group-type="author"><name><surname>Latif</surname><given-names>A.A.</given-names></name><name><surname>Mullen</surname><given-names>J.L.</given-names></name><name><surname>Alkuzweny</surname><given-names>M.</given-names></name><name><surname>Tsueng</surname><given-names>G.</given-names></name><name><surname>Cano</surname><given-names>M.</given-names></name><name><surname>Haag</surname><given-names>E.</given-names></name><name><surname>Zhou</surname><given-names>J.</given-names></name><name><surname>Zeller</surname><given-names>M.</given-names></name><name><surname>Hufbauer</surname><given-names>E.</given-names></name><name><surname>Matteson</surname><given-names>N.</given-names></name><etal/></person-group>          <article-title>4 Lineage Report, Outbreak.Info</article-title>          <comment>Available online: <ext-link xlink:href="https://outbreak.info/situation-reports?pango=AY.4.2" ext-link-type="uri">https://outbreak.info/situation-reports?pango=AY.4.2</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-11-23">(accessed on 23 November 2021)</date-in-citation>        ) [19](          <person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name></person-group>          <article-title>Covariants: SARS-CoV-2 Mutations and Variants of Interest</article-title>          <comment>Available online: <ext-link xlink:href="https://covariants.org/" ext-link-type="uri">https://covariants.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-12">(accessed on 12 July 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36579069">36579069</a> <b>(1)</b> Prevalence of SARS-CoV-2 and co-infection with malaria during the first wave of the pandemic (the Burkina Faso case)<ul><li><bold>(<b>Nextstrain</b>)</bold></th>              <th rowspan="1" colspan="1"/>              <th rowspan="1" colspan="1"/>            </tr>          </thead>          <tbody>            <tr>              <td valign="top" align="left" rowspan="1" colspan="1"><ext-link xlink:href="SAMEA110648965" ext-link-type="DDBJ/EMBL/GenBank">SAMEA110648965</ext-link>.<li> Phylogenetic tree generated with Nextclade online software v2.5.0 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) (accessed on 13 September 2022) and visualized using Auspice v2.37.3 (<ext-link xlink:href="https://auspice.us/" ext-link-type="uri">https://auspice.us/</ext-link>) (accessed on 13 September 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36507832">36507832</a> <b>(3)</b> Emergence and Potential Extinction of Genetic Lineages of Human Metapneumovirus between 2005 and 2021<ul><li> For SARS-CoV-2, the Pangolin nomenclature system provides detailed cluster information (<xref rid="B29" ref-type="bibr"><u>29</u></xref>), similar to the Global Initiative on Sharing Avian Influenza Data (GISAID) nomenclature system (<ext-link xlink:href="https://www.gisaid.org/resources/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://www.gisaid.org/resources/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link>), while the <b>Nextstrain</b> nomenclature system offers a more large-scale view of clade trends (<ext-link xlink:href="https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming" ext-link-type="uri">https://<b>nextstrain</b>.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>).<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36507680">36507680</a> <b>(2)</b> Genome Sequences of SARS-CoV-2 Strains from the Republic of Moldova<ul><li><b>Nextstrain</b> clade</th>                <th rowspan="1" colspan="1">No.<li>Phylogenetic analyses were performed through the <b>Nextstrain</b> bioinformatics platform (<xref rid="B12" ref-type="bibr"><u>12</u></xref>).<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560768">36560768</a> <b>(1)</b> Bayesian Molecular Dating Analyses Combined with Mutational Profiling Suggest an Independent Origin and Evolution of SARS-CoV-2 Omicron BA.1 and BA.2 Sub-Lineages<ul><li> The current nomenclature schemes for these variants are the Global Initiative on Sharing All Influenza Data (GISAID) [<xref rid="B1-viruses-14-02764" ref-type="bibr"><u>1</u></xref>], Pango [<xref rid="B2-viruses-14-02764" ref-type="bibr"><u>2</u></xref>], and <b>Nextstrain</b> [<xref rid="B3-viruses-14-02764" ref-type="bibr"><u>3</u></xref>].<li> [1](          <person-group person-group-type="author"><collab>GISIAD</collab></person-group>          <article-title>Clade and Lineage Nomenclature Aids in Genomic Epidemiology Studies of Active hCoV-19 Viruses</article-title>          <comment>Available online: <ext-link xlink:href="https://go.nature.com/3pgSIt6" ext-link-type="uri">https://go.nature.com/3pgSIt6</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-10-21">(accessed on 21 October 2022)</date-in-citation>        ) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [3](          <person-group person-group-type="author"><name><surname>Trevor</surname><given-names>B.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Updated Nextstrain SARS-CoV-2 Clade Naming Strategy</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming" ext-link-type="uri">https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-10-21">(accessed on 21 October 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36451891">36451891</a> <b>(0)</b> SARS-CoV-2 exposure in Norway rats (<italic toggle="yes">Rattus norvegicus)</italic> from New York City<ul><li> Nucleotide sequences were aligned using MAFFT v7.471, and the mutations were analyzed using nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36554028">36554028</a> <b>(1)</b> An Analysis of Body Language of Patients Using Artificial Intelligence<ul><li> HealthMap, UpCode, Thebaselab, <b>NextStrain</b>, the BBC, Johns Hopkins&#x2019; CSSE, and the New York Times have the best dashboards, according to them.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36555300">36555300</a> <b>(13)</b> Quasispecies Analysis of SARS-CoV-2 of 15 Different Lineages during the First Year of the Pandemic Prompts Scratching under the Surface of Consensus Genome Sequences<ul><li> It was observed to be &#x2248;30.4 substitutions/genome/year (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m?l=clock">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m?l=clock</uri> (accessed on 30 September 2022)) [<xref rid="B10-ijms-23-15658" ref-type="bibr"><u>10</u></xref>] or 9.8 &#xD7; 10<sup>&#x2212;4</sup> substitutions per site per year [<xref rid="B11-ijms-23-15658" ref-type="bibr"><u>11</u></xref>].<li> they included 22 genomes of <b>Nextstrain</b> lineage 20A; 9 of lineage 20B, and 6 of lineage 20C (<xref rid="app1-ijms-23-15658" ref-type="app"><u>Supplementary Table S1</u></xref>).<li> and one of the <b>Nextstrain</b> lineage 20A (<xref rid="ijms-23-15658-f002" ref-type="fig"><u>Figure 2</u></xref>).<li>To try correlating at a given nucleotide position the intra-sample genetic diversity in the SARS-CoV-2 genomes studied here and the inter-sample and inter-patient diversity at the largest scale, we superimposed the mean intra-sample genetic diversity for the 110 genomes analyzed here and the global diversity at all SARS-CoV-2 nucleotide positions available from the <b>Nextstrain</b> online tool (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time">https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time</uri> (accessed on 30 September 2022)).<li> Finally, approximately 1 out of 2500 genome positions were identified that showed intra-sample diversity in the present work and concurrently inter-sample diversity between genomes obtained from samples collected worldwide as measured by <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time">https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time</uri> (accessed on 30 September 2022)) [<xref rid="B10-ijms-23-15658" ref-type="bibr"><u>10</u></xref>].<li> The SARS-CoV-2 genotype had been identified on the basis of the consensus genome sequence with the Nextclade tool v1.6.0 (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri> (accessed on 30 September 2022)) [<xref rid="B29-ijms-23-15658" ref-type="bibr"><u>29</u></xref>].<li> To study the correlation between genetic diversity at nucleotide positions either intra-sample or inter-samples, we compared intra-sample diversity in the genomes studied here and genetic diversity at the largest available scale that was recovered as a .tsv file from the <b>Nextstrain</b> website (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time">https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time</uri> (accessed on 30 September 2022)) [<xref rid="B10-ijms-23-15658" ref-type="bibr"><u>10</u></xref>] that displays genomic epidemiology of SARS-CoV-2.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33239633">33239633</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36569096">36569096</a> <b>(2)</b> Nucleic acid testing of SARS-CoV-2: A review of current methods, challenges, and prospects<ul><li> As of June 2021, <b>Nextstrain</b> has identified 13 major clades (19A&#x2013;B, 20A&#x2013;20J and 21A) (<xref rid="B40" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="B60" ref-type="bibr"><b>Nextstrain</b>, 2022</xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36458547">36458547</a> <b>(2)</b> Molecular evolution of SARS&#x2010;CoV&#x2010;2 from December 2019 to August 2022<ul><li>The established nomenclature systems for naming and tracking SARS&#x2010;CoV&#x2010;2 genetic lineages are GISAID (which has allowed the comparison of viral genome variation in various regions of the world), <b>Nextstrain</b>, and Pango.<li>SARS&#x2010;CoV&#x2010;2&#xA0;WGS with available country and year of sampling were downloaded from GenBank and <b>nextstrain</b><xref rid="jmv28366-bib-0020" ref-type="bibr"><sup>20</sup></xref>, <xref rid="jmv28366-bib-0021" ref-type="bibr"><sup>21</sup></xref> (Supporting Information&#xA0;material).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36198312">36198312</a> <b>(6)</b> Multimodal surveillance of SARS-CoV-2 at a university enables development of a robust outbreak response framework<ul><li><hr/></td></tr><tr><td>Study-specific analyses</td><td>GitHub (Broad Institute)</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/broadinstitute/sc2-cmu-study" id="intref0075">https://github.com/broadinstitute/sc2-cmu-study</ext-link></td></tr><tr><td>LoFreq version 2.1.5</td><td><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1093/nar/gks918" id="intref0080">https://doi.org/10.1093/nar/gks918</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://csb5.github.io/lofreq/" id="intref0085">https://csb5.github.io/lofreq/</ext-link></td></tr><tr><td>viral-ngs 2.1.28</td><td>Broad Institute</td><td><ext-link ext-link-type="uri" xlink:href="http://dockstore.org/organizations/BroadInstitute/collections/pgs" id="intref0090">dockstore.org/organizations/BroadInstitute/collections/pgs</ext-link></td></tr><tr><td>Freyja v1.3.4</td><td>GitHub (Andersen Lab)</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/andersen-lab/Freyja" id="intref0095">https://github.com/andersen-lab/Freyja</ext-link></td></tr><tr><td>outbreaker2 (version 1.1.2)</td><td><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1186/s12859-018-2330-z" id="intref0100">https://doi.org/10.1186/s12859-018-2330-z</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/outbreaker2/index.html" id="intref0105">https://cran.r-project.org/web/packages/outbreaker2/index.html</ext-link></td></tr><tr><td><b>Nextstrain</b>.<li><hr/></td></tr><tr><td>Study-specific analyses</td><td>GitHub (Broad Institute)</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/broadinstitute/sc2-cmu-study" id="intref0075">https://github.com/broadinstitute/sc2-cmu-study</ext-link></td></tr><tr><td>LoFreq version 2.1.5</td><td><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1093/nar/gks918" id="intref0080">https://doi.org/10.1093/nar/gks918</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://csb5.github.io/lofreq/" id="intref0085">https://csb5.github.io/lofreq/</ext-link></td></tr><tr><td>viral-ngs 2.1.28</td><td>Broad Institute</td><td><ext-link ext-link-type="uri" xlink:href="http://dockstore.org/organizations/BroadInstitute/collections/pgs" id="intref0090">dockstore.org/organizations/BroadInstitute/collections/pgs</ext-link></td></tr><tr><td>Freyja v1.3.4</td><td>GitHub (Andersen Lab)</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/andersen-lab/Freyja" id="intref0095">https://github.com/andersen-lab/Freyja</ext-link></td></tr><tr><td>outbreaker2 (version 1.1.2)</td><td><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1186/s12859-018-2330-z" id="intref0100">https://doi.org/10.1186/s12859-018-2330-z</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/outbreaker2/index.html" id="intref0105">https://cran.r-project.org/web/packages/outbreaker2/index.html</ext-link></td></tr><tr><td><b>Nextstrain</b></td><td><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1093/bioinformatics/bty407" id="intref0110">https://doi.org/10.1093/bioinformatics/bty407</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="http://github.com/<b>nextstrain</b>/ncov" id="intref0115">github.com/<b>nextstrain</b>/ncov</ext-link>.<li> We used entropy data from a CDC-curated <b>Nextstrain</b> analysis focused on data from Colorado as of August 2021.<li>Using the <b>Nextstrain</b> augur pipeline, a maximum likelihood (ML) tree was created via IQ-Tree using a GTR mutation model, as well as a time-resolved tree via TreeTime, both rooted to the ancestral reference genome, NC_045512.2.<li> The contextual genomes were obtained as part of the open dataset of pre-aligned sequences curated by <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36962792">36962792</a> <b>(1)</b> Genomics-informed outbreak investigations of SARS-CoV-2 using civet<ul><li> <b>NextStrain</b> takes an alternative approach and downsamples the dataset heavily, leaving a manageable amount of data to display [<xref rid="pgph.0000704.ref012" ref-type="bibr"><u>12</u></xref>].<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36552295">36552295</a> <b>(2)</b> Interpretable and Predictive Deep Neural Network Modeling of the SARS-CoV-2 Spike Protein Sequence to Predict COVID-19 Disease Severity<ul><li> This metadata file included the collection date, annotated Pangolin lineage [<xref rid="B55-biology-11-01786" ref-type="bibr"><u>55</u></xref>], <b>Nextstrain</b>-identified clade, and geographical information.<li> All records further include additional metadata: host, continent/country/region of collection, Pango nomenclature lineage, <b>NextStrain</b> clade, sample collection and submission date, patient age, and patient gender.<li> [55](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560738">36560738</a> <b>(7)</b> Molecular Epidemiology of the Norwegian SARS-CoV-2 Delta Lineage AY.63<ul><li>Phylogenetic analyses of Delta sequences were conducted using <b>NextStrain</b> [<xref rid="B18-viruses-14-02734" ref-type="bibr"><u>18</u></xref>] and its specific pipeline for SARS-CoV-2 (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</uri>; accessed on 10 October 2022).<li> To visualize AY.63 in the context of all Delta strains, the <b>NextStrain</b> analysis was set to include all AY.63 sequences and to subsample evenly across all other lineages.<li> To focus on AY.63, the <b>NextStrain</b> analysis was set to include all sequences from the AY.63 dataset and to sample only a small selection of the most closely related Delta sequences to provide a reasonable outgroup.<li>To calculate pairwise SNP distances, sequence alignments were generated for each of the lineages B.617.2, AY.63, AY.4, AY.43, AY.122, and AY.127 using <b>NextStrain</b> [<xref rid="B18-viruses-14-02734" ref-type="bibr"><u>18</u></xref>] on the same dataset as described above.<li> As inputs for the analysis, the phylogenetic trees and metadata files generated by <b>NextStrain</b> were used.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36481486">36481486</a> <b>(1)</b> Host microRNAs exhibit differential propensity to interact with SARS-CoV-2 and variants of concern<ul><li> 2020&#x2013;2022 <b>Nextstrain</b> developers <xref rid="bb0165" ref-type="bibr"><u>[33]</u></xref>.<li> [[33]](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36473508">36473508</a> <b>(3)</b> Structural patterns of SARS-CoV-2 variants of concern (alpha, beta, gamma, delta) spike protein are influenced by variant-specific amino acid mutations: A computational study with implications on viral evolution<ul><li> Network design following the coronavirus disease 2019 (COVID-19) genomic epidemiology based on the <b>Nextstrain</b> website (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org" id="ir005"><b>nextstrain</b>.org</ext-link>) between December 2019- July 2021 was analyzed with a particular focus on the chronological occurrence of mutating residues among the SARS2 VOC (alpha, beta, gamma, delta) spike proteins.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560718">36560718</a> <b>(2)</b> ViralVar: A Web Tool for Multilevel Visualization of SARS-CoV-2 Genomes<ul><li> To determine if these clusters can also be identified using phylogenetic analysis, we examined these mutations using the <b>Nextstrain</b> webserver (<xref rid="viruses-14-02714-f005" ref-type="fig"><u>Figure 5</u></xref>D).<li> The time-resolved phylogenetic trees from <b>Nextstrain</b> suggest that the Spike K1191N mutation is monophyletic, whereas the P227L mutation arose in at least two distinct branches (<xref rid="viruses-14-02714-f005" ref-type="fig"><u>Figure 5</u></xref>D).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36499592">36499592</a> <b>(7)</b> Genetic and Structural Data on the SARS-CoV-2 Omicron BQ.1 Variant Reveal Its Low Potential for Epidemiological Expansion<ul><li>Phylogenomic reconstruction (<xref rid="ijms-23-15264-f001" ref-type="fig"><u>Figure 1</u></xref>) indicates that the genomes of BQ.1 (GSAID Clade 22E) are clustered within the not-monophyletic GSAID Clade 21L, with a close relationship with its direct progenitor BA.5 (See <xref rid="ijms-23-15264-t001" ref-type="table"><u>Table 1</u></xref> for details on <b>Nextstrain</b> clade, Pango lineage and WHO labels).<li>The original Omicron, named 21K and BA.1, according to <b>Nextstrain</b> clade and Pango lineage nomenclature, respectively (<uri xlink:href="https://www.who.int/activities/tracking-SARS-CoV-2-variants">https://www.who.int/activities/tracking-SARS-CoV-2-variants</uri>, accessed on 5 November 2022), appears to have arisen in November 2021, probably in South Africa [<xref rid="B17-ijms-23-15264" ref-type="bibr"><u>17</u></xref>].<li>The first genomic epidemiology of the SARS-CoV-2 BQ.1 Omicron variant (GSAID Clade 22E) was reconstructed by using a subsampling focused globally over the past 6 months, built with <b>nextstrain</b>/ncov (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</uri>, accessed on 9 November 2022), available at <uri xlink:href="https://gisaid.org/phylodynamics/global/<b>nextstrain</b>/">https://gisaid.org/phylodynamics/global/<b>nextstrain</b>/</uri> (accessed on 9 November 2022), including all genomes belonging to the GSAID Clade 21 M (Omicron).<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34897752">34897752</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36846035">36846035</a> <b>(4)</b> SARS-CoV-2 Monitoring on Mink Farms in Poland<ul><li> Sequences of SARS-CoV-2 were analysed with use of online tools: the <b>Nextstrain</b> database (<ext-link xlink:href="http://https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org),</ext-link> Pangolin (<ext-link xlink:href="http://https://Pangolin.cog-uk.io/" ext-link-type="uri">https://Pangolin.cog-uk.io/),</ext-link> the Global Initiative on Sharing All Influenza Data (GISAID &#x2013; <ext-link xlink:href="http://https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>) and the University of California Santa Cruz Genome Browser (<ext-link xlink:href="http://https://hgwdev.gi.ucsc.edu/index.html" ext-link-type="uri">https://hgwdev.gi.ucsc.edu/index.html).<li> <ext-link xlink:href="http://https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>) and the University of California Santa Cruz Genome Browser (<ext-link xlink:href="http://https://hgwdev.gi.ucsc.edu/index.html" ext-link-type="uri">https://hgwdev.gi.ucsc.edu/index.html).</ext-link> Phylogenetic analysis, determination of the clade and lineage affiliation of individual virus strains and matching of the most closely related genomes were performed in the GISAID EpiCoV database (<ext-link xlink:href="http://https://www.epicov.org/,version" ext-link-type="uri">https://www.epicov.org/,version</ext-link> 2022-04-05) and verified in <b>Nextstrain</b> (<ext-link xlink:href="http://https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) databases (v1.14.0).<li> The tree was processed with the Nextclade open source tool (<ext-link xlink:href="http://https://clades.<b>nextstrain</b>.org/tree" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/tree</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36582501">36582501</a> <b>(1)</b> Genome Evolution and Early Introductions of the SARS-CoV-2 Omicron Variant in Mexico<ul><li>A time-scaled phylogenetic tree was built using TreeTime v0.7.4 (<xref rid="R42" ref-type="bibr"><u>Sagulenko, Puller, and Neher 2018</u></xref>), which specified a clock rate of 8&#x2009;&#xD7;&#x2009;10<sup>&#x2212;4</sup> substitutions per site per year according to the <b>Nextstrain</b> workflow (<xref rid="R17" ref-type="bibr">Hadfield et&#xA0;al.<li> [Sagulenko, Puller, and Neher 2018](<person-group person-group-type="author"><string-name><surname>Sagulenko</surname><given-names>P.</given-names></string-name>, <string-name><surname>Puller</surname><given-names>V.</given-names></string-name>, and <string-name><surname>Neher</surname><given-names>R. A.</given-names></string-name></person-group> (<year>2018</year>) &#x2018;<article-title>TreeTime: Maximum-Likelihood Phylodynamic Analysis</article-title>&#x2019;, <source><italic toggle="yes">Virus Evolution</italic></source>, <volume>4</volume>: <fpage>1</fpage>&#x2013;<lpage>9</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36322955">36322955</a> <b>(0)</b> Probable Aerosol Transmission of SARS-CoV-2 through Floors and Walls of Quarantine Hotel, Taiwan, 2021<ul><li> We processed the obtained viral sequences using DRAGEN COVID Lineage application version 3.5.5 on Illumina&#x2019;s Basespace cloud analytical system and compared those with Nextclade software (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36178142">36178142</a> <b>(0)</b> Possible Occupational Infection of Healthcare Workers with Monkeypox Virus, Brazil<ul><li> Analysis using the Nextclade tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) showed that the sequenced genomes were 100% identical and belonged to MPXV clade IIb sublineage B.1.1 (<xref rid="SD1" ref-type="supplementary-material"><u>Appendix</u></xref> Figures 1&#x2013;3).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36695462">36695462</a> <b>(3)</b> Severe mpox (formerly monkeypox) disease in five patients after recent vaccination with MVA-BN vaccine, Belgium, July to October 2022<ul><li> Phylogenetic analysis assigned the genomes to both existing and different MPXV Clade IIb lineages (<ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link> and <xref rid="f2" ref-type="fig"><u>Figure 2</u></xref>).<li> Our four genomes were also analysed by <b>Nextstrain</b> (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>), which assigned following lineages: B.1 to PEPV patient and PPV patient 4; B.1.3 to PPV patient 1; B.1.7 to PPV patient 2.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36568782">36568782</a> <b>(4)</b> Detection of SARS-CoV-2 infection clusters: The useful combination of spatiotemporal clustering and genomic analyses<ul><li>Pairwise single nucleotide variant (SNV) distances were computed from quality-checked sequences using <b>Nextstrain</b> SARS-CoV-2 multiple sequence alignment (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="B35" ref-type="bibr"><u>35</u></xref>) and pairsnp (<ext-link xlink:href="https://github.com/gtonkinhill/pairsnp" ext-link-type="uri">https://github.com/gtonkinhill/pairsnp</ext-link>).<li>Phylogenetic analysis and visualization were conducted with Augur and Auspice, respectively, which are parts of <b>Nextstrain</b> that allows for customization and interactive web visualization (<xref rid="B35" ref-type="bibr"><u>35</u></xref>).<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36456577">36456577</a> <b>(2)</b> Risk of SARS-CoV-2 reinfection: a systematic review and meta-analysis<ul><li>The first viral genome belongs to GISAID clade V, <b>Nextstrain</b> clade 19A, and Pangolin lineage B.2 with a probability of 0.99.<li> The second viral genome belongs to GISAID clade G, <b>Nextstrain</b> clade 20A, and Pangolin lineage B.1.79 with a probability of 0.70.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36447206">36447206</a> <b>(2)</b> Identification of herbal teas and their compounds eliciting antiviral activity against SARS-CoV-2 in vitro<ul><li> Delta was the most prevalent variant in 2021, but in 2022, Omicron has overtaken Delta as the predominant variant (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>, [<xref ref-type="bibr" rid="CR21"><u>21</u></xref>]).<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36547242">36547242</a> <b>(3)</b> Genomic Evidence Suggests Viral Persistence of SARS-CoV-2 for 386 Days in Health Worker: A Case Report from Santiago of Chile<ul><li> SARS-CoV-2 genome-wide analysis indicated that the SARS-CoV-2 variant in the second COVID-19 diagnosis was classified in the clade <b>Nextstrain</b> 20A, with 0% circulation in Chile, from May 2021 [<xref rid="B7-idr-14-00096" ref-type="bibr"><u>7</u></xref>].<li> The analysis using RNA sequencing of both positive samples (0806-191 and 230921-099) indicated that both sequences belong to the same clade <b>Nextstrain</b> 20A lineage (<xref rid="idr-14-00096-f001" ref-type="fig"><u>Figure 1</u></xref>).<li> The results and analysis indicated that it belonged to the Clade <b>Nextstrain</b> 20A lineage, one of the reported SARS-CoV-2 circulating in Chile.<li> [7](          <article-title>Consorcio Genomas COV-2. Secuencias de SARS-CoV-2 de Chile</article-title>          <comment>Available online: <ext-link xlink:href="https://auspice.cov2.cl/ncov/chile-global" ext-link-type="uri">https://auspice.cov2.cl/ncov/chile-global</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-08-15">(accessed on 15 August 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36448812">36448812</a> <b>(10)</b> Genome Sequence of a Recombinant SARS-CoV-2 Lineage XAM (BA.1.1/BA.2.9) Strain from a Clinical Sample in Santo Domingo, Dominican Republic<ul><li> The phylogenetic tree was built using <b>Nextstrain</b> (<xref rid="B17" ref-type="bibr"><u>17</u></xref>).<li><ext-link xlink:href="https://github.com/<b>nextstrain</b>/augur" ext-link-type="uri">https://github.com/<b>nextstrain</b>/augur</ext-link>.<li><ext-link xlink:href="https://github.com/<b>nextstrain</b>/nextclade" ext-link-type="uri">https://github.com/<b>nextstrain</b>/nextclade</ext-link>.<li><b>Nextstrain</b>.<li><ext-link xlink:href="https://github.com/<b>nextstrain</b>/<b>nextstrain</b>.org" ext-link-type="uri">https://github.com/<b>nextstrain</b>/<b>nextstrain</b>.org</ext-link>.<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36448779">36448779</a> <b>(0)</b> Epidemiological and Genomic Analysis of a Large SARS-CoV-2 Outbreak in a Long-Term Care Facility in Catalonia, Spain<ul><li> Amino acid substitutions and deletions were identified with Nextclade v0.14.1 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560693">36560693</a> <b>(0)</b> What&#x2019;s Next for Flu? Out-of-Season Circulation of Influenza Viruses in Southern Italy, August 2022<ul><li> Amino acid substitutions were identified by comparing the HA gene sequences of the identified strains and the HA gene sequences of the reference strains through the Nextclade tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 30 November 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36467017">36467017</a> <b>(1)</b> Pathogen diversity in meta-population networks<ul><li> The right vertical coordinates, e.g.&#xA0;22C, 22B, are the nomenclature of <b>Nextstrain</b>&#xA0;<xref rid="b50" ref-type="bibr"><u>[50]</u></xref> for the SARS-CoV-2 mutant strain.<li> [[50]](Nextstrain, <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/gisaid/global/all-time" id="interref4">https://nextstrain.org/ncov/gisaid/global/all-time</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36507266">36507266</a> <b>(1)</b> Nanomaterials to combat SARS-CoV-2: Strategies to prevent, diagnose and treat COVID-19<ul><li> <xref rid="B442" ref-type="bibr"><u>Lucas et al., 2020a</u></xref>), started after at that moment by fostering a continuous communication and classification of the obtained viral sequences through different resources (e.g., GISAID, GenBank, <b>Nextstrain</b>, Pangolin, among others) (<xref rid="B257" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="B28" ref-type="bibr"><u>Andersen et al., 2020</u></xref>; <xref rid="B422" ref-type="bibr"><u>Liu et al., 2020d</u></xref>; <xref rid="B345" ref-type="bibr"><u>Khare et al., 2021</u></xref>; <xref rid="B536" ref-type="bibr"><u>O'Toole et al., 2021</u></xref>; <xref rid="B637" ref-type="bibr"><u>Sayers et al., 2021</u></xref>; <xref rid="B711" ref-type="bibr"><u>Underwood et al., 2022</u></xref>)), where WGS, GWAS (genome-wide association study) and nanopore sequencing approaches and technologies have gain relevance for surveillance and diagnose (<xref rid="B37" ref-type="bibr"><u>Arora et al., 2022</u></xref>) (<xref rid="F4" ref-type="fig"><u>Figure 4</u></xref>).<li> [Lucas et al., 2020a](<person-group person-group-type="author"><name><surname>Lucas</surname><given-names>C.</given-names></name><name><surname>Klein</surname><given-names>J.</given-names></name><name><surname>Sundaram</surname><given-names>M.</given-names></name><name><surname>Liu</surname><given-names>F.</given-names></name><name><surname>Wong</surname><given-names>P.</given-names></name><name><surname>Silva</surname><given-names>J.</given-names></name><etal/></person-group> (<year>2020a</year>). <article-title>Kinetics of antibody responses dictate COVID-19 outcome</article-title>. <comment>medRxiv</comment>. <pub-id pub-id-type="doi">10.1101/2020.12.18.20248331</pub-id>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Andersen et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32284615">32284615</a>) [Liu et al., 2020d](<person-group person-group-type="author"><name><surname>Liu</surname><given-names>W.</given-names></name><name><surname>Zhang</surname><given-names>Q.</given-names></name><name><surname>Chen</surname><given-names>J.</given-names></name><name><surname>Xiang</surname><given-names>R.</given-names></name><name><surname>Song</surname><given-names>H.</given-names></name><name><surname>Shu</surname><given-names>S.</given-names></name><etal/></person-group> (<year>2020d</year>). <article-title>Detection of covid-19 in children in early january 2020 in wuhan, China</article-title>. <source>N. Engl. J. Med. Overseas. Ed.</source><volume>382</volume>, <fpage>1370</fpage>&#x2013;<lpage>1371</lpage>. <pub-id pub-id-type="doi">10.1056/nejmc2003717</pub-id>) [Khare et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34934514">34934514</a>) [O'Toole et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [Sayers et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33196830">33196830</a>) [Underwood et al., 2022](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35230423">35230423</a>) [Arora et al., 2022](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35477025">35477025</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560632">36560632</a> <b>(1)</b> Evaluation of Rapid Lateral-Flow Tests Directed against the SARS-CoV-2 Nucleoprotein Using Viral Suspensions Belonging to Different Lineages of SARS-CoV-2<ul><li>The selected, different SARS-CoV-2 strains were sequenced and deposited at GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri> accessed 20 November 2022) with the following accession numbers (<b>Nextstrain</b>/Pango classifications): EPI_ISL_1707038 (19A/B.38); EPI_ISL_1785075 (20A/B.1.160); EPI_ISL_1707039 Alpha variant (20I/B.1.1.7); EPI_ISL_768828 Beta variant (20H/B.1.351); EPI_ISL_1359892 Gamma variant (20J/P.1); EPI_ISL_1904989 Delta variant (21A/B.1.617.2); and EPI_ISL_7608613 Omicron variant (21K/BA.1).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36483099">36483099</a> <b>(1)</b> Global challenge with the SARS-CoV-2 omicron BA.2 (B.1.1.529.2) subvariant: Should we be concerned?<ul><li>According to <b>Nextstrain</b> online server data, the omicron variant comprises three distinct branches: 21K (or Pangolin lineage12 BA.1), 21L (or BA.2), and BA.3 of the 21M omicron clade[<xref rid="B4" ref-type="bibr"><u>4</u></xref>] (Figure <xref rid="F4" ref-type="fig"><u>4</u></xref>).<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35680610">35680610</a>) </ul></p>
<p><u>No PMID</u> <b>(2)</b> Third booster vaccination and stopping the Omicron, a new variant of concern<ul><li>One and a half years have passed since the emergence of SARS-CoV-2 (severe acute respiratory syndrome Coronavirus 2), while according to WHO, more than 418 million infected cases and 5.85 million deaths have been recorded globally due to different variants of this virus.<xref rid="bb0045" ref-type="bibr"><sup>9</sup></xref> Omicron is the latest variant of concern (VOC, Pango lineage B.1.1.529, <b>Nextstrain</b> clade identifier 21K) that was first characterized and reported on November 2, 2021 from Botswana, South Africa (GISAID sequence accession ID: EPI_ISL_8182767).<li> Although many countries now have banned travel from South Africa, continuous monitoring of <b>Nextstrain</b> databases reflects the unbridled expansion of the VOC Omicron around the world so that it has replaced the earlier Delta variant (<xref rid="f0005" ref-type="fig">Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36434068">36434068</a> <b>(1)</b> Comparison of antibody responses to SARS-CoV-2 variants in Australian children<ul><li> The Delta cohort was recruited between July 2021 and November 2021, while the Omicron cohort was recruited between January 2022-July 2022 based on data from <b>Nextstrain</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36435688">36435688</a> <b>(2)</b> Measurably recombining malaria parasites<ul><li> see <ext-link xlink:href="https://<b>nextstrain</b>.org/pathogens" ext-link-type="uri">https://<b>nextstrain</b>.org/pathogens</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36817495">36817495</a> <b>(0)</b> Systematic sequencing of imported cases leads to detection of SARS-CoV-2 B.1.1.529 (Omicron) variant in central Viet Nam<ul><li> The Dragen Covid Lineage software (Illumina) was used for genomic analysis and phylogenetic trees were generated using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36418984">36418984</a> <b>(0)</b> Convalescent or standard plasma versus standard of care in the treatment of COVID-19 patients with respiratory impairment: short and long-term effects. A&#xA0;three-arm randomized controlled clinical trial<ul><li> CCP was collected in May and June 2020 from donors recovered from first-wave COVID-19 infection, when the predominant variant in our area was the 20A S614G lineage (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36415455">36415455</a> <b>(2)</b> Resistance of Omicron subvariants BA.2.75.2, BA.4.6 and BQ.1.1 to neutralizing antibodies<ul><li> Sequences of the variants after one passage on IGROV-1 cells identified BA.4.6 and BQ.1.1 (Pango lineage B.1.1.529.4.6 and B.1.1.529.5.3.1.1.1.1.1.1, respectively according to <b>Nextstrain</b>, GISAID accession ID: BA.4.6: EPI_ISL_15729633 and BQ.1.1: EPI_ISL_15731523), indicating that no adaptative mutations were generated during this short culture period.<li> Sequencing of the virus confirmed the presence of BA.2.75.2 (Pango lineage B.1.1.529.2.75.2, according to <b>Nextstrain</b>, GISAID accession ID: E EPI_ISL_15731524).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36451887">36451887</a> <b>(6)</b> Evolution of the SARS-CoV-2 mutational spectrum<ul><li> We tallied counts separately for each <b>Nextstrain</b> clade (<xref rid="R1" ref-type="bibr">Aksamentov et al.<li> 2021</xref>) to subset the mutation-annotated tree on samples from each <b>Nextstrain</b> clade, and then extract the mutations on each branch of the subsetted mutation-annotated trees.<li> We next tallied the counts of each mutation on all branches for that clade, excluding mutations on any branches with &gt;4 total mutations, &gt;1 mutation that was a reversion to either the Wuhan-Hu-1 reference genome (Genbank <ext-link xlink:href="NC_045512.2" ext-link-type="DDBJ/EMBL/GenBank">NC_045512.2</ext-link>), or &gt;1 mutation that was a reversion to the founder for that <b>Nextstrain</b> clade as defined by (<xref rid="R33" ref-type="bibr"><u>Neher 2022</u></xref>) (see <ext-link xlink:href="https://github.com/neherlab/SC2_variant_rates/blob/62c525dc4238385ec0755b40658f3007fdbfab1a/data/clade_gts.json" ext-link-type="uri">https://github.com/neherlab/SC2_variant_rates/blob/62c525dc4238385ec0755b40658f3007fdbfab1a/data/clade_gts.json</ext-link>).<li>The predicted and observed nucleotide frequencies for human influenza virus in <xref rid="F3" ref-type="fig"><u>Figure 3C</u></xref> were calculated from phylogenetic analyses of the different influenza virus lineages at <ext-link xlink:href="https://<b>nextstrain</b>.org/groups/neherlab" ext-link-type="uri"><b>nextstrain</b>.org/groups/neherlab</ext-link> and influenza reference genomes used in the <b>nextstrain</b> seasonal influenza workflow.<li> [Neher 2022](<name><surname>Neher</surname><given-names>RA</given-names></name>. <year>2022</year>. <source>Contributions of adaptation and purifying selection to SARS-CoV-2 evolution</source>. :<comment>2022.08.22.504731.</comment><comment>Available from: <ext-link xlink:href="https://www.biorxiv.org/content/10.1101/2022.08.22.504731v1" ext-link-type="uri">https://www.biorxiv.org/content/10.1101/2022.08.22.504731v1</ext-link></comment>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36409093">36409093</a> <b>(12)</b> Tracing the First Days of the SARS-CoV-2 Pandemic in Greece and the Role of the First Imported Group of Travelers<ul><li>Demographic characteristics of the data set and taxonomic features of the isolated SARS-CoV-2 sequences, using the Pango lineage (26) and the <b>Nextstrain</b> clade (25) nomenclature.<li><b>Nextstrain</b> clade</th>                  <th rowspan="1" colspan="1">GISAID accession</th>                </tr>              </thead>              <tbody>                <tr>                  <td rowspan="1" colspan="1">Imported group from Egypt &#x2003;and Israel to Greece.<li> As for the molecular taxonomy of the sequences derived from the group, all of the sequences were identified as B.1 Pango lineage and 20A <b>Nextstrain</b> clade.<li> The molecular taxonomy presented for each MTC is based on <b>Nextstrain</b>&#x2019;s Clade nomenclature.<li> As presented in <xref rid="fig2" ref-type="fig"><u>Fig.&#xA0;2</u></xref>, the distinct five clusters were classified under the 19A, 20A, 20B, and 20C <b>Nextstrain</b> clades.<li> All sequences of this cluster are classified as 20A <b>Nextstrain</b> clade and all sequences of the group fall within it.<li> The largest molecular transmission cluster (MTC) is classified under the 20A <b>Nextstrain</b> clade and consists of 39 sequences, including all the sequences collected from the imported cases (23/39, 58.97%) and 16 (41.03%) sequences from Greece obtained during the subsampling.<li> Yellow arrows depict the <b>Nextstrain</b>&#x2019;s Clade in which the sequences are classified.<li> Yellow arrows depict the <b>Nextstrain</b>&#x2019;s Clade in which the sequences are classified.<li> command that is available from <b>Nextstrain</b>&#x2019;s &#x201C;augur&#x201D; pipeline (<xref rid="B29" ref-type="bibr"><u>29</u></xref>).<li>NextClade CLI (Version 1.11.0) (<xref rid="B38" ref-type="bibr"><u>38</u></xref>) was used in order to perform quality control (QC) of the FASTA sequences, assignment to a <b>Nextstrain</b> clade and nucleotide substitution profile, and identification of changes in the viral proteins relative to the SARS-CoV-2 reference.<li> <b>Nextstrain</b>&#x2019;s molecular taxonomy nomenclature is based on a year-letter clade assignment when (i) a clade reaches &gt;20% global frequency for 2 or more months; (ii) a clade reaches &gt;30% regional frequency for 2 or more months; or (iii) a variant of concern (VOC) (<xref rid="B39" ref-type="bibr"><u>39</u></xref>) is recognized (<xref rid="B40" ref-type="bibr"><u>40</u></xref>).<li> [29](<person-group person-group-type="author"><string-name><surname>Huddleston</surname><given-names>J</given-names></string-name>, <string-name><surname>Hadfield</surname><given-names>J</given-names></string-name>, <string-name><surname>Sibley</surname><given-names>TR</given-names></string-name>, <string-name><surname>Lee</surname><given-names>J</given-names></string-name>, <string-name><surname>Fay</surname><given-names>K</given-names></string-name>, <string-name><surname>Ilcisin</surname><given-names>M</given-names></string-name></person-group>, <etal>et al</etal>. <year>2021</year>. <article-title>Augur: a bioinformatics toolkit for phylogenetic analyses of human pathogens</article-title>. <source>J Open Source Softw</source><volume>6</volume>. doi:<pub-id pub-id-type="doi">10.21105/joss.02906</pub-id>.) [38](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. Available at <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>. doi:<pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>.) [39](<collab>World Health Organization</collab>. <year>2022</year>. <article-title>Tracking SARS-CoV-2 variants</article-title>. Available at <ext-link xlink:href="https://www.who.int/activities/tracking-SARS-CoV-2-variants" ext-link-type="uri">https://www.who.int/activities/tracking-SARS-CoV-2-variants</ext-link>.) [40](<person-group person-group-type="author"><string-name><surname>Bedford</surname><given-names>T</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Updated Nextstrain SARS-CoV-2 clade naming strategy</article-title>. Available at <ext-link xlink:href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming" ext-link-type="uri">https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36409091">36409091</a> <b>(3)</b> Two-Period Study Results from a Large Italian Hospital Laboratory Attesting SARS-CoV-2 Variant PCR Assay Evolution<ul><li> Sequencing reads were analyzed using the SOPHiA GENETICS platform for COVID-19, Covid Analyst software (SmartSeq s.r.l., Novara, Italy), or Illumina DRAGEN COVIDSeq Pipeline software as appropriate, and their quality was checked using <b>NextStrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36422639">36422639</a> <b>(5)</b> Re-Emergence of Circulation of Seasonal Influenza during COVID-19 Pandemic in Russia and Receptor Specificity of New and Dominant Clade 3C.2a1b.2a.2 A(H3N2) Viruses in 2021&#x2013;2022<ul><li>Influenza circulation in Russia and the world decreased by the end of January, which coincided with the peak of the SARS-CoV-2 Omicron wave (<uri>worldometers.info/coronavirus/country/russia/</uri> (accessed on 1 September 2022), <uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri> (accessed on 1 September 2022)).<li> The rising dominance of the 5a2 group globally is expected based on the circulation dynamic, which shows a continuous increase in circulation of this group in 2022 based on analysis by the <b>NextStrain</b> of available genetic information in GISAID (Next Strain <uri xlink:href="https://<b>nextstrain</b>.org/flu/seasonal/h1n1pdm/ha/2y">https://<b>nextstrain</b>.org/flu/seasonal/h1n1pdm/ha/2y</uri>, accessed on 1 September 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36415680">36415680</a> <b>(1)</b> Cases of BA.2.75 and recent BA.2.75.2 subvariant of Omicron are increasing in India: Is it alarming at the global level?<ul><li> Fig.1b and 1c were developed using the <b>Nextstrain</b> server.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36467724">36467724</a> <b>(0)</b> Survival and detection of SARS-CoV-2 variants on dry swabs post storage<ul><li> Sequence data (FASTA files) were then uploaded to <uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> for variant determination.<li> Raw sequences were then uploaded to <uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> for variant determination.<li> <bold>(A)</bold> The genome of each strain was sequenced and the strain type confirmed using <uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36423054">36423054</a> <b>(1)</b> Clinical Presentation of COVID-19 and Antibody Responses in Bangladeshi Patients Infected with the Delta or Omicron Variants of SARS-CoV-2<ul><li> The variants are assigned Greek alphabetical letters by the WHO [<xref rid="B2-vaccines-10-01959" ref-type="bibr"><u>2</u></xref>] and cross-referenced to the clade nomenclature of international reference network laboratories GISAID, <b>Nextstrain</b>, and Pango [<xref rid="B3-vaccines-10-01959" ref-type="bibr"><u>3</u></xref>].<li> [2](          <person-group person-group-type="author"><collab>World Health Organization</collab></person-group>          <article-title>Tracking SARS-CoV-2 Variants</article-title>          <comment>Available online: <ext-link xlink:href="https://www.who.int/activities/tracking-SARS-CoV-2-variants" ext-link-type="uri">https://www.who.int/activities/tracking-SARS-CoV-2-variants</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-06-29">(accessed on 29 June 2022)</date-in-citation>        ) [3](          <person-group person-group-type="author"><collab>Pango Network</collab></person-group>          <article-title>Rules for the Designation and Naming of Pango Lineages</article-title>          <comment>Available online: <ext-link xlink:href="https://www.pango.network/the-pangonomenclature-system/statement-of-nomenclature-rules/" ext-link-type="uri">https://www.pango.network/the-pangonomenclature-system/statement-of-nomenclature-rules/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-06-29">(accessed on 29 June 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36398277">36398277</a> <b>(2)</b> SARS-CoV-2 and the role of close contact in transmission: a systematic review<ul><li>Phylogenetic relationships, including distinct<break/>clustering of viral whole genome sequences,<break/>were inferred based on nucleotide<break/>differences via IQ-TREE using general time<break/>reversible substitution models as a part of<break/>the <b>Nextstrain</b> workflow.<li> Clade<break/>annotation was included for the Pangolin,<break/>GISAID and <b>NextStrain</b> systems.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36395093">36395093</a> <b>(4)</b> Molecular source attribution<ul><li> Much of this knowledge translation was mediated through the open-source project <ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri"><b>Nextstrain</b></ext-link> [<xref rid="pcbi.1010649.ref092" ref-type="bibr"><u>92</u></xref>] that performs phylogenetic analyses on pathogen sequence data as they become available on public and access-restricted databases, and uses the results to update web documents in real time.<li> On March 4, 2020, <b>Nextstrain</b> developers released a phylogeny in which a SARS-CoV-2 genome that was isolated from a German patient occupied an ancestral position relative to a monophyletic clade of sequences sampled from Europe and Mexico.<li> Users of the <ext-link xlink:href="https://en.wikipedia.org/wiki/Twitter" ext-link-type="uri">Twitter</ext-link> social media platform soon commented on the related post from <b>Nextstrain</b> that onward transmission from the German individual seemed to have "led directly to some fraction of the widespread outbreak circulating in Europe today" [<xref rid="pcbi.1010649.ref093" ref-type="bibr"><u>93</u></xref>].<li> [92](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [93](Trevor Bedford [@trvrb]. Incredibly, it appears that this cluster containing Germany/BavPat1/2020 is the direct ancestor of these later viruses and thus led directly to some fraction of the widespread outbreak circulating in Europe today. 5/7 <ext-link xlink:href="https://t.co/mobsvnx2PZ" ext-link-type="uri">https://t.co/mobsvnx2PZ</ext-link>. In: Twitter [Internet]. 2020. Available: <ext-link xlink:href="https://twitter.com/trvrb/status/1235105849841872897" ext-link-type="uri">https://twitter.com/trvrb/status/1235105849841872897</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36395156">36395156</a> <b>(1)</b> Analysis of two choir outbreaks acting in concert to characterize long- range transmission risks through SARS-CoV-2, Berlin, Germany, 2020<ul><li>Phylogenetic trees were constructed by aligning the consensus sequences to chosen SARS-CoV-2 genomes sampled before April 2020 and retrieved from GISAID (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org</ext-link>), maximum likelihood analysis with MAFFT and visualization with Auspice, as described by the standard protocol for analysis of SARS-CoV-2 genomes provided by <b>Nextstrain</b> (<ext-link xlink:href="https://www.<b>nextstrain</b>.org/" ext-link-type="uri">https://www.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36383605">36383605</a> <b>(1)</b> SARS-CoV-2 variant Alpha has a spike-dependent replication advantage over the ancestral B.1 strain in human cells with low ACE2 expression<ul><li> T<sup>716</sup>I mutation is a uniquely VOC Alpha-defining mutation (<b>Nextstrain</b> analysis conducted in August 2022), suggesting that the T<sup>716</sup>I-related spike stabilization strategy is unique to VOC Alpha.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36423141">36423141</a> <b>(2)</b> Challenges and Opportunities for Global Genomic Surveillance Strategies in the COVID-19 Era<ul><li> <b>Nextstrain</b> is one such example, where the software can take raw sequence data and offer flexible analyses that output interpretable results for researchers to understand the genome of their specific SARS-CoV-2 samples [<xref rid="B56-viruses-14-02532" ref-type="bibr"><u>56</u></xref>].<li> The messaging platforms used to relay surveillance information can broadly be categorized into: (1) platforms curated by and targeted towards specialists for SARS-CoV-2 information (i.e., GISAID, <b>Nextstrain</b>, or Pubmed), (2) data tracking and messaging platforms curated by specialists and targeted towards general audiences (i.e., the New York Times Data Tracker, CDC website/press releases, the John Hopkins Coronavirus Resource Center, the Yale SARS-CoV-2 Genomic Surveillance Initiative, etc.<li> [56](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36561335">36561335</a> <b>(1)</b> Teaching computational genomics and bioinformatics on a high performance computing cluster&#x2014;a primer<ul><li> Additionally, the intuitive visualizations of the evolution of SARS-CoV-2 and other viruses offered by <ext-link xlink:href="http://www.<b>nextstrain</b>.org" ext-link-type="uri">www.<b>nextstrain</b>.org</ext-link> provided an exciting and timely learning tool for the extensive utility of real-time genomics and strain and spread monitoring.<li> In the following laboratory session, students were led through estimation of haploid nucleotide diversity, Tajima&#x2019;s <italic toggle="yes">D</italic>, and a comparison of evolutionary rates of HIV and SARS-CoV-2 with the results from <b>NextStrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36377483">36377483</a> <b>(3)</b> Taxonium, a web-based tool for exploring large phylogenetic trees<ul><li> This has been an important approach, and has allowed <b>Nextstrain</b> (<xref rid="bib12" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) to provide one of the most widely-used and important tools for exploring SARS-CoV-2 genetic diversity.<li> Briefly, the <b>Nextstrain</b> pipeline downsamples sequences in a rational way, and then uses iqtree (<xref rid="bib24" ref-type="bibr"><u>Minh et al., 2020</u></xref>) to assemble them into a tree, assigns chronology and ancestral states to this tree using TreeTime (<xref rid="bib28" ref-type="bibr"><u>Sagulenko et al., 2018</u></xref>), and displays the results in a user-friendly interactive interface using Auspice (<xref rid="bib12" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> All three of these post-downsampling stages are bottlenecks which prevent the use of full datasets, and so <b>Nextstrain</b> analyses are typically limited to ~4000 sequences &#x2013; sampled in a structured way to ensure they either provide an overview of the pandemic or hone in on particular sequences of interest.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Minh et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [Sagulenko et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Taxonium, a web-based tool for exploring large phylogenetic trees<ul><li>Examples: "Taxonium scales to trees with millions of nodes' and "<b>NextStrain</b> analyses are typically limited to ~4,000 nodes".<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(3)</b> Taxonium, a web-based tool for exploring large phylogenetic trees<ul><li>Examples: "Taxonium scales to trees with millions of nodes' and "<b>NextStrain</b> analyses are typically limited to ~4,000 nodes".<li> I have changed the <b>Nextstrain</b> sentence to refer to sequences, and am sorry for that error.<li> (I have also belatedly corrected my mis-capitalisation of <b>Nextstrain</b> in the final manuscript file!) I then went through the manuscript and considered each use of the word "node", making some changes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36377913">36377913</a> <b>(0)</b> SARS-CoV-2 Genomic Diversity in Households Highlights the Challenges of Sequence-Based Transmission Inference<ul><li>Defined using nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36406862">36406862</a> <b>(1)</b> Relative infectivity of the SARS-CoV-2 Omicron variant in human alveolar cells<ul><li>During the global spread of the coronavirus disease 2019 (COVID-19), many novel SARS-CoV-2 variants of concern (VOC) have emerged, posing an increased risk to global public health and of quarantine.<xref rid="bib1" ref-type="bibr"><sup>1</sup></xref><sup>,</sup><xref rid="bib2" ref-type="bibr"><sup>2</sup></xref><sup>,</sup><xref rid="bib3" ref-type="bibr"><sup>3</sup></xref> International communities, such as GISAID,<xref rid="bib4" ref-type="bibr"><sup>4</sup></xref> PANGO,<xref rid="bib5" ref-type="bibr"><sup>5</sup></xref> and <b>Nextstrain</b>,<xref rid="bib6" ref-type="bibr"><sup>6</sup></xref> have been monitoring and assessing the evolution of SARS-CoV-2 using periodic genomic sequencing of viral samples.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36423117">36423117</a> <b>(0)</b> Epidemiological and Clinical Features of SARS-CoV-2 Variants Circulating between April&#x2013;December 2021 in Italy<ul><li> 2.4.1 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 3 November 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37007166">37007166</a> <b>(2)</b> Antigenic evolution of SARS-CoV-2 in immunocompromised hosts<ul><li> Data downloaded from <b>Nextstrain</b> (<b>nextstrain</b>.org) on 08/02/2022 [<xref rid="CIT0018" ref-type="bibr"><u>18</u></xref>, <xref rid="CIT0045" ref-type="bibr"><u>45</u></xref>] and provided by the Global Initiative for Sharing All Influenza Data (GISAID, gisaid.org) [<xref rid="CIT0046" ref-type="bibr"><u>46&#x2013;48</u></xref>].<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [45](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [46&#x2013;48](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34934514">34934514</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36375791">36375791</a> <b>(3)</b> Emergence of the Omicron SARS-CoV-2 subvariants during the COVID-19 pandemic<ul><li>Phylogenetic relationships of <b>Nextstrain</b> SARS-CoV-2 clades and the rising number of BA.2 subvariants.<li> Adopted from <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0055">https://<b>nextstrain</b>.org</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36324055">36324055</a> <b>(1)</b> IgG autoantibodies against ACE2 in SARS&#x2010;CoV&#x2010;2 infected patients<ul><li> It should also be noted that the present study consisted of samples collected between September and December 2020, a period dominated by <b>Nextstrain</b> SARS&#x2010;CoV&#x2010;2 clades 20A, 20B, and 20C.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36438034">36438034</a> <b>(4)</b> SARS-CoV-2 variants: Impact on biological and clinical outcome<ul><li> The most accepted and widely used systems are GISAID (global initiative on sharing avian flu data), <b>Nextstrain</b>, and Pango or Pangolin (Phylogenetic Assignment of Named Global Outbreak Lineages) (<xref rid="B21" ref-type="bibr"><u>21</u></xref>).<li> Similarly, <b>Nextstrain</b> is a viral genome database that is comprised of data curation and analysis, as well as visualization components (<xref rid="B24" ref-type="bibr"><u>24</u></xref>).<li>Phylogenetic relationships of SARS-CoV-2 clades, as defined by <b>Nextstrain</b> (as of September 15, 2022) showing evolutionary relationships of SARS-CoV-2 viruses from the ongoing COVID-19 pandemic.<li><bold><b>Nextstrain</b> clade</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>Relationship to circulating VOC lineages</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>Genetic features</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>Earliest documented samples</bold></th>            </tr>          </thead>          <tbody>            <tr>              <td valign="top" align="left" rowspan="1" colspan="1">BA.4<sup>#</sup>.<li> [21](<person-group person-group-type="author"><collab>WHO</collab></person-group>. <source>Tracking SARS-CoV-2 Variants.</source> (<year>2022</year>). Available online at: <ext-link xlink:href="https://www.who.int/activities/tracking-SARS-CoV-2-variants" ext-link-type="uri">https://www.who.int/activities/tracking-SARS-CoV-2-variants</ext-link> (accessed September 15, 2022).) [24](<person-group person-group-type="author"><collab>Nextstrain</collab></person-group>. <source>Real-Time Tracking of Pathogen Evolution.</source> (<year>2022</year>). Available online at: <ext-link xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link> (accessed September 15, 2022).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36357713">36357713</a> <b>(3)</b> Divergent SARS-CoV-2 variant emerges in white-tailed deer with deer-to-human transmission<ul><li>Maximum-likelihood (ML) and parsimony-based phylogenetic analyses were used to examine lineage origins and relatedness, and showed that these deer-derived viral genomes formed a highly divergent clade within the B.1 PANGO lineage/20C <b>Nextstrain</b> clade (100% ultrafast bootstrap (UFB)) that shared a most recent common ancestor (MRCA) between May and July 2021 (95% highest posterior density (HPD) node age bounds; Extended Data Fig.<li> The second experiment used only the subset of samples belonging to <b>Nextstrain</b> clade 20C (which contains the B.1.641 sequences).<li> An analysis using the entire Pangolin B.1 lineage was not appropriate since this lineage also includes <b>Nextstrain</b> clades 20A and 20B and the inclusion of these samples could potentially obfuscate important patterns since the results would have to be interpreted in a context wider than necessary.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36355753">36355753</a> <b>(1)</b> Ten simple rules for developing visualization tools in genomics<ul><li> Coronavirus Disease 2019 (COVID-19) visualizations, good and bad [<xref rid="pcbi.1010622.ref002" ref-type="bibr"><u>2</u></xref>], became omnipresent in the public debates, with interactive and dynamic visualization platforms like <b>Nextstrain</b> [<xref rid="pcbi.1010622.ref003" ref-type="bibr"><u>3</u></xref>] gaining a lot of popularity.<li> [2](<name><surname>Krzywinski</surname><given-names>M.</given-names></name><source>A pandemic of bad charts</source>. <year>2022</year>. Available from: <ext-link xlink:href="https://www.youtube.com/watch?v=_YGmfsKL8N8" ext-link-type="uri">https://www.youtube.com/watch?v=_YGmfsKL8N8</ext-link>.) [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36371334">36371334</a> <b>(0)</b> Seroepidemiological and genomic investigation of SARS-CoV-2 spread in North East region of India<ul><li> Mutation calling and lineage assignment were carried out using NextClade v.154 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0015">https://clades.<b>nextstrain</b>.org/</ext-link>) and Pangolin Lineage Assigner v3.1.16 (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="intref0020">https://pangolin.cog-uk.io</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36407474">36407474</a> <b>(16)</b> The Potential Protective Role of GS-441524, a Metabolite of the Prodrug Remdesivir, in Vaccine Breakthrough SARS-CoV-2 Infections<ul><li> <b>Nextstrain</b> clade label: 21&#xA0;K indicates Omicron BA.1 lineage; 21L indicates Omicron BA.2 lineage.<li> <bold>B</bold>, <bold>C</bold> Generated using <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>, <xref ref-type="bibr" rid="CR5"><u>5</u></xref>].<li><sup>*</sup>, &#x2020;, &#x2021;, &#xA7;, &#x2016;: Amino acid residues that have mutations in VOCs reported on <b>Nextstrain</b> as of March 20th, 2022.<li>To evaluate the mutations in amino acid residues of major GS-441524-interacting SARS-CoV-2 NSPs (RdRp, NSP3, NSP5, and NSP14), the online database <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) was used (up to March 20th, 2022) [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>, <xref ref-type="bibr" rid="CR5"><u>5</u></xref>].<li> As of March 20th, 2022, no mutation in amino acids of RdRp that interact with GS-441524 had been found in VOCs (Table <xref rid="Tab2" ref-type="table"><u>2</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>, <xref ref-type="bibr" rid="CR5"><u>5</u></xref>].<li> As of March 20th, 2022, two different mutations (lineage significance cut-off at 10% global prevalence) in two samples of Omicron variant lineages (A154T: BA.1 and A154N: BA.1.1) were identified in an single amino acid residue of NSP3 that interacts with NTP (Table <xref rid="Tab2" ref-type="table"><u>2</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>, <xref ref-type="bibr" rid="CR5"><u>5</u></xref>].<li> We also found several other mutations in GS-441524-NSP3 interacting sites of Delta and Omicron variant cases on <b>Nextstrain</b>; however, all these mutations occurred in patchy cases and with low prevalence (&lt;&#x2009;0.5&#x2013;2%) (Table <xref rid="Tab2" ref-type="table"><u>2</u></xref> and Supplementary Table S2).<li>Similar to RdRp, since no mutation of NTP-interacting residues in NSP5 had been found as of March 20th, 2022, the interactions between GS-441524 and NSP5 of VOCs (Alpha, Beta, Gamma, Delta, and Omicron variants) would appear similar to that between GS-441524 and Wuhan wild-type NSP5 (Table <xref rid="Tab2" ref-type="table"><u>2</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>, <xref ref-type="bibr" rid="CR5"><u>5</u></xref>].<li>As of March 20th, 2022, one mutated NSP14 residue that interacts with GS-441524 is observed on <b>Nextstrain</b> in a sample of the Omicron variant sub-lineage BA.1 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>, <xref ref-type="bibr" rid="CR5"><u>5</u></xref>].<li> [4](Nextstrain/ncov/global. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](Nextstrain/ncov/global. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](Nextstrain/ncov/global. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](Nextstrain/ncov/global. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](Nextstrain/ncov/global. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](Nextstrain/ncov/global. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36368344">36368344</a> <b>(1)</b> SpikeSeq: A rapid, cost efficient and simple method to identify SARS-CoV-2 variants of concern by Sanger sequencing part of the spike protein gene<ul><li> Then, to gain more information from the mutation profile, we suggest using the fantastic resources available at <b>nextstrain</b>.org/ncov (<xref rid="bib8" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) to filter the GISAID deposited genome sequences to the samples with a similar mutation profile, which will give information on the geographical spread and novelty of the mutation profile.<li> [Hadfield et al., 2018](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinforma. Oxf. Engl.</source>          <volume>34</volume>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36346252">36346252</a> <b>(0)</b> Detection of SARS-CoV-2 in Saliva and Nasopharyngeal Swabs According to Viral Variants<ul><li> Among the 134 samples also sequenced via whole-genome sequencing (WGS), Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) retrieved the following lineages: 29 Wuhan (no VOC), 19 Alpha (including 4 Alpha&#x2009;+&#x2009;484K), 11 Beta, 33 Delta, and 42 Omicron (including 40 from the BA.1 lineage and 2 from the BA.2 lineage).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36346321">36346321</a> <b>(4)</b> Swiss public health measures associated with reduced SARS-CoV-2 transmission using genome data<ul><li> Last, we followed the <b>Nextstrain</b> pipeline&#x2019;s recommendation to mask the first 100 and last 50 sites of the alignment (<xref rid="R39" ref-type="bibr"><italic toggle="yes">39</italic></xref>) because the start and end of SARS-CoV-2 sequences are prone to sequencing errors (<xref rid="R40" ref-type="bibr"><italic toggle="yes">40</italic></xref>).<li> We considered all non-Swiss sequences from each lineage available on GISAID that passed the quality filtering steps detailed above and applied the <b>Nextstrain</b> priority script (<xref rid="R39" ref-type="bibr"><italic toggle="yes">39</italic></xref>) to rank these sequences by their genetic similarity to Swiss sequences in each lineage alignment.<li> 10<sup>&#x2212;4</sup> substitutions per site per year (s/s/y), based on estimates by <b>Nextstrain</b> (<xref rid="R45" ref-type="bibr"><italic toggle="yes">45</italic></xref>).<li> We constrained the most-recent common ancestor to be between 15 November and 24 December 2019, also based on estimates by <b>Nextstrain</b> (<xref rid="R45" ref-type="bibr"><italic toggle="yes">45</italic></xref>), and set the minimum branch length to zero.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36348341">36348341</a> <b>(1)</b> Whole-genome sequence analysis reveals the circulation of multiple SARS-CoV-2 variants of concern in Nairobi and neighboring counties, Kenya between March and July 2021<ul><li>The SARS-CoV-2 genome sequences were grouped into genetic lineages based on the Phylogenetic Assignment of Named Global Outbreak LiNeages (PANGOLIN) [<xref ref-type="bibr" rid="CR32"><u>32</u></xref>] and <b>NextStrain</b> [<xref ref-type="bibr" rid="CR33"><u>33</u></xref>] classification methods.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36370519">36370519</a> <b>(5)</b> Mutations in SARS-CoV-2: Insights on structure, variants, vaccines, and biomedical interventions<ul><li> the most common are PANGO, GISAID, and <b>Nextstrain</b>, in addition to WHO and emerging variants systems <xref rid="bib131" ref-type="bibr"><u>[131]</u></xref>, <xref rid="bib132" ref-type="bibr"><u>[132]</u></xref>, <xref rid="bib133" ref-type="bibr"><u>[133]</u></xref> (<xref rid="fig0015" ref-type="fig">Fig.<li><b>Nextstrain</b> nomenclature.<li>The <b>Nextstrain</b> nomenclature currently recognises 12 major clades include: 19&#x2009;A clade, 19B clade, 20&#x2009;A clade, 20B clade, 20&#x2009;C clade, 20D clade, 20E clade, 20&#x2009;F clade, 20&#x2009;G clade, 20&#x2009;H/501Y.V2 clade, 20I/501Y.V1 clade, and 20&#x2009;J/501Y.V3 clade <xref rid="bib140" ref-type="bibr"><u>[140]</u></xref>, <xref rid="bib141" ref-type="bibr"><u>[141]</u></xref>.<li> As a result, <b>Nextstrain</b> added regional frequency (&gt;30%) and known variations of concern to their main clade designation criteria <xref rid="bib142" ref-type="bibr"><u>[142]</u></xref>.<li> the GISAID, <b>Nextstrain</b>, and Pangolin nomenclatures for SARS-CoV-2 sequences from worldwide.<li> [[131]](          <person-group person-group-type="author">            <name>              <surname>Liu</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Liu</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Zhang</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Zhang</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Bian</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>Huang</surname>              <given-names>L.</given-names>            </name>          </person-group>          <article-title>CoV-Seq, a New Tool for SARS-CoV-2 genome analysis and visualization: development and usability study</article-title>          <source>J. Med. Internet Res.</source>          <volume>22</volume>          <year>2020</year>          <object-id pub-id-type="publisher-id">e22299</object-id>        ) [[132]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33514928">33514928</a>) [[133]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [[140]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[141]](Bedford T., Hodcroft E., Neher R. Updated Nextstrain SARS CoV-2 clade naming strategy. <ext-link ext-link-type="uri" xlink:href="https://nextstrainorg/blog/2021-01-06-updated-SARS-CoV-2-clade-naming" id="ir0005">&#x3008;https://nextstrainorg/blog/2021&#x2013;01-06-updated-SARS-CoV-2-clade-naming&#x3009;</ext-link> 2021.) [[142]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34200331">34200331</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36342956">36342956</a> <b>(1)</b> Risk and symptoms of COVID-19 in health professionals according to baseline immune status and booster vaccination during the Delta and Omicron waves in Switzerland&#x2014;A multicentre cohort study<ul><li> During the local emergence of the Delta (October to December 2021) and Omicron B.1.1.529.1 (<b>Nextstrain</b> 21K; BA.1) variant (January to March 2022), the follow-up survey through questionnaires was continued at monthly intervals.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36366559">36366559</a> <b>(2)</b> In Silico Genome Analysis Reveals the Evolution and Potential Impact of SARS-CoV-2 Omicron Structural Changes on Host Immune Evasion and Antiviral Therapeutics<ul><li> sequences, created by <b>Nextstrain</b> (accessed on 22 October 2022), was used (<xref rid="app1-viruses-14-02461" ref-type="app"><u>Table S1</u></xref>) [<xref rid="B17-viruses-14-02461" ref-type="bibr"><u>17</u></xref>].<li>To study phylogenetic relationships, the subset data of globally circulating five VOCs sequences were analyzed in <b>Nextstrain</b> (<xref rid="viruses-14-02461-f001" ref-type="fig"><u>Figure 1</u></xref>F).<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36344418">36344418</a> <b>(1)</b> Double N501Y mutation hindering its detection by RT-PCR kit widely used at Clinical Microbiology laboratories<ul><li> The readings obtained were subsequently assembled using the ARTIC <italic>pipeline</italic> for data obtained from sequencers with Oxford Nanopore technology and analysed using the <b>Nextstrain</b> web app<xref rid="bib0045" ref-type="bibr"><sup>9</sup></xref> with consistency verified using the Integrative Genomics Viewer (IGV) program<xref rid="bib0050" ref-type="bibr"><sup>10</sup></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36333340">36333340</a> <b>(5)</b> SARS-CoV-2 quasi-species analysis from patients with persistent nasopharyngeal shedding<ul><li> Whole genome quality was also assessed on <b>Nextstrain</b> and Auspice (Supplementary Fig.&#xA0;<xref rid="MOESM2" ref-type="media"><u>2</u></xref>).<li> According to <b>Nextstrain</b> analysis, we have obtained a clade distribution consisting of 47% and 25% of 20A, 15% and 6% of 20E, 19% and 32% of 20I/Alpha variant, 12% and 32% of 20&#xA0;J/Beta variant, 7% and 6% of other clades, in persistent and non-persistent groups, respectively (Supplementary Fig.&#xA0;<xref rid="MOESM2" ref-type="media"><u>2</u></xref>).<li>Phylogenetic trees were constructed using the <b>nextstrain</b>/ncov tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) and visualized with Auspice (2.36.0) software (<ext-link ext-link-type="uri" xlink:href="https://auspice.us/">https://auspice.us/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36405967">36405967</a> <b>(2)</b> SARS-CoV-2 variants of concern surveillance including Omicron using RT-PCR&#x2013;based genotyping offers comparable performance to whole genome sequencing<ul><li> The distribution of these 1,040 samples is shown as phylogenic tree of <b>Nextstrain</b> clades in <xref rid="f2" ref-type="fig"><bold>Figure&#xA0;2A</bold></xref>.<li> <bold>(A)</bold> Nextclade phylogenetic radial tree from the <b>Nextstrain</b> global reference tree.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36336246">36336246</a> <b>(0)</b> SARS-CoV-2 genomic surveillance in Malaysia: displacement of B.1.617.2 with AY lineages as the dominant Delta variants and the introduction of Omicron during the fourth epidemic wave<ul><li> Clade assignment, mutation calling, and sequence quality control score were performed using the open-source Nextclade pipelines comprising the phylodynamic analysis component Augur and the interactive visualization component Auspice (https://clades.<b>nextstrain</b>.org) (<xref rid="bib0001" ref-type="bibr">Aksamentov <italic>et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36322581">36322581</a> <b>(4)</b> RASCL: Rapid Assessment of Selection in CLades through molecular sequence analysis<ul><li> We compiled our background dataset from the globally subsampled <b>Nextstrain</b> [<xref rid="pone.0275623.ref039" ref-type="bibr"><u>39</u></xref>] (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time</ext-link>) build (last accessed June 12, 2022, genomes were sampled from the beginning of the SARS-CoV-2 pandemic) (<xref rid="pone.0275623.s001" ref-type="supplementary-material"><u>S1 File</u></xref>).<li>We also report the GISAID accession ID&#x2019;s for our <b>Nextstrain</b> background dataset.<li> [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36366530">36366530</a> <b>(1)</b> Systematic Genomic Surveillance of SARS-CoV-2 Virus on Illumina Sequencing Platforms in the Slovak Republic&#x2014;One Year Experience<ul><li> Nomenclature systems from the Global Initiative on Sharing All Influenza Data (GISAID) [<xref rid="B10-viruses-14-02432" ref-type="bibr"><u>10</u></xref>], <b>Nextstrain</b> [<xref rid="B11-viruses-14-02432" ref-type="bibr"><u>11</u></xref>] and Pango [<xref rid="B12-viruses-14-02432" ref-type="bibr"><u>12</u></xref>] are currently used to name and track SARS-CoV-2 genetic lineages.<li> [10](          <article-title>GISAID&#x2014;hCov19 Variants</article-title>          <comment>Available online: <ext-link xlink:href="https://www.gisaid.org/hcov19-variants" ext-link-type="uri">https://www.gisaid.org/hcov19-variants</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-06-29">(accessed on 29 June 2022)</date-in-citation>        ) [11](          <article-title>CoVariants</article-title>          <comment>Available online: <ext-link xlink:href="https://covariants.org/" ext-link-type="uri">https://covariants.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-06-29">(accessed on 29 June 2022)</date-in-citation>        ) [12](          <article-title>Cov-Lineages</article-title>          <comment>Available online: <ext-link xlink:href="https://cov-lineages.org/lineage_list.html" ext-link-type="uri">https://cov-lineages.org/lineage_list.html</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-06-29">(accessed on 29 June 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36150508">36150508</a> <b>(1)</b> Human Parainfluenza Virus in Homeless Shelters before and during the COVID-19 Pandemic, Washington, USA<ul><li> We aligned shelter consensus genomes generated with corresponding HPIV type genomes from GenBank, generated type-specific phylogenetic trees, and visualized trees by using <b>NextStrain</b> Auspice software (<ext-link xlink:href="https://github.com" ext-link-type="uri">https://github.com</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36274370">36274370</a> <b>(3)</b> Incidence, risk factors, and whole-genome sequence of SARs-CoV-2 and influenza virus among the Egyptian pilgrims returning from Umrah mass gathering in Saudi Arabia, April-May 2022<ul><li> Similar findings were reported in this study where Omicron was the only variant found in the sequenced isolates including BA.1, BA.1.1 (or <b>Nextstrain</b> clade 21&#x2009;K) and BA.2 (or <b>Nextstrain</b> clade 21&#x2009;L) and C.36 (or <b>Nextstrain</b> clade 20D) sublineages.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36314610">36314610</a> <b>(3)</b> Framing the future of the COVID-19 response operations in 2022 in the WHO African region<ul><li>Virus evolution Genomic spread of SARS CoV 2 VOC.</p></caption><alternatives><graphic xlink:href="ZGHA_A_2130528_F0001_OC" content-type="color" specific-use="web-only" position="float"/><graphic xlink:href="ZGHA_A_2130528_F0001_PB" content-type="black-white" specific-use="print-only" position="float"/></alternatives><attrib>Source: <b>Nextstrain</b> based on GISAID data: <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global?f_region=Africa" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global?f_region=Africa</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36315517">36315517</a> <b>(3)</b> One health genomic surveillance and response to a university-based outbreak of the SARS-CoV-2 Delta AY.25 lineage, Arizona, 2021<ul><li> SARS-CoV-2 genomes publicly available through GISAID (<xref rid="pone.0272830.s001" ref-type="supplementary-material"><u>S1 Table</u></xref>) from these sequencing efforts were utilized to build phylogenetic trees using the SARS-CoV-2 specific workflows (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov/" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov/</ext-link>) in <b>Nextstrain</b> [<xref rid="pone.0272830.ref028" ref-type="bibr"><u>28</u></xref>].<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36366521">36366521</a> <b>(1)</b> Retrospective Analysis of the SARS-CoV-2 Infection Profile in COVID-19 Positive Patients in Vitoria da Conquista, Northeast Brazil<ul><li> A unique set of external references (n = 7992) were obtained by including all the non-Brazilian sequences from the global and South American <b>NextStrain</b> builds (date of access: 31 July 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36366515">36366515</a> <b>(1)</b> Characterizing Longitudinal Antibody Responses in Recovered Individuals Following COVID-19 Infection and Single-Dose Vaccination: A Prospective Cohort Study<ul><li> A phylogenetic tree was constructed using Fasttree [<xref rid="B28-viruses-14-02416" ref-type="bibr"><u>28</u></xref>] and visualized in <b>Nextstrain</b> [<xref rid="B29-viruses-14-02416" ref-type="bibr"><u>29</u></xref>] and lineage assignment was performed using the Phylogenetic Assignment of Named Global Outbreak Lineages tool (Pango/Usher Version 1.15.1) [<xref rid="B30-viruses-14-02416" ref-type="bibr"><u>30</u></xref>].<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20224823">20224823</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36366490">36366490</a> <b>(3)</b> ORF-Interrupting Mutations in Monkeypox Virus Genomes from Washington and Ohio, 2022<ul><li> Additionally, phylogenetic analysis using the <b>Nextstrain</b> v4.2.0 [<xref rid="B20-viruses-14-02393" ref-type="bibr"><u>20</u></xref>] monkeypox workflow (<uri xlink:href="https://github.com/<b>nextstrain</b>/monkeypox">https://github.com/<b>nextstrain</b>/monkeypox</uri>, accessed on 5 October 2022) was performed on 25 samples containing ORF-disrupting mutations and 129 other samples sequenced by our laboratory.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36360202">36360202</a> <b>(1)</b> Membrane Clustering of Coronavirus Variants Using Document Similarity<ul><li> In [<xref rid="B10-genes-13-01966" ref-type="bibr"><u>10</u></xref>], the authors created a protocol for the analysis and phylogenetic clustering of <italic toggle="yes">SARS-CoV-2</italic> genomes using an open-source tool, <italic toggle="yes"><b>Nextstrain</b></italic>, for real-time interactive visualization of genome sequencing data.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34124300">34124300</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36511336">36511336</a> <b>(1)</b> Transmission dynamics of SARS-CoV-2 Omicron variant infections in Hangzhou, Zhejiang, China, January-February 2022<ul><li> The epidemiological investigation showed that all cases identified in this outbreak could be traced back to an index case, and the gene sequence analysis revealed that all patients were infected with the Omicron variant sublineage of BA.1.1 (or <b>Nextstrain</b> clade 21K).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36364158">36364158</a> <b>(1)</b> Homology Modeling and Molecular Dynamics-Driven Search for Natural Inhibitors That Universally Target Receptor-Binding Domain of Spike Glycoprotein in SARS-CoV-2 Variants<ul><li> According to the subsampling of globally circulating sequences created by <b>NextStrain</b>, the spread of most variants was diminished by December 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36317070">36317070</a> <b>(2)</b> Evaluation of immunoprotection against coronavirus disease 2019: Novel variants, vaccine inoculation, and complications<ul><li>According to phylodynamic analysis of SARS-CoV-2 performed using the <b>Nextstrain</b> platform [<xref rid="bib6" ref-type="bibr"><u>[6]</u></xref>, <xref rid="bib7" ref-type="bibr"><u>[7]</u></xref>], in 2021, SARS-CoV-2 has evolved from 20H (Beta/B.1.351.3, V2) to 22C (Omicron/B.1.1.529).<li>Mutations in the genomic regions of the spike (S), envelope (E), membrane (M), and nucleocapsid (N) proteins in severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) variants reported in 2021 (data derived from the <b>Nextstrain</b> platform) [<xref rid="bib7" ref-type="bibr"><u>7</u></xref>].<li> [[6]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[7]](The Nextstrain team, Genomic epidemiology of SARS-CoV-2 with subsampling focused globally over the past 6 months. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/gisaid/global/6m" id="intref0020">https://nextstrain.org/ncov/gisaid/global/6m</ext-link>. (Accessed 16 June 2022).) [7](The Nextstrain team, Genomic epidemiology of SARS-CoV-2 with subsampling focused globally over the past 6 months. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/gisaid/global/6m" id="intref0020">https://nextstrain.org/ncov/gisaid/global/6m</ext-link>. (Accessed 16 June 2022).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36313592">36313592</a> <b>(6)</b> A comprehensive review on variants of SARS-CoVs-2: Challenges, solutions and open issues<ul><li>The naming of SARS-CoV-2 variants is currently done by GISAD, <b>Nextstrain</b>, and Pango.<li> <b>Nextstrain</b> is an organization for real-time tracking of pathogens, their evolution, and their genomic data.<li><b>Nextstrain</b></th><th align="left">Earliest documented</th><th align="left">Date of designation</th></tr></thead><tbody><tr><td align="left">Epsilon.<li><b>Nextstrain</b></th><th align="left">Earliest documented</th><th align="left">Date of designation</th></tr></thead><tbody><tr><td align="left">Alpha.<li> The most prevalent of them are BA.1, BA.1.1 (also known as <b>Nextstrain</b> clade 21K), and BA.2 (or <b>Nextstrain</b> clade 21L).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36282886">36282886</a> <b>(4)</b> Fully AutomatedCRISPR-LAMP Platform for SARS-CoV-2Delta and Omicron Variants<ul><li> Comparing the B.1617.2 (Delta variant) sequence andOmicron variant among the <b>Nextstrain</b> clade, GISAID clade, and Pangolineage, we found that <italic>Spike</italic> T478K, D614G, P681R,and P681H were special point mutations for the widespread characteristicsall over these clades.<li> Also, studies have shown that mutations atthe <italic>Spike</italic> residue positions 478, 614, and 681 (locatedat a furin cleavage site) may result in higher pathogenicity and transmissibility.<sup><xref ref-type="bibr" rid="ref20"><u>20</u></xref>&#x2212;<xref ref-type="bibr" rid="ref22"><u>22</u></xref></sup> Summarized from <b>Nextstrain</b> CoVariants resources, residue positions478 and 681 existed only in Delta and Omicron strains and residueposition 614 existed in almost all VOCs.<li> The judgment standardsof Delta and Omicron (according to the <b>Nextstrain</b> CoVariants resources)variants are shown in the left bottom of <xref rid="fig5" ref-type="fig">Figure <xref rid="fig5" ref-type="fig"><u>5</u></xref></xref>.<li> Points mutationsin Del (Delta) and Omi (Omicron) are summarized according to the <b>Nextstrain</b>CoVariants (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://covariants.org/">https://covariants.org/</uri>); NC: negative control; Nor: normal SARS-CoV-2 samples; and Neg:negative SARS-CoV-2 samples.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34601723">34601723</a>) [22](<name><surname>Saito</surname><given-names>A.</given-names></name>; <name><surname>Irie</surname><given-names>T.</given-names></name>; <name><surname>Suzuki</surname><given-names>R.</given-names></name>; <name><surname>Maemura</surname><given-names>T.</given-names></name>; <name><surname>Nasser</surname><given-names>H.</given-names></name>; <name><surname>Uriu</surname><given-names>K.</given-names></name>; <name><surname>Kosugi</surname><given-names>Y.</given-names></name>; <name><surname>Shirakawa</surname><given-names>K.</given-names></name>; <name><surname>Sadamasu</surname><given-names>K.</given-names></name>; <name><surname>Kimura</surname><given-names>I.</given-names></name>; <name><surname>Ito</surname><given-names>J.</given-names></name>; <name><surname>Wu</surname><given-names>J.</given-names></name>; <name><surname>Iwatsuki-Horimoto</surname><given-names>K.</given-names></name>; <name><surname>Ito</surname><given-names>M.</given-names></name>; <name><surname>Yamayoshi</surname><given-names>S.</given-names></name>; <name><surname>Ozono</surname><given-names>S.</given-names></name>; <name><surname>Butlertanaka</surname><given-names>E. P.</given-names></name>; <name><surname>Tanaka</surname><given-names>Y. L.</given-names></name>; <name><surname>Shimizu</surname><given-names>R.</given-names></name>; <name><surname>Shimizu</surname><given-names>K.</given-names></name>; <name><surname>Yoshimatsu</surname><given-names>K.</given-names></name>; <name><surname>Kawabata</surname><given-names>R.</given-names></name>; <name><surname>Sakaguchi</surname><given-names>T.</given-names></name>; <name><surname>Tokunaga</surname><given-names>K.</given-names></name>; <name><surname>Yoshida</surname><given-names>I.</given-names></name>; <name><surname>Asakura</surname><given-names>H.</given-names></name>; <name><surname>Nagashima</surname><given-names>M.</given-names></name>; <name><surname>Kazuma</surname><given-names>Y.</given-names></name>; <name><surname>Nomura</surname><given-names>R.</given-names></name>; <name><surname>Horisawa</surname><given-names>Y.</given-names></name>; <name><surname>Yoshimura</surname><given-names>K.</given-names></name>; <name><surname>Takaori-Kondo</surname><given-names>A.</given-names></name>; <name><surname>Imai</surname><given-names>M.</given-names></name>; <name><surname>The Genotype to PhenotypeJapan</surname><given-names>C.</given-names></name>; <name><surname>Nakagawa</surname><given-names>S.</given-names></name>; <name><surname>Ikeda</surname><given-names>T.</given-names></name>; <name><surname>Fukuhara</surname><given-names>T.</given-names></name>; <name><surname>Kawaoka</surname><given-names>Y.</given-names></name>; <name><surname>Sato</surname><given-names>K.</given-names></name><article-title>SARS-CoV-2 spike P681R mutation,a hallmark of the Delta variant, enhances viral fusogenicity and pathogenicity</article-title>. <source>bioRxiv</source><year>2021</year>, <pub-id pub-id-type="doi">10.1101/2021.06.17.448820</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36412834">36412834</a> <b>(1)</b> Shared 6mer Peptides of Human and Omicron (21K and 21L) at SARS-CoV-2 Mutation Sites<ul><li> Consequently, the possible molecular mimicry-based autoimmunity risk of the Omicron <b>Nextstrain</b> clades 21K and 21L was investigated here.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36361690">36361690</a> <b>(6)</b> A Timeframe for SARS-CoV-2 Genomes: A Proof of Concept for Postmortem Interval Estimations<ul><li> Furthermore, it is important to note that there are in fact large genome databases for other pathogens (e.g., influenza, zika and enterovirus) for which there are already large phylogenetic developments (see, e.g., <uri xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</uri>, accessed on 17 May 2022).<li> The associated metadata for these genomes was downloaded (<uri xlink:href="https://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/">https://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/</uri> accessed on 17 May 2022), and includes information on sampling date and location, GenBank accession number, sequence length, pangolin, <b>Nextstrain</b> and UShER clade.<li> 10<sup>&#x2212;3</sup> s/s/y (<b>Nextstrain</b>; <uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri>, accessed on 17 May 2022) and the average length of the SARS-CoV-2, which is 28.5Kb.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36272063">36272063</a> <b>(1)</b> SARS-CoV-2 Omicron BA.1, BA.2, and XAG identification during routine surveillance on a university campus in Belo Horizonte, Brazil, 2022<ul><li> A dataset (<italic>n</italic>&#x2009;=&#x2009;1468) containing public reference genomes classified by <b>Nextstrain</b> as Omicron clades (Supplementary Table <xref rid="MOESM2" ref-type="media"><u>2</u></xref>) was aligned using Minimap2, and a maximum likelihood phylogeny was inferred using IQ-tree v2.0.3 [<xref ref-type="bibr" rid="CR12"><u>12</u></xref>] under the GTR&#x2009;+&#x2009;F&#x2009;+&#x2009;I&#x2009;+&#x2009;G4 model to corroborate the classification.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36312039">36312039</a> <b>(3)</b> Variation analysis of SARS-CoV-2 complete sequences from Iran<ul><li> GISAID [<xref rid="B9" ref-type="bibr"><u>9</u></xref>,<xref rid="B10" ref-type="bibr"><u>10</u></xref>], <b>Nextstrain</b> [<xref rid="B10" ref-type="bibr"><u>10</u></xref>] and Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin) [<xref rid="B11" ref-type="bibr"><u>11</u></xref>] are SARS-CoV-2 nomenclature systems.<li> The Pangolin nomenclature system is more inclusive of phylogenetic clades and has the potential to supplement two other nomenclature systems (<b>NextStrain</b> and GISAID) [<xref rid="B13" ref-type="bibr"><u>13</u></xref>].<li> <b>Nextstrain</b> was also used to analyze the genomes and compare the results [<xref rid="B10" ref-type="bibr"><u>10</u></xref>].<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36271389">36271389</a> <b>(2)</b> A comprehensive update on CIDO: the community-based coronavirus infectious disease ontology<ul><li> The classifications of viral variants and amino acid variants were obtained from GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</ext-link>), <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://www.<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>), and WHO.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36296366">36296366</a> <b>(2)</b> SARS-CoV-2 Omicron BA.2.75 Variant May Be Much More Infective than Preexisting Variants Based on In Silico Model<ul><li>We obtained the information for the amino acid substitutions of the spike proteins from the CoVariants website, which classifies the variants according to <b>Nextstrain</b> clades [<xref rid="B7-microorganisms-10-02090" ref-type="bibr"><u>7</u></xref>,<xref rid="B8-microorganisms-10-02090" ref-type="bibr"><u>8</u></xref>].<li> We focused on only the S gene sequence to calculate the evolutionary distance and did not use the genome sequence, which is used for the phylogeny provided by <b>Nextstrain</b> [<xref rid="B7-microorganisms-10-02090" ref-type="bibr"><u>7</u></xref>] because it is the spike protein that binds to ACE2 on human cells.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](          <person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name></person-group>          <article-title>CoVariants: SARS-CoV-2 Mutations and Variants of Interest</article-title>          <comment>Available online: <ext-link xlink:href="https://covariants.org/" ext-link-type="uri">https://covariants.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-10-16">(accessed on 16 October 2021)</date-in-citation>        ) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298869">36298869</a> <b>(3)</b> Impact of Early Pandemic SARS-CoV-2 Lineages Replacement with the Variant of Concern P.1 (Gamma) in Western Bahia, Brazil<ul><li> Initially, phylogenetic reconstructions were performed using datasets containing sequences from the study (n = 112) and 1004 representative <b>Nextstrain</b>&#x2019;s subsampling SARS-CoV-2 genomic sequences from South America countries and their territories (n = 16).<li> Such sequences were retrieved from the beginning of the pandemic until August 2022 (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 1 September 2022), representing multiple SARS-CoV-2 circulating strains.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298865">36298865</a> <b>(0)</b> SARS-CoV-2 from COVID-19 Patients in the Republic of Moldova: Whole-Genome Sequencing Results<ul><li> The Nextclade Pangolin lineage option was used when determining lineages using the Nextlade software, which was identified by applying a Nextclade web tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri> (accessed on 1 August 2022)) [<xref rid="B6-viruses-14-02310" ref-type="bibr"><u>6</u></xref>].<li> [6](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36332633">36332633</a> <b>(1)</b> Evidence for SARS-CoV-2 Delta and Omicron co-infections and recombination<ul><li> Of note, ORF1A:P2046L is not a defining mutation of Delta (<b>Nextstrain</b> clade 21A) as it is present in only 90.47% of the Delta sequences based on <ext-link ext-link-type="uri" xlink:href="https://covariants.org/variants/21A.Delta" id="intref0060">https://covariants.org/variants/21A.Delta</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298856">36298856</a> <b>(3)</b> Two Years of Genomic Surveillance in Belgium during the SARS-CoV-2 Pandemic to Attain Country-Wide Coverage and Monitor the Introduction and Spread of Emerging Variants<ul><li>In order to inform authorities, the scientific community, and the general public on the evolution and spread of SARS-CoV-2 in Belgium, we used the available genomes to construct our first Belgian <b>Nextstrain</b> instance, which became publicly available on 8 January 2021 (see <xref rid="viruses-14-02301-f002" ref-type="fig"><u>Figure 2</u></xref>).<li> <b>Nextstrain</b>&#x2019;s joint temporal and spatial visualizations integrate sequence data with geographic information, lineage nomenclature, and mutations of interest to show how the pandemic unfolded over time, delivering important insights to health professionals, epidemiologists, virologists, and the general public via easily shareable links (for example, through social media) [<xref rid="B19-viruses-14-02301" ref-type="bibr"><u>19</u></xref>].<li> These sequences were used to update both global and Belgium-focused <b>Nextstrain</b> builds (e.g., <xref rid="viruses-14-02301-f002" ref-type="fig"><u>Figure 2</u></xref>) and to transparently inform public health authorities on the circulation of the different variants of the virus (e.g., results of these analyses are often used in the weekly reports of the consortium).<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298616">36298616</a> <b>(3)</b> Variants of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) and Vaccine Effectiveness<ul><li> The established nomenclature systems for naming and tracking SARS-CoV-2 lineages by GISAID, <b>Nextstrain</b>, and Pango are currently in use among scientific community [<xref rid="B15-vaccines-10-01751" ref-type="bibr"><u>15</u></xref>].<li> The <b>Nextstrain</b> nomenclature system is based on the diversity of SARS-CoV-2 patterns and label clades that can persist for at least several months; and have a significant geographic spread [<xref rid="B15-vaccines-10-01751" ref-type="bibr"><u>15</u></xref>].<li> WHO along with group of scientists (WHO Virus Evolution Working Group, the WHO COVID-19 reference laboratory network), representatives (GISAID, <b>Nextstrain</b>, Pango) and experts (virological, microbial nomenclature) from several countries and agencies, developed an easy-to-pronounce and non-stigmatizing labels for variants to assist discussion among public [<xref rid="B15-vaccines-10-01751" ref-type="bibr"><u>15</u></xref>].<li> [15](          <person-group person-group-type="author"><collab>World Health Organization</collab></person-group>          <article-title>Tracking SARS-CoV-2 variants</article-title>          <comment>Available online: <ext-link xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" ext-link-type="uri">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-08-25">(accessed on 25 August 2021)</date-in-citation>        ) [15](          <person-group person-group-type="author"><collab>World Health Organization</collab></person-group>          <article-title>Tracking SARS-CoV-2 variants</article-title>          <comment>Available online: <ext-link xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" ext-link-type="uri">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-08-25">(accessed on 25 August 2021)</date-in-citation>        ) [15](          <person-group person-group-type="author"><collab>World Health Organization</collab></person-group>          <article-title>Tracking SARS-CoV-2 variants</article-title>          <comment>Available online: <ext-link xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" ext-link-type="uri">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-08-25">(accessed on 25 August 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37144832">37144832</a> <b>(4)</b> Validation of a new strategy for the identification of SARS-CoV-2 variants by sequencing the spike gene by Sanger<ul><li> Similarly, there are platforms that enable genomic and phylogenetic analyses of the variants, such as <b>Nextstrain</b><xref rid="bib0045" ref-type="bibr"><sup>9</sup></xref>.<li> They were aligned with the Sequencher 5.4.6 program (Gene Codes Corporation) and the variants were identified with the <b>Nextstrain</b>&#x2013;Nextclade Web 1.10.0 web-based program, Nextclade CLI 1.7.0 (09-12-2021) (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intr0025">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> Variant nomenclature was performed according to the <b>Nextstrain</b>&#x2013;Nextclade Web 1.10.0 online program, Nextclade CLI 1.7.0 (2021-12-09) and PANGO Lineages (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/" id="intr0030">https://cov-lineages.org/</ext-link>).<li> the Sanger and NGS results analysed in <b>Nextstrain</b> generated the classification 21A and 21J, respectively, for sample 36211.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36251275">36251275</a> <b>(2)</b> High temporal resolution Nanopore sequencing dataset of SARS-CoV-2 and host cell RNAs<ul><li> The obtained draft assembly (SARS-CoV-2_Hun-1_GenomeDraft_v1) was analyzed for mutations and characterized phylogenetically with the <b>Nextstrain</b> [<xref rid="bib35" ref-type="bibr"><u>35</u></xref>] program, along with the genome from the VirStrain result (Fig.&#xA0;<xref rid="fig3" ref-type="fig"><u>3</u></xref>; <xref rid="sup9" ref-type="supplementary-material">Supplementary Fig.<li> The tree was generated by the <b>Nextstrain</b> pipeline.<li> [35](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <etal>et al.</etal></person-group><article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source>. <year>2021</year>;<volume>6</volume>:<fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36250864">36250864</a> <b>(1)</b> Genome Sequence of an Early Imported Case of SARS-CoV-2 Delta Variant (B.1.617.2 AY.122) in Iraq in April 2021<ul><li><b>Nextstrain</b> clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36250860">36250860</a> <b>(4)</b> Complete Genome Sequence of Monkeypox Virus Strain MPXV-ROK-P1-2022 Isolated from the First Monkeypox Patient in the Republic of Korea<ul><li> The maximum-likelihood-based phylogenetic tree constructed with the general time-reversible (GTR) substitution model by IQ-TREE 2.1.2 (<xref rid="B7" ref-type="bibr"><u>7</u></xref>) packaged into the <b>Nextstrain</b> analysis pipeline (<xref rid="B8" ref-type="bibr"><u>8</u></xref>) shows that strain MPXV-ROK-P1-2022 belongs to lineage B.1.1 in clade IIb, which includes 70 strains from the United States (<italic toggle="yes">n</italic> = 14), Germany (<italic toggle="yes">n</italic> = 53), the United Kingdom (<italic toggle="yes">n</italic> = 1), Switzerland (<italic toggle="yes">n</italic> = 1), and Austria (<italic toggle="yes">n</italic> = 1) (<xref rid="fig1" ref-type="fig"><u>Fig.&#xA0;1</u></xref>).<li>Phylogenetic tree of the monkeypox viruses using the <b>Nextstrain</b> analysis pipeline (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/monkeypox" ext-link-type="uri">https://github.com/<b>nextstrain</b>/monkeypox</ext-link>) (<xref rid="B8" ref-type="bibr"><u>8</u></xref>).<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36253478">36253478</a> <b>(1)</b> The effects of the COVID-19 pandemic on community respiratory virus activity<ul><li>Interestingly, <xref rid="Glos10" ref-type="list"><b>Nextstrain</b></xref> last reported influenza B/Yamagata lineage virus in March 2020 (ref.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36240259">36240259</a> <b>(1)</b> Genetic diversity and evolutionary convergence of cryptic SARS- CoV-2 lineages detected via wastewater sequencing<ul><li> Surprisingly, the sewershed sequences often lacked two of the four consensus sequences that define the B.1 PANGO lineage (GISAID G clades or <b>Nextstrain</b> &#x2018;20&#x2019; clades) of SARS-COV-2 [<xref rid="ppat.1010636.ref036" ref-type="bibr"><u>36</u></xref>].<li> [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36263061">36263061</a> <b>(5)</b> A pseudovirus system enables deep mutational scanning of the full SARS-CoV-2 spike<ul><li> We then used matUtils (<xref rid="R63" ref-type="bibr"><u>Turakhia et al., 2021</u></xref>) to separate the tree by the pre-annotated <b>Nextstrain</b> clades, and extract the number of unique <bold><italic toggle="yes">occurrences</italic></bold> of each nucleotide mutation along the tree branches for each clade.<li> Overall, this process provided a list of the counts of nucleotide mutations for each <b>Nextstrain</b> clade after filtering branches and mutations/sites expected to be affected by various possible errors.<li>We then calculated the relative rates of each type of the 12 possible nucleotide mutations at four-fold synonymous third codon positions by counting the mutation at such positions in each <b>Nextstrain</b> clade, and then normalizing the fraction of these positions that had each nucleotide identity at each site, retaining only <b>Nextstrain</b> clades with at least 5000 such mutation counts.<li> Next, we aggregated the expected and observed counts across all <b>Nextstrain</b> clades, and also across all nucleotide mutations that encoded for the same amino-acid mutation to get expected and observed counts for amino-acid mutations, and only retained amino-acid mutations with at least 20 expected counts.<li> [Turakhia et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36227810">36227810</a> <b>(1)</b> New rules for genomics-informed COVID-19 responses&#x2013;Lessons learned from the first waves of the Omicron variant in Australia<ul><li> Presently, there are no guidelines available for utilising the immense (and sometimes overwhelming) dataset, however, commonly used approaches to attempt to reduce sampling bias include subsampling the available genomes geographically and temporally [<xref rid="pgen.1010415.ref026" ref-type="bibr"><u>26</u></xref>] or using global diversity via the <b>Nextstrain</b> &#x201C;backbone&#x201D; [<xref rid="pgen.1010415.ref029" ref-type="bibr"><u>29</u></xref>].<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33679622">33679622</a>) [29](<name><surname>Tay</surname><given-names>JH</given-names></name>, <name><surname>Porter</surname><given-names>AF</given-names></name>, <name><surname>Wirth</surname><given-names>W</given-names></name>, <name><surname>Duchene</surname><given-names>S</given-names></name>. <article-title>The emergence of SARS-CoV-2 variants of concern is driven by acceleration of the evolutionary rate</article-title>. <source>medRxiv</source>. <year>2021</year>:<fpage>2021.08.29.21262799</fpage>. <comment>doi: </comment><pub-id pub-id-type="doi">10.1101/2021.08.29.21262799</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36227884">36227884</a> <b>(1)</b> Comparison of the kinetics and magnitude of antibody responses to different SARS-CoV-2 proteins in Sinopharm/BBIBP-CorV vaccinees following the BNT162b2 booster or natural infection<ul><li> All sequencing data of our laboratory was uploaded to GISAID and the figures were obtained from <b>Nextstrain</b> [<xref rid="pone.0274845.ref011" ref-type="bibr"><u>11</u></xref>].<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(3)</b> Comparison of the kinetics and magnitude of antibody responses to different SARS-CoV-2 proteins in Sinopharm/BBIBP-CorV vaccinees following the BNT162b2 booster or natural infection<ul><li> All sequencing data are uploaded to <b>Nextstrain</b> and GISAID <ext-link xlink:href="https://<b>nextstrain</b>.org/community/aicbu/ncov/srilanka" ext-link-type="uri">https://<b>nextstrain</b>.org/community/aicbu/ncov/srilanka</ext-link>.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36699377">36699377</a> <b>(2)</b> Evidente&#x2014;a visual analytics tool for data enrichment in SNP-based phylogenetic trees<ul><li> A prominent example for this is <b>Nextstrain</b> (<xref rid="vbac075-B11" ref-type="bibr">Hadfield <italic toggle="yes">et al.<li> <b>Nextstrain</b> has been instrumental in understanding the evolution and spread of SARS-CoV-2, the virus that has caused the COVID-19 pandemic.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36246573">36246573</a> <b>(1)</b> Linking contact tracing with genomic surveillance to deconvolute SARS-CoV-2 transmission on a university campus<ul><li> UTRs were clipped from all 58 samples, and multiple sequence alignment was performed with <b>Nextstrain</b> Nextalign (<xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>).<li> [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/JOSS.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36296293">36296293</a> <b>(2)</b> The Spike Protein of SARS-coV2 19B (S) Clade Mirrors Critical Features of Viral Adaptation and Coevolution<ul><li> It was evaluated using the GISAID EpiCoV database and <b>NEXTSTRAIN</b> web tool [<xref rid="B16-microorganisms-10-02017" ref-type="bibr"><u>16</u></xref>,<xref rid="B17-microorganisms-10-02017" ref-type="bibr"><u>17</u></xref>].<li>Sequences were analyzed using the Next-Generation Sequencing (NGS) analysis packages targeting areas of variations in all sequences included in the analysis, the COVID-19 genome annotator online tool for annotation and the <b>NEXTSTRAIN</b> online tool for defining clades for each sequence [<xref rid="B16-microorganisms-10-02017" ref-type="bibr"><u>16</u></xref>,<xref rid="B18-microorganisms-10-02017" ref-type="bibr"><u>18</u></xref>].<li> [16](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [17](          <article-title>GISAID&#x2014;Gisaid</article-title>          <year>2021</year>          <comment>Available online: <ext-link xlink:href="https://gisaid.org/" ext-link-type="uri">https://gisaid.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-08-15">(accessed on 15 August 2022)</date-in-citation>        ) [16](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33205830">33205830</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36328240">36328240</a> <b>(0)</b> Investigation of SARS-CoV-2 lineages and mutations circulating in a university-affiliated hospital in South Korea analyzed using Oxford Nanopore MinION sequencing<ul><li> The genomes were classified into PANGO lineages using Pangolin v.4.0.6 [<xref rid="b7-j-phrp-2022-0183" ref-type="bibr"><u>7</u></xref>], and mutation annotation was performed using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) and Coronapp (<ext-link xlink:href="http://giorgilab.unibo.it/coronannotator/" ext-link-type="uri">http://giorgilab.unibo.it/coronannotator/</ext-link>).<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36220486">36220486</a> <b>(1)</b> Genomic characteristics and viral load dynamics of a SARS-CoV-2 Omicron BA.2.2 variant from a hospitalized patient treated with molnupiravir<ul><li>BA.2.2</td></tr><tr><td><b>Nextstrain</b> clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36205601">36205601</a> <b>(3)</b> Phytest: quality control for phylogenetic analyses<ul><li> These include Phytest for quality control in a <b>Nextstrain</b>/Snakemake pipeline (<xref rid="btac664-B4" ref-type="bibr">Hadfield <italic toggle="yes">et al.<li> <xref rid="btac664-B7" ref-type="bibr">M&#xF6;lder <italic toggle="yes">et al.</italic>, 2021</xref>) (<ext-link xlink:href="https://github.com/phytest-devs/phytest-<b>nextstrain</b>-example" ext-link-type="uri">https://github.com/phytest-devs/phytest-<b>nextstrain</b>-example</ext-link>), Testing phylogenetic data with Continuous Integration features on GitHub (<ext-link xlink:href="https://github.com/phytest-devs/phytest-continuous-testing-example" ext-link-type="uri">https://github.com/phytest-devs/phytest-continuous-testing-example</ext-link>) and using Phytest to test for temporal signal (<ext-link xlink:href="https://github.com/phytest-devs/phytest-temporal-signal-example" ext-link-type="uri">https://github.com/phytest-devs/phytest-temporal-signal-example</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36246576">36246576</a> <b>(2)</b> Host transcriptional responses in nasal swabs identify potential SARS-CoV-2 infection in PCR negative patients<ul><li><b>Nextstrain</b> clade analysis of PCR positive and negative samples that passed QC for genome analysis and submitted to GISAID.<li><b>Nextstrain</b> clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36297216">36297216</a> <b>(3)</b> The Re-Identification of Previously Unidentifiable Clinical Non-Tuberculous Mycobacterial Isolates Shows Great Species Diversity and the Presence of Other Acid-Fast Genera<ul><li> Several online databases that track the pathogen identification and evolution processes in real time are already available, including the platform <b>Nextstrain</b> (available online: <uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>; accessed on 20 September 2022),which tracks the pathogenic evolution of several viruses and <italic toggle="yes">Yersinia pestis,</italic> or the Pango Network (available online: <uri xlink:href="https://www.pango.network/">https://www.pango.network/</uri>; accessed on 20 September 2022, developed for identifying SARS-CoV-2 genetic lineages of epidemiological relevance.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36325031">36325031</a> <b>(2)</b> Global estimates of the fitness advantage of SARS-CoV-2 variant Omicron<ul><li> The known problematic samples were removed according to the <b>Nextstrain</b> &#x2018;ncov&#x2019; quality control pipeline (<xref rid="R10" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>; <xref rid="R16" ref-type="bibr"><b>Nextstrain</b>, 2022</xref>).<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298754">36298754</a> <b>(0)</b> SARS-CoV-2 Transmission in Belgian French-Speaking Primary Schools: An Epidemiological Pilot Study<ul><li> Online Nextclade version 2.3.0 software as a first sequence aligner, allowing comparison to the Wuhan-hu-1/2019 (MN908947) SARS-CoV-2 reference genome and permitting a clade assignment (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, accessed on 25 August 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36276383">36276383</a> <b>(2)</b> Guiding the design of SARS-CoV-2 genomic surveillance by estimating the resolution of outbreak detection<ul><li>Single nucleotide polymorphism (SNP) distances were determined from the consensus sequences using <monospace>snp-dists</monospace> (<ext-link xlink:href="https://github.com/tseemann/snp-dists" ext-link-type="uri">https://github.com/tseemann/snp-dists</ext-link> version 0.7.0) after aligning to the reference sequence (RefSeq accession NC_045512.2) with <monospace>nextalign</monospace> [<ext-link xlink:href="https://github.com/<b>nextstrain</b>/nextclade" ext-link-type="uri">https://github.com/<b>nextstrain</b>/nextclade</ext-link> version 2.2.0 (<xref rid="B32" ref-type="bibr"><u>32</u></xref>)] and masking the first 100 and final 200 positions.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35905853">35905853</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36194597">36194597</a> <b>(25)</b> Phylodynamics of a regional SARS-CoV-2 rapid spreading event in Colorado in late 2020<ul><li> To better understand the larger, continental context of our sampling of genomes, plausible ancestors of the CU and USAFA genomes were qualitatively compared to the <b>NextStrain</b> &#x201C;ncov&#x201D; database sample of 3983 genomes accessed on May 11, 2021 [<xref rid="pone.0274050.ref022" ref-type="bibr"><u>22</u></xref>].<li> Of these, 3923 were submitted in North America between March 5, 2020 and May 10, 2021, and will be referred to as North American <b>NextStrain</b> (NANS).<li> Each mutation event was also associated with one of the eleven major <b>NextStrain</b> clades based on the ancestral context in which it first appeared.<li>Although <b>NextStrain</b> calls its groupings &#x201C;clades&#x201D;, meaning they are collections of all viruses inferred to be descended from a single common ancestor, their tracking and labelling system confusingly goes against common usage of the word &#x2018;clade&#x2019; by removing named clades from within larger clades, leading to a paraphyletic naming convention.<li>A large number of the USAFA samples are descended from a series of closely related plausible common ancestors beginning with A12 that are mostly closely related to ancestor A3 (corresponding to <b>NextStrain</b> cluster 20G) but separated by nine mutation events mostly not seen elsewhere in the <b>NextStrain</b> database.<li> Color shading indicates which main phylogroup genomes belong to, A1-A3, corresponding to <b>NextStrain</b> clades 20A, 20C and 20G, respectively.<li> All newly sequenced Colorado genomes appear to be descended from what we label the A1 ancestor, which is the likely ancestor of <b>NextStrain</b>&#x2019;s clade 20A.<li> Based on analysis using <b>NextStrain</b> [<xref rid="pone.0274050.ref022" ref-type="bibr"><u>22</u></xref>], this variant&#x2019;s early origin and competitive advantage over the original virus caused its descendants to represent 99% of genome sequences throughout the world as of August 2020, when the first of our samples were collected (<xref rid="pone.0274050.s003" ref-type="supplementary-material"><u>S3 Fig</u></xref>).<li>15 of the 21 CU sequences and 75% of the USAFA sequences are descended from A2, which is itself a descendant of A1 and is the likely ancestor of <b>NextStrain</b>&#x2019;s clade 20C, estimated to have originated in April 2020.<li> 18 have documented amino acid substitutions already in the <b>NextStrain</b> database.<li>Mutation frequency in US [<b>NextStrain</b>]</th>                  <th align="center" rowspan="1" colspan="1"/>                  <th align="left" rowspan="1" colspan="1"/>                </tr>                <tr>                  <th align="center" rowspan="1" colspan="1">Position<xref rid="t001fn002" ref-type="table-fn"><sup>1</sup></xref></th>                  <th align="center" rowspan="1" colspan="1">ORF</th>                  <th align="center" rowspan="1" colspan="1">REF<xref rid="t001fn003" ref-type="table-fn"><sup>2</sup></xref></th>                  <th align="center" rowspan="1" colspan="1">ALT<xref rid="t001fn003" ref-type="table-fn"><sup>2</sup></xref></th>                  <th align="center" rowspan="1" colspan="1">Amino acid change<xref rid="t001fn004" ref-type="table-fn"><sup>3</sup></xref></th>                  <th align="center" rowspan="1" colspan="1">August 2020 [8&#x2013;14]</th>                  <th align="center" rowspan="1" colspan="1">November 2020 [11&#x2013;14]</th>                  <th align="center" rowspan="1" colspan="1">February 2021 [2&#x2013;13]</th>                  <th align="center" rowspan="1" colspan="1">May 2021 [5&#x2013;5]</th>                  <th align="center" rowspan="1" colspan="1">Ancestor introduced<xref rid="t001fn005" ref-type="table-fn"><sup>4</sup></xref></th>                  <th align="center" rowspan="1" colspan="1"><b>NextStrain</b> clade</th>                </tr>              </thead>              <tbody>                <tr>                  <td align="center" rowspan="1" colspan="1">241.<li><sup>4</sup>The clade corresponding to an ancestral sequence was determined as the named clade in the <b>NextStrain</b> database at the time of evaluation with the most mutations shared with the &#x201C;ancestor introduced&#x201D; prior to that clade on the <b>NextStrain</b> phylogenetic tree.<li>These mutations were qualitatively cross referenced against the <b>NextStrain</b> ncov database North American frequencies (henceforth, ncov) to evaluate their mutation frequencies and determine which of the eleven major clades, as defined by <b>NextStrain</b> at the time of evaluation, were represented in our sampling (<xref rid="pone.0274050.s004" ref-type="supplementary-material"><u>S4 Fig</u></xref>, <xref rid="pone.0274050.t001" ref-type="table"><u>Table 1</u></xref>).<li>Of the 18 Colorado lineage-defining mutations that were more common in ncov, all were on the lineages leading to the series of ancestors A1, A2, or A3, corresponding to <b>NextStrain</b> clades 20A, 20C, and 20G.<li> A3, which corresponds to the common ancestor of <b>NextStrain</b>&#x2019;s 20G clade, increased rapidly from August to November 2020 to a peak of 25% in North America (<xref rid="pone.0274050.t001" ref-type="table"><u>Table 1</u></xref>, <xref rid="pone.0274050.g004" ref-type="fig"><u>Fig 4</u></xref>), and its dominant presence in the later USAFA samples may be due to stochastic events in the local community.<li> = 1% in <b>NextStrain</b> (<xref rid="pone.0274050.t001" ref-type="table"><u>Table 1</u></xref>), and three result in non-synonymous amino acid substitutions.<li> Prior sequences from the nearby Denver/Aurora area and pre-outbreak USAFA samples (likely reflecting the Colorado Spring community) indicate slow and potentially neutral evolution of variant twigs, which come from common ancestors of known expanding variants previously identified by <b>NextStrain</b>.<li> The <b>NextStrain</b> database (derived from the GISAID database) is a highly incomplete collection of existing viral strains, making it difficult to determine whether this mutation was imported to the Colorado area or arose <italic toggle="yes">de novo</italic> locally.<li>Geographic location of the Orf3a K67N mutation in <b>NextStrain</b>.<li>The only other documented instance of this variant in the <b>NextStrain</b> repository occurs in Northern Mexico.<li> Map image downloaded from <b>NextStrain</b>, which uses OpenStreetMap&#xAE;.<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36260961">36260961</a> <b>(7)</b> The Omicron variant of concern: Diversification and convergent evolution in spike protein, and escape from anti-Spike monoclonal antibodies<ul><li>On November 8, 2021, a novel SARS-CoV-2 lineage named B.1.1.529 in PANGOLIN phylogeny, 21&#xA0;M in <b>NextStrain</b>, VUI-21NOV-01 in Public Heath England (PHE), and belonging to GISAID clade GR/484&#xA0;A was reported from 100 cases in South Africa.<li> 21&#xA0;K in <b>NextStrain</b>) has the hallmark mutations listed in <xref rid="tbl0005" ref-type="table"><u>Table 1</u></xref>.<li> 21&#xA0;L in <b>NextStrain</b>, VUI-22JAN-01 in PHE, or colloquially as &#x201C;Omicron 2&#x201D;) was first reported in Gauteng on Nov 17, 2021 and carries almost all the spike RBD mutations first noted in Omicron, and both furin cleavage adjacent mutations as well as the NSP6 deletion seen in other VOCs.<li> 22&#xA0;A in <b>NextStrain</b>, V-22APR-03 first and VOC-22APR-03 later in PHE) shares all mutations/deletions with the BA.2 lineage except the following: NSP4: L438F reverted to WT (wild type); S: 69/70 deletion, L452R, F486V, Q493 (WT); ORF 6: D61 (WT); ORF 7b: L11F; N: P151S.<li> 22B in <b>NextStrain</b>, V-22APR-04 first and VOC-22APR-04 later in PHE) shares the same mutations/deletions as BA.4 except the following: M: D3N; ORF7b: L11 (WT); N: P151 (WT); synonymous SNPs: A27038G, and C27889T.<li> 22&#xA0;C in <b>NextStrain</b> and V-22MAY-01 in PHE) has been a prevalent lineage in USA in Spring 2022, and carries L452Q and S704F.<li>Omicron<hr/></th></tr><tr><th>PANGOLIN name</th><th>BA.1</th><th>BA.2</th><th>BA.2.12.1</th><th>BA.2.75</th><th>BA.4/BA.5</th><th>BA.4.6</th><th>BA.4.7</th><th>BA.5.9</th></tr></thead><tbody><tr><td><bold><b>NextStrain</b> name</bold>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36263030">36263030</a> <b>(2)</b> Fast-track development of vaccines for SARS-CoV-2: The shots that saved the world<ul><li> <bold>(A)</bold> Radial graph of emerging SARS-CoV-2 variants from December 2021 to August 2022 (built with <b>nextstrain</b>/ncov).<li> <bold>(B)</bold> Frequencies of clades from March 2022 to August 2022 (from <b>nextstrain</b>.org), and <bold>(C)</bold> classification of SARS-CoV-2 omicron variant based on a clade tree (created with Biorender.com).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36292077">36292077</a> <b>(1)</b> SARS-CoV-2 Variants Monitoring Using Real-Time PCR<ul><li>The genomes variants were analyzed by applying data published in international databases (GISAID [<xref rid="B18-diagnostics-12-02388" ref-type="bibr"><u>18</u></xref>], PangoLineages [<xref rid="B19-diagnostics-12-02388" ref-type="bibr"><u>19</u></xref>], <b>Nextstrain</b> [<xref rid="B20-diagnostics-12-02388" ref-type="bibr"><u>20</u></xref>]), as well as data created at the Central Research Institute of Epidemiology platform &#x201C;VGARus (Virus Genome Aggregator of Russia).<li> [18](          <article-title>Global Initiative on Sharing Avian Influenza Data (GISAID)</article-title>          <comment>Available online: <ext-link xlink:href="https://gisaid.org/" ext-link-type="uri">https://gisaid.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-09-27">(accessed on 27 September 2022)</date-in-citation>        ) [19](          <article-title>Pango Lineages: Latest Epidemiological Lineages of SARS-CoV-2</article-title>          <comment>Available online: <ext-link xlink:href="https://cov-lineages.org/" ext-link-type="uri">https://cov-lineages.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-09-27">(accessed on 27 September 2022)</date-in-citation>        ) [20](          <article-title>Real-Time Tracking of Pathogen Evolution, (Nextrain)</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-09-27">(accessed on 27 September 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36184049">36184049</a> <b>(6)</b> Dynamics of SARS-CoV-2 lineages in French Guiana in 2020&#x2013;2021: 4 epidemic waves with cross-influences from Europe and South America<ul><li> Consensus sequences were then submitted to Nextclade (<xref rid="bb0110" ref-type="bibr"><u>O'Toole et al., 2021</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0020">https://clades.<b>nextstrain</b>.org/</ext-link>) and Pangolin (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="ir0025">https://pangolin.cog-uk.io/</ext-link>) in order to assign them to the different clades and lineages (<xref rid="bb0110" ref-type="bibr"><u>O'Toole et al., 2021</u></xref>).<li>We used the <b>Nextstrain</b> (<xref rid="bb0075" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) ncov pipeline to infer a global phylogeny including all SARS-CoV-2 sequences from FG, and contextualized using a subsampling of sequences available on the GISAID EpiCoV database (<xref rid="bb0125" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>).<li> However, from mid-April 2020 to September, 15 out of 19 sequences were 20B, a dominant lineage in Brazil, but almost absent from continental France at that period(<xref rid="bb0100" ref-type="bibr"><b>Nextstrain</b>, 2022</xref>).<li> All SARS-CoV-2 sequences available on the GISAID EpiCov database as if January 30th, 2021 were retrieved, and included in a phylogenetic analysis using the <b>Nextstrain</b> pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="ir0005">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> Within <b>Nextstrain</b>, a random subsampling approach capping a maximum number of sequences was used.<li> [O'Toole et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [O'Toole et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [Hadfield et al., 2018](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinforma. Oxf. Engl.</source>          <volume>34</volume>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [Shu and McCauley, 2017](          <person-group person-group-type="author">            <name>              <surname>Shu</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>McCauley</surname>              <given-names>J.</given-names>            </name>          </person-group>          <article-title>G.I.S.A.I.D.-Global initiative on sharing all influenza data - from vision to reality</article-title>          <source>Euro Surveill. Bull. Eur. Sur Mal. Transm. Eur. Commun. Dis. Bull.</source>          <volume>22</volume>          <year>2017</year>          <fpage>30494</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36296228">36296228</a> <b>(1)</b> First Detection of the SARS-CoV-2 Omicron BA.5/22B in Monaco<ul><li> accessed on 12 July 2022) [<xref rid="B3-microorganisms-10-01952" ref-type="bibr"><u>3</u></xref>])/21K (Nextclade classification ((<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>; accessed on 11 July 2022) [<xref rid="B4-microorganisms-10-01952" ref-type="bibr"><u>4</u></xref>,<xref rid="B5-microorganisms-10-01952" ref-type="bibr"><u>5</u></xref>]) and BA.2/21L (<uri xlink:href="https://covariants.org">https://covariants.org</uri>; accessed on 19 July 2022) [<xref rid="B6-microorganisms-10-01952" ref-type="bibr"><u>6</u></xref>,<xref rid="B7-microorganisms-10-01952" ref-type="bibr"><u>7</u></xref>].<li> Nucleotide and amino acid changes compared to the Wuhan-Hu-1 genome were identified with the Nextclade tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>; accessed on 11 July 2022) [<xref rid="B4-microorganisms-10-01952" ref-type="bibr"><u>4</u></xref>,<xref rid="B5-microorganisms-10-01952" ref-type="bibr"><u>5</u></xref>].<li> <b>Nextstrain</b> clades and pangolin lineages were determined with the Nextclade web application (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>; accessed on 11 July 2022) [<xref rid="B4-microorganisms-10-01952" ref-type="bibr"><u>4</u></xref>,<xref rid="B5-microorganisms-10-01952" ref-type="bibr"><u>5</u></xref>] and the pangolin tool (<uri xlink:href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</uri>; accessed on 11 July 2022) [<xref rid="B22-microorganisms-10-01952" ref-type="bibr"><u>22</u></xref>], respectively.<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34292871">34292871</a>) [4](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](          <person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>E.</given-names></name></person-group>          <article-title>CoVariants: SARS-CoV-2 Mutations and Variants of Interest</article-title>          <year>2021</year>          <comment>Available online: <ext-link xlink:href="https://covariants.org/https://covariants.org/" ext-link-type="uri">https://covariants.org/https://covariants.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-19">(accessed on 19 July 2022)</date-in-citation>        ) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35083761">35083761</a>) [4](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><u>No PMID</u> <b>(8)</b> Rapid evolution of SARS-CoV-2 in domestic cats<ul><li> Custom <b>Nextstrain</b> builds were prepared separately for sequences from domestic cats alone&#xA0;(<xref rid="F6" ref-type="fig"><u>Fig.&#xA0;6</u></xref>) and all felids (<xref rid="sup1" ref-type="supplementary-material">Supplementary Fig.<li> Sequences are colored by clade as assigned by <b>Nextstrain</b>.<li> The interactive version of this phylogeny is hosted at: <ext-link xlink:href="https://<b>nextstrain</b>.org/community/laurabashor/SARS2felids/catshumans" ext-link-type="uri">https://<b>nextstrain</b>.org/community/laurabashor/SARS2felids/catshumans</ext-link>.<li> Subsampling employed metadata available from GISAID and the <b>Nextstrain</b> (<xref rid="R22" ref-type="bibr">Hadfield et&#xA0;al.<li> Maximum likelihood time-resolved phylogenies were reconstructed and visualized from these sequence data using open source <b>Nextstrain</b> bioinformatic tools.<li> Publicly available interactive versions of these custom <b>Nextstrain</b> builds are hosted online.<li> To query the presence and location of specific consensus variants throughout the phylogeny, we used the interactive tools provided by <b>Nextstrain</b> to filter the displayed data by genotype.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36876764">36876764</a> <b>(4)</b> The diverse role of heparan sulfate and other GAGs in SARS-CoV-2 infections and therapeutics<ul><li> The image was adapted from <ext-link ext-link-type="uri" xlink:href="http://viralzone.expasy.org/9556" id="ir0050">viralzone.expasy.org/9556</ext-link> and extended with information from the open-source <ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.com" id="ir0055"><b>nextstrain</b>.com</ext-link> project and (<xref rid="bb0130" ref-type="bibr"><u>Gobeil et al., 2021</u></xref>; <xref rid="bb0205" ref-type="bibr"><u>Huang et al., 2020</u></xref>; <xref rid="bb0385" ref-type="bibr"><u>Peng et al., 2021</u></xref>).<li>Among others, HS interaction with SARS-CoV-2 involves two motifs in the wild-type Wuhan strain, at residues 453&#x2013;459 (YRLFRKS) and at residues 681&#x2013;686 (PRRARS) (<xref rid="bb0240" ref-type="bibr"><u>Kim et al., 2020</u></xref>), Compared to the SARS-CoV-2 strain Wuhan-Hu-1/2019 (accessed from the NCBI <italic>SARS-CoV-2 Variants Overview</italic> (<ext-link ext-link-type="uri" xlink:href="http://ncbi.nlm.nih.gov/activ" id="ir0105">ncbi.nlm.nih.gov/activ</ext-link>) and the <ext-link ext-link-type="uri" xlink:href="http://nexstrain.org" id="ir0110">nexstrain.org</ext-link> project (<ext-link ext-link-type="uri" xlink:href="http://<b>nextstrain</b>.org/nextrain.org" id="ir0115"><b>nextstrain</b>.org/nextrain.org</ext-link>), last accessed April 15, 2022), the highest rate of alterations in these motifs was found at residue P681, located at the S1/S2 furin cleavage site.<li> [Gobeil et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Gobeil</surname>              <given-names>S.M.-C.</given-names>            </name>            <name>              <surname>Janowska</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>McDowell</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Mansouri</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Parks</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Stalls</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Acharya</surname>              <given-names>P.</given-names>            </name>          </person-group>          <article-title>Effect of natural mutations of SARS-CoV-2 on spike structure, conformation, and antigenicity</article-title>          <source>Science</source>          <volume>373</volume>          <issue>6555</issue>          <year>2021</year>          <pub-id pub-id-type="doi">10.1126/science.abi6226</pub-id>        ) [Huang et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32747721">32747721</a>) [Peng et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34187722">34187722</a>) [Kim et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Kim</surname>              <given-names>S.Y.</given-names>            </name>            <name>              <surname>Jin</surname>              <given-names>W.</given-names>            </name>            <name>              <surname>Sood</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Montgomery</surname>              <given-names>D.W.</given-names>            </name>            <name>              <surname>Grant</surname>              <given-names>O.C.</given-names>            </name>            <name>              <surname>Fuster</surname>              <given-names>M.M.</given-names>            </name>            <name>              <surname>Linhardt</surname>              <given-names>R.J.</given-names>            </name>          </person-group>          <article-title>Characterization of heparin and severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) spike glycoprotein binding interactions</article-title>          <source>Antiviral Research</source>          <volume>181</volume>          <year>2020</year>          <object-id pub-id-type="publisher-id">104873</object-id>          <pub-id pub-id-type="doi">10.1016/j.antiviral.2020.104873</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36179885">36179885</a> <b>(0)</b> Genome characterization of monkeypox cases detected in India: Identification of three sub clusters among A.2 lineage<ul><li> C 34472 T reported during earlier studies (https://master.clades.<b>nextstrain</b>.org) (<xref rid="fig0002" ref-type="fig"><u>Fig.&#xA0;2</u></xref>B).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36962555">36962555</a> <b>(1)</b> The impact of sampling bias on viral phylogeographic reconstruction<ul><li> in <b>nextstrain</b>) [<xref rid="pgph.0000577.ref020" ref-type="bibr"><u>20</u></xref>].<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36170362">36170362</a> <b>(6)</b> Microfluidic affinity selection of active SARS-CoV-2 virus particles<ul><li> Data are found at <ext-link xlink:href="http://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/" ext-link-type="uri">www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/</ext-link>.<li>In December 2020 (<ext-link xlink:href="http://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>" ext-link-type="uri">www.gisaid.org/phylodynamics/global/<b>nextstrain</b></ext-link>) (<xref rid="R38" ref-type="bibr"><italic toggle="yes">38</italic></xref>), most of samples collected in eastern KS and western MO, which is the source of the samples analyzed in this study, also showed the presence of the 20A clade of SARS-CoV-2 (<xref rid="F1" ref-type="fig">Fig.<li> On the basis of sequencing data (<ext-link xlink:href="http://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>" ext-link-type="uri">www.gisaid.org/phylodynamics/global/<b>nextstrain</b></ext-link>) (<xref rid="R38" ref-type="bibr"><italic toggle="yes">38</italic></xref>), the most frequently detected SARS-CoV-2 clade in KS and MO at that time was the Delta variant (<xref rid="F1" ref-type="fig">Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36167869">36167869</a> <b>(7)</b> Detecting SARS-CoV-2 variants in wastewater and their correlation with circulating variants in the communities<ul><li> SARS-CoV-2 sequences of the randomly selected clinical samples from the individuals of the Reno-Sparks metropolitan area between November 1, 2020, and June 30, 2021, were displayed through <ext-link ext-link-type="uri" xlink:href="http://www.auspice.us">http://www.auspice.us</ext-link> (genomic epidemiology of novel coronavirus built on <ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>/ncov">http://www.<b>nextstrain</b>/ncov</ext-link>) (Fig.&#xA0;<xref rid="Fig3" ref-type="fig"><u>3</u></xref>).<li> Figure generated using <b>NextStrain</b><sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup>.<li> Figure generated using <b>NextStrain</b><sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup>.<li> Figure generated using <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>)<sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup>.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298486">36298486</a> <b>(1)</b> Relationship between Humoral Response in COVID-19 and Seasonal Influenza Vaccination<ul><li> This period was dominated by infections with <b>Nextstrain</b> clades 20A, 20B, and 20C [<xref rid="B15-vaccines-10-01621" ref-type="bibr"><u>15</u></xref>], which did not reveal major differences in clinical outcomes [<xref rid="B16-vaccines-10-01621" ref-type="bibr"><u>16</u></xref>,<xref rid="B17-vaccines-10-01621" ref-type="bibr"><u>17</u></xref>].<li> [15](          <article-title>Genomic Epidemiology of SARS-CoV-2 with Subsampling Focused Globally since Pandemic Start</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org/ncov/gisaid/global/" ext-link-type="uri">https://nextstrain.org/ncov/gisaid/global/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-09-22">(accessed on 22 September 2022)</date-in-citation>        ) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34571275">34571275</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35011858">35011858</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298686">36298686</a> <b>(1)</b> GPS Tracking of Free-Roaming Cats (<italic toggle="yes">Felis catus</italic>) on SARS-CoV-2-Infected Mink Farms in Utah<ul><li>Phylogenetic relations of the August 2020 investigation sequences were inferred using maximum likelihood analyses implemented in Treetime using the <b>NextStrain</b> pipeline [<xref rid="B39-viruses-14-02131" ref-type="bibr"><u>39</u></xref>].<li> [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298683">36298683</a> <b>(1)</b> Intrahost SARS-CoV-2 k-mer Identification Method (iSKIM) for Rapid Detection of Mutations of Concern Reveals Emergence of Global Mutation Patterns<ul><li> An additional custom set of 3243 background reference samples were obtained from the <b>NextStrain</b> SARS-CoV-2 global build [<xref rid="B49-viruses-14-02128" ref-type="bibr"><u>49</u></xref>] and added.<li> [49](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36161327">36161327</a> <b>(1)</b> <italic toggle="yes">tidy tree</italic>: A New Layout for Phylogenetic Trees<ul><li>Second, we retrieved the phylogeny of 79 measles virus genomes from the United States, using the <b>NextStrain</b> platform (<xref rid="msac204-B4" ref-type="bibr">Hadfield et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36325034">36325034</a> <b>(3)</b> SPREAD 4: online visualisation of pathogen phylogeographic reconstructions<ul><li> Such web-based technologies have also permeated the research fields of phylogenetic and phylodynamic inference, and several popular tools have emerged in the past decade such as Evolview (<xref rid="R20" ref-type="bibr"><u>He et&#xA0;al., 2016</u></xref>), iTol (<xref rid="R26" ref-type="bibr"><u>Letunic and Bork, 2019</u></xref>), Microreact (<xref rid="R2" ref-type="bibr"><u>Argim&#xF3;n et&#xA0;al., 2016</u></xref>), and <b>Nextstrain</b> (<xref rid="R1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>).<li>Owing to their flexibility in visualizing analyses for a wide range of pathogens and the ease with which the results can be shared via social media, <b>Nextstrain</b> (<xref rid="R18" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) and Nextclade (<xref rid="R1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>) have become primary examples that achieved broad visibility during the COVID-19 pandemic caused by the severe acute respiratory syndrome coronavirus 2 virus (SARS-CoV-2).<li> The <b>Nextstrain</b> package offers a pipeline that combines data collection, phylogenetic analysis, and visualisation of the resulting trees as well as dedicated post-processing packages that focus on the visualisation task.<li> [He et&#xA0;al., 2016](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27131786">27131786</a>) [Letunic and Bork, 2019](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30931475">30931475</a>) [Argim&#xF3;n et&#xA0;al., 2016](<person-group person-group-type="author"><string-name><surname>Argim&#xF3;n</surname><given-names>S.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2016</year>) &#x2018;<article-title>Microreact: visualizing and sharing data for genomic epidemiology and phylogeography</article-title>&#x2019;, <source><italic toggle="yes">Microbial genomics</italic></source>, <volume>2</volume>: <page-range>e000093</page-range>.) [Aksamentov et&#xA0;al., 2021](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2021</year>) &#x2018;<article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>&#x2019;, <source><italic toggle="yes">Journal of open source software</italic></source>, <volume>6</volume>: <page-range>3773</page-range>. doi: <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et&#xA0;al., 2021](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2021</year>) &#x2018;<article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>&#x2019;, <source><italic toggle="yes">Journal of open source software</italic></source>, <volume>6</volume>: <page-range>3773</page-range>. doi: <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36167254">36167254</a> <b>(11)</b> Characteristics of replication and pathogenicity of SARS-CoV-2 Alpha and Delta isolates<ul><li> The phylogenetic reconstruction was performed using 3087 representative global sequences retrieved from <b>Nextstrain</b> (https://<b>nextstrain</b>.org; as of 8<sup>th</sup> February 2022) and the two consensus genomes obtained in the current study using IQTREE (v1.6.12) (<xref rid="bib28" ref-type="bibr"><u>Nguyen et&#xA0;al., 2015</u></xref>).<li> The sequencing data (3087 genomes) were assembled and aligned with viral genomes of the existed SARS-CoV-2 variants by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0010">https://<b>nextstrain</b>.org/</ext-link>) on Feb 8, 2022.<li> Phylogenetic tree analysis within <b>Nextstrain</b> database (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0015">https://<b>nextstrain</b>.org/</ext-link>) showed that the Alpha (DWS1216) and Delta (DWSJ070) isolates were most close to one isolated from Cambodia (KHM B117 2021) (<xref rid="appsec1" ref-type="sec"><u>Supplementary Fig.&#xA0;S1</u></xref>) and from Georgia (USA GA GPHL 2509 2021) and Utah (USA UT CDC ASC210489950 2021) (<xref rid="appsec1" ref-type="sec"><u>Supplementary Fig.&#xA0;S2</u></xref>), respectively.<li> <bold>A</bold> Phylogenetic analysis of the spike sequences of the SARS-CoV-2 isolates within the existed SARS-CoV-2 variants by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="intref0025">https://<b>nextstrain</b>.org/</ext-link>).<li> [Nguyen et&#xA0;al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36213661">36213661</a> <b>(1)</b> Genomic profile of SARS-CoV-2 Omicron variant and its correlation with disease severity in Rajasthan<ul><li>Fast alignment sequence test for application (FASTA) sequences generated from generate consensus plugin was downloaded and these were aligned with Wuhan-Hu-1/2019 (Genbank: <ext-link xlink:href="MN908947" ext-link-type="DDBJ/EMBL/GenBank">MN908947</ext-link>) as a reference, and the sequences were downloaded from GISAID (<xref rid="B7" ref-type="bibr"><u>7</u></xref>) using Nextclade (<ext-link xlink:href="https://clades.<b>Nextstrain</b>.org" ext-link-type="uri">https://clades.<b>Nextstrain</b>.org</ext-link>).<li> The phylogenetic tree was constructed, downloaded, and visualized using <b>Nextstrain</b> Auspice (accessed on 05 February 2022) web software.<li> [7](<person-group person-group-type="author"><collab>Global Initiative on Sharing All Influenza Data (GISAID)</collab></person-group>. <source>2020 Clade and lineage nomenclature aids in genomic epidemiology studies of active hCoV-19 viruses</source>. Available online at: <ext-link xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link> (accessed February 02, 2022).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36213636">36213636</a> <b>(2)</b> Whole genome sequencing analysis of SARS-CoV-2 from Malaysia: From alpha to Omicron<ul><li>To facilitate the tracking of SARS-CoV-2 genetic lineages at local and global levels, several nomenclature systems are currently in use including the World Health Organization (WHO) label (<xref rid="B12" ref-type="bibr"><u>12</u></xref>), GISAID (<xref rid="B13" ref-type="bibr"><u>13</u></xref>), <b>NextStrain</b> (<xref rid="B14" ref-type="bibr"><u>14</u></xref>), and Phylogenetic Assignment of Named Global Outbreak LINeages (Pango lineages) (<xref rid="B15" ref-type="bibr"><u>15</u></xref>).<li> The 8,897 complete genomes with high coverage (see <xref rid="TS3" ref-type="supplementary-material"><u>Supplementary Table 3</u></xref>) were then used as input in the <b>Nextstrain</b> v3.0.3 SARS-CoV-2 workflow (<xref rid="B14" ref-type="bibr"><u>14</u></xref>) to construct a phylogenetic tree.<li> [12](<collab>World Health Organization.</collab><source><italic>WHO Announces Simple, Easy-To-Say Labels for SARS-CoV-2 Variants of Interest and Concern.</italic></source> (<year>2021</year>). Available online at: <ext-link xlink:href="https://www.who.int/news/item/31-05-2021-who-announces-simple-easy-to-say-labels-for-sars-cov-2-variants-of-interest-and-concern" ext-link-type="uri">https://www.who.int/news/item/31-05-2021-who-announces-simple-easy-to-say-labels-for-sars-cov-2-variants-of-interest-and-concern</ext-link><comment>(accessed May 15, 2022)</comment>.) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36151076">36151076</a> <b>(1)</b> Accumulation of mutations in antibody and CD8 T cell epitopes in a B cell depleted lymphoma patient with chronic SARS-CoV-2 infection<ul><li>The phylogenetic tree was obtained from a local <b>Nextstrain</b> installation using the supplied Snakemake workflow<sup><xref ref-type="bibr" rid="CR78"><u>78</u></xref></sup>.<li> [78](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36151084">36151084</a> <b>(1)</b> Using multiple sampling strategies to estimate SARS-CoV-2 epidemiological parameters from genomic sequencing data<ul><li> The first 130 base pairs (bp) and last 50&#x2009;bps of the aligned sequences were trimmed to remove potential sequencing artefacts in line with the <b>Nextstrain</b> protocol<sup><xref ref-type="bibr" rid="CR64"><u>64</u></xref></sup>.<li> Both datasets were then processed using the Nextclade pipeline for quality control (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [64](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36185993">36185993</a> <b>(2)</b> Chronic neuropsychiatric sequelae of SARS&#x2010;CoV&#x2010;2: Protocol and methods from the Alzheimer's Association Global Consortium<ul><li> High rates of mutation in SARS&#x2010;CoV&#x2010;2 (<ext-link xlink:href="https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/" ext-link-type="uri">https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/</ext-link>) strongly suggest that viral infectivity, including neurotropism, may not be uniform across countries impacted.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36195045">36195045</a> <b>(12)</b> Genomic and structural mechanistic insight to reveal the differential infectivity of omicron and other variants of concern<ul><li> <b>Nextstrain</b> categorized them into 13 major clades including 19A to 19B, 20A to 20J and 21A whereas PANGO (Phylogenetic Assignment of Named Global Outbreak) divided these into 1517 lineages and sub lineages [<xref rid="bib14" ref-type="bibr"><u>[14]</u></xref>, <xref rid="bib15" ref-type="bibr"><u>[15]</u></xref>, <xref rid="bib16" ref-type="bibr"><u>[16]</u></xref>].<li> Additionally, a collection of 2365 high-quality reference genomes comprising different lineages in Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0015">https://clades.<b>nextstrain</b>.org/</ext-link>) were included as a reference for phylogenetic interpretation [<xref rid="bib36" ref-type="bibr"><u>36</u></xref>].<li>The pipeline from <b>Nextstrain</b> was employed to screen the presence of Single Nucleotide Polymorphisms (SNPs) and Insertions and Deletions (InDels) in the various lineages of SARS-CoV-2.<li> A genome-wide mutation detection analysis was also performed from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0020">https://github.com/<b>nextstrain</b>/ncov</ext-link>) [<xref rid="bib36" ref-type="bibr"><u>36</u></xref>].<li> Phylogenomic analysis was also performed with the <b>Nextstrain</b> workflow available at <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0030">https://github.com/<b>nextstrain</b>/ncov</ext-link>.<li> The phylogenetic analysis was performed using <b>Nextstrain</b> pipelines, which contain an integrated Augur software for analysis and auspice viewer to visualise the data [<xref rid="bib40" ref-type="bibr"><u>40</u></xref>].<li> The <b>Nextstrain</b> uses its reference genomes for each lineage which are characterised by mapping with the reference genome Wuhan 2019-1 [<xref rid="bib41" ref-type="bibr"><u>41</u></xref>].<li> The phylogenetic relationship amongst each lineage was inferred by running a comparative approach using all the <b>Nextstrain</b> reference genomes as a representative of all PANGOLIN lineages [<xref rid="bib15" ref-type="bibr"><u>15</u></xref>].<li> The resulting tree was visualized using <b>Nextstrain</b> auspice and annotated with iTol [<xref rid="bib44" ref-type="bibr"><u>44</u></xref>].<li> [[14]](          <person-group person-group-type="author">            <collab>GISAID</collab>            <collab>GISAID Initiative</collab>          </person-group>          <source>Adv. Virus Res.</source>          <volume>2008</volume>          <year>2020</year>        ) [[15]](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>NextStrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics</source>          <volume>34</volume>          <year>2018</year>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [[16]](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Holmes</surname>              <given-names>E.C.</given-names>            </name>            <name>              <surname>O'Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.T.</given-names>            </name>            <name>              <surname>Ruis</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>du Plessis</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Pybus</surname>              <given-names>O.G.</given-names>            </name>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>          <source>Nat. Microbiol.</source>          <volume>5</volume>          <year>2020</year>          <pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id>        ) [36](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [36](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [40](          <person-group person-group-type="author">            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Sibley</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Lee</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Fay</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Ilcisin</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Harkins</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>          </person-group>          <article-title>Augur: a bioinformatics toolkit for phylogenetic analyses of human pathogens</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <pub-id pub-id-type="doi">10.21105/joss.02906</pub-id>        ) [41](          <person-group person-group-type="author">            <name>              <surname>Graham Carlos</surname>              <given-names>W.</given-names>            </name>            <name>              <surname>Dela Cruz</surname>              <given-names>C.S.</given-names>            </name>            <name>              <surname>Cao</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Pasnick</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Jamil</surname>              <given-names>S.</given-names>            </name>          </person-group>          <article-title>Novel wuhan (2019-NCoV) coronavirus</article-title>          <source>Am. J. Respir. Crit. Care Med.</source>          <volume>201</volume>          <year>2020</year>          <pub-id pub-id-type="doi">10.1164/rccm.2014P7</pub-id>        ) [15](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>NextStrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics</source>          <volume>34</volume>          <year>2018</year>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [44](          <person-group person-group-type="author">            <name>              <surname>Letunic</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Bork</surname>              <given-names>P.</given-names>            </name>          </person-group>          <article-title>Interactive Tree of Life (iTOL) v4: recent updates and new developments</article-title>          <source>Nucleic Acids Res.</source>          <volume>47</volume>          <year>2019</year>          <pub-id pub-id-type="doi">10.1093/nar/gkz239</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36134610">36134610</a> <b>(0)</b> Characterization of Severe Acute Respiratory Syndrome Coronavirus 2 Omicron Variant Shedding and Predictors of Viral Culture Positivity on Vaccinated Healthcare Workers With Mild Coronavirus Disease 2019<ul><li> Sequences with &gt;50 times genome coverage were used to lineage classification by Pangolin version 3.1.5 (<ext-link xlink:href="http://pangolin.cog-uk.io/" ext-link-type="uri">http://pangolin.cog-uk.io/</ext-link>)[<xref rid="jiac391-B5" ref-type="bibr"><u>5</u></xref>] and Nextclade version 1.4.0 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) and confirmed by manual genotyping.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298455">36298455</a> <b>(3)</b> Quantitative Analysis of SARS-CoV-2 Serological Responses Post Three Doses of Immunization and Prior to Breakthrough COVID-19 Infections<ul><li> Phylogenies were created using Augur from <b>Nextstrain</b>&#x2019;s bioinformatic toolkit (<uri xlink:href="https://github.com/<b>nextstrain</b>/augur">https://github.com/<b>nextstrain</b>/augur</uri>, accessed on 20 August 2022) using the ancestral genome (MN908947.3) as the reference and root for the phylogenetic trees.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298654">36298654</a> <b>(5)</b> Molecular Epidemiology of HIV-1 in Eastern Europe and Russia<ul><li>Phylogenetic trees were generated and visualised using a local instance of the <b>Nextstrain</b> workflow [<xref rid="B21-viruses-14-02099" ref-type="bibr"><u>21</u></xref>].<li> The local <b>Nextstrain</b> instance was constructed with Python v3.7.5 using the <b>nextstrain</b> augur v 6.1.1 tool suite.<li> When we compared this analysis to the <b>nextstrain</b> analysis, which has divergence-based branch lengths and is rooted on an outgroup, we observed that, although the general topology of the trees was highly similar, there were small differences within subclusters, which is probably at least partially due to the fact that, for the Georgian sequences, no exact date was available (only a range).<li>The most likely transmission routes were inferred using the <b>Nextstrain</b> workflow (<xref rid="viruses-14-02099-f002" ref-type="fig"><u>Figure 2</u></xref>B).<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36097349">36097349</a> <b>(2)</b> The outbreak of SARS&#x2010;CoV&#x2010;2 Omicron lineages, immune escape, and vaccine effectivity<ul><li>The earliest BA.2 sequence was uploaded from South Africa on November 27, 2021.<xref rid="jmv28138-bib-0021" ref-type="bibr"><sup>21</sup></xref> On December 7, the Omicron sublineage was detected in South Africa, and the <b>Nextstrain</b> defined it as 21&#x2009;L, Omicron (21&#x2009;L, BA.2).<li><b>Nextstrain</b></th>                <th align="left" valign="bottom" rowspan="1" colspan="1">GISAID clade</th>                <th align="left" valign="bottom" rowspan="1" colspan="1">Amino acid mutations in spike</th>                <th align="left" valign="bottom" rowspan="1" colspan="1">Earliest document samples</th>                <th align="left" valign="bottom" rowspan="1" colspan="1">Spread faster in some countries</th>              </tr>            </thead>            <tbody valign="top">              <tr>                <td valign="top" rowspan="1" colspan="1">BA.1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36152668">36152668</a> <b>(1)</b> Evolutionary consequences of delaying intervention for monkeypox<ul><li> Retrospective phylogenetic analysis of the large Ebola outbreak from 2013 to 2016 revealed a small number of amino acid changes associated with increases in in-vitro growth in cell culture and transmission in the population.<xref rid="bib1" ref-type="bibr"><sup>1</sup></xref> The delta and omicron variants of SARS-CoV-2 contain extensive mutations, as illustrated in <b>NextStrain</b>,<xref rid="bib2" ref-type="bibr"><sup>2</sup></xref> and are associated with increases in effective R.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36131001">36131001</a> <b>(0)</b> Tracking SARS-COV-2 variants using Nanopore sequencing in Ukraine in 2021<ul><li> Mutation calling and sequence quality were analysed using Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) and Geneious Prime 2021.2.2 software (Biomatters, New Zealand).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36130056">36130056</a> <b>(3)</b> Reproducible acquisition, management and meta-analysis of nucleotide sequence (meta)data using q2-fondue<ul><li>The initial list of SARS-CoV-2 whole genome sequencing data accession IDs was generated based on the metadata obtained from the <b>Nextstrain</b>.org platform (<xref rid="btac639-B15" ref-type="bibr">Hadfield <italic toggle="yes">et al.<li> <ext-link xlink:href="https://data.<b>nextstrain</b>.org/files/ncov/open/metadata.tsv.gz" ext-link-type="uri">https://data.<b>nextstrain</b>.org/files/ncov/open/metadata.tsv.gz</ext-link>, access date: February 08, 2022).<li> Obtained metadata was then supplemented with the original <b>Nextstrain</b> metadata by merging the two datasets on a common SRA run ID and only samples sequenced using single-end reads on the Illumina NextSeq 500/550 platforms were retained.<li> We used all of the pre-processed metadata obtained through the <b>Nextstrain</b>.org platform (<xref rid="btac639-B15" ref-type="bibr">Hadfield <italic toggle="yes">et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36150783">36150783</a> <b>(2)</b> Data capture and sharing in the COVID-19 pandemic: a cause for concern<ul><li> For example, sequencing of SARS-CoV-2 variants has been facilitated through several viral sequencing consortia including <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="interrefs60">GISAID</ext-link>, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="interrefs70"><b>Nextstrain</b></ext-link>, and <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/" id="interrefs80">Pango</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36297127">36297127</a> <b>(3)</b> SARS-CoV-2 Alpha-Variant Outbreak Amongst a Partially Vaccinated Long-Term Care Facility Population in The Netherlands&#x2014;Phylogenetic Analysis and Infection Control Observations<ul><li> Samples with a reference coverage of 70% or more were analyzed using Pangolin (<uri xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</uri>, accessed on 9 September 2021) and <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 9 September 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36714870">36714870</a> <b>(1)</b> Frequent occurrence of mutations in nsp3 and nsp4 of SARS-CoV-2, presumably caused by the inhaled asthma drug ciclesonide<ul><li> The whole genome sequences of these mutants, registered in the database between 2020 January and 2022 March were downloaded and phylogenetic trees were constructed using the <b>Nextstrain</b> interface.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36417247">36417247</a> <b>(0)</b> Molecular Epidemiological Investigations of Localized SARS-CoV-2 Outbreaks-Utility of Public Algorithms<ul><li>The clustering of patient CoV-2 sequences locally and within global databases was performed by Nextclade&#x2019;s algorithm (v1.14.1) <uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>) [<xref rid="B12-epidemiologia-03-00031" ref-type="bibr"><u>12</u></xref>] and UShER (<uri xlink:href="https://genome.ucsc.edu/cgi-bin/hgPhyloPlace">https://genome.ucsc.edu/cgi-bin/hgPhyloPlace</uri>) [<xref rid="B10-epidemiologia-03-00031" ref-type="bibr"><u>10</u></xref>].<li> [12](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36121299">36121299</a> <b>(4)</b> The Spike-Stabilizing D614G Mutation Interacts with S1/S2 Cleavage Site Mutations To Promote the Infectious Potential of SARS-CoV-2 Variants<ul><li> The phylogenetic tree was generated by the <b>Nextstrain</b> open-source project (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>), with data points colorized according to spike genotype at position 681 (turquoise, P681; yellow, H681; blue, R681; orange, L681).<li> The branches are labeled with SARS-CoV-2 clades according to the <b>Nextstrain</b> nomenclature, with variant names in parentheses.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36141993">36141993</a> <b>(3)</b> Monkeypox Virus Infections in Southern Italy: Is There a Risk for Community Spread?<ul><li> Phylogenetic analysis was performed using the <b>Nextstrain</b> web app (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 17 August 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36145490">36145490</a> <b>(2)</b> SARS-CoV-2 Variants Identification: Overview of Molecular Existing Methods<ul><li> Originally, the scientist identified these emerging variants using genetic lineages determined by programs based on phylogenetic framework such as GISAID [<xref rid="B51-pathogens-11-01058" ref-type="bibr"><u>51</u></xref>], <b>Nextstrain</b> [<xref rid="B52-pathogens-11-01058" ref-type="bibr"><u>52</u></xref>] or Pango [<xref rid="B53-pathogens-11-01058" ref-type="bibr"><u>53</u></xref>].<li> These algorithms were developed by the two principal teams of expert (<b>Nextstrain</b> and Pango) that are monitoring and assigning the clades/lineage variation.<li> [51](          <person-group person-group-type="author"><collab>Gisaid</collab></person-group>          <article-title>Gisaid</article-title>          <comment>Available online: <ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-27">(accessed on 27 July 2022)</date-in-citation>        ) [52](          <article-title>Nextstrain-Real Time RT Tracking of Pathogen Evolution</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org" ext-link-type="uri">https://nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-27">(accessed on 27 July 2022)</date-in-citation>        ) [53](          <person-group person-group-type="author"><name><surname>O&#x2019;Toole</surname><given-names>&#xC1;.</given-names></name><name><surname>Scher</surname><given-names>E.</given-names></name><name><surname>Underwood</surname><given-names>A.</given-names></name><name><surname>Jackson</surname><given-names>B.</given-names></name><name><surname>Hill</surname><given-names>V.</given-names></name><name><surname>McCrone</surname><given-names>J.T.</given-names></name><name><surname>Colquhoun</surname><given-names>R.</given-names></name><name><surname>Ruis</surname><given-names>C.</given-names></name><name><surname>Abu-Dahab</surname><given-names>K.</given-names></name><name><surname>Taylor</surname><given-names>B.</given-names></name><etal/></person-group>          <article-title>Assignment of Epidemiological Lineages in an Emerging Pandemic Using the Pangolin Tool</article-title>          <comment>Available online: <ext-link xlink:href="https://cov-lineages.org/" ext-link-type="uri">https://cov-lineages.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-27">(accessed on 27 July 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36187667">36187667</a> <b>(3)</b> Understanding spatiotemporal symptom onset risk of Omicron BA.1, BA.2 and hamster-related Delta AY.127<ul><li> The Whole Genome Sequencing data can be found at the <b>Nextstrain</b> repository: <ext-link xlink:href="https://<b>nextstrain</b>.org/groups/ncovHK/ncov/Latest?c=transmission_alert&amp;p%24=%24full" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/ncovHK/ncov/Latest?c=transmission_alert&amp;p%24=%24full</ext-link>; The real-time effective reproductive number data can be found at <ext-link xlink:href="https://covid19.sph.hku.hk/" ext-link-type="uri">https://covid19.sph.hku.hk/</ext-link>; The vaccination data can be found at <ext-link xlink:href="https://www.covidvaccine.gov.hk/en/dashboard" ext-link-type="uri">https://www.covidvaccine.gov.hk/en/dashboard</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36122669">36122669</a> <b>(3)</b> Non-severe COVID-19 complicated by cytotoxic lesions of the corpus callosum (mild encephalitis/encephalopathy with a reversible splenial lesion): a case report and literature review<ul><li>A 61-year-old Japanese woman presented to the emergency department with acute stroke-like symptoms in January 2022 (Day 1), the predominant phase of SARS-CoV-2 Omicron sublineage BA.1.1 (Omicron BA 1.1 or <b>Nextstrain</b> clade 21K: 91%; Delta or <b>Nextstrain</b> clade 21J: 9% in Japan(<xref rid="bib0018" ref-type="bibr"><italic><b>Nextstrain</b>/Ncov/Gisaid/Global/6m</italic>, n.d.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36143099">36143099</a> <b>(1)</b> Evaluation of a Triage Checklist for Mild COVID-19 Outpatients in Predicting Subsequent Emergency Department Visits and Hospitalization during the Isolation Period: A Single-Center Retrospective Study<ul><li> The tree was generated by <b>Nextstrain</b> [<xref rid="B10-jcm-11-05444" ref-type="bibr"><u>10</u></xref>,<xref rid="B11-jcm-11-05444" ref-type="bibr"><u>11</u></xref>].<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36113162">36113162</a> <b>(2)</b> Outbreak of SARS-CoV-2 B.1.617.2 (Delta Variant) in a Youth Camp Associated With Community Spread, Nebraska, June&#x2013;July 2021<ul><li> NPHL used the <b>NextStrain</b> pipeline to align SARS-CoV-2 consensus sequences and generate the phylogenetic tree.<li> We used Auspice.us (<b>NextStrain</b>) for further visualization.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36111556">36111556</a> <b>(1)</b> Emergence of genetically linked vaccine-originated poliovirus type 2 in the absence of oral polio vaccine, Jerusalem, April to July 2022<ul><li> Phylogenetic analysis via <b>Nextstrain</b> augur pipeline [<xref rid="r7" ref-type="bibr"><u>7</u></xref>] and mutational analyses of the P1 region revealed additional mutations shared across all 2022 sequences that were not detected in the sequences from Bnei Brak or those from 2020 and 2021: T981C (VP2), A1590G (VP2), G1635A (VP2), T2460C (VP3) (<xref rid="f3" ref-type="fig"><u>Figure 3</u></xref>).<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36106981">36106981</a> <b>(3)</b> Phylodynamic analysis of SARS-CoV-2 spread in Rio de Janeiro, Brazil, highlights how metropolitan areas act as dispersal hubs for new variants<ul><li> As representative sequences of the global diversity, we used the genomes belonging to the Delta clade in the SARS-CoV-2 global phylogeny (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) generated through the <b>Nextstrain</b> pipeline [<xref rid="R27" ref-type="bibr"><u>27</u></xref>].<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36108049">36108049</a> <b>(9)</b> The evolving SARS-CoV-2 epidemic in Africa: Insights from rapidly expanding genomic surveillance<ul><li>The African regional- (table S1) and country-specific (table S2) <b>NextStrain</b> builds also clearly support the changing nature of the pandemic over time.<li>Regional and country specific <b>NextStrain</b> builds.<li>In order to investigate more granular changes in lineage dynamics within a specific country or region in Africa we utilized the <b>NextStrain</b> pipeline (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) to generate the regional and country-specific builds for African countries (<xref rid="R75" ref-type="bibr"><italic toggle="yes">75</italic></xref>).<li> The methodological pipeline for <b>NextStrain</b> is well documented and performs all analyses within one workflow, including filtering of sequences, alignment, tree inference, molecular clock and ancestral state reconstruction.<li> For more information please visit, <ext-link xlink:href="https://docs.<b>nextstrain</b>.org/en/latest/index.html" ext-link-type="uri">https://docs.<b>nextstrain</b>.org/en/latest/index.html</ext-link>.<li> All builds are publicly available under the links provided in tables S1 and S2 as well as on the <b>NextStrain</b> webpage (<ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/#datasets" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2/#datasets</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35931073">35931073</a> <b>(5)</b> Determinants of Spike infectivity, processing, and neutralization in SARS-CoV-2 Omicron subvariants BA.1 and BA.2<ul><li> Retrieved from <b>Nextstrain</b> on June 24, 2022 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/open/global/6m?l=radial" id="intref0150">https://<b>nextstrain</b>.org/ncov/open/global/6m?l=radial</ext-link>) and modified.<li> Retrieved and modified from <b>Nextstrain</b> on June 24, 2022.<li> Retrieved and modified from <b>Nextstrain</b> on April 7, 2022.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36140815">36140815</a> <b>(3)</b> Simultaneous SARS-CoV-2 Genome Sequencing of 384 Samples on an Illumina MiSeq Instrument through Protocol Optimization<ul><li>The consensus sequences in fasta format were analyzed using Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 13 September 2022) [<xref rid="B15-genes-13-01648" ref-type="bibr"><u>15</u></xref>] to determine the <b>Nextstrain</b> clades and identify mutations (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 13 September 2022 [<xref rid="B15-genes-13-01648" ref-type="bibr"><u>15</u></xref>].<li> [15](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [15](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36191361">36191361</a> <b>(1)</b> Quantitative analysis of RT-PCR test results for SARS-CoV-2 diagnostics across Poland during COVID-19 pandemic: Comparison between early stage and major pandemic waves in 2020 and 2021 with reference to SARS-CoV-2 variants<ul><li> SARS-CoV-2 variants were identified with the Pangolin tool, while <b>Nextstrain</b> was used to identify the GISAID clades and amino acid (aa) substitutions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36100631">36100631</a> <b>(15)</b> The roles of APOBEC-mediated RNA editing in SARS-CoV-2 mutations, replication and fitness<ul><li> With the direct experimental evidence described here that APOBECs can target specific sites on SARS-CoV-2 for editing, we analyzed the publicly available SARS-CoV-2 genome sequence data (the <b>Nextstrain</b> datasets from Dec.<li> 22nd, 2022 downloaded from the GISIAD database, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>)<sup><xref ref-type="bibr" rid="CR37"><u>37</u></xref></sup>, with hope to detect some obvious effects of APOBEC-mediated viral mutations on the current viral strain variants and fitness.<li> (<bold>A</bold>) The number of mutational events (all single nucleotide variants) on SARS-CoV-2 RNA segment 5&#x2019;UTR-Orf1a (segment 1 in Fig.&#xA0;<xref rid="Fig1" ref-type="fig"><u>1</u></xref>A) from the SARS-CoV-2 genome sequence data (the <b>Nextstrain</b> datasets from Dec.<li> 22nd, 2022 downloaded from the GISAID database, <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/hcov19-variants/">https://www.gisaid.org/hcov19-variants/</ext-link> and <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> The sequencing frequency is represented by C in blue and U in yellow (referred to the <b>Nextstrain</b> datasets: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>).<li> The number of mutational events on SARS-CoV-2 RNA segment 5&#x2032;UTR-Orf1a (in Fig.&#xA0;<xref rid="Fig4" ref-type="fig"><u>4</u></xref>A) was counted from the SARS-CoV-2 genome sequence data (the <b>Nextstrain</b> datasets from Dec.<li> 22nd, 2022 downloaded from the GISAID database, <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/hcov19-variants/">https://www.gisaid.org/hcov19-variants/</ext-link> and <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) using parameters set with nucleotide (x-axis) and events (y-axis).<li> The sequencing frequency chart (in Fig.&#xA0;<xref rid="Fig4" ref-type="fig"><u>4</u></xref>C) was indicated by designating the corresponding nucleotide as colored by Genotype (referred to the <b>Nextstrain</b> datasets: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>, <sup><xref ref-type="bibr" rid="CR37"><u>37</u></xref></sup>.<li> [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36094771">36094771</a> <b>(4)</b> Evolution, epidemiology, geographical distribution, and mutational landscape of newly emerging monkeypox virus<ul><li>We used the <b>Nextstrain</b> server for the data analysis and retrieval [<xref ref-type="bibr" rid="CR39"><u>39</u></xref>], which uses the metadata and curated sequence data directly fetched by the NCBI virus.<li> Different lineages of the virus were denoted as per the <b>Nextstrain</b> server, and Happi et al.<li> We analyzed our data using the <b>Nextstrain</b> server.<li> Currently, the <b>Nextstrain</b> server used the lineage and sublineage instead of the clade.<li> [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36056719">36056719</a> <b>(6)</b> Whole&#x2010;genome sequencing of SARS&#x2010;CoV&#x2010;2: Comparison of target capture and amplicon single molecule real&#x2010;time sequencing protocols<ul><li> Consensus sequences were then analyzed with Pangolin lineages (v3.1.20, <ext-link xlink:href="https://cov-lineages.org/resources/pangolin.html" ext-link-type="uri">https://cov-lineages.org/resources/pangolin.html</ext-link>) and <b>Nextstrain</b> clades (v1.11.0, <ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2</ext-link>).<li> The consensus sequences were then analyzed with Pangolin (v3.1.20, <ext-link xlink:href="https://cov-lineages.org/resources/pangolin.html" ext-link-type="uri">https://cov-lineages.org/resources/pangolin.html</ext-link>) for lineages and <b>Nextstrain</b> (v1.11.0, <ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2</ext-link>) for clades.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36031728">36031728</a> <b>(5)</b> The emergence, spread and vanishing of a French SARS&#x2010;CoV&#x2010;2 variant exemplifies the fate of RNA virus epidemics and obeys the Mistigri rule<ul><li> We identified in July 2020 at the University Hospital Institute (IHU) M&#xE9;diterran&#xE9;e Infection, Marseille, Southern France (which generated about 20% of the genomes deposited in GISAID (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>)<xref rid="jmv28102-bib-0012" ref-type="bibr"><sup>12</sup></xref> by France as on December 16, 2021), a new SARS&#x2010;CoV&#x2010;2 variant, named Marseille&#x2010;4 (later classified as lineage 20A.EU2 and B.1.160 in <b>Nextstrain</b><xref rid="jmv28102-bib-0013" ref-type="bibr"><sup>13</sup></xref> and Pangolin<xref rid="jmv28102-bib-0014" ref-type="bibr"><sup>14</sup></xref> classifications).<li> (A)&#xA0;The phylogenetic tree is adapted from a screenshot of the nextclade web application (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).<li> Mutation detection was performed using the Nextclade tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) and freebayes (<ext-link xlink:href="https://github.com/freebayes/freebayes" ext-link-type="uri">https://github.com/freebayes/freebayes</ext-link>)<xref rid="jmv28102-bib-0053" ref-type="bibr"><sup>53</sup></xref> using a mapping quality score of 20 and results filtered by the Python script based on major nucleotide frequencies &#x2265;70% and nucleotide depths&#x2009;&#x2265;10 (when sequence reads were generated on the NovaSeq Illumina instrument) or &#x2265;5 (when sequence reads were generated on the MiSeq Illumina instrument).<li> <b>Nextstrain</b> clades and Pangolin lineages provided in the present study were determined using the Nextclade web application (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>)<xref rid="jmv28102-bib-0013" ref-type="bibr"><sup>13</sup></xref>, <xref rid="jmv28102-bib-0054" ref-type="bibr"><sup>54</sup></xref> and the Pangolin web application (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>),<xref rid="jmv28102-bib-0014" ref-type="bibr"><sup>14</sup></xref> respectively.<li>Phylogenetic reconstruction based on SARS&#x2010;CoV&#x2010;2 genomes were performed for genome sequences &gt;24&#x2009;000 nucleotide&#x2010;long using the <b>Nextstrain</b>/ncov tool (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) and then were visualized using Auspice (<ext-link xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/" ext-link-type="uri">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36059082">36059082</a> <b>(2)</b> On the SARS&#x2010;CoV&#x2010;2 BA.2.75 variant: A genetic and structural point of view<ul><li>The phylogenomic reconstruction, available at <ext-link xlink:href="https://gisaid.org/phylodynamics/global/<b>nextstrain</b>/" ext-link-type="uri">https://gisaid.org/phylodynamics/global/<b>nextstrain</b>/</ext-link>, indicates that the evolutionary condition of BA.2.75 seems to be very similar to the sublineage BA.2.12.1, which represents an evolutionary blind background with no further descendant.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36164552">36164552</a> <b>(0)</b> Occurrence of SARS-CoV-2 reinfections at regular intervals in Ecuador<ul><li> Consensus sequence quality was assessed by Nextclade v1.14.0 parameters (clades.<b>nextstrain</b>.org) and were uploaded to the GISAID repository (gisaid.org).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36483428">36483428</a> <b>(2)</b> Factors contributing to a coronavirus disease 2019 (COVID-19) outbreak on a mixed medical-surgical unit in a Canadian acute-care hospital<ul><li> Consensus sequences were generated using the nCoV-tools analysis pipeline, based on Next-Strain/Augur (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri> and <uri xlink:href="https://github.com/jts/ncov-tools">https://github.com/jts/ncov-tools</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36691215">36691215</a> <b>(7)</b> SARS-CoV-2 chronological genomic evolution and epidemiology in the Middle East and North Africa (MENA) region as affected by vaccination, conflict and socioeconomical disparities: a population-based cohort study<ul><li> The phylogenetic tree was visualised using the interactive Tree of Life online tool.<xref rid="R27" ref-type="bibr"><u>27</u></xref> For the phylogeographical analysis, we used <b>Nextstrain</b> V.3.2.2 and the Augur pipeline.<li> An animation can be viewed at <ext-link xlink:href="https://<b>nextstrain</b>.org/community/obeiddx/MENA/ncov" ext-link-type="uri">https://<b>nextstrain</b>.org/community/obeiddx/MENA/ncov</ext-link>.<li><xref rid="F8" ref-type="fig"><u>Figure 8B</u></xref> shows the phylogeographical analysis of SARS-CoV-2 using the <b>Nextstrain</b> pipeline.<li> An animation of the transmission analysis can be viewed on a <b>Nextstrain</b> webpage at <ext-link xlink:href="https://<b>nextstrain</b>.org/community/obeiddx/MENA/ncov" ext-link-type="uri">https://<b>nextstrain</b>.org/community/obeiddx/MENA/ncov</ext-link>.<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33885785">33885785</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36087642">36087642</a> <b>(1)</b> Critical pediatric neurological illness associated with COVID-19 (Omicron BA.2.3.7 variant) infection in Taiwan: immunological assessment and viral genome analysis in tertiary medical center<ul><li> To investigate the amino acid variations in the S region of these Taiwanese strains and classify pathogen evolution relationships, further sequence analyses were executed using Nextclade (<ext-link ext-link-type="uri" xlink:href="http://clades.<b>nextstrain</b>.org/" id="interref0001a">http://clades.<b>nextstrain</b>.org/</ext-link>) with Augur developed by <b>Nextstrain</b> (<xref rid="bib0004" ref-type="bibr">Hadfield <italic>et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36074769">36074769</a> <b>(8)</b> COVID-19 infection and transmission includes complex sequence diversity<ul><li> Data provided through the Global Initiative on Sharing All Influenza Data (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">GISAID</ext-link>), and <b>NextStrain</b> provides a continuously updating report on SARS-CoV-2 lineage classification and illustrates that an apparently low mutation rate is no guarantee that the virus will exhibit limited capacity for variation, particularly once it has become so widely dispersed through millions of daily infections [<xref rid="pgen.1010200.ref005" ref-type="bibr"><u>5</u></xref>,<xref rid="pgen.1010200.ref019" ref-type="bibr"><u>19</u></xref>,<xref rid="pgen.1010200.ref020" ref-type="bibr"><u>20</u></xref>].<li> When we queried the database of 1,801,208 SARS-Cov-2 sequences accessible through <b>Nextstrain</b>/GISAID (available up to October 3, 2021), while the repository reports extensive nucleotide substitutions, results showed that 85.9% reported no ambiguity sequence codes suggesting no biallelic polymorphism in their SARS-CoV-2 sequences; 0.46% reported &gt;10 suggesting potential for a complex multi-strain infection (<bold><xref rid="pgen.1010200.s006" ref-type="supplementary-material"><u>S6 Table</u></xref></bold>).<li> In head-to-head comparisons, nucleotide-specific substitution was higher in <b>Nextstrain</b>/GISAID compared to our study (average substitution rate = 1.69, st.<li> We greatly appreciate the work of international data repositories such as GISAID and <b>NextStrain</b> for curating the millions of reported infections and believe that these repositories are a tremendous and invaluable asset for the monitoring of SARS-CoV-2.<li>Evaluation of ambiguity code usage of sequence data available through <b>Nextstrain</b> was performed using <italic toggle="yes">awk</italic> (a command line tool for text-based pattern searching) on metadata available for 1,801,208 SARS-Cov-2 sequences accessible through <b>Nextstrain</b> up to October 3, 2021; data accessed on October 10, 2021.<li> <bold>S6E Table Average Difference in Substitutions <b>Nextstrain</b>/GISAID Compared to Current VA Study</bold>.<li> <bold>S6F Table Average Difference in Ambiguity Code Usage <b>Nextstrain</b>/GISAID Compared to Current VA Study</bold>.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33239633">33239633</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32087114">32087114</a>) [20](<name><surname>Bosch</surname><given-names>J</given-names></name>, <name><surname>Wilson</surname><given-names>A</given-names></name>, <name><surname>O&#x201D;Neil</surname><given-names>K</given-names></name>, <name><surname>Zimmerman</surname><given-names>PA</given-names></name>. <article-title>COVID-19Predict&#x2014;Predicting Pandemic Trends [Dashboard].</article-title><source>medRxiv</source><year>2020</year> [updated December 29, 2020December 29, 2020]. Available from: <ext-link xlink:href="https://covid19predict.com" ext-link-type="uri">https://covid19predict.com</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36069609">36069609</a> <b>(0)</b> A Robust, Highly Multiplexed Mass Spectrometry Assay to Identify SARS-CoV-2 Variants<ul><li> All FASTA consensus genome sequences underwent mutation calling and phylogenetic lineage assignment by the Nextclade Web Interface (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>, accessed April 18, 2022) and the Pangolin COVID-19 Lineage Assigner (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>, accessed April 18, 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36091850">36091850</a> <b>(1)</b> The Omicron Variant of the Sars-Cov-2 Virus As the Dominant Agent of a New Risk of Disease amid the COVID-19 Pandemic<ul><li> According to the GISAID project, variant B.1.1.529 was assigned the identifier clade GR/484A [<xref ref-type="bibr" rid="CR7"><u>7</u></xref>], and the <b>Nextstrain</b> project was assigned the identifier clade 21K [<xref ref-type="bibr" rid="CR8"><u>8</u></xref>].<li> [7](The unique mix of spike amino acid changes in Omicron GR/484A (B.1.1.529) is of interest as it comprises several that were previously known to affect receptor binding and antibody escape. https://www.gisaid.org/. Cited November 28, 2021.) [8](Variant 21K (Omicron) appears to have arisen in November 2021, possibly in South Africa. https://covariants.org/variants/21K.Omicron. Cited November 28, 2021.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36091861">36091861</a> <b>(2)</b> COVID-19 Epidemiology and Diagnosis: Monitoring Evolutionary Changes in the SARS-COV-2 Virus<ul><li> Phylogenetic analysis has been carried out using the Nexstrain (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>) and Pangolin (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/resources/pangolin.html">https://cov-lineages.org/resources/pangolin.html</ext-link>) classifiers.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36116149">36116149</a> <b>(8)</b> Bioinformatics pipeline unveils genetic variability to synthetic vaccine design for Indian SARS-CoV-2 genomes<ul><li> In this regard, we have performed multiple sequence alignment of 4996 Indian SARS-CoV-2 sequences as a case study using MAFFT followed by phylogenetic analysis of the aligned sequences using <b>Nextstrain</b>.<li> Thus, based on the comprehensive bioinformatics pipeline, the main contributions of this work can be summarised as: (a) phylogenetic analysis in <b>Nextstrain</b> to identify virus clades, (b) identification of SNPs in the aligned sequences, (c) based on frequency, top 10 signature SNPs identification in each virus clade, (d) identification of conserved regions and based on length selecting one such region as potential target for reporting the corresponding primers and probes to detect SARS-CoV-2 and (e) identification of T-cell and B-cell epitopes for peptide based synthetic vaccine design.<li> To achieve these goals, in the first phase of the pipeline, multiple sequence alignment of 4996 Indian SARS-CoV-2 genomes as a case study using MAFFT <xref rid="b0095" ref-type="bibr"><u>[19]</u></xref> is carried out followed by the phylogenetic analyses using <b>Nextstrain</b> <xref rid="b0100" ref-type="bibr"><u>[20]</u></xref>.<li> On the other hand, analysis of the evolution and spread of pathogens is done using <b>Nextstrain</b> by considering phylogenomic and phylogeographic data.<li> The spread and evolution of virus genomes can be visualised at <b>nextstrain</b>.org using auspice.<li> Once the identification of the virus clades are performed using <b>Nextstrain</b>, clade specific aligned sequences are used to identify mutation points as substitutions especially SNPs in each clade.<li>To achieve the first step of the bioinformatics pipeline, multiple sequence alignment of 4996 Indian SARS-CoV-2 genomes is performed using MAFFT followed by phylogenetic analysis with the help of <b>Nextstrain</b>.<li> In this regard, initially, multiple sequence alignment of 4996 Indian SARS-CoV-2 genomes as a case study are carried out using MAFFT followed by phylogenetic analysis by <b>Nextstrain</b> to identify virus clades, resulting in 5 virus clades; 19A, 19B, 20A, 20B and 20C.<li> [[19]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [[20]](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics (Oxford, England)</source>          <volume>34</volume>          <year>2018</year>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36075374">36075374</a> <b>(1)</b> SARS-CoV-2 dual infection with Delta and Omicron variants in an immunocompetent host: a case report<ul><li> PANGO lineage and <b>Nextstrain</b> clade were determined using workflows devised by the Galaxy Project (<xref rid="bib0005" ref-type="bibr">Maier <italic>et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36075557">36075557</a> <b>(1)</b> Overlapping outbreak of COVID-19 and monkeypox in 2022: Warning for immediate preparedness in Iran<ul><li> <b>Nextstrain</b> databases showed that all the human monkeypox viruses around Iran belonged to the B.1 lineage; it has been estimated that B.1 originated in Europe and after a short time it spread around the world [<xref rid="bib6" ref-type="bibr"><u>6</u></xref>].<li> [6](          <person-group person-group-type="author">            <name>              <surname>Luna</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Ram&#xED;rez</surname>              <given-names>A.L.</given-names>            </name>            <name>              <surname>Mu&#xF1;oz</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Ballesteros</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Pati&#xF1;o</surname>              <given-names>L.H.</given-names>            </name>            <name>              <surname>Casta&#xF1;eda</surname>              <given-names>S.A.</given-names>            </name>            <etal/>          </person-group>          <article-title>Phylogenomic analysis of the monkeypox virus (MPXV) 2022 outbreak: emergence of a novel viral lineage?</article-title>          <source>Trav. Med. Infect. Dis.</source>          <volume>49</volume>          <year>2022</year>          <object-id pub-id-type="publisher-id">102402</object-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36068236">36068236</a> <b>(6)</b> Genomic surveillance of SARS-CoV-2 Omicron variants on a university campus<ul><li>Maximum likelihood phylogenetic trees were constructed using the <b>Nextstrain</b> Augur software package<sup><xref ref-type="bibr" rid="CR45"><u>45</u></xref></sup> using default parameters for SARS-CoV-2 as outlined on the <b>Nextstrain</b> GitHub webpage<sup><xref ref-type="bibr" rid="CR46"><u>46</u></xref></sup>.<li> Finally, <b>Nextstrain</b> Auspice software was used for tree visualization.<li> The latter 1174 were randomly selected from among 15,406 available GISAID genomes using the <b>Nextstrain</b> Augur filter command, with the maximum number of genomes per month for September 2021 to February 2022 capped at approximately 250.<li> The latter 1512 were randomly selected from among 15,406 available GISAID genomes using the <b>Nextstrain</b> Augur filter command with the maximum number of genomes per month for December 2021 to February 2022 capped at approximately 500.<li> The <b>Nextstrain</b> Augur &#x201C;traits&#x201D; subcommand was used to infer campus versus non-campus states for all internal nodes.<li> [45](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinforma. Oxf. Engl.</source>          <year>2018</year>          <volume>34</volume>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [46](Bedford, T. et al. <italic>Nextstrain build for novel coronavirus SARS-CoV-2</italic>. <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</ext-link> (2022).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36070806">36070806</a> <b>(3)</b> A 21L/BA.2-21K/BA.1 &#x201C;MixOmicron&#x201D; SARS-CoV-2 hybrid undetected by qPCR that screen for variant in routine diagnosis<ul><li> Substantial rates of co-incidence over periods of several weeks have thus been reported recently worldwide for the Delta [WHO denomination (<ext-link ext-link-type="uri" xlink:href="https://www.who.int/fr/activities/tracking-SARS-CoV-2-variants" id="ir0010">https://www.who.int/fr/activities/tracking-SARS-CoV-2-variants</ext-link>)]/21J [Nextclade classification (<xref rid="bb0005" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/" id="ir0015">https://<b>nextstrain</b>.org/</ext-link>)] and Omicron 21K/BA.1 [Pangolin classification (<xref rid="bb0190" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/resources/pangolin.html" id="ir0020">https://cov-lineages.org/resources/pangolin.html</ext-link>)] variants, and thereafter, to a lesser extent, for the Omicron BA.1 and BA.2 variants (<ext-link ext-link-type="uri" xlink:href="https://covariants.org/per-country" id="ir0025">https://covariants.org/per-country</ext-link>) (<xref rid="bb0100" ref-type="bibr"><u>Hodcroft, 2021</u></xref>; <xref rid="bb0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), including in our geographical area (<xref rid="f0005" ref-type="fig">Fig.<li> Nucleotide and amino acid changes compared to the Wuhan-Hu-1 isolate genome were determined using the Nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0070">https://clades.<b>nextstrain</b>.org/</ext-link>) (<xref rid="bb0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bb0005" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>).<li> <b>Nextstrain</b> clades and Pangolin lineages were identified with the Nextclade web application (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0075">https://clades.<b>nextstrain</b>.org/</ext-link>) (<xref rid="bb0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bb0005" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) and the Pangolin tool (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/pangolin.html" id="ir0080">https://cov-lineages.org/pangolin.html</ext-link>) (<xref rid="bb0190" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), respectively.<li> Adapted from screenshots of the nextclade web application output (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="ir0005">https://clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="bb0005" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>; <xref rid="bb0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Zenodo</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hodcroft, 2021](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>          </person-group>          <article-title>CoVariants: SARS-CoV-2 Mutations and Variants of Interest</article-title>          <comment>Available from:</comment>          <ext-link ext-link-type="uri" xlink:href="https://covariants.org/https://covariants.org/" id="ir0130">https://covariants.org/https://covariants.org/</ext-link>          <year>2021</year>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Zenodo</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Zenodo</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Zenodo</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><u>No PMID</u> <b>(1)</b> Abstracts of the papers presented in the international conference of Indian Virological Society, VIROCON 2021 on &#x201C;Emerging and Re-emerging Viral Diseases &#x2013; Climate Change Impacts and Mitigation&#x201D;, during March 26&#x2013;28, 2022, at AIIMS, Hyderabad, India (Virtual Mode)<ul><li> To identify the global prevalence of Omicron SARS&#x2010;CoV&#x2010;2, we have used the phylogenetic assignment of named global outbreak (PANGO) lineage based on the <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36144387">36144387</a> <b>(41)</b> Towards Efficient and Accurate SARS-CoV-2 Genome Sequence Typing Based on Supervised Learning Approaches<ul><li> In&#xA0;addition, other developed sequence typing tools, <b>Nextstrain</b>&#xA0;[<xref rid="B23-microorganisms-10-01785" ref-type="bibr"><u>23</u></xref>], GISAID&#xA0;[<xref rid="B7-microorganisms-10-01785" ref-type="bibr"><u>7</u></xref>], Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin) [<xref rid="B11-microorganisms-10-01785" ref-type="bibr"><u>11</u></xref>,<xref rid="B24-microorganisms-10-01785" ref-type="bibr"><u>24</u></xref>], COVID-19 Genotyping Tool&#xA0;[<xref rid="B15-microorganisms-10-01785" ref-type="bibr"><u>15</u></xref>], and&#xA0;Genome Detective Coronavirus Typing Tool&#xA0;[<xref rid="B25-microorganisms-10-01785" ref-type="bibr"><u>25</u></xref>], are critical for tracking emerging diversity and spread of certain lineages.<li> There are 25 <b>Nextstrain</b> clades, 1725 Pango lineages, and&#xA0;11 GISAID clades as of 20 April 2022.<li> As&#xA0;for the single nucleotide polymorphism (SNP)-based classification methods, including Chinese lineage&#xA0;[<xref rid="B14-microorganisms-10-01785" ref-type="bibr"><u>14</u></xref>] and <b>Nextstrain</b>&#xA0;[<xref rid="B23-microorganisms-10-01785" ref-type="bibr"><u>23</u></xref>], are not enough to fully address the complex genetic diversity of SARS-CoV-2, for those two methods depend on mutations with significant geographic distribution and frequency or marker mutations&#xA0;[<xref rid="B27-microorganisms-10-01785" ref-type="bibr"><u>27</u></xref>].<li> Different from the previous work, the&#xA0;focus of this study is not to discover new evolutionary branches, but&#xA0;to provide a model with improved efficiency and accuracy based on existing <b>Nextstrain</b>, GISAID, and&#xA0;Pangolin classification standards.<li> In&#xA0;summary, the&#xA0;main contributions of this study are listed as follows: (1) Supervised learning-based identification models are constructed for the three typing strategies of <b>Nextstrain</b>, GISAID, and&#xA0;Pangolin, respectively, achieving rapid and accurate SARS-CoV-2 genome sequence typing.<li> (2) A multilayer template matching algorithm is proposed for SARS-CoV-2 genome sequence typing, achieving ideal results for the <b>Nextstrain</b> and GISAID clades.<li> Given the different classification densities of the three typing tools (GISAID, <b>Nextstrain</b>, and&#xA0;Pangolin), sequences of each clade or lineage were uniformly sampled according to the collection date.<li> Overall, the&#xA0;amounts of downsampled sequences are 91,772, 203,740, and&#xA0;279,899 for GISAID, <b>Nextstrain</b>, and&#xA0;Pangolin,&#xA0;respectively.<li> For&#xA0;the <b>Nextstrain</b> part, labels of sequences were obtained from the Nextclade system (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 12 July 2022)).<li> We referred to the nucleotide site screening protocol provided by Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 12 July 2022)) and PangoLEARN (<uri xlink:href="https://github.com/cov-lineages/pangoLEARN/">https://github.com/cov-lineages/pangoLEARN/</uri> (accessed on 12 July 2022)) and selected feature extraction strategies suitable for different typing methods through subsequent model training and cross-validation.<li>The template matching method was proposed for GISAID and <b>Nextstrain</b> clades.<li>The difference matrix <inline-formula><mml:math id="mm63" overflow="scroll"><mml:mrow><mml:mi mathvariant="script">D</mml:mi></mml:mrow></mml:math></inline-formula> of the 25 <b>Nextstrain</b> clades is shown in <xref rid="microorganisms-10-01785-f004" ref-type="fig"><u>Figure 4</u></xref>.<li> Taking the <b>Nextstrain</b> clade typing task as an example, <inline-formula><mml:math id="mm79" overflow="scroll"><mml:mrow><mml:msubsup><mml:mi>p</mml:mi><mml:mi>j</mml:mi><mml:mi>i</mml:mi></mml:msubsup></mml:mrow></mml:math></inline-formula> represents the probability that the <italic toggle="yes">i</italic>th classifier predicts the <italic toggle="yes">j</italic>th clade.<li> In addition, for GISAID and <b>Nextstrain</b>, this study also applies the average difference <inline-formula><mml:math id="mm85" overflow="scroll"><mml:mrow><mml:mover accent="true"><mml:mi>D</mml:mi><mml:mo stretchy="false">&#xAF;</mml:mo></mml:mover></mml:mrow></mml:math></inline-formula> to test the classifier:<disp-formula id="FD9-microorganisms-10-01785"><label>(9)</label><mml:math id="mm86" display="block" overflow="scroll"><mml:mrow><mml:mrow><mml:mover accent="true"><mml:mi>D</mml:mi><mml:mo stretchy="false">&#xAF;</mml:mo></mml:mover><mml:mo>=</mml:mo><mml:mfrac><mml:mn>1</mml:mn><mml:msub><mml:mi>N</mml:mi><mml:mi>F</mml:mi></mml:msub></mml:mfrac><mml:munderover><mml:mo>&#x2211;</mml:mo><mml:mrow><mml:mi>k</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn></mml:mrow><mml:msub><mml:mi>N</mml:mi><mml:mi>F</mml:mi></mml:msub></mml:munderover><mml:msubsup><mml:mi mathvariant="script">D</mml:mi><mml:mrow><mml:mi>i</mml:mi><mml:mi>j</mml:mi></mml:mrow><mml:mi>k</mml:mi></mml:msubsup><mml:mo>,</mml:mo></mml:mrow></mml:mrow></mml:math></disp-formula>where <inline-formula><mml:math id="mm87" overflow="scroll"><mml:mrow><mml:msub><mml:mi>N</mml:mi><mml:mi>F</mml:mi></mml:msub></mml:mrow></mml:math></inline-formula> represents the number of samples with incorrect predictions.<li>This study applied the sites provided by <b>Nextstrain</b> and Pangolin as the basis for feature extraction.<li> As of 20 April 2022, <b>Nextstrain</b> provides 83 (<inline-formula><mml:math id="mm88" overflow="scroll"><mml:mrow><mml:msub><mml:mi>F</mml:mi><mml:mrow><mml:mi>m</mml:mi><mml:mi>i</mml:mi><mml:mi>n</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula>) nucleotide sites for the classification of 25 SARS-CoV-2 clades.<li>Given the successful application of the RF classifier in Pangolin [<xref rid="B24-microorganisms-10-01785" ref-type="bibr"><u>24</u></xref>], the RF classifier was trained to classify the 25 clades of <b>Nextstrain</b>, and the initial feature scale was set to <inline-formula><mml:math id="mm92" overflow="scroll"><mml:mrow><mml:mrow><mml:msub><mml:mi>l</mml:mi><mml:mn>5</mml:mn></mml:msub><mml:mo>=</mml:mo><mml:msub><mml:mi>F</mml:mi><mml:mrow><mml:mi>m</mml:mi><mml:mi>a</mml:mi><mml:mi>x</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:math></inline-formula>.<li> Obviously, <inline-formula><mml:math id="mm101" overflow="scroll"><mml:mrow><mml:msub><mml:mi>l</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:mrow></mml:math></inline-formula> corresponds to the 83 effective sites provided by <b>Nextstrain</b>, and <inline-formula><mml:math id="mm102" overflow="scroll"><mml:mrow><mml:msub><mml:mi>l</mml:mi><mml:mn>5</mml:mn></mml:msub></mml:mrow></mml:math></inline-formula> corresponds to the 4544 sites offered by Pangolin.<li> To simplify the model training, this study applied the same feature settings as <b>Nextstrain</b> for the construction of the GISAID clade typing model.<li> <b>Nextstrain</b> Clade Typing&#xA0;Results.<li>The datasets used for the <b>Nextstrain</b> typing experiment consist of two groups: the first group (<inline-formula><mml:math id="mm103" overflow="scroll"><mml:mrow><mml:msubsup><mml:mi mathvariant="script">S</mml:mi><mml:mn>1</mml:mn><mml:mi>N</mml:mi></mml:msubsup></mml:mrow></mml:math></inline-formula>) includes 50,935 sequences for model training; the second group (<inline-formula><mml:math id="mm104" overflow="scroll"><mml:mrow><mml:msubsup><mml:mi mathvariant="script">S</mml:mi><mml:mn>2</mml:mn><mml:mi>N</mml:mi></mml:msubsup></mml:mrow></mml:math></inline-formula>) includes 152,805 sequences for model testing.<li> Training of the <b>Nextstrain</b> Clade Typing&#xA0;Models.<li> In addition to the optimized parameter <inline-formula><mml:math id="mm118" overflow="scroll"><mml:mrow><mml:msub><mml:mi>N</mml:mi><mml:mi>l</mml:mi></mml:msub></mml:mrow></mml:math></inline-formula> for the template matching algorithm, the difference matrix of the <b>Nextstrain</b> clades as shown in <xref rid="microorganisms-10-01785-f004" ref-type="fig"><u>Figure 4</u></xref> was also obtained.<li> Testing of the <b>Nextstrain</b> Clade Typing&#xA0;Models.<li> Compared with <b>Nextstrain</b>&#x2019;s difference matrix (<xref rid="microorganisms-10-01785-f004" ref-type="fig"><u>Figure 4</u></xref>), GISAID&#x2019;s <inline-formula><mml:math id="mm172" overflow="scroll"><mml:mrow><mml:mi mathvariant="script">D</mml:mi></mml:mrow></mml:math></inline-formula> shows clear discrimination.<li> Based on the good performance of the RF and Catboost classifiers in the <b>Nextstrain</b> clade typing, we applied these two methods for feature scale selection.<li> Furthermore, the accuracy of the GISAID clade typing is lower than that of <b>Nextstrain</b>, and the <inline-formula><mml:math id="mm194" overflow="scroll"><mml:mrow><mml:mover accent="true"><mml:mi>D</mml:mi><mml:mo stretchy="false">&#xAF;</mml:mo></mml:mover></mml:mrow></mml:math></inline-formula> of misclassified samples of the GISAID clade typing is larger.<li> Different from <b>Nextstrain</b>, the number of features used by the GISAID classification models is <inline-formula><mml:math id="mm196" overflow="scroll"><mml:mrow><mml:msub><mml:mi>l</mml:mi><mml:mn>5</mml:mn></mml:msub></mml:mrow></mml:math></inline-formula>.<li>Unlike the <b>Nextstrain</b> and GISAID typing issues, the number of lineages defined by Pangolin is significantly increased [<xref rid="B24-microorganisms-10-01785" ref-type="bibr"><u>24</u></xref>], and TM is no longer suitable for Pango lineage typing.<li> Comparing it with the feature distributions of <b>Nextstrain</b> and GISAID, Pangolin (green) has the widest distribution of effective features, with only 53 of the 4544 sites weighting 0.<li> Taking <b>Nextstrain</b> as an example, we obtained 561 sequences with high coverage of two new clades (22A (Omicron) and 22B (Omicron)) from the GISAID database.<li> According to Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 12 July 2022)), 22A (Omicron) and 22B (Omicron) evolved from 21L (Omicron).<li>Facing the SARS-CoV-2 genome sequence typing problem, this study built classifiers for three typing strategies of GISAID, <b>Nextstrain</b>, and Pangolin.<li> In addition to the machine learning-based methods, this study has proposed a method based on template matching for GISAID and <b>Nextstrain</b>.<li> Overall, our study achieved the highest accuracy on <b>Nextstrain</b> clade typing (precision: 99.879%, recall: 99.879%, F-score: 99.879%), followed by the Pangolin (precision: 97.889%, recall: 97.732%, F-score: 97.766%) and the GISAID (precision: 96.433%, recall: 96.291%, F-score: 96.235%).<li>(1) <b>Nextstrain</b>: Our study has studied the classification of 25 <b>Nextstrain</b> clades, using seven machine learning-based methods and a template matching-based method.<li> Compared with the <b>Nextstrain</b> clade typing, TM performs worse in the GISAID clade classification, and the F-score is lower than 85%.<li> It indicates that the separability between GISAID clades is lower than that of <b>Nextstrain</b> clades.<li> Overall, the system aims to achieve rapid and accurate SARS-CoV-2 genome sequence typing for the three typing strategies of <b>Nextstrain</b>, GISAID, and Pangolin, respectively.<li> For <b>Nextstrain</b> and GISAID, this study has proposed a method based on template matching.<li> The template matching method achieved satisfactory results in the <b>Nextstrain</b> clade typing.<li> Finally, verified by a large number of testing datasets, the ensemble model proposed in this study helps to improve the accuracy of the classification system (<b>Nextstrain</b>: 99.879%, Pangolin: 97.732%, GISAID: 96.291%).<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32835191">32835191</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32108862">32108862</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34676127">34676127</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [27](          <person-group person-group-type="author"><name><surname>Song</surname><given-names>Y.</given-names></name><name><surname>Xu</surname><given-names>W.</given-names></name></person-group>          <article-title>Genomic analysis platforms and typing methods for SARS-CoV-2 genome sequences</article-title>          <source>Chin. J. Virol.</source>          <year>2021</year>          <volume>37</volume>          <fpage>181</fpage>          <lpage>190</lpage>        ) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36052992">36052992</a> <b>(2)</b> Vaccination decreases the risk of influenza A virus reassortment but not genetic variation in pigs<ul><li> Due to the continued evolution, the H1N1pdm09 virus has generated a complex hemagglutinin (HA) clade system (<ext-link xlink:href="https://<b>nextstrain</b>.org/flu/seasonal/h1n1pdm/ha/12y" ext-link-type="uri">https://<b>nextstrain</b>.org/flu/seasonal/h1n1pdm/ha/12y</ext-link>; <xref rid="bib31" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36118859">36118859</a> <b>(3)</b> A high scale SARS-CoV-2 profiling by its whole-genome sequencing using Oxford Nanopore Technology in Kazakhstan<ul><li> Figures were drawn and edited by <b>Nextstrain</b>, BioRender and OriginPro tools.<li> Lineages were assigned to the sequenced genomes according to the <b>Nextstrain</b> SARS-CoV-2 clades.<li> <bold>(A)</bold> A phylogenetic tree is generated by <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35863699">35863699</a> <b>(4)</b> Clinical Performance Characteristics of the Swift Normalase Amplicon Panel for Sensitive Recovery of Severe Acute Respiratory Syndrome Coronavirus 2 Genomes<ul><li>For detection of within-sample variation, two previously sequenced culture isolates from different clades were used: WA-UW-20236 TM, EPI_ISL_4926371, <b>Nextstrain</b> clade 20A, and WA-UW-19433 TM, EPI_ISL_4926374, 20B.<li> Consensus sequences are analyzed using Pangolin (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="intref0050"><italic>https://pangolin.cog-uk.io</italic></ext-link>, last accessed July 5, 2022) to obtain PANGO lineage designations and Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="intref0055"><italic>https://clades.<b>nextstrain</b>.org</italic></ext-link>, last accessed July 5, 2022) to generate <b>Nextstrain</b> clade assignments and a list of mutations.<li> For SARSEQ orders, the Phylogenetic Assignment of Named Global Outbreak (PANGO) lineage, <b>Nextstrain</b> clade, and list of amino acid changes in spike are returned along with a free-text interpretation of these results.<li>RNA was extracted from two SARS-CoV-2 cultured viral isolates from distinct clades (WA-UW-20236 TM, <b>Nextstrain</b> clade 20A, and WA-UW-19433 TM, 20B) with multiple distinguishing mutations (see <xref rid="sec1" ref-type="sec"><u>Materials and Methods</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35779308">35779308</a> <b>(2)</b> Mutational dynamics across VOCs in International travellers and Community transmission underscores importance of Spike-ACE2 interaction<sup><xref ref-type="fn" rid="d35e188">&#x2606;</xref></sup><ul><li>The <b>Nextstrain</b> augur pipeline (https://github.com/<b>nextstrain</b>/ncov) was used to perform the phylogenetic analysis of 1567 sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36048827">36048827</a> <b>(2)</b> A Putative long-range RNA-RNA interaction between <italic toggle="yes">ORF8</italic>&#xA0;and <italic toggle="yes">Spike</italic> of SARS-CoV-2<ul><li> For population-based sequence-covariance analyses, a set of 2,348,494 aligned full-length SARS-CoV-2 genome sequences were taken from the <b>Nextstrain</b> project [<xref rid="pone.0260331.ref043" ref-type="bibr"><u>43</u></xref>] on December 9, 2021.<li> The sequences were originally from the Global Initiative on Sharing All Influenza Data (GISAID) platform [<xref rid="pone.0260331.ref044" ref-type="bibr"><u>44</u></xref>&#x2013;<xref rid="pone.0260331.ref046" ref-type="bibr"><u>46</u></xref>] (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>) and were subsequently filtered for high quality sequence (<b>nextstrain</b>.org, filename: filtered.fasta.xz).<li> [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34934514">34934514</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36118194">36118194</a> <b>(5)</b> Spatiotemporal dynamics of HIV-1 CRF63_02A6 sub-epidemic<ul><li>To estimate ancestral state reconstruction and identify time to the most recent common ancestor (tMRCA) of the CRF63_02A6, a local instance of <b>Nextstrain</b> (<xref rid="ref19" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) was used.<li> We also estimated tMRCA and investigated geographic distribution of the CRF63_02A6 using BEAST v1.10.4 and compared it with the results obtained by the <b>Nextstrain</b>.<li>Phylogenetic and evolutionary analysis of CRF63_02A6 872 <italic>pol</italic> gene sequences using <b>Nextstrain</b> identified that CRF63_02A6 originated in Novosibirsk around February 2005 (confidence interval: December, 2004 &#x2013; July, 2005; <xref rid="fig3" ref-type="fig"><u>Figure 3</u></xref>).<li><b>Nextstrain</b> maximum likelihood analysis of HIV-1 CRF63_02A6 872 <italic>pol</italic> gene sequences.<li> Both results from the <b>Nextstrain</b> and Bayesian inference agreed and placed Novosibirsk as the most probable ancestral location of the HIV-1 CRF63_02A6 sub-epidemic.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36111122">36111122</a> <b>(3)</b> Implemented occupational health surveillance limits the spread of SARS-CoV-2 Omicron at the workplace<ul><li> SARS-CoV-2 consensus genomes were aligned and assigned to global outbreak lineages with Pangolin v.3.1.17 (<xref rid="B12" ref-type="bibr"><u>12</u></xref>), and to <b>Nextstrain</b> clades with NextClade v.1.5.1 (<xref rid="B13" ref-type="bibr"><u>13</u></xref>).<li>Our analyses revealed that only four samples were related to Delta (<b>Nextstrain</b> clade 21J) (<xref rid="F2" ref-type="fig"><u>Figure 2A</u></xref>).<li> The other 91 samples were related to the emerging Omicron variant (<b>Nextstrain</b> clade 21K), all of which were assigned to pangolin lineage BA.1 (aka.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [13](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J Open Source Softw.</italic></source> (<year>2021</year>) <volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36055471">36055471</a> <b>(5)</b> Genomic and epidemiologic surveillance of SARS-CoV-2 in Southern Brazil and identification of a new Omicron-L452R sublineage<ul><li> Phylogenetic tree was constructed including all SARS-CoV-2 complete genomes available from GISAID through the Nextclade tool on <b>Nextstrain</b> server (<xref rid="bib0006" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>).<li> Regarding genetic characterization, the sequences were aligned with complete SARS-CoV-2 genomes of different lineages through the NextClade online tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0006">https://clades.<b>nextstrain</b>.org/</ext-link>) and the phylogenetic tree was inferred (<xref rid="fig0001" ref-type="fig"><u>Fig.&#xA0;1</u></xref>a).<li><bold>Phylogenetic and phylodynamic of SARS-CoV-2 variant analysis in Rio Grande do Sul State.</bold> Phylogenetic tree of all circulating lineages, developed through the <b>Nextstrain</b> server (Nextclade tool), using GISAID data.<li> The current global estimate including multiple variants of concern/interest suggests a similar rate of approximately 25.941 substitutions per year (derived from a global <b>Nextstrain</b> build, <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="interref0008">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>, accessed April 1st, 2022).<li> [Hadfield&#xA0;et&#xA0;al., 2018](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics</source>          <volume>34</volume>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36039616">36039616</a> <b>(1)</b> SARS&#x2010;CoV&#x2010;2 does not infect pigs, but this has to be verified regularly<ul><li> However, there was no difference between these two SARS&#x2010;CoV&#x2010;2 variants.<xref rid="xen12772-bib-0089" ref-type="bibr"><sup>89</sup></xref> When the Delta and Omicron variants were also compared in this model, infection with the Delta variant resulted in quicker weight loss and more rapid distribution of SARS&#x2010;CoV&#x2010;2 protein in the lungs, resulting in increased pathogenicity.<xref rid="xen12772-bib-0090" ref-type="bibr"><sup>90</sup></xref> In ferret models, it was found that both the prototype Wuhan strain and the D614G variant can infect ferrets, but the D614G variant was dominant in terms of progeny virus produced from these animals during dual infection.<xref rid="xen12772-bib-0091" ref-type="bibr"><sup>91</sup></xref> Non&#x2010;human primates have also been infected with Beta, Gamma, and Delta variants to assess vaccine efficacy against different variants.<xref rid="xen12772-bib-0092" ref-type="bibr"><sup>92</sup></xref> Established nomenclature systems for naming and tracking SARS&#x2010;CoV&#x2010;2 genetic lineages by GISAID, <b>Nextstrain</b> and Pango are currently and will remain in use by scientists and in scientific research.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36038845">36038845</a> <b>(3)</b> Increased transmissibility of SARS-CoV-2 alpha variant (B.1.1.7) in children: three large primary school outbreaks revealed by whole genome sequencing in the Netherlands<ul><li> The consensus sequences were used to construct a phylogenetic tree with ncov pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) v3 of <b>nextstrain</b> with all B.1.1.7 and Dutch genomes in the Global Initiative on Sharing All Influenza Data (15-apr-2021) as a reference.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36145416">36145416</a> <b>(6)</b> Strain Variation Based on Spike Glycoprotein Gene of SARS-CoV-2 in Kuwait from 2020 to 2021<ul><li>To define the circulating clades of SARS-CoV-2 in Kuwait, we applied the <b>Nextstrain</b> nomenclature that includes the original clade 19A and its derivation 19B and the emerging clade 20B and its derivation 20I.<li> The phylogenetic analytical tree of 35 SARS-CoV-2 isolates from Kuwait and the global strains based on the S glycoprotein gene was performed by Nextclade <sup>beta</sup> of <b>Nextstrain</b> platform, which is based on banded Smith&#x2013;Waterman alignment with an affine gap-penalty.<li> Clade assignment, mutation calling, and sequence quality checks were performed using the <b>Nextstrain</b> platform (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 21 August 2022) and CoVsurver mutations App-enabled by GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>, accessed on 21 August 2022).<li> The generated consensus file was used to align the sequence of KW-35A with the reference strain (NC_045512) using MEGA software version 7, while sequence variations analysis and mutation detection were performed using the <b>Nextstrain</b> platform and CoVsurver mutations App-enabled by GISAID.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36060621">36060621</a> <b>(8)</b> Safety, Efficacy, and Relapse of Nirmatrelvir-Ritonavir in Kidney Transplant Recipients Infected With SARS-CoV-2<ul><li> Several sublineages were identified: 8 patients with a BA.2 (<b>Nextstrain</b> clade 21L), 1 patient with a BA.2.3 (<b>Nextstrain</b> clade 21L), 2 patients with a BA.2.9 (<b>Nextstrain</b> clade 21L), 1 patient with a BA.2.36 (<b>Nextstrain</b> clade 21L) and 1 patient with a BA.2.12.1 (<b>Nextstrain</b> clade 22C) (<xref rid="tbl2" ref-type="table"><u>Table&#xA0;2</u></xref>).<li>Baseline and relapse <b>nextstrain</b> clade, pangolin sublineage assignment and GISAID submission references.<li>For patient 1, although the viral strains sequenced on D0 and at relapse are Omicron variants with the same sublineage assignment BA.2.3 (or <b>Nextstrain</b> clade: 21L) (<xref rid="tbl2" ref-type="table"><u>Table&#xA0;2</u></xref>), the strain present at the time of relapse (D22) shows additional variations: one amino acid substitution in the Spike protein compared with the starting strain S:R408S as well as the deletions ORF1a:G82-, ORF1a:G83-, ORF1a:G84-, ORF1a:G85-, ORF1a:G86-, and S:Y144-.<li> For patient 4, the viral strains sequenced on D0 and at relapse (D11) are Omicron variants with the same sublineage assignment BA.2.9 (or <b>Nextstrain</b> clade: 21L) (<xref rid="tbl2" ref-type="table"><u>Table&#xA0;2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36092929">36092929</a> <b>(1)</b> Editorial: SARS-CoV-2: From Genetic Variability to Vaccine Design<ul><li> In fact, tools like <b>Nextstrain</b> have been used to visualize the virus evolution as well.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36090111">36090111</a> <b>(2)</b> Molecular epidemiology and genetic characterization of SARS-CoV-2 in Kuwait: A descriptive study<ul><li> Clade nomenclature was attained from <b>Nextstrain</b> (<xref rid="ref14" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> According to the <b>Nextstrain</b> classification, 98 (46.8%) of viral genomes out of the 209 genomes were clustered as clade 20A, while the rest of the genomes sequences were classified in the clades 20I (Alpha, V1; <italic>n</italic>&#x2009;=&#x2009;50, 24%); 21I (Delta; <italic>n</italic>&#x2009;=&#x2009;38, 18.2%); 19A (<italic>n</italic>&#x2009;=&#x2009;8, 3.8%); 21A (Delta; <italic>n</italic>&#x2009;=&#x2009;7, 3.3%); 21D (Eta; <italic>n</italic>&#x2009;=&#x2009;6, 3%); and 21&#x2009;J (Delta; <italic>n</italic>&#x2009;=&#x2009;3, 1.4%; <xref rid="fig1" ref-type="fig"><u>Figure 1</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36091553">36091553</a> <b>(4)</b> A bibliometric analysis of the 100 most cited articles describing SARS-CoV-2 variants<ul><li> The current VOCs and VOIs were summarized in <xref rid="T7" ref-type="table"><u>Table 7</u></xref> based on their Pango lineage, GISAID clade (<ext-link xlink:href="http://www.gisaid.org/" ext-link-type="uri">www.gisaid.org</ext-link>), and <b>Nextstrain</b> clade (<ext-link xlink:href="http://<b>nextstrain</b>.org/ncov" ext-link-type="uri">http://<b>nextstrain</b>.org/ncov</ext-link>; <ext-link xlink:href="https://outbreak.info/" ext-link-type="uri">https://outbreak.info/</ext-link>) information.<li><bold><b>Nextstrain</b> clade</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>(Genetic) features</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>Earliest documented samples</bold></th>            </tr>          </thead>          <tbody>            <tr>              <td valign="top" align="left" rowspan="1" colspan="1">Variants of concern (VOCs).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36017801">36017801</a> <b>(1)</b> Delineating the Spread and Prevalence of SARS-CoV-2 Omicron Sublineages (BA.1&#x2013;BA.5) and Deltacron Using Wastewater in the Western Cape, South Africa<sup><xref rid="jiac356-FM1" ref-type="author-notes">&#xA0;</xref></sup><ul><li> On 4 April 2022, a new Omicron sublineage was announced in South Africa, and the <b>Nextstrain</b> database defined it as Omicron (BA.4 and BA.5) harboring the following spike protein amino acid substitutions and insertion: BA.4 and BA.5 (T19I, L24S, ins25PPA, A27, 69/70 DEL, G142D, V143/145DEL, V213I, G339D, S371L, S373P, S375F, T376A, D405N, R408S, K417N, N440K, L452R, S477N, T478K, E484A, Q498R, N501Y, Y505H, D614G, H655Y, N679K, P681H, N764K, D796Y, Q954H, N969K) (<ext-link xlink:href="https://www.nicd.ac.za/wp-content/uploads/2022/04/Update-of-SA-sequencing-data-from-GISAID-8-Apr-2022.pdf" ext-link-type="uri">https://www.nicd.ac.za/wp-content/uploads/2022/04/Update-of-SA-sequencing-data-from-GISAID-8-Apr-2022.pdf</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36029839">36029839</a> <b>(1)</b> Molecular evidence for SARS-CoV-2 in samples collected from patients with morbilliform eruptions since late 2019 in Lombardy, northern Italy<ul><li> This strain is now classified as 20&#xA0;A in <b>NextStrain</b> (<xref rid="bib34" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and B.1 in Pangolin (<xref rid="bib60" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), has an &#x3B1;&#x3B2; mutational signature (<xref rid="bib38" ref-type="bibr"><u>Kumar et al., 2021</u></xref>), and is also referred to as DG1111 haplotype (<xref rid="bib62" ref-type="bibr"><u>Ruan et al., 2021</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Kumar et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33942847">33942847</a>) [Ruan et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Ruan</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>Wen</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Hou</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>He</surname>              <given-names>Z.</given-names>            </name>            <name>              <surname>Lu</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Xue</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>He</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Zhang</surname>              <given-names>Y.P.</given-names>            </name>            <name>              <surname>Wu</surname>              <given-names>C.I.</given-names>            </name>          </person-group>          <article-title>The twin-beginnings of COVID-19 in Asia and Europe-one prevails quickly</article-title>          <source>Natl. Sci. Rev.</source>          <volume>9</volume>          <year>2021</year>          <pub-id pub-id-type="doi">10.1093/nsr/nwab223</pub-id>          <comment>nwab223</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36008511">36008511</a> <b>(16)</b> SARS-CoV-2 genome variations and evolution patterns in Egypt: a multi-center study<ul><li> <b>Nextstrain</b>, Global Initiative on Sharing Avian Influenza Data (GISAID) and Phylogenetic Assignment of Named Global Outbreak Lineages (PANGO).<li> Downstream analysis were performed in R<sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup> in which heatmap computed using ComplexHeatmap package<sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>, phylogenetic tree visualized using ggtree<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup> and Nextclade online tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup>.<li>To determine phylogenetic characteristics of our SARS-CoV-2 isolates, we performed maximum likelihood tree based on aligned full length sequence of our genomic sequences using IQ-tree and Nextclade online tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) (Fig.&#xA0;<xref rid="Fig1" ref-type="fig"><u>1</u></xref>).<li> <b>Nextstrain</b> clade, PANGO lineage, date of sampling, hospitalization and clinical severity are color indicated on the phylogenetic tree in Fig.&#xA0;<xref rid="Fig1" ref-type="fig"><u>1</u></xref>a.<li> <b>Nextstrain</b> classifies strains as follows; 19A is considered the parent strain with no mutations compared to the Wuhan-1 reference strain, <italic>C8782T</italic> and <italic>T28144C</italic> (ORF8 L84S) define clade 19B, and <italic>C3037T</italic>, <italic>C14408T</italic> (ORF1ab P4715L) and <italic>A23403G</italic> (S D614G) define clade 20A.<li> Most of our samples belong to <b>Nextstrain</b> clades 20A and 20D, 34.5% and 40% respectively (Fig.&#xA0;<xref rid="Fig1" ref-type="fig"><u>1</u></xref>a, b) and are equally distributed across our sampling dates from May 2020 to January 2021.<li> For each sample, date of sample collection, clinical severity, hospitalization, <b>nextstrain</b> clade and PANGO lineage are indicated by the circular color strip around the tree according to the legend.<li> (<bold>b</bold>) 110 sequenced samples included in this study were placed on Nextclade <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link><sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup>.<li> <b>Nextstrain</b> clades are broken down according to the indicated color codes.<li> Samples belonging to <b>Nextstrain</b> clade 19B also belonged to PANGO lineage A.28 and showed all its characteristic mutations (i) in ORF1ab; V86F (leader protein), A3529T (nsp5), A4977V (RdRp), A5376V and D5584Y (helicase), (ii) in spike; IHV68I, N501T and H655Y, (iii) in ORF3a; S171L, (iv) in ORF8; L84S, and (v) in nucleocapsid; A35V and S202N.<li> These samples also showed mutations not related to their PANGO lineage or <b>Nextstrain</b> clade; ORF1ab P2110S (nsp3) and A4977V (RdRp).<li> Samples belonging to <b>Nextstrain</b> clade 20A and its derivatives 20B and 20D, showed clade characteristic mutations S D614G and ORF1ab P4715L.<li> ORF3a Q57H was prevalent in samples belonging to <b>Nextstrain</b> clade 20A (PANGO lineage B.1, B.1.170 and B.1.466) although not a defining mutation for this clade.<li> Samples belonging to <b>Nextstrain</b> clade 20B and its derivative clade 20D showed its characteristic mutations; nucleocapsid R203K and G204R.<li> Samples belonging to <b>Nextstrain</b> clade 20D showed two additional clade characteristic mutations; ORF1ab T1246I (nsp3) and and G3278S (nsp5) and those of which belonged to PANGO lineage C.36 also showed nucleocapsid G212V.<li> The third group of mutations (Fig.&#xA0;<xref rid="Fig4" ref-type="fig"><u>4</u></xref>c) contained seven mutations that were defining mutations for PANGO lineage C.36 and <b>Nextstrain</b> clade 20D, and prevalent to a lesser extent in PANGO lineage B.1 and <b>Nextstrain</b> 20A.<li> P681H mutation, which is near the furin cleavage site, was found in eight samples in our study belonging to <b>Nextstrain</b> clade 20D and PANGO lineage C.36, and is considered one of the clade defining mutations for the highly contagious delta variant&#x2014;<b>Nextstrain</b> clade 20I.<li> [20](R Core Team. <italic>R: A Language and Environment for Statistical Computing</italic>. R Foundation for Statistical Computing, Vienna, Austria (2020).) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27207943">27207943</a>) [28](          <person-group person-group-type="author">            <name>              <surname>Yu</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Using ggtree to visualize data on tree-like structures</article-title>          <source>Curr. Protoc. Bioinform.</source>          <year>2020</year>          <volume>69</volume>          <fpage>e96</fpage>          <pub-id pub-id-type="doi">10.1002/cpbi.96</pub-id>        ) [26](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [26](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36146685">36146685</a> <b>(4)</b> SARS-CoV-2 Genetic Diversity and Lineage Dynamics in Egypt during the First 18 Months of the Pandemic<ul><li> Lineages and clades were assigned using the dynamic lineage classification tool Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin) [<xref rid="B9-viruses-14-01878" ref-type="bibr"><u>9</u></xref>] and <b>NextStrain</b> [<xref rid="B10-viruses-14-01878" ref-type="bibr"><u>10</u></xref>].<li> Global reference sequences were chosen based on their previous inclusion in the global and African phylogenetic builds of the <b>NextStrain</b> platform (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri>; date of access: 9 August 2021).<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36140162">36140162</a> <b>(3)</b> Clinical Impact of Monoclonal Antibodies in the Treatment of High-Risk Patients with SARS-CoV-2 Breakthrough Infections: The ORCHESTRA Prospective Cohort Study<ul><li> Consensus viral sequences were extracted, clades and lineages assigned using <b>NextStrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/nextclade/sars-cov-2?branchLabel=aa">https://<b>nextstrain</b>.org/nextclade/sars-cov-2?branchLabel=aa</uri> (accessed on 3 August 2022)) and Phylogenic Assignment of named Global Outbreak LINeages (PANGOLIN) [<xref rid="B16-biomedicines-10-02063" ref-type="bibr"><u>16</u></xref>].<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34095513">34095513</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36002181">36002181</a> <b>(2)</b> Comparison of prognostic scores for inpatients with COVID-19: a retrospective monocentric cohort study<ul><li> During the study period, infections in Switzerland were caused by COVID-19 claves 19A, 19B, 20A, 20B and 20A (EU1) (<ext-link xlink:href="https://<b>nextstrain</b>.org/groups/swiss/ncov/switzerland" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/swiss/ncov/switzerland</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36140458">36140458</a> <b>(1)</b> SARS-CoV-2 RT-PCR to Screen for B.1.617.2 (Delta) Variant of Concern<ul><li> Notably, the rapid sharing of SARS-CoV-2 genomic sequences and related clinical and epidemiological data to The GISAID Initiative has made it possible to track current strains in real-time [<xref rid="B8-diagnostics-12-02056" ref-type="bibr"><u>8</u></xref>], and other online databases, such as <italic toggle="yes">outbreak.info</italic>, <italic toggle="yes"><b>nextstrain</b>.org, and cov-lineages.org</italic>, have combined genetic SARS-CoV-2 data from several data sources (including The GISAID Initiative) to provide essential analysis of sequence variability and geo-temporal analysis [<xref rid="B9-diagnostics-12-02056" ref-type="bibr"><u>9</u></xref>,<xref rid="B10-diagnostics-12-02056" ref-type="bibr"><u>10</u></xref>].<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35637329">35637329</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36000731">36000731</a> <b>(1)</b> Multiple Introductions of SARS-CoV-2 Alpha and Delta Variants into White-Tailed Deer in Pennsylvania<ul><li> Mutation C7303U was found in 86% of these PA deer and 29% of previously published deer, but in only 0.04% of humans in Delaware Valley and 0.09% of global SARS-CoV-2 genomes reported by <b>NextStrain</b> (data from 1/26/2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36123226">36123226</a> <b>(2)</b> A Preliminary Genomic Analysis of the Omicron Variants of SARS-CoV-2 in Central India During the third wave of the COVID-19 Pandemic<ul><li> The sequences were also submitted to <b>Nextstrain</b>: real-time tracking of pathogen evolution, to check the name of the clade and lineage.<li> This revealed that 29 samples belonged to the clade 21L Omicron group and the same samples belonged to the pangolin BA.2 lineage, while the remaining two samples belonged to clade 21K in <b>Nextstrain</b> and the pangolin BA.1 lineage.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35997285">35997285</a> <b>(2)</b> Genomic Epidemiology and Serology Associated with a SARS-CoV-2 R.1 Variant Outbreak in New Jersey<ul><li> Using established nomenclature, all 11 strains were classified as R.1 lineage, sublineage of B.1.1.316, and belonging to GR clade (GISAID) and 20B clade (<b>Nextstrain</b>).<li>The resulting SARS-CoV-2 viral genome sequences were characterized by Nextclade CLI (v1) (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) to assign <b>Nextstrain</b> clades (<xref rid="B24" ref-type="bibr"><u>24</u></xref>).<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35998195">35998195</a> <b>(3)</b> Urgent need for a non-discriminatory and non-stigmatizing nomenclature for monkeypox virus<ul><li> Designated lineages can be visualized on <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/monkeypox/hmpxv1" ext-link-type="uri">https://<b>nextstrain</b>.org/monkeypox/hmpxv1</ext-link>) and novel sequences can be assigned using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35677069">35677069</a> <b>(0)</b> Imprinted antibody responses against SARS-CoV-2 Omicron sublineages<ul><li> Sequence quality was determined using Nextclade version 1.0.0-alpha.8 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35989365">35989365</a> <b>(5)</b> A comprehensive analysis of the mutational landscape of the newly emerging Omicron (B.1.1.529) variant and comparison of mutations with VOCs and VOIs<ul><li> We used the COVID-19 CG server [<xref ref-type="bibr" rid="CR48"><u>48</u></xref>], <b>Nextstrain</b> server [<xref ref-type="bibr" rid="CR49"><u>49</u></xref>], <b>Nextstrain</b> (2021)[<xref ref-type="bibr" rid="CR50"><u>50</u></xref>], and Outbreak.info [<xref ref-type="bibr" rid="CR51"><u>51</u></xref>] for further analysis.<li> The <b>Nextstrain</b> server was used to develop a scatter plot for VOCs and VOIs.<li> The scatter plot was developed for Omicron and all VOCs and VOIs using the <b>Nextstrain</b> server [<xref ref-type="bibr" rid="CR49"><u>49</u></xref>, <xref ref-type="bibr" rid="CR50"><u>50</u></xref>].<li> The scatter plot was developed for VOCs and VOIs using the <b>Nextstrain</b> server [<xref ref-type="bibr" rid="CR49"><u>49</u></xref>, <xref ref-type="bibr" rid="CR50"><u>50</u></xref>].<li> [48](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33620031">33620031</a>) [49](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972767">33972767</a>) [50](Nextstrain. Nextstrain SARS-CoV-2 resources, on Nextstrain<italic>.</italic> 2021. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link>. Acessed on 5 Jan 2022.) [51](Karthik Gangavarapu et al. Outbreak.info genomic reports: scalable and dynamic surveillance of SARS-CoV-2 variants and mutations. 2022. 10.1101/2022.01.27.22269965.) [49](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972767">33972767</a>) [50](Nextstrain. Nextstrain SARS-CoV-2 resources, on Nextstrain<italic>.</italic> 2021. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link>. Acessed on 5 Jan 2022.) [49](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972767">33972767</a>) [50](Nextstrain. Nextstrain SARS-CoV-2 resources, on Nextstrain<italic>.</italic> 2021. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link>. Acessed on 5 Jan 2022.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35995785">35995785</a> <b>(0)</b> Antiviral and clinical activity of bamlanivimab in a randomized trial of non-hospitalized adults with COVID-19<ul><li> SARS-COV-2 variant calling was done using 3 different variant calling platforms, namely, CoVDB<sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup>, Scorpio call v1.2.123 (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>)<sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>, and Nextclade v.1.13.2 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup>.<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35263335">35263335</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [28](Aksamentov, I., Roemer, C., Hodcroft, E. &amp; Neher, R. Nextclade: clade assignment, mutation calling and quality control for viral genomes. <italic>J. Open Source Softw.</italic><bold>6</bold>, 3773 (2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35989600">35989600</a> <b>(1)</b> A scalable analytical approach from bacterial genomes to epidemiology<ul><li> The same methods based on the structured coalescent can also be applied to a dated phylogeny in order to reconstruct past geographical migrations [<xref rid="RSTB20210246C9" ref-type="bibr"><u>9</u></xref>], although such phylogeographic inference is much more often based on discrete trait analysis, for example using the ace command from the R package ape [<xref rid="RSTB20210246C68" ref-type="bibr"><u>68</u></xref>] or in the <b>NextStrain</b> platform [<xref rid="RSTB20210246C69" ref-type="bibr"><u>69</u></xref>].<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26267488">26267488</a>) [68](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30016406">30016406</a>) [69](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35989603">35989603</a> <b>(1)</b> Scalable Bayesian phylogenetics<ul><li> This has motivated the development of analytic and visualization tools such as <b>Nextstrain</b> for real-time tracking of pathogen evolution and spread [<xref rid="RSTB20210242C5" ref-type="bibr"><u>5</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35995311">35995311</a> <b>(1)</b> A distinct phylogenetic cluster of Monkeypox genomes suggests an early and cryptic spread of the virus<ul><li>(A) Phylogenetic tree of genome isolates from GISAID belonging to the hMPXV-1 clade of monkeypox virus constructed using <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36014093">36014093</a> <b>(2)</b> Uninvited Guest: Arrival and Dissemination of Omicron Lineage SARS-CoV-2 in St. Petersburg, Russia<ul><li>On the general tree of <b>Nextstrain</b> strains, all identified strains were included in the 20K cluster (Omicron), with the formation of subclusters corresponding to sublineages within BA.1 (<xref rid="microorganisms-10-01676-f004" ref-type="fig"><u>Figure 4</u></xref>).<li> On the general tree of <b>Nextstrain</b> strains, all identified strains were included in cluster 20L (Omicron) (<xref rid="microorganisms-10-01676-f002" ref-type="fig"><u>Figure 2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36006289">36006289</a> <b>(3)</b> Transmission Dynamics and Genomic Epidemiology of Emerging Variants of SARS-CoV-2 in Bangladesh<ul><li> We used a web-based program, &#x201C;<b>Nextstrain</b>&#x201D; (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, accessed on 25 March 2022), to generate the clade-designated tree.<li> Each branch tip bearing a Bangladeshi genome found in accessible <b>Nextstrain</b> clades was recognized in the original phylogeny.<li> We used the automated global map-based analysis in auspice version 0.5.0 (<uri xlink:href="https://auspice.us">https://auspice.us</uri>, accessed on 25 March 2022) by <b>Nextstrain</b> [<xref rid="B36-tropicalmed-07-00197" ref-type="bibr"><u>36</u></xref>] to visualize the transmission paths.<li> [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35984845">35984845</a> <b>(1)</b> TreeKnit: Inferring ancestral reassortment graphs of influenza viruses<ul><li> In the future, we plan to extend TreeKnit to multiple and complement it with ARG visualization tools in <b>Nextstrain</b> [<xref rid="pcbi.1010394.ref028" ref-type="bibr"><u>28</u></xref>].<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36111270">36111270</a> <b>(4)</b> Containing novel SARS-CoV-2 variants at source is possible with high-intensity sequencing<ul><li>Data on the variant membership metadata for United States and England for SARS-CoV-2 sequences were downloaded from <ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri"><b>nextstrain</b>.org</ext-link> on 2022 February 4 (<xref rid="bib44" ref-type="bibr"><u>44</u></xref>).<li> Variant membership data for sequences from England were also downloaded from <ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri"><b>nextstrain</b>.org</ext-link>, subsampled from the open dataset sourced from GenBank (<xref rid="bib45" ref-type="bibr"><u>45</u></xref>).<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [45](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25414350">25414350</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36090771">36090771</a> <b>(1)</b> Lineage BA.2 dominated the Omicron SARS-CoV-2 epidemic wave in the Philippines<ul><li> For each data set, 1,500 global proximal strains genetically similar to the Philippine strains were sampled by the <b>Nextstrain</b> bioinformatic pipeline (<xref rid="R7" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35996440">35996440</a> <b>(1)</b> The clinical and genomic epidemiology of seasonal human coronaviruses in congregate homeless shelter settings: A repeated cross-sectional study<ul><li> We used SAS software version 9&#xB7;4 (Cary, NC, USA) for general analysis and <b>NextStrain</b> software for genome consensus processing and visualization of phylogenetic trees.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35982313">35982313</a> <b>(1)</b> Genomic epidemiology of the SARS-CoV-2 epidemic in Brazil<ul><li> A unique set of external references (<italic>n</italic>&#x2009;=&#x2009;7,992) was obtained by including all the non-Brazilian and non-Paraguayan sequences from the global and South American <b>NextStrain</b> builds (date of access, 30 June 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35982065">35982065</a> <b>(2)</b> Considering epitopes conservity in targeting SARS-CoV-2 mutations in variants: a novel immunoinformatics approach to vaccine design<ul><li> Our approach is similar to CoVariants and <b>Nextstrain</b> sites to depict diversities and mutations in the genome of different viruses.<li>In order to translate sequences and their mutations into Amino Acids, we used the augur translate module [augur/translate.py at master <b>nextstrain</b>/augur (github.com)].</ul></p>
<p><u>No PMID</u> <b>(2)</b> Unsupervised clustering of SARS-CoV-2 using deep convolutional autoencoder<ul><li>There are currently three nomenclature systems to classify and monitor SARS-CoV-2 genetic lineages: the Global Initiative on Sharing All Influenza Data (GISAID) [<xref ref-type="bibr" rid="CR18"><u>18</u></xref>], <b>Nextstrain</b>, and Pango [<xref ref-type="bibr" rid="CR19"><u>19</u></xref>].<li>Currently designated variants ofconcern (VOC)</p></caption><table frame="hsides" rules="groups"><thead><tr><th>WHO label</th><th>Variant type</th><th>First detection</th><th>GISAID</th><th><b>Nextstrain</b></th><th>Pango</th></tr></thead><tbody><tr><td>Alpha.<li> [18](GISAID. <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</ext-link>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35982756">35982756</a> <b>(4)</b> Temporal lineage replacements and dominance of imported variants of concern during the COVID-19 pandemic in Kenya<ul><li> The <b>Nextstrain</b> clade system<sup><xref ref-type="bibr" rid="CR31"><u>31</u></xref></sup> uses a year-letter nomenclature on a clade exceeding 20% global representation and &gt;2 positional differences from its parent clade while considering clade persistence with time as well as the extent of its geographical spread.<li> Due to the huge number of genomes present in the GISAID, we opted to utilize the globally sampled genomes maintained by <b>Nextstrain</b>.<li>All trees were reconstructed with augur v14 as implemented in the <b>Nextstrain</b> pipeline version 3.0.6<sup><xref ref-type="bibr" rid="CR31"><u>31</u></xref></sup>.<li> Within <b>Nextstrain</b>, a random subsampling method was used to cap the maximum number of context sequences from the rest of the world &#x2013; to provide phylogenetic context, based on genomic proximity.<li> [31](Hadfield, J. et al. NextStrain: Real-time tracking of pathogen evolution. <italic>Bioinformatics</italic>10.1093/bioinformatics/bty407 (2018).) [31](Hadfield, J. et al. NextStrain: Real-time tracking of pathogen evolution. <italic>Bioinformatics</italic>10.1093/bioinformatics/bty407 (2018).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36041271">36041271</a> <b>(1)</b> Predicting COVID-19 disease severity from SARS-CoV-2 spike protein sequence by mixed effects machine learning<ul><li> as well as metadata fields generally available for all SARS-CoV-2 sequences on GISAID, which include <italic>inter alia</italic> host, the continent/country/region of collection, Pango nomenclature lineage, <b>NextStrain</b> clade, sample collection and submission date, patient age, and patient gender.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35992368">35992368</a> <b>(5)</b> Co-existence and co-infection of influenza A viruses and coronaviruses: Public health challenges<ul><li> <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/flu/seasonal/" id="intref0030">https://<b>nextstrain</b>.org/flu/seasonal/</ext-link>).<li> Figures&#xA0;are reannotated from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/influenza/" id="intref0075">https://<b>nextstrain</b>.org/influenza/</ext-link>, adapted from and courtesy of Creative Commons Attribution Licensing).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35978025">35978025</a> <b>(1)</b> Infections with the SARS-CoV-2 Delta variant exhibit fourfold increased viral loads in the upper airways compared to Alpha or non-variants of concern<ul><li> Nevertheless, it has to be noted that the former study<sup><xref ref-type="bibr" rid="CR5"><u>5</u></xref></sup> comprised a much smaller dataset (62 samples harboring Delta variant vs 63 samples harboring clade 19A/19B isolates) which was part of an outbreak and compared the Delta variant to a mixture of samples harboring <b>nextstrain</b> clade 19A/19B variants, which did not harbor the D614G mutation that enhances infectivity<sup><xref ref-type="bibr" rid="CR13"><u>13</u></xref>&#x2013;<xref ref-type="bibr" rid="CR15"><u>15</u></xref></sup> and were much closer related to the original Wuhan strain than the non-VOC isolates in our study.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35075154">35075154</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33243994">33243994</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36032969">36032969</a> <b>(1)</b> Intrahost SARS-CoV-2 k-mer identification method (iSKIM) for rapid detection of mutations of concern reveals emergence of global mutation patterns<ul><li> An additional custom set of 3,243 background reference samples were obtained from the <b>NextStrain</b> SARS-CoV-2 global build [<xref rid="R49" ref-type="bibr"><u>49</u></xref>] and added.<li> [49](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36075217">36075217</a> <b>(0)</b> Remdesivir-induced emergence of SARS-CoV2 variants in patients with prolonged infection<ul><li> (2021).</td><td><ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0085">https://clades.<b>nextstrain</b>.org/</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35972143">35972143</a> <b>(1)</b> The New SARS-CoV-2 Variants and Their Epidemiological Impact in Mexico<ul><li> most available SARS-CoV-2 whole genomes are deposited in the EpiCoV database of the Global Initiative on Sharing All Influenza Data (GISAID) (<xref rid="B30" ref-type="bibr"><u>30</u></xref>), a standardized international collaboration including clade classification by <b>Nextstrain</b> (<xref rid="B31" ref-type="bibr"><u>31</u></xref>) and lineages with the Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin) universal nomenclature classification (<xref rid="B32" ref-type="bibr"><u>32</u></xref>).<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36030585">36030585</a> <b>(7)</b> Omicron variant (B.1.1.529) and its sublineages: What do we know so far amid the emergence of recombinant variants of SARS-CoV-2?<ul><li> 1</alt-text><graphic xlink:href="gr1_lrg"/><attrib>Source: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global?m=div" id="ir0005">https://<b>nextstrain</b>.org/ncov/gisaid/global?m=div</ext-link>.<li> 3</alt-text><graphic xlink:href="gr3_lrg"/><attrib>Source: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m?c=emerging_lineage&amp;l=scatter" id="ir0010">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m?c=emerging_lineage&amp;l=scatter</ext-link>.<li> 4</alt-text><graphic xlink:href="gr4_lrg"/><attrib>Source: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global?m=div" id="ir0015">https://<b>nextstrain</b>.org/ncov/gisaid/global?m=div</ext-link></attrib></fig>.<li> The prompt deployment of GISAID and <b>Nextstrain</b> in response to COVID-19 is one prominent example.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35962276">35962276</a> <b>(4)</b> Delta variant (B.1.617.2) of SARS-CoV-2: current understanding of infection, transmission, immune escape, and mutational landscape<ul><li> The mutations are derived from the <b>Nextstrain</b> server (<b>Nextstrain</b> <xref ref-type="bibr" rid="CR52"><u>2021</u></xref>) and the CoVariants server.<li> The figure was developed by the <b>Nextstrain</b> server (<b>Nextstrain</b> <xref ref-type="bibr" rid="CR52"><u>2021</u></xref>).<li> [2021](Nextstrain (2021) Nextstrain SARS-CoV-2 resources, on Nextstrain. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link> (Acessed on 27 Oct&#xA0;2021)) [2021](Nextstrain (2021) Nextstrain SARS-CoV-2 resources, on Nextstrain. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link> (Acessed on 27 Oct&#xA0;2021)) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35962405">35962405</a> <b>(2)</b> Improved SARS-CoV-2 sequencing surveillance allows the identification of new variants and signatures in infected patients<ul><li>The phylogenetic analysis was generated using <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR19"><u>19</u></xref>] standard pipeline on a random subsample of sequences generated until 2022-03-30.<li> To further investigate this finding, we performed a phylogenetic analysis over our entire dataset using <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR19"><u>19</u></xref>] and found that all B.1.177.x samples carrying the Spike E484K substitution (B.1.177<sup>E484K</sup> samples) clustered in a specific and monophyletic clade branching within the B.1.177.x lineage (Fig.<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35961956">35961956</a> <b>(3)</b> Protection of COVID-19 vaccination and previous infection against Omicron BA.1, BA.2 and Delta SARS-CoV-2 infections<ul><li>Since November 2021 the Omicron BA.1 SARS-CoV-2 variant (<b>Nextstrain</b>&#xA0;clade 21K, Pango lineage B.1.1.529) has rapidly spread causing high numbers of infections across Europe, due to higher transmissibility and immune escape<sup><xref ref-type="bibr" rid="CR1"><u>1</u></xref></sup>.<li> Several studies indicate that Omicron variants cause less severe disease than the Delta variant&#xA0;(<b>Nextstrain</b> clade&#xA0;21A, Pango lineage&#xA0;B.1.617.2), also when adjusted for infection- and vaccine-induced immunity<sup><xref ref-type="bibr" rid="CR3"><u>3</u></xref>&#x2013;<xref ref-type="bibr" rid="CR5"><u>5</u></xref></sup>.<li> from January 2022 a new variant from Omicron, BA.2 (<b>Nextstrain</b>&#xA0;clade 21L), started to replace Omicron BA.1.<li> [1](European Centre for Disease Prevention and Control (ECDC). Assessment of the further emergence and potential impact of the SARS-CoV-2 Omicron variant of concern in the context of ongoing transmission of the Delta variant of concern in the EU/EEA, 18th update. Available from: <ext-link ext-link-type="uri" xlink:href="https://www.ecdc.europa.eu/sites/default/files/documents/covid-19-assessment-further-emergence-omicron-18th-risk-assessment-december-2021.pdf">https://www.ecdc.europa.eu/sites/default/files/documents/covid-19-assessment-further-emergence-omicron-18th-risk-assessment-december-2021.pdf</ext-link>.) [3](Davies, M.-A. et al. Outcomes of laboratory-confirmed SARS-CoV-2 infection in the Omicron-driven fourth wave compared with previous waves in the Western Cape Province, South Africa. medRxiv. 10.1101/2022.01.12.22269148.) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35065011">35065011</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016382">36016382</a> <b>(0)</b> Development and Validation of an In-House Real-Time Reverse-Transcriptase Polymerase Chain Reaction Assay for SARS-CoV-2 Omicron Lineage Subtyping between BA.1 and BA.2<ul><li> SARS-CoV-2 genomic lineages were inferred using Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) (<uri xlink:href="https://github.com/hCoV-2019/pangolin">https://github.com/hCoV-2019/pangolin</uri>) and NextClade (<uri xlink:href="https://docs.<b>nextstrain</b>.org/projects/nextclade">https://docs.<b>nextstrain</b>.org/projects/nextclade</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35968047">35968047</a> <b>(6)</b> Genomic surveillance of SARS-CoV-2 in Puerto Rico enabled early detection and tracking of variants<ul><li> We then used the standard ncov augur/auspice multiple input workflow available in the <b>Nextstrain</b> platform<sup><xref ref-type="bibr" rid="CR35"><u>35</u></xref></sup> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) to subsample contextual genomes and phylogenetic inference with time-stamped trees.<li> The resulting global ML tree was visualized with <b>Nextstrain</b> auspice<sup><xref ref-type="bibr" rid="CR35"><u>35</u></xref></sup> and annotated with iTol for region of origin and emerging variants<sup><xref ref-type="bibr" rid="CR39"><u>39</u></xref></sup>.<li> A custom subsampling scheme was selected on the <b>Nextstrain</b> ncov workflow to select genomes from samples collected between November 1st, 2020 and February 28th, 2021.<li>The date of the most recent common ancestor (tMRCA) determined by the <b>Nextstrain</b> ML phylogenetic inference was confirmed with Bayesian coalescent analyses for B.1.588, Alpha VBM and Delta VOC lineage trees.<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17050570">17050570</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35971553">35971553</a> <b>(2)</b> Engineering SARS-CoV-2 neutralizing antibodies for increased potency and reduced viral escape pathways<ul><li> Since the discovery of SARS-CoV-2 in 2019, thousands of viral variants containing synonymous and non-synonymous mutations have been documented (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="intref0040">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35953484">35953484</a> <b>(0)</b> SARS-CoV-2 and other respiratory pathogens are detected in continuous air samples from congregate settings<ul><li> Lineage/clade assignments were also confirmed using NextClade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>, version 1.14.0) and Pangolin COVID-19 Lineage Assigner (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>, version 3.1.20) by uploading obtained FASTA files<sup><xref ref-type="bibr" rid="CR76"><u>76</u></xref>,<xref ref-type="bibr" rid="CR77"><u>77</u></xref></sup>.<li> [76](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [77](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35947257">35947257</a> <b>(1)</b> SARS-COV-2/COVID-19: scenario, epidemiology, adaptive mutations, and environmental factors<ul><li> Our ability to evaluate genetic variations quickly during an epidemic is made possible by genomic tracking technologies such as <b>Nextstrain</b> (Hadfield et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36034694">36034694</a> <b>(3)</b> How concerning is a SARS-CoV-2 variant of concern? Computational predictions and the variants labeling system<ul><li> and communicate more effectively with the public about the adequate response to the emergence of new variants of SARS-CoV-2 (<uri xlink:href="https://www.who.int/">https://www.who.int/</uri>).&#xA0;WHO has first developed a system to facilitate naming SARS-CoV-2 variants in addition to the existing nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages established by GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>), <b>Nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>), and Pango (<uri xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36033872">36033872</a> <b>(2)</b> Genomic and virologic characterization of samples from a shipboard outbreak of COVID-19 reveals distinct variants within limited temporospatial parameters<ul><li>Virus lineage/<b>nextStrain</b> clade/GISAID clade</th>              </tr>            </thead>            <tbody>              <tr>                <td align="left" valign="top" rowspan="1" colspan="1">TR1.<li> all of them belonged to Pangolin lineage B.1.1, <b>NextStrain</b> clade 20B, and GISAID clade GR (<xref rid="tab2" ref-type="table"><u>Table 2</u></xref>; <xref rid="fig2" ref-type="fig"><u>Figure 2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35963550">35963550</a> <b>(2)</b> First two cases of Monkeypox virus infection in travellers returned from UAE to India, July 2022<ul><li> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/monkeypox/blob/master/config/clades.tsv" id="interref09041">https://github.com/<b>nextstrain</b>/monkeypox/blob/master/config/clades.tsv</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35943268">35943268</a> <b>(2)</b> Impact of SARS-CoV-2 Omicron on Rapid Antigen Testing Developed for Early-Pandemic SARS-CoV-2 Variants<ul><li> However, sensitivity of RATs may be impaired for novel SCoV2 variants, such as the now globally dominating Omicron variants (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35963158">35963158</a> <b>(1)</b> Host genetic diversity and genetic variations of SARS-CoV-2 in COVID-19 pathogenesis and the effectiveness of vaccination<ul><li>In addition, according to the <b>NextStrain</b>, recent evidence of monitoring SARS-CoV-2 genome showed new massive mutations in a 56&#xA0;years old man infected with the Delta variant include 34 mutations in the spike <xref rid="b1105" ref-type="bibr"><u>[221]</u></xref>.<li> [[221]](NextStrain, (2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35952818">35952818</a> <b>(3)</b> Omicron (B.1.1.529) - A new heavily mutated variant: Mapped location and probable properties of its mutations with an emphasis on S-glycoprotein<ul><li> The COVID-19 data was also gathered from several countries and <b>Nextstrain</b> <xref rid="bb0245" ref-type="bibr"><u>[49]</u></xref>.<li> <b>Nextstrain</b> SARS-CoV-2 resources are available on <b>Nextstrain</b> <xref rid="bb0250" ref-type="bibr"><u>[50]</u></xref>.<li> [[49]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33473201">33473201</a>) [[50]](          <person-group person-group-type="author">            <name>              <surname>Nextstrain</surname>            </name>          </person-group>          <part-title>Nextstrain SARS-CoV-2 Resources on Nextstrain</part-title>          <year>2021</year>          <comment>(Accessed on 7/12/2021)</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35981541">35981541</a> <b>(0)</b> Monitoring of the SARS-CoV-2 Omicron BA.1/BA.2 lineage transition in the Swedish population reveals increased viral RNA levels in BA.2 cases<ul><li> Analysis Software</td><td>Applied Biosystems</td><td>v1.5.2</td></tr><tr><td>fastp</td><td><ext-link ext-link-type="uri" xlink:href="http://github.com/OpenGene/fastp" id="intref0015">github.com/OpenGene/fastp</ext-link></td><td>v0.20.0</td></tr><tr><td>MEGAHIT</td><td><ext-link ext-link-type="uri" xlink:href="http://github.com/voutcn/megahit" id="intref0020">github.com/voutcn/megahit</ext-link></td><td>v1.2.9</td></tr><tr><td>RagTag</td><td><ext-link ext-link-type="uri" xlink:href="http://github.com/malonge/RagTag" id="intref0025">github.com/malonge/RagTag</ext-link></td><td>v2.1.0</td></tr><tr><td>Pangolin</td><td><ext-link ext-link-type="uri" xlink:href="http://pangolin.cog-uk.io" id="intref0030">pangolin.cog-uk.io</ext-link></td><td>v3.1.20 / v4.0.6</td></tr><tr><td>Minimap2</td><td><ext-link ext-link-type="uri" xlink:href="http://github.com/lh3/minimap2" id="intref0035">github.com/lh3/minimap2</ext-link></td><td>v2.24-r1122</td></tr><tr><td>Nextclade</td><td><ext-link ext-link-type="uri" xlink:href="http://clades.<b>nextstrain</b>.org" id="intref0040">clades.<b>nextstrain</b>.org</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016360">36016360</a> <b>(3)</b> SARS-CoV-2 in a Mink Farm in Italy: Case Description, Molecular and Serological Diagnosis by Comparing Different Tests<ul><li>Phylogenetic and molecular analyses were performed using Nextclade Web 1.5.3 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 23 August 2021) (Ref, strain Wuhan/WH01/2019) [<xref rid="B24-viruses-14-01738" ref-type="bibr"><u>24</u></xref>].<li> On the reference tree, clades were assigned using lists of clade-defining mutations via the augur workflow (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv</uri>, accessed on 23 August 2021).<li>A whole-genome-based phylogenetic analysis showed that the Italian worker and mink SARS-CoV-2 sequences belonged to different clades, which corresponded to two distinct <b>Nextstrain</b> clades.<li> [24](          <article-title>Nextclade Web 1.5.3</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-08-10">(accessed on 10 August 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35906185">35906185</a> <b>(1)</b> Genomic annotation and molecular evolution of monkeypox virus outbreak in 2022<ul><li>The nucleotide mutations, including nucleotide substitutes, deletions, and insertions of each MPXV strain, were called by Nextclade (<ext-link xlink:href="https://master.clades.<b>nextstrain</b>.org" ext-link-type="uri">https://master.clades.<b>nextstrain</b>.org</ext-link>) on June 22, 2022.<li> These nucleotide mutations were sectioned referring to sequence lineages from <b>NextStrain</b> on June 27, 2022, and the clustering result above.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35944840">35944840</a> <b>(1)</b> Unique mutations in SARS-CoV-2 Omicron subvariants' non-spike proteins: Potential impacts on viral pathogenesis and host immune evasion<ul><li> The mutations were singled out from <b>Nextstrain</b> site and literature review.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35945925">35945925</a> <b>(1)</b> Analysis of co-occurring and mutually exclusive amino acid changes and detection of convergent and divergent evolution events in SARS-CoV-2<ul><li> Many organizations, such as the Global Initiative on Sharing all Influenza Data (GISAID) <xref rid="b0210" ref-type="bibr"><u>[42]</u></xref>, <b>Nextstrain</b> <xref rid="b0090" ref-type="bibr"><u>[18]</u></xref> and Pangolin <xref rid="b0200" ref-type="bibr"><u>[40]</u></xref>, are studying the phylodynamics of SARS-CoV-2 genomes to track and define new variants.<li> [[42]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [[18]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[40]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36533145">36533145</a> <b>(3)</b> covSampler: A subsampling method with balanced genetic diversity for large-scale SARS-CoV-2 genome data sets<ul><li> Several nomenclature systems for SARS-CoV-2 variants have been introduced to classify and track these variants, such as <b>Nextstrain</b> clades (<xref rid="R9" ref-type="bibr">Hadfield et&#xA0;al.<li> <b>Nextstrain</b> (<xref rid="R9" ref-type="bibr">Hadfield et&#xA0;al.<li> 2021</xref>) is used for sequence alignment, mutation calling, <b>Nextstrain</b> clade assignment, and quality control for SARS-CoV-2 genomes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35921539">35921539</a> <b>(2)</b> Clinical Evaluation of Severe Acute Respiratory Syndrome Coronavirus 2 Rapid Antigen Tests During the Omicron Wave in South Africa<sup><xref rid="jiac333-FM1" ref-type="author-notes">&#xA0;</xref></sup><ul><li> The majority (98.39%) of the SARS-CoV-2&#x2013;positive samples were classified as Omicron sublineage BA.2 (<b>Nextstrain</b> clade 21K).<li> The majority (73.37%) of the SARS-CoV-2&#x2013;positive cases were identified as Omicron sublineage BA.2 (<b>Nextstrain</b> clade 21K).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35916373">35916373</a> <b>(1)</b> Genomic epidemiology of the first two waves of SARS-CoV-2 in Canada<ul><li> An initial cleaning step was applied to remove sequences if they were listed in the <b>Nextstrain</b> exclude list on the day of data download (n=6272) (<xref rid="bib6" ref-type="bibr"><u>Bedford, 2021</u></xref>; <xref rid="bib31" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), duplicate entries (n=790), from a non-human host (n=2057), environmental samples (n=965), likely to contain sequencing errors based on previous temporal analyses (n=2) (<xref rid="bib64" ref-type="bibr"><u>Rambaut, 2020</u></xref>), or had incomplete sample collection dates (n=60,076).<li> [Bedford, 2021](          <person-group person-group-type="author"><name><surname>Bedford</surname><given-names>T</given-names></name></person-group>          <year iso-8601-date="2021">2021</year>          <data-title>Nextstrain/ncovnextstrain/ncov</data-title>          <version designator="v12">v12</version>          <source>Github</source>          <ext-link xlink:href="https://github.com/nextstrain/ncov" ext-link-type="uri">https://github.com/nextstrain/ncov</ext-link>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut, 2020](          <person-group person-group-type="author"><name><surname>Rambaut</surname><given-names>A</given-names></name></person-group>          <year iso-8601-date="2020">2020</year>          <article-title>Phylodynamic analysis | 176 genomes |</article-title>          <source>Virological.org</source>          <ext-link xlink:href="https://virological.org/t/phylodynamic-analysis-176-genomes-6-mar-2020/356/1" ext-link-type="uri">https://virological.org/t/phylodynamic-analysis-176-genomes-6-mar-2020/356/1</ext-link>          <date-in-citation iso-8601-date="2020-05-06">May 6, 2020</date-in-citation>        ) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35705189">35705189</a> <b>(1)</b> Investigation of COVID-19 Outbreak among Wildland Firefighters during Wildfire Response, Colorado, USA, 2020<ul><li> Phylogenetic inference of the consensus sequences was performed using IQTree version 2.0.3 (<ext-link xlink:href="http://www.iqtree.org" ext-link-type="uri">http://www.iqtree.org</ext-link>) with 1,000 ultrafast bootstrap replicates and phylogenetic tree visualization was performed using the python module ete3 version 3.1.2 (<ext-link xlink:href="https://pypi.org/project/ete3" ext-link-type="uri">https://pypi.org/project/ete3</ext-link>).&#xA0;Pangolin v.2.4.2<sup>5</sup> (<xref rid="R9" ref-type="bibr"><italic>9</italic></xref>) and <b>Nextstrain</b>&#x2019;s Nextclade tools (<xref rid="R10" ref-type="bibr"><italic>10</italic></xref>) were used to assign lineage and clade designations to each assembled genome.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36039088">36039088</a> <b>(3)</b> MPoxVR: A comprehensive genomic resource for monkeypox virus variant surveillance<ul><li> <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/labs/virus/vssi/" id="intref0020">https://www.ncbi.nlm.nih.gov/labs/virus/vssi/</ext-link>), <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/monkeypox/hmpxv1" id="intref0025">https://<b>nextstrain</b>.org/monkeypox/hmpxv1</ext-link>), Virus Pathogen Resource (ViPR; <ext-link ext-link-type="uri" xlink:href="https://www.viprbrc.org/brc/home.spg?decorator=pox" id="intref0030">https://www.viprbrc.org/brc/home.spg?decorator=pox</ext-link>), and Viralzone (<ext-link ext-link-type="uri" xlink:href="https://viralzone.expasy.org/9957" id="intref0035">https://viralzone.expasy.org/9957</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35913920">35913920</a> <b>(4)</b> Genomic surveillance, evolution and global transmission of SARS-CoV-2 during 2019&#x2013;2022<ul><li>Global Initiative on Sharing All Influenza Data (GISAID) and <b>Nextstrain</b> have suggested two different nomenclature systems for clade of COVID-19 [<xref rid="pone.0271074.ref039" ref-type="bibr"><u>39</u></xref>,<xref rid="pone.0271074.ref040" ref-type="bibr"><u>40</u></xref>].<li>The data used in this study were from Worldometer (<ext-link xlink:href="https://www.worldometers.info/coronavirus/" ext-link-type="uri">https://www.worldometers.info/coronavirus/</ext-link>), Bing COVID-19 dashboard (<ext-link xlink:href="https://www.bing.com/covid/" ext-link-type="uri">https://www.bing.com/covid/</ext-link>), Johns Hopkins Coronavirus Resource Center (<ext-link xlink:href="https://coronavirus.jhu.edu/map.html" ext-link-type="uri">https://coronavirus.jhu.edu/map.html</ext-link>), UK governmental site (<ext-link xlink:href="https://coronavirus.data.gov.uk/" ext-link-type="uri">https://coronavirus.data.gov.uk/</ext-link>), GitHub COVID-19 database (<ext-link xlink:href="https://github.com/CSSEGISandData/COVID-19" ext-link-type="uri">https://github.com/CSSEGISandData/COVID-19</ext-link>), CDC COVID-19 Research Articles Downloadable Database (<ext-link xlink:href="https://www.cdc.gov/library/researchguides/2019novelcoronavirus/researcharticles.html" ext-link-type="uri">https://www.cdc.gov/library/researchguides/2019novelcoronavirus/researcharticles.html</ext-link>), World Health Organization Situation Report (<ext-link xlink:href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports" ext-link-type="uri">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports</ext-link>), GISAID (<ext-link xlink:href="https://www.epicov.org/epi3/frontend#" ext-link-type="uri">https://www.epicov.org/epi3/frontend#</ext-link>), NCBI Virus (<ext-link xlink:href="https://www.ncbi.nlm.nih.gov/sars-cov-2/" ext-link-type="uri">https://www.ncbi.nlm.nih.gov/sars-cov-2/</ext-link>), and <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global</ext-link>) [<xref rid="pone.0271074.ref039" ref-type="bibr"><u>39</u></xref>,<xref rid="pone.0271074.ref040" ref-type="bibr"><u>40</u></xref>,<xref rid="pone.0271074.ref043" ref-type="bibr"><u>43</u></xref>&#x2013;<xref rid="pone.0271074.ref045" ref-type="bibr"><u>45</u></xref>].<li> [39](GISAID. 2022. [Cited 2022 January 31]. <ext-link xlink:href="https://www.epicov.org/epi3/frontend#" ext-link-type="uri">https://www.epicov.org/epi3/frontend#</ext-link>.) [40](Nextstrain. 2022. [Cited 2022 January 31]. <ext-link xlink:href="https://nextstrain.org/ncov/global" ext-link-type="uri">https://nextstrain.org/ncov/global</ext-link>.) [39](GISAID. 2022. [Cited 2022 January 31]. <ext-link xlink:href="https://www.epicov.org/epi3/frontend#" ext-link-type="uri">https://www.epicov.org/epi3/frontend#</ext-link>.) [40](Nextstrain. 2022. [Cited 2022 January 31]. <ext-link xlink:href="https://nextstrain.org/ncov/global" ext-link-type="uri">https://nextstrain.org/ncov/global</ext-link>.) [43](NCBI Virus. 2022. [Cited 2022 January 31]. <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/sars-cov-2/" ext-link-type="uri">https://www.ncbi.nlm.nih.gov/sars-cov-2/</ext-link>.) [45](World Health Organization. SARS-CoV-2 variants. 2022. [Cited 2022 January 31]. <ext-link xlink:href="http://www.who.int/csr/don/31-december-2020-sars-cov2-variants/en" ext-link-type="uri">www.who.int/csr/don/31-december-2020-sars-cov2-variants/en</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36009386">36009386</a> <b>(2)</b> The First Identification in Italy of SARS-CoV-2 Omicron BA.4 Harboring KSF141_del: A Genomic Comparison with Omicron Sub-Variants<ul><li> The <b>nextstrain</b> GSAID was used with <uri><b>nextstrain</b>/ncov</uri> (accessed on 11:46 pm, 25 April 2022) to visualize the current situation worldwide.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35950011">35950011</a> <b>(1)</b> Confirming Multiplex RT-qPCR Use in COVID-19 with Next-Generation Sequencing: Strategies for Epidemiological Advantage<ul><li>The FASTQ sequence file was analyzed and visualized for evolutionary relationships through the open-source toolkit <b>Nextstrain</b> (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35914602">35914602</a> <b>(0)</b> Temporal dynamics of SARS-CoV-2 genome and detection of variants of concern in wastewater influent from two metropolitan areas in Arkansas<ul><li> Lineage/clade analysis of SARS-CoV-2 wastewater samples with &gt;80 % genome coverage was performed using Pangolin and NextClade tools, v1.13.1, <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0030">https://clades.<b>nextstrain</b>.org/</ext-link> (<xref rid="bb0030" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>).<li> Clades were assigned by using the NextClade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="ir0005">https://clades.<b>nextstrain</b>.org</ext-link>).<li> [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>JOSS</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35982659">35982659</a> <b>(1)</b> Immune Heterogeneity and Epistasis Explain Punctuated Evolution of SARS-CoV-2<ul><li> We have used the open, GenBank-derived dataset of aligned SARS-CoV-2 sequences from <b>Nextstrain</b> (<xref rid="R13" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016297">36016297</a> <b>(7)</b> Evaluation of EPISEQ SARS-CoV-2 and a Fully Integrated Application to Identify SARS-CoV-2 Variants from Several Next-Generation Sequencing Approaches<ul><li> The percentage of coverage of the consensus sequence to the reference genome was calculated and SARS-CoV-2 variant clade and lineage were identified according to the <b>Nextstrain</b> clade and Pango lineage nomenclatures [<xref rid="B31-viruses-14-01674" ref-type="bibr"><u>31</u></xref>,<xref rid="B32-viruses-14-01674" ref-type="bibr"><u>32</u></xref>,<xref rid="B33-viruses-14-01674" ref-type="bibr"><u>33</u></xref>] using Nextclade v1.11.0 and Pangolin v3.1.20, respectively.<li> SARS-CoV-2 variant clade and lineage were identified in EPISEQ SARS-CoV-2, according to the <b>Nextstrain</b> clade and Pango lineage nomenclatures [<xref rid="B31-viruses-14-01674" ref-type="bibr"><u>31</u></xref>,<xref rid="B32-viruses-14-01674" ref-type="bibr"><u>32</u></xref>,<xref rid="B33-viruses-14-01674" ref-type="bibr"><u>33</u></xref>] using the same version of Nextclade (v1.11.0) and Pangolin (v3.1.20), respectively, as that used for the reference pipeline.<li> Variants are identified according to the up-to-date <b>Nextstrain</b> clade and Pango lineage nomenclatures using the Nextclade and Pangolin public tools, respectively [<xref rid="B31-viruses-14-01674" ref-type="bibr"><u>31</u></xref>,<xref rid="B32-viruses-14-01674" ref-type="bibr"><u>32</u></xref>,<xref rid="B33-viruses-14-01674" ref-type="bibr"><u>33</u></xref>].<li> 95% (based on the reference method) were considered to assess the concordance in variant call (<b>Nextstrain</b> clade and Pango lineage) by the EPISEQ SARS-CoV-2 pipeline vs.<li> 0.0001), <b>Nextstrain</b> clade and Pango lineage classifications (&gt;99% concordance), and amino acid substitution identification (&gt;99% concordance within the spike protein), over 1362 NGS data covering alpha to omicron SARS-CoV-2 variants.<li>The comparison of 40 NGS data generated by a variety of sequencing approaches (ARTIC and VSS kits on Illumina and ONT platforms) using EPISEQ SARS-CoV-2 revealed a perfect concordance in <b>Nextstrain</b> clade and Pango lineage classifications.<li> The platform is updated weekly to evolve with the reporting of new SARS-CoV-2 <b>Nextstrain</b> clades, Pango lineages, and VOC.<li> [31](          <article-title>Nextstrain&#x2014;Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [32](          <article-title>Centre for Genomic Pathogen Surveillance&#x2014;Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin)</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [33](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [31](          <article-title>Nextstrain&#x2014;Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [32](          <article-title>Centre for Genomic Pathogen Surveillance&#x2014;Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin)</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [33](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [31](          <article-title>Nextstrain&#x2014;Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [32](          <article-title>Centre for Genomic Pathogen Surveillance&#x2014;Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin)</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [33](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016294">36016294</a> <b>(1)</b> Sequence Similarity Network Analysis Provides Insight into the Temporal and Geographical Distribution of Mutations in SARS-CoV-2 Spike Protein<ul><li> Another popular tool, <b>NextStrain</b> [<xref rid="B21-viruses-14-01672" ref-type="bibr"><u>21</u></xref>], also uses phylogenetic tree analysis to study pathogen evolution [<xref rid="B22-viruses-14-01672" ref-type="bibr"><u>22</u></xref>].<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](          <person-group person-group-type="author"><name><surname>Paudel</surname><given-names>S.</given-names></name><name><surname>Dahal</surname><given-names>A.</given-names></name><name><surname>Kumar Bhattarai</surname><given-names>H.</given-names></name><name><surname>Yavropoulou</surname><given-names>M.</given-names></name><name><surname>Paraskevis</surname><given-names>D.</given-names></name><name><surname>Tsiodras</surname><given-names>S.</given-names></name></person-group>          <article-title>Temporal Analysis of SARS-CoV-2 Variants during the COVID-19 Pandemic in Nepal</article-title>          <source>COVID</source>          <year>2021</year>          <volume>1</volume>          <fpage>423</fpage>          <lpage>434</lpage>          <pub-id pub-id-type="doi">10.3390/covid1020036</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016298">36016298</a> <b>(10)</b> Identifying HIV-1 Transmission Clusters in Uzbekistan through Analysis of Molecular Surveillance Data<ul><li> the ML trees in Newick format were importing into MicrobeTrace after the reconstruction by <b>Nextstrain</b> tool (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>, accessed on 20 May 2021) under GTR (general time reversible) model.<li>To evaluate the potential origins of HIV-1 clusters in Uzbekistan, all sequences were combined with those from other countries and analyzed with the <b>Nextstrain</b> tool (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>) that uses ML analysis implemented in Tree-Time [<xref rid="B18-viruses-14-01675" ref-type="bibr"><u>18</u></xref>,<xref rid="B19-viruses-14-01675" ref-type="bibr"><u>19</u></xref>].<li> The total dataset for <b>Nextstrain</b> ML analysis contained 1055 HIV-1 sequences, comprising 316 sequences from Uzbekistan and 739 global HIV-1 <italic toggle="yes">pol</italic> sequences, plus one HIV-1 subtype C sequence (GenBank accession numbers: AF443091) used as the outgroup.<li> The output JASON files from <b>Nextstrain</b> pipeline were viewed using Auspice (<uri xlink:href="https://auspice.us">https://auspice.us</uri>, accessed on 20 May 2021).<li>Here, we combined NS-HIVDR subtype A6 and CRF02_AG sequences with similar sequences of the same subtypes identified by BLAST search in HIV Databases at LANL (Los-Alamos National Laboratory), followed by data analysis in <b>Nextstrain</b>.<li> Using an evolutionary time scale, the <b>Nextstrain</b> analysis indicated that the CRF02_AG outbreak cluster in Uzbekistan evolved from a MRCA introduced around 25-05-2000 [12-02-1998, 14-04-2001].<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35966675">35966675</a> <b>(1)</b> Dynamics of Viral Infection and Evolution of SARS-CoV-2 Variants in the Calabria Area of Southern Italy<ul><li>The established nomenclature system for tracking SARS-CoV-2 lineages is based on specific platforms (i.e., GISAID, <b>Nextstrain</b>, Pango).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35967580">35967580</a> <b>(1)</b> An outbreak of infection due to severe acute respiratory corona virus-2 in a neonatal unit from a low and middle income setting<ul><li> The bcftools 1.7-2 mpileup method was then used to filter out the low-quality mutations, and all the sequences were deposited in GISAID.<sup><xref rid="footnote2" ref-type="fn"><u>2</u></xref></sup> A custom pipeline based on a local version of <b>NextStrain</b> was used for the phylogenetic analysis using a South African dataset with genomes collected during the same time as the study period.<li> [2](<a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35896605">35896605</a> <b>(3)</b> A ricin-based peptide BRIP from <italic>Hordeum vulgare</italic> inhibits M<sup>pro</sup> of SARS-CoV-2<ul><li> The prevalent mutations found in M<sup>pro</sup> of the Omicron variant are at positions 135, 842, and 856 (retrieved from <b>nextstrain</b>; <ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35996591">35996591</a> <b>(2)</b> A single early introduction governed viral diversity in the second wave of SARS-CoV-2 epidemic in Hungary<ul><li> <xref rid="R53" ref-type="bibr"><u>Shu and McCauley 2017</u></xref>), and analysed by many research groups using, for example, the <b>Nextstrain</b> pipeline (<xref rid="R16" ref-type="bibr">Hadfield et&#xA0;al.<li> While the first wave was driven by multiple variants circulating at low to intermediate frequencies, the second wave showed a markedly different pattern: it was dominated by a single, high-frequency variant, B.1.160 (20A.EU2 <b>Nextstrain</b> clade), suggesting a lower genetic diversity.<li> [Shu and McCauley 2017](<person-group person-group-type="author"><string-name><surname>Shu</surname><given-names>Y.</given-names></string-name>, and <string-name><surname>McCauley</surname><given-names>J.</given-names></string-name></person-group> (<year>2017</year>) &#x2018;<article-title>GISAID: Global Initiative on Sharing All Influenza Data: From Vision to Reality</article-title>&#x2019;, <source><italic toggle="yes">Eurosurveillance</italic></source>, <volume>22</volume>: <page-range>30494</page-range>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35996592">35996592</a> <b>(3)</b> Tracking the turnover of SARS-CoV-2 VOCs Gamma to Delta in a Brazilian state (Minas Gerais) with a high-vaccination status<ul><li> This data set encompasses all Delta (Pango lineage B.1.617.2*, <b>NextStrain</b> clade 21A) sequences from this variant NexStrain build (as captured on 12 October 2021, <italic toggle="yes">n</italic>&#x2009;=&#x2009;5,349).<li> These estimates were then recalculated as proportions (divided by the estimated total number of Delta infections), which were used to calculate the number of sequences to sample, considering a fixed target (<italic toggle="yes">n&#x2009;=</italic>&#x2009;5,000, to approximately match the amount of <b>NextStrain</b> background sequences).<li>We assembled an independent secondary data set using the <b>NextStrain</b> background sequences and a uniform sampling scheme for Brazilian sequences to verify the influence of data set composition on the obtained results.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35919871">35919871</a> <b>(2)</b> Evolution of ACE2-independent SARS-CoV-2 infection and mouse adaption after passage in cells expressing human and mouse ACE2<ul><li>The GISAID (<ext-link xlink:href="https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/" ext-link-type="uri">https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/</ext-link>) (<xref rid="R20" ref-type="bibr"><u>Elbe and Buckland-Merrett 2017</u></xref>) EpiCoV &#x2018;search&#x2019; function (<ext-link xlink:href="https://www.epicov.org/epi3/frontend#577a0f" ext-link-type="uri">https://www.epicov.org/epi3/frontend#577a0f</ext-link>) was used to identify the number of GISAID SARS-CoV-2 sequence submissions containing each amino acid change in the MA viruses.<li> [Elbe and Buckland-Merrett 2017](<person-group person-group-type="author"><string-name><surname>Elbe</surname><given-names>S.</given-names></string-name>, and <string-name><surname>Buckland-Merrett</surname><given-names>G.</given-names></string-name></person-group> (<year>2017</year>) &#x2018;<article-title>Data, Disease and Diplomacy: GISAID&#x2019;s Innovative Contribution to Global Health</article-title>&#x2019;, <source><italic toggle="yes">Global Challenges (Hoboken, NJ)</italic></source>, <volume>1</volume>: <fpage>33</fpage>&#x2013;<lpage>46</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36942016">36942016</a> <b>(1)</b> Estimating COVID-19 Vaccination and Booster Effectiveness Using Electronic Health Records From an Academic Medical Center in Michigan<ul><li> The variants are colored by their WHO variant names, whereas the <b>Nextstrain</b> clade names are shown in the figure.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35905853">35905853</a> <b>(5)</b> Analysis of the mutation dynamics of SARS-CoV-2 genome in the samples from Georgia State of the United States<ul><li>The SARS-CoV-2 Genome Sequences were downloaded in the FASTA format and online tools NextClade of <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir035">https://clades.<b>nextstrain</b>.org/</ext-link>) (<xref rid="b0115" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>, <xref rid="b0010" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) were used for the analysis of the genome sequences.<li>The phylogenetic tree was visualized by <b>NextStrain</b> Auspice (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/tree" id="ir040">https://clades.<b>nextstrain</b>.org/tree</ext-link>).<li>The <b>NextStrain</b> clade analysis revealed 16 different types of clades ranging from an occurrence rate of 0.03% (1 out of 3,970) to 49.6% (1,968 out of 3,970).<li> The phylogenetic tree generated with the <b>NextStrain</b> analysis tools is shown in the <xref rid="f0015" ref-type="fig">Fig.<li> The phylogenetic tree was generated using the <b>NextStrain</b> Auspice online tools (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/tree" id="ir005">https://clades.<b>nextstrain</b>.org/tree</ext-link>).<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Aksamentov et al., 2021](Aksamentov, I., Roemer, C., Hodcroft, E.B. and Neher, R.A., 2021. Nextclade: clade assignment, mutation calling and quality control for viral genomes.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35894057">35894057</a> <b>(5)</b> Insight into Genetic Characteristics of Identified SARS-CoV-2 Variants in Egypt from March 2020 to May 2021<ul><li> In the same context, several online classification and analysis tools have been developed such as GISAID, <b>Nextstrain</b> SARS-CoV-2, Pangolin (<uri xlink:href="https://cov-lineages.org/resources/pangolin.html">https://cov-lineages.org/resources/pangolin.html</uri> accessed on 3 January 2022), Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> accessed on 3 January 2022), and others.<li>According to Nextclade analysis, 529 of the identified genomes belonged to <b>Nextstrain</b> clade 20D, 484 belonged to clade 20A, 94 belonged to clade 20B, 59 belonged to clade 19A, 28 belonged to clade 19B, and 3 belonged to clade 20C.<li> Pangolin (<uri xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</uri> accessed on 3 January 2022) and <b>NextStrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming">https://<b>nextstrain</b>.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</uri> (accessed on 3 January 2022)) criteria were used to classify the lineage and sublineage of the collected samples.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35876490">35876490</a> <b>(1)</b> Replacement of the Alpha variant of SARS-CoV-2 by the Delta variant in Lebanon between April and June 2021<ul><li>The <b>NextStrain</b> [<xref rid="R22" ref-type="bibr"><u>22</u></xref>] open dataset (version 2022-03-09), based on genomes deposited with INSDC public databases [<xref rid="R32" ref-type="bibr"><u>32</u></xref>], consisting of 4&#x200A;119&#x200A;969 genomes, was analysed to understand frequencies of mutations present in Lebanon with respect to the global frequencies, for each of the major variants of concern, Alpha, Delta and Omicron.<li> [22](          <person-group person-group-type="author"><name><surname>Li</surname><given-names>H</given-names></name></person-group>          <article-title>Aligning Sequence Reads, Clone Sequences and Assembly Contigs with 175 BWA-MEM</article-title>          <year>2013</year>          <ext-link xlink:href="http://arxiv.org/abs/1303.3997" ext-link-type="uri">http://arxiv.org/abs/1303.3997</ext-link>        ) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26657633">26657633</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35877668">35877668</a> <b>(0)</b> Post-vaccination antibody evaluation for nosocomial SARS-CoV-2 delta variant breakthrough infection<ul><li> The variant assignment was performed on Pangolin, UCSC Genome Browser (<ext-link xlink:href="https://genome.ucsc.edu/cgi-bin/hgPhyloPlace" ext-link-type="uri">https://genome.ucsc.edu/cgi-bin/hgPhyloPlace</ext-link>) and Nextclade (<ext-link xlink:href="http://clades.<b>nextstrain</b>.org/" ext-link-type="uri">http://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35935779">35935779</a> <b>(3)</b> Mild reinfection with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) Delta variant: First case report from Indonesia<ul><li> Clade information as inferred by GISAID, <b>Nextstrain</b>, and Pangolin nomenclatures, are shown.<li> S-gene sequencing revealed mutations in the S gene (T19R, G142D, E156G, Del157/158, A222V, L452R, T478K, D614G, P681R, and D950N) that were consistent with the Delta variant, belonging to Pango Lineage B.1.617.2 or clade 21A on <b>Nextstrain</b> (<xref rid="B22" ref-type="bibr"><u>22</u></xref>).<li> The US CDC maintains that the best evidence of reinfection is differing clades/lineages/variants of SARS-CoV-2, as defined in <b>Nextstrain</b> and GISAID, between an initial and subsequent infection (<xref rid="B9" ref-type="bibr"><u>9</u></xref>).<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [9](<collab>CDC.</collab><source><italic>Investigative Criteria for Suspected Cases of SARS-CoV-2 Reinfection (ICR).</italic></source> (<year>2020</year>). Available online at: <ext-link xlink:href="https://www.cdc.gov/coronavirus/2019-ncov/php/invest-criteria.html" ext-link-type="uri">https://www.cdc.gov/coronavirus/2019-ncov/php/invest-criteria.html</ext-link> (<comment>accessed on March 14, 2022</comment>).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35867852">35867852</a> <b>(2)</b> Novel Human Parechovirus 3 Diversity, Recombination, and Clinical Impact Across 7 Years: An Australian Story<ul><li> <b>Nextstrain</b> [<xref rid="jiac311-B20" ref-type="bibr"><u>20</u></xref>] was used to analyze AR genome evolution over time.<li> The <b>Nextstrain</b> plot shows the progressive evolution of the AR strain for most of the disease outbreak periods.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=2513255">2513255</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35901580">35901580</a> <b>(2)</b> Investigation of healthcare-associated COVID-19 in a large French hospital group by whole-genome sequencing<ul><li> <b>Nextstrain</b> classification (<xref rid="bib6" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) was used to assign a lineage to each SARS-CoV-2 strain.<li> According to the <b>Nextstrain</b> classification, 59 (51.3 %) belonged to clade 20&#x2009;A, 31 (27 %) to clade 20E, 19 (16.5 %) to clade 20B, and 6 (5.2 %) to clade 20&#x2009;C.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35893668">35893668</a> <b>(1)</b> Immune Escape Associated with RBD Omicron Mutations and SARS-CoV-2 Evolution Dynamics<ul><li>Using the web-based <b>NextStrain</b> toolkit [<xref rid="B10-viruses-14-01603" ref-type="bibr"><u>10</u></xref>], we analyzed the spread of SARS-CoV-2 based on 3257 SARS-CoV-2 genomes, sampled between February 2019 and February 2022 from GISAID [<xref rid="B10-viruses-14-01603" ref-type="bibr"><u>10</u></xref>,<xref rid="B11-viruses-14-01603" ref-type="bibr"><u>11</u></xref>] (<xref rid="viruses-14-01603-f002" ref-type="fig"><u>Figure 2</u></xref>).<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35879932">35879932</a> <b>(1)</b> Gestion de l&#x2019;&#xE9;pid&#xE9;mie de COVID-19&#xA0;au sein du groupe a&#xE9;ronaval (janvier&#x2013;avril 2020) par le Centre d&#x2019;&#xE9;pid&#xE9;miologie et de sant&#xE9; publique des arm&#xE9;es*<sup><xref ref-type="fn" rid="d36e1833">&#x2606;</xref></sup><ul><li> Pour la reconstruction phylog&#xE9;n&#xE9;tique, les techniques d&#x2019;analyse <b>NEXTSTRAIN</b> <xref rid="bib0240" ref-type="bibr"><u>[12]</u></xref>, MAFFT <xref rid="bib0240" ref-type="bibr"><u>[12]</u></xref>, <xref rid="bib0245" ref-type="bibr"><u>[13]</u></xref>, IQ-TREE <xref rid="bib0250" ref-type="bibr"><u>[14]</u></xref> et TREETIME <xref rid="bib0255" ref-type="bibr"><u>[15]</u></xref> ont &#xE9;t&#xE9; mises en &#x153;uvre.<li> [[12]](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinforma Oxf Engl</source>          <volume>34</volume>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>        ) [[12]](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinforma Oxf Engl</source>          <volume>34</volume>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>        ) [[13]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [[14]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [[15]](          <person-group person-group-type="author">            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Puller</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>TreeTime: maximum-likelihood phylodynamic analysis</article-title>          <source>Virus Evol</source>          <volume>4</volume>          <year>2018</year>          <comment>[vex042]</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35868481">35868481</a> <b>(1)</b> Inhaled therapy for COVID-19: Considerations of drugs, formulations and devices<ul><li><bold>SARS-CoV-2 variants</bold><hr/></th></tr><tr><th>WHO classification</th><th>Alpha</th><th>Beta</th><th>Gamma</th><th>Delta</th><th>Omicron</th></tr></thead><tbody><tr><td>Pango Lineage</td><td>B.1.1.7</td><td>B.1.351</td><td>P.1</td><td>B.1.617.2</td><td>B.1.1.529</td></tr><tr><td>Origin of detection</td><td>United Kingdom</td><td>South Africa</td><td>Brazil</td><td>India</td><td>Multiple</td></tr><tr><td>Date of designation</td><td>18 December 2020</td><td>18 December 2020</td><td>11 January 2021</td><td>11 May 2021</td><td>26 November 2021</td></tr><tr><td>GISAID clade</td><td>GRY</td><td>GH/501Y.V2</td><td>GR/501Y.V3</td><td>G/478&#xA0;K.V1</td><td>GRA</td></tr><tr><td><b>Nextstrain</b> clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35857745">35857745</a> <b>(1)</b> Spread of endemic SARS-CoV-2 lineages in Russia before April 2021<ul><li> After that, we excluded sequences (1) shorter than 29,000 bp, (2) with more than 3,000 (for Russian sequences) or 300 (for all other countries) positions of missing data (Ns), (3) excluded by <b>Nextstrain</b> [<xref rid="pone.0270717.ref009" ref-type="bibr"><u>9</u></xref>], (4) in non-human animals, (5) with a genetic distance to the reference genome more than four standard deviations from the epi-week mean genetic distance to the reference, or (6) with incomplete collection dates.<li> [9](<ext-link xlink:href="https://github.com/nextstrain/ncov/blob/master/defaults/exclude.txt" ext-link-type="uri">https://github.com/nextstrain/ncov/blob/master/defaults/exclude.txt</ext-link>. In: GitHub [Internet]. [cited 23 Mar 2021]. Available: <ext-link xlink:href="https://github.com/nextstrain/ncov/blob/master/defaults/exclude.txt" ext-link-type="uri">https://github.com/nextstrain/ncov/blob/master/defaults/exclude.txt</ext-link>.) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35899232">35899232</a> <b>(3)</b> A Unique Trinucleotide-Bloc Mutation-Based Two SARS-CoV-2 Genotypes with Potential Pathogenic Impacts<ul><li> Although one SARS-CoV-2a affected person was reported in Italy in January [<xref rid="B11" ref-type="bibr"><u>11</u></xref>], an analysis of the sequences deposited in the GISAID database using the <b>Nextstrain</b> web tool (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/6m" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global/6m</ext-link>) which allows to select a time window and look at the nucleotide/amino acid changes in the deposited sequences shows that the 28881&#x2013;28883: GGG position has higher entropy (a measure of changes) if the sequences from December 2019 to February 2020 are compared with those from March 2020 to June 2020.<li> [11](          <person-group person-group-type="author"><name><surname>Stefanelli</surname><given-names>P.</given-names></name><name><surname>Faggioni</surname><given-names>G.</given-names></name><name><surname>Lo Presti</surname><given-names>A.</given-names></name><etal/></person-group>          <article-title>Whole genome and phylogenetic analysis of two sars-cov-2 strains isolated in italy in january and february 2020: additional clues on multiple introductions and further circulation in europe</article-title>          <source><italic>Euro Surveill</italic></source>          <year>2020</year>          <volume>25</volume>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35868457">35868457</a> <b>(1)</b> Optimized infection control practices augment the robust protective effect of vaccination for ESRD patients during a hemodialysis facility SARS-CoV-2 outbreak<ul><li> Libraries were sequenced and analyzed as previously described.<xref rid="bib0005" ref-type="bibr"><sup>5</sup></xref><sup>,</sup><xref rid="bib0008" ref-type="bibr"><sup>8</sup></xref><sup>,</sup><xref rid="bib0009" ref-type="bibr"><sup>9</sup></xref> For phylogenetic inference (ie to determine the hierarchy of case relationships), sequences were integrated with associated metadata and aligned on a local implementation of <b>NextStrain</b><xref rid="bib0010" ref-type="bibr"><sup>10</sup></xref> using augur and displayed via a web browser using auspice.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35852336">35852336</a> <b>(12)</b> Contrasting Distribution of SARS-CoV-2 Lineages across Multiple Rounds of Pandemic Waves in West Bengal, the Gateway of East and North-East States of India<ul><li> The rate of viral transmission was found to be variable during the ongoing pandemic (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).<li> All quality control (QC) passed genomes were evaluated through the command-line version of <b>Nextstrain</b>&#x2019;s NextClade (v1.2) software (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/nextclade" ext-link-type="uri">https://github.com/<b>nextstrain</b>/nextclade</ext-link>) to obtain a list of all nucleotide and amino-acid substitutions.<li> <b>Nextstrain</b>&#x2019;s Augur (<xref rid="B40" ref-type="bibr"><u>40</u></xref>) version 12.0.0 pipeline utilized by <b>Nextstrain</b> filtered out genomes that have a minimum length of 27,000&#x2009;bp before aligning them using the MAFFT package.<li> As a reference, the <b>Nextstrain</b> nCoV toolkit (<xref rid="B41" ref-type="bibr"><u>41</u></xref>) (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov</ext-link>) aligns SARS-CoV-2 sequences to the SARS-CoV-2 isolate Wuhan-Hu-1 complete genome (MN908947).<li> The resulting files are combined and used by Auspice version 2.23.0 (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/auspice" ext-link-type="uri">https://github.com/<b>nextstrain</b>/auspice</ext-link>) to visualize phylogenetic relationships and geographic distributions of SARS-CoV-2 across time.<li> [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35350209">35350209</a> <b>(1)</b> Maximum likelihood pandemic-scale phylogenetics<ul><li> <b>NextStrain</b><sup><xref rid="R35" ref-type="bibr"><u>35</u></xref></sup>) and many other genomic analyses often downsample global SARS-CoV-2 datasets to a few thousand genomes, leading to loss of power and resolution<sup><xref rid="R36" ref-type="bibr"><u>36</u></xref>,<xref rid="R37" ref-type="bibr"><u>37</u></xref></sup>.<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12228001">12228001</a>) [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12066312">12066312</a>) <li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12228001">12228001</a>) [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12066312">12066312</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35843590">35843590</a> <b>(2)</b> SARS-CoV-2 and the central nervous system: Emerging insights into hemorrhage-associated neurological consequences and therapeutic considerations<ul><li> The delta variant of clade 21&#x2009;A exhibits mutation clusters in ORF1a (D691A, G1125C, M2796L, K2894E, V4102I, etc.), ORF1b (S598G, K774N, D884N, P1095S, P1975L, etc.), ORF3 (S26L, V259L), ORF7a (P45L, V82A, T120I), ORF7b (T40I), ORF8 (D119-, F120-), S (T19R, D138-, P139-, G142D, E156-, F157-, R158G, K417N/T, L452R, T478K, E484K, N501Y, T547I, D614G, V615I/F, P681R, A831V, T95I, P1263L, etc.), N (D63G, L161F, R203M, G215C, D377Y), and M (I82T, S94G, D209Y) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="ir0015">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35840884">35840884</a> <b>(1)</b> Genomic surveillance of SARS-CoV-2 in US military compounds in Afghanistan reveals multiple introductions and outbreaks of Alpha and Delta variants<ul><li> All GISAID sequences originating from Afghanistan were added to the top-BLAST genomes and our sequenced genomes, as were reference genomes from the <b>NextStrain</b> Global set [<xref ref-type="bibr" rid="CR29"><u>29</u></xref>].<li> [29](NextStrain SARS-CoV-2 Resources. Available from: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global">https://nextstrain.org/ncov/global</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35889149">35889149</a> <b>(3)</b> SARS-CoV-2 Amino Acid Mutations Detection in Greek Patients Infected in the First Wave of the Pandemic<ul><li> These specific mutations have also been reported in large-scale meta-analytic studies, where data were extracted from the GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>), <b>NEXTSTRAIN</b> (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>) and National Center for Biotechnology Information (NCBI) (<uri xlink:href="https://www.ncbi.nlm.nih.gov/sars-cov-2/">https://www.ncbi.nlm.nih.gov/sars-cov-2/</uri>) databases or were tracked during our study in a high number of GISAID SARS-CoV-2 sequences by using the CoV-GLUE web bioinformatics application (<uri xlink:href="https://cov-glue.cvr.gla.ac.uk/">https://cov-glue.cvr.gla.ac.uk/</uri>, accessed on 25 June 2022) or were published by other groups that used their own large cohorts of patients [<xref rid="B36-microorganisms-10-01430" ref-type="bibr"><u>36</u></xref>,<xref rid="B37-microorganisms-10-01430" ref-type="bibr"><u>37</u></xref>,<xref rid="B57-microorganisms-10-01430" ref-type="bibr"><u>57</u></xref>,<xref rid="B58-microorganisms-10-01430" ref-type="bibr"><u>58</u></xref>,<xref rid="B59-microorganisms-10-01430" ref-type="bibr"><u>59</u></xref>,<xref rid="B60-microorganisms-10-01430" ref-type="bibr"><u>60</u></xref>,<xref rid="B61-microorganisms-10-01430" ref-type="bibr"><u>61</u></xref>,<xref rid="B62-microorganisms-10-01430" ref-type="bibr"><u>62</u></xref>,<xref rid="B63-microorganisms-10-01430" ref-type="bibr"><u>63</u></xref>,<xref rid="B64-microorganisms-10-01430" ref-type="bibr"><u>64</u></xref>,<xref rid="B65-microorganisms-10-01430" ref-type="bibr"><u>65</u></xref>,<xref rid="B66-microorganisms-10-01430" ref-type="bibr"><u>66</u></xref>,<xref rid="B67-microorganisms-10-01430" ref-type="bibr"><u>67</u></xref>,<xref rid="B68-microorganisms-10-01430" ref-type="bibr"><u>68</u></xref>,<xref rid="B69-microorganisms-10-01430" ref-type="bibr"><u>69</u></xref>,<xref rid="B70-microorganisms-10-01430" ref-type="bibr"><u>70</u></xref>,<xref rid="B71-microorganisms-10-01430" ref-type="bibr"><u>71</u></xref>,<xref rid="B72-microorganisms-10-01430" ref-type="bibr"><u>72</u></xref>,<xref rid="B73-microorganisms-10-01430" ref-type="bibr"><u>73</u></xref>,<xref rid="B74-microorganisms-10-01430" ref-type="bibr"><u>74</u></xref>,<xref rid="B75-microorganisms-10-01430" ref-type="bibr"><u>75</u></xref>,<xref rid="B76-microorganisms-10-01430" ref-type="bibr"><u>76</u></xref>,<xref rid="B77-microorganisms-10-01430" ref-type="bibr"><u>77</u></xref>,<xref rid="B78-microorganisms-10-01430" ref-type="bibr"><u>78</u></xref>].<li> [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33170902">33170902</a>) [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34570776">34570776</a>) [57](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32793182">32793182</a>) [58](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33083157">33083157</a>) [59](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32742035">32742035</a>) [60](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34606987">34606987</a>) [61](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33347989">33347989</a>) [62](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32814791">32814791</a>) [63](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) [64](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32991842">32991842</a>) [65](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33060796">33060796</a>) [66](          <person-group person-group-type="author"><name><surname>Martin</surname><given-names>J.</given-names></name><name><surname>Klapsa</surname><given-names>D.</given-names></name><name><surname>Wilton</surname><given-names>T.</given-names></name><name><surname>Zambon</surname><given-names>M.</given-names></name><name><surname>Bentley</surname><given-names>E.</given-names></name><name><surname>Bujaki</surname><given-names>E.</given-names></name><name><surname>Fritzsche</surname><given-names>M.</given-names></name><name><surname>Mate</surname><given-names>R.</given-names></name><name><surname>Majumdar</surname><given-names>M.</given-names></name></person-group>          <article-title>Tracking SARS-CoV-2 in Sewage: Evidence of Changes in Virus Variant Predominance during COVID-19 Pandemic</article-title>          <source>Viruses</source>          <year>2020</year>          <volume>12</volume>          <elocation-id>1144</elocation-id>          <pub-id pub-id-type="doi">10.3390/v12101144</pub-id>        ) [67](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32451533">32451533</a>) [68](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33755704">33755704</a>) [69](          <person-group person-group-type="author"><name><surname>Ugurel</surname><given-names>O.M.</given-names></name><name><surname>Ata</surname><given-names>O.</given-names></name><name><surname>Turgut-Balik</surname><given-names>D.</given-names></name></person-group>          <article-title>An updated analysis of variations in SARS-CoV-2 genome</article-title>          <source>Turk. J. Biol. Turk Biyol. Derg.</source>          <year>2020</year>          <volume>44</volume>          <fpage>157</fpage>          <lpage>167</lpage>          <pub-id pub-id-type="doi">10.3906/biy-2005-111</pub-id>        ) [70](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33353955">33353955</a>) [71](          <person-group person-group-type="author"><name><surname>Leary</surname><given-names>S.</given-names></name><name><surname>Gaudieri</surname><given-names>S.</given-names></name><name><surname>Parker</surname><given-names>M.D.</given-names></name><name><surname>Chopra</surname><given-names>A.</given-names></name><name><surname>James</surname><given-names>I.</given-names></name><name><surname>Pakala</surname><given-names>S.</given-names></name><name><surname>Alves</surname><given-names>E.</given-names></name><name><surname>John</surname><given-names>M.</given-names></name><name><surname>Lindsey</surname><given-names>B.B.</given-names></name><name><surname>Keeley</surname><given-names>A.J.</given-names></name><etal/></person-group>          <article-title>Generation of a novel SARS-CoV-2 sub-genomic RNA due to the R203K/G204R variant in nucleocapsid: Homologous recombination has potential to change SARS-CoV-2 at both protein and RNA level</article-title>          <source>bioRxiv</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.20411/pai.v6i2.460</pub-id>        ) [72](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33564691">33564691</a>) [73](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33015402">33015402</a>) [74](          <person-group person-group-type="author"><name><surname>Nzivo</surname><given-names>M.M.</given-names></name><name><surname>Budambula</surname><given-names>N.L.M.</given-names></name></person-group>          <article-title>Mutations and Epidemiology of SARS-CoV-2 Compared to Selected Corona Viruses during the First Six Months of the COVID-19 Pandemic: A Review</article-title>          <source>J. Pure Appl. Microbiol.</source>          <year>2021</year>          <volume>15</volume>          <fpage>524</fpage>          <lpage>533</lpage>          <pub-id pub-id-type="doi">10.22207/JPAM.15.2.19</pub-id>        ) [75](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33239633">33239633</a>) [76](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33101264">33101264</a>) [77](          <person-group person-group-type="author"><name><surname>Alouane</surname><given-names>T.</given-names></name><name><surname>Laamarti</surname><given-names>M.</given-names></name><name><surname>Essabbar</surname><given-names>A.</given-names></name><name><surname>Hakmi</surname><given-names>M.</given-names></name><name><surname>Bouricha</surname><given-names>E.M.</given-names></name><name><surname>Chemao-Elfihri</surname><given-names>M.W.</given-names></name><name><surname>Kartti</surname><given-names>S.</given-names></name><name><surname>Boumajdi</surname><given-names>N.</given-names></name><name><surname>Bendani</surname><given-names>H.</given-names></name><name><surname>Laamarti</surname><given-names>R.</given-names></name><etal/></person-group>          <article-title>Genomic Diversity and Hotspot Mutations in 30,983 SARS-CoV-2 Genomes: Moving Toward a Universal Vaccine for the &#x201C;Confined Virus&#x201D;?</article-title>          <source>Pathogens</source>          <year>2020</year>          <volume>9</volume>          <elocation-id>829</elocation-id>          <pub-id pub-id-type="doi">10.3390/pathogens9100829</pub-id>        ) [78](          <person-group person-group-type="author"><name><surname>Patro</surname><given-names>L.P.P.</given-names></name><name><surname>Sathyaseelan</surname><given-names>C.</given-names></name><name><surname>Uttamrao</surname><given-names>P.P.</given-names></name><name><surname>Rathinavelan</surname><given-names>T.</given-names></name></person-group>          <article-title>Global variation in SARS-CoV-2 proteome and its implication in pre-lockdown emergence and dissemination of 5 dominant SARS-CoV-2 clades</article-title>          <source>Infect. Genet. Evol. J. Mol. Epidemiol. Evol. Genet. Infect. Dis.</source>          <year>2021</year>          <volume>93</volume>          <fpage>104973</fpage>          <pub-id pub-id-type="doi">10.1016/j.meegid.2021.104973</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35855306">35855306</a> <b>(2)</b> A comprehensive review on Covid-19 Omicron (B.1.1.529) variant<ul><li> As of June 2021, the Next strain had been used to track SARS-CoV-2, recognizing 13 significant clades (19A&#x2013;B, 20A&#x2013;20J, and 21A) (<xref rid="b0415" ref-type="bibr"><b>Nextstrain</b> &#x201C;<b>Nextstrain</b> COVID-19&#x2033;, 2022</xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35836127">35836127</a> <b>(4)</b> Phylogenetic analysis and comparative genomics of SARS-CoV-2 from survivor and non-survivor COVID-19 patients in Cordoba, Argentina<ul><li>Epidemiological data of the genome sequences of SARS-CoV-2 2obtained from COVID-19 patients in Cordoba</p></caption><table frame="hsides" rules="groups"><thead><tr><th>Genebank ID</th><th>Isolation site</th><th>N Ct value</th><th>Age</th><th>Gender</th><th>Symp- toms</th><th>Comor- bidities</th><th>Hospita- lization</th><th>Disease status</th><th>Pangolin lineage<break/>(<b>Nextstrain</b> Clade)</th><th>Putative Origin (% identity)</th></tr></thead><tbody><tr><td>MW633892.1.<li>The sequences were analyzed using <b>NextStrain</b> tools (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link>), such as NextClade V1.6.0 [<xref ref-type="bibr" rid="CR14"><u>14</u></xref>], and classified by Pangolin lineages.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35910603">35910603</a> <b>(4)</b> Mapping Genetic Events of SARS-CoV-2 Variants<ul><li> The WHO named the variants that significantly affected the current public health by integrating the nomenclature systems including GISAID (<ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>), <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/" ext-link-type="uri">https://<b>nextstrain</b>.org/sars-cov-2/</ext-link>) and Pango (<xref rid="ref23" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>).<li> On the other hand, our classification was based on the co-mutations, which avoided the statistical uncertainty and limitations in computation and visualization of the phylogenetic tree-based classification like GISAID, <b>Nextstrain</b>, and Pango.<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35856045">35856045</a> <b>(2)</b> Relatively rapid evolution rates of SARS-CoV-2 spike gene at the primary stage of massive vaccination<ul><li> However, the causative agent of COVID-19, severe acute respiratory syndrome coronavirus&#xA0;2 (SARS-CoV-2), has been undergoing genetic evolution and mutations, and multiple virus variants continuously emerged, which challenged the pandemic control and the effectiveness of COVID-19 vaccines designed based on the early isolated SARS-CoV-2 strain (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="ir010">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>) <xref rid="b0035" ref-type="bibr"><u>[7]</u></xref>.<li> [[7]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32756549">32756549</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35831773">35831773</a> <b>(5)</b> Continent-wide evolutionary trends of emerging SARS-CoV-2 variants: dynamic profiles from Alpha to Omicron<ul><li>The <b>NextStrain</b> server was used for the phylogenetic analysis of SARS-CoV-2 [<xref ref-type="bibr" rid="CR38"><u>38</u></xref>].<li> The <b>NextStrain</b> clades were analyzed continuously in this study, providing information on the continent-wise SARS-CoV-2 evolution.<li>source of the <b>NextStrain</b> server, CoVerage, and GISAID database [<xref ref-type="bibr" rid="CR38"><u>38</u></xref>, <xref ref-type="bibr" rid="CR42"><u>42</u></xref>, <xref ref-type="bibr" rid="CR50"><u>50</u></xref>].<li>source of the <b>NextStrain</b> server, CoVerage, and GISAID database [<xref ref-type="bibr" rid="CR38"><u>38</u></xref>, <xref ref-type="bibr" rid="CR42"><u>42</u></xref>, <xref ref-type="bibr" rid="CR50"><u>50</u></xref>].<li>source of the <b>NextStrain</b> server, CoVerage, and GISAID database [<xref ref-type="bibr" rid="CR38"><u>38</u></xref>, <xref ref-type="bibr" rid="CR42"><u>42</u></xref>, <xref ref-type="bibr" rid="CR50"><u>50</u></xref>].<li> [38](Nextstrain. Nextstrain SARS-CoV-2 resources, on Nextstrain. 2021. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link> (Access date 25/01/2022).) [38](Nextstrain. Nextstrain SARS-CoV-2 resources, on Nextstrain. 2021. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link> (Access date 25/01/2022).) [42](CoVerage. CoVerage: SARS-CoV-2 lineage dynamics. 2021. <ext-link ext-link-type="uri" xlink:href="https://sarscoverage.org/">https://sarscoverage.org/</ext-link> (Access date 25&#x2013;01&#x2013;2022).) [50](WHO. Tracking SARS-CoV-2 variants. 2022. <ext-link ext-link-type="uri" xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link> (Access date 31/01/2022).) [38](Nextstrain. Nextstrain SARS-CoV-2 resources, on Nextstrain. 2021. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link> (Access date 25/01/2022).) [42](CoVerage. CoVerage: SARS-CoV-2 lineage dynamics. 2021. <ext-link ext-link-type="uri" xlink:href="https://sarscoverage.org/">https://sarscoverage.org/</ext-link> (Access date 25&#x2013;01&#x2013;2022).) [50](WHO. Tracking SARS-CoV-2 variants. 2022. <ext-link ext-link-type="uri" xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link> (Access date 31/01/2022).) [38](Nextstrain. Nextstrain SARS-CoV-2 resources, on Nextstrain. 2021. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/sars-cov-2/">https://nextstrain.org/sars-cov-2/</ext-link> (Access date 25/01/2022).) [42](CoVerage. CoVerage: SARS-CoV-2 lineage dynamics. 2021. <ext-link ext-link-type="uri" xlink:href="https://sarscoverage.org/">https://sarscoverage.org/</ext-link> (Access date 25&#x2013;01&#x2013;2022).) [50](WHO. Tracking SARS-CoV-2 variants. 2022. <ext-link ext-link-type="uri" xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link> (Access date 31/01/2022).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35862764">35862764</a> <b>(1)</b> Genetic Surveillance of SARS-CoV-2 M<sup>pro</sup> Reveals High Sequence and Structural Conservation Prior to the Introduction of Protease Inhibitor Paxlovid<ul><li>Because selection analysis tools are computationally intensive, the genome data set retrieved from GISAID was randomly downsampled to a manageable subset (~80,000) using the <b>Nextstrain</b> Augur pipeline (<xref rid="B54" ref-type="bibr"><u>54</u></xref>) with a maximum of 100,000 sequences equally sampled by geographic region and month from December 1, 2020, through January 1, 2022.<li> [54](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35855895">35855895</a> <b>(1)</b> Significance of interferon signaling based on mRNA-microRNA integration and plasma protein analyses in critically ill COVID-19 patients<ul><li> The SARS-CoV2 clades were based on <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35857510">35857510</a> <b>(2)</b> Genomic epidemiological models describe pathogen evolution across fitness valleys<ul><li> However, it is worth noting that our results show that no special population compartments (such as chronically infected individuals or other animals) are needed to explain the fact that the chain of mutations that led to Omicron went undetected for over a year, the time separating the last common ancestor between Omicron and other sequenced variants [<ext-link xlink:href="http://<b>nextstrain</b>.org/groups/neherlab/ncov/21K.Omicron" ext-link-type="uri"><b>nextstrain</b>.org/groups/neherlab/ncov/21K.Omicron</ext-link>, (<xref rid="R22" ref-type="bibr"><italic toggle="yes">22</italic></xref>, <xref rid="R63" ref-type="bibr"><italic toggle="yes">63</italic></xref>)].</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35840078">35840078</a> <b>(3)</b> Phylogenomic analysis of the monkeypox virus (MPXV) 2022 outbreak: Emergence of a novel viral lineage?<ul><li> Moreover, results from our analyses suggest a more recent emergence for the 2022 MPXV than that reported by <b>Nextstrain</b> (May 03) (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/monkeypox/hmpxv1?dmin=2021-07-03&amp;label=clade:B.1" id="intref0010">https://<b>nextstrain</b>.org/monkeypox/hmpxv1?dmin=2021-07-03&amp;label=clade:B.1</ext-link> (accessed on 01 July 2022)), which performed the dating analysis from the genomes of the MPXV Clade 3 lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35862982">35862982</a> <b>(1)</b> Performance of 22 Rapid Lateral Flow Tests for SARS-CoV-2 Antigen Detection and Influence of &#x201C;Variants of Concern&#x201D;: Implications for Clinical Use<ul><li> Lineages and clades were interpreted using the Lineage Assigner web application (Centre for Genomic Pathogen Surveillance, Cambridge, United Kingdom) and the Nextclade web application (<b>Nextstrain</b> Project, University of Basel, Basel, Switzerland) and then submitted to the GISAID international database (<ext-link xlink:href="https://www.gisaid.org" ext-link-type="uri">https://www.gisaid.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35835159">35835159</a> <b>(1)</b> Gamma variant vertically transmitted from a mild symptomatic pregnant woman associated with fatal neonatal COVID<ul><li>A representative subset of 3,874 genomes obtained from GISAID was obtained following the <b>Nextstrain</b> guidelines.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35863047">35863047</a> <b>(4)</b> Co-circulation of Multiple SARS-CoV-2 Variants of Concern in Dhaka, Bangladesh, during the Second Wave of the COVID-19 Pandemic<ul><li> Sequences were uploaded to the Nextclade web server (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) for variant calling, lineage determination, and detection of nucleotide and amino acid mutations in reference to the Wuhan-Hu-1/2019 strain (GenBank accession number <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MN908947" ext-link-type="uri" assigning-authority="insdc">MN908947</ext-link>) (<xref rid="B5" ref-type="bibr"><u>5</u></xref>).<li> The remaining 2 sequences (6.7%) were assigned to <b>Nextstrain</b> clade 20B.<li> Among Delta variants, <b>Nextstrain</b> clade 21A (Phylogenetic Assignment of Named Global Outbreaks [PANGO] lineage B.1.617.2) was the predominant clade.<li> The sequence identifier, Global Initiative on Sharing All Influenza Data (GISAID) accession number, GenBank accession number, amino acid substitutions, <b>Nextstrain</b> clade, and PANGO lineage for each sequence are listed in <xref rid="tab1" ref-type="table"><u>Table&#xA0;1</u></xref>.<li><b>Nextstrain</b> clade (WHO designation)</th>                <th rowspan="1" colspan="1">PANGO lineage (Nextclade)</th>              </tr>            </thead>            <tbody>              <tr>                <td rowspan="1" colspan="1">BSMMU_Viro_1(258).<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35816514">35816514</a> <b>(2)</b> The structural role of SARS-CoV-2 genetic background in the emergence and success of spike mutations: The case of the spike A222V mutation<ul><li> 4,993,996 SARS-CoV-2 sequences were aligned with the Nextalign package (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/nextclade/tree/master/packages/nextalign_cli" ext-link-type="uri">https://github.com/<b>nextstrain</b>/nextclade/tree/master/packages/nextalign_cli</ext-link>).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35810174">35810174</a> <b>(2)</b> Clinical characteristics and immune profile alterations in vaccinated individuals with breakthrough Delta SARS-CoV-2 infections<ul><li> thus, Delta VOC had become the dominant strain, accounting for 95% of newly reported cases by 30 Nov 2021 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891491">35891491</a> <b>(0)</b> SARS-CoV-2 Testing of Aircraft Wastewater Shows That Mandatory Tests and Vaccination Pass before Boarding Did Not Prevent Massive Importation of Omicron Variant into Europe<ul><li> Sequences from complete genomes and clade assignments were analysed using the Nextclade web tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> accessed on 14 June 2022) [<xref rid="B26-viruses-14-01511" ref-type="bibr"><u>26</u></xref>].<li> [26](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35872788">35872788</a> <b>(2)</b> Advances in COVID-19 Vaccines and New Coronavirus Variants<ul><li>There are various methods for naming and classify new variants of SARS-CoV-2, including <b>Nextstrain</b> typing, Pango genealogy nomenclature, and Global Initiative of Sharing All Influenza Data (GISAID) typing, among others (<xref rid="B9" ref-type="bibr"><u>9</u></xref>).<li><bold><b>Nextstrain</b> clade</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>GISAID clade</bold></th>              <th valign="top" align="center" rowspan="1" colspan="1"><bold>Earliest record time</bold></th>              <th valign="top" align="left" rowspan="1" colspan="1"><bold>The first detection area</bold></th>            </tr>          </thead>          <tbody>            <tr>              <td valign="top" align="left" rowspan="1" colspan="1">VOC.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35890025">35890025</a> <b>(2)</b> From Alpha to Delta&#x2014;Genetic Epidemiology of SARS-CoV-2 (hCoV-19) in Southern Poland<ul><li> The established nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages by GISAID, <b>Nextstrain</b>, and Pango are currently in use [<xref rid="B4-pathogens-11-00780" ref-type="bibr"><u>4</u></xref>,<xref rid="B5-pathogens-11-00780" ref-type="bibr"><u>5</u></xref>,<xref rid="B6-pathogens-11-00780" ref-type="bibr"><u>6</u></xref>].<li> Clade assignment, mutation calling, and sequence quality checks were performed using the tools available on the website: <uri xlink:href="https://clades.<b>nextstrain</b>.org/results">https://clades.<b>nextstrain</b>.org/results</uri> (accessed on 27 January 2022).<li> The <b>Nextstrain</b> naming system for variants was used.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35875574">35875574</a> <b>(0)</b> The Mutational Landscape of SARS-CoV-2 Variants of Concern Recovered From Egyptian Patients in 2021<ul><li>Rectangular phylogenetic tree showing the distribution of sequences among different clades (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/tree" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/tree</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35798890">35798890</a> <b>(3)</b> SARS-CoV-2 Omicron is an immune escape variant with an altered cell entry pathway<ul><li> <bold>c</bold>, Inferred evolutionary relationships of SARS-CoV-2 from <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>), with the VOCs labelled.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35818430">35818430</a> <b>(2)</b> Third booster vaccination and stopping the Omicron, a new variant of concern<ul><li>One and a half years have passed since the emergence of SARS-CoV-2 (severe acute respiratory syndrome Coronavirus 2), while according to WHO, more than 418 million infected cases and 5.85 million deaths have been recorded globally due to different variants of this virus.<xref rid="bb0045" ref-type="bibr"><sup>9</sup></xref> Omicron is the latest variant of concern (VOC, Pango lineage B.1.1.529, <b>Nextstrain</b> clade identifier 21K) that was first characterized and reported on November 2, 2021 from Botswana, South Africa (GISAID sequence accession ID: EPI_ISL_8182767).<li> Although many countries now have banned travel from South Africa, continuous monitoring of <b>Nextstrain</b> databases reflects the unbridled expansion of the VOC Omicron around the world so that it has replaced the earlier Delta variant (<xref rid="f0005" ref-type="fig">Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35895415">35895415</a> <b>(4)</b> Genomic Sequencing of Dengue Virus Strains Associated with Papua New Guinean Outbreaks in 2016 Reveals Endemic Circulation of DENV-1 and DENV-2<ul><li> All complete dengue genomes from NCBI containing a minimum set of meta-data were used for both DENV-1 and DENV-2 analyses using a modified <b>Nextstrain</b> workflow.<li> All complete dengue genomes from NCBI containing a minimum set of meta-data were used for both DENV-1 and DENV-2 analyses using a modified <b>Nextstrain</b> workflow.<xref rid="b13" ref-type="bibr"><sup>13</sup></xref> The dengue repository from <b>Nextstrain</b> (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/dengue.git" ext-link-type="uri">https://github.com/<b>nextstrain</b>/dengue.git</ext-link>) was cloned using git and the config and meta-data files modified to include the NCBI genomes, plus those generated here.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35788093">35788093</a> <b>(1)</b> Functional mutations of SARS-CoV-2: implications to viral transmission, pathogenicity and immune escape<ul><li> <b>Nextstrain</b> is a powerful tool for real-time tracking of viral evolution by providing the worldwide distributions and evolution of SARS-CoV-2.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35865920">35865920</a> <b>(9)</b> Two Years of Evolutionary Dynamics of SARS-CoV-2 in Mexico, With Emphasis on the Variants of Concern<ul><li> Maximum Likelihood phylogenetic trees were constructed using the <b>Nextstrain</b> platform.<li> Maximum Likelihood phylogenetic trees were constructed using the <b>Nextstrain</b> platform.<xref rid="fn0008" ref-type="fn"><sup>5</sup></xref> This tool provided information to differentiate between a group of sequences selected by the <b>Nextstrain</b> platform (which were used as reference sequences) and our Mexican SARS-CoV-2 genomes.<li> For tree exploration, Auspice v0.6.0<xref rid="fn0009" ref-type="fn"><sup>6</sup></xref> by <b>Nextstrain</b> was used.<li> The branches with red circles represent the 56 genomes analyzed, and branches without a circle represent the 1,794 genomes that the <b>Nextstrain</b> took as reference.<li> the isolates of the same VOC taken from other parts of the world that the <b>Nextstrain</b> took as references are shown in other colors.<li> the isolates of the same VOC taken from other parts of the world that the <b>Nextstrain</b> took as references are shown in other colors.<li> the isolates of the same VOC taken from other parts of the world that the <b>Nextstrain</b> took as references are shown in other colors.<li> The isolates of Omicron VOCs taken from other parts of the world that the <b>Nextstrain</b> took as references are shown in other colors.<li> The isolates that the <b>Nextstrain</b> took as references are shown in black.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35889075">35889075</a> <b>(8)</b> Determining the International Spread of B.1.1.523 SARS-CoV-2 Lineage with a Set of Mutations Highly Associated with Reduced Immune Neutralization<ul><li> Fasta files and metadata were extracted using the nCoV-ingest tool [<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov-ingest">https://github.com/<b>nextstrain</b>/ncov-ingest</uri> (cloned on 19 April 2021)].<li> The general sequence quality evaluation, extraction and alignment of S protein sequences, variant calling was performed using Nextclade 1.3.020 [<uri xlink:href="https://github.com/<b>nextstrain</b>/nextclade">https://github.com/<b>nextstrain</b>/nextclade</uri> (accessed on 5 May 2021)] [<xref rid="B8-microorganisms-10-01356" ref-type="bibr"><u>8</u></xref>].<li> The maximum likelihood tree was calculated using a modified version of <b>Nextstrain</b> workflow [<uri xlink:href="https://github.com/<b>nextstrain</b>/zika">https://github.com/<b>nextstrain</b>/zika</uri> (cloned at 1 May 2021)].<li> Additionally, a set of full genome sequences was composed matching the leaves of the aforementioned trimmed tree, and a corresponding maximum likelihood tree was calculated using the aforementioned <b>Nextstrain</b> workflow.<li> [8](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35790838">35790838</a> <b>(2)</b> Whole-genome analysis and mutation pattern of SARS-CoV-2 during first and second wave outbreak in Gwangju, Republic of Korea<ul><li><b>Nextstrain</b></th><th align="left">Nucleotide</th><th align="left">Amino acid</th></tr></thead><tbody><tr><td align="left">M/46.<li> According to <b>Nextstrain</b>, there are 11 major clades or lineages (19A, 19B, and 20A&#x2013;20I), which have been used to track SARS-CoV-2<sup><xref ref-type="bibr" rid="CR25"><u>25</u></xref></sup>.<li> [25](Nextclade" (What are the clades?). clades.nextstrain.org. Archived from the original on 19 January 2021. Retrieved 19 January 2021.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891460">35891460</a> <b>(1)</b> The Divergent Pattern of SARS-CoV-2 Variant Predominance and Transmission Dynamics in the Brazilian Island of Ilhabela<ul><li> The phylogenetic analysis was performed using the <b>Nextstrain</b> v3.0.3 SARS-CoV-2 workflow [<xref rid="B17-viruses-14-01481" ref-type="bibr"><u>17</u></xref>].<li> [17](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35794895">35794895</a> <b>(3)</b> Accelerated SARS-CoV-2 intrahost evolution leading to distinct genotypes during chronic infection<ul><li> <xref rid="R59" ref-type="bibr"><u>Nadeau et al., 2021</u></xref>), and the current (June 2022) <b>Nextstrain</b> estimate is approximately 9.9&#xD7;10<sup>&#x2212;4</sup> s/s/y.<li> To understand the genetic relationship of the consensus SARS-CoV-2 genomes from the chronic infection and other WHO designated SARS-CoV-2 variants (<ext-link xlink:href="https://www.who.int/activities/tracking-SARS-CoV-2-variants" ext-link-type="uri">https://www.who.int/activities/tracking-SARS-CoV-2-variants</ext-link>), we constructed phylogenetic trees with branches resolved by time and genetic divergence, i.e., number of mutations, using the <b>Nextstrain</b> pipeline (version 3.0.3) (<xref rid="R28" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> We used Nextalign (version 1.10.2) (<ext-link xlink:href="https://github.com/neherlab/nextalign" ext-link-type="uri">https://github.com/neherlab/nextalign</ext-link>) and Augur (version 11.1.2)(<xref rid="R35" ref-type="bibr"><u>Huddleston et al., 2021</u></xref>), implemented in the <b>Nextstrain</b> pipeline, to filter out the genomes based on sampling dates, construct maximum likelihood phylogenies using IQ-TREE (version 2.0.3) (<xref rid="R61" ref-type="bibr"><u>Nguyen et al., 2015</u></xref>), refine and reconstruct mutations on the phylogeny, and estimate the effective population size (N<sub>e</sub>).<li> [Nadeau et al., 2021](<name><surname>Nadeau</surname><given-names>S.A.</given-names></name>, <name><surname>Vaughan</surname><given-names>T.G.</given-names></name>, <name><surname>Scire</surname><given-names>J.</given-names></name>, <name><surname>Huisman</surname><given-names>J.S.</given-names></name>, and <name><surname>Stadler</surname><given-names>T.</given-names></name> (<year>2021</year>). <article-title>The origin and early spread of SARS-CoV-2 in Europe</article-title>. <source>Proc. Natl. Acad. Sci. U. S. A.</source><volume>118</volume>. <pub-id pub-id-type="doi">10.1073/pnas.2012008118</pub-id>.) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Huddleston et al., 2021](<name><surname>Huddleston</surname><given-names>J.</given-names></name>, <name><surname>Hadfield</surname><given-names>J.</given-names></name>, <name><surname>Sibley</surname><given-names>T.R.</given-names></name>, <name><surname>Lee</surname><given-names>J.</given-names></name>, <name><surname>Fay</surname><given-names>K.</given-names></name>, <name><surname>Ilcisin</surname><given-names>M.</given-names></name>, <name><surname>Harkins</surname><given-names>E.</given-names></name>, <name><surname>Bedford</surname><given-names>T.</given-names></name>, <name><surname>Neher</surname><given-names>R.A.</given-names></name>, and <name><surname>Hodcroft</surname><given-names>E.B.</given-names></name> (<year>2021</year>). <article-title>Augur: a bioinformatics toolkit for phylogenetic analyses of human pathogens</article-title>. <source>J Open Source Softw</source><volume>6</volume>. <pub-id pub-id-type="doi">10.21105/joss.02906</pub-id>.) [Nguyen et al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35778549">35778549</a> <b>(1)</b> Viral isolation allows characterization of early samples of SARS-CoV-2 lineage B1.1.33 with unique mutations (S: H655Y and T63N) circulating in Southern Brazil in 2020<ul><li> PANGO and <b>Nextstrain</b> lineage assignments were applied to characterize the consensus sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35642476">35642476</a> <b>(2)</b> Analyzing and Modeling the Spread of SARS-CoV-2 Omicron Lineages BA.1 and BA.2, France, September 2021&#x2013;February 2022<ul><li>The first Omicron lineage to dominate was BA.1 (B.1.1.529.1, <b>Nextstrain</b> clade 21K).<li> However, in some countries, such as Denmark, its sister lineage BA.2 (former B.1.1.529.2, <b>Nextstrain</b> clade 21L) rapidly became dominant.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35771751">35771751</a> <b>(5)</b> Genomic surveillance of SARS-CoV-2 in patients presenting neurological manifestations<ul><li> Classification into clades was performed through <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/SARS-CoV-2/" ext-link-type="uri">https://<b>nextstrain</b>.org/SARS-CoV-2/</ext-link> accessed on the 15th of March 2021), while lineage assessment was conducted using the Phylogenetic Assignment of Named Global Outbreak LINeages (PANGOLIN) tool available at <ext-link xlink:href="https://github.com/hCoV-2019/pangolin" ext-link-type="uri">https://github.com/hCoV-2019/pangolin</ext-link> [<xref rid="pone.0270024.ref032" ref-type="bibr"><u>32</u></xref>] (accessed on the 15<sup>th</sup> of March 2021).<li>Furthermore, a clade classification by <b>Nextstrain</b> was also obtained.<li> The <b>Nextstrain</b> classification into clades showed that the viral sequences from patients As and Da, together with patients Em, Fs and Gs were all classified into clade 20A, while those from patients Ba and Cm were classified into clade 20B.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35794888">35794888</a> <b>(1)</b> Remdesivir resistance in transplant recipients with persistent COVID-19<ul><li> Nextclade v2.0.0 and Auspice v2.37.3 phylogenomic visualization <b>Nextstrain</b> project) were used to examine viral genome clade assignment and mutation calling.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35794893">35794893</a> <b>(1)</b> Outbreak.info genomic reports: scalable and dynamic surveillance of SARS-CoV-2 variants and mutations<ul><li> Existing tools like <b>NextStrain</b><sup><xref rid="R22" ref-type="bibr"><u>22</u></xref></sup> and frameworks such as Microreact<sup><xref rid="R23" ref-type="bibr"><u>23</u></xref></sup> primarily rely on a phylogeny to elucidate transmission chains and monitor the evolution of the virus.<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28348833">28348833</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35764603">35764603</a> <b>(4)</b> Molecular characteristics, immune evasion, and impact of SARS-CoV-2 variants<ul><li>The phylogenetic classification is widely used as a fundamental method for emergent SARS-CoV-2 strain classification in the clade-nomenclature system (terming the major strain as clade code such as GR) by Global Initiative of Sharing All Influenza Data (GISAID)<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup> or <b>NextStrain</b><sup><xref ref-type="bibr" rid="CR29"><u>29</u></xref></sup> or Pango lineage system (terming the major strain as letter and number with point interval such as B.1.1.7) by Pango Network<sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup> (Fig.&#xA0;<xref rid="Fig1" ref-type="fig"><u>1a</u></xref>).<li> The phylogenetic analysis and sequential frequency data come from the <b>Nextstrain</b> GISAID database (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>), and figures in related (<bold>a</bold>, <bold>b</bold>) are generated under the CC-BY 4.0 permission.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33875719">33875719</a>) [29](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;</given-names>            </name>            <name>              <surname>Pybus</surname>              <given-names>OG</given-names>            </name>            <name>              <surname>Abram</surname>              <given-names>ME</given-names>            </name>            <name>              <surname>Kelly</surname>              <given-names>EJ</given-names>            </name>            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>          </person-group>          <article-title>Pango lineage designation and assignment using SARS-CoV-2 spike gene nucleotide sequences</article-title>          <source>BMC Genom.</source>          <year>2022</year>          <volume>23</volume>          <fpage>121</fpage>          <pub-id pub-id-type="doi">10.1186/s12864-022-08358-2</pub-id>        ) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891393">35891393</a> <b>(4)</b> Identifying Structural Features of Nucleotide Analogues to Overcome SARS-CoV-2 Exonuclease Activity<ul><li> Profiles are constructed by the <b>Nextstrain</b> webtool using genomic sequences from GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>; <b>Nextstrain</b> datasets: &#x201C;early-19A&#x201D; and &#x201C;22B Omicron&#x201D;; webtool accessed on 25 May 2022 at <uri xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</uri> [<xref rid="B61-viruses-14-01413" ref-type="bibr"><u>61</u></xref>]).<li> [61](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891388">35891388</a> <b>(1)</b> Sequencing during Times of Change: Evaluating SARS-CoV-2 Clinical Samples during the Transition from the Delta to Omicron Wave<ul><li> A phylogeny tree was generated using the <b>Nextstrain</b> [<xref rid="B29-viruses-14-01408" ref-type="bibr"><u>29</u></xref>] SARS-CoV-2-specific procedures and visualized in auspice (<uri xlink:href="https://auspice.us">https://auspice.us</uri>, accessed on 6 April 2022).<li>To identify the SARS-CoV-2 lineages in the 37 RNA samples, we generated consensus sequences from each sample and classified lineages based on results from Pangolin (v3.1.2) (<uri xlink:href="https://pangolin.cog-uk.io">https://pangolin.cog-uk.io</uri>, accessed on 22 March 2022) and Nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, accessed on 6 April 2022) (<xref rid="app1-viruses-14-01408" ref-type="app"><u>Supplemental Table S1</u></xref>).<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35761219">35761219</a> <b>(0)</b> Safety and efficacy of convalescent plasma for severe COVID-19: a randomized, single blinded, parallel, controlled clinical study<ul><li>Comparative genomics analyses were developed to evaluate the phylogenomic relationships for the 43 genomes obtained in this study in the context of 2,170 genomes from a representative selection of all SARS-CoV-2 lineages obtained from NextClade tool v 1.5.4 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35760080">35760080</a> <b>(5)</b> Emergence of SARS-CoV-2 Omicron lineages BA.4 and BA.5 in South Africa<ul><li>We initially analyzed genomes from South Africa against the global reference dataset using a custom pipeline based on a local version of <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>)<sup><xref ref-type="bibr" rid="CR24"><u>24</u></xref></sup>.<li> <sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup>), tree dating and ancestral state construction and annotation (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35759503">35759503</a> <b>(6)</b> Genome sequence diversity of SARS-CoV-2 obtained from clinical samples in Uzbekistan<ul><li>The Nextclade-based phylogenetic analysis of the SARS-CoV-2 genome samples sequenced in our study has revealed that the most currently distributed SARS-CoV-2 variants in our dataset belong to the tree clades: 20A (Alpha, V1), 20I (Alpha, V1), and 21A (Delta) in consonance with &#x201C;<b>Nextstrain</b>&#x201D;&#x2018;s nomenclature system [<xref rid="pone.0270314.ref016" ref-type="bibr"><u>16</u></xref>].<li> Neighbor Joining analysis and the &#x201C;<b>Nextstrain</b>&#x201D; classification has grouped 82% of our SARS-CoV-2 sample genome sequences into the Delta variant clade (n = 32; 82%), followed by Alpha (n = 4; 10.3%) and 20A (n = 3; 7.7%) clades (Figs <xref rid="pone.0270314.g006" ref-type="fig"><u>6</u></xref> and <xref rid="pone.0270314.g007" ref-type="fig"><u>7</u></xref>).<li> Recently, <b>Nextstrain</b> updated its clade designation, partitioning the 21A into the two subclades of 21J and 21I.<li> According to <b>Nextstrain</b> database the Delta 21J variant became predominant worldwide (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>, accessed in December 2021).<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35760380">35760380</a> <b>(1)</b> Classification of Omicron BA.1, BA.1.1, and BA.2 sublineages by TaqMan assay consistent with whole genome analysis data<ul><li>To determine the viral clade and lineage classifications, the consensus FASTA files were downloaded and processed through <b>Nextstrain</b> (<xref rid="bib0004" ref-type="bibr">Hadfield <italic>et&#xA0;al</italic>., 2018</xref>) and Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) (<xref rid="bib0019" ref-type="bibr">Rambaut <italic>et&#xA0;al</italic>., 2020</xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35760381">35760381</a> <b>(0)</b> Rapid and qualitative identification of SARS-CoV-2 mutations associated with variants of concern using a multiplex RT-PCR assay coupled with melting analysis<ul><li> Lineage and clade data, quality controls, mutations, missing nucleotides, and gaps were analyzed and confirmed with the support of Pangolin (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="interref0006">https://pangolin.cog-uk.io/</ext-link> accessed on March 6, 2022) and Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0007">https://clades.<b>nextstrain</b>.org/</ext-link> accessed on March 6, 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35749674">35749674</a> <b>(3)</b> Advancing Precision Vaccinology by Molecular and Genomic Surveillance of Severe Acute Respiratory Syndrome Coronavirus 2 in Germany, 2021<sup><xref rid="ciac399-FM3" ref-type="author-notes">&#xA0;</xref></sup><ul><li> The VOC topology in our phylogeny overall agrees with the one based on all global GISAID sequences, as shown in the <b>Nextstrain</b> website build (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>; accessed 17 January 2022): Omicron branching together with Alpha and B.1.177 branching together with Delta.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35752633">35752633</a> <b>(1)</b> Detection of SARS-CoV-2 intra-host recombination during superinfection with Alpha and Epsilon variants in New York City<ul><li> 10<sup>&#x2212;4</sup> substitutions/site/year under a skyline coalescent model (per <b>NextStrain</b> default parameters for SARS-CoV-2).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35750157">35750157</a> <b>(3)</b> Phylogenomic characterization and signs of microevolution in the 2022 multi-country outbreak of monkeypox virus<ul><li>Reads were human-depleted using BMTagger<sup><xref ref-type="bibr" rid="CR32"><u>32</u></xref></sup> and subsequently mapped to the reference genome MPXV-UK_P2 (<ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MT903344.1">MT903344.1</ext-link>, also being used as reference in the newest monkeypox <b>nextstrain</b> build: <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/monkeypox">https://<b>nextstrain</b>.org/monkeypox</ext-link>) using the INSaFLU pipeline<sup><xref ref-type="bibr" rid="CR33"><u>33</u></xref></sup> (<ext-link ext-link-type="uri" xlink:href="https://insaflu.insa.pt/">https://insaflu.insa.pt/</ext-link>).<li>Rapid extraction and/or visualization of variant sites from sequence alignments was performed using ReporTree (<ext-link ext-link-type="uri" xlink:href="https://github.com/insapathogenomics/ReporTree">https://github.com/insapathogenomics/ReporTree</ext-link>), snipit (<ext-link ext-link-type="uri" xlink:href="https://github.com/aineniamh/snipit">https://github.com/aineniamh/snipit</ext-link>) and NextClade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> [32](Rotmistrovsky, K. &amp; Agarwala, R. BMTagger: best match tagger for removing human reads from metagenomics datasets. <ext-link ext-link-type="uri" xlink:href="https://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/">https://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/</ext-link> (2011).) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29954441">29954441</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891359">35891359</a> <b>(0)</b> Description of a One-Year Succession of Variants of Interest and Concern of SARS-CoV-2 in Venezuela<ul><li> The variant assignment was performed with the Dragen COVID-19 program, Nextclade Web 1.14.1 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 30 April 2022), or Pangolin COVID-19 Lineage Assigner (<uri xlink:href="https://cov-lineages.org/resources/pangolin.html">https://cov-lineages.org/resources/pangolin.html</uri>, accessed on 30 April 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35753602">35753602</a> <b>(0)</b> Host genetic loci <italic>LZTFL1</italic> and <italic>CCL2</italic> associated with SARS-CoV-2 infection and severity of COVID-19<ul><li> The Nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="interref0002">https://clades.<b>nextstrain</b>.org</ext-link>) was used to identify nucleotide, and amino acid substitutions and lineages were obtained using the Pangolin online tool (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/resources/pangolin.html" id="interref0003">https://cov-lineages.org/resources/pangolin.html</ext-link>) (<xref rid="bib0034" ref-type="bibr"><u>Rambaut&#xA0;et&#xA0;al., 2020</u></xref>).<li> [Rambaut&#xA0;et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35843112">35843112</a> <b>(7)</b> Rapid repeat infection of SARS-CoV-2 by two highly distinct delta-lineage viruses<ul><li> The resulting consensus assemblies were then analyzed with Pangolin [<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="interref0008">https://github.com/cov-lineages/pangolin</ext-link>] and NextClade [<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade" id="interref0009">https://github.com/<b>nextstrain</b>/nextclade</ext-link>] to perform lineage and clade designations, respectively.<li> Titan_Augur_Run</italic> executes subcommands from the <b>NextStrain</b> Augur Toolkit [github.com/<b>nextstrain</b>/ncov] through the use of a modified version of The Broad Institute's sarscov2_<b>nextstrain</b> WDL workflow [<ext-link ext-link-type="uri" xlink:href="https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/workflows/sarscov2_<b>nextstrain</b>.wdl" id="interref0011">https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/workflows/sarscov2_<b>nextstrain</b>.wdl</ext-link>] to construct SARS-CoV-2 maximum likelihood [ML] and time trees as well as an Auspice-compatible JSON file for interactive visualization.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35729228">35729228</a> <b>(0)</b> Transmission of B.1.617.2 Delta variant between vaccinated healthcare workers<ul><li> Following this, all sequences were screened for number of gaps and N-regions using the Nextclade v1.5.4 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) server.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35726117">35726117</a> <b>(0)</b> Monkeypox: considerations for the understanding and containment of the current outbreak in non-endemic countries<ul><li> However, when MPV nucleotide sequences obtained from the USA and shared by CDC were included, the genomic epidemiology analysis and phylogenetic relatedness (<ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org/monkeypox">www.<b>nextstrain</b>.org/monkeypox</ext-link>; [<xref ref-type="bibr" rid="CR48"><u>48</u></xref>]) show that two distinguishable outbreaks appear to coexist.<li> [48](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35723286">35723286</a> <b>(0)</b> Automated Library Construction and Analysis for High-Throughput Nanopore Sequencing of SARS-CoV-2<ul><li>Nextclade, part of the ARTIC analysis pipeline (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>), classifies samples into clades and assigns a quality score of Good, Mediocre, or Bad based on several factors, including the fraction of unambiguously sequence-covered genome and the number of mutations compared to currently known clades.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35725921">35725921</a> <b>(11)</b> Drivers of adaptive evolution during chronic SARS-CoV-2 infections<ul><li>We searched for patterns of evolution across all 27 patients with chronic infection and compared this pattern to the pattern observed under (1) mostly neutral evolution, in the first approximately 9&#x2009;months of viral circulation<sup><xref ref-type="bibr" rid="CR19"><u>19</u></xref>,<xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup> (data were obtained from a sample of ~3,500 sequences generated by <b>NextStrain</b> <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link><sup><xref ref-type="bibr" rid="CR21"><u>21</u></xref></sup> (<xref rid="Sec8" ref-type="sec"><u>Methods</u></xref>)) and under (2) presumed positive selection, which occurred in the lineages leading to the five currently defined VOCs (Alpha, Beta, Gamma, Delta and Omicron) (data on lineage-defining mutations (LDMs) of VOCs were obtained from <ext-link ext-link-type="uri" xlink:href="https://covariants.org">https://covariants.org</ext-link> (Fig.<li> Data on the proportion of a given substitution in the general global viral samples were taken from <ext-link ext-link-type="uri" xlink:href="http://cov-glue-viz.cvr.gla.ac.uk/">http://cov-glue-viz.cvr.gla.ac.uk/</ext-link>, and counts of substitutions along a sampled phylogeny of approximately 3,500 sequences were taken from <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR21"><u>21</u></xref></sup> (Supplementary Table <xref rid="MOESM3" ref-type="media"><u>9</u></xref>).<li>A subset of sequences from <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR21"><u>21</u></xref></sup> (Supplementary Table <xref rid="MOESM3" ref-type="media"><u>9</u></xref>).<li>A subset of sequences of the three globally dominant VOCs (Alpha, Delta and Omicron 21K) after their emergence, from <b>NextStrain</b> (Supplementary Table <xref rid="MOESM3" ref-type="media"><u>9</u></xref>).<li>To eliminate sequencing artifacts, all events that occurred in (c) or (d) in the first 130 bases or last 50 bases of the virus genome were trimmed off, as suggested in the <b>NextStrain</b> protocol<sup><xref ref-type="bibr" rid="CR21"><u>21</u></xref></sup>.<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34292871">34292871</a>) [20](MacLean, O. A. et al. Natural selection in the evolution of SARS-CoV-2 in bats created a generalist virus and highly capable human pathogen. <italic>PLoS Biol</italic>. <bold>19</bold>, e3001115 (2021).) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35872966">35872966</a> <b>(5)</b> The past, current and future epidemiological dynamic of SARS-CoV-2<ul><li> <italic toggle="yes">Notes</italic>: (<bold>A</bold>) Phylogenetic tree of &#x223C;200,000 SARS-CoV-2 strains coloured by their <b>Nextstrain</b> clade assignment with the inset providing the presence of the Spike L452R and N501Y mutations highlighting the recurrence of independent emergence of mutations in multiple lineages.<li> (<bold>B</bold>) Distributions of distances (in number of mutations) from the root of the tree (Wuhan-Hu-1) of <b>Nextstrain</b> clades (ordered by emergence).<li> (<bold>C</bold>) Daily prevalence of each <b>NextStrain</b> clade estimated as the proportion of uploads to the GISAID genome database highlighting the dynamic of lineage replacement in SARS-CoV-2.<li> Among the two most widely used SARS-CoV-2 classification schemes are the <b>Nextstrain</b> clade naming system, which provides an up-to-date nomenclature based on phylogeny, VoC status and global and regional frequency [<xref rid="iqac003-B161" ref-type="bibr"><u>161</u></xref>] and the PANGO nomenclature scheme which relies on the phylogeny and carriage of diagnostic mutations [<xref rid="iqac003-B162" ref-type="bibr"><u>162</u></xref>].<li> Interestingly, Omicron comprises two deeply diverging lineages, BA.1 and BA.2 (<b>nextstrain</b> clades 21K and 21L).<li> [161](<person-group person-group-type="author"><string-name><surname>Bedford</surname><given-names>T</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA.</given-names></string-name></person-group><italic toggle="yes">Updated Nextstrain SARS-CoV-2 clade naming strategy</italic><year>2021</year>. <ext-link xlink:href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming" ext-link-type="uri">https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>.) [162](<person-group person-group-type="author"><string-name><surname>Rambaut</surname><given-names>A</given-names></string-name>, <string-name><surname>Holmes</surname><given-names>EC</given-names></string-name>, <string-name><surname>Hill</surname><given-names>V</given-names></string-name></person-group><etal>et al</etal><article-title>A dynamic nomenclature proposal for SARS-CoV-2 to assist genomic epidemiology</article-title>. <source>bioRxiv</source>. <year>2020</year>:2020.04.17.046086. <pub-id pub-id-type="doi">10.1101/2020.04.17.046086</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35718768">35718768</a> <b>(1)</b> Investigation of target sequencing of SARS-CoV-2 and immunogenic GWAS profiling in host cells of COVID-19 in Vietnam<ul><li> Nextclade (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org">https://docs.<b>nextstrain</b>.org</ext-link>) is a tool that helps to identifies the differences between target sequences and a reference sequence by <b>Nextstrain</b> to assign clades to these sequences [<xref ref-type="bibr" rid="CR28"><u>28</u></xref>].<li> [28](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling andquality control for viral genomes</article-title>          <source>J Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <issue>67</issue>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35652479">35652479</a> <b>(1)</b> Nosocomial COVID: the moral and clinical imperative for worldwide data collection and action<ul><li> Indeed, the COVID pandemic has revealed how well countries are able to work together in data-sharing, taking for example the global tracking of gene mutations by the regular uploading of genomic sequencing on public databases like GISAID or <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35779343">35779343</a> <b>(0)</b> Rapid and accurate identification of SARS-CoV-2 Omicron variants using droplet digital PCR (RT-ddPCR)<ul><li> Phylogenetic lineage was assigned using the PANGOLIN (Phylogenetic Assignment of Named Global Outbreak LINeages, <ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="interref0002">https://pangolin.cog-uk.io/</ext-link>) and NextClade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0003">https://clades.<b>nextstrain</b>.org/</ext-link>) tools.</ul></p>
<p><u>No PMID</u> <b>(1)</b> Early Warning Measurement of SARS-CoV-2 Variants of Concern in Wastewaters byMass Spectrometry<ul><li> The open-sourcetoolkit <b>Nextstrain</b> also confirmed that D3L_del was the dominant Alpha mutant inCanada.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35735762">35735762</a> <b>(2)</b> COVID-19 Variants in Critically Ill Patients: A Comparison of the Delta and Omicron Variant Profiles<ul><li> BA.1, BA.1.1 (or <b>Nextstrain</b> clade 21K), and BA.2 (or <b>Nextstrain</b> clade 21L) are the most common [<xref rid="B5-idr-14-00052" ref-type="bibr"><u>5</u></xref>].<li> [5](          <article-title>Statement on Omicron Sublineage, BA.2</article-title>          <comment>Available online: <ext-link xlink:href="https://www.who.int/news/item/22-02-2022-statement-on-omicron-sublineage-ba.2" ext-link-type="uri">https://www.who.int/news/item/22-02-2022-statement-on-omicron-sublineage-ba.2</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-04-02">(accessed on 2 April 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35873063">35873063</a> <b>(2)</b> Emergence of omicron variant's sublineages BA.4 and BA.5: risks assessment and possible countermeasures<ul><li> (Source (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global?branchLabel=emerging_lineage&amp;l=unrooted&amp;m=div" id="intref0025">https://<b>nextstrain</b>.org/ncov/gisaid/global?branchLabel=emerging_lineage&amp;l=unrooted&amp;m=div</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35734085">35734085</a> <b>(7)</b> Evolution of host protease interactions among SARS-CoV-2 variants of concern and related coronaviruses<ul><li> Beginning in December 2019, viral genomes have been collected globally and made available by GISAID and <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link>), of which we visualized a subsample (<xref rid="R10" ref-type="bibr"><u>Elbe and Buckland-Merrett, 2017</u></xref>; <xref rid="R16" ref-type="bibr">Hadfield <italic toggle="yes">et al</italic>., 2018</xref>).<li>A subsampled collection of 3043 samples from between Dec 2109 and May 2022 from GISAID was obtained and visualized using <b>Nextstrain</b> on June 3, 2022 (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time?c=gt-S_681&amp;l=radial" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global/all-time?c=gt-S_681&amp;l=radial</ext-link>) (<xref rid="R10" ref-type="bibr"><u>Elbe and Buckland-Merrett, 2017</u></xref>; <xref rid="R16" ref-type="bibr">Hadfield <italic toggle="yes">et al</italic>., 2018</xref>).<li> Sample clades and phylogeny were defined using default settings of <b>Nextstrain</b> and displayed in radial mode.<li> [Elbe and Buckland-Merrett, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [Elbe and Buckland-Merrett, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35747328">35747328</a> <b>(2)</b> A comprehensive account of SARS-CoV-2 genome structure, incurred mutations, lineages and COVID-19 vaccination program<ul><li> Three nomenclature systems were established by the Global initiative on sharing all influenza data (GISAID), <b>Nextstrain</b>&#xA0;and Phylogenetic Assignment of Named Global Outbreak Lineages (PANGO) for naming and tracking SARS-CoV-2 genetic lineages [<xref rid="B20" ref-type="bibr"><u>20&#x2013;23</u></xref>].<li> The variant known as 501Y.V1 belongs to <b>Nextstrain</b> clade 20B, GISAID clade GR [<xref rid="B20" ref-type="bibr"><u>20</u></xref>,<xref rid="B21" ref-type="bibr"><u>21</u></xref>]&#xA0;and lineage B.1.1.7 and is now known as Alpha strain.<li> [20&#x2013;23](<article-title>GISAID</article-title>. (<year>2020</year>). <ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>) [20](<article-title>GISAID</article-title>. (<year>2020</year>). <ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link>) [21](<collab>WHO</collab>. <article-title>Track SARS-CoV-2 variants</article-title> (<year>2021</year>). <ext-link xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" ext-link-type="uri">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35734094">35734094</a> <b>(2)</b> Global estimates of the fitness advantage of SARS-CoV-2 variant Omicron<ul><li> Known problematic samples were removed according to the <b>Nextstrain</b> &#x2018;ncov&#x2019; quality control pipeline (<xref rid="R11" ref-type="bibr">Hadfield et al.<li> <xref rid="R16" ref-type="bibr"><b>Nextstrain</b> 2022</xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35744748">35744748</a> <b>(2)</b> SARS-CoV-2: Two Years in the Pandemic: What Have We Observed from Genome Sequencing Results in Lithuania?<ul><li>Fasta files and metadata were extracted using ncov-ingest (<uri xlink:href="https://github.com/<b>nextstrain</b>/ncov-ingest">https://github.com/<b>nextstrain</b>/ncov-ingest</uri> (accessed on 19 April 2021)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35710544">35710544</a> <b>(0)</b> Tracking the molecular evolution and transmission patterns of SARS-CoV-2 lineage B.1.466.2 in Indonesia based on genomic surveillance data<ul><li> Taking Wuhan-Hu-1 (NC_045512.2) as the reference genome, single nucleotide variants (SNVs) were called and annotated using the Nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) [<xref ref-type="bibr" rid="CR31"><u>31</u></xref>].<li> [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35732100">35732100</a> <b>(5)</b> Potential linear B-cells epitope change to a helix structure in the spike of Omicron 21L or BA.2 predicts increased SARS-CoV-2 antibodies evasion<ul><li>The spike protein sequences of <b>Nextstrain</b> clades 19A (WA1)(<xref rid="bib33" ref-type="bibr"><u>Wu et al., 2020</u></xref>), clade 20A (which includes the dominant mutation D614G) besides the clade 21K (Omicron; B.1.1.529) have been retrieved from the EpiCoV database of GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="intref0010">https://www.gisaid.org/</ext-link>).<li> Three versions of Omicron variants have been found in the EpiCoV database of GISAID and <b>Nextstrain</b>, as Omicron B.1.1.529 (BA.1 or clade 21K), Omicron B.1.1.529 (BA.3 or clade 21M) and Omicron B.1.1.529 (BA.2 or clade 21L).<li>The DALI server (<ext-link ext-link-type="uri" xlink:href="http://ekhidna2.biocenter.helsinki.fi/dali/" id="intref0020">http://ekhidna2.biocenter.helsinki.fi/dali/</ext-link>) (<xref rid="bib36" ref-type="bibr"><u>Yang et al., 2014</u></xref>) was used to perform all-against-all protein databank (PDB) structure comparison for the wild type spike protein (clade 19A) versus the spike protein from variants of concern (DALI code: <b>Nextstrain</b> clade code): s001A: 20A, s002A: 19A, s003A: Beta, s004A: Iota, s005A: Delta, s006A: Gamma, s007A: Alpha, s008A: Epsilon, s009A: Lambda, s010A: Kappa, s011A: Eta, s012A: Theta, s013A: Omicron BA.1 or clade 21M, s014A: Omicron BA.3 or probable Omicron, s015A: Omicron BA.2 or clade 21L.<li><italic>Normal Mode Analysis (NMA)</italic> by DynaMut for Omicron mutations.</p></caption><alt-text id="alttext0055">Table 1</alt-text><table frame="hsides" rules="groups"><thead><tr><th>#</th><th>Spike protein domains</th><th>Omicron mutations</th><th>&#x394;&#x394;G DynaMut kcal/mol</th><th>&#x394;&#x394;S ENCoM kcal.mol<sup>&#x2212;1</sup>.K<sup>&#x2212;1</sup></th><th>If it has been found in Previous WHO or <b>Nextstrain</b> -named Variants</th><th>Interatomic interaction with surrounding Amino Acids for increasing flexibility only (&#x2212;) Lost interatomic interaction<break/>(+) Gained interatomic interaction</th></tr></thead><tbody><tr><td align="left">1.<li> Actually, there are three <b>Nextstrain</b> (or PANGO Lineage) clades represent the Omicron variant now, which are 21K (BA.1), 21M (BA.3) and 21L (BA.2) from <ext-link ext-link-type="uri" xlink:href="http://www.<b>nextstrain</b>.org" id="intref0040">www.<b>nextstrain</b>.org</ext-link>.<li> [Wu et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32015508">32015508</a>) [Yang et al., 2014](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25240437">25240437</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746787">35746787</a> <b>(0)</b> SARS-CoV-2 Delta Variant N Gene Mutations Reduce Sensitivity to the TaqPath COVID-19 Multiplex Molecular Diagnostic Assay<ul><li> Phylogenetic trees were created using the online Nexstrain analysis program (<uri xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>, accessed on 25 April 2022) [<xref rid="B19-viruses-14-01316" ref-type="bibr"><u>19</u></xref>].<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35777362">35777362</a> <b>(2)</b> Atlas of currently available human neutralizing antibodies against SARS-CoV-2 and escape by Omicron sub-variants BA.1/BA.1.1/BA.2/BA.3<ul><li> So far, Omicron has spread to all six geographic regions, surpassed Delta as the dominant VOC in many countries (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="intref0020">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>), and developed several sub-lineages (e.g., BA.1, BA.1.1, BA.2, BA.3, BA.4, BA.5, and BA.2.12.1).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35705624">35705624</a> <b>(1)</b> A simulation of geographic distribution for the emergence of consequential SARS-CoV-2 variant lineages<ul><li> However, as implied by the data obtained from the <b>Nextstrain</b> Study Group, this possibility seems less likely.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35699426">35699426</a> <b>(0)</b> Transmission networks of SARS-CoV-2 in Coastal Kenya during the first two waves: A retrospective genomic study<ul><li> Amino acid sequence changes in the Coastal Kenya genomes were investigated using the Nextclade tool v0.14.2 (<xref rid="bib12" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>): <ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link> (last accessed August 3, 2021).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35701537">35701537</a> <b>(1)</b> Harmonization and standardization of data for a pan-European cohort on SARS- CoV-2 pandemic<ul><li>Details of the 15 concepts that were submitted to NCI to request creation of codes and the newly created codes.</p></caption><table frame="hsides" rules="groups"><thead><tr><th>Concept</th><th>New NCI Code</th></tr></thead><tbody><tr><td>Insert Size</td><td>C180312</td></tr><tr><td>Read Length</td><td>C153362</td></tr><tr><td>Read Group Identifier</td><td>C180313</td></tr><tr><td>Platform Unit</td><td>C180315</td></tr><tr><td>Nucleotide Sequence Sample Name</td><td>C180316</td></tr><tr><td>Nucleotide Sequencing Kit</td><td>C180317</td></tr><tr><td>Quality Control</td><td>C15311</td></tr><tr><td>Illumina Sentrix ID</td><td>C180318</td></tr><tr><td>Illumina Sentrix Position</td><td>C180319</td></tr><tr><td>Nucleotide Sequencing Plate ID Number</td><td>C180320</td></tr><tr><td>Pangolin Lineage Nomenclature</td><td>C180321</td></tr><tr><td>Pangolin Database Version Identifier</td><td>C180322</td></tr><tr><td><b>Nextstrain</b> Clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746774">35746774</a> <b>(1)</b> Evaluation of Screening Program and Phylogenetic Analysis of SARS-CoV-2 Infections among Hospital Healthcare Workers in Li&#xE8;ge, Belgium<ul><li>To investigate the evolutionary relationships among HCW infectious cases, phylogenetic analysis were performed based on an alignment made of (i) the 45 viral genomes obtained from the infections of HCWs and sequenced in the context of the present study, (ii) all Belgian sequences available on GISAID (<uri xlink:href="www.gisaid.org">www.gisaid.org</uri> (accessed on 1 March 2022)) and collected until 1 December 2020 (<italic toggle="yes">n</italic> = 3163), as well as (iii) all genomic sequences that were used in the European <b>Nextstrain</b> [<xref rid="B44-viruses-14-01302" ref-type="bibr"><u>44</u></xref>] build dating of 1 December 2020 (<italic toggle="yes">n</italic> = 3721).<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35695776">35695776</a> <b>(1)</b> SPEAR: Systematic ProtEin AnnotatoR<ul><li> Currently, these VoCs are defined by health authorities, World Health Organization and/or by lineages, such as PANGO Lineages (<xref rid="btac391-B15" ref-type="bibr">Rambaut <italic toggle="yes">et al.</italic>, 2020</xref>) or <b>Nextstrain</b> (<xref rid="btac391-B11" ref-type="bibr">Hadfield <italic toggle="yes">et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35698014">35698014</a> <b>(1)</b> Molecular and computational analysis of spike protein of newly emerged omicron variant in comparison to the delta variant of SARS-CoV-2 in Iraq<ul><li> The phylogenetic tree was constricted based on sequence date in GISAID data using the <b>Nextstrain</b> server (Nextclade tool).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35822110">35822110</a> <b>(17)</b> Evolutionary traits of Tick-borne encephalitis virus: Pervasive non-codingRNA structure conservation and molecular epidemiology<ul><li> By considering also strains that do not cluster with the establishedsubtypes, we provide a unified picture of functional RNA conservation in TBEV 3&#x2B9;UTRs.Moreover, we contextualize genomic and epidemiological data through <b>Nextstrain</b>, providingan updated online resource to study the molecular evolution of TBEV.<li><b>Nextstrain</b> phylogeography.<li>The workflow management system Snakemake v5.32.1 (<xref rid="R41" ref-type="bibr"><u>K&#xF6;sterand Rahmann 2012</u></xref>) was used to build a pipeline for rapid and reproducibledeployment of <b>Nextstrain</b> (<xref rid="R29" ref-type="bibr">Hadfield et&#xA0;al.<li> 2014</xref>), and TreeTime v0.8.4 (<xref rid="R55" ref-type="bibr"><u>Sagulenko, Puller, and Neher 2018</u></xref>), for MSA, MLphylogeny, and timed tree inference, respectively, resulting in a JSON file for <b>Nextstrain</b>visualization.<li> TheTBEV <b>Nextstrain</b> build is available at <ext-link xlink:href="https://<b>nextstrain</b>.org/groups/ViennaRNA/TBEVnext/" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/ViennaRNA/TBEVnext/</ext-link>.<li> To this end, we compiled an interactive visualization of the global TBEV spread in<b>Nextstrain</b>, termed TBEVnext, which is publicly available at <ext-link xlink:href="https://<b>nextstrain</b>.org/groups/ViennaRNA/TBEVnext" ext-link-type="uri">https://<b>nextstrain</b>.org/groups/ViennaRNA/TBEVnext</ext-link>.<li>Panel a: Visualization of the TBEV phylodynamics in <b>Nextstrain</b> and Panel b: Unrootedphylogenetic tree, exposing the divergence of TBEV subtypes and novel lineages.<li>The <b>Nextstrain</b> framework is particularly useful for studying phylogeographic traits.<li>The TBEV <b>Nextstrain</b> data set features a timed tree, which provides estimates for the timeof the most recent common ancestor (TMRCA) associated with each clade (<xref rid="T1" ref-type="table"><u>Table&#xA0;1</u></xref>).<li>TMRCA estimates of distinct clades and associated subtypes of the TBEV phylogenyextracted from the TBEV <b>Nextstrain</b> data set (all dates are given as year CE).<li>Animation of historic dispersal events across Eurasia within <b>Nextstrain</b> suggests thatTBEV originated in Central Russia almost 2,700&#x2009;years ago, and subsequently moved botheastwards, forming the TBEV-Sib and TBEV-FE lineages, and westwards into Europe.<li>One of <b>Nextstrain</b>&#x2019;s strengths is the possibility to augment the data set with customtraits, in addition to default attributes like date and place of collection.<li> To this end, weprovide TBEVnext, a <b>Nextstrain</b> dataset that reveals spatiotemporal and phylodynamiccharacteristics of the global TBEV spread.<li> [K&#xF6;sterand Rahmann 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22908215">22908215</a>) [Sagulenko, Puller, and Neher 2018](<person-group person-group-type="author"><string-name><surname>Sagulenko</surname><given-names>P.</given-names></string-name>,<string-name><surname>Puller</surname><given-names>V.</given-names></string-name>, and<string-name><surname>Neher</surname><given-names>R.A.</given-names></string-name></person-group> (<year>2018</year>)&#x2018;<article-title>TreeTime: Maximum-Likelihood Phylodynamic Analysis</article-title>&#x2019;,<source><italic toggle="yes">Virus Evolution</italic></source>, <volume>4</volume>:<page-range>vex042</page-range>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35698660">35698660</a> <b>(3)</b> The phylodynamics of SARS-CoV-2 during 2020 in Finland<ul><li> These were the GISAID clades G, GR, L and V, of which the first corresponds with <b>Nextstrain</b> clade 20A, the second with 20B, and the two latter ones with 19&#x2009;A.<li> Out of these, <b>Nextstrain</b> clade 20C grew relatively fast into the dominant clade, starting from week 14 (beginning of April), when the number of 20C detections among the weekly cases nearly tripled from 17 (23%) to 59 (54%) from the previous week (Supplementary Fig.&#xA0;<xref rid="MOESM5" ref-type="media"><u>1</u></xref>).<li> Descendant taxa from this single Spanish introduction belong to <b>Nextstrain</b> clade 20A, which is the second most dominant clade during the first wave epidemic in Finland (Supplementary Fig.&#xA0;<xref rid="MOESM5" ref-type="media"><u>1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35644902">35644902</a> <b>(0)</b> Genomic surveillance of the Lambda SARS&#x2010;CoV&#x2010;2 variant in a global phylogenetic context<ul><li>The quality of reads was assessed and removal of reads contamination was performed using FastQC v0.11.9 (<ext-link xlink:href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" ext-link-type="uri">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</ext-link>) and Kraken2 v2.0.8, respectively.<xref rid="jmv27889-bib-0007" ref-type="bibr"><sup>7</sup></xref> The filter reads were mapped to the NCBI reference sequence (NC_045512) isolated from Wuhan using the software package BWA v.0.7.17.<xref rid="jmv27889-bib-0008" ref-type="bibr"><sup>8</sup></xref> The consensus sequence was obtained using Samtools v.1.9 and IVAR v.1.<xref rid="jmv27889-bib-0009" ref-type="bibr"><sup>9</sup></xref>, <xref rid="jmv27889-bib-0010" ref-type="bibr"><sup>10</sup></xref> Finally, the annotation of consensus sequences was performed using NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>), and lineage designation was defined using the software Pangolin v.3.1.2 (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35679181">35679181</a> <b>(1)</b> Monkeypox Outbreak &#x2014; Nine States, May 2022<ul><li>Genome sequencing results from virus recovered from the patient in Massachusetts display similarities to other published genomes in this outbreak from Europe (<b>Nextstrain</b>/monkeypox)<xref rid="FN8" ref-type="fn"><sup>&#xB6;&#xB6;</sup></xref> and are related to the 2017&#x2013;2018 monkeypox outbreak in Nigeria.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35688998">35688998</a> <b>(2)</b> Mutation induced infection waves in diseases like COVID-19<ul><li> The time point at which a variant has reached <inline-formula id="IEq123"><alternatives><tex-math id="M277">\documentclass[12pt]{minimal}				\usepackage{amsmath}				\usepackage{wasysym} 				\usepackage{amsfonts} 				\usepackage{amssymb} 				\usepackage{amsbsy}				\usepackage{mathrsfs}				\usepackage{upgreek}				\setlength{\oddsidemargin}{-69pt}				\begin{document}$$5\%$$\end{document}</tex-math><mml:math id="M278"><mml:mrow><mml:mn>5</mml:mn><mml:mo>%</mml:mo></mml:mrow></mml:math><inline-graphic xlink:href="41598_2022_13137_Article_IEq123.gif"/></alternatives></inline-formula> in the sequenced genomes reported in<sup><xref ref-type="bibr" rid="CR4"><u>4</u></xref></sup> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link>) is used as the emergence time.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746737">35746737</a> <b>(4)</b> Sequential Appearance and Isolation of a SARS-CoV-2 Recombinant between Two Major SARS-CoV-2 Variants in a Chronically Infected Immunocompromised Patient<ul><li> <b>Nextstrain</b> clade (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>; accessed on 20 October 2021) [<xref rid="B32-viruses-14-01266" ref-type="bibr"><u>32</u></xref>] 20A.EU2, or Marseille-4 [<xref rid="B27-viruses-14-01266" ref-type="bibr"><u>27</u></xref>]) and Alpha (according to the WHO denomination (<uri xlink:href="https://www.who.int/fr/activities/tracking-SARS-CoV-2-variants">https://www.who.int/fr/activities/tracking-SARS-CoV-2-variants</uri>; accessed on 20 October 2021) (a.k.a.<li> The clade was designated at the consensus level with the Nextclade online tool (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>; accessed on 18 May 2022, 2020 <b>Nextstrain</b> developers, Swiss Institute of Bioinformatics) [<xref rid="B32-viruses-14-01266" ref-type="bibr"><u>32</u></xref>,<xref rid="B47-viruses-14-01266" ref-type="bibr"><u>47</u></xref>] and an in-house Python script allowed for the detection of variants and the hybrids of variants.<li> [32](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35197938">35197938</a>) [32](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [47](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35756003">35756003</a> <b>(3)</b> SARS-CoV-2 Whole-Genome Sequencing Using Oxford Nanopore Technology for Variant Monitoring in Wastewaters<ul><li> This diversity is described by the <b>Nextstrain</b> project<sup><xref rid="footnote1" ref-type="fn"><u>1</u></xref></sup> which divides SARS-CoV-2 into 25 major clades (19A-B, 20A-20J, and 21A-M) based on high prevalence, signature mutations and geographic spread (<xref rid="B18" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The detected SNVs were filtered by position and compared with the signature mutations for alpha, beta, gamma and delta VOC described in <ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri">https://<b>nextstrain</b>.org</ext-link> and listed in <xref rid="TS1" ref-type="supplementary-material"><u>Supplementary Table 3</u></xref>.<li> [1](<a href="https://nextstrain.org">https://nextstrain.org</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35678049">35678049</a> <b>(2)</b> Therapeutics to tackle Omicron outbreak<ul><li> of them, BA.1, BA.1.1 (or <b>Nextstrain</b> clade 21K) and BA.2 (or <b>Nextstrain</b> clade 21L) are the most common.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746526">35746526</a> <b>(4)</b> Biological Properties of SARS-CoV-2 Variants: Epidemiological Impact and Clinical Consequences<ul><li> The global initiative on sharing all influenza data (GISAID), <b>Nextstrain</b>, and Pango are three scientific nomenclature systems that have been used to identify and track SARS-CoV-2 genotypes [<xref rid="B33-vaccines-10-00919" ref-type="bibr"><u>33</u></xref>].<li> This variant is also known by the names 20I/501Y.V1 in the <b>Nextstrain</b> database and GR/501Y.V1 in the GISAID database [<xref rid="B94-vaccines-10-00919" ref-type="bibr"><u>94</u></xref>].<li> is now known as the Beta variant (B.1.351) or 20H/501Y.V2 in the <b>Nextstrain</b> database and GH/501Y.V2 in the GISAID database [<xref rid="B94-vaccines-10-00919" ref-type="bibr"><u>94</u></xref>].<li> The P.1 variety is also known as 20J/501Y.V3 in the <b>Nextstrain</b> database and GR/501Y.V3 in the GISAID database.<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [94](          <person-group person-group-type="author"><name><surname>Chakraborty</surname><given-names>C.</given-names></name><name><surname>Bhattacharya</surname><given-names>M.</given-names></name><name><surname>Sharma</surname><given-names>A.R.</given-names></name></person-group>          <article-title>Present variants of concern and variants of interest of severe acute respiratory syndrome coronavirus 2: Their significant mutations in S-glycoprotein, infectivity, re-infectivity, immune escape and vaccines activity</article-title>          <source>Rev. Med. Virol.</source>          <year>2021</year>          <volume>32</volume>          <fpage>e2270</fpage>          <pub-id pub-id-type="doi">10.1002/rmv.2270</pub-id>        ) [94](          <person-group person-group-type="author"><name><surname>Chakraborty</surname><given-names>C.</given-names></name><name><surname>Bhattacharya</surname><given-names>M.</given-names></name><name><surname>Sharma</surname><given-names>A.R.</given-names></name></person-group>          <article-title>Present variants of concern and variants of interest of severe acute respiratory syndrome coronavirus 2: Their significant mutations in S-glycoprotein, infectivity, re-infectivity, immune escape and vaccines activity</article-title>          <source>Rev. Med. Virol.</source>          <year>2021</year>          <volume>32</volume>          <fpage>e2270</fpage>          <pub-id pub-id-type="doi">10.1002/rmv.2270</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35436444">35436444</a> <b>(6)</b> SARS-CoV-2 Omicron triggers cross-reactive neutralization and Fc effector functions in previously vaccinated, but not unvaccinated, individuals<ul><li><hr/></td></tr><tr><td>Genome Detective 1.132</td><td>Genome Detective</td><td><ext-link ext-link-type="uri" xlink:href="https://www.genomedetective.com" id="intref0045">https://www.genomedetective.com</ext-link></td></tr><tr><td>Geneious software</td><td>Biomatters</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com" id="intref0050">https://www.geneious.com</ext-link></td></tr><tr><td><b>NextStrain</b>.<li> (2018)</xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0055">https://github.com/<b>nextstrain</b>/ncov</ext-link>.<li> Phylogenetic clade classification of the genomes in this study consisted of analyzing them against a global reference dataset using a custom pipeline based on a local version of <b>NextStrain</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0095">https://github.com/<b>nextstrain</b>/ncov</ext-link>) (<xref rid="bib9" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35732171">35732171</a> <b>(4)</b> Cryo-EM structures of SARS-CoV-2 Omicron BA.2 spike<ul><li>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) Omicron B.1.1.529 (or <b>Nextstrain</b> 21M) variant, first detected in November 2021, includes several sub-lineages, including BA.1 (B.1.1.529.1 or <b>Nextstrain</b> clade 21K), BA.2 (B.1.1.529.2 or <b>Nextstrain</b> clade 21L), and BA.3 (B.1.1.529.3 or <b>Nextstrain</b> clade 21M) (<xref rid="fig1" ref-type="fig"><u>Figure&#xA0;1</u></xref>; <xref rid="mmc1" ref-type="supplementary-material"><u>Data S1</u></xref>; <xref rid="bib16" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>; <xref rid="bib32" ref-type="bibr"><u>Sagulenko et&#xA0;al., 2018</u></xref>).<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Sagulenko et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35729053">35729053</a> <b>(1)</b> <italic>De novo</italic> emergence of the mutation E484K in a SARS-CoV-2 B.1.1.7 lineage variant<sup><xref ref-type="fn" rid="d36e743">&#x2606;</xref></sup><ul><li> The <b>Nextstrain</b> webApp<xref rid="bib0050" ref-type="bibr"><sup>10</sup></xref> was also used for both clade assignment and visualisation of mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35690234">35690234</a> <b>(1)</b> The influence of new SARS-CoV-2 variant Omicron (B.1.1.529) on vaccine efficacy, its correlation to Delta variants: A computational approach<ul><li> From <b>Nextstrain</b> database, the PANGO lineage AY.1 contains T19R, T95I, G142D, E156-, F157-, R158G, W258L, K417&#xA0;N, L452R, T478K, D614G, P681R, D950&#xA0;N and PANGO AY.2 contains T19R, G142D, E156-, F157-, R158G, A222V, K417&#xA0;N, L452R, T478K, D614G, P681R, D950&#xA0;N and PANGO lineage AY.3 contains T19R, E156-, F157-, R158G, L452R, T478K, D614G, P681R, D950&#xA0;N mutations in S protein.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35742840">35742840</a> <b>(0)</b> Evolution of SARS-CoV-2 in Spain during the First Two Years of the Pandemic: Circulating Variants, Amino Acid Conservation, and Genetic Variability in Structural, Non-Structural, and Accessory Proteins<ul><li>For the correct assignment of the SARS-CoV-2 variants, the quality of all the downloaded sequences was checked using Nextclade v.1.11.0 (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri> (accessed on 10 May 2022)), and the sequences classified as &#x201C;bad&#x201D; quality were removed from the analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35703299">35703299</a> <b>(0)</b> Household Secondary Transmission of the Severe Acute Respiratory Syndrome Coronavirus 2 Alpha Variant From a Community Cluster in a Nursery in Japan<ul><li> A phylogenetic tree was constructed based on the whole-genome sequence (29,809 nucleotides) of virus strains obtained in Niigata Prefecture between February 1, 2021, and February 28, 2021, using Nextclade version 1.13.2 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746705">35746705</a> <b>(3)</b> Genomic Diversity of SARS-CoV-2 Omicron Variant in South American Countries<ul><li> These schemes consisted of SARS-CoV-2 lineage typing using the PANGOLIN-v1.9 tool (<uri xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</uri> (accessed on 10 May 2022)), and alignment and phylogenetic analysis using the NextClade tool v1.5.4 default command line (<uri xlink:href="https://docs.<b>nextstrain</b>.org/projects/nextclade/en/stable/user/nextclade-cli.html">https://docs.<b>nextstrain</b>.org/projects/nextclade/en/stable/user/nextclade-cli.html</uri> (accessed on 10 May 2022)), which performs sequence alignment, variant calling, clade assignment and maximum-likelihood (ML) tree placement.<li> However, this finding contrasts with that reported in <b>Nextstrain</b> (7 September 2021 in Guyana (CI = 8 June 2021 to 16 October 2021)) (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/south-america/all-time">https://<b>nextstrain</b>.org/ncov/gisaid/south-america/all-time</uri> (accessed on 10 May 2022)) and is more recent than the first case in the continent (24 November 2021) (<xref rid="app1-viruses-14-01234" ref-type="app"><u>Table S5</u></xref>) [<xref rid="B17-viruses-14-01234" ref-type="bibr"><u>17</u></xref>].<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34921354">34921354</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746710">35746710</a> <b>(3)</b> Recombination in Coronaviruses, with a Focus on SARS-CoV-2<ul><li> <b>NextStrain</b> uses a year-letter system (<uri xlink:href="https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/reference/naming_clades.html">https://docs.<b>nextstrain</b>.org/projects/ncov/en/latest/reference/naming_clades.html</uri>, accessed on 29 April 2022).<li> The first detailed case was described in February 2021 as co-infection from <b>NextStrain</b> 20A and 20B lineages, which was followed up for kinetics of relative abundance: a Portuguese patient had a prolonged viral shedding case (97 days long), first with a severe disease manifestation followed by a short second hospitalization episode, in an otherwise healthy young female [<xref rid="B43-viruses-14-01239" ref-type="bibr"><u>43</u></xref>].<li> To date, neither WHO nor <b>NextStrain</b> phylogenies have a scheming name for SARS-CoV-2 recombinants.<li> [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33540596">33540596</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35438176">35438176</a> <b>(5)</b> Molecular signature of postmortem lung tissue from COVID-19 patients suggests distinct trajectories driving mortality<ul><li> Phylogenetic analyses assigned these sequences to the GISAID clade GH or <b>Nextstrain</b> clade 20A in PANGO lineage B.1.36 (<xref rid="DMM049572F6" ref-type="fig"><u>Fig.&#xA0;6</u></xref>A) circulating in Europe, Asia and North America from September 2020 to March 2021 (<uri xlink:href="https://journals.biologists.com/dmm/article-lookup/DOI/10.1242/dmm.049572">Fig.&#xA0;S8C</uri>).<li><bold>Phylogenetic classification and analysis of recovered SARS-CoV-2 genomes.</bold> (A) Phylogenetic analysis of the three high confidence SARS-CoV-2 genomes show that they all fall within the same <b>nextstrain</b> clade 20A and are next to each other in the phylogenetic tree.<li> The SARS-CoV-2 genomes were classified using GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>), PANGO database (<uri xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</uri>) and nextclade (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>) and were placed in a phylogenetic tree created using <b>nextstrain</b> (<uri xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35733956">35733956</a> <b>(2)</b> A Rapid, Whole Genome Sequencing Assay for Detection and Characterization of Novel Coronavirus (SARS-CoV-2) Clinical Specimens Using Nanopore Sequencing<ul><li> For comparison to work by others, we also included <b>Nextstrain</b> clade and Pango lineage classifications.<li><b>Nextstrain</b> Clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35658571">35658571</a> <b>(3)</b> Trivalent NDV-HXP-S Vaccine Protects against Phylogenetically Distant SARS-CoV-2 Variants of Concern in Mice<ul><li> Graphics were adapted from the website <ext-link xlink:href="http://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri"><b>nextstrain</b>.org/ncov/gisaid/global</ext-link> (accessed 5 Februry 2022; CC-BY <xref rid="B30" ref-type="bibr"><u>30</u></xref>, <xref rid="B31" ref-type="bibr"><u>31</u></xref>).<li> This is of special interest because Omicron (B.1.1.529 or BA.1) has quickly diverged into different sublineages (BA.1.1, BA.2, BA.2.12.1, BA.3, BA.4 and BA.5) and has gained prevalence globally (<ext-link xlink:href="http://www.<b>nextstrain</b>.org/ncov" ext-link-type="uri">www.<b>nextstrain</b>.org/ncov</ext-link>).<li> Phylogenetic tree and frequency timelines were obtained from the <b>Nextstrain</b>/ncov Project within the GISAID initiative (<ext-link xlink:href="http://www.<b>nextstrain</b>.org/ncov" ext-link-type="uri">www.<b>nextstrain</b>.org/ncov</ext-link>) under CC-BY (<xref rid="B30" ref-type="bibr"><u>30</u></xref>, <xref rid="B31" ref-type="bibr"><u>31</u></xref>).<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746700">35746700</a> <b>(3)</b> National Scale Real-Time Surveillance of SARS-CoV-2 Variants Dynamics by Wastewater Monitoring in Israel<ul><li> SC-2 variant B.1.1.7 (<uri xlink:href="https://<b>nextstrain</b>.org/sars-cov-2/">https://<b>nextstrain</b>.org/sars-cov-2/</uri> (accessed on 1 May 2022), Alpha variant) was first identified in the UK during December 2020, where it became the dominant SC-2 variant within four weeks [<xref rid="B6-viruses-14-01229" ref-type="bibr"><u>6</u></xref>].<li> This mutation is almost exclusive to variant B.1.1.7 and is not currently associated with any other major SC-2 variant, as evident from the <b>NextStrain</b> global mutation analysis (<xref rid="app1-viruses-14-01229" ref-type="app"><u>Figure S1A</u></xref>).<li> [6](          <person-group person-group-type="author"><name><surname>Rambaut</surname><given-names>A.</given-names></name><name><surname>Loman</surname><given-names>N.</given-names></name><name><surname>Pybus</surname><given-names>O.</given-names></name><name><surname>Barclay</surname><given-names>W.</given-names></name><name><surname>Barrett</surname><given-names>J.</given-names></name><name><surname>Carabelli</surname><given-names>A.</given-names></name><name><surname>Connor</surname><given-names>T.</given-names></name><name><surname>Peacock</surname><given-names>T.</given-names></name><name><surname>Robertson</surname><given-names>D.L.</given-names></name><name><surname>Volz</surname><given-names>E.</given-names></name></person-group>          <article-title>Preliminary Genomic Characterisation of an Emergent SARS-CoV-2 Lineage in the UK Defined by a Novel Set of Spike Mutations</article-title>          <year>2020</year>          <comment>Available online: <ext-link xlink:href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563" ext-link-type="uri">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-05-01">(accessed on 1 May 2022)</date-in-citation>        ) </ul></p>
<p><u>No PMID</u> <b>(1)</b> Multiple drugs<ul><li> All seven haplotypes were categorised in 20B in <b>Nextstrain</b> clade or B.1.1.214 in Pangolin Lineage V.3.1.11.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35691101">35691101</a> <b>(1)</b> The role of airborne transmission in a large single source outbreak of SARS-CoV-2 in a Belgian nursing home in 2020<ul><li> SARS-CoV-2 lineage assignments were derived using the Pangolin tool as well as classification by Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="ir0005">https://clades.<b>nextstrain</b>.org</ext-link>) (<xref rid="bib13" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>, <xref rid="bib12" ref-type="bibr"><u>O&#x2019;Toole et al., 2021</u></xref>).<li>The sequences were aligned with MAFFT v7.310, and phylogenetic analysis was conducted with IQ-TREE v1.6.12, accompanying the set of 115 newly obtained genomes with a representative set of 1467 international SARS-CoV-2 sequences available on GISAID at the time of the analysis selected by using <b>NextStrain</b> (<xref rid="bib9" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>).<li> [Rambaut et al., 2020](Rambaut, A., Holmes, E., O&#x2019;Toole, &#xC1;, Hill, V., McCrone, J., Ruis, C., et al., 2020. A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology. Nat. Microbiol. [Internet], vol. 5(issue 11), pp. 1403&#x2013;7. [cited 2021 Oct 27]. Available from: &#x3008;<ext-link ext-link-type="uri" xlink:href="https://pubmed.ncbi.nlm.nih.gov/32669681/" id="ir0070">https://pubmed.ncbi.nlm.nih.gov/32669681/</ext-link>&#x3009;.) [O&#x2019;Toole et al., 2021](O&#x2019;Toole, &#xC1;, Scher, E., Underwood, A., Jackson, B., Hill, V., McCrone, J.T., et al., 2021. Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool. Virus Evol. [Internet], vol. 7(issue 2). [cited 2021 Oct 13]. Available from: &#x3008;<ext-link ext-link-type="uri" xlink:href="https://academic.oup.com/ve/article/7/2/veab064/6315289" id="ir0065">https://academic.oup.com/ve/article/7/2/veab064/6315289</ext-link>&#x3009;.) [Katoh and Standley, 2013](Katoh, K., Standley, D. , 2013. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol. Biol. Evol. [Internet], vol. 30(issue 4), pp. 772&#x2013;80. [cited 2021 Oct 27]. Available from: &#x3008;<ext-link ext-link-type="uri" xlink:href="https://pubmed.ncbi.nlm.nih.gov/23329690/" id="ir0055">https://pubmed.ncbi.nlm.nih.gov/23329690/</ext-link>&#x3009;.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35677072">35677072</a> <b>(1)</b> Genetic Diversity and Evolutionary Convergence of Cryptic SARS-CoV-2 Lineages Detected Via Wastewater Sequencing<ul><li> Surprisingly, the sewershed sequences often lacked two of the four consensus sequences that define the B.1 PANGO lineage (GISAID G clades or <b>Nextstrain</b> &#x2018;20&#x2019; clades) of SARS-COV-2 [<xref rid="R36" ref-type="bibr"><u>36</u></xref>].<li> [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35667598">35667598</a> <b>(1)</b> Covid-19, an unfinished story<ul><li> This surveillance and the nomenclature of new variants are carried out by the Technical Advisory Group on the evolution of the SARS-CoV-2 virus (TAG-VE for Technical Advisory Group on SARS-CoV-2 Virus Evolution) in liaison with the laboratory networks GISAID, <b>Nextstrain</b> and Pango.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35719386">35719386</a> <b>(3)</b> Increased Frequency of Indels in Hypervariable Regions of SARS-CoV-2 Proteins&#x2014;A Possible Signature of Adaptive Selection<ul><li>We mapped the number of indels for each genome (between one and six indels) on the <b>Nextstrain</b> time-resolved tree (<xref rid="B13" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), which includes 3475 genomes sampled between December 2019 and Dec 27<sup>th</sup>, 2021.<li>Distribution of indels in SARS-CoV-2 genomes <bold>(A)</bold> and <bold>(B)</bold> Increase in the number of deletion <bold>(D)</bold> and insertion <bold>(I)</bold> events in newly emerged lineages illustrated on <b>Nextstrain</b>&#x2019;s time-resolved phylogenetic tree, respectively <bold>(C)</bold> and <bold>(D)</bold> Percentage of PANGO lineages with and without deletion and insertion events over time, respectively <bold>(E)</bold> Distribution of the most common deletions along the SARS-CoV-2 genome (red) compared to insertions (blue) and missense substitutions (green).<li> <bold>(A)</bold> <b>Nextstrain</b> time-resolved tree, which includes 3475 genomes sampled between December 2019 and Dec 27<sup>th</sup>, 2021) displays the presence and distribution of the most frequent deletions positioned on NSP1-HVRs and NSP6-HVR as red dots <bold>(B)</bold> Top SARS-CoV-2 variants harbor the most frequent and potentially recurrent deletions of NSP1 and NSP6.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35682933">35682933</a> <b>(2)</b> SARS-CoV-2 Continuous Genetic Divergence and Changes in Multiplex RT-PCR Detection Pattern on Positive Retesting Median 150 Days after Initial Infection<ul><li> (v1.11.0, <uri xlink:href="https://github.com/<b>nextstrain</b>/nextclade">https://github.com/<b>nextstrain</b>/nextclade</uri>, assessed on 20 April 2022) as &#x201C;good&#x201D; [<xref rid="B15-ijms-23-06254" ref-type="bibr"><u>15</u></xref>] and are subjected to mutation analysis.<li> [15](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35677076">35677076</a> <b>(2)</b> RNA polymerase inaccuracy underlies SARS-CoV-2 variants and vaccine heterogeneity<ul><li> In support of this, <b>Nextstrain</b> data shows this region to be highly conserved across SARS-CoV-2 lineages and other members of the Coronaviridae family.<li> Phylogenetic trees were constructed in <b>Nextstrain</b> v2.35.0<sup><xref rid="R32" ref-type="bibr"><u>32</u></xref></sup> from genomes sequenced between Dec.<li> [32](R32) <li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35775026">35775026</a> <b>(1)</b> Epidemiological inference from pathogen genomes: A review of phylodynamic models and applications<ul><li> This is in part facilitated by shared sequence repositories such as GISAID, <b>Nextstrain</b>, and forums such as Virological.org (<xref rid="R94" ref-type="bibr"><u>Shu and McCauley 2017</u></xref>; <xref rid="R41" ref-type="bibr">Hadfield et&#xA0;al.<li> [Shu and McCauley 2017](<person-group person-group-type="author"><string-name><surname>Shu</surname><given-names>Y.</given-names></string-name>, and <string-name><surname>McCauley</surname><given-names>J.</given-names></string-name></person-group> (<year>2017</year>) &#x2018;<article-title>GISAID: Global Initiative on Sharing All Influenza Data &#x2013; from Vision to Reality</article-title>&#x2019;, <source><italic toggle="yes">Eurosurveillance</italic></source>, <volume>22</volume>: <page-range>30494</page-range>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35656100">35656100</a> <b>(3)</b> A Paradigm Shift in the Combination Changes of SARS-CoV-2 Variants and Increased Spread of Delta Variant (B.1.617.2) across the World<ul><li> The COVID-19 data were gathered from several countries and also from <b>Nextstrain</b> [<xref rid="b39-2152-5250-13-3-927" ref-type="bibr"><u>39</u></xref>-<xref rid="b42-2152-5250-13-3-927" ref-type="bibr"><u>42</u></xref>].<li>The study used servers, such as <b>Nextstrain</b> (SARS-CoV-2 resources) [<xref rid="b41-2152-5250-13-3-927" ref-type="bibr"><u>41</u></xref>, <xref rid="b42-2152-5250-13-3-927" ref-type="bibr"><u>42</u></xref>], CoVariants [<xref rid="b38-2152-5250-13-3-927" ref-type="bibr"><u>38</u></xref>,<xref rid="b43-2152-5250-13-3-927" ref-type="bibr"><u>43</u></xref>] and 2019 Novel Coronavirus Resource (2019nCoVR) (developed by CNCB) [<xref rid="b44-2152-5250-13-3-927" ref-type="bibr"><u>44</u></xref>].<li> 12A</xref>), and our study analyzed the location-wise distribution of 21A (<b>Nextstrain</b> clade) or G/478K.V1 (GISAID clade), resulting in two distinct clusters in the haplotype network.<li> [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972767">33972767</a>) [42](<collab>Nextstrain</collab> (<year>2021</year>). <article-title>Nextstrain SARS-CoV-2 resources, on Nextstrain</article-title>. <ext-link xlink:href="https://nextstrain.org/sars-cov-2/" ext-link-type="uri">https://nextstrain.org/sars-cov-2/</ext-link>. Accessed September 10 2021) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [42](<collab>Nextstrain</collab> (<year>2021</year>). <article-title>Nextstrain SARS-CoV-2 resources, on Nextstrain</article-title>. <ext-link xlink:href="https://nextstrain.org/sars-cov-2/" ext-link-type="uri">https://nextstrain.org/sars-cov-2/</ext-link>. Accessed September 10 2021) [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [43](<name><surname>Hodcroft</surname><given-names>EB</given-names></name> (<year>2021</year>). <article-title>CoVariants: SARS-CoV-2 Mutations and Variants of Interest</article-title>. <ext-link xlink:href="https://covariants.org/" ext-link-type="uri">https://covariants.org/</ext-link>. Accessed September 10 2021) [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33704069">33704069</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35298900">35298900</a> <b>(2)</b> SARS-CoV-2 incidence, transmission, and reinfection in a rural and an urban setting: results of the PHIRST-C cohort study, South Africa, 2020&#x2013;21<ul><li> and confirmed reinfection as distinct <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" id="interrefs50"><b>Nextstrain</b></ext-link> clades on sequencing or variant PCR between RT-rtPCR-positive specimens meeting the temporal criteria for possible or probable.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35650205">35650205</a> <b>(6)</b> CoV2K model, a comprehensive representation of <italic>SARS-CoV-2</italic> knowledge and data interplay<ul><li> Names are assigned by WHO<sup><xref ref-type="bibr" rid="CR2"><u>2</u></xref></sup> (using Greek letters), but they are also provided by Pangolin<sup><xref ref-type="bibr" rid="CR3"><u>3</u></xref></sup> as <italic>lineages</italic>, GISAID<sup><xref ref-type="bibr" rid="CR4"><u>4</u></xref></sup> and <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR5"><u>5</u></xref></sup> as <italic>clades</italic>, or by Public Health England (PHE<sup><xref ref-type="bibr" rid="CR6"><u>6</u></xref></sup>) based on the variant importance for genomic surveillance.<li> <b>Nextstrain</b> names are in turn reported by portals of both WHO and the Centers for Disease Control and Prevention (CDC<sup><xref ref-type="bibr" rid="CR7"><u>7</u></xref></sup>), with different notations for many lineages/variants (e.g., Alpha is mapped to 20I(V1) and to 20I/501Y.V1); GISAID also provides heterogeneous names (e.g., Alpha is currently assigned to GRY; formerly it was assigned to GR/501Y.V1).<li> At that time, WHO was reporting the most updated <b>Nextstrain</b> names<sup><xref ref-type="bibr" rid="CR8"><u>8</u></xref></sup> whereas the CDC names were outdated.<li> CoVariants<sup><xref ref-type="bibr" rid="CR10"><u>10</u></xref></sup> (employing the <b>Nextstrain</b> system) reports both amino acid changes and synonymous (nucleotide) mutations.<li> are typical of <b>Nextstrain</b>, while &#x2018;VUI-21JUN-01&#x2019; are typical of PHE).<li>We refer to the JSON at <ext-link ext-link-type="uri" xlink:href="https://github.com/hodcroftlab/covariants/blob/master/web/data/clusters.json">https://github.com/hodcroftlab/covariants/blob/master/web/data/clusters.json</ext-link>, where variants are clustered and identified with unique names assigned by <b>Nextstrain</b>/Covariants, enriched with a variant characterization in terms of amino acid changes and nucleotide mutations.<li> [2](World Health Organization. Tracking SARS-CoV-2 variants. <ext-link ext-link-type="uri" xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link>. Last accessed: March 8th, 2022.) [3](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>            <etal/>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>          <source>Nature Microbiology</source>          <year>2020</year>          <volume>5</volume>          <fpage>1403</fpage>          <lpage>1407</lpage>          <pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id>        ) [4](Shu, Y. &amp; McCauley, J. GISAID: Global initiative on sharing all influenza data&#x2013;from vision to reality. <italic>Eurosurveillance</italic><bold>22</bold> (2017).) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](Public Health England. COVID-19 variants: genomically confirmed case numbers. <ext-link ext-link-type="uri" xlink:href="https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers">https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers</ext-link>. Last accessed: March 8th, 2022.) [7](Centers for Disease Control and Prevention. SARS-CoV-2 Variant Classifications and Definitions. <ext-link ext-link-type="uri" xlink:href="https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-info.html">https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-info.html</ext-link>. Last accessed: March 8th, 2022.) [8](Bedford, T., Hodcroft, E. &amp; Neher, R. Updated Nextstrain SARS-CoV-2 clade naming strategy. <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming">https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming</ext-link>. (2021-01-06). Last accessed: March 8th, 2022.) [10](Hodcroft, E. B. CoVariants: SARS-CoV-2 Mutations and Variants of Interest. <ext-link ext-link-type="uri" xlink:href="https://covariants.org/">https://covariants.org/</ext-link>. Last accessed: March 8th, 2022.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35649381">35649381</a> <b>(1)</b> Antigenic and virological properties of an H3N2 variant that continues to dominate the 2021&#x2013;22 Northern Hemisphere influenza season<ul><li>(A) <b>Nextstrain</b> phylogenetic analysis of the HA gene of contemporary (2019&#x2013;21) H3N2 viruses.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35638826">35638826</a> <b>(3)</b> Genome Sequencing of Omicron Variants of SARS-CoV-2 Circulating in Bangladesh during the Third Wave of the COVID-19 Pandemic<ul><li> Sequence identifier (ID), GISAID accession no., GenBank accession no., SRA accession no., total genome length, <b>Nextstrain</b> clade, and Pango Lineage for each sample were listed in <xref rid="tab1" ref-type="table"><u>Table&#xA0;1</u></xref>.<li> All sequences were assigned to WHO-designated variant of concern (VOC) Omicron and <b>Nextstrain</b> clades 21K and 21L.<li><b>Nextstrain</b> clade</th>                <th rowspan="1" colspan="1">PANGO<xref rid="ngtab1-4" ref-type="table-fn"><sup><italic toggle="yes">d</italic></sup></xref> lineage</th>              </tr>            </thead>            <tbody>              <tr>                <td rowspan="1" colspan="1">BSMMU_Viro-01.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35639811">35639811</a> <b>(10)</b> Evolution, geographic spreading, and demographic distribution of Enterovirus D68<ul><li> Continuously updated data and phylogenies are available on the <b>Nextstrain</b> platform at <ext-link xlink:href="http://<b>nextstrain</b>.org/enterovirus" ext-link-type="uri"><b>nextstrain</b>.org/enterovirus</ext-link>, which provides a comprehensive phylogenetic analysis pipeline, a visualization tool and a public web-interface [<xref rid="ppat.1010515.ref034" ref-type="bibr"><u>34</u></xref>].<li> We analyzed the combined sequence data set with the <b>Nextstrain</b> pipeline [<xref rid="ppat.1010515.ref034" ref-type="bibr"><u>34</u></xref>].<li>Interactive near real-time phylogenetic analysis with <b>Nextstrain</b>.<li>To enable global genomic surveillance of EV-D68, we implemented an automated phylogenetic analysis pipeline using <b>Nextstrain</b> which generates an interactive visualization integrating a phylogeny with sample metadata such as geographic location or host age.<li> This analysis is available at <ext-link xlink:href="http://<b>nextstrain</b>.org/enterovirus" ext-link-type="uri"><b>nextstrain</b>.org/enterovirus</ext-link> and was updated whenever new data became available and we intend to keep it up-to-date going forward.<li> Instead, some subgroups sampled in Europe in 2018 had their closest relatives in 2016 in Asia (see <xref rid="ppat.1010515.g001" ref-type="fig"><u>Fig 1A</u></xref> and the <ext-link xlink:href="https://<b>nextstrain</b>.org/community/enterovirus-phylo/evd68-archive/vp1/2018-Fig1-4" ext-link-type="uri">online phylogeny</ext-link> for more detail).<li>Platforms like <b>nextstrain</b>.org allow the near real-time analysis and dissemination of sequence data integrated with relevant metadata, which we hope will be a valuable resource to the community.<li> [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Evolution, geographic spreading, and demographic distribution of Enterovirus D68<ul><li> The paper is very well written, with clear figures and very balanced explanations, and I laud the authors for making the data (as always) publicly available and accessible through <b>NextStrain</b>.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36570438">36570438</a> <b>(2)</b> Quand Omicron fait des petits<ul><li> 1.1 (ou <italic><b>Nextstrain</b> clade</italic> 21 K), BA.2 (ou <italic><b>Nextstrain</b> clade</italic> 21L).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746671">35746671</a> <b>(6)</b> Two Separate Clusters of SARS-CoV-2 Delta Variant Infections in a Group of 41 Students Travelling from India: An Illustration of the Need for Rigorous Testing and Quarantine<ul><li> Initially, we constructed a dedicated <b>Nextstrain</b> build by extending the global <b>Nextstrain</b> build (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri> (accessed on 12 July 2021)) with the genomes of 24 infected traveling students, as well as all other available Belgian Delta variant genomes, and the French and Indian Delta variant genomes until the end of April 2021 [<xref rid="B13-viruses-14-01198" ref-type="bibr"><u>13</u></xref>].<li> We used the default settings of the <b>Nextstrain</b> build process to generate a large-scale maximum-likelihood (ML) phylogeny consisting of 13,389 genomes.<li> We assumed default priors on all parameters as defined in the BEAUti interface to BEAST, but added a calibration prior to each root node corresponding to the inferred age of these nodes in our preceding <b>Nextstrain</b> analysis [<xref rid="B18-viruses-14-01198" ref-type="bibr"><u>18</u></xref>].<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29942656">29942656</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35656201">35656201</a> <b>(2)</b> System analysis of the fast global coronavirus disease 2019 (COVID-19) spread. Can we avoid future pandemics under global climate change?<ul><li> The genome of the virus is about 30,000 nucleotides with a high rate of mutations at hotspots of the genome which helped to trace the viral distribution and origin from country to country around the world (e.g [<xref rid="cit0003" ref-type="bibr"><u>3</u></xref>], based on GISAID database <ext-link xlink:href="https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/" ext-link-type="uri">https://www.gisaid.org/phylodynamics/global/<b>nextstrain</b>/</ext-link> [<xref rid="cit0004" ref-type="bibr"><u>4</u></xref>]).<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32269081">32269081</a>) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33510015">33510015</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35707526">35707526</a> <b>(0)</b> Analysis of the Transmission of SARS-CoV-2 Delta VOC in Yantai, China, August 2021<ul><li> For clade assignment and mutation calling, we imported all sequences into Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/tree" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/tree</ext-link>) and the web-application Phylogenetic Assignment of Named Global Outbreak Lineages (pangolin: <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35677393">35677393</a> <b>(1)</b> Early detection and improved genomic surveillance of SARS-CoV-2 variants from deep sequencing data<ul><li> The data generated are made available on portals such as GISAID (<xref rid="bib41" ref-type="bibr"><u>Shu and McCauley, 2017</u></xref>), <b>Nextstrain</b> (<xref rid="bib20" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>), <ext-link ext-link-type="uri" xlink:href="http://Cov-Lineages.org" id="intref0020">Cov-Lineages.org</ext-link> (<xref rid="bib32" ref-type="bibr"><u>O&#x2019;Toole et&#xA0;al., 2021a</u></xref>, <xref rid="bib33" ref-type="bibr"><u>2021b</u></xref>), NCBI SARS-CoV-2 Resources (<xref rid="bib31" ref-type="bibr"><u>National Center for Biotechnology Information, 2021</u></xref>), or the EMBL-EBI COVID-19 Data Portal (<xref rid="bib13" ref-type="bibr"><u>EMBL-EBI Covid-19 Data Portal, 2021</u></xref>).<li> [Shu and McCauley, 2017](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [O&#x2019;Toole et&#xA0;al., 2021a](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Pybus</surname>              <given-names>O.G.</given-names>            </name>            <name>              <surname>Watts</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Bogoch</surname>              <given-names>I.I.</given-names>            </name>            <name>              <surname>Khan</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Messina</surname>              <given-names>J.P.</given-names>            </name>            <name>              <surname>COVID</surname>              <given-names>T.</given-names>            </name>            <etal/>          </person-group>          <article-title>Tracking the International Spread of SARS-CoV-2 Lineages B. 1.1. 7 and b. 1.351/501y-v2</article-title>          <source>Wellcome Open Res.</source>          <volume>6</volume>          <year>2021</year>        ) [2021b](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [National Center for Biotechnology Information, 2021](          <person-group person-group-type="author">            <collab>National Center for Biotechnology Information</collab>          </person-group>          <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/sars-cov-2/" id="intref0080">https://www.ncbi.nlm.nih.gov/sars-cov-2/</ext-link>          <year>2021</year>        ) [EMBL-EBI Covid-19 Data Portal, 2021](          <person-group person-group-type="author">            <collab>EMBL-EBI Covid-19 Data Portal</collab>          </person-group>          <ext-link ext-link-type="uri" xlink:href="https://www.covid19dataportal.org/" id="intref0065">https://www.covid19dataportal.org/</ext-link>          <year>2021</year>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35637329">35637329</a> <b>(2)</b> Platform for isolation and characterization of SARS-CoV-2 variants enables rapid characterization of Omicron in Australia<ul><li> In 2021, Delta VOC globally dominated, supplanting all VOCs Alpha, Beta and Gamma (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/open/global">https://<b>nextstrain</b>.org/ncov/open/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35665019">35665019</a> <b>(0)</b> A robust, highly multiplexed mass spectrometry assay to identify SARS-CoV-2 variants<ul><li> All FASTA consensus genome sequences underwent mutation calling and phylogenetic lineage assignment by the Nextclade Web Interface (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>, last accessed 4/18/22) and the Pangolin COVID-19 Lineage Assigner (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>, last accessed 4/18/22).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35679788">35679788</a> <b>(0)</b> Real-time allelic assays of SARS-CoV-2 variants to enhance sewage surveillance<ul><li> The specific lineage of the called consensus sequence was identified by Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0002">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> The phylogenetic tree was created using Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="interref0003">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35643449">35643449</a> <b>(6)</b> ClusTRace, a bioinformatic pipeline for analyzing clusters in virus phylogenies<ul><li> Pangolin [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>], <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR21"><u>21</u></xref>], Nextclade [<xref ref-type="bibr" rid="CR22"><u>22</u></xref>], Jovian [<xref ref-type="bibr" rid="CR23"><u>23</u></xref>], HaVoC [<xref ref-type="bibr" rid="CR24"><u>24</u></xref>] and Lazypipe [<xref ref-type="bibr" rid="CR25"><u>25</u></xref>].<li> <b>Nextstrain</b> offers a comprehensive analysis, but is heavily dependent on sequence metadata and dataset pre-filtering.<li>Popular pipelines for tracking viral outbreak phylodynamics include Augur, Auspice, <b>Nextstrain</b>, Nextclade and Pangolin [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>&#x2013;<xref ref-type="bibr" rid="CR22"><u>22</u></xref>, <xref ref-type="bibr" rid="CR39"><u>39</u></xref>].<li> <b>Nextstrain</b> is an integration of several toolkits, including Augur for analysing sequence and phylogeographical data, and Auspice for visualising results.<li> Unlike <b>Nextstrain</b>/Auspice visualizations, ClusTRace focuses directly on parts of the phylogeny that are picked out by the unsupervised cluster analysis and provides no details on the likely origin of the mutations in the tree.<li> unlike <b>Nextstrain</b>/Augur, ClusTRace has no need for down sampling the sequence sets.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Sour Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [23](Zwagemaker F, Schmitz D, Nooij S, kroonma, Laros JFJ. DennisSchmitz/Jovian: 1.2.07. Zenodo; 2021.) [24](          <person-group person-group-type="author">            <name>              <surname>Nguyen</surname>              <given-names>PTT</given-names>            </name>            <name>              <surname>Plyusnin</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Sironen</surname>              <given-names>T</given-names>            </name>            <name>              <surname>Vapalahti</surname>              <given-names>O</given-names>            </name>            <name>              <surname>Kant</surname>              <given-names>R</given-names>            </name>            <name>              <surname>Smura</surname>              <given-names>T</given-names>            </name>          </person-group>          <article-title>HAVoC, a bioinformatic pipeline for reference-based consensus assembly and lineage assignment for SARS-CoV-2 sequences</article-title>          <source>BMC Bioinform</source>          <year>2021</year>          <volume>22</volume>          <fpage>1</fpage>          <lpage>8</lpage>          <pub-id pub-id-type="doi">10.1186/s12859-020-03881-z</pub-id>        ) [25](          <person-group person-group-type="author">            <name>              <surname>Plyusnin</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Kant</surname>              <given-names>R</given-names>            </name>            <name>              <surname>J&#xE4;&#xE4;skel&#xE4;inen</surname>              <given-names>AJ</given-names>            </name>            <name>              <surname>Sironen</surname>              <given-names>T</given-names>            </name>            <name>              <surname>Holm</surname>              <given-names>L</given-names>            </name>            <name>              <surname>Vapalahti</surname>              <given-names>O</given-names>            </name>            <etal/>          </person-group>          <article-title>Novel NGS pipeline for virus discovery from a wide spectrum of hosts and sample types</article-title>          <source>Virus Evolut</source>          <year>2020</year>          <volume>6</volume>          <fpage>veaa091</fpage>          <pub-id pub-id-type="doi">10.1093/ve/veaa091</pub-id>        ) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [22](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Sour Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [39](          <person-group person-group-type="author">            <name>              <surname>Huddleston</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Hadfield</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Sibley</surname>              <given-names>TR</given-names>            </name>            <name>              <surname>Lee</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Fay</surname>              <given-names>K</given-names>            </name>            <name>              <surname>Ilcisin</surname>              <given-names>M</given-names>            </name>            <etal/>          </person-group>          <article-title>Augur: a bioinformatics toolkit for phylogenetic analyses of human pathogens</article-title>          <source>J Open Sour Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>2906</fpage>          <pub-id pub-id-type="doi">10.21105/joss.02906</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35747620">35747620</a> <b>(3)</b> Heterogeneity of the Omicron variant in Senegal<ul><li> Nexclade website (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intref0015">https://clades.<b>nextstrain</b>.org/</ext-link>) and Pangolin web platform (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="intref0020">https://pangolin.cog-uk.io/</ext-link>) were used to determine the clade and the lineage of each genomic sequence respectively.<li> <bold>B.</bold> Evolution of confirmed cases over time according to the WHO data and the different lineages of Omicron variant isolated at IRESSEF from the first detection of Omicron variant to last lineage of this variant and <bold>C.</bold> Phylogenetic information of the different lineage of different variants from Senegal isolated at IRESSEF (Phylogeny reconstruction was performed using the <b>nextstrain</b>/ncov tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="intref0045">https://github.com/<b>nextstrain</b>/ncov</ext-link>) and then visualized with Auspice (<ext-link ext-link-type="uri" xlink:href="https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/" id="intref0050">https://docs.<b>nextstrain</b>.org/projects/auspice/en/stable/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35643510">35643510</a> <b>(0)</b> Identification and application of a pair of noncompeting monoclonal antibodies broadly binding to the nucleocapsid proteins of SARS-CoV-2 variants including Omicron<ul><li> The clade of the isolated Omicron variant was determined using the Nextclade web application (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) and the sequence information has not been uploaded yet.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36699387">36699387</a> <b>(2)</b> CATHAI: cluster analysis tool for healthcare-associated infections<ul><li>Several existing solutions for genomic epidemiology and real-time surveillance of human viral (<b>Nextstrain</b>) (<xref rid="vbac040-B5" ref-type="bibr">Hadfield <italic toggle="yes">et al.<li> Both <b>Nextstrain</b> and pathogenwatch provide tools for the analysis and visualization of pathogen evolution and transmission on a global scale.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35711666">35711666</a> <b>(3)</b> Perspectives: SARS-CoV-2 Spike Convergent Evolution as a Guide to Explore Adaptive Advantage<ul><li> The phylogenetic pattern of mutations at each of these AA positions was assessed visually in the representative global subsample of SARS-CoV-2 genomes (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</uri>) (<xref rid="B9" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>).<li> <b>NextStrain</b> was scanned repeatedly between March to June 2021;lineages were confirmed to show a recent localized rise to high frequency (indicative of selective advantage).<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36962352">36962352</a> <b>(1)</b> Investigation of an outbreak of COVID-19 among U.S. military personnel and beneficiaries stationed in the Republic of Korea, June-July 2021<ul><li> USFK genomes were aligned to a SARS-CoV-2 reference dataset from <b>NextSTrain</b> and to the top-matching BLAST genomes using MAFFT [<xref rid="pgph.0000236.ref009" ref-type="bibr"><u>9</u></xref>].<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35620998">35620998</a> <b>(1)</b> Detection of SARS-CoV-2 in young children attending day-care centres in Belgium, May 2020 to February 2022<ul><li> Consensus sequences were extracted, and clade and lineage assignment performed using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) and Pangolin (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>), respectively.<li><b>Nextstrain</b> clade (variant)</th>            </tr>          </thead>          <tbody>            <tr>              <td valign="bottom" align="left" style="border-left: solid 0.50pt; border-top: solid 0.50pt; border-right: solid 0.50pt; border-bottom: solid 0.50pt" scope="row" rowspan="1" colspan="1">1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35722299">35722299</a> <b>(1)</b> Characteristics of 24 SARS-CoV-2-Sequenced Reinfection Cases in a Tertiary Hospital in Spain<ul><li> (b) the number of acquired nucleotide mutations was similar to or higher than the expected to happen during the time period between episodes according to the evolution rate estimated by <b>Nextstrain</b> (2 mutations per month approximately) (<xref rid="B12" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>); or (c) the presence of new compatible COVID-19 symptoms in the second episode or a history of recent contact with an infected person.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35644262">35644262</a> <b>(2)</b> Evolving strategy for an evolving virus: Development of real-time PCR assays for detecting all SARS-CoV-2 variants of concern<ul><li> pangolin was used to assign lineages (<ext-link ext-link-type="uri" xlink:href="https://www.nature.com/articles/s41564-020-0770-5" id="ir0035">https://www.nature.com/articles/s41564&#x2013;020&#x2013;0770&#x2013;5</ext-link>) and nextclade was used to detect mutations (<ext-link ext-link-type="uri" xlink:href="http://github.com/<b>nextstrain</b>/nextclade" id="ir0040">github.com/<b>nextstrain</b>/nextclade</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35664999">35664999</a> <b>(2)</b> A repeat pattern of founder events for SARS-CoV-2 variants in Alaska<ul><li>Visualizations, statistical analyses, and <b>Nextstrain</b> build.<li> We generated a <b>Nextstrain</b> (cli v-3.2.4) build to examine the phylogenetic relationship of BA.2.3 in Alaska compared to global sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35818365">35818365</a> <b>(0)</b> Genomic Surveillance of SARS-CoV-2 in a University Community: Insights Into Tracking Variants, Transmission, and Spread of Gamma (P.1) Variant<ul><li> Customized scripts were used to calculate quality metrics, such as percentage of the genome sequenced and coverage depth, and a sequence quality check was performed (using <ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>) [<xref rid="ofac268-B33" ref-type="bibr"><u>33</u></xref>].<li> [33](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname> &#xA0;<given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname> &#xA0;<given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname> &#xA0;<given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname> &#xA0;<given-names>RA</given-names></string-name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source> &#xA0;<year>2021</year>; <volume>6</volume>:<fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35617193">35617193</a> <b>(1)</b> High prevalence of an alpha variant lineage with a premature stop codon in ORF7a in Iraq, winter 2020&#x2013;2021<ul><li> Lineage and clade assignment were identified using Pangolin (version v.3.1.7) [<xref rid="pone.0267295.ref024" ref-type="bibr"><u>24</u></xref>] and the <b>Nextstrain</b> web server [<xref rid="pone.0267295.ref025" ref-type="bibr"><u>25</u></xref>].<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35619142">35619142</a> <b>(2)</b> First description of a multisystemic and lethal SARS-CoV-2 variant of concern P.1 (Gamma) infection in a FeLV-positive cat<ul><li> The genomes were analysed using the <b>NextStrain</b> pipeline [<xref ref-type="bibr" rid="CR7"><u>7</u></xref>].<li>Time tree of the SARS-CoV-2 genomic sequences from the cat-derived sequence and global dataset set of 2,334 high-quality genomes from GISAID was inferred using <b>nextstrain</b> pipeline.<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35608305">35608305</a> <b>(1)</b> The Occurrence Birth&#x2013;Death Process for Combined-Evidence Analysis in Macroevolution and Epidemiology<ul><li> Following the standard <b>NextStrain</b> pipeline, sites 13402, 24389, and 24390 as well as 150 bases at the ends of the genomes were masked, thought to be sequencing artifacts that would bias the alignment (<xref rid="B18" ref-type="bibr">Hadfield et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35645440">35645440</a> <b>(1)</b> Doble mutaci&#xF3;n en N501Y que impide su identificaci&#xF3;n mediante un kit de RT-PCR de uso habitual en los laboratorios de Microbiolog&#xED;a Cl&#xED;nica<ul><li> Las lecturas obtenidas fueron posteriormente ensambladas mediante el <italic>pipeline</italic> de ARTIC para datos obtenidos de secuenciadores con tecnolog&#xED;a de Oxford Nanopore y se analizaron mediante la webApp de <b>Nextstrain</b><xref rid="bib0095" ref-type="bibr"><sup>9</sup></xref> comprobando su consistencia mediante el programa Integrative Genomics Viewer (IGV)<xref rid="bib0100" ref-type="bibr"><sup>10</sup></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35609706">35609706</a> <b>(10)</b> Unraveling the hurdles of a large COVID-19 epidemiological investigation by viral genomics<ul><li> Phylogenetic analysis and clade assignment was performed with <b>Nextstrain</b> pipeline<xref rid="bib0030" ref-type="bibr"><sup>30</sup></xref> version from March 27, 2021 (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov" id="interref0001">https://github.com/<b>nextstrain</b>/ncov</ext-link>), using default settings.<li>Genomic analysis of the 128 available sequences revealed four main viral genetic profiles: 95 sequences forming a sub-clade within <b>Nextstrain</b> clade 20A due to a shared mutation (G24077T, leading to the Spike D839Y amino acid change) (hereafter designated as genetic profile 1), 31 sequences belonging to <b>Nextstrain</b> clade 20B (genetic profile 2) and two genetically distant sequences belonging to <b>Nextstrain</b> clade 19A (genetic profiles 3 and 4) (<xref rid="fig0003" ref-type="fig"><u>Fig.&#xA0;3</u></xref>, Supplementary Fig.<li> In contrast, SARS-CoV-2 with the genetic profile 2 (including <b>Nextstrain</b> clade 20B root sequences and 1&#x2013;3 SNPs descendants) was highly prevalent worldwide (especially in Europe) during the study period (March-June 2021),<xref rid="bib0036" ref-type="bibr"><sup>36</sup></xref> and was introduced multiple times (and in multiple locations) in Portugal during the early epidemics.<li> Four main viral genetic profiles were identified, revealing 74% of samples belonging to a sub-clade within <b>Nextstrain</b> clade 20A due to the shared D839Y mutation in the Spike protein (profile 1), 24% of sequences belonging to <b>Nextstrain</b> clade 20B (profile 2) and two genetically distant sequences belonging to <b>Nextstrain</b> clade 19A (profiles 3 and 4).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36398130">36398130</a> <b>(1)</b> Wastewater Reveals the Spatiotemporal Spread of SARS-CoV-2in the Canton of Ticino (Switzerland) during the Onset of the COVID-19Pandemic<ul><li>Bioinformatic Analysisand <b>Nextstrain</b> SARS-CoV-2 Clade Assignment.<li> Finally,curated consensus sequences and forward or reverse chains with goodquality were used to assign a SARS-CoV-2 clade by using the Nextcladeweb tool v1.7.1 (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35605952">35605952</a> <b>(1)</b> Genomic evidence of co-identification with Omicron and Delta SARS-CoV-2 variants: a report of two cases<ul><li>Since mid-January 2022, the Omicron variant has become the dominant lineage in most countries worldwide, with a growing tendency to displace lineage B.1.617.2, designated as the Delta variant (<xref rid="bib0007" ref-type="bibr"><b>Nextstrain</b>,&#xA0;2022</xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35598737">35598737</a> <b>(3)</b> Whole-genome sequencing of SARS-CoV-2 reveals diverse mutations in circulating Alpha and Delta variants during the first, second, and third waves of COVID-19 in South Kivu, east of the Democratic Republic of the Congo<ul><li>A total of 399 SARS-CoV-2 reference sequences subsampled by <b>Nextstrain</b> were retrieved from the National Center for Biotechnology Information and included in the analysis.<li> The subsampling of the reference genomes was done using the <b>Nextstrain</b> command-line Augur filter tool based on an already subsampled dataset of 2867 that were evenly distributed across time and geography.<li> Nextclade CLI 1.4.2 tool (<xref rid="bib0001" ref-type="bibr"><u>Aksamentov&#xA0;and Neher,&#xA0;2021</u></xref>) was used to assign SARS-CoV-2 &#x201C;<b>Nextstrain</b> clades&#x201D; to the genomes based on predetermined mutations.<li> [Aksamentov&#xA0;and Neher,&#xA0;2021](Aksamentov I, Neher R, Nextclade. <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="interref0013">https://clades.nextstrain.org</ext-link>, XXX (accessed 13 September 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35611117">35611117</a> <b>(1)</b> Optimizing the Illumina COVIDSeq laboratorial and bioinformatics pipeline on thousands of samples for SARS-CoV-2 Variants of Concern tracking<ul><li> Viral sequences are not the only resources used for detection, the results from the lab need to be continuously compared to public resources such as <b>Nextstrain</b> or the COVID-miner, in order to avoid irreproducible announcements caused by technical artifacts such as inter-sample contamination <xref rid="b0095" ref-type="bibr"><u>[19]</u></xref>.<li> [[19]](F. Kreier. &#x201C;Deltacron: the story of the variant that wasn&#x2019;t.&#x201D; <italic>Nature</italic>. vol. 602, no. 7895. 19&#x2013;19. Feb. 2022. doi: 10.1038/D41586-022-00149-9.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35663867">35663867</a> <b>(3)</b> SARS-CoV-2 Within-Host and <italic>in vitro</italic> Genomic Variability and Sub-Genomic RNA Levels Indicate Differences in Viral Expression Between Clinical Cohorts and <italic>in vitro</italic> Culture<ul><li>Consensus SARS-CoV-2 genomes were processed using the <b>Nextstrain</b> Conda environment.<li> As a comparison, a representative global subset of SARS-CoV-2 genomes curated by <b>Nextstrain</b> between September 2019 and August 2021 was included in our phylogenetic analysis.<li> The SARS-CoV-2 phylogeny was constructed in a local version of <b>Nextstrain</b> (auspice.us) and shows the evolutionary relationship of SARS-CoV-2 genomes sequenced within this study (black dots) and a subsample of publicly available SARS CoV-2 genomes collected globally (<italic>n</italic> = 1,079).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35589813">35589813</a> <b>(1)</b> Broad-spectrum CRISPR-mediated inhibition of SARS-CoV-2 variants and endemic coronaviruses in vitro<ul><li> SN1 is able to perfectly target 95.5% of SARS-CoV-2 isolates, regardless of the variations in different SARS-CoV-2 clades as identified in <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup>.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35655960">35655960</a> <b>(4)</b> Evolutionary Traits and Genomic Surveillance of SARS-CoV-2 in South America<ul><li> This is evident in the nomenclature used by the main databases or organizations in charge of surveillance, like WHO, GISAID, and <b>Nextstrain</b>.<li> The two main databases for SARS-CoV-2 genomes and genomic epidemiology, GISAID and <b>Nextstrain</b>, respectively, settled on their own nomenclature [<xref rid="B18" ref-type="bibr"><u>18</u></xref>, <xref rid="B39" ref-type="bibr"><u>39</u></xref>].<li>The third most frequent variant is shared by Lambda (C.37, <b>Nextstrain</b>: 21G) and Mu (B.1.621, <b>Nextstrain</b>: 21H), which are about 3 to 5% frequent each.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35585202">35585202</a> <b>(1)</b> Co-infection with SARS-CoV-2 Omicron and Delta variants revealed by genomic surveillance<ul><li>Representative SARS-CoV-2 genomes collected between December 2020 and 31st December 2022 (<italic>n</italic>&#x2009;=&#x2009;1300, &#x2265;27,000-bp in length) were downloaded from the Global Initiative on Sharing All Influenza Data (GISAID)<sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup> EpiCoV, using a global subsampling strategy developed by <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>.<li> [26](Koehorst, J. J. et al. GISAID Global Initiative on Sharing All Influenza Data. Phylogeny of SARS-like betacoronaviruses including novel coronavirus (nCoV). <italic>Oxford</italic><bold>34</bold>, 1401&#x2013;1403 (2017).) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35677574">35677574</a> <b>(12)</b> Comparative phylodynamics reveals the evolutionary history of SARS-CoV-2 emerging variants in the Arabian Peninsula<ul><li> gisaid.org), <b>NextStrain</b> (<b>nextstrain</b>.org), and Pangolin (cov-lineages.org).<li> Next, we downloaded the metadata of the sequences used to construct the Novel Coronavirus-Global Subsampling (n-cov) tree from the <b>NextStrain</b> web interface (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>) in order the retrieve the corresponding sequences from the GISAID database (file S2 for the accession numbers).<li> The <b>NextStrain</b> sequence dataset comprised 2971 globally representative sequences collected between 26 December 2019 and 1 July 2021.<li> Here, we adapted the terminology suggested by <b>NextStrain</b> documentation (<xref rid="R27" ref-type="bibr">Hadfield et&#xA0;al.<li> sequences obtained locally), while the <b>NextStrain</b> global dataset as context sequences (i.e.<li> The <b>NextStrain</b> algorithm randomly subsamples approximately 600 sequence predefined continental regions, with approximately 400 from the previous 4&#x2009;months and approximately 200 from earlier strains, to generate representative inferences that are computationally efficient for the average users (see <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).<li> We chose the <b>NextStrain</b> genomic dataset as our context sequences because they ensure a more equitable representation of sequence distribution across different continental regions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632828">35632828</a> <b>(1)</b> Detecting Potentially Adaptive Mutations from the Parallel and Fixed Patterns in SARS-CoV-2 Evolution<ul><li>Phylogenetic trees from 27 time-points between June 2020 and July 2021 were acquired from <b>Nextstrain</b> [<xref rid="B7-viruses-14-01087" ref-type="bibr"><u>7</u></xref>], and the related sequences for each phylogenetic tree were downloaded from Global Initiative on Sharing All Influenza Data (GISAID) [<xref rid="B8-viruses-14-01087" ref-type="bibr"><u>8</u></xref>,<xref rid="B9-viruses-14-01087" ref-type="bibr"><u>9</u></xref>,<xref rid="B10-viruses-14-01087" ref-type="bibr"><u>10</u></xref>].<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34934514">34934514</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><u>No PMID</u> <b>(10)</b> The emergence of SARS-CoV-2 variants of concern in Australia by haplotype coalescence reveals a continental link to COVID-19 seasonality<ul><li> In parallel, the open-source <b>Nextstrain</b> project (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="ir0020">https://<b>nextstrain</b>.org</ext-link>) made available a continuously updated phylogenomic view of this data alongside with powerful and portable analysis and visualization tools.<li> As of February 2022, genome sequences have been clustered into 11 GISAID clades (L, S, O, V, G, GH, GR, GV, GRY, GK, and GRA) or 23 <b>Nextstrain</b> clades defined by a year-letter nomenclature.<li> 1</xref>A uses the <b>Nextstrain</b> nomenclature to pinpoint the evolutionary appearance of major VOCs along a timeline that originated when the first two clades (19A and 19B) diverged from each other.<li> <b>Nextstrain</b> clades of VOC omicron correspond to the recent GISAID clade GRA.<li> The timetree of 3347 genomes randomly sampled between December 2019 and January 2022 was obtained from <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="ir0010">https://<b>nextstrain</b>.org</ext-link>) on January 15, 2022.<li> VOC alpha (also known as <b>Nextstrain</b> clade 20I or Pangolin lineage B.1.1.7) appeared in the United Kingdom and was the first to expand quickly worldwide, probably correlated with significant increases in transmissibility and infection rates (<xref rid="bb0060" ref-type="bibr"><u>Davies et al., 2021</u></xref>).<li> [Davies et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Davies</surname>              <given-names>N.G.</given-names>            </name>            <name>              <surname>Abbott</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Barnard</surname>              <given-names>R.C.</given-names>            </name>            <name>              <surname>Jarvis</surname>              <given-names>C.I.</given-names>            </name>            <name>              <surname>Kucharski</surname>              <given-names>A.J.</given-names>            </name>            <etal/>          </person-group>          <article-title>Estimated transmissibility and impact of SARS-CoV-2 lineage B.1.1.7 in England</article-title>          <source>Science</source>          <volume>            <italic>372</italic>          </volume>          <issue>6538</issue>          <year>2021</year>          <comment>eabg3055</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35579740">35579740</a> <b>(1)</b> Bioactivity and Pharmacological Potential of Date Palm (<italic>Phoenix dactylifera</italic> L.) Against Pandemic COVID-19: a Comprehensive Review<ul><li> The nomenclature system by GISAID, <b>Nextstrain</b>, and Pango is currently used for naming and tracing SARS-CoV-2 lineage variation [<xref ref-type="bibr" rid="CR33"><u>33</u></xref>].<li> [33](Konings, F., Perkins,&#xA0;M. D., Kuhn,&#xA0;J. H., Pallen,&#xA0;M. J., Alm,&#xA0;E. J., Archer, B. N., Barakat,&#xA0;A., et al. (2021). SARS-CoV-2 Variants of interest and concern naming scheme conducive for global discourse. <italic>Nature Microbiology</italic>, <italic>6</italic>(7), 821&#x2013;823. 10.1038/s41564-021-00932-w) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35655849">35655849</a> <b>(5)</b> Screening and Whole Genome Sequencing of SARS-CoV-2 Circulating During the First Three Waves of the COVID-19 Pandemic in Libreville and the Haut-Ogoou&#xE9; Province in Gabon<ul><li>In September 2021, the four major VOCs were Alpha (Pangolin lineage B.1.1.7, V1, clade 20I), Beta (B.1.351, <b>Nextstrain</b> clade 20H), Gamma (P.1, V3, clade 20J) and Delta (B.1.617.2, clade 21A/ clade 21J), identified for the first time in the United Kingdom, South Africa, Brazil and India, respectively.<li> The analysis of the sequences obtained showed that the first wave began in March 2020 with the circulation of viruses belonging to lineage A (<b>Nextstrain</b> clade 19B, GISAID clade S) (2/64), B.1 (<b>Nextstrain</b> clade 20A/25563T/8371T, GISAID clade G) (1/64) and B.1.356 (<b>Nextstrain</b> clade 20C, GISAID clade GH) (1/64).<li> <b>Nextstrain</b> clade 20I; GISAID clade GR), while one third of the genomes belonged to lineage B.1.1.318 (56/154) (36.4%) (<xref rid="F5" ref-type="fig"><u>Figure 5</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35594871">35594871</a> <b>(1)</b> European Congress of Clinical Microbiology &amp; Infectious Diseases 2022<ul><li> Subsequent whole-genome sequencing of samples from both episodes showed that the woman had been infected with two different strains of SARS-CoV-2: the first, from the test performed on Dec 20, 2021, was from the <b>Nextstrain</b> clade 21A (delta; GISAID Accession ID: EPI_ISL_9316722), whereas the second, from the test performed on Jan 10, 2022, was from clade 21K (omicron; GISAID Accession ID: EPI_ISL_9316721).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35579429">35579429</a> <b>(0)</b> Increased Abundance of <named-content content-type="genus-species">Achromobacter xylosoxidans</named-content> and <named-content content-type="genus-species">Bacillus cereus</named-content> in Upper Airway Transcriptionally Active Microbiome of COVID-19 Mortality Patients Indicates Role of Co-Infections in Disease Severity and Outcome<ul><li> Clade assignment to all the genomes was done using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35510932">35510932</a> <b>(3)</b> Genome sequencing and analysis of genomic diversity in the locally transmitted SARS&#x2010;CoV&#x2010;2 in Pakistan<ul><li>We also determined the viral strains according to the <b>Nextstrain</b> clades (Aksamentov et&#xA0;al., <xref rid="tbed14586-bib-0001" ref-type="bibr"><u>2021</u></xref>).<li>Figure S8: Number of genomes (according to the <b>Nextstrain</b> clades) in the four waves of COVID&#x2010;19.<li>Figure S9: Phylogenetic tree representing the placement of SARS&#x2010;CoV&#x2010;2 genomes according to the <b>NextStrain</b> clades.<li> [2021](<string-name><surname>Aksamentov</surname>, <given-names>I.</given-names></string-name>, <string-name><surname>Roemer</surname>, <given-names>C.</given-names></string-name>, <string-name><surname>Hodcroft</surname>, <given-names>E. B.</given-names></string-name>, &amp; <string-name><surname>Neher</surname>, <given-names>R. A.</given-names></string-name> (<year>2021</year>). <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source>, <volume>6</volume>(<issue>67</issue>), <fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632819">35632819</a> <b>(3)</b> Coxsackievirus A6 Recombinant Subclades D3/A and D3/H Were Predominant in Hand-Foot-And-Mouth Disease Outbreaks in the Paediatric Population, France, 2010&#x2013;2018<ul><li> The geographic spread of CVA6 was explored with an automated phylogenetic approach using <b>Nextstrain</b> Augur toolkit (<uri xlink:href="https://docs.<b>nextstrain</b>.org/projects/augur/en/stable/index.html">https://docs.<b>nextstrain</b>.org/projects/augur/en/stable/index.html</uri>; accessed on 1 April 2020) [<xref rid="B34-viruses-14-01078" ref-type="bibr"><u>34</u></xref>] and was applied to the P1 genome region.<li> The <b>Nextstrain</b> pipeline generates an interactive visualization integrating phylogeny and metadata such as the geographic location of clinical samples used to obtain genomic data.<li> The geographic spread of CVA6 was reconstructed with the <b>NextStrain</b> pipeline across 17 countries, using the nucleotide sequences derived from the P1 region (<xref rid="viruses-14-01078-f005" ref-type="fig"><u>Figure 5</u></xref>).<li> [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632815">35632815</a> <b>(5)</b> Molecular Analysis of SARS-CoV-2 Lineages in Armenia<ul><li>Phylogenetic analysis, <b>Nextstrain</b> clade, and PANGO lineage identification was performed using the SARS-CoV-2 genomic epidemiology-specific pipeline implemented in <b>Nextstrain</b> 3.0.6 [<xref rid="B25-viruses-14-01074" ref-type="bibr"><u>25</u></xref>].<li> These 145 sequences represented 6 <b>Nextstrain</b> clades and 23 PANGO lineages (<xref rid="viruses-14-01074-f002" ref-type="fig"><u>Figure 2</u></xref>A,B).<li> The <b>Nextstrain</b> pipeline identified Jordan and Germany as the main transmission route for the 20I (Alpha) clade sequences.<li> We compared the <b>Nextstrain</b> clade and PANGO lineage assignment for the consensus these sequences produced (<xref rid="viruses-14-01074-t002" ref-type="table"><u>Table 2</u></xref>, <xref rid="app1-viruses-14-01074" ref-type="app"><u>Supplementary Materials</u></xref>&#xA0;<xref rid="app1-viruses-14-01074" ref-type="app"><u>Table S7</u></xref>).<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35602577">35602577</a> <b>(4)</b> Validaci&#xF3;n de una nueva estrategia para la identificaci&#xF3;n de variantes de SARS-CoV-2 mediante secuenciaci&#xF3;n del gen espiga por Sanger<ul><li> mismo, existen plataformas que permiten realizar an&#xE1;lisis gen&#xF3;micos y filogen&#xE9;ticos de las variantes, como <b>Nextstrain</b><xref rid="bib0195" ref-type="bibr"><sup>9</sup></xref>.<li> con el programa web <b>Nextstrain</b>-Nextclade Web 1.10.0, Nextclade CLI 1.7.0 (2021-12-09) (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="intr0025">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> de acuerdo con el programa en l&#xED;nea <b>Nextstrain</b>-Nextclade Web 1.10.0, Nextclade CLI 1.7.0 (2021-12-09) y PANGO Lineages (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/" id="intr0030">https://cov-lineages.org/</ext-link>).<li> los resultados de Sanger y NGS analizados en <b>Nextstrain</b> generaron la clasificaci&#xF3;n 21A y 21J, respectivamente, para la muestra 36211.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35599988">35599988</a> <b>(4)</b> Evolution of the SARS-CoV-2 pandemic in India<ul><li> Differing nomenclatures such as GISAID,<xref rid="bib3" ref-type="bibr"><sup>3</sup></xref> <b>NextStrain</b>,<xref rid="bib4" ref-type="bibr"><sup>4</sup></xref> PangoLIN,<xref rid="bib5" ref-type="bibr"><sup>5</sup></xref> based on varied approaches were proposed.<li>The <b>NextStrain</b>, which is a Year-Letter nomenclature provides large-scale diversity patterns, and clades that continue for several months and show considerable geographic spread are labeled.<li>Correspondence of different clade nomenclature systems and the marker mutations.</p></caption><alt-text id="alttext0030">Table 1</alt-text><table frame="hsides" rules="groups"><thead><tr><th>WHO name</th><th>GISAID clade</th><th><b>Nextstrain</b> clade</th><th>Pangolin lineage</th><th>Major Marker</th></tr></thead><tbody><tr><td/><td>L.<li>The variants of concern and variants of interest.</p></caption><alt-text id="alttext0035">Table 2</alt-text><table frame="hsides" rules="groups"><thead><tr><th>WHO label</th><th>Pango lineage</th><th>GISAID clade</th><th><b>Nextstrain</b> clade</th><th>Additional amino acid changes monitored</th><th>Earliest documented samples</th></tr></thead><tbody><tr><td colspan="6">Variants of concern (VoCs).</ul></p>
</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35476822">35476822</a> <b>(2)</b> Construction of a Linked Data Set of COVID-19 Knowledge Graphs: Development and Applications<ul><li>Specifically, we analyzed some data of related biodatabases (eg, National Center for Biotechnology Information [NCBI] [<xref rid="ref31" ref-type="bibr"><u>31</u></xref>], GISAID [<xref rid="ref32" ref-type="bibr"><u>32</u></xref>], China National Center for Bioinformation [<xref rid="ref33" ref-type="bibr"><u>33</u></xref>], DrugBank [<xref rid="ref34" ref-type="bibr"><u>34</u></xref>], and <b>Nextstrain</b> [<xref rid="ref35" ref-type="bibr"><u>35</u></xref>]) and related biological KGs such as SNAP [<xref rid="ref36" ref-type="bibr"><u>36</u></xref>] at Stanford University.<li> Similarly, the SARS-CoV-2 phylogeny KG was constructed by referencing <b>Nextstrain</b> metadata.<li> [31](          <source>National Center for Biotechnology Information</source>          <date-in-citation content-type="access-date">2022-02-13</date-in-citation>          <comment><ext-link xlink:href="https://www.ncbi.nlm.nih.gov" ext-link-type="uri">https://www.ncbi.nlm.nih.gov</ext-link></comment>        ) [32](          <source>GISAID</source>          <date-in-citation content-type="access-date">2022-02-13</date-in-citation>          <comment><ext-link xlink:href="https://www.gisaid.org/" ext-link-type="uri">https://www.gisaid.org/</ext-link></comment>        ) [33](          <source>Database Commons: A Catalog of Biological Databases</source>          <date-in-citation content-type="access-date">2022-02-13</date-in-citation>          <comment><ext-link xlink:href="https://ngdc.cncb.ac.cn/databasecommons/database/id/1796" ext-link-type="uri">https://ngdc.cncb.ac.cn/databasecommons/database/id/1796</ext-link></comment>        ) [34](          <source>Drug-Bank. Building the foundation for better health outcomes</source>          <date-in-citation content-type="access-date">2022-02-13</date-in-citation>          <comment><ext-link xlink:href="https://go.drugbank.com/" ext-link-type="uri">https://go.drugbank.com/</ext-link></comment>        ) [35](          <source>Nextstrain</source>          <date-in-citation content-type="access-date">2022-02-13</date-in-citation>          <comment><ext-link xlink:href="https://nextstrain.org/" ext-link-type="uri">https://nextstrain.org/</ext-link></comment>        ) [36](          <person-group person-group-type="author">            <name>              <surname>Leskovec</surname>              <given-names>J</given-names>            </name>          </person-group>          <article-title>Stanford Biomedical Network Dataset Collection</article-title>          <source>Stanford University</source>          <date-in-citation content-type="access-date">2022-02-13</date-in-citation>          <comment><ext-link xlink:href="http://snap.stanford.edu/biodata/" ext-link-type="uri">http://snap.stanford.edu/biodata/</ext-link></comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35411706">35411706</a> <b>(2)</b> Ecological super&#x2010;spreaders drive host&#x2013;range oscillations: Omicron and risk space for emerging infectious disease<ul><li> Phylogenetic and mutation analyses indicates that it diverged early in the outbreak and dissemination of SARS&#x2010;COV&#x2010;2 (<b>Nextstrain</b> phylogeny in 7 January 2022 &#x2013; Hadfield et&#xA0;al., <xref rid="tbed14557-bib-0027" ref-type="bibr"><u>2018</u></xref>).<li> (left) and the phylogeny of the putative pathway of differentiation (accumulation of mutations of the S1 protein derived from <b>Nextstrain</b> &#x2013; <ext-link xlink:href="http://www.nextrain.com" ext-link-type="uri">www.nextrain.com</ext-link>) of the variant Omicron (pink area and visual regression area in red) compared to the pathway recovered for all lineages of SARS&#x2010;COV&#x2010;2 (grey area and regression area in black).<li> [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632784">35632784</a> <b>(1)</b> Clearance of Persistent SARS-CoV-2 RNA Detection in a NF&#x3BA;B-Deficient Patient in Association with the Ingestion of Human Breast Milk: A Case Report<ul><li> We performed a maximum-likelihood phylogenetic analysis using <b>NextStrain</b> (Auspice tool) [<xref rid="B8-viruses-14-01042" ref-type="bibr"><u>8</u></xref>] and confirmed all Gamma lineage signatures (<xref rid="app1-viruses-14-01042" ref-type="app"><u>Supplementary Material Table S1</u></xref>) using the CoV-GLUE online resource.<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32278790">32278790</a>) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35551247">35551247</a> <b>(1)</b> Human-to-dog transmission of SARS-CoV-2, Colombia<ul><li> Each assembly was typed based on the PANGOLIN nomenclature lineage allocator and SNPs identification was performed using the clade assignment, mutation calling and sequence quality checking tool, NextClade, which identifies differences between sequences and a reference sequence used by <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35571557">35571557</a> <b>(1)</b> EpiBuilder: A Tool for Assembling, Searching, and Classifying B-Cell Epitopes<ul><li> The genomes obtained from GISAID were submitted to <b>NextStrain</b><sup><xref rid="bibr33-11779322221095221" ref-type="bibr"><u>33</u></xref></sup> for assembly, protein prediction, and lineage confirmation (<ext-link xlink:href="https://journals.sagepub.com/doi/suppl/10.1177/11779322221095221" ext-link-type="uri">Supplementary Figure 1</ext-link>).<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35543892">35543892</a> <b>(3)</b> Genome sequencing reveals existence of SARS-CoV-2 B.1.1.529 variant in Egypt<ul><li> (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org">https://www.gisaid.org</ext-link> ) that elucidate its genomic characteristics, and <b>Nextstrain</b>; (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org">https://<b>nextstrain</b>.org</ext-link> ) which monitors its spread within the global population.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35543509">35543509</a> <b>(2)</b> SARS-CoV-2 Variants of Concern Hijack IFITM2 for Efficient Replication in Human Lung Cells<ul><li> Since its first identification in November 2021 in South Africa, it has outcompeted the previously dominating Delta VOC at amazing speed (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35574268">35574268</a> <b>(9)</b> Reconstruction of the origin of the first major SARS-CoV-2 outbreak in Germany<ul><li> In addition, we performed a <b>Nextstrain</b> <xref rid="b0115" ref-type="bibr"><u>[23]</u></xref> phylogenetic analysis <xref rid="b0120" ref-type="bibr"><u>[24]</u></xref> and lineage annotation for integrative analysis purposes.<li> Using default parameters for subsampling and analysis, we ran the <b>Nextstrain</b> workflow by setting the geographic areas to Europe, Germany, and North-Rhine Westphalia (NRW).<li>To further investigate the origin of the virus on the state and national level, we performed a phylogenetic analysis using <b>Nextstrain</b>.<li><underline>Phylogenetic tree reconstruction using <b>Nextstrain</b> and the GISAID database- NRW and Germany.<li><underline>Phylogenetic tree reconstruction using <b>Nextstrain</b> and the GISAID database- NRW and Germany.</underline> A) <b>Nextstrain</b>-based phylogenetic tree analysis using a subsampling schema based on state NRW level from December 2020 to March 2021.<li> B) <b>Nextstrain</b>-based phylogenetic tree analysis using a subsampling schema based on national level from December 2020 to March 2021.<li> However, the lack of information on the travel history and the subsampling approach applied in the <b>Nextstrain</b> workflow limits the analysis.<li><underline>Phylogenetic tree reconstruction using <b>Nextstrain</b> and the GISAID database.<li><underline>Phylogenetic tree reconstruction using <b>Nextstrain</b> and the GISAID database.</underline> A) <b>Nextstrain</b>-based phylogenetic tree analysis using a subsampling schema based on European country level from January to March 2020.<li> [[23]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[24]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35552003">35552003</a> <b>(1)</b> Phylogenetic and amino acid signature analysis of the SARS-CoV-2s lineages circulating in Tunisia<ul><li>Clades were assigned to SARS-CoV-2 genome sequences (<italic>n</italic>&#xA0;=&#xA0;58) using the Phylogenetic Assignment of Named Global Outbreak LINeages tool (PANGOLIN), <b>NextStrain</b> and GISAID nomenclature systems.<li> Next Strain clade designations, (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/" id="ir0010">https://clades.<b>nextstrain</b>.org/</ext-link>), separate viruses that originated in China in 2019 (Clade 19) from those that subsequently introduced into Europe in early 2020 (Clade 20).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35538074">35538074</a> <b>(4)</b> Cumulative SARS-CoV-2 mutations and corresponding changes in immunity in an immunocompromised patient indicate viral evolution within the host<ul><li>All NGS sequences were shown to belong to the prevalent Pangolin lineage B.1.1 and the <b>Nextstrain</b> clade 20B.<li> Viral genome assembly and screening for distinct mutations was performed online using <b>nextstrain</b>.org (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv</ext-link>; <ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35536870">35536870</a> <b>(4)</b> The distinguishing NS5-M114V mutation in American Zika virus isolates has negligible impacts on virus replication and transmission potential<ul><li>This analysis was constructed using <b>Nextstrain</b> (<ext-link xlink:href="https://<b>nextstrain</b>.org/zika" ext-link-type="uri">https://<b>nextstrain</b>.org/zika</ext-link>, with permission from Trevor Bedford) with the use of 667 ZIKV genomes globally sampled from February 2013 to July 2019 (accurate at the time of writing) [<xref rid="pntd.0010426.ref027" ref-type="bibr"><u>27</u></xref>].<li> The geographic map data constructed using <b>Nextstrain</b> was derived from OpenStreetMap (<ext-link xlink:href="https://www.openstreetmap.org/#map=2/33.3/143.1" ext-link-type="uri">https://www.openstreetmap.org/#map=2/33.3/143.1</ext-link>), made available under the Open Database License (<ext-link xlink:href="https://www.openstreetmap.org/copyright" ext-link-type="uri">https://www.openstreetmap.org/copyright</ext-link>) and used under a CC-BY-4.0 license.<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><u>No PMID</u> <b>(0)</b> Molecular Monitoring of SARS-CoV-2 in Different Sewage Plants in Venice andthe Implications for Genetic Surveillance<ul><li> Lineage assignment was performed using the Pangolin tool<sup><xref ref-type="bibr" rid="ref50"><u>50</u></xref></sup> andconfirmed with Nextclade Web<sup><xref ref-type="bibr" rid="ref51"><u>51</u></xref></sup> (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</uri>).<li> [50](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [51](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35544862">35544862</a> <b>(1)</b> First report and genome sequencing of SARS-CoV-2 in a cat (<italic>Felis catus</italic>) in Colombia<ul><li> The genomic analysis for SARS-CoV-2 showed that it belongs to the phylogenetic <b>Nextstrain</b> clade 20A, and its lineages are currently circulating in Colombia.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35766502">35766502</a> <b>(9)</b> Genetic and Antigenic Characterization of an Expanding H3 Influenza A Virus Clade in U.S. Swine Visualized by <b>Nextstrain</b><ul><li> Concurrently, we adapted the <b>Nextstrain</b> platform (<xref rid="B28" ref-type="bibr"><u>28</u></xref>) to IAV in swine to provide near-real-time phylogenetic visualization of surveillance data for the H3 subtype.<li> The <b>Nextstrain</b> platform can be used to visualize the tanglegram in finer detail at <ext-link xlink:href="https://flu-crew.org/" ext-link-type="uri">https://flu-crew.org/</ext-link>.<li>The time-scaled tree annotated with amino acid substitutions created with <b>Nextstrain</b> showed two amino acid substitutions associated with the expansion of the major clade, N156H and K368E, dating to November 2016 (95% CI, 4 June 2016, 18 May 2017) (<xref rid="fig3" ref-type="fig"><u>Fig.&#xA0;3</u></xref>).<li>Deployment of <b>Nextstrain</b> for H3 IAV in swine.<li>The <b>Nextstrain</b> (<xref rid="B28" ref-type="bibr"><u>28</u></xref>) platform was adapted for H3 IAV in swine.<li> A separate time-scaled tree was estimated for paired NA nucleotide sequences, and the two trees were then compared using the Auspice visualization platform (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/auspice" ext-link-type="uri">https://github.com/<b>nextstrain</b>/auspice</ext-link>).<li> For the major clade, we identified an amino acid substitution at position 156 on the backbone of the phylogeny using <b>Nextstrain</b>.<li> <b>Nextstrain</b> for H3 swine IAV is hosted at <ext-link xlink:href="https://flu-crew.org/" ext-link-type="uri">https://flu-crew.org/</ext-link>.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35441368">35441368</a> <b>(3)</b> Identification of SARS&#x2010;CoV&#x2010;2 Omicron variant using spike gene target failure and genotyping assays, Gauteng, South Africa, 2021<ul><li> Sequences were then downloaded from GISAID for further analysis using the <b>Nextstrain</b> (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>) online tool for the construction of the phylogenetic trees and the Pangolin lineage assigner (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>) was used to confirm the lineages.<li> (A) Virology, NHLS sequence data are plotted against SARS&#x2010;CoV&#x2010;2 global data from the next clade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).<li> <ext-link xlink:href="https://<b>nextstrain</b>.org" ext-link-type="uri"><b>nextstrain</b>.org</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36417255">36417255</a> <b>(0)</b> Similarity Index&#x2013;Probabilistic Confidence Estimation of SARS-CoV-2 Strain Relatedness in Localized Outbreaks<ul><li> NextClade can be found at (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35467028">35467028</a> <b>(1)</b> Culture and identification of a &#x201C;Deltamicron&#x201D; SARS&#x2010;CoV&#x2010;2 in a three cases cluster in southern France<ul><li> Nucleotide and amino acid changes relative to the Wuhan&#x2010;Hu&#x2010;1 isolate genome were obtained using the Nextclade tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> Nucleotide and amino acid changes relative to the Wuhan&#x2010;Hu&#x2010;1 isolate genome were obtained using the Nextclade tool (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<xref rid="jmv27789-bib-0025" ref-type="bibr"><sup>25</sup></xref>, <xref rid="jmv27789-bib-0026" ref-type="bibr"><sup>26</sup></xref> <b>Nextstrain</b> clades and Pangolin lineages were determined using the Nextclade web application (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>)<xref rid="jmv27789-bib-0025" ref-type="bibr"><sup>25</sup></xref>, <xref rid="jmv27789-bib-0026" ref-type="bibr"><sup>26</sup></xref> and the Pangolin tool (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>),<xref rid="jmv27789-bib-0027" ref-type="bibr"><sup>27</sup></xref> respectively.<li> Adapted from screenshots of the nextclade web application output (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35523782">35523782</a> <b>(4)</b> Genetic diversity of SARS-CoV-2 infections in Ghana from 2020-2021<ul><li> Delta lineages, Alpha, B.1.1.318, and Beta, based on <b>Nextstrain</b>&#x2019;s emerging lineages designations.<li>Phylogenetic analysis of the SARS-CoV-2 genomes was performed using the <b>Nextstrain</b> pipelines (v11)<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup>.<li> Briefly, the <b>nextstrain</b> pipeline incorporates various quality control processes, including; validation of the clinical metadata, aligning sequences using nextalign to identify gaps compared to the SARS-CoV-2 reference genome (MN908947.3 and LR757998.1), and performing pangolin to assign lineages labels<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup>.<li> Molecular clock estimation for all the lineages and mutational fitness analysis for the B.1.1.318 lineage was also performed using <b>nextstrain</b> pipelines (v11).<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35508965">35508965</a> <b>(4)</b> A database for retrieving information on SARS-CoV-2 S protein mutations based on correlation network analysis<ul><li> Over a million genomes and mutational analyses of the virus are provided in public databases such as the NCBI GenBank virus database [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>] and <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR5"><u>5</u></xref>], and tens of thousands of new genomes are being uploaded daily [<xref ref-type="bibr" rid="CR6"><u>6</u></xref>].<li>There are several public databases that contain information on the mutations of the virus, such as GenBank, <b>Nextstrain</b>, and SARS-CoV-2 MAT [<xref ref-type="bibr" rid="CR8"><u>8</u></xref>].<li> GenBank and <b>Nextstrain</b> associate variants of the virus with the geographical locations from which specimens infected by the variants were collected.<li>GenBank [<xref ref-type="bibr" rid="CR4"><u>4</u></xref>], <b>Nextstrain</b> [<xref ref-type="bibr" rid="CR5"><u>5</u></xref>], and SARS-CoV-2 MAT [<xref ref-type="bibr" rid="CR8"><u>8</u></xref>] also provide information on mutations of the virus and refined phylogenetic trees to help trace the lineage of the mutations and the variants.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33196830">33196830</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33893460">33893460</a>) [8](McBroome J, Thornlow B, Hinrichs AS, Kramer A, De Maio N, Goldman N, Haussler D, Corbett-Detig R, Turakhia Y. A Daily-udated Database and tools for comprehensive SARSCoV-2 mutation-annotated trees.&#xA0;Mol Biol Evol. 2021;38:5819-24.) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33196830">33196830</a>) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [8](McBroome J, Thornlow B, Hinrichs AS, Kramer A, De Maio N, Goldman N, Haussler D, Corbett-Detig R, Turakhia Y. A Daily-udated Database and tools for comprehensive SARSCoV-2 mutation-annotated trees.&#xA0;Mol Biol Evol. 2021;38:5819-24.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35320700">35320700</a> <b>(0)</b> Rapid Replacement of SARS-CoV-2 Variants by Delta and Subsequent Arrival of Omicron, Uganda, 2021<ul><li> We performed mutation calling by using Nextclade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org" ext-link-type="uri">https://clades.<b>nextstrain</b>.org</ext-link>), followed by SARS-CoV-2 lineage analysis with Pangolin (<ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35572963">35572963</a> <b>(0)</b> SARS-CoV-2 Molecular Epidemiology Can Be Enhanced by Occupational Health: The Experience of Monitoring Variants of Concern in Workplaces in Rio de Janeiro, Brazil<ul><li> Aligned FASTA files were also assigned to variants and quality checked by NextClade (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>, version 1.10.0) (<xref rid="B22" ref-type="bibr"><u>22</u></xref>).<li> [22](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J. Open Source Softw.</source> (<year>2021</year>) <volume>6</volume>:<fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35601074">35601074</a> <b>(1)</b> Clinical challenges of SARS-CoV-2 variants (Review)<ul><li>The Global Initiative on Sharing All Influenza Data, <b>Nextstrain</b>, and Pango systems have specified SARS-CoV-2 lineage nomenclature.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35572594">35572594</a> <b>(7)</b> Analysis of the Protective Efficacy of Approved COVID-19 Vaccines Against Various Mutants<ul><li> The <b>Nextstrain</b> platform provides a time-calibrated worldwide SARS-CoV-2 phylogeny (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri>).<li> On May 31, 2021, B.1.351 was renamed the Beta variant virus (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri>).<li> P.1 was named the Gamma Variant Virus on 31 May 2021 (<uri xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global">https://<b>nextstrain</b>.org/ncov/gisaid/global</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35570946">35570946</a> <b>(1)</b> Impact of SARS-CoV-2 Mutations on PCR Assay Sequence Alignment<ul><li>The Global Initiative on Sharing Avian Influenza Data (GISAID) organization and <b>Nextstrain</b> project provide genomic data, metadata, and phylodynamic analysis for monitoring ongoing outbreaks (<xref rid="B11" ref-type="bibr"><u>11</u></xref>, <xref rid="B12" ref-type="bibr"><u>12</u></xref>).<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35482816">35482816</a> <b>(2)</b> A combination of potently neutralizing monoclonal antibodies isolated from an Indian convalescent donor protects against the SARS-CoV-2 Delta variant<ul><li> Among all of the SARS-CoV-2 VOCs, the Delta variant (B.1.617.2), first detected in India in late 2020 in the state of Maharashtra [<xref rid="ppat.1010465.ref001" ref-type="bibr"><u>1</u></xref>,<xref rid="ppat.1010465.ref002" ref-type="bibr"><u>2</u></xref>], became the globally dominant circulating strain during the period from July 2021 to December 2021 (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).<li> [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34488225">34488225</a>) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34648303">34648303</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35572563">35572563</a> <b>(1)</b> <italic>In Silico</italic> Analysis Predicts a Limited Impact of SARS-CoV-2 Variants on CD8 T Cell Recognition<ul><li> Visualization generated using the <b>Nextstrain</b> platform (<xref rid="B22" ref-type="bibr"><u>22</u></xref>).<li> [22](<person-group person-group-type="author"><name><surname>Hadfield</surname><given-names>J</given-names></name><name><surname>Megill</surname><given-names>C</given-names></name><name><surname>Bell</surname><given-names>SM</given-names></name><name><surname>Huddleston</surname><given-names>J</given-names></name><name><surname>Potter</surname><given-names>B</given-names></name><name><surname>Callender</surname><given-names>C</given-names></name><etal/></person-group>. <article-title>Nextstrain: Real-Time Tracking of Pathogen Evolution</article-title>. <source>Bioinformatics</source> (<year>2018</year>) <volume>34</volume>:<page-range>4121&#x2013;3</page-range>. doi: <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35528635">35528635</a> <b>(3)</b> First Identification of Reinfection by a Genetically Different Variant of SARS-CoV-2 in a Homeless Person from the Metropolitan Area of Santiago, Chile<ul><li> In addition, the <b>Nextstrain</b> clade assignment was performed using NextClade version 1.9.0 to compare and visualize our samples globally.<li> The phylogenetic analysis showed that 0606-201 and 0308-063 samples were members of <b>Nextstrain</b> clades 20B and 20A, respectively (<xref rid="fig1" ref-type="fig"><u>Figure 1(c)</u></xref>).<li> These mutations are related to different <b>Nextstrain</b> and PANGO lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35355267">35355267</a> <b>(1)</b> &#x201C;Is Omicron mild&#x201D;? Testing this narrative with the mutational landscape of its three lineages and response to existing vaccines and therapeutic antibodies<ul><li> In the three nomenclature systems proposed by phylogenetic assignment of named global outbreak lineages, <b>Nextstrain</b> and global initiative on sharing all influenza data (GISAID), Omicron belongs to Pango lineage B.1.1.529 with BA.1, BA.2, and BA.3 included as its three descendent lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35502206">35502206</a> <b>(1)</b> Alterations in SARS-CoV-2 Omicron and Delta peptides presentation by HLA molecules<ul><li> According to <b>Nextstrain</b> project, at the beginning of April 2022 most of new COVID-19 cases were driven by Omicron and Delta variants.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35625733">35625733</a> <b>(0)</b> Resilience of Spike-Specific Immunity Induced by COVID-19 Vaccines against SARS-CoV-2 Variants<ul><li> To identify the mutations in the selected variants genomes, the Bioinformatic tool Nextclade v.1.14.0 was used (<uri xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</uri>, accessed on 17 Frebruary 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35815933">35815933</a> <b>(2)</b> Safety and immunogenicity of the Pfizer/BioNTech SARS-CoV-2 mRNA third booster vaccine dose against the BA.1 and BA.2 Omicron variants<ul><li> At present, Omicron is composed of several sublineages, such as BA.1, BA.1.1 (or <b>Nextstrain</b> clade 21 K), and BA.2 (or <b>Nextstrain</b> clade 21 L).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35474215">35474215</a> <b>(0)</b> Tracking SARS-CoV-2 Omicron diverse spike gene mutations identifies multiple inter-variant recombination events<ul><li> Mutation and Pango lineage of &#x201C;Deltacron&#x201D;-like variants with L452R mutation was annotated with Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35496653">35496653</a> <b>(4)</b> Genetic differentiation and diversity of SARS-CoV-2 Omicron variant in its early outbreak<ul><li>The Phylogenic tree was downloaded from <b>NextStrain</b> <xref rid="b0070" ref-type="bibr"><u>[14]</u></xref>, accessed on December 22, 2021, as a NEXUS file, including detailed lineage information.<li> In addition, systematic studies from the <b>NextStrain</b> website revealed that Omicron did not come from the previous dominant strain Delta <xref rid="b0025" ref-type="bibr"><u>[5]</u></xref>, <xref rid="b0070" ref-type="bibr"><u>[14]</u></xref>, <xref rid="b0090" ref-type="bibr"><u>[18]</u></xref> but was an individually emerging variant (<xref rid="f0005" ref-type="fig">Fig.<li> B) A phylogenetic tree from the <b>NextStrain</b> is displayed, whose five VOCs are labeled in different colors.<li>The Omicron variant was further divided into four lineages, namely B.1.1.529, BA.1, BA.2, and BA.3, based on the <b>NextStrain</b> <xref rid="b0070" ref-type="bibr"><u>[14]</u></xref>.<li> [[14]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[5]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34897752">34897752</a>) [[14]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[18]](W. Ma, J. Yang, H. Fu, C. Su, C. Yu, Q. Wang, A.T.R. de Vasconcelos, G.A. Bazykin, Y. Bao, M. Li, Genomic perspectives on the emerging SARS-CoV-2 omicron variant [Preprint], Genomics Proteomics Bioinf. (2022), S1672-0229(22)00002-X. <pub-id pub-id-type="doi">10.1016/j.gpb.2022.01.001</pub-id>.) [[14]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35468739">35468739</a> <b>(3)</b> AutoCoV: tracking the early spread of COVID-19 in terms of the spatial and temporal patterns from embedding space by K-mer based deep learning<ul><li>According to <b>Nextstrain</b>&#x2019;s August 2020 SARS-CoV-2 situation<xref rid="MOESM1" ref-type="media"><u>1</u></xref> reports, the virus has mutated over time as the virus spreads to different regions [<xref ref-type="bibr" rid="CR11"><u>11</u></xref>].<li> This trend was similar to the current outbreak of SARS-CoV-2 until August, and it was consistent with the August 2020 report of <b>NextStrain</b> [<xref ref-type="bibr" rid="CR11"><u>11</u></xref>].<li> All continents spread more widely than those in the NCBI dataset, but this seemed to indicate the influx of mutant viruses by intercontinental migration, as mentioned in <b>NextStrain</b> [<xref ref-type="bibr" rid="CR11"><u>11</u></xref>].<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35547225">35547225</a> <b>(6)</b> Identification and Analysis of SARS-CoV-2 Alpha Variants in the Largest Taiwan COVID-19 Outbreak in 2021<ul><li> The KMUH-4 to KMUH-7 sequences belong to the 20I (alpha, V1)/B.1.1.7/GRY (<b>NextStrain</b>_clade/pangolin_lineage/GISAID_clade) lineage, and KMUH-3 is 20I (alpha, V1)/B.1.1.7/GR; these were at that time the most important VOCs determined by the European Centre for Disease Prevention and Control (ECDC) and the World Health Organization (WHO).<li> Viruses are shown as the virus name| Accession ID| Collection date| City/<b>Nextstrain</b>_clade/pangolin_lineage/GISAID_clade).<li> <bold>(B)</bold> A partial tree exhibited phylogenetic correlation between <b>Nextstrain</b> clades 20H beta-V2, 20C, 21C epsilon, and 20G.<li> <bold>(C)</bold> A partial tree exhibited phylogenetic correlation between <b>Nextstrain</b> clades 20A, 21A delta, 21I delta, 21J delta, and 20E EU1.<li> <bold>(D)</bold> A partial tree exhibited phylogenetic correlation between <b>Nextstrain</b> clades 19A and 19B.<li> <bold>(E)</bold> A partial tree exhibited phylogenetic correlation between <b>Nextstrain</b> clades 20B, 20J, and 20I alpha-V1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35465708">35465708</a> <b>(1)</b> Evaluation of a Rapid and Accessible Reverse Transcription-Quantitative PCR Approach for SARS-CoV-2 Variant of Concern Identification<ul><li>Nextclade phylogenetic tree of 3,097 SARS-CoV-2 genomes, including all 732 of the sequenced genomes from this study and 2,365 genomes from the <b>Nextstrain</b> global reference tree as of 2 February 2022.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632625">35632625</a> <b>(4)</b> Phylodynamic Dispersal of SARS-CoV-2 Lineages Circulating across Polish&#x2013;German Border Provinces<ul><li>For SARS-CoV-2 variant identification, we employed two lineage assignment tools: PANGOLIN v3.0 (Available online: <uri xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</uri> (accessed on 1 July 2021)) and NEXTCLADE v01.2.0 (Available online: <uri xlink:href="https://github.com/<b>nextstrain</b>/nextclade">https://github.com/<b>nextstrain</b>/nextclade</uri> (accessed on 1 July 2021)).<li>Then, we used the <b>Nextstrain</b> [<xref rid="B3-viruses-14-00884" ref-type="bibr"><u>3</u></xref>] focal SARS-CoV-2 sequences for Europe and the Alpha Cluster built on 14 June 2021.<li> Following the <b>Nextstrain</b> workflow, we specified a clock rate of 8 &#xD7; 10<sup>&#x2212;4</sup> in TreeTime and excluded samples that differed by more than four times from a root-to-tip regression.<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35472367">35472367</a> <b>(1)</b> High rate of reinfection with the SARS-CoV-2 Omicron variant<ul><li> The first epidemic from February to early June 2020 was driven by three mutants derived from the Wuhan-Hu-1 isolate that were classified into <b>Nextstrain</b> clades (Pangolin lineages) 20A (B.1), 20B (B.1.1) and 20C (B.1).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35459088">35459088</a> <b>(0)</b> Comparison of SARS-CoV-2 sequencing using the ONT GridION and the Illumina MiSeq<ul><li>Consensus genomes produced by both platforms were uploaded to Nextclade Online Tool v1.4.2 (2021-10-26) (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>) for genome clade assignments, mutation calling, quality checks, and to determine the genome position on the SARS-CoV-2 phylogenetic tree.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35459226">35459226</a> <b>(2)</b> Human endogenous retrovirus K in the respiratory tract is associated with COVID-19 physiopathology<ul><li>The SARS-CoV-2 emerging clades circulating in Brazil during 2020 (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global?dmax=2020-12-16&amp;dmin=2020-01-16&amp;f_country=Brazil">https://<b>nextstrain</b>.org/ncov/global?dmax=2020-12-16&amp;dmin=2020-01-16&amp;f_country=Brazil</ext-link>) were found to activate HERV-K in the lower respiratory tract of critically ill COVID-19 patients under IMV.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35459859">35459859</a> <b>(4)</b> Phylogenetic and phylodynamic approaches to understanding and combating the early SARS-CoV-2 pandemic<ul><li> Although nomenclatures such as that currently adopted by the WHO assign names to definitive constellations of substitutions that commonly occur together (for example, VOC delta), most other current nomenclatures are lineage based (for example, Pango and <b>Nextstrain</b>).<li>Pango lineages of interest or concern during the first year of the pandemic</p></caption><table frame="hsides" rules="groups"><thead><tr><th>Pango lineage</th><th><b>Nextstrain</b> clade</th><th>WHO and Public Health England denotations</th><th>Territory of first reporting</th></tr></thead><tbody><tr><td>A.23.1.<li> Phylogeny from <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR146"><u>146</u></xref>,<xref ref-type="bibr" rid="CR147"><u>147</u></xref></sup> (which used data from the Europe ncov GISAID data set<sup><xref ref-type="bibr" rid="CR148"><u>148</u></xref></sup>), visualized in Figtree.<li> Acknowledgements of authors responsible for the genetic sequence data generated, shared via the GISAID initiative and used to generate the <b>Nextstrain</b> tree, may be found in Supplementary Table <xref rid="MOESM1" ref-type="media"><u>1</u></xref>.<li> [146](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [147](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [148](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35469875">35469875</a> <b>(1)</b> Use of wastewater surveillance for early detection of Alpha and Epsilon SARS-CoV-2 variants of concern and estimation of overall COVID-19 infection burden<ul><li> Sequences were then classified by all defined viral lineages categorized both by Pangolin lineage classifications and major <b>NextStrain</b> (<xref rid="bb0125" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) clades.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632615">35632615</a> <b>(1)</b> SARS-CoV-2 Variants in Paraguay: Detection and Surveillance with an Economical and Scalable Molecular Protocol<ul><li> The genotype of only K417 was found in samples of lineages B.1.1.28, B.1.1.33, B.1.1.277, and N3, all within the 20B <b>Nextstrain</b> clade and lineage B.1.499 (clade 20C) [<xref rid="B29-viruses-14-00873" ref-type="bibr"><u>29</u></xref>].<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35569431">35569431</a> <b>(0)</b> Presumptive positive with the Cepheid Xpert Xpress SARS-CoV-2 assay due to N mutations in the Delta variant<ul><li> Consensus FASTA files generated against reference SARS-Cov-2 sequence (NC_045512) were uploaded for variant analysis and detection of mutation at Pangolin lineage (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="interref0001">https://pangolin.cog-uk.io</ext-link>) and Nextclade (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="interref0003">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35451365">35451365</a> <b>(2)</b> The impact of SARS-CoV-2 on respiratory syndromic and sentinel surveillance in Israel, 2020: a new perspective on established systems<ul><li>The lack of viruses belonging to the 19A and 19B clades, which were predominant in Asia during the initial period of the SARS-CoV-2 outbreak, is consistent with their global decrease, as clade 20 became the dominant strain worldwide, around March 2020 (<ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global?dmax=2020-12-3" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global?dmax=2020-12-3</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35779562">35779562</a> <b>(3)</b> Delays in the arrival of the waves of COVID-19: a comparison between Gabon and the African continent<ul><li>For more on <bold><b>nextstrain</b></bold> see <ext-link ext-link-type="uri" xlink:href="http://https://<b>nextstrain</b>.org/ncov/gisaid/africa" id="interrefs10">https://<b>nextstrain</b>.org/ncov/gisaid/africa</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35460989">35460989</a> <b>(0)</b> The displacement of the SARS-CoV-2 variant Delta with Omicron: An investigation of hospital admissions and upper respiratory viral loads<ul><li> 70%, Tables S1 and S2 detail the quality of the genomes, sequences with coverage &gt;90% and mean depth &gt;100: Delta 840/ 908 and Omicron 987/ 1,119).Query mutations were manually confirmed with Integrated Genomics viewer (IGV) (Version 2.8.10), clades were determined using Nextclade beta v 1.13.2 (clades.<b>nextstrain</b>.org, Last accessed March 30, 2022), and lineages were determined with Pangolin COVID-19 lineage Assigner.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35442444">35442444</a> <b>(1)</b> Recombination between SARS-CoV-2 Omicron BA.1 and BA.2 variants identified in a traveller from Nepal at the airport quarantine facility in Japan<ul><li> The top 50 similar sequences were further analysed using <b>Nextstrain</b> to visualize the phylogenetic relationship in the country (B) and Phylogenetic Assignment of Named Global Outbreak (PANGO) lineage (C).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35443965">35443965</a> <b>(4)</b> Questioning the seasonality of SARS-COV-2: a Fourier spectral analysis<ul><li> These data are extrapolated, respectively, by the two following initiatives: Covariants.org (specifically, <ext-link xlink:href="https://covariants.org/per-country" ext-link-type="uri">https://covariants.org/per-country</ext-link>)<xref rid="R19" ref-type="bibr"><u>19</u></xref> and <b>Nextstrain</b>.org (specifically, <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/gisaid/global</ext-link>).<li> It is worth concluding by pointing out that each mentioned variant in <xref rid="T1" ref-type="table"><u>table 1</u></xref> has been identified based on the conventional names proposed by the genome sequencing initiatives mentioned above (ie, Covariants.org and <b>Nextstrain</b>.org.).<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34098568">34098568</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35516436">35516436</a> <b>(3)</b> Whole-Genome Sequence Approach and Phylogenomic Stratification Improve the Association Analysis of Mutations With Patient Data in Influenza Surveillance<ul><li> In a first study, this dataset was used to demonstrate that using powerful phylogenomic tools such as BEAST and <b>Nextstrain</b>, allows substantially improved phylogenetic classification when considering the whole genome rather than solely the HA segment (<xref rid="B59" ref-type="bibr"><u>Van Poelvoorde et al., 2021</u></xref>).<li>In order to compare results from the Belgian strains with the international context, a local <b>Nextstrain</b> instance (<xref rid="B25" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), allowing light-weight phylogenomics, was built using the in-house sequenced samples complemented with GISAID sequences.<li> were excluded from this analysis based on the custom-built <b>Nextstrain</b> instance of 10,583 samples because these strains were genetically too distant from the Belgian sequenced samples (<xref rid="DS2" ref-type="supplementary-material"><u>Data Sheet 2</u></xref>: <xref rid="DS2" ref-type="supplementary-material"><u>Supplementary Figure 1</u></xref> and <xref rid="TS2" ref-type="supplementary-material"><u>Supplementary Table 2</u></xref>: GISAID_Samples).<li> [Van Poelvoorde et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34477544">34477544</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35440671">35440671</a> <b>(3)</b> Rapid evolution of SARS-CoV-2 challenges human defenses<ul><li>We use the phylogenetic tree provided by <b>nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/global">https://<b>nextstrain</b>.org/ncov/global</ext-link> accessed 05/10/2021), which provides the phylogeny of ~&#x2009;3.8&#x2009;&#xD7;&#x2009;10<sup>6</sup> genomes sampled between December 2019 and October 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35447085">35447085</a> <b>(4)</b> Impact of previous exposure to SARS-CoV-2 and of S-Trimer (SCB-2019) COVID-19 vaccination on the risk of reinfection: a randomised, double-blinded, placebo-controlled, phase 2 and 3 trial<ul><li>Sourced from <b>Nextstrain</b> and the Philippines Genome center .<li>For more on <bold><b>Nextstrain</b></bold> see <ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org" id="interrefs30">https://<b>nextstrain</b>.org</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35420469">35420469</a> <b>(1)</b> The Delta SARS-CoV-2 Variant of Concern Induces Distinct Pathogenic Patterns of Respiratory Disease in K18-hACE2 Transgenic Mice Compared to the Ancestral Strain from Wuhan<ul><li> The two isolates were therefore both genetically close to the original ancestral strain (<b>Nextstrain</b>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35421164">35421164</a> <b>(2)</b> Wastewater surveillance of SARS-CoV-2 mutational profiles at a university and its surrounding community reveals a 20G outbreak on campus<ul><li> Four of the five signature mutations of the 20C clade in <b>NextStrain</b> [<xref rid="pone.0266407.ref029" ref-type="bibr"><u>29</u></xref>], also referred to as the GH clade in GISAID [<xref rid="pone.0266407.ref030" ref-type="bibr"><u>30</u></xref>].<li> The high frequency of these mutations across both university and WWTP influent samples is in agreement with clinical data from July to September 2020, where the 20A, 20C, and 20G <b>NextStrain</b> clades were dominant in the United States [<xref rid="pone.0266407.ref029" ref-type="bibr"><u>29</u></xref>].<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> Wastewater surveillance of SARS-CoV-2 mutational profiles at a university and its surrounding community reveals a 20G outbreak on campus<ul><li> If these 3 are all going in the same direction (are they?), they may be evidence that either the University or the City were undergoing a wave of 20G that the other was not (<ext-link xlink:href="https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv" ext-link-type="uri">https://github.com/<b>nextstrain</b>/ncov/blob/master/defaults/clades.tsv</ext-link>).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35422113">35422113</a> <b>(10)</b> Computational prediction of the molecular mechanism of statin group of drugs against SARS-CoV-2 pathogenesis<ul><li> Sequences were mapped against the SARS-CoV-2 reference sequence (Wuhan/WH01/2019) using the NextAlign aligner tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/nextclade">https://github.com/<b>nextstrain</b>/nextclade</ext-link>).<li> To predict the time-dependent emergence of a specific mutation, we performed a phylodynamic analysis of global subsamples of SARS-CoV-2 sequences (N&#x2009;=&#x2009;3525) using the next train/ncov pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> Entropy for each position on SARS-CoV-2 Spike protein, helicase, RdRp, and main protease was calculated using a method encoded in the <b>nextstrain</b>/ncov pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).<li> <bold>(B)</bold> Lineage-defining Spike protein mutations for SARS-CoV-2 variant of concerns (VOCs) that are most prevalent worldwide, determined from the SARS-CoV-2 sequences and metadata submitted in GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</ext-link>), using the <b>Nextstrain</b> ncov pipeline (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35427787">35427787</a> <b>(6)</b> Comparative genomics, evolutionary epidemiology, and RBD-hACE2 receptor binding pattern in B.1.1.7 (Alpha) and B.1.617.2 (Delta) related to their pandemic response in UK and India<ul><li>The data was also collected using several databases and servers such as Github and Pango lineages (<xref rid="bb0345" ref-type="bibr"><u>O'Toole and McCrone, 2020</u></xref>), Pango lineages (<xref rid="bb0350" ref-type="bibr"><u>O'Toole et al., 2020</u></xref>), SARS-CoV-2 resources GISAID (<xref rid="bb0410" ref-type="bibr"><u>Velazquez et al., 2020</u></xref>), and <b>Nextstrain</b> server (<xref rid="bb0230" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bb0335" ref-type="bibr"><b>Nextstrain</b>, 2020</xref>).<li> The <b>NextStrain</b> server fetches the records from the GISAID server.<li>Here, we used servers and databases like <b>Nextstrain</b> (<xref rid="bb0230" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bb0425" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) and GISAID (<xref rid="bb0425" ref-type="bibr"><u>Velazquez et al., 2020</u></xref>) to analyze the molecular phylogenetics, the molecular clock of evolution, country-wise geographical distribution, and transmission pattern, and entropy diversity of these two variants.<li> The <b>Nextstrain</b> server is used to evaluate a pathogen's real-time evolution and is a very efficient tool for phylodynamics analysis (<xref rid="bb0230" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li>This study used the <b>Nextstrain</b> server and GISAID to access the molecular phylogenetics, the molecular clock of evolution, country-wise geographical distribution, transmission pattern, etc., two variants.<li> [O'Toole and McCrone, 2020](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.</given-names>            </name>          </person-group>          <part-title>Software Package for Assigning SARS-CoV-2 Genome Sequences to Global Lineages</part-title>          <year>2020</year>          <publisher-name>GitHub</publisher-name>          <ext-link ext-link-type="uri" xlink:href="https://github.com/hCoV-2019/pangolin" id="ir0035">https://github.com/hCoV-2019/pangolin</ext-link>          <comment>(Accessed on September 15, 2021)</comment>        ) [O'Toole et al., 2020](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Underwood</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Jackson</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Ruis</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Abu-Dahab</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Taylor</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Yeats</surname>              <given-names>C.</given-names>            </name>          </person-group>          <article-title>Pangolin: lineage assignment in an emerging pandemic as an epidemiological tool</article-title>          <source>Virus Evol.</source>          <volume>7</volume>          <year>2020</year>          <comment>veab064</comment>        ) [Velazquez et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Velazquez</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Bustria</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Ouyang</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>Moshiri</surname>              <given-names>N.</given-names>            </name>          </person-group>          <article-title>An analysis of clinical and geographical metadata of over 75,000 records in the GISAID COVID-19 database</article-title>          <source>medRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.09.22.20199497</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut et al., 2020](          <person-group person-group-type="author">            <name>              <surname>WHO</surname>            </name>          </person-group>          <part-title>Weekly Epidemiological Update&#x2013;October 27</part-title>          <volume>579</volume>          <year>2020</year>          <publisher-name>WHO</publisher-name>          <publisher-loc>Geneva</publisher-loc>          <fpage>580</fpage>        ) [Velazquez et al., 2020](          <person-group person-group-type="author">            <name>              <surname>WHO</surname>            </name>          </person-group>          <part-title>Weekly Epidemiological Update&#x2013;October 27</part-title>          <volume>579</volume>          <year>2020</year>          <publisher-name>WHO</publisher-name>          <publisher-loc>Geneva</publisher-loc>          <fpage>580</fpage>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35412591">35412591</a> <b>(1)</b> Associations Between Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Variants and Risk of Coronavirus Disease 2019 (COVID-19) Hospitalization Among Confirmed Cases in Washington State: A Retrospective Cohort Study<sup><xref rid="ciac279-FM2" ref-type="author-notes">&#xA0;</xref></sup><ul><li> Sequence quality was determined using Nextclade version 1.0.1 (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).<li> These were all assigned a <b>Nextstrain</b> clade making the distinction clear [<xref rid="ciac279-B15" ref-type="bibr"><u>15</u></xref>].<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34108654">34108654</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35414157">35414157</a> <b>(2)</b> Reverse vaccinology approach for multi-epitope centered vaccine design against delta variant of the SARS-CoV-2<ul><li> The notable effect caused by these mutations is the increased transmission rate (within 10&#xA0;s, 60% more transmissible than alpha variant) rendering it as most dangerous and dominant variant globally (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/ncov/gisaid/global?l=clock">https://<b>nextstrain</b>.org/ncov/gisaid/global?l=clock</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35495156">35495156</a> <b>(1)</b> Case Report: Omicron BA.2 Subvariant of SARS-CoV-2 Outcompetes BA.1 in Two Co-infection Cases<ul><li> Consensus sequences were used to generate a phylogenetic tree in <b>Nextstrain</b> (<ext-link xlink:href="https://clades.<b>nextstrain</b>.org/" ext-link-type="uri">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35441170">35441170</a> <b>(3)</b> The Roles of APOBEC-mediated RNA Editing in SARS-CoV-2 Mutations, Replication and Fitness<ul><li> With the direct experimental evidence that APOBECs can target specific sites on SARS-CoV-2 for editing, we analyzed the publicly available SARS-CoV-2 genome sequence data (the <b>Nextstrain</b> datasets from Dec.<li> 22<sup>nd</sup>, 2022 downloaded from the GISIAD database, <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global</ext-link>)(<xref rid="R35" ref-type="bibr"><u>35</u></xref>), with hope to detect some obvious effects of APOBEC-mediated viral mutations on the current viral strain variants and fitness.<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35410128">35410128</a> <b>(1)</b> DeepSARS: simultaneous diagnostic detection and genomic surveillance of SARS-CoV-2<ul><li> The beginning and the end of the alignment are masked respectively by 100 and 50 sequences as well as sites 13,402, 24,389 and 24,390, identified by <b>Nextstrain</b> as prone to sequencing errors [<xref ref-type="bibr" rid="CR35"><u>35</u></xref>].<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35455582">35455582</a> <b>(1)</b> Critically Ill Pediatric Patient and SARS-CoV-2 Infection<ul><li> For epidemiologic and virologic purposes, several nomenclatures have been adopted (Pango lineage, GISAID clade, <b>Nextstrain</b> clade) to describe so-called SARS-CoV-2 variants of concern (VOC), with increased transmissibility, virulence, severe clinical presentation and/or with decreased potential for available treatment options [<xref rid="B16-children-09-00538" ref-type="bibr"><u>16</u></xref>].<li> [16](          <person-group person-group-type="author"><collab>World Health Organization</collab></person-group>          <article-title>Tracking SARS-CoV-2 Variants</article-title>          <comment>Available online: <ext-link xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" ext-link-type="uri">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-02-05">(accessed on 5 February 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35479940">35479940</a> <b>(1)</b> Advanced Functions Embedded in the Second Version of Database, Global Evaluation of SARS-CoV-2/hCoV-19 Sequences 2<ul><li> The &#x201C;<b>Nextstrain</b>&#x201D;<sup><xref rid="footnote4" ref-type="fn"><u>4</u></xref></sup> (<xref rid="B7" ref-type="bibr"><u>7</u></xref>) is another web tool that can conduct a viral evolution analysis on COVID-19 and others such as influenza.<li> [4](<a href="http://nextstrain.org/">http://nextstrain.org/</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35421481">35421481</a> <b>(0)</b> Wastewater and marine bioindicators surveillance to anticipate COVID-19 prevalence and to explore SARS-CoV-2 diversity by next generation sequencing: One-year study<ul><li> Mutations were analyzed in comparison to the Wuhan reference using the Nextclade tool (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org" id="ir0030">https://clades.<b>nextstrain</b>.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35397689">35397689</a> <b>(2)</b> Should routine risk reduction procedures for the prevention and control of pandemics become a standard in all oncological outpatient clinics? The prospective COVID-19 cohort study: protect-CoV<ul><li> Phylogenetic analyses were achieved with the web- and analysis platform Auspice using the SARS-CoV-2 build (<ext-link ext-link-type="uri" xlink:href="https://github.com/<b>nextstrain</b>/ncov">https://github.com/<b>nextstrain</b>/ncov</ext-link>) and the bioinformatic toolkit augur [<xref ref-type="bibr" rid="CR18"><u>18</u></xref>, <xref ref-type="bibr" rid="CR19"><u>19</u></xref>].<li> [18](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>SM</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>P</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics (Oxford, England)</source>          <year>2018</year>          <volume>34</volume>          <issue>23</issue>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [19](          <person-group person-group-type="author">            <name>              <surname>Billion</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Ghai</surname>              <given-names>R</given-names>            </name>            <name>              <surname>Chakraborty</surname>              <given-names>T</given-names>            </name>            <name>              <surname>Hain</surname>              <given-names>T</given-names>            </name>          </person-group>          <article-title>Augur&#x2014;a computational pipeline for whole genome microbial surface protein prediction and classification</article-title>          <source>Bioinformatics (Oxford, England)</source>          <year>2006</year>          <volume>22</volume>          <issue>22</issue>          <fpage>2819</fpage>          <lpage>2820</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/btl466</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35456974">35456974</a> <b>(10)</b> SARS-CoV-2 Pandemic Tracing in Italy Highlights Lineages with Mutational Burden in Growing Subsets<ul><li> The analysis was carried out by running the <b>Nextstrain</b> pipeline [<xref rid="B17-ijms-23-04155" ref-type="bibr"><u>17</u></xref>], with parameters described under the Materials and Methods section, on a dataset containing all the Italian sequences submitted to GISAID [<xref rid="B18-ijms-23-04155" ref-type="bibr"><u>18</u></xref>,<xref rid="B19-ijms-23-04155" ref-type="bibr"><u>19</u></xref>,<xref rid="B20-ijms-23-04155" ref-type="bibr"><u>20</u></xref>] at the time when the analysis was performed (8 January 2021), and a sample of the sequences from the neighboring countries, selected on the basis of population and spreading degree of SARS-CoV-2 infection, in order to avoid overrepresentation of sequences from countries, including Switzerland, who, following a different sequencing policy, had submitted many more sequences than others.<li>The relationship between subsets and known <b>Nextstrain</b> clades was assessed by evaluating the fraction of leaves shared between each subset and each clade; the results, reported in full in <xref rid="app1-ijms-23-04155" ref-type="app"><u>Table S1 in the Supplementary Materials</u></xref>, are used to fill the two clade columns of <xref rid="ijms-23-04155-t001" ref-type="table"><u>Table 1</u></xref>.<li> column&#x201D;, S4 corresponds with the <b>Nextstrain</b> clade 20I/501Y.V1 (the &#x201C;UK&#x201D; variant), while S7 corresponds to the above mentioned clade 20A.EU2.<li> S4 and S7, previously identified as <b>Nextstrain</b> clades 20I/501Y.V1 and 20A.EU2, also correspond with the equivalent Pango lines B.1.1.7 and B.1.160, respectively.<li> For the other subsets (in yellow in <xref rid="ijms-23-04155-t001" ref-type="table"><u>Table 1</u></xref>), the analysis failed to give further information, as they were identified as the Pango lines corresponding to the <b>Nextstrain</b> clades from which they derive, as reported in column &#x201C;parent&#x201D; in the same table.<li> Phylodynamic analysis was based on the <italic toggle="yes"><b>Nextstrain</b>/ncov</italic> workflow, optimized for SARS-CoV-2 analysis [<xref rid="B17-ijms-23-04155" ref-type="bibr"><u>17</u></xref>,<xref rid="B60-ijms-23-04155" ref-type="bibr"><u>60</u></xref>].<li> <italic toggle="yes"><b>nextstrain</b>.cli</italic> version 2.0.0.post1 was used to analyze 2020 data, while <italic toggle="yes">nexstrain.cli 3.0.3</italic> was used for the datasets included in the tracing analysis.<li> Subset Relationships with <b>Nextstrain</b> Clades/Pango Lineages.<li>Subsets were initially assigned to a <b>Nextstrain</b> clade and/or a Pango lineage, using the information annotated for each leaf in the tree file.<li>Membership of a subset in a <b>Nextstrain</b> clade is assigned on the basis of the presence in their consensus sequence of the clade signatures used by the <italic toggle="yes">ncov</italic> workflow version 3.0.3, reported in <italic toggle="yes">clade_signatures.txt</italic> in the <xref rid="app1-ijms-23-04155" ref-type="app"><u>Supplementary Materials</u></xref>.<li> [17](          <person-group person-group-type="author"><name><surname>Hadfield</surname><given-names>J.</given-names></name><name><surname>Megill</surname><given-names>C.</given-names></name><name><surname>Bell</surname><given-names>S.M.</given-names></name><name><surname>Huddleston</surname><given-names>J.</given-names></name><name><surname>Potter</surname><given-names>B.</given-names></name><name><surname>Callender</surname><given-names>C.</given-names></name><name><surname>Sagulenko</surname><given-names>P.</given-names></name><name><surname>Bedford</surname><given-names>T.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextstrain: Real-Time Tracking of Pathogen Evolution</article-title>          <source>Bioinforma</source>          <year>2018</year>          <volume>34</volume>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34934514">34934514</a>) [19](          <person-group person-group-type="author"><name><surname>Elbe</surname><given-names>S.</given-names></name><name><surname>Buckland-Merrett</surname><given-names>G.</given-names></name></person-group>          <article-title>Data, Disease and Diplomacy: GISAID&#x2019;s Innovative Contribution to Global Health</article-title>          <source>Glob. Chall. Hoboken NJ</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382917">28382917</a>) [17](          <person-group person-group-type="author"><name><surname>Hadfield</surname><given-names>J.</given-names></name><name><surname>Megill</surname><given-names>C.</given-names></name><name><surname>Bell</surname><given-names>S.M.</given-names></name><name><surname>Huddleston</surname><given-names>J.</given-names></name><name><surname>Potter</surname><given-names>B.</given-names></name><name><surname>Callender</surname><given-names>C.</given-names></name><name><surname>Sagulenko</surname><given-names>P.</given-names></name><name><surname>Bedford</surname><given-names>T.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextstrain: Real-Time Tracking of Pathogen Evolution</article-title>          <source>Bioinforma</source>          <year>2018</year>          <volume>34</volume>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [60](          <article-title>NCOV, Nextstrain Build for Novel Coronavirus SARS-CoV-2</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/ncov" ext-link-type="uri">https://github.com/nextstrain/ncov</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-29">(accessed on 29 March 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35464852">35464852</a> <b>(1)</b> Genomic Surveillance of COVID-19 Variants With Language Models and Machine Learning<ul><li> Earlier, initiatives such as <italic><b>Nextstrain</b></italic> (<xref rid="B5" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) have focused on providing high-quality tracking information for the strains and lineages as these emerge without forecasting or predictions.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35396511">35396511</a> <b>(10)</b> Emergence and phenotypic characterization of the global SARS-CoV-2 C.1.2 lineage<ul><li><bold>a</bold> Phylogenetic tree of 6,192 global sequences (1,991 sequences from South Africa), including variants of concern (VOC), variants of interest (VOI), and the C.1.2 lineage, colored by <b>Nextstrain</b> clade (shown in the key; VOCs and VOIs are in color, other clades in greyscale) and scaled by divergence (number of mutations).<li> The tree is colored by month of collection, with tip symbols indicating country of collection, as indicated in the key (Figure generated from a <b>Nextstrain</b> build of global C.1.2 sequences with &#x2265;90% coverage).<li><bold>a</bold> Visualization of C.1.2 spike mutations, highlighting those shared with VOCs and VOIs (colored by the <b>Nextstrain</b> clade).<li> The SARS-CoV-2 genomes in our dataset were also classified using the clade classification proposed by <b>Nextstrain</b> (<ext-link ext-link-type="uri" xlink:href="https://<b>nextstrain</b>.org/">https://<b>nextstrain</b>.org/</ext-link>) built for real-time tracking of the pathogen evolution<sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup>.<li> This dataset was further down-sampled using a custom build of the <b>Nextstrain</b> SARS-CoV-2 pipeline<sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup> to produce a final dataset of 6192 genomes.<li>Phylogenetic analysis was conducted with a custom <b>Nextstrain</b> SARS-CoV-2 build (using <b>NextStrain</b> v7)<sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup>.<li> Briefly, the pipeline filters sequences, aligns these sequences with Nextalign (<ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/nextalign">https://github.com/neherlab/nextalign</ext-link>), sub-samples the datasets (resulting in the dataset described above), constructs a phylogenetic tree with IQ-TREE<sup><xref ref-type="bibr" rid="CR31"><u>31</u></xref></sup>, refines and dates the tree with TreeTime<sup><xref ref-type="bibr" rid="CR33"><u>33</u></xref></sup>, reconstructs ancestral states, and assigns <b>Nextstrain</b> clades to the sequences.<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [33](          <person-group person-group-type="author">            <name>              <surname>Sagulenko</surname>              <given-names>P</given-names>            </name>            <name>              <surname>Puller</surname>              <given-names>V</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>TreeTime: maximum-likelihood phylodynamic analysis</article-title>          <source>Virus Evol.</source>          <year>2018</year>          <volume>4</volume>          <fpage>042</fpage>          <pub-id pub-id-type="doi">10.1093/ve/vex042</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34981048">34981048</a> <b>(3)</b> The Roles of APOBEC-mediated RNA Editing in SARS-CoV-2 Mutations, Replication and Fitness<ul><li> With the direct experimental evidence that APOBECs can target specific sites on SARS-CoV-2 for editing, we analyzed the publicly available SARS-CoV-2 genome sequence data (the <b>Nextstrain</b> datasets from Dec.<li> 22<sup>nd</sup>, 2022 downloaded from the GISIAD database, <ext-link xlink:href="https://<b>nextstrain</b>.org/ncov/global" ext-link-type="uri">https://<b>nextstrain</b>.org/ncov/global</ext-link>)(<xref rid="R35" ref-type="bibr"><u>35</u></xref>), with hope to detect some obvious effects of APOBEC-mediated viral mutations on the current viral strain variants and fitness.<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35456454">35456454</a> <b>(2)</b> Using an Unsupervised Clustering Model to Detect the Early Spread of SARS-CoV-2 Worldwide<ul><li>We followed the <b>Nextstrain</b> pipeline [<xref rid="B20-genes-13-00648" ref-type="bibr"><u>20</u></xref>] to map the downloaded SARS-CoV-2 strains to the reference genome (GenBank Accession Number: NC_045512.2).<li> Considering putatively artefactual substitutions and gaps located at the beginning and end of the sequencing, we masked the first 130 bps and last 50 bps in the mutation calling step based on the <b>Nextstrain</b> pipeline [<xref rid="B20-genes-13-00648" ref-type="bibr"><u>20</u></xref>].<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35430092">35430092</a> <b>(1)</b> Prolonged shedding of infectious viruses with haplotype switches of SARS-CoV-2 in an immunocompromised patient<ul><li> All seven haplotypes were classified in 20B in <b>Nextstrain</b> clade or B.1.1.214 in Pangolin Lineage V.3.1.11, which were the dominant circulating strains in Japan in December 2020 [<xref rid="bib9" ref-type="bibr"><u>9</u></xref>].<li> [9](          <person-group person-group-type="author">            <name>              <surname>Takashi</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Kentaro</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Koji</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>Rina</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Satsuki</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Risa</surname>              <given-names>S.</given-names>            </name>            <etal/>          </person-group>          <article-title>Molecular analysis of SARS-CoV-2 (2021.1.14)</article-title>          <source>Infectious Agents Surveillance Reports</source>          <volume>42</volume>          <year>2011</year>          <fpage>61</fpage>          <lpage>64</lpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35389245">35389245</a> <b>(2)</b> The Alpha Variant (B.1.1.7) of SARS-CoV-2 Failed to Become Dominant in Mexico<ul><li> Additionally, to explore further the relationship between B.1.1.7 viruses of Mexico and the United States, a phylogeny was built in <b>Nextstrain</b> using all high-quality sequences from Mexico&#x2019;s northern states plus 100 sequences per month from the United States and the global subsampling from <b>Nextstrain</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35393464">35393464</a> <b>(2)</b> Host&#x2013;pathogen dynamics in longitudinal clinical specimens from patients with COVID-19<ul><li>Of the seven most commonly observed variants in our dataset (Table <xref rid="Tab3" ref-type="table"><u>3</u></xref>), the three most frequent define the Washington state outbreak clade<sup><xref ref-type="bibr" rid="CR16"><u>16</u></xref>,<xref ref-type="bibr" rid="CR17"><u>17</u></xref></sup> and the rest of the variants are clade-defining mutations in <b>Nextstrain</b> clades 20A and 20C.<li> Clade assignments were generated using Pangolin (<ext-link ext-link-type="uri" xlink:href="http://github.com/cov-lineages/pangolin">http://github.com/cov-lineages/pangolin</ext-link>) and <b>Nextstrain</b><sup><xref ref-type="bibr" rid="CR40"><u>40</u></xref></sup> in December 2020.<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32913002">32913002</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32912998">32912998</a>) [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35875697">35875697</a> <b>(1)</b> Genomic characterisation reveals a dominant lineage of SARS-CoV-2 in PapuaNew Guinea<ul><li> This dataset included the genomes generated in this study withsufficient sequence quality and associated date of collection, and a sample of globalgenomic diversity focussed on the region of Oceania by using the latest <b>NextStrain</b> Oceaniabuild, that included 489 genomes from other countries (as of 20 March 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35388091">35388091</a> <b>(0)</b> Phylogeography and genomic epidemiology of SARS-CoV-2 in Italy and Europe with newly characterized Italian genomes between February-June 2020<ul><li> 0.14.1 (<ext-link ext-link-type="uri" xlink:href="https://clades.<b>nextstrain</b>.org/">https://clades.<b>nextstrain</b>.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35277864">35277864</a> <b>(2)</b> SARS&#x2010;CoV&#x2010;2 variants and vulnerability at the global level<ul><li> For viral variant&#x2010;related information, we have also referred to <ext-link xlink:href="https://<b>nextstrain</b>.org/" ext-link-type="uri">https://<b>nextstrain</b>.org/</ext-link> and GISAID.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35316576">35316576</a> <b>(1)</b> Evaluation of canine detection of COVID&#x2010;19 infected individuals under controlled settings<ul><li> Genomic data of the variants circulating in the world were accessed on <b>nextstrain</b>.org on a weekly basis during the life of this project and highlighted the evolution and diversity of SARS&#x2010;CoV&#x2010;2 variants from the source populations.</ul></p>
</body>
