<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<title>Sentences that contain nextclade</title>
</head>
<body>
<h1>Sentences that contain nextclade</h1>
<p><u>No PMID</u> <b>(1)</b> Prolonged infective SARS-CoV-2 omicron variant shedding in a patient with diffuse large B cell lymphoma successfully cleared after three courses of remdesivir<ul><li> Base substitutions T911A and C912A lead to amino acid substitution S36N in non-structural protein (NSP)2, T3161C and 6513-6515Del lead to S36N and SL1265I in NSP3, G8868A leads to G105D in NSP4, G23048A, C24054T, or G24521T lead to G496S, A831V, or V987F in spike protein, and T28417C leads to I45T in open-reading frame (ORF)9b, respectively, according to data from <b>Nextclade</b> v2.10.0 [<xref rid="bib13" ref-type="bibr"><u>13</u></xref>].<li> [13](          <person-group person-group-type="author">            <collab>Nextcladev2.11.0</collab>          </person-group>          <article-title>Clade assignment, mutation calling, and sequence quality Checks</article-title>          <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0010">https://clades.nextstrain.org/</ext-link>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37149667">37149667</a> <b>(5)</b> Global mpox lineage discovery and rapid outbreak tracking with nanopore sequencing<ul><li>After sequencing, all consensus sequences were analyzed using <b>NextClade</b> software [<xref ref-type="bibr" rid="CR10"><u>10</u></xref>].<li> As of October 2022, there are twelve sub-lineages, as defined by the <b>NextClade</b> group [<xref ref-type="bibr" rid="CR11"><u>11</u></xref>] (Fig.&#xA0;<xref rid="Fig1" ref-type="fig"><u>1</u></xref>B).<li> It has subsequently been designated as lineage B.1.11 by <b>Nextclade</b>.<li>The ARTIC bioinformatics pipeline (v1.2.1) was used to consolidate FASTQ reads from MinKNOW and align them against the <b>NextClade</b> B.1 Mpox template genome (Additional file <xref rid="MOESM2" ref-type="media"><u>2</u></xref>: Working Protocol 10).<li><b>NextClade</b> software was used to identify mutations in each sequence against the template genome (Additional file <xref rid="MOESM2" ref-type="media"><u>2</u></xref>: Working Protocol 11).<li> [10](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Source Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [11](mpxv-lineages/lineage-designation (<ext-link ext-link-type="uri" xlink:href="https://github.com/mpxv-lineages/lineage-designation">https://github.com/mpxv-lineages/lineage-designation</ext-link>; 2022).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37133864">37133864</a> <b>(1)</b> Efficacy and Safety of a Protein-Based SARS-CoV-2 Vaccine<ul><li> Participants with RT-PCR&#x2013;positive testing with an N gene cycle threshold value less than 25 underwent an S gene sequence analysis.<sup><xref rid="zoi230329r14" ref-type="bibr"><u>14</u></xref></sup> SARS-CoV-2 variant determination was conducted using the <b>Nextclade</b> application.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33738620">33738620</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37039029">37039029</a> <b>(2)</b> Reinfection rate and disease severity of the BA.5 Omicron SARS-CoV-2 lineage compared to previously circulating variants of concern in the Canary Islands (Spain)<ul><li> Lineages were assigned using Pangolin v.4.1.2 in the Usher mode [<xref rid="CIT0009" ref-type="bibr"><u>9</u></xref>], while sequence quality control (QC) was carried out using <b>Nextclade</b> (v2.3.1) [<xref rid="CIT0010" ref-type="bibr"><u>10</u></xref>].<li> The tree also includes relationships with other 1,504 genome sequences representative of major clades (light grey), labelled based on Nextstrain definitions, from <b>Nextclade</b> v2.13.0 and Auspice v2.24.2 (as of 2023-03-24), and sampled between December 2019 and July 2022 for reference.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [10](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <etal>et al.</etal></person-group><article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source>. <year>2021</year>;<volume>6</volume>:<fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37117393">37117393</a> <b>(13)</b> Minimal requirements for ISO15189 validation and accreditation of three next generation sequencing procedures for SARS-CoV-2 surveillance in clinical setting<ul><li>The molecular typing of SARS-CoV-2 strains was carried out using different classifications: WHO, <b>Nextclade</b> and Pangolin with <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup> (version 2.2.0) and Pango-designation<sup><xref ref-type="bibr" rid="CR31"><u>31</u></xref></sup> (version 1.9, lineages version 2022-05-13) databases available online.<li> The comparison was carried out at two levels: typing according to WHO guidelines and two databases (Pangolin<sup><xref ref-type="bibr" rid="CR31"><u>31</u></xref></sup> and <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup>), and comparison of amino acid mutations.<li> Typing results for sample UZL_CV10_89 are identical in our lab and both labs to which it has been compared when using <b>Nextclade</b> or according to WHO guidelines but differ from lab to lab by using the Pangolin database (Table <xref rid="Tab1" ref-type="table"><u>1</u></xref>).<li><b>NextClade</b>.<li><b>NextClade</b>.<li> Reproducibility was assessed at three levels: typing according to WHO guidelines and two databases (Pangolin and <b>Nextclade</b>), amino acid (aa) mutations and SNVs identification.<li> The repeatability was assessed at three levels: typing according to WHO guidelines and two databases (PangoLineage and <b>Nextclade</b>), aa mutations and SNVs identification.<li>The samples were sequenced according to the three SOPs and compared at two levels: typing according to WHO guidelines and two databases (Pangolin and <b>Nextclade</b>), and SNVs identification.<li> 25, the WHO and <b>NextClade</b> nomenclatures show 100% concordance between the three methods.<li><b>NextClade</b></th><th align="left" colspan="3">Pangolin</th></tr><tr><th align="left">Artic v3</th><th align="left">Midnight v1</th><th align="left">Nimagen</th><th align="left">Artic v3</th><th align="left">Midnight v1</th><th align="left">Nimagen</th><th align="left">Artic v3</th><th align="left">Midnight v1</th><th align="left">Nimagen</th></tr></thead><tbody><tr><td align="left">CV2031222739.<li> 100% typing concordance was obtained with the <b>Nextclade</b> and Pangolin databases.<li> 100% typing concordance with the previous typing results (WHO, <b>Nextclade</b> and Pangolin) must be obtained.<li> [30](Aksamentov, I., Roemer, C., Hodcroft, E. &amp; Neher, R. Nextclade: Clade assignment, mutation calling and quality control for viral genomes. <italic>J. Open Source Softw.</italic><bold>6</bold>, 3773. 10.21105/joss.03773 (2021).) [31](O&#x2019;Toole, A. <italic>et&#xA0;al.</italic> Assignment of Epidemiological Lineages in an Emerging Pandemic Using the Pangolin Tool. <italic>Virus Evol.</italic> veab064, 10.1093/ve/veab064 (2021).) [31](O&#x2019;Toole, A. <italic>et&#xA0;al.</italic> Assignment of Epidemiological Lineages in an Emerging Pandemic Using the Pangolin Tool. <italic>Virus Evol.</italic> veab064, 10.1093/ve/veab064 (2021).) [30](Aksamentov, I., Roemer, C., Hodcroft, E. &amp; Neher, R. Nextclade: Clade assignment, mutation calling and quality control for viral genomes. <italic>J. Open Source Softw.</italic><bold>6</bold>, 3773. 10.21105/joss.03773 (2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37101215">37101215</a> <b>(1)</b> SARS-CoV-2 surveillance-based on municipal solid waste leachate in Brazil<ul><li> <xref ref-type="bibr" rid="CR41"><u>2021</u></xref>) and <b>Nextclade</b> version v.0.14.2 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) (Aksamentov et al.<li> [2021](O&#x2019;Toole &#xC1;, Scher E, Underwood A, Jackson B, Hill V, McCrone JT, Colquhoun R, Ruis C, Abu-Dahab K, Taylor B, Yeats C, du Plessis L, Maloney D, Medd N, Attwood SW, Aanensen DM, Holmes EC, Pybus OG, Rambaut A (2021) Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool. Virus Evol 7(2):veab064. 10.1093/ve/veab064) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35923324">35923324</a> <b>(1)</b> Genomic epidemiology reveals the dominance of Hennepin County in transmission of SARS-CoV-2 in Minnesota from 2020&#x2013;2022<ul><li> 2) given an overall quality score by <b>Nextclade</b> [<xref rid="R16" ref-type="bibr"><u>16</u></xref>] as <italic toggle="yes">bad</italic>; 3) potentially contaminated based on presence of unusual allele frequencies (&lt; 0.9); 3) duplicate runs; 4) positive or negative controls.<li> [16](<name><surname>Aksamentov</surname><given-names>I.</given-names></name>, <etal/>, <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. . <source>zenodo</source>, <year>2021</year>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37099540">37099540</a> <b>(4)</b> Evaluation of miniaturized Illumina DNA preparation protocols for SARS-CoV-2 whole genome sequencing<ul><li> QC metrics were generated using <b>Nextclade</b>, a tool that identifies differences between your sequences and a reference sequence used by Nextstrain [<xref rid="pone.0283219.ref021" ref-type="bibr"><u>21</u></xref>], which uses these differences to assign your sequences to clades, and reports potential sequence quality issues in your data.<li>To compare the quality of sequencing between the two methods, consensus QC reports were generated using <b>Nextclade</b> [<xref rid="pone.0283219.ref022" ref-type="bibr"><u>22</u></xref>].<li>In this study, we used <b>Nextclade</b> to determine the quality profiles of the genomes produced.<li> <b>Nextclade</b> is part of the Nextstrain host of tools that have been pivotal in the real-time tracking of SARS-CoV-2 and other infectious disease pathogens globally.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](<name><surname>Aksamentov</surname><given-names>IR</given-names></name>, <name><surname>Cornelius</surname></name> &amp; <name><surname>Hodcroft</surname></name>, <name><surname>Emma</surname><given-names>Neher</given-names></name>, <name><surname>Richard</surname></name>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source>. <year>2021</year>;<fpage>6</fpage>.) <p><u>No PMID</u> <b>(1)</b> Timely course of SARS-CoV-2 infections and vaccinations in patients with hemato-oncological diseases: analysis of a real-life cohort<ul><li>Pandemic phases with regard to viral variants were defined based on <b>Nextclade</b>/GISAID data which aggregate reported results of SARS-CoV-2 genome sequencing<xref rid="bib21" ref-type="bibr"><sup>21</sup></xref><sup>,</sup><xref rid="bib22" ref-type="bibr"><sup>22</sup></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37095145">37095145</a> <b>(1)</b> SARS-CoV-2 variants and mutational patterns: relationship with risk of ventilator-associated pneumonia in critically ill COVID-19 patients in the era of dexamethasone<ul><li> Lineages and clades were interpreted using Pangolin and <b>NextClade</b>, before being submitted to the GISAID database (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org">https://www.gisaid.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37111512">37111512</a> <b>(1)</b> SARS-CoV-2 Variants by Whole-Genome Sequencing in a University Hospital in Bangkok: First to Third COVID-19 Waves<ul><li> The consensus sequences were assessed using <b>NextClade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, version 1.9.0, accessed on 1 October 2021) for sequence clade assignment, identification quantification of mutations, and sequence quality analyses.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37151582">37151582</a> <b>(3)</b> NGS implementation for monitoring SARS-CoV-2 variants in Chicagoland: An institutional perspective, successes and challenges<ul><li> This labor-intensive effort was quickly abandoned when we began using more specialized open source software packages provided by Nextstrain, Pangolin, and <b>Nextclade</b>, reducing the necessity of manual analysis.<li> <b>Nextclade</b> (<xref rid="ref12" ref-type="bibr"><u>12</u></xref>) was also used to compare and summarize variant classifications by uploading our merged FASTA files (<xref rid="ref13" ref-type="bibr"><u>13</u></xref>).<li> Open source databases, like <b>NextClade</b> and Pangolin, are routinely updated and made available for analysis through the application, and the data is easily viewable and managed by multiple users.<li> [12](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling, and quality control for viral genomes</article-title>. <source>J Open Source Softw</source>. (<year>2021</year>) <volume>6</volume>:<fpage>3773</fpage>. doi: <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) [13](<person-group person-group-type="author"><collab id="coll1">Nextstrain</collab></person-group>. Clade assignment, mutation calling, and sequence quality checks. Available at: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link> (Accessed 02/01/2023)) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37081087">37081087</a> <b>(14)</b> Comparison of SARS-CoV-2 whole genome sequencing using tiled amplicon enrichment and bait hybridization<ul><li>The consensus sequences were annotated for virus clade information using <b>Nextclade</b> (NEXTstrain version 1.14.1) and Pangolin (pangolin lineage version v3.1.20 with pangoLEARN version 2022-04-28).<li> Consensus quality control (qc.overall and qc.missing) was extracted from the <b>Nextclade</b> output.<li> The first 300 missing sites are not penalized (<ext-link ext-link-type="uri" xlink:href="https://docs.nextstrain.org/projects/<b>nextclade</b>/en/stable/user/algorithm/07-quality-control.html">https://docs.nextstrain.org/projects/<b>nextclade</b>/en/stable/user/algorithm/07-quality-control.html</ext-link>).<li>Pangolin and <b>Nextclade</b> assessment and quality control metrics were generated from consensus sequences of SARS-CoV-2 (Supplementary Table <xref rid="MOESM1" ref-type="media"><u>2</u></xref>).<li> All Pangolin and <b>Nextclade</b> designations were in concordance across methods when reads were sufficient for variant classification.<li> For the Twist protocol, 42/47 samples were classified using Pangolin and 46/47 samples were classified using <b>Nextclade</b>.<li> according to <b>Nextclade</b> in Supplementary Table <xref rid="MOESM1" ref-type="media"><u>2</u></xref>).<li> Pangolin and <b>Nextclade</b> successfully classified all 47/47 ARTIC samples.<li> Using the Midnight protocol, Pangolin was able to classify 18/24 samples and <b>Nextclade</b> classified 21/24 samples.<li>The phylogeny, <b>Nextclade</b>, and Pangolin lineages for the consensus sequences are shown in Fig.&#xA0;<xref rid="Fig5" ref-type="fig"><u>5</u></xref>.<li> Pangolin lineage (Pango), <b>Nextclade</b> (Clade) and <b>Nextclade</b> quality control (QC) measures to flag potential problematic sequences, shown in circles.<li> Another three samples had&#x2009;&gt;&#x2009;20,000 reads but were assigned as bad quality according to <b>Nextclade</b>, and could not be assigned a lineage.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112997">37112997</a> <b>(6)</b> Epidemiological Surveillance Reveals the Rise and Establishment of the Omicron SARS-CoV-2 Variant in Brazil<ul><li>The consensus sequences were classified using the Pangolin tool v.4.1.3 [<xref rid="B40-viruses-15-01017" ref-type="bibr"><u>40</u></xref>] and the <b>NextClade</b> web application v.2.3.0 [<xref rid="B41-viruses-15-01017" ref-type="bibr"><u>41</u></xref>].<li> A Maximum Likelihood phylogeny was inferred using the IQ-tree program v2.0.3 [<xref rid="B43-viruses-15-01017" ref-type="bibr"><u>43</u></xref>] under the GTR+F+I+G4 model to corroborate the classification generated by Pangolin and <b>NextClade</b> [<xref rid="B44-viruses-15-01017" ref-type="bibr"><u>44</u></xref>].<li> The sequences were initially analyzed by Pangolin and <b>NextClade</b> tools.<li> Sequencing metrics and Pangolin/<b>Nextclade</b> classification are available in <xref rid="app1-viruses-15-01017" ref-type="app"><u>Supplementary Table S3</u></xref>.<li> To corroborate the results found in the Pangolin/<b>Nextclade</b> classification, we performed a maximum likelihood analysis.<li> Our phylogenic reconstruction confirmed the Pango/<b>NextClade</b> for all samples analyzed (<xref rid="viruses-15-01017-f003" ref-type="fig"><u>Figure 3</u></xref>).<li> [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [41](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [43](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=7932792">7932792</a>) </ul></p>
<p><u>No PMID</u> <b>(4)</b> A CRISPR-enhanced metagenomic NGS test to improve pandemic preparedness<ul><li> To evaluate clade assignment accuracy, we compared our consensus genome-based <b>NextClade</b><xref rid="bib22" ref-type="bibr"><sup>22</sup></xref> approach, for both mock and depleted samples from site A, against clade identification via an independent PCR-based method (Variant-Seq, PerkinElmer).<li> We applied a 5% genome breadth cutoff as the minimum requirement for accepting clade assignments from <b>Nextclade</b>.<li><hr/></td></tr><tr><td>Kraken2</td><td>Wood et&#xA0;al., 2019<xref rid="bib39" ref-type="bibr"><sup>34</sup></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/DerrickWood/kraken2/wiki/Manual" id="intref0010">https://github.com/DerrickWood/kraken2/wiki/Manual</ext-link></td></tr><tr><td>Bracken</td><td>Lu et&#xA0;al., 2017<xref rid="bib38" ref-type="bibr"><sup>35</sup></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://ccb.jhu.edu/software/bracken/index.shtml" id="intref0015">https://ccb.jhu.edu/software/bracken/index.shtml</ext-link></td></tr><tr><td>CZ ID</td><td>Chan Zuckerberg Biohub, San Francisco, CA</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/chanzuckerberg/idseq-workflows" id="intref0020">https://github.com/chanzuckerberg/idseq-workflows</ext-link></td></tr><tr><td><b>NextClade</b>.<li> The reconstructed genome sequence was used to identify SARS-CoV-2 clades using <b>Nextclade</b> (v1.9.0).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36927408">36927408</a> <b>(2)</b> Phylogenetic landscape of Monkeypox Virus (MPV) during the early outbreak in New York City, 2022<ul><li> GenBank accession number: NC_063383.1) using the <b>NextClade</b> tool v1.5.4 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<li>To identify nucleotide variations, the 23 genomes included in this study were compared with the reference genome NC_063383.1 using the <b>NextClade</b> tool v1.5.4.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37077184">37077184</a> <b>(1)</b> Temporal dynamics and fatality of SARS&#x2010;CoV&#x2010;2 variants in Bangladesh<ul><li> The clade and Pango lineages were assigned by using <b>Nextclade</b> v2.8.1 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37066129">37066129</a> <b>(1)</b> Influenza A(H3N2) infection followed by separate COVID-19 infection<ul><li>We used the <b>Nextclade</b> web tool v.2.6.21 (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) to reconstruct a phylogenetic tree with the four sequenced genomes and all of the 366 SARS-CoV-2 sequences from Rio Grande do Sul sampled during January 2022 and available in the GISAID (Global Initiative on Sharing All Influenza Data) database (<ext-link xlink:href="https://www.gisaid.org" ext-link-type="uri">https://www.gisaid.org</ext-link>) on April 19, 2022.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37052679">37052679</a> <b>(1)</b> A third vaccine dose equalises the levels of effectiveness and immunogenicity of heterologous or homologous COVID-19 vaccine regimens, Lyon, France, December 2021 to March 2022<ul><li> Clades and lineages were determined on samples with genome coverage&#x2009;&gt;&#x2009;90% using <b>Nextclade</b> and PangoLEARN, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37041231">37041231</a> <b>(1)</b> Structural basis of spike RBM-specific human antibodies counteracting broad SARS-CoV-2 variants<ul><li> Variant calling and annotation were performed using the <b>Nextclade</b> website (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37039557">37039557</a> <b>(0)</b> Evolution of the SARS-CoV-2 Mutational Spectrum<ul><li> 2021</xref>) from November 7, 2022 (<ext-link xlink:href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/11/07/public-2022-11-07.all.masked.<b>nextclade</b>.pangolin.pb.gz" ext-link-type="uri">http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/11/07/public-2022-11-07.all.masked.<b>nextclade</b>.pangolin.pb.gz</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37041143">37041143</a> <b>(1)</b> Comparative epidemic expansion of SARS-CoV-2 variants Delta and Omicron in the Brazilian State of Amazonas<ul><li> Genomes were evaluated for mutation calling and quality with <b>Nextclade</b> Web 1.13.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>)<sup><xref ref-type="bibr" rid="CR57"><u>57</u></xref></sup>.<li> [57](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open-Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112914">37112914</a> <b>(2)</b> Whole-Genome Characterization of SARS-CoV-2 Reveals Simultaneous Circulation of Three Variants and a Putative Recombination (20B/20H) in Pets, Brazzaville, Republic of the Congo<ul><li> <b>Nextclade</b> analysis classified it in clade 20B (B1533 lineage), while Bayesian phylogenetic analysis positions it in clade 20H (V2, B.1351, Beta variant) (<xref rid="viruses-15-00933-f004" ref-type="fig"><u>Figure 4</u></xref>).<li> This virus clustered within clade 20B according to <b>Nextclade</b> analyses, whereas phylogenetic analyses found it to cluster within the 20H clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37110367">37110367</a> <b>(1)</b> Novel Virus Air Sampler Based on Electrostatic Precipitation and Air Sampling of SARS-CoV-2<ul><li> The sequencing analysis of the S1 region was performed using the Sanger method [<xref rid="B21-microorganisms-11-00944" ref-type="bibr"><u>21</u></xref>], and lineages were determined based on the sequence data using the <b>Nextclade</b> web application [<xref rid="B22-microorganisms-11-00944" ref-type="bibr"><u>22</u></xref>].<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32074516">32074516</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37143627">37143627</a> <b>(1)</b> Clinical Characteristics of SARS-CoV-2 Omicron Cases in Pune, Maharashtra, India<ul><li> Lineage identification and clade analysis were performed using Pangolin (<ext-link xlink:href="https://github.com/hCoV-2019/pangolin" ext-link-type="uri">https://github.com/hCoV-2019/pangolin</ext-link>) and <b>Nextclade</b> software (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>/).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36696624">36696624</a> <b>(1)</b> Monkeypox Virus Infection in 2 Female Travelers Returning to Vietnam from Dubai, United Arab Emirates, 2022<ul><li> We determined viral lineage by using <b>NextClade</b> (<xref rid="R6" ref-type="bibr"><italic>6</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36754279">36754279</a> <b>(3)</b> Rapid Lineage Assignment of Severe Acute Respiratory Syndrome Coronavirus 2 Cases through Automated Library Preparation, Sequencing, and Bioinformatic Analysis<ul><li> The resulting consensus assemblies were then analyzed with Pangolin<xref rid="bib5" ref-type="bibr"><sup>5</sup></xref> (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0055"><italic>https://github.com/cov-lineages/pangolin</italic></ext-link>, last accessed December 29, 2022) and <b>NextClade</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" id="intref0060"><italic>https://github.com/nextstrain/<b>nextclade</b></italic></ext-link>, last accessed October 12, 2022) to perform lineage and clade designations, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37092436">37092436</a> <b>(1)</b> Whole Sequencing and Detailed Analysis of SARS-CoV-2 Genomes in Southeast Spain: Identification of Recurrent Mutations in the 20E (EU1) Variant with Some Clinical Implications<ul><li> Next, it performs lineage/clade analysis using Pangolin (version 3.1.7) and <b>NextClade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37000821">37000821</a> <b>(5)</b> Genomic surveillance identifies SARS-CoV-2 transmission patterns in local university populations, Wisconsin, USA, 2020&#x2013;2022<ul><li>The clade and lineage assignment for each of 537 SARS-CoV-2 sequences were uniformly performed using <b>Nextclade</b> Pango Classifier in <b>Nextclade</b> version web-1.14.1 [<xref rid="R25" ref-type="bibr"><u>25</u></xref>].<li> The existing SARS-CoV-2 sequences that were subsampled from the global dataset (Nextstrain, accessed on 10 October 2022) and 537 sequences from this study were illustrated in the <b>Nextclade</b> to discover the nature of clustering of sequences (Table S3).<li> The clades for 537 genome sequences of SARS-CoV-2 were identified in <b>Nextclade</b> and classified based on the sampling date.<li> Clades are colour coded following the naming convention and branching colours in <b>Nextclade</b>.<li> [25](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Source Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37002990">37002990</a> <b>(1)</b> Emergence and antibody evasion of BQ, BA.2.75 and SARS-CoV-2 recombinant sub-lineages in the face of maturing antibody breadth at the population level<ul><li> GISAID sequences were processed and assigned to clades via <b>NextClade</b> (using the BA.2 reference set, &#x201C;sars-cov-2-21L&#x2033;), and compiled into counts per country per lineage per day.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36975897">36975897</a> <b>(4)</b> Implementation and Evaluation of the Clear Dx Platform for Sequencing SARS-CoV-2 Genomes in a Public Health Laboratory<ul><li> The maximum likelihood circular tree showed two major clades, one consisting of three major clusters, including 21A (Delta), 21I (Delta), and 21J (Delta), and a second one consisting of two clusters, including 21K (Omicron) and 21L (Omicron) clade strains, following the naming convention and branching colors in <b>Nextclade</b>.<li> The corresponding Pango lineages and sublineages for each of these five <b>Nextclade</b>s were provided in the box.<li> The clades for 1,224 SARS-CoV-2 genome sequences were identified in <b>Nextclade</b> and classified based on the sampling date.<li> The <b>Nextclade</b>-based analysis confirmed the accuracy in clade assignment for these 12 specimens, indicating the efficiency and stability of Clear Dx in characterizing SARS-CoV-2 genomes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37051600">37051600</a> <b>(1)</b> A framework for real-time monitoring, analysis and adaptive sampling of viral amplicon nanopore sequencing<ul><li> The SNPs were called using <b>nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) with the output data available in <xref rid="s10" ref-type="sec"><u>Supplementary Files S3.3, S3.4</u></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36971577">36971577</a> <b>(1)</b> Complete Genome Sequences of Monkeypox Virus from a French Clinical Sample and the Corresponding Isolated Strain, Obtained Using Nanopore Sequencing<ul><li> Both viral genomic sequences belonged to lineage B.1 and clade IIb, according to <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) (<xref rid="B8" ref-type="bibr"><u>8</u></xref>).<li> [8](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><volume>6</volume>:<fpage>3773</fpage>. doi:<pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37112835">37112835</a> <b>(1)</b> Immunosuppression as a Hub for SARS-CoV-2 Mutational Drift<ul><li> Assignment to phylogenetic clades and quality control were performed by <b>Nextclade</b> 2.10.1 [<xref rid="B24-viruses-15-00855" ref-type="bibr"><u>24</u></xref>], with sequences missing over 3000 nucleotides removed from further analysis.<li> [24](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36972867">36972867</a> <b>(1)</b> Perspective on the application of genome sequencing for monkeypox virus surveillance<ul><li> The pipeline can be found at <ext-link ext-link-type="uri" xlink:href="https://github.com/BioWu/Monkeypox-genome-assembly" id="intref0010">https://github.com/BioWu/Monkeypox-genome-assembly</ext-link>.Variant analysis and molecular typing can be performed using <b>Nextclade</b> on the MPXV genome sequence (<xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>).<li> [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36974160">36974160</a> <b>(3)</b> Molecular evolutionary model based on phylogenetic and mutation analysis of SARS-CoV-2 spike protein sequences from Asian countries: A phylogenomic approach<ul><li> The nucleotide alignment obtained was subjected to <b>NextClade</b> v2.1.0 tool [<xref rid="bib30" ref-type="bibr"><u>30</u></xref>].<li> Further, the output obtained from <b>NextClade</b> was used to generate a maximum likelihood (ML) in IQ-TREE multicore v1.6.12 [<xref rid="bib31" ref-type="bibr"><u>31</u></xref>] for phylogenomic analysis, using the best substitution model and other default parameters.<li>The alignment output file was subjected to <b>NextClade</b> for the construction of phylogenetic tree.<li> [30](          <person-group person-group-type="author">            <name>              <surname>Salimi</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Gomar</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Heshmati</surname>              <given-names>B.</given-names>            </name>          </person-group>          <article-title>The COVID-19 outbreak in Iran</article-title>          <source>J. Glob. Health</source>          <volume>10</volume>          <issue>1</issue>          <year>2020</year>          <object-id pub-id-type="publisher-id">10365</object-id>        ) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35883172">35883172</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36992509">36992509</a> <b>(2)</b> Evaluation of SARS-CoV-2 ORF7a Deletions from COVID-19-Positive Individuals and Its Impact on Virus Spread in Cell Culture<ul><li> Lineage and genome variants were accessed using the web tool <b>Nextclade</b> (c) 2020&#x2013;2022 version 1.14.0 (Nextstrain developers) [<xref rid="B32-viruses-15-00801" ref-type="bibr"><u>32</u></xref>] through fasta files generated from NGS sequencing.<li>Nevertheless, data from the assembled virus genomes submitted to <b>Nextclade</b> on the Nextstrain website showed the same variants indicating that the viruses infecting all the family members belong to the B.1.1.33 lineage.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36993216">36993216</a> <b>(1)</b> Reduced control of SARS-CoV-2 infection is associated with lower mucosal antibody responses in pregnant women<ul><li> Clades were determined using <b>Nextclade</b> beta v 1.13.2 (<ext-link xlink:href="http://clades.nextstrain.org" ext-link-type="uri">clades.nextstrain.org</ext-link>, Last accessed March 30, 2022), and lineages were determined with Pangolin COVID-19 lineage Assigner [<xref rid="R19" ref-type="bibr"><u>19</u></xref>].<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36932280">36932280</a> <b>(1)</b> Subregional origins of emerging SARS-CoV-2 variants during the second pandemic wave in C&#xF4;te d&#x2019;Ivoire<ul><li> We next excluded sequences identified as being of low quality by <b>NextClade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>), those with missing sampling dates, those with&#x2009;&lt;&#x2009;90% coverage, those with&#x2009;&gt;&#x2009;40 SNPs, those with&#x2009;&gt;&#x2009;10 ambiguous base-calls per genome, and those with clustered SNPs.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36868218">36868218</a> <b>(0)</b> A pseudovirus system enables deep mutational scanning of the full SARS-CoV-2 spike<ul><li>The approach for calculating the enrichment of observed versus expected mutations is inspired by the strategy used in Neher<xref rid="bib75" ref-type="bibr"><sup>75</sup></xref> Briefly, we downloaded the pre-built UShER tree of all publicly available SARS-CoV-2 sequences<sup>25</sup> as of Sept-26-2022from <ext-link ext-link-type="uri" xlink:href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/09/26/public-2022-09-26.all.masked.<b>nextclade</b>.pangolin.pb.gz" id="intref0230">http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/09/26/public-2022-09-26.all.masked.<b>nextclade</b>.pangolin.pb.gz</ext-link>We then used matUtils25 to separate the tree by the pre-annotated Nextstrain clades, and extract the number of unique occurrences of each nucleotide mutation along the tree branches for each clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36933724">36933724</a> <b>(2)</b> Wastewater surveillance uncovers regional diversity and dynamics of SARS-CoV-2 variants across nine states in the USA<ul><li> The genomes were then processed by <b>Nextclade</b> CLI (v.2.4.0) (<xref rid="bb0010" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) which generates a multiple sequence alignment against the reference genome GenBank accession # MN908947; RefSeq ID <ext-link ext-link-type="uri" xlink:href="ncbi-n:NC_045512.2" id="ir0025">NC_045512.2</ext-link> (Wuhan-Hu-1/2019) and provides a list of SNVs, insertions, and deletions associated with each genome sequence.<li> applied by <b>Nextclade</b> and that passed pangolin quality control for lineage assignment were used for downstream analysis.<li> [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36989935">36989935</a> <b>(1)</b> Genomic characterization and molecular evolution of SARS-CoV-2 in Rio Grande do Sul State, Brazil<ul><li> The strains were identified using the dynamic nomenclature implemented in Pangolin (Rambaut, 2020) (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0035">https://github.com/cov-lineages/pangolin</ext-link>) and global clades and mutations using Nextstrain from <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0040">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36924839">36924839</a> <b>(2)</b> SARS-CoV-2 coinfection in immunocompromised host leads to the generation of recombinant strain<ul><li> The consensus sequences were submitted to Pangolin and <b>NextClade</b> for lineage assignment.<li> The sequence analysis with <b>NextClade</b> offered further confirmation, illustrating a clear breakpoint between ORF1ab and spike (approximate breakpoint site: 20418-21618).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36916974">36916974</a> <b>(1)</b> Transmission of SARS-CoV-2 Omicron Variant under a Dynamic Clearance Strategy in Shandong, China<ul><li> The genetic clade for each SARS-CoV-2 consensus sequence was determined using <b>Nextclade</b> v1.11.0 (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36980977">36980977</a> <b>(4)</b> Molecular Epidemiology and Diversity of SARS-CoV-2 in Ethiopia, 2020&#x2013;2022<ul><li> <b>Nextclade</b> was used to determine the quality control profiles of the sequences.<li> Using the quality control reports generated by <b>Nextclade</b>, all mutations flagged as private mutations were either confirmed or resolved by manually inspecting the BAM files in Geneious Prime.<li> <b>Nextclade</b> was further employed to assign clades and lineage classifications to the sequences [<xref rid="B16-genes-14-00705" ref-type="bibr"><u>16</u></xref>,<xref rid="B32-genes-14-00705" ref-type="bibr"><u>32</u></xref>].<li> Using the quality control reports generated by <b>Nextclade</b>, all mutations flagged as private mutations were either confirmed or resolved by manually inspecting the BAM files in Geneious Prime.<li> [16](          <person-group person-group-type="author"><name><surname>Shu</surname><given-names>Y.</given-names></name><name><surname>McCauley</surname><given-names>J.</given-names></name></person-group>          <article-title>GISAID: Global initiative on sharing all influenza data&#x2014;From vision to reality</article-title>          <source>Eurosurveillance Bull. Eur. Sur Les Mal. Transm.</source>          <year>2017</year>          <volume>22</volume>          <fpage>30494</fpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>        ) [32](          <person-group person-group-type="author"><collab>Nextclade</collab></person-group>          <article-title>Next Clade</article-title>          <year>2021</year>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-12-15">(accessed on 15 December 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36914815">36914815</a> <b>(21)</b> Benchmarking machine learning robustness in Covid-19 genome sequence classification<ul><li> There are other methods in the literature for SARS-CoV-2 subtyping, such as Covidex<sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup> and <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR21"><u>21</u></xref></sup>.<li>Some bioinformatics tools have been proposed in the literature for SARS-CoV-2 subtyping, such as Covidex<sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup> and <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR21"><u>21</u></xref></sup>.<li>We also use the tools such as Covidex<sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup> and <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR21"><u>21</u></xref></sup> for classifying the lineages.<li> The predictive results of Covidex and <b>Nextclade</b> are reported in Table&#xA0;<xref rid="Tab3" ref-type="table"><u>3</u></xref> for the original data.<li>Accuracy results on 8220 (original) nucleotide sequences (without any error) using Covidex and <b>Nextclade</b>.<li><b>Nextclade</b>.<li> The robustness results for Covidex and <b>Nextclade</b> are reported in Table&#xA0;<xref rid="Tab7" ref-type="table"><u>7</u></xref>.<li> Moreover, <b>Nextclade</b> failed for both depth 5 and depth 10 data completely.<li> The robustness results of Covidex and <b>Nextclade</b> for depth 15 and 20 datasets are reported in Table&#xA0;<xref rid="Tab9" ref-type="table"><u>9</u></xref>.<li>Robustness results on PBSIM data with 5 and 10 as read depth using Covidex and <b>Nextclade</b>.<li><b>Nextclade</b>.<li>Robustness results on PBSIM data with 15 and 20 as read depth using Covidex and <b>Nextclade</b>.<li><b>Nextclade</b>.<li> The robustness results for the same data using Covidex and <b>Nextclade</b> are reported in Table&#xA0;<xref rid="Tab11" ref-type="table"><u>11</u></xref>.<li> Opposite to the results for the PBSIM simulator, we can observe that both Covidex and <b>Nextclade</b> show higher robustness results compared to the embedding methods.<li> Moreover, the robustness results using Covidex and <b>Nextclade</b> on the same datasets are reported in Table&#xA0;<xref rid="Tab13" ref-type="table"><u>13</u></xref>.<li> Other than Macro F1, both Covidex and <b>Nextclade</b> outperforms the embedding methods for all other evaluation metrics, hence showing better generalizability over the noisy sequences.<li>Robustness results on Covidex and <b>Nextclade</b> using illumina-based errored sequences with 5000 and 10,&#xA0;000 short reads used in the simulation process.<li><b>Nextclade</b>.<li>Robustness results on Covidex and <b>Nextclade</b> using illumina-based errored sequences with 15,000 and 20,000 short reads used in the simulation process.<li><b>Nextclade</b>.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35231666">35231666</a>) [21](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35231666">35231666</a>) [21](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35231666">35231666</a>) [21](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36897843">36897843</a> <b>(3)</b> The clinical characteristics of pediatric patients infected by SARS-CoV-2 Omicron variant and whole viral genome sequencing analysis<ul><li> All of the analyzable strains belonged to the PANGO lineage B.1.1.529 as classified by Pangolin (version 4.0.2) and <b>Nextclade</b> clade 21L (version 1.11.0), Omicron variant.<li> Phylogenetic clades of the consensus genomes were mapped using <b>Nextclade</b> (version 1.11.0) [<xref rid="pone.0282389.ref014" ref-type="bibr"><u>14</u></xref>].<li> All of the strains belonged to the PANGO lineage B.1.1.529 (b) and <b>Nextclade</b> clade 21L (a), Omicron variant.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36980953">36980953</a> <b>(4)</b> High Throughput SARS-CoV-2 Genome Sequencing from 384 Respiratory Samples Using the Illumina COVIDSeq Protocol<ul><li> All nucleotide and deduced amino acid sequences were compared to Wuhan-Hu-1 isolate genome, and sequence modifications were detected using <b>Nextclade</b> tool [<xref rid="B26-genes-14-00681" ref-type="bibr"><u>26</u></xref>,<xref rid="B27-genes-14-00681" ref-type="bibr"><u>27</u></xref>,<xref rid="B28-genes-14-00681" ref-type="bibr"><u>28</u></xref>].<li> Nextstrain clades and Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) were identified with the <b>Nextclade</b> web application [<xref rid="B26-genes-14-00681" ref-type="bibr"><u>26</u></xref>,<xref rid="B27-genes-14-00681" ref-type="bibr"><u>27</u></xref>,<xref rid="B28-genes-14-00681" ref-type="bibr"><u>28</u></xref>] and Pangolin tool [<xref rid="B29-genes-14-00681" ref-type="bibr"><u>29</u></xref>,<xref rid="B30-genes-14-00681" ref-type="bibr"><u>30</u></xref>], respectively.<li>Of 376 samples processed for sequencing with the modified Illumina COVIDSeq protocol, 356 (94.7%) yielded interpretable genome sequences as defined by <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 2 September 2022).<li>Of 376 samples sequenced using the standard Illumina COVIDSeq protocol, 357 (94.9%) yielded interpretable genome sequences as defined by <b>Nextclade</b>.<li> [26](          <article-title>Nextclade Web 2.12.0, Nextclade CLI 2.12.0</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-09-12">(accessed on 12 September 2022)</date-in-citation>        ) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [28](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [26](          <article-title>Nextclade Web 2.12.0, Nextclade CLI 2.12.0</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-09-12">(accessed on 12 September 2022)</date-in-citation>        ) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [28](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [29](          <article-title>Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN)</article-title>          <comment>Available online: <ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-09-12">(accessed on 12 September 2022)</date-in-citation>        ) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36892291">36892291</a> <b>(1)</b> SARS-CoV-2 Exposure in Norway Rats (<named-content content-type="genus-species">Rattus norvegicus</named-content>) from New York City<ul><li> Nucleotide sequences were aligned using MAFFT v7.471, and the mutations were analyzed using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36995936">36995936</a> <b>(11)</b> Rapid escape of new SARS-CoV-2 Omicron variants from BA.2-directed antibody responses<ul><li> BA.1 caused the first Omicron wave, closely followed by a combination of BA.1 sub-lineages BA.1.1 and BA.2 in early 2021 (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/lineage_list" id="intref0045">https://cov-lineages.org/lineage_list</ext-link>, <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/<b>nextclade</b>/sars-cov-2" id="intref0050">https://nextstrain.org/<b>nextclade</b>/sars-cov-2</ext-link>).<li> BA.2 variants (including derivatives of BA.5)&#x2014;variants with long phylogenetic branch lengths, multiple antigenic mutations, and a lack of genetic intermediates, for example BA.2.75, BJ.1, BS.1, BA.2.10.4, and BA.2.3.20,<xref rid="bib19" ref-type="bibr"><sup>19</sup></xref> and (2) antigenic drift, seen both in BA.5<xref rid="bib24" ref-type="bibr"><sup>24</sup></xref> and within these second-generation BA.2 lineages, notably BQ.1 and BA.2.75 (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/<b>nextclade</b>/sars-cov-2/21L" id="intref0060">https://nextstrain.org/<b>nextclade</b>/sars-cov-2/21L</ext-link>) (<xref rid="fig1" ref-type="fig"><u>Figures&#xA0;1</u></xref>A&#x2013;1C).<li> The rooted phylogenetic tree<xref rid="bib26" ref-type="bibr"><sup>26</sup></xref> (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/<b>nextclade</b>/sars-cov-2" id="intref0075">https://nextstrain.org/<b>nextclade</b>/sars-cov-2</ext-link>) shown in <xref rid="fig1" ref-type="fig"><u>Figure&#xA0;1</u></xref>C provides a helpful grouping of the lineages.<li>The phylogenetic tree was generated by pruning the <b>Nextclade</b> reference tree (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/<b>nextclade</b>/sars-cov-2/21L" id="intref0215">https://nextstrain.org/<b>nextclade</b>/sars-cov-2/21L</ext-link>) which contains one sequence per Pango lineage.<li> The workflow is available at: <ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/<b>nextclade</b>_data_workflows/tree/27bf7e0b4f62cbbbc9a8ac96db1587cd76b3ae10" id="intref0220">https://github.com/neherlab/<b>nextclade</b>_data_workflows/tree/27bf7e0b4f62cbbbc9a8ac96db1587cd76b3ae10</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36992400">36992400</a> <b>(1)</b> Systematic Guidelines for Effective Utilization of COVID-19 Databases in Genomic, Epidemiologic, and Clinical Research<ul><li> The WHO, Pangolin, and <b>Nextclade</b>, powered by Nextstrain, are considered to be the main classifying entities [<xref rid="B26-viruses-15-00692" ref-type="bibr"><u>26</u></xref>,<xref rid="B27-viruses-15-00692" ref-type="bibr"><u>27</u></xref>,<xref rid="B28-viruses-15-00692" ref-type="bibr"><u>28</u></xref>].<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33735881">33735881</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34149046">34149046</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36863763">36863763</a> <b>(1)</b> SARS-CoV-2 infection in newborn infants: descriptive epidemiological analysis of cases reported to the Colombian national surveillance system during the first pandemic year, March 2020&#x2013;February 2021<ul><li> The spatiotemporal distribution of SARS-CoV-2 lineages circulating in Colombia between March 2020 and February 2021 was determined following the National Programme for the Genomic Characterisation of SARS-CoV-2 based on representativeness and virological criteria for probabilistic sampling.<xref rid="R10" ref-type="bibr"><u>10</u></xref> Sequences and metadata from the Global Initiative on Sharing Avian Influenza Data for SARS-CoV-2 were obtained, and samples were processed by the National Genomic Characterisation Programme for SARS-CoV-2 in Colombia using Oxford Nanopore Technology.<xref rid="R11" ref-type="bibr"><u>11</u></xref> The sequence lineage was estimated with Pangolin.<xref rid="R12" ref-type="bibr"><u>12</u></xref> Other sequence information, such as insertions and deletions (INDELs), single nucleotide polymorphisms (SNPs), Clade and Lineage, was obtained using <b>NextClade</b>.<li> [10](<person-group person-group-type="author"><string-name><surname>Mercado-Reyes</surname><given-names>M</given-names></string-name>, <string-name><surname>Laiton-Donato</surname><given-names>K</given-names></string-name>, <string-name><surname>Franco-Mu&#xF1;oz</surname><given-names>C</given-names></string-name>, <etal>et al</etal></person-group>. <source>Estrategia de caracterizaci&#xF3;n gen&#xF3;mica por muestreo probabil&#xED;stico para SARS-cov-2 en</source>. <publisher-loc>Colombia</publisher-loc>: <publisher-name>Primer muestreo</publisher-name>, <year>2021</year>: <fpage>1</fpage>&#x2013;<lpage>10</lpage>. <comment>Available</comment>: <uri xlink:href="http://www.ins.gov.co/BibliotecaDigital/Estrategia-caracterizacion-genomica-muestreo-probabilistico-SARS-CoV-2-Primer-muestreo.pdf">http://www.ins.gov.co/BibliotecaDigital/Estrategia-caracterizacion-genomica-muestreo-probabilistico-SARS-CoV-2-Primer-muestreo.pdf</uri>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34403832">34403832</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36732066">36732066</a> <b>(1)</b> Genomic Analysis of Early Monkeypox Virus Outbreak Strains, Washington, USA<ul><li> We used Augur, Auspice, and <b>Nextclade</b> to perform phylogenetic analysis (<xref rid="R4" ref-type="bibr"><italic>4</italic></xref>,<xref rid="R5" ref-type="bibr"><italic>5</italic></xref>), and we used UShER (<xref rid="R6" ref-type="bibr"><italic>6</italic></xref>) to perform phylogenetic placement on a global tree (<xref rid="SD1" ref-type="supplementary-material"><u>Appendix</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36979680">36979680</a> <b>(1)</b> Evaluation of Four Rapid Antigen Tests for the Detection of SARS-CoV-2 Infection with Nasopharyngeal Swabs<ul><li> Genetic mutations in <italic toggle="yes">N</italic> were identified by <b>Nextclade</b> (ver.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36820641">36820641</a> <b>(2)</b> SARS-CoV-2 molecular epidemiology in Slovenia, January to September 2021<ul><li> SARS-CoV-2 variants were called with locally installed Pangolin (daily updated version [<xref rid="r5" ref-type="bibr"><u>5</u></xref>]) and mutations were obtained with online <b>Nextclade</b> analysis [<xref rid="r6" ref-type="bibr"><u>6</u></xref>].<li> We present amino acid substitutions and deletions based on <b>Nextclade</b> analysis (v1.7.4).<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [6](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>R</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source>J Open Source Softw</source>. <year>2021</year>;<volume>6</volume>(<issue>67</issue>):<fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36746267">36746267</a> <b>(2)</b> Monkeypox virus (MPXV) genomics: A mutational and phylogenomic analyses of B.1 lineages<ul><li> This consists of an alignment using <b>NextClade</b> v2.4.2 tool [<xref rid="bib11" ref-type="bibr"><u>11</u></xref>], geographic analysis of B.1 lineages, mutational analysis of coding regions, especially for those genes in which unique mutations have been found for the MPXV B.1 lineages, and analysis on the phylogenomic relationships of genome sequences publicly available.<li> For phylogenomic analysis, the aligned dataset obtained from <b>NextClade</b> was used to build a maximum likelihood (ML) in IQ-TREE multicore v1.6.12 [<xref rid="bib12" ref-type="bibr"><u>12</u></xref>], using as the best substitution model and other parameters by default.<li> [11](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Source Softw</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36992320">36992320</a> <b>(2)</b> Dissecting Phenotype from Genotype with Clinical Isolates of SARS-CoV-2 First Wave Variants<ul><li> 20 as determined using <b>NextClade</b> v.0.13.0 [<xref rid="B33-viruses-15-00611" ref-type="bibr"><u>33</u></xref>,<xref rid="B34-viruses-15-00611" ref-type="bibr"><u>34</u></xref>].<li> OL411643-OL411673) with <b>NextClade</b> [<xref rid="B34-viruses-15-00611" ref-type="bibr"><u>34</u></xref>] quality scores ranging from 0&#x2013;18 (<xref rid="viruses-15-00611-f001" ref-type="fig"><u>Figure 1</u></xref>D).<li> [33](          <article-title>NextClade</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-12-15">(accessed on 15 December 2021)</date-in-citation>        ) [34](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [34](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36963002">36963002</a> <b>(1)</b> Implementation of genomic surveillance of SARS-CoV-2 in the Caribbean: Lessons learned for sustainability in resource-limited settings<ul><li> CoV-GLUE (<ext-link xlink:href="http://cov-glue.cvr.gla.ac.uk/#/home" ext-link-type="uri">http://cov-glue.cvr.gla.ac.uk/#/home</ext-link>), UShER [<xref rid="pgph.0001455.ref014" ref-type="bibr"><u>14</u></xref>] and <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) were used for exploratory analysis prior to reporting results to the originating institutions, for example to inspect mutations, resolve ambiguities, and check for long branches.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36968876">36968876</a> <b>(2)</b> Clinical Characteristics and Outcomes of Laboratory-Confirmed SARS-CoV-2 Cases Infected With Omicron Subvariants and the XBB Recombinant Variant<ul><li> Lineage analysis was done using Phylogenetic Assignment of Named Global Outbreak LINeages (PangoLIN) COVID-19 lineage assigner, version v4.1.3, pangolin-data version v1.17, and clade was analyzed using <b>Nextclade</b> software, version 2.9.1.<li><b>Nextclade</b> Clade assigned.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36992302">36992302</a> <b>(2)</b> Combined Use of RT-qPCR and NGS for Identification and Surveillance of SARS-CoV-2 Variants of Concern in Residual Clinical Laboratory Samples in Miami-Dade County, Florida<ul><li> The second part utilized the <b>Nextclade</b> web-based software (all versions: 10.5281/zenodo.5726680) [<xref rid="B42-viruses-15-00593" ref-type="bibr"><u>42</u></xref>].<li> <b>Nextclade</b> performs alignment of the FASTA file against the SARS-CoV-2 reference and quality control, calls variants, and assigns clades for each sample.<li> [42](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32546606">32546606</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36845037">36845037</a> <b>(3)</b> One-year monitoring SARS-CoV-2 RNA surface contamination in hospitals reveals no correlation with organic material and negative pressure as a limiting factor for contamination<ul><li> PANGO lineages were determined using Pangolin v3.1.5 [<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0020">https://github.com/cov-lineages/pangolin</ext-link>, <ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1038/s41564-020-0770-5" id="intref0025">https://doi.org/10.1038/s41564-020-0770-5</ext-link>] and Nextstrain clade assignment was performed using <b>Nextclade</b> CLI v1.10.3 [<ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" id="intref0030">https://github.com/nextstrain/<b>nextclade</b></ext-link>, <ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1093/bioinformatics/bty407" id="intref0035">https://doi.org/10.1093/bioinformatics/bty407</ext-link>].</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37009519">37009519</a> <b>(1)</b> Genomic Surveillance for SARS-CoV-2 &#x2014; China, September 26, 2022 to January 29, 2023<ul><li> Qiagen, Germany) and <b>Nextclade</b> webservers.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36800345">36800345</a> <b>(1)</b> Transcriptionally active nasopharyngeal commensals and opportunistic microbial dynamics define mild symptoms in the COVID 19 vaccination breakthroughs<ul><li> Clade assignment to all the genomes was done using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36824718">36824718</a> <b>(1)</b> Accelerated evolution of SARS-CoV-2 in free-ranging white-tailed deer<ul><li> The dataset was aligned using <b>NextClade</b> with Wuhan-Hu-1 as a reference.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36796985">36796985</a> <b>(1)</b> Genomic epidemiology of SARS-CoV-2 infections in The Gambia: an analysis of routinely collected surveillance data between March, 2020, and January, 2022<ul><li> Lineages were assigned with Pangolin and quality of the genomes was assessed with <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36860855">36860855</a> <b>(1)</b> Risk of hospitalization and vaccine effectiveness among COVID-19 patients in the UAE during the Delta and Omicron outbreaks<ul><li> Further quality checks were undertaken using <b>Nextclade</b> 0.14.1 default quality control analyses, that is those for missing data, mixed sites, private mutations, mutation clusters, stop codons, and frameshifts.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36844451">36844451</a> <b>(4)</b> Delta-Omicron recombinant escapes therapeutic antibody neutralization<ul><li><hr/></td></tr><tr><td>GraphPad Prism 8.0e</td><td>GraphPad Prism Software, Inc.</td><td>N/A</td></tr><tr><td>IQ-TREE v2.2.0</td><td>IQ-TREE</td><td>N/A</td></tr><tr><td>BEAST v1.10.5</td><td>BEAST</td><td>N/A</td></tr><tr><td>Nextstrain CLI v6.0.0</td><td>Nextstrain</td><td>N/A</td></tr><tr><td><b>Nextclade</b> v1.14.<li><hr/></td></tr><tr><td>GraphPad Prism 8.0e</td><td>GraphPad Prism Software, Inc.</td><td>N/A</td></tr><tr><td>IQ-TREE v2.2.0</td><td>IQ-TREE</td><td>N/A</td></tr><tr><td>BEAST v1.10.5</td><td>BEAST</td><td>N/A</td></tr><tr><td>Nextstrain CLI v6.0.0</td><td>Nextstrain</td><td>N/A</td></tr><tr><td><b>Nextclade</b> v1.14</td><td><b>Nextclade</b>.<li> The web-based <b>Nextclade</b> v1.14<xref rid="bib40" ref-type="bibr"><sup>40</sup></xref> and Auspice v2.36 phylogenomic visualization features of Nextstrain<xref rid="bib14" ref-type="bibr"><sup>14</sup></xref> were used to examine viral genome clade assignment and mutation calling.<li> Mutations were identified with <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0030">https://clades.nextstrain.org/</ext-link>) and lineage assignment was performed with Pangolin v3.1.19 (pangoLearn 2022-01-22)<xref rid="bib22" ref-type="bibr"><sup>22</sup></xref> for both consensus genomes, which identified the first part of the recombinant genome as AY.45, i.e., a sublineage of the Delta variant of concern (VOC), and the second part as Omicron BA.1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36779717">36779717</a> <b>(1)</b> Coding-Complete Genome Sequences of 40 SARS-CoV-2 Omicron XBB, XBB.1, and XBB.2 Sublineage Strains in Bangladesh<ul><li> however, the sequenced reads were assembled utilizing the ARTIC gupplyplex code script with Medaka v1.4 using the EPI2ME desktop agent v3.5.7 (FastQ quality control plus ARTIC plus <b>NextClade</b>) (<ext-link xlink:href="https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html" ext-link-type="uri">https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36779742">36779742</a> <b>(1)</b> Severe Acute Respiratory Syndrome Coronavirus-2 Lambda Variant Collected from a Child from Arkansas and Sequenced<ul><li><b>Nextclade</b> v1.10.2 (<xref rid="B7" ref-type="bibr"><u>7</u></xref>) with default parameters was used to characterize mutations and amino acid changes in the consensus sequence generated from both Nanopore and Illumina sequencing and identified all 7 of the synonymous mutations in the sample from both sequencing platforms.<li> [7](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>IR</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><volume>6</volume>:<fpage>3773</fpage>. doi:<pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36781860">36781860</a> <b>(1)</b> Emergence and spread of two SARS-CoV-2 variants of interest in Nigeria<ul><li> <b>Nextclade</b> v1.3.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) was used to assign the sequences to globally circulating viral clades and to investigate mutations in the genome that could affect primer amplification during PCR.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36834888">36834888</a> <b>(1)</b> Accurate Detection of SARS-CoV-2 by Next-Generation Sequencing in Low Viral Load Specimens<ul><li> automatic plugin (Thermo Fisher Scientific), as well as the <b>Nextclade</b> tool (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>), against the reference genome from Wuhan (GenBank accession number NC_045512.2), to achieve the complete viral genome sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36797096">36797096</a> <b>(3)</b> Persistent SARS-COV-2 infection in vaccinated individual with three doses of COVID-19 vaccine<ul><li> Lineage prediction and variant calling was performed using Pangolin v4.1.2 (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="ir020">https://github.com/cov-lineages/pangolin</ext-link>) and <b>Nextclade</b> v2.6.0 (<ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" id="ir025">https://github.com/nextstrain/<b>nextclade</b></ext-link>) software.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36773619">36773619</a> <b>(2)</b> Characterisation of SARS-CoV-2 variants in Beijing during 2022: an epidemiological and phylogenetic analysis<ul><li>The evaluation of the quality of genomes, genomic alignment, clade, and Pango lineage assignment, and the genetic variation annotations of SARS-CoV-2 genomes were performed by <b>Nextclade</b> version 2.9.1.<li> A phylogenetic tree containing the remaining SARS-CoV-2 genomes was also reconstructed by <b>Nextclade</b> version 2.9.1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36758698">36758698</a> <b>(1)</b> Identification and genome sequencing of an influenza H3N2 variant in wastewater from elementary schools during a surge of influenza A cases in Las Vegas, Nevada<ul><li> The segment 1 consensus sequences were analyzed with <b>Nextclade</b> (v.1.8.1) and the phylogenetic tree was visualized on the Auspice webserver.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36825152">36825152</a> <b>(1)</b> Variant-specific deleterious mutations in the SARS-CoV-2 genome reveal immune responses and potentials for prophylactic vaccine development<ul><li>Sequence analysis, alignment, and clustering were performed using <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>), an advanced tool for SARS-CoV-2 sequence analysis (<xref rid="B4" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>).<li> [Aksamentov et al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group> (<year>2021</year>). <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J. Open Source Softw.</source><volume>6</volume> (<issue>67</issue>), <fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36750714">36750714</a> <b>(15)</b> Magnetic hydrogel particles improve nanopore sequencing of SARS-CoV-2 and other respiratory viruses<ul><li> <b>Nextclade</b> and Pangolin analysis was performed through the EPI2ME Labs developed ARTIC SARS-CoV-2 Workflow<sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup>.<li> to perform Pangolin and <b>Nextclade</b> analysis to further characterize the potential improvement gained by magnetic hydrogel particle (Nanotrap particle) enrichment.<li> Similar to Pangolin, <b>Nextclade</b> is a multi-tiered alignment tool that compares an input sequence against the Wuhan sequence identifying mutations that best match previously detected variants, resulting in: a lineage report, mutation count, an alignment map indicating genome coverage, and a general QC metric of the data being analyzed.<li> Samples underwent sequencing on a ONT MinION R.9 flow cell and were analyzed by Pangolin <bold>(a)</bold>, and <b>Nextclade</b>, which assessed: QC score comparison of all 10 samples <bold>(b)</bold>, total genome locations unable to be characterized per sample (missing n&#x2019;s, less&#x2009;=&#x2009;better) <bold>(c)</bold>, total mutations detected per sample <bold>(d)</bold>.<li><b>Nextclade</b> analysis revealed a similar trend with the use of Nanotrap particles enabling more samples to be characterized at a higher quality rating.<li> <b>Nextclade</b> uses a three tier QC system: &#x201C;Good&#x201D;, &#x201C;Mediocre&#x201D;, and &#x201C;Bad&#x201D;.<li> The QC tier indicates how much of the data provided was able to be characterized by <b>Nextclade</b> to accurately assess the viral genome and determine an accurate classification.<li> Furthermore, Nanotrap particle processing improved <b>Nextclade</b> analysis by decreasing the amount of missing bases by an average of 15,556 &#x201C;n&#x201D;s, or the number of viral genome locations unable to be characterized by <b>Nextclade</b> (Fig.&#xA0;<xref rid="Fig5" ref-type="fig"><u>5</u></xref>C).<li> also contribute to the <b>Nextclade</b> quality rating.<li> The fewer &#x201C;missing n&#x2019;s&#x201D;, the more accurate and useful the <b>Nextclade</b> tool becomes allowing for the analysis of a greater portion of the SARS-CoV-2 genome.<li> For Nanotrap particle processed samples, <b>Nextclade</b> detected an average of 9.2 more mutations per sample compared to samples processed without Nanotrap particles (Fig.&#xA0;<xref rid="Fig5" ref-type="fig"><u>5</u></xref>D).<li>To further assess the practical utility of the improvement seen when using the Nanotrap particles, we also analyzed sequencing data using Pangolin and <b>Nextclade</b>, two bioinformatics tools that are commonly used globally to track and analyze SARS-CoV-2 variants<sup><xref ref-type="bibr" rid="CR32"><u>32</u></xref>&#x2013;<xref ref-type="bibr" rid="CR34"><u>34</u></xref></sup>.<li> We observed that for a majority of the diagnostic remnant samples tested, the Nanotrap particle process enabled these bioinformatics tools to work more effectively and, for certain low-titer samples, allowed both Pangolin and <b>Nextclade</b> to complete a successful analysis that would have otherwise failed due to insufficient data.<li> <b>Nextclade</b> was also able to better map the entire SARS-CoV-2 genome for Nanotrap samples, identifying more total mutations per sample relative to the [&#x2212;&#x2009;NT] control.<li> [26](EPI2ME Labs Workflows. <italic>EPI2ME Labs</italic><ext-link ext-link-type="uri" xlink:href="https://labs.epi2me.io/wfindex/">https://labs.epi2me.io/wfindex/</ext-link> (2020).) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [34](Nextclade. <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36747014">36747014</a> <b>(1)</b> Simultaneous detection of omicron and other SARS-CoV-2 variants by multiplex PCR MassARRAY technology<ul><li> SARS-CoV-2 lineages and variants were determined using Pangolin v3.1.20 with lineages version 2022-02-28<sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>, GISAID clade nomenclature<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup> and <b>Nextclade</b> v1.14.0<sup><xref ref-type="bibr" rid="CR29"><u>29</u></xref></sup>.<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36778275">36778275</a> <b>(2)</b> Investigations on SARS-CoV-2 and other coronaviruses in mink farms in France at the end of the first year of COVID-19 pandemic<ul><li> EPI_ISL_10036487&#x2013;97) were classified as 20A based on a panel of mutations compared to reference sequence NC_045512 and a <b>Nextclade</b> analysis (Fig.<li>All twelve SARS-CoV-2 genome sequences obtained, despite some variability, belonged to the 20A clade, according to <b>Nextclade</b> and two Bayesian phylogenetic reconstructions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36692451">36692451</a> <b>(4)</b> Early Introduction and Community Transmission of SARS-CoV-2 Omicron Variant, New York, New York, USA<ul><li> we used consensus sequences of these viruses as input to <b>Nextclade</b> version 1.10.1 (<xref rid="R19" ref-type="bibr"><italic>19</italic></xref>) for quality control, mutation calling, and Nextstrain clade assignment.<li> Viruses &lt;29,000 nt in length or with <b>Nextclade</b>-assessed QC.overallStatus below good were considered low quality and removed.<li> We performed mutation calling of these contextual sequences using <b>Nextclade</b> version 1.10.1 (<xref rid="R19" ref-type="bibr"><italic>19</italic></xref>).<li> Sequences that were &lt;29,000 nt long or had <b>Nextclade</b>-assessed QC.overallStatus value below good were considered low quality and removed.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36819024">36819024</a> <b>(3)</b> Molecular determinants associated with temporal succession of SARS-CoV-2 variants in Uttar Pradesh, India<ul><li>The trimmed sequences were checked in Pango, <b>Nextclade</b> and GISAID-CoVsurver mutation app for clade and lineage characterization.<li> <b>Nextclade</b> works by identifying the differences between the query sequence and the original SARS-CoV-2 Wuhan sequence to identify matches and mutations and characterizes the clades.<li><b>Nextclade</b> and GISAID majorly annotated G142D, V143del, Y144del, Y145del, E156G, F157del, R158del, Q173R, A222V, A262S, Q414K, G446V, L452R, R454K, T478K, E484Q, S494P, P499R, D614G, Q677H, P681R, R683W, D950N, S1242I mutations in the Spike glycoprotein of 21A Delta.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36778462">36778462</a> <b>(0)</b> Fitness effects of mutations to SARS-CoV-2 proteins<ul><li>We counted occurrences of each mutation in each viral clade using the UShER pre-built mutation-annotated tree [<xref rid="R28" ref-type="bibr"><u>28</u></xref>, <xref rid="R29" ref-type="bibr"><u>29</u></xref>, <xref rid="R30" ref-type="bibr"><u>30</u></xref>] from Dec-18-2022 (<ext-link xlink:href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/12/18/public-2022-12-18.all.masked.<b>nextclade</b>.pangolin.pb.gz" ext-link-type="uri">http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/12/18/public-2022-12-18.all.masked.<b>nextclade</b>.pangolin.pb.gz</ext-link>), which contains all ~6.5-million SARS-CoV-2 sequences that are available in public databases.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34469548">34469548</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) [30](<name><surname>Lanfear</surname><given-names>R.</given-names></name><article-title>A global phylogeny of SARS-CoV-2 sequences from GISAID</article-title>. <source>Zenodo</source>. <year>2020</year>;DOI <pub-id pub-id-type="doi">10.5281/zenodo.3958883</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36032980">36032980</a> <b>(2)</b> Demographic and Viral-Genetic Analyses of COVID-19 Severity in Bahrain Identify Local Risk Factors and a Protective Effect of Polymerase Mutations<ul><li> We then filtered sequences based on quality summaries implemented in <b>Nextclade</b> using default parameters for SARS-CoV-2 (<xref rid="R2" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>, <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).<li>We assigned samples to phylogenetic clades using Nextstrain clade definitions as implemented in the <b>Nextclade</b> software (<xref rid="R2" ref-type="bibr">Aksamentov et al.<li> [Aksamentov et al., 2021](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <name><surname>Roemer</surname><given-names>C</given-names></name>, <name><surname>Hodcroft</surname><given-names>EB</given-names></name>, &amp; <name><surname>Neher</surname><given-names>RA</given-names></name>, (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source>, <volume>6</volume>(<issue>67</issue>), <fpage>3773</fpage>, <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851588">36851588</a> <b>(1)</b> SARS-CoV-2 Genome Variations in Viral Shedding of an Immunocompromised Patient with Non-Hodgkin&#x2019;s Lymphoma<ul><li> Oxford Nanopore Raw Data Processing and Sequencing Data Quality Assessment Basecalling and barcode demultiplexing were performed with EPI2ME Agent v3.5.4 using the Fastq QC+ARTIC+<b>NextClade</b> app.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851150">36851150</a> <b>(3)</b> COVID-19 in a Pre-Omicron Era: A Cross-Sectional Immuno-Epidemical and Genomic Evaluation<ul><li> algorithms (developed by the authors) and the &#x201C;<b>Nextclade</b>&#x201D; application were used.<li> Thus, the genomes were inserted into the &#x201C;<b>Nextclade</b>&#x201D; software that compared them to the genome of the original SARS-CoV-2 Wuhan strain and then provided a TSV file containing information about the mutations that occurred, besides the entropy of genome information and the phylogenetic tree.<li> script was responsible for processing the TSV file generated by &#x201C;<b>Nextclade</b>&#x201D;.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36768752">36768752</a> <b>(2)</b> Detection of High Level of Co-Infection and the Emergence of Novel SARS CoV-2 Delta-Omicron and Omicron-Omicron Recombinants in the Epidemiological Surveillance of Andalusia<ul><li>Lineage and clade assignment to each consensus genome was generated by the Pangolin [<xref rid="B28-ijms-24-02419" ref-type="bibr"><u>28</u></xref>] (pangolin v4.0.6, pangolin-data v1.8) and <b>Nextclade</b> [<xref rid="B29-ijms-24-02419" ref-type="bibr"><u>29</u></xref>] (Version 1.11.0) tools, respectively.<li>Known recombinant variants were detected using the <b>Nextclade</b> [<xref rid="B29-ijms-24-02419" ref-type="bibr"><u>29</u></xref>] (Version 1.11.0) tool.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [29](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [29](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36761900">36761900</a> <b>(1)</b> Photodynamic nasal SARS-CoV-2 decolonization shortens infectivity and influences specific T-Cell responses<ul><li> The phylogenetic tree was performed with <b>Nextclade</b>.org.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851541">36851541</a> <b>(1)</b> Retrospective Insights of the COVID-19 Epidemic in the Major Latin American City, S&#xE3;o Paulo, Southeastern Brazil<ul><li> The quality of the consensus genome sequences was finally assessed using <b>Nextclade</b> [<xref rid="B25-viruses-15-00327" ref-type="bibr"><u>25</u></xref>].<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36755883">36755883</a> <b>(1)</b> Mutational characterization of Omicron SARS-CoV-2 lineages circulating in Chhattisgarh, a central state of India<ul><li> The <b>Nextclade</b> software analysis of 108 sequences of SARS-CoV-2 revealed 91 (84%) sequences belonged to clade 21L, 12 (11%) sequences to clade 22B, and five (5%) sequences to clade 22D (<xref rid="F1" ref-type="fig"><u>Figure 1A</u></xref>) (<xref rid="B10" ref-type="bibr"><u>10</u></xref>).<li> [10](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J Open Source Softw.</italic></source> (<year>2021</year>) <volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36700580">36700580</a> <b>(1)</b> Clinical and epidemiological aspects of Delta and Gamma SARS-CoV-2 variant of concern from the western Brazilian Amazon<ul><li> The tree was visualised and edited with FigTree v.1.4.4.<xref rid="B15" ref-type="bibr"><sup>15</sup></xref> SARS-CoV-2 genomes were classified into lineages using the available software Pangolin<xref rid="B7" ref-type="bibr"><sup>7</sup></xref> and mutations were analysed with <b>Nextclade</b> Beta.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36656015">36656015</a> <b>(1)</b> Mucosal Gene Expression in Response to SARS-CoV-2 Is Associated with Viral Load<ul><li> All SARS-CoV-2 genomes identified were placed in the B.1 lineage in the <b>Nextclade</b> tree (<xref rid="B13" ref-type="bibr"><u>13</u></xref>) (see Table S2 in the supplemental material).<li> [13](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><volume>6</volume>:<fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36633196">36633196</a> <b>(3)</b> Whole&#x2010;genome sequencing of hMPXV1 in five Italian cases confirms the occurrence of the predominant epidemic lineage<ul><li>The mutation pattern was analyzed by the <b>NextClade</b> online tool using as a reference the strain MPXV_USA_2022_MA001 in NC_063383 coordinates (pseudo_ON563414) (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>, v2.8.1, last access&#xA0;November 25, 2022).<li> <b>Nextclade</b> was used for clade and lineage assignment, identification of single nucleotide variant mutations, insertions, and deletions.<li><b>Nextclade</b> was used for lineage assignment.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36839431">36839431</a> <b>(1)</b> Epidemiological, Clinical, and Virological Investigation of the First Four Cases of Monkeypox in Cartagena during the 2022 Outbreak<ul><li> Consensus sequences were deposited in NCBI, and clades and lineages were determined using <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 3 November 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36851091">36851091</a> <b>(3)</b> Incipient Parallel Evolution of SARS-CoV-2 Deltacron Variant in South Brazil<ul><li> To evaluate the quality and determine the lineage of the genome sequences, we analysed them on <b>Nextclade</b> Web version 2.3.1 [<xref rid="B19-vaccines-11-00212" ref-type="bibr"><u>19</u></xref>].<li> After this, we visually inspected the mutation pattern of the top hits on the <b>Nextclade</b> Web, using the putative Brazilian recombinant sequences as references.<li> omicron segments in the <b>Nextclade</b> Web and Pangolin COVID-19 Lineage Assigner version 4.1.1 [<xref rid="B22-vaccines-11-00212" ref-type="bibr"><u>22</u></xref>].<li> [19](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36650445">36650445</a> <b>(1)</b> A new and efficient enrichment method for metagenomic sequencing of Monkeypox virus<ul><li><b>Nextclade</b> (v.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36650162">36650162</a> <b>(1)</b> Rapid transmission and tight bottlenecks constrain the evolution of highly transmissible SARS-CoV-2 variants<ul><li> We used <b>Nextclade</b> to annotate clades and variants of concern<sup><xref ref-type="bibr" rid="CR58"><u>58</u></xref></sup>.<li> [58](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36711810">36711810</a> <b>(1)</b> Simultaneous Detection of Omicron and Other SARS-CoV-2 Variants by Multiplex PCR MassARRAY Technology<ul><li> SARS-CoV-2 lineages and variants were determined using Pangolin v3.1.20 with lineages version 2022-02-28 [<xref rid="R27" ref-type="bibr"><u>27</u></xref>], GISAID clade nomenclature [<xref rid="R28" ref-type="bibr"><u>28</u></xref>] and <b>Nextclade</b> v1.14.0 [<xref rid="R29" ref-type="bibr"><u>29</u></xref>].<li> [27](<name><surname>Rambaut</surname><given-names>A.</given-names></name><etal/><article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>. <source>Nature Microbiology</source><volume>5</volume>, <fpage>1403</fpage>&#x2013;<lpage>1407</lpage>. https://doi: <pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id> (<year>2020</year>).) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32794443">32794443</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36648012">36648012</a> <b>(1)</b> Evoluci&#xF3;n temporal de las variantes de SARSCoV-2 circulantes en el &#xE1;rea sanitaria de C&#xE1;diz<ul><li> en el &#xC1;rea de Bioinfoirm&#xE1;tica Cl&#xED;nica de la Fundaci&#xF3;n Progreso y Salud, ensamblando con la herramienta nf-core/viralrecon (<ext-link xlink:href="https://nfco.re/viralrecon" ext-link-type="uri">https://nfco.re/viralrecon</ext-link>) [<xref rid="ref5" ref-type="bibr"><u>5</u></xref>], asignando los diferentes clados y anotando las mutaciones presentes con <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) [<xref rid="ref6" ref-type="bibr"><u>6</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32055031">32055031</a>) [6](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>E</given-names></string-name>, <string-name><surname>Neher</surname><given-names>R</given-names></string-name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source>. <year>2021</year><month>Nov</month><day>30</day>;<volume>6</volume>(<issue>67</issue>):<fpage>3773</fpage>. doi: <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36686884">36686884</a> <b>(1)</b> Characterization and phylogenetic analysis of Iranian SARS&#x2010;CoV&#x2010;2 genomes: A phylogenomic study<ul><li>According to the <b>Nextclade</b> quality criteria, the sequences that were suspected to sequencing errors (265/1 sequences) were excluded.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680285">36680285</a> <b>(5)</b> Detection and Molecular Characterization of the SARS-CoV-2 Delta Variant and the Specific Immune Response in Companion Animals in Switzerland<ul><li>In household USZ22, both companion animals, dog 22.1 and cat 22.2, as well as the four owners were infected with the Delta variant AY.129 (<b>Nextclade</b> clade 21J).<li>In household USZ29, we identified the SARS-CoV-2 Delta-variant pangolin lineage AY.4 (<b>Nextclade</b> clade 21J) in dog 29.1 and owner 1.<li>In the second household, USZ38, we identified the Delta-variant pangolin lineage AY.4 (<b>Nextclade</b> clade 21J).<li>In household USZ36, the sequences were classified as pangolin lineage AY.43 (<b>Nextclade</b> clade 21J).<li>In household USZ39, SARS-CoV-2 Delta-variant pangolin lineage B.1.617.2 (<b>Nextclade</b> clade 21J) in cats 39.1 and 39.2 as well as in the four infected owners using NGS was identified.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680283">36680283</a> <b>(2)</b> Omicron-BA.1 Dispersion Rates in Mexico Varied According to the Regional Epidemic Patterns and the Diversity of Local Delta Subvariants<ul><li> BA.1, labeled clade 21 K by <b>Nextclade</b>, was the first of the two clades to spread, and it caused epidemic outbreaks throughout the world between December 2021 and January 2022.<li>Lineage clade assignment was carried out using Pangolin PLEARN v1.8 [<xref rid="B25-viruses-15-00243" ref-type="bibr"><u>25</u></xref>] and mutation annotation was done using the <b>NextClade</b> tool v.2.0.0-beta.3 alignment [<xref rid="B26-viruses-15-00243" ref-type="bibr"><u>26</u></xref>].<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [26](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36642191">36642191</a> <b>(1)</b> Genomic evidence for reinfection with different Omicron subvariants<ul><li> The data were uploaded and visualized using <b>Nextclade</b> (https://clades.nextstrain.org).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36579775">36579775</a> <b>(6)</b> Dynamics of nosocomial SARS&#x2010;CoV&#x2010;2 transmissions: Facing the challenge of variants of concern in a Brazilian reference hospital<ul><li><b>Nextclade</b>.<li> The characteristics of this patient were as follows: 2 vaccine doses, positive test results 72 days after the second dose, comorbidities smoking and HTN, <b>Nextclade</b> 21J, Pango AY.101.<li>Considering only the sequenced samples, which correspond to the year 2021, 42 (45.7%) of the individuals had not received any vaccine dose before diagnosis, with 31 (73.8%) belonging to <b>Nextclade</b> 20J (Gamma), 7 (16.7%) to 20B, and 4 (9.5%) to 21J (Delta).<li> In the 31 individuals who had received 2 doses, only 6 (19.3%) were assigned to the 20J clade, and the others (<italic toggle="yes">n</italic>&#x2009;=&#x2009;25, 80.6%) to <b>Nextclade</b> 21J.<li> (A) Distribution of sequenced samples (Nextstrain server, <b>Nextclade</b> tool).<li>Accordingly, we also identified a higher rate of deaths in sequenced samples attributed to <b>Nextclade</b> 20J, especially to the P.1 lineage.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36625603">36625603</a> <b>(3)</b> Rapid and Flexible RT-qPCR Surveillance Platforms To Detect SARS-CoV-2 Mutations<ul><li> Consensus genome mutation calling with reference to Wuhan-Hu-1/2019 (GenBank accession number <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MN908947" ext-link-type="uri" assigning-authority="insdc">MN908947</ext-link>) was performed with <b>Nextclade</b> CLI (<ext-link xlink:href="https://github.com/nextstrain/<b>nextclade</b>" ext-link-type="uri">https://github.com/nextstrain/<b>nextclade</b></ext-link>) (v.1.2.0), and lineage designations were performed using Pangolin (<ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link>) (v.3.1.3) with the accompanying pangoLEARN model (<ext-link xlink:href="https://github.com/cov-lineages/pangoLEARN" ext-link-type="uri">https://github.com/cov-lineages/pangoLEARN</ext-link>) (v.1.2.6).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36751652">36751652</a> <b>(4)</b> Challenges in Harnessing Shared Within-Host Severe Acute Respiratory Syndrome Coronavirus 2 Variation for Transmission Inference<ul><li> and assigned <b>Nextclade</b> lineages [<xref rid="ofad001-B29" ref-type="bibr"><u>29</u></xref>].<li> Clade assignments are made with <b>Nextclade</b> [<xref rid="ofad001-B46" ref-type="bibr"><u>46</u></xref>] through the nf-core viralrecon pipeline [<xref rid="ofad001-B24" ref-type="bibr"><u>24</u></xref>].<li> Clade, pairs of individuals outside households infected with the same <b>Nextclade</b> clade; and Outside clade, pairs of individuals outside households infected with different <b>Nextclade</b> clades.<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [46](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><year>2021</year>; <volume>6</volume>:<fpage>3773</fpage>.) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32055031">32055031</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36687432">36687432</a> <b>(1)</b> Probable vertical transmission of Alpha variant of concern (B.1.1.7) with evidence of SARS-CoV-2 infection in the syncytiotrophoblast, a case report<ul><li> RNA mutation identification used <b>Nextclade</b> version v0.14.2 (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36870749">36870749</a> <b>(1)</b> An ultrastructural and genomic study on the SARS-CoV-2 variant B.1.210 circulating during the first wave of COVID-19 pandemic in India<ul><li> Mutations were checked using the online tool <b>Nextclade</b> and compared with the reference sequence (NC_045512.2).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36604461">36604461</a> <b>(1)</b> Expanding repertoire of SARS-CoV-2 deletion mutations contributes to evolution of highly transmissible variants<ul><li> <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR33"><u>33</u></xref></sup> for viral clade assignment, phylogenetic analysis, and S codon mutation calling, in comparison to the wild-type reference sequence of SARS-CoV-2 Wuhan-Hu-1 (lineage B, clade 19A).<li> [33](Nextclade. <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36687433">36687433</a> <b>(3)</b> Whole-genome sequencing of SARS-CoV-2 isolates from symptomatic and asymptomatic individuals in Tanzania<ul><li> <b>NextClade</b> Web 1.10.0 (<xref rid="B15" ref-type="bibr"><u>15</u></xref>) with default settings was used to assess sequence quality and amplicon drop-outs.<li> <b>NextClade</b> Web 1.10.0 (<xref rid="B15" ref-type="bibr"><u>15</u></xref>) was used to assign clades and WHO designations.<li> <b>NextClade</b> Web 1.10.0 (<xref rid="B15" ref-type="bibr"><u>15</u></xref>) output was used to visualize the phylogenetic placement of the isolates in NextStrain Auspice 2.32.0 (<xref rid="B22" ref-type="bibr"><u>22</u></xref>).<li> [15](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J Open Source Softw.</italic></source> (<year>2021</year>) <volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) [15](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J Open Source Softw.</italic></source> (<year>2021</year>) <volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) [15](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J Open Source Softw.</italic></source> (<year>2021</year>) <volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36684874">36684874</a> <b>(3)</b> The Delta variant wave in Tunisia: Genetic diversity, spatio-temporal distribution and evidence of the spread of a divergent AY.122 sub-lineage<ul><li> The web-based software <b>Nextclade</b> v1.12.0 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) was used for clade assignment, diversity and gene-wise amino acid mutations analysis (<xref rid="B32" ref-type="bibr"><u>32</u></xref>).<li>Using <b>Nextclade</b> program, the 468 complete sequences belonged to clades 21J, 21I and 21A which correspond to the Delta variant.<li> The web-based software <b>Nextclade</b> v1.12.0 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) was used for gene-wise amino acid mutations analysis (<xref rid="B32" ref-type="bibr"><u>32</u></xref>).<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36597102">36597102</a> <b>(1)</b> A traveling SARS-CoV-2 laboratory as part of a pandemic response among vulnerable Brazilian populations<ul><li> We assessed the consensus genome sequence quality using <b>Nextclade</b> version 0.8.1 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36571266">36571266</a> <b>(1)</b> Mpox diagnostics: Review of current and emerging technologies<ul><li> however, additional consideration in using data polishing tools (Canu, Medaka, etc.) to correct indels should be considered due to issues with base&#x2010;calling in homopolymer&#x2010;rich regions.<xref rid="jmv28429-bib-0076" ref-type="bibr"><sup>76</sup></xref> In addition, due to the presence of repetitive and hypervariable regions in the ITS, masking of the first 1500&#xA0;and 7000&#x2009;bp of the genome is often performed to prevent any read mapping error.<xref rid="jmv28429-bib-0078" ref-type="bibr"><sup>78</sup></xref> Open source bioinformatics workflow (masking, pairwise alignments to reference sequence, phylogenetic reconstruction, clade and lineage assignments) is available through <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36529453">36529453</a> <b>(1)</b> SARS-CoV-2 Omicron BA.5 Infections in Vaccinated Persons, Rural Uganda<ul><li> Nine cases yielded full genome SARS-CoV-2 sequences that we lineage-typed using <b>Nextclade</b> (<xref rid="R9" ref-type="bibr"><italic>9</italic></xref>) and Pangolin (<xref rid="R10" ref-type="bibr"><italic>10</italic></xref>) software; 8 of the 9 genomes were from the BA.5 lineage and 1 was BA.2.31, all variants within the Omicron variant-of-concern lineage.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680170">36680170</a> <b>(1)</b> Phylogenomic and Structural Analysis of the Monkeypox Virus Shows Evolution towards Increased Stability<ul><li> The data were downloaded from <b>Nextclade</b> [<xref rid="B19-viruses-15-00127" ref-type="bibr"><u>19</u></xref>] after clade assignment, mutation calling, and sequence quality checks.<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35000296">35000296</a> <b>(1)</b> Anti-SARS-CoV-2 spike protein and neutralizing antibodies at 1 and 3 months after three doses of SARS-CoV-2 vaccine in a large cohort of solid organ transplant patients<ul><li>Neutralizing antibody titers were assessed using a live virus neutralization assay and a clinical SARS-CoV-2 strain (GISAID: EPI_ISL_804378, GISAID Clade:GH, Pango lineage: B.1.160, <b>Nextclade</b>: 20A.EU2) infecting Vero cells (ATCC, CCL-81<sup>TM</sup>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35384272">35384272</a> <b>(1)</b> Neutralizing antibody response against the B.1.617.2 (delta) and the B.1.1.529 (omicron) variants after a third mRNA SARS-CoV-2 vaccine dose in kidney transplant recipients<ul><li> Sequencing adapters were trimmed using trim_galore and host-read contamination was assessed and filtered using kraken2, as described previously.<xref rid="bib31" ref-type="bibr"><sup>31</sup></xref><sup>,</sup><xref rid="bib32" ref-type="bibr"><sup>32</sup></xref>Reads were aligned to the SARS-CoV-2 reference genome using bwa and alignments were sorted and indexed using samtools and quality-controlled using alfred.<xref rid="bib33" ref-type="bibr"><u>33</u></xref>, <xref rid="bib34" ref-type="bibr"><u>34</u></xref>, <xref rid="bib35" ref-type="bibr"><u>35</u></xref>Priming regions were masked with iVar, followed by variant calling with FreeBayes, normalization of variants with bcftools, and annotation of variants with the Ensembl Variant Effect Predictor (VEP).<xref rid="bib36" ref-type="bibr"><u>36</u></xref>, <xref rid="bib37" ref-type="bibr"><u>37</u></xref>, <xref rid="bib38" ref-type="bibr"><u>38</u></xref>, <xref rid="bib39" ref-type="bibr"><u>39</u></xref>iVar was employed to compute a viral consensus sequence that was then classified by Pangolin and <b>Nextclade</b> to determine the viral lineage and clade, respectively.<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19451168">19451168</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19505943">19505943</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30520945">30520945</a>) [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30621750">30621750</a>) [37](Garrison E, Marth G. Haplotype-based variant detection from short-read sequencing. <italic>Arxiv</italic>. 2012. Published online.) [38](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21903627">21903627</a>) [39](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27268795">27268795</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35150193">35150193</a> <b>(2)</b> Severe relapse of SARS-CoV-2 infection in a kidney transplant recipient with negative nasopharyngeal SARS-CoV-2 RT-PCR after rituximab<ul><li> (A) Summary of SARS-CoV-2 virological results from the first and second COVID-19 infections including RT-PCR results in a nasal swab and BAL, infectious titer and clade according to <b>Nextclade</b> classification.<li> Genomes were classified into clades using <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36680148">36680148</a> <b>(6)</b> Genomic Epidemiology of the SARS-CoV-2 Epidemic in Cyprus from November 2020 to October 2021: The Passage of Waves of Alpha and Delta Variants of Concern<ul><li>Mutations in the sequences in this study were identified using <b>Nextclade</b> Webtool Web version 1.14.1 (<uri xlink:href="https://clades.nextstrain.org/<b>Nextclade</b>">https://clades.nextstrain.org/<b>Nextclade</b></uri> accessed on 7 May 2022) [<xref rid="B35-viruses-15-00108" ref-type="bibr"><u>35</u></xref>].<li> To identify the characteristic and most common mutations of these lineages, sequences were input into <b>Nextclade</b> Webtool [<xref rid="B35-viruses-15-00108" ref-type="bibr"><u>35</u></xref>], and the output was sorted to isolate S protein mutations (deletions and substitutions).<li> Mutations were identified using the <b>Nextclade</b> webtool (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>, date accessed 9 May 2022) [<xref rid="B35-viruses-15-00108" ref-type="bibr"><u>35</u></xref>].<li> The maximum likelihood tree was generated in <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>, date accessed 05 August 2022 [<xref rid="B35-viruses-15-00108" ref-type="bibr"><u>35</u></xref>]) and highlights the emergence of mutations.<li> [35](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [35](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [35](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [35](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36575217">36575217</a> <b>(2)</b> Genomic epidemiology of SARS- CoV-2 Omicron variants in the Republic of Korea<ul><li> We used <b>Nextclade</b> v1.13.2 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) for quality control (QC) of 582 sequences isolated in South Korea.<li> <b>Nextclade</b> uses an algorithm that considers missing data, mixed sites, private mutations, mutation clusters, stop codons, and frameshifts in calculating a score.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36581284">36581284</a> <b>(1)</b> Parallel deployment of passive and composite samplers for surveillance and variant profiling of SARS-CoV-2 in sewage<ul><li> <b>Nextclade</b> Web v2.3.0 was applied for the clade assignments of the consensus sequences (<xref rid="bb0065" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36620039">36620039</a> <b>(1)</b> Genomic landscape of the SARS-CoV-2 pandemic in Brazil suggests an external P.1 variant origin<ul><li> The alignment was made using the <b>Nextclade</b> online tool (Hadfield et al., <xref rid="B22" ref-type="bibr"><u>2018</u></xref>).<li> [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36611300">36611300</a> <b>(3)</b> Complete Genomic Characterisation and Mutation Patterns of Iraqi SARS-CoV-2 Isolates<ul><li> Furthermore, the clades were identified using the <b>Nextclade</b> sequence analysis and the GISAID database platform tools [<xref rid="B5-diagnostics-13-00008" ref-type="bibr"><u>5</u></xref>,<xref rid="B21-diagnostics-13-00008" ref-type="bibr"><u>21</u></xref>].<li> The second nomenclature system used was the <b>Nextclade</b> tool of the Nextrain platform; the resulting clade for the Duhok isolates in this study were 21J (Delta variant of concern) for the third wave and 22K (Omicron) for the fifth wave, which was the leading isolate of the fifth wave.<li> A phylogenetic tree was constructed using the <b>Nextclade</b> web-based application that uses global sequences from the GISAID database for phylogenetic analysis.<li> [5](          <person-group person-group-type="author"><name><surname>Elbe</surname><given-names>S.</given-names></name><name><surname>Buckland-Merrett</surname><given-names>G.</given-names></name></person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Glob. Challenges</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ) [21](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36668909">36668909</a> <b>(2)</b> First Case of Monkeypox in Venezuela: Partial Complete Genome Sequence Allowed Its Grouping into the West African Clade II<ul><li>The FASTA file obtained from Genome Detective was analyzed using the <b>Nextclade</b> web-tool [<xref rid="B11-tropicalmed-08-00002" ref-type="bibr"><u>11</u></xref>,<xref rid="B12-tropicalmed-08-00002" ref-type="bibr"><u>12</u></xref>].<li> The JSON file containing the phylogenomic datasets was downloaded from <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> last accessed 20 September 2022), and was analyzed at Auspice (<uri xlink:href="https://auspice.us/">https://auspice.us/</uri> last accessed 20 September 2022) [<xref rid="B13-tropicalmed-08-00002" ref-type="bibr"><u>13</u></xref>].<li> [11](          <article-title>Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-26">(accessed on 26 July 2022)</date-in-citation>        ) [12](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [13](          <article-title>Auspice.Us</article-title>          <comment>Available online: <ext-link xlink:href="https://auspice.us/" ext-link-type="uri">https://auspice.us/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-27">(accessed on 27 July 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36751558">36751558</a> <b>(1)</b> Surveillance and Analysis of SARS-CoV-2 Variant Importation &#x2014; China, January&#x2013;June 2022<ul><li>Consensus sequences were assessed for quality control by the <b>Nextclade</b> web tool (https://clades.nextstrain.org; version 2.4.1) with parameters for missing data, mixed sites, private mutations, mutation clusters, frameshifts, and stop codons.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36561171">36561171</a> <b>(2)</b> Local-Scale phylodynamics reveal differential community impact of SARS-CoV-2 in metropolitan US county<ul><li>A temporally-resolve phylogeny was created using the Nextstrain (<xref rid="R13" ref-type="bibr"><u>13</u></xref>) SARS-CoV-2 workflow (<ext-link xlink:href="https://github.com/nextstrain/ncov" ext-link-type="uri">https://github.com/nextstrain/ncov</ext-link>), which aligns sequences against the Wuhan Hu-1 reference using nextalign (<ext-link xlink:href="https://github.com/nextstrain/<b>nextclade</b>" ext-link-type="uri">https://github.com/nextstrain/<b>nextclade</b></ext-link>), infers a maximum-likelihood phylogeny using IQ-TREE (<xref rid="R14" ref-type="bibr"><u>14</u></xref>) with a GTR nucleotide substitution model, and estimates molecular clock branch lengths using TreeTime (<xref rid="R15" ref-type="bibr"><u>15</u></xref>).<li> [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [15](<name><surname>Sagulenko</surname><given-names>P</given-names></name>, <name><surname>Puller</surname><given-names>V</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>. <article-title>TreeTime: Maximum-likelihood phylodynamic analysis</article-title>. <source>Virus Evol</source>. <year>2018</year><month>Jan</month><day>8</day>;<volume>4</volume>(<issue>1</issue>):<comment>vex042.</comment>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36654682">36654682</a> <b>(1)</b> Genomic epidemiology of SARS-CoV-2 in Cambodia, January 2020 to February 2021<ul><li> Lineages and specific mutations were determined from consensus sequences using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and PANGOLIN (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) webservers.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36557043">36557043</a> <b>(4)</b> Identification of Genomic Variants of SARS-CoV-2 Using Nanopore Sequencing<ul><li> (Oxford, UK), Version 2022.04.26-13521, which uses Artic, <b>Nextclade</b>, Pangolin sub-sections (Artic + <b>Nextclade</b> + Pangolin-v3.3.1).<li> <b>Nextclade</b> software identifies the genetic variants compared to the reference genome and provides data concerning quality control [<xref rid="B22-medicina-58-01841" ref-type="bibr"><u>22</u></xref>].<li> The alignment, lineage, and clade identification were performed using <b>Nextclade</b> and SARS-CoV-2 as a reference genome (MN908947).<li> [22](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560792">36560792</a> <b>(3)</b> Genomic Epidemiology of SARS-CoV-2 in Western Burkina Faso, West Africa<ul><li>Genomic sequences were assigned to phylogenetic clades using <b>NextClade</b> (version 0.13.0) and to Pango lineages using the pangolin toolkit (version 2.3.2) with pangoLEARN (version v10 February 2021).<li>The 372 genomes were assigned to 10 different phylogenetic clades (<xref rid="viruses-14-02788-f001" ref-type="fig"><u>Figure 1</u></xref>) using the <b>NextClade</b> classification system that included 30 clades described worldwide on 20 October 2022 [<xref rid="B13-viruses-14-02788" ref-type="bibr"><u>13</u></xref>].<li> Although nearly 50% of sequences (<italic toggle="yes">n</italic> = 184) had a relatively high proportion of missing data (&gt;3000 Ns), clade signature mutations were in sufficient numbers to ensure good phylogenetic support for the <b>NextClade</b> classification (<xref rid="app1-viruses-14-02788" ref-type="app"><u>Figure S1a</u></xref>).<li> [13](          <article-title>Nextstrain Clade Assignment</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/ncov/blob/9a1cf941872a421d7b69d98787c6aed73083e2cc/defaults/clades.tsv" ext-link-type="uri">https://github.com/nextstrain/ncov/blob/9a1cf941872a421d7b69d98787c6aed73083e2cc/defaults/clades.tsv</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-10-20">(accessed on 20 October 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36579069">36579069</a> <b>(3)</b> Prevalence of SARS-CoV-2 and co-infection with malaria during the first wave of the pandemic (the Burkina Faso case)<ul><li> Phylogenetic tree was generated with <b>Nextclade</b> v2.5.0<xref rid="fn0008" ref-type="fn"><sup>8</sup></xref>, the tree was downloaded in JSON format and visualized using Auspice v2.37.3 online tool<xref rid="fn0009" ref-type="fn"><sup>9</sup></xref> (<xref rid="B15" ref-type="bibr"><u>15</u></xref>, <xref rid="B16" ref-type="bibr"><u>16</u></xref>).<li> To assign the sequences to lineages and clades, we used the dynamic lineage classification method called Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) (<xref rid="B13" ref-type="bibr"><u>13</u></xref>) and the sequence analysis web tool, <b>Nextclade</b> (<xref rid="B14" ref-type="bibr"><u>14</u></xref>).<li> Phylogenetic tree generated with <b>Nextclade</b> online software v2.5.0 (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) (accessed on 13 September 2022) and visualized using Auspice v2.37.3 (<ext-link xlink:href="https://auspice.us/" ext-link-type="uri">https://auspice.us/</ext-link>) (accessed on 13 September 2022).<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35905853">35905853</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36451891">36451891</a> <b>(1)</b> SARS-CoV-2 exposure in Norway rats (<italic toggle="yes">Rattus norvegicus)</italic> from New York City<ul><li> Nucleotide sequences were aligned using MAFFT v7.471, and the mutations were analyzed using <b>nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36555300">36555300</a> <b>(2)</b> Quasispecies Analysis of SARS-CoV-2 of 15 Different Lineages during the First Year of the Pandemic Prompts Scratching under the Surface of Consensus Genome Sequences<ul><li> They were classified in various lineages according to the <b>Nextclade</b> [<xref rid="B10-ijms-23-15658" ref-type="bibr"><u>10</u></xref>,<xref rid="B29-ijms-23-15658" ref-type="bibr"><u>29</u></xref>], Pangolin [<xref rid="B30-ijms-23-15658" ref-type="bibr"><u>30</u></xref>], and World Health Organization (WHO) (<uri xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/</uri> (accessed on 3 October 2022)) classifications.<li> The SARS-CoV-2 genotype had been identified on the basis of the consensus genome sequence with the <b>Nextclade</b> tool v1.6.0 (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri> (accessed on 30 September 2022)) [<xref rid="B29-ijms-23-15658" ref-type="bibr"><u>29</u></xref>].<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [29](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36495397">36495397</a> <b>(1)</b> Systematic In-Silico Evaluation of the Diagnostic Impact of Mpox Genome Variants in the Current Outbreak<ul><li>Genetic variants in the MPXV genomes were called using <b>Nextclade</b> [<xref ref-type="bibr" rid="CR40"><u>40</u></xref>] against the reference genome.<li> [40](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560750">36560750</a> <b>(1)</b> Monitoring the Establishment of VOC Gamma in Minas Gerais, Brazil: A Retrospective Epidemiological and Genomic Surveillance Study<ul><li> The consensus sequences generated in our study were classified using the Pangolin tool v.3.1.14 and the <b>NextClade</b> web application v.1.7.0 [<xref rid="B30-viruses-14-02747" ref-type="bibr"><u>30</u></xref>].<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560738">36560738</a> <b>(1)</b> Molecular Epidemiology of the Norwegian SARS-CoV-2 Delta Lineage AY.63<ul><li> The A222V mutation probably has a deep origin in Delta, close to the split between the two major groups (21I and 21J according to the <b>NextClade</b> nomenclature [<xref rid="B30-viruses-14-02734" ref-type="bibr"><u>30</u></xref>]), while the deletion in nsp1 is less common but has been observed several times in the 21I clade as well (<xref rid="viruses-14-02734-f003" ref-type="fig"><u>Figure 3</u></xref>A).<li> [30](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36472422">36472422</a> <b>(1)</b> Genome Sequences of 23 SARS-CoV-2 Omicron-Lineage Strains from Bangladesh<ul><li> Processed reads were assembled with the ARTIC guppyplex code script with Medaka v1.4, using the ARTIC EPI2ME v3.3.0 SARS-CoV-2 pipeline (FastQC plus ARTIC plus <b>NextClade</b>) (<ext-link xlink:href="https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html" ext-link-type="uri">https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36481486">36481486</a> <b>(2)</b> Host microRNAs exhibit differential propensity to interact with SARS-CoV-2 and variants of concern<ul><li>We uploaded each whole sequence information to <b>Nextclade</b> v1.14.0 &#xA9; 2020&#x2013;2022 Nextstrain developers <xref rid="bb0165" ref-type="bibr"><u>[33]</u></xref>.<li> We performed phylogenetic analysis using <b>Nextclade</b> (v1.14.1) to determine which variants to compare in our study.<li> [[33]](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560720">36560720</a> <b>(1)</b> SARS-CoV-2 Genomic Surveillance in Brazil: A Systematic Review with Scientometric Analysis<ul><li> Pangolin [<xref rid="B10-viruses-14-02715" ref-type="bibr"><u>10</u></xref>] was the leading variant-calling software (<italic toggle="yes">n</italic> = 75, 70.75%), followed by IQTREE [<xref rid="B124-viruses-14-02715" ref-type="bibr"><u>124</u></xref>] (<italic toggle="yes">n</italic> = 59, 50.66%) and <b>Nextclade</b> [<xref rid="B125-viruses-14-02715" ref-type="bibr"><u>125</u></xref>] (<italic toggle="yes">n</italic> = 29, 27.35%).<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [124](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [125](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560718">36560718</a> <b>(2)</b> ViralVar: A Web Tool for Multilevel Visualization of SARS-CoV-2 Genomes<ul><li> ViralVar accepts input data from the GISAID in a tabular format that includes three sets of information for each genome: (1) PANGO lineage (users can opt to manually add <b>Nextclade</b> designations), (2) sample collection date, and (3) a list of protein mutations (denoted as &#x201C;AA Substitutions&#x201D; in the GISAID data; required for &#x201C;Mutational Analysis&#x201D;).<li> The phylogenetic nomenclature option allows users to customize output data to use PANGO lineage, <b>Nextclade</b> clade, or World Health Organization-defined VOC nomenclature.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36846035">36846035</a> <b>(1)</b> SARS-CoV-2 Monitoring on Mink Farms in Poland<ul><li> The tree was processed with the <b>Nextclade</b> open source tool (<ext-link xlink:href="http://https://clades.nextstrain.org/tree" ext-link-type="uri">https://clades.nextstrain.org/tree</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36558796">36558796</a> <b>(1)</b> Genomic Variability of SARS-CoV-2 Omicron Variant Circulating in the Russian Federation during Early December 2021 and Late January 2022<ul><li> All genomes were aligned with the reference genome Wuhan-Hu-1/2019 (GenBank: MN908947) using <b>Nextclade</b>CLI v.1.11.0 [<xref rid="B14-pathogens-11-01461" ref-type="bibr"><u>14</u></xref>] and a FastTree 2.1.11 [<xref rid="B15-pathogens-11-01461" ref-type="bibr"><u>15</u></xref>] with the following command: -gtr -nt alignment_file &gt; tree_file.<li> [14](          <article-title>Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli.html" ext-link-type="uri">https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli.html</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-05-10">(accessed on 10 May 2022)</date-in-citation>        ) [15](          <article-title>FastTree 2.1.11</article-title>          <comment>Available online: <ext-link xlink:href="http://www.microbesonline.org/fasttree/" ext-link-type="uri">http://www.microbesonline.org/fasttree/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-05-10">(accessed on 10 May 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36322955">36322955</a> <b>(1)</b> Probable Aerosol Transmission of SARS-CoV-2 through Floors and Walls of Quarantine Hotel, Taiwan, 2021<ul><li> We processed the obtained viral sequences using DRAGEN COVID Lineage application version 3.5.5 on Illumina&#x2019;s Basespace cloud analytical system and compared those with <b>Nextclade</b> software (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36178142">36178142</a> <b>(1)</b> Possible Occupational Infection of Healthcare Workers with Monkeypox Virus, Brazil<ul><li> Analysis using the <b>Nextclade</b> tool (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) showed that the sequenced genomes were 100% identical and belonged to MPXV clade IIb sublineage B.1.1 (<xref rid="SD1" ref-type="supplementary-material"><u>Appendix</u></xref> Figures 1&#x2013;3).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36453913">36453913</a> <b>(1)</b> Direct Nasal Swab for Rapid Test and Saliva as an Alternative Biological Sample for RT-PCR in COVID-19 Diagnosis<ul><li> <b>Nextclade</b> v1.5.2 was used to discover the SARS-CoV-2 clade (accessed on 18 March 2022) (<xref rid="B33" ref-type="bibr"><u>33</u></xref>).<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36547242">36547242</a> <b>(1)</b> Genomic Evidence Suggests Viral Persistence of SARS-CoV-2 for 386 Days in Health Worker: A Case Report from Santiago of Chile<ul><li> In addition, a clade assignment was performed using <b>NextClade</b> version 1.9.0 software to compare and visualize our samples globally.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36448812">36448812</a> <b>(6)</b> Genome Sequence of a Recombinant SARS-CoV-2 Lineage XAM (BA.1.1/BA.2.9) Strain from a Clinical Sample in Santo Domingo, Dominican Republic<ul><li> In October 2022, XBB, a BA.2.10 and BA.2.75 recombinant, reached dominance in Singapore (<ext-link xlink:href="https://cov-spectrum.org/explore/Singapore/AllSamples/Y2022/variants?variantQuery=<b>nextclade</b>PangoLineage%3AXBB*&amp;" ext-link-type="uri">https://cov-spectrum.org/explore/Singapore/AllSamples/Y2022/variants?variantQuery=<b>nextclade</b>PangoLineage%3AXBB*&amp;</ext-link>).<li> Lineage assignment according to the Pangolin nomenclature (<xref rid="B10" ref-type="bibr"><u>10</u></xref>) was conducted with <b>Nextclade</b> (v.2.8.0) (<xref rid="B14" ref-type="bibr"><u>14</u></xref>).<li><b>Nextclade</b>.<li><ext-link xlink:href="https://github.com/nextstrain/<b>nextclade</b>" ext-link-type="uri">https://github.com/nextstrain/<b>nextclade</b></ext-link>.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [14](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><volume>6</volume>:<fpage>3773</fpage>. doi:<pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36448779">36448779</a> <b>(1)</b> Epidemiological and Genomic Analysis of a Large SARS-CoV-2 Outbreak in a Long-Term Care Facility in Catalonia, Spain<ul><li> Amino acid substitutions and deletions were identified with <b>Nextclade</b> v0.14.1 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36450812">36450812</a> <b>(3)</b> Pattern of SARS-CoV-2 variant B.1.1.519 emergence in Alaska<ul><li> The number of Spike protein amino acid substitutions was determined using <b>Nextclade</b> (v 1.13.2) with &#x2018;bad&#x2019; quality genomes, as determined by <b>Nextclade</b>&#x2019;s algorithm for quality, removed prior to analyses<sup><xref ref-type="bibr" rid="CR11"><u>11</u></xref></sup>.<li> <b>Nextclade</b>&#x2019;s quality control incorporates six individual rules based on missing data, mutations, and amino acid changes as described in the online documentation.<li> [11](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Sour. Softw.</source>          <year>2021</year>          <volume>6</volume>          <issue>67</issue>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560693">36560693</a> <b>(1)</b> What&#x2019;s Next for Flu? Out-of-Season Circulation of Influenza Viruses in Southern Italy, August 2022<ul><li> Amino acid substitutions were identified by comparing the HA gene sequences of the identified strains and the HA gene sequences of the reference strains through the <b>Nextclade</b> tool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 30 November 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36523785">36523785</a> <b>(1)</b> Identification of severe acute respiratory syndrome coronavirus 2 breakthrough infections by anti-nucleocapsid antibody among fully vaccinated non-healthcare workers during the transition from the delta to omicron wave<ul><li> The genomes were classified into lineages using Pangolin (<xref rid="B33" ref-type="bibr"><u>33</u></xref>) and into clades using <b>Nextclade</b> (<xref rid="B34" ref-type="bibr"><u>34</u></xref>).<li> [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [34](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J. Open Source Softw.</italic></source> (<year>2021</year>) <volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36448028">36448028</a> <b>(1)</b> Dynamics of SARS-CoV-2 variants characterized during different COVID-19 waves in Mali<ul><li> The <b>Nextclade</b> tool (Version 1.14.0) was used for SARS-CoV-2 S gene alignment, mutation calling, quality checks, phylogenetic placement and clade assignment.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36448029">36448029</a> <b>(1)</b> A tale of two waves: characteristics and outcomes of COVID-19 admissions during the Omicron-driven fourth wave in Cape Town, South Africa, and implications for the future<ul><li> The assignment of clades and identification of mutations were performed using <b>Nextclade</b> (<xref rid="bib0002" ref-type="bibr"><u>Aksamentov&#xA0;et&#xA0;al., 2021</u></xref>) and lineages were determined using the web version of the Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) software (<xref rid="bib0015" ref-type="bibr"><u>O'Toole&#xA0;et&#xA0;al., 2021</u></xref>).<li> [Aksamentov&#xA0;et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Journal of Open Source Software</source>          <volume>6</volume>          <issue>67</issue>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [O'Toole&#xA0;et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>O'Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <etal/>          </person-group>          <article-title>Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool</article-title>          <source>Virus Evolution</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36560613">36560613</a> <b>(1)</b> Comparison of a Prototype SARS-CoV-2 Lateral Flow IMMUNOASSAY with the BinaxNOW<sup>TM</sup> COVID-19 Antigen CARD<ul><li>SARS-CoV-2 Omicron-positive remnants clinical specimens were sequenced and analyzed for lineages through <b>NextClade</b> [<xref rid="B30-viruses-14-02609" ref-type="bibr"><u>30</u></xref>].<li> [30](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/JOSS.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36817495">36817495</a> <b>(1)</b> Systematic sequencing of imported cases leads to detection of SARS-CoV-2 B.1.1.529 (Omicron) variant in central Viet Nam<ul><li> The Dragen Covid Lineage software (Illumina) was used for genomic analysis and phylogenetic trees were generated using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36451887">36451887</a> <b>(0)</b> Evolution of the SARS-CoV-2 mutational spectrum<ul><li> 2021</xref>) from November-7&#x2013;2022(<ext-link xlink:href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/11/07/public-2022-11-07.all.masked.<b>nextclade</b>.pangolin.pb.gz" ext-link-type="uri">http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/11/07/public-2022-11-07.all.masked.<b>nextclade</b>.pangolin.pb.gz</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36409093">36409093</a> <b>(1)</b> Tracing the First Days of the SARS-CoV-2 Pandemic in Greece and the Role of the First Imported Group of Travelers<ul><li><b>NextClade</b> CLI (Version 1.11.0) (<xref rid="B38" ref-type="bibr"><u>38</u></xref>) was used in order to perform quality control (QC) of the FASTA sequences, assignment to a Nextstrain clade and nucleotide substitution profile, and identification of changes in the viral proteins relative to the SARS-CoV-2 reference.<li> [38](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. Available at <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>. doi:<pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36398277">36398277</a> <b>(1)</b> SARS-CoV-2 and the role of close contact in transmission: a systematic review<ul><li> Samples from the same<break/>household were considered to be related<break/>if they were assigned to the same larger<break/>clade by <b>Nextclade</b> as well as the same clade<break/>in MrBayes.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36415653">36415653</a> <b>(2)</b> Genome sequence of SARS-CoV-2 sublineage BA.2.12.1 strain from Bangladesh<ul><li> The phylogenetic tree was created using <ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/<b>nextclade</b>_data_workflows/tree/master/sars-cov-2" id="intref0015">https://github.com/neherlab/<b>nextclade</b>_data_workflows/tree/master/sars-cov-2</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36383192">36383192</a> <b>(1)</b> Quantifying the impact of immune history and variant on SARS-CoV-2 viral kinetics and infection rebound: A retrospective cohort study<ul><li> Lineage assignment and phylogenetics analysis using the most updated version of Pangolin (<xref rid="bib38" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) and <b>NextClade</b> (<xref rid="bib1" ref-type="bibr"><u>Aksamentov and Neher, 2021</u></xref>), respectively.<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Aksamentov and Neher, 2021](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>          <year iso-8601-date="2021">2021</year>          <article-title>NextClade</article-title>          <source>The Journal of Open Source Software</source>          <volume>6</volume>          <elocation-id>3773</elocation-id>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36430621">36430621</a> <b>(1)</b> Cytokine Profiling in Different SARS-CoV-2 Genetic Variants<ul><li> The <b>Nextclade</b> tool was used to assess the quality of assembled sequences and to assign genomes to lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36423145">36423145</a> <b>(1)</b> SARS-CoV-2 and Its Variants in Thrice-Infected Health Workers: A Case Series from an Italian University Hospital<ul><li> The tree was generated by the <b>Nextclade</b> site comparing the two different Omicron lineages of the second and third infection of patients 3, 4, and 9, displaying the phylogenetic distance between the two samples.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36377913">36377913</a> <b>(2)</b> SARS-CoV-2 Genomic Diversity in Households Highlights the Challenges of Sequence-Based Transmission Inference<ul><li><b>Nextclade</b> clade<xref rid="ngtab2-1" ref-type="table-fn"><sup><italic toggle="yes">a</italic></sup></xref></th>                <th rowspan="1" colspan="1">PANGO lineage<xref rid="ngtab2-2" ref-type="table-fn"><sup><italic toggle="yes">b</italic></sup></xref></th>                <th rowspan="1" colspan="1">Mean consensus diff<xref rid="ngtab2-3" ref-type="table-fn"><sup><italic toggle="yes">c</italic></sup></xref></th>              </tr>            </thead>            <tbody>              <tr>                <td rowspan="1" colspan="1">PR1.<li>Defined using <b>nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36374079">36374079</a> <b>(1)</b> Whole-Genome Sequences of Human Monkeypox Virus Strains from Two 2022 Global Outbreak Cases in Western New York State<ul><li>The two sequences were B.1.3 lineage, clade IIb (<b>Nextclade</b> v2.6.1), consistent with other sequences from the 2022 MPXV outbreak, descendants of the A.1.1 lineage exported from Nigeria in 2021 (<xref rid="B17" ref-type="bibr"><u>17</u></xref>).<li> [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35263559">35263559</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36375707">36375707</a> <b>(1)</b> Impact of COVID-19 RT-PCR testing of asymptomatic health care workers on absenteeism and hospital transmission during the pandemic<ul><li>Consensus sequences were initially submitted to a quality control check using <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36423117">36423117</a> <b>(2)</b> Epidemiological and Clinical Features of SARS-CoV-2 Variants Circulating between April&#x2013;December 2021 in Italy<ul><li> 4.1.1 (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>, accessed on 3 November 2022) and <b>Nextclade</b> v.<li> Mutations were identified using <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36514661">36514661</a> <b>(1)</b> An Early and Preliminary Assessment of the Clinical Severity of the Emerging SARS-CoV-2 Omicron Variants in Maharashtra, India<ul><li> The clade analysis was done using the <b>Nextclade</b> software, version 2.3.0.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36354320">36354320</a> <b>(2)</b> Genetic Study of SARS-CoV-2 Non Structural Protein 12 in COVID-19 Patients Non Responders to Remdesivir<ul><li> Mutations were identified by aligning the consensus sequence with the Wuhan-Hu-1 reference genome (GenBank accession number <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MN908947.3" ext-link-type="uri" assigning-authority="insdc">MN908947.3</ext-link>) using <b>Nextclade</b> v.1.14.0.<li>The SARS-CoV-2 lineage was identified using <b>Nextclade</b> v.1.14.0 according to the amino acid replacement determinant of each variant, as classified by the SARS-CoV-2 Interagency Group (SIG) and the Centers of Disease Control and Infection (CDC) (<xref rid="B24" ref-type="bibr"><u>24</u></xref>).<li> [24](<collab>Centers for Disease Control and Prevention</collab>. <year>2022</year>. <article-title>SARS-CoV-2 variant classifications and definitions</article-title>. <ext-link xlink:href="https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-classifications.html#:~:text=On%20November%2030%2C%202021%2C%20the,among%20those%20without%20travel%20history" ext-link-type="uri">https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-classifications.html#:~:text=On%20November%2030%2C%202021%2C%20the,among%20those%20without%20travel%20history</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36357713">36357713</a> <b>(3)</b> Divergent SARS-CoV-2 variant emerges in white-tailed deer with deer-to-human transmission<ul><li> The mutations identified by the <b>Nextclade</b> (v1.10.2) (ref.<li> <b>Nextclade</b> (v1.10.2) analysis was used to determine amino acid mutations and missing or low/no coverage regions from the sample genome sequences.<li> Sequence statistics such as C&#x2009;&gt;&#x2009;T rate were directly calculated from <b>nextclade</b> results (v1.10.2 with 2022-01-05 database).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36371334">36371334</a> <b>(1)</b> Seroepidemiological and genomic investigation of SARS-CoV-2 spread in North East region of India<ul><li> Mutation calling and lineage assignment were carried out using <b>NextClade</b> v.154 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0015">https://clades.nextstrain.org/</ext-link>) and Pangolin Lineage Assigner v3.1.16 (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="intref0020">https://pangolin.cog-uk.io</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36346252">36346252</a> <b>(1)</b> Detection of SARS-CoV-2 in Saliva and Nasopharyngeal Swabs According to Viral Variants<ul><li> Among the 134 samples also sequenced via whole-genome sequencing (WGS), <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) retrieved the following lineages: 29 Wuhan (no VOC), 19 Alpha (including 4 Alpha&#x2009;+&#x2009;484K), 11 Beta, 33 Delta, and 42 Omicron (including 40 from the BA.1 lineage and 2 from the BA.2 lineage).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36356728">36356728</a> <b>(1)</b> Identification of a rare SARS-CoV-2 XL hybrid variant in wastewater and the subsequent discovery of two infected individuals in Nevada<ul><li>Using <b>Nextclade</b>, we determined the lineage components of the recombinant XL genome by taking each fragment that was separated at the breakpoint position.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36346321">36346321</a> <b>(2)</b> Swiss public health measures associated with reduced SARS-CoV-2 transmission using genome data<ul><li> We also filtered sequences flagged by the <b>Nextclade</b> tool (<xref rid="R37" ref-type="bibr"><italic toggle="yes">37</italic></xref>) for suspiciously clustered SNPs (QC SNP clusters status metric not &#x201C;good&#x201D;; &gt;&#xA0;=&#xA0;6 mutations in 100 bases), too many private mutations (QC private mutations status metric not &#x201C;good&#x201D;; &gt;&#xA0;=&#xA0;10 mutations from the nearest tree node), or overall bad quality (<b>Nextclade</b> QC overall status &#x201C;bad&#x201D;).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36405967">36405967</a> <b>(1)</b> SARS-CoV-2 variants of concern surveillance including Omicron using RT-PCR&#x2013;based genotyping offers comparable performance to whole genome sequencing<ul><li> <bold>(A)</bold> <b>Nextclade</b> phylogenetic radial tree from the Nextstrain global reference tree.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36336246">36336246</a> <b>(1)</b> SARS-CoV-2 genomic surveillance in Malaysia: displacement of B.1.617.2 with AY lineages as the dominant Delta variants and the introduction of Omicron during the fourth epidemic wave<ul><li> Clade assignment, mutation calling, and sequence quality control score were performed using the open-source <b>Nextclade</b> pipelines comprising the phylodynamic analysis component Augur and the interactive visualization component Auspice (https://clades.nextstrain.org) (<xref rid="bib0001" ref-type="bibr">Aksamentov <italic>et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36366530">36366530</a> <b>(2)</b> Systematic Genomic Surveillance of SARS-CoV-2 Virus on Illumina Sequencing Platforms in the Slovak Republic&#x2014;One Year Experience<ul><li> <b>NextClade</b> [<xref rid="B26-viruses-14-02432" ref-type="bibr"><u>26</u></xref>] was then used to check quality of consensus sequences.<li> Consensus sequences were visualized on the backbone of the global SARS-CoV-2 lineage tree using the interactive <b>Nextclade</b> v1.14.1 platform [<xref rid="B26-viruses-14-02432" ref-type="bibr"><u>26</u></xref>,<xref rid="B30-viruses-14-02432" ref-type="bibr"><u>30</u></xref>,<xref rid="B31-viruses-14-02432" ref-type="bibr"><u>31</u></xref>].<li> [26](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [26](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [30](          <article-title>Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-06-29">(accessed on 29 June 2022)</date-in-citation>        ) [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36314806">36314806</a> <b>(1)</b> Analytical Sensitivity of Lateral Flow Devices against SARS-CoV-2 Omicron Subvariants BA.4, BA.5, and BA.2.75<ul><li> Lineage designation was assigned using both <b>Nextclade</b> and Pangolin v3.1.16.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36467294">36467294</a> <b>(1)</b> Genomic Investigation to Identify Sources of Severe Acute Respiratory Syndrome Coronavirus 2 Infection Among Healthcare Personnel in an Acute Care Hospital<ul><li> The circulating SARS-CoV-2 lineage was the Wuhan lineage (including <b>Nextclade</b> [<xref rid="ofac581-B9" ref-type="bibr"><u>9</u></xref>] clades 20A, B, C, G) with the d614G mutation.<li> [9](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><year>2021</year>; <volume>6</volume>:<fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36301107">36301107</a> <b>(1)</b> Influenza Infection in Ferrets with SARS-CoV-2 Infection History<ul><li>The SARS-CoV-2 beta variant (B.1.351) was isolated from a COVID-19 patient (hCoV-19/Netherlands/NH-RIVM-20159/2021, <b>nextclade</b> 20H/501Y.V2).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36303187">36303187</a> <b>(1)</b> Case report: BA.1 subvariant showing a BA.2-like pattern using a variant-specific PCR assay due to a single point mutation downstream the spike 69/70 deletion<ul><li> That is the case of databases such as CoVSpectrum and <b>Nextclade</b> [<xref ref-type="bibr" rid="CR6"><u>6</u></xref>, <xref ref-type="bibr" rid="CR10"><u>10</u></xref>].<li> [6](covSPECTRUM [Internet]. [cited 2022 May 16]. Available from: https://cov-spectrum.org.) [10](Nextclade [Internet]. [cited 2022 May 25]. Available from: <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36366473">36366473</a> <b>(1)</b> Genomic Analysis of SARS-CoV-2 Alpha, Beta and Delta Variants of Concern Uncovers Signatures of Neutral and Non-Neutral Evolution<ul><li>The sequences were aligned using <b>Nextclade</b> sequence aligner [<xref rid="B27-viruses-14-02375" ref-type="bibr"><u>27</u></xref>], with NC_045512.2, the first sequenced SARS-CoV-2 genome from Wuhan [<xref rid="B28-viruses-14-02375" ref-type="bibr"><u>28</u></xref>], as a reference sequence to accelerate the calculations and to identify gene positions inside the Multiple Sequence Alignment (MSA) created.<li> [27](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [28](          <article-title>Severe Acute Respiratory Syndrome Coronavirus 2 Isolate Wuhan-Hu-1, Complete Genome. NCBI Reference Sequence: NC_045512.2</article-title>          <comment>Available online: <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/1798174254" ext-link-type="uri">https://www.ncbi.nlm.nih.gov/nuccore/1798174254</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-08-27">(accessed on 27 August 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36309317">36309317</a> <b>(2)</b> Genomic evolution of SARS-CoV-2 in Reunion Island<ul><li>Lineage assignments were performed using PANGOLIN (Phylogenetic Assignment of Named Global Outbreak LINeages) v3.1.2 (<xref rid="bb0090" ref-type="bibr"><u>O'Toole et al., 2021</u></xref>), and the <b>NextClade</b> CLI v1.7.0 (<xref rid="bb0005" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>).<li> The output from <b>NextClade</b> was used for sequence alignment and to identify the presence of specific substitutions, deletions, and insertions for each genome.<li> [O'Toole et al., 2021](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Underwood</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Jackson</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.T.</given-names>            </name>            <etal/>          </person-group>          <article-title>Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool</article-title>          <source>Virus Evol.</source>          <year>2021</year>          <fpage>7</fpage>          <pub-id pub-id-type="doi">10.1093/VE/VEAB064</pub-id>        ) [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/JOSS.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36048527">36048527</a> <b>(3)</b> SARS-CoV-2 reinfections during the Delta and Omicron waves<ul><li> Clade determination was performed via <b>NextClade</b> CLI v1.4.5 (<xref rid="B27" ref-type="bibr"><u>27</u></xref>).<li> Samples were given an initial determination as reinfection or persistence of initial genome based on whether there was a change in clade assigned by <b>NextClade</b>.<li> Afterward, the consensus genomes for each patient were manually reviewed in <b>NextClade</b> and/or Integrated Genomics Viewer to provide a final determination on reinfection or persistence.<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36272063">36272063</a> <b>(2)</b> SARS-CoV-2 Omicron BA.1, BA.2, and XAG identification during routine surveillance on a university campus in Belo Horizonte, Brazil, 2022<ul><li> 4.1.2) [<xref ref-type="bibr" rid="CR11"><u>11</u></xref>] and <b>Nextclade</b> web application (v.2.3.0) (Supplementary Table <xref rid="MOESM1" ref-type="media"><u>1</u></xref>).<li>According to the <b>Nextclade</b> classification, we identified 10 samples from the BA.1 clade: BA.1 (<italic>n</italic>&#x2009;=&#x2009;2), BA.1.1 (<italic>n</italic>&#x2009;=&#x2009;6), and BA.1.14.1 (<italic>n</italic>&#x2009;=&#x2009;2) and 45 samples in the BA.2 clade: BA.2 (<italic>n</italic>&#x2009;=&#x2009;33), BA.2.9 (<italic>n</italic>&#x2009;=&#x2009;2), BA.2.23 (<italic>n</italic>&#x2009;=&#x2009;1), BA.2.10 (<italic>n</italic>&#x2009;=&#x2009;1), BA.2.56 (<italic>n</italic>&#x2009;=&#x2009;2), BA.2.62 (<italic>n</italic>&#x2009;=&#x2009;2), and BA.2.81 (<italic>n</italic>&#x2009;=&#x2009;4) (Supplementary Table <xref rid="MOESM1" ref-type="media"><u>1</u></xref>).<li> [11](O&#x2019;Toole &#xC1;, Scher E, Underwood A, et al. (2021) Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool. Virus Evol 7(2):veab064) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36356586">36356586</a> <b>(1)</b> Antigenic cartography of well-characterized human sera shows SARS-CoV-2 neutralization differences based on infection and vaccination history<ul><li> The Pango classification tool (version 4.0.6) was used for lineage classification.<xref rid="bib59" ref-type="bibr"><sup>59</sup></xref> In a small minority of SARS-CoV-2 infections (<xref rid="mmc2" ref-type="supplementary-material"><u>Table&#xA0;S1</u></xref>B), a Pangolin lineage was unable to be ascertained and either a <b>Nextclade</b> clade<xref rid="bib59" ref-type="bibr"><sup>59</sup></xref> was used and/or a Pangolin lineage was inferred by manual inspection of key lineage-defining amino acid substitutions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298865">36298865</a> <b>(2)</b> SARS-CoV-2 from COVID-19 Patients in the Republic of Moldova: Whole-Genome Sequencing Results<ul><li> The <b>Nextclade</b> Pangolin lineage option was used when determining lineages using the Nextlade software, which was identified by applying a <b>Nextclade</b> web tool (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri> (accessed on 1 August 2022)) [<xref rid="B6-viruses-14-02310" ref-type="bibr"><u>6</u></xref>].<li> [6](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36299431">36299431</a> <b>(1)</b> A collaborative approach to improve representation in viral genomic surveillance<ul><li> 800 ng DNA) were sequenced on an R9.4.1 flow cell using a MinION Mk1B or Mk1C device, basecalled in real-time via MinKNOW, and analyzed using the wf-artic workflow (medaka, minimap2, bcftools, samtools, <b>nextclade</b>, artic, pangolin) according to the ONT protocol.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36332633">36332633</a> <b>(1)</b> Evidence for SARS-CoV-2 Delta and Omicron co-infections and recombination<ul><li> Another is to review every instance where a sample has good sequencing metrics but where methods like <b>Nextclade</b><xref rid="bib26" ref-type="bibr"><sup>26</sup></xref> or Pangolearn<xref rid="bib23" ref-type="bibr"><sup>23</sup></xref> have difficulty attributing a clade or a lineage to the sequence.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=37144832">37144832</a> <b>(4)</b> Validation of a new strategy for the identification of SARS-CoV-2 variants by sequencing the spike gene by Sanger<ul><li> They were aligned with the Sequencher 5.4.6 program (Gene Codes Corporation) and the variants were identified with the Nextstrain&#x2013;<b>Nextclade</b> Web 1.10.0 web-based program, <b>Nextclade</b> CLI 1.7.0 (09-12-2021) (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intr0025">https://clades.nextstrain.org/</ext-link>).<li> Variant nomenclature was performed according to the Nextstrain&#x2013;<b>Nextclade</b> Web 1.10.0 online program, <b>Nextclade</b> CLI 1.7.0 (2021-12-09) and PANGO Lineages (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/" id="intr0030">https://cov-lineages.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36251275">36251275</a> <b>(1)</b> High temporal resolution Nanopore sequencing dataset of SARS-CoV-2 and host cell RNAs<ul><li> <xref rid="sup9" ref-type="supplementary-material"><u>Supplementary figures</u></xref>, tables, and information files, including the assembled genome sequence and annotation and the full output of the <b>Nextclade</b>, Virstrain, and Readstatistc programs, as well as other supporting data, be found at the <italic toggle="yes">GigaScience</italic> GigaDB Database [<xref rid="bib45" ref-type="bibr"><u>45</u></xref>].<li> [45](<person-group person-group-type="author"><string-name><surname>Tomb&#xE1;cz</surname><given-names>D</given-names></string-name>, <string-name><surname>D&#xF6;rm&#x151;</surname><given-names>A</given-names></string-name>, <string-name><surname>Guly&#xE1;s</surname><given-names>G</given-names></string-name>, <etal>et al.</etal></person-group><article-title>Supporting data for &#x201C;High Temporal-Resolution Nanopore Sequencing Dataset of SARS-CoV-2 and Host Cell RNAs.&#x201D;</article-title>. <source>GigaScience Database</source>. <year>2022</year>. <pub-id pub-id-type="doi">10.5524/102256</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36250864">36250864</a> <b>(1)</b> Genome Sequence of an Early Imported Case of SARS-CoV-2 Delta Variant (B.1.617.2 AY.122) in Iraq in April 2021<ul><li> Lineages and clades were assigned using Pangolin v3.1.7 and the <b>Nextclade</b> webserver (<xref rid="tab1" ref-type="table"><u>Table&#xA0;1</u></xref>) (<xref rid="B8" ref-type="bibr"><u>8</u></xref>, <xref rid="B9" ref-type="bibr"><u>9</u></xref>).<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36263068">36263068</a> <b>(3)</b> Rapid transmission and tight bottlenecks constrain the evolution of highly transmissible SARS-CoV-2 variants<ul><li> We used <b>Nextclade</b> to annotate clades and variants of concern (<xref rid="R31" ref-type="bibr"><u>31</u></xref>).<li> The genomes were visualized using <b>Nextclade</b>.<li> Both <b>Nextclade</b> and Pango lineages are reported.<li> [31](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <name><surname>Roemer</surname><given-names>C</given-names></name>, <name><surname>Hodcroft</surname><given-names>EB</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><volume>6</volume>:<fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36263061">36263061</a> <b>(0)</b> A pseudovirus system enables deep mutational scanning of the full SARS-CoV-2 spike<ul><li> Briefly, we downloaded the pre-built UShER tree of all publicly available SARS-CoV-2 sequences (<xref rid="R63" ref-type="bibr"><u>Turakhia et al., 2021</u></xref>) as of Sept-26-2022 from <ext-link xlink:href="http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/09/26/public-2022-09-26.all.masked.<b>nextclade</b>.pangolin.pb.gz" ext-link-type="uri">http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2022/09/26/public-2022-09-26.all.masked.<b>nextclade</b>.pangolin.pb.gz</ext-link>.<li> [Turakhia et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36312916">36312916</a> <b>(1)</b> Limited permissibility of ENL-R and Mv-1-Lu mink cell lines to SARS-CoV-2<ul><li>Lineage (<b>Nextclade</b>)</th>                <th align="left" valign="top" rowspan="1" colspan="1">WHO Label</th>                <th align="left" valign="top" rowspan="1" colspan="1">CDC classification</th>                <th align="left" valign="top" rowspan="1" colspan="1">Country first identified (community)</th>              </tr>            </thead>            <tbody>              <tr>                <td align="char" valign="top" char="." rowspan="1" colspan="1">3.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36224216">36224216</a> <b>(1)</b> Clinical phenotypes and outcomes associated with SARS-CoV-2 variant Omicron in critically ill French patients with COVID-19<ul><li> Lineages and clades were interpreted using Pangolin and <b>NextClade</b>, before being submitted to the GISAID international database (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org">https://www.gisaid.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36328240">36328240</a> <b>(2)</b> Investigation of SARS-CoV-2 lineages and mutations circulating in a university-affiliated hospital in South Korea analyzed using Oxford Nanopore MinION sequencing<ul><li> The genomes were classified into PANGO lineages using Pangolin v.4.0.6 [<xref rid="b7-j-phrp-2022-0183" ref-type="bibr"><u>7</u></xref>], and mutation annotation was performed using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) and Coronapp (<ext-link xlink:href="http://giorgilab.unibo.it/coronannotator/" ext-link-type="uri">http://giorgilab.unibo.it/coronannotator/</ext-link>).<li> In total, 6 SARS-CoV-2 lineages were identified in the 58 samples based on Pangolin [<xref rid="b7-j-phrp-2022-0183" ref-type="bibr"><u>7</u></xref>] and <b>Nextclade</b> [<xref rid="b11-j-phrp-2022-0183" ref-type="bibr"><u>11</u></xref>].<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [11](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><etal/></person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Source Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36221029">36221029</a> <b>(1)</b> Local monitoring of SARS-CoV-2 variants in two large California counties in 2021<ul><li> Consensus genomes assessed with pangolin and <b>nextclade</b><sup><xref ref-type="bibr" rid="CR15"><u>15</u></xref></sup>.<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36217024">36217024</a> <b>(1)</b> Dynamic SARS-CoV-2 emergence algorithm for rationally-designed logical next-generation vaccines<ul><li> The SNP were inputted into the SnapGene sequence feature and <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR61"><u>61</u></xref></sup> to determine amino acid substitutions (AAS).<li> [61](Aksamentov, I. et al. Nextclade Web 1.5.0. Accessed July, 2021. <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36220486">36220486</a> <b>(2)</b> Genomic characteristics and viral load dynamics of a SARS-CoV-2 Omicron BA.2.2 variant from a hospitalized patient treated with molnupiravir<ul><li> All three genomes were assigned to clade 21&#xA0;L (Omicron, Pango lineage BA.2.2) by <b>Nextclade</b> (<xref rid="bb0045" ref-type="bibr"><b>Nextclade</b>, 2022</xref>) (Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36246576">36246576</a> <b>(1)</b> Host transcriptional responses in nasal swabs identify potential SARS-CoV-2 infection in PCR negative patients<ul><li> <b>Nextclade</b> analysis identified 19B (1), 20B (1), and 20C (5) strains of the SARS-CoV-2 virus.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36298754">36298754</a> <b>(1)</b> SARS-CoV-2 Transmission in Belgian French-Speaking Primary Schools: An Epidemiological Pilot Study<ul><li> Online <b>Nextclade</b> version 2.3.0 software as a first sequence aligner, allowing comparison to the Wuhan-hu-1/2019 (MN908947) SARS-CoV-2 reference genome and permitting a clade assignment (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>, accessed on 25 August 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36276383">36276383</a> <b>(2)</b> Guiding the design of SARS-CoV-2 genomic surveillance by estimating the resolution of outbreak detection<ul><li>Single nucleotide polymorphism (SNP) distances were determined from the consensus sequences using <monospace>snp-dists</monospace> (<ext-link xlink:href="https://github.com/tseemann/snp-dists" ext-link-type="uri">https://github.com/tseemann/snp-dists</ext-link> version 0.7.0) after aligning to the reference sequence (RefSeq accession NC_045512.2) with <monospace>nextalign</monospace> [<ext-link xlink:href="https://github.com/nextstrain/<b>nextclade</b>" ext-link-type="uri">https://github.com/nextstrain/<b>nextclade</b></ext-link> version 2.2.0 (<xref rid="B32" ref-type="bibr"><u>32</u></xref>)] and masking the first 100 and final 200 positions.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35905853">35905853</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36194133">36194133</a> <b>(1)</b> SARS-CoV-2 3CL<sup>pro</sup> mutations selected in a VSV-based system confer resistance to nirmatrelvir, ensitrelvir, and GC376<ul><li> A visualization application (<b>Nextclade</b> 2.4.0) was used to check mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36210871">36210871</a> <b>(1)</b> Molecular characterization of SARS&#x2010;CoV&#x2010;2 from the saliva of patients in the Republic of Korea in 2020<ul><li>SARS&#x2010;CoV&#x2010;2 sequences were classified into different lineages using the PANGOLIN webtool (Pango v.4.0.6), <b>Nextclade</b> (v.2.2.0), and phylogenetic analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36190984">36190984</a> <b>(1)</b> Validation of reduced S-gene target performance and failure for rapid surveillance of SARS-CoV-2 variants<ul><li> Variant calling, consensus sequence generation, and lineage/clade analysis is completed via Pangolin and <b>NextClade</b> pipelines.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36184049">36184049</a> <b>(5)</b> Dynamics of SARS-CoV-2 lineages in French Guiana in 2020&#x2013;2021: 4 epidemic waves with cross-influences from Europe and South America<ul><li> <b>Nextclade</b> 20I/V1) (<xref rid="bb0065" ref-type="bibr"><u>Genomic Epidemiology, 2022</u></xref>) was characterized by a higher intrinsic transmissibility than the prior variants (<xref rid="bb0140" ref-type="bibr"><u>Volz et al., 2021</u></xref>; <xref rid="bb0035" ref-type="bibr"><u>Davies et al., 2021</u></xref>; <xref rid="bb0105" ref-type="bibr"><u>Obermeyer et al., 2022</u></xref>).<li> <b>Nextclade</b> 20H/V2) was detected as early as May 2020 in South Africa and was characterized by an important immune evasion component (<xref rid="bb0060" ref-type="bibr"><u>Garcia-Beltran et al., 2021</u></xref>; <xref rid="bb0085" ref-type="bibr"><u>Hoffmann et al., 2021</u></xref>).<li> <b>Nextclade</b> 20&#xA0;J/V3), descendant of lineage B.1.1.28.<li> <b>Nextclade</b> 21A,I and J), first identified in India in October 2020 (<xref rid="bb0025" ref-type="bibr"><u>Cherian et al., 2021</u></xref>), became dominant worldwide from March 2021 on and proved to outcompete pre-existing lineages, including the other VOCs Alpha, Beta and Gamma.<li> Consensus sequences were then submitted to <b>Nextclade</b> (<xref rid="bb0110" ref-type="bibr"><u>O'Toole et al., 2021</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir0020">https://clades.nextstrain.org/</ext-link>) and Pangolin (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="ir0025">https://pangolin.cog-uk.io/</ext-link>) in order to assign them to the different clades and lineages (<xref rid="bb0110" ref-type="bibr"><u>O'Toole et al., 2021</u></xref>).<li> [Genomic Epidemiology, 2022](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A.</given-names>            </name>            <etal/>          </person-group>          <issue-title>Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations - SARS-CoV-2 Coronavirus / NCoV-2019</issue-title>          <source>Genomic Epidemiology</source>          <year>2022</year>          <comment>Available online:</comment>          <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563" id="ir0035">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</ext-link>          <comment>(accessed on 3 September 2021)</comment>        ) [Volz et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Volz</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Mishra</surname>              <given-names>S.</given-names>            </name>            <name>              <surname>Chand</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Barrett</surname>              <given-names>J.C.</given-names>            </name>            <name>              <surname>Johnson</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Hinsley</surname>              <given-names>W.R.</given-names>            </name>            <name>              <surname>Laydon</surname>              <given-names>D.J.</given-names>            </name>            <name>              <surname>Dabrera</surname>              <given-names>G.</given-names>            </name>            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>Ragonnet-Cronin</surname>              <given-names>M.</given-names>            </name>            <etal/>          </person-group>          <part-title>Transmission of SARS-CoV-2 Lineage B.1.1.7 in England: Insights from Linking Epidemiological and Genetic Data</part-title>          <year>2021</year>          <publisher-name>medRxiv</publisher-name>        ) [Davies et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33658326">33658326</a>) [Obermeyer et al., 2022](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35608456">35608456</a>) [Garcia-Beltran et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Garcia-Beltran</surname>              <given-names>W.F.</given-names>            </name>            <name>              <surname>Lam</surname>              <given-names>E.C.</given-names>            </name>            <name>              <surname>St. Denis</surname>              <given-names>K.</given-names>            </name>            <name>              <surname>Nitido</surname>              <given-names>A.D.</given-names>            </name>            <name>              <surname>Garcia</surname>              <given-names>Z.H.</given-names>            </name>            <name>              <surname>Hauser</surname>              <given-names>B.M.</given-names>            </name>            <name>              <surname>Feldman</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Pavlovic</surname>              <given-names>M.N.</given-names>            </name>            <name>              <surname>Gregory</surname>              <given-names>D.J.</given-names>            </name>            <name>              <surname>Poznansky</surname>              <given-names>M.C.</given-names>            </name>            <etal/>          </person-group>          <article-title>Multiple SARS-CoV-2 variants escape neutralization by vaccine-induced humoral immunity</article-title>          <source>Cell</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.1016/j.cell.2021.03.013</pub-id>        ) [Hoffmann et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Hoffmann</surname>              <given-names>M.</given-names>            </name>            <name>              <surname>Arora</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Gro&#xDF;</surname>              <given-names>R.</given-names>            </name>            <name>              <surname>Seidel</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>H&#xF6;rnich</surname>              <given-names>B.F.</given-names>            </name>            <name>              <surname>Hahn</surname>              <given-names>A.S.</given-names>            </name>            <name>              <surname>Kr&#xFC;ger</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Graichen</surname>              <given-names>L.</given-names>            </name>            <name>              <surname>Hofmann-Winkler</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Kempf</surname>              <given-names>A.</given-names>            </name>            <etal/>          </person-group>          <article-title>SARS-CoV-2 variants B.1.351 and P.1 escape from neutralizing antibodies</article-title>          <source>Cell</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.1016/j.cell.2021.03.036</pub-id>          <comment>S0092867421003676</comment>        ) [Cherian et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34361977">34361977</a>) [O'Toole et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [O'Toole et al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36296228">36296228</a> <b>(5)</b> First Detection of the SARS-CoV-2 Omicron BA.5/22B in Monaco<ul><li> accessed on 12 July 2022) [<xref rid="B3-microorganisms-10-01952" ref-type="bibr"><u>3</u></xref>])/21K (<b>Nextclade</b> classification ((<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>; accessed on 11 July 2022) [<xref rid="B4-microorganisms-10-01952" ref-type="bibr"><u>4</u></xref>,<xref rid="B5-microorganisms-10-01952" ref-type="bibr"><u>5</u></xref>]) and BA.2/21L (<uri xlink:href="https://covariants.org">https://covariants.org</uri>; accessed on 19 July 2022) [<xref rid="B6-microorganisms-10-01952" ref-type="bibr"><u>6</u></xref>,<xref rid="B7-microorganisms-10-01952" ref-type="bibr"><u>7</u></xref>].<li> Nucleotide and amino acid changes compared to the Wuhan-Hu-1 genome were identified with the <b>Nextclade</b> tool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>; accessed on 11 July 2022) [<xref rid="B4-microorganisms-10-01952" ref-type="bibr"><u>4</u></xref>,<xref rid="B5-microorganisms-10-01952" ref-type="bibr"><u>5</u></xref>].<li> Nextstrain clades and pangolin lineages were determined with the <b>Nextclade</b> web application (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>; accessed on 11 July 2022) [<xref rid="B4-microorganisms-10-01952" ref-type="bibr"><u>4</u></xref>,<xref rid="B5-microorganisms-10-01952" ref-type="bibr"><u>5</u></xref>] and the pangolin tool (<uri xlink:href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</uri>; accessed on 11 July 2022) [<xref rid="B22-microorganisms-10-01952" ref-type="bibr"><u>22</u></xref>], respectively.<li> Next-generation genome sequencing allowed obtaining a full-length genome that harbored all signature nucleotide substitutions and deletions of the Omicron BA.5/22B variant (<xref rid="microorganisms-10-01952-f001" ref-type="fig"><u>Figure 1</u></xref>) and was identified as of this genotype by the <b>Nextclade</b> and Pangolin tools, and by the phylogenetic analysis (<xref rid="microorganisms-10-01952-f002" ref-type="fig"><u>Figure 2</u></xref>).<li> Mutation Q556K in ORF1a was present in 5751 genomes overall, classified in several lineages including the B.1.177 lineage (or <b>Nextclade</b> 20A.EU1, or Marseille-2) and some Delta and BA.1 and BA.2 genomes.<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34292871">34292871</a>) [4](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [6](          <person-group person-group-type="author"><name><surname>Hodcroft</surname><given-names>E.</given-names></name></person-group>          <article-title>CoVariants: SARS-CoV-2 Mutations and Variants of Interest</article-title>          <year>2021</year>          <comment>Available online: <ext-link xlink:href="https://covariants.org/https://covariants.org/" ext-link-type="uri">https://covariants.org/https://covariants.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-07-19">(accessed on 19 July 2022)</date-in-citation>        ) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35083761">35083761</a>) [4](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [4](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36212105">36212105</a> <b>(1)</b> COVID-19 Whole-Genome Resequencing with Redundant Tiling PCR and Subtract-Based Amplicon Normalization Successfully Characterized SARS-CoV-2 Variants in Clinical Specimens<ul><li> The clades, SNVs, insertions, and deletions were annotated using the <b>Nextclade</b> web service [<xref rid="B22" ref-type="bibr"><u>22</u></xref>].<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36167869">36167869</a> <b>(1)</b> Detecting SARS-CoV-2 variants in wastewater and their correlation with circulating variants in the communities<ul><li> These variants are classified based on the <b>Nextclade</b> as well as pangolin lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36226173">36226173</a> <b>(1)</b> SARS-CoV-2 coinfections with variant genomic lineages identified by multiplex fragment analysis<ul><li> <b>Nextclade</b>, and non-amplicon-based platforms and bioinformatic tools such as RAT (Recombination Analysis Tool) or RDP (Recombination Detection Program) may also aid in the characterization of SARS-CoV-2 recombinatory events.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36325034">36325034</a> <b>(1)</b> SPREAD 4: online visualisation of pathogen phylogeographic reconstructions<ul><li>Owing to their flexibility in visualizing analyses for a wide range of pathogens and the ease with which the results can be shared via social media, Nextstrain (<xref rid="R18" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) and <b>Nextclade</b> (<xref rid="R1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>) have become primary examples that achieved broad visibility during the COVID-19 pandemic caused by the severe acute respiratory syndrome coronavirus 2 virus (SARS-CoV-2).<li> [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et&#xA0;al., 2021](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2021</year>) &#x2018;<article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>&#x2019;, <source><italic toggle="yes">Journal of open source software</italic></source>, <volume>6</volume>: <page-range>3773</page-range>. doi: <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36213661">36213661</a> <b>(3)</b> Genomic profile of SARS-CoV-2 Omicron variant and its correlation with disease severity in Rajasthan<ul><li>Fast alignment sequence test for application (FASTA) sequences generated from generate consensus plugin was downloaded and these were aligned with Wuhan-Hu-1/2019 (Genbank: <ext-link xlink:href="MN908947" ext-link-type="DDBJ/EMBL/GenBank">MN908947</ext-link>) as a reference, and the sequences were downloaded from GISAID (<xref rid="B7" ref-type="bibr"><u>7</u></xref>) using <b>Nextclade</b> (<ext-link xlink:href="https://clades.Nextstrain.org" ext-link-type="uri">https://clades.Nextstrain.org</ext-link>).<li> <b>Nextclade</b> carries out different processes, such as sequence alignment (pairwise alignment using a variation of the Smith-Waterman algorithm), clade assignment, and phylogenetic placement of these sequences, which can be visualized and/or downloaded.<li>Phylogenetic tree of Omicron built using the <b>Nextclade</b> online tool.<li> [7](<person-group person-group-type="author"><collab>Global Initiative on Sharing All Influenza Data (GISAID)</collab></person-group>. <source>2020 Clade and lineage nomenclature aids in genomic epidemiology studies of active hCoV-19 viruses</source>. Available online at: <ext-link xlink:href="https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/" ext-link-type="uri">https://www.gisaid.org/references/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/</ext-link> (accessed February 02, 2022).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36151084">36151084</a> <b>(3)</b> Using multiple sampling strategies to estimate SARS-CoV-2 epidemiological parameters from genomic sequencing data<ul><li> Both datasets were then processed using the <b>Nextclade</b> pipeline for quality control (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).<li> Briefly, the <b>Nextclade</b> pipeline examines the completeness, divergence, and ambiguity of bases in each genetic sequence.<li> by the <b>Nextclade</b> pipeline were selected.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36195045">36195045</a> <b>(7)</b> Genomic and structural mechanistic insight to reveal the differential infectivity of omicron and other variants of concern<ul><li> Additionally, a collection of 2365 high-quality reference genomes comprising different lineages in <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0015">https://clades.nextstrain.org/</ext-link>) were included as a reference for phylogenetic interpretation [<xref rid="bib36" ref-type="bibr"><u>36</u></xref>].<li> This complete dataset with 5609 sequences was used to perform phylogenetic inference using the <b>Nextclade</b> client interface (CLI) tool [<xref rid="bib36" ref-type="bibr"><u>36</u></xref>].<li>All the genomic sequences were aligned to the reference genome Wuhan-Hu-1 and the phylogenetic clade and lineage assignment was accomplished using <b>Nextclade</b> version 0.7.2 [<xref rid="bib36" ref-type="bibr"><u>36</u></xref>].<li><b>Nextclade</b> operates on banded Smith&#x2013;Waterman alignment and uses affine gap-penalty for sequence analysis [<xref rid="bib15" ref-type="bibr"><u>15</u></xref>].<li> The sequences analysed using <b>Nextclade</b> command-line interface (<b>Nextclade</b> CLI) contain a reference database of 2365 sequences representing the various lineages and use Nextalign tool to align the query sequences.<li>The present study includes data retrieval from GISAID, <b>Nextclade</b> and RCSB which are freely available public repositories and permit free data access for analysis.<li> [36](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [36](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [36](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [15](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>NextStrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics</source>          <volume>34</volume>          <year>2018</year>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36134610">36134610</a> <b>(1)</b> Characterization of Severe Acute Respiratory Syndrome Coronavirus 2 Omicron Variant Shedding and Predictors of Viral Culture Positivity on Vaccinated Healthcare Workers With Mild Coronavirus Disease 2019<ul><li> Sequences with &gt;50 times genome coverage were used to lineage classification by Pangolin version 3.1.5 (<ext-link xlink:href="http://pangolin.cog-uk.io/" ext-link-type="uri">http://pangolin.cog-uk.io/</ext-link>)[<xref rid="jiac391-B5" ref-type="bibr"><u>5</u></xref>] and <b>Nextclade</b> version 1.4.0 (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) and confirmed by manual genotyping.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36131001">36131001</a> <b>(1)</b> Tracking SARS-COV-2 variants using Nanopore sequencing in Ukraine in 2021<ul><li> Mutation calling and sequence quality were analysed using <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) and Geneious Prime 2021.2.2 software (Biomatters, New Zealand).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36417247">36417247</a> <b>(12)</b> Molecular Epidemiological Investigations of Localized SARS-CoV-2 Outbreaks-Utility of Public Algorithms<ul><li> We present three case examples where methods involving <b>Nextclade</b> and UShER algorithms, which are publicly available for clinical laboratory personnel, were used [<xref rid="B10-epidemiologia-03-00031" ref-type="bibr"><u>10</u></xref>,<xref rid="B11-epidemiologia-03-00031" ref-type="bibr"><u>11</u></xref>,<xref rid="B12-epidemiologia-03-00031" ref-type="bibr"><u>12</u></xref>].<li> Outbreak samples underwent phylogenetic clustering with locally sequenced samples utilizing <b>Nextclade</b>.<li>The clustering of patient CoV-2 sequences locally and within global databases was performed by <b>Nextclade</b>&#x2019;s algorithm (v1.14.1) <uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>) [<xref rid="B12-epidemiologia-03-00031" ref-type="bibr"><u>12</u></xref>] and UShER (<uri xlink:href="https://genome.ucsc.edu/cgi-bin/hgPhyloPlace">https://genome.ucsc.edu/cgi-bin/hgPhyloPlace</uri>) [<xref rid="B10-epidemiologia-03-00031" ref-type="bibr"><u>10</u></xref>].<li> <b>Nextclade</b> was also used for the determination of CoV-2 strain mutations in comparison to the original Wuhan strain.<li>The goal of this step is to assess or screen for signs of identity within the outbreak strains when compared to other sequences obtained from same laboratory (i.e., local) utilizing <b>Nextclade</b>.<li>In this step, we show the use of <b>Nextclade</b> nucleotide sequence alignment interface.<li> <italic toggle="yes">COVIDCG</italic> data, as well as <b>Nextclade</b> alignment, show that the Delta variant carries high frequency mutations in ORF1a.<li>This method is similar to local clustering by <b>Nextclade</b>; it produces a secondary layer of confirmation in strain-association.<li> <b>Nextclade</b> nucleotide alignment system was then used to analyze ORF1a gene of CoV-2 within the ten strains in-question in comparison to other non-related Delta-isolates.<li> <b>Nextclade</b> displayed four unique mutations that were not present in other Delta strains (<xref rid="epidemiologia-03-00031-f002" ref-type="fig"><u>Figure 2</u></xref>B&#x2013;red bars).<li> <b>Nextclade</b> nucleotide alignment system was used to analyze the ORF1a gene of CoV-2 (<xref rid="epidemiologia-03-00031-f003" ref-type="fig"><u>Figure 3</u></xref>B).<li> For example, one cannot produce a quantitative estimate for strain relatedness since no direct statistical comparison is involved other than qualitative grouping of the samples based on genetic similarities (<b>NextClade</b>, UShER) [<xref rid="B10-epidemiologia-03-00031" ref-type="bibr"><u>10</u></xref>,<xref rid="B11-epidemiologia-03-00031" ref-type="bibr"><u>11</u></xref>,<xref rid="B12-epidemiologia-03-00031" ref-type="bibr"><u>12</u></xref>].<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34469548">34469548</a>) [12](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [12](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33972780">33972780</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34469548">34469548</a>) [12](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36165924">36165924</a> <b>(2)</b> The virology of human monkeypox virus (hMPXV): A brief overview<ul><li> A compiled version of the genomes for visualization is available at <ext-link ext-link-type="uri" xlink:href="https://<b>nextclade</b>.org/monkeypox" id="interref0002">https://<b>nextclade</b>.org/monkeypox</ext-link> (Accessed 29th May 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36688184">36688184</a> <b>(1)</b> Occurrence of the Omicron variant of SARS-CoV-2 in northern Viet Nam in early 2022<ul><li> The sublineages were assigned using <b>Nextclade</b> V1.14.0 and Pangolin (lineage version 2022&#x2013;02&#x2013;28).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36145490">36145490</a> <b>(1)</b> SARS-CoV-2 Variants Identification: Overview of Molecular Existing Methods<ul><li>However, in most of these tools, the clades/lineages are assigned by one of two principal tools: <b>Nextclade</b> or Pangolin.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36108049">36108049</a> <b>(2)</b> The evolving SARS-CoV-2 epidemic in Africa: Insights from rapidly expanding genomic surveillance<ul><li> Briefly, all African sequences were passed through the <b>NextClade</b> analysis pipeline (<xref rid="R65" ref-type="bibr"><italic toggle="yes">65</italic></xref>) in order to identify and exclude: (<italic toggle="yes">i</italic>) sequences missing &gt;10% of the SARS-CoV-2 genome, (<italic toggle="yes">ii</italic>) sequences that deviate by &gt;70 nucleotides from the Wuhan reference strain, (<italic toggle="yes">iii</italic>) sequences with &gt;10 ambiguous bases, (<italic toggle="yes">iv</italic>) clustered mutations, and (<italic toggle="yes">v</italic>) sequences flagged with private mutations by <b>NextClade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36140622">36140622</a> <b>(2)</b> Molecular Characterization of Whole-Genome SARS-CoV-2 from the First Suspected Cases of the XE Variant in the Lazio Region, Italy<ul><li> after PANGOLIN analysis and classified into the XE clade by the <b>Nextclade</b> program.<li>Finally, the RDP4 analysis of INMI sequences showed a recombinant breakpoint between BA.1 and BA.2, approximately located at the nucleotide position 11,556 (99% confidence interval: 9460&#x2013;13,866), that is in accordance with a previously observed XE recombinant breakpoint [<xref rid="B19-diagnostics-12-02219" ref-type="bibr"><u>19</u></xref>], and thus compatible with XE classification of INMI sequences by the <b>Nextclade</b> program (<xref rid="app1-diagnostics-12-02219" ref-type="app"><u>Figure S1</u></xref>).<li> [19](          <person-group person-group-type="author"><collab>UK Health Security Agency</collab></person-group>          <article-title>Technical briefing 39</article-title>          <source>SARS-CoV-2 Variants of Concern and Variants under Investigation in England</source>          <comment>Crown Copyright 2022, England, March 2022; GOV-11753</comment>          <publisher-name>Department of Health and Social Care</publisher-name>          <publisher-loc>London, UK</publisher-loc>          <year>2022</year>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36140815">36140815</a> <b>(1)</b> Simultaneous SARS-CoV-2 Genome Sequencing of 384 Samples on an Illumina MiSeq Instrument through Protocol Optimization<ul><li>The consensus sequences in fasta format were analyzed using <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 13 September 2022) [<xref rid="B15-genes-13-01648" ref-type="bibr"><u>15</u></xref>] to determine the Nextstrain clades and identify mutations (<uri xlink:href="https://nextstrain.org/">https://nextstrain.org/</uri>, accessed on 13 September 2022 [<xref rid="B15-genes-13-01648" ref-type="bibr"><u>15</u></xref>].<li> [15](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [15](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36097810">36097810</a> <b>(3)</b> Regional importation and asymmetric within-country spread of SARS-CoV-2 variants of concern in the Netherlands<ul><li> Of note, four such VOCs arose into global prominence in late 2020, including Alpha (<b>Nextclade</b> 20I; PANGO lineage B.1.1.7), Beta (20 H; B.1.351), Gamma (20 J; P.1), and Delta (21 J; B.1.617.2), causing substantial levels of transmission worldwide, with Alpha and Delta being the most common VOCs globally in 2021 (<xref rid="bib5" ref-type="bibr"><u>Campbell et al., 2021</u></xref>).<li> Lineage proportions of sequences colored by <b>NextClade</b> genotype designations (3rd panel).<li>Using <b>NextClade</b> lineage assignment (<xref rid="bib1" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>), the viruses sampled at the start of the study period were largely genotyped as clade 20 A and its daughter lineages, 20B and 20E (EU1) (<xref rid="fig1" ref-type="fig"><u>Figure 1B&#x2013;C</u></xref>).<li> [Campbell et al., 2021](          <person-group person-group-type="author"><name><surname>Campbell</surname><given-names>F</given-names></name><name><surname>Archer</surname><given-names>B</given-names></name><name><surname>Laurenson-Schafer</surname><given-names>H</given-names></name><name><surname>Jinnai</surname><given-names>Y</given-names></name><name><surname>Konings</surname><given-names>F</given-names></name><name><surname>Batra</surname><given-names>N</given-names></name><name><surname>Pavlin</surname><given-names>B</given-names></name><name><surname>Vandemaele</surname><given-names>K</given-names></name><name><surname>Kerkhove</surname><given-names>MD</given-names></name><name><surname>Jombart</surname><given-names>T</given-names></name><name><surname>Morgan</surname><given-names>O</given-names></name><name><surname>Waroux</surname><given-names>OLP</given-names></name></person-group>          <year iso-8601-date="2021">2021</year>          <article-title>Increased transmissibility and global spread of sarscov- 2 variants of concern as at june 2021</article-title>          <source>Eurosurveillance</source>          <volume>26</volume>          <fpage>1</fpage>          <lpage>6</lpage>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2021.26.24.2100509/CITE/PLAINTEXT</pub-id>        ) [Aksamentov et al., 2021](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>          <year iso-8601-date="2021">2021</year>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Journal of Open Source Software</source>          <volume>6</volume>          <elocation-id>3773</elocation-id>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36031728">36031728</a> <b>(3)</b> The emergence, spread and vanishing of a French SARS&#x2010;CoV&#x2010;2 variant exemplifies the fate of RNA virus epidemics and obeys the Mistigri rule<ul><li> (A)&#xA0;The phylogenetic tree is adapted from a screenshot of the <b>nextclade</b> web application (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).<li> Mutation detection was performed using the <b>Nextclade</b> tool (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and freebayes (<ext-link xlink:href="https://github.com/freebayes/freebayes" ext-link-type="uri">https://github.com/freebayes/freebayes</ext-link>)<xref rid="jmv28102-bib-0053" ref-type="bibr"><sup>53</sup></xref> using a mapping quality score of 20 and results filtered by the Python script based on major nucleotide frequencies &#x2265;70% and nucleotide depths&#x2009;&#x2265;10 (when sequence reads were generated on the NovaSeq Illumina instrument) or &#x2265;5 (when sequence reads were generated on the MiSeq Illumina instrument).<li> Nextstrain clades and Pangolin lineages provided in the present study were determined using the <b>Nextclade</b> web application (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>)<xref rid="jmv28102-bib-0013" ref-type="bibr"><sup>13</sup></xref>, <xref rid="jmv28102-bib-0054" ref-type="bibr"><sup>54</sup></xref> and the Pangolin web application (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>),<xref rid="jmv28102-bib-0014" ref-type="bibr"><sup>14</sup></xref> respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36164552">36164552</a> <b>(3)</b> Occurrence of SARS-CoV-2 reinfections at regular intervals in Ecuador<ul><li> Consensus sequence quality was assessed by <b>Nextclade</b> v1.14.0 parameters (clades.nextstrain.org) and were uploaded to the GISAID repository (gisaid.org).<li>Clades and linages assignments were performed using <b>Nextclade</b> and Pangolin COVID-19 Lineage Assigner v4.0.5 (pangolin.cog-uk.io), respectively.<li> <b>Nextclade</b> was used to identify nucleotide mutations and amino acid substitutions on each sequence relative to the reference genome.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36159654">36159654</a> <b>(2)</b> Tracing the trajectories of SARS-CoV-2 variants of concern between December 2020 and September 2021 in the Canary Islands (Spain)<ul><li> <b>Nextclade</b> v.0.14.3 (<xref rid="B1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>) was used for variant calling and functional predictions.<li> by <b>Nextclade</b> software were excluded.<li> [Aksamentov et&#xA0;al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><name><surname>Neher</surname><given-names>R. A.</given-names></name></person-group>. (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J. Open Source Softw.</source><volume>6</volume> (<issue>67</issue>), <elocation-id>3773</elocation-id>. doi:&#xA0;<pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36087642">36087642</a> <b>(3)</b> Critical pediatric neurological illness associated with COVID-19 (Omicron BA.2.3.7 variant) infection in Taiwan: immunological assessment and viral genome analysis in tertiary medical center<ul><li> To investigate the amino acid variations in the S region of these Taiwanese strains and classify pathogen evolution relationships, further sequence analyses were executed using <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="http://clades.nextstrain.org/" id="interref0001a">http://clades.nextstrain.org/</ext-link>) with Augur developed by Nextstrain (<xref rid="bib0004" ref-type="bibr">Hadfield <italic>et&#xA0;al.<li><b>Nextclade</b> showed that the phylogenetic relationships of all the sequences were clustered as a new node different from other BA.2.3.7 lineage strains (<xref rid="fig0001" ref-type="fig"><u>Figure&#xA0;1</u></xref>a).<li>(a) Phylogenetic relationship of 21 Taiwan Omicron sequences based on <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36069609">36069609</a> <b>(1)</b> A Robust, Highly Multiplexed Mass Spectrometry Assay to Identify SARS-CoV-2 Variants<ul><li> All FASTA consensus genome sequences underwent mutation calling and phylogenetic lineage assignment by the <b>Nextclade</b> Web Interface (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>, accessed April 18, 2022) and the Pangolin COVID-19 Lineage Assigner (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>, accessed April 18, 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36068236">36068236</a> <b>(1)</b> Genomic surveillance of SARS-CoV-2 Omicron variants on a university campus<ul><li> The latter sequences were screened using <b>Nextclade</b> version 1.10.0<sup><xref ref-type="bibr" rid="CR44"><u>44</u></xref></sup> and any sequences deemed to be of &#x201C;bad&#x201D; or &#x201C;mediocre&#x201D; quality by this tool (due to missing data, mixed sites, private mutations, mutation clusters, frameshifts, or stop codons) were excluded from further analyses.<li> [44](Aksamentov, I., Roemer, C., Hodcroft, E. B. &amp; Neher, R. A. Nextclade: clade assignment, mutation calling and quality control for viral genomes. <italic>J. Open Source Softw</italic>. <bold>6</bold>, 3773 (2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36090517">36090517</a> <b>(1)</b> Clinical outcomes in individuals hospitalized with SARS-CoV-2 Delta variant (B.1.617.2) who had been vaccinated with Covishield (ChAdOx1) and Covaxin (BBV-152)<ul><li> Mutations in the sequences were identified using <b>Nextclade</b> v1.1.0 (<xref rid="bib0013" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36070806">36070806</a> <b>(4)</b> A 21L/BA.2-21K/BA.1 &#x201C;MixOmicron&#x201D; SARS-CoV-2 hybrid undetected by qPCR that screen for variant in routine diagnosis<ul><li> Substantial rates of co-incidence over periods of several weeks have thus been reported recently worldwide for the Delta [WHO denomination (<ext-link ext-link-type="uri" xlink:href="https://www.who.int/fr/activities/tracking-SARS-CoV-2-variants" id="ir0010">https://www.who.int/fr/activities/tracking-SARS-CoV-2-variants</ext-link>)]/21J [<b>Nextclade</b> classification (<xref rid="bb0005" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/" id="ir0015">https://nextstrain.org/</ext-link>)] and Omicron 21K/BA.1 [Pangolin classification (<xref rid="bb0190" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/resources/pangolin.html" id="ir0020">https://cov-lineages.org/resources/pangolin.html</ext-link>)] variants, and thereafter, to a lesser extent, for the Omicron BA.1 and BA.2 variants (<ext-link ext-link-type="uri" xlink:href="https://covariants.org/per-country" id="ir0025">https://covariants.org/per-country</ext-link>) (<xref rid="bb0100" ref-type="bibr"><u>Hodcroft, 2021</u></xref>; <xref rid="bb0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), including in our geographical area (<xref rid="f0005" ref-type="fig">Fig.<li> Nucleotide and amino acid changes compared to the Wuhan-Hu-1 isolate genome were determined using the <b>Nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir0070">https://clades.nextstrain.org/</ext-link>) (<xref rid="bb0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bb0005" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>).<li> Nextstrain clades and Pangolin lineages were identified with the <b>Nextclade</b> web application (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir0075">https://clades.nextstrain.org/</ext-link>) (<xref rid="bb0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="bb0005" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) and the Pangolin tool (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/pangolin.html" id="ir0080">https://cov-lineages.org/pangolin.html</ext-link>) (<xref rid="bb0190" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), respectively.<li> Adapted from screenshots of the <b>nextclade</b> web application output (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="ir0005">https://clades.nextstrain.org</ext-link>) (<xref rid="bb0005" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>; <xref rid="bb0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Zenodo</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hodcroft, 2021](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>          </person-group>          <article-title>CoVariants: SARS-CoV-2 Mutations and Variants of Interest</article-title>          <comment>Available from:</comment>          <ext-link ext-link-type="uri" xlink:href="https://covariants.org/https://covariants.org/" id="ir0130">https://covariants.org/https://covariants.org/</ext-link>          <year>2021</year>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Zenodo</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Zenodo</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>Zenodo</source>          <year>2021</year>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36093336">36093336</a> <b>(4)</b> Benchmark datasets for SARS-CoV-2 surveillance bioinformatics<ul><li> For lineage detection, these platforms include Pangolin and <b>Nextclade</b> (<xref rid="ref-28" ref-type="bibr"><u>O&#x2019;Toole et al., 2021</u></xref>; <xref rid="ref-17" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li><b>nextclade</b>_aa_dels.<li><b>nextclade</b>_aa_subs.<li><b>nextclade</b>_version.<li> [O&#x2019;Toole et al., 2021](          <person-group person-group-type="author"><name><surname>O&#x2019;Toole</surname><given-names>&#xC1;</given-names></name><name><surname>Scher</surname><given-names>E</given-names></name><name><surname>Underwood</surname><given-names>A</given-names></name><name><surname>Jackson</surname><given-names>B</given-names></name><name><surname>Hill</surname><given-names>V</given-names></name><name><surname>McCrone</surname><given-names>JT</given-names></name><name><surname>Colquhoun</surname><given-names>R</given-names></name><name><surname>Ruis</surname><given-names>C</given-names></name><name><surname>Abu-Dahab</surname><given-names>K</given-names></name><name><surname>Taylor</surname><given-names>B</given-names></name><name><surname>Yeats</surname><given-names>C</given-names></name><name><surname>Du Plessis</surname><given-names>L</given-names></name><name><surname>Maloney</surname><given-names>D</given-names></name><name><surname>Medd</surname><given-names>N</given-names></name><name><surname>Attwood</surname><given-names>SW</given-names></name><name><surname>Aanensen</surname><given-names>DM</given-names></name><name><surname>Holmes</surname><given-names>EC</given-names></name><name><surname>Pybus</surname><given-names>OG</given-names></name><name><surname>Rambaut</surname><given-names>A</given-names></name></person-group>          <year iso-8601-date="2021">2021</year>          <article-title>Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool</article-title>          <source>Virus Evolution</source>          <volume>7</volume>          <issue>2</issue>          <pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>        ) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36144387">36144387</a> <b>(6)</b> Towards Efficient and Accurate SARS-CoV-2 Genome Sequence Typing Based on Supervised Learning Approaches<ul><li> For&#xA0;the Nextstrain part, labels of sequences were obtained from the <b>Nextclade</b> system (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 12 July 2022)).<li> We referred to the nucleotide site screening protocol provided by <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 12 July 2022)) and PangoLEARN (<uri xlink:href="https://github.com/cov-lineages/pangoLEARN/">https://github.com/cov-lineages/pangoLEARN/</uri> (accessed on 12 July 2022)) and selected feature extraction strategies suitable for different typing methods through subsequent model training and cross-validation.<li> Specifically, <b>Nextclade</b> defined each clade by the combination of signature mutations, providing a total of 83 mutations for 25 clades.<li> According to <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 12 July 2022)), 22A (Omicron) and 22B (Omicron) evolved from 21L (Omicron).<li> On the other hand, <b>Nextclade</b> [<xref rid="B44-microorganisms-10-01785" ref-type="bibr"><u>44</u></xref>] is an open-source project for viral genome alignment, mutation calling, clade assignment, quality checks, and phylogenetic placement.<li> Compared with <b>Nextclade</b> and UShER, this study does not construct the evolutionary tree but focuses on the typing of genomes.<li> [44](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36118859">36118859</a> <b>(1)</b> A high scale SARS-CoV-2 profiling by its whole-genome sequencing using Oxford Nanopore Technology in Kazakhstan<ul><li> The pipeline output includes <b>NextClade</b> and Pangolin analysis that includes the clade designation according to GISIAD and Pangolin nomenclature.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36050650">36050650</a> <b>(2)</b> Genomic surveillance of SARS-COV-2 reveals diverse circulating variant lineages in Nairobi and Kiambu Counties, Kenya<ul><li> Nucleotide mutations and amino acid changes within 12 SARS-CoV-2 genes for all samples were subsequently detected through <b>NextClade</b> (v0.13.0), using standard parameters [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>].<li> Sequence variants across the 55 isolates using <b>NextClade</b> described nucleotide mutations (range: 5 to 48) leading to 4 amino acid changes (range: 4 to 23) and deletions (range: 2 to 8).<li> [20](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Source Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35925013">35925013</a> <b>(1)</b> Highly Divergent SARS-CoV-2 Alpha Variant in Chronically Infected Immunocompromised Person<ul><li> Pangolin version 4.0.6 PLEARN-v1.8 classification using default settings demonstrated that all sequenced viruses belonged to the Alpha (B.1.1.7) variant of concern (VOC) (<xref rid="R5" ref-type="bibr"><italic>5</italic></xref>), <b>Nextclade</b> version 1.14.1 strain 20I (<xref rid="R6" ref-type="bibr"><italic>6</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35863699">35863699</a> <b>(3)</b> Clinical Performance Characteristics of the Swift Normalase Amplicon Panel for Sensitive Recovery of Severe Acute Respiratory Syndrome Coronavirus 2 Genomes<ul><li> Consensus sequences are analyzed using Pangolin (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="intref0050"><italic>https://pangolin.cog-uk.io</italic></ext-link>, last accessed July 5, 2022) to obtain PANGO lineage designations and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="intref0055"><italic>https://clades.nextstrain.org</italic></ext-link>, last accessed July 5, 2022) to generate Nextstrain clade assignments and a list of mutations.<li> The mutation list generated from <b>Nextclade</b> is parsed to extract amino acid changes in spike.<li> PANGO and <b>Nextclade</b> lineages were compared for consensus sequences generated by our pipeline against benchmark deposited consensus genomes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36037469">36037469</a> <b>(1)</b> The Association of Baseline Plasma SARS-CoV-2 Nucleocapsid Antigen Level and Outcomes in Patients Hospitalized With COVID-19<ul><li> Among 811 samples that were characterized as Delta via PCR, 810 (99.9%) were confirmed as Delta by either Pangolin or <b>Nextclade</b> sequencing; 3 samples among 364 identified as non-Delta by quantitative PCR were found to be Delta by sequence analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36111122">36111122</a> <b>(2)</b> Implemented occupational health surveillance limits the spread of SARS-CoV-2 Omicron at the workplace<ul><li> SARS-CoV-2 consensus genomes were aligned and assigned to global outbreak lineages with Pangolin v.3.1.17 (<xref rid="B12" ref-type="bibr"><u>12</u></xref>), and to Nextstrain clades with <b>NextClade</b> v.1.5.1 (<xref rid="B13" ref-type="bibr"><u>13</u></xref>).<li> <bold>(A)</bold> Maximum-Likelihood phylogeny of the isolated genome consensus sequences and publicly available datasets under <b>NextClade</b> v.1.5.1.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [13](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J Open Source Softw.</italic></source> (<year>2021</year>) <volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36055471">36055471</a> <b>(3)</b> Genomic and epidemiologic surveillance of SARS-CoV-2 in Southern Brazil and identification of a new Omicron-L452R sublineage<ul><li> Phylogenetic tree was constructed including all SARS-CoV-2 complete genomes available from GISAID through the <b>Nextclade</b> tool on Nextstrain server (<xref rid="bib0006" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>).<li> Regarding genetic characterization, the sequences were aligned with complete SARS-CoV-2 genomes of different lineages through the <b>NextClade</b> online tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0006">https://clades.nextstrain.org/</ext-link>) and the phylogenetic tree was inferred (<xref rid="fig0001" ref-type="fig"><u>Fig.&#xA0;1</u></xref>a).<li><bold>Phylogenetic and phylodynamic of SARS-CoV-2 variant analysis in Rio Grande do Sul State.</bold> Phylogenetic tree of all circulating lineages, developed through the Nextstrain server (<b>Nextclade</b> tool), using GISAID data.<li> [Hadfield&#xA0;et&#xA0;al., 2018](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>S.M.</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B.</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Sagulenko</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinformatics</source>          <volume>34</volume>          <year>2018</year>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36146724">36146724</a> <b>(2)</b> SARS-CoV-2 Infections in Vaccinated and Unvaccinated Populations in Camp Lemonnier, Djibouti, from April 2020 to January 2022<ul><li> The clade assignments and consensus mutations were determined using <b>Nextclade</b> CLI 1.2.0 and Nextalign CLI 1.2.0.<li> The metadata (collection date, consensus genome length, Pango lineage, <b>NextClade</b>, and GISAID accessions) associated with 589 samples (the coding-complete and those with a consensus genome length &#x2265; 20,000 nt) can be found in <xref rid="app1-viruses-14-01918" ref-type="app"><u>Supplementary Table S1</u></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36145416">36145416</a> <b>(3)</b> Strain Variation Based on Spike Glycoprotein Gene of SARS-CoV-2 in Kuwait from 2020 to 2021<ul><li> The phylogenetic analytical tree of 35 SARS-CoV-2 isolates from Kuwait and the global strains based on the S glycoprotein gene was performed by <b>Nextclade</b> <sup>beta</sup> of Nextstrain platform, which is based on banded Smith&#x2013;Waterman alignment with an affine gap-penalty.<li>According to <b>Nextclade</b> sequence analysis, the 19B variant in Kuwait showed 7&#x2013;9% divergence at the nucleotide level from the original clade 19A (<xref rid="pathogens-11-00985-f001" ref-type="fig"><u>Figure 1</u></xref>C).<li> In addition, sequence analysis based on the <b>Nextclade</b> platform demonstrated that 20I (Alpha, V1) variant in Kuwait showed 28&#x2013;30% divergence at the nucleotide level from the original clade 19A (<xref rid="pathogens-11-00985-f001" ref-type="fig"><u>Figure 1</u></xref>B).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36038563">36038563</a> <b>(5)</b> Genomic epidemiology and emergence of SARS-CoV-2 variants of concern in the United Arab Emirates<ul><li> The generated FASTA of each samples were assigned for particular lineages using Pangolin COVID-19 lineage assigner (v 3.1.19) and <b>Nextclade</b> 0.14.1<sup><xref ref-type="bibr" rid="CR29"><u>29</u></xref>,<xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup>, and labeled according to WHO nomenclature.<li> <b>Nextclade</b> 0.14.1 default quality control was adopted in this study for mutation call<sup><xref ref-type="bibr" rid="CR30"><u>30</u></xref></sup>.<li> Samples that did not pass <b>Nextclade</b> quality check were excluded.<li> The quality control used by <b>Nextclade</b> includes the number missing and ambiguous nucleotides, degree of divergence, and clustered differences.<li> The total number of samples that failed <b>Nextclade</b> quality check were 47.<li> [29](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;</given-names>            </name>            <etal/>          </person-group>          <article-title>Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool</article-title>          <source>Virus Evolut.</source>          <year>2021</year>          <volume>7</volume>          <issue>2</issue>          <fpage>veab064</fpage>          <pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>        ) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36146700">36146700</a> <b>(1)</b> Assessing the Impact of SARS-CoV-2 Lineages and Mutations on Patient Survival<ul><li>With the aim of having all the genomic variants in our dataset, the whole set of consensus genomes, regardless of missing data, has been used as input to the <b>Nextclade</b> software [<xref rid="B45-viruses-14-01893" ref-type="bibr"><u>45</u></xref>].<li> [45](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade. Viral Genome Clade Assignment, Mutation Calling, and Sequence Quality Checks</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-08-03">(accessed on 3 August 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36046362">36046362</a> <b>(1)</b> STArS (STrain-Amplicon-Seq), a targeted nanopore sequencing workflow for SARS-CoV-2 diagnostics and genotyping<ul><li> The final consensus sequences were also uploaded in <b>Nextclade</b> [<xref rid="bpac020-B49" ref-type="bibr"><u>49</u></xref>], to obtain strain identification.<li> [49](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>E</given-names></string-name></person-group><etal>et al</etal><article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Sorce Softw</source><year>2021</year>;<volume>6</volume>:<fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36140181">36140181</a> <b>(1)</b> SARS-CoV-2 Variability in Patients and Wastewaters&#x2014;Potential Immuno-Modulation during the Shift from Delta to Omicron<ul><li> then variant proportions in each mixture were determined according to covariants, CoV-Spectrum data, <b>Nextclade</b> assignment [<xref rid="B11-biomedicines-10-02080" ref-type="bibr"><u>11</u></xref>] and alignment with Wuhan reference strain (EPI ISL 402124), Delta (EPI ISL 11448607), and Omicron sequences (EPI_ISL_11352845; EPI_ISL_12179603) from GISAID (Global Initiative on Sharing Avian Influenza Data, <uri xlink:href="http://gisaid.org">http://gisaid.org</uri>, assessed on 1 March 2022).<li> [11](          <person-group person-group-type="author"><name><surname>Chen</surname><given-names>C.</given-names></name><name><surname>Nadeau</surname><given-names>S.</given-names></name><name><surname>Yared</surname><given-names>M.</given-names></name><name><surname>Voinov</surname><given-names>P.</given-names></name><name><surname>Xie</surname><given-names>N.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Stadler</surname><given-names>T.</given-names></name></person-group>          <article-title>CoV-Spectrum: Analysis of Globally Shared SARS-CoV-2 Data to Identify and Characterize New Variants</article-title>          <source>Bioinformatics</source>          <year>2021</year>          <volume>38</volume>          <fpage>1735</fpage>          <lpage>1737</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/btab856</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36008511">36008511</a> <b>(5)</b> SARS-CoV-2 genome variations and evolution patterns in Egypt: a multi-center study<ul><li>Finally, clade assignment were performed using Ultrafast Sample placement on Existing tRee (UShER)<sup><xref ref-type="bibr" rid="CR25"><u>25</u></xref></sup> and <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup>.<li> Downstream analysis were performed in R<sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup> in which heatmap computed using ComplexHeatmap package<sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>, phylogenetic tree visualized using ggtree<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup> and <b>Nextclade</b> online tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup>.<li>To determine phylogenetic characteristics of our SARS-CoV-2 isolates, we performed maximum likelihood tree based on aligned full length sequence of our genomic sequences using IQ-tree and <b>Nextclade</b> online tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) (Fig.&#xA0;<xref rid="Fig1" ref-type="fig"><u>1</u></xref>).<li> (<bold>b</bold>) 110 sequenced samples included in this study were placed on <b>Nextclade</b> <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link><sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup>.<li> The first group (Fig.&#xA0;<xref rid="Fig4" ref-type="fig"><u>4</u></xref>a) comprised 14 mutations that were prevalent in PANGO lineage A.28 and <b>Nextclade</b> 19B and are highly correlated.<li> [25](          <person-group person-group-type="author">            <name>              <surname>Turakhia</surname>              <given-names>Y</given-names>            </name>            <etal/>          </person-group>          <article-title>Ultrafast sample placement on existing trees (UShER) empowers real-time phylogenetics for the SARS-CoV-2 pandemic</article-title>          <source>bioRxiv preprint Server Biol.</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.09.26.314971</pub-id>        ) [26](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [20](R Core Team. <italic>R: A Language and Environment for Statistical Computing</italic>. R Foundation for Statistical Computing, Vienna, Austria (2020).) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27207943">27207943</a>) [28](          <person-group person-group-type="author">            <name>              <surname>Yu</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Using ggtree to visualize data on tree-like structures</article-title>          <source>Curr. Protoc. Bioinform.</source>          <year>2020</year>          <volume>69</volume>          <fpage>e96</fpage>          <pub-id pub-id-type="doi">10.1002/cpbi.96</pub-id>        ) [26](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [26](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36140162">36140162</a> <b>(2)</b> Clinical Impact of Monoclonal Antibodies in the Treatment of High-Risk Patients with SARS-CoV-2 Breakthrough Infections: The ORCHESTRA Prospective Cohort Study<ul><li> Consensus viral sequences were extracted, clades and lineages assigned using NextStrain (<uri xlink:href="https://nextstrain.org/<b>nextclade</b>/sars-cov-2?branchLabel=aa">https://nextstrain.org/<b>nextclade</b>/sars-cov-2?branchLabel=aa</uri> (accessed on 3 August 2022)) and Phylogenic Assignment of named Global Outbreak LINeages (PANGOLIN) [<xref rid="B16-biomedicines-10-02063" ref-type="bibr"><u>16</u></xref>].<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34095513">34095513</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36000731">36000731</a> <b>(3)</b> Multiple Introductions of SARS-CoV-2 Alpha and Delta Variants into White-Tailed Deer in Pennsylvania<ul><li>To assess relationships among deer and human-derived SARS-CoV-2 sequences, we used <b>NextClade</b> for sequence alignment and IQ-TREE to construct a maximum-likelihood phylogenetic tree for viral sequences from alpha (<xref rid="fig3" ref-type="fig"><u>Fig.&#xA0;3</u></xref>) and delta (<xref rid="fig4" ref-type="fig"><u>Fig.&#xA0;4</u></xref>) variants, in each case comparing PA deer to their 100 nearest genetic neighbors from NCBI&#x2019;s global human data set and contemporaneous human-derived sequences from the same variant in the Delaware Valley region.<li> Each data set was aligned using <b>NextClade</b> with Wuhan-Hu-1 as a reference (<ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512.2" ext-link-type="uri" assigning-authority="insdc">NC_045512.2</ext-link>).<li> <b>NextClade</b> was used to align sequences to the Wuhan reference (<xref rid="B53" ref-type="bibr"><u>53</u></xref>).<li> [53](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source><volume>6</volume>. doi:<pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36000897">36000897</a> <b>(1)</b> Emergence and Spread of the SARS-CoV-2 Variant of Concern Delta across Different Brazilian Regions<ul><li>The <b>Nextclade</b> algorithm (<xref rid="B41" ref-type="bibr"><u>41</u></xref>) was used to access the molecular composition of the Brazilian genomes.<li> [41](<article-title>Nextclade</article-title>. <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link>. <date-in-citation content-type="access-date">Accessed 20 June 2022</date-in-citation>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36146662">36146662</a> <b>(1)</b> Effective Approaches to Study the Genetic Variability of SARS-CoV-2<ul><li> The <b>Nextclade</b> [<xref rid="B27-viruses-14-01855" ref-type="bibr"><u>27</u></xref>] tool was used to identify the strains present in the studied samples.<li> [27](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36146671">36146671</a> <b>(1)</b> Genomic Surveillance of SARS-CoV-2 in the Southern Province of Zambia: Detection and Characterization of Alpha, Beta, Delta, and Omicron Variants of Concern<ul><li> The NextSeq 2000 optionally performs lineage/clade analysis using Pangolin and <b>NextClade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36052371">36052371</a> <b>(1)</b> Mucosal gene expression in response to SARS-CoV-2 is associated with early viral load<ul><li> All SARS-CoV-2 genomes were identified were placed to the B.1 lineage in the <b>Nextclade</b> tree [<xref rid="R13" ref-type="bibr"><u>13</u></xref>].<li> [13](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <article-title>Roemer, C., Hodcroft, E. B., &amp; Neher, R. A.,. Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source>. <year>2021</year>;<volume>6</volume>(<issue>67</issue>):<fpage>3773</fpage>. doi: <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35997285">35997285</a> <b>(1)</b> Genomic Epidemiology and Serology Associated with a SARS-CoV-2 R.1 Variant Outbreak in New Jersey<ul><li>The resulting SARS-CoV-2 viral genome sequences were characterized by <b>Nextclade</b> CLI (v1) (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) to assign Nextstrain clades (<xref rid="B24" ref-type="bibr"><u>24</u></xref>).<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35998195">35998195</a> <b>(1)</b> Urgent need for a non-discriminatory and non-stigmatizing nomenclature for monkeypox virus<ul><li> Designated lineages can be visualized on Nextstrain (<ext-link xlink:href="https://nextstrain.org/monkeypox/hmpxv1" ext-link-type="uri">https://nextstrain.org/monkeypox/hmpxv1</ext-link>) and novel sequences can be assigned using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36146659">36146659</a> <b>(3)</b> Healthcare-Associated SARS-CoV-2 Reinfection after 3 Months with a Phylogenetically Distinct Omicron Variant: A Case Report<ul><li> variant was detected, both using the pangolin and <b>Nextclade</b> lineage classification with high confidence.<li> <b>Nextclade</b> [<xref rid="B20-viruses-14-01852" ref-type="bibr"><u>20</u></xref>] Web 2.2.0 was used to create the viral base variation figure (<xref rid="viruses-14-01852-f004" ref-type="fig"><u>Figure 4</u></xref>A) and <b>Nextclade</b> CLI 2.2.0 was also used for lineage analysis.<li> [20](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35677069">35677069</a> <b>(1)</b> Imprinted antibody responses against SARS-CoV-2 Omicron sublineages<ul><li> Sequence quality was determined using <b>Nextclade</b> version 1.0.0-alpha.8 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35993116">35993116</a> <b>(1)</b> Understanding Early Pandemic Severe Acute Respiratory Syndrome Coronavirus 2 Transmission in a Medical Center by Incorporating Public Sequencing Databases to Mitigate Bias<ul><li>We aligned samples using MAFFT 7.490 software (via <b>Nextclade</b>) [<xref rid="jiac348-B15" ref-type="bibr"><u>15</u></xref>], masked the 5&#x2019; and 3&#x2019; ends of each genome and generated a pairwise distance matrix of genomes using the K80 model of evolution using the ape R package [<xref rid="jiac348-B16" ref-type="bibr"><u>16</u></xref>, <xref rid="jiac348-B17" ref-type="bibr"><u>17</u></xref>].<li> [15](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>E</given-names></string-name>, <string-name><surname>Neher</surname><given-names>R</given-names></string-name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><year>2021</year>; <volume>6</volume>:<fpage>3773</fpage>.) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30016406">30016406</a>) [17](<person-group person-group-type="author"><collab>R Core Team</collab></person-group>. <source>R: a language and environment for statistical computing</source>. <publisher-loc>Vienna, Austria</publisher-loc>: <publisher-name>R Foundation for Statistical Computing</publisher-name>; <year>2021</year>. <comment>Available at</comment><source>:</source><ext-link xlink:href="https://www.R-project.org/" ext-link-type="uri">https://www.R-project.org/</ext-link>. Accessed 20 August 2022.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35995785">35995785</a> <b>(1)</b> Antiviral and clinical activity of bamlanivimab in a randomized trial of non-hospitalized adults with COVID-19<ul><li> SARS-COV-2 variant calling was done using 3 different variant calling platforms, namely, CoVDB<sup><xref ref-type="bibr" rid="CR26"><u>26</u></xref></sup>, Scorpio call v1.2.123 (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>)<sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>, and <b>Nextclade</b> v.1.13.2 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR28"><u>28</u></xref></sup>.<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35263335">35263335</a>) [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [28](Aksamentov, I., Roemer, C., Hodcroft, E. &amp; Neher, R. Nextclade: clade assignment, mutation calling and quality control for viral genomes. <italic>J. Open Source Softw.</italic><bold>6</bold>, 3773 (2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35995311">35995311</a> <b>(1)</b> A distinct phylogenetic cluster of Monkeypox genomes suggests an early and cryptic spread of the virus<ul><li> Further reports suggest that the early transmission in this outbreak was largely amongst gays, bisexuals and other men who have sex with men (MSM) with some exceptions.<xref rid="bib0004" ref-type="bibr"><sup>4</sup></xref><sup>,</sup><xref rid="bib0005" ref-type="bibr"><sup>5</sup></xref> However, the recently deposited genome sequences from the United States of America, Thailand and much more recently from India suggest a distinctly different phylogenetic cluster of genomes within clade IIb, classified by <b>Nextclade</b> as lineage A.2, in contrast to the cluster encompassing the majority of genomes (<italic>N</italic>&#xA0;=&#xA0;547) that are classified as lineage B.1 (<xref rid="fig0001" ref-type="fig"><u>Fig.&#xA0;1</u></xref>A).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36014093">36014093</a> <b>(2)</b> Uninvited Guest: Arrival and Dissemination of Omicron Lineage SARS-CoV-2 in St. Petersburg, Russia<ul><li> The <b>Nextclade</b> tool was used to assess the quality of assembled sequences and to assign genomes to lineages [<xref rid="B19-microorganisms-10-01676" ref-type="bibr"><u>19</u></xref>].<li> A global phylogenetic tree of SARS-CoV-2 variants was constructed using the tools implemented in <b>Nextclade</b> [<xref rid="B19-microorganisms-10-01676" ref-type="bibr"><u>19</u></xref>].<li> [19](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [19](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36032964">36032964</a> <b>(3)</b> Within-host genetic diversity of SARS-CoV-2 in the context of large-scale hospital-associated genomic surveillance<ul><li> We also limited all our analyses to samples with at least 100x coverage over at least 98% of the genome, and excluded samples where the consensus sequence was flagged as poor quality by <b>Nextclade</b><sup><xref rid="R15" ref-type="bibr"><u>15</u></xref></sup> or that did not have a lineage assigned by Pango<sup><xref rid="R16" ref-type="bibr"><u>16</u></xref></sup>, and used the earliest available sample from patients from whom multiple samples were collected.<li> We also excluded samples where the consensus sequence was flagged as poor quality under the default quality control criteria of <b>Nextclade</b><sup><xref rid="R15" ref-type="bibr"><u>15</u></xref></sup> (QC score &gt;100) or that did not have a lineage assigned by Pango<sup><xref rid="R16" ref-type="bibr"><u>16</u></xref></sup>.<li> Consensus sequence quality control was carried out in <b>Nextclade</b> 1.4.1<sup><xref rid="R15" ref-type="bibr"><u>15</u></xref></sup>.<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [16](<name><surname>O&#x2019;Toole</surname><given-names>&#xC1;.</given-names></name><etal/><article-title>Assignment of Epidemiological Lineages in an Emerging Pandemic Using the Pangolin Tool</article-title>. <source>Virus Evol</source>. <comment>veab064</comment> (<year>2021</year>) doi:<pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>.) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [16](<name><surname>O&#x2019;Toole</surname><given-names>&#xC1;.</given-names></name><etal/><article-title>Assignment of Epidemiological Lineages in an Emerging Pandemic Using the Pangolin Tool</article-title>. <source>Virus Evol</source>. <comment>veab064</comment> (<year>2021</year>) doi:<pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>.) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36090771">36090771</a> <b>(3)</b> Lineage BA.2 dominated the Omicron SARS-CoV-2 epidemic wave in the Philippines<ul><li> The quality of the compiled genomic data was evaluated by <b>Nextclade</b> CLI v1.10.3 (<xref rid="R1" ref-type="bibr">Aksamentov et&#xA0;al.<li> Sequences shorter than 27,000 nucleotides or sequences excluded by <b>Nextclade</b> due to too many ambiguous sites were also removed from the data sets.<li> A more strictly filtered BA.2 data set was prepared with sequences annotated as good quality by <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35982756">35982756</a> <b>(3)</b> Temporal lineage replacements and dominance of imported variants of concern during the COVID-19 pandemic in Kenya<ul><li> The consensus sequences were further curated using <b>Nextclade</b> Web v1.35.1<sup><xref ref-type="bibr" rid="CR29"><u>29</u></xref></sup> to screen out samples with too much missing information (Ns &gt; 30% of the genome), mixed sites and private mutations.<li> Clades were assigned to each consensus sequence using <b>Nextclade</b> Web v 1.13.1.<li>By <b>Nextclade</b> nomenclature, 13 clades were identified from the 1034 study genomes and are shown in the Supplementary Fig.&#xA0;<xref rid="MOESM1" ref-type="media"><u>2</u></xref>: 21A-Delta contributed 298 genomes (28.8%), 192 from 20I-Alpha V1 (18.6%), 188 from 21K-Omicron (18.2%), 98 from 20C (9.5%), 94 from 21J-Delta (9.1%), 60 from 20A (5.8%), 34 from 20B (3.3%), 24 from 19B (2.3%), 19 from 21D-Eta (1.8%), 13 from 20H-Beta, V2 (1.3%), 12 from 21I-Delta (1.2%) and two from 20D (0.2%).<li> [29](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36060750">36060750</a> <b>(6)</b> Rapid and sensitive amplicon-based genome sequencing of SARS-CoV-2<ul><li> The quality of the draft genome assemblies was checked and aligned using the <b>Nextclade</b> web application (<xref rid="ref9" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The viral genomic sequences including hCoV-19/Wuhan/WIV04/2019 (GISAID: EPI_ISL_402124) and hCoV-19/bat/Yunnan/RaTG13/2013 (GISAID: EPI_ISL_402131) were aligned and classified using the <b>Nextclade</b> web application (<xref rid="ref9" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> All the amino acid differences of the genomes sequenced in this study did not affect the variant classification results of the GISAID and <b>Nextclade</b>.<li> The genome sequence analysis from <b>Nextclade</b> is summarized in <xref rid="SM3" ref-type="supplementary-material"><u>Supplementary Table S10</u></xref>.<li> According to the <b>Nextclade</b> analysis, the difference nucleotides of MinION were not private mutations which meant that the nucleotide differences were also observed in other strains.<li><b>Nextclade</b> analysis showed that the nucleotide substitutions of the viral genome sequences in this study were not derived from PCR bias or sequencing error.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36030585">36030585</a> <b>(1)</b> Omicron variant (B.1.1.529) and its sublineages: What do we know so far amid the emergence of recombinant variants of SARS-CoV-2?<ul><li> To calculate the prevalence of mutations, the sequences obtained from GISAID were processed by using the <b>NextClade</b> CLI and/or an in-house Python script.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016382">36016382</a> <b>(3)</b> Development and Validation of an In-House Real-Time Reverse-Transcriptase Polymerase Chain Reaction Assay for SARS-CoV-2 Omicron Lineage Subtyping between BA.1 and BA.2<ul><li> SARS-CoV-2 genomic lineages were inferred using Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) (<uri xlink:href="https://github.com/hCoV-2019/pangolin">https://github.com/hCoV-2019/pangolin</uri>) and <b>NextClade</b> (<uri xlink:href="https://docs.nextstrain.org/projects/<b>nextclade</b>">https://docs.nextstrain.org/projects/<b>nextclade</b></uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35953484">35953484</a> <b>(1)</b> SARS-CoV-2 and other respiratory pathogens are detected in continuous air samples from congregate settings<ul><li> Lineage/clade assignments were also confirmed using <b>NextClade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>, version 1.14.0) and Pangolin COVID-19 Lineage Assigner (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>, version 3.1.20) by uploading obtained FASTA files<sup><xref ref-type="bibr" rid="CR76"><u>76</u></xref>,<xref ref-type="bibr" rid="CR77"><u>77</u></xref></sup>.<li> [76](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [77](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36033872">36033872</a> <b>(1)</b> Genomic and virologic characterization of samples from a shipboard outbreak of COVID-19 reveals distinct variants within limited temporospatial parameters<ul><li> The remaining samples were processed through <b>nextclade</b> (v10.0.10) to identify SNVs present in the samples.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35902787">35902787</a> <b>(2)</b> Initial introduction and spread of the SARS&#x2010;CoV&#x2010;2 AY.4.2.1 Delta variant in Bulgaria, a genomic insight<ul><li> The mutation pattern of the VUI was analyzed using the <b>NextClade</b> online tool.<li> The mutational pattern of the newly generated Bulgarian strains (Figure&#xA0;<xref rid="jmv28033-fig-0001" ref-type="fig"><u>1B</u></xref>) defined using <b>NextClade</b> highlighted the characterized AY.4.2 substitutions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016787">36016787</a> <b>(1)</b> Dynamicity and persistence of severe acute respiratory syndrome coronavirus-2 antibody response after double dose and the third dose with BBV-152 and AZD1222 vaccines: A prospective, longitudinal cohort study<ul><li> ARTIC field bioinformatics and <b>Nextclade</b> were the other two pipelines used for bioinformatic analysis (<xref rid="B13" ref-type="bibr"><u>Loman et al., 2020</u></xref>; <xref rid="B1" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>).<li> [Loman et al., 2020](<person-group person-group-type="author"><name><surname>Loman</surname><given-names>N.</given-names></name><name><surname>Rowe</surname><given-names>W.</given-names></name><name><surname>Rambaut</surname><given-names>A.</given-names></name></person-group> (<year>2020</year>). <source><italic>nCoV-2019 Novel Coronavirus Bioinformatics Protocol. Arctic Network.</italic></source><comment>Available online at:</comment><ext-link xlink:href="https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html" ext-link-type="uri">https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html</ext-link><comment>(accessed April 30, 2022)</comment>.) [Aksamentov et al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><name><surname>Neher</surname><given-names>R. A.</given-names></name></person-group> (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J. Open Source Softw.</italic></source><volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35991589">35991589</a> <b>(1)</b> Impact of Age and Symptom Development on SARS-CoV-2 Transmission in Households With Children&#x2014;Maryland, New York, and Utah, August 2020&#x2013;October 2021<ul><li> We performed clade assignments using <b>Nextclade</b>, version 1.13.2 [<xref rid="ofac390-B31" ref-type="bibr"><u>31</u></xref>], and assigned lineages using Pangolin, version 3.1.20 (pangoLEARN 1.2.123, Scorpio 0.3.16) [<xref rid="ofac390-B32" ref-type="bibr"><u>32</u></xref>].<li> [31](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>E</given-names></string-name>, <string-name><surname>Neher</surname><given-names>R</given-names></string-name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><year>2021</year>; <volume>6</volume>:<fpage>3773</fpage>.) [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35970304">35970304</a> <b>(2)</b> Sequencing and mutations analysis of the first recorded SARS-CoV-2 Omicron variant during the fourth wave of pandemic in Iraq<ul><li> The phylogenetic tree was built using the neighbour-joining method implemented in the Molecular Evolutionary Genetics Analysis program (MEGA 11).<xref rid="bib0015" ref-type="bibr"><sup>15</sup></xref> The variants and linage identification were made using the <b>Nextclade</b> sequence analysis and Pangolin system version (v3.1.14), respectively.<li>Two isolates of this study were found to be Omicron VOC using <b>Nextclade</b> platform,<xref rid="bib0016" ref-type="bibr"><sup>16</sup></xref> and the Pangolin tool has identified these sequences as BA.1.1 lineage of Omicron.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35981541">35981541</a> <b>(2)</b> Monitoring of the SARS-CoV-2 Omicron BA.1/BA.2 lineage transition in the Swedish population reveals increased viral RNA levels in BA.2 cases<ul><li> Analysis Software</td><td>Applied Biosystems</td><td>v1.5.2</td></tr><tr><td>fastp</td><td><ext-link ext-link-type="uri" xlink:href="http://github.com/OpenGene/fastp" id="intref0015">github.com/OpenGene/fastp</ext-link></td><td>v0.20.0</td></tr><tr><td>MEGAHIT</td><td><ext-link ext-link-type="uri" xlink:href="http://github.com/voutcn/megahit" id="intref0020">github.com/voutcn/megahit</ext-link></td><td>v1.2.9</td></tr><tr><td>RagTag</td><td><ext-link ext-link-type="uri" xlink:href="http://github.com/malonge/RagTag" id="intref0025">github.com/malonge/RagTag</ext-link></td><td>v2.1.0</td></tr><tr><td>Pangolin</td><td><ext-link ext-link-type="uri" xlink:href="http://pangolin.cog-uk.io" id="intref0030">pangolin.cog-uk.io</ext-link></td><td>v3.1.20 / v4.0.6</td></tr><tr><td>Minimap2</td><td><ext-link ext-link-type="uri" xlink:href="http://github.com/lh3/minimap2" id="intref0035">github.com/lh3/minimap2</ext-link></td><td>v2.24-r1122</td></tr><tr><td><b>Nextclade</b>.<li> Identification of SARS-CoV-2 mutations was performed using <b>Nextclade</b><xref rid="bib19" ref-type="bibr"><sup>19</sup></xref> v1.14.0.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016360">36016360</a> <b>(9)</b> SARS-CoV-2 in a Mink Farm in Italy: Case Description, Molecular and Serological Diagnosis by Comparing Different Tests<ul><li>Phylogenetic and molecular analyses were performed using <b>Nextclade</b> Web 1.5.3 (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 23 August 2021) (Ref, strain Wuhan/WH01/2019) [<xref rid="B24-viruses-14-01738" ref-type="bibr"><u>24</u></xref>].<li> <b>Nextclade</b> assigns clades by placing the sequence in a tree representing the currently circulating SARS-CoV-2.<li> The results of the <b>Nextclade</b> analysis, namely quality control evaluation, clade assignment, mutations, number of missing nucleotides, gaps, and insertions in the Italian sequences, as compared to the reference strain Wuhan 1, are shown in <xref rid="viruses-14-01738-f003" ref-type="fig"><u>Figure 3</u></xref>.<li> The SARS-CoV-2 sequences obtained from the worker and the mink sampled in August 2020 were assigned to <b>Nextclade</b> 20 B while the sequence from October 2020 was assigned to a different <b>Nextclade</b>, 20A (<xref rid="viruses-14-01738-f004" ref-type="fig"><u>Figure 4</u></xref>).<li> However, although the two August sequences belonged to the same <b>Nextclade</b>, different nucleotide and amino acid (aa) mutations were evident in the two sequences, suggesting a non-correlation between the worker and the first case detected in mink.<li> Several changes were also found that differentiated the three Italian sequences (one from the worker and two from the minks), evidenced by both their <b>Nextclade</b> assignments (20B for human and mink1 and 20A for mink2) and their varied positions in the phylogenetic tree.<li> <b>Nextclade</b> 20A emerged from 19A, dominating the European outbreak in March, and has since spread globally; additionally, <b>Nextclade</b> 20B is a large, genetically distinct 20A subclade that emerged in early 2020.<li> [24](          <article-title>Nextclade Web 1.5.3</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-08-10">(accessed on 10 August 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35906185">35906185</a> <b>(1)</b> Genomic annotation and molecular evolution of monkeypox virus outbreak in 2022<ul><li>The nucleotide mutations, including nucleotide substitutes, deletions, and insertions of each MPXV strain, were called by <b>Nextclade</b> (<ext-link xlink:href="https://master.clades.nextstrain.org" ext-link-type="uri">https://master.clades.nextstrain.org</ext-link>) on June 22, 2022.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36533145">36533145</a> <b>(3)</b> covSampler: A subsampling method with balanced genetic diversity for large-scale SARS-CoV-2 genome data sets<ul><li><b>Nextclade</b> (<xref rid="R1" ref-type="bibr">Aksamentov et&#xA0;al.<li> The reference genome is set as the default in <b>Nextclade</b> (<xref rid="R1" ref-type="bibr">Aksamentov et&#xA0;al.<li> Genomes less than 27,000 nucleotides in length or with <b>Nextclade</b>-assessed errors or warnings will be removed.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35932968">35932968</a> <b>(1)</b> Exhaled breath SARS-CoV-2 shedding patterns across variants of concern<ul><li> Complete sequences were recovered using the ARTIC analysis pipeline and were typed using Pangolin and <b>NextClade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35705189">35705189</a> <b>(1)</b> Investigation of COVID-19 Outbreak among Wildland Firefighters during Wildfire Response, Colorado, USA, 2020<ul><li> Phylogenetic inference of the consensus sequences was performed using IQTree version 2.0.3 (<ext-link xlink:href="http://www.iqtree.org" ext-link-type="uri">http://www.iqtree.org</ext-link>) with 1,000 ultrafast bootstrap replicates and phylogenetic tree visualization was performed using the python module ete3 version 3.1.2 (<ext-link xlink:href="https://pypi.org/project/ete3" ext-link-type="uri">https://pypi.org/project/ete3</ext-link>).&#xA0;Pangolin v.2.4.2<sup>5</sup> (<xref rid="R9" ref-type="bibr"><italic>9</italic></xref>) and Nextstrain&#x2019;s <b>Nextclade</b> tools (<xref rid="R10" ref-type="bibr"><italic>10</italic></xref>) were used to assign lineage and clade designations to each assembled genome.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35914602">35914602</a> <b>(2)</b> Temporal dynamics of SARS-CoV-2 genome and detection of variants of concern in wastewater influent from two metropolitan areas in Arkansas<ul><li> Lineage/clade analysis of SARS-CoV-2 wastewater samples with &gt;80 % genome coverage was performed using Pangolin and <b>NextClade</b> tools, v1.13.1, <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir0030">https://clades.nextstrain.org/</ext-link> (<xref rid="bb0030" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>).<li> Clades were assigned by using the <b>NextClade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="ir0005">https://clades.nextstrain.org</ext-link>).<li> [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>JOSS</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36016297">36016297</a> <b>(5)</b> Evaluation of EPISEQ SARS-CoV-2 and a Fully Integrated Application to Identify SARS-CoV-2 Variants from Several Next-Generation Sequencing Approaches<ul><li> The percentage of coverage of the consensus sequence to the reference genome was calculated and SARS-CoV-2 variant clade and lineage were identified according to the Nextstrain clade and Pango lineage nomenclatures [<xref rid="B31-viruses-14-01674" ref-type="bibr"><u>31</u></xref>,<xref rid="B32-viruses-14-01674" ref-type="bibr"><u>32</u></xref>,<xref rid="B33-viruses-14-01674" ref-type="bibr"><u>33</u></xref>] using <b>Nextclade</b> v1.11.0 and Pangolin v3.1.20, respectively.<li> SARS-CoV-2 variant clade and lineage were identified in EPISEQ SARS-CoV-2, according to the Nextstrain clade and Pango lineage nomenclatures [<xref rid="B31-viruses-14-01674" ref-type="bibr"><u>31</u></xref>,<xref rid="B32-viruses-14-01674" ref-type="bibr"><u>32</u></xref>,<xref rid="B33-viruses-14-01674" ref-type="bibr"><u>33</u></xref>] using the same version of <b>Nextclade</b> (v1.11.0) and Pangolin (v3.1.20), respectively, as that used for the reference pipeline.<li> Variants are identified according to the up-to-date Nextstrain clade and Pango lineage nomenclatures using the <b>Nextclade</b> and Pangolin public tools, respectively [<xref rid="B31-viruses-14-01674" ref-type="bibr"><u>31</u></xref>,<xref rid="B32-viruses-14-01674" ref-type="bibr"><u>32</u></xref>,<xref rid="B33-viruses-14-01674" ref-type="bibr"><u>33</u></xref>].<li> Mutations are screened in all SARS-CoV-2 genes, including the S gene, using <b>Nextclade</b> [<xref rid="B31-viruses-14-01674" ref-type="bibr"><u>31</u></xref>,<xref rid="B33-viruses-14-01674" ref-type="bibr"><u>33</u></xref>].<li> <b>NextClade</b>, ONT), albeit reliable, are more complex to use and their results are more difficult to locate, extract and interpret for a non-specialist in bioinformatics.<li> [31](          <article-title>Nextstrain&#x2014;Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [32](          <article-title>Centre for Genomic Pathogen Surveillance&#x2014;Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin)</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [33](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [31](          <article-title>Nextstrain&#x2014;Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [32](          <article-title>Centre for Genomic Pathogen Surveillance&#x2014;Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin)</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [33](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [31](          <article-title>Nextstrain&#x2014;Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [32](          <article-title>Centre for Genomic Pathogen Surveillance&#x2014;Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin)</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [33](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [31](          <article-title>Nextstrain&#x2014;Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-30">(accessed on 30 March 2022)</date-in-citation>        ) [33](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35966714">35966714</a> <b>(1)</b> A SARS-CoV-2 Negative Antigen Rapid Diagnostic in RT-qPCR Positive Samples Correlates With a Low Likelihood of Infectious Viruses in the Nasopharynx<ul><li> Whole-genome sequences were then classified using both the pangolin tool version 3.1.14 (Pango version 1.2.81) (O'Toole et al., <xref rid="B37" ref-type="bibr"><u>2021</u></xref>) and <b>NextClade</b> version 1.7.3 (Hadfield et al., <xref rid="B21" ref-type="bibr"><u>2018</u></xref>).<li> [2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35996592">35996592</a> <b>(3)</b> Tracking the turnover of SARS-CoV-2 VOCs Gamma to Delta in a Brazilian state (Minas Gerais) with a high-vaccination status<ul><li> 2021</xref>) or the <b>NextClade</b> web application v.1.7.0 (<xref rid="R1" ref-type="bibr">Aksamentov et&#xA0;al.<li> The <b>Nextclade</b> tool identified all the novel genome sequences as Clade 21A (VOC Delta), in perfect agreement with the genotyping RT-qPCR results.<li> The pangolin lineage report and the complete <b>Nextclade</b> analysis outputs are available on our GitHub page (<ext-link xlink:href="https://github.com/LBI-lab/Establishment-VOC-Delta-in-Minas-Gerais-Brazil" ext-link-type="uri">https://github.com/LBI-lab/Establishment-VOC-Delta-in-Minas-Gerais-Brazil</ext-link>&#x2014;<xref rid="s6" ref-type="sec"><u>Supplementary Material 2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35881313">35881313</a> <b>(1)</b> Repurposing Positive SARS-CoV-2 Antigen Test Devices for Variant Tracking<ul><li> <b>Nextclade</b> [<xref ref-type="bibr" rid="CR15"><u>15</u></xref>] and Pangolin [<xref ref-type="bibr" rid="CR16"><u>16</u></xref>] webApp were used to assign the corresponding clades or lineages.<li> [15](          <person-group person-group-type="author">            <name>              <surname>Hadfield</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Megill</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Bell</surname>              <given-names>SM</given-names>            </name>            <name>              <surname>Huddleston</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Potter</surname>              <given-names>B</given-names>            </name>            <name>              <surname>Callender</surname>              <given-names>C</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>          <source>Bioinforma Oxf Engl</source>          <year>2018</year>          <volume>1</volume>          <issue>34</issue>          <fpage>4121</fpage>          <lpage>4123</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/bty407</pub-id>        ) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35905853">35905853</a> <b>(2)</b> Analysis of the mutation dynamics of SARS-CoV-2 genome in the samples from Georgia State of the United States<ul><li>The SARS-CoV-2 Genome Sequences were downloaded in the FASTA format and online tools <b>NextClade</b> of NextStrain (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir035">https://clades.nextstrain.org/</ext-link>) (<xref rid="b0115" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>, <xref rid="b0010" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) were used for the analysis of the genome sequences.<li> <b>NextClade</b> uses SARS-CoV-2 isolate Wuhan-Hu-1/2019 (Accession # MN908947) as a reference genome.<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Aksamentov et al., 2021](Aksamentov, I., Roemer, C., Hodcroft, E.B. and Neher, R.A., 2021. Nextclade: clade assignment, mutation calling and quality control for viral genomes.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35894057">35894057</a> <b>(2)</b> Insight into Genetic Characteristics of Identified SARS-CoV-2 Variants in Egypt from March 2020 to May 2021<ul><li> In the same context, several online classification and analysis tools have been developed such as GISAID, Nextstrain SARS-CoV-2, Pangolin (<uri xlink:href="https://cov-lineages.org/resources/pangolin.html">https://cov-lineages.org/resources/pangolin.html</uri> accessed on 3 January 2022), <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> accessed on 3 January 2022), and others.<li>According to <b>Nextclade</b> analysis, 529 of the identified genomes belonged to Nextstrain clade 20D, 484 belonged to clade 20A, 94 belonged to clade 20B, 59 belonged to clade 19A, 28 belonged to clade 19B, and 3 belonged to clade 20C.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35877668">35877668</a> <b>(2)</b> Post-vaccination antibody evaluation for nosocomial SARS-CoV-2 delta variant breakthrough infection<ul><li> The variant assignment was performed on Pangolin, UCSC Genome Browser (<ext-link xlink:href="https://genome.ucsc.edu/cgi-bin/hgPhyloPlace" ext-link-type="uri">https://genome.ucsc.edu/cgi-bin/hgPhyloPlace</ext-link>) and <b>Nextclade</b> (<ext-link xlink:href="http://clades.nextstrain.org/" ext-link-type="uri">http://clades.nextstrain.org/</ext-link>).<li> The phylogenetic tree was obtained from worldwide data on <b>Nextclade</b> in September of 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35893668">35893668</a> <b>(1)</b> Immune Escape Associated with RBD Omicron Mutations and SARS-CoV-2 Evolution Dynamics<ul><li>According to the <b>NextClade</b> classification [<xref rid="B8-viruses-14-01603" ref-type="bibr"><u>8</u></xref>], the following SARS-CoV-2 clades can be distinguished (<xref rid="viruses-14-01603-f001" ref-type="fig"><u>Figure 1</u></xref>).<li> [8](          <article-title>Clade Naming &amp; Definitions-SARS-CoV-2 Workflow Documentation</article-title>          <comment>Available online: <ext-link xlink:href="https://docs.nextstrain.org/projects/ncov/en/latest/reference/naming_clades.html" ext-link-type="uri">https://docs.nextstrain.org/projects/ncov/en/latest/reference/naming_clades.html</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-05-13">(accessed on 13 May 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35862508">35862508</a> <b>(1)</b> Elevated risk of infection with SARS-CoV-2 Beta, Gamma, and Delta variant compared to Alpha variant in vaccinated individuals<ul><li> These genomes were analyzed with Pangolin (version 3.1.11) and <b>NextClade</b> (version 1.3.0) to get a final variant call (<xref rid="R40" ref-type="bibr"><italic toggle="yes">40</italic></xref>, <xref rid="R41" ref-type="bibr"><italic toggle="yes">41</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35886640">35886640</a> <b>(4)</b> Multifaceted Assessment of Wastewater-Based Epidemiology for SARS-CoV-2 in Selected Urban Communities in Davao City, Philippines: A Pilot Study<ul><li> All consensus sequences were submitted to the <b>Nextclade</b> Web v.1.12.0 (<uri xlink:href="https://clades.nextrain.org">https://clades.nextrain.org</uri> (accessed on 26 May 2022)) online server bioinformatics tool for the identification of viral mutations (single nucleotide polymorphisms or SNPs) using the SARS-CoV-2 isolate (Wuhan-Hu-1/2019) complete genome (GenBank Accession ID: MN908947) as the reference sequence [<xref rid="B28-ijerph-19-08789" ref-type="bibr"><u>28</u></xref>].<li> Clade assignment results from the <b>Nextclade</b> analysis were not considered because of low genome coverage and the high possibility of chimeric consensus genomes from mixed environmental samples.<li> The resulting FASTA file was submitted to <b>Nextclade</b> for SNP tracking.<li><b>Nextclade</b> analysis detected a total of 29 non-synonymous mutations among all assembled consensus sequences, two of which were indels (<xref rid="app1-ijerph-19-08789" ref-type="app"><u>Table S9, Supplementary Material</u></xref>).<li> [28](          <person-group person-group-type="author"><name><surname>Singer</surname><given-names>J.B.</given-names></name><name><surname>Gifford</surname><given-names>R.J.</given-names></name><name><surname>Cotten</surname><given-names>M.</given-names></name><name><surname>Robertson</surname><given-names>D.</given-names></name></person-group>          <article-title>CoV-GLUE: A Web Application for Tracking SARS-CoV-2 Genomic Variation</article-title>          <source>Preprints</source>          <year>2020</year>          <fpage>2020060225</fpage>          <pub-id pub-id-type="doi">10.20944/preprints202006.0225.v1</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35852336">35852336</a> <b>(3)</b> Contrasting Distribution of SARS-CoV-2 Lineages across Multiple Rounds of Pandemic Waves in West Bengal, the Gateway of East and North-East States of India<ul><li> All quality control (QC) passed genomes were evaluated through the command-line version of Nextstrain&#x2019;s <b>NextClade</b> (v1.2) software (<ext-link xlink:href="https://github.com/nextstrain/<b>nextclade</b>" ext-link-type="uri">https://github.com/nextstrain/<b>nextclade</b></ext-link>) to obtain a list of all nucleotide and amino-acid substitutions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35841447">35841447</a> <b>(1)</b> SARS-CoV-2 BA.1 and BA.2 coinfection detected by genomic surveillance in Brazil, January 2022<ul><li> However, <b>NextClade</b> pointed to several individual mutations that differed between the query sequence and the nearest neighbor sequence that are typically associated with the BA.2 clade, although the PANGO designation assigned it as lineage BA.1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35836127">35836127</a> <b>(2)</b> Phylogenetic analysis and comparative genomics of SARS-CoV-2 from survivor and non-survivor COVID-19 patients in Cordoba, Argentina<ul><li>Phylogenetic analyses were performed against a background of 1129 SARS-CoV-2 sequences from Argentina in January&#x2013;December 2020 (GISAID EpiCoV database, [<xref ref-type="bibr" rid="CR13"><u>13</u></xref>], <ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org">https://www.gisaid.org</ext-link>) and analyzed with <b>NextClade</b> V1.6.0 [<xref ref-type="bibr" rid="CR14"><u>14</u></xref>].<li>The sequences were analyzed using NextStrain tools (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org">https://nextstrain.org</ext-link>), such as <b>NextClade</b> V1.6.0 [<xref ref-type="bibr" rid="CR14"><u>14</u></xref>], and classified by Pangolin lineages.<li> [13](          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID's innovative contribution to global health</article-title>          <source>Global Chall</source>          <year>2017</year>          <volume>1</volume>          <issue>1</issue>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35840078">35840078</a> <b>(2)</b> Phylogenomic analysis of the monkeypox virus (MPXV) 2022 outbreak: Emergence of a novel viral lineage?<ul><li> The analyses of these genomes were carried out according to the following scheme: An alignment using <b>NextClade</b> v2.1.0 tool [<xref rid="bib25" ref-type="bibr"><u>25</u></xref>], a descriptive analysis of the genomes studied and the number of confirmed MPXV cases [<xref rid="bib26" ref-type="bibr"><u>26</u></xref>] and an analysis of the phylogenomic relationships of genome sequences publicly available.<li> For phylogenomic analysis, the aligned dataset obtained from <b>NextClade</b> was used to build a maximum likelihood (ML) in IQ-TREE multicore v1.6.12 [<xref rid="bib27" ref-type="bibr"><u>27</u></xref>], using the best substitution model and other parameters by default.<li> [25](          <person-group person-group-type="author">            <name>              <surname>Zumla</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Valdoleiros</surname>              <given-names>S.R.</given-names>            </name>            <name>              <surname>Haider</surname>              <given-names>N.</given-names>            </name>            <name>              <surname>Asogun</surname>              <given-names>D.</given-names>            </name>            <name>              <surname>Ntoumi</surname>              <given-names>F.</given-names>            </name>            <name>              <surname>Petersen</surname>              <given-names>E.</given-names>            </name>            <etal/>          </person-group>          <article-title>Monkeypox outbreaks outside endemic regions: scientific and social priorities</article-title>          <source>Lancet Infect Dis</source>          <year>2022</year>        ) [26](          <person-group person-group-type="author">            <name>              <surname>World Health Organization</surname>            </name>          </person-group>          <part-title>Disease outbreak news; multi-country monkeypox outbreak in non-endemic countries [Internet]</part-title>          <year>2022</year>          <comment>[cited 2022 Jun 11];Available from:</comment>          <ext-link ext-link-type="uri" xlink:href="https://www.who.int/emergencies/disease-outbreak-news/item/2022-DON385" id="intref0055">https://www.who.int/emergencies/disease-outbreak-news/item/2022-DON385</ext-link>        ) [27](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Source Software</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891509">35891509</a> <b>(1)</b> SARS-CoV-2 Genomic Characteristics and Clinical Impact of SARS-CoV-2 Viral Diversity in Critically Ill COVID-19 Patients: A Prospective Multicenter Cohort Study<ul><li> Lineages and clades were interpreted using Pangolin and <b>NextClade</b>, before being submitted to the GISAID database (<uri xlink:href="https://www.gisaid.org">https://www.gisaid.org</uri>; accessed on 1 June 2021; the GISAID IDs of the sequenced sampled are listed in the <xref rid="app1-viruses-14-01529" ref-type="app"><u>Supplementary Materials</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35862982">35862982</a> <b>(1)</b> Performance of 22 Rapid Lateral Flow Tests for SARS-CoV-2 Antigen Detection and Influence of &#x201C;Variants of Concern&#x201D;: Implications for Clinical Use<ul><li> Lineages and clades were interpreted using the Lineage Assigner web application (Centre for Genomic Pathogen Surveillance, Cambridge, United Kingdom) and the <b>Nextclade</b> web application (Nextstrain Project, University of Basel, Basel, Switzerland) and then submitted to the GISAID international database (<ext-link xlink:href="https://www.gisaid.org" ext-link-type="uri">https://www.gisaid.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35835159">35835159</a> <b>(1)</b> Gamma variant vertically transmitted from a mild symptomatic pregnant woman associated with fatal neonatal COVID<ul><li>Mutational profiles were investigated using the <b>Nextclade</b> tool to describe substitutions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35863047">35863047</a> <b>(2)</b> Co-circulation of Multiple SARS-CoV-2 Variants of Concern in Dhaka, Bangladesh, during the Second Wave of the COVID-19 Pandemic<ul><li> Sequences were uploaded to the <b>Nextclade</b> web server (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) for variant calling, lineage determination, and detection of nucleotide and amino acid mutations in reference to the Wuhan-Hu-1/2019 strain (GenBank accession number <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MN908947" ext-link-type="uri" assigning-authority="insdc">MN908947</ext-link>) (<xref rid="B5" ref-type="bibr"><u>5</u></xref>).<li>PANGO lineage (<b>Nextclade</b>)</th>              </tr>            </thead>            <tbody>              <tr>                <td rowspan="1" colspan="1">BSMMU_Viro_1(258).<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35816514">35816514</a> <b>(2)</b> The structural role of SARS-CoV-2 genetic background in the emergence and success of spike mutations: The case of the spike A222V mutation<ul><li> 4,993,996 SARS-CoV-2 sequences were aligned with the Nextalign package (<ext-link xlink:href="https://github.com/nextstrain/<b>nextclade</b>/tree/master/packages/nextalign_cli" ext-link-type="uri">https://github.com/nextstrain/<b>nextclade</b>/tree/master/packages/nextalign_cli</ext-link>).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891491">35891491</a> <b>(1)</b> SARS-CoV-2 Testing of Aircraft Wastewater Shows That Mandatory Tests and Vaccination Pass before Boarding Did Not Prevent Massive Importation of Omicron Variant into Europe<ul><li> Sequences from complete genomes and clade assignments were analysed using the <b>Nextclade</b> web tool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> accessed on 14 June 2022) [<xref rid="B26-viruses-14-01511" ref-type="bibr"><u>26</u></xref>].<li> [26](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35890025">35890025</a> <b>(3)</b> From Alpha to Delta&#x2014;Genetic Epidemiology of SARS-CoV-2 (hCoV-19) in Southern Poland<ul><li> An illustration of phylogenetic relationships of SARS-CoV-2 clades from Southern Poland is presented in <xref rid="pathogens-11-00780-f007" ref-type="fig"><u>Figure 7</u></xref>, but due to the low transparency of the phylogeny generated in <b>Nextclade</b>, a phylogenetic tree construction in R v.<li> Phylogenetic trees were also generated using the <b>Nextclade</b> tools.<li> The maximum likelihood phylogeny of Southern Poland whole-genome sequences generated in <b>Nextclade</b> was performed.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891473">35891473</a> <b>(1)</b> Comparative Dynamics of Delta and Omicron SARS-CoV-2 Variants across and between California and Mexico<ul><li> The remaining sequences were curated and aligned using Nextalign implemented in <b>Nextclade</b> [<xref rid="B14-viruses-14-01494" ref-type="bibr"><u>14</u></xref>].<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35875574">35875574</a> <b>(3)</b> The Mutational Landscape of SARS-CoV-2 Variants of Concern Recovered From Egyptian Patients in 2021<ul><li>For viral genome alignment, quality inspection, clade determination, and phylogenetic studies, the online tool &#x201C;<b>Nextclade</b>&#x201D; was utilized.<li>The <b>Nextclade</b> Tool was used for the detection of genetic mutations in the SARS-CoV-2 genome, including S, E, M, N, ORF1a, ORF1b, ORF3a, ORF6, ORF7a, ORF7b, ORF8, and ORF9b (see <xref rid="SM1" ref-type="supplementary-material"><u>Supplementary Table S1</u></xref>).<li>A rectangular phylogenetic tree constructed by the online <b>Nextclade</b> Tool revealed the most common clades were 20D (34.4%) and 21&#x2009;J (32.8%), followed by 20I (12.5%), 20A (9.4%), and 19B (4.7%).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35818397">35818397</a> <b>(1)</b> First case of within-host co-infection of different SARS-CoV-2 variants in Ecuador<ul><li> For the three different consensus sequences obtained, the online tool <b>NextClade</b> (v.1.6.0) [<xref rid="bib5" ref-type="bibr"><u>5</u></xref>] was used to assign the sequences to clades, identify mutations, and to determine sequence quality control.<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35867411">35867411</a> <b>(2)</b> Decreased Efficiency of Neutralizing Antibodies from Previously Infected or Vaccinated Individuals against the B.1.617.2 (Delta) SARS-CoV-2 Variant<ul><li>The strains used were sequenced and the data deposited in GISAID under accession numbers EPI_ISL_804378 (GISAID clade: GH, Pango lineage: B.1.160, <b>Nextclade</b>: 20A.EU2) and EPI_ISL_4276187 (GISAID clade: GK, Pango lineage: B.1.617.2, <b>Nextclade</b>: 21A).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35889075">35889075</a> <b>(6)</b> Determining the International Spread of B.1.1.523 SARS-CoV-2 Lineage with a Set of Mutations Highly Associated with Reduced Immune Neutralization<ul><li> The general sequence quality evaluation, extraction and alignment of S protein sequences, variant calling was performed using <b>Nextclade</b> 1.3.020 [<uri xlink:href="https://github.com/nextstrain/<b>nextclade</b>">https://github.com/nextstrain/<b>nextclade</b></uri> (accessed on 5 May 2021)] [<xref rid="B8-microorganisms-10-01356" ref-type="bibr"><u>8</u></xref>].<li> The sequences with poor overall quality control status or with more than 1000 bps missing (as indicated by <b>Nextclade</b> analysis) were discarded.<li>The S protein-based phylogeny was based on the S protein sequences extracted from GISAID by <b>Nextclade</b> and aligned to the reference COVID-19 sequence.<li> The sequence alignment that was used to construct the tree was composed of the alignment produced by <b>Nextclade</b>, leaving only the representative sequences of the clusters and CAT approximation with 20 rate categories: lg -gamma.<li> [8](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35792280">35792280</a> <b>(1)</b> Evidence of co-infections during Delta and Omicron SARS-CoV-2 variants co-circulation through prospective screening and sequencing<ul><li> In our study, <b>Nextclade</b> assigned a clade from consensus sequences for all sequenced D1 samples but reported the presence of additional mutations as shown by high private mutation scores.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35779200">35779200</a> <b>(1)</b> Long-term SARS-CoV-2 Asymptomatic Carriage in an Immunocompromised Host: Clinical, Immunological, and Virological Implications<ul><li>Following the genetic investigation performed on nasopharyngeal samples collected on days 35, 143, and 207, Pangolin v3.1.16 and <b>Nextclade</b> v.1.7.2 assigned all the sequenced samples in the B.1 lineage and 20A clade, respectively, the origin of which is considered linked with the Northern Italian outbreak in February 2020.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35551714">35551714</a> <b>(3)</b> SARS-CoV-2 Delta&#x2013;Omicron Recombinant Viruses, United States<ul><li><b>Nextclade</b> (<xref rid="R11" ref-type="bibr"><italic>11</italic></xref>) classified the 9 whole genomes as 21K (Omicron/BA.1).<li> <b>Nextclade</b> classified the first 22150 base fragment as clade 21J (Delta) and the remainder as clade 21K (Omicron/BA.1).<li>To visualize the parents of the recombinant genomes, we split all candidate recombinant genomes at position 22150, within the predicted breakpoint, and used <b>Nextclade</b> (<xref rid="R11" ref-type="bibr"><italic>11</italic></xref>) to place each genome fragment (1&#x2013;22150 and 22151 through the 3&#x2032; end) onto a reference tree.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35640400">35640400</a> <b>(1)</b> A qualitative RT-PCR assay for the specific identification of the SARS-CoV-2 B.1.1.529 (Omicron) Variant of Concern<ul><li> and with less than 5% undetermined bases have been downloaded and analysed with <b>Nextclade</b> <xref rid="bib0016" ref-type="bibr"><u>[16]</u></xref> (Supplementary Material S1).<li> [[16]](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <issue>67</issue>          <year>2021</year>          <fpage>3773</fpage>          <comment>https://doiorg/1021105/joss03773</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35794888">35794888</a> <b>(1)</b> Remdesivir resistance in transplant recipients with persistent COVID-19<ul><li> <b>Nextclade</b> v2.0.0 and Auspice v2.37.3 phylogenomic visualization Nextstrain project) were used to examine viral genome clade assignment and mutation calling.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891388">35891388</a> <b>(5)</b> Sequencing during Times of Change: Evaluating SARS-CoV-2 Clinical Samples during the Transition from the Delta to Omicron Wave<ul><li> and analyzed through <b>Nextclade</b> for private mutation information; all private mutations identified from consensus sequences of each sample were also compared to mutation calling results and/or the corresponding BAM file to ensure the accuracy of consensus calling.<li>To identify the SARS-CoV-2 lineages in the 37 RNA samples, we generated consensus sequences from each sample and classified lineages based on results from Pangolin (v3.1.2) (<uri xlink:href="https://pangolin.cog-uk.io">https://pangolin.cog-uk.io</uri>, accessed on 22 March 2022) and <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>, accessed on 6 April 2022) (<xref rid="app1-viruses-14-01408" ref-type="app"><u>Supplemental Table S1</u></xref>).<li> We first analyzed the consensus sequences through <b>Nextclade</b> to identify the private mutations for each patient (<xref rid="viruses-14-01408-f005" ref-type="fig"><u>Figure 5</u></xref>, <xref rid="app1-viruses-14-01408" ref-type="app"><u>Supplemental Table S2</u></xref>).<li> Private mutations are mutations that are not observed in the nearest neighbor on the <b>Nextclade</b>&#x2019;s reference tree and are therefore very likely to be specific to individual samples.<li> We found that both ArticV4.1 and VarSkip2 reached near-full-genome coverage (<xref rid="viruses-14-01408-f001" ref-type="fig"><u>Figure 1</u></xref>) and resulted in identical lineage assignments using both Pangolin and <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35761219">35761219</a> <b>(4)</b> Safety and efficacy of convalescent plasma for severe COVID-19: a randomized, single blinded, parallel, controlled clinical study<ul><li>Comparative genomics analyses were developed to evaluate the phylogenomic relationships for the 43 genomes obtained in this study in the context of 2,170 genomes from a representative selection of all SARS-CoV-2 lineages obtained from <b>NextClade</b> tool v 1.5.4 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).<li> For this, the total dataset with 2,213 genomes was aligned and used to build a maximum likelihood (ML) in <b>NextClade</b> following the pipeline of analysis previously published [<xref ref-type="bibr" rid="CR19"><u>19</u></xref>].<li>A mutational analysis was performed over the 43 SARS-CoV-2 whole genome sequences obtained in this study using the <b>NextClade</b> tool.<li> <bold>A</bold> Phylogenomic relationships using a maximum likelihood phylogeny obtained in <b>Nextclade</b>.<li> [19](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33467048">33467048</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35758686">35758686</a> <b>(1)</b> Identification of SARS-CoV-2 Variants of Concern Using Amplicon Next-Generation Sequencing<ul><li> Consensus sequences were submitted to Pangolin command-line tool (pangolin version 2.2.2 and pangoLEARN version 2021-02-12) and <b>Nextclade</b> (version 0.12.0) to determine the PANGO lineage and clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35759503">35759503</a> <b>(5)</b> Genome sequence diversity of SARS-CoV-2 obtained from clinical samples in Uzbekistan<ul><li> The <b>Nextclade</b> Web tool v.1.11.1 [<xref rid="pone.0270314.ref016" ref-type="bibr"><u>16</u></xref>] was used to compare study sequences to SARS-CoV-2 reference sequences, assign them to clades, and determine their position within the SARS-CoV-2 phylogenetic tree.<li>The <b>Nextclade</b>-based phylogenetic analysis of the SARS-CoV-2 genome samples sequenced in our study has revealed that the most currently distributed SARS-CoV-2 variants in our dataset belong to the tree clades: 20A (Alpha, V1), 20I (Alpha, V1), and 21A (Delta) in consonance with &#x201C;Nextstrain&#x201D;&#x2018;s nomenclature system [<xref rid="pone.0270314.ref016" ref-type="bibr"><u>16</u></xref>].<li>The <b>Nextclade</b> based phylogenetic tree of the SARS-CoV-2 variants isolated in Uzbekistan.<li> Clades are assigned according to <b>Nextclade</b> nomenclature.<li> The <b>Nextclade</b> based analysis of geographical distribution indicated that all samples in the subclade 21A were mostly clustered close to the SARS-CoV-2 genotypes from Bangladesh, whereas all samples in the subclade 21J were mostly clustered with the genome sequence samples of coronavirus from Egypt and Russia.<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35760381">35760381</a> <b>(1)</b> Rapid and qualitative identification of SARS-CoV-2 mutations associated with variants of concern using a multiplex RT-PCR assay coupled with melting analysis<ul><li> Lineage and clade data, quality controls, mutations, missing nucleotides, and gaps were analyzed and confirmed with the support of Pangolin (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="interref0006">https://pangolin.cog-uk.io/</ext-link> accessed on March 6, 2022) and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0007">https://clades.nextstrain.org/</ext-link> accessed on March 6, 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35751044">35751044</a> <b>(2)</b> Longitudinal alterations of the gut mycobiota and microbiota on COVID-19 severity<ul><li> The obtained genome sequences illustrated that all SARS-CoV-2 variants were assigned to clade 20B on the <b>Nextclade</b> lineage (Table <xref rid="Tab2" ref-type="table"><u>2</u></xref>).<li><b>NextClade</b> lineage</th><th align="left">Spike amino acid changes</th></tr></thead><tbody><tr><td align="left">Cov1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35750157">35750157</a> <b>(1)</b> Phylogenomic characterization and signs of microevolution in the 2022 multi-country outbreak of monkeypox virus<ul><li>Rapid extraction and/or visualization of variant sites from sequence alignments was performed using ReporTree (<ext-link ext-link-type="uri" xlink:href="https://github.com/insapathogenomics/ReporTree">https://github.com/insapathogenomics/ReporTree</ext-link>), snipit (<ext-link ext-link-type="uri" xlink:href="https://github.com/aineniamh/snipit">https://github.com/aineniamh/snipit</ext-link>) and <b>NextClade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35891359">35891359</a> <b>(1)</b> Description of a One-Year Succession of Variants of Interest and Concern of SARS-CoV-2 in Venezuela<ul><li> The variant assignment was performed with the Dragen COVID-19 program, <b>Nextclade</b> Web 1.14.1 (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 30 April 2022), or Pangolin COVID-19 Lineage Assigner (<uri xlink:href="https://cov-lineages.org/resources/pangolin.html">https://cov-lineages.org/resources/pangolin.html</uri>, accessed on 30 April 2022).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35737946">35737946</a> <b>(1)</b> Characterization of Virologic Rebound Following Nirmatrelvir-Ritonavir Treatment for Coronavirus Disease 2019 (COVID-19)<ul><li> Notable amino acid mutations and sequence quality were analyzed using <b>Nextclade</b> v1.14.1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35753602">35753602</a> <b>(1)</b> Host genetic loci <italic>LZTFL1</italic> and <italic>CCL2</italic> associated with SARS-CoV-2 infection and severity of COVID-19<ul><li> The <b>Nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="interref0002">https://clades.nextstrain.org</ext-link>) was used to identify nucleotide, and amino acid substitutions and lineages were obtained using the Pangolin online tool (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/resources/pangolin.html" id="interref0003">https://cov-lineages.org/resources/pangolin.html</ext-link>) (<xref rid="bib0034" ref-type="bibr"><u>Rambaut&#xA0;et&#xA0;al., 2020</u></xref>).<li> [Rambaut&#xA0;et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35753659">35753659</a> <b>(1)</b> Brief dispersion of a putative B.1.1.28-derived SARS-CoV-2 lineage harboring additional N234P and E471Q spike protein mutations in individuals crossing the Argentina-Brazil border<ul><li> On the left, a global SARS-CoV-2 complete genome phylogeny performed on the <b>Nextclade</b> online platform.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35843112">35843112</a> <b>(3)</b> Rapid repeat infection of SARS-CoV-2 by two highly distinct delta-lineage viruses<ul><li> The resulting consensus assemblies were then analyzed with Pangolin [<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="interref0008">https://github.com/cov-lineages/pangolin</ext-link>] and <b>NextClade</b> [<ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" id="interref0009">https://github.com/nextstrain/<b>nextclade</b></ext-link>] to perform lineage and clade designations, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35729228">35729228</a> <b>(1)</b> Transmission of B.1.617.2 Delta variant between vaccinated healthcare workers<ul><li> Following this, all sequences were screened for number of gaps and N-regions using the <b>Nextclade</b> v1.5.4 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) server.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35723286">35723286</a> <b>(18)</b> Automated Library Construction and Analysis for High-Throughput Nanopore Sequencing of SARS-CoV-2<ul><li> The endpoint of the analysis was to classify samples into clades using <b>Nextclade</b> (<xref rid="jfac054-B21" ref-type="bibr"><u>21</u></xref>).<li> Here, we used the <b>Nextclade</b> clade classification quality scores as a proxy for sequencing data quality.<li> The guppyplex and minion commands were combined with <b>Nextclade</b> assignment and quality control (<xref rid="jfac054-B21" ref-type="bibr"><u>21</u></xref>) in a nextflow 0.1.1 (<xref rid="jfac054-B22" ref-type="bibr"><u>22</u></xref>) workflow to allow easy parallelization on our network.<li><b>Nextclade</b>, part of the ARTIC analysis pipeline (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>), classifies samples into clades and assigns a quality score of Good, Mediocre, or Bad based on several factors, including the fraction of unambiguously sequence-covered genome and the number of mutations compared to currently known clades.<li> <xref rid="sup1" ref-type="supplementary-material"><u>Supplementary Table 1</u></xref> shows the change in <b>Nextclade</b>&#x2019;s quality classification for reads obtained at 5&#x2005;h (lower redundancy) and 24&#x2005;h (higher redundancy) of sequencing.<li>Classification of samples via <b>Nextclade</b> after 24&#x2005;h of sequencing, showing the 25 plates/pools in the order in which they were processed and sequenced.<li> <b>Nextclade</b>&#x2019;s good/mediocre/bad quality classification is based on genome coverage and mutational profile.<li> (B) <b>Nextclade</b> quality score changes showing the effect of post-PCR concentration enhancement for 12 high Ct samples.<li> 4</xref>, occasionally resulted in underrepresented amplicons, although the genomes could still be assigned a clade using <b>Nextclade</b>.<li>(A) Regions of missing coverage for samples classified by <b>Nextclade</b> as good.<li><xref rid="jfac054-F3" ref-type="fig"><u>Figure 3, A</u></xref> presents results from 2 representative 96-well plates that underwent mass quantification, illustrating the relationship in yield from ARTIC PCR between the sample&#x2019;s starting Ct and <b>Nextclade</b> quality classification.<li>WGS QC<sup><xref rid="jfac054-tblfn1" ref-type="table-fn"><u>a</u></xref></sup> score (<b>Nextclade</b>)</th>              </tr>            </thead>            <tbody>              <tr>                <td rowspan="1" colspan="1">Detected by WGS.<li>Concordance is 99.8% (1655/1658) for samples classified as good by <b>Nextclade</b>.<li>Another limitation in this analysis is that we used the <b>Nextclade</b> quality score as a metric of sequencing success, because we determined that it best reflects what is of immediate value to the clinical laboratory.<li> <b>Nextclade</b> does reflect typical genome quality metrics in that in that the breadth of unambiguous genome coverage is a key factor in the quality score.<li> However, to assign clades, <b>Nextclade</b> also requires that the number of mutations does not exceed a certain threshold and deducts from the quality score if the mutations count is higher.<li> We also note that other tools for phylogenetic classification may be readily employed, but <b>Nextclade</b> was sufficient for the purposes of this comparison of WGS data to a SNP PCR assay.<li> Samples with a small number of regions with reduced sequence coverage are still classifiable by <b>Nextclade</b>, but in cases where there is particular interest in specific regions, extra amplicons or repositioned amplicons may be required.<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28398311">28398311</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35718768">35718768</a> <b>(3)</b> Investigation of target sequencing of SARS-CoV-2 and immunogenic GWAS profiling in host cells of COVID-19 in Vietnam<ul><li> <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://docs.nextstrain.org">https://docs.nextstrain.org</ext-link>) is a tool that helps to identifies the differences between target sequences and a reference sequence by Nextstrain to assign clades to these sequences [<xref ref-type="bibr" rid="CR28"><u>28</u></xref>].<li> <b>Nextclade</b> was used for alignment to reference SARS-CoV-2 Wuhan-1 (MN908947.3), and for detecting variants and protein mutations on these datasets.<li> These samples were also analyzed by <b>Nextclade</b> to find out which strains and mutations prevail among others.<li> [28](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling andquality control for viral genomes</article-title>          <source>J Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <issue>67</issue>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35779343">35779343</a> <b>(1)</b> Rapid and accurate identification of SARS-CoV-2 Omicron variants using droplet digital PCR (RT-ddPCR)<ul><li> Phylogenetic lineage was assigned using the PANGOLIN (Phylogenetic Assignment of Named Global Outbreak LINeages, <ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="interref0002">https://pangolin.cog-uk.io/</ext-link>) and <b>NextClade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0003">https://clades.nextstrain.org/</ext-link>) tools.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35744748">35744748</a> <b>(1)</b> SARS-CoV-2: Two Years in the Pandemic: What Have We Observed from Genome Sequencing Results in Lithuania?<ul><li> General sequence quality evaluation, extraction, and alignment of S protein sequences and variation calling was carried out using <b>Nextclade</b> 1.3.0 [<xref rid="B5-microorganisms-10-01229" ref-type="bibr"><u>5</u></xref>].<li> [5](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35710544">35710544</a> <b>(1)</b> Tracking the molecular evolution and transmission patterns of SARS-CoV-2 lineage B.1.466.2 in Indonesia based on genomic surveillance data<ul><li> Taking Wuhan-Hu-1 (NC_045512.2) as the reference genome, single nucleotide variants (SNVs) were called and annotated using the <b>Nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) [<xref ref-type="bibr" rid="CR31"><u>31</u></xref>].<li> [31](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746789">35746789</a> <b>(1)</b> Genomic Epidemiology of SARS-CoV-2 in Seychelles, 2020&#x2013;2021<ul><li>Additional quality control, clade assignment and mutation profiles were obtained using the <b>NextClade</b> tool v1.13.2 [<xref rid="B8-viruses-14-01318" ref-type="bibr"><u>8</u></xref>] using a SARS-CoV-2 reference genome (accession NC_045512).<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35699426">35699426</a> <b>(1)</b> Transmission networks of SARS-CoV-2 in Coastal Kenya during the first two waves: A retrospective genomic study<ul><li> Amino acid sequence changes in the Coastal Kenya genomes were investigated using the <b>Nextclade</b> tool v0.14.2 (<xref rid="bib12" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>): <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link> (last accessed August 3, 2021).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746765">35746765</a> <b>(1)</b> Nationwide Harmonization Effort for Semi-Quantitative Reporting of SARS-CoV-2 PCR Test Results in Belgium<ul><li> WGS classified the strain to the type 20A according to <b>Nextclade</b> [<xref rid="B34-viruses-14-01294" ref-type="bibr"><u>34</u></xref>], and to the Pangolin [<xref rid="B35-viruses-14-01294" ref-type="bibr"><u>35</u></xref>] lineage B.1.160.<li> [34](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35695095">35695095</a> <b>(1)</b> From the Wuhan-Hu-1 strain to the XD and XE variants: is targeting the SARS-CoV-2 spike protein still a pharmaceutically relevant option against COVID-19?<ul><li> In the case of newly discovered variants XD, XE, and XF, the sequence was chosen according to the one reported by the <b>Nextclade</b> project<xref rid="CIT0048" ref-type="bibr"><sup>48</sup></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35698014">35698014</a> <b>(4)</b> Molecular and computational analysis of spike protein of newly emerged omicron variant in comparison to the delta variant of SARS-CoV-2 in Iraq<ul><li> <b>Nextclade</b> was used to perform phylogenetic analysis based on the GISAID clades [<xref ref-type="bibr" rid="CR12"><u>12</u></xref>].<li> The phylogenetic tree was constricted based on sequence date in GISAID data using the Nextstrain server (<b>Nextclade</b> tool).<li> The first one belongs to <b>nextclade</b> 21J (delta), GISAID clade O, VOC delta (B.1.617.2.).<li> The second virus belongs to <b>nextclade</b> 21K (Omega), GISAID clade GRA, VOC omicron (B.1.1.529+BA.1) (Fig.<li> [12](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <etal/>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Sour Softw</source>          <year>2021</year>          <volume>6</volume>          <issue>67</issue>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35698660">35698660</a> <b>(1)</b> The phylodynamics of SARS-CoV-2 during 2020 in Finland<ul><li> Clade and lineage assignment was done using <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR20"><u>20</u></xref></sup> and pangolin.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35644902">35644902</a> <b>(1)</b> Genomic surveillance of the Lambda SARS&#x2010;CoV&#x2010;2 variant in a global phylogenetic context<ul><li>The quality of reads was assessed and removal of reads contamination was performed using FastQC v0.11.9 (<ext-link xlink:href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" ext-link-type="uri">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</ext-link>) and Kraken2 v2.0.8, respectively.<xref rid="jmv27889-bib-0007" ref-type="bibr"><sup>7</sup></xref> The filter reads were mapped to the NCBI reference sequence (NC_045512) isolated from Wuhan using the software package BWA v.0.7.17.<xref rid="jmv27889-bib-0008" ref-type="bibr"><sup>8</sup></xref> The consensus sequence was obtained using Samtools v.1.9 and IVAR v.1.<xref rid="jmv27889-bib-0009" ref-type="bibr"><sup>9</sup></xref>, <xref rid="jmv27889-bib-0010" ref-type="bibr"><sup>10</sup></xref> Finally, the annotation of consensus sequences was performed using <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>), and lineage designation was defined using the software Pangolin v.3.1.2 (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746737">35746737</a> <b>(1)</b> Sequential Appearance and Isolation of a SARS-CoV-2 Recombinant between Two Major SARS-CoV-2 Variants in a Chronically Infected Immunocompromised Patient<ul><li> The clade was designated at the consensus level with the <b>Nextclade</b> online tool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>; accessed on 18 May 2022, 2020 Nextstrain developers, Swiss Institute of Bioinformatics) [<xref rid="B32-viruses-14-01266" ref-type="bibr"><u>32</u></xref>,<xref rid="B47-viruses-14-01266" ref-type="bibr"><u>47</u></xref>] and an in-house Python script allowed for the detection of variants and the hybrids of variants.<li> [32](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [47](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35676259">35676259</a> <b>(2)</b> Rapid, adaptable and sensitive Cas13-based COVID-19 diagnostics using ADESSO<ul><li> Frequency analysis of viral genetic variations was performed using <b>Nextclade</b> tool with standard parameters (<b>Nextclade</b> CLI).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35742840">35742840</a> <b>(1)</b> Evolution of SARS-CoV-2 in Spain during the First Two Years of the Pandemic: Circulating Variants, Amino Acid Conservation, and Genetic Variability in Structural, Non-Structural, and Accessory Proteins<ul><li>For the correct assignment of the SARS-CoV-2 variants, the quality of all the downloaded sequences was checked using <b>Nextclade</b> v.1.11.0 (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 10 May 2022)), and the sequences classified as &#x201C;bad&#x201D; quality were removed from the analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35693461">35693461</a> <b>(1)</b> From Delta to Omicron SARS-CoV-2 variant: Switch to saliva sampling for higher detection rate<ul><li> Clade calling on the consensus FASTA was done by both Pangolin lineage assignment (v2.3.2) and <b>Nextclade</b> (v0.14.1) webtools.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35703299">35703299</a> <b>(2)</b> Household Secondary Transmission of the Severe Acute Respiratory Syndrome Coronavirus 2 Alpha Variant From a Community Cluster in a Nursery in Japan<ul><li> A phylogenetic tree was constructed based on the whole-genome sequence (29,809 nucleotides) of virus strains obtained in Niigata Prefecture between February 1, 2021, and February 28, 2021, using <b>Nextclade</b> version 1.13.2 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<li> The phylogenetic tree was constructed with <b>Nextclade</b> version 1.13.2 and was based on the whole-genome sequence (29,809 nucleotides) of virus strains obtained in Niigata Prefecture between February 1, 2021, and February 28, 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746705">35746705</a> <b>(5)</b> Genomic Diversity of SARS-CoV-2 Omicron Variant in South American Countries<ul><li> These schemes consisted of SARS-CoV-2 lineage typing using the PANGOLIN-v1.9 tool (<uri xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</uri> (accessed on 10 May 2022)), and alignment and phylogenetic analysis using the <b>NextClade</b> tool v1.5.4 default command line (<uri xlink:href="https://docs.nextstrain.org/projects/<b>nextclade</b>/en/stable/user/<b>nextclade</b>-cli.html">https://docs.nextstrain.org/projects/<b>nextclade</b>/en/stable/user/<b>nextclade</b>-cli.html</uri> (accessed on 10 May 2022)), which performs sequence alignment, variant calling, clade assignment and maximum-likelihood (ML) tree placement.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35438176">35438176</a> <b>(1)</b> Molecular signature of postmortem lung tissue from COVID-19 patients suggests distinct trajectories driving mortality<ul><li> The SARS-CoV-2 genomes were classified using GISAID (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>), PANGO database (<uri xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</uri>) and <b>nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>) and were placed in a phylogenetic tree created using nextstrain (<uri xlink:href="https://nextstrain.org/">https://nextstrain.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35733956">35733956</a> <b>(3)</b> A Rapid, Whole Genome Sequencing Assay for Detection and Characterization of Novel Coronavirus (SARS-CoV-2) Clinical Specimens Using Nanopore Sequencing<ul><li> <b>Nextclade</b><sup><xref rid="footnote6" ref-type="fn"><u>6</u></xref></sup> was used to generate CoV-2 clade assignments.<li> After P12 and P15 sequences were corrected, the analyses were performed again on all the sequences using the most current versions of each of the apps: GISAID CoVsurver, <b>Nextclade</b> v.14.1, and Pangolin v4.0.6.<li>Mutation analysis of assembled sequences using CoVsurver and comparison to Pangolin and <b>Nextclade</b> classifications.<li> [6](<a href="clades.nextstrain.org">clades.nextstrain.org</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35691101">35691101</a> <b>(2)</b> The role of airborne transmission in a large single source outbreak of SARS-CoV-2 in a Belgian nursing home in 2020<ul><li> SARS-CoV-2 lineage assignments were derived using the Pangolin tool as well as classification by <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="ir0005">https://clades.nextstrain.org</ext-link>) (<xref rid="bib13" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>, <xref rid="bib12" ref-type="bibr"><u>O&#x2019;Toole et al., 2021</u></xref>).<li> All 115 SARS-CoV-2 sequences could be assigned to clade 20B according to <b>Nextclade</b>, while phylogenetically the lineage B.1.1 was derived.<li> [Rambaut et al., 2020](Rambaut, A., Holmes, E., O&#x2019;Toole, &#xC1;, Hill, V., McCrone, J., Ruis, C., et al., 2020. A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology. Nat. Microbiol. [Internet], vol. 5(issue 11), pp. 1403&#x2013;7. [cited 2021 Oct 27]. Available from: &#x3008;<ext-link ext-link-type="uri" xlink:href="https://pubmed.ncbi.nlm.nih.gov/32669681/" id="ir0070">https://pubmed.ncbi.nlm.nih.gov/32669681/</ext-link>&#x3009;.) [O&#x2019;Toole et al., 2021](O&#x2019;Toole, &#xC1;, Scher, E., Underwood, A., Jackson, B., Hill, V., McCrone, J.T., et al., 2021. Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool. Virus Evol. [Internet], vol. 7(issue 2). [cited 2021 Oct 13]. Available from: &#x3008;<ext-link ext-link-type="uri" xlink:href="https://academic.oup.com/ve/article/7/2/veab064/6315289" id="ir0065">https://academic.oup.com/ve/article/7/2/veab064/6315289</ext-link>&#x3009;.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35500140">35500140</a> <b>(1)</b> Overlapping Delta and Omicron Outbreaks During the COVID-19 Pandemic: Dynamic Panel Data Estimates<ul><li> We used <b>Nextclade</b> nomenclature [<xref rid="ref46" ref-type="bibr"><u>46</u></xref>] to collect clade designations from sequences and Pangolin nomenclature for lineage designations of SARS-CoV-2 [<xref rid="ref47" ref-type="bibr"><u>47</u></xref>,<xref rid="ref48" ref-type="bibr"><u>48</u></xref>].<li> [46](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [47](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [48](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35682933">35682933</a> <b>(3)</b> SARS-CoV-2 Continuous Genetic Divergence and Changes in Multiplex RT-PCR Detection Pattern on Positive Retesting Median 150 Days after Initial Infection<ul><li> 7586 isolates out of 9516 sequenced isolates were filtered out by <b>Nextclade</b>-assessed &#x201C;qc.overallStatus&#x201D; (v1.11.0, <uri xlink:href="https://github.com/nextstrain/<b>nextclade</b>">https://github.com/nextstrain/<b>nextclade</b></uri>, assessed on 20 April 2022) as &#x201C;good&#x201D; [<xref rid="B15-ijms-23-06254" ref-type="bibr"><u>15</u></xref>] and are subjected to mutation analysis.<li> [15](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35452383">35452383</a> <b>(1)</b> Detection of SARS-CoV-2 B.1.351 (Beta) Variant through Wastewater Surveillance before Case Detection in a Community, Oregon, USA<ul><li> We visualized and compared clinical specimen genomes from GISAID and putative viral isolate genomes by using <b>Nextclade</b> version 1.5.2 and used the output to manually construct the maximum-parsimony tree (<xref rid="R29" ref-type="bibr"><italic>29</italic></xref>) (<xref rid="F1" ref-type="fig"><u>Figure 1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35356875">35356875</a> <b>(1)</b> Serum Neutralization of SARS-CoV-2 Omicron BA.1 and BA.2 after BNT162b2 Booster Vaccination<ul><li> We obtained lineage and mutation calls by using Pangolin version 3.1.17 with pangoLEARN database 06&#x2013;12&#x2013;2021 and <b>Nextclade</b> version 0.14.4 (<xref rid="R5" ref-type="bibr"><italic>5</italic></xref>,<xref rid="R6" ref-type="bibr"><italic>6</italic></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35362410">35362410</a> <b>(1)</b> Detecting SARS-CoV-2 Omicron B.1.1.529 Variant in Wastewater Samples by Using Nanopore Sequencing<ul><li> We used <b>NextClade</b> (<xref rid="R8" ref-type="bibr"><italic>8</italic></xref>) for typing consensus sequences and mutation detection (<xref rid="T1" ref-type="table"><u>Table</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746671">35746671</a> <b>(1)</b> Two Separate Clusters of SARS-CoV-2 Delta Variant Infections in a Group of 41 Students Travelling from India: An Illustration of the Need for Rigorous Testing and Quarantine<ul><li> SARS-CoV-2 lineage assignments were determined using the Pangolin COVID-19 Lineage Assigner as well as <b>Nextclade</b> [<xref rid="B12-viruses-14-01198" ref-type="bibr"><u>12</u></xref>].<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35707526">35707526</a> <b>(1)</b> Analysis of the Transmission of SARS-CoV-2 Delta VOC in Yantai, China, August 2021<ul><li> For clade assignment and mutation calling, we imported all sequences into <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/tree" ext-link-type="uri">https://clades.nextstrain.org/tree</ext-link>) and the web-application Phylogenetic Assignment of Named Global Outbreak Lineages (pangolin: <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35665019">35665019</a> <b>(1)</b> A robust, highly multiplexed mass spectrometry assay to identify SARS-CoV-2 variants<ul><li> All FASTA consensus genome sequences underwent mutation calling and phylogenetic lineage assignment by the <b>Nextclade</b> Web Interface (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>, last accessed 4/18/22) and the Pangolin COVID-19 Lineage Assigner (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>, last accessed 4/18/22).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35679788">35679788</a> <b>(2)</b> Real-time allelic assays of SARS-CoV-2 variants to enhance sewage surveillance<ul><li> The specific lineage of the called consensus sequence was identified by <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0002">https://clades.nextstrain.org/</ext-link>).<li> The phylogenetic tree was created using <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0003">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35643449">35643449</a> <b>(5)</b> ClusTRace, a bioinformatic pipeline for analyzing clusters in virus phylogenies<ul><li> Pangolin [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>], Nextstrain [<xref ref-type="bibr" rid="CR21"><u>21</u></xref>], <b>Nextclade</b> [<xref ref-type="bibr" rid="CR22"><u>22</u></xref>], Jovian [<xref ref-type="bibr" rid="CR23"><u>23</u></xref>], HaVoC [<xref ref-type="bibr" rid="CR24"><u>24</u></xref>] and Lazypipe [<xref ref-type="bibr" rid="CR25"><u>25</u></xref>].<li> Tools like Pangolin and <b>Nextclade</b> are primarily designed for tracking large accumulations of mutation events that are rare and may be preceded by the less visible sub-lineage genetic changes.<li>Popular pipelines for tracking viral outbreak phylodynamics include Augur, Auspice, Nextstrain, <b>Nextclade</b> and Pangolin [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>&#x2013;<xref ref-type="bibr" rid="CR22"><u>22</u></xref>, <xref ref-type="bibr" rid="CR39"><u>39</u></xref>].<li> Pangolin and <b>Nextclade</b> are primarily concerned with classifying viral genomes into lineages or clades, while ClusTRace is designed to track mutations within lineages.<li> <b>Nextclade</b> also offers mutation calling for large clades, which is similar to ClusTRace mutation calling for lineages.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [22](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Sour Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [23](Zwagemaker F, Schmitz D, Nooij S, kroonma, Laros JFJ. DennisSchmitz/Jovian: 1.2.07. Zenodo; 2021.) [24](          <person-group person-group-type="author">            <name>              <surname>Nguyen</surname>              <given-names>PTT</given-names>            </name>            <name>              <surname>Plyusnin</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Sironen</surname>              <given-names>T</given-names>            </name>            <name>              <surname>Vapalahti</surname>              <given-names>O</given-names>            </name>            <name>              <surname>Kant</surname>              <given-names>R</given-names>            </name>            <name>              <surname>Smura</surname>              <given-names>T</given-names>            </name>          </person-group>          <article-title>HAVoC, a bioinformatic pipeline for reference-based consensus assembly and lineage assignment for SARS-CoV-2 sequences</article-title>          <source>BMC Bioinform</source>          <year>2021</year>          <volume>22</volume>          <fpage>1</fpage>          <lpage>8</lpage>          <pub-id pub-id-type="doi">10.1186/s12859-020-03881-z</pub-id>        ) [25](          <person-group person-group-type="author">            <name>              <surname>Plyusnin</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Kant</surname>              <given-names>R</given-names>            </name>            <name>              <surname>J&#xE4;&#xE4;skel&#xE4;inen</surname>              <given-names>AJ</given-names>            </name>            <name>              <surname>Sironen</surname>              <given-names>T</given-names>            </name>            <name>              <surname>Holm</surname>              <given-names>L</given-names>            </name>            <name>              <surname>Vapalahti</surname>              <given-names>O</given-names>            </name>            <etal/>          </person-group>          <article-title>Novel NGS pipeline for virus discovery from a wide spectrum of hosts and sample types</article-title>          <source>Virus Evolut</source>          <year>2020</year>          <volume>6</volume>          <fpage>veaa091</fpage>          <pub-id pub-id-type="doi">10.1093/ve/veaa091</pub-id>        ) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [22](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Sour Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [39](          <person-group person-group-type="author">            <name>              <surname>Huddleston</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Hadfield</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Sibley</surname>              <given-names>TR</given-names>            </name>            <name>              <surname>Lee</surname>              <given-names>J</given-names>            </name>            <name>              <surname>Fay</surname>              <given-names>K</given-names>            </name>            <name>              <surname>Ilcisin</surname>              <given-names>M</given-names>            </name>            <etal/>          </person-group>          <article-title>Augur: a bioinformatics toolkit for phylogenetic analyses of human pathogens</article-title>          <source>J Open Sour Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>2906</fpage>          <pub-id pub-id-type="doi">10.21105/joss.02906</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35643510">35643510</a> <b>(1)</b> Identification and application of a pair of noncompeting monoclonal antibodies broadly binding to the nucleocapsid proteins of SARS-CoV-2 variants including Omicron<ul><li> The clade of the isolated Omicron variant was determined using the <b>Nextclade</b> web application (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) and the sequence information has not been uploaded yet.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746651">35746651</a> <b>(2)</b> Omicron Sub-Lineages (BA.1.1.529 + BA.*) Current Status in Ecuador<ul><li>Sequences were classified by epidemiological week and then submitted to <b>Nextclade</b> [<xref rid="B14-viruses-14-01177" ref-type="bibr"><u>14</u></xref>] and Pangolin COVID-19 [<xref rid="B15-viruses-14-01177" ref-type="bibr"><u>15</u></xref>] for clade and lineage assignment.<li> A phylogenetic tree was built in <b>Nextclade</b> using the nearest neighbor method and visualized by Nextrain Auspice [<xref rid="B14-viruses-14-01177" ref-type="bibr"><u>14</u></xref>].<li> [14](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [14](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35746637">35746637</a> <b>(6)</b> Dominance of Three Sublineages of the SARS-CoV-2 Delta Variant in Mexico<ul><li> Synonymous/nonsynonymous substitutions were identified using the <b>Nextclade</b> single nucleotide variant (SNV) calling system [<xref rid="B20-viruses-14-01165" ref-type="bibr"><u>20</u></xref>].<li> Finally, the odds ratio between <b>Nextclade</b> clades and categorical variables, at a 95% confidence level, was calculated using binary logistic regression models in SPSS v22.<li> Interestingly, AY.20 and AY.100 belong to the 21J <b>Nextclade</b> Delta subclade, the most widespread in Mexico, whereas AY.26 falls within subclade 21I which started decreasing before Omicron appeared.<li> Additionally, AY.20 and AY.100, which are placed in clade 21J of the <b>Nextclade</b> classification, also share ten high-frequency substitutions (six in ORF1a, and one in each ORF1b, S, ORF7b, and N).<li> Interestingly, the prevalence of AY.26 decreased by November 2021 (&lt;10%), as did the majority of worldwide Delta sublineages belonging to clade 21I (of <b>Nextclade</b>), while AY.20 remained the most prevalent (&gt;44%) until the expansion of the Omicron variant.<li> Here, we identified 16 aa changes in ORF1a, 6 of which were shared by AY.20 and AY.100, probably due to their placement within the <b>Nextclade</b> subclade 21J.<li> [20](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35620998">35620998</a> <b>(1)</b> Detection of SARS-CoV-2 in young children attending day-care centres in Belgium, May 2020 to February 2022<ul><li> Consensus sequences were extracted, and clade and lineage assignment performed using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) and Pangolin (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>), respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35644262">35644262</a> <b>(3)</b> Evolving strategy for an evolving virus: Development of real-time PCR assays for detecting all SARS-CoV-2 variants of concern<ul><li> pangolin was used to assign lineages (<ext-link ext-link-type="uri" xlink:href="https://www.nature.com/articles/s41564-020-0770-5" id="ir0035">https://www.nature.com/articles/s41564&#x2013;020&#x2013;0770&#x2013;5</ext-link>) and <b>nextclade</b> was used to detect mutations (<ext-link ext-link-type="uri" xlink:href="http://github.com/nextstrain/<b>nextclade</b>" id="ir0040">github.com/nextstrain/<b>nextclade</b></ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35578378">35578378</a> <b>(2)</b> Genomics of postvaccination SARS&#x2010;CoV&#x2010;2 infections during the Delta dominated second wave of COVID&#x2010;19 pandemic, from Mumbai Metropolitan Region (MMR), India<ul><li>The consensus FASTA files from the SARS&#x2010;CoV&#x2010;2 were aligned using <italic toggle="yes">MAFFT</italic> (v7.489)<xref rid="jmv27861-bib-0022" ref-type="bibr"><sup>22</sup></xref> and clustered using <italic toggle="yes">Augur</italic> (v13.0.0).<xref rid="jmv27861-bib-0023" ref-type="bibr"><sup>23</sup></xref> Maximum likelihood trees were constructed with default parameters using <italic toggle="yes">IQ&#x2010;TREE</italic> (v2.1.3)<xref rid="jmv27861-bib-0024" ref-type="bibr"><sup>24</sup></xref> and visualized with <italic toggle="yes">Auspice</italic> (v2.32.1).<xref rid="jmv27861-bib-0025" ref-type="bibr"><sup>25</sup></xref> <b>Nextclade</b><xref rid="jmv27861-bib-0026" ref-type="bibr"><sup>26</sup></xref> was used to assign clades to the sequences.<li> The samples were found to be distributed in three clades based on the <b>Nextclade</b> classification, that is, 21A (Delta), 21I (Delta), 21J (Delta).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35623374">35623374</a> <b>(1)</b> Co-infection with SARS-CoV-2 omicron BA.1 and BA.2 subvariants in a non-vaccinated woman<ul><li> <b>Nextclade</b> (version 1.14.0) analysis confirmed the presence of two distinct isolates (<xref rid="sec1" ref-type="sec"><u>appendix p 2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36398130">36398130</a> <b>(3)</b> Wastewater Reveals the Spatiotemporal Spread of SARS-CoV-2in the Canton of Ticino (Switzerland) during the Onset of the COVID-19Pandemic<ul><li> Finally,curated consensus sequences and forward or reverse chains with goodquality were used to assign a SARS-CoV-2 clade by using the <b>Nextclade</b>web tool v1.7.1 (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>).<li>To explore the identity ofSARS-CoV-2 strain(s) circulating at that time, we performed an analysisof the curated Sanger sequences of the spike locus using <b>Nextclade</b>.The <b>Nextclade</b> tool assigned most of the consensus sequences to clade19A, the same clade as the original Wuhan strain.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35605952">35605952</a> <b>(1)</b> Genomic evidence of co-identification with Omicron and Delta SARS-CoV-2 variants: a report of two cases<ul><li>Although we have been able to obtain the full genome sequences, it was not possible to assign the two sequences to a specific SARS-CoV-2 variant or lineage using either the <b>Nextclade</b> tool or the Pangolin lineage classification software (<xref rid="bib0001" ref-type="bibr"><u>Aksamentov&#xA0;et&#xA0;al., 2021</u></xref>; <xref rid="bib0008" ref-type="bibr"><u>O'Toole&#xA0;et&#xA0;al., 2021</u></xref>) owing to the high number of sites where different nucleotides were called at one position.<li> [Aksamentov&#xA0;et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Source Softw</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) [O'Toole&#xA0;et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>O'Toole</surname>              <given-names>&#xC1;</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Underwood</surname>              <given-names>A</given-names>            </name>            <name>              <surname>Jackson</surname>              <given-names>B</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>V</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>JT</given-names>            </name>            <name>              <surname>Colquhoun</surname>              <given-names>R</given-names>            </name>            <name>              <surname>Ruis</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Abu-Dahab</surname>              <given-names>K</given-names>            </name>            <name>              <surname>Taylor</surname>              <given-names>B</given-names>            </name>            <name>              <surname>Yeats</surname>              <given-names>C</given-names>            </name>            <name>              <surname>du Plessis</surname>              <given-names>L</given-names>            </name>            <name>              <surname>Maloney</surname>              <given-names>D</given-names>            </name>            <name>              <surname>Medd</surname>              <given-names>N</given-names>            </name>            <name>              <surname>Attwood</surname>              <given-names>SW</given-names>            </name>            <name>              <surname>Aanensen</surname>              <given-names>DM</given-names>            </name>            <name>              <surname>Holmes</surname>              <given-names>EC</given-names>            </name>            <name>              <surname>Pybus</surname>              <given-names>OG</given-names>            </name>            <name>              <surname>Rambaut</surname>              <given-names>A.</given-names>            </name>          </person-group>          <article-title>Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool</article-title>          <source>Virus Evol</source>          <volume>7</volume>          <year>2021</year>          <comment>veab064</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35598737">35598737</a> <b>(4)</b> Whole-genome sequencing of SARS-CoV-2 reveals diverse mutations in circulating Alpha and Delta variants during the first, second, and third waves of COVID-19 in South Kivu, east of the Democratic Republic of the Congo<ul><li> Pangolin UShER was used further downstream to analyze consensus sequences for lineage assignments based on parsimony and <b>Nextclade</b> (<xref rid="bib0014" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>) for clade specification.<li> The tree was generated through <b>nextclade</b>s&#x2019; command-line workflow and visualized using a local Auspice installation.<li> <b>Nextclade</b> CLI 1.4.2 tool (<xref rid="bib0001" ref-type="bibr"><u>Aksamentov&#xA0;and Neher,&#xA0;2021</u></xref>) was used to assign SARS-CoV-2 &#x201C;Nextstrain clades&#x201D; to the genomes based on predetermined mutations.<li>Based on the identified mutation profiles, the <b>Nextclade</b> analysis clustered the sequences in four main Pango lineages.<li> [Hadfield&#xA0;et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov&#xA0;and Neher,&#xA0;2021](Aksamentov I, Neher R, Nextclade. <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="interref0013">https://clades.nextstrain.org</ext-link>, XXX (accessed 13 September 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35640456">35640456</a> <b>(2)</b> COVID-19 &amp; differential effects in twins: Insights from Placenta Pathology<ul><li> The genomes were submitted to GISAID database and were input into <b>NextClade</b> and UCSC UShER for phylogeny analysis.<li> In the <b>NextClade</b> database, both genomes were clustered into the 21J (Delta) branch.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35611117">35611117</a> <b>(1)</b> Optimizing the Illumina COVIDSeq laboratorial and bioinformatics pipeline on thousands of samples for SARS-CoV-2 Variants of Concern tracking<ul><li> In order to achieve these performances, several non-critical steps of the pipeline were skipped via <italic>--skip_kraken2 --skip_fastp --skip_<b>nextclade</b> --skip_snpeff --skip_variants_quast --skip_plasmidid --skip_blast --skip_bandage &#x2013;skip_asciigenome.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35665008">35665008</a> <b>(3)</b> Sequence Proven Reinfections with SARS-CoV-2 at a Large Academic Center<ul><li> Clade determination was performed via <b>NextClade</b> CLI v1.4.5[<xref rid="R18" ref-type="bibr"><u>18</u></xref>].<li> Samples were given an initial determination as reinfection or persistence of initial genome based on whether there was a change in clade assigned by <b>NextClade</b>.<li> Afterward, the consensus genomes for each patient were manually reviewed in <b>NextClade</b> and/or IGV to provide a final determination on reinfection or persistence.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35582905">35582905</a> <b>(1)</b> Partial ORF1ab Gene Target Failure with Omicron BA.2.12.1<ul><li> Viral lineage classification, identification of mutations, and phylogenetic analyses were performed using Pangolin v4.0.5 and <b>Nextclade</b> v1.11.0.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35582907">35582907</a> <b>(2)</b> Putative Host-Derived Insertions in the Genomes of Circulating SARS-CoV-2 Variants<ul><li> The sequences were then processed by <b>Nextclade</b> CLI v1.7.0 (<xref rid="B49" ref-type="bibr"><u>49</u></xref>), which generated a multiple-sequence alignment against the reference genome (Wuhan-Hu-1/2019) and provided a list of single nucleotide polymorphisms, insertions, and deletions associated with each genome sequence.<li> applied by <b>Nextclade</b> were used for further analysis (<italic toggle="yes">n</italic>&#x2009;=&#x2009;5,226,229).<li> [49](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Joss</source><volume>6</volume>:<fpage>3773</fpage>. doi:<pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35655849">35655849</a> <b>(2)</b> Screening and Whole Genome Sequencing of SARS-CoV-2 Circulating During the First Three Waves of the COVID-19 Pandemic in Libreville and the Haut-Ogoou&#xE9; Province in Gabon<ul><li> The genome sequences were classified into lineages using Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin) and <b>Nextclade</b> (<xref rid="B30" ref-type="bibr"><u>30</u></xref>).<li> It was assigned to VOC Delta according <b>Nextclade</b> (<xref rid="F4" ref-type="fig"><u>Figure 4</u></xref>).<li> [30](<person-group person-group-type="author"><collab>Nextclade</collab></person-group>. Available online at: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link> (accessed December 08, 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35579429">35579429</a> <b>(1)</b> Increased Abundance of <named-content content-type="genus-species">Achromobacter xylosoxidans</named-content> and <named-content content-type="genus-species">Bacillus cereus</named-content> in Upper Airway Transcriptionally Active Microbiome of COVID-19 Mortality Patients Indicates Role of Co-Infections in Disease Severity and Outcome<ul><li> Clade assignment to all the genomes was done using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632815">35632815</a> <b>(5)</b> Molecular Analysis of SARS-CoV-2 Lineages in Armenia<ul><li>To perform preliminary QC of Armenian samples and select contextual samples, we initially screened our sequences with <b>Nextclade</b> [<xref rid="B25-viruses-14-01074" ref-type="bibr"><u>25</u></xref>] and PANGOLIN [<xref rid="B26-viruses-14-01074" ref-type="bibr"><u>26</u></xref>].<li>We performed the functional annotation of analyzed sequences using <b>Nextclade</b> [<xref rid="B25-viruses-14-01074" ref-type="bibr"><u>25</u></xref>] and CorGAT [<xref rid="B35-viruses-14-01074" ref-type="bibr"><u>35</u></xref>] tools (<xref rid="app1-viruses-14-01074" ref-type="app"><u>Supplementary Materials</u></xref>&#xA0;<xref rid="app1-viruses-14-01074" ref-type="app"><u>Tables S4 and S5</u></xref>).<li> (reversions to reference, mutations ascribed to different clades, and mutations that are for reversions or belonging to other clades) as defined by the <b>Nextclade</b> app.<li> Therefore, we conducted a reassessment of published diagnostic PCR assays, including those recommended by the World Health Organization (WHO), through the evaluation of the possible effect of identified mutations on the efficacy of recommended primers and probes used for PCR detection of SARS-CoV-2 with the <b>Nextclade</b> app.<li> First, we assessed the number of missed nucleotides in the <b>Nextclade</b> app analysis, which can indicate gaps in genomes because of insufficient read coverage.<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [35](          <person-group person-group-type="author"><name><surname>Chiara</surname><given-names>M.</given-names></name><name><surname>Zambelli</surname><given-names>F.</given-names></name><name><surname>Tangaro</surname><given-names>M.A.</given-names></name><name><surname>Mandreoli</surname><given-names>P.</given-names></name><name><surname>Horner</surname><given-names>D.S.</given-names></name><name><surname>Pesole</surname><given-names>G.</given-names></name></person-group>          <article-title>CorGAT: A Tool for the Functional Annotation of SARS-CoV-2 Genomes</article-title>          <source>Bioinformatics</source>          <year>2020</year>          <volume>36</volume>          <fpage>5522</fpage>          <lpage>5523</lpage>          <pub-id pub-id-type="doi">10.1093/bioinformatics/btaa1047</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35602577">35602577</a> <b>(4)</b> Validaci&#xF3;n de una nueva estrategia para la identificaci&#xF3;n de variantes de SARS-CoV-2 mediante secuenciaci&#xF3;n del gen espiga por Sanger<ul><li> con el programa web Nextstrain-<b>Nextclade</b> Web 1.10.0, <b>Nextclade</b> CLI 1.7.0 (2021-12-09) (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intr0025">https://clades.nextstrain.org/</ext-link>).<li> de acuerdo con el programa en l&#xED;nea Nextstrain-<b>Nextclade</b> Web 1.10.0, <b>Nextclade</b> CLI 1.7.0 (2021-12-09) y PANGO Lineages (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/" id="intr0030">https://cov-lineages.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35628343">35628343</a> <b>(2)</b> Complex Mutation Pattern of Omicron BA.2: Evading Antibodies without Losing Receptor Interactions<ul><li> In addition, we used <b>NextClade</b> [<xref rid="B9-ijms-23-05534" ref-type="bibr"><u>9</u></xref>] and an in-house Python script to process sequences for the identification of the BA.2 signature mutations.<li> These sequences were analyzed for amino acid changes using <b>Nextclade</b> and later processed using an in-house Python script to identify the prevalence of mutations in BA.2.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35605437">35605437</a> <b>(5)</b> Advancing genomic epidemiology by addressing the bioinformatics bottleneck: Challenges, design principles, and a Swiss example<ul><li> For example, a Pango lineage assigner requires the pangolin tool&#xA0;(<xref rid="b20" ref-type="bibr"><u>O&#x2019;Toole et al., 2021</u></xref>), a <b>Nextclade</b> importer needs <b>Nextclade</b>&#xA0;(<xref rid="b1" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>), and the metadata importer has to mount a network folder.<li> For example, we have a <b>Nextclade</b> importer service that runs the <b>Nextclade</b> program and imports resulting quality scores and mutations.<li> This service queries the database every ten minutes and looks for entries in the sequence table where <b>Nextclade</b> quality scores were previously unpopulated.<li> [O&#x2019;Toole et al., 2021](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>A.</given-names>            </name>            <etal/>          </person-group>          <article-title>Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool</article-title>          <source>Virus Evol.</source>          <issn>2057-1577</issn>          <volume>7</volume>          <issue>2</issue>          <year>2021</year>          <pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>          <comment>veab064</comment>        ) [Aksamentov et al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <issue>67</issue>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632784">35632784</a> <b>(1)</b> Clearance of Persistent SARS-CoV-2 RNA Detection in a NF&#x3BA;B-Deficient Patient in Association with the Ingestion of Human Breast Milk: A Case Report<ul><li> Finally, we used the open-source interactive tool <b>NextClade</b>(v0.4.0) (Seattle, WA, USA) [<xref rid="B18-viruses-14-01042" ref-type="bibr"><u>18</u></xref>] to assign the sequences to clades.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35551247">35551247</a> <b>(1)</b> Human-to-dog transmission of SARS-CoV-2, Colombia<ul><li> Each assembly was typed based on the PANGOLIN nomenclature lineage allocator and SNPs identification was performed using the clade assignment, mutation calling and sequence quality checking tool, <b>NextClade</b>, which identifies differences between sequences and a reference sequence used by Nextstrain.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35543892">35543892</a> <b>(1)</b> Genome sequencing reveals existence of SARS-CoV-2 B.1.1.529 variant in Egypt<ul><li><b>Nextclade</b> web tool v.012.0 [<xref ref-type="bibr" rid="CR17"><u>17</u></xref>] was used to compare the assembly sequence to SARS-CoV-2 reference sequences and to assign it to clades.<li> [17](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>EB</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>RA</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J Open Source Softw</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/JOSS.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35552003">35552003</a> <b>(1)</b> Phylogenetic and amino acid signature analysis of the SARS-CoV-2s lineages circulating in Tunisia<ul><li> According to the <b>Nextclade</b> nomenclature, the 58 Tunisian isolates were dispersed in 7 different SARS-CoV-2 subclades including 19A, 19B, 20A, 20B, 20C, 20 E and 20I (<xref rid="f0010" ref-type="fig">Fig.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35536777">35536777</a> <b>(4)</b> SARS-CoV-2 variants of concern Alpha and Delta show increased viral load in saliva<ul><li> Some of the samples in this analysis exceeded this threshold, which prevented database upload, but all had sufficient information to confidently assign clade by Pangolin and <b>Nextclade</b> [<xref rid="pone.0267750.ref025" ref-type="bibr"><u>25</u></xref>, <xref rid="pone.0267750.ref026" ref-type="bibr"><u>26</u></xref>].<li> To maintain phylogenetic independence, we only compare Ct values for variants at branch tips within the <b>NextClade</b> phylogeny [<xref rid="pone.0267750.ref026" ref-type="bibr"><u>26</u></xref>].<li> Therefore, the four <b>Nextclade</b>s we compare do not have parent-offspring relation.<li> We did notice that samples within the higher Ct range (28&#x2013;33) had more regions with ambiguous nucleotides and were therefore less likely to have received a clade assignment via <b>Nextclade</b>.<li> [25](<ext-link xlink:href="https://github.com/CUGBF/SARS-CoV-2_Ct-vs-Clade" ext-link-type="uri">https://github.com/CUGBF/SARS-CoV-2_Ct-vs-Clade</ext-link>) [26](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <name><surname>Roemer</surname><given-names>C</given-names></name>, <name><surname>Hodcroft</surname><given-names>EB</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source><italic toggle="yes">Journal of Open Source Software</italic></source>, <year>2021</year>; <volume>6</volume>(<issue>67</issue>), <fpage>3773</fpage>.) [26](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <name><surname>Roemer</surname><given-names>C</given-names></name>, <name><surname>Hodcroft</surname><given-names>EB</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source><italic toggle="yes">Journal of Open Source Software</italic></source>, <year>2021</year>; <volume>6</volume>(<issue>67</issue>), <fpage>3773</fpage>.) </ul></p>
<p><u>No PMID</u> <b>(1)</b> Molecular Monitoring of SARS-CoV-2 in Different Sewage Plants in Venice andthe Implications for Genetic Surveillance<ul><li> Lineage assignment was performed using the Pangolin tool<sup><xref ref-type="bibr" rid="ref50"><u>50</u></xref></sup> andconfirmed with <b>Nextclade</b> Web<sup><xref ref-type="bibr" rid="ref51"><u>51</u></xref></sup> (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>).<li> [50](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [51](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=36417255">36417255</a> <b>(3)</b> Similarity Index&#x2013;Probabilistic Confidence Estimation of SARS-CoV-2 Strain Relatedness in Localized Outbreaks<ul><li> In the latter case, public algorithms such as <b>NextClade</b> (v1.14.1) can be useful, qualitatively, for defining similarity or dissimilarity in localized CoV-2 outbreak strains [<xref rid="B3-epidemiologia-03-00019" ref-type="bibr"><u>3</u></xref>].<li> The consensus FASTA files were then uploaded to <b>NextClade</b>&#x2019;s algorithm (v1.14.1) in order to define each strain&#x2019;s mutational repertoire.<li> <b>NextClade</b> can be found at (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>).<li> [3](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35467028">35467028</a> <b>(4)</b> Culture and identification of a &#x201C;Deltamicron&#x201D; SARS&#x2010;CoV&#x2010;2 in a three cases cluster in southern France<ul><li>Two pandemic variants, Delta and Omicron 21K (<b>Nextclade</b> classification<xref rid="jmv27789-bib-0025" ref-type="bibr"><sup>25</sup></xref>, <xref rid="jmv27789-bib-0026" ref-type="bibr"><sup>26</sup></xref>)/BA.1 (Pangolin classification<xref rid="jmv27789-bib-0027" ref-type="bibr"><sup>27</sup></xref>), recently succeeded each other as the predominant viruses but co&#x2010;circulated for a period of several weeks, creating conditions for co&#x2010;infections and subsequently recombinations.<li> Nucleotide and amino acid changes relative to the Wuhan&#x2010;Hu&#x2010;1 isolate genome were obtained using the <b>Nextclade</b> tool (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<li> Nucleotide and amino acid changes relative to the Wuhan&#x2010;Hu&#x2010;1 isolate genome were obtained using the <b>Nextclade</b> tool (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<xref rid="jmv27789-bib-0025" ref-type="bibr"><sup>25</sup></xref>, <xref rid="jmv27789-bib-0026" ref-type="bibr"><sup>26</sup></xref> Nextstrain clades and Pangolin lineages were determined using the <b>Nextclade</b> web application (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>)<xref rid="jmv27789-bib-0025" ref-type="bibr"><sup>25</sup></xref>, <xref rid="jmv27789-bib-0026" ref-type="bibr"><sup>26</sup></xref> and the Pangolin tool (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>),<xref rid="jmv27789-bib-0027" ref-type="bibr"><sup>27</sup></xref> respectively.<li> Adapted from screenshots of the <b>nextclade</b> web application output (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35523782">35523782</a> <b>(2)</b> Genetic diversity of SARS-CoV-2 infections in Ghana from 2020-2021<ul><li> A coverage map was generated by comparing the Ghanaian SARS-CoV-2 genomes and the reference genome (Wuhan-Hu-1/2019) using <b>Nextclade</b> CLI (version 1.4.0).<li> <b>Nextclade</b> was also used to perform clade assignment and overall quality assessment of the genomes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35528048">35528048</a> <b>(1)</b> Development and performance of a point-of-care rapid antigen test for detection of SARS-COV-2 variants<ul><li> <b>Nextclade</b> and Pangolin were used to identify lineages and genomic mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35592078">35592078</a> <b>(2)</b> Inactivation and Recovery of High Quality RNA From Positive SARS-CoV-2 Rapid Antigen Tests Suitable for Whole Virus Genome Sequencing<ul><li> For phylogenetic analysis, sequences were aligned using a pipeline used previously (<xref rid="B5" ref-type="bibr"><u>5</u></xref>, <xref rid="B6" ref-type="bibr"><u>6</u></xref>) which included the analysis with <b>Nextclade</b> (<xref rid="B14" ref-type="bibr"><u>14</u></xref>) to identify differences between sequences and report sequence quality, while the pangolin tool was used for the assignment of epidemiological lineages (<xref rid="B15" ref-type="bibr"><u>15</u></xref>).<li> According to <b>Nextclade</b> Pango nomenclature, were identified seven different lineages: 9 BA.1.1.15 (30.0 %), 5 BA.1 (16.7 %), 4 BA.1.19 (13.3 %), 4 BA.1.1 (13.3 %), 4 BA (13.3 %), 3 BA.1.17 (10.0 %), and 1 BA.1.10 (3.3%).<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32954414">32954414</a>) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34161223">34161223</a>) [14](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Software.</source> (<year>2021</year>) <volume>6</volume>:<fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35320700">35320700</a> <b>(1)</b> Rapid Replacement of SARS-CoV-2 Variants by Delta and Subsequent Arrival of Omicron, Uganda, 2021<ul><li> We performed mutation calling by using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>), followed by SARS-CoV-2 lineage analysis with Pangolin (<ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35572963">35572963</a> <b>(4)</b> SARS-CoV-2 Molecular Epidemiology Can Be Enhanced by Occupational Health: The Experience of Monitoring Variants of Concern in Workplaces in Rio de Janeiro, Brazil<ul><li> Aligned FASTA files were also assigned to variants and quality checked by <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>, version 1.10.0) (<xref rid="B22" ref-type="bibr"><u>22</u></xref>).<li> The output JSON file from <b>NextClade</b> was used to generate phylogeny through <ext-link xlink:href="https://auspice.us/" ext-link-type="uri">https://auspice.us/</ext-link> (version 0.8.0) (<xref rid="B23" ref-type="bibr"><u>23</u></xref>, <xref rid="B24" ref-type="bibr"><u>24</u></xref>).<li> We obtained 50 high-quality, full-length consensus genomes, with quality scores above 30 for base calling, at least 10x of depth, and adequate <b>NextClade</b> assignment.<li> <bold>(B)</bold> Distribution of the Gamma and Delta variants, assigned by <b>NextClade</b> version 1.10.0, per month.<li> [22](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J. Open Source Softw.</source> (<year>2021</year>) <volume>6</volume>:<fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632673">35632673</a> <b>(3)</b> Epidemiological Features of COVID-19 in Northwest Russia in 2021<ul><li> The <b>Nextclade</b> tool was used to assess the quality of assembled sequences and to assign genomes into lineages [<xref rid="B10-viruses-14-00931" ref-type="bibr"><u>10</u></xref>].<li> A global phylogenetic tree of SARS-CoV-2 variants was constructed using the tools implemented in <b>Nextclade</b> [<xref rid="B10-viruses-14-00931" ref-type="bibr"><u>10</u></xref>].<li>Genomic analysis based on <b>Nextclade</b> SARS-CoV-2 Clade Assigner showed seven lineages (<xref rid="viruses-14-00931-f008" ref-type="fig"><u>Figure 8</u></xref>) circulating in the region in 2021: 20A, 20B, 20H (Beta); 20I (Alpha); 21A (Delta); 21J (Delta); and 21K (Omicron).<li> [10](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [10](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632677">35632677</a> <b>(1)</b> A Short Series of Case Reports of COVID-19 in Immunocompromised Patients<ul><li> accessed on 15 March 2022), GISAID [<xref rid="B14-viruses-14-00934" ref-type="bibr"><u>14</u></xref>], and <b>NextClade</b> [<xref rid="B15-viruses-14-00934" ref-type="bibr"><u>15</u></xref>].<li> [14](          <person-group person-group-type="author"><collab>GISAID</collab></person-group>          <article-title>Tracking of Variants</article-title>          <comment>Available online: <ext-link xlink:href="https://www.gisaid.org/hcov19-variants" ext-link-type="uri">https://www.gisaid.org/hcov19-variants</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-09-15">(accessed on 15 September 2021)</date-in-citation>        ) [15](          <person-group person-group-type="author"><collab>Next-Strain</collab></person-group>          <article-title>Real-Time Tracking of Pathogen Evolution</article-title>          <comment>Available online: <ext-link xlink:href="https://nextstrain.org" ext-link-type="uri">https://nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-09-15">(accessed on 15 September 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35043112">35043112</a> <b>(2)</b> Putative host-derived insertions in the genomes of circulating SARS-CoV-2 variants<ul><li> The sequences were then processed by <b>NextClade</b> CLI v1.7.0 (<xref rid="R49" ref-type="bibr"><u>49</u></xref>) which generated a multiple sequence alignment against the reference genome (Wuhan-Hu-1/2019) and provided a list of single nucleotide polymorphisms, insertions, and deletions associated with each genome sequence.<li> applied by <b>NextClade</b> were used for further analysis (n=5,226,229).<li> [49](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <name><surname>Roemer</surname><given-names>C</given-names></name>, <name><surname>Hodcroft</surname><given-names>EB</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source><volume>6</volume>:<fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35528635">35528635</a> <b>(1)</b> First Identification of Reinfection by a Genetically Different Variant of SARS-CoV-2 in a Homeless Person from the Metropolitan Area of Santiago, Chile<ul><li> In addition, the Nextstrain clade assignment was performed using <b>NextClade</b> version 1.9.0 to compare and visualize our samples globally.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35625733">35625733</a> <b>(3)</b> Resilience of Spike-Specific Immunity Induced by COVID-19 Vaccines against SARS-CoV-2 Variants<ul><li> To identify the mutations in the selected variants genomes, the Bioinformatic tool <b>Nextclade</b> v.1.14.0 was used (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 17 Frebruary 2022).<li> <b>Nextclade</b> is a Web application that can identify differences between a query and a reference sequence (SARS-CoV-2 isolate Wuhan-Hu-1, NCBI accession NC_045512).<li> Genome sequence mutations resulting in amino acid changes were identified using <b>Nextclade</b> (details in Material and Methods).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35471044">35471044</a> <b>(1)</b> An engineered ACE2 decoy neutralizes the SARS-CoV-2 Omicron variant and confers protection against infection in vivo<ul><li> Time course of variant distribution was analyzed by <b>Nextclade</b> ver 1.7.0 (<ext-link xlink:href="https://joss.theoj.org/papers/10.21105/joss.03773" ext-link-type="uri">https://joss.theoj.org/papers/10.21105/joss.03773</ext-link>) from SARS-CoV-2 nucleic acid sequences which were downloaded from GISAID database as of December 17, 2021.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35474215">35474215</a> <b>(1)</b> Tracking SARS-CoV-2 Omicron diverse spike gene mutations identifies multiple inter-variant recombination events<ul><li> Mutation and Pango lineage of &#x201C;Deltacron&#x201D;-like variants with L452R mutation was annotated with <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35468749">35468749</a> <b>(2)</b> Identification of SARS-CoV-2 variants using viral sequencing for the Centers for Disease Control and Prevention genomic surveillance program<ul><li>Consensus sequences were analyzed with the Pangolin and <b>NextClade</b> pipelines to generate the lineage and clade information, respectively.<li> By default, the latest version of the PANGOLearn and <b>NextClade</b> databases were downloaded for the most up to date lineage and clade information.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35547225">35547225</a> <b>(2)</b> Identification and Analysis of SARS-CoV-2 Alpha Variants in the Largest Taiwan COVID-19 Outbreak in 2021<ul><li> To verify the SNV and InDel results analyzed in our genome workstation, we deposited these five genomic RNA sequences into GISAID EpiCoV, as named in the order of the above numbers KMUH-3 to KMUH-7 (<xref rid="T1" ref-type="table"><u>Table 1</u></xref>), and analyzed them by using <b>Nextclade</b> v1.10.0<sup>3</sup> (<xref rid="B48" ref-type="bibr"><u>48</u></xref>).<li> These viruses share similar amino acid deletions (e.g., NSP6 3675&#x2013;3677del, spike 69&#x2013;70del, and spike 144del) and non-synonymous amino acid variations (e.g., spike N501Y, D614G, and P681H) predicted by using Dindel (v1.01) (<xref rid="B36" ref-type="bibr"><u>36</u></xref>), Lofreq (v2.1.5) (<xref rid="B37" ref-type="bibr"><u>37</u></xref>), GISAID and <b>Nextclade</b> v1.10.0.<li> [48](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA.</given-names></name></person-group><source><italic>Nextclade: Clade Assignment, Mutation Calling and Quality Control for Viral Genomes.</italic></source> (<year>2021</year>). Available online at: <ext-link xlink:href="https://zenodo.org/record/5607694" ext-link-type="uri">https://zenodo.org/record/5607694</ext-link> (accessed January 10, 2022).) [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20980555">20980555</a>) [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23066108">23066108</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35465708">35465708</a> <b>(3)</b> Evaluation of a Rapid and Accessible Reverse Transcription-Quantitative PCR Approach for SARS-CoV-2 Variant of Concern Identification<ul><li> PANGO lineage assignment was performed using the Pangolin Coronavirus Disease 2019 (COVID-19) Lineage Assigner (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>) running Pangolin v3.1.17, while <b>Nextclade</b> Web v1.13.1 and auspice.us v0.8.0 were used to perform phylogenetic placement (<xref rid="B3" ref-type="bibr"><u>3</u></xref>, <xref rid="B29" ref-type="bibr"><u>29</u></xref>, <xref rid="B30" ref-type="bibr"><u>30</u></xref>).<li> Mapping the genotyping results for the cohort based on RT-qPCR mutation analysis onto the <b>Nextclade</b> phylogenetic tree demonstrated close correlation with their WHO variant designations (<xref rid="F2" ref-type="fig">Fig.<li><b>Nextclade</b> phylogenetic tree of 3,097 SARS-CoV-2 genomes, including all 732 of the sequenced genomes from this study and 2,365 genomes from the Nextstrain global reference tree as of 2 February 2022.<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26511519">26511519</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632430">35632430</a> <b>(1)</b> Characteristics of the First 284 Patients Infected with the SARS-CoV-2 Omicron BA.2 Subvariant at a Single Center in the Apulia Region of Italy, January&#x2013;March 2022<ul><li> Phylogenetic analysis to investigate the relationships among strains of the BA.2 lineage was performed using the <b>Nextclade</b> sequence analysis web app [<xref rid="B19-vaccines-10-00674" ref-type="bibr"><u>19</u></xref>].<li> [19](          <article-title>Nextclade Sequence Analysis Web App</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-02-14">(accessed on 14 February 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632625">35632625</a> <b>(3)</b> Phylodynamic Dispersal of SARS-CoV-2 Lineages Circulating across Polish&#x2013;German Border Provinces<ul><li>For SARS-CoV-2 variant identification, we employed two lineage assignment tools: PANGOLIN v3.0 (Available online: <uri xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</uri> (accessed on 1 July 2021)) and <b>NEXTCLADE</b> v01.2.0 (Available online: <uri xlink:href="https://github.com/nextstrain/<b>nextclade</b>">https://github.com/nextstrain/<b>nextclade</b></uri> (accessed on 1 July 2021)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35459088">35459088</a> <b>(16)</b> Comparison of SARS-CoV-2 sequencing using the ONT GridION and the Illumina MiSeq<ul><li>Consensus genomes produced by the GridION and the MiSeq were uploaded to <b>Nextclade</b> to determine the genome quality.<li> <b>Nextclade</b> classifies genomes as either good, mediocre, or bad, based on the amount of missing data, and the number of mixed sites, private mutations, clustered mutations, frameshifts, and misplaced stop codons.<li>Comparison of consensus genome quality obtained from the GridION and the MiSeq and analyzed on <b>Nextclade</b>: To compare the quality of consensus genomes obtained from the GridION and the MiSeq, consensus genomes from both platforms were uploaded to <b>Nextclade</b> and the results plotted on a double bar graph.<li>Analysis of mutations in samples sequenced on the GridION and the MiSeq: Consensus genomes produced by Genome Detective were uploaded to <b>Nextclade</b> and the results were analyzed.<li> A consensus file for all runs, for each platform, was produced and uploaded to <b>Nextclade</b> and a Wilcoxon rank sum test was performed to compare the number and type of mutations detected by both platforms (Fig.<li> A total of 93 consensus genomes from both the GridION and the MiSeq were uploaded to <b>Nextclade</b> and the results were compared.<li>Comparison of the genome coverage and assigned clade for run116 samples on <b>Nextclade</b>.<li>Phylogenetic comparison between identical samples sequenced using both the GridION and MiSeq: A phylogenetic tree was created using IQTREE and visualized using FigTree for samples from Run116 sequenced on both the GridION and the MiSeq but classified in different clades by <b>Nextclade</b>.<li>Consensus genomes produced by both platforms were uploaded to <b>Nextclade</b> Online Tool v1.4.2 (2021-10-26) (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) for genome clade assignments, mutation calling, quality checks, and to determine the genome position on the SARS-CoV-2 phylogenetic tree.<li> <b>Nextclade</b> is built on Nextalign and consists of three tools; <b>Nextclade</b> Web, <b>Nextclade</b> CLI, and Nextalign CLI, which all share the common C++ library of algorithms.<li> <b>Nextclade</b> starts by performing a pairwise alignment of the query sequence to a reference sequence using Nextalign that uses a banded local alignment algorithm with affine gap-cost that are determined through seed matching.<li> <b>Nextclade</b> places each query sequence on the reference phylogenetic tree by comparing the mutations on the query sequence with the mutations of every node and tip in the reference tree, and finding the node which has the most similar set of mutations.<li>The non-parametric nature of the data influenced the use of a Wilcoxon test to compare the number of consensus genomes produced by the GridION and the MiSeq classified within each category of the online <b>Nextclade</b> sequence analysis tool.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35459226">35459226</a> <b>(1)</b> Human endogenous retrovirus K in the respiratory tract is associated with COVID-19 physiopathology<ul><li> Consensus fasta sequences were aligned with ClustalW in Unipro UGENE [<xref ref-type="bibr" rid="CR40"><u>40</u></xref>] (version 38), and phylogenies were constructed with <b>Nextclade</b> [<xref ref-type="bibr" rid="CR41"><u>41</u></xref>] to assign the emerging clades (Supplementary Table <xref rid="MOESM2" ref-type="media"><u>2</u></xref>).<li> [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22368248">22368248</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35538682">35538682</a> <b>(1)</b> Phylogenetic and genome-wide mutational analysis of SARS-CoV-2 strains circulating in Nigeria: no implications for attenuated COVID-19 outcomes<ul><li> The sequence IDs (GISAID, Pango, and <b>Nextclade</b>) of the 230 genomes, as well as the acknowledgement list of the sequence submitters of all 230 isolates, are shown in the supplementary data (<xref rid="SD1-j-phrp-2021-0329" ref-type="supplementary-material"><u>Tables S1</u></xref> and <xref rid="SD2-j-phrp-2021-0329" ref-type="supplementary-material"><u>S2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35469875">35469875</a> <b>(1)</b> Use of wastewater surveillance for early detection of Alpha and Epsilon SARS-CoV-2 variants of concern and estimation of overall COVID-19 infection burden<ul><li> Genome consensus sequences were assembled using Medaka via the ARTIC pipeline (<xref rid="bb0185" ref-type="bibr"><u>Lu et al., 2020</u></xref>), and mutations and deletions were identified using <b>NextClade</b>.<li> [Lu et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32359424">32359424</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632615">35632615</a> <b>(1)</b> SARS-CoV-2 Variants in Paraguay: Detection and Surveillance with an Economical and Scalable Molecular Protocol<ul><li> The <b>Nextclade</b> tool was used for clade definition and mutation calling [<xref rid="B29-viruses-14-00873" ref-type="bibr"><u>29</u></xref>] and lineages were assigned using the Pangolin web server [<xref rid="B30-viruses-14-00873" ref-type="bibr"><u>30</u></xref>].<li> [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35569431">35569431</a> <b>(3)</b> Presumptive positive with the Cepheid Xpert Xpress SARS-CoV-2 assay due to N mutations in the Delta variant<ul><li> Consensus FASTA files generated against reference SARS-Cov-2 sequence (NC_045512) were uploaded for variant analysis and detection of mutation at Pangolin lineage (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="interref0001">https://pangolin.cog-uk.io</ext-link>) and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="interref0003">https://clades.nextstrain.org</ext-link>).<li><b>NextClade</b></th><th valign="top">Genome coverage</th><th valign="top">Coverage depth</th></tr></thead><tbody><tr><td valign="top">PV2_93.<li> The samples were identified as Delta (B.1.617.2) and Delta sub lineage (AY.3 and AY.24) based on PANGO lineage <xref rid="bib0013" ref-type="bibr"><u>[13]</u></xref> and 21A based on <b>NextClade</b> (<xref rid="tbl0001" ref-type="table"><u>Table 1</u></xref>).<li> [[13]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35460989">35460989</a> <b>(2)</b> The displacement of the SARS-CoV-2 variant Delta with Omicron: An investigation of hospital admissions and upper respiratory viral loads<ul><li> 70%, Tables S1 and S2 detail the quality of the genomes, sequences with coverage &gt;90% and mean depth &gt;100: Delta 840/ 908 and Omicron 987/ 1,119).Query mutations were manually confirmed with Integrated Genomics viewer (IGV) (Version 2.8.10), clades were determined using <b>Nextclade</b> beta v 1.13.2 (clades.nextstrain.org, Last accessed March 30, 2022), and lineages were determined with Pangolin COVID-19 lineage Assigner.<li><b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35442444">35442444</a> <b>(2)</b> Recombination between SARS-CoV-2 Omicron BA.1 and BA.2 variants identified in a traveller from Nepal at the airport quarantine facility in Japan<ul><li> or &#x2018;none&#x2019;) in the PANGO lineage.<xref rid="ref2" ref-type="bibr"><sup>2</sup></xref> Detailed genome alignment using the <b>Nextclade</b> tool suggested that despite belonging to the 21 L (Omicron, BA.2) clade, the IC-5720 isolate has mutation profiles in the whole ORF1a gene identical to those in 21 K (Omicron, BA.1).<li> <b>Nextclade</b> alignment assigned the IC-5720 to isolate (highlighted in red) to the 21 L (Omicron) clade; however, the mutation profile in the <italic toggle="yes">ORF1a</italic> gene suggested a profile identical to that of BA.1.1 (EPI_ISL_11057296).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35632589">35632589</a> <b>(4)</b> Delta Variant of SARS-CoV-2 Replacement in Brazil: A National Epidemiologic Surveillance Program<ul><li>The consensus genomes were classified in Pango lineages using pangolin tool v3.1.17 [<xref rid="B24-viruses-14-00847" ref-type="bibr"><u>24</u></xref>] and <b>NextClade</b> web application v.1.7.0 [<xref rid="B25-viruses-14-00847" ref-type="bibr"><u>25</u></xref>].<li> The sequences, initially analyzed by Pangolin and <b>NextClade</b> tools, were classified as Gamma (P.1) (91.30%; 63/69), Alpha (B.1.1.7) (1.45%; 1/69), P.4 (1.45%; 1/69), and Delta (B.1.617.2) (5.79%; 4/69).<li> Sequencing metrics and the pangolin/<b>nextclade</b> classification are available in <xref rid="app1-viruses-14-00847" ref-type="app"><u>Supplementary Table S3</u></xref>.<li> We performed a maximum likelihood phylogeny to corroborate the data found in Pangolin and <b>Nextclade</b> classification.<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [25](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35422113">35422113</a> <b>(2)</b> Computational prediction of the molecular mechanism of statin group of drugs against SARS-CoV-2 pathogenesis<ul><li> Sequences were mapped against the SARS-CoV-2 reference sequence (Wuhan/WH01/2019) using the NextAlign aligner tool (<ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>">https://github.com/nextstrain/<b>nextclade</b></ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35412591">35412591</a> <b>(1)</b> Associations Between Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Variants and Risk of Coronavirus Disease 2019 (COVID-19) Hospitalization Among Confirmed Cases in Washington State: A Retrospective Cohort Study<sup><xref rid="ciac279-FM2" ref-type="author-notes">&#xA0;</xref></sup><ul><li> Sequence quality was determined using <b>Nextclade</b> version 1.0.1 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35421481">35421481</a> <b>(1)</b> Wastewater and marine bioindicators surveillance to anticipate COVID-19 prevalence and to explore SARS-CoV-2 diversity by next generation sequencing: One-year study<ul><li> Mutations were analyzed in comparison to the Wuhan reference using the <b>Nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="ir0030">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35396511">35396511</a> <b>(5)</b> Emergence and phenotypic characterization of the global SARS-CoV-2 C.1.2 lineage<ul><li> Samples sequenced from Oxford Nanopore GridION were assembled according to the ARTIC&#xA0;nCoV-2019 novel coronavirus bioinformatics protocol or using the Fastq QC&#x2009;+&#x2009;ARTIC&#x2009;+&#x2009;<b>NextClade</b> pipeline on Epi2Me (Oxford Nanopore Technologies).<li> Quality control reports were obtained from <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>.<li> Clade and lineage assignment was determined using <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup> and Pangolin<sup><xref ref-type="bibr" rid="CR29"><u>29</u></xref></sup>.<li> the corresponding <b>Nextclade</b> classification is 20D (<b>Nextclade</b> version v1.5.3, clades version 2021-07-28).<li> [27](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [27](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35389245">35389245</a> <b>(2)</b> The Alpha Variant (B.1.1.7) of SARS-CoV-2 Failed to Become Dominant in Mexico<ul><li>Genomic changes of Mexican B.1.1.7 and B.1.1.519 variants compared to the reference Wuhan-Hu-1 estimated by the <b>Nextclade</b> tool using sample collection date.<li>The <b>Nextclade</b> single nucleotide variant (SNV) calling system was used to identify synonymous/nonsynonymous substitutions in the Mexican sequences (<xref rid="B33" ref-type="bibr"><u>33</u></xref>), enabling us to determine the association of nucleotide SNPs to a particular haplotype or geographical group.<li> [33](<person-group person-group-type="author"><string-name><surname>Hodcroft</surname><given-names>EB</given-names></string-name>, <string-name><surname>Hadfield</surname><given-names>J</given-names></string-name>, <string-name><surname>Neher</surname><given-names>RA</given-names></string-name>, <string-name><surname>Bedford</surname><given-names>T</given-names></string-name></person-group>. <year>2020</year>. <article-title>Year-letter genetic clade naming for SARS-CoV-2 on Nextstrain.org</article-title>.<ext-link xlink:href="https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming" ext-link-type="uri">https://nextstrain.org/blog/2020-06-02-SARSCoV2-clade-naming</ext-link>. <date-in-citation content-type="access-date">Accessed September 4, 2021</date-in-citation>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35393464">35393464</a> <b>(1)</b> Host&#x2013;pathogen dynamics in longitudinal clinical specimens from patients with COVID-19<ul><li>Clade (<b>Nextclade</b>/Pangolin)</th><th align="left">Number of nt differences relative to first sample</th></tr></thead><tbody><tr><td align="left" rowspan="2">P001.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35385575">35385575</a> <b>(1)</b> High Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Household Transmission Rates Detected by Dense Saliva Sampling<ul><li>Other factors that may have affected SARs are the study period including vaccination status of the participants (the COVID-19 vaccination program had not started yet), quarantine policies, and dominating variants (<b>Nextclade</b> 20A, 20B, 20E [EU1]) associated with different transmission rates [<xref rid="CIT0036" ref-type="bibr"><u>36</u></xref>].<li> [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34448865">34448865</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35388091">35388091</a> <b>(2)</b> Phylogeography and genomic epidemiology of SARS-CoV-2 in Italy and Europe with newly characterized Italian genomes between February-June 2020<ul><li> The <b>Nextclade</b> classification showed a high prevalence of the clades 20A (n&#x2009;=&#x2009;207, 44.5%) and 20B (n&#x2009;=&#x2009;141, 30.3%), followed by 19A (n&#x2009;=&#x2009;84, 18.1%), and 20D (n&#x2009;=&#x2009;29, 6.2%).<li> 2.3.2 (last access 15 April 2021, <ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>) and <b>Nextclade</b> v.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35479614">35479614</a> <b>(2)</b> Outbreak of Severe Acute Respiratory Syndrome Coronavirus-2 B.1.620 Lineage in the General Hospital of Jeju Island, Republic of Korea<ul><li> SARS-CoV-2 B.1.620 lineage genomes from the outbreak cases of the hospital in Jeju Island were classified using <b>Nextclade</b><sup><xref rid="footnote2" ref-type="fn"><u>2</u></xref></sup>.<li> The <b>Nextclade</b> analysis revealed that SARS-CoV-2 genomes from outbreak cases of hospitals in Jeju belong to clade 20A (<xref rid="TS1" ref-type="supplementary-material"><u>Supplementary Table 1</u></xref>).<li> [2](<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2022.860535/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fmicb.2022.860535/full#supplementary-material</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35444655">35444655</a> <b>(5)</b> Genetic Surveillance of Five SARS-CoV-2 Clinical Samples in Henan Province Using Nanopore Sequencing<ul><li> Meanwhile, clade analysis was done using <b>Nextclade</b> tool, and the phylogenetic tree was performed and visualized with Prism GraphPad 9.<li> The lineage of genome was analyzed using the online website pangolin (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>), and <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>) was used for tracking the clade.<li> Furthermore, five genome sequences were analyzed using <b>Nextclade</b>.<li> The phylogenetic tree was generated using <b>Nextclade</b> and visualized using Prism GraphPad 9.<li> By default, all sequences divided into 21A and 21I subclades in the <b>Nextclade</b> tool were displayed.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35458482">35458482</a> <b>(1)</b> Viral Evolution and Immunology of SARS-CoV-2 in a Persistent Infection after Treatment with Rituximab<ul><li> Clade assignment and amino acid and nucleotide comparison to the reference genome were performed using <b>NextClade</b> v0.7.2, (Basel, Switzerland) [<xref rid="B20-viruses-14-00752" ref-type="bibr"><u>20</u></xref>].<li> [20](          <article-title>NextClade v0.7.2</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-12-12">(accessed on 12 December 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35433558">35433558</a> <b>(3)</b> Clinical Epidemiology of Pediatric COVID-19 Delta Variant Cases From North Sumatra, Indonesia<ul><li> FASTA files were analyzed by Pangolin version 3.1.5 and <b>Nextclade</b> 0.13.0 prior to submission to GISAID.<li> The SARS-CoV-2 phylogenetic tree was constructed with <b>Nextclade</b> v.1.5.4, and descriptive statistics were used to describe the results (<xref rid="B13" ref-type="bibr"><u>13</u></xref>).<li>Phylogenetic tree built with <b>Nextclade</b> v1.5.4.<li> [13](<person-group person-group-type="author"><name><surname>Nick Loman</surname><given-names>ZP</given-names></name><collab><italic>ARTIC Project</italic></collab></person-group>. <source>The ARTIC Field Bioinformatics Pipeline</source>. (<year>2021</year>). Available at: <ext-link xlink:href="https://github.com/quadram-institute-bioscience/fieldbioinformatics" ext-link-type="uri">https://github.com/quadram-institute-bioscience/fieldbioinformatics</ext-link> (accessed August 27, 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35433760">35433760</a> <b>(3)</b> Epidemiological Dynamics of SARS-CoV-2 Variants During Social Protests in Cali, Colombia<ul><li> The mutation search was performed by means of Clade assignment, mutation calling, and sequence quality checks <b>NextClade</b> v 1.5.4.<li> and 1,785 reference genomes representing the diversity of SARS-CoV-2 lineages available in the <b>NextClade</b> tool.<li>Single-nucleotide polymorphism (SNP) analysis was performed by comparing the 228 genomes from Cali downloaded from the GISAID database (ranged from 4 April 2021 to 31 July 2021) with the reference genome from Wuhan, China (hCoV-19/Wuhan/Hu-1/2019, GenBank accession number: <ext-link xlink:href="NC_045512.2" ext-link-type="DDBJ/EMBL/GenBank">NC_045512.2</ext-link>) using the <b>NextClade</b> tool v 1.5.4 (see text footnote 11) (<xref rid="B10" ref-type="bibr"><u>10</u></xref>), and the UGENE v.33.0 software (<xref rid="B17" ref-type="bibr"><u>17</u></xref>).<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [17](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22368248">22368248</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35367360">35367360</a> <b>(2)</b> Identification, propagation and molecular characterization of SARS-CoV-2 delta variant isolated from Egyptian COVID-19 patients<ul><li> <b>Nextclade</b> Tool (version 1.5.2) was also used for viral genome alignment, quality checks and clade assignment.<li>Further analysis was performed to detect the genetic mutation using <b>Nextclade</b> Tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir0010">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35378751">35378751</a> <b>(1)</b> SARS-CoV-2 and other respiratory pathogens are detected in continuous air samples from congregate settings<ul><li> Lineage/clade assignments were also confirmed using <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain" ext-link-type="uri">https://clades.nextstrain</ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35458441">35458441</a> <b>(1)</b> Comparative Analysis of SARS-CoV-2 Variants of Concern, Including Omicron, Highlights Their Common and Distinctive Amino Acid Substitution Patterns, Especially at the Spike ORF<ul><li> More specifically, for Delta, we calculated the ancestral sequence of clades 21A, 21I, and 21J separately (based on <b>Nextclade</b>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35358727">35358727</a> <b>(1)</b> Genomic profiles of vaccine breakthrough SARS-CoV-2 strains from Odisha, India<ul><li> Although two of the cases were classified as B.1, we found that both of them were classified as 21A (Delta) by <b>Nextclade</b>, having S:D614G and S:P681R mutations along with other lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35343761">35343761</a> <b>(1)</b> Genome Sequences of 408 SARS-CoV-2 Strains Obtained from Nasopharyngeal Swabs in La Araucan&#xED;a Region, Southern Chile<ul><li> Finally, the lineage and clade assignments were performed using the Pangolin tool (<xref rid="B7" ref-type="bibr"><u>7</u></xref>), considering the PANGO nomenclature system (<xref rid="B8" ref-type="bibr"><u>8</u></xref>) and viral clades by <b>Nextclade</b> (<xref rid="B9" ref-type="bibr"><u>9</u></xref>).<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [9](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Roemer</surname><given-names>C</given-names></string-name>, <string-name><surname>Hodcroft</surname><given-names>E</given-names></string-name>, <string-name><surname>Neher</surname><given-names>R</given-names></string-name></person-group>. <year>2021</year>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>J Open Source Softw</source><volume>6</volume>:<fpage>3773</fpage>. doi:<pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35458424">35458424</a> <b>(1)</b> Emergence of Two Distinct SARS-CoV-2 Gamma Variants and the Rapid Spread of P.1-like-II SARS-CoV-2 during the Second Wave of COVID-19 in Santa Catarina, Southern Brazil<ul><li> The assembled genomes were analyzed in the <b>Nextclade</b> [<xref rid="B23-viruses-14-00695" ref-type="bibr"><u>23</u></xref>] and Pangolin [<xref rid="B24-viruses-14-00695" ref-type="bibr"><u>24</u></xref>] platforms to classify the SARS-CoV-2 variants.<li> [23](          <article-title>Nextclade Web 1.10.0</article-title>          <year>2020</year>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-11-09">(accessed on 9 November 2021)</date-in-citation>        ) [24](          <article-title>Pangolin COVID-19 Lineage Assigner</article-title>          <year>2021</year>          <comment>Available online: <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-11-09">(accessed on 9 November 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35368908">35368908</a> <b>(2)</b> Phylogenetic-based inference reveals distinct transmission dynamics of SARS-CoV-2 lineages Gamma and P.2 in Brazil<ul><li><hr/></td></tr><tr><td>Viralflow v.0.0.6</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/dezordi/ViralFlow" id="intref0035">https://github.com/dezordi/ViralFlow</ext-link></td><td><xref rid="bib6" ref-type="bibr"><u>Dezordi et&#xA0;al., 2022</u></xref></td></tr><tr><td>Geneious v10.2.6</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com/" id="intref0040">https://www.geneious.com/</ext-link></td><td>N/A</td></tr><tr><td>Pangolin</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0045">https://github.com/cov-lineages/pangolin</ext-link></td><td><xref rid="bib26" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td></tr><tr><td><b>Nextclade</b>.<li> Mutational composition of these sequences was analyzed in <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0155">https://clades.nextstrain.org</ext-link>) (<xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>) and due to highly variable patterns of substitutions, we have retained in the dataset only those genomes with the full-set of P.2 and Gamma lineages defining synapomorphies, thus excluding from the analysis all Gamma-like sequences (<xref rid="mmc1" ref-type="supplementary-material"><u>Table&#xA0;S2</u></xref>.<li> [Dezordi et&#xA0;al., 2022](          <person-group person-group-type="author">            <name>              <surname>Dezordi</surname>              <given-names>F.Z.</given-names>            </name>            <name>              <surname>Neto</surname>              <given-names>A.M.D.S.</given-names>            </name>            <name>              <surname>Campos</surname>              <given-names>T.L.</given-names>            </name>            <name>              <surname>Jeronimo</surname>              <given-names>P.M.C.</given-names>            </name>            <name>              <surname>Aksenen</surname>              <given-names>C.F.</given-names>            </name>            <name>              <surname>Almeida</surname>              <given-names>S.P.</given-names>            </name>            <name>              <surname>Wallau</surname>              <given-names>G.L.</given-names>            </name>            <collab>Fiocruz Covid-Genomic Surveillance Network</collab>          </person-group>          <article-title>ViralFlow: a versatile automated workflow for SARS-CoV-2 genome assembly, lineage assignment, mutations and intrahost variant detection</article-title>          <source>Viruses</source>          <volume>14</volume>          <year>2022</year>          <pub-id pub-id-type="doi">10.3390/v14020217</pub-id>        ) [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35243660">35243660</a> <b>(2)</b> First cases of infection with the 21L/BA.2 Omicron variant in Marseille, France<ul><li>Nucleotide and amino acid changes in viral genomes relative to the Wuhan&#x2010;Hu&#x2010;1 isolate genome were obtained using the <b>Nextclade</b> tool (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<li>Nucleotide and amino acid changes in viral genomes relative to the Wuhan&#x2010;Hu&#x2010;1 isolate genome were obtained using the <b>Nextclade</b> tool (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<xref rid="jmv27695-bib-0010" ref-type="bibr"><sup>10</sup></xref>, <xref rid="jmv27695-bib-0011" ref-type="bibr"><sup>11</sup></xref> Nextstrain clades and Pangolin lineages were determined using the <b>Nextclade</b> web application (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>)<xref rid="jmv27695-bib-0010" ref-type="bibr"><sup>10</sup></xref>, <xref rid="jmv27695-bib-0011" ref-type="bibr"><sup>11</sup></xref> and Pangolin web application (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>),<xref rid="jmv27695-bib-0012" ref-type="bibr"><sup>12</sup></xref> respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35194615">35194615</a> <b>(4)</b> SARS-CoV-2 variants of concern Alpha and Delta show increased viral load in saliva<ul><li> Some of the samples in this analysis exceeded this threshold, which prevented database upload, but all had sufficient information to confidently assign clade by Pangolin and <b>Nextclade</b> [<xref rid="R23" ref-type="bibr"><u>23</u></xref>&#x2013;<xref rid="R24" ref-type="bibr"><u>24</u></xref>].<li> To maintain phylogenetic independence, we only compare Ct values for variants at branch tips within the <b>NextClade</b> phylogeny [<xref rid="R24" ref-type="bibr"><u>24</u></xref>].<li> Therefore, the four <b>Nextclade</b>s we compare do not have parent-offspring relation.<li> We did notice that samples within the higher Ct range (28&#x2013;33) had more regions with ambiguous nucleotides and were therefore less likely to have received a clade assignment via <b>Nextclade</b>.<li> [23](<comment><ext-link xlink:href="https://github.com/CUGBF/SARS-CoV-2_Ct-vs-Clade" ext-link-type="uri">https://github.com/CUGBF/SARS-CoV-2_Ct-vs-Clade</ext-link></comment>) [24](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <name><surname>Roemer</surname><given-names>C</given-names></name>, <name><surname>Hodcroft</surname><given-names>EB</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source>, <year>2021</year>; <volume>6</volume>(<issue>67</issue>), <fpage>3773</fpage>.) [24](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <name><surname>Roemer</surname><given-names>C</given-names></name>, <name><surname>Hodcroft</surname><given-names>EB</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source>, <year>2021</year>; <volume>6</volume>(<issue>67</issue>), <fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35323016">35323016</a> <b>(1)</b> Genome Sequences of 25 SARS-CoV-2 Sublineage B.1.1.529 Omicron Strains in Bangladesh<ul><li> Processed reads were assembled using the artic gupplyplex script with Medaka v1.4 using the ARTIC EPI2ME v3.3.0 SARS-CoV-2 pipeline (FastQC plus ARTIC plus <b>NextClade</b>) (<ext-link xlink:href="https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html" ext-link-type="uri">https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35458389">35458389</a> <b>(1)</b> Genomic Epidemiology of SARS-CoV-2 in Tocantins State and the Diffusion of P.1.7 and AY.99.2 Lineages in Brazil<ul><li>All sequences were analyzed with <b>Nextclade</b> v1.7.2 [<xref rid="B17-viruses-14-00659" ref-type="bibr"><u>17</u></xref>] (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 12 January 2022) to determine the clade and the number of gap regions.<li> [17](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35364015">35364015</a> <b>(6)</b> Rapid and parallel adaptive mutations in spike S1 drive clade success in SARS-CoV-2<ul><li><hr/></td></tr><tr><td>Augur, version 12.0.0</td><td><xref rid="bib12" ref-type="bibr"><u>Huddleston et&#xA0;al., 2021</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/augur" id="intref0030">https://github.com/nextstrain/augur</ext-link></td></tr><tr><td>TreeTime, version 0.8.2</td><td><xref rid="bib27" ref-type="bibr"><u>Sagulenko et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/neherlab/treetime" id="intref0035">https://github.com/neherlab/treetime</ext-link></td></tr><tr><td>IQ-TREE, version 2.1.2</td><td><xref rid="bib24" ref-type="bibr"><u>Nguyen et&#xA0;al., 2015</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/Cibiv/IQ-TREE" id="intref0040">https://github.com/Cibiv/IQ-TREE</ext-link></td></tr><tr><td><b>Nextclade</b>.<li> These genomes were aligned with Nextalign (<ext-link ext-link-type="uri" xlink:href="https://docs.nextstrain.org/projects/<b>nextclade</b>/en/latest/user/nextalign-cli.html" id="intref0070">https://docs.nextstrain.org/projects/<b>nextclade</b>/en/latest/user/nextalign-cli.html</ext-link>) and masked to minimize error in phylogenetic inference associated with problematic amplicon sites.<li> shorter than 27,000&#xA0;bp of A, C, T, or G bases, missing complete date information, annotated with a date prior to October 2019, flagged with more than 20 mutations above the expected number based on the mutational clock rate, or flagged by <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://docs.nextstrain.org/projects/<b>nextclade</b>/en/latest/user/algorithm/07-quality-control.html" id="intref0075">https://docs.nextstrain.org/projects/<b>nextclade</b>/en/latest/user/algorithm/07-quality-control.html</ext-link>; <xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>) with one or more clusters of 6 or more private differences in a 100-nucleotide window.<li> [Huddleston et&#xA0;al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [Sagulenko et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) [Nguyen et&#xA0;al., 2015](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35233783">35233783</a> <b>(1)</b> Active surveillance and early detection of community transmission of SARS&#x2010;CoV&#x2010;2 Mu variant (B.1.621) in the Brazilian Amazon<ul><li> Samples with cycle threshold (Ct) values &lt;30 for <italic toggle="yes">N</italic> gene target were transferred in dry ice to Fiocruz, Amazonas, where nucleotide sequencing was performed using Illumina MiSeq or NextSeq platform and the COVIDSEQ Kit (Illumina).<xref rid="jmv27686-bib-0011" ref-type="bibr"><sup>11</sup></xref> Genomic sequences were analyzed using <b>Nextclade</b> software v.1.5.4 (<ext-link xlink:href="http://clades.nettrain.org" ext-link-type="uri">http://clades.nettrain.org</ext-link>) and Pangolin COVID&#x2010;19 Linea Assigner v.2.1.7 (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35350194">35350194</a> <b>(2)</b> Regional importation and asymmetric within-country spread of SARS-CoV-2 variants of concern in the Netherlands<ul><li> Of note, four such VOCs arose into global prominence in late 2020, including Alpha (<b>Nextclade</b> 20I; PANGO lineage B.1.1.7), Beta (20H; B.1.351), Gamma (20J; P.1) and Delta (21J; B.1.617.2), causing substantial levels of transmission worldwide, with Alpha and Delta being the most common variants globally in 2021<sup><xref rid="R5" ref-type="bibr"><u>5</u></xref></sup>.<li>Using <b>NextClade</b> lineage assignment<sup><xref rid="R17" ref-type="bibr"><u>17</u></xref></sup>, the viruses sampled at the start of the study period were largely genotyped as clade 20A and its daughter lineages, 20B and 20E (EU1) (<xref rid="F1" ref-type="fig"><u>Figure 1B</u></xref>&#x2013;<xref rid="F1" ref-type="fig"><u>C</u></xref>).<li> [5](<name><surname>Campbell</surname><given-names>F</given-names></name>, <name><surname>Archer</surname><given-names>B</given-names></name>, <name><surname>Laurenson-Schafer</surname><given-names>H</given-names></name>, <etal/><article-title>Increased transmissibility and global spread of SARS-CoV-2 variants of concern as at June 2021</article-title>. <source>Eurosurveillance</source><year>2021</year>; <volume>26</volume>: <fpage>1</fpage>&#x2013;<lpage>6</lpage>.) [17](<name><surname>Aksamentov</surname><given-names>I</given-names></name>, <name><surname>Roemer</surname><given-names>C</given-names></name>, <name><surname>Hodcroft</surname><given-names>EB</given-names></name>, <name><surname>Neher</surname><given-names>RA</given-names></name>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source><year>2021</year>; <volume>6</volume>: <fpage>3773</fpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35458384">35458384</a> <b>(1)</b> Analytical Sensitivity of Six SARS-CoV-2 Rapid Antigen Tests for Omicron versus Delta Variant<ul><li> Following WGS, the sequences were aligned with complete SARS-CoV-2 genomes of different lineages through the <b>NextClade</b> online tool [<xref rid="B10-viruses-14-00654" ref-type="bibr"><u>10</u></xref>].<li> [10](          <article-title>Nextclade v1.14.0, Clade Assignment, Mutation Calling, and Sequence Quality Checks</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-02-15">(accessed on 15 February 2022)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35387187">35387187</a> <b>(1)</b> Description of Symptoms Caused by the Infection of the SARS-CoV-2 B.1.621 (Mu) Variant in Patients With Complete CoronaVac Vaccination Scheme: First Case Report From Santiago of Chile<ul><li> Representative MiSeq and NextSeq 2000 sequencing confirmed the presence of the Mu variant of SARS-CoV-2, being classified on the Nextstrain Clade 21H (<b>NextClade</b> software version 1.9, Biozentrum, University of Basel, Swiss) (<xref rid="F1" ref-type="fig"><u>Figure 1E</u></xref>) and following the World Health Organization (WHO) recommendations (<xref rid="B16" ref-type="bibr"><u>16</u></xref>).<li> [16](<person-group person-group-type="author"><collab>World Health Organization. Tracking SARS-CoV-2 Variants.</collab></person-group> (<year>2021</year>). Available online at: <ext-link xlink:href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants" ext-link-type="uri">https://www.who.int/en/activities/tracking-SARS-CoV-2-variants</ext-link> (accessed October 18, 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35335115">35335115</a> <b>(3)</b> A Rapid and Consistent Method to Identify Four SARS-CoV-2 Variants during the First Half of 2021 by RT-PCR<ul><li> Mutations of external datasets were defined by uploading the FASTA sequences to the <b>NextClade</b> website (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 1 January 2022).<li> The FASTA file sequences were submitted to Pangolin tools for variant calling and to the <b>NextClade</b> website for mutation calling.<li> Afterward, the 162 FASTA file sequences were submitted to the <b>NextClade</b> website for SNP calling, and all data were compared with our designed genotyping panel.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35371561">35371561</a> <b>(3)</b> The early SARS-CoV-2 epidemic in Senegal was driven by the local emergence of B.1.416 and the introduction of B.1.1.420 from Europe<ul><li> First, the sequences were run through a local sandboxed Docker deployment of <b>NextClade</b> (<xref rid="R29" ref-type="bibr"><u>Orf GS 2021</u></xref>); those with &#x2018;bad&#x2019; QC scores (using default scoring metrics in <b>NextClade</b>) or total mutations &gt;40 were discarded.<li> Sequence headers were renamed with relevant metadata information, including accession, country, PANGOLIN lineage, <b>NextClade</b> classification, and date of collection.<li> [Orf GS 2021](<person-group person-group-type="author"><string-name><surname>Orf</surname> &#xA0;<given-names>G. S.</given-names></string-name></person-group> &#xA0;<etal>et&#xA0;al.</etal> (<year>2021</year>) &#x2018;<article-title>SNP and Phylogenetic Characterization of Low Viral Load SARS-CoV-2 Specimens by Target Enrichment</article-title>&#x2019;, <source><italic toggle="yes">Front Virology</italic></source>, <volume>1</volume>: <page-range>765974</page-range>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35300748">35300748</a> <b>(1)</b> COVID-19 cases in spectators returning to Finland from UEFA Euro 2020 matches in Saint Petersburg<ul><li> Adapter, low quality (quality score &lt;30) and short (&lt;50&#xA0;nt) sequences were removed using Trimmomatic [<xref rid="ref2" ref-type="bibr"><u>2</u></xref>], followed by assembly using BWA-MEM [<xref rid="ref3" ref-type="bibr"><u>3</u></xref>], variant calling using LoFreq [<xref rid="ref4" ref-type="bibr"><u>4</u></xref>] and consensus calling using SAMtools [<xref rid="ref5" ref-type="bibr"><u>5</u></xref>] implemented in HaVoC pipeline [<xref rid="ref6" ref-type="bibr"><u>6</u></xref>], followed by lineage annotation with <b>Nextclade</b> (clades.nextstrain.org) and Pangolin (<uri xlink:href="https://cov-lineages.org/resources/pangolin.html">https://cov-lineages.org/resources/pangolin.html</uri><underline>)</underline>.<li> [2](<string-name><surname>Bolger</surname><given-names>AM</given-names></string-name>, <string-name><surname>Lohse</surname><given-names>M</given-names></string-name> and <string-name><surname>Usadel</surname><given-names>B</given-names></string-name> (<year>2014</year>) <article-title>Trimmomatic: a flexible trimmer for Illumina sequence data</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>30</volume>, <fpage>2114</fpage>&#x2013;<lpage>2120</lpage>.) [3](<string-name><surname>Li</surname><given-names>H</given-names></string-name> (<year>2013</year>) Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv:1303.3997v2.) [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23066108">23066108</a>) [5](<string-name><surname>Li</surname><given-names>H</given-names></string-name><etal/> (<year>2009</year>) <article-title>The sequence alignment/map format and SAMtools</article-title>. <source>Bioinformatics (Oxford, England)</source><volume>25</volume>, <fpage>2078</fpage>&#x2013;<lpage>2079</lpage>.) [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34273961">34273961</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35328676">35328676</a> <b>(2)</b> High-Integrity Sequencing of Spike Gene for SARS-CoV-2 Variant Determination<ul><li> Furthermore, such segment-missing amplicons resulted in mutation miscalling using the ARTIC protocol followed by the <b>Nextclade</b> (as shown in <xref rid="app1-ijms-23-03257" ref-type="app"><u>Supplementary Materials</u></xref>).<li> A file containing consensus sequences of spike genes was uploaded to <b>Nextclade</b> (v0.14.2) for clade assignment and variant calling [<xref rid="B42-ijms-23-03257" ref-type="bibr"><u>42</u></xref>].<li> [42](          <article-title>Nextclade v0.14.2</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-04-30">(accessed on 30 April 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33594384">33594384</a> <b>(1)</b> Genomic surveillance of SARS-CoV-2 during the first year of the pandemic in the Bronx enabled clinical and epidemiological inference<ul><li> We called variants with the <b>NextClade</b> tool and annotated lineages using the October 13th, 2021 update of PANGOLIN version 3.1.14 (<xref rid="R6" ref-type="bibr"><u>6</u></xref>, <xref rid="R7" ref-type="bibr"><u>7</u></xref>).<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35297757">35297757</a> <b>(2)</b> Unusual SARS-CoV-2 intrahost diversity reveals lineage superinfection<ul><li> The consensus genomes were automatically analysed with PangoLineage v1.1.23 with pangoLEARN updated at 28 May 2021 [<xref rid="R30" ref-type="bibr"><u>30</u></xref>] and to <b>Nextclade</b> [<xref rid="R31" ref-type="bibr"><u>31</u></xref>] tools.<li> The excess or lack of mutations are then compared with the mutations annotated with <b>NextClade</b> using an <italic toggle="yes">in house</italic> R script (<ext-link xlink:href="https://github.com/dezordi/SARS-CoV-2_tools/" ext-link-type="uri">https://github.com/dezordi/SARS-CoV-2_tools/</ext-link> blob/master/compare_mutation.R).<li> [30](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) [31](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>          <article-title>Nextclade</article-title>          <year>2021</year>          <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35337031">35337031</a> <b>(7)</b> Molecular Epidemiology of SARS-CoV-2 in Tunisia (North Africa) through Several Successive Waves of COVID-19<ul><li>The Fasta format of whole genome sequences was used for clade and lineage assignment using online tools: <b>Nextclade</b> [<xref rid="B10-viruses-14-00624" ref-type="bibr"><u>10</u></xref>] and Pangolin (version 3.1.16, lineages version 2021-11-25) [<xref rid="B25-viruses-14-00624" ref-type="bibr"><u>25</u></xref>].<li> A <b>Nextclade</b> tree was also constructed using the online tool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 10 March 2022)).<li> According to the <b>Nextclade</b> classification [<xref rid="B10-viruses-14-00624" ref-type="bibr"><u>10</u></xref>], 14 different clades were detected in Tunisia: 19A, 19B, 20A-E, 20G-I, 21A, 21D, 21I and 21J.<li> <xref rid="viruses-14-00624-f001" ref-type="fig"><u>Figure 1</u></xref> shows the phylogenetic tree built using the <b>Nextclade</b> online tool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 10 March 2022)).<li> <xref rid="viruses-14-00624-f002" ref-type="fig"><u>Figure 2</u></xref> shows the phylogenetic tree based on Pangolin lineage classification and <xref rid="viruses-14-00624-t001" ref-type="table"><u>Table 1</u></xref> illustrates the correspondence between the WHO, Pangolin lineage and <b>Nextclade</b> assignments according to the months of sample collection.<li> B.1.398, B.1.214, B.520 and B.4, corresponding to the 19A and 20A <b>Nextclade</b> classifications.<li> it was characterized by a higher genetic diversity with the circulation of at least 20 different lineages, namely: B.1.1.50, B.1.597, B.1.1.1, B.1.22, B.1.428.2, B.1.1.25, B.1.1.198, B.1.1.189, B.1.1.354, B.1.177, B.1.160, B.1.111, B.1.389, B.1.575, B.1.473, B.1.595, B.1.623, B.1.236, B.1.535, B.1.533 and B.1.416 (20A to 20E according to the <b>Nextclade</b> classification) (<xref rid="viruses-14-00624-t001" ref-type="table"><u>Table 1</u></xref>).<li> [10](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [25](          <article-title>Pangolin COVID-19 Lineage Assigner</article-title>          <comment>Available online: <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2022-03-10">(accessed on 10 March 2022)</date-in-citation>        ) [10](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35304093">35304093</a> <b>(1)</b> Tracking SARS-CoV-2 variants by entire S-gene analysis using long-range RT-PCR and Sanger sequencing<ul><li>The amino acid mutations and types of VOC were determined by <b>Nextclade</b> (v1.12.0, <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="ir010">https://clades.nextstrain.org</ext-link>) <xref rid="b0055" ref-type="bibr"><u>[11]</u></xref>.<li> [[11]](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35294336">35294336</a> <b>(8)</b> Targeted Sanger sequencing to recover key mutations in SARS-CoV-2 variant genome assemblies produced by next-generation sequencing<ul><li> The resulting consensus sequences were analysed in <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and the mutational sets were compared.<li> <b>Nextclade</b> produces a report detailing sequence quality and coverage and identifies mutational changes observed at the nucleotide and protein levels relative to the Wuhan reference.<li>To assess the merging step, we generated a <b>Nextclade</b> report of the whole-genome sequence before merging to determine the detectable mutations present at baseline.<li> Next, we generated a <b>Nextclade</b> report of the Sanger sequences to identify the mutations recovered within these.<li> Finally, we generated a <b>Nextclade</b> report of the mutations present in the consensus genomes after merging.<li>The <b>Nextclade</b> report of the Sanger sequences enabled us to identify instances where merging the Sanger and NGS data yielded a composite genome sequence assembly within which the expected mutation profiles were and were not successfully detected, while the report after merging was used to confirm a successful merge.<li> The success of the merge process was confirmed using <b>Nextclade</b> reports.<li> Table S3 shows a <b>Nextclade</b> report for all the samples submitted to GISAID.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35328562">35328562</a> <b>(1)</b> An Upgrade on the Surveillance System of SARS-CoV-2: Deployment of New Methods for Genetic Inspection<ul><li>The consensus sequence was then uploaded in <b>Nextclade</b> (<uri>clades.nextstrain.org</uri>, last accession 14 December 2021) and the mutation list was analyzed against the results obtained from the other tools.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35243662">35243662</a> <b>(1)</b> Hotspots for SARS&#x2010;CoV&#x2010;2 Omicron variant spread: Lessons from New York City<ul><li> All Omicron sequences clustered in a larger, well&#x2010;defined group from the <b>NextClade</b> reference genomes (representative of all SARS&#x2010;CoV&#x2010;2 lineages) (Figure&#xA0;<xref rid="jmv27691-fig-0002" ref-type="fig"><u>2A</u></xref>) and its corresponding sublineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35229308">35229308</a> <b>(1)</b> SARS&#x2010;COV&#x2010;2 genomic monitoring in the state of S&#xE3;o Paulo unveils two emerging AY.43 sublineages<ul><li> We assessed the consensus genome sequence quality using <b>Nextclade</b> v0.8.1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35336987">35336987</a> <b>(2)</b> On the Origin and Propagation of the COVID-19 Outbreak in the Italian Province of Trento, a Tourist Region of Northern Italy<ul><li> Due to the low number of mutations present in this dataset (306 mutations), we ran <b>Nextclade</b> v0.14.2 [<xref rid="B3-viruses-14-00580" ref-type="bibr"><u>3</u></xref>] to filter sequences for their quality, keeping only sequences with a good overall quality, according to the output of the program.<li> This discrepancy is likely related to the different taxon sampling and to the further quality control steps that we carried out for the second dataset, in which we only included high quality sequences according to the <b>Nextclade</b> output (see <xref rid="sec2-viruses-14-00580" ref-type="sec"><u>Section 2</u></xref>).<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35272746">35272746</a> <b>(2)</b> Molecular epidemiology of the SARS-CoV-2 variant Omicron BA.2 sub-lineage in Denmark, 29 November 2021 to 2 January 2022<ul><li>Lineage-specific mutations were derived from analysis of WGS consensus genomes with <b>Nextclade</b> CLI 1.9.0 [<xref rid="r19" ref-type="bibr"><u>19</u></xref>] relative to the Wuhan-Hu-1 reference genome and filtered for substitutions and deletions which comprise&#x2009;at least&#x2009;50% of one lineage and&#x2009;less than&#x2009;5% of the other.<li> Mutations were derived from consensus genomes by use of <b>Nextclade</b> CLI (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) [<xref rid="r19" ref-type="bibr"><u>19</u></xref>].<li> [19](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source>J Open Source Softw</source>. <year>2021</year>;<volume>6</volume>(<issue>67</issue>):<fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) [19](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>E</given-names></name><name><surname>Neher</surname><given-names>R</given-names></name></person-group>. <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source>J Open Source Softw</source>. <year>2021</year>;<volume>6</volume>(<issue>67</issue>):<fpage>3773</fpage>. <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35262410">35262410</a> <b>(1)</b> Vaccine-Induced Antibody Responses against SARS-CoV-2 Variants-Of-Concern Six Months after the BNT162b2 COVID-19 mRNA Vaccination<ul><li> Sequence reads were mapped to the reference genome with BWA-MEM, and amino acid changes were identified with <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35104245">35104245</a> <b>(1)</b> Adaptive immune responses in vaccinated patients with symptomatic SARS-CoV-2 Alpha infection<ul><li> <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) was used for clade assignment, sequence quality check, and phylogenetic tree construction.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35336952">35336952</a> <b>(1)</b> Early Genomic, Epidemiological, and Clinical Description of the SARS-CoV-2 Omicron Variant in Mexico City<ul><li> The <b>Nextclade</b> tool was used to determine amino acid substitutions in the viral genome and generate histogram plots of mutation frequency and phylogenies highlighting mutation occurrence [<xref rid="B11-viruses-14-00545" ref-type="bibr"><u>11</u></xref>].<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33570181">33570181</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35308526">35308526</a> <b>(2)</b> Case Report: SARS-CoV-2 Gamma Isolation From Placenta of a Miscarriage in Midwest, Brazil<ul><li> The nucleotide and amino acid features were observed by the online tools <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and CoVSurver (<ext-link xlink:href="https://mendel.bii.a-star.edu.sg/METHODS/corona/beta/" ext-link-type="uri">https://mendel.bii.a-star.edu.sg/METHODS/corona/beta/</ext-link>).<li> Additionally, an assessment of the molecular composition of both sequences by the <b>NextClade</b> algorithm revealed that as they bare all synapomorphic signatures of the VOC Gamma, two additional amino-acid substitutions are shared, both in ORF1a, P885L and T2121I.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35257682">35257682</a> <b>(2)</b> Fast detection of SARS-CoV-2 variants including Omicron using one-step RT-PCR and Sanger sequencing<ul><li> The resulting FASTA-files were analyzed with <b>NextClade</b> to identify signature mutations in the spike gene (<xref rid="bib7" ref-type="bibr"><u>Hodcroft, 2021</u></xref>).<li> Because the Omicron variant has a lot of mutations in the S gene, the amplicon sequence allows unambiguous identification using <b>NextClade</b>.<li> [Hodcroft, 2021](Hodcroft , &#x201C;Nextclade,&#x201D; 2021. https://clades.nextstrain.org/ (Accessed 15 June 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35308391">35308391</a> <b>(1)</b> Increase in Viral Load in Patients With SARS-CoV-2 Delta Variant Infection in the Republic of Korea<ul><li> The lineages and clades of the SARS-CoV-2 sequences were assigned using <b>Nextclade</b> v1.7.1 (<xref rid="B18" ref-type="bibr"><u>Nextstrain Project</u></xref>) and PANGOLIN (<xref rid="B23" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>).<li> [Nextstrain Project](<collab>Nextstrain Project</collab> Available online at: <ext-link xlink:href="https://nextstrain.org/ncov/gisaid/global" ext-link-type="uri">https://nextstrain.org/ncov/gisaid/global</ext-link><comment>(accessed October 26, 2021)</comment>.) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35309205">35309205</a> <b>(1)</b> Genomic Snapshot of SARS-CoV-2 in Migrants Entering Through Mediterranean Sea Routes<ul><li> Viral clade and lineage have been inferred using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and Pangolin (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>) assigners, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35262089">35262089</a> <b>(1)</b> Duration of viable virus shedding in SARS-CoV-2 omicron variant infection<ul><li> Amino acid variants were identified at the codon level with perl code and the resulting variant file was used to determine SARS-CoV-2 variant type using <b>Nextclade</b> version 1.13.1 (<ext-link xlink:href="10.21105/joss.03773" ext-link-type="doi">https://doi.org/10.21105/joss.03773</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35262086">35262086</a> <b>(2)</b> Imported SARS-COV-2 Variants of Concern Drove Spread of Infections Across Kenya During the Second Year of the Pandemic<ul><li> Further downstream, consensus sequences were used by Pangolin USHER for lineage assignments based on parsimony and <b>Nextclade</b> [<xref rid="R25" ref-type="bibr"><u>25</u></xref>] for clade specification.<li> To compare with global sequences, consensus sequences were aligned using NextAlign, embedded in <b>Nextclade</b>, and resulting multiple sequence alignment (MSA) fed into IQ-TREE [<xref rid="R26" ref-type="bibr"><u>26</u></xref>] for inferring maximum likelihood to determine the most likely phylogram describing the combined dataset.<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35231666">35231666</a> <b>(4)</b> Covidex: An ultrafast and accurate tool for SARS-CoV-2 subtyping<ul><li> These sequences were classified with Pangolin and <b>NextClade</b> (<xref rid="bb0040" ref-type="bibr"><b>Nextclade</b>, 2021</xref>; <xref rid="bb0045" ref-type="bibr"><u>Pangolin, 2021</u></xref>).<li>Additionally, the time performance of Covidex was compared with two other subtyping tools available for SARS-CoV-2 lineage classification: Pangolin 2.0 and <b>NextClade</b>.<li> In turn, Pangolin 2.0 and <b>NextClade</b> took 21&#xA0;s and 52&#xA0;s, respectively.<li> [Pangolin, 2021](          <person-group person-group-type="author">            <name>              <surname>Pangolin</surname>            </name>          </person-group>          <article-title>Cov-lineages</article-title>          <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/resources/pangolin.html" id="ir0045">https://cov-lineages.org/resources/pangolin.html</ext-link>          <year>2021</year>          <comment>(accessed 12.16.21)</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35213571">35213571</a> <b>(4)</b> Whole genome sequence analysis showing unique SARS-CoV-2 lineages of B.1.524 and AU.2 in Malaysia<ul><li>Mutation analyses were carried out using <b>Nextclade</b> v.1.5.2, a web-based analysis server (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) by comparing against a wild-type of Wuhan-Hu-1 (NC_045512.2).<li>Total 419 complete genomes of B.1.524 and 311 complete genomes of AU.2 were uploaded to <b>Nextclade</b> v.1.5.2 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) to analyse dominant mutations occur in spike protein of Malaysian lineages.<li>Using <b>Nextclade</b> v.1.5.2 for clade assignment, mutation calling and sequence quality checks (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org</ext-link>), 1356 complete genomes of the D614G variant were analysed for sequence quality and mutations.<li> Of that, 986 complete genomes passed <b>Nextclade</b>&#x2019;s sequence quality control highlighting different mutations in the spike protein.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35334985">35334985</a> <b>(1)</b> Humoral and Cellular Immune Responses of Solid Organ Transplant Patients on Belatacept to Three Doses of mRNA-Based Anti-SARS-CoV-2 Vaccine<ul><li>Neutralizing antibody titers were determined using a live virus neutralization assay and a clinical SARS-CoV-2 strain (GISAID: EPI_ISL_804378, GISAID Clade:GH, Pango lineage: B.1.160, <b>Nextclade</b>: 20A.EU2) infecting Vero cells (ATCC, CCL-81<sup>TM</sup>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35281046">35281046</a> <b>(1)</b> Case Report: Evolution of Humoral and Cellular Immunity in Two COVID-19 Breakthrough Infections After BNT162b2 Vaccine<ul><li> Lineages and clades were interpreted using Pangolin and <b>NextClade</b>, before being submitted to the GISAID database.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35196783">35196783</a> <b>(1)</b> Spread of Gamma (P.1) Sub-Lineages Carrying Spike Mutations Close to the Furin Cleavage Site and Deletions in the N-Terminal Domain Drives Ongoing Transmission of SARS-CoV-2 in Amazonas, Brazil<ul><li> Subsequently, we evaluated the consensus files for quality using <b>Nextclade</b> tool v1.5.2 (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>), those with more than 1% of ambiguities &#x201C;Ns&#x201D; had the FASTQ files imported into Geneious Prime 2021 for trimming and assembling using a customized workflow employing BBDuk and BBMap tools (v38.84) and the NC_045512.2 RefSeq as a template with carefully visually inspection.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35194675">35194675</a> <b>(2)</b> Misidentification of the SARS-CoV-2 Mu variant using commercial mutation screening assays<ul><li> The resulting sequences were then run on <b>NextClade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) and Pangolin COVID-19 Lineage Assigner (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>) for final classification.<li> All three strains were classified as clade 21H in <b>NextClade</b> and as B.1.621 in Pango, which is compatible with the Mu variant and not with the Beta variant.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35224469">35224469</a> <b>(3)</b> Assessment of the Abbott BinaxNOW SARS-CoV-2 rapid antigen test against viral variants of concern<ul><li><hr/></td></tr><tr><td>Rosalind Diagnostic Monitoring (DxM) system</td><td>RadX</td><td><ext-link ext-link-type="uri" xlink:href="https://radx.rosalind.bio/" id="intref0020">https://radx.rosalind.bio</ext-link></td></tr><tr><td>GraphPad Prism</td><td>GraphPad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.graphpad.com/scientific-software/prism/" id="intref0025">https://www.graphpad.com/scientific-software/prism/</ext-link></td></tr><tr><td><b>NextClade</b>.<li><hr/></td></tr><tr><td>Rosalind Diagnostic Monitoring (DxM) system</td><td>RadX</td><td><ext-link ext-link-type="uri" xlink:href="https://radx.rosalind.bio/" id="intref0020">https://radx.rosalind.bio</ext-link></td></tr><tr><td>GraphPad Prism</td><td>GraphPad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.graphpad.com/scientific-software/prism/" id="intref0025">https://www.graphpad.com/scientific-software/prism/</ext-link></td></tr><tr><td><b>NextClade</b></td><td><b>NextClade</b>.<li>Heat-inactivated RCS (provided by Helix OpCo, LLC) from individuals positive for SARS-CoV-2 (N&#xA0;= 8) known to be infected with the Mu variant were selected based on genomic sequence quality (as determined by <b>NextClade</b>) and cycle threshold (Ct) value of N protein.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35178586">35178586</a> <b>(3)</b> Emergence in southern France of a new SARS-CoV-2 variant harbouring both N501Y and E484K substitutions in the spike protein<ul><li> SARS-CoV-2 genomes were classified into <b>Nextclade</b> and Pangolin lineages using web applications (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>; <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</ext-link>) [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>&#x2013;<xref ref-type="bibr" rid="CR11"><u>11</u></xref>].<li> This figure is adapted from screenshots of the <b>nextclade</b> web application (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>) [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>, <xref ref-type="bibr" rid="CR10"><u>10</u></xref>].<li><b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>)&#xA0;identified a 20A lineage.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](          <name>            <surname>Aksamentov</surname>            <given-names>I</given-names>          </name>          <name>            <surname>Roemer</surname>            <given-names>C</given-names>          </name>          <name>            <surname>Hodcroft</surname>            <given-names>EB</given-names>          </name>          <name>            <surname>Neher</surname>            <given-names>RA</given-names>          </name>          <year>2021</year>          <data-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</data-title>          <source>Zenodo</source>          <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35208920">35208920</a> <b>(7)</b> Genomic Diversity of SARS-CoV-2 in Algeria and North African Countries: What We Know So Far and What We Expect?<ul><li> The viral clades were assigned by the <b>Nextclade</b> tool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> accessed on 20 December 2021) [<xref rid="B20-microorganisms-10-00467" ref-type="bibr"><u>20</u></xref>] and through the UShER web interface from the University of California, Santa Cruz (<uri xlink:href="https://genome.ucsc.edu/cgi-bin/hgPhyloPlace">https://genome.ucsc.edu/cgi-bin/hgPhyloPlace</uri> accessed on 20 December 2021).<li>The <b>Nextclade</b> web tool (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri> accessed on 20 December 2021) and the online COVID-19 genome annotator &#x2018;coronapp&#x2019; [<xref rid="B23-microorganisms-10-00467" ref-type="bibr"><u>23</u></xref>] were used to perform mutation signature calling and SNP profile defining of total genome sequences from North African countries by checking amino acid substitutions, deletion or insertion mutations on specific regions, including; spike surface glycoprotein (S), polyprotein 1ab (nsp1-nsp16), structural proteins (S, E, M, and N) and other accessory proteins.<li> The <b>Nextclade</b> analysis revealed that seventeen of the 36 SARS-CoV-2 genomes belonged to the Delta clade (21J), with the rest being part of clades 20A, 20B and 20C.<li> The mutation-resolved ML phylogenetic tree confirmed the PANGO and <b>Nextclade</b> lineages assigned, since 17 genomes grouped with the 21J representatives, six (16.6%) with the 20A clade, four (11.1%) belong to the 20B clade, eight (22.2%) with the 20C sequences and two (5.4%) with the 21K (Omicron) clade (<xref rid="microorganisms-10-00467-f002" ref-type="fig"><u>Figure 2</u></xref>).<li> The <b>Nextclade</b> analysis revealed that 41.2% of the 971 SARS-CoV-2 genomes belonged to clade 20D, followed by 20A (29.1%) and 21J (Delta) (13.1%) with the rest being part of clades 19B, 20B, 20C, 21I (Delta), 20I (Alpha, V1) and only one genome classified as 21K (Omicron) submitted in December 12, 2021, isolated from a hospitalized male patient aged 67 years.<li> As is shown <xref rid="microorganisms-10-00467-f005" ref-type="fig"><u>Figure 5</u></xref>A, the global phylogeny of SARS-CoV-2 sequences (Delta and Omicron) from North Africa (as of 20 December 2021) showed that Omicron sequences (21K) could have been a progeny of the <b>nextclade</b> 20B.<li> The results revealed the presence of different clades and variants as defined by GISAID, PINGOLIN and <b>Nextclade</b> tools that could be involved in the varied exacerbation of symptoms and disease severity in local residents.<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32793182">32793182</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35252269">35252269</a> <b>(1)</b> Optimization of the SARS-CoV-2 ARTIC Network V4 Primers and Whole Genome Sequencing Protocol<ul><li> <b>NextClade</b> (version 0.13.0) was used for clade assignment and overall quality control metrics are shown in <xref rid="SM1" ref-type="supplementary-material"><u>Supplementary Table 1</u></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35216014">35216014</a> <b>(1)</b> SARS-CoV-2 Delta Variant (AY.3) in the Feces of a Domestic Cat<ul><li>To construct phylogenetic trees, <b>NextClade</b> was used for alignment with default parameters [<xref rid="B24-viruses-14-00421" ref-type="bibr"><u>24</u></xref>].<li> [24](          <person-group person-group-type="author"><name><surname>Ivan Aksamentov</surname><given-names>C.R.</given-names></name><name><surname>Hodcroft</surname><given-names>E.B.</given-names></name><name><surname>Neher</surname><given-names>R.A.</given-names></name></person-group>          <article-title>Nextclade: Clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Res. Softw.</source>          <year>2021</year>          <volume>6</volume>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34729567">34729567</a> <b>(1)</b> Associations between SARS-CoV-2 variants and risk of COVID-19 hospitalization among confirmed cases in Washington State: a retrospective cohort study<ul><li> Sequence quality was determined using <b>Nextclade</b> version 1.0.1 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35171035">35171035</a> <b>(1)</b> Genomic Surveillance of SARS-CoV-2 Lineages Indicates Early Circulation of P.1 (Gamma) Variant of Concern in Southern Brazil<ul><li> All the amino acid mutations are based on <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35242782">35242782</a> <b>(1)</b> Comparative Genomics and Characterization of SARS-CoV-2 P.1 (Gamma) Variant of Concern From Amazonas, Brazil<ul><li> The clade assignment, mutation calling, and sequence quality check were performed by the <b>Nextclade</b> web server (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35317348">35317348</a> <b>(1)</b> Unrecognized introductions of SARS-CoV-2 into the US state of Georgia shaped the early epidemic<ul><li>We assigned all sequences in our dataset to a given clade using <b>Nextclade</b> v.0.13.0 (<xref rid="R24" ref-type="bibr">Hadfield et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35157057">35157057</a> <b>(1)</b> Molecular and Serologic Investigation of the 2021 COVID-19 Case Surge Among Vaccine Recipients in Mongolia<ul><li> Reads were mapped to the SARS-CoV-2 reference sequence (GenBank accession <ext-link xlink:href="http://www.ncbi.nlm.nih.gov/nuccore/1798174254" ext-link-type="uri">NC_045512</ext-link>), and variant analyses were performed using the GISAID database and the <b>NextClade</b> tool (eAppendix in the <xref rid="note-ZLD210328-1-s" ref-type="supplementary-material"><u>Supplement</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35494175">35494175</a> <b>(2)</b> Emergence and spread of a sub-lineage of SARS-CoV-2 Alpha variant B.1.1.7 in Europe, and with further evolution of spike mutation accumulations shared with the Beta and Gamma variants<ul><li> These are the standard quality assessment parameters utilized in <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) (<xref rid="R1" ref-type="bibr">Aksamentov et&#xA0;al.<li> Briefly, it allows for the filtering of genomes, the alignment of genomes in <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) (<xref rid="R1" ref-type="bibr">Aksamentov et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35214741">35214741</a> <b>(1)</b> Autochthonous Outbreak of SARS-CoV-2 Omicron Variant in Booster-Vaccinated (3 Doses) Healthcare Workers in Southern Italy: Just the Tip of the Iceberg?<ul><li> Phylogenetic analysis was performed using the <b>Nextclade</b> sequence analysis webapp (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 14 February 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35167916">35167916</a> <b>(1)</b> Drug repurposing for identification of potential spike inhibitors for SARS-CoV-2 using molecular docking and molecular dynamics simulations<ul><li>The genomic sequences of the specific SARS-CoV-2 lineages were downloaded from the GISAID database (https:/gisaid.org) and the spike protein sequences were obtained using the <b>nextclade</b> tool v 1.7.0 <xref rid="b0190" ref-type="bibr"><u>[38]</u></xref>.<li> [[38]](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Sour. Softw.</source>          <volume>6</volume>          <issue>67</issue>          <year>2021</year>          <fpage>3773</fpage>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35158042">35158042</a> <b>(1)</b> Introduction of the SARS-CoV-2 Beta variant from Comoros into the Marseille geographical area<ul><li> Assembled sequences were analysed for their classification using the <b>Nextclade</b> online tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0025">https://clades.nextstrain.org/</ext-link>) [<xref rid="bib14" ref-type="bibr"><u>14</u></xref>], the Pangolin online tool (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/pangolin.html" id="intref0030">https://cov-lineages.org/pangolin.html</ext-link>) [<xref rid="bib15" ref-type="bibr"><u>15</u></xref>], and an in-house script written in Python as previously described [<xref rid="bib2" ref-type="bibr"><u>2</u></xref>].<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [2](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35197938">35197938</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35685574">35685574</a> <b>(1)</b> Genomic Analysis of AZD1222 (ChAdOx1) Vaccine Breakthrough Infections in the City of Mumbai<ul><li> <b>Nextclade</b> (v0.9.0) was used for assigning our sequences to clade and for mutation calling using recommended parameters [<xref rid="B15" ref-type="bibr"><u>15</u></xref>].<li> [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35144722">35144722</a> <b>(1)</b> Investigation of outbreak cases infected with the SARS-CoV-2 B.1.640 variant in a fully vaccinated elderly population, Normandy, France, November to December 2021<ul><li> The 20A clade/B.1.640 lineage was assigned after <b>Nextclade</b> and Pango lineage analysis [<xref rid="r1" ref-type="bibr"><u>1</u></xref>,<xref rid="r4" ref-type="bibr"><u>4</u></xref>].<li> [1](Cov-Lineages. Lineage B.1.640. PANGO lineages: latest epidemiological lineages of SARS-CoV-2. [Accessed: 13 Jan 2022]. Available from: <ext-link xlink:href="https://cov-lineages.org/lineage.html?lineage=B.1.640" ext-link-type="uri">https://cov-lineages.org/lineage.html?lineage=B.1.640</ext-link>) [4](Nextclade. Clade assignment, mutation calling, and sequence quality checks. [Accessed: 20 Jan 2022]. Available from: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35136154">35136154</a> <b>(5)</b> Development and validation of a high throughput SARS-CoV-2 whole genome sequencing workflow in a clinical laboratory<ul><li><sup>a</sup>SARS-CoV-2 clades were assigned with <b>Nextclade</b> v1.3.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).<li><sup>a</sup>SARS-CoV-2 clades were assigned with <b>Nextclade</b> v1.3.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) and lineages were assigned with Pangolin v3.1.11 (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>).<li><sup>b</sup>SARS-CoV-2 clades were assigned with <b>Nextclade</b> v1.3.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).<li> For clade and lineage assignment, <b>Nextclade</b> version 1.3.0 and Pangolin version 3.1.11 with pangoLEARN 2021&#x2013;08-24 were used.<li> Clade and lineage assignments initially performed with earlier versions of <b>Nextclade</b> and Pangolin were subsequently reprocessed with the above-indicated versions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35197938">35197938</a> <b>(3)</b> Analysis of SARS-CoV-2 Variants From 24,181 Patients Exemplifies the Role of Globalization and Zoonosis in Pandemics<ul><li> Detection of mutations was performed using the <b>Nextclade</b> tool (see footnote 4), and with freebayes<sup><xref rid="footnote16" ref-type="fn"><u>16</u></xref></sup> (<xref rid="B32" ref-type="bibr"><u>Garrison and Marth, 2012</u></xref>) using a mapping quality score of 20 and results filtering by the Python script based on majoritary nucleotide frequencies &#x2265; 70% and nucleotide depths &#x2265; 10 [when sequence reads were generated on the NovaSeq Illumina instrument (Illumina Inc.)] or &#x2265;5 [when sequences reads were generated on a MiSeq Illumina instrument (Illumina Inc.)].<li> Nextstrain clades and Pangolin lineages provided in the present study were determined with the <b>Nextclade</b> web application (see footnote 4) (<xref rid="B34" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="B2" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) and the Pangolin web application (see footnote 5) (<xref rid="B58" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>), respectively.<li> <ext-link xlink:href="NC_045512.2" ext-link-type="DDBJ/EMBL/GenBank">NC_045512.2</ext-link>) were obtained using the <b>Nextclade</b> tool (see footnote 4) (<xref rid="B34" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>; <xref rid="B2" ref-type="bibr"><u>Aksamentov et al., 2021</u></xref>) for SARS-CoV-2 genomes used in the phylogenetic analyses.<li> [16](<a href="https://github.com/freebayes/freebayes">https://github.com/freebayes/freebayes</a>) [Garrison and Marth, 2012](<person-group person-group-type="author"><name><surname>Garrison</surname><given-names>E.</given-names></name><name><surname>Marth</surname><given-names>G.</given-names></name></person-group> (<year>2012</year>). <source><italic>Haplotype-Based Variant Detection From Short-Read Sequencing.</italic></source> Available online at: <ext-link xlink:href="https://arxiv.org/abs/1207.3907" ext-link-type="uri">https://arxiv.org/abs/1207.3907</ext-link><comment>(accessed September 9, 2021)</comment>.) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><name><surname>Neher</surname><given-names>R. A.</given-names></name></person-group> (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J. Open Source Softw.</italic></source><volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Aksamentov et al., 2021](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Roemer</surname><given-names>C.</given-names></name><name><surname>Hodcroft</surname><given-names>E. B.</given-names></name><name><surname>Neher</surname><given-names>R. A.</given-names></name></person-group> (<year>2021</year>). <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes.</article-title><source><italic>J. Open Source Softw.</italic></source><volume>6</volume>:<issue>3773</issue>. <pub-id pub-id-type="doi">10.5281/zenodo.5607694</pub-id>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35242361">35242361</a> <b>(1)</b> Limited genomic reconstruction of SARS-CoV-2 transmission history within local epidemiological clusters<ul><li> We assigned the consensus sequences to a SARS-CoV-2 clade with <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>; last accessed 20 December 2021) and to a SARS-CoV-2 PANGO lineage (<xref rid="R54" ref-type="bibr">Rambaut et&#xA0;al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35051367">35051367</a> <b>(4)</b> Transmission from vaccinated individuals in a large SARS-CoV-2 Delta variant outbreak<ul><li><hr/></td></tr><tr><td>LoFreq version 2.1.5</td><td><xref rid="bib27" ref-type="bibr"><u>Wilm et&#xA0;al., 2012</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://csb5.github.io/lofreq/" id="intref0025">https://csb5.github.io/lofreq/</ext-link></td></tr><tr><td>viral-ngs 2.1.28</td><td>Broad Institute</td><td><ext-link ext-link-type="uri" xlink:href="https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master" id="intref0030">https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master</ext-link></td></tr><tr><td>Nextstrain v3.0.3</td><td><xref rid="bib12" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_nextstrain_aligned_input:master" id="intref0035">https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_nextstrain_aligned_input:master</ext-link></td></tr><tr><td>FastTree version 2.1.11</td><td><xref rid="bib19" ref-type="bibr"><u>Price et&#xA0;al., 2009</u></xref> and <xref rid="bib20" ref-type="bibr"><u>2010</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="http://microbesonline.org/fasttree/" id="intref0040">http://www.microbesonline.org/fasttree/</ext-link></td></tr><tr><td>baltic</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/evogytis/baltic" id="intref0045">https://github.com/evogytis/baltic</ext-link></td></tr><tr><td>EpiEstim v4.0.1</td><td><xref rid="bib7" ref-type="bibr"><u>Cori et&#xA0;al., 2013</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/EpiEstim/index.html" id="intref0050">https://cran.r-project.org/web/packages/EpiEstim/index.html</ext-link></td></tr><tr><td>outbreaker2 (version 1.1.2)</td><td><xref rid="bib3" ref-type="bibr"><u>Campbell et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/outbreaker2/index.html" id="intref0055">https://cran.r-project.org/web/packages/outbreaker2/index.html</ext-link></td></tr><tr><td><b>Nextclade</b> CLI version 1.3.0.<li><hr/></td></tr><tr><td>LoFreq version 2.1.5</td><td><xref rid="bib27" ref-type="bibr"><u>Wilm et&#xA0;al., 2012</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://csb5.github.io/lofreq/" id="intref0025">https://csb5.github.io/lofreq/</ext-link></td></tr><tr><td>viral-ngs 2.1.28</td><td>Broad Institute</td><td><ext-link ext-link-type="uri" xlink:href="https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master" id="intref0030">https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_illumina_full:master</ext-link></td></tr><tr><td>Nextstrain v3.0.3</td><td><xref rid="bib12" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_nextstrain_aligned_input:master" id="intref0035">https://dockstore.org/workflows/github.com/broadinstitute/viral-pipelines/sarscov2_nextstrain_aligned_input:master</ext-link></td></tr><tr><td>FastTree version 2.1.11</td><td><xref rid="bib19" ref-type="bibr"><u>Price et&#xA0;al., 2009</u></xref> and <xref rid="bib20" ref-type="bibr"><u>2010</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="http://microbesonline.org/fasttree/" id="intref0040">http://www.microbesonline.org/fasttree/</ext-link></td></tr><tr><td>baltic</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/evogytis/baltic" id="intref0045">https://github.com/evogytis/baltic</ext-link></td></tr><tr><td>EpiEstim v4.0.1</td><td><xref rid="bib7" ref-type="bibr"><u>Cori et&#xA0;al., 2013</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/EpiEstim/index.html" id="intref0050">https://cran.r-project.org/web/packages/EpiEstim/index.html</ext-link></td></tr><tr><td>outbreaker2 (version 1.1.2)</td><td><xref rid="bib3" ref-type="bibr"><u>Campbell et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/outbreaker2/index.html" id="intref0055">https://cran.r-project.org/web/packages/outbreaker2/index.html</ext-link></td></tr><tr><td><b>Nextclade</b> CLI version 1.3.0</td><td><xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" id="intref0060">https://github.com/nextstrain/<b>nextclade</b></ext-link>.<li> All sequences were processed with <b>Nextclade</b> CLI version 1.3.0 (<xref rid="bib1" ref-type="bibr"><u>Aksamentov et&#xA0;al., 2021</u></xref>) and custom python code.<li> [Wilm et&#xA0;al., 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23066108">23066108</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Price et&#xA0;al., 2009](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19377059">19377059</a>) [2010](          <person-group person-group-type="author">            <name>              <surname>Price</surname>              <given-names>M.N.</given-names>            </name>            <name>              <surname>Dehal</surname>              <given-names>P.S.</given-names>            </name>            <name>              <surname>Arkin</surname>              <given-names>A.P.</given-names>            </name>          </person-group>          <article-title>FastTree 2--approximately maximum-likelihood trees for large alignments</article-title>          <source>PLOS One</source>          <volume>5</volume>          <year>2010</year>          <object-id pub-id-type="publisher-id">e9490</object-id>          <pub-id pub-id-type="doi">10.1371/journal.pone.0009490</pub-id>        ) [Cori et&#xA0;al., 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24043437">24043437</a>) [Campbell et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30343663">30343663</a>) [Wilm et&#xA0;al., 2012](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23066108">23066108</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Price et&#xA0;al., 2009](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19377059">19377059</a>) [2010](          <person-group person-group-type="author">            <name>              <surname>Price</surname>              <given-names>M.N.</given-names>            </name>            <name>              <surname>Dehal</surname>              <given-names>P.S.</given-names>            </name>            <name>              <surname>Arkin</surname>              <given-names>A.P.</given-names>            </name>          </person-group>          <article-title>FastTree 2--approximately maximum-likelihood trees for large alignments</article-title>          <source>PLOS One</source>          <volume>5</volume>          <year>2010</year>          <object-id pub-id-type="publisher-id">e9490</object-id>          <pub-id pub-id-type="doi">10.1371/journal.pone.0009490</pub-id>        ) [Cori et&#xA0;al., 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24043437">24043437</a>) [Campbell et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30343663">30343663</a>) [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [Aksamentov et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J.&#xA0;Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35124261">35124261</a> <b>(1)</b> Neutralizing antibody activity against the B.1.617.2 (delta) variant 8&#xA0;months after two-dose vaccination with BNT162b2 in health care workers<ul><li> For the B.1.617.2 (delta) variant, next-generation sequencing (NGS) sequencing was performed, and lineage, variant assignment, and clade classification of the SARS-CoV-2 sequence was carried out using Pangolin and <b>Nextclade</b> as part of the Nextstrain framework.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35060146">35060146</a> <b>(2)</b> Characteristics of the first 1119 SARS&#x2010;CoV&#x2010;2 Omicron variant cases, in Marseille, France, November&#x2212;December 2021<ul><li> Mutation detection was performed with <b>Nextclade</b> web application (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>)<xref rid="jmv27613-bib-0008" ref-type="bibr"><sup>8</sup></xref> and freebayes (<ext-link xlink:href="https://github.com/freebayes/freebayes" ext-link-type="uri">https://github.com/freebayes/freebayes</ext-link>).<li> Nextstrain clades and Pangolin lineages were determined using <b>Nextclade</b> and Pangolin web application (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>),<xref rid="jmv27613-bib-0010" ref-type="bibr"><sup>10</sup></xref> respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35112910">35112910</a> <b>(1)</b> Identification of the First SARS-CoV-2 Lineage B.1.1.529 Virus Detected in Europe<ul><li> <b>Nextclade</b> v1.10.0 was used for comparison with the RefSeq sequence (<xref rid="B7" ref-type="bibr"><u>7</u></xref>).<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35112895">35112895</a> <b>(2)</b> Coding-Complete Genome Sequences of a Delta Subvariant (AY.33) of SARS-CoV-2 Obtained from Moroccan COVID-19 Patients<ul><li> The quality of the sequences was monitored using the <b>Nextclade</b> web tool version 1.7.2 (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>), although the determination of the mutations was carried out using the web application Cov-GLUE (<ext-link xlink:href="http://cov-glue.cvr.gla.ac.uk/" ext-link-type="uri">http://cov-glue.cvr.gla.ac.uk/</ext-link>), and the lineage and sublineage were obtained using the web application Pangolin (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>).<li>Phylogenetic tree of SARS-CoV-2 generated from Nextstrain using the <b>Nextclade</b> maximum likelihood module.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35115515">35115515</a> <b>(2)</b> Multiple expansions of globally uncommon SARS-CoV-2 lineages in Nigeria<ul><li>Monthly clade frequency distribution using <b>Nextclade</b> nomenclature for clades with a frequency &gt;1% in both Nigeria and West Africa compared to the main clades observed in Europe and South Africa.<li> <b>Nextclade</b> nomenclature was used due to its clarity to represent major clades.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35132417">35132417</a> <b>(1)</b> Detection and Interspecies Comparison of SARS-CoV-2 Delta Variant (AY.3) in Feces from a Domestic Cat and Human Samples<ul><li>To construct phylogenetic trees, <b>NextClade</b> was used for alignment [<xref rid="R24" ref-type="bibr"><u>24</u></xref>], IQ-Tree (v1.6.12) was used to generate the phylogenetic tree [<xref rid="R25" ref-type="bibr"><u>25</u></xref>&#x2013;<xref rid="R28" ref-type="bibr"><u>28</u></xref>], and FigTree v1.4.4 was used to visualize the tree.<li> [24](<name><surname>Ivan Aksamentov</surname><given-names>C.R.</given-names></name>, <name><surname>Emma</surname><given-names>B.</given-names></name><article-title>Hodcroft, Richard A. Neher, Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>. <source>Journal of Open Source Software</source>, <year>2021</year>. <volume>6</volume>(<issue>67</issue>).) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28481363">28481363</a>) [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27084950">27084950</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35134129">35134129</a> <b>(1)</b> Prolonged Shedding of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) at High Viral Loads Among Hospitalized Immunocompromised Persons Living With Human Immunodeficiency Virus (HIV), South Africa<ul><li> Clade and lineage assignments were made using the online <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and Pangolin (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) applications, which also enable identification of known variants of concern as well as novel mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35123044">35123044</a> <b>(2)</b> Clinical and genomic data of sars-cov-2 detected in maternal&#x2013;fetal interface during the first wave of infection in Brazil<ul><li> v.3.1.14) and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0010">https://clades.nextstrain.org/</ext-link>; v.1.5.4).<li> Mutation profile identified on <b>NEXTCLADE</b> was compared with the most frequent lineage mutations using outbreak.info Lineage Reports [<xref rid="bib22" ref-type="bibr"><u>22</u></xref>].<li> [22](Mullen J, Tsueng G, Latif A, Alkuzweny M, Cano M. et&#xA0;al. outbreak.info. <ext-link ext-link-type="uri" xlink:href="https://outbreak.info/" id="intref0065">https://outbreak.info/</ext-link>(Accessed September 10 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35105380">35105380</a> <b>(1)</b> Neutrophils predominate the immune signature of cerebral thrombi in COVID-19 stroke patients<ul><li>Importantly, we further confirmed the viral PCR detection by sequencing, which revealed presence of the SARS-CoV-2 clade 20A, identified by <b>NextClade</b> v 1.6.0 (clades.nextstrain.org) (data not shown).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34906290">34906290</a> <b>(4)</b> SARS-CoV-2 Circulation, Guinea, March 2020&#x2013;July 2021<ul><li> We analyzed the remaining 136 (54%) sequences by using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) and Pangolin software (<ext-link xlink:href="https://cov-lineages.org" ext-link-type="uri">https://cov-lineages.org</ext-link>).<li> Branches and the sequence names are colored according to <b>Nextclade</b> assigned clades: 20A, light gray; 20B, medium gray; 20C, dark gray; 20D, black; 20I/B.1.1.7/Alpha, blue; 21A/B.1.617.2/Delta, azure; 21D/B.1.525/Eta, yellow.<li> The 136 selected sequences are assigned by <b>Nextclade</b> and classified according to sampling date from March 31, 2020, to July 16, 2021.<li>*Clades and lineages are respectively assigned according to <b>Nextclade</b> definition (<ext-link xlink:href="https://github.com/nextstrain/ncov/blob/master/docs/src/reference/naming_clades.md" ext-link-type="uri">https://github.com/nextstrain/ncov/blob/master/docs/src/reference/naming_clades.md</ext-link>) and PANGO lineages list (<ext-link xlink:href="https://github.com/cov-lineages/pangolin" ext-link-type="uri">https://github.com/cov-lineages/pangolin</ext-link>) at the same assignment date (August 14, 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35105893">35105893</a> <b>(1)</b> SARS-CoV-2 genomes from Saudi Arabia implicate nucleocapsid mutations in host response and increased viral load<ul><li> The phylogeny was constructed using IQ-TREE (v2.0.5)<sup><xref ref-type="bibr" rid="CR67"><u>67</u></xref></sup>, clades were assigned using <b>Nextclade</b> and internal node dates were inferred and sequences pruned using TreeTime (v0.7.5)<sup><xref ref-type="bibr" rid="CR68"><u>68</u></xref></sup>.<li> [67](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25371430">25371430</a>) [68](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35072638">35072638</a> <b>(1)</b> Has Omicron Changed the Evolution of the Pandemic?<ul><li> We collected clade designations from sequences using <b>Nextclade</b> nomenclature [<xref rid="ref40" ref-type="bibr"><u>40</u></xref>] and lineage designations using Pangolin nomenclature for SARS-CoV-2 [<xref rid="ref41" ref-type="bibr"><u>41</u></xref>,<xref rid="ref42" ref-type="bibr"><u>42</u></xref>].<li> [40](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) [41](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33514928">33514928</a>) [42](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35104582">35104582</a> <b>(3)</b> Early introduction, dispersal and evolution of Delta SARS-CoV-2 in Southern Brazil, late predominance of AY.99.2 and AY.101 related lineages<ul><li> Phylogenetic tree was constructed including all SARS-CoV-2 complete genomes available from GISAID (between November 2019 and December 2021) through the <b>Nextclade</b> tool on Nextstrain server (<xref rid="bib0001" ref-type="bibr"><u>Aksamentov&#xA0;et&#xA0;al., 2021</u></xref>).<li> The 183 Delta variant genomes were aligned against the Wuhan-Hu-1 reference through <b>Nextclade</b> tool, and maximum-likelihood phylogenetic tree was constructed in IQ-TREE33.<li> Panel A shows the phylogenetic tree of all circulating lineages, developed through the Nextstrain server (<b>Nextclade</b> tool), using GISAID data.<li> [Aksamentov&#xA0;et&#xA0;al., 2021](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J. Open Source Softw.</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35118480">35118480</a> <b>(1)</b> A Quick Displacement of the SARS-CoV-2 variant Delta with Omicron: Unprecedented Spike in COVID-19 Cases Associated with Fewer Admissions and Comparable Upper Respiratory Viral Loads<ul><li> Query mutations were manually confirmed with Integrated Genomics viewer (IGV) (Version 2.8.10), clades were determined using <b>Nextclade</b> beta v 0.12.0 (<ext-link xlink:href="http://clades.nextstrain.org" ext-link-type="uri">clades.nextstrain.org</ext-link>), and lineages were determined with Pangolin COVID-19 lineage Assigner (COG-UK (cog-uk.io)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35107520">35107520</a> <b>(1)</b> Open-source real-time quantitative RT-PCR-based on a RNA standard for the assessment of SARS-CoV-2 viral load<ul><li> <b>Nextclade</b> CLI was utilized for determining the nucleotide substitutions of the global sample.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35208761">35208761</a> <b>(1)</b> Comparative Analysis of Five Multiplex RT-PCR Assays in the Screening of SARS-CoV-2 Variants<ul><li> Lineage and clade data, quality controls, mutations, missing nucleotides and gaps were analyzed and confirmed with the support of Pangolin (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri> accessed on 15 November 2021) and <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> accessed on 15 November 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35081137">35081137</a> <b>(1)</b> ASPICov: An automated pipeline for identification of SARS-Cov2 nucleotidic variants<ul><li>Several new features are currently under development, such as a global HTML report, phylogenetic analysis, integration of ONT and MGI sequencing technologies, highlight of genotype percentage, PANGO lineage determination and <b>Nextclade</b>/Gisaid data comparison.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35093719">35093719</a> <b>(3)</b> Comparing susceptibility and contagiousness in concurrent outbreaks with a non-VOC and the VOC SARS-CoV-2 variant B.1.1.7 in daycare centers in Hamburg, Germany<ul><li> Pangolin lineage and Nextstrain clade assignment of consensus sequences were performed using the pangolin (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0010">https://github.com/cov-lineages/pangolin</ext-link>) and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" id="intref0015">https://github.com/nextstrain/<b>nextclade</b></ext-link>) packages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35215820">35215820</a> <b>(1)</b> Spike Gene Evolution and Immune Escape Mutations in Patients with Mild or Moderate Forms of COVID-19 and Treated with Monoclonal Antibodies Therapies<ul><li> The mutations found in the S gene and the Nextstrain clade were determined for each strain with the <b>Nextclade</b> online tool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35215811">35215811</a> <b>(2)</b> ViralFlow: A Versatile Automated Workflow for SARS-CoV-2 Genome Assembly, Lineage Assignment, Mutations and Intrahost Variant Detection<ul><li> The consensus quality and set of mutations are evaluated using <b>nextclade</b> v.0.14.2 [<xref rid="B28-viruses-14-00217" ref-type="bibr"><u>28</u></xref>].<li> If the analyzed sample shows intrahost signals, Pangolin and <b>nextclade</b> analyses are performed for both consensus versions (with major and minor allele frequencies), while it will run only for a single consensus genome in the absence of iSNV sites.<li> and &#x2018;.<b>nextclade</b>.csv&#x2019;, which provide the virus lineage and the mutations found, respectively.<li> columns present in the &#x2018;.<b>nextclade</b>.csv&#x2019; table.<li> [28](          <person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I.</given-names></name><name><surname>Neher</surname><given-names>R.</given-names></name></person-group>          <article-title>Nextclade</article-title>          <comment>Available online: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-09-02">(accessed on 2 September 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35123650">35123650</a> <b>(1)</b> SARS-CoV-2 breakthrough infections elicit potent, broad, and durable neutralizing antibody responses<ul><li> Sequence quality was determined using <b>Nextclade</b> version 1.0.0-alpha.8 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0035">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35086043">35086043</a> <b>(1)</b> Detection of SARS-CoV-2 variants by Abbott molecular, antigen, and serological tests<ul><li> Sequences containing lineage-defining mutations were obtained from GISAID <xref rid="bib0011" ref-type="bibr"><u>[11]</u></xref> for <italic>in silico</italic> analysis by customized application of the <b>NextClade</b> tool (clades.nextstrain.org) to compare individual assay target sequences for Abbott ID NOW COVID-19, <italic>m</italic>2000 RealTi<italic>m</italic>e SARS-CoV-2, Alinity m SARS-CoV-2, and Alinity m Resp-4-Plex molecular assays; the BinaxNOW COVID-19 Ag Card and Panbio COVID-19 Ag Rapid Test Device; and the ARCHITECT/Alinity i SARS-CoV-2 IgG and AdviseDx IgM assays, Panbio COVID-19 IgG assay, and ARCHITECT/Alinity i AdviseDx SARS-CoV-2 IgG II assay.<li> [[11]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35065705">35065705</a> <b>(1)</b> Efficacy of the adjuvanted subunit protein COVID-19 vaccine, SCB-2019: a phase 2 and 3 multicentre, double-blind, randomised, placebo-controlled trial<ul><li> Pangolin and <b>NextClade</b> were used for viral variants and lineages identification.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34545361">34545361</a> <b>(5)</b> Rapid and parallel adaptive mutations in spike S1 drive clade success in SARS-CoV-2<ul><li> These genomes were aligned with Nextalign (<ext-link xlink:href="https://docs.nextstrain.org/projects/<b>nextclade</b>/en/latest/user/nextalign-cli.html" ext-link-type="uri">docs.nextstrain.org/projects/<b>nextclade</b>/en/latest/user/nextalign-cli.html</ext-link>) and masked to minimize error in phylogenetic inference associated with problematic amplicon sites.<li> Masked alignments were filtered to exclude strains that were known outliers, sequenced due to &#x2018;S dropout&#x2019;, mis-annotated with an admin division of &#x2018;USA&#x2019;, shorter than 27,000 bp of A, C, T, or G bases, missing complete date information, annotated with a date prior to October 2019, flagged with more than 20 mutations above the expected number based on the mutational clock rate, or flagged by <b>Nextclade</b> (<ext-link xlink:href="https://docs.nextstrain.org/projects/<b>nextclade</b>/en/latest/user/algorithm/07-quality-control.html" ext-link-type="uri">docs.nextstrain.org/projects/<b>nextclade</b>/en/latest/user/algorithm/07-quality-control.html</ext-link>) with one or more clusters of 6 or more private differences in a 100-nucleotide window.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34731022">34731022</a> <b>(3)</b> Clinical and Infection Prevention Applications of Severe Acute Respiratory Syndrome Coronavirus 2 Genotyping: an Infectious Diseases Society of America/American Society for Microbiology Consensus Review Document<ul><li>As most clinicians are now familiar with the predominant circulating VOI and VOC, it is reasonable to consider including variant lineages and clades in clinical reports, including if they were derived from the PANGO or <b>Nextclade</b> classification systems, as well as the genotyping method used (<xref rid="B9" ref-type="bibr"><u>9</u></xref>).<li> The data generated from these basic programs require additional analysis to produce PANGO lineage and <b>Nextclade</b> assignment as well as potential clinical interpretation of identified variants.<li> access to the ARTIC and <b>Nextclade</b> pipelines (<xref rid="B13" ref-type="bibr"><u>13</u></xref>).<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33101244">33101244</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34757834">34757834</a> <b>(1)</b> External Quality Assessment of SARS-CoV-2 Sequencing: an ESGMD-SSM Pilot Trial across 15 European Laboratories<ul><li> differently and instead reported the <b>Nextclade</b> assignment (<xref rid="B10" ref-type="bibr"><u>10</u></xref>); center 14 in turn deemed samples NGS1 and NGS4 to belong to a single cluster.<li> [10](<person-group person-group-type="author"><string-name><surname>Aksamentov</surname><given-names>I</given-names></string-name>, <string-name><surname>Neher</surname><given-names>R</given-names></string-name></person-group>. <year>2020</year>. <article-title>Nextclade</article-title>. <ext-link xlink:href="https://github.com/nextstrain/nextclade" ext-link-type="uri">https://github.com/nextstrain/nextclade</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35045124">35045124</a> <b>(9)</b> Genomic surveillance of SARS-CoV-2 in the state of Delaware reveals tremendous genomic diversity<ul><li> Consensus sequences were then submitted to <b>NextClade</b> and Pangolin for phylogenetic analysis and GISAID for mutation analysis.<li> A lineage analysis was performed using both <b>Nextclade</b>&#x2019;s clade assigner (<xref rid="pone.0262573.s002" ref-type="supplementary-material"><u>S2 Table</u></xref>) and the Pangolin COVID-19 lineage assigner (<xref rid="pone.0262573.s003" ref-type="supplementary-material"><u>S3 Table</u></xref>).<li> <b>Nextclade</b>&#x2019;s results contained 16 unique clades whereas Pangolin&#x2019;s contained 88 lineages (<xref rid="pone.0262573.g001" ref-type="fig"><u>Fig 1</u></xref>).<li> Panel B: The distribution of the <b>NextClade</b> Clades for all samples analyzed.<li> The second largest clade identified by <b>Nextclade</b> was the global 20C with 273 samples (19%); Pangolin categorized these into 31 unique lineages with most belonging to B.1, B.1.369, and B.1.311.<li> <b>Nextclade</b> identified 209 samples as members of the 20G US specific clade originally identified by Pater et al.<li> <b>Nextclade</b>&#x2019;s phylogenetic analysis combined their global database with our Delaware specific data (<xref rid="pone.0262573.g002" ref-type="fig"><u>Fig 2B</u></xref>).<li> Panel B: <b>Nextclade</b> phylogenetic tree of samples from the Delaware cohort superimposed on global sample set.<li><b>Nextclade</b> results.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35065122">35065122</a> <b>(2)</b> Use of sewage surveillance for COVID-19 to guide public health response: A case study in Hong Kong<ul><li> The called consensus was identified to the specific lineage by <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir0010">https://clades.nextstrain.org/</ext-link>).<li> The phylogenetic tree was created by <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir0015">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35464127">35464127</a> <b>(1)</b> Dominant mutations in the severe acute respiratory syndrome coronavirus&#x2010;2 genome challenge polymerase chain reaction detection<ul><li> Web&#x2010;based tools, such as <b>NextClade</b> and PrimerChecker of GISAID, are available to evaluate the mismatches of SARS&#x2010;CoV&#x2010;2 RT&#x2010;PCR primers/probes, which is very helpful for checking and renewing primers/probes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35095827">35095827</a> <b>(3)</b> Molecular Characterization of Severe Acute Respiratory Syndrome Coronavirus 2 Isolates From Central Inner Sardinia<ul><li> The sequences were then attributed to a PANGOLIN (Phylogenetic Assignment of Named Global Outbreak LINeages) lineage with PANGOLIN COVID-19 Lineage Assigner v.2.4.2 software (<xref rid="B17" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) and to a NextStrain clade with the <b>Nextclade</b> v0.14.2 web tool (<xref rid="B5" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li>However, a notable exception to this general correlation was found, since COV1241 was assigned to the B.1.1.7 lineage by PANGOLIN COVID-19 Lineage Assigner v.2.4.2 and to the 20B clade by the <b>Nextclade</b> v0.14.2 algorithm.<li> T in <italic>ORF1ab</italic>, are included in the set of SNPs that are used by the <b>Nextclade</b> v0.14.2 algorithm to identify clade 20I.<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062359">35062359</a> <b>(1)</b> Prevalence of SARS-CoV-2 Variants of Concern and Variants of Interest in COVID-19 Breakthrough Infections in a Hospital in Monterrey, Mexico<ul><li> Sequence quality was determined with <b>Nextclade</b> web tool, and 58 low-quality sequences were removed.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35055707">35055707</a> <b>(1)</b> Symptomatic SARS-CoV-2 Infection with Ageusia after Two mRNA Vaccine Doses<ul><li> <b>Nextclade</b> software (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 21 April 2021) was used to assign clade and call mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35087848">35087848</a> <b>(7)</b> HiSpike Method for High-Throughput Cost Effective Sequencing of the SARS-CoV-2 Spike Gene<ul><li> <xref rid="SM2" ref-type="supplementary-material"><u>Supplementary File 2</u></xref>) into <b>Nextclade</b> branches that were similar to those assigned via ARTIC V3 whole genome sequencing.<li> Spike gene sequences of 441 samples generated by the HiSpike method were uploaded to the <b>Nextclade</b> platform.<li> HiSpike sequences are represented by the full circles on a background of the <b>Nextclade</b>'s global representative clade tree of December 8 2021.<li> Nevertheless, even in these samples we obtained enough sequence information to detect the variant's mutation signatures and determine clades with the <b>nextclade</b> web tool (<xref rid="SM13" ref-type="supplementary-material"><u>Supplementary File 13</u></xref>; <xref rid="F4" ref-type="fig"><u>Figure 4</u></xref>).<li> FASTA sequences were uploaded to <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<li> <b>Nextclade</b> is a web-tool that identifies differences between reference and user uploaded genomes.<li> For each uploaded sequence, <b>Nextclade</b> provided a complete list of established and new spike gene mutations, both in nucleotide and amino acid notation.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35056592">35056592</a> <b>(1)</b> SARS-CoV-2 Evolution and Spike-Specific CD4+ T-Cell Response in Persistent COVID-19 with Severe HIV Immune Suppression<ul><li> We built the consensus sequences using iVar consensus, with a minimum variant allele frequency (VAF) threshold of 0.5 and a minimum depth of 10 reads, assigning them to a clade with <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>, accessed on 19 October 2021) and to a lineage using Pangolin [<xref rid="B7-microorganisms-10-00143" ref-type="bibr"><u>7</u></xref>].<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062324">35062324</a> <b>(1)</b> SARS-CoV-2 Reverse Zoonoses to Pumas and Lions, South Africa<ul><li> <b>NextClade</b> analysis as well as multiple sequence alignment (MSA) of the Spike glycoproteins revealed that each of the infections was a Delta variant (B.1.617.1) of SARS-CoV-2.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35017107">35017107</a> <b>(3)</b> Evidence of SARS-CoV-2 symptomatic reinfection in four healthcare professionals from the same hospital despite the presence of antibodies<ul><li> Major SARS-CoV-2 clade memberships and the worldwide distribution of S477 mutation were predicted using <b>Nextclade</b> (<xref rid="bib0009" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al.&#xA0;2018</u></xref>).<li><b>Nextclade</b> analysis of the 8 SARS-CoV-2 isolates from the 4 reinfection cases.<li> (A) Geographic distribution of SARS-CoV-2 clades (<b>Nextclade</b>, accessed on March 21, 2021); (B) Time evolution of clades; (C) Time evolution of S477 frequency.<li> [Hadfield&#xA0;et&#xA0;al.&#xA0;2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35000004">35000004</a> <b>(1)</b> Emergence of two distinct variants of SARS-CoV-2 and an explosive second wave of COVID-19: the experience of a tertiary care hospital in Pune, India<ul><li> The <b>Nextclade</b> web tool (v0.14.2) (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>) was used to compare the sequences to those of reference strains for clade assignment.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35018222">35018222</a> <b>(2)</b> First detection of SARS-CoV-2 variant B.1.1.529 (Omicron) in Ecuador<ul><li>Following the <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0010">https://clades.nextstrain.org/</ext-link>) [<xref rid="bib8" ref-type="bibr"><u>8</u></xref>] and Pangolin (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0015">https://github.com/cov-lineages/pangolin</ext-link>) package [<xref rid="bib9" ref-type="bibr"><u>9</u></xref>], the annotated genome clustered into 21K clade and BA.1 lineage respectively, which correspond to Omicron VOC.<li> Phylogenetic analyzes were performed in CoVizue-GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="intref0025">https://www.gisaid.org/</ext-link>) and <b>Nextclade</b> [<xref rid="bib9" ref-type="bibr"><u>9</u></xref>,<xref rid="bib10" ref-type="bibr"><u>10</u></xref>].<li> [8](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Roemer</surname>              <given-names>C.</given-names>            </name>            <name>              <surname>Hodcroft</surname>              <given-names>E.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade: clade assignment, mutation calling and quality control for viral genomes</article-title>          <source>J&#xA0;Open Source Softw</source>          <volume>6</volume>          <year>2021</year>          <fpage>3773</fpage>          <pub-id pub-id-type="doi">10.21105/joss.03773</pub-id>        ) [9](Centre for Genomic Pathogen and Surveillance. Pangolin app 2021:1. Centre for genomic pathogen and surveillance (accessed 21 December, 2021).) [9](Centre for Genomic Pathogen and Surveillance. Pangolin app 2021:1. Centre for genomic pathogen and surveillance (accessed 21 December, 2021).) [10](          <person-group person-group-type="author">            <name>              <surname>Shu</surname>              <given-names>Y.</given-names>            </name>            <name>              <surname>McCauley</surname>              <given-names>J.</given-names>            </name>          </person-group>          <article-title>GISAID: global initiative on sharing all influenza data - from vision to reality</article-title>          <source>Euro Surveill Bull Eur Sur Les Mal Transm&#xA0;= Eur Commun Dis Bull</source>          <volume>22</volume>          <year>2017</year>          <pub-id pub-id-type="doi">10.2807/1560-7917.ES.2017.22.13.30494</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34999740">34999740</a> <b>(2)</b> High-Throughput Adaptable SARS-CoV-2 Screening for Rapid Identification of Dominant and Emerging Regional Variants<ul><li> Within 2 days, we used sequencing and clustering to elucidate that the sample is a Delta variant based on its Pangolin lineage B.1.617.2 (ambiguity score: 0.9995) and <b>Nextclade</b> clade 21A <bold><xref rid="F3" ref-type="fig"><u> Figure 3A </u></xref></bold>.<li> Interestingly, when phylogenetically arranged by <b>Nextclade</b>, our sample did not cluster with the samples from Turkey, making direct spread from Turkey less likely (<xref rid="sup1" ref-type="supplementary-material"><u>Supplemental Figure 2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35071285">35071285</a> <b>(3)</b> Isolation and Identification of a Rare Spike Gene Double-Deletion SARS-CoV-2 Variant From the Patient With High Cycle Threshold Value<ul><li> Alignment and sequence comparisons of KMUH-1 and KMUH-2 to the reference sequence NC_045512.2 The SARS-CoV-2 isolate Wuhan-Hu-1/2019 was performed using <b>Nextclade</b> v1.7.3 (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) (<xref rid="B38" ref-type="bibr"><u>38</u></xref>) to verify the results of the virus detail report from GISAID.<li> The spike gene variations in the 72 closely related SARS-CoV-2 genomes were analyzed using <b>Nextclade</b> v1.7.3 using NC_045512.2 SARS-CoV-2 isolate Wuhan-Hu-1/2019 as a reference sequence.<li> These three subsets of SARS-CoV-2 sequences were downloaded and validated for the presence of the double-deletion variation by using <b>Nextclade</b> v1.7.3 with the Wuhan-Hu-1/2019 sequence used as the reference sequence.<li> [38](<person-group person-group-type="author"><name><surname>Aksamentov</surname><given-names>I</given-names></name><name><surname>Roemer</surname><given-names>C</given-names></name><name><surname>Hodcroft</surname><given-names>EB</given-names></name><name><surname>Neher</surname><given-names>RA</given-names></name></person-group>. <source>Nextclade: Clade Assignment, Mutation Calling Quality Control for Viral Genomes.</source> (<year>2021</year>). Available online at: <ext-link xlink:href="https://zenodo.org/record/5607694" ext-link-type="uri">https://zenodo.org/record/5607694</ext-link> (accessed December 20, 2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34842496">34842496</a> <b>(1)</b> A year living with SARS-CoV-2: an epidemiological overview of viral lineage circulation by whole-genome sequencing in Barcelona city (Catalonia, Spain)<ul><li> Furthermore, it performs lineage/clade determination and mutation characterization by using updated Pangolin (<ext-link xlink:href="https://cov-lineages.org/pangolin.html" ext-link-type="uri">https://cov-lineages.org/pangolin.html</ext-link>) and <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) nomenclatures.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35074623">35074623</a> <b>(1)</b> Rapid and sensitive detection of SARS-CoV-2 variants in nasopharyngeal swabs and wastewaters<ul><li> Clade assignment was performed using COVIDEX (<xref rid="bib0002" ref-type="bibr"><u>Cacciabue&#xA0;et al., 2020</u></xref>) and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0004">https://clades.nextstrain.org/</ext-link>).<li> [Cacciabue&#xA0;et al., 2020](          <person-group person-group-type="author">            <name>              <surname>Cacciabue</surname>              <given-names>M</given-names>            </name>            <name>              <surname>Aguilera</surname>              <given-names>P</given-names>            </name>            <name>              <surname>Gismondi</surname>              <given-names>MI</given-names>            </name>            <name>              <surname>Taboga</surname>              <given-names>O.</given-names>            </name>          </person-group>          <article-title>Covidex: an ultrafast and accurate tool for virus subtyping</article-title>          <source>bioRxiv</source>          <year>2020</year>          <pub-id pub-id-type="doi">10.1101/2020.08.21.261347</pub-id>          <comment>2020.08.21.261347</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35045382">35045382</a> <b>(1)</b> A deletion in the N gene of SARS-CoV-2 may reduce test sensitivity for detection of SARS-CoV-2<ul><li> The genetic lineages of the sequences was determined using Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin, pangolin.cog-uk.io) and <b>Nextclade</b> (clades.nextstrain.org).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34954792">34954792</a> <b>(1)</b> CoV-Spectrum: analysis of globally shared SARS-CoV-2 data to identify and characterize new variants<ul><li> We clean the location data with Nextstrain&#x2019;s geo location rules (<ext-link xlink:href="https://github.com/nextstrain/ncov-ingest/blob/master/source-data/gisaid_geoLocationRules.tsv" ext-link-type="uri">https://github.com/nextstrain/ncov-ingest/blob/master/source-data/gisaid_geoLocationRules.tsv</ext-link>) and we run <b>Nextclade</b> (<xref rid="btab856-B1" ref-type="bibr">Aksamentov <italic toggle="yes">et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35047415">35047415</a> <b>(1)</b> Learning From Biological and Computational Machines: Importance of SARS-CoV-2 Genomic Surveillance, Mutations and Risk Stratification<ul><li> SARS-CoV-2 clades were assigned using <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35062227">35062227</a> <b>(3)</b> Emerging SARS-CoV-2 Genotypes Show Different Replication Patterns in Human Pulmonary and Intestinal Epithelial Cells<ul><li> The classification of the isolates was determined using the <b>Nextclade</b> tool (Nextstrain, <b>Nextclade</b>: <uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 20 May 2021 and 27 November 2021) and the Pangolin classification tool (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri> (accessed on 20 May 2021), and for the name of the mutation.<li>The phylogenetic analysis confirmed that the isolates classified as belonging to the same lineage by the <b>Nextclade</b> tool were close relatives and clustered (<xref rid="viruses-14-00023-f003" ref-type="fig"><u>Figure 3</u></xref>a).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34937195">34937195</a> <b>(1)</b> Evaluation of Three Quantitative Anti-SARS-CoV-2 Antibody Immunoassays<ul><li> <b>Nextclade</b> 20A.EU2) infecting Vero cells (ATCC, CCL-81).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34922338">34922338</a> <b>(1)</b> Infection With the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Delta Variant Is Associated With Higher Recovery of Infectious Virus Compared to the Alpha Variant in Both Unvaccinated and Vaccinated Individuals<ul><li> Mutations were visually confirmed with Integrated Genomics viewer (IGV) (Version 2.8.10), clades were determined using <b>Nextclade</b> beta v 0.12.0 (clades.nextstrain.org), and lineages were determined with Pangolin COVID-19 lineage Assigner (COG-UK (cog-uk.io)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34960813">34960813</a> <b>(1)</b> Prediction of SARS-CoV-2 Variant Lineages Using the S1-Encoding Region Sequence Obtained by PacBio Single-Molecule Real-Time Sequencing<ul><li> Clades were identified by <b>NextClade</b> and lineages were determined using the PANGOLIN classification based on the analysis of full-length genome sequences (<xref rid="viruses-13-02544-t002" ref-type="table"><u>Table 2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34976008">34976008</a> <b>(1)</b> Geographical Landscape and Transmission Dynamics of SARS-CoV-2 Variants Across India: A Longitudinal Perspective<ul><li> For clade classification assignment, we used the <b>NextClade</b> command line interface (<xref rid="B14" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), and the lineage classification was performed with the Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) using command line (<xref rid="B33" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34975897">34975897</a> <b>(1)</b> Anti-SARS-CoV-2 Immunoglobulin Isotypes, and Neutralization Activity Against Viral Variants, According to BNT162b2-Vaccination and Infection History<ul><li> SARS-CoV-2 sequences were classified using the Pangolin COVID-19 Lineage Assigner tool v.2.3.2 (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>) and <b>Nextclade</b> v.0.14.1 (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34923158">34923158</a> <b>(2)</b> Genome sequencing of SARS-CoV-2 reveals the prevalence of variant B 1.1.7 in Egypt<ul><li> Then <b>Nextclade</b> Tool (version 1.5.2) was used for viral genome alignment, mutation calling, quality checks, phylogenetic placement, and clade assignment.<li>The genetic mutation using <b>Nextclade</b> Tool showed the viral genome, including ORF1a, ORF1b, ORF3a, ORF6, ORF7a, ORF7b, ORF8, S, E, M, and N.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34946210">34946210</a> <b>(1)</b> Genomic Epidemiology of SARS-CoV-2 Divulge B.1, B.1.36, and B.1.1.7 as the Most Dominant Lineages in First, Second, and Third Wave of SARS-CoV-2 Infections in Pakistan<ul><li> A coverage map was created using <b>Nextclade</b> (version 0.7.5), which performs banded Smith&#x2013;Waterman alignment with an affine gap penalty.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35262032">35262032</a> <b>(3)</b> SARS-CoV-2 loads in urine, sera and stool specimens in association with clinical features of COVID-19 patients<ul><li> The FASTA file with consensus sequences was uploaded in the <b>Nextclade</b> Beta (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0001">https://clades.nextstrain.org/</ext-link>) <xref rid="bib0011" ref-type="bibr"><u>[11]</u></xref> to identify mutations profile, considering PANGO linages.<li> By submitting the nucleotide sequences on the <b>NextClade</b> platform, it was possible to observe the profile of mutations found in these genomes.<li>The profile of mutations was obtained by <b>NextClade</b> database platform.<li> [[11]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34913861">34913861</a> <b>(2)</b> Context-aware genomic surveillance reveals hidden transmission of a carbapenemase-producing <italic toggle="yes">Klebsiella pneumoniae</italic><ul><li> Positive selection was assessed by first aligning all sequences for a particular gene using <named-content content-type="mono-space">nextalign</named-content> (no version, unpublished, <ext-link xlink:href="https://github.com/nextstrain/<b>nextclade</b>" ext-link-type="uri">github.com/nextstrain/<b>nextclade</b></ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34946199">34946199</a> <b>(2)</b> Simplified Point-of-Care Full SARS-CoV-2 Genome Sequencing Using Nanopore Technology<ul><li>Consensus FASTA files were uploaded to the Nextstrain webapp (<b>Nextclade</b>.<li>Single or batch high-quality consensus FASTA formatted sequences were used for phylogenetic tree visualization and variant calling using the Nextstrain webapp (<b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34931174">34931174</a> <b>(1)</b> Striking lineage diversity of severe acute respiratory syndrome coronavirus 2 from non-human sources<ul><li> These reference genomes were downloaded from <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir0015">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34911747">34911747</a> <b>(2)</b> Monitoring the SPREAD of the SARS-CoV-2 lineage B.1.621 in Udine, Italy<ul><li> As evidenced in <xref rid="F1" ref-type="fig"><u>figure 1A</u></xref>, our samples overlap with the other B.1.621 sequences in <b>Nextclade</b> V.2.29.1web application (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<li> (A) Clades distribution according to the <b>Nextclade</b> online tool.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34910368">34910368</a> <b>(4)</b> Transmission of SARS&#x2010;CoV&#x2010;2 from humans to a 16&#x2010;year&#x2010;old domestic cat with comorbidities in Pennsylvania, USA<ul><li>The data sets of 1343 SARS&#x2010;CoV&#x2010;2 genomes deposited in GISAID from North America (sampled between December 2019 and December 2020) were used for phylogenetic analysis using the standard protocol from Nextstrain/<b>Nextclade</b> (<ext-link xlink:href="https://nextstrain.org/sars-cov-2/" ext-link-type="uri">https://nextstrain.org/sars&#x2010;cov&#x2010;2/</ext-link>).<li> The <b>NextClade</b> analysis (Bosco&#x2010;Lauth et&#xA0;al., <xref rid="vms3695-bib-0003" ref-type="bibr"><u>2020</u></xref>; Shi et&#xA0;al., <xref rid="vms3695-bib-0031" ref-type="bibr"><u>2020</u></xref>; Oude Munnink et&#xA0;al., <xref rid="vms3695-bib-0024" ref-type="bibr"><u>2021</u></xref>) performed confirmed the clade as GH defined by D614G and Q57H mutations (Pereson et&#xA0;al., <xref rid="vms3695-bib-0026" ref-type="bibr"><u>2021</u></xref>).<li> <b>NextClade</b> analysis performed confirmed the clade as GH defined by D614G and Q57H mutations.<li> The <b>NextClade</b> analysis performed confirmed the clade as GH defined by D614G and Q57H mutations.<li> [2020](<string-name><surname>Bosco&#x2010;Lauth</surname>, <given-names>A. M.</given-names></string-name>, <string-name><surname>Hartwig</surname>, <given-names>A. E.</given-names></string-name>, <string-name><surname>Porter</surname>, <given-names>S. M.</given-names></string-name>, <string-name><surname>Gordy</surname>, <given-names>P. W.</given-names></string-name>, <string-name><surname>Nehring</surname>, <given-names>M.</given-names></string-name>, <string-name><surname>Byas</surname>, <given-names>A. D.</given-names></string-name>, <string-name><surname>VandeWoude</surname>, <given-names>S.</given-names></string-name>, <string-name><surname>Ragan</surname>, <given-names>I. K.</given-names></string-name>, <string-name><surname>Maison</surname>, <given-names>R. M.</given-names></string-name>, &amp; <string-name><surname>Bowen</surname>, <given-names>R. A.</given-names></string-name> (<year>2020</year>). <article-title>Pathogenesis, transmission and response to re&#x2010;exposure of SARS&#x2010;CoV&#x2010;2 in domestic cats</article-title>. <italic toggle="yes">bioRxiv</italic>2020.05.28.120998.) [2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32269068">32269068</a>) [2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33172935">33172935</a>) [2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32966646">32966646</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35039782">35039782</a> <b>(1)</b> Identification of a novel SARS-CoV-2 P.1 sub-lineage in Brazil provides new insights about the mechanisms of emergence of variants of concern<ul><li> 2017</xref>) and their mutational profile was investigated using the <b>Nextclade</b> tool (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34946187">34946187</a> <b>(2)</b> Comprehensive Contact Tracing, Testing and Sequencing Show Limited Transmission of SARS-CoV-2 between Children in Schools in Norway, August 2020 to May 2021<ul><li> Sequences were aligned in a codon-aware manner to the reference genome Wuhan-Hu-1 (GenBank accession no: MN908947.3) using the program Nextalign v.0.2.0 (available at <uri xlink:href="https://github.com/nextstrain/<b>nextclade</b>">https://github.com/nextstrain/<b>nextclade</b></uri>, accessed on 1 December 2021), and a maximum-likelihood phylogenetic tree was created using IQTREE v.2.0.3 (available at <uri xlink:href="https://iqtree.org">https://iqtree.org</uri>, accessed on 1 December 2021) [<xref rid="B18-microorganisms-09-02587" ref-type="bibr"><u>18</u></xref>].<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34905574">34905574</a> <b>(3)</b> Local occurrence and fast spread of B.1.1.7 lineage: A glimpse into Friuli Venezia Giulia<ul><li> Clade and lineages of 479 specimens were defined with the <b>Nextclade</b> v2.29.1 web application (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>), the GISAID CoVsurver mutation App (<ext-link xlink:href="https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/" ext-link-type="uri">https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/</ext-link>), the pangolin web app developed by the Centre for Genomic Pathogen Surveillance&#x2013;version v.3.1.11 - (<ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) and the Ultrafast Sample placement on Existing tRee (UShER) web tool by UCSC (<ext-link xlink:href="https://hgwdev-angie.gi.ucsc.edu/cgi-bin/hgPhyloPlace" ext-link-type="uri">https://hgwdev-angie.gi.ucsc.edu/cgi-bin/hgPhyloPlace</ext-link>).<li> Eventually, our 479 sequences have been analyzed with the <b>Nextclade</b> online tool in order to map them within the SARS-CoV-2 global landscape (<xref rid="pone.0261229.g002" ref-type="fig"><u>Fig 2</u></xref>, panel C).<li> C, clades distribution according to the <b>Nextclade</b> online tool.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34931192">34931192</a> <b>(1)</b> Delta breakthrough infections elicit potent, broad and durable neutralizing antibody responses<ul><li> Sequence quality was determined using <b>Nextclade</b> version 1.0.0-alpha.8 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34915422">34915422</a> <b>(1)</b> Omicron SARS-CoV-2 variant: Unique features and their impact on pre-existing antibodies<ul><li>We used the high-quality and complete sequences (n&#xA0;=&#xA0;77) of the Omicron (B.1.1.529) variant downloaded from the GISAID repository (November 26, 2021) [<xref rid="bib8" ref-type="bibr"><u>8</u></xref>], and processed the sequences using the <b>NextClade</b> CLI [<xref rid="bib9" ref-type="bibr"><u>9</u></xref>] and/or an in-house Python script to calculate the prevalence of mutations.<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34897035">34897035</a> <b>(2)</b> Multicenter study evaluating one multiplex RT-PCR assay to detect SARS-CoV-2, influenza A/B, and respiratory syncytia virus using the LabTurbo AIO open platform: epidemiological features, automated sample-to-result, and high-throughput testing<ul><li>To identify pathogen evolution relationships, assembly sequences were uploaded to the <b>Nextclade</b> website (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) developed by Nextstrain [<xref rid="r28" ref-type="bibr"><u>28</u></xref>].<li> TSGH sequences were aligned to the reference sequences of major clades of SARS-CoV (20I) grouped using Nextstrain and phylogenetic tree annotated with these alignment definitions, using lists of grouped clade-defining mutations via Augur workflow [<xref rid="r29" ref-type="bibr"><u>29</u></xref>] supported by <b>Nextclade</b>.<li> [28](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [29](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34189396">34189396</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34893946">34893946</a> <b>(3)</b> Case Series: Convalescent Plasma Therapy for Patients with COVID-19 and Primary Antibody Deficiency<ul><li> The clades were classified with the webservers of <b>Nextclade</b> (clades.nextstrain.org) and Pangolin (pangolin.cog-uk.io).<li> Phylogenetic analysis clustered all three patients into the clade 20B (<b>Nextclade</b> nomenclature) (Fig.&#xA0;<xref rid="Fig2" ref-type="fig"><u>2</u></xref>) which was prevalent in Germany at the time of the initial infection (October to November).<li> The circularized maximum-likelihood phylogenetic tree was constructed with IQ-Tree (GTR&#x2009;+&#x2009;F&#x2009;+&#x2009;I) and rooted on the Wuhan-Hu-1 reference sequence (NC_045512), tree branches were colored according to their <b>Nextclade</b> classification.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34960743">34960743</a> <b>(1)</b> Nucleocapsid (N) Gene Mutations of SARS-CoV-2 Can Affect Real-Time RT-PCR Diagnostic and Impact False-Negative Results<ul><li> Viral strains were classified and mutations analyzed using the software tools Pangolin (<uri xlink:href="http://pangolin.cog-uk.io/">http://pangolin.cog-uk.io/</uri>, accessed on 5 August 2021), and <b>nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 5 August 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34881982">34881982</a> <b>(3)</b> SARS-CoV-2 Genome Sequences, Including One with an ORF7a Deletion, Obtained from Patients in Bangladesh<ul><li>The sequences were analyzed and assigned to clades using the <b>Nextclade</b> Web service (<xref rid="B6" ref-type="bibr"><u>6</u></xref>).<li> The phylogenetic analysis of 1,139 sequences originating from Bangladesh available in the Global Initiative on Sharing Avian Influenza Data (GISAID) database in June 2021, with a guide reference tree provided by <b>Nextclade</b>, was conducted and visualized using <b>Nextclade</b>.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34943537">34943537</a> <b>(1)</b> Clinical Evaluation of a New Antigen-Based COVID-19 Rapid Diagnostic Test from Symptomatic Patients<ul><li> The SARS-CoV-2 clade was detected using the <b>Nextclade</b> v1.5.2 (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 3 October 2021)) with default parameters [<xref rid="B25-diagnostics-11-02300" ref-type="bibr"><u>25</u></xref>].<li> [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34880295">34880295</a> <b>(1)</b> Emergence of novel combinations of SARS-CoV-2 spike receptor binding domain variants in Senegal<ul><li> Genomes were assembled using BLAST and sequence NC_045512 as a reference, followed by clade assignment and mutation analysis with the <b>NextClade</b> tool (clades.nextstrain.org) and lineage assignments with the Pangolin tool<sup><xref ref-type="bibr" rid="CR12"><u>12</u></xref></sup>.<li> [12](O&#x2019;Toole, &#xC1;., Scher, E., Underwood, A., Jackson, B., Hill, V., McCrone, J. T., Ruis, C., Abu-Dahab, K., Taylor, B., Yeats, C., du Plessis, L., Aanensen, D., Holmes, E., Pybus, O. &amp; Rambaut, A. Pangolin: Lineage assignment in an emerging pandemic as an epidemiological tool. githubcom/cov-lineages/pangolin (2021).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34952247">34952247</a> <b>(1)</b> Genomic analysis of SARS-CoV-2 variants of concern identified from the ChAdOx1 nCoV-19 immunized patients from Southwest part of Bangladesh<ul><li> Mutations and amino acid substitutions were retrieved from <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intr0005">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34908852">34908852</a> <b>(2)</b> First Report of SARS-CoV-2 Lineage B.1.1.7 (Alpha Variant) in Ecuador, January 2021<ul><li> The genome was processed using the Artic bioinformatics pipeline, assembled in the online tool Genome Detective using default parameters<xref rid="cit0009" ref-type="bibr"><u>9</u></xref> and submitted to <b>Nextclade</b> (<underline><ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link></underline>) for preliminary taxonomic analysis.<li> Following the <b>Nextclade</b> analysis, the annotated genome clustered into 501.YV1 clade which correspond to Alpha VOC.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33298935">33298935</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34854726">34854726</a> <b>(2)</b> Genome Sequencing of the SARS-CoV-2 Delta (B.1.617.2) Variant of Concern Detected in Bangladesh<ul><li> The Pangolin lineage was determined and the phylogenetic analysis performed using the default parameters of <b>Nextclade</b> version 1.7.1 (<ext-link xlink:href="https://clades.nextstrain.org/tree" ext-link-type="uri">https://clades.nextstrain.org/tree</ext-link>) (<xref rid="B7" ref-type="bibr"><u>7</u></xref>).<li> The FASTA file was uploaded in <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/results" ext-link-type="uri">https://clades.nextstrain.org/results</ext-link>), and a phylogenetic tree was created using <ext-link xlink:href="https://www.ncbi.nlm.nih.gov/nuccore/MZ437368" ext-link-type="uri" assigning-authority="insdc">MZ437368</ext-link> to represent the Delta lineage (B.1.617.2/20A/S:478K) in Bangladesh.<li> [7](<collab>Nextstrain Developers</collab>. <year>2020</year>. <article-title>Nextclade: clade assignment, mutation calling, and sequence quality checks</article-title>. <ext-link xlink:href="https://clades.nextstrain.org/tree" ext-link-type="uri">https://clades.nextstrain.org/tree</ext-link>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34854728">34854728</a> <b>(2)</b> Coding-Complete Genome Sequences of Alpha and Delta SARS-CoV-2 Variants from Kamphaeng Phet Province, Thailand, from May to July 2021<ul><li> <b>Nextclade</b> v1.6.0 (<xref rid="B14" ref-type="bibr"><u>14</u></xref>) was used to identify variants.<li> <b>Nextclade</b> v1.6.0 (<xref rid="B14" ref-type="bibr"><u>14</u></xref>) was used to identify variants.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34855869">34855869</a> <b>(6)</b> Patient health records and whole viral genomes from an early SARS-CoV-2 outbreak in a Quebec hospital reveal features associated with favorable outcomes<ul><li> The most popular lineage assignment tools are Pangolin [<xref rid="pone.0260714.ref012" ref-type="bibr"><u>12</u></xref>], which uses a supervised learning approach to classify sequences, and Nextstrain/<b>Nextclade</b> [<xref rid="pone.0260714.ref013" ref-type="bibr"><u>13</u></xref>], which use condensed phylogeny and phylogenetic placement methods, respectively.<li> <b>NextClade</b> lineage assignment was more conservative, emitting classifications for 205 genomes: 7x 19A, 169x 20A and 29x 20C (<xref rid="pone.0260714.s008" ref-type="supplementary-material"><u>S1 Table</u></xref>).<li>However, discrepancies were observed between the phylogenetic analysis and Pangolin/<b>NextClade</b> lineage classification, prompting us to employ an alternative genotyping strategy by grouping viral haplotypes based on the 22 most common (allele frequency above 10%) SARS-CoV-2 variants observed in the global community between December 2019 and July 2020, as reported in GISAID on January 20th 2021 (see <xref rid="sec012" ref-type="sec"><u>Methods</u></xref>).<li> The resulting haplotype groups were more consistent with the phylogenetic analysis than the Pangolin/<b>NextClade</b> classifications, prompting us to retain this genotyping strategy in subsequent analyses.<li>The genomic PHATE embeddings clearly delineate two subclades of haplotype group II, which are supported by the phylogenetic analysis but not by Pangolin lineage assignment (almost all genomes classified as B.1) or <b>NextClade</b> (all 20C).<li> However, we found that different lineage assignment methods we employed (Pangolin, <b>NextClade</b>, phylogeny, dominant haplogroups and PHATE dimensionality reduction) produced disparate results.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Patient health records and whole viral genomes from an early SARS-CoV-2 outbreak in a Quebec hospital reveal features associated with favorable outcomes<ul><li> In fact, there are clade assignment methodologies other than pangoling such as Cov-Glue and <b>NextClade</b>.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(5)</b> Patient health records and whole viral genomes from an early SARS-CoV-2 outbreak in a Quebec hospital reveal features associated with favorable outcomes<ul><li> The reviewer might appreciate that this methodology is very similar to the ones proposed by pangolin and <b>nextclade</b>, where a set of discriminative mutation features are used to categorize viral subclades (continued in the next response).<li> In fact, there are clade assignment methodologies other than pangoling such as Cov-Glue and <b>NextClade</b>.<li> The haplogroup method we describe is in essence highly similar to the <b>NextClade</b> method (established in September 2020; not available when we started the study).<li> We have nonetheless added the <b>NextClade</b> assignments to Table S1 in the revised manuscript, next to Pangolin and Haplogroup classifications, as well as in the Methods section.<li> Moreover, following the reviewers interest in describing amino acid variants, we have uploaded the metadata and .json configuration files associated with the NextStrain/Auspice visualization we report in the Supplementary Figure 2 to the study&#x2019;s github repository so that users can explore these data interactively using the NextStrain software packages (alternatively, the fasta sequences can be directly uploaded to the <b>NextClade</b> webportal).<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35017872">35017872</a> <b>(1)</b> Phylodynamic Pattern of Genetic Clusters, Paradigm Shift on Spatio-Temporal Distribution of Clades, and Impact of Spike Glycoprotein Mutations of SARS-CoV-2 Isolates from India<ul><li> The <b>Nextclade</b>-Nextstrain pipeline (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) was used for studies on phylogenetic analysis and clustering patterns of the S gene.[<xref rid="ref3" ref-type="bibr"><u>3</u></xref>] Further, the Average evolutionary divergence was estimated using Kimura-2 parameter model.<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34811754">34811754</a> <b>(1)</b> Dynamics of spreading of SARS&#x2010;CoV&#x2010;2 in a Belgian hemodialysis facility: The importance of the analysis of viral strains<ul><li><b>Nextclade</b> Beta version 0.8.1 was used as a first sequence aligner, allowing comparison to other documented SARS&#x2010;CoV&#x2010;2 strains and clade assignment (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34873595">34873595</a> <b>(1)</b> Tracking SARS-COV-2 Variants Using Nanopore Sequencing in Ukraine in Summer 2021<ul><li> Mutation calling and sequence quality were analysed using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and Geneious Prime 2021.2.2 software (Biomatters, New Zealand).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34863056">34863056</a> <b>(1)</b> Performance of a high-throughput, automated enzyme immunoassay for the detection of SARS-CoV-2 antigen, including in viral &#x201C;variants of concern&#x201D;: Implications for clinical use<ul><li> Lineages and clades were interpreted using Pangolin and <b>NextClade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34839413">34839413</a> <b>(2)</b> Occurrence of a substitution or deletion of SARS-CoV-2 spike amino acid 677 in various lineages in Marseille, France<ul><li> The nature and number of nucleotide changes in the SARS-CoV-2 genomes and of amino acid changes in the SARS-CoV-2 proteins were obtained using the <b>Nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/results">https://clades.nextstrain.org/results</ext-link>).<li> Classification into Nextstrain clades was performed with the <b>Nextclade</b> online tool at URL: <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) and classification into Pangolin lineages was performed using the Pangolin online tool at URL: <ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/pangolin.html">https://cov-lineages.org/pangolin.html</ext-link> [<xref ref-type="bibr" rid="CR22"><u>22</u></xref>].<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34848213">34848213</a> <b>(1)</b> Genomic monitoring of the SARS-CoV-2 B1.1.7 (WHO VOC Alpha) in the Sao Paulo state, Brazil<ul><li> We assessed genome consensus sequence quality using <b>Nextclade</b> v0.8.1 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="interref0005">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34895912">34895912</a> <b>(1)</b> Parenchymal involvement on CT pulmonary angiography in SARS-CoV-2 Alpha variant infection and correlation of COVID-19 CT severity score with clinical disease severity and short-term prognosis in a UK cohort<ul><li> Consensus sequences were generated using ARTIC field bioinformatics v1.2.1<xref rid="bib26" ref-type="bibr"><sup>26</sup></xref> All sequences underwent quality control, requiring &gt;50% consensus genome coverage at &#x2265;20 depth, and agreement between Pangolin and <b>Nextclade</b> v2.3.24 assignments of lineage B.1.1.7.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34833135">34833135</a> <b>(1)</b> Emerging Mutations Potentially Related to SARS-CoV-2 Immune Escape: The Case of a Long-Term Patient<ul><li> <b>Nextclade</b> and Pangolin analyses assigned the surveyed genome sequence to clade 20E (EU1) and lineage B.1.177.51, respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34788739">34788739</a> <b>(2)</b> The Dominance of Severe Acute Respiratory Syndrome Coronavirus 2 B.1.617 and Its Sublineages and Associations with Mortality during the COVID-19 Pandemic in India between 2020 and 2021<ul><li>A phylogenetic dendrogram was constructed based on the complete genome sequences of 15740 Indian SARS-COV-2 strains using the <b>Nextclade</b> web-based tool (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<li> The selected sequences were uploaded to <b>Nextclade</b> webtool and aligned against genome sequence of SARS-COV-2 (Wuhan-Hu-1/2019) to assign Nextstrain clades and construct phylogeny trees of Indian SARS-COV-2 strains.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34798321">34798321</a> <b>(1)</b> SARS-CoV-2 introduction and lineage dynamics across three epidemic peaks in Southern Brazil: massive spread of P.1<ul><li> Consensus sequences were submitted to <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="ir0010">https://clades.nextstrain.org</ext-link>) for quality inspection and to determine the mutational profile.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34766651">34766651</a> <b>(2)</b> SARS&#x2010;CoV&#x2010;2 AY.4.2 variant circulating in Italy: Genomic preliminary insight<ul><li> The mutation pattern of the VUI was analyzed using the <b>NextClade</b> online tool.<li> Additionally, using <b>NextClade</b>, the mutation pattern of the newly generated strains (Figure&#xA0;<xref rid="jmv27451-fig-0001" ref-type="fig"><u>1C</u></xref>) was determined, and no newly acquired mutations were identified.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34787497">34787497</a> <b>(1)</b> SARS-CoV-2 Variants and Their Relevant Mutational Profiles: Update Summer 2021<ul><li> The alignment of sequences containing no nucleotide ambiguities was imported into and analyzed in the <b>Nextclade</b> tool in order to define mutations and to reveal which variant they belong to, while Seqscape software was used for sequences containing nucleotide ambiguities.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34787439">34787439</a> <b>(1)</b> Postvaccination SARS-COV-2 among Health Care Workers in New Jersey: A Genomic Epidemiological Study<ul><li>The resulting SARS-CoV-2 viral genome sequences were characterized by <b>Nextclade</b> CLI (v1) (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) to assign Nextstrain clades (<xref rid="B37" ref-type="bibr"><u>37</u></xref>).<li> [37](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34787498">34787498</a> <b>(2)</b> Prolonged SARS-CoV-2 Positivity in Immunocompetent Patients: Virus Isolation, Genomic Integrity, and Transmission Risk<ul><li> (A) Panel exhibiting all nonsynonymous mutations identified in each sequence by <b>Nextclade</b> v1.7.1 (&#x201C;x&#x201D; marks presence), indicating the high degree of similarity of the characterized genomes.<li>The novel genome sequences were classified with the pangolin tool v3.1.11 (pango v1.2.76, pangoLEARN model from 17 September 2021) (<xref rid="B44" ref-type="bibr"><u>44</u></xref>) and the <b>NextClade</b> Web application v1.7.1 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>), which was also used to annotate all mutations.<li> [44](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34789293">34789293</a> <b>(2)</b> Re-emergence of Gamma-like-II and emergence of Gamma-S:E661D SARS-CoV-2 lineages in the south of Brazil after the 2021 outbreak<ul><li> Variants were identified after read mapping in the reference genome for SARS-CoV-2 (RefSeq accession number NC <underline>0</underline>45512), and the analysis was performed on the Pangolin and <b>Nextclade</b> platforms.<li> In parallel, we also assigned the viral lineages using the <b>Nextclade</b> web application (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34828410">34828410</a> <b>(1)</b> Relative Consolidation of the Kappa Variant Pre-Dates the Massive Second Wave of COVID-19 in India<ul><li> This alignment was mapped on the global all SARS-CoV-2 sequence alignment in the <b>Nextclade</b> database to determine the lineage and clade of the 20 sequenced samples.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34785663">34785663</a> <b>(3)</b> Organ-specific genome diversity of replication-competent SARS-CoV-2<ul><li> Clades for each sequence were assigned via <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>/">https://github.com/nextstrain/<b>nextclade</b>/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34755363">34755363</a> <b>(1)</b> Evolution of spike mutations following antibody treatment in two immunocompromised patients with persistent COVID&#x2010;19 infection<ul><li> The global phylogenetic placement was determined using the <b>Nextclade</b> database (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34757384">34757384</a> <b>(1)</b> A stem-loop RNA RIG-I agonist protects against acute and chronic SARS-CoV-2 infection in mice<ul><li> We used <b>Nextclade</b> v0.14.2 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) to generate a phylogenetic tree and to compile a list of amino acid changes in the virus isolates as compared with the Wuhan-Hu-1 reference strain (<xref rid="sup1" ref-type="supplementary-material"><u>Table S1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34757863">34757863</a> <b>(1)</b> Whole-genome sequencing of SARS-COV-2 showed wide spread of B.1.525 in February 2021 in Libya<ul><li> Determination of strains, number of variants in coding and non-coding regions, as well as evolutionary origin and geolocation was performed through <b>Nextclade</b> and Pangolin applications [<xref rid="cit0006" ref-type="bibr"><u>6</u></xref>].<li> [6](<person-group person-group-type="author"><string-name><surname>O&#x2019;Toole</surname><given-names>&#xC1;</given-names></string-name>, <string-name><surname>Scher</surname><given-names>E</given-names></string-name>, <string-name><surname>Underwood</surname><given-names>A</given-names></string-name></person-group>, <italic toggle="yes">et al</italic>. <article-title>Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool</article-title>. <source>Virus Evol</source>. <year>2021</year>;<volume>7</volume>. DOI:<pub-id pub-id-type="doi">10.1093/VE/VEAB064</pub-id>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34743764">34743764</a> <b>(1)</b> Transmission of severe acute respiratory coronavirus virus 2 (SARS-CoV-2), delta variant, between two fully vaccinated healthcare personnel<ul><li> (B) Genomic alignment of the 2 HCW genomes that shows the amino acid changes compared to the reference genome using <b>Nextclade</b> version 1.5.2 software.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34755818">34755818</a> <b>(2)</b> SARS-CoV-2 variant N.9 identified in Rio de Janeiro, Brazil<ul><li> The mutational profile was investigated using the <b>Nextclade</b> v.0.14.2 (https://clades.nextstrain.org, <b>Nextclade</b> 2020-2021 Nextstrain developers), and the CoVSurver mutations App (https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/, A*STAR Bioinformatics Institute, Singapore).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34835044">34835044</a> <b>(4)</b> Circulation and Evolution of SARS-CoV-2 in India: Let the Data Speak<ul><li> Lineages were assigned to these entries using <b>NextClade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>; Last accessed 30 October 2021) [<xref rid="B6-viruses-13-02238" ref-type="bibr"><u>6</u></xref>].<li> The equivalence of WHO lineages with the clade definitions of <b>NextClade</b> and GISAID is provided for Dataset_3 (<xref rid="app1-viruses-13-02238" ref-type="app"><u>Supplementary File S4</u></xref>).<li> Overall, the predominance of <b>NextClade</b> lineages 20A, 20B, and 21A was observed, which corresponds to GISAID clades G, GR, and GRY in the pan-India dataset.<li> However, no data corresponding to WHO lineages Gamma and Iota (<b>NextClade</b>: 20J/501Y.V3 and 21F; GISAID: G, GR (Gamma), GH (Iota)) are part of our study, as they were eliminated during curation due to poor sequencing coverage and quality.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34988250">34988250</a> <b>(1)</b> In-depth Characterization of Vaccine Breakthrough Infections With SARS-CoV-2 Among Health Care Workers in a Dutch Academic Medical Center<ul><li> <b>Nextclade</b> (version 1.1.0) was used for phylogenetic analysis of the IRMA consensus sequences under the terms and conditions of the Creative Commons (CC BY) license (<ext-link xlink:href="https://creativecommons.org/licenses/by/4.0/" ext-link-type="uri">https://creativecommons.org/licenses/by/4.0/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34737266">34737266</a> <b>(1)</b> Within-host evolution of SARS-CoV-2 in an immunosuppressed COVID-19 patient as a source of immune escape variants<ul><li> The clades were classified with the webservers of <b>Nextclade</b> (clades.nextstrain.org) and Pangolin (pangolin.cog-uk.io).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34731234">34731234</a> <b>(3)</b> Clinical and Infection Prevention Applications of Severe Acute Respiratory Syndrome Coronavirus 2 Genotyping: An Infectious Diseases Society of America/American Society for Microbiology Consensus Review Document<ul><li>As most clinicians are now familiar with the predominant circulating VOIs and VOCs, it is reasonable to consider including variant lineages and clades in clinical reports, including whether they were derived from the PANGO or <b>Nextclade</b> classification systems, as well as the genotyping method used [<xref rid="CIT0009" ref-type="bibr"><u>9</u></xref>].<li>The data generated from these basic programs require additional analysis to produce PANGO lineage and <b>Nextclade</b> assignment as well as potential clinical interpretation of identified variants.<li> access to the ARTIC and <b>Nextclade</b> pipelines [<xref rid="CIT0013" ref-type="bibr"><u>13</u></xref>].<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [13](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33101244">33101244</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34742926">34742926</a> <b>(1)</b> Emergence of B.1.1.318 SARS-CoV-2 viral lineage and high incidence of alpha B.1.1.7 variant of concern in the Republic of Gabon<ul><li> SARS-CoV-2 genomes were classified into lineages using Phylogenetic Assignment of Named Global Outbreak LINeages (pangolin) (<xref rid="bib0005" ref-type="bibr"><u>O'Toole&#xA0;et&#xA0;al.&#xA0;2021</u></xref>) and viral clades by <b>Nextclade</b> Beta (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0002">https://clades.nextstrain.org/</ext-link>).<li> [O'Toole&#xA0;et&#xA0;al.&#xA0;2021](          <person-group person-group-type="author">            <name>              <surname>O'Toole</surname>              <given-names>&#xC1;ine</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>Emily</given-names>            </name>            <name>              <surname>Underwood</surname>              <given-names>Anthony</given-names>            </name>            <name>              <surname>Jackson</surname>              <given-names>Ben</given-names>            </name>            <name>              <surname>Hill</surname>              <given-names>Verity</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>John T</given-names>            </name>            <name>              <surname>Colquhoun</surname>              <given-names>Rachel</given-names>            </name>            <etal/>          </person-group>          <article-title>Assignment of Epidemiological Lineages in an Emerging Pandemic Using the Pangolin Tool</article-title>          <source>Virus Evolution</source>          <volume>7</volume>          <issue>2</issue>          <year>2021</year>          <fpage>1</fpage>          <lpage>9</lpage>          <pub-id pub-id-type="doi">10.1093/ve/veab064</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34728747">34728747</a> <b>(2)</b> Genomic diversity of SARS-CoV-2 in Oxford during United Kingdom&#x2019;s first national lockdown<ul><li> The phylogenetic tree was generated using <b>Nextclade</b> (version 0.14.2)<sup><xref ref-type="bibr" rid="CR27"><u>27</u></xref></sup>.<li>Phylogenetic analysis was performed via Nextstrain&#x2019;s online tool <b>Nextclade</b> (version 0.14.2).<li> [27](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34834987">34834987</a> <b>(1)</b> The Evolutionary Landscape of SARS-CoV-2 Variant B.1.1.519 and Its Clinical Impact in Mexico City<ul><li>According to the phylogenetic analysis, this variant is grouped in an independent clade derived from the clade 20B <b>NextClade</b> classification (<xref rid="viruses-13-02182-f001" ref-type="fig"><u>Figure 1</u></xref>C).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34718395">34718395</a> <b>(1)</b> A comprehensive review of the analysis and integration of omics data for SARS-CoV-2 and COVID-19<ul><li> The database provides the <b>Nextclade</b> tool to compare sequences to the SARS-CoV-2 reference sequence, assign them to clades and see where they fall on the SARS-CoV-2 tree [<xref rid="ref35" ref-type="bibr"><u>35</u></xref>].<li> [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34700068">34700068</a> <b>(3)</b> Dynamics of SARS-CoV-2 variants of concern (VOC) in Bangladesh during the first half of 2021<ul><li>The file of sequence data in FASTA format was then imported into the web-based software <b>Nextclade</b> v1.5.2 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0035">https://clades.nextstrain.org/</ext-link>) for quality assurance, clade assignment, lineage distribution, and diversity analysis.<li> A phylogenetic tree was generated from 820 high quality Bangladeshi SARS-CoV-2 strains in comparison with the global reference strains (n&#xA0;=&#xA0;1794) from different sub-continent that are automatically incorporated within the <b>Nextclade</b> platform (<xref rid="bib13" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The plot was generated in online Auspice platform (<ext-link ext-link-type="uri" xlink:href="https://auspice.us/" id="intref0090">https://auspice.us/</ext-link>) by given input from <b>Nextclade</b> generated &#x201C;JSON_Auspice&#x201D; data.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34672705">34672705</a> <b>(2)</b> Genome Sequences of SARS-CoV-2 Sublineage B.1.617.2 Strains from 12 Children in Chattogram, Bangladesh<ul><li> A consensus FASTA file was generated using a FASTQ QC + ARTIC + <b>Nextclade</b> r1.0.4 workflow on the cloud-based analysis platform EPI2ME Desktop Agent v3.3.0 with default parameters, based on ARTIC Field Bioinformatics software (<ext-link xlink:href="https://github.com/artic-network/fieldbioinformatics" ext-link-type="uri">https://github.com/artic-network/fieldbioinformatics</ext-link>).<li>A phylogenetic tree was constructed on 3 August 2021 using the <b>Nextclade</b> v1.5.2 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) with default parameters (<xref rid="B3" ref-type="bibr"><u>3</u></xref>).<li> [3](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34673905">34673905</a> <b>(1)</b> In-Depth Investigation of Conjunctival Swabs and Tear Fluid of Symptomatic COVID-19 Patients, an Observational Cohort Study<ul><li> Genome quality control was performed using <b>Nextclade</b>, phylogenetic analysis was carried out utilizing Nextstrain pipelines, phylogenetic tree visualization was generated in Auspice, and Pangolin (Phylogenetic Assignment of Named Global Outbreak Lineages) lineages were assigned using the Pangolin COVID-19 Lineage Assigner web application.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34834921">34834921</a> <b>(1)</b> Mutational Hotspot in the SARS-CoV-2 Spike Protein N-Terminal Domain Conferring Immune Escape Potential<ul><li> Clades were assigned to the genotypes using <b>Nextclade</b> (version 0.13.0) [<xref rid="B34-viruses-13-02114" ref-type="bibr"><u>34</u></xref>] and lineages using Pangolin (version 2.3.2) [<xref rid="B35-viruses-13-02114" ref-type="bibr"><u>35</u></xref>].<li> [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [35](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34678187">34678187</a> <b>(2)</b> SARS-CoV-2 sequencing collaboration in west Africa shows best practices<ul><li> <b>Nextclade</b> assigned clades to 43 sequences, whereas Pangolin assigned lineages to only 29 sequences.<li> 19 sequences had less than 3000 Ns on <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34696534">34696534</a> <b>(2)</b> Near-Complete Genome of SARS-CoV-2 Delta (AY.3) Variant Identified in a Dog in Kansas, USA<ul><li>SARS-CoV-2 sequence lineages were determined in Pangolin (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>) and the <b>Nextclade</b> tool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>; accessed 25 August 2021) assigned clades and were used to perform the phylogenetic analysis.<li>A phylogenetic tree was constructed from hCoV-19/dog/USA/KS-8074/2021 and the Kansas (human) delta variant references using the <b>Nextclade</b> bioinformatics pipeline (locally) (<xref rid="viruses-13-02104-f002" ref-type="fig"><u>Figure 2</u></xref>).</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34596904">34596904</a> <b>(1)</b> Temporal spread and evolution of SARS&#x2010;CoV&#x2010;2 in the second pandemic wave in Brazil<ul><li> We identified nucleotide and amino acids substitutions for P.1 and P.2 lineages using <b>Nextclade</b> v0.8.1 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and Pangolin v2 (<ext-link xlink:href="https://github.com/covlineages/pangolin" ext-link-type="uri">https://github.com/covlineages/pangolin</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34596257">34596257</a> <b>(1)</b> Surveillance genome sequencing reveals multiple SARS&#x2010;CoV&#x2010;2 variants circulating in central Texas, USA, with a predominance of delta variant and review of vaccine breakthrough cases<ul><li> Data were manually uploaded to the <b>NextClade</b> bioinformatics pipeline for analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34642697">34642697</a> <b>(2)</b> Year-long COVID-19 infection reveals within-host evolution of SARS-CoV-2 in a patient with B cell depletion<ul><li> Lineages were called using <b>Nextclade</b> and Pangolin software packages [<xref rid="R18" ref-type="bibr"><u>18</u></xref>, <xref rid="R20" ref-type="bibr"><u>20</u></xref>, <xref rid="R21" ref-type="bibr"><u>21</u></xref>].<li> Assembled consensus sequences of virus genomes were assigned lineage and all samples from this patient mapped to the Pango lineage B.1.332 (<b>Nextclade</b> lineage 20C).<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33514928">33514928</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34593004">34593004</a> <b>(1)</b> Transmission event of SARS-CoV-2 delta variant reveals multiple vaccine breakthrough infections<ul><li> Swift analysis pipeline produced variant vcf file, consensus genome, pangolin lineage, and <b>Nextclade</b> assignment (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34660520">34660520</a> <b>(1)</b> The Emergence of the New P.4 Lineage of SARS-CoV-2 With Spike L452R Mutation in Brazil<ul><li>Preliminary mutation analyses were performed using <b>Nextclade</b> (<xref rid="B25" ref-type="bibr"><u>25</u></xref>).<li> [25](<person-group person-group-type="author"><collab>Nextclade</collab></person-group>. Available online at: <ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link> (accessed May 19, 2021)) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34599175">34599175</a> <b>(2)</b> Emergence and spread of SARS-CoV-2 lineage B.1.620 with variant of concern-like mutations and deletions<ul><li>Lineage B.1.620 attracted our attention due to large numbers of unique mutations in B.1.620 genomes from Lithuania in <b>nextclade</b> analyses (its genomes are 18 mutations away from nearest relatives and 26 from reference strain Wuhan-Hu-1), and those genomes initially being assigned to clade 20A, corresponding to B.1 in Pangolin nomenclature<sup><xref ref-type="bibr" rid="CR4"><u>4</u></xref>,<xref ref-type="bibr" rid="CR5"><u>5</u></xref></sup>.<li> Initially identified cases of B.1.620 were mistakenly classified by pangolin as B.1.177 or B.1.177.57, while <b>nextclade</b><sup><xref ref-type="bibr" rid="CR34"><u>34</u></xref></sup> assigned it to clade 20A rather than the expected 20E (EU1), while highlighting that B.1.620 sequences bore many unique mutations compared to the closest sequence.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [5](O&#x2019;Toole, A. et al. Assignment of epidemiological lineages in an emerging pandemic using the pangolin tool. <italic>Virus Evol</italic>. <bold>7</bold>, veab064 (2021).) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34725568">34725568</a> <b>(4)</b> Epidemiological dynamics of SARS-CoV-2 VOC Gamma in Rio de Janeiro, Brazil<ul><li> <b>Nextclade</b> 20I/V1), Beta (Pango lineage B.1.351; <b>Nextclade</b> 20H/V2), Gamma (Pango lineage P.1; <b>Nextclade</b> 20J/V3), and Delta (Pango lineage B.1.617.2; <b>Nextclade</b> 21A) in the UK, South Africa, Brazil, and India, respectively (<xref rid="R50" ref-type="bibr"><u>Tegally et&#xA0;al., 2020</u></xref>; <xref rid="R7" ref-type="bibr"><u>Cherian et&#xA0;al., 2021</u></xref>; <xref rid="R54" ref-type="bibr"><u>Volz et&#xA0;al., 2021</u></xref>; <xref rid="R13" ref-type="bibr"><u>Faria et&#xA0;al., 2021</u></xref>; <xref rid="R33" ref-type="bibr"><u>Naveca et&#xA0;al., 2021</u></xref>).<li> [Tegally et&#xA0;al., 2020](<person-group person-group-type="author"><string-name><surname>Tegally</surname>&#xA0;<given-names>H.</given-names></string-name></person-group>&#xA0;<etal>et&#xA0;al.</etal> (<year>2020</year>) &#x2018;<article-title>Emergence and Rapid Spread of a New Severe Acute Respiratory Syndrome-related Coronavirus 2 (Sars-cov-2) Lineage with Multiple Spike Mutations in South Africa</article-title>&#x2019;. Preprint from medRxiv.doi: <pub-id pub-id-type="doi">10.1101/2020.12.21.20248640</pub-id>.) [Cherian et&#xA0;al., 2021](<person-group person-group-type="author"><string-name><surname>Cherian</surname>&#xA0;<given-names>S.</given-names></string-name></person-group>&#xA0;<etal>et&#xA0;al.</etal> (<year>2021</year>) &#x2018;<article-title>Convergent Evolution of SARS-CoV-2 Spike Mutations, L452R, E484Q and P681R, in the Second Wave of COVID-19 in Maharashtra, India</article-title>&#x2019;, <source><italic toggle="yes">BioRxiv</italic></source>.doi: <pub-id pub-id-type="doi">10.1101/2021.04.22.440932</pub-id>.) [Volz et&#xA0;al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33767447">33767447</a>) [Faria et&#xA0;al., 2021](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33853970">33853970</a>) [Naveca et&#xA0;al., 2021](<person-group person-group-type="author"><string-name><surname>Naveca</surname>&#xA0;<given-names>F. G.</given-names></string-name></person-group>&#xA0;<etal>et&#xA0;al.</etal> (<year>2021</year>) &#x2018;<article-title>COVID-19 in Amazonas, Brazil, Was Driven by the Persistence of Endemic Lineages and P.1 Emergence</article-title>&#x2019;, <source><italic toggle="yes">Nature Medicine</italic></source>, 27: 1230&#x2013;8.doi: <pub-id pub-id-type="doi">10.1038/s41591-021-01378-7</pub-id>.) </ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34631911">34631911</a> <b>(3)</b> High throughput nanopore sequencing of SARS-CoV-2 viral genomes from patient samples<ul><li> Finally, lineage and clade of the consensus genomes can be assigned using online or locally implemented tools such as Pangolin and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="www.pangolin.cog-uk.io">www.pangolin.cog-uk.io</ext-link>) [<xref rid="ref14" ref-type="bibr"><u>14</u></xref>].<li> A convenient way to observe sequences and determine the quality is to use the <b>NextClade</b> website (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).<li><bold>Working Protocol S13.</bold> Visualizing data in <b>Nextclade</b> and Nextstrain.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34680431">34680431</a> <b>(1)</b> OmniSARS2: A Highly Sensitive and Specific RT-qPCR-Based COVID-19 Diagnostic Method Designed to Withstand SARS-CoV-2 Lineage Evolution<ul><li> We then assigned them to a SARS-CoV-2 clade with <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri> (accessed on 6 July 2021)) and to a SARS-CoV-2 PANGO lineage [<xref rid="B5-biomedicines-09-01314" ref-type="bibr"><u>5</u></xref>] with Pangolin [<xref rid="B18-biomedicines-09-01314" ref-type="bibr"><u>18</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a> <b>(3)</b> OmniSARS2: A Highly Sensitive and Specific RT-qPCR-Based COVID-19 Diagnostic Method Designed to Withstand SARS-CoV-2 Lineage Evolution<ul><li> Variants were called using Snippy (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/tseemann/snippy" ext-link-type="uri">https://github.com/tseemann/snippy</ext-link>) and <b>Nextclade</b>.<li> Local Nextstrain/<b>Nextclade</b> v0.13.0 was also implemented for clade assignment and variant annotation.<li>Variants were called using Snippy (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/tseemann/snippy" ext-link-type="uri">https://github.com/tseemann/snippy</ext-link>) and <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34527285">34527285</a> <b>(1)</b> OmniSARS2: A Highly Sensitive and Specific RT-qPCR-Based COVID-19 Diagnostic Method Designed to Withstand SARS-CoV-2 Lineage Evolution<ul><li> <b>Nextclade</b> was used to identify the position of the mutation in the genome compared to the SARS-CoV-2 original Wuhan Hu1 strain (NC_045512).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34578452">34578452</a> <b>(1)</b> Multiple Occurrences of a 168-Nucleotide Deletion in SARS-CoV-2 ORF8, Unnoticed by Standard Amplicon Sequencing and Variant Calling Pipelines<ul><li> All sequences possess at least 21 SNVs compared to the Wuhan-Hu-1 reference (MN908947.3) and were assigned to Clades 20B by <b>Nextclade</b> (clades.nextstrain.org) and M.1 (B.1.1.294.1) by Pangolin (cov-lineages.org, accessed on 17 September 2021), respectively.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34578452">34578452</a> <b>(1)</b> Multiple Occurrences of a 168-Nucleotide Deletion in SARS-CoV-2 ORF8, Unnoticed by Standard Amplicon Sequencing and Variant Calling Pipelines<ul><li> <b>NextClade</b> v1.5.0 web tool (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">clades.nextstrain.org/</ext-link>) was employed for clade determination and mutation calling.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34535691">34535691</a> <b>(3)</b> Exploiting genomic surveillance to map the spatio-temporal dispersal of SARS-CoV-2 spike mutations in Belgium across 2020<ul><li>In addition to the three main steps detailed above, we used <b>Nextclade</b> v.0.10.1 (<ext-link ext-link-type="uri" xlink:href="https://www.npmjs.com/package/@neherlab/<b>nextclade</b>">https://www.npmjs.com/package/@neherlab/<b>nextclade</b></ext-link>) to identify all amino acid mutations relative to the Wuhan-Hu-1/2019 reference genome (MN908947.3), and build on our spatially-explicit phylogeographic reconstruction to map clades of the tree associated with specific mutations in the spike protein.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34536502">34536502</a> <b>(1)</b> Case-control study evaluating risk factors for SARS-CoV-2 outbreak amongst healthcare personnel at a tertiary care center<ul><li> <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="interref0001">https://clades.nextstrain.org</ext-link>) and GISAID EpiCoV (<ext-link ext-link-type="uri" xlink:href="https://www.epicov.org" id="interref0002">https://www.epicov.org</ext-link>) were then used to track the origin of this strain and assign it to its corresponding clade.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34760282">34760282</a> <b>(1)</b> Genomic epidemiology reveals the reduction of the introduction and spread ofSARS-CoV-2 after implementing control strategies in Republic of Korea,2020<ul><li> The lineages and clades ofSARS-CoV-2 sequences were assigned by PANGOLIN (<xref rid="R22" ref-type="bibr"><u>Rambautet&#xA0;al., 2020</u></xref>) (<ext-link xlink:href="https://cov-lineages.org/" ext-link-type="uri">https://cov-lineages.org/</ext-link>) and <b>Nextclade</b> Beta (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<li> [Rambautet&#xA0;al., 2020](<person-group person-group-type="author"><string-name><surname>Rambaut</surname><given-names>A.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2020</year>) &#x2018;<article-title>A Dynamic Nomenclature Proposalfor SARS-CoV-2 Lineages to Assist Genomic Epidemiology</article-title>&#x2019;,<source><italic toggle="yes">Nature Microbiology</italic></source>, <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>7</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=<person-group person-group-type="author"><string-name><surname>Rambaut</surname><given-names>A.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2020</year>) &#x2018;<article-title>A Dynamic Nomenclature Proposalfor SARS-CoV-2 Lineages to Assist Genomic Epidemiology</article-title>&#x2019;,<source><italic toggle="yes">Nature Microbiology</italic></source>, <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>7</lpage>."><person-group person-group-type="author"><string-name><surname>Rambaut</surname><given-names>A.</given-names></string-name></person-group><etal>et&#xA0;al.</etal> (<year>2020</year>) &#x2018;<article-title>A Dynamic Nomenclature Proposalfor SARS-CoV-2 Lineages to Assist Genomic Epidemiology</article-title>&#x2019;,<source><italic toggle="yes">Nature Microbiology</italic></source>, <volume>5</volume>:<fpage>1403</fpage>&#x2013;<lpage>7</lpage>.</a> <b>(3)</b> Genomic epidemiology reveals the reduction of the introduction and spread ofSARS-CoV-2 after implementing control strategies in Republic of Korea,2020<ul><li> Mutations of interest were determined using a local instance of <b>nextclade</b>-cli version 0.14.2 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" ext-link-type="uri">https://github.com/nextstrain/<b>nextclade</b></ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34488872">34488872</a> <b>(2)</b> Late reinfection with a different severe acute respiratory syndrome coronavirus-2 clade in a patient with refractory arterial hypertension: a case report<ul><li>Phylogenetic analysis of both specimens was done using web server Nextstrain (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">https://nextstrain.org/</ext-link>), with the SARS-CoV-2 database <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) to identify the clade, mutation calling, and phylogenetic placement of the SARS-CoV-2 genomes.<li> Phylogenetic analysis of both samples against GISAID genomes database using <b>Nextclade</b> web server analysis is shown in Fig.&#xA0;<xref rid="Fig2" ref-type="fig"><u>2</u></xref>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34488225">34488225</a> <b>(1)</b> SARS-CoV-2 B.1.617.2 Delta variant replication and immune evasion<ul><li> Following this, all sequences were passed through <b>Nextclade</b> v0.15 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) to determine the number of gap regions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34509927">34509927</a> <b>(2)</b> Evaluation of two RT-PCR screening assays for identifying SARS-CoV-2 variants<ul><li> Analysis included the SARS CoV-2 consensus sequence, plus generation of <b>Nextclade</b> and Pangolin lineage.<li> We used phylogenetic analysis of this alignment to confirm <b>Nextclade</b> and Pangolin analyses.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34509927">34509927</a> <b>(1)</b> Evaluation of two RT-PCR screening assays for identifying SARS-CoV-2 variants<ul><li> Using the <b>Nextclade</b> tool, the sequence was compared to the Wuhan-Hu-1 strain; a total of 23 mutations with 10 amino acid substitutions, three deletions, and one stop codon were observed in our analyzed sequence (<xref rid="tab1" ref-type="table"><u>Table&#xA0;1</u></xref>) (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34540874">34540874</a> <b>(3)</b> Whole Genome Sequencing of SARS-CoV-2 Strains in COVID-19 Patients From Djibouti Shows Novel Mutations and Clades Replacing Over Time<ul><li> Sequences were analyzed using <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) (<xref rid="B20" ref-type="bibr"><u>20</u></xref>) and an in-house Python script.<li>Phylogenetic tree (obtained using the Nexcladevv0.14.4 software (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>), including four South African variants of SARS-CoV-2 strains from the second COVID-19 wave in Djibouti compared to forty SARS-CoV-2 sequences (clade 20H) available through GISAID and automatically selected using <b>Nextclade</b>.<li> During the summer of 2020, the dynamic tracking of the SARS-CoV-2 mutants in the global pandemic highlighted a rapid evolution of the virus with an increase in the frequency of mutations and an epidemic pattern characterized by the generation of new variants including the Marseille 4 variant <b>Nextclade</b> 20A.EU2 (<xref rid="B7" ref-type="bibr"><u>7</u></xref>), the UK variant 20I/501Y.V1 (<xref rid="B8" ref-type="bibr"><u>8</u></xref>), the South African variant 20H/501Y.V2 (<xref rid="B9" ref-type="bibr"><u>9</u></xref>), the Brazilian variant 484K.V2 (<xref rid="B25" ref-type="bibr"><u>25</u></xref>), and others (<xref rid="B10" ref-type="bibr"><u>10</u></xref>).<li> [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33785459">33785459</a>) [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33383088">33383088</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33690265">33690265</a>) [25](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33548358">33548358</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33915216">33915216</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33835528">33835528</a> <b>(2)</b> Multiple clades of SARS&#x2010;CoV&#x2010;2 were introduced to Thailand during the first quarter of 2020<ul><li> The phylogenetic tree was constructed using <b>Nextclade</b> webapp (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).<li>Sequencing data were uploaded to <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>, accessed on April 4, 2020) for phylogenetic analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34468193">34468193</a> <b>(1)</b> Targeted Hybridization Capture of SARS-CoV-2 and Metagenomics Enables Genetic Variant Discovery and Nasal Microbiome Insights<ul><li> (D) Phylogenetic tree generated using <b>Nextclade</b> (version 04-22-21) and modified BioPhylo methods.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34469548">34469548</a> <b>(0)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li> These trees were obtained by running the command &#x201C;matUtils extract -i public-2021-06-09.all.masked.<b>nextclade</b>.pangolin.pb.gz -c B.1.1.7 -m G23012A -H &#x2018;(USA.*)&#x2019; -N 500 -j clade_trees -d clade_out,&#x201D; which selects all samples from clade B.1.1.7 which acquired this mutation and are from the United States, then identifies the minimum set of 500 sample subtrees which contain all of these samples, creating an Auspice v2 format JSON for each subtree (<xref rid="msab264-B8" ref-type="bibr"><u>Hadfield et al 2018</u></xref>).<li> [Hadfield et al 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li> Complete genomic sequences were analyzed with the <b>NextClade</b> online software (<uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>, accessed on 21 July 2021) to assign the clade and call amino acid mutations.<li> Phylogenetic analysis of the complete genomic sequences using the <b>NextClade</b> online software revealed four distinct lineages of SARS-CoV-2 (<xref rid="viruses-13-01743-f001" ref-type="fig"><u>Figure 1</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li> We identified three clades according to the <b>Nextclade</b> classification, i.e., 19A, 20A, and 20B.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li>The tree was constructed using <b>Nextclade</b>.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> A Daily-Updated Database and Tools for Comprehensive SARS-CoV-2 Mutation-Annotated Trees<ul><li> However, at the time of analysis, we ran all the sequences against the available genomes on <b>Nextclade</b> (n=4645) and constructed a phylogenetic tree.<li> We also added a phylogenetic tree showing all the genomes in <b>Nextclade</b> between march and April 2020.<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34448987">34448987</a> <b>(1)</b> A SARS-CoV-2 mutant from B.1.258 lineage with &#x2206;H69/&#x2206;V70 deletion in the Spike protein circulating in Central Europe in the fall 2020<ul><li> <b>Nextclade</b> lineages [<xref ref-type="bibr" rid="CR20"><u>20</u></xref>] in color.<li> [20](          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Global Chall</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Global Chall</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ">          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Global Chall</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        </a> <b>(2)</b> A SARS-CoV-2 mutant from B.1.258 lineage with &#x2206;H69/&#x2206;V70 deletion in the Spike protein circulating in Central Europe in the fall 2020<ul><li>Sequence typing tools developed so far such as Pangolin, <b>Nextclade</b>, Genome Detective Coronavirus Typing Tool, and COVID-19 Genotyping Tool (CGT; <xref rid="evab197-B25" ref-type="bibr">Hadfield et al.<li> <b>Nextclade</b> (<xref rid="evab197-B25" ref-type="bibr">Hadfield et al.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Global Chall</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        ">          <person-group person-group-type="author">            <name>              <surname>Elbe</surname>              <given-names>S</given-names>            </name>            <name>              <surname>Buckland-Merrett</surname>              <given-names>G</given-names>            </name>          </person-group>          <article-title>Data, disease and diplomacy: GISAID&#x2019;s innovative contribution to global health</article-title>          <source>Global Chall</source>          <year>2017</year>          <volume>1</volume>          <fpage>33</fpage>          <lpage>46</lpage>          <pub-id pub-id-type="doi">10.1002/gch2.1018</pub-id>        </a> <b>(1)</b> A SARS-CoV-2 mutant from B.1.258 lineage with &#x2206;H69/&#x2206;V70 deletion in the Spike protein circulating in Central Europe in the fall 2020<ul><li>The resulting SARS-CoV-2 viral genome sequences were uploaded to <b>Nextclade</b> server (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) to assign Nextstrain clades (<xref rid="B23" ref-type="bibr"><u>23</u></xref>).<li> [23](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34428777">34428777</a> <b>(2)</b> Emergence and expansion of SARS-CoV-2 B.1.526 after identification in New York<ul><li> <xref rid="Fig2" ref-type="fig"><u>2a</u></xref>) was conducted using the Nextstrain<sup><xref ref-type="bibr" rid="CR32"><u>32</u></xref></sup> workflow at <ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/ncov">https://github.com/nextstrain/ncov</ext-link>, which aligns sequences against the Wuhan Hu-1 reference using nextalign (<ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>">https://github.com/nextstrain/<b>nextclade</b></ext-link>), constructs a maximum-likelihood phylogenetic tree via IQ-TREE<sup><xref ref-type="bibr" rid="CR33"><u>33</u></xref></sup>, estimates molecular clock branch lengths via TreeTime<sup><xref ref-type="bibr" rid="CR34"><u>34</u></xref></sup> and reconstructs nucleotide and amino acid changes (also via TreeTime).<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34458642">34458642</a> <b>(2)</b> Molecular characterization of SARS-CoV-2 from Bangladesh: implications in genetic diversity, possible origin of the virus, and functional significance of the mutations<ul><li> Pairwise sequence alignment and clade assignment was done using web based Nextstrain tool <b>Nextclade</b> beta version 0.4.9 (<xref rid="bib46" ref-type="bibr"><b>Nextclade</b>, 2020</xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34419470">34419470</a> <b>(1)</b> CovidPhy: A tool for phylogeographic analysis of SARS-CoV-2 variation<ul><li> The tool <b>Nextclade</b>, which is part of the Nextstrain project, allows to classify SARS-CoV-2 fasta sequences into 18 major clades.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34409452">34409452</a> <b>(1)</b> Cluster characterization of SARS-CoV-2 in military personnel deployed to Egypt and subsequent introduction of B.1.1.7 and C.36 lineages to Colombia<ul><li> In addition to the B.1.1.7 groups, a collection of 1902 high-quality reference genomes embracing different lineages used in <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) for phylogenetic inferences was included and built from GISAID (Global Initiative on Sharing All Influenza Data) database (downloaded on 19 May 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34391446">34391446</a> <b>(2)</b> Circulating SARS-CoV-2 variants in Italy, October 2020&#x2013;March 2021<ul><li> 3.1.5 (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>) and <b>Nextclade</b> v.<li> Mutations were identified using <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34433082">34433082</a> <b>(1)</b> Acute SARS-CoV-2 infection is associated with an increased abundance of bacterial pathogens, including <italic>Pseudomonas aeruginosa</italic> in the nose<ul><li> Amino acid changes and clade assignments were identified using <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34433082">34433082</a> <b>(1)</b> Acute SARS-CoV-2 infection is associated with an increased abundance of bacterial pathogens, including <italic>Pseudomonas aeruginosa</italic> in the nose<ul><li> During this wave, a decline of 25%&#x2013;70% mobility from baseline was noted in Google mobility indicators even before the institution of restrictions by the government.<xref rid="R2" ref-type="bibr"><u>2</u></xref> Genomic surveillance shows that <b>Nextclade</b> 20B, first observed in Europe in February 2020, dominated the circulating variants during the first wave.<li> [2](<article-title>Community mobility reports</article-title>. Available: <uri xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.google.com/covid19/mobility/">https://www.google.com/covid19/mobility/</uri> [Accessed <comment>03 Aug 2021</comment>].) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34390857">34390857</a> <b>(1)</b> SARS-CoV-2 variant of concern P.1 (Gamma) infection in young and middle-aged patients admitted to the intensive care units of a single hospital in Salvador, Northeast Brazil, February 2021<ul><li> Genomes were submitted to the <b>Nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="interref0005">https://clades.nextstrain.org</ext-link>) for quality control assessment and the Pangolin tool (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/" id="interref0006">https://pangolin.cog-uk.io/</ext-link>) for SARS-CoV-2 lineage assignment.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34399188">34399188</a> <b>(1)</b> Evolutionary analysis of the Delta and Delta Plus variants of the SARS-CoV-2 viruses<ul><li> All available high coverage, complete sequences of the Delta variant collected during July 6&#x2013;13, 2021, were downloaded from GISAID [<xref rid="bib5" ref-type="bibr"><u>5</u></xref>] and processed through <b>NextClade</b> [<xref rid="bib15" ref-type="bibr"><u>15</u></xref>].<li> [5](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31565258">31565258</a>) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34376689">34376689</a> <b>(1)</b> Tracking the introduction and spread of SARS-CoV-2 in coastal Kenya<ul><li> Phylogenetic clades were assigned using <b>NextClade</b> (version 0.13.0).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34451453">34451453</a> <b>(2)</b> Predominance of the SARS-CoV-2 Lineage P.1 and Its Sublineage P.1.2 in Patients from the Metropolitan Region of Porto Alegre, Southern Brazil in March 2021<ul><li> Finally, we assessed the consensus sequences quality using <b>Nextclade</b> v0.14.2 (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>; accessed on 4 May 2021).<li> accessed on 4 May 2021) and global clades and mutations using <b>Nextclade</b> v0.14.2 (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>; accessed on 4 May 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33655278">33655278</a> <b>(2)</b> Emergence and Expansion of the SARS-CoV-2 Variant B.1.526 Identified in New York<ul><li> 2A</xref>) was conducted using the Nextstrain<sup><xref rid="R32" ref-type="bibr"><u>32</u></xref></sup> workflow at <ext-link xlink:href="https://github.com/nextstrain/ncov" ext-link-type="uri">https://github.com/nextstrain/ncov</ext-link> which aligns sequences against the Wuhan-Hu-1 reference via nextalign (<ext-link xlink:href="https://github.com/nextstrain/<b>nextclade</b>" ext-link-type="uri">https://github.com/nextstrain/<b>nextclade</b></ext-link>), constructs a maximum-likelihood phylogenetic tree via IQ-TREE<sup><xref rid="R33" ref-type="bibr"><u>33</u></xref></sup>, estimates molecular clock branch lengths via TreeTime<sup><xref rid="R34" ref-type="bibr"><u>34</u></xref></sup> and reconstructs nucleotide and amino acid changes also via TreeTime.<li> [32](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [33](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32011700">32011700</a>) [34](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34452380">34452380</a> <b>(1)</b> Characterization of a Lineage C.36 SARS-CoV-2 Isolate with Reduced Susceptibility to Neutralization Circulating in Lombardy, Italy<ul><li> whole-genome sequencing (WGS) on the first NPS identified the strain (hCoV-19/Italy/LOM-UnINSU/2021, hereafter named C.36_3) as belonging to clade 20D and lineage C.36 (<b>Nextclade</b> and Pangolin definitions as of 18 May 2021, respectively).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34394197">34394197</a> <b>(1)</b> poreCov-An Easy to Use, Fast, and Robust Workflow for SARS-CoV-2 Genome Reconstruction <italic>via</italic> Nanopore Sequencing<ul><li> The last columns of the table highlight the determined lineages based on the pangolin and nextstrain nomenclatures and include information about substitutions and deletions based on the <b>nextclade</b> alignments (<xref rid="B9" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> poreCov-An Easy to Use, Fast, and Robust Workflow for SARS-CoV-2 Genome Reconstruction <italic>via</italic> Nanopore Sequencing<ul><li> Clade and lineage assignments were performed using <b>Nextclade</b> (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin) (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) (<xref rid="B24" ref-type="bibr"><u>24</u></xref>), respectively.<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34508652">34508652</a> <b>(3)</b> Emergence of an early SARS-CoV-2 epidemic in the United States<ul><li><hr/></td></tr><tr><td>Pangolin v2.0</td><td><xref rid="bib50" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0025">https://github.com/cov-lineages/pangolin</ext-link></td></tr><tr><td><b>NextClade</b> v0.12.0.<li><hr/></td></tr><tr><td>Pangolin v2.0</td><td><xref rid="bib50" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0025">https://github.com/cov-lineages/pangolin</ext-link></td></tr><tr><td><b>NextClade</b> v0.12.0</td><td><xref rid="bib64" ref-type="bibr"><u>Hadfield et al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" id="intref0030">https://github.com/nextstrain/<b>nextclade</b></ext-link>.<li> [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34316508">34316508</a> <b>(2)</b> First detection of an Italian human-to-cat outbreak of SARS-CoV-2 Alpha variant &#x2013; lineage B.1.1.7<ul><li> The consensus sequence was analysed on PANGO lineages [<xref rid="bb0095" ref-type="bibr"><u>19</u></xref>] and <b>Nextclade</b> [<xref rid="bb0100" ref-type="bibr"><u>20</u></xref>] software for lineage and clade identification.<li> Sequence analyses on PANGO lineages and <b>Nextclade</b> showed that both cat and owners SARS-CoV-2 genomes belong to the lineage B.1.1.7 and the phylogenetic clade 20I/501Y.V1.<li> [19](O'Toole A, Scher E, Underwood A, Jackson B, Hill V, McCrone JT, et al. Pangolin: lineage assignment in an emerging pandemic as an epidemiological tool. Available from: <ext-link ext-link-type="uri" xlink:href="http://github.com/cov-lineages/pangolin" id="ir0030">github.com/cov-lineages/pangolin</ext-link>.) [20](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34354952">34354952</a> <b>(3)</b> Case Report: Discovery a Novel SARS-CoV-2 Variant in a Six-Months Long-Term Swab Positive Female Suffering From Non-Hodgkin Lymphoma<ul><li> Furthermore, <b>Nextclade</b> (<uri xlink:type="simple" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>) and PANGO (<uri xlink:type="simple" xlink:href="https://cov-lineages.org/">https://cov-lineages.org/</uri>) free tools were used to identify the clade or lineage, respectively, which are defined by specific mutation signatures.<li>In fact, both PANGO and <b>Nextclade</b> software indicated that this virus derives from clade 20E (EU1), but it differs from the initial 19A strain by 22 mutations.<li> Data obtained by using the GISAID, <b>Nextclade</b> and PANGO databases, confirm the unicity of our variant since the mutations mainly recurrent in ORF3a region are not present in our isolate.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34260066">34260066</a> <b>(1)</b> SARS&#x2010;CoV&#x2010;2 reinfection in a cancer patient with a defective neutralizing humoral response<ul><li> (C) Viral genome classification and amino acid mutations were identified according to <b>Nextclade</b> and Pangolin among the three specimens harvested on Days 0, 110, and 115.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34372615">34372615</a> <b>(1)</b> Viral Load and Patterns of SARS-CoV-2 Dissemination to the Lungs, Mediastinal Lymph Nodes, and Spleen of Patients with COVID-19 Associated Lymphopenia<ul><li>Identification of SARS-CoV-2 subtype was carried out by comparing sequence data with the reference genome on the <b>Nextclade</b> v1.1.0 (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 20 April 2021) platform.</ul></p>
</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34241897">34241897</a> <b>(1)</b> Genomic monitoring unveil the early detection of the SARS&#x2010;CoV&#x2010;2 B.1.351 (beta) variant (20H/501Y.V2) in Brazil<ul><li>Mutational profile was investigated by using the <b>Nextclade</b> tool (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) to describe substitutions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34273961">34273961</a> <b>(1)</b> <italic>HAVoC</italic>, a bioinformatic pipeline for reference-based consensus assembly and lineage assignment for SARS-CoV-2 sequences<ul><li> To identify to what variant a sequenced SARS-CoV-2 belongs to, it is usually analyzed and classified with pangolin [<xref ref-type="bibr" rid="CR15"><u>15</u></xref>] or <b>Nextclade</b>, which is a part of Nextstrain [<xref ref-type="bibr" rid="CR16"><u>16</u></xref>].<li> [15](O&#x2019;Toole &#xC1;, Scher E, Underwood A, Jackson B, Hill V, McCrone JT, et al. pangolin: lineage assignment in an emerging pandemic as an epidemiological tool. <ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</ext-link>. Accessed 12 Feb 2021.) [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34272947">34272947</a> <b>(1)</b> An Update on Severe Acute Respiratory Syndrome Coronavirus 2 Diversity in the US National Capital Region: Evolution of Novel and Variants of Concern<ul><li> Clades were determined using <b>Nextclade</b> beta version 0.12.0 (clades.nextstrain.org) [<xref rid="CIT0026" ref-type="bibr"><u>26</u></xref>], and lineages were determined with the Pangolin coronavirus disease 2019 (COVID-19) lineage assigner (COG-UK; cog-uk.io).<li> [26](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34464903">34464903</a> <b>(1)</b> Diagnostic pre-screening method based on N-gene dropout or delay to increase feasibility of SARS-CoV-2 VOC B.1.1.7 detection<ul><li> Clade assignment and annotated mutations are generated through the <b>nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0002">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34274478">34274478</a> <b>(2)</b> No association between the SARS-CoV-2 variants and mortality rates in the Eastern Mediterranean Region<ul><li> For clade analysis, we used the <b>Nextclade</b> tool from the Nextstrain project (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir015">https://clades.nextstrain.org/</ext-link>) (<xref rid="b0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and GISAID clade designation provided by CoVsurver analysis.<li>Next, we used the <b>Nextclade</b> tool from Nextstrain to assign clades (GISAID nomenclature) to the sequences (<xref rid="f0015" ref-type="fig">Fig.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(3)</b> No association between the SARS-CoV-2 variants and mortality rates in the Eastern Mediterranean Region<ul><li> Genome Detective Virus Tool is a web&#x2010;based software that evaluates the read quality, removes low&#x2010;quality reads, performs read mapping, assembles the reads, and generates consensus sequences for NGS data.<xref rid="jmv27188-bib-0016" ref-type="bibr"><sup>16</sup></xref> <b>Nextclade</b> tool was used to check for sequencing errors and the quality of the assembled sequences.<li> <b>Nextclade</b> tool utilizes different metrics to analyze the quality of consensus sequences.<li>Sequences with suspected underlying sequencing errors (10/45 sequences) according to <b>Nextclade</b> quality metrics were excluded from the analysis.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(2)</b> No association between the SARS-CoV-2 variants and mortality rates in the Eastern Mediterranean Region<ul><li> The quality of the genome sequences was assessed using <b>Nextclade</b> version v.<li> Nucleotide and amino acid mutations were mapped using <b>Nextclade</b> version 0.14.1.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> No association between the SARS-CoV-2 variants and mortality rates in the Eastern Mediterranean Region<ul><li> Processed reads were assembled using the ARTIC guppyplex script with Medaka v1.4 using the ARTIC EPI2ME v3.3.0 SARS-CoV-2 pipeline (FastQ quality control plus ARTIC plus <b>NextClade</b>) (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html" ext-link-type="uri">https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> No association between the SARS-CoV-2 variants and mortality rates in the Eastern Mediterranean Region<ul><li>Annotation of variants were performed using COVID-19 genome annotator,<xref rid="R11" ref-type="bibr"><u>11</u></xref> while clade assignment was performed using COVIDEX<xref rid="R12" ref-type="bibr"><u>12</u></xref> and <b>Nextclade</b>.<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21903627">21903627</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33205830">33205830</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34271202">34271202</a> <b>(1)</b> Monoclonal Antibody Therapy in a Vaccine Breakthrough SARS-CoV-2 Hospitalized Delta (B.1.617.2) Variant Case<ul><li> The phylogenetic tree was generated using COVID-DX and <b>Nextclade</b> (version 4-22-2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34372565">34372565</a> <b>(2)</b> Variant Analysis of SARS-CoV-2 Genomes from Belgian Military Personnel Engaged in Overseas Missions and Operations<ul><li> using the <b>Nextclade</b> tool [<xref rid="B28-viruses-13-01359" ref-type="bibr"><u>28</u></xref>].<li> Currently, <b>Nextclade</b> defines 12 major clades.<li> [28](          <person-group person-group-type="author">            <name>              <surname>Aksamentov</surname>              <given-names>I.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.</given-names>            </name>          </person-group>          <article-title>Nextclade</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-05-13">(accessed on 13 May 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34359323">34359323</a> <b>(1)</b> Limitation of Screening of Different Variants of SARS-CoV-2 by RT-PCR<ul><li> For a complete analysis and lineage determination, we employed two websites: <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 10 May 2021) and pangolineage (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>, accessed on 10 May 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34268529">34268529</a> <b>(1)</b> Transmission event of SARS-CoV-2 Delta variant reveals multiple vaccine breakthrough infections<ul><li> Swift analysis pipeline produced variant vcf file, consensus genome, pangolin lineage and <b>Nextclade</b> assignment (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34358195">34358195</a> <b>(1)</b> Neutralizing Activity of Sera from Sputnik V-Vaccinated People against Variants of Concern (VOC: B.1.1.7, B.1.351, P.1, B.1.617.2, B.1.617.3) and Moscow Endemic SARS-CoV-2 Variants<ul><li> Resulting FASTA files were analyzed in <b>Nextclade</b> web-service [<xref rid="B17-vaccines-09-00779" ref-type="bibr"><u>17</u></xref>] for the presence of amino acid substitutions, corresponding CSV file was downloaded and analyzed similarly to the GISAID data.<li> [17](          <person-group person-group-type="author">            <collab>Nextclade</collab>          </person-group>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-04-16">(accessed on 16 April 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=          <person-group person-group-type="author">            <collab>Nextclade</collab>          </person-group>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-04-16">(accessed on 16 April 2021)</date-in-citation>        ">          <person-group person-group-type="author">            <collab>Nextclade</collab>          </person-group>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-04-16">(accessed on 16 April 2021)</date-in-citation>        </a> <b>(3)</b> Neutralizing Activity of Sera from Sputnik V-Vaccinated People against Variants of Concern (VOC: B.1.1.7, B.1.351, P.1, B.1.617.2, B.1.617.3) and Moscow Endemic SARS-CoV-2 Variants<ul><li> <b>Nextclade</b> beta version 0.14.2 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://clades.nextstrain.org" ext-link-type="uri">clades.nextstrain.org</ext-link>) by Nextstrain (<xref rid="B11" ref-type="bibr"><u>11</u></xref>) was utilized in the phylogenetic analysis, which assigned both sequences to SARS-CoV-2 clade 20B (<xref rid="fig1" ref-type="fig"><u>Fig.&#xA0;1</u></xref>).<li> EPI_ISL_682280) according to <b>Nextclade</b>.<li> <b>Nextclade</b> beta version 0.14.2 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://clades.nextstrain.org" ext-link-type="uri">clades.nextstrain.org</ext-link>) was used to construct the tree on 4 May 2021, with the red and dark blue circles representing the positions of the hCoV-19/Bangladesh/JU-WMSRC-Dhaka-5/2020 (GISAID accession no.<li> [11](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34273860">34273860</a> <b>(1)</b> Rapid SARS-CoV-2 variant monitoring using PCR confirmed by whole genome sequencing in a high-volume diagnostic laboratory<ul><li> The lineages of the samples were assigned based on the consensus sequences, using <italic>Panglin</italic> and <italic><b>Nextclade</b></italic><xref rid="bib0016" ref-type="bibr"><u>[16]</u></xref>, <xref rid="bib0017" ref-type="bibr"><u>[17]</u></xref>, <xref rid="bib0018" ref-type="bibr"><u>[18]</u></xref>, <xref rid="bib0019" ref-type="bibr"><u>[19]</u></xref>, <xref rid="bib0020" ref-type="bibr"><u>[20]</u></xref>.<li> [[16]](O'Toole &#xC1;, Scher E, Underwood A, Jackson B, Hill V, McCrone J, et&#xA0;al. Pango lineages n.d. <ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="interref0006">https://github.com/cov-lineages/pangolin</ext-link> (accessed February 2021).) [[17]](Nextstrain n.d. https://clades.nextstrain.org/(accessed February 2021).) [[18]](Nextclade n.d. https://github.com/nextstrain/nextclade (accessed February 2021).) [[19]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[20]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a> <b>(1)</b> Rapid SARS-CoV-2 variant monitoring using PCR confirmed by whole genome sequencing in a high-volume diagnostic laboratory<ul><li> Clades were determined using <b>Nextclade</b> beta v 0.12.0 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://clades.nextstrain.org" ext-link-type="uri">clades.nextstrain.org</ext-link>) (<xref rid="R7" ref-type="bibr"><u>7</u></xref>), and lineages were determined with Pangolin COVID-19 lineage Assigner (COG-UK (cog-uk.io)).<li> [7](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(4)</b> Rapid SARS-CoV-2 variant monitoring using PCR confirmed by whole genome sequencing in a high-volume diagnostic laboratory<ul><li> Clade assignment was performed using <b>Nextclade</b> v0.12.0 (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" ext-link-type="uri">https://github.com/nextstrain/<b>nextclade</b></ext-link>).<li> Both samples were classified as 20I/501Y.V1 using <b>Nextclade</b> (v.0.12.0) and had 100% pairwise nucleotide identity in a whole&#x2010;genome alignment).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34594942">34594942</a> <b>(1)</b> Genome Characterization of the First Outbreak of COVID-19 Delta Variant B.1.617.2 &#x2014; Guangzhou City, Guangdong Province, China, May 2021<ul><li> Nucleotide (nt) and amino acid (AA) differences between the 5 virus genome sequences from this study and the reference sequence Wuhan-Hu-1 were analyzed using the programs BioEdit and online tool &#x201C;<b>Nextclade</b>&#x201D; (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) (<xref rid="b1" ref-type="bibr"><u>1</u></xref>).<li> [1](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34372500">34372500</a> <b>(3)</b> Molecular Evolution and Epidemiological Characteristics of SARS COV-2 in (Northwestern) Poland<ul><li> referred to as PANGO) and <b>NEXTCLADE</b> v0.14.2 (<uri xlink:href="https://github.com/nextstrain/<b>nextclade</b>">https://github.com/nextstrain/<b>nextclade</b></uri> (accessed on 15 March 2021); referred to as Nextstrain).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33979566">33979566</a> <b>(2)</b> Possible Human-to-Dog Transmission of SARS-CoV-2, Italy, 2020<ul><li> The Pangolin COVID-19 Lineage Assigner (<ext-link xlink:href="https://pangolin.cog-uk.io" ext-link-type="uri">https://pangolin.cog-uk.io</ext-link>) and <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>) assigned the sequences to the lineage B.1.177 (denoted by <b>Nextclade</b> as 20A.EU1) in Europe.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34212838">34212838</a> <b>(1)</b> Neutralising capacity against Delta (B.1.617.2) and other variants of concern following Comirnaty (BNT162b2, BioNTech/Pfizer) vaccination in health care workers, Israel<ul><li> Identification of the variants were done with Pangolin COVID-19 lineage assigner (<ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</ext-link>) and specific mutations were identified with a custom python code and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34197207">34197207</a> <b>(1)</b> New SARS-CoV-2 Variant from Jordan<ul><li> The phylogenetic tree was constructed using the <b>Nextclade</b> tool v 0.14.3 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34181711">34181711</a> <b>(1)</b> Characterization of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection Clusters Based on Integrated Genomic Surveillance, Outbreak Analysis and Contact Tracing in an Urban Setting<ul><li>Sample clades, variants and other summary statistics were computed with the <b>NextClade</b> tool of NextStrain [<xref rid="CIT0021" ref-type="bibr"><u>21</u></xref>].<li> [21](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33906227">33906227</a> <b>(1)</b> Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Sequence Characteristics of Coronavirus Disease 2019 (COVID-19) Persistence and Reinfection<ul><li>Sequences were analyzed for mutations using <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and snp-sites (<ext-link xlink:href="https://github.com/sanger-pathogens/snp-sites" ext-link-type="uri">https://github.com/sanger-pathogens/snp-sites</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34172783">34172783</a> <b>(1)</b> Limited intestinal inflammation despite diarrhea, fecal viral RNA and SARS-CoV-2-specific IgA in patients with acute COVID-19<ul><li> The mutation profiles were analyzed with the tool <b>NextClade</b> v0.3.6 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34166484">34166484</a> <b>(2)</b> Variants of Concern Are Overrepresented Among Postvaccination Breakthrough Infections of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) in Washington State<ul><li> <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>) and <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) tools.<li> Amino acid changes were annotated according to <b>NextClade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34188776">34188776</a> <b>(2)</b> Evolution, correlation, structural impact and dynamics of emerging SARS-CoV-2 variants<ul><li>We also used Nextstrain&#x2019;s <b>NextClade</b> <xref rid="b0220" ref-type="bibr"><u>[44]</u></xref> tool to further analyze and validate the evolutionary relationships between variants from variant sequences (n&#xA0;=&#xA0;6,052); B.1.1.7 (n&#xA0;=&#xA0;1,637), B.1.351 (n&#xA0;=&#xA0;1,069), P.1 (n&#xA0;=&#xA0;1,926), and Cal.20C (n&#xA0;=&#xA0;4,182).<li> The resultant alignment was submitted into NextStrain&#x2019;s <b>NextClade</b> <xref rid="b0220" ref-type="bibr"><u>[44]</u></xref>, which generated a Phylogeny using the sequences.<li> [[44]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [[44]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34174479">34174479</a> <b>(1)</b> Molecular surveillance of the on-going SARS-COV-2 epidemic in Ribeirao Preto City, Brazil<ul><li> The reference SARS-CoV-2 strains composing the phylogenetic tree were obtained from an available dataset from the <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="ir0020">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34163035">34163035</a> <b>(1)</b> Emergence and spread of a SARS-CoV-2 lineage A variant (A.23.1) with altered spike protein in Uganda<ul><li>The genomes generated in this study were classified into Pango lineages<sup><xref ref-type="bibr" rid="CR4"><u>4</u></xref></sup> using the pangolin module pangoLEARN (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</ext-link>) and into Nextstrain clades using <b>Nextclade</b><sup><xref ref-type="bibr" rid="CR9"><u>9</u></xref></sup> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Emergence and spread of a SARS-CoV-2 lineage A variant (A.23.1) with altered spike protein in Uganda<ul><li> According to <b>Nextclade</b> analysis [<xref rid="CIT0012" ref-type="bibr"><u>12</u></xref>], it contains 7 known amino acid substitutions (S:L18F, S:A222V, S:D614G, nsp12:P323L, N:A220V, ORF14:L67F, ORF10:V30L) and 14 nucleotide mutations, 11 of which were private.<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34110974">34110974</a> <b>(1)</b> Symptomatic SARS-CoV-2 infections after full schedule BNT162b2 vaccination in seropositive healthcare workers: a case series from a single institution<ul><li> Sequences were deposited on GenBank and characterized using <b>NextClade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>), showing the synchronous circulation of different SARS-CoV-2 strains in the Lombardy region.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34110974">34110974</a> <b>(2)</b> Symptomatic SARS-CoV-2 infections after full schedule BNT162b2 vaccination in seropositive healthcare workers: a case series from a single institution<ul><li> The clades were determined using <b>Nextclade</b><sup>beta</sup> (version 0.14.2; <ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).<li> Finally, it is possible that we underestimated the sequence relatedness because we used strict filtering criteria and both the <b>Nextclade</b> and Bionumerics 7.6 phylogenetic tree methodologies for concurrence.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34220784">34220784</a> <b>(2)</b> The First Molecular Characterization of Serbian SARS-CoV-2 Isolates From a Unique Early Second Wave in Europe<ul><li> Clades were assigned by using GISAID platform and also by the <b>Nextclade</b> sequence analysis webapp<sup><xref ref-type="fn" rid="footnote3"><u>3</u></xref></sup> (<xref rid="B16" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The tree was made online with <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) without any restrictions on use.<li> [3](<a href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34207314">34207314</a> <b>(1)</b> Isolation of Viable SARS-CoV-2 Virus from Feces of an Immunocompromised Patient Suggesting a Possible Fecal Mode of Transmission<ul><li> The consensus sequence was analyzed with <b>Nextclade</b> web interface (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 24 May 2021) (version 0.7.5)).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34159330">34159330</a> <b>(2)</b> A stem-loop RNA RIG-I agonist confers prophylactic and therapeutic protection against acute and chronic SARS-CoV-2 infection in mice<ul><li> We used <b>Nextclade</b> v0.14.2 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) to generate a phylogenetic tree and to compile a list of amino acid changes in the virus isolates as compared to the Wuhan-Hu-1 reference strain (<xref rid="SD1" ref-type="supplementary-material"><u>Extended Data Table 1</u></xref>; <xref rid="F10" ref-type="fig">Extended Data Fig.<li>We used <b>NextClade</b> (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) to generate a phylogenetic tree to show the evolutionary relations between the four cultured viruses and other SARS-CoV-2 lineages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34176716">34176716</a> <b>(3)</b> Virological and genomic analysis of SARS-CoV-2 from a favipiravir clinical trial cohort<ul><li>The genomic epidemiology of SARS-CoV-2 identified in this study was analyzed using the <b>Nextclade</b> webpage (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0025">https://clades.nextstrain.org/</ext-link>) and the Nextstrain tools (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.github.io/ncov/" id="intref0030">https://nextstrain.github.io/ncov/</ext-link>).<li> Sequencing results were evaluated in the <b>Nextclade</b> webpage (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0035">https://clades.nextstrain.org/</ext-link>), and sequence data indicated as good with all QC status under the default settings were defined as high-quality.<li> The amino acid substitutions on SARS-CoV-2 genome were also evaluated on the <b>Nextclade</b> webpage (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0045">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34136351">34136351</a> <b>(4)</b> SARS-CoV-2 detection and sequencing in heart tissue associated with myocarditis and persistent arrhythmia: A case report<ul><li> Finally, the online tools <b>NextClade</b> (v0.4.0) [<xref rid="bib0045" ref-type="bibr"><u>9</u></xref>] and CoV-GLUE [<xref rid="bib0050" ref-type="bibr"><u>10</u></xref>] were used to determine the sequence clade and to perform lineage classification and mutation resolution.<li> Phylogenetic analysis placed the strain in the 20A clade according to <b>NextClade</b> (<xref rid="fig0015" ref-type="fig">Fig.<li> The tree was acquired from <b>NextClade</b> v0.4.0, which uses a banded Smith-Waterman algorithm with an affine gap-penalty [<xref rid="bib0060" ref-type="bibr"><u>12</u></xref>].<li> The sequence was compared with the reference strain Wuhan-1 (GenBank accession number MN908947), and a phylogenetic tree was obtained with <b>Nextclade</b>.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29291722">29291722</a>) [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32697968">32697968</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34207490">34207490</a> <b>(2)</b> A Comprehensive Molecular Epidemiological Analysis of SARS-CoV-2 Infection in Cyprus from April 2020 to January 2021: Evidence of a Highly Polyphyletic and Evolving Epidemic<ul><li>Mutations of the sequences in this study were identified using the <b>Nextclade</b> webtool (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 15 February 2021) [<xref rid="B18-viruses-13-01098" ref-type="bibr"><u>18</u></xref>].<li> To perform this analysis, the sequences were input into the <b>nextclade</b> webtool [<xref rid="B18-viruses-13-01098" ref-type="bibr"><u>18</u></xref>], and the output data were sorted to isolate S protein mutations and deletions.<li> [18](          <article-title>Nextstrain Nextstrain</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">https://nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-02-24">(accessed on 24 February 2021)</date-in-citation>        ) [18](          <article-title>Nextstrain Nextstrain</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/">https://nextstrain.org/</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-02-24">(accessed on 24 February 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34166617">34166617</a> <b>(1)</b> Recurrent emergence of SARS-CoV-2 spike deletion H69/V70 and its role in the Alpha variant B.1.1.7<ul><li> Major SARS-CoV-2 clade memberships were assigned to all sequences using the <b>Nextclade</b> server v0.13 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0060">https://clades.nextstrain.org/</ext-link>), Pangolin v2.4.2 (<xref rid="bib56" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020a</u></xref>); <ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0065">https://github.com/cov-lineages/pangolin</ext-link>) and a local instance of the PangoLEARN model, dated 18<sup>th</sup> April 21:49 (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangoLEARN" id="intref0070">https://github.com/cov-lineages/pangoLEARN</ext-link>).<li> [Rambaut et&#xA0;al., 2020a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34042186">34042186</a> <b>(1)</b> Genome&#x2010;wide analysis of SARS&#x2010;CoV&#x2010;2 strains circulating in Vietnam: Understanding the nature of the epidemic and role of the D614G mutation<ul><li> Variant analysis was done using CLC (v10.1.1) and <b>Nextclade</b> (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34122366">34122366</a> <b>(1)</b> Respiratory Co-Infections: Modulators of SARS-CoV-2 Patients&#x2019; Clinical Sub-Phenotype<ul><li> We identified two clades according to the <b>Nextclade</b> classification, i.e., 19A and 20B (<xref ref-type="fig" rid="F3"><u>Figure 3C</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33990970">33990970</a> <b>(2)</b> Genomic surveillance activities unveil the introduction of the SARS&#x2010;CoV&#x2010;2 B.1.525 variant of interest in Brazil: Case report<ul><li> All sequences were aligned using the MAFFT tool,<xref rid="jmv27086-bib-0006" ref-type="bibr"><sup>6</sup></xref> and IQ&#x2010;TREE<xref rid="jmv27086-bib-0007" ref-type="bibr"><sup>7</sup></xref> was used for phylogenetic analysis using the maximum likelihood approach.<xref rid="jmv27086-bib-0008" ref-type="bibr"><sup>8</sup></xref> The mutation pattern of the VUI was analyzed using the <b>NextClade</b> online tool.<li> Using <b>NextClade</b>, the mutation pattern of BA125 (Figure&#xA0;<xref rid="jmv27086-fig-0001" ref-type="fig"><u>1C</u></xref>) was determined, and no newly acquired mutations were identified.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33990970">33990970</a> <b>(1)</b> Genomic surveillance activities unveil the introduction of the SARS&#x2010;CoV&#x2010;2 B.1.525 variant of interest in Brazil: Case report<ul><li> The data downloaded from this website can then be uploaded into <b>Nextclade</b>, which translates sequencing data into amino acid substitutions to allow for the assessment of the frequency of key RBD mutations in statewide samples.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34016042">34016042</a> <b>(2)</b> Genomic epidemiology of SARS-CoV-2 in Esteio, Rio Grande do Sul, Brazil<ul><li> Finally, we assessed genome consensus sequence quality using <b>Nextclade</b> v0.8.1 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) and CoV-GLUE (<ext-link ext-link-type="uri" xlink:href="http://cov-glue.cvr.gla.ac.uk/">http://cov-glue.cvr.gla.ac.uk/</ext-link> [<xref ref-type="bibr" rid="CR48"><u>48</u></xref>];).<li> Moreover, we identified global virus lineages using <b>Nextclade</b> v0.8.1 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) and Pangolin v2.1.3 (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</ext-link> [<xref ref-type="bibr" rid="CR15"><u>15</u></xref>];).<li> [48](Singer J, Gifford R, Cotten M, Robertson D. CoV-GLUE: A Web Application for Tracking SARS-CoV-2 Genomic Variation. 2020. 10.20944/preprints202006.0225.v1.) [15](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34093495">34093495</a> <b>(4)</b> Genomic Epidemiology of SARS-CoV-2 From Mainland China With Newly Obtained Genomes From Henan Province<ul><li>To identify differences between 19 genome sequences from Henan Province and reference sequences, and to find out which clades they are from and where on the SARS-CoV-2 tree they fall, we used the <b>Nextclade</b> web tool (beta v0.10.0, <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) to perform mutation calling and clade assignment.<li> Of note, the <b>Nextclade</b> does not do a real phylogenetic analysis.<li> Clade names are assigned by the <b>Nextclade</b></italic>.<li>Clade assignment of the 19 Henan sequences analyzed by the <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34016683">34016683</a> <b>(1)</b> Complete Genome Sequence of a SARS-CoV-2 Strain Sampled in Morocco in May 2020, Obtained Using Sanger Sequencing<ul><li>Sequence quality control, clade assignment, and mutation determination were done using the <b>Nextclade</b> tool v0.12.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34043946">34043946</a> <b>(3)</b> SARS-CoV-2 genomic surveillance identifies naturally occurring truncation of ORF7a that limits immune suppression<ul><li><hr/></td></tr><tr><td>SDS software v1.4</td><td>Applied Biosystems</td><td>4379633</td></tr><tr><td>RStudio v1.2.1335</td><td>The R project</td><td><ext-link ext-link-type="uri" xlink:href="https://www.r-project.org/" id="intref0035">https://www.r-project.org/</ext-link></td></tr><tr><td>artic-ncov2019</td><td>ARTIC network</td><td><ext-link ext-link-type="uri" xlink:href="https://artic.network/ncov-2019" id="intref0040">https://artic.network/ncov-2019</ext-link></td></tr><tr><td>MinKNOW software</td><td>Oxford Nanopore Technologies</td><td><ext-link ext-link-type="uri" xlink:href="https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016" id="interref0010">https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revbm_11apr2016</ext-link></td></tr><tr><td>PrimePCR analysis software</td><td>Bio-Rad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a" id="intref0045">https://www.bio-rad.com/en-us/category/qpcr-analysis-software?ID=42a6560b-3ad7-43e9-bb8d-6027371de67a</ext-link></td></tr><tr><td>CD-HIT</td><td>GitHub</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/weizhongli/cdhit" id="intref0050">https://github.com/weizhongli/cdhit</ext-link></td></tr><tr><td>BioStrings</td><td>Bioconductor</td><td><ext-link ext-link-type="uri" xlink:href="https://bioconductor.org/packages/release/bioc/html/Biostrings.html" id="intref0055">https://bioconductor.org/packages/release/bioc/html/Biostrings.html</ext-link></td></tr><tr><td>ggplot2</td><td>CRAN</td><td><ext-link ext-link-type="uri" xlink:href="https://cran.r-project.org/web/packages/ggplot2/index.html" id="intref0060">https://cran.r-project.org/web/packages/ggplot2/index.html</ext-link></td></tr><tr><td>Augur</td><td>Nextstrain</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/augur" id="intref0065">https://github.com/nextstrain/augur</ext-link></td></tr><tr><td><b>Nextclade</b>.<li> Then, the <b>Nextclade</b> online tool was used to determine the quality of 73 SARS-CoV-2 genomes (% genome covered) sequenced from Bozeman patients.<li> Clades were assigned to SARS-CoV-2 genomes using <b>Nextclade</b> and pangolin (<ext-link ext-link-type="uri" xlink:href="https://cov-lineages.org/" id="intref0140">https://cov-lineages.org/</ext-link>) utilities.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34029990">34029990</a> <b>(4)</b> Implementation of an in-house real-time reverse transcription-PCR assay to detect the emerging SARS-CoV-2 N501Y variants<ul><li> Ten cDNA samples, one for each of the 10 Marseille variants [<xref rid="bib0006" ref-type="bibr"><u>6</u></xref>,<xref rid="bib0007" ref-type="bibr"><u>7</u></xref>], including <b>Nextclade</b>s 20A.EU2 and 20E, identified in our institute by genome next-generation sequencing, and three nasopharyngeal samples of the Marseille-4 variant/<b>Nextclade</b> 20A.EU2 that predominated locally <xref rid="bib0008" ref-type="bibr"><u>[8]</u></xref> tested negative.<li> All 112 and 26 samples from patients infected with a SARS-CoV-2 20I/501Y.V1 and a 20H/501Y.V2 variant, respectively, were positive with the N501Y-specific qPCR, while all 23 samples from patients infected with a N501Y-negative SARS-CoV-2 (Marseille-2/<b>Nextclade</b> 20E (<italic>n</italic>&#xA0;=&#xA0;20); Marseille-8 (<italic>n</italic>&#xA0;=&#xA0;2); Marseille-4/<b>Nextclade</b> 20A.EU2 (<italic>n</italic>&#xA0;=&#xA0;1)) tested negative with our N501Y-specific qPCR.<li> [6](          <person-group person-group-type="author">            <name>              <surname>Colson</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Levasseur</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Delerce</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Chaudet</surname>              <given-names>H.</given-names>            </name>            <name>              <surname>Bossi</surname>              <given-names>V.</given-names>            </name>            <name>              <surname>Ben Kheder</surname>              <given-names>M.</given-names>            </name>            <etal/>          </person-group>          <part-title>Dramatic Increase in the SARS-CoV-2 Mutation Rate and Low Mortality Rate During the Second Epidemic in Summer in Marseille</part-title>          <year>2020</year>          <publisher-name>IHU preprint</publisher-name>          <pub-id pub-id-type="doi">10.35088/68c3-ew82</pub-id>        ) [7](Fournier P.E., Colson P., Levasseur A., Gautret P., Bedotto M., Filosa V., et&#xA0;al., Genome sequence analysis enabled deciphering the atypical evolution of COVID-19 epidemics in Marseille, France, IHU preprint (2021), doi:<pub-id pub-id-type="doi">10.35088/kmct-tj43</pub-id>.) [[8]](          <person-group person-group-type="author">            <name>              <surname>Fournier</surname>              <given-names>P.E.</given-names>            </name>            <name>              <surname>Colson</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Levasseur</surname>              <given-names>A.</given-names>            </name>            <name>              <surname>Devaux</surname>              <given-names>C.A.</given-names>            </name>            <name>              <surname>Gautret</surname>              <given-names>P.</given-names>            </name>            <name>              <surname>Bedotto</surname>              <given-names>M.</given-names>            </name>            <etal/>          </person-group>          <article-title>Emergence and outcome of the SARS-CoV-2 "Marseille-4" variant</article-title>          <source>Int. J. Infect. Dis.</source>          <volume>S1201-9712</volume>          <issue>21</issue>          <year>2021</year>          <comment>00286&#x2013;00281</comment>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33988184">33988184</a> <b>(1)</b> Identification of and Surveillance for the SARS-CoV-2 Variants B.1.427 and B.1.429 &#x2014; Colorado, January&#x2013;March 2021<ul><li> COVID-19 B.1.427/B.1.429 variant cases were identified through tiled amplicon WGS.<xref rid="FN2" ref-type="fn"><sup>&#x2020;</sup></xref><sup>,</sup><xref rid="FN3" ref-type="fn"><sup>&#xA7;</sup></xref> Assembly of sequencing data into whole genomes was performed using CDPHE&#x2019;s publicly available Illumina and Nanopore data workflows.<xref rid="FN4" ref-type="fn"><sup>&#xB6;</sup></xref> Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin)<xref rid="FN5" ref-type="fn"><u>**</u></xref> (<xref rid="R5" ref-type="bibr"><italic>5</italic></xref>) and Nextstrain&#x2019;s <b>Nextclade</b> tools (<xref rid="R6" ref-type="bibr"><italic>6</italic></xref>) were used to assign lineage designations to each assembled genome.<li> [**](<a href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33985433">33985433</a> <b>(1)</b> IPD 2.0: To derive insights from an evolving <italic>SARS-CoV-2</italic> genome<ul><li> Clade assessment of the sequences was performed using the <b>NextClade</b> module of the NextStrain package [<xref ref-type="bibr" rid="CR9"><u>9</u></xref>] and used as a truth set.<li> [9](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33986354">33986354</a> <b>(1)</b> Surge of severe acute respiratory syndrome coronavirus 2 infections linked to single introduction of a virus strain in Myanmar, 2020<ul><li> The variant calling was done by CoV-GLUE (<ext-link ext-link-type="uri" xlink:href="http://cov-glue.cvr.gla.ac.uk/">http://cov-glue.cvr.gla.ac.uk/</ext-link>)<sup><xref ref-type="bibr" rid="CR9"><u>9</u></xref></sup> and <b>NextClade</b> v0.5.0 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>)<sup><xref ref-type="bibr" rid="CR10"><u>10</u></xref></sup> to assign the clade.<li> [9](Singer, J., Gifford, R., Cotten, M. &amp; Robertson, D. CoV-GLUE: A web application for tracking SARS-CoV-2 genomic variation. 2020060225. 10.20944/preprints202006.0225.v1 (2020).) [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33991677">33991677</a> <b>(1)</b> Spreading of a new SARS-CoV-2 N501Y spike variant in a new lineage<ul><li> Sequences were analysed using <sc><b>Nextclade</b></sc> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0025">https://clades.nextstrain.org/</ext-link>) [<xref rid="bib6" ref-type="bibr"><u>6</u></xref>] and an in-house <sc>Python</sc> script.<li> [6](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34065789">34065789</a> <b>(2)</b> Molecular Analysis of SARS-CoV-2 Circulating in Bangladesh during 2020 Revealed Lineage Diversity and Potential Mutations<ul><li> <b>Nextclade</b> beta (<uri xlink:href="https://clades.nextstrain.org/results">https://clades.nextstrain.org/results</uri> (accessed on 23 February 2021) was used to check the sequence quality and to generate the clade designated tree.<li> We further analyzed the AA diversity using the <b>Nextclade</b> web tools.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34066205">34066205</a> <b>(1)</b> Symptomatic SARS-CoV-2 Reinfection in a Healthy Healthcare Worker in Italy Confirmed by Whole-Genome Sequencing<ul><li>The phylogenetic analysis was performed using the <b>Nextclade</b> sequence analysis webapp (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed on 14 February 2021).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33797393">33797393</a> <b>(1)</b> Genetic Evidence and Host Immune Response in Persons Reinfected with SARS-CoV-2, Brazil<ul><li> We determined SARS-CoV-2 clades using the <b>Nextclade</b> software, beta version 0.14 (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33900177">33900177</a> <b>(2)</b> Monitoring SARS-CoV-2 Circulation and Diversity through Community Wastewater Sequencing, the Netherlands and Belgium<ul><li> Clades were assigned by using the <b>Nextclade</b> tool.<li> Clades (19A, 19B, 20A, 20B, and 20C) were assigned by using the <b>Nextclade</b> tool (<ext-link xlink:href="https://clades.nextstrain.org" ext-link-type="uri">https://clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33914720">33914720</a> <b>(1)</b> COVID-19 Outbreak Associated with a SARS-CoV-2 R.1 Lineage Variant in a Skilled Nursing Facility After Vaccination Program &#x2014; Kentucky, March 2021<ul><li> Genomes were assembled using the StaPH-B Monroe pipeline,<xref rid="FN4" ref-type="fn"><sup>&#xB6;</sup></xref> followed by <b>Nextclade</b><xref rid="FN5" ref-type="fn"><u>**</u></xref> for clade assignment and mutation calling, Pangolin<xref rid="FN6" ref-type="fn"><sup>&#x2020;&#x2020;</sup></xref> for lineage assignment, and Nextstrain for phylogenetic analysis (<xref rid="R3" ref-type="bibr"><italic>3</italic></xref>).<li> [**](<a href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34796242">34796242</a> <b>(2)</b> The Longest Persistence of Viable SARS-CoV-2 With Recurrence of Viremia and Relapsing Symptomatic COVID-19 in an Immunocompromised Patient&#x2014;A Case Study<ul><li> We used the <b>Nextclade</b> and Pangolin systems (freely available at <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link> and <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>, respectively) [<xref rid="CIT0013" ref-type="bibr"><u>13</u></xref>, <xref rid="CIT0014" ref-type="bibr"><u>14</u></xref>] for strain classification.<li>Lineage Classification of Strains Included in the Study According to <b>Nextclade</b> and Pangolin Identified Mutations With Cutoff &gt;90%.<li> [13](Nextclade. Available at: <ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>. <comment>Accessed 17 December 2020</comment>.) [14](Pangolin COVID-19 Lineage Assigner. Available at: <ext-link xlink:href="https://pangolin.cog-uk.io/" ext-link-type="uri">https://pangolin.cog-uk.io/</ext-link>. <comment>Accessed 17 December 2020</comment>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33915337">33915337</a> <b>(1)</b> Increased viral variants in children and young adults with impaired humoral immunity and persistent SARS-CoV-2 infection: A consecutive case series<ul><li> Phylogenetic clade analysis was performed using <b>Nextclade</b> (version 0.3.7).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33922257">33922257</a> <b>(6)</b> Genomic Surveillance of Circulating SARS-CoV-2 in South East Italy: A One-Year Retrospective Genetic Study<ul><li>An additional classification effort has been provided by <b>Nextclade</b> [<xref rid="B11-viruses-13-00731" ref-type="bibr"><u>11</u></xref>], which initially employed a year-letter nomenclature to facilitate the discussion of large-scale diversity patterns of SARS-CoV-2 and labeled genetically well-defined clades that have achieved significant frequency and geographic spread.<li><b>Nextclade</b> provides a classification, which supports the PANGOLIN lineage nomenclature system, based on non-synonymous amino acid mutations that characterize the two clades that emerged in 2019 [<xref rid="B11-viruses-13-00731" ref-type="bibr"><u>11</u></xref>] and the nine clades identified during 2020 [<xref rid="B12-viruses-13-00731" ref-type="bibr"><u>12</u></xref>].<li> Two hundred and ninety-eight consensus genome sequences passed the quality control assessment by <b>Nextclade</b>, whose parameters include missing data, mixed sites, private mutations, and mutation clusters, and were deposited in the GISAID database (<uri xlink:href="https://www.gisaid.org/">https://www.gisaid.org/</uri>, accessed: 28 January 2021), with accession numbers provided in <xref ref-type="app" rid="app1-viruses-13-00731"><u>Supplementary Table S1</u></xref>.<li>The <b>Nextclade</b> tool was used for isolate clades definition, mutation calling, and phylogenetic analyses (Nextstrain, <b>Nextclade</b>: <uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>, accessed: 28 January 2021).<li> Among these, <b>Nextclade</b> analysis allowed to identify three main clades: 51.27% (<italic>n</italic> = 181) of the genomes belonged to Nextstrain clade 20E(EU1), 29.18% (<italic>n</italic> = 103) belonged to clade 20B, and 9.63% (<italic>n</italic> = 34) belonged to clade 20A.<li> [11](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>          </person-group>          <article-title>Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstrain.org</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-02-14">(accessed on 14 February 2021)</date-in-citation>        ) [11](          <person-group person-group-type="author">            <name>              <surname>Hodcroft</surname>              <given-names>E.B.</given-names>            </name>            <name>              <surname>Hadfield</surname>              <given-names>J.</given-names>            </name>            <name>              <surname>Neher</surname>              <given-names>R.A.</given-names>            </name>            <name>              <surname>Bedford</surname>              <given-names>T.</given-names>            </name>          </person-group>          <article-title>Year-letter Genetic Clade Naming for SARS-CoV-2 on Nextstrain.org</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming">https://nextstrain.org//blog/2020-06-02-SARSCoV2-clade-naming</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-02-14">(accessed on 14 February 2021)</date-in-citation>        ) [12](          <article-title>Updated Nextstain SARS-CoV-2 Clade Naming Strategy-SARS-CoV-2 Coronavirus/nCoV-2019 Genomic Epidemiology</article-title>          <comment>Available online: <ext-link ext-link-type="uri" xlink:href="https://virological.org/t/updated-nextstain-sars-cov-2-clade-naming-strategy/581">https://virological.org/t/updated-nextstain-sars-cov-2-clade-naming-strategy/581</ext-link></comment>          <date-in-citation content-type="access-date" iso-8601-date="2021-02-15">(accessed on 15 February 2021)</date-in-citation>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33882219">33882219</a> <b>(1)</b> Vaccine Breakthrough Infections with SARS-CoV-2 Variants<ul><li> The SARS-CoV-2 genome was assembled with MEGAHIT with default parameters, and the longest sequence (30,005 nucleotides) was analyzed with <b>Nextclade</b> software (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) in order to assign the clade and call mutations.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33919314">33919314</a> <b>(1)</b> A Potential SARS-CoV-2 Variant of Interest (VOI) Harboring Mutation E484K in the Spike Protein Was Identified within Lineage B.1.1.33 Circulating in Brazil<ul><li> The mutational profile was investigated using the <b>Nextclade</b> tool (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>, accessed on 1 March 2021) and temporal signal was assessed by the regression analysis of the root-to-tip genetic distance against sampling dates using the program Tempest [<xref rid="B12-viruses-13-00724" ref-type="bibr"><u>12</u></xref>].<li> [12](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27774300">27774300</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33857576">33857576</a> <b>(1)</b> Monitoring the rise of the SARS-CoV-2 lineage B.1.1.7 in Tenerife (Spain) since mid-December 2020<ul><li> The PANGOLIN software suite v2.1.7 (<ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="interref0002">https://github.com/cov-lineages/pangolin</ext-link>) was then used for lineage and clade classification following the nomenclature of Rambaut et&#xA0;al.<xref rid="bib0009" ref-type="bibr"><sup>9</sup></xref> <b>Nextclade</b> v.0.12.0 was used for variant calling and functional predictions.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33857576">33857576</a> <b>(1)</b> Monitoring the rise of the SARS-CoV-2 lineage B.1.1.7 in Tenerife (Spain) since mid-December 2020<ul><li> Each sample&#x2019;s consensus sequence was visually inspected using Geneious Prime software (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.geneious.com" ext-link-type="uri">https://www.geneious.com</ext-link>) and/or Nextstrain&#x2019;s <b>Nextclade</b> online tool (<ext-link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33853970">33853970</a> <b>(1)</b> Genomics and epidemiology of the P.1 SARS-CoV-2 lineage in Manaus, Brazil<ul><li> Viral lineages were classified by using the Pangolin (<xref rid="R26" ref-type="bibr"><italic>26</italic></xref>) software tool (<ext-link ext-link-type="uri" xlink:href="http://pangolin.cog-uk.io">http://pangolin.cog-uk.io</ext-link>), <b>nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>), and standard phylogenetic analysis using complete reference genomes.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33921216">33921216</a> <b>(3)</b> SARS-CoV-2 Reinfection in a Healthcare Worker Despite the Presence of Detectable Neutralizing Antibodies<ul><li> Pangolin lineage and nextstrain clade assignment of consensus sequences were performed using the pangolin (<uri xlink:href="https://github.com/cov-lineages/pangolin">https://github.com/cov-lineages/pangolin</uri>, accessed on 7 March 2021) and <b>nextclade</b> (<uri xlink:href="https://github.com/nextstrain/<b>nextclade</b>">https://github.com/nextstrain/<b>nextclade</b></uri>, accessed on 7 March 2021) packages.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33835177">33835177</a> <b>(3)</b> Genome Analysis for Sequence Variants in SARS-CoV-2 Among Asymptomatic Individuals in a Long-term Care Facility<ul><li> Whole genome sequencing (WGS) was performed using the MinION platform.<sup><xref rid="zld210061r4" ref-type="bibr"><u>4</u></xref></sup> After uploading consensus genomes to the GISAID database, <b>Nextclade</b> version 0.9.0 was used for quality checks and data analysis (<xref rid="zld210061t1" ref-type="table"><u>Table</u></xref>).<li><b>NextClade</b><xref ref-type="table-fn" rid="zld210061t1n4"><sup>d</sup></xref></th>            </tr>          </thead>          <tbody>            <tr>              <td valign="top" align="left" scope="row" rowspan="1" colspan="1">NE-001-17.<li>All 7 SARS-CoV-2 strains were grouped in 20C by <b>NextClade</b>.<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33060796">33060796</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33857895">33857895</a> <b>(16)</b> Rapid screening for SARS-CoV-2 variants of concern in clinical and environmental samples using nested RT-PCR assays targeting key mutations of the spike protein<ul><li> Preliminary studies on sequences available in GISAID and <b>Nextclade</b> were performed before application on field samples.<li> This was done using GISAID (<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/" id="interref0005">https://www.gisaid.org/</ext-link>) and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/" id="interref0006">https://nextstrain.org/</ext-link>).<li>To further explore the discriminatory potential of our newly-designed long nested PCR for the different SARS-CoV-2 variants, we compared the ~1500&#xA0;bp fragments to their corresponding full-length sequences in terms of how they would be classified by <b>Nextclade</b> web tool.<li> Both files were then analyzed using the <b>Nextclade</b> web tool (nextstrain.org) in order to determine whether our nested assay (PCR ID 980) would yield similar results to those given by a full sequence analysis.<li> <b>Nextclade</b> web tool assigned all the variants correctly on the sole basis of the genome region amplified by the PCR ID 980.<li> The only significant exception was the 20I/501Y.V1 variant, which was incorrectly assigned to clade 20B This misclassification, however, was due to a limitation of the <b>Nextclade</b> web tool, which, for the purposes of phylogenetic placement and clade assignment, uses only mutations and no gaps (<ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>/issues/311#issuecomment-765904855" id="interref0007">https://github.com/nextstrain/<b>nextclade</b>/issues/311#issuecomment-765904855</ext-link>), and was therefore unable to take into consideration the deletions HV69&#x2013;70del and Y144del for a correct assignment of our partial sequences of the S gene.<li> Similarly, the two trees built with <b>Nextclade</b>, one based on full-length genomes and the other based on ~1500&#xA0;bp fragments, both correctly assigned all of the variants to the branches corresponding to clades 19A, 20H/501Y.V2, 20&#xA0;J/501Y.V3 and 20E.EU1.<li><b>Nextclade</b> web tool phylogenetic trees for representative strains of the different variants using the full genome or the Spike partial gene fragment amplified with PCR ID 980 - nextstrain.org with ad hoc modifications.<li> The <b>Nextclade</b> web tool v.012.0 (<xref rid="bib0017" ref-type="bibr"><u>Hadfield&#xA0;et&#xA0;al., 2018</u></xref>) was used to compare study sequences to SARS-CoV-2 reference sequences, assign them to clades, and determine their position within the SARS-CoV-2 phylogenetic tree.<li> In the <b>Nextclade</b> tree constructed using the ~1500&#xA0;bps sequence, this isolate fell at the base of the branch corresponding to clade 20I/501Y.V1, as predicted by the preliminary study we conducted with the UK variant in Nextrain prior to primer design (<xref rid="fig0002" ref-type="fig"><u>Fig.&#xA0;2</u></xref>).<li><b>Nextclade</b></th></tr></thead><tbody><tr><td valign="top">swab_1.<li> The UK variant, however, could only be assigned to its correct lineage using the GISAID BLAST tool, given the aforementioned limitation of the <b>Nextclade</b> tool.<li> Indeed, they showed 100% nt identity with different clades/lineages in GISAID, and were all classified as 19A by <b>Nextclade</b> (<xref rid="tbl0003" ref-type="table"><u>Table&#xA0;3</u></xref>).<li>The discriminatory potential of the long PCR assay was initially explored in silico using GISAID and <b>Nextclade</b>.<li> [Hadfield&#xA0;et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33916205">33916205</a> <b>(1)</b> Nosocomial Outbreak of SARS-CoV-2 in a &#x201C;Non-COVID-19&#x201D; Hospital Ward: Virus Genome Sequencing as a Key Tool to Understand Cryptic Transmission<ul><li> Clade and lineage assignments were performed using <b>Nextclade</b> (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri>) and Phylogenetic Assignment of Named Global Outbreak Lineages (Pangolin) (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri>, accessed on 16 December 2020) [<xref rid="B22-viruses-13-00604" ref-type="bibr"><u>22</u></xref>], respectively.<li> [22](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33861950">33861950</a> <b>(4)</b> Emergence and rapid transmission of SARS-CoV-2 B.1.1.7 in the United States<ul><li><hr/></td></tr><tr><td>DRAGEN COVIDSeq Test Pipeline v.1.3.0.28</td><td>Illumina</td><td>N/A</td></tr><tr><td>Pangolin v2.0</td><td><xref rid="bib24" ref-type="bibr"><u>O&#x2019;Toole et&#xA0;al., 2021b</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0025">https://github.com/cov-lineages/pangolin</ext-link></td></tr><tr><td><b>NextClade</b> v0.12.0.<li><hr/></td></tr><tr><td>DRAGEN COVIDSeq Test Pipeline v.1.3.0.28</td><td>Illumina</td><td>N/A</td></tr><tr><td>Pangolin v2.0</td><td><xref rid="bib24" ref-type="bibr"><u>O&#x2019;Toole et&#xA0;al., 2021b</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/cov-lineages/pangolin" id="intref0025">https://github.com/cov-lineages/pangolin</ext-link></td></tr><tr><td><b>NextClade</b> v0.12.0</td><td><xref rid="bib18" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" id="intref0030">https://github.com/nextstrain/<b>nextclade</b></ext-link>.<li>Consensus FASTA for each sample was used as input to <b>NextClade</b> v0.12.0 (<xref rid="bib18" ref-type="bibr"><u>Hadfield et&#xA0;al., 2018</u></xref>) and Pangolin v2.0 (<xref rid="bib24" ref-type="bibr"><u>O&#x2019;Toole et&#xA0;al., 2021b</u></xref>) to label phylogenetic clades and lineages based on (<xref rid="bib28" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020b</u></xref>), respectively.<li> [O&#x2019;Toole et&#xA0;al., 2021b](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.T.</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>E.</given-names>            </name>          </person-group>          <part-title>Phylogenetic Assignment of Named Global Outbreak LINeages</part-title>          <year>2021</year>          <publisher-name>Pangolin</publisher-name>        ) [O&#x2019;Toole et&#xA0;al., 2021b](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.T.</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>E.</given-names>            </name>          </person-group>          <part-title>Phylogenetic Assignment of Named Global Outbreak LINeages</part-title>          <year>2021</year>          <publisher-name>Pangolin</publisher-name>        ) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [Hadfield et&#xA0;al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [O&#x2019;Toole et&#xA0;al., 2021b](          <person-group person-group-type="author">            <name>              <surname>O&#x2019;Toole</surname>              <given-names>&#xC1;.</given-names>            </name>            <name>              <surname>McCrone</surname>              <given-names>J.T.</given-names>            </name>            <name>              <surname>Scher</surname>              <given-names>E.</given-names>            </name>          </person-group>          <part-title>Phylogenetic Assignment of Named Global Outbreak LINeages</part-title>          <year>2021</year>          <publisher-name>Pangolin</publisher-name>        ) [Rambaut et&#xA0;al., 2020b](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33794262">33794262</a> <b>(1)</b> Incidence of delayed asymptomatic COVID-19 recurrences in a 6-month longitudinal study<ul><li> Genome sequencing of SARS-CoV-2 was performed on nasopharyngeal samples following ARTIC amplicon sequencing protocol for MinIon version V3- Phylogenetic analysis was done using webserver Nextstrain (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/" id="interref0001">https://nextstrain.org/</ext-link>), with the SARS-CoV-2 database <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0002">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33855294">33855294</a> <b>(1)</b> Possible Involvement of Central Nervous System in COVID-19 and Sequence Variability of SARS-CoV-2 Revealed in Autopsy Tissue Samples: A Case Report<ul><li> The phylogenetic analysis (<xref ref-type="fig" rid="fig2-2632010X211006096"><u>Figure 2</u></xref>) showed that all 4 virus samples belonged to the A2a clade (corresponding to clade 20B by the classification system implemented in <b>Nextclade</b>: <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/results">https://clades.nextstrain.org/results</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33785459">33785459</a> <b>(1)</b> Emergence and outcomes of the SARS-CoV-2 &#x2018;Marseille-4&#x2019; variant<ul><li>The 1038 SARS-CoV-2 whole genome sequences obtained in our laboratory were analysed using the <b>Nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intr0040">https://clades.nextstrain.org/</ext-link>) (<xref rid="bib0085" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and an in-house script written in Python.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33791722">33791722</a> <b>(1)</b> SARS-CoV-2 genome sequencing from COVID-19 in Ecuadorian patients: a whole country analysis<ul><li> The online tool <b>NextClade</b> (<xref rid="R14" ref-type="bibr"><u>14</u></xref>) was used to determine the genomes clades.<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33766711">33766711</a> <b>(3)</b> SARS-CoV-2 variants lacking ORF8 occurred in farmed mink and pangolin<ul><li> vison</italic>) with ORF8 nonsense mutations were classified into clades using the <b>Nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="ir010">https://clades.nextstrain.org/</ext-link>) and the Phylogenetic Assignment of Named Global Outbreak LINeages (PANGOLIN) tool running in GISAID (<xref rid="b0165" ref-type="bibr"><u>Rambaut et al., 2020a</u></xref>).<li>SARS-CoV-2 and related viruses with variants lacking ORF8 identified in wild or domestic species.</p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th>Sequence</th><th>GISAID Accession ID</th><th>Collection date</th><th>Location</th><th>Clade (<b>Nextclade</b>)</th><th>Lineage (Pangolin)</th><th>ORF8 nonsense mutation</th><th>Spike amino acid substitutions</th></tr></thead><tbody><tr><td>Pangolin (<italic>Manis javanica</italic>).<li> The sequences belong to clade 20A (<b>Nextclade</b>) or lineage B.1 that dominated the European outbreak in March 2020 (<xref rid="b0165" ref-type="bibr"><u>Rambaut et al., 2020a</u></xref>).<li> [Rambaut et al., 2020a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Rambaut et al., 2020a](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33766552">33766552</a> <b>(3)</b> Discovery of a SARS-CoV-2 variant from the P.1 lineage harboring K417T/E484K/N501Y mutations in Kofu, Japan<ul><li> The sequence data were subjected to phylogenetic analysis using <b>Nextclade</b>, identifying five types of clades: 19A (<italic>n</italic>&#x202F;=&#x202F;1), 20A (<italic>n</italic>&#x202F;=&#x202F;3), 20B (<italic>n</italic>&#x202F;=&#x202F;59), 20C (<italic>n</italic>&#x202F;=&#x202F;6), and 20&#x202F;J/501Y.V3 (<italic>n</italic>&#x202F;=&#x202F;1) (<xref rid="fig0001" ref-type="fig"><u>Fig.&#xA0;1</u></xref>).<li><bold>Phylogenetic tree analysis of SARS-CoV-2.</bold> (<bold>A</bold>) Sequencing data were uploaded to <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="interref0001">https://clades.nextstrain.org/</ext-link>) for phylogenetic analysis.<li> <bold>(C)</bold> A total of 119 sequencing data were analyzed on <b>Nextclade</b>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33810168">33810168</a> <b>(1)</b> Multiple Early Introductions of SARS-CoV-2 to Cape Town, South Africa<ul><li><b>Nextclade</b> v0.14.0 (<uri xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</uri>, last accessed 21 March 2021) [<xref rid="B18-viruses-13-00526" ref-type="bibr"><u>18</u></xref>] was used as quality assurance to report potential sequence quality issues, to identify differences between the Tygerberg sequences and the Wuhan-Hu-1 reference sequence, and to identify clades.<li> [18](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33740028">33740028</a> <b>(1)</b> SARS-CoV-2 outbreak in a tri-national urban area is dominated by a B.1 lineage variant linked to a mass gathering event<ul><li> We used <b>Nextclade</b> version 0.3.5 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org</ext-link>) to infer genomic mutations and filtered for sequences that contained C15324T.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33737129">33737129</a> <b>(4)</b> Genomic surveillance of SARS-CoV-2 in the Republic of Congo<ul><li> The genomes were classified into lineages using PANGOLIN and into clades using <b>Nextclade</b>.<li> SARS-CoV-2 genomes were classified into lineages using Phylogenetic Assignment of Named Global Outbreak LINeages (PANGOLIN) (<xref rid="bib0060" ref-type="bibr"><u>Rambaut et al., 2020</u></xref>) and viral clades were assigned by <b>Nextclade</b> Beta (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intr0010">https://clades.nextstrain.org/</ext-link>) and Nextstrain (<xref rid="bib0025" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>).<li> The <b>Nextclade</b> analysis revealed that nine of the 11 SARS-CoV-2 genomes belonged to clade 20A, with the rest being part of clade 20C (<xref rid="fig0005" ref-type="fig"><u>Figure 1</u></xref>).<li>Lineage/<b>Nextclade</b></th><th align="left">Mutations observed</th></tr></thead><tbody><tr><td align="left">1.<li> [Rambaut et al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33688657">33688657</a> <b>(1)</b> SARS-CoV-2 501Y.V2 (B.1.351) elicits cross-reactive neutralizing antibodies<ul><li> via the Phylogenetic Assignment of named Global Outbreak LINeages (PANGOLIN) software suite (<ext-link ext-link-type="uri" xlink:href="https://github.com/hCoV-2019/pangolin">https://github.com/hCoV-2019/pangolin</ext-link>) (<xref rid="R27" ref-type="bibr"><u>Rambaut et al., 2020a</u></xref>), and <b>Nextclade</b> (<xref rid="R11" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>), respectively.<li> [Rambaut et al., 2020a](<name><surname>RAMBAUT</surname><given-names>A.</given-names></name>, <name><surname>HOLMES</surname><given-names>E. C.</given-names></name>, <name><surname>O&#x2019;TOOLE</surname><given-names>&#xC1;.</given-names></name>, <name><surname>HILL</surname><given-names>V.</given-names></name>, <name><surname>MCCRONE</surname><given-names>J. T.</given-names></name>, <name><surname>RUIS</surname><given-names>C.</given-names></name>, <name><surname>DU PLESSIS</surname><given-names>L.</given-names></name> &amp; <name><surname>PYBUS</surname><given-names>O. G.</given-names></name><year>2020a</year>. <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>. <source>Nature microbiology</source>, <volume>5</volume>, <fpage>1403</fpage>&#x2013;<lpage>1407</lpage>.) [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33655280">33655280</a> <b>(1)</b> SARS-CoV-2 genomic surveillance identifies naturally occurring truncations of ORF7a that limit immune suppression.<ul><li> Then, the <b>NextClade</b> online tool was used to determine the quality of 73 SARS-CoV-2 genomes (% genome covered) sequenced from Bozeman patients.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33817389">33817389</a> <b>(1)</b> Mink SARS-CoV-2 Infection in Poland &#x2013; Short Communication<ul><li> 14-02-2021, <ext-link ext-link-type="uri" xlink:href="http://https://www.epicov.org">https://www.epicov.org</ext-link>) and <b>NextClade</b> (ver.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33842111">33842111</a> <b>(1)</b> A Case of Moderately Severe COVID-19 in a Healthcare Worker in Russia: Virus Isolation and Full Genome Sequencing<ul><li>The phylogram was provided by the <b>Nextclade</b> tool of the Nextstrain web-service (<ext-link ext-link-type="uri" xlink:href="clades.nextstrain.org">clades.nextstrain.org</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33688689">33688689</a> <b>(2)</b> Estimation of secondary household attack rates for emergent SARS-CoV-2 variants detected by genomic surveillance at a community-based testing site in San Francisco<ul><li><bold>Supplementary Figure 1: <b>Nextclade</b> Phylogeny.<li><bold>Supplementary Figure 1: <b>Nextclade</b> Phylogeny.</bold> Radial tree phylogeny as visualized via <b>Nextclade</b> [<xref rid="R26" ref-type="bibr"><u>26</u></xref>] using full length genomes (n=614) from the Jan 10th - 29th sampling period and the San Francisco public reference tree (downloaded Feb 17, 2021).<li> [26](<name><surname>Hadfield</surname><given-names>J</given-names></name>, <name><surname>Megill</surname><given-names>C</given-names></name>, <name><surname>Bell</surname><given-names>SM</given-names></name>, <etal/><article-title>Nextstrain: real-time tracking of pathogen evolution</article-title>. <source>Bioinforma Oxf Engl</source><year>2018</year>; <volume>34</volume>:<fpage>4121</fpage>&#x2013;<lpage>4123</lpage>.) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33676232">33676232</a> <b>(2)</b> SARS-CoV-2 variants combining spike mutations and the absence of ORF8 may be more transmissible and require close monitoring<ul><li>All sequences have been classified into clades using the <b>Nextclade</b> tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0015">https://clades.nextstrain.org/</ext-link>).<li> According to the <b>Nextclade</b> tool, all sequences in this cluster had 19 or more mutations in relation to the root of the tree build with reference sequences.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33671631">33671631</a> <b>(1)</b> Genomic Epidemiology of SARS-CoV-2 in Madrid, Spain, during the First Wave of the Pandemic: Fast Spread and Early Dominance by D614G Variants<ul><li>Lineages were assigned following the &#x2019;PANGO scheme [<xref rid="B16-microorganisms-09-00454" ref-type="bibr"><u>16</u></xref>] and using the Pangolin 2.0 web app (<uri xlink:href="https://pangolin.cog-uk.io/">https://pangolin.cog-uk.io/</uri> (accessed on 3 February 2021)), and the NextStrain year-letter scheme in the <b>NextClade</b> web page (<uri xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</uri> (accessed on 3 February 2021)).<li> [16](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33580111">33580111</a> <b>(1)</b> SARS-CoV-2 genomic surveillance in Rond&#xF4;nia, Brazilian Western Amazon<ul><li> <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) and CoVsurver reports&#xA0;(<ext-link ext-link-type="uri" xlink:href="https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/">https://www.gisaid.org/epiflu-applications/covsurver-mutations-app/</ext-link>) are shown in the Supplementary Material (table <xref rid="MOESM3" ref-type="media"><u>S2</u></xref>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33564780">33564780</a> <b>(1)</b> Genomic epidemiology identifies emergence and rapid transmission of SARS-CoV-2 B.1.1.7 in the United States<ul><li>Consensus FASTA for each sample was used as input to <b>NextClade</b> v0.12.0 (<xref rid="R18" ref-type="bibr"><u>Hadfield et al., 2018</u></xref>) and Pangolin v2.0 (<xref rid="R23" ref-type="bibr"><u>O&#x2019;Toole et al., 2021</u></xref>) to label phylogenetic clades and lineages based on (<xref rid="R27" ref-type="bibr"><u>Rambaut et al., 2020b</u></xref>), respectively.<li> [Hadfield et al., 2018](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) [O&#x2019;Toole et al., 2021](<name><surname>O&#x2019;Toole</surname><given-names>&#xC1;.</given-names></name>, <name><surname>McCrone</surname><given-names>J.T.</given-names></name>, and <name><surname>Scher</surname><given-names>E.</given-names></name> (<year>2021</year>). <source>Phylogenetic Assignment of Named Global Outbreak LINeages (Pangolin)</source>.) [Rambaut et al., 2020b](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33545711">33545711</a> <b>(1)</b> SARS-CoV-2 evolution during treatment of chronic infection<ul><li> Major SARS-CoV-2 clade memberships were assigned to all sequences using both the <b>Nextclade</b> server v0.9 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) and Phylogenetic Assignment Of Named Global Outbreak Lineages (pangolin)<sup><xref rid="R32" ref-type="bibr"><u>32</u></xref></sup>.<li> [32](          <person-group person-group-type="author">            <name>              <surname>Rambaut</surname>              <given-names>A</given-names>            </name>            <etal/>          </person-group>          <article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>          <source>Nature Microbiology</source>          <year>2020</year>          <volume>5</volume>          <fpage>1403</fpage>          <lpage>1407</lpage>          <pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id>        ) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33535105">33535105</a> <b>(2)</b> Introduction into the Marseille geographical area of a mild SARS-CoV-2 variant originating from sub-Saharan Africa: An investigational study<ul><li>Sequences from complete genomes were analyzed using the <b>Nextclade</b> web-tool (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0025">https://clades.nextstrain.org/</ext-link>) <xref rid="bib7" ref-type="bibr"><u>[7]</u></xref>.<li> Phylogenetic trees were reconstructed by using <b>Nextclade</b> and visualized with iTOL (<ext-link ext-link-type="uri" xlink:href="https://itol.embl.de/" id="intref0035">https://itol.embl.de/</ext-link>).<li> [[7]](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33509997">33509997</a> <b>(1)</b> Genome Sequence of a SARS-CoV-2 VUI 202012/01 Strain Identified from a Patient Returning from London, England, to the Apulia Region of Italy<ul><li>The phylogenetic analysis was performed by using the <b>Nextclade</b> sequence analysis Web app (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org">https://clades.nextstrain.org</ext-link>) (<xref rid="B4" ref-type="bibr"><u>4</u></xref>).<li> [4](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33509990">33509990</a> <b>(1)</b> Genome Sequencing of a Novel Coronavirus SARS-CoV-2 Isolate from Iraq<ul><li>The <b>Nextclade</b> v0.8.1 Web tool (<xref rid="B10" ref-type="bibr"><u>10</u></xref>) was employed to generate a phylogenetic tree and a clade assignment.<li> [10](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a> <b>(1)</b> Genome Sequencing of a Novel Coronavirus SARS-CoV-2 Isolate from Iraq<ul><li> According to the <b>NextClade</b> tool from Nextstrain (<xref rid="B8" ref-type="bibr"><u>8</u></xref>), all variants were classified in the 20B clade.<li> [8](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33398302">33398302</a> <b>(1)</b> Neutralising antibodies in Spike mediated SARS-CoV-2 adaptation<ul><li> Major SARS-CoV-2 clade memberships were assigned to all sequences using both the <b>Nextclade</b> server v0.9 (<ext-link xlink:href="https://clades.nextstrain.org/" ext-link-type="uri">https://clades.nextstrain.org/</ext-link>) and Phylogenetic Assignment Of Named Global Outbreak Lineages (pangolin)<sup><xref rid="R35" ref-type="bibr"><u>35</u></xref></sup>.<li> [35](<name><surname>Rambaut</surname><given-names>A.</given-names></name><etal/><article-title>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</article-title>. <source>Nature Microbiology</source><volume>5</volume>, <fpage>1403</fpage>&#x2013;<lpage>1407</lpage>, doi:<pub-id pub-id-type="doi">10.1038/s41564-020-0770-5</pub-id> (<year>2020</year>).) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33325750">33325750</a> <b>(4)</b> Genomic analysis of SARS-CoV-2 reveals local viral evolution in Ghana<ul><li>A coverage map was generated by comparing the Ghanaian SARS-CoV-2 genomes and the reference genome (Wuhan-Hu-1/2019) using <b>Nextclade</b> (version 0.7.5).<li>A coverage map was generated by comparing the Ghanaian SARS-CoV-2 genomes and the reference genome (Wuhan-Hu-1/2019) using <b>Nextclade</b> (version 0.7.5).<sup><xref rid="bibr24-1535370220975351" ref-type="bibr"><u>24</u></xref></sup> <b>Nextclade</b> is a web-based tool which performs banded Smith&#x2013;Waterman alignment with an affine gap-penalty.<li> We used <b>Nextclade</b>&#x2014;a webtool that identifies the differences between sequences such as mutations, Ns, and gaps.<li> <b>Nextclade</b> uses an algorithm which estimates the alignment of each genome to the reference genome using short 21-mers along the sequence thereby performing a banded Smith&#x2013;Waterman alignment with an affine gap-penalty.<li> [24](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790939">29790939</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32909002">32909002</a> <b>(1)</b> Limited intestinal inflammation despite diarrhea, fecal viral RNA and SARS-CoV-2-specific IgA in patients with acute COVID-19<ul><li> The mutation profiles were analyzed with the tool <b>NextClade</b> v0.3.6 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33131453">33131453</a> <b>(2)</b> Massive dissemination of a SARS-CoV-2 Spike Y839 variant in Portugal<ul><li> and Portugal phylogeny using Nextstrain (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov">https://nextstrain.org/ncov</ext-link>) and <b>Nextclade</b> (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).<li> phylogeny (using Nextstrain <ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov">https://nextstrain.org/ncov</ext-link> and <b>Nextclade</b> <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) pointed out that the D839Y amino acid change likely emerged independently in two other instances.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34095334">34095334</a> <b>(1)</b> High Prevalence of SARS-CoV-2 Genetic Variation and D614G Mutation in Pediatric Patients With COVID-19<ul><li> Phylogenetic clade analysis was performed using <b>Nextclade</b> (version 0.3.7).</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33176093">33176093</a> <b>(1)</b> SARS-CoV-2 Transmission among Marine Recruits during Quarantine<ul><li> Transmission and outbreak events were identified on the basis of clustering of the SARS-CoV-2 genomes obtained from study participants within the Nextstrain phylogenetic tree, visualized with TreeTime.<sup><xref rid="r14" ref-type="bibr"><u>14</u></xref></sup> A comparative analysis of mutation profiles relative to the SARS-CoV-2 Wuhan reference genome was performed with the use of <b>Nextclade</b> software, version 0.3.6 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>).<li> [14](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29340210">29340210</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33248470">33248470</a> <b>(5)</b> Case Study: Prolonged Infectious SARS-CoV-2 Shedding from an Asymptomatic Immunocompromised Individual with Cancer<ul><li> The quality of the sequences was determined by <b>Nextclade</b> v.0.7.5 (<ext-link ext-link-type="uri" xlink:href="https://nextstrain.org/ncov/global" id="interref0015">https://nextstrain.org/ncov/global</ext-link>), and 1,789 sequences on April 20, 385 sequences between April 20 and May 11, 268 sequences between May 11 and May 26, and 709 sequences between May 26 and June 15 were kept for further phylogenetic analysis.<li><hr/></td></tr><tr><td>MAFFT align</td><td>(<xref rid="bib31" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>, <xref rid="bib32" ref-type="bibr"><u>Katoh et&#xA0;al., 2002</u></xref>)</td><td>Geneious Prime 2020.1.2 plugin</td></tr><tr><td>Multalin sequence alignment</td><td>(<xref rid="bib13" ref-type="bibr"><u>Corpet, 1988</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://multalin.toulouse.inra.fr/multalin/" id="intref0075">http://multalin.toulouse.inra.fr/multalin/</ext-link></td></tr><tr><td>ESPript 3.0</td><td>(<xref rid="bib57" ref-type="bibr"><u>Robert and Gouet, 2014</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://espript.ibcp.fr/ESPript/ESPript/" id="intref0080">http://espript.ibcp.fr/ESPript/ESPript/</ext-link></td></tr><tr><td>Geneious Prime 2020.1.2</td><td>Geneious</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com" id="intref0085">https://www.geneious.com</ext-link></td></tr><tr><td>PhyML 3.320180621</td><td><xref rid="bib18" ref-type="bibr"><u>Guindon et&#xA0;al., 2010</u></xref></td><td>Geneious Prime 2020.1.2 plugin</td></tr><tr><td>FigTree v1.4.4</td><td><ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/figtree/" id="intref0090">http://tree.bio.ed.ac.uk/software/figtree/</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/rambaut/figtree/" id="intref0095">https://github.com/rambaut/figtree/</ext-link></td></tr><tr><td>Pangolin COVID-19 Lineage Assigner</td><td><xref rid="bib56" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="intref0100">https://pangolin.cog-uk.io</ext-link></td></tr><tr><td>Pymol Molecular Graphics System version 2.0.1</td><td>Schr&#xF6;dinger</td><td><ext-link ext-link-type="uri" xlink:href="https://www.schrodinger.com/pymol" id="intref0105">https://www.schrodinger.com/pymol</ext-link></td></tr><tr><td>Prism 8.2.0</td><td>GraphPad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.graphpad.com:443/" id="intref0110">https://www.graphpad.com:443/</ext-link></td></tr><tr><td><b>NextClade</b> v0.7.5.<li><hr/></td></tr><tr><td>MAFFT align</td><td>(<xref rid="bib31" ref-type="bibr"><u>Katoh and Standley, 2013</u></xref>, <xref rid="bib32" ref-type="bibr"><u>Katoh et&#xA0;al., 2002</u></xref>)</td><td>Geneious Prime 2020.1.2 plugin</td></tr><tr><td>Multalin sequence alignment</td><td>(<xref rid="bib13" ref-type="bibr"><u>Corpet, 1988</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://multalin.toulouse.inra.fr/multalin/" id="intref0075">http://multalin.toulouse.inra.fr/multalin/</ext-link></td></tr><tr><td>ESPript 3.0</td><td>(<xref rid="bib57" ref-type="bibr"><u>Robert and Gouet, 2014</u></xref>)</td><td><ext-link ext-link-type="uri" xlink:href="http://espript.ibcp.fr/ESPript/ESPript/" id="intref0080">http://espript.ibcp.fr/ESPript/ESPript/</ext-link></td></tr><tr><td>Geneious Prime 2020.1.2</td><td>Geneious</td><td><ext-link ext-link-type="uri" xlink:href="https://www.geneious.com" id="intref0085">https://www.geneious.com</ext-link></td></tr><tr><td>PhyML 3.320180621</td><td><xref rid="bib18" ref-type="bibr"><u>Guindon et&#xA0;al., 2010</u></xref></td><td>Geneious Prime 2020.1.2 plugin</td></tr><tr><td>FigTree v1.4.4</td><td><ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/figtree/" id="intref0090">http://tree.bio.ed.ac.uk/software/figtree/</ext-link></td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/rambaut/figtree/" id="intref0095">https://github.com/rambaut/figtree/</ext-link></td></tr><tr><td>Pangolin COVID-19 Lineage Assigner</td><td><xref rid="bib56" ref-type="bibr"><u>Rambaut et&#xA0;al., 2020</u></xref></td><td><ext-link ext-link-type="uri" xlink:href="https://pangolin.cog-uk.io" id="intref0100">https://pangolin.cog-uk.io</ext-link></td></tr><tr><td>Pymol Molecular Graphics System version 2.0.1</td><td>Schr&#xF6;dinger</td><td><ext-link ext-link-type="uri" xlink:href="https://www.schrodinger.com/pymol" id="intref0105">https://www.schrodinger.com/pymol</ext-link></td></tr><tr><td>Prism 8.2.0</td><td>GraphPad</td><td><ext-link ext-link-type="uri" xlink:href="https://www.graphpad.com:443/" id="intref0110">https://www.graphpad.com:443/</ext-link></td></tr><tr><td><b>NextClade</b> v0.7.5</td><td><ext-link ext-link-type="uri" xlink:href="https://github.com/nextstrain/<b>nextclade</b>" id="intref0115">https://github.com/nextstrain/<b>nextclade</b></ext-link>.<li> The sequences were analyzed by the <b>Nextclade</b> server v0.7.5 (<ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/" id="intref0230">https://clades.nextstrain.org/</ext-link>) for quality and sequences that were not of sufficient quality were discarded.<li> [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Katoh et&#xA0;al., 2002](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [Corpet, 1988](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=2849754">2849754</a>) [Robert and Gouet, 2014](          <person-group person-group-type="author">            <name>              <surname>Robert</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Gouet</surname>              <given-names>P.</given-names>            </name>          </person-group>          <article-title>Deciphering key features in protein structures with the new ENDscript server</article-title>          <source>Nucleic Acids Res.</source>          <volume>42</volume>          <year>2014</year>          <comment>W320-4</comment>        ) [Guindon et&#xA0;al., 2010](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20525638">20525638</a>) [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) [Katoh and Standley, 2013](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23329690">23329690</a>) [Katoh et&#xA0;al., 2002](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12136088">12136088</a>) [Corpet, 1988](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=2849754">2849754</a>) [Robert and Gouet, 2014](          <person-group person-group-type="author">            <name>              <surname>Robert</surname>              <given-names>X.</given-names>            </name>            <name>              <surname>Gouet</surname>              <given-names>P.</given-names>            </name>          </person-group>          <article-title>Deciphering key features in protein structures with the new ENDscript server</article-title>          <source>Nucleic Acids Res.</source>          <volume>42</volume>          <year>2014</year>          <comment>W320-4</comment>        ) [Guindon et&#xA0;al., 2010](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20525638">20525638</a>) [Rambaut et&#xA0;al., 2020](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32669681">32669681</a>) </ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33058797">33058797</a> <b>(1)</b> Genomic evidence for reinfection with SARS-CoV-2: a case study<ul><li> uncalled bases (Ns) from specimen A and 98 Ns from specimen B, genomic sequences were aligned and related using NGPhylogeny.fr PhyML+SMS.<xref rid="bib16" ref-type="bibr"><sup>16</sup></xref> Sequences were then first-aligned using MAFFT with automatic flavour selection.<xref rid="bib17" ref-type="bibr"><sup>17</sup></xref> Informative regions were selected using Block Mapping and Gathering with Entropy, a sliding window size of 3, and maximum entropy of 0&#xB7;5.<xref rid="bib18" ref-type="bibr"><sup>18</sup></xref> Unrooted trees were constructed by PhyML with Smart Model Selection, the Akaike information criterion, and Subtree Pruning and Regrafting.<xref rid="bib19" ref-type="bibr"><sup>19</sup></xref> Newick trees were visualised using Interactive Tree Of Live version 4 and rooted at the Wuhan reference strain.<xref rid="bib20" ref-type="bibr"><sup>20</sup></xref> Major SARS-CoV-2 clade memberships were predicted using <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org" id="interrefs10"><b>Nextclade</b></ext-link>.</ul></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33026453">33026453</a> <b>(2)</b> Analysis of Genomic Characteristics and Transmission Routes of Patients With Confirmed SARS-CoV-2 in Southern California During the Early Stage of the US COVID-19 Pandemic<ul><li> These genomes passed quality control assessment by <b>Nextclade</b><sup><xref rid="zoi200793r28" ref-type="bibr"><u>28</u></xref></sup> and were retained for downstream phylogenetic analysis.<li> Tree visualizations were performed with FigTree version 1.4.4<sup><xref rid="zoi200793r35" ref-type="bibr"><u>35</u></xref></sup> and iTOL version 5.6.1.<sup><xref rid="zoi200793r36" ref-type="bibr"><u>36</u></xref></sup> Subsampling with global background was performed by <b>NextClade</b> with CSMC samples.<li> [28](<person-group><collab>Nextstrain</collab></person-group> Nextclade. Accessed September 9, 2020. <ext-link ext-link-type="uri" xlink:href="https://clades.nextstrain.org/">https://clades.nextstrain.org/</ext-link>) [35](<person-group><name name-style="western"><surname>Rambaut</surname><given-names>A</given-names></name></person-group> FigTree. Accessed September 9, 2020. <ext-link ext-link-type="uri" xlink:href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</ext-link>) [36](<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16513982">16513982</a>) </ul></p>
</body>
