---
layout: post
title: "RISmed: auto fetching PubMed Content"
description: ""
category: 
tags: []
---
{% include JB/setup %}

**The What**: 

RISmed is an R package for fetching data from PubMed. 

**The So-What**:

Scripting methods for fetching pubmed data can help a researcher quickly screen and identify important publications to read, resulting in faster publication cycles, and less wasted time.

**The Now-What**:

A very basic tutorial of RISmed is included below:

Install and load the RISmed package.

```
install.packages("RISmed");
library(RISmed);
```

Then you can iterate through a list of keywords saving information on the papers and the coauthor network.

```
query_term <- "alzheimer"
start <- 1996
end <- 2016
res <- EUtilsSummary(query_term, type='esearch', db="pubmed", datetype='pdat',
                     mindate=start, maxdate=end, retmax=10000)
QueryCount(res) # get number of results

```

However to turn into a more human readable data.table, although this takes some time (minutes).

```
big <- EUtilsGet(res)
slotNames(big) # lists all possible fetchable annotation.
 [1] "Query"                "PMID"                 "YearReceived"        
 [4] "MonthReceived"        "DayReceived"          "HourReceived"        
 [7] "MinuteReceived"       "YearAccepted"         "MonthAccepted"
 ... many more results      

```

Pull out interesting items into a dataframe.

```
data <- data.frame(pmid=PMID(big), year=YearPubmed(big), title=ArticleTitle(big),
                  journal=Title(big), affiliation=Affiliation(big), country=Country(big))

# Export to tsv, save for later
write.table(data, file=paste(query_term,"-papers.tsv", sep=""), sep="\t",
              row.names=FALSE, col.names=TRUE, quote=FALSE)
```
The resulting table is similar to the following:

|pmid|year|title|journal|affiliation|country|
|----|----|-----|-------|-----------|-------|
|28003598 | 2016 | An iTRAQ-based quantit... | Japanese journal of... | Guangzhou ... | Japan|

```
28000890        2016    Combined analysis of gene expression, miRNA expression and DNA methylation profiles of osteosarcoma.    Oncology reports        Department of Joint Surgery, The Affiliated Hospital of Qingdao University, Qingdao, Shandong 266003, P.R. China.       Greece

The total pipeline can be looped over multiple query terms.

```
terms <- c("alzheimer","parkinson")
for(i in 1:length(terms)){
  query_term <- terms[i]
  start <- 1996
  end <- 2016
  res <- EUtilsSummary(query_term,type='esearch',db='pubmed',datetype='pdat',
                       mindate=start,maxdate=end,retmax=10000)
  
  big <- EUtilsGet(res) # takes time
  
  # slotNames(big) # list all names of paper annotations
  data <- data.frame(pmid=PMID(big),year=YearPubmed(big),title=ArticleTitle(big),
                     journal=Title(big),affiliation=Affiliation(big),country=Country(big))
  write.table(data,file=paste(query_term,"-papers.tsv",sep=""),sep="\t",
              row.names=FALSE,col.names=TRUE,quote=FALSE)
  
  coauthors <- data.frame(Author(pmids)[1])
  names(coauthors)<-c("LastName","ForeName","Initials","order")
  coauthors$author<-str_replace_all(paste(coauthors$ForeName,"_",coauthors$LastName,
                                    sep=""), " ","_")
  coauthors$pmid=PMID(pmids)[1]
  coa <- data.frame(pmid=coauthors$pmid,author=coauthors$author,order=coauthors$order)

  ll <- length(PMID(pmids))
  for(j in 2:ll){
    coauthors <- data.frame(Author(pmids)[j])
    names(coauthors)<-c("LastName","ForeName","Initials","order")
    coauthors$author<-str_replace_all(paste(coauthors$ForeName,"_",coauthors$LastName,
                                      sep=""), " ","_")
    coauthors$pmid=PMID(pmids)[j]
    coa<-rbind(coa,data.frame(pmid=coauthors$pmid,author=coauthors$author,
               order=coauthors$order))
  }
  write.table(coa,file=paste(query_term,"-authors.tsv",sep=""),sep="\t",row.names=FALSE,
              col.names=TRUE,quote=FALSE)
}
```

