<!DOCTYPE html>
<html lang="en" dir="ltr">

<head>
  <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1">
  <title>Jennifer Chang</title>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="style.css">
</head>

<body>
  <div class="content">
    <div class="navbar" id="mynav">
      <a href="index.html">Jennifer Chang</a>
      <a href="projects.html" class="active">Projects</a>
      <a href="posts.html">Posts</a>
      <a href="CV.pdf">CV</a>
      <a href="javascript:void(0);" style="font-size:15px;" class="icon" onclick="navFunction()">&#9776;</a>
    </div>

    <div class="article">

      <h1 style="padding:0px;margin:0px;">Projects</h1>

      <hr />

      <h2>2020-present Bioinformatics Workbook: tutorial contributions.</h2>

      <p>
      GitHub Repo: <a href="https://github.com/ISUgenomics/bioinformatics-workbook">ISUgenomics/bioinformatics-workbook</a>
    </br>
        Website: <a href="https://bioinformaticsworkbook.org/#gsc.tab=0">https://bioinformaticsworkbook.org</a>
      </p>

      <p>
        The Bioinformatics Workbook is maintained by the Genome Informatics Facility at Iowa State University. Through collaboration, contributed the following tutorials:
    </p>

      <ul>
        <li><a href="https://bioinformaticsworkbook.org/dataWrangling/R/r-setup.html#gsc.tab=0" target="_blank">Setting up an R and RStudio environment</a></li>
        <li><a href="https://bioinformaticsworkbook.org/dataAnalysis/RNA-Seq/RNA-SeqIntro/wgcna.html#gsc.tab=0" target="_blank">WGCNA: weighted gene correlation network analysis</a></li>
        <li><a href="https://geospatial.101workbook.org/Workshops/R_Image_Processing.html" target="_blank">Image Processing Fundamentals</a></li>
      </ul>

      <hr />

      <h2>2020-present HuffordLab/Maize_WGS_Build: GATK pipeline wrapped in Nextflow</h2>

      <p>
      GitHub Repo: <a href="https://github.com/HuffordLab/Maize_WGS_Build"> HuffordLab/Maize_WGS_Build</a>
      </p>

      <p>
      Whole genome sequences (WGS) of various lines of Maize are becoming more and more available. In order to make this information available on the MaizeGDB website, a reproducible, scaleable method to call variants against a reference was needed. In 2020, via a collaboration, the GATK (Genome Analysis Tool Kit) pipeline was wrapped in Nextflow (groovy-based workflow language). The resulting pipeline is avaiable at the following github repo:<a href="https://github.com/HuffordLab/Maize_WGS_Build"> HuffordLab/Maize_WGS_Build</a>. A fork of the repo is also available at:<a href="https://github.com/isugifNF/GATK"> isugifNF/GATK</a>.
    </p>

<!--
      <p style="text-align:center;">
        <iframe class="responsive_video" width="560" height="315" src="https://www.youtube.com/embed/X6bnS1H-eYg" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      </p>
    -->

<hr />



      <h2>2017-2020 OctoFLU: a pipeline for classifying swine Influenza into the USA clades</h2>

      <p>
      GitHub Repo: <a href="https://github.com/flu-crew/octoFLU">flu-crew/octoFLU</a>
      </p>

      <p>
      Influenza A Viruses are made up of 8 genomic segments (HA, NA, PB2, PB1, PA, NP, M, NS) and classifying them into their respective clades can be a tedius process, especially as clade names tend to change over time. octoFLU provides a one-stop-shop to classify the 8 (octo) swine influenza genomic segments into currently accepted U.S. swine influenza clades. The pipeline is mainly available as a commandline tool and can be run natively or in a Docker/Singularity image. A GUI version of the tool is in development and available at the github repo:<a href="https://github.com/j23414/octoFLU"> j23414/octoFLU</a> although it will probably be replaced with a version on flu-crew.
    </p>

      <p style="text-align:center;">
        <iframe class="responsive_video" width="560" height="315" src="https://www.youtube.com/embed/X6bnS1H-eYg" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      </p>

      <hr />
      <h2>2016-2017 Cavatica: a pipeline for literature mining PubMed and PubMed Central</h2>
      <p>
      GitHub Repo: <a href="https://github.com/incertae-sedis/cavatica">incertae-sedis/cavatica</a>
      <p>

      <p>Cavatica is a set of scripts for pulling PubMed and PubMed Central articles to compare adoption of methods/tools in a research community. The below example is comparing the publications using different network analysis software (IPA, Cytoscape, etc). Green nodes represent authors while black nodes represent papers. The horizontal blue lines, going across network tools, indicate individuals who have published a paper in one software, switching to a different software. Cavatica has several submodule scripts for analyzing these software/method adoption trends.</p>

      <p style="text-align:center;">
        <img class="responsive_small" src="imgs/cavatica.png" width="560" alt="3 Ideas"></img>
      </p>

      <hr />

      <h2>2014-2017 Mango: an environment for combining heterogeneous networks</h2>

      <p>Mango Graph Studio development was motivated by the need for fast heterogeneous graph integration and analysis. The software has since been licensed to Complex Computation, LLC. I conceived and wrote the first line of Mango code in January
        2014 while working toward a Ph.D. in Bioinformatics at Iowa State University.</p>

      <ul>
        <li><a href="http://www.complex.iastate.edu/download/Mango/index.html">Download Mango</a></li>
        <li><a href="https://www.gitbook.com/book/j23414/mango-user-guide/details">Mango User Guide</a></li>
      </ul>

      <p>Mango was accepted as a Computer Demo at the 2016 Plant and Animal Genome (PAG) meeting.</p>
      <p style="text-align:center;">
      <iframe class="responsive_video" height="315" width="560" src="https://www.youtube.com/embed/HTNfK0HIfWw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
    </p>
      <!--<iframe  src="https://www.youtube.com/embed/HTNfK0HIfWw" frameborder="0" allowfullscreen></iframe>
      width="560" height="315" -->
      <hr />

      <h2>2012-2014 Lucy2 v2.20: DNA cleaning and trimming</h2>

      <p>Lucy2 is the GUI version of the Lucy command line program. To learn C++ and wxWidgets, I worked on the GUI update while part of Dr Hui-Hsien Chou's lab at Iowa State University in 2012.</p>
      <ul>
        <li><a href="http://www.complex.iastate.edu/download/Lucy2/index.html">Download Lucy2</a></li>
        <li>Updated the GUI using wxWidgets. Updates included allowing multiple sequence files to be analyzed concurrently, new nucleotide color scheme, new FASTQ reader, and creating a settings dialog box validator.</li>
      </ul>

      <hr/>


      <h2>2011 Visualizing Exon Junction Reads in R</h2>

      <p>During my rotation in <a href="http://dicook.org/" target="_blank">Dr. Di Cook's</a> lab at Iowa State University in 2011, I had just started learning R and github and was documenting this mini-project in a blog.</p>

      <ul>
        <li><a href="https://github.com/j23414/Exon-Junction-Arches">GitHub Site</a></li>
        <li><a href="https://jcrotone.wordpress.com">Blog</a></li>
      </ul>

  </div>
  <p style="font-size:11px; text-align:center; color:#aaa;">&copy; Jennifer Chang 2020 &middot; Last Update: 2020-10-15
  </p>
</body>

<script src="script.js"></script>

</html>
